



<!DOCTYPE html>
<html lang="en">
  
  <head>
    
      <meta charset="utf-8">
      <title>Bokeh Plot</title>
      
      
        
          
        
        
          
        <script type="text/javascript" src="https://cdn.bokeh.org/bokeh/release/bokeh-2.0.1.min.js" integrity="sha384-JpP8FXbgAZLkfur7LiK3j9AGBhHNIvF742meBJrjO2ShJDhCG2I1uVvW+0DUtrmc" crossorigin="anonymous"></script>
        <script type="text/javascript" src="https://cdn.bokeh.org/bokeh/release/bokeh-widgets-2.0.1.min.js" integrity="sha384-xZlADit0Q04ISQEdKg2k3L4W9AwQBAuDs9nJL9fM/WwzL1tEU9VPNezOFX0nLEAz" crossorigin="anonymous"></script>
        <script type="text/javascript" src="https://cdn.bokeh.org/bokeh/release/bokeh-tables-2.0.1.min.js" integrity="sha384-4BuPRZkdMKSnj3zoxiNrQ86XgNw0rYmBOxe7nshquXwwcauupgBF2DHLVG1WuZlV" crossorigin="anonymous"></script>
        <script type="text/javascript">
            Bokeh.set_log_level("info");
        </script>
        
      
      
    
  </head>
  
  
  <body>
    
      
        
          
          
            
              <div class="bk-root" id="f8f52611-6bdc-4254-810d-62f68f22f0ca" data-root-id="26201"></div>
            
          
        
      
      
        <script type="application/json" id="26438">
          {"ffe6f363-40d3-48ab-bde3-138b6ae224d7":{"roots":{"references":[{"attributes":{"args":{"sd":{"id":"26117"},"sp":{"id":"26118"}},"code":"\n    if (sd.selected.indices.length == 0) {\n      return;\n    }\n    var sel = sd.selected.indices[0];\n    var new_selected = [];\n    for (var i = 0; i &lt; sp.data['doc'].length; i++) {\n      if (sp.data['doc'][i] == sd.data['doc'][sel]) {\n        new_selected.push(i);\n      }\n    }\n    if (JSON.stringify(new_selected) != JSON.stringify(sp.selected.indices) &amp;&amp;\n    !(sp.selected.indices.length == 1 &amp;&amp; new_selected.includes(sp.selected.indices[0]))) {\n      sp.selected.indices = new_selected;\n      sp.change.emit();\n    }\n    "},"id":"26190","type":"CustomJS"},{"attributes":{},"id":"26133","type":"BasicTicker"},{"attributes":{},"id":"26129","type":"BasicTicker"},{"attributes":{"children":[{"id":"26152"},{"id":"26119"}]},"id":"26198","type":"Row"},{"attributes":{"data_source":{"id":"26118"},"glyph":{"id":"26179"},"hover_glyph":null,"muted_glyph":null,"nonselection_glyph":{"id":"26180"},"selection_glyph":null,"view":{"id":"26182"}},"id":"26181","type":"GlyphRenderer"},{"attributes":{},"id":"26137","type":"ResetTool"},{"attributes":{},"id":"26122","type":"Range1d"},{"attributes":{"callback":null,"tooltips":[["Title","@title"],["Passage","@text"],["Published","@date"],["Authors","@authors"],["Journal","@journal"],["URL","@url"]]},"id":"26173","type":"HoverTool"},{"attributes":{"axis":{"id":"26132"},"dimension":1,"ticker":null},"id":"26135","type":"Grid"},{"attributes":{},"id":"26168","type":"BasicTicker"},{"attributes":{"children":[{"id":"26195"}]},"id":"26199","type":"Row"},{"attributes":{"formatter":{"id":"26223"},"ticker":{"id":"26168"}},"id":"26167","type":"LinearAxis"},{"attributes":{},"id":"26206","type":"StringFormatter"},{"attributes":{"js_property_callbacks":{"change:indices":[{"id":"26190"}]}},"id":"26189","type":"Selection"},{"attributes":{"fill_alpha":{"value":0.1},"fill_color":{"field":"color"},"line_alpha":{"value":0.1},"line_color":{"field":"color"},"radius":{"field":"rad","units":"data"},"x":{"field":"x"},"y":{"field":"y"}},"id":"26145","type":"Circle"},{"attributes":{"fill_color":{"field":"color"},"line_color":{"field":"color"},"radius":{"field":"rad","units":"data"},"x":{"field":"x"},"y":{"field":"y"}},"id":"26179","type":"Circle"},{"attributes":{"below":[{"id":"26128"}],"center":[{"id":"26131"},{"id":"26135"}],"left":[{"id":"26132"}],"plot_height":340,"plot_width":900,"renderers":[{"id":"26146"}],"title":{"id":"26202"},"toolbar":{"id":"26139"},"toolbar_location":"below","x_range":{"id":"26120"},"x_scale":{"id":"26124"},"y_range":{"id":"26122"},"y_scale":{"id":"26126"}},"id":"26119","subtype":"Figure","type":"Plot"},{"attributes":{"sizing_mode":"stretch_width","text":"&lt;h2&gt;Technology roadmap for diagnostics.&lt;/h2&gt;"},"id":"26193","type":"Div"},{"attributes":{"callback":null},"id":"26136","type":"TapTool"},{"attributes":{},"id":"26217","type":"StringEditor"},{"attributes":{"js_property_callbacks":{"change:indices":[{"id":"26192"}]}},"id":"26191","type":"Selection"},{"attributes":{"children":[{"id":"26193"}]},"id":"26196","type":"Row"},{"attributes":{},"id":"26225","type":"BasicTickFormatter"},{"attributes":{"editor":{"id":"26217"},"field":"scr","formatter":{"id":"26216"},"title":"Score","width":30},"id":"26185","type":"TableColumn"},{"attributes":{},"id":"26120","type":"Range1d"},{"attributes":{"fill_alpha":{"value":0.1},"fill_color":{"field":"color"},"line_alpha":{"value":0.1},"line_color":{"field":"color"},"radius":{"field":"rad","units":"data"},"x":{"field":"x"},"y":{"field":"y"}},"id":"26180","type":"Circle"},{"attributes":{"columns":[{"id":"26150"},{"id":"26151"}],"height":340,"source":{"id":"26117"},"view":{"id":"26153"},"width":900},"id":"26152","type":"DataTable"},{"attributes":{"text":""},"id":"26204","type":"Title"},{"attributes":{},"id":"26213","type":"BasicTickFormatter"},{"attributes":{"children":[{"id":"26194"}]},"id":"26197","type":"Row"},{"attributes":{"editor":{"id":"26207"},"field":"scr","formatter":{"id":"26206"},"title":"Score","width":30},"id":"26150","type":"TableColumn"},{"attributes":{},"id":"26208","type":"StringFormatter"},{"attributes":{"children":[{"id":"26196"},{"id":"26197"},{"id":"26198"},{"id":"26199"},{"id":"26200"}]},"id":"26201","type":"Column"},{"attributes":{"formatter":{"id":"26213"},"ticker":{"id":"26133"}},"id":"26132","type":"LinearAxis"},{"attributes":{"axis":{"id":"26167"},"dimension":1,"ticker":null},"id":"26170","type":"Grid"},{"attributes":{"active_drag":"auto","active_inspect":"auto","active_multi":null,"active_scroll":"auto","active_tap":"auto","tools":[{"id":"26171"},{"id":"26172"},{"id":"26173"}]},"id":"26174","type":"Toolbar"},{"attributes":{},"id":"26124","type":"LinearScale"},{"attributes":{"data":{"authors":["Memish, Ziad A; Perlman, Stanley; Van Kerkhove, Maria D; Zumla, Alimuddin","Qiao, Jie","Nour, Mohamed; Alhajri, Mohd; Farag, Elmoubasher A. B. A.; Al-Romaihi, Hamad E.; Al-Thani, Mohamed; Al-Marri, Salih; Savoia, Elena","Nour, Mohamed; Alhajri, Mohd; Farag, Elmoubasher A. B. A.; Al-Romaihi, Hamad E.; Al-Thani, Mohamed; Al-Marri, Salih; Savoia, Elena","Qiao, Jie","Yadi Zhou; Yuan Hou; Jiayu Shen; Yin Huang; William Martin; Feixiong Cheng","Yang, Yongshi; Peng, Fujun; Wang, Runsheng; Guan, Kai; Jiang, Taijiao; Xu, Guogang; Sun, Jinlyu; Chang, Christopher","Chen, Shih-Cheng; Olsthoorn, Ren\u00e9 C.L.","Bruna GG Pinto; Antonio ER Oliveira; Youvika Singh; Leandro Jimenez; Andre NA Goncalves; Rodrigo LT Ogava; Rachel Creighton; Jean PS Peron; Helder I Nakaya","Mengying Dong; Xiaojun Cao; Mingbiao Liang; Lijuan Li; Huiying Liang; Guangjian Liu","Mengying Dong; Xiaojun Cao; Mingbiao Liang; Lijuan Li; Huiying Liang; Guangjian Liu","Bruna GG Pinto; Antonio ER Oliveira; Youvika Singh; Leandro Jimenez; Andre NA Goncalves; Rodrigo LT Ogava; Rachel Creighton; Jean PS Peron; Helder I Nakaya","Xiong Ding; Kun Yin; Ziyue Li; Changchun Liu","Parkes, Margot W.; Bienen, Leslie; Breilh, Jaime; Hsu, Lee-Nah; McDonald, Marian; Patz, Jonathan A.; Rosenthal, Joshua P.; Sahani, Mazrura; Sleigh, Adrian; Waltner-Toews, David; Yassi, Annalee","Zhou, Yadi; Hou, Yuan; Shen, Jiayu; Huang, Yin; Martin, William; Cheng, Feixiong","Carr, Kristen M; Rosenblatt, Kevin; Petricoin, Emanuel F; Liotta, Lance A","Conor Jenkins; Ben Orsburn","Dong, J.; Ismail, N.; Walker, D.H.","HIETPAS, JACK; MCMULLAN, LAURA K.; HANSON, HOLLY L.; RICE, CHARLES M.; MINDELL, DAVID P.","Martin, R.R.; Tzanetakis, I.E.","Carr, Kristen M; Rosenblatt, Kevin; Petricoin, Emanuel F; Liotta, Lance A","Robson, B.","Clark, David P.; Pazdernik, Nanette J.","Malik, Yashpal Singh; Sircar, Shubhankar; Bhat, Sudipta; Sharun, Khan; Dhama, Kuldeep; Dadar, Maryam; Tiwari, Ruchi; Chaicumpa, Wanpen","Wei, Pengfei; Cai, Zelang; Hua, Jinwen; Yu, Weijia; Chen, Jiajie; Kang, Kang; Qiu, Congling; Ye, Lanlan; Hu, Jiayun; Ji, Kunmei","Wernery, Ulrich; Lau, Susanna K.P.; Woo, Patrick C.Y.","","Yinhua Zhang; Nelson Odiwuor; Jin Xiong; Luo Sun; Raphael Ohuru Nyaruaba; Hongping Wei; Nathan A Tanner","Uttamchandani, Mahesh; Neo, Jia Ling; Ong, Brandon Ngiap Zhung; Moochhala, Shabbir","Barnard, Ross Thomas; Hall, Roy A; Gould, Ernest A","Kenneth N. Hass; Mengdi Bao; Qian He; Myeongkee Park; Peiwu Qin; Ke Du","Roux, Simon; Matthijnssens, Jelle; Dutilh, Bas E.","McCullough, Ann E.; Leslie, Kevin O.","","Lee, Vernon J; Aguilera, Ximena; Heymann, David; Wilder-Smith, Annelies; Lee, Vernon J.; Aguilera, Ximena; Heymann, David L.; Wilder-Smith, Annelies; Bausch, Daniel G.; Briand, Sylvie; Bruschke, Christianne; Carmo, Eduardo H.; Cleghorn, Sean; Dandona, Lalit; Donnelly, Christl; Fall, Ibrahima Soc\u00e9; Halton, Jane; Hatchett, Richard; Hong, Felicia; Horby, Peter; Ihekweazu, Chikwe; Jacobs, Michael; Khan, Kamran; Lin, Yijun; Leung, Gabriel; Low, Constance; McDonald, Bethan F.; Memish, Ziad A.; Morhard, Ryan; Ng, Deborah HL; Nkengasong, John; Pang, Junxiong; Redd, Stephen C.; Tan, Karen; Yeo, Wen Qing","","Uttamchandani, Mahesh; Neo, Jia Ling; Ong, Brandon Ngiap Zhung; Moochhala, Shabbir","Zumla, Alimuddin; Gant, Vanya; Bates, Matthew; Mwaba, Peter; Maeurer, Markus; Memish, Ziad A","","Bai, Li; Yang, Dawei; Wang, Xun; Tong, Lin; Zhu, Xiaodan; Zhong, Nanshan; Bai, Chunxue; Powell, Charles A.; Chen, Rongchang; Zhou, Jian; Song, Yuanlin; Zhou, Xin; Zhu, Huili; Han, Baohui; Li, Qiang; Shi, Guochao; Li, Shengqing; Wang, Changhui; Qiu, Zhongmin; Zhang, Yong; Xu, Yu; Liu, Jie; Zhang, Ding; Wu, Chaomin; Li, Jing; Yu, Jinming; Wang, Jiwei; Dong, Chunling; Wang, Yaoli; Wang, Qi; Zhang, Lichuan; Zhang, Min; Ma, Xia; Zhao, Lin; Yu, Wencheng; Xu, Tao; Jin, Yang; Wang, Xiongbiao; Wang, Yuehong; Jiang, Yan; Chen, Hong; Xiao, Kui; Zhang, Xiaoju; Song, Zhenju; Zhang, Ziqiang; Wu, Xueling; Sun, Jiayuan; Shen, Yao; Ye, Maosong; Tu, Chunlin; Jiang, Jinjun; Yu, Hai; Tan, Fei","Laine, Outi; Joutsi-Korhonen, Lotta; Lassila, Riitta; Huhtala, Heini; Vaheri, Antti; M\u00e4kel\u00e4, Satu; Mustonen, Jukka","","Kim, Mi-Na; Ko, Young Jin; Seong, Moon-Woo; Kim, Jae-Seok; Shin, Bo-Moon; Sung, Heungsup","Beigel, John H.; Nam, Hannah H.; Adams, Peter L.; Krafft, Amy; Ince, William L.; El-Kamary, Samer S.; Sims, Amy C.","Cao, Zhiliang; Liu, Lifeng; Du, Lanying; Zhang, Chao; Jiang, Shibo; Li, Taisheng; He, Yuxian","Song, Yuanlin; Jiang, Jinjun; Wang, Xun; Yang, Dawei; Bai, Chunxue","Pang, Junxiong; Wang, Min Xian; Ang, Ian Yi Han; Tan, Sharon Hui Xuan; Lewis, Ruth Frances; Chen, Jacinta I. Pei; Gutierrez, Ramona A.; Gwee, Sylvia Xiao Wei; Chua, Pearleen Ee Yong; Yang, Qian; Ng, Xian Yi; Yap, Rowena K. S.; Tan, Hao Yi; Teo, Yik Ying; Tan, Chorh Chuan; Cook, Alex R.; Yap, Jason Chin-Huat; Hsu, Li Yang","Rauch, Susanne; Jasny, Edith; Schmidt, Kim E.; Petsch, Benjamin","",""],"color":["#ef8a62","#ef8a62","#ef8a62","#ef8a62","#ef8a62","#ef8a62","#ef8a62","#ef8a62","#ef8a62","#ef8a62","#ef8a62","#ef8a62","#ef8a62","#ef8a62","#ef8a62","#ef8a62","#ef8a62","#ef8a62","#ef8a62","#ef8a62","#ef8a62","#ef8a62","#ef8a62","#ef8a62","#ef8a62","#ef8a62","#ef8a62","#ef8a62","#ef8a62","#ef8a62","#ef8a62","#ef8a62","#ef8a62","#ef8a62","#ef8a62","#ef8a62","#ef8a62","#ef8a62","#ef8a62","#ef8a62","#ef8a62","#ef8a62","#ef8a62","#ef8a62","#ef8a62","#ef8a62","#ef8a62","#ef8a62","#ef8a62","#ef8a62"],"date":["2020-03-04","2020-03-13","2017-12-19","2017-12-19","2020-03-13","2020-02-05","2020-03-03","2010-05-25","2020-03-27","2020-03-30","2020-03-30","2020-03-27","2020-03-21","2005-11-10","2020-03-16","2004-01-01","2020-03-10","2017-12-31","2005-12-31","2014-12-31","2004-01-01","2020-02-26","2016-12-31","2020-02-08","2016-07-20","2017-02-28","2006-12-31","2020-02-29","2009-01-31","2014-01-09","2020-03-20","2019-12-31","2018-12-31","2006-12-31","2020-01-31","2006-12-31","2009-01-31","2013-06-30","2006-12-31","2020-12-31","2016-12-30","2006-12-31","2016-06-24","2019-07-31","2010-11-04","2020-12-31","2020","2018-09-19","2008-05-31","2020-03-01"],"doc":["izolg0pm","qpwvn2fi","gp6gz0bw","gp6gz0bw","qpwvn2fi","b4mdiont","szsb1oan","obes9kjl","2un9aggj","3wuh6k6g","3wuh6k6g","2un9aggj","vlqd192g","7z10l0m1","4yuw7jo3","aza0pzud","te049lxh","1h87wldg","qma1lsuw","z0bxsjug","aza0pzud","13jupb26","dbuy7vvw","1qkwsh6a","zd011ca5","9zx8mr0f","cgcvfftf","cv3qgno3","vgizgq2y","59gmu16g","d840uu3e","1qj9ri18","j13p0ezj","cgcvfftf","55snzm9h","cgcvfftf","vgizgq2y","vpq4yghn","aei5jyp8","wyz5jyjh","jhcc6eex","aei5jyp8","1le537yd","h97dui9n","bd2ndoug","sqyhzxty","8gncbgot","rc3ymu9d","39vjafky","nb0gkwku"],"journal":["The Lancet","The Lancet","Int J Environ Res Public Health","Int J Environ Res Public Health","The Lancet","","Journal of Autoimmunity","Virology","","","","","","Ecohealth","Cell Discov","Hum Genomics","","Diagnostic Molecular Pathology","Microbial Forensics","Encyclopedia of Agriculture and Food Systems","Hum Genomics","Computers in Biology and Medicine","Biotechnology","Vet Q","Biomed Res Int","The Veterinary Journal","Clinical Microbiology and Infection","","Trends in Biotechnology","Expert Rev Mol Diagn","","Reference Module in Life Sciences","Practical Pulmonary Pathology: A Diagnostic Approach","Clinical Microbiology and Infection","The Lancet Infectious Diseases","Clinical Microbiology and Infection","Trends in Biotechnology","The Lancet Respiratory Medicine","Clinical Microbiology and Infection","Clinical eHealth","Medicine (Baltimore)","Clinical Microbiology and Infection","Ann Lab Med","Antiviral Research","Virol J","Clinical eHealth","Journal of Clinical Medicine","Front Immunol","Clinical Microbiology and Infection","Bull World Health Organ"],"rad":[0.02,0.0182708365571112,0.012172759263845461,0.01124140891387803,0.00907018713596907,0.009009168410174833,0.008795199086968356,0.008626254816394577,0.008621494544049051,0.008574092061495439,0.008559941931581123,0.008485018945550461,0.008467517585716906,0.008279096096906496,0.007635966781770583,0.007310797508470865,0.006899621619269412,0.006867788127366274,0.006224473989911304,0.0058549966469016645,0.00553720840497629,0.005374446519321677,0.005100290069116101,0.004987303967448054,0.00493355207412801,0.004880455691702103,0.004869702978936298,0.004808496648852785,0.004763324273628711,0.004731348322627397,0.004630615793048169,0.004581138998718413,0.00432487514235261,0.004281779382582805,0.004180197167189842,0.0039475950401112,0.0038134366092527996,0.003748530399606458,0.003725732119869041,0.0036411313052413493,0.003631957631707963,0.0034224511015150043,0.0034194728703140355,0.0033897243730755472,0.0033098286963083186,0.003272387198540663,0.0032162751378926754,0.003055263234467514,0.0030121357354381856,0.003],"scr":[1.0,0.898,0.54,0.485,0.357,0.353,0.341,0.331,0.331,0.328,0.327,0.323,0.322,0.311,0.273,0.254,0.229,0.228,0.19,0.168,0.149,0.14,0.124,0.117,0.114,0.111,0.11,0.106,0.104,0.102,0.096,0.093,0.078,0.075,0.069,0.056,0.048,0.044,0.043,0.038,0.037,0.025,0.025,0.023,0.018,0.016,0.013,0.003,0.001,0.0],"text":["WHO guidelines for testing should be followed. 77 MERS-CoV testing must be performed in appropriately equipped biosafety laboratories by staff trained in the relevant technical and safety procedures. National or WHO guidelines on the laboratory biosafety should be followed in all circumstances. 83 Three real-time RT-PCR (rRT-PCR) assays for routine detection of MERS-CoV have been developed targeting upstream of the E protein gene (upE) and open reading frame (ORF)1b and ORF1a. The assay for the upE target is considered highly sensitive and is recommended for screening, with the ORF1a assay (ORF1a assay is considered more sensitive than ORF1b assay). 77 An updated roadmap for MERS-CoV product development lists all available diagnostics and other diagnostics in the developmental phase. 84 Several diagnostic tests are under development to accelerate turnaround times from sample analysis to result. [85] [86] [87] A 2018 rapid and specific assay for the detection of MERS-CoV combines the reverse transcription loop-mediated isothermal amplification technique and a vertical flow visualisation strip (RT-LAMP-VF) to detect the nucleocapsid gene of MERS-CoV. 86 The RT-LAMP-VF assay provides detection of MERS-CoV within 35 min and is easier to perform than the WHO-recommended rRT-PCR method. 86 Several serological assays are available for detection of MERS-CoV: ELISA, recombinant spike immunofluorescent assay, and spike pseudoparticle neutralisation and microneutralisation assay. [87] [88] [89] [90] A case confirmed by serology requires evidence of seroconversion in two samples, ideally taken at least 14 days apart, by a screening (eg, ELISA, immunofluorescence assay) and a neutralisation assay. 77, 78 Imaging A range of abnormal but non-specific chest x-ray findings are seen in patients with MERS. 91, 92 These abnormal ities include unilateral or bilateral bronchovascular shadowing, interstitial infiltrates, reticular opacities, reticulonodular shadowing, nodules, pleural effusions, and patchy to confluent consolidation (appendix pp [18] [19] . Lower lobes tend to be affected more than upper lobes early in the course of MERS and rapid opacification of lungs and progression to acute respiratory distress syndrome can occur.","Fortunately, scientists around the world are working at breakneck speed to figure out how to detect, treat, and control the new coronavirus. On Feb 10-12, 2020, WHO brought almost 400 scientists together for a research and innovation forum on the new coronavirus. 4 The meeting covered the topics of diagnostics, vaccines, and therapeutics for COVID19, alongside questions of how to best integrate social science into the response and protection of healthcare workers from infection. The forum generated a research roadmap, due to be published at the end of February, 2020, to develop tools to help control the outbreak, reduce deaths, and minimise damage to economies and the social fabric of communities.","The National Outbreak Control Taskforce (NOCT): There was an urgent need for information to support the decision-making process of the NOCT, which was formed by the SCH to respond to the outbreak. As there were only a small number of reported MERS-CoV cases, the main information sources used by the decision-makers were WHO, KSA, and HPA in the UK, and the media. This information served as a basis for the development of the risk assessment plan and the roadmap for a response. The roadmap was developed by WHO and the Food and Agriculture Organization of the United Nations (FAO) providing technical support in response to Qatar\u2019s request. The roadmap emphasized: enhanced surveillance; case management, isolation, infection prevention and control; health education; and, risk communication. Despite these efforts, public fear was palpable due to the approaching Hajj season. Therefore, a significant part of the technical discussion that involved the Medical Committee for the Hajj focused on considering potential epidemic scenarios.","Assessing the potential risk was a critical component of the WHO support, as the roadmap that it helped develop eased securing the resources required for investigation and response measures. Thus, decision makers approved the proposed roadmap despite the overwhelming uncertainty regarding both the nature of the epidemic and the indefinite outcomes of the ratified proposed actions.","The roadmap is intended to enable scientists, researchers, and funders to coordinate and align","We posit that the human PPIs provide an unbiased, rational roadmap for repurposing drugs for potential treatment of HCoVs in which they were not originally approved.","We can always improve on the handling of global pandemics or epidemics. From the lessons learned during the SARS and now the SARS-CoV-2 epidemics, we can nearly provide a roadmap for the response to future outbreaks. The development of a vaccine for coronavirus is a critical step in prevention, but it may not be effective for future strains, and we must be ready for the next epidemic.","The unique and conserved features of the 5\u2032 UTR and SL5 highlight the role of RNA structure in the evolution of CoVs and may serve as a roadmap for further studies. Future experiments should also verify whether the conserved UUYCGU motifs in SL5 function as PS in group I and II CoVs by interacting with nucleocapsid and/or membrane proteins (Molenkamp and Spaan, 1997; Narayanan and Makino, 2001; Narayanan et al., 2003) . The absence of these or other conserved motifs in the 5\u2032 UTR of group III CoVs suggests that their PSs are located elsewhere in the genome. This possibility is currently being explored.","Pathway enrichment analysis revealed that several of the genes positively associated with ACE2 were regulated by KDM5B, and by specific histone acetylation (H3K27ac) and histone methylation (H3K4me1 and H3K4me3) (Figure 4b ). In fact, KMD5B demethylates lysine 4 of histone H3 (i.e. H3K4) and is involved in transcriptional regulation and DNA repair [32] . We then checked in the Roadmap Epigenomics Project database [33] to see whether ACE2 locus contained ChIP-seq information for these histone markers. In the . CC-BY-ND 4.0 International license It is made available under a author/funder, who has granted medRxiv a license to display the preprint in perpetuity.","Understanding of COVID-19 is evolving rapidly. It is essential to understand the emerging scientific knowledge to coordinate COVID-19 research globally. Recently, the WHO has published a Global Research Roadmap with immediate, mid-term and longer-term priorities to enable the implementation of priority research [8] . Bonilla-Aldana DK et al. [9] and Md Mahbub Hossain MBBS [10] have performed bibliometric analysis to evaluate the scientific literature on coronavirus infection and COVID-19, basing on indicators such as the number of articles, the productivity of authors, geographic distribution of articles and prominent keywords. However, to the best of our knowledge, there is still no quantitative thematic analysis available to date that focuses on CoVs.","Similar to the study of Md Mahbub Hossain MBBS [10] , we did not identify any topic about social, economic, political, psychological, or cultural consequences of COVID-19 as well as other CoV infections. The WHO roadmap has noticed this and taken 'social sciences in the outbreak response' as a priority. The social science research community should act to assess the impact of a major public health crisis like COVID-19 on the complex network of modern societies [23, 24] . [25] . By highlighting such theme interactions within coronavirus research, we hope to inspire new collaborations among researchers, by sharing data, information, technologies and ideas. The fight against the COVID-19 pandemic will benefit from active efforts to cross fertilize.","is the (which was not peer-reviewed) The copyright holder for this preprint . https://doi.org/10.1101/2020.03.21.20040261 doi: medRxiv preprint human lung, peaks for H3K4me1 and H3K4me3, as well as H3K27ac, were identified in ACE2 locus (Figure 4c ), suggesting that ACE2 may be epigenetically regulated in the lung. Figure 4 . Insights of ACE2 regulation in the lung. a. Genes whose expression is correlated with ACE2 in the lung. Selected genes that were negatively (blue) or positively (red) correlated with ACE2 are highlighted. b. Pathway enrichment analysis using the ACE2-positively correlated genes. Pathways from the \"ChIP-X Enrichment Analysis\" and \"Epigenomics Roadmap\" databases with Adjusted P-value &lt; 10 -10 were selected. The size of the red circles is proportional to the -log10 Adjusted P-value of the enrichment. c.","Polymerase chain reaction (PCR) method is the most commonly used technology for pathogen nucleic acid detection and has been considered as a \"gold standard\" for disease diagnostics due to high sensitivity and specificity. 4, 5, 6 However, it typically relies on expensive equipment and well-trained personnel, all of which is not suitable for point of care diagnostic application. In recent decades, several isothermal amplification methods, such as recombinase polymerase amplification (RPA) 7 , loop-mediated isothermal amplification (LAMP) 8 , have been developed as attractive alternatives to conventional PCR method because of their simplicity, rapidity and low cost. However, there is still a challenge to apply it to develop a reliable POC diagnostics for clinical applications due to non-specific signals (e.g., false-positive). 9, 10 Recently, RNA-guided CRISPR/Cas nuclease-based nucleic acid detection has shown great promise for the development of next-generation molecular diagnostics technology due to its high sensitivity, specificity and reliability. 11, 12 For example, some Cas nucleases (e.g., Cas12a, Cas12b and Cas13a) perform strong collateral cleavage activities in which a crRNA-targetbinding activated Cas can indiscriminately cleave surrounding non-target single-stranded nucleic acids. 13, 14, 15, 16, 17 By combining with RPA preamplification, Cas13 and Cas12a have, respectively, been used to develop SHERLOCK (Specific High-sensitivity Enzymatic Reporter UnLOCKing) system 18 and DETECTR (DNA Endonuclease-Targeted CRISPR Trans Reporter) system 14 for highly sensitive nucleic acid detection. Apart from RPA method, some CRISPR-Cas-based nucleic acid sensors utilized the LAMP and PCR approaches, for instance, the CRISPR-Cas12b-assisted HOLMESv2 platform. 16 However, these CRISPR-Cas-based detection methods typically require separate nucleic acid pre-amplification and multiple manual operations, which undoubtedly complicates the procedures and brings about contaminations.","A symposium entitled \u201cEmerging Infectious Diseases and Social\u2013Ecological Systems\u201d was held in Honolulu, Hawaii in March 2005. The meeting was designed to facilitate interdisciplinary integration as part of the National Institutes of Health Roadmap initiative \u201cResearch Teams of the Future\u201d and was hosted by the East West Centre and the Asia-Pacific Institute of Tropical Medicine and Infectious Diseases. The ideas expressed here reflect the experiences and lessons of a group of scholars with expertise in fields ranging from social science to international development, ecology, and various branches of veterinary and human medicine. The article represents the collective ideas of a working group whose task was to focus on the role of \u201ctransdisciplinary approaches\u201d as a means to integrate knowledge across and beyond academic disciplines; to enhance our understanding of the social and ecological contexts of infectious disease and, ultimately, to improve prevention and control of emerging infections. The meeting themes of interdisciplinary integration and application of a coupled, human\u2013natural systems approach to emerging infectious disease are elaborated further by Lewis (2005) and Wilcox and Colwell (2005).","We posit that the human PPIs provide an unbiased, rational roadmap for repurposing drugs for potential treatment of HCoVs in which they were not originally approved. Given C, the set of host genes associated with a specific HCoV, and T, the set of drug targets, we computed the network proximity of C with the target set T of each drug using the \u201cclosest\u201d method:1\\documentclass[12pt]{minimal} \t\t\t\t\\usepackage{amsmath} \t\t\t\t\\usepackage{wasysym}  \t\t\t\t\\usepackage{amsfonts}  \t\t\t\t\\usepackage{amssymb}  \t\t\t\t\\usepackage{amsbsy} \t\t\t\t\\usepackage{mathrsfs} \t\t\t\t\\usepackage{upgreek} \t\t\t\t\\setlength{\\oddsidemargin}{-69pt} \t\t\t\t\\begin{document}$$\\left\\langle d_{CT} \\right\\rangle = \\frac{1}{\\vert\\vert C \\vert\\vert +\\vert\\vert T \\vert\\vert}\\left({\\sum\\limits_{c \\in C}} {\\min}_{t \\in T}\\,d\\left( {c,t} \\right) + {\\sum\\limits_{t \\in T}} {\\min}_{c \\in C}\\,d\\left(c,t\\right) \\right),$$\\end{document}dCT=1\u2223\u2223C\u2223\u2223+\u2223\u2223T\u2223\u2223\u2211c\u2208Cmint\u2208Tdc,t+\u2211t\u2208Tminc\u2208Cdc,t,where d(c, t) is the shortest distance between gene c and t in the human protein interactome. The network proximity was converted to Z-score based on permutation tests:2\\documentclass[12pt]{minimal} \t\t\t\t\\usepackage{amsmath} \t\t\t\t\\usepackage{wasysym}  \t\t\t\t\\usepackage{amsfonts}  \t\t\t\t\\usepackage{amssymb}  \t\t\t\t\\usepackage{amsbsy} \t\t\t\t\\usepackage{mathrsfs} \t\t\t\t\\usepackage{upgreek} \t\t\t\t\\setlength{\\oddsidemargin}{-69pt} \t\t\t\t\\begin{document}$$Z_{d_{CT}} = \\frac{{d_{CT} - \\overline {d_r} }}{{\\sigma _r}},$$\\end{document}ZdCT=dCT\u2212dr\u00af\u03c3r,where \\documentclass[12pt]{minimal} \t\t\t\t\\usepackage{amsmath} \t\t\t\t\\usepackage{wasysym}  \t\t\t\t\\usepackage{amsfonts}  \t\t\t\t\\usepackage{amssymb}  \t\t\t\t\\usepackage{amsbsy} \t\t\t\t\\usepackage{mathrsfs} \t\t\t\t\\usepackage{upgreek} \t\t\t\t\\setlength{\\oddsidemargin}{-69pt} \t\t\t\t\\begin{document}$$\\overline {d_r}$$\\end{document}dr\u00af and \u03c3r were the mean and standard deviation of the permutation test repeated 1000 times, each time with two randomly selected gene lists with similar degree distributions to those of C and T. The corresponding P value was calculated based on the permutation test results. Z-score &lt; \u22121.5 and P &lt; 0.05 were considered significantly proximal drug\u2013HCoV associations. All networks were visualized using Gephi 0.9.2 (https://gephi.org/).","The encouraging early results from the DNA microarray technology has inspired the development of a variety of array-based platforms [38-42] and contributed to the development of a system-wide study of proteins -- the field of modern pro-teomics. Traditionally, two-dimensional gel electrophoresis (2-D PAGE) with mass spectrometry has been the basis of proteomic technology [43]. Although this technique has provided rich information about individual proteins, its clinical usefulness for the study of the proteome is limited. The limitations of this technique for clinical diagnostics are primarily due to low sample throughput compared with other techniques. Additionally, there are technical challenges relating to the reproducibility of the 2-D gels, the relatively low numbers of proteins that can be resolved on each gel and the limited sensitivity (ie low abundant proteins are not easily identified). Modern proteomics technology, as discussed in several recent reviews, is developing quickly and is focused on mass spec-trometry-based serum pattern profiling and protein micro-arrays in continuing efforts to discover new molecular markers and therapeutic targets in human cancer [44-46]. The use of serum proteomic pattern analysis has created diagnostic signatures for ovarian [47,48], breast [49], prostate [50] and liver cancers [51]. In the Laboratory of Pathology at the National Cancer Institute (NCI), in collaboration with the Food and Drug Administration (FDA), this proteome-based strategy is being used for early detection of ovarian cancer in high-risk women. Preliminary data is so encouraging that the technology is on track to enter clinical trials in the very near future [47]. For women living with the spectre of an aggressive disease, such as ovarian cancer, a reliable screening test would represent a major step forward in the diagnostic capabilities of the physicians who treat them.","The detection of viral proteins in body fluids can be a rapid and specific diagnostic for infection in severe acute respiratory syndrome (SARS). 1-3 During the 2003 (SARS) outbreak, non-MS based methods of protein detection proved to be more successful 4,5 than LCMS methods. [6] [7] [8] Non-MS based methods, such as western blots, enzyme-linked immunosorbent assays (ELISAs), and protein arrays, rely on antibodies for the detection of proteins. Given recent studies concerning high variability in antibody production, LCMS-based methods are an attractive alternative approach for the rapid identification of small molecules, proteins, and peptides in clinical settings where consistency is paramount. 9, 10 In the 15 years since the 2003 SARS outbreak, LCMS technology has experienced a revolution led primarily by increases in the speed, sensitivity, and resolution of MS instruments. Today, protein array and antibody-based methods are falling out of favor in both research and clinical diagnostics, due in large part to the improvements in LCMS technology. 11, 12 A review of this growth by Grebe and Singh described a clinical lab with no LCMS systems in 1998 that completed over 2 million individual LCMS clinical assays in 2010. 13 Incremental improvements in rapid sample preparation techniques, chromatography, and data processing have also contributed to the increasing use of LCMS-based clinical testing. A 2013 study demonstrated the level of advance by identifying 4,000 yeast proteins in one hour of LCMS run time, identifying approximately 75 proteins/min at a rate 100 times faster than studies a decade prior. 14 LCMS methods can measure protein quantity by either the intact protein (protein-centric) or the analysis of proteolytic peptides (peptide-centric). While the mass to charge (m/z) ratio of a peptide or protein (MS1) may be a specific diagnostic in some materials, the majority of LCMS methods employ tandem MS in which the peptide or protein parent ion is subjected to gas phase collision to produce fragment ions. The measurement of the fragment ions (MS2) has a higher specificity and lower level of false positives and is the method of choice in clinical diagnostics. 12, 13 Protein-centric assays, such as intact parallel reaction monitoring or proteoform monitoring, performs well on small GTPases like KRAS, but larger proteins cannot currently be monitored with the same relative ease and accuracy. 15 For this reason, peptide-centric assays are currently the dominant proteomics approach.","The advances of sequencing technology, nanotechnology, and bioinformatics have driven molecular tests including assays for emerging pathogens to be more comprehensive and precise. For example, the availability of various sequence databases permits quick identification of sequence identity and variations. For example, the HIV database http://www.hiv.lanl.gov/ contains data on HIV genetic sequences and drug resistance associated mutations. It is valuable for HIV epidemiological studies, research, development, and clinical validation studies of HIV clinical assays [38, 71] . It is well known that there are significant variations of clinical phenotypes in the presence of emerging infections ranging from asymptomatic carrier to lethal infection. Recently, assays to examine multiple pathogen panels have been developed [97\u00c0102], which should increase the diagnostic yield for many pathogens. A critical need for emerging pathogen analysis is quicker, easier, cost-effect assays that can be used in a point-of-care setting. New assays that are performed on platforms with a small footprint and detect pathogens quickly (in minutes instead of hours or days) have entered clinical use. For example, the FilmArray (BioFire Diagnostics, Inc.) and Simplexa (Focus Diagnostics, Inc.) molecular assays can generate results in approximately 60 min. The user-friendly Alere i (Alere Inc.) and Cobas Liat (Roche Molecular Systems) platforms are compact and portable, generate rapid molecular results in 15\u00c020 min, can use electricity or rechargeable battery, and therefore are completely mobile and suited for point-of-care testing. It is obvious that the current rapid development of new technologies will further enhance the utility of molecular diagnostics in various emerging infectious diseases.","In our unpredictable world, viral diagnostics and forensics are becoming increasingly important and sophisticated. The purpose of this chapter is to introduce basic concepts in virology and to provide several examples of how current technology has been used to identify and track viral pathogens. The first part of the chapter discusses virus structure, replication strategies, classification, and evolution. The second part highlights the varied modes of transmission, infection, and disease manifestations as they relate to the different diagnostic methods for virus identification. We follow this with examples of how emerging viruses were identified and tracked (Sin Nombre, Nipah, West Nile, SARS coronavirus, and monkeypox). With regard to viral forensics, the Schmidt case is a fascinating example illustrating how the study of RNA virus evolution and relatedness was used in a criminal investigation. Finally, we provide a few snapshot views into new technologies that allow the creation of manmade or \"engineered'' viruses, some exciting new frontiers in viral diagnostics, and a few thoughts on the future of virology.","Potentially, within the next five years as 'Deep' or 'NexGen' sequencing becomes more widely adapted (Studholme et al., 2011; Kreuze et al., 2009) , and universal plant microarrays are perfected (Hammond, 2011) they could replace indexing. This would require extensive validation of these technologies to ensure they are as good or better than current methods. The advantage of these technologies is that they have the potential to provide information on any virus in a plant without any a priori knowledge of the virus, in contrast to other laboratory techniques that detect known viruses (Kreuze et al., 2009; Al Rwahnih et al., 2009; Kashif et al., 2012; Thekke-Veetil et al., 2013; Seguin et al., 2014; Hammond, 2011; Esteban et al., 2010) . Wang et al. (2002) demonstrated the feasibility and utility of the microarray technology to identify and characterize new viruses. They developed microarrays containing oligonucleotides that represented conserved sequences of all fully sequenced human respiratory viruses, which at the time represented a few hundred viruses. Using this array they identified a novel Coronavirus and showed that it caused severe acute respiratory syndrome, a newly emerging disease at the time (Wang et al., 2003) . NexGen sequencing has a huge impact on many aspects of biology and is used in virus discovery and is being investigated for virus diagnostics. Sequencing of total nucleic acids in plants has led to the identification of multiple viruses and viroids in single plants (Adams et al., 2009; Al Rwahnih et al., 2009 Kreuze et al., 2009; Sequin et al., 2014) and offers the potential to certify plants as virus-free rather than virus-tested. Obtaining the 'virome' of a plant provides much information on the range of viruses and their diversity within a single plant. However, work with grapevines has shown that many of the viruses were related to mycoviruses rather than plant viruses (Al Rwahnih et al., 2011; Coetzee et al., 2010) . This leads to questions on the significance of these viruses in plant health. The new technology will allow for rapid discovery of many new viruses; unfortunately, characterizing the biological significance of these viruses will take much longer. Mycoviruses in grapevines may actually be modifying endophytes and indirectly impacting the health status of grapevines. Thus, eventually how viruses and other microorganisms impact the whole plant (understanding the microbiome) may be important in certification and quarantine, but we are not there yet. Although it is good to know what is in the plant, this technology is resulting in the identification of many new viruses for which there is no biological information. For the next decade it is likely that the biological significance will lag behind the discovery of microorganisms in plants, and it is important that for certification and quarantine programs, any changes are made in response to the biology of these organisms rather than their presence. It seems reasonable that new plant viruses related to known plant pathogenic viruses should be considered as the highest priority for evaluating their biological significance and determining if they should be part of quarantine and certification regulations.","The preliminary data provide an exciting start for the integration of genomics approaches into the clinic, and they demonstrate the potential of the technology to improve diagnostics and treatment. It will be interesting to learn whether or not the gene expression profile will be validated in a larger group of women, given that the expression profile was generated from young women (&lt; 53 years of age), while the median age for the diagnosis of breast cancer is between 60-65 years of age. In addition, as anticipated when making treatment decisions for stage I-II patients with breast cancer, a small percentage of women will be misclassified as having a 'good-prognosis signature' -- ie a low risk for developing distant metastases, will not receive adjuvant systemic therapy and will develop metastases within five years. Although this misclassification error rate is not significantly higher than the misclassification rate that occurs today, using the best clinical criteria available to the physician, further work should be done so that this technology can be introduced into the clinic without denying women available treatments. An upcoming study to be directed by Daniel Haber at the Massachusetts General Hospital underscores this point, as he will not use the gene expression profiles from his study to decide on treatment options, but will rather validate the Dutch prognostic profile. Lastly, it is interesting that 97 per cent of the tumours in the 'good-prognosis signature' were OR positive. Other gene expression profiling studies have observed that the OR status of a breast tumour has a strong influence on the subsequent classifications [60]. As 63 per cent of the tumours in the 'poor-prognosis signature' identified by van de Vijver et al [7]. were OR positive, however, further studies are required to detect the additional pathways -- along with the oestrogen signalling pathway -- that determine the classification. The addition of future proteomic data to the breast cancer gene expression analysis may help to identify those additional signalling pathways that are important for a good prognosis.","This paper is provided to the community to promote the more general applications of the thinking of Professor Paul A. M. Dirac in human and animal medicine in accordance with the charter of The Dirac Foundation, to emphasize the advantages and simplicity of the basic form of the Hyperbolic Dirac Net, to encourage its use, and to propose at least some of the principles of the associated Q-UEL, a universal exchange language for medicine, as a basis for a standard for interoperability. These mathematical and engineering principles are used, amongst many others in an integrated way, in the algorithms and internal architectural features of the BioIngine.com, a distributed system developed by Ingine Inc. Cleveland, Ohio, for the mining of, and inference from, Very Big Data for commercial purposes. Barry Robson BSc(Hons) PhD DSc, Professor Emeritus Medicine was five years as Chief Scientific Officer IBM Global Healthcare, Pharmaceutical, and Life Sciences and, prior to that, six years as the Strategic Advisor at IBM Global Research Headquarters (T. J. Watson Research Centre). For most of those 11 years he held the prestigious title of IBM Distinguished Engineer. According to Barry's two page biography written by journalist Brendan Horton in Nature (389,418-420, 1997), Barry was a pioneer in bioinformatics, protein modelling, and computer-aided drug design. He is the recipient of several honours including the Asklepios Award for Outstanding Vision in Science and Technology at the Future of Health Technology Congress at M.I.T. in 2002. He has helped start up several other companies or divisions in the UK and USA. Barry continues as CEO of The Dirac Foundation in the UK, and Distinguished Scientist (Admin.) at the University of Wisconsin-Stout Department of Mathematics, Statistics, and Computer Science. He is also cofounder of Ingine Inc., Virginia USA, a medical A.I. company. While continuing to work for, and then collaborate with IBM, he was also University Research Director and Professor of EBM, Biostatistics and Epidemiology at St. Matthew's University School of Medicine which he helped established in its earlier days in the Cayman Islands. Barry also holds a Harvard-Macy Diploma in the Business of Medical Education. Immediately prior to joining IBM in 1998 he was hired as Principal Scientist at MDL Information Systems in California to help put together the technology for the multimillion sale of a bioinformatics system to the holding company forming Craig Venter's Celera Genomics that produced the first draft of the human genome. Prior to that, he was CSO of Gryphon Sciences (later Gryphon Pharmaceuticals) in South San Francisco, California, a bionanotechnology ultrastructural chemistry start-up largely held and then acquired by SmithKline Beecham. Before moving to the US, Barry was the scientific founder of Proteus International plc in the UK, designing and leading the development of the PROMETHEUS Expert System and its underlying GLOBAL Expert System, bioinformatics and simulation language for drug, vaccine, and diagnostic discovery. It sold for the equivalent of $9.4 million to the pharmaceutical industry in the mid-1990s. At Proteus, he also led the team that used the above Expert System to invent and patent several diagnostics and vaccines including the Mad Cow disease diagnostic subsequently marketed worldwide by Abbott. He has over 300 scientific publications in Nature, Science, J. Mol. Biol. Biochemical J., including some 50 patents and two books: \"The Engines of Hippocrates. From the Dawn of Medicine to Medical and Pharmaceutical Informatics\" Robson and Baek, 2009, Wiley, 600 pages)\" and \"Introduction to Proteins and Protein Engineering\" (B. Robson and J. Garnier, 1984 Garnier, , 1988 , Elsevier, 700 pages). He has contributed to several reports to governments (EU, US, Denmark) including Panels of the National Innovation Initiative for \"Innovate America\" published by The Council on Competitiveness, Washington D.C. (2004) as a whitepaper to the President of the United States. He was also an advisor in relation to a major scientific computer-aided drug design collaboration and network for Peter Feinstein Consultants between work between US scientists and the Russian Science City Arzamas. ","Rather than attempting to grow and identify disease-causing agents using classical microbiological techniques, molecular diagnostics analyzes molecules; typically DNA, but RNA, proteins, and volatile organic compounds can also be used. (Other diagnostic methods involve the use of antibody technology and are discussed in Chapter 6.) Molecular techniques have the advantage of being quicker, more accurate, and more sensitive.","The emergency pathogens could be counteracted by opting immediate and timely international collaborative efforts, cooperative efforts between human and animal health sectors. Other effective measures include One health approach, implementation of effective prevention and control strategies, rapid communication and networking, and exploring advances in science and technology for developing rapid and confirmatory diagnostics, enhancing disease surveillance and monitoring, implementation of strict biosecurity measures, and timely efforts toward designing appropriate and effective vaccines and therapeutics (Cheng et al. 2020; Cohen 2020; Cyranoski 2020; Lu 2020; Munjal et al. 2017; Singh et al. 2017).","In conclusion, the SARS tragedy compelled the Chinese government to reform its administrative rules, promote biomedical technology, and enhance its public health systems. These improvements involve three fundamental, mutually reinforcing components: (1) enduring government administration reforms, including legislation establishing a unified public health emergency management system; (2) prioritized funding for biotechnology and biomedicine industrialization, especially in the areas of pathogen identification, drug production, and the development of vaccines and diagnostics; and (3) increasing investment for public health and establishment of a rapid-response infectious diseases prevention and control system. China's experience in combating emerging epidemics would benefit human security and public health over the world if shared internationally.","Rapid diagnosis of MERS-CoV infections in dromedaries can also be achieved by direct antigen detection assays. In 2015, we published a monoclonal antibody-based ELISA for the detection of the nucleocapsid antigen of MERS-CoV in respiratory samples, with high sensitivity and specificity (Chen et al., 2015) . Recently, we have used a lateral flow immunoassay platform for rapid detection of nucleocapsid antigen in respiratory samples of dromedaries; this has a sensitivity of~80% and a specificity of 100% when compared to RT-qPCR (Chen et al., 2016) . This technology has enabled the on-site screening of a large number of dromedaries in a short period of time. It also avoids the use of expensive equipment and does not require expertise in molecular diagnostics.","Progress in detection of bacteria responsible for infections in challenging clinical situations and of antimicrobial resistance R. Leclercq (Caen, FR) Identification by conventional techniques of bacteria responsible for infections remains challenging in situations when these bacteria are rare in the clinical sample or difficult to grow or when antibiotic treatment has been administered early. Bone and joint infections, endocarditis and meningitis are examples of these types of situation. Another challenging issue is the rapid detection of bacteria in case of severe infection or for prevention of neonatal infection by group B. streptococci. Several molecular homemade or commercially available techniques can now be used which provide reliable results. Detection of antimicrobial resistance might be useful when resistance is poorly expressed in conventional tests or for epidemiologic purposes. In particular, detection of methicillin resistance in staphylococci has been the subject of extensive works. Several methods are now available which allow the detection of PLP2a expression or of the mecA gene. Several commercial kits are available for the detection of the mecA gene that is generally based on DNA amplification followed by hybridization. These kits generally allow species identification in the same experiment. Similarly, identification of vancomycin resistance genes in enterococci can be performed using either homemade multiplex PCR or commercial kits. The cost of these techniques is decreasing but remains to be balanced with the benefits that are expected from their use. The advantages of molecular diagnostics in virology has become clear for a growing number of clinically important questions, and has left from the research state to become integrated in the routine diagnostic setting. The repertoire of assays available was mostly focussing on the detection of the blood-borne viruses (HBV, , but due to the availability of new and easy-to-use technologies (mostly real-time amplification based) the detection of every target is possible. Targets like the herpesviruses and a large panel of viruses infecting the respiratory tract, are already routinely detected in larger (university) hospitals. This is also pushed by the \"urgent\" need to be prepared for the detection of new and emerging viruses, like SARS and influenza virus (whether H7N7 or H5N1), but also due to the possibilities to develop assays within a relative short period of time for the newly detected viruses, like the coronaviruses HKU and NL-63, and BOCA virus, which has increased our insight in the clinical implications of these viruses in respiratory tract infections. More knowledge with easy-to-use real time technologies, the improved isolation and detection platforms, the availability of better controlled proficiency panels for an increasing number of targets, and the introduction of universal internal controls, have all resulted in a better and more standardized assay performance. This resulted not only in a discrepancy of the number of commercially available assays used for virological targets compared to in-house develop assays, but also in the acceptability of molecular diagnostics, simply because clinical decisions can now be better made due to the information provided. The last two hurdles, costs and timeto-result are very close to be solved. The developments of molecular technologies and assays during the last years, as well as the detection of new pathogens which in most cases cannot be detected by other technologies like virus culture or serology, and the improved quality control possibilities, has generated from virology a dynamic discipline with an impact in good clinical practice. Although improvements are always possible, like multiplexing, small volumes, very fast turn-around-time, DNA chip technology, the finish is just around the corner.","Here we describe a molecular diagnostic approach for SARS-CoV-2 RNA detection using loop-mediated isothermal amplification (LAMP) and simple visual detection of amplification for potential use in rapid, field applications. LAMP was developed as a rapid and reliable method to amplify from a small amount target sequence at a single reaction temperature, obviating the need for sophisticated thermal cycling equipment (5) . Since the initial description of LAMP, a number of advancements in detection technology have helped establish LAMP as a standard method for simple isothermal diagnostics. These detection methods have allowed detection by visual examination without instrumentation using dyes that utilize inherent by-products of the extensive DNA synthesis, such as malachite green (6), calcein (7) and hydroxynaphthol . CC-BY 4.0 International license It is made available under a author/funder, who has granted medRxiv a license to display the preprint in perpetuity.","The applications of microarrays described here are broad-ranging and span from DNA-or protein-based detection of pathogens to epidemic outbreak surveillance and vaccine development, thus showcasing the full spectrum of microarray potential ( Table 1 ). The progress made in the past several years has brought microarrays to the forefront of rapid diagnostics and medical research. Further integration of upstream sample preparation with downstream data processing is expected to transform microarrays into compact, field expedient solutions for analysis and monitoring in the near future. The large assortments of biomolecules now utilized in microarrays provide novel opportunities in molecular forensics and comparative profiling, especially for the newer and less well-understood pathogens. As we equip ourselves with better capabilities and countermeasures against potential biological threats, we hope to become more agile and responsive whenever such threats emerge in the future. Microarrays have improved our confidence in this respect and will continue to play a decisive part in biodefence research and technology.","A range of new detection technologies has emerged over the last decade. The ones that will make the most impact will be compatible with assay designs that can support a high degree of multiplexing and adaptability to the detection of new viruses. Some of the adaptability will reside in astute primer design using new bioinformatics approaches (see earlier). High degrees of multiplexing, in turn, depend on resolution of signals from simultaneous assays, either by detection at different energy frequencies or by spatially/temporally resolving signals in one, two, three or four dimensions. Azzazy et al. reviewed some of the core technologies in nanodiagnostics, comparing quantum dots, cantilevers and nanoparticles [93]. Quantum dots and nanoparticle-based assays offer the potential for large-scale multiplexing of nucleic acid-based diagnostic tests, generally via the use of capture oligonucleotides to address and separate reactions for analysis. Nucleic acid amplification on microspheres is the core technology of new-generation sequencing [94], but is yet to be fully exploited in first-line clinical molecular diagnostics.","Alternatively, the emerging additive manufacturing technology can also be used to create such ultra-high aspect ratio microstructures as an efficient target capture platform. 27, 28 One of the main advantages of the IMPACT chip compared to traditional CRISPR assays is its ability to limit the background caused by dye-quencher probes, which is typically seen in CRISPR detection in the liquid state and needs to be designed around to lower the detection limit. 29, 30 Our device utilizing solid-phase CRISPR does not need to tether a quencher on the probe. The cleaved CRISPR products are sent to a separate reservoir for detection thus completely avoiding the fluorescence background caused by the tradition \"onepot\" detection. As shown in Fig. 5 , without the presence of ASFV target DNA, no fluorescence background signal was detected, demonstrating this powerful backgroundfree detection. This is an important improvement for molecular diagnostics, especially in low light settings in which the amount of background fluorescence present can significantly affect the detection limit . 31,32 Leveraging this advantage with increasing the aspect ratio either with PDMS or a silicon substrate will be able to further increase the accuracy and robustness of the device and improve the limit of detection.","Among the expected technological improvements, two stand out as likely to benefit the field of viral metagenomics in the near future. First, long-read sequencing technologies are progressively amenable to the sequencing of environmental viral communities. Pragmatically, this means that instead of having to assemble virus genomes from short reads, a process which can yield potentially erroneous and/or incomplete genome sequences, a complete viral genome could be sequenced as a single read. Once broadly available, these long-reads metagenomes will not only bypass assembly issues but also provide valuable information about virus genome evolution by enabling whole-genome phasing of polymorphisms. Meanwhile, in an epidemiological context, long-read sequencing technologies associated with miniaturized devices, streamlined sample preparation, and live scanning of the sequencing results offers unique possibility for real-time surveillance or diagnostics. This is especially the case for the MinION sequencer based on Nanopore sequencing technology, allowing the identification of viral pathogens from a patient sample in less than 6 h, compared to more than 20 h for other sequencing technologies. The computational framework to analyze and share these types of data in a timely, safe, and meaningful way remains to be built, however it is likely that metagenomics through portable genome sequencers will become a major component of the epidemiological toolkit in the near future.","The key morphologic and microbiologic features of the bacterial pneumonias are summarized in Table 7 .5. The presence of purulent exudates or significant numbers of neutrophils in biopsy or cytologic samples should always trigger a search for bacterial infection. Because lung biopsies are usually performed late in the clinical course after procedures have been performed and bacterial infections have been excluded and/or treated with antibiotics, neutrophilic exudates may not signify bacterial infection unless accompanied by necrosis, as in an abscess. Instead, consideration should be given to one of several noninfectious acute inflammatory diseases, with an immunologic basis, that can mimic bacterial infection. Some of these include granulomatosis with polyangiitis, Goodpasture syndrome, systemic lupus erythematosus, and microscopic polyangiitis, all conditions that can produce acute inflammation predominantly involving alveolar septal blood vessels (\"capillaritis\"). On occasion, capillaritis can result in airspace accumulation of neutrophils, further raising concern for bronchopneumonia. Centrally necrotic or cavitary neoplasms of various types may mimic as a \"last resort\" effort. 142, 143 In recent years, delays in the diagnosis of mycobacterial infection have markedly decreased, thanks in part to recommendations from the CDC for improving laboratory turnaround time and to the response of the diagnostics industry with better methods and technology. Because direct acid-fast smears of respiratory specimens yield negative findings in at least half of the cases, 144 and because many mycobacterial species are fastidious and slow-growing, the biopsy results may be the first suggestion of a mycobacterial infection. The biopsy findings can also define the organism's relationship to a histopathologic lesion and host response-important information in evaluating the significance of a culture result. Although an isolate of M. tuberculosis is always taken seriously, obtaining a single isolate of a nontuberculous mycobacterium from the respiratory tract does not necessarily implicate the organism as the cause of disease. 145 ","Rapid methods for the identification of bacterial clones and pathogens Objectives: Previously we identified a genotypically distinct subset of epidemical Enterococcus faecium (Efm), designated CC17 using multilocus sequence typing (MLST). We investigated whether a whole genome microarray approach would type strains similarly and if epidemic specific markers could be identified. Methods: A shotgun library was constructed of 9 Efm strains of which 5700 \u00b1 1.2 kb DNA fragments were spotted on a slide. Cy5 labeled DNA of 100 MLST typed Efm isolates from different sources (outbreaks, infections, surveys and animals) and 5 continents, vancomycin resistant and susceptible, were hybridized to the array together with Cy3 labelled DNA of the library strains. 2919 spots met quality criteria and were included in normalization and EPP data transformation (Kim et al. 2002) . Transversal hierarchical clustering (HC) was performed based on strains and spots (inserts). Inserts differentiating between CC17 and non-CC17 were identified from the visualized data matrix, PCR amplified and sequenced. Sequences were blasted in Genebank (NCBI). Results: HC resulted in 9 different clusters and a small cluster of 2 strains and resembled MLST genogrouping. Four clusters consisted of mainly CC17 strains, including all but two epidemic strains. Approximately 60 % of inserts was conserved while 413 inserts (14.1%) were associated with the epidemic clusters. From these, 190 were selected for sequencing. This revealed that differentiating inserts were gene fragments encoding potential virulence factors (24), resistance determinants (3), transposons or IS elements (32), proteins involved in replication (13) or regulation (3), as well as hypothetical proteins (10). 44 (23%) of hits showed no significant homology. Conclusion: Comparative genomics of Efm based on a mixed whole genome microarray discerned the epidemic MLST subpopulation. From the analysis 14.1% of inserts were identified differential for CC17 including both regulatory and potential virulence genes. This technology may provide new targets for diagnostics and for the development of selective drugs for hospital-adapted Efm.","Failure of host countries to protect the health of refugees and migrants; epidemic of devastating rumours and fake news on social media due to increased digital connectivity; emergence of social influencers exerting influence on politicians and institutions; increased resistance and hesitancy within communities to health interventions because of opposition by local experts Environment Geography affecting biological diversity; planned and unplanned urbanisation; climate change; interaction between humans, animals, and vectors; human development; state of the economy; state of health systems Climate change resulting in increased flooding with failed sanitation and safe water; altered distribution of zoonotic disease reservoirs and vectors; emerging zoonosis with increased agricultural production and human encroachment into animal environments; changing national priorities resulting in sharply reduced investment in health systems Pathogenesis Changing disease biomes; relationship between hosts and pathogens; pathogen evolution and changes; technologies such as synthetic biology, and the risks of manufacturing pathogens and their accidental or deliberate release; characteristics of a population such as underlying disease condition Increased opportunities for mutation or reassortment of infectious agents; increasingly reduced effectiveness of conventional vaccines and therapeutics for prevention and treatment of diseases; failure of conventional control measures to break the chain of transmission of infectious agents To tackle the wide-ranging issues, the Commission has brought together experts from academic, public health, policy making, international, non-governmental, and corporate institutions. They bring local and global knowledge and experience, including policy-making and field response, human and animal health (including One Health) approaches, and novel developments in communications, information technology, analytics, public health, diagnostics, and therapeutics. The Commission aims to deliver the report by 2021 and will include key recommendations for countries and international stakeholders, and monitoring indicators to evaluate the effectiveness of preparedness initiatives over time. The Mok Hing Yiu Charitable Foundation is the sole sponsor for the Lancet Infectious Diseases Commission meetings and travel for some commissioners. The foundation is not involved in the technical discussions, data analysis, and drafting of the report. This Comment was written on the behalf of the Lancet Infectious Diseases Commission on Preparedness for Emerging Epidemic Threats; see appendix (pp 1-2) for the Commissions author list.","The goal of rational use of antimicrobial agents is effective treatment with minimal amount of harmful side-effects. To deal with bacterial infections we have to use antimicrobial agents, but we also know that there is a substantial overuse of these drugs in most parts of the world. The side-effects include not only allergic reactions, antibiotic diarrhoea, costs and development of bacterial resistance but also disturbance of the normal human microbiota (normal flora) that may be more important than we have previously thought. Bacterial resistance is linked with consumption of antimicrobial agents both in hospitals and in the community. On the country level, less antimicrobial agents is linked with less bacterial resistance, but resistance bacteria may spread despite of low level of antibiotic use. Several articles show that use of certain antibiotics has lead to increased resistance against these drugs. However, there are only few studies that show effect of reduction of antibiotic use on high level of resistance. Due to multiresistance, reduction of antibiotic use may not be the best way to decrease antibiotic resistance. The most rational way to tackle antibiotic resistance is to try to prevent development and spread of resistant bacteria. This includes optimal antimicrobial prescribing but also improved hygienic measures especially in child day care centres. Optimal antimicrobial prescribing needs national current care guidelines that are uniformly or widely accepted. The term 'optimal or rational use' includes careful clinical and laboratory diagnostics. Automatic computer notifications may be the best way to guide prudent antimicrobial prescribing. Current care guidelines have also to be flexible to react when bacterial resistance situation is changing. Economical approaches, pricing and especially reimbursement policy, may also be reasonable approaches. There is a need to increase public awareness on antibiotic use and harmful effects linked to improper use. For this purpose, several interesting publications have recently been published. These reports show that healthy human microbiota seems to protect use against allergy, obesity and cancer and antibiotic use may greatly disturb this balance. Paradoxically, antibiotic use may also increase risk of bacterial infections. Once more data is accumulated, we may better understand the value of our microbiota, and, hopefully, patients may not anymore be so eager to ask for antimicrobial prescriptions. Bacterial resistance is emerging, and we have to protect our normal microbiota. Therefore, new therapeutic options should be developed to treat and prevent bacterial infections. There are already interesting new study fields, like bacteriotherapy and therapy with bacteriophages or their enzymes. Future will show the importance of these approaches. Finally, I will also put my hope on unexpected discoveries and totally new solutions that medical technology may create. We need creative innovations.","Microarray technology has also been brought outside the laboratory to the point-of-care, a development that has widening implications for threat detection. DNA-based detection systems generally rely on the ability of PCR to amplify and fluorescently tag the tiny amounts of target DNA present in the specimen. Advances in miniaturizing this initial PCR step, for instance the development of Review Glossary Biodefence: defensive measures against biological threats, including natural/ emerging pathogens and bioterror agents, that have significant potential to endanger public health Detection: identifying the presence of target pathogen(s) from clinical or environmental samples. Diagnostics: tests used to detect a medical condition, for example to test for the causative pathogen responsible for an infection. Sandwich immunoassay: a biochemical assay for detecting the presence and/ or abundance of a target substance using the antibody-antigen reaction. Two antibodies are used; the first is immobilized and the other, free antibody carries a reporter group, thereby providing a positive readout when the target is recognized and 'sandwiched' between the two antibodies. Sensitivity: probability of a positive result when the pathogen is indeed present; high sensitivity is related to a low Type 2 error. Serovars: a group of microorganisms distinguished by the presence of specific surface antigens. Specificity: probability of a negative result when the pathogen is not present; high specificity is related to a low Type 1 error. Tiling arrays: tiling arrays are a type of DNA microarray in which short probe segments that have been designed to cover the entire genome are used. The extent of probe overlap will translate to the mapping resolution and might range from highly overlapped probes across each individual nucleotide base (for resequencing applications) to non-overlapping probes (for genome-wide expression analysis). Vaccinia virus: a poxvirus that is closely related to the virus that causes cowpox. It was the first human vaccine and has been extensively used for vaccination against smallpox. Corresponding author: Uttamchandani, M. (mahesh@dso.org.sg). micro-PCR (mPCR), followed by hybridization and identification against probes on DNA microarrays have spun-off viable chip-based platforms that are able to successfully perform biological threat detection and analysis. Such portable systems, as will be described, are already appearing on the market and competing to attract lucrative biodefence funding. Alternative detection platforms have exploited quantitative PCR in 'fieldable' real-time detectors, where positive PCR amplification would indicate the presence of the corresponding target DNA [4] . These have included devices such as GeneXpert (Cepheid), Rapid (Idaho Technologies) or BioSeeq (Smiths Detection), but these systems provide relatively limited throughput [7] . Microarray systems are, by contrast, more definitive and highly scalable because hundreds to tens of thousands of possible DNA elements can be interrogated in a single experiment. Their performance nevertheless hinges on both the adoption of robust panels of probes that can accurately identify DNA from organisms of interest and the successful extraction and amplification of pathogen DNA from the relevant clinical sample or isolates [8, 9] .","Advances in molecular diagnostics have been applied to simultaneous detection of pathogens and their antibiotic resistance from one sputum sample. The absence of rapid accurate diagnostic tests for pulmonary tuberculosis was further compounded by the widespread inability to screen for drug-resistant bacteria. The introduction of the GeneXpert MTB/RIF Assay, which detects Mycobacterium tuberculosis DNA and rifampicin resistance from sputum samples in 2 h, could revolutionise tuber culosis diagnostics. 10 A lateral fl ow immunoassay for detection of meticillin-resistant Staphylococcus aureus in sputum is also now available. 11 Although several new rapid molecular diagnostic plat forms capable of simultaneously identifying both pathogens and the genetic determinants of anti microbial resistance are emerging, 12 they do not seem to be well suited to clinical presentations or downstream manage ment strategies for patients with RTIs. Furthermore, none can identify new emerging organisms for which genetic sequences are unavailable. However, rapid sequencing technologies linked to bio informatics have provided proof of diagnostic principle in a short time. 13 Until new diagnostics are developed to serve clinical need, the mainstay of clinical management for any RTI will remain dependent on imperfect diagnostics and linked to empirical broad-spectrum antibiotics. Thus, clinicians have to maintain clinical awareness of other possible causes of RTIs, whether they are opportunistic or not.","Objectives: The detection of heterophile antibodies, and specific antibodies against Epstein-Barr virus (EBV) antigens, has proven useful to categorize patients with no history of infectious mononucleosis, patients who are acutely infected, patients with evidence of a prior or remote infection, and patients with reactivation of latent infection. The objective of this study is to compare the results of a fluorescence-based automated multiplexed assay for heterophile determination to results acquired with a conventional latex agglutination spot test. Methods: Using the Bio-Rad BioPlex 2200 system, a fully automated high throughput multiplex platform, we report the detection of IgM antibodies to heterophile antigens and compare them to the Wampole Mono-Latex \u00d2 test, a qualitative subjective methodology. The BioPlex 2200 results are reported as a semiquantitative index value relative to negative and positive calibrators. For each sample, three internal quality control beads are evaluated simultaneously to verify instrument and chemistry performance, supplementing traditional quality control samples and calibration curve checks. The multiplex assay uses 5 ll of specimen and the instrument processes one hundred patient samples per hour. Results: Serum specimens (N = 523 non-randomly selected) were analyzed by both assay methods. About 12% of the sera were positive for heterophile antibody. Compared to the Mono-Latex assay, the BioPlex 2200 gave a sensitivity of 91.5%, specificity of 99.3% and overall agreement of 98.3%. Six samples were initially categorized as false-negatives and three as falsepositives. Upon repeat testing of the discrepant specimens, the automated test system exhibited a sensitivity of 97.1%, specificity of 99.6%, and overall agreement of 99.2%. The Mono-Latex test changed status on five of the nine discrepant samples, highlighting the difficulties interpreting visual readouts. No status changes for the discrepant samples were observed with the automated system. Additionally, the BioPlex 2200 system has excellent precision with CVs of less than 5% for samples near the cut-off. Conclusion: The two heterophile assay systems showed excellent concordance. The BioPlex 2200 system, however, offers practical advantages that allow for rapid and fully automated evaluation of heterophile and IgM EBV antibodies. Background: Management of therapy for hepatitis C virus (HCV) infection is based on quantitative measurement of HCV RNA and the decision of treatment discontinuation on the basis of a 2 log decline at week 12 is a widely accepted rule. Aims and methods: In the present study, we evaluated the performance of the completely automated system Cobas Ampliprep-Cobas TaqMan (CAP/CTM, Roche Diagnostics, Branchburg, NJ) for HCV RNA quantification in clinical serum specimens. CAP/CTM is a real time Polymerase Chain assay that relies on an automated nucleic acid extraction from 1050 ll of serum followed by RNA capture using magnetic particles, purification and elution. The system has a reported lower detection limit of 15 IU/ml and a dynamic range from 43 to 6.9E7 IU/ml. CAP/CTM results were compared for quantification of HCV genotype 1-4 with those from a signal amplification assay based on the branched DNA (bDNA) technology, the Versant Quantitative 3.0 (Bayer Diagnostic, Tarrytown, NY, detection limit 615 IU/ml and dynamic range from 1185 to 1.5E7 IU/ml).","To reflect the three basic processes of ''comprehensive perception ? reliable transmission ? intelligent processing\" of the IoT technology and assist the GPU to manage the nCapp assisted three-level linked cloud plus terminal platform, we designed 15 easy-to-use questionnaires and nucleic acid detection information for deep mining and intelligent processing (Fig. 2) . The diagnosis and treatment recommendations are automatically generated and transmitted to doctors and experts for reference. Physicians and experts can use smartphone nCapp assistant software to participate in the three-linkage IoT cloud plus platform according to their needs (Fig. 2) . 14, 19, 20 The three-cascade IoT ''cloud plus terminal\" nCapp 14 can assist in the intelligent management, command, and diagnosis of COVID-19. ''Cloud\" is a general term for cloud technology, which can be further subdivided into network technology, information technology, integration technology, management platform technology, and application technology based on the application of the cloud computing model. The terminal is a physician's smartphone that can implement the following eight functions in real-time online communication with the ''cloud\" through the page selection key (Fig. 3) :","Fibrinogen was measured with a modification of the Clauss method (Multifibren U, Siemens Healthcare Diagnostics), d-dimer with an immunoturbidimetric assay (Tina-quant D-Dimer, Roche Diagnostics, Mannheim, Germany), and F1 + 2 with an enzyme immunoassay (Enzygnost F1 + 2, monoclonal, Siemens Healthcare Diagnostics). The reference values were 1.7 to 4.0 g/L for fibrinogen, \u22640.5 mg/L for d-dimer, and 69 to 229 pmol/L for F1 + 2.","Objectives: Determine the limit of detection, linearity, reproducibility and correlation between clinical results of the COBAS AMPLICOR HCV test v2.0 (CA HCV) using the MagNA Pure LC (MPLC) for automated specimen preparation. Methods: A range of analytical standards and clinical specimens were used to determine each of the evaluation parameters as follows: Limit of detection and linearity 16 replicates of a 7 member dilution panel of the WHO 2nd International Standard HCV RNA covering the range 0-1000 IU/mL. Reproducibility 80 replicates of HCV 100 IU/mL WHO 2nd International Standard HCV RNA.Concordance Split sample analyses of 100 clinical specimens tested using both methods of sample preparation. Sample preparation was performed using the Total Nucleic Acid Isolation Kit (Roche Diagnostics Australia) on the automated MPLC (Roche Diagnostics Australia) platform according to a modified in-house validated protocol. Nucleic acid from the 100 clinical samples was also extracted in duplicate using the COBAS AMPLICOR HCV test v2.0 (Roche Diagnostics Australia) manual sample preparation protocols. Testing for HCV was performed using the COBAS AMPLICOR HCV test v2.0 (Roche Diagnostics Australia) on the Roche COBAS Amplicor (Roche Diagnostics Australia) according to the manufacturer's instructions. Results: The performance criteria for the CA HCV assay following sample preparation by MPLC were determined. Preliminary results to date demonstrate that the linearity across the range of HCV RNA levels tested (0-1000 IU/mL) is acceptable and that the lower limit of detection is comparable to the manufacturer's claimed sensitivity for manually processed samples (50 IU/mL). For the 100 clinical specimens tested using both methods of specimen preparation, results were concordant for 99/100 specimens. After resolution of the discordant results the sensitivity and specificity of CA HCV following MPLC specimen preparation were 99.0% and 100%, respectively, when compared to manual specimen preparation. Conclusions: The MPLC instrument is a suitable front end platform for use with the COBAS AMPLICOR HCV test v2.0. The modified protocol using the Total Nucleic Acid Isolation Kit on the MPLC resulted in a reliable and labour-saving method for the extraction of nucleic acid. The performance of CA HCV PCR testing on samples processed on the MPLC was comparable to that on samples processed manually. Background: Human cytomegalovirus (HCMV) is an important pathogen capable of establishing lifelong persistent infections, which normally remain asymptomatic. Previous studies indicated that sequence variation among CMV strains frequently occurs even in highly conserved genes' regions. Genetic variation of functionally important genes may complicate CMV diagnostics. Objective: The purpose of this work was designing primers and probes for simultaneously Real-Time PCR diagnostics and genotyping of CMV. Materials and methods: In this study 527 serum samples obtained from pregnant women and 3 samples from children with CMV congenital infection were used. Virus DNA was extracted by using the MagNA Pure DNA purification kit (Roche, Indianapolis, IN, USA). 4 sets of primers directed to IE2, gN, gO and gB CMV genes and 10 sets of probes for Real-Time detection with LidhtCycler instrument (Roche, Indianapolis, IN, USA) have been designed and evaluated. Genotyping was being carried out by sequencing analysis on ABI 3100 Avant Genetic Analyzer instrument (ABI, Foster City, CA, USA). Phylogenetic analysis of the nucleotide sequences was conducted with Clastal X, the Tamura-Nei substitution model, Grow tree based on Neighbour-Joining tree building method, and Maximum Parsimony method implemented in MEGA 3.0 package. Results: The Real-Time PCR analysis with IE2 gene detected CMV activity in 72 isolates among 527 analyzed samples The PCR tests with previously described primers to gN, gO and gB genes revealed 7, 1 and 2 positive samples accordingly. The PCR test sensitivity was defined with quantitative CMV control (ABI, Foster City, CA, USA). The sensitivity of PCR test on IE2 gene was 20 copies per 50 ll. The phylogenetic analysis of IE2 region sequences demonstrated that this region could be successfully used for virus genotyping. The Real-Time FRET test divided all analyzed samples into two groups, those that had a melting peak like laboratory strain Davies and those that had a melting peak like Towne and AD169 laboratory adapted strain. Two pairs of specific hybridisation probe covering two mutations inside IE2 gene's region were used to confirm it. Conclusion: Nested PCR with primers to IE2 gene incorporated with Real-Time FRET analyse described here, provides a sensitive and specific assay for detecting CMV in clinical isolates. The IE2 gene can serve as a target for simultaneously detecting and genotyping of CMV using the Real-Time PCR opportunities.","The WHO and United States Centers for Disease Control and Prevention (US CDC) provided guidelines for the molecular diagnosis of MERS-CoV [34], and since June 6, 2013 the US CDC has made novel coronavirus rRT-PCR assays [5] available free of charge under emergency use authorization [6]. Although at least three commercial rRT-PCR assays for MERS-CoV detection were available from Altona Diagnostics, Fast Track Diagnostics [3], and PrimerDesign (http://www.genesig.com) before the 2015 outbreak in Korea, only RealStar MERS-CoV (Altona Diagnostics, Hamburg, Germany) had been approved for the in vitro diagnosis of MERS-CoV by Conformit\u00e9 Europ\u00e9enne (CE) and authorized for emergency use only in the United States. Therefore, all six commercial kits evaluated in this study had not been validated for diagnostic use. This study was designed to analytically and clinically validate the six above-mentioned commercial MERS CoV RNA detection kits.","For any outbreak, the faster the identification, the faster and more robust the response. BARDA is supporting work to improve diagnostics with the goal of recognizing illness before any symptoms develop. This may include wearable diagnostics or biosensors. The high mortality associated with human infections with novel coronaviruses MERS-CoV and SARS-CoV and avian influenza strains (influenza A(H5N1) and A(H7N9)) have raised questions about the possible role of a dysregulated immune response, a so called \"cytokine storm\", in the pathogenesis of severe respiratory disease. As some patients may not respond to typical anti-pathogen therapeutics (e.g. antivirals), BARDA is pursuing therapeutics which target the host in order to modify the immune/host response. More work needs to be done to identify these host targets and develop therapeutics towards the dysregulated immune response. The field also needs better diagnostics to guide treatment as over-suppression of the immune response can be just as dangerous as over-stimulation.","SARS-CoV is an enveloped positive-stranded RNA virus and its \"crown\"-like spike (S) protein has two major biological functions: 1) mediating receptor (angiotensin converting enzyme 2, ACE2) binding and membrane fusion; 2) inducing neutralizing antibody responses [2,3]. The S protein was considered as an important target for developing diagnostics, vaccines and therapeutics [4-12]. The receptor-binding domain (RBD) of S protein was defined as a fragment corresponding to the residues 318 - 510 of the S protein, which mediates viral binding to cell receptor ACE2 [13-15]. Coincidently, we identified the RBD as a major target of neutralizing antibodies [16-19], and proposed it as an ideal vaccine antigen for clinical application [20-22]. The immunogenicity and protective efficacy of RBD-based vaccine candidates have been evaluated in animal models [17,23-25]. However, the antigenicity and immunogenicity of RBD in humans need to be characterized in detail toward developing the RBD-based vaccines and diagnostics. In this short communication, we found that patients recovered from SARS developed potent and persistent RBD-specific antibody responses, highlighting the potentials of clinical applications of RBD-based vaccines and diagnostics.","Wireless sensing technology was first used in fetal electrocardiograms, 6 and the use of wireless sensing technology to monitor the breathing status of patients dates back to the 1970s. Pope et al. 7 used a temperature sensor at the head end of a tracheotomy to send a patient's breathing parameters to a signal receiver in real time through an FM transmitter. The new technology was then used in the intensive care unit (ICU) to monitor newborns and children, and assist health care workers identify apnea in a timely manner. Wireless stethoscopes designed for anesthesiologists to listen to a patient's heart rate and breathing sounds during surgery have also been developed. 8 as heart rate, breathing rate, electrocardiogram, body temperature, and blood pressure. 9 With the development of technology, physiological parameter recorders based on wireless sensing technology are being introduced to the medical field, such as esophagus pH, 10 pulse blood volume, 11 CO2 decompression, 12 and wireless breathing frequency monitors during movement. 13 Moreover, with the development of microelectronics and integrated industrial technology, numerous sensing technologies have been developed, such as the ''smart T-shirt\". 14, 15 The concept of instant ''plug and play\" modular installation has further accelerated the development of the ''sensor network,\" which can be easily applied for heart, lung function, and other joint monitoring.","Examples of search strings can be found in Table S1 . We searched for randomized controlled trials (RCTs) and validation trials (for diagnostics test) published in English, that measured (a) the sensitivity and/or specificity of a rapid diagnostic test or a point-of-care testing kit, (b) the impact of drug therapy or (c) vaccine efficacy against either of these diseases with no date restriction applied. For the 2019-nCoV, we searched for all in vitro, animal, or human studies published in English between 1 December 2019 and 6 February 2020, on the same outcomes of interest. In addition, we reviewed the references of retrieved articles in order to identify additional studies or reports not retrieved by the initial searches. Studies that examined the mechanisms of diagnostic tests, drug therapy or vaccine efficacy against SARS, MERS and 2019-nCoV were excluded. A Google search for 2019-nCoV diagnostics (as of 6 February 2020; Table S2 ) yielded five webpage links from government and international bodies with official information and guidelines (WHO, Europe CDC, US CDC, US FDA), three webpage links on diagnostic protocols and scientific commentaries, and five webpage links on market news and press releases. Six protocols for diagnostics using reverse transcriptase polymerase chain reaction (RT-PCR) from six countries were published on WHO's website [9] . Google search for 2019-nCoV vaccines yielded 19 relevant articles.","Pandemics such as HIV, Ebola, and Zika have raised the awareness of global threats to human health posed by known as well as newly emerging pathogens and can provide the impetus to prepare against future pandemics by promoting the development of vaccine platforms that can tackle the challenges of outbreak situations. New platforms, such as viral vector and nucleic acid based vaccines meet the prerequisites to provide solutions for some of these challenges by representing highly versatile technologies that allow fast vaccine manufacturing. Each vaccine technology has its own advantages and disadvantages related to its ability to induce certain immune responses, manufacturing capacity and safety for human use (Table 4). Viral vector based vaccines are able to induce potent immune responses against the encoded target antigen. Indeed, a number of clinical trials have demonstrated that viral vector based vaccines such as VSV-ZEBOV show great promise for inducing protective responses in humans. However, antigen delivery in the context of an unrelated virus renders this technology relatively complex in terms of manufacturing. Furthermore, the presence of immune targets other than the target antigen can lead to unfavorable effects such as pre-existing immunity hampering immune responses, as seen for Ad5 vectors, or the inability to use the same technology for repeated vaccinations. In addition, delivery of attenuated viral vectors raises safety concerns due to the risk of adverse events and residual viral replication upon delivery, as detected in a small number of subjects in a clinical trial testing VSV-ZEBOV. DNA based vaccines offer the advantage of allowing a relatively simple, fully synthetic production process. While the presence of non-functional sequences in original DNA vectors raised regulatory safety concerns, newer developments allow minimal constructs that exclusively encode for the target antigen. Several studies have demonstrated the safety of DNA vaccines for human use and clinical trials testing vaccines against influenza and Zika have furthermore highlighted the speed of vaccine development supported by this technology. However, the potential for long term persistence and genomic integration and the dependence on injection devices or electroporation represent some important disadvantages of this technology. Some, especially early, clinical studies testing DNA based vaccines have yielded somewhat discouraging results in terms of immunogenicity, while newer trials, such as studies testing DNA vaccines against Zika virus, have demonstrated that this technology is able to induce promising immune responses. Like DNA vaccines, RNA based vaccine technologies support a comparably simple, fully synthetic manufacturing process that allows production of different vaccines using the same established production process and facility. Their inability for genomic integration and lack of persistence in the cells of the vaccinee offers important advantages in terms of vaccine safety. However, since RNA vaccines represent the most recently developed technology described here, their use in humans is less well characterized than for viral vector or DNA based vaccines. Although further studies will be required to fully characterize this technology in humans, clinical studies conducted so far have yielded overall encouraging results in terms of safety and immunogenicity and provide support for further clinical exploration.","S. Jemielity, M. Barbani, T. Staub, D. Grandgirard, M. Gorgievski-Hrisoho (Berne, CH) Respiratory viruses are among the most frequent human pathogens worldwide, leading to a significant number of hospitalisations, especially in children and in immune-compromised patients. Several new diagnostics tools for respiratory viruses have recently appeared on the market, including the xTag Respiratory Viral Panel (RVP) by Luminex Molecular Diagnostics. The goal of the present study was to compare this new multiplex PCR method with conventional direct immunofluorescence (DIF) in children's nasopharyngeal aspirates for the following viruses: respiratory sincytial virus (RSV), adenovirus (ADV), parainfluenza viruses 1\u22123 (PIF), influenza viruses A and B (IF), as well as human metapneumovirus (hMPV). The RVP kit permits in addition the detection of entero-/rhinoviruses, coronaviruses and parainfluenza virus 4. In total 240 nasopharyngeal aspirates (126 DIF negative and 114 DIF positive samples) were analysed. All RSV, PIVA, IF and hMPV DIF positive samples (43, 15, 18 and 11, respectively) were confirmed as positive for the same virus by RVP. For ADV, however, only 13 out of 27 DIF positive samples (48%) were also ADV positive by RVP. The samples with discrepant results are currently being reanalysed by DIF and by a second, independent PCR. Of note is the finding that over 64% (81/126) of all DIF negative samples were positive when analysed with RVP. This was in great part due to the additional entero-/rhinoviruses detected by RVP (75/81), but also because of additional RSV, IF, PIF, ADV and hMPV positive samples that had been missed with the DIF assays (21/81). Finally, using RVP we detected as many as 10.4% (25/240) of double and 0.5% (1/240) of triple infections, versus a single double infection detected by DIF. The implications of our findings for both diagnostics as well as clinical practices are discussed.","WHO and the Foundation for Innovative New Diagnostics (FIND) have established a formal collaboration on universal access to essential diagnostics. Announced on 10 February, the aim of the collaboration is to close diagnostic gaps at country level and bolster disease surveillance that will inform public health initiatives and enhance outbreak preparedness and response."],"title":["Middle East respiratory syndrome","What are the risks of COVID-19 infection in pregnant women?","How Do the First Days Count? A Case Study of Qatar Experience in Emergency Risk Communication during the MERS-CoV Outbreak","How Do the First Days Count? A Case Study of Qatar Experience in Emergency Risk Communication during the MERS-CoV Outbreak","What are the risks of COVID-19 infection in pregnant women?","Network-based Drug Repurposing for Human Coronavirus","The deadly coronaviruses: The 2003 SARS pandemic and the 2020 novel coronavirus epidemic in China","Group-specific structural features of the 5\u2032-proximal sequences of coronavirus genomic RNAs","ACE2 Expression is Increased in the Lungs of Patients with Comorbidities Associated with Severe COVID-19","Understand Research Hotspots Surrounding COVID-19 and Other Coronavirus Infections Using Topic Modeling","Understand Research Hotspots Surrounding COVID-19 and Other Coronavirus Infections Using Topic Modeling","ACE2 Expression is Increased in the Lungs of Patients with Comorbidities Associated with Severe COVID-19","All-in-One Dual CRISPR-Cas12a (AIOD-CRISPR) Assay: A Case for Rapid, Ultrasensitive and Visual Detection of Novel Coronavirus SARS-CoV-2 and HIV virus","All Hands on Deck: Transdisciplinary Approaches to Emerging Infectious Disease","Network-based drug repurposing for novel coronavirus 2019-nCoV/SARS-CoV-2","Genomic and proteomic approaches for studying human cancer: Prospects for true patient-tailored therapy","In silico approach to accelerate the development of mass spectrometry-based proteomics methods for detection of viral proteins: Application to COVID-19","Chapter 15 Molecular Testing in Emerging Infectious Diseases","CHAPTER 4 Keeping Track of Viruses","Pathogen-Tested Planting Material","Genomic and proteomic approaches for studying human cancer: Prospects for true patient-tailored therapy","Computers and viral diseases. Preliminary bioinformatics studies on the design of a synthetic vaccine and a preventative peptidomimetic antagonist against the SARS-CoV-2 (2019-nCoV, COVID-19) coronavirus","Chapter 22 Biological Warfare: Infectious Disease and Bioterrorism","Emerging novel coronavirus (2019-nCoV)\u2014current scenario, evolutionary perspective based on genome analysis and recent developments","Pains and Gains from China's Experiences with Emerging Epidemics: From SARS to H7N9","Middle East respiratory syndrome (MERS) coronavirus and dromedaries","S8 \u2013 S47 and O50 \u2013 O440","Rapid Molecular Detection of SARS-CoV-2 (COVID-19) Virus RNA Using Colorimetric LAMP","Applications of microarrays in pathogen detection and biodefence","Expecting the unexpected: nucleic acid-based diagnosis and discovery of emerging viruses","Integrated Micropillar Polydimethylsiloxane Accurate CRISPR Detection (IMPACT) System for Rapid Viral DNA Sensing","Metagenomics in Virology","7 Lung Infections","S8 \u2013 S47 and O50 \u2013 O440","Preparedness for emerging epidemic threats: a Lancet Infectious Diseases Commission","S8 \u2013 S47 and O50 \u2013 O440","Applications of microarrays in pathogen detection and biodefence","Rapid diagnostics urgently needed for killer infections","P441 \u2013 P678","Chinese experts\u2019 consensus on the Internet of Things-aided diagnosis and treatment of coronavirus disease 2019 (COVID-19)","Elevated thrombopoietin and platelet indices confirm active thrombopoiesis but fail to predict clinical severity of puumala hantavirus infection","P441 \u2013 P678","Analytical and Clinical Validation of Six Commercial Middle East Respiratory Syndrome Coronavirus RNA Detection Kits Based on Real-Time Reverse-Transcription PCR","Advances in respiratory virus therapeutics \u2013 A meeting report from the 6th isirv Antiviral Group conference","Potent and persistent antibody responses against the receptor-binding domain of SARS-CoV spike protein in recovered patients","Prospect and application of Internet of Things technology for prevention of SARIs","Potential Rapid Diagnostics, Vaccine and Therapeutics for 2019 Novel Coronavirus (2019-nCoV): A Systematic Review","New Vaccine Technologies to Combat Outbreak Situations","Oral presentations","Public health round-up"],"url":["https://doi.org/10.1016/s0140-6736(19)33221-0","https://doi.org/10.1016/s0140-6736(20)30365-2","https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5751014/","https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5751014/","https://doi.org/10.1016/s0140-6736(20)30365-2","https://doi.org/10.1101/2020.02.03.20020263","https://doi.org/10.1016/j.jaut.2020.102434","https://doi.org/10.1016/j.virol.2010.02.007","https://doi.org/10.1101/2020.03.21.20040261","https://doi.org/10.1101/2020.03.26.20044164","https://doi.org/10.1101/2020.03.26.20044164","https://doi.org/10.1101/2020.03.21.20040261","https://doi.org/10.1101/2020.03.19.998724","https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7087652/","https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7073332/","https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3525069/","https://doi.org/10.1101/2020.03.08.980383","https://doi.org/10.1016/b978-0-12-800886-7.00015-7","https://doi.org/10.1016/b978-012088483-4/50007-x","https://doi.org/10.1016/b978-0-444-52512-3.00173-x","https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3525069/","https://doi.org/10.1016/j.compbiomed.2020.103670","https://doi.org/10.1016/b978-0-12-385015-7.00022-3","https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7054940/","https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4971293/","https://doi.org/10.1016/j.tvjl.2016.12.020","https://doi.org/10.1111/j.1470-9465.2006.12_4_1426.x","https://doi.org/10.1101/2020.02.26.20028373","https://doi.org/10.1016/j.tibtech.2008.09.004","https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7103685/","https://doi.org/10.1101/2020.03.17.994137","https://doi.org/10.1016/b978-0-12-809633-8.20957-6","https://doi.org/10.1016/b978-0-323-44284-8.00007-7","https://doi.org/10.1111/j.1470-9465.2006.12_4_1426.x","https://doi.org/10.1016/s1473-3099(19)30674-7","https://doi.org/10.1111/j.1470-9465.2006.12_4_1426.x","https://doi.org/10.1016/j.tibtech.2008.09.004","https://doi.org/10.1016/s2213-2600(13)70099-7","https://doi.org/10.1111/j.1470-9465.2006.12_4_1427.x","https://doi.org/10.1016/j.ceh.2020.03.001","https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5207557/","https://doi.org/10.1111/j.1470-9465.2006.12_4_1427.x","https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4940488/","https://doi.org/10.1016/j.antiviral.2019.04.006","https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2988023/","https://doi.org/10.1016/j.ceh.2020.02.001","https://doi.org/10.3390/jcm9030623","https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6156540/","https://doi.org/10.1111/j.1469-0691.2008.02006.x","https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7047037/"],"x":[0.8640297018447771,0.5318692764227305,0.5113364258834464,0.5130105786973932,0.42363609507701305,0.3652895031241683,0.7034305137591086,0.7973487094028436,0.3952533389373239,0.4306494941100863,0.46343393099430213,0.8106331336033704,0.48465174550745277,0.5072684418309846,0.46139723498383156,0.4290283704329818,0.6871489733978324,0.46481774099682654,0.599212942294082,0.8931654825435634,0.2951613413816856,0.4617968738396965,0.5057424463223826,0.8765844008170718,0.4900905865766029,0.8594331189177317,0.4695376356199569,0.4097056589973628,0.46891109443031015,0.4761391908134879,0.47375412348833634,0.5848231193216268,0.1350753995489734,0.5817432311988745,0.5179197391981162,0.285029792848869,0.4853017605851694,0.46442375847540074,0.4507718457916185,0.4582608385978487,0.358736507481679,0.49145219376394533,0.8628532684053972,0.7227371014368541,0.7574482672984162,0.4644181361911335,0.5236772877732152,0.6845581358764699,0.4980010856596869,0.5034827827759368],"y":[0.49098244933987495,0.6625671557120264,0.7330778183142542,0.6553613406710591,0.5152914475580284,0.4792814665931502,0.556418461519075,0.378461005455641,0.19737958000188954,0.8404222099039079,0.8099219903645118,0.24888084156459434,0.3846094544033029,0.6893469842710139,0.4786747506824922,0.6101868293569418,0.2588139636726128,0.415405802645114,0.4698872291376686,0.36320040796055436,0.6698993219236916,0.6299663803600714,0.4283203398357994,0.40954263577893296,0.7766660412247163,0.47967509724598806,0.3993999087029547,0.19276581825388006,0.5911897503978182,0.40853477733228094,0.40662726744597977,0.42998348173281986,0.5934027601480033,0.39100635876173734,0.7264438136013377,0.49614224404245105,0.3790269538231133,0.41449658065758177,0.36723720524170533,0.5994808256914419,0.3519323439626596,0.3614170559943945,0.49362460883750175,0.3990915974906302,0.339214481530971,0.5729026206622526,0.4472187324891997,0.4378971255228298,0.4128152547195836,0.726613621041017]},"selected":{"id":"26191"},"selection_policy":{"id":"26220"}},"id":"26118","type":"ColumnDataSource"},{"attributes":{},"id":"26161","type":"LinearScale"},{"attributes":{"axis":{"id":"26163"},"ticker":null},"id":"26166","type":"Grid"},{"attributes":{"axis":{"id":"26128"},"ticker":null},"id":"26131","type":"Grid"},{"attributes":{},"id":"26219","type":"StringEditor"},{"attributes":{"editor":{"id":"26209"},"field":"title","formatter":{"id":"26208"},"title":"Document title","width":770},"id":"26151","type":"TableColumn"},{"attributes":{},"id":"26220","type":"UnionRenderers"},{"attributes":{"formatter":{"id":"26225"},"ticker":{"id":"26164"}},"id":"26163","type":"LinearAxis"},{"attributes":{},"id":"26126","type":"LinearScale"},{"attributes":{"fill_color":{"field":"color"},"line_color":{"field":"color"},"radius":{"field":"rad","units":"data"},"x":{"field":"x"},"y":{"field":"y"}},"id":"26144","type":"Circle"},{"attributes":{"source":{"id":"26117"}},"id":"26147","type":"CDSView"},{"attributes":{"source":{"id":"26118"}},"id":"26188","type":"CDSView"},{"attributes":{},"id":"26207","type":"StringEditor"},{"attributes":{"children":[{"id":"26187"},{"id":"26154"}]},"id":"26200","type":"Row"},{"attributes":{"data_source":{"id":"26117"},"glyph":{"id":"26144"},"hover_glyph":null,"muted_glyph":null,"nonselection_glyph":{"id":"26145"},"selection_glyph":null,"view":{"id":"26147"}},"id":"26146","type":"GlyphRenderer"},{"attributes":{"text":""},"id":"26202","type":"Title"},{"attributes":{},"id":"26159","type":"LinearScale"},{"attributes":{"columns":[{"id":"26185"},{"id":"26186"}],"height":340,"source":{"id":"26118"},"view":{"id":"26188"},"width":900},"id":"26187","type":"DataTable"},{"attributes":{"active_drag":"auto","active_inspect":"auto","active_multi":null,"active_scroll":"auto","active_tap":"auto","tools":[{"id":"26136"},{"id":"26137"},{"id":"26138"}]},"id":"26139","type":"Toolbar"},{"attributes":{},"id":"26215","type":"BasicTickFormatter"},{"attributes":{"formatter":{"id":"26215"},"ticker":{"id":"26129"}},"id":"26128","type":"LinearAxis"},{"attributes":{},"id":"26172","type":"ResetTool"},{"attributes":{"text":"&lt;h3&gt;Paragraph ranking&lt;/h3&gt;"},"id":"26195","type":"Div"},{"attributes":{"source":{"id":"26117"}},"id":"26153","type":"CDSView"},{"attributes":{},"id":"26155","type":"Range1d"},{"attributes":{"callback":null,"tooltips":[["Title","@title"],["Abstract","@text"],["Published","@date"],["Authors","@authors"],["Journal","@journal"],["URL","@url"]]},"id":"26138","type":"HoverTool"},{"attributes":{"below":[{"id":"26163"}],"center":[{"id":"26166"},{"id":"26170"}],"left":[{"id":"26167"}],"plot_height":340,"plot_width":900,"renderers":[{"id":"26181"}],"title":{"id":"26204"},"toolbar":{"id":"26174"},"toolbar_location":"below","x_range":{"id":"26155"},"x_scale":{"id":"26159"},"y_range":{"id":"26157"},"y_scale":{"id":"26161"}},"id":"26154","subtype":"Figure","type":"Plot"},{"attributes":{},"id":"26209","type":"StringEditor"},{"attributes":{},"id":"26218","type":"StringFormatter"},{"attributes":{"args":{"sd":{"id":"26117"},"sp":{"id":"26118"}},"code":"\n    if (sp.selected.indices.length == 0) {\n      return;\n    }\n    var sel = sp.selected.indices[0];\n    var new_selected = [];\n    for (var i = 0; i &lt; sd.data['doc'].length; i++) {\n      if (sd.data['doc'][i] == sp.data['doc'][sel]) {\n        new_selected.push(i);\n      }\n    }\n    if (JSON.stringify(new_selected) != JSON.stringify(sd.selected.indices) &amp;&amp;\n    !(sd.selected.indices.length == 1 &amp;&amp; new_selected.includes(sd.selected.indices[0]))) {\n      sd.selected.indices = new_selected;\n      sd.change.emit();\n    }\n    "},"id":"26192","type":"CustomJS"},{"attributes":{},"id":"26216","type":"StringFormatter"},{"attributes":{"data":{"authors":["Jacob Kames; David Dillon Holcomb; Ofer Kimchi; Michael DiCuccio; Nobuko Hamasaki-Katagiri; Tony Wang; Anton A Komar; Aikaterini Alexaki; Chava Kimchi-Sarfaty","Ward, Kenneth","Tieying Hou; Weiqi Zeng; Minling Yang; Wenjing Chen; Lili Ren; Jingwen Ai; Ji Wu; Yalong Liao; Xuejing Gou; Yongjun Li; Xiaorui Wang; Hang Su; Jianwei Wang; Bing Gu; Teng Xu","HIETPAS, JACK; MCMULLAN, LAURA K.; HANSON, HOLLY L.; RICE, CHARLES M.; MINDELL, DAVID P.","Uzoma, Ijeoma; Zhu, Heng","Rajakaruna, Sanjeewa Jayachandra; Liu, Wen-Bin; Ding, Yi-Bo; Cao, Guang-Wen","Ganova-Raeva, Lilia M; Khudyakov, Yury E","Kiselev, Daniel; Matsvay, Alina; Abramov, Ivan; Dedkov, Vladimir; Shipulin, German; Khafizov, Kamil","Schildgen, O.; Schildgen, V.","Dong, Jianli; Olano, Juan P.; McBride, Jere W.; Walker, David H.","Song, Yuanlin; Jiang, Jinjun; Wang, Xun; Yang, Dawei; Bai, Chunxue","Calton, Brook; Abedini, Nauzley; Fratkin, Michael","Phan, Tung","Domachowske, Joseph B.","Timokratis Karamitros; Gethsimani Papadopoulou; Maria Bousali; Anastasios Mexias; Sotiris Tsiodras; Andreas Mentis","Dahlhausen, Bob","Cornelius Christian; Francis Christian","Dong, J.; Ismail, N.; Walker, D.H.","Pan, Xingchen; Ojcius, David M.; Gao, Tianyue; Li, Zhongsheng; Pan, Chunhua; Pan, Chungen","Pedersen, Niels C.","Nur, Suza Mohammad; Hasan, Md. Anayet; Amin, Mohammad Al; Hossain, Mehjabeen; Sharmin, Tahmina","Gilbert, Gwendolyn L.; Degeling, Chris; Johnson, Jane","Poon, Leo L.M.; Chan, Kwok Hung; Wong, On Kei; Yam, Wing Cheong; Yuen, Kwok Yung; Guan, Yi; Lo, Y.M.Dennis; Peiris, Joseph S.M.","Mengying Dong; Xiaojun Cao; Mingbiao Liang; Lijuan Li; Huiying Liang; Guangjian Liu","Pang, Junxiong; Wang, Min Xian; Ang, Ian Yi Han; Tan, Sharon Hui Xuan; Lewis, Ruth Frances; Chen, Jacinta I. Pei; Gutierrez, Ramona A.; Gwee, Sylvia Xiao Wei; Chua, Pearleen Ee Yong; Yang, Qian; Ng, Xian Yi; Yap, Rowena K. S.; Tan, Hao Yi; Teo, Yik Ying; Tan, Chorh Chuan; Cook, Alex R.; Yap, Jason Chin-Huat; Hsu, Li Yang","Singer, Donald R.J.; Redekop, Ken","Lin, Chun-Yu; Cheng, Chih-Hung; Lu, Po-Liang; Shih, Deng-Chiung; Hung, Ching-Tzu; Lo, Hsin-Hui; Tsai, Ming-Ju; Hung, Jen-Yu","Tian, Sijia; Hu, Nan; Lou, Jing; Chen, Kun; Kang, Xuqin; Xiang, Zhenjun; Chen, Hui; Wang, Dali; Liu, Ning; Liu, Dong; Chen, Gang; Zhang, Yongliang; Li, Dou; Li, Jianren; Lian, Huixin; Niu, Shengmei; Zhang, Luxi; Zhang, Jinjun","Zhou, Fei; Yu, Ting; Du, Ronghui; Fan, Guohui; Liu, Ying; Liu, Zhibo; Xiang, Jie; Wang, Yeming; Song, Bin; Gu, Xiaoying; Guan, Lulu; Wei, Yuan; Li, Hui; Wu, Xudong; Xu, Jiuyang; Tu, Shengjin; Zhang, Yi; Chen, Hua; Cao, Bin","Huang, Chaolin; Wang, Yeming; Li, Xingwang; Ren, Lili; Zhao, Jianping; Hu, Yi; Zhang, Li; Fan, Guohui; Xu, Jiuyang; Gu, Xiaoying; Cheng, Zhenshun; Yu, Ting; Xia, Jiaan; Wei, Yuan; Wu, Wenjuan; Xie, Xuelei; Yin, Wen; Li, Hui; Liu, Min; Xiao, Yan; Gao, Hong; Guo, Li; Xie, Jungang; Wang, Guangfa; Jiang, Rongmeng; Gao, Zhancheng; Jin, Qi; Wang, Jianwei; Cao, Bin","Zumla, Alimuddin; Gant, Vanya; Bates, Matthew; Mwaba, Peter; Maeurer, Markus; Memish, Ziad A","Mackay, Ian M.; Arden, Katherine E.","Saglio, Francesco; Hanley, Patrick J.; Bollard, Catherine M.","Humphries, Linda","Zhang, Jun; Wu, Weili; Zhao, Xin; Zhang, Wei","Rabelo, Luiza A.; Nunes-Souza, Val\u00e9ria; Bader, Michael","Duggan, Jacqueline M.; Brooks, Timothy J.G.","Holmes, Kathryn V.; Dominguez, Samuel R.","Konrad, Regina; Eberle, Ute; Dangel, Alexandra; Treis, Bianca; Berger, Anja; Bengs, Katja; Fingerle, Volker; Liebl, Bernhard; Ackermann, Nikolaus; Sing, Andreas","Yang, Ning; Ma, Ping; Lang, Jianshe; Zhang, Yanli; Deng, Jiejie; Ju, Xiangwu; Zhang, Gongyi; Jiang, Chengyu","Grigoriev, Andrei","Lau, Susanna K. P.; Chan, Jasper F. W.","Anna S Fomsgaard; Maiken W Rosenstierne","Corman, Victor M; Landt, Olfert; Kaiser, Marco; Molenkamp, Richard; Meijer, Adam; Chu, Daniel KW; Bleicker, Tobias; Br\u00fcnink, Sebastian; Schneider, Julia; Schmidt, Marie Luisa; Mulders, Daphne GJC; Haagmans, Bart L; van der Veer, Bas; van den Brink, Sharon; Wijsman, Lisa; Goderski, Gabriel; Romette, Jean-Louis; Ellis, Joanna; Zambon, Maria; Peiris, Malik; Goossens, Herman; Reusken, Chantal; Koopmans, Marion PG; Drosten, Christian","Carr, Kristen M; Rosenblatt, Kevin; Petricoin, Emanuel F; Liotta, Lance A","Zhang, Yuanjiang; Li, Tieshi; Fu, Ling; Yu, Changming; Li, Yinghua; Xu, Xialian; Wang, Yinyin; Ning, Hongxiu; Zhang, Shuping; Chen, Wei; Babiuk, Lorne A; Chang, Zhijie","Goldenberg, Simon","Uttamchandani, Mahesh; Neo, Jia Ling; Ong, Brandon Ngiap Zhung; Moochhala, Shabbir","McLoughlin, Kevin S.","Gardy, Jennifer L.; Loman, Nicholas J."],"color":["#67a9cf","#67a9cf","#67a9cf","#67a9cf","#67a9cf","#67a9cf","#67a9cf","#67a9cf","#67a9cf","#67a9cf","#67a9cf","#67a9cf","#67a9cf","#67a9cf","#67a9cf","#67a9cf","#67a9cf","#67a9cf","#67a9cf","#67a9cf","#67a9cf","#67a9cf","#67a9cf","#67a9cf","#67a9cf","#67a9cf","#67a9cf","#67a9cf","#67a9cf","#67a9cf","#67a9cf","#67a9cf","#67a9cf","#67a9cf","#67a9cf","#67a9cf","#67a9cf","#67a9cf","#67a9cf","#67a9cf","#67a9cf","#67a9cf","#67a9cf","#67a9cf","#67a9cf","#67a9cf","#67a9cf","#67a9cf","#67a9cf","#67a9cf"],"date":["2020-03-31","2006-08-31","2020-02-23","2005-12-31","2013-01-17","2017-10-27","2013-05-01","2020-02-14","2016-12-31","2008-05-31","2020-12-31","2020-03-31","2020-04-30","2004-10-15","2020-03-28","2010-04-30","2020-03-27","2017-12-31","2020-03-31","2014-08-31","2015-07-30","2019-06-10","2003-12-31","2020-03-30","2020","2020-03-31","2020-03-31","2020-02-27","2020-03-11","2020-02-21","2013-06-30","2015-12-22","2014-02-28","2005-11-30","2020","2015-12-31","2005-03-18","2013-01-08","2020-03-05","2012-01-17","2004-03-31","2015-12-22","2020-03-30","2020-01-23","2004-01-01","2004-02-27","2013-11-30","2009-01-31","2011-09-19","2017-11-13"],"doc":["2fn25l6m","jia6dqld","hth8f5sn","qma1lsuw","t96j8qt0","hiw9unh3","qh2kuzdc","u2n8so5w","mg7k1pt8","ijdl4toy","sqyhzxty","91weyqmm","s9mqv4lo","4dqz8req","jxm69ndw","o976nab9","ieobv7q8","1h87wldg","d13bq4ij","j3yb3k57","86ltgee4","o1fa8dfw","86el3qwn","3wuh6k6g","8gncbgot","4bt6hq3x","qi7l4beq","k3f7ohzg","k36rymkv","0nhgxoim","vpq4yghn","mwj2qby4","3rbjwjxu","gfcmn5it","030bc0h3","hrgtaunt","kzj263xj","krvei97r","zlhu1jit","1i12ff55","g68g6xxu","zhxrlopp","vou46eie","r16y83dm","aza0pzud","3me20gsi","n8hlrknq","vgizgq2y","9ntgqyhw","th1da1bb"],"journal":["","American Journal of Obstetrics and Gynecology","","Microbial Forensics","Genomics Proteomics Bioinformatics","Mil Med Res","Expert Review of Molecular Diagnostics","Viruses","The Microbiology of Respiratory System Infections","The Journal of Molecular Diagnostics","Clinical eHealth","Journal of Pain and Symptom Management","Infection, Genetics and Evolution","Clinical Microbiology Newsletter","","Journal of Exotic Pet Medicine","","Diagnostic Molecular Pathology","Microbes and Infection","The Veterinary Journal","Interdiscip Sci","Asian Bioeth Rev","Journal of Clinical Virology","","Journal of Clinical Medicine","Health Policy and Technology","Journal of Hospital Infection","Journal of Infection","The Lancet","The Lancet","The Lancet Respiratory Medicine","Virol J","Cytotherapy","AORN Journal","Precision Clinical Medicine","The Protective Arm of the Renin Angiotensin System (RAS)","Vaccine","mBio","Euro Surveill","Journal of Biological Chemistry","Trends in Genetics","Virol J","","Euro Surveill","Hum Genomics","FEBS Letters","Medicine","Trends in Biotechnology","Brief Funct Genomics","Nat Rev Genet"],"rad":[0.02,0.010224623307738687,0.00734914571044366,0.0069306082931390196,0.005850526984437539,0.005707915917387249,0.005481527795387478,0.005306015427796495,0.005287827511440014,0.005283288940226428,0.005161431777654827,0.004912048262560478,0.004871852817990947,0.004795591391277013,0.004778657760482973,0.004626737434807052,0.0045950426198843935,0.004556045157882283,0.004501522334737785,0.004490548482468135,0.004335299594002239,0.004327114656443245,0.004323026624745895,0.0039842366781383445,0.003928978714771959,0.0037982426985286514,0.003793950217737007,0.0037489239283906815,0.0037158868090998703,0.0036359791957590254,0.0035217253472523253,0.003480841970804253,0.0034251384857737254,0.0034211378679659425,0.003267476879778079,0.003251972991793406,0.0032249537254687077,0.0031904235469718874,0.003157629276493504,0.003156126156195285,0.0031420320472330954,0.0031342514086642546,0.003114852021281671,0.00308073139629725,0.0030620575550238107,0.0030471682973652655,0.0030418078262470444,0.0030418078262470444,0.0030037862984641874,0.003],"scr":[1.0,0.425,0.256,0.231,0.168,0.159,0.146,0.136,0.135,0.134,0.127,0.112,0.11,0.106,0.105,0.096,0.094,0.092,0.088,0.088,0.079,0.078,0.078,0.058,0.055,0.047,0.047,0.044,0.042,0.037,0.031,0.028,0.025,0.025,0.016,0.015,0.013,0.011,0.009,0.009,0.008,0.008,0.007,0.005,0.004,0.003,0.002,0.002,0.0,0.0],"text":["As the SARS-CoV-2 pandemic is rapidly progressing, the need for the development of an effective vaccine is critical. A promising approach for vaccine development is to generate, through codon pair deoptimization, an attenuated virus. This approach carries the advantage that it only requires limited knowledge specific to the virus in question, other than its genome sequence. Therefore, it is well suited for emerging viruses for which we may not have extensive data. We performed comprehensive in silico analyses of several features of SARS-CoV-2 genomic sequence (e.g., codon usage, codon pair usage, dinucleotide/junction dinucleotide usage, RNA structure around the frameshift region) in comparison with other members of the coronaviridae family of viruses, the overall human genome, and the transcriptome of specific human tissues such as lung, which are primarily targeted by the virus. Our analysis identified the spike (S) and nucleocapsid (N) proteins as promising targets for deoptimizatio...","Microarrays can be constructed with dozens to millions of probes on their surface to allow high-throughput analyses of many biologic processes to be performed simultaneously on the same sample. Microarrays are now widely used for gene expression analysis, deoxyribonucleic acid resequencing, single-nucleotide polymorphism genotyping, and comparative genomic hybridization. Microarray technology is accelerating research in many fields and now microarrays are moving into clinical application. This review discusses the emerging role of microarrays in molecular diagnostics, pathogen detection, oncology, and pharmacogenomics.","Background: The recent outbreak of infections by the 2019 novel coronavirus (2019-nCoV), the third zoonotic CoV has raised great public health concern. The demand for rapid and accurate diagnosis of this novel pathogen brought significant clinical and technological challenges. Currently, metagenomic next-generation sequencing (mNGS) and reverse-transcription PCR (RT-PCR) are the most widely used molecular diagnostics. Methods: 2019-nCoV infections were confirmed in 52 specimens by mNGS. Genomic information was analyzed and used for the design and development of an isothermal, CRISPR-based diagnostic for this novel virus. The diagnostic performance of CRISPR-nCoV was assessed and compared across three technology platforms (mNGS, RT-PCR and CRISPR). Results: 2019-nCoVs sequenced in our study were conserved with the Wuhan strain, and shared certain genetic similarity with SARS-CoV. A high degree of variation in the level of viral RNA was observed in clinical specimens. CRISPR-nCoV demonst...","Publisher Summary This chapter introduces basic concepts of virology and exemplifies the usage of current technology for identifying and tracking viral pathogens. Virus structure, lifestyle, replication strategies, classication, the varied modes of transmission, infection, and disease manifestations are focussed upon in the chapter. Viruses are extremely simple \u201clife\u201d forms without metabolic capacity, organelles, translational machinery, or autonomous replicative potential; the essential elements of a virus are consequently minimal. A virus enters the host through any opportunistic opening of the body\u2014; such as minute breaks in the skin barrier or mucosal linings of the eye, nose, etc. Antibodies produced during an infection often have the ability to bind the virus and reduce infectivity. An immediate question to solve during a virus outbreak is the mechanism by which the virus spreads. Viruses frequently infect animal or insect vectors that serve to pass the virus to humans. By identi...","A major focus of systems biology is to characterize interactions between cellular components, in order to develop an accurate picture of the intricate networks within biological systems. Over the past decade, protein microarrays have greatly contributed to advances in proteomics and are becoming an important platform for systems biology. Protein microarrays are highly flexible, ranging from large-scale proteome microarrays to smaller customizable microarrays, making the technology amenable for detection of a broad spectrum of biochemical properties of proteins. In this article, we will focus on the numerous studies that have utilized protein microarrays to reconstruct biological networks including protein\u2013DNA interactions, posttranslational protein modifications (PTMs), lectin\u2013glycan recognition, pathogen\u2013host interactions and hierarchical signaling cascades. The diversity in applications allows for integration of interaction data from numerous molecular classes and cellular states, pr...","Hospital-acquired infections (HAIs) are serious problems for healthcare systems, especially in developing countries where public health infrastructure and technology for infection preventions remain undeveloped. Here, we characterized how strategy and technology could be mobilized to improve the effectiveness of infection prevention and control in hospitals during the outbreaks of Ebola, Middle East respiratory syndrome (MERS), and severe acute respiratory syndrome (SARS) in Asia and West Africa. Published literature on the hospital-borne outbreaks of SARS, Ebola, and MERS in Asia and West Africa was comprehensively reviewed. The results showed that healthcare systems and hospital management in affected healthcare facilities had poor strategies and inadequate technologies and human resources for the prevention and control of HAIs, which led to increased morbidity, mortality, and unnecessary costs. We recommend that governments worldwide enforce disaster risk management, even when no ou...","Mass spectrometry (MS) has found numerous applications in life sciences. It has high accuracy, sensitivity and wide dynamic range in addition to medium- to high-throughput capabilities. These features make MS a superior platform for analysis of various biomolecules including proteins, lipids, nucleic acids and carbohydrates. Until recently, MS was applied for protein detection and characterization. During the last decade, however, MS has successfully been used for molecular diagnostics of microbial and viral infections with the most notable applications being identification of pathogens, genomic sequencing, mutation detection, DNA methylation analysis, tracking of transmissions, and characterization of genetic heterogeneity. These new developments vastly expand the MS application from experimental research to public health and clinical fields. Matching of molecular techniques with specific requirements of the major MS platforms has produced powerful technologies for molecular diagnosti...","Viruses are evolving at an alarming rate, spreading and inconspicuously adapting to cutting-edge therapies. Therefore, the search for rapid, informative and reliable diagnostic methods is becoming urgent as ever. Conventional clinical tests (PCR, serology, etc.) are being continually optimized, yet provide very limited data. Could high throughput sequencing (HTS) become the future gold standard in molecular diagnostics of viral infections? Compared to conventional clinical tests, HTS is universal and more precise at profiling pathogens. Nevertheless, it has not yet been widely accepted as a diagnostic tool, owing primarily to its high cost and the complexity of sample preparation and data analysis. Those obstacles must be tackled to integrate HTS into daily clinical practice. For this, three objectives are to be achieved: (1) designing and assessing universal protocols for library preparation, (2) assembling purpose-specific pipelines, and (3) building computational infrastructure to s...","Abstract The human bocavirus is one of the most common respiratory viruses and occurs in all age groups. It is associated with upper and lower respiratory tract infections, and causes clinical symptoms from the mild common cold to life threatening respiratory diseases. Besides its ability to persist the virus appears to trigger chronic lung disease and increase the clinical symptoms, while being a putative trigger for fibrotic lung diseases. Laboratory diagnostics should include serological diagnostics in order to rule out a viremia because due to prolonged viral shedding acute and chronic infections cannot be differentiated on the detection of viral nucleic acids in respiratory specimen alone. Although Koch\u2019s postulates cannot be formally fulfilled due to the lack of an animal model and the chance for clinical trials with volunteers are limited due to the long term effects of HBoV infections, there is no doubt that the virus is a serious pathogen and requires attention. The aim of the...","More than 50 emerging and reemerging pathogens have been identified during the last 40 years. Until 1992 when the Institute of Medicine issued a report that defined emerging infectious diseases, medicine had been complacent about such infectious diseases despite the alarm bells of infections with human immunodeficiency virus. Molecular tools have proven useful in discovering and characterizing emerging viruses and bacteria such as Sin Nombre virus (hantaviral pulmonary syndrome), hepatitis C virus, Bartonella henselae (cat scratch disease, bacillary angiomatosis), and Anaplasma phagocytophilum (human granulocytotropic anaplasmosis). The feasibility of applying molecular diagnostics to dangerous, fastidious, and uncultivated agents for which conventional tests do not yield timely diagnoses has achieved proof of concept for many agents, but widespread use of cost-effective, validated commercial assays has yet to occur. This review presents representative emerging viral respiratory infect...","Abstract The Internet of Things (IoT) includes three core procedures: full spectrum perception, reliable transmission, and intelligent processing. It can be applied for the prevention and control of SARI (severe acute respiratory infection). By combining sensors, information technology, artificial intelligence, and available dynamic networking devices, IoT could realize long-distance communication between hospitals, patients, and medical devices, which could ultimately improve current medical conditions.","ABSTRACT Within weeks, COVID-19 has transformed our practice of palliative care and clinical medicine as we know it. Telemedicine has emerged as a critical technology to bring medical care to patients while attempting to reduce the transmission of COVID-19 among patients, families, and clinicians. It is also increasingly necessary to preserve scarce resources like personal protective equipment. In this article, we share just-in-time tips to support palliative care clinicians and program leaders in providing the best care possible by telemedicine. These quick, practical tips cover telemedicine set-up, patient considerations, and clinician considerations. Next steps include ensuring equitable access to affordable telemedicine technology for vulnerable populations through creative solutions and financing, and dedicated attention to telemedicine evaluation and quality improvement.","Abstract A novel coronavirus designated as 2019-nCoV first appeared in Wuhan, China in late December 2019. Dozens of people died in China, and thousands of people infected as 2019-nCoV continues to spread around the world. We have described the discovery, emergence, genomic characteristics, and clinical diagnostics of 2019-nCoV.","Abstract Advances in molecular biology, bioinformatics, and robotics have allowed microarray technology to be used for in-depth, basic science studies in all fields of microbiology. Recently, translation of these basic science applications to clinical microbiology and infectious diseases has also progressed. From a clinical infectious disease perspective, genome-based organism identification, pathogen discovery, and antimicrobial susceptibility testing of problematic organisms offer the potential to yield diagnostic information that may not otherwise become available. Moreover, microarray-based studies have the ability to provide \u201csignatures\u201d of host cell transcriptional responses for individual pathogens and/or groups of pathogens. This type of information has the potential to confirm difficult diagnoses, to monitor responses to therapeutic intervention, or even to predict prognosis and sequelae following an infectious disease. Examples are presented to illustrate ways in which microa...","In December 2019, an outbreak of atypical pneumonia (Coronavirus disease 2019 -COVID-19) associated with a novel coronavirus (SARS-CoV-2) was reported in Wuhan city, Hubei province, China. The outbreak was traced to a seafood wholesale market and human to human transmission was confirmed. The rapid spread and the death toll of the new epidemic warrants immediate intervention. The intra-host genomic variability of SARS-CoV-2 plays a pivotal role in the development of effective antiviral agents and vaccines, but also in the design of accurate diagnostics. We analyzed NGS data derived from clinical samples of three Chinese patients infected with SARS-CoV-2, in order to identify small- and large-scale intra-host variations in the viral genome. We identified tens of low- or higher-frequency single nucleotide variations (SNVs) with variable density across the viral genome, affecting 7 out of 10 protein-coding viral genes. The majority of these SNVs corresponded to missense changes. The annot...","Abstract The development of rapid, accurate, and sensitive diagnostic methods for detecting pathogens is the basis for treating, controlling, and eradicating infectious diseases of veterinary importance. Scientific and technological advancements have revolutionized the field of veterinary diagnostics. Genome sequencing has allowed efficient, sensitive, and specific diagnostic assays to be developed based on the detection of nucleic acids. The integration of advances in biochemistry, proteomics, engineering, and medicine offers enormous potential for the rapid and accurate diagnosis of viral, microbial, genetic, and metabolic disease. In the future, polymerase chain reaction assays, microarray testing, genomic analysis, and metabolic profiling will be accomplished in a rapid, portable, sensitive, and cost-efficient manner.","COVID-19 has spread with unequal efficiency in various parts of the world. In several European countries including Italy, the increase in the number of COVID-19 cases has followed a consistent, exponential pattern of spread. However, some countries, notably Taiwan and Hong Kong, have achieved a different outcome and have managed to bring the COVID-19 outbreak in their countries rapidly under control, without entering the exponential pattern and with very few cases. They have used several different approaches to COVID-19 outbreak control, including the innovative use of smartphone technology and the widespread use of surgical face masks. We show through our models, that Canada has followed the same, consistent COVID-19 exponential growth pattern that is seen in Italy. Both nationally and in its most heavily affected provinces, there is exponential growth of COVID-19 cases, making it possible to make predictions for the future, if no further interventions are made in public health policy...","Abstract It was widely believed in the late 1960s that infectious diseases had been conquered by vaccines and antibiotics and humans were no longer under threat by microbial pathogens. Yet, since that time more than 60 pathogens have been discovered that can cause serious emerging infectious diseases. Molecular methods have played critical roles in the discovery, monitoring, and clinical diagnostics of emerging pathogens. In this chapter, we present well-recognized emerging pathogens. We provide examples of the utility of molecular assays in research and clinical care of emerging infectious diseases. We also discuss some theoretical and practical limitations of molecular tests and the future prospects of expanding molecular diagnostics for emerging pathogens based on new advances of knowledge and technologies.","Abstract Only a month after the outbreak of pneumonia caused by 2019-nCoV, more than forty-thousand people were infected. This put enormous pressure on the Chinese government, medical healthcare provider, and the general public, but also made the international community deeply nervous. On the 25th day after the outbreak, the Chinese government implemented strict traffic restrictions on the area where the 2019-nCoV had originated\u2014Hubei province, whose capital city is Wuhan. Ten days later, the rate of increase of cases in Hubei showed a significant difference (p = 0.0001) compared with the total rate of increase in other provinces of China. These preliminary data suggest the effectiveness of a traffic restriction policy for this pandemic thus far. At the same time, solid financial support and improved research ability, along with network communication technology, also greatly facilitated the application of epidemic prevention measures. These measures were motivated by the need to provid...","Abstract This review is concerned with what has been learned about feline infectious peritonitis (FIP) diagnostics and therapeutics since the publication of an extensive overview of literature covering the period 1963\u20132009. Although progress has been made in both areas, obtaining a definitive diagnosis of FIP remains a problem for those veterinarians and/or cat owners who require absolute certainty. This review will cover both indirect and direct diagnostic tests for the disease and will emphasize their limitations, as well as their specificity and sensitivity. There is still no effective treatment for FIP, although there are both claims that such therapies exist and glimmers of hope coming from new therapies that are under research. FIP has also been identified in wild felids and FIP-like disease is now a growing problem among pet ferrets.","The Middle East respiratory syndrome coronavirus (MERS-CoV) is a virus that manifests itself in viral infection with fever, cough, shortness of breath, renal failure and severe acute pneumonia, which often result in a fatal outcome. MERS-CoV has been shown to spread between people who are in close contact. Transmission from infected patients to healthcare personnel has also been observed and is irredeemable with present technology. Genetic studies on MERS-CoV have shown that ORF1ab encodes replicase polyproteins and play a foremost role in viral infection. Therefore, ORF1ab replicase polyprotein may be used as a suitable target for disease control. Viral activity can be controlled by RNA interference (RNAi) technology, a leading method for post transcriptional gene silencing in a sequence-specific manner. However, there is a genetic inconsistency in different viral isolates; it is a great challenge to design potential RNAi (miRNA and siRNA) molecules which can silence the respective ta...","Surveillance is essential for communicable disease prevention and control. Traditional notification of demographic and clinical information, about individuals with selected (notifiable) infectious diseases, allows appropriate public health action and is protected by public health and privacy legislation, but is slow and insensitive. Big data\u2013based electronic surveillance, by commercial bodies and government agencies (for profit or population control), which draws on a plethora of internet- and mobile device\u2013based sources, has been widely accepted, if not universally welcomed. Similar anonymous digital sources also contain syndromic information, which can be analysed, using customised algorithms, to rapidly predict infectious disease outbreaks, but the data are nonspecific and predictions sometimes misleading. However, public health authorities could use these online sources, in combination with de-identified personal health data, to provide more accurate and earlier warning of infectio...","Abstract Background: A novel coronavirus was recently identified as the aetiological agent of Severe Acute Respiratory Syndrome (SARS). Molecular assays currently available for detection of SARS-coronavirus (SARS-Cov) have low sensitivity during the early stage of the illness. Objective: To develop and evaluate a sensitive diagnostic test for SARS by optimizing the viral RNA extraction methods and by applying real-time quantitative RT-PCR technology. Study design: 50 nasopharyngeal aspirate (NPA) samples collected from days 1\u20133 of disease onset from SARS patients in whom SARS CoV infections was subsequently serologically confirmed and 30 negative control samples were studied. Samples were tested by: (1) our first generation conventional RT-PCR assay with a routine RNA extraction method (Lancet 361 (2003) 1319), (2) our first generation conventional RT-PCR assay with a modified RNA extraction method, (3) a real-time quantitative RT-PCR assay with a modified RNA extraction method. Result...","Background: Severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) is a virus that causes severe respiratory illness in humans, which eventually results in the current outbreak of novel coronavirus disease (COVID-19) around the world. The research community is interested to know what are the hotspots in coronavirus (CoV) research and how much is known about COVID-19. This study aimed to evaluate the characteristics of publications involving coronaviruses as well as COVID-19 by using a topic modeling analysis. Methods: We extracted all abstracts and retained the most informative words from the COVID-19 Open Research Dataset, which contains all the 35,092 pieces of coronavirus related literature published up to March 20, 2020. Using Latent Dirichlet Allocation modeling, we trained an eight-topic model from the corpus. We then analyzed the semantic relationships between topics and compared the topic distribution between COVID-19 and other CoV infections. Results: Eight topics emerge...","Rapid diagnostics, vaccines and therapeutics are important interventions for the management of the 2019 novel coronavirus (2019-nCoV) outbreak. It is timely to systematically review the potential of these interventions, including those for Middle East respiratory syndrome-Coronavirus (MERS-CoV) and severe acute respiratory syndrome (SARS)-CoV, to guide policymakers globally on their prioritization of resources for research and development. A systematic search was carried out in three major electronic databases (PubMed, Embase and Cochrane Library) to identify published studies in accordance with the Preferred Reporting Items for Systematic Reviews and Meta-Analyses (PRISMA) guidelines. Supplementary strategies through Google Search and personal communications were used. A total of 27 studies fulfilled the criteria for review. Several laboratory protocols for confirmation of suspected 2019-nCoV cases using real-time reverse transcription polymerase chain reaction (RT-PCR) have been publ...","","","Abstract Background Since the first case of a novel coronavirus (COVID-19) infection pneumonia was detected in Wuhan, China, a series of confirmed cases of the COVID-19 were found in Beijing. We analyzed the data of 262 confirmed cases to determine the clinical and epidemiological characteristics of COVID-19 in Beijing. Methods We collected patients who were transferred by Beijing Emergency Medical Service to the designated hospitals. The information on demographic, epidemiological, clinical, laboratory test for the COVID-19 virus, diagnostic classification, cluster case and outcome were obtained. Furthermore we compared the characteristics between severe and common confirmed cases which including mild cases, no-pneumonia cases and asymptomatic cases, and we also compared the features between COVID-19 and 2003 SARS. Findings By Feb 10, 2020, 262 patients were transferred from the hospitals across Beijing to the designated hospitals for special treatment of the COVID-19 infected by Beij...","Summary Background Since December, 2019, Wuhan, China, has experienced an outbreak of coronavirus disease 2019 (COVID-19), caused by the severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2). Epidemiological and clinical characteristics of patients with COVID-19 have been reported but risk factors for mortality and a detailed clinical course of illness, including viral shedding, have not been well described. Methods In this retrospective, multicentre cohort study, we included all adult inpatients (\u226518 years old) with laboratory-confirmed COVID-19 from Jinyintan Hospital and Wuhan Pulmonary Hospital (Wuhan, China) who had been discharged or had died by Jan 31, 2020. Demographic, clinical, treatment, and laboratory data, including serial samples for viral RNA detection, were extracted from electronic medical records and compared between survivors and non-survivors. We used univariable and multivariable logistic regression methods to explore the risk factors associated with in-hosp...","Summary Background A recent cluster of pneumonia cases in Wuhan, China, was caused by a novel betacoronavirus, the 2019 novel coronavirus (2019-nCoV). We report the epidemiological, clinical, laboratory, and radiological characteristics and treatment and clinical outcomes of these patients. Methods All patients with suspected 2019-nCoV were admitted to a designated hospital in Wuhan. We prospectively collected and analysed data on patients with laboratory-confirmed 2019-nCoV infection by real-time RT-PCR and next-generation sequencing. Data were obtained with standardised data collection forms shared by WHO and the International Severe Acute Respiratory and Emerging Infection Consortium from electronic medical records. Researchers also directly communicated with patients or their families to ascertain epidemiological and symptom data. Outcomes were also compared between patients who had been admitted to the intensive care unit (ICU) and those who had not. Findings By Jan 2, 2020, 41 ad...","","The first known cases of Middle East respiratory syndrome (MERS), associated with infection by a novel coronavirus (CoV), occurred in 2012 in Jordan but were reported retrospectively. The case first to be publicly reported was from Jeddah, in the Kingdom of Saudi Arabia (KSA). Since then, MERS-CoV sequences have been found in a bat and in many dromedary camels (DC). MERS-CoV is enzootic in DC across the Arabian Peninsula and in parts of Africa, causing mild upper respiratory tract illness in its camel reservoir and sporadic, but relatively rare human infections. Precisely how virus transmits to humans remains unknown but close and lengthy exposure appears to be a requirement. The KSA is the focal point of MERS, with the majority of human cases. In humans, MERS is mostly known as a lower respiratory tract (LRT) disease involving fever, cough, breathing difficulties and pneumonia that may progress to acute respiratory distress syndrome, multiorgan failure and death in 20 % to 40 % of tho...","Abstract Adoptive immunotherapy\u2014in particular, T-cell therapy\u2014has recently emerged as a useful strategy with the potential to overcome many of the limitations of antiviral drugs for the treatment of viral complications after hematopietic stem cell transplantation. In this review, we briefly summarize the current methods for virus-specific T-cell isolation or selection and we report results from clinical trials that have used these techniques, focusing specifically on the strategies aimed to broaden the application of this technology.","ABSTRACT \u2022 ADVANCED TECHNOLOGY and improved surgical techniques have led to new therapeutic uses for allografts. \u2022 DISEASE TRANSMISSION via allograft tissue transplants has prompted federal intervention in the tissue banking industry and resulted in federal regulations. \u2022 NEW STANDARDS from the Joint Commission on Accreditation of Healthcare Organizations became effective July 1,2005, and apply to all hospitals that store or implant allograft tissues. These standards include mandatory policies on all aspects of hospital transplantation programs, including tissue ordering, receipt, storage, issuance, and record keeping. AORN J 82 (November 2005) 806\u2013814.","The novel coronavirus pneumonia (COVID-19)epidemic has brought serious social psychological impact to the Chinese people, especially those quarantined and thus with limited access to face-to-face communication and traditional social psychological interventions. To better deal with the urgent psychological problems of people involved in the COVID-19 epidemic, we developed a new psychological crisis intervention model by utilizing internet technology. This new model, one of West China Hospital, integrates physicians, psychiatrists, psychologists and social workers into Internet platforms to carry out psychological intervention to patients, their families and medical staff. We hope this model will make a sound basis for developing a more comprehensive psychological crisis intervention response system that is applicable for urgent social and psychological problems.","Abstract The aim of this chapter is to describe the animal models generated by transgenic technology for the functional analysis of the protective axis of the renin\u2013angiotensin system, consisting of angiotensin-converting enzyme 2 (ACE2), angiotensin (Ang)-(1-7), and Mas. Transgenic overexpression of the components of this axis in general led to an ameliorated cardiac and vascular damage in disease states and to an improved metabolic profile. Knockout models for ACE2 and Mas, however, show aggravated cardiovascular pathologies and a metabolic syndrome-like state. In particular, the local production of Ang-(1-7) in the vascular wall, in the heart, and in the brain was found to be of high physiological relevance by the use of transgenic animals overexpressing ACE2 or Ang-(1-7) in these tissues.","Abstract This paper describes a proposed Strategic Vaccine Facility (SVF) to provide a capability to the UK to deal with new and emerging disease threats. It would underpin the vaccine manufacturing industry by developing expertise and technology to enable rapid manufacture of small batches of vaccines for emergency use against agents, such as bioterrorist agents and emerging diseases. It would have a rare ability to work with dangerous pathogens under containment, allowing the production of inactivated and live vaccines, which would be difficult in a conventional plant. The facility's output will include vaccine candidates and manufacturing protocols for transfer to industry, small vaccine batches for emergency use or clinical trials, and vaccine reference standards. It would also be available for manufacturing small batches of experimental and public health vaccines for the UK and the developing world, allowing clinical trials to be undertaken against key diseases.","A new human betacoronavirus in lineage c, tentatively called HCoV-EMC, was isolated from a patient from the Kingdom of Saudi Arabia who died from acute severe pneumonia and renal failure. The viral RNA has been detected in eight additional cases. Sequencing and bioinformatic analysis of the viral genomic RNA showed that it is a novel virus not previously detected in any other species and that its closest relatives are two Asian bat coronaviruses. HCoV-EMC may represent a sporadic spillover to humans from an unknown animal reservoir. In a recent article, van Boheemen et al. demonstrated how state-of-the-art sequencing technology and bioinformatic analysis can quickly provide critical insight into the viral genome sequence, phylogeny, replication strategy, and potential drug and vaccine targets and generate tools to evaluate the possible epidemic risk associated with this novel human virus.","The need for timely establishment of diagnostic assays arose when Germany was confronted with the first travel-associated outbreak of severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) in Europe. We describe our laboratory experiences during a large contact tracing investigation, comparing previously published real-time RT-PCR assays in different PCR systems and a commercial kit. We found that assay performance using the same primers and probes with different PCR systems varied and the commercial kit performed well.","Phosphatidylinositol kinases (PI kinases) play an important role in the life cycle of several viruses after infection. Using gene knockdown technology, we demonstrate that phosphatidylinositol 4-kinase III\u03b2 (PI4KB) is required for cellular entry by pseudoviruses bearing the severe acute respiratory syndrome-coronavirus (SARS-CoV) spike protein and that the cell entry mediated by SARS-CoV spike protein is strongly inhibited by knockdown of PI4KB. Consistent with this observation, pharmacological inhibitors of PI4KB blocked entry of SARS pseudovirions. Further research suggested that PI4P plays an essential role in SARS-CoV spike-mediated entry, which is regulated by the PI4P lipid microenvironment. We further demonstrate that PI4KB does not affect virus entry at the SARS-CoV S-ACE2 binding interface or at the stage of virus internalization but rather at or before virus fusion. Taken together, these results indicate a new function for PI4KB and suggest a new drug target for preventing SA...","Abstract Focused efforts by several international laboratories have resulted in the sequencing of the genome of the causative agent of severe acute respiratory syndrome (SARS), novel coronavirus SARS-CoV, in record time. Using cumulative skew diagrams, I found that mutational patterns in the SARS-CoV genome were strikingly different from other coronaviruses in terms of mutation rates, although they were in general agreement with the model of the coronavirus lifecycle. These findings might be relevant for the development of sequence-based diagnostics and the design of agents to treat SARS.","The severe acute respiratory syndrome coronavirus (SARS-CoV) and recently emerged Middle East respiratory syndrome coronavirus (MERS-CoV) epidemics have proven the ability of coronaviruses to cross species barrier and emerge rapidly in humans. Other coronaviruses such as porcine epidemic diarrhea virus (PEDV) are also known to cause major disease epidemics in animals wiith huge economic loss. This special issue in Virology Journal aims to highlight the advances and key discoveries in the animal origin, viral evolution, epidemiology, diagnostics and pathogenesis of the emerging and re-emerging coronaviruses in both humans and animals.","Abstract The World Health Organisation has declared a pandemic caused by the newly discovered SARS-CoV-2. Due to growing demand for reagents used for SARS-CoV-2 RNA extraction for subsequent molecular diagnostics, there is a worldwide risk of kit- and/or reagent-shortages for extraction. With a detection sensitivity of 97.4% (95% CI=86.2-99.9%), we describe a simple, fast, alternative workflow for molecular detection of SARS-CoV-2, where samples are simply heat-processed for 5 minutes at 98\u00b0C prior to the RT-qPCR reaction.","BACKGROUND: The ongoing outbreak of the recently emerged novel coronavirus (2019-nCoV) poses a challenge for public health laboratories as virus isolates are unavailable while there is growing evidence that the outbreak is more widespread than initially thought, and international spread through travellers does already occur. AIM: We aimed to develop and deploy robust diagnostic methodology for use in public health laboratory settings without having virus material available. METHODS: Here we present a validated diagnostic workflow for 2019-nCoV, its design relying on close genetic relatedness of 2019-nCoV with SARS coronavirus, making use of synthetic nucleic acid technology. RESULTS: The workflow reliably detects 2019-nCoV, and further discriminates 2019-nCoV from SARS-CoV. Through coordination between academic and public laboratories, we confirmed assay exclusivity based on 297 original clinical specimens containing a full spectrum of human respiratory viruses. Control material is mad...","Global gene expression analysis is beginning to move from the laboratories of basic investigators to large-scale clinical trials. The potential of this technology to improve diagnosis and tailored treatment of human disease may soon be realised, now that several comprehensive studies have demonstrated the utility of gene expression profiles for the classification of tumours into distinct, clinically relevant subtypes and the prediction of clinical outcomes. In addition, new data from the emerging proteomics platforms add another layer of molecular information to the study of human disease, as scientists attempt to catalogue a complete inventory of the proteins encoded by the genome and to establish a 'biosignature' profile of human health and disease. As a result, it is anticipated that, together, these technologies will facilitate the comprehensive study of genes, gene products and signalling pathways so that the objective of personalised molecular medicine can be achieved. This paper...","Abstract The severe acute respiratory syndrome (SARS) has been one of the most epidemic diseases threatening human health all over the world. Based on clinical studies, SARS-CoV (the SARS-associated coronavirus), a novel coronavirus, is reported as the pathogen responsible for the disease. To date, no effective and specific therapeutic method can be used to treat patients suffering from SARS-CoV infection. RNA interference (RNAi) is a process by which the introduced small interfering RNA (siRNA) could cause the degradation of mRNA with identical sequence specificity. The RNAi methodology has been used as a tool to silence genes in cultured cells and in animals. Recently, this technique was employed in anti-virus infections in human immunodeficiency virus and hepatitis C/B virus. In this study, RNAi technology has been applied to explore the possibility for prevention of SARS-CoV infection. We constructed specific siRNAs targeting the S gene in SARS-CoV. We demonstrated that the siRNAs ...","Abstract Microbiology laboratories have traditionally relied upon phenotypic methods involving culture and biochemical testing to identify and characterize clinically important pathogens. These techniques have several disadvantages including poor sensitivity and long turn-around time. Molecular and mass spectroscopy techniques are rapidly changing infection diagnosis and management. Compared with conventional culture-based techniques, these modern approaches provide substantially more rapid and specific information on organism identification and on the presence of resistance mechanisms. These methods are expected to contribute substantially to enhancing antibiotic stewardship and to improving \u2018time to appropriate antibiotics\u2019, one of the most important factors in improving the prognosis of patients with life-threatening infections. This article gives an overview of some of the practical applications of these newer technologies.","The microarray is a platform with wide-ranging potential in biodefence. Owing to the high level of throughput attainable through miniaturization, microarrays have accelerated the ability to respond in an epidemic or crisis. Extending beyond diagnostics, recent studies have applied microarrays as a research tool towards understanding the etiology and pathogenicity of dangerous pathogens, as well as in vaccine development. The original emphasis was on DNA microarrays, but the range now includes protein, antibody and carbohydrate microarrays, and research groups have exploited this diversity to further extend microarray applications in the area of biodefence. Here, we discuss the impact and contributions of the growing range of microarrays and emphasize the concepts that might shape the future of biodefence research.","DNA microarrays have emerged as a viable platform for detection of pathogenic organisms in clinical and environmental samples. These microbial detection arrays occupy a middle ground between low cost, narrowly focused assays such as multiplex PCR and more expensive, broad-spectrum technologies like high-throughput sequencing. While pathogen detection arrays have been used primarily in a research context, several groups are aggressively working to develop arrays for clinical diagnostics, food safety testing, environmental monitoring and biodefense. Statistical algorithms that can analyze data from microbial detection arrays and provide easily interpretable results are absolutely required in order for these efforts to succeed. In this article, we will review the most promising array designs and analysis algorithms that have been developed to date, comparing their strengths and weaknesses for pathogen detection and discovery.","The recent Ebola and Zika epidemics demonstrate the need for the continuous surveillance, rapid diagnosis and real-time tracking of emerging infectious diseases. Fast, affordable sequencing of pathogen genomes \u2014 now a staple of the public health microbiology laboratory in well-resourced settings \u2014 can affect each of these areas. Coupling genomic diagnostics and epidemiology to innovative digital disease detection platforms raises the possibility of an open, global, digital pathogen surveillance system. When informed by a One Health approach, in which human, animal and environmental health are considered together, such a genomics-based system has profound potential to improve public health in settings lacking robust laboratory capacity. SUPPLEMENTARY INFORMATION: The online version of this article (doi:10.1038/nrg.2017.88) contains supplementary material, which is available to authorized users."],"title":["Sequence analysis of SARS-CoV-2 genome reveals features important for vaccine design","Microarray technology in obstetrics and gynecology: A guide for clinicians","Development and Evaluation of A CRISPR-based Diagnostic For 2019-novel Coronavirus","CHAPTER 4 Keeping Track of Viruses","Interactome Mapping: Using Protein Microarray Technology to Reconstruct Diverse Protein Networks","Strategy and technology to prevent hospital-acquired infections: Lessons from SARS, Ebola, and MERS in Asia and West Africa","Application of mass spectrometry to molecular diagnostics of viral infections","Current Trends in Diagnostics of Viral Infections of Unknown Etiology","Chapter 5 Respiratory infections of the human bocavirus","Emerging Pathogens: Challenges and Successes of Molecular Diagnostics","Prospect and application of Internet of Things technology for prevention of SARIs","Telemedicine in the Time of Coronavirus","Novel coronavirus: From discovery to clinical diagnostics","Microarrays and gene expression profiling in microbiology and infectious diseases: a clinician's perspective","SARS-CoV-2 exhibits intra-host genomic plasticity and low-frequency polymorphic quasispecies","Future Veterinary Diagnostics","COVID-19 in Canada: Predictions for the future and control lessons from Asia","Chapter 15 Molecular Testing in Emerging Infectious Diseases","Lessons learned from the 2019-nCoV epidemic on prevention of future infectious diseases","An update on feline infectious peritonitis: Diagnostics and therapeutics","Design of Potential RNAi (miRNA and siRNA) Molecules for Middle East Respiratory Syndrome Coronavirus (MERS-CoV) Gene Silencing by Computational Method","Communicable Disease Surveillance Ethics in the Age of Big Data and New Technology","Early diagnosis of SARS Coronavirus infection by real time RT-PCR","Understand Research Hotspots Surrounding COVID-19 and Other Coronavirus Infections Using Topic Modeling","Potential Rapid Diagnostics, Vaccine and Therapeutics for 2019 Novel Coronavirus (2019-nCoV): A Systematic Review","What is ahead for health policy and technology in the 2020s?","Active Surveillance for Suspected COVID-19 Cases in Inpatients with Information Technology","Characteristics of COVID-19 infection in Beijing","Clinical course and risk factors for mortality of adult inpatients with COVID-19 in Wuhan, China: a retrospective cohort study","Clinical features of patients infected with 2019 novel coronavirus in Wuhan, China","Rapid diagnostics urgently needed for killer infections","MERS coronavirus: diagnostics, epidemiology and transmission","The time is now: moving toward virus-specific T cells after allogeneic hematopoietic stem cell transplantation as the standard of care","Decreasing Latitude and Increasing Regulation in Transplantable Tissue Programs","Recommended psychological crisis intervention response to the 2019 novel coronavirus pneumonia outbreak in China: a model of West China Hospital","Chapter 22 Animal Models with a Genetic Alteration of the ACE2/Ang-(1-7)/Mas Axis","A Strategic Vaccine Facility for the UK","The New Age of Virus Discovery: Genomic Analysis of a Novel Human Betacoronavirus Isolated from a Fatal Case of Pneumonia","Rapid establishment of laboratory diagnostics for the novel coronavirus SARS-CoV-2 in Bavaria, Germany, February 2020","Phosphatidylinositol 4-Kinase III\u03b2 Is Required for Severe Acute Respiratory Syndrome Coronavirus Spike-mediated Cell Entry","Mutational patterns correlate with genome organization in SARS and other coronaviruses","Coronaviruses: emerging and re-emerging pathogens in humans and animals","An alternative workflow for molecular detection of SARS-CoV-2 - escape from the NA extraction kit-shortage","Detection of 2019 novel coronavirus (2019-nCoV) by real-time RT-PCR","Genomic and proteomic approaches for studying human cancer: Prospects for true patient-tailored therapy","Silencing SARS-CoV Spike protein expression in cultured cells by RNA interference","Molecular-based diagnostics, including future trends","Applications of microarrays in pathogen detection and biodefence","Microarrays for Pathogen Detection and Analysis","Towards a genomics-informed, real-time, global pathogen surveillance system"],"url":["https://doi.org/10.1101/2020.03.30.016832","https://doi.org/10.1016/j.ajog.2005.12.014","https://doi.org/10.1101/2020.02.22.20025460","https://doi.org/10.1016/b978-012088483-4/50007-x","https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3968920/","https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5659033/","http://europepmc.org/articles/pmc5831079?pdf=render","https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7077230/","https://doi.org/10.1016/b978-0-12-804543-5.00005-1","https://doi.org/10.2353/jmoldx.2008.070063","https://doi.org/10.1016/j.ceh.2020.02.001","https://doi.org/10.1016/j.jpainsymman.2020.03.019","https://doi.org/10.1016/j.meegid.2020.104211","https://doi.org/10.1016/j.clinmicnews.2004.09.004","https://doi.org/10.1101/2020.03.27.009480","https://doi.org/10.1053/j.jepm.2010.05.006","https://doi.org/10.1101/2020.03.21.20040667","https://doi.org/10.1016/b978-0-12-800886-7.00015-7","https://doi.org/10.1016/j.micinf.2020.02.004","https://doi.org/10.1016/j.tvjl.2014.04.016","https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7090891/","https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7091643/","https://doi.org/10.1016/j.jcv.2003.08.004","https://doi.org/10.1101/2020.03.26.20044164","https://doi.org/10.3390/jcm9030623","https://doi.org/10.1016/j.hlpt.2020.02.003","https://doi.org/10.1016/j.jhin.2020.03.027","https://doi.org/10.1016/j.jinf.2020.02.018","https://doi.org/10.1016/s0140-6736(20)30566-3","https://doi.org/10.1016/s0140-6736(20)30183-5","https://doi.org/10.1016/s2213-2600(13)70099-7","https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4687373/","https://doi.org/10.1016/j.jcyt.2013.11.010","https://doi.org/10.1016/s0001-2092(06)60273-1","https://doi.org/10.1093/pcmedi/pbaa006","https://doi.org/10.1016/b978-0-12-801364-9.00022-5","https://doi.org/10.1016/j.vaccine.2005.01.066","https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3546555/","https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7068163/","http://www.jbc.org/content/287/11/8457.full.pdf","https://doi.org/10.1016/j.tig.2004.01.009","https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4687117/","https://doi.org/10.1101/2020.03.27.20044495","https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6988269/","https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3525069/","https://doi.org/10.1016/s0014-5793(04)00087-0","https://doi.org/10.1016/j.mpmed.2013.08.012","https://doi.org/10.1016/j.tibtech.2008.09.004","https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3245552/","https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7097748/"],"x":[0.6215699291681775,0.4012543386613147,0.8450964443514554,0.41108454380901754,0.6959474672334902,0.48986133327224957,0.40374231503120955,0.40409383241138586,0.42767348901622376,0.3652478009370751,0.48148501469859667,0.4570070492317263,0.23535527750218543,0.3727181213338299,0.628689707069077,0.8580373178900748,0.36384963956890476,0.33553717182786097,0.308575430174524,0.39843134295360083,0.9147629596042705,0.23439536413190606,0.862304781971212,0.5450373513633756,0.8456498854988137,0.2636718083974626,0.43887653921792597,0.6484635039101382,0.6967829840009211,0.738200010403311,0.4013455517016946,0.9816871048096769,0.615858409164676,0.429657954677696,0.35337376444490526,0.747890810501781,0.4922302149101807,0.4803501279995843,0.8495545562650394,0.7546128901386496,0.6615609043299657,0.6917601413704977,0.8322424854996182,0.2320383665383705,0.3817565329350351,0.6991159844419369,0.40117761935566687,0.4032713675043241,0.8576739587332147,0.23278608236054535],"y":[0.3080392718836421,0.3560169305255111,0.5414581106868969,0.21068323151696952,0.18103248594400465,0.5404070708022779,0.34680773217488753,0.34453598993568213,0.28962756890905944,0.3009524954241369,0.5308276218462266,0.5960067439053032,0.4602809413807584,0.35812793418313027,0.2929591809173555,0.5567685760619223,0.8649950129710403,0.33560158284107094,0.623727358256833,0.38832562853789493,0.39346884985560227,0.5498723520101517,0.617469323505646,0.7524022840907872,0.5378786444049526,0.5707421599746703,0.7876629155622118,0.8349393212462295,0.7942314468474788,0.7587105542070992,0.31407714299557865,0.39013664627097877,0.2259718004640924,0.46719864127397404,0.6925019232105887,0.04547304602804997,0.33936497263436793,0.07476187654915503,0.6006809789077675,0.1625498586404054,0.3640657640654383,0.3845252368608811,0.5846985881739207,0.43926005859863365,0.3653971835764181,0.29226990535418956,0.348294695741071,0.3586764550992798,0.5593214920984487,0.5385340137303893]},"selected":{"id":"26189"},"selection_policy":{"id":"26210"}},"id":"26117","type":"ColumnDataSource"},{"attributes":{"callback":null},"id":"26171","type":"TapTool"},{"attributes":{"source":{"id":"26118"}},"id":"26182","type":"CDSView"},{"attributes":{},"id":"26164","type":"BasicTicker"},{"attributes":{"editor":{"id":"26219"},"field":"text","formatter":{"id":"26218"},"title":"Passage","width":770},"id":"26186","type":"TableColumn"},{"attributes":{},"id":"26210","type":"UnionRenderers"},{"attributes":{},"id":"26223","type":"BasicTickFormatter"},{"attributes":{"text":"&lt;h3&gt;Document ranking&lt;/h3&gt;"},"id":"26194","type":"Div"},{"attributes":{},"id":"26157","type":"Range1d"}],"root_ids":["26201"]},"title":"Bokeh Application","version":"2.0.1"}}
        </script>
        <script type="text/javascript">
          (function() {
            var fn = function() {
              Bokeh.safely(function() {
                (function(root) {
                  function embed_document(root) {
                    
                  var docs_json = document.getElementById('26438').textContent;
                  var render_items = [{"docid":"ffe6f363-40d3-48ab-bde3-138b6ae224d7","root_ids":["26201"],"roots":{"26201":"f8f52611-6bdc-4254-810d-62f68f22f0ca"}}];
                  root.Bokeh.embed.embed_items(docs_json, render_items);
                
                  }
                  if (root.Bokeh !== undefined) {
                    embed_document(root);
                  } else {
                    var attempts = 0;
                    var timer = setInterval(function(root) {
                      if (root.Bokeh !== undefined) {
                        clearInterval(timer);
                        embed_document(root);
                      } else {
                        attempts++;
                        if (attempts > 100) {
                          clearInterval(timer);
                          console.log("Bokeh: ERROR: Unable to run BokehJS code because BokehJS library is missing");
                        }
                      }
                    }, 10, root)
                  }
                })(window);
              });
            };
            if (document.readyState != "loading") fn();
            else document.addEventListener("DOMContentLoaded", fn);
          })();
        </script>
    
  </body>
  
</html>