



<!DOCTYPE html>
<html lang="en">
  
  <head>
    
      <meta charset="utf-8">
      <title>Bokeh Plot</title>
      
      
        
          
        
        
          
        <script type="text/javascript" src="https://cdn.bokeh.org/bokeh/release/bokeh-2.0.1.min.js" integrity="sha384-JpP8FXbgAZLkfur7LiK3j9AGBhHNIvF742meBJrjO2ShJDhCG2I1uVvW+0DUtrmc" crossorigin="anonymous"></script>
        <script type="text/javascript" src="https://cdn.bokeh.org/bokeh/release/bokeh-widgets-2.0.1.min.js" integrity="sha384-xZlADit0Q04ISQEdKg2k3L4W9AwQBAuDs9nJL9fM/WwzL1tEU9VPNezOFX0nLEAz" crossorigin="anonymous"></script>
        <script type="text/javascript" src="https://cdn.bokeh.org/bokeh/release/bokeh-tables-2.0.1.min.js" integrity="sha384-4BuPRZkdMKSnj3zoxiNrQ86XgNw0rYmBOxe7nshquXwwcauupgBF2DHLVG1WuZlV" crossorigin="anonymous"></script>
        <script type="text/javascript">
            Bokeh.set_log_level("info");
        </script>
        
      
      
    
  </head>
  
  
  <body>
    
      
        
          
          
            
              <div class="bk-root" id="8aea6f0b-6974-4dc2-ba1a-4096df311729" data-root-id="3983"></div>
            
          
        
      
      
        <script type="application/json" id="4220">
          {"4accd84c-b012-404b-9079-49e6d3180efa":{"roots":{"references":[{"attributes":{"columns":[{"id":"3967"},{"id":"3968"}],"height":340,"source":{"id":"3900"},"view":{"id":"3970"},"width":900},"id":"3969","type":"DataTable"},{"attributes":{"children":[{"id":"3976"}]},"id":"3979","type":"Row"},{"attributes":{},"id":"3906","type":"LinearScale"},{"attributes":{},"id":"4007","type":"BasicTickFormatter"},{"attributes":{},"id":"3941","type":"LinearScale"},{"attributes":{},"id":"4005","type":"BasicTickFormatter"},{"attributes":{"below":[{"id":"3910"}],"center":[{"id":"3913"},{"id":"3917"}],"left":[{"id":"3914"}],"plot_height":340,"plot_width":900,"renderers":[{"id":"3928"}],"title":{"id":"3984"},"toolbar":{"id":"3921"},"toolbar_location":"below","x_range":{"id":"3902"},"x_scale":{"id":"3906"},"y_range":{"id":"3904"},"y_scale":{"id":"3908"}},"id":"3901","subtype":"Figure","type":"Plot"},{"attributes":{},"id":"4002","type":"UnionRenderers"},{"attributes":{"js_property_callbacks":{"change:indices":[{"id":"3974"}]}},"id":"3973","type":"Selection"},{"attributes":{},"id":"3904","type":"Range1d"},{"attributes":{"args":{"sd":{"id":"3899"},"sp":{"id":"3900"}},"code":"\n    if (sp.selected.indices.length == 0) {\n      return;\n    }\n    var sel = sp.selected.indices[0];\n    var new_selected = [];\n    for (var i = 0; i &lt; sd.data['doc'].length; i++) {\n      if (sd.data['doc'][i] == sp.data['doc'][sel]) {\n        new_selected.push(i);\n      }\n    }\n    if (JSON.stringify(new_selected) != JSON.stringify(sd.selected.indices) &amp;&amp;\n    !(sd.selected.indices.length == 1 &amp;&amp; new_selected.includes(sd.selected.indices[0]))) {\n      sd.selected.indices = new_selected;\n      sd.change.emit();\n    }\n    "},"id":"3974","type":"CustomJS"},{"attributes":{},"id":"4001","type":"StringEditor"},{"attributes":{"fill_alpha":{"value":0.1},"fill_color":{"field":"color"},"line_alpha":{"value":0.1},"line_color":{"field":"color"},"radius":{"field":"rad","units":"data"},"x":{"field":"x"},"y":{"field":"y"}},"id":"3962","type":"Circle"},{"attributes":{"axis":{"id":"3910"},"ticker":null},"id":"3913","type":"Grid"},{"attributes":{},"id":"4000","type":"StringFormatter"},{"attributes":{"children":[{"id":"3978"},{"id":"3979"},{"id":"3980"},{"id":"3981"},{"id":"3982"}]},"id":"3983","type":"Column"},{"attributes":{"formatter":{"id":"3995"},"ticker":{"id":"3915"}},"id":"3914","type":"LinearAxis"},{"attributes":{"data_source":{"id":"3900"},"glyph":{"id":"3961"},"hover_glyph":null,"muted_glyph":null,"nonselection_glyph":{"id":"3962"},"selection_glyph":null,"view":{"id":"3964"}},"id":"3963","type":"GlyphRenderer"},{"attributes":{},"id":"3915","type":"BasicTicker"},{"attributes":{},"id":"3908","type":"LinearScale"},{"attributes":{"sizing_mode":"stretch_width","text":"&lt;h2&gt;Tools and studies to monitor phenotypic change and potential adaptation of the virus&lt;/h2&gt;"},"id":"3975","type":"Div"},{"attributes":{},"id":"3937","type":"Range1d"},{"attributes":{},"id":"3939","type":"Range1d"},{"attributes":{"text":"&lt;h3&gt;Document ranking&lt;/h3&gt;"},"id":"3976","type":"Div"},{"attributes":{},"id":"3995","type":"BasicTickFormatter"},{"attributes":{"axis":{"id":"3914"},"dimension":1,"ticker":null},"id":"3917","type":"Grid"},{"attributes":{},"id":"3919","type":"ResetTool"},{"attributes":{"callback":null},"id":"3918","type":"TapTool"},{"attributes":{"js_property_callbacks":{"change:indices":[{"id":"3972"}]}},"id":"3971","type":"Selection"},{"attributes":{"source":{"id":"3900"}},"id":"3970","type":"CDSView"},{"attributes":{},"id":"3997","type":"BasicTickFormatter"},{"attributes":{},"id":"3998","type":"StringFormatter"},{"attributes":{},"id":"3989","type":"StringEditor"},{"attributes":{"fill_color":{"field":"color"},"line_color":{"field":"color"},"radius":{"field":"rad","units":"data"},"x":{"field":"x"},"y":{"field":"y"}},"id":"3926","type":"Circle"},{"attributes":{},"id":"3988","type":"StringFormatter"},{"attributes":{"source":{"id":"3900"}},"id":"3964","type":"CDSView"},{"attributes":{},"id":"3902","type":"Range1d"},{"attributes":{"callback":null,"tooltips":[["Title","@title"],["Passage","@text"],["Published","@date"],["Authors","@authors"],["Journal","@journal"],["URL","@url"]]},"id":"3955","type":"HoverTool"},{"attributes":{"text":"&lt;h3&gt;Paragraph ranking&lt;/h3&gt;"},"id":"3977","type":"Div"},{"attributes":{"editor":{"id":"4001"},"field":"text","formatter":{"id":"4000"},"title":"Passage","width":770},"id":"3968","type":"TableColumn"},{"attributes":{"data":{"authors":["Mackay, Ian M.; Arden, Katherine E.","Barnard, Ross Thomas; Hall, Roy A; Gould, Ernest A","Gorbalenya, Alexander E.; Baker, Susan C.; Baric, Ralph S.; de Groot, Raoul J.; Drosten, Christian; Gulyaeva, Anastasia A.; Haagmans, Bart L.; Lauber, Chris; Leontovich, Andrey M.; Neuman, Benjamin W.; Penzar, Dmitry; Perlman, Stanley; Poon, Leo L. M.; Samborskiy, Dmitry V.; Sidorov, Igor A.; Sola, Isabel; Ziebuhr, John; Coronaviridae Study Group of the International Committee on Taxonomy of, Viruses","Beigel, John H.; Nam, Hannah H.; Adams, Peter L.; Krafft, Amy; Ince, William L.; El-Kamary, Samer S.; Sims, Amy C.","Brisse, Sylvain; Brehony, Carina; Concei\u00e7\u00e3o, Teresa; Cubero, Meritxell; Glasner, Corinna; Le Gouil, Meriadeg; Renvois\u00e9, Aur\u00e9lie; Sheppard, Sam; Weinert, Lucy A.","Clarke, Nicola E.; Turner, Anthony J.","Villarreal, Luis P.","Axel G Rossberg; Robert J. Knell","Musazzi, Umberto M.; Di Giorgio, Domenico; Minghetti, Paola","Danielle E Anderson; Jin Cui; Qian Ye; Baoying Huang; Wenhong Zu; Jing Gong; Weiqiang Liu; So Young Kim; Biao Guo Yan; Kristmundur Sigmundsson; Xiao Fang Lim; Fei Ye; Peihua Niu; Xuming Zhou; Wenjie Tan; Lin-Fa Wang; Xu Tan","Lackie, J.M.","Beigel, John H.; Nam, Hannah H.; Adams, Peter L.; Krafft, Amy; Ince, William L.; El-Kamary, Samer S.; Sims, Amy C.","Mackay, Ian M.; Arden, Katherine E.","Phillips, J. E.; Jackwood, M. W.; McKinley, E. T.; Thor, S. W.; Hilt, D. A.; Acevedol, N. D.; Williams, S. M.; Kissinger, J. C.; Paterson, A. H.; Robertson, J. S.; Lemke, C.","Villarreal, L.P.","Brisse, Sylvain; Brehony, Carina; Concei\u00e7\u00e3o, Teresa; Cubero, Meritxell; Glasner, Corinna; Le Gouil, Meriadeg; Renvois\u00e9, Aur\u00e9lie; Sheppard, Sam; Weinert, Lucy A.","Gorbalenya, Alexander E.; Baker, Susan C.; Baric, Ralph S.; de Groot, Raoul J.; Drosten, Christian; Gulyaeva, Anastasia A.; Haagmans, Bart L.; Lauber, Chris; Leontovich, Andrey M.; Neuman, Benjamin W.; Penzar, Dmitry; Perlman, Stanley; Poon, Leo L. M.; Samborskiy, Dmitry V.; Sidorov, Igor A.; Sola, Isabel; Ziebuhr, John; Coronaviridae Study Group of the International Committee on Taxonomy of, Viruses","Moni, Mohammad Ali; Li\u00f2, Pietro","Mayer, J.D.","Yonghua Wu","Sun, Kaiyuan; Chen, Jenny; Viboud, C\u00e9cile","Sun, Kaiyuan; Chen, Jenny; Viboud, C\u00e9cile","Robson, B.","Charrel, R.N.","Villarreal, Luis P.","Alexander E. Gorbalenya; Susan C. Baker; Ralph S. Baric; Raoul J. de Groot; Christian Drosten; Anastasia A. Gulyaeva; Bart L. Haagmans; Chris Lauber; Andrey M Leontovich; Benjamin W. Neuman; Dmitry Penzar; Stanley Perlman; Leo L.M. Poon; Dmitry Samborskiy; Igor A. Sidorov; Isabel Sola; John Ziebuhr","Mackay, Ian M.; Arden, Katherine E.","Yamada, Yoshiyuki; Liu, Xiao Bo; Fang, Shou Guo; Tay, Felicia P. L.; Liu, Ding Xiang","Boulos, Maged N Kamel","Wenjun Wang; Yikai Wang; Xin Zhang; Yaping Li; Xiaoli Jia; Shuangsuo Dang","Weber, David J.; Rutala, William A.; Fischer, William A.; Kanamori, Hajime; Sickbert-Bennett, Emily E.","","Majid Einian; Hamid Reza Tabarraei","Drexler, Jan Felix; Corman, Victor Max; Drosten, Christian","Yamada, Yoshiyuki; Liu, Xiao Bo; Fang, Shou Guo; Tay, Felicia P. L.; Liu, Ding Xiang","Jones, Samuel L.; Blikslager, Anthony T.","","Mackay, Ian M.; Arden, Katherine E.","Plunkett, Signe J.","","Habibi, Roojin; Burci, Gian Luca; de Campos, Thana C; Chirwa, Danwood; Cin\u00e0, Margherita; Dagron, St\u00e9phanie; Eccleston-Turner, Mark; Forman, Lisa; Gostin, Lawrence O; Meier, Benjamin Mason; Negri, Stefania; Ooms, Gorik; Sekalala, Sharifah; Taylor, Allyn; Yamin, Alicia Ely; Hoffman, Steven J","Jessica A. Plante; Kenneth S. Plante; Lisa E. Gralinski; Anne Beall; Martin T. Ferris; Daniel Bottomly; Richard Green; Shannon K. McWeeney; Mark T. Heise; Ralph S. Baric; Vineet D. Menachery","Sun, Jiumeng; He, Wan-Ting; Wang, Lifang; Lai, Alexander; Ji, Xiang; Zhai, Xiaofeng; Li, Gairu; Suchard, Marc A.; Tian, Jin; Zhou, Jiyong; Veit, Michael; Su, Shuo","Truyen, Uwe; Parrish, Colin R.; Harder, Timm C.; Kaaden, Oskar-R\u00fcger","Parkes, Margot W.; Bienen, Leslie; Breilh, Jaime; Hsu, Lee-Nah; McDonald, Marian; Patz, Jonathan A.; Rosenthal, Joshua P.; Sahani, Mazrura; Sleigh, Adrian; Waltner-Toews, David; Yassi, Annalee","Totura, Allison L; Baric, Ralph S","Wu, Kailang; Peng, Guiqing; Wilken, Matthew; Geraghty, Robert J.; Li, Fang","","Minghua Jiang; Wenjie Fang; Amir Aratehfar; Xiaojing Li; Liyan ling; Hua Fang; Farnaz Farnaz Daneshnia; Jian Yu; Wanqing Liao; Hao Pei; Weihua Pan; Cornelia Lass-Florl"],"color":["#ef8a62","#ef8a62","#ef8a62","#ef8a62","#ef8a62","#ef8a62","#ef8a62","#ef8a62","#ef8a62","#ef8a62","#ef8a62","#ef8a62","#ef8a62","#ef8a62","#ef8a62","#ef8a62","#ef8a62","#ef8a62","#ef8a62","#ef8a62","#ef8a62","#ef8a62","#ef8a62","#ef8a62","#ef8a62","#ef8a62","#ef8a62","#ef8a62","#ef8a62","#ef8a62","#ef8a62","#ef8a62","#ef8a62","#ef8a62","#ef8a62","#ef8a62","#ef8a62","#ef8a62","#ef8a62","#ef8a62","#ef8a62","#ef8a62","#ef8a62","#ef8a62","#ef8a62","#ef8a62","#ef8a62","#ef8a62","#ef8a62"],"date":["2015-12-22","2014-01-09","2020","2019-07-31","2014-03-31","2011-11-10","2008-12-31","2020-04-01","2020-04-15","2020-03-30","2013-12-31","2019-07-31","2015-12-22","2011-09-10","2008-12-31","2014-03-31","2020","2014-10-24","2008-12-31","2020-03-04","2020","2020","2020-02-26","2014-05-31","2008-12-31","2020-02-11","2015-04-16","2009-07-02","2004-01-28","2020-02-26","2016-05-02","2008-05-31","2020-03-30","2014-01-31","2009-07-02","2004-12-31","2017-11-30","2015-04-16","2000-12-31","2011-05-31","2020-03-06","2020-02-20","2020-03-21","1995-02-28","2005-11-10","2012-06-30","2012-01-30","1985-12-31","2020-03-20"],"doc":["mwj2qby4","59gmu16g","vvhhpchm","h97dui9n","a4mf2vnp","cuz5bqy6","ydkigome","vy1mu04l","k0kha22e","0l33i6s4","hau0cshe","h97dui9n","mwj2qby4","432ixb7g","01a8er2v","a4mf2vnp","vvhhpchm","3qgzulgj","t99l3zii","4ihv80au","faec051u","faec051u","13jupb26","3bzvqc13","ydkigome","ptbk69rk","6xyipioz","gkm7i62s","kuybfc1y","g3asp1ps","ljggnelt","39vjafky","tnw82hbm","sxbmd0df","gkm7i62s","j9kg00qf","ojmhkt5o","6xyipioz","qtz3nund","poamqquy","w8ch013c","99sbhbuo","9il7coyk","19z4ej76","7z10l0m1","4hsbt4bl","c4q5gtw2","eh21tdhp","eo2pcgix"],"journal":["Virol J","Expert Rev Mol Diagn","Nature Microbiology","Antiviral Research","Research in Microbiology","Int J Hypertens","Origin and Evolution of Viruses","","International Journal of Pharmaceutics","","The Dictionary of Cell &amp; Molecular Biology","Antiviral Research","Virol J","Virus Genes","Encyclopedia of Virology","Research in Microbiology","Nature Microbiology","BMC Bioinformatics","International Encyclopedia of Public Health","","The Lancet Digital Health","The Lancet Digital Health","Computers in Biology and Medicine","Clinical Microbiology and Infection","Origin and Evolution of Viruses","","Virus Research","PLoS One","Int J Health Geogr","","American Journal of Infection Control","Clinical Microbiology and Infection","","Antiviral Research","PLoS One","Equine Internal Medicine","International Journal of Antimicrobial Agents","Virus Research","Emergency Procedures for the Small Animal Veterinarian","Clinical Microbiology and Infection","The Lancet","","Trends in Molecular Medicine","Veterinary Microbiology","Ecohealth","Current Opinion in Virology","Journal of Biological Chemistry","Perspectives in Medical Virology",""],"rad":[0.02,0.013639330275703472,0.01311655013174138,0.012281995420323401,0.011827706291387526,0.009057427192265037,0.0074808703871623505,0.006889857926686727,0.0067231600762898205,0.006542793006100566,0.005888514307135485,0.005835588847761261,0.005698912597108649,0.0056092094198387545,0.005601449247701005,0.0055406019070459835,0.00543479548149462,0.005402587730234272,0.005299566987950789,0.005134633842183545,0.005040715076015385,0.005040715076015385,0.005014403835898673,0.004980825626821095,0.004865297965360195,0.0046100153730654625,0.004598063822982344,0.00454733358817708,0.004313472978797434,0.004250744688205415,0.004063177641894865,0.003948698629636446,0.0038696254331380448,0.0038403987316034775,0.003663850526929672,0.0036554853018535374,0.0035860936692766582,0.003557513454211348,0.0034707260414063206,0.0034500862974031887,0.0033497980394640147,0.003341982521156643,0.003300948403272383,0.0031786868593676647,0.0031257393064802776,0.0030934880358252205,0.0030668053026893967,0.0030432519938399173,0.003],"scr":[1.0,0.626,0.595,0.546,0.519,0.356,0.264,0.229,0.219,0.208,0.17,0.167,0.159,0.153,0.153,0.149,0.143,0.141,0.135,0.126,0.12,0.12,0.118,0.117,0.11,0.095,0.094,0.091,0.077,0.074,0.063,0.056,0.051,0.049,0.039,0.039,0.034,0.033,0.028,0.026,0.021,0.02,0.018,0.011,0.007,0.005,0.004,0.003,0.0],"text":["Changes in genome sequence may herald alterations to virus transmissibility, replication, persistence, lethality or response to future drugs. If we have prior knowledge of the impact of genetic changes because of thorough characterization studies, we can closely monitor the genomic regions and better understand any changes in transmission or disease patterns as they occur. Genetic mutations noted during the largest of human outbreaks, Jeddah-2014, did not impart any major replicative or immunomodulatory changes when compared to earlier viral variants in vitro [156, 176]. However, we understand very little of the phenotypic outcomes that result from subtle genetic change in MERS-CoV genomes. To date no clinical relevance or obvious in vivo changes to viral replication, shedding or transmission has been reported or attributed to mutations or to new recombinant viruses [156]. But vigilance and larger, more contemporary and in vivo studies are needed.","It is reasonable to suggest that climate change will impact on the distribution of disease vectors and viruses [8]. However, the geographic directions of change and local impact due to climate are unlikely to be predictable in a simple, latitude-dependent manner. Change in vector distribution, for example, the incursions of Aedes albopictus from Torres Strait into northern Australia, are likely to lead to permanent colonization [9]. These mosquitoes have the capacity to spread a suite of viruses (West Nile virus [WNV], yellow fever virus, St. Louis encephalitis, dengue fever and chikungunya virus [CHIKV]) into higher latitudes. Irrespective of whether this change is related to climate change, to the rapid adaptation of mosquitoes to cooler climates or the adaptation of viruses to new vectors [10,11], the need for continued adaptation of our diagnostic methods is evident. Extreme climatic events, such as flooding, have an impact on the local disease incidence, and moreover, flooding provides water corridors for the distribution of insect vectors over large distances.","The currently known viruses of the species Severe acute respiratory syndrome-related coronavirus may be as (poorly) representative for this particular species as the few individuals that we selected to represent H. sapiens in Fig. 1 . It is thus reasonable to assume that this biased knowledge of the natural diversity of the species Severe acute respiratory syndrome-related coronavirus limits our current understanding of fundamental aspects of the biology of this species and, as a consequence, our abilities to control zoonotic spillovers to humans. Future studies aimed at understanding the ecology of these viruses and advancing the accuracy and resolution of evolutionary analyses 41 would benefit greatly from adjusting our research and sampling strategies. This needs to include an expansion of our current research focus on human pathogens and their adaptation to specific hosts to other viruses in this species. To illustrate the great potential of species-wide studies, it may again be instructive to draw a parallel to H. sapiens, and specifically to the impressive advancements in personalized medicine in recent years. Results of extensive genetic analyses of large numbers of individuals representing diverse populations from all continents have been translated into clinical applications and greatly contribute to optimizing patient-specific diagnostics and therapy. They were instrumental in identifying reliable predictive markers for specific diseases as well as genomic sites that are under selection. It thus seems reasonable to expect that genome-based analyses with a comparable species coverage will be similarly insightful for coronaviruses. Also, additional diagnostic tools that target the entire species should be developed to complement existing tools optimized to detect individual pathogenic variants (a proactive approach). Technical solutions to this problem are already available; for example, in the context of multiplex PCR-based assays 42 . The costs for developing and applying (combined or separate) species-and virus-specific diagnostic tests in specific clinical and/or epidemiological settings may help to better appreciate the biological diversity and zoonotic potential of specific virus species and their members. Also, the further reduction of time required to identify the causative agents of novel virus infections will contribute to limiting the enormous social and economic consequences of large outbreaks. To advance such studies, innovative fundraising approaches may be required.","Routine virologic surveillance is carried out through NGS analysis to monitor known determinants of reduced drug susceptibility and by empirical assessments in phenotypic drug susceptibility assays. Most cell culture-based assays are slow and cumbersome, requiring multiple rounds of virus replication. To facilitate surveillance efforts, the CDC laboratory has developed a single-cycle assay: high-content imaging-based neutralization test (HINT). In this assay, MDCK-SIAT1 cells are infected with influenza virus in the absence of trypsin to limit virus spread. After 16-24 h of incubation, cells are fixed, immuno-fluorescently stained with anti-NP antibody, and imaged using an automated fluorescent cell counter. Baloxavir EC 50 values measured using the HINT assay were within approximately 2-fold of values obtained by focus reduction and plaque reduction assays . The HINT assay revealed that the PA-I38L substitution, detected in two A(H1N1)pdm09 viruses, but as yet un-characterized, conferred a 7-fold reduction in baloxavir susceptibility, which is comparable to the 10fold reduction observed for the previously characterized treatmentemergent I38M substitution . For pimodivir, the HINT EC 50 values were 8-fold higher than EC 50 values determined by focus reduction assay; nevertheless, the fold changes conferred by PB2-S324C and S324R substitutions remained consistent between two assays. The HINT assay provides a higher throughput option for phenotypic surveillance of seasonal and other low pathogenic influenza viruses; HINT does not work with highly pathogenic avian influenza viruses. As the HINT assay is not yet available for influenza C and D viruses, their susceptibility to baloxavir was assessed using the virus yield reduction assay. Baloxavir potently inhibited replication of all viruses tested and showed the highest activity against type A and the lowest (\u223c8-fold) against influenza D viruses.","Keeping on the topic of virulence emergence, Lucy Weinert (University of Cambridge, Cambridge, UK) reported on the genomic differences between commensal and pathogenic Streptococcus suis. S. suis causes respiratory and systemic disease in pigs, but is also zoonotic and the leading cause of human meningitis in Vietnam. Using genome sequences from 400 isolates, L. Weinert showed there was a complete lack of phylogeographical structuring of S. suis and that the genomes were very diverse, highly recombinogenic and had experienced dynamic gene gain and loss. There was strong genetic differentiation among commensal, respiratory and systemic disease isolates, although no SNP or gene was able to unambiguously resolve the three phenotypic groups. In contrast, there seemed to be little difference between human and pig isolates and, consequently, little evidence for adaptation to humans.","Soon after its discovery, gene deletion studies established ACE2 not only as a modulator of blood pressure but also as an essential regulator of cardiovascular function [31]. The progressive cardiac dysfunction observed in the first ACE2 mouse knockout model resembled that of tissue subjected to long-term hypoxia of the type that occurs after coronary artery disease or bypass surgery in humans [32]. As a result of these observations ACE2 was immediately proposed as a cardioprotective protein. This hypothesis was strengthened by the observation that ACE2-null phenotypes were reversed by concurrent knockout of the ACE gene. This evidence appeared to demonstrate unequivocally that the primary role of ACE2 was to counterbalance that of ACE [31]. The initial hypothesis that ACE2 plays a critical role in cardiac function primarily by counterbalancing the effects of ACE was not, however, entirely supported by subsequent gene deletion models. The discrepancies between the initial study and the phenotypes are described elsewhere, which saw no obvious functional or morphological changes [33, 34], were initially proposed to be due to differing genetic backgrounds in their models. This potential mechanism was investigated by backcrossing the hybrid model used in both studies with an initial parental line; however, both backcrossed models showed no cardiac changes [34]. Interestingly subsequent studies using the original ACE2-deficient mice described by Crackower et al. also showed no overt cardiac changes, suggesting that the phenotype is lost over time [35, 36]. Despite seeing no overt phenotypic change in deletion models, subsequent groups have shown a reduced ability to respond to injury in ACE2-null mice. Together these studies suggest that, rather than being a key mediator of cardiac phenotype, ACE2 is essential in modulating responses to injury [33, 37]. ","However, phylogenetic approaches necessarily assume the master template is the fi ttest type and that mutations or variants in the RNA populations are a source of genetic load that are deleterious and limiting to virus adaptation ( Pybus et al. , 2007 ) . Such variation is mostly due to \" unfi t \" mutations, which indicates that a viral cloud is mostly and unfi t consortia. It would seem that such conclusions go against the concept of quasispecies as being fi t per se as described above. In this consideration we see a major weakness of extant phylogenetic methods. They were not developed to access the evolutionary relationship and fi tness of interacting mixtures. Nor were they designed to follow the evolution of systems with high rates of recombination between numerous parental templates. We currently lack the analytical tools for such a population analysis. Without such tools, however, it seems we can only evaluate those parameters we can defi ne and will remain confused by those we cannot. Evolution of a consortia thus provides a new directions for theoretical and laboratory research. We should seek to investigate the mixture, not just its average.","The use of detailed simulation modelling to assess the potential outcomes of the current SARS-CoV-2 pandemic and its associated disease, COVID-19, has produced crucial information for policy makers and healthcare managers, most notably in the UK where a detailed modelling study (Ferguson et al. 2020 ) has led to a significant change in government policy regarding the pandemic (Gallagher 2020) . The model used, an adaptation of one that was previously used to model potential influenza pandemics (Ferguson et al. 2006) , is a complex individual-based simulation with many thousand lines of code which are not yet available openly ( Tweet from @neil_ferguson 22nd March 2020 ). While such models can make accurate predictions for given assumptions on individual behaviour, their computationally intensive nature means that they do not easily capture a major source of uncertainty determining disease dynamics: when and how non-pharmaceutical interventions (NPI) such as social distancing measures are enacted by governments and how the population change their behaviour as a result. To address this aspect of the problem, we propose to use simpler models and perform \"Management Strategy Evaluation\" (MSE: (Bunnefeld, Hoshino &amp; Milner-Gulland 2011; Punt et al. 2016) , an adaptive framework for comparing multiple management strategies in a variety of circumstances. Originally developed for assessing fisheries management schemes, this framework has the potential to be applied widely for assessing best practice in other important management scenarios (Bunnefeld et al. 2011 ) .","Moreover, the promising results obtained by some National competent Authorities (e.g., AIFA) suggested that setting up cooperation schemes between administration and other stakeholders, fostering a responsible approach in managing crisis in medicines supply, also allowing, for instance, surveillance/verification programs that may represent rational and effective preventive strategies to monitor the activities of exporting wholesalers and distributors for reducing the risks of product unavailability. If such tools are strongly necessary for highshortage-impact products, their adoption as a preventive action to monitor medium-and low-impact products is also desirable. However, for low-and medium-impact products, the development of protocols to rationalize the clinical medicine substitution or to activate importation procedures may be more effective. In this context, education programs should be performed by competent Authorities to ensure update of healthcare professionals on these topics.","infections. For example, the DNA damage checkpoint pathway and the NF-kB pathway are 25 known to be positively selected in the bat genomes 9 . In addition, in the type I interferon gene 26 family, the MHC class I genes and natural killer cell receptor family have been reported to be 27 highly expanded in the bat genomes 10 . These studies provided evidence to support the hypothesis 28 that bats have evolved a highly specific genetic configuration, especially in the innate immunity 29 pathways. A major caveat is the lack of systematic examination of the functions of these genes 30 due to shortage of functional genomic tools. 31 CRISPR and RNAi technologies have enabled genome-wide screening of gene functions for a 1 variety of cell types, yielding an unprecedented wealth of information revealing the complexity of 2 biological processes 11, 12 . This is especially true in the field of virus-host interactions, where a 3 plethora of host factors have been identified for a variety of viruses [13] [14] [15] . Both technologies have 4 advantages and disadvantages and provide complementary approaches to achieve comprehensive 5 genetic perturbations [16] [17] [18] . CRISPR permits complete knockout of a gene, thus affords generation 6 of truly null genotype. A caveat is that the lethality associated with knockout of essential genes 7 prevents functional parsing of these genes in different biological processes. In addition, CRISPR 8 screening is usually used a pool-based format that requires the generation of stable cell lines 9 through long term culturing. Therefore, CRISPR is suited for studying phenotypes that require a 10 longer time to manifest. Based on previous studies, CRISPR screens demonstrated excellent 11 specificity but limited sensitivity 17 . In contrast, siRNA screening relies on temporary knockdown 12 of genes in an individual well format, a methodology suited for examining short-term phenotypic 13 changes. Due to the transient nature of the assay, siRNA screening can tolerate growth defects due 14 to gene knockdown and can elucidate hypomorphic phenotypes induced by siRNAs with varying 15 degrees of knockdown. In general, RNAi screens allow comprehensive identification of genes in 16 a biological process at the cost of more false positives due to pervasive off-target effects 17 . To fully 17 realize the power of CRISPR and RNAi screening as complementary strategies to interrogate gene 18 function in bats, here we developed a pool-based CRISPR library and a plate-based siRNA library 19","adaptation A change in sensory or excitable cells upon repeated stimulation, that reduces their sensitivity to continued stimulation. Those cells that show rapid adaptation are known as phasic; those that adapt slowly are known as tonic. Can also be used in a more general sense for any system that changes responsiveness with time \u00c0 for example by downregulation of receptors (tachyphylaxis) or through internal modulation of the signalling system, as in bacterial chemotaxis.","Resistant variants can be detected using phenotypic assays that determine the 50% effective or inhibitory concentration of drug on virus growth (EC 50 ) or enzymatic activity (IC 50 ) of the target (e.g. NA or PA), or using genotypic assays, which detect known genetic changes associated with reduced susceptibility. Phenotypic NAI assays are wellcharacterized but require a cultivated isolate, which make them inadequate as a point-of-care diagnostic, and they are generally less sensitive for detecting resistant variants in mixed populations (Wetherall et al., 2003) ; however, newer technologies have attempted to reduce turn-around time and increase the sensitivity of the standard NAI assay (Gubareva et al., 2017) . Lack of standardization and a clear relationship between the level of reduced susceptibility and clinical outcomes make phenotypic assays difficult to interpret, but this also is a continuing area of study. Genotypic assays are generally rapid and sensitive, but require prior knowledge of the effect of certain mutations on virus susceptibility. Genotypic assays can vary in scope, from realtime PCR SNP detection to Sanger or 'next-generation' sequencing of whole genes or genomes. The advent of influenza antivirals with distinct mechanisms of action will require new assays and approaches. Phenotypic assays for baloxavir are currently neither rapid nor highthroughput, although advances are currently being made, and the genetic determinants of reduced susceptibility to baloxavir are still being defined. As with all classes of drugs, surveillance will involve the use of genotypic and phenotypic assays to account for the emergence and spread of resistance.","Genomic sequence can also be used to define the geographic boundaries of a cluster or outbreak and monitor its progress, based on the similarity of the variants found among infected humans and animals when occurring together, or between different sites and times (Fig. 6) [169]. This approach was employed when defining the geographically constrained MERS hospital outbreak in Al-Ahsa, which occurred between 1st April and 23rd May 2013, as well as clusters in Buraidah and a community outbreak in Hafr Al-Batin, the KSA. Genomic sequencing identified that approximately 12 MERS-CoV detections from a community outbreak in Hafr Al-Batin between June and August 2013 may have been triggered by an index case becoming infected through DC contact [175]. Sequencing MERS-CoV genomes from the 2013 Al-Ahsa hospital outbreak indicated that multiple viral variants contributed to the cases but that most were similar enough to each other to be consistent with human-to-human transmission. Molecular epidemiology has revealed otherwise hidden links in transmission chains encompassing a period of up to five months [179]. However, most outbreaks have not continued for longer than two to three months and so opportunities for the virus to adapt further to humans through co-infection and sustained serial passage have been rare [169]. In Riyadh-2014, genetic evidence supported the likelihood of multiple external introductions of virus, implicating a range of healthcare facilities in an event that otherwise looked contiguous [23, 168, 179]. Riyadh is a nexus for camel and human travel and has had more MERS cases than any other region of the KSA to date but also harbours a wide range of MERS-CoV variants [128, 167, 179]. However the South Korean outbreak originated from a single infected person, resulting in three to four generations of cases [180, 181]. Studies of this apparently recombinant viral variant did not find an increased evolutionary rate and no sign of virus adaptation thus the outbreak seems to have been driven by circumstance rather than circumstance together with mutation [181].","Many studies have examined sequence changes in the structural proteins of IBV and found that most of the changes associated with adaptation to a particular host or with a particular virus pathotype occur in the spike glycoprotein [18, 19, 30]. But only a few studies have examined changes across the entire genome associated with biological characteristics of the virus [22, 31]. Ammayappan et al. [22] found a total of 17 amino acid changes between the genomes of Ark DPI 11, a pathogenic virus and Ark DPI 101 an attenuated virus, with four amino changes in nsp 3 and six amino acid changes in the S1 glycoprotein. Based on that data, it was suggested that changes in the replicase sequence in addition to structural proteins might play a role in pathogenicity. Fang et al. [31] found 53.06% of all amino acid substitutions across the entire genome were located in the spike glycoprotein following adaptation of an attenuated avian coronavirus to primate cells, suggesting that spike plays a role in host adaptation.","The study of virus variation and evolution is an applied science that allows the observation of evolutionary change in real time. For example, human individuals (or populations) infected with either human immunodeficiency virus 1 (HIV-1) or hepatitis C virus (HCV) show progressive or geographical evolutionary adaptation associated with the emergence of specific viral clades that affect disease therapy and progression (such as resistance to antiviral drugs). Figure 1 shows HIV variation in an individual human patient whereas Figure 2 shows HCV variation in the human population. Virus evolution is also important for the commercial growth of various organisms, such as the dairy industry (lactose fermenting bacteria), the brewing industry, agriculture, aquaculture, and farming. In all these applied cases major losses can result from virus adaptation to the cultivated species, often from viruses of wild species. Some organisms appear much less prone to viral adaptations (e.g., nematodes, ferns, sharks). Virus evolution can also be applied to technological innovation, as in phage display. This is a process in which a terminal surface protein of some filamentous bacterial virus can be genetically engineered for novel surface protein expression. By generating diversity in vitro (with up to 10 15 types), and applying the principles of evolution (random variation) to biochemical selection (such as binding to a chemical substrate), a reiterative amplification can find solutions to problems in biochemistry, such as surface interactions or catalytic activity.","Typing methods should have three important properties that favor their wide application: high discriminatory power, simplicity of use and low cost. In light of the whooping cough resurgence in several countries, Norman Fry (PHE, Colindale, UK) reviewed existing recommendations for Bordetella pertussis strain molecular characterization [37] . The observed increase in infection could reflect waning immunity, improved surveillance or pathogen adaptation. Understanding these factors is an important public health objective. Of the typing methods that are used, serotyping offers low discrimination, but provides an easy way of detecting changes in B. pertussis populations. PFGE, MLVA and DNA sequencing offer higher discrimination. More importantly, DNA sequence-based methods can monitor changes in particular genes e for example genes coding for cellular vaccine components, but are more labor-intensive. As for other microorganisms, wholegenome sequencing offers the greatest amount of information. For monitoring of the emergence of vaccine antigendeficient variants, N. Fry and colleagues recommend a combination of DNA sequence analysis (Sanger or HTS) and protein detection techniques, which must ideally conform to guidelines for the validation and application of typing methods used in bacterial epidemiology [46] .","However, the host of a given virus may be uncertain, and virus pathogenicity remains unknown for a major (and fast-growing) proportion of viruses, including many coronaviruses discovered in metagenomics studies using next-generation sequencing technology of environmental samples 47, 48 . These studies have identified huge numbers of viruses that circulate in nature and have never been characterized at the phenotypic level. Thus, the genome sequence is the only characteristic that is known for the vast majority of viruses, and needs to be used in defining specific viruses. In this framework, a virus is defined by a genome sequence that is capable of autonomous replication inside cells and dissemination between cells or organisms under appropriate conditions. It may or may not be harmful to its natural host. Experimental studies may be performed for a fraction of known viruses, while computational comparative genomics is used to classify (and deduce characteristics of) all viruses. Accordingly, virus naming is not necessarily connected to disease but rather informed by other characteristics.","To observe the trend of phenotypic relative risk corresponding to the number of shared genes between 2 diseases, we have computed the number of shared genes between two diseases and their corresponding phenotypic relative risk of the occurrence of comorbidities as shown in Figure 10. We observed that with increasing number of shared biomarker genes between 2 diseases, the phenotypic relative risk is also increased. We may predict existing diseases of a patient and the prospective disease comorbidities through the identification of highly up and down dysregulated genes. So based on the available data we could predict the disease comorbidities and the level of the comorbidities using the regression model as Figure 10.","Briefly, the factors that this committee identified were the following: human demographics and behavior; technology and industry; economic development and land use; international travel and commerce; microbial adaptation and change; and the breakdown of public health measures. It is notable that five of these six factors are social factors that are consequences of changes in society. Even microbial adaptation and change, such as the development of antimicrobial resistance as a response to selective pressure, has a large behavioral dimension. This is partly a response to a technical innovation -the development of antimicrobials -and partly a response to a behaviorthe prescribing of those antimicrobials. Of course, one dimension of this factor is the nonselective and improper prescribing of antimicrobials. This has several dimensions: The prescription of antibiotics when none are needed, the prescription of broad-spectrum antibiotics when narrowspectrum antibiotics are sufficient, the free availability of antibiotics in many developing countries on the street and in pharmacies where no prescription is needed, and the free use of late-generation antibiotics in the food industry to promote the growth of cattle, chickens, and other animals intended for human consumption. So, in fact, all of the six factors of emergence are social and behavioral in nature.","Considering that 2019-nCoV and SARS-CoV are distantly related, but show high similarity in the RBD protein structure and can use the same cell receptor, ACE2 5-11 , an evolutionary convergence may have occurred between them. In the present study, we employ a recently developed molecular phyloecological approach 14-16 , which uses a comparative phylogenetic analysis of functional gene sequences to determine the genetic basis of phenotypic evolution, and we examine the possible molecular basis underlying the phenotypic convergence between 2019-nCoV and SARS-CoV. Our results reveal positive selection signals and evolutionary convergent amino acid sites of the spike protein in both 2019-nCoV and SARS-CoV and their related coronaviruses, providing new insights into understanding the evolutionary origin of their phenotypic convergence.","In conclusion, crowdsourced epidemiological data can be useful to monitor emerging outbreaks, such as COVID-19 and, as previously, Ebola virus. 7 These efforts can help generate and disseminate detailed information in the early stages of an outbreak when little other data are available, enabling independent estimation of key parameters that affect interventions. Based on our small sample of patients with COVID-19, we note an intriguing age distribution, reminiscent of that of SARS, which warrants further epidemiological and serological studies. We also report early signs that the response is strengthening in China on the basis of a decrease in case detection time, and rapid management of travel-related infections that are identified internationally. This is an early report of a rapidly evolving situation and the parameters discussed here could change quickly. In the coming weeks, we will continue to monitor the epidemiology of this outbreak using data from news reports and official sources.","In conclusion, crowdsourced epidemiological data can be useful to monitor emerging outbreaks, such as COVID-19 and, as previously, Ebola virus. 7 These efforts can help generate and disseminate detailed information in the early stages of an outbreak when little other data are available, enabling independent estimation of key parameters that affect interventions. Based on our small sample of patients with COVID-19, we note an intriguing age distribution, reminiscent of that of SARS, which warrants further epidemiological and serological studies. We also report early signs that the response is strengthening in China on the basis of a decrease in case detection time, and rapid management of travel-related infections that are identified internationally. This is an early report of a rapidly evolving situation and the parameters discussed here could change quickly. In the coming weeks, we will continue to monitor the epidemiology of this outbreak using data from news reports and official sources.","Some care is needed in deducing mechanisms of activation and cell entry, and hence potential therapies, on the basis of using different types of virus as reference models. This is because there are four main types of coronavirus with differences in spike proteins and it is known that here are some differences in activation and lung cell entry. Renaming the believed causative agent of COVID-19 to indicate it as a type of SARS does not alleviate the concern that different strains use different mechanisms: very few point mutations might change the modes of cell entry. The entry types with represented examples are as follows [4] . Genus Alphacoronavirus: human coronavirus NL63 (HCoV-NL63), porcine transmissible gastroenteritis coronavirus (TGEV), porcine epidemic diarrhea coronavirus (PEDV), and porcine respiratory coronavirus (PRCV). Genus Betacoronavirus: severe acute respiratory syndrome coronavirus (SARS-CoV), Middle East respiratory syndrome coronavirus (MERS-CoV), bat coronavirus HKU4, mouse hepatitis coronavirus (MHV), bovine coronavirus (BCoV), and human coronavirus OC43. Genus Gammacoronavirus avian infectious bronchitis coronavirus (IBV). Genus Deltacoronavirus: porcine deltacoronavirus (PdCV). The bioinformatics-based approach can also be reproduced by standard tools, but is facilitated by Q-UEL tools, especially with the explosive growth in studies regarding 2019-nCoV.","The last decade has witnessed a large number of emerging or re-emerging infections. It is obvious that improving our capacity to respond is an important goal. It is important for preparedness and response programmes to be developed, and gaps in capacities must be identified and corrected [5] . These programmes have to provide the medical community with detection tools that are applicable in routine diagnostic laboratories to enable the rapid detection of cases and to monitor, in a timely manner, the unpredictable dissemination of emerging pathogens.","This is an inherent problem in the analysis of virus evolution: a small selected set of hallmark genes with some similarity are assumed to trace an apparently linear (tree-based) viral lineage whereas the larger number of genes are not included and cannot be traced. If most of RNA virus evolution is indeed mediated by a mixed cloud of genomes, any role for mutant mixtures thus becomes obscure. But perhaps there is little else we can currently do given the lack of information. How might we explain the increased fi delity and genome size of the nidoviruses? Was there some change in viral adaptation in which quasispecies and generation of mixtures was no longer as important for adaptation? Did the need and selection for a larger genome override the use of error to generate adaptability as seen in poliovirus and HIV-1? If so, what selective pressures might have changed this seemingly basic feature? What do we know about the natural biology of these viruses, which might provide some insight into this?","However, the host is not definitive nor is the pathogenicity known for a major (and fast growing) share of viruses, including many coronaviruses discovered in metagenomics studies using next generation sequencing technology 35, 36 . These studies analyze diverse environmental specimens and assemble genomic sequence of viruses, which circulate in nature and have never been characterized on the phenotypic level. Thus, the genome sequence is the only characteristic that is known for the vast majority of viruses, and its use in defining virus identity in the virosphere is the only available choice going forward. In this framework, a virus is defined by its genome sequence that instructs the synthesis of polynucleotide molecules capable of autonomous replication inside cells and dissemination between cells or organisms under appropriate conditions. It may or may not be harmful to its natural host. Experimental studies may be performed for a fraction of known viruses, while computational comparative genomics is used to classify (and deduce characteristics of) all viruses.","To date, the RBD of MERS-CoV have not undergone noteworthy genetic change, which could indicate that the RBD is not a crucial factor for species adaptation (Barlan et al., 2014) , or that time spent in humans has been too short to see such change develop. There is wide genetic diversity among the DPP4 sequences from different animal species, but those with the strongest affinity for MERS-CoV recombinant S protein are more closely related and have the most conservation among 14 amino acids within the human DPP4 sequence predicted to come into contact with the S protein (Barlan et al., 2014) .","Based on data generated from adaptation [10] and cell\u2013cell fusion studies presented here, a model of two-step adaptation process is proposed (Fig. 7). In this model, the adaptation was divided into primary and secondary adaptation (Fig. 7). Early passages of Vero-adapted IBV, including p7, p12, p14 and p20, belong to the primarily adapted group (Fig. 7). Other cell culture-adapted strains, including the CK-adapted and Beaudette-US strains CAC39114 and CAC39300 [8], [9], and the Vero-adapted strain AAV98206 described by Youn et al. [13], also belong to this group (Fig. 7). Late passages of Vero-adapted IBV, including p36, p50 and p65, contain the additional G405-D amino acid substitution and belong to the secondarily adapted group (Fig. 7).","Since the first maps used by Dr John Snow in 1854 to trace the origin of a cholera outbreak in London's Soho district, it has become evident that carefully planned and designed maps can be very powerful decision support and spatio-temporal analysis tools [10]. In the case of infectious disease epidemics, they allow public health decision makers, travellers and local populations at risk to visually monitor and appreciate at a glance changes, trends and patterns buried in large datasets that are continuously varying with time (like SARS datasets \u2013 see below). This kind of support is vital for making well-informed decisions when designing and following up epidemic control strategies or issuing and updating travel advisories.","is the (which was not peer-reviewed) The copyright holder for this preprint . https://doi.org/10.1101/2020.02.24.20026682 doi: medRxiv preprint Intelligence Network (GPHIN), identified the early SARS outbreak in China in 2003 more than two months earlier and first alerted the outbreak of Middle East Respiratory Syndrome Coronavirus (known as MERS-CoV) in 2012 [8] . As far as we know, GPHIN and other established tools do not gather data from WeChat, the dominant Chinese social media. The current study shows that gathering and analyzing data from WeChat may be amazing to early detect disease outbreak. This may have an advantage especially for detecting outbreaks in China considering the population of WeChat users. Previous studies used search query data, including Baidu Index, to monitor the activity of emerging infectious diseases [15] [16] [17] . As far as we know, this is the first time to use WeChat data in this field. As the current study indicated, using WeChat data may perform better than using Baidu search query data in early detecting the outbreak of a new disease because people may communicate first in WeChat in the times of WeChat as a lifestyle of Chinese","(2) human demographics and behavior (societal events: population growth and migration; war or civil conflicts; urban decay; sexual behavior; intravenous drug use; use of high-density facilities); (3) international travel and commerce (worldwide movement of goods and people; air travel); (4) technology and industry (globalization of food supplies; changes in food processing and packaging; organ or tissue transplantation; drugs causing immunosuppression; widespread use of antibiotics); (5) microbial adaptation and change (microbial evolution, response to selection in the environment); and (6) breakdown in public health measures (curtailment or reduction in prevention programs; inadequate sanitation and vector control measures). 9,20 A more recent article listed the following factors as influencing new and re-emerging infections: exotic pets, exotic foods (eg, bush meat), companion animals, alterations in livestock management, acquisition of new virulence factors, pathogen adaptation to new host species, changes in land use, tourism, translocation of infected animals or persons, and climate changes influencing arthropods. 21","We found that in this mouse model, DAP is more effective than VAN treatment of severe soft tissue infections caused by USA300 strains of CA-MRSA. A. Novais, I. Comas, R. Cant\u00f3n, T.M. Coque, F. Baquero, A. Moya, J.C. Gal\u00e1n (Madrid, Valencia, ES) Objectives: To define the different mutational pathways driving the diversification of blaCTX-M-1-like genes, and to establish a hypothetical evolutionary scenario among CTX-M-1-cluster enzymes. Methods: All known CTX-M-1-like sequences were aligned using clustalW. MODELTEST programme was used to establish the best model of DNA substitution and parameters for phylogenetic reconstruction. Maximum likelihood trees were estimated with PhymL. The obtained consensus tree was used to reconstruct the most likely ancestral sequence by Mesquite software. Changes in positions 77, 114, 140, 167, 240 and 288 were considered as they are frequently found among CTX-M-1-like enzymes. As distinct phenotypic profiles might be observed depending on the allelic background (epistasis), 30 combinations of mutations on these positions were constructed by site-directed mutagenesis using blaCTX-M-3 as template DNA. blaCTX-M mutants were cloned in pBGS18 plasmid and transformed into E. coli MI1443 (delta-ampC). Cefotaxime (CTX), ceftazidime (CAZ), and cefepime (FEP) MICs were determined by E-test. Results: Phylogenetic reconstruction analysis predicted that CTX-M-3 might be the ancestor of all CTX-M-1-like enzymes, except of CTX-M-10, -34, -37 and -53. The branch-site model indicated a strong positive selection in CTX-M-3 sub-branch, with five positions (A77V, N114D, P167S, D240G and D288N) being implicated. Four of these mutations (A77V, N114D, P167S, and D288N) were required in successive steps to obtain CTX-M-58, which showed the highest CAZ MIC (256 mg/mL). Distinct evolutionary pathways to obtain CTX-M-58 are possible. Moreover, A140S change (also present in CTX-M-58) was highly beneficial to the CAZ/CTX resistance phenotype when introduced in the last step, although its presence in intermediate mutants yielded a neutral or even deleterious effect (sign epistasis). In evolutionary routes involving D240G change, lower increases in MIC values than those observed among P167S route were observed, being the presence of both changes in the same background selectively inaccessible. This result would suggest that the fitness landscape in CTX-M-1 cluster is twopeaked. Conclusions: All aminoacid changes associated with positive selection by bioinformatics tools contributed to favourable resistance phenotypes in specific contexts. Different evolutionary paths leading to more efficient enzymes have been identified, indicating a rugged fitness landscape for CTX-M-1-like enzymes. (85Kb), while the plasmid with RFLP pattern J showed two replicons (RepFII+RepFIA) and its size was 100 Kb. Only plasmid belonging to pattern A is epidemic in our environment (highly related to pC15\u22121a). The plasmids were transformed in two Escherichia coli isogenic strains, REL606(ara\u2212) and REL607(ara+). Competition experiments were performed in Davis minimal (DM) broth supplemented with glucose during seven days. Each day 50 mcl from coculture were inoculated into 20mL of fresh DM and another sample was spread onto MacConkey plates with arabinose to estimate the relative fitness (W) of each plasmid alone and between them (in this case 1 mg/L cefotaxime was added to plates). The values expressed the mean of six independent experiments. The magnitude of the fitness difference corresponds to how much more slowly one strain grew relative to the other during the course of the experiments. Results: Strains harbouring the epidemic plasmid (pattern A) yielded a growth rate 9% slower than its parental strain (W=0.91\u00b10.09). On the contrary, strains harbouring the non-epidemic plasmids (pattern M and J) showed higher cost than epidemic plasmid, growing 16% and 17% slower than its parental strain (W=0.84\u00b10.08 and W=0.82\u00b10.02) respectively. However, in competition assay between isogenic strains harbouring epidemic (pattern A) and non-epidemic plasmid (pattern M), the relative fitness was next to 1. The competition assay between strains harbouring plasmids belonging to pattern A and J showed a slight advantage ( 10%) for the plasmid carrying two replicons (W=1.09\u00b10.10), although this advantage was detected after five days of competition. Conclusion: The epidemic plasmid show higher fitness, which might facilitate its dissemination. Although other posibilities can not be excluded, the presence of plasmids with multireplicons could give even more advantage in stable environments.","While the paper simulates various scenarios of containment, it does not address the economic cost of each scenario. In line with recent findings, the note argues that a fast and aggressive multi-layer intervention is needed to have a substantial dent of the transmission. What the paper calls for is a clear and transparent strategy by governments to implement social distancing and aggressive testing. The testing, should be targeted to geographic locations that the virus is wide spread, to airports and cruise ships, to road travellers, and anyone who has been in areas that are hot spot or has been in any contact with someone in affected locations over the past two weeks. The strategy should not \"just\" focus on those that show symptoms but should include \"all\" who might have been exposed. In this exceptional environment, governments can use temporarily a combination of location data and credit card information to track COVID-19 in their countries and use surveillance tools to monitor quarantined individuals.","One way to rapidly acquire novel genes that potentially facilitate host switching is recombination between different viruses. A canonical example of recombination in CoVs is FIPV type 2. The prototype strains of this feline virus are recombinants between FIPV type 1 and CCoV in different parts of the ORF1b and Spike genes (Herrewegh et al., 1998) . Recombination has also been hypothesized to be involved in the emergence of the SARS-CoV Hon et al., 2008; Lau et al., 2010a; Yuan et al., 2010) and HCoV-OC43 genotypes (Lau et al., 2011) . However, the observed recombination events in SARS-CoV and HCoV-OC43 are restricted to genetically closely related viruses. The exact recombination partners giving rise to SARS-CoV have never been detected. (Simmons, 2005) . Boxes indicate descriptions of alpha-and betacoronaviruses according to the coloring in Fig. 2 . Another phenomenon putatively associated with CoV adaptation to distinct hosts and cells is exemplified by the variable deletions spanning over 600 nucleotides in the globular S1 domain of the Spike gene that is associated with a change from enteric (TGEV) to respiratory tract (Porcine respiratory coronavirus, PRCV) tropism. Whether the 290-nucleotide deletion downstream from the Spike gene in HCoV-OC43, in comparison to its likely ancestral virus BCoV (Vijgen et al., 2006 , represents an adaptation to a human host remains unknown.","Interspecies adaptation, replication and transmission in cells are essential steps for an animal virus to emerge successfully in a human population. Virus-cell and cell\u2013cell membrane fusion, mediated by fusion proteins associated with viral envelope, is crucial for the entry of enveloped viruses into cells and for rapid spread of infection to the neighboring cells. This membrane fusion process may, therefore, be a limiting point for efficient adaptation and infection of an animal virus in cells from a different host species. In this study, we report that acquisition of the cell\u2013cell fusion activity by point mutations in the spike (S) protein of avian coronavirus infectious bronchitis virus (IBV) plays a critical role in adaptation and/or selection of a variant that infects cultured cells.","Treatments aimed at inhibiting lipopolysaccharide interaction with cells or turning off intracellular signaling pathways are under investigation. Nontoxic lipopolysaccharide or lipid A structures can act as endotoxin antagonists, if they competitively inhibit binding to LBP or cellular receptors or inhibit cellular activation by other mechanisms. Of the potential antagonists that have been evaluated experimentally, lipopolysaccharide and lipid A from the phototrophic bacterium Rhodobacter sphaeroides, and a synthetic compound (E5531) the structure of which is based on R. sphaeroides lipopolysaccharide, have been most promising. [135] [136] [137] [138] [139] Unfortunately, species differences exist regarding cellular response to these structures, and R. sphaeroides lipopolysaccharide acts as a potent inducer of cytokine expression in equine cells. 140 Based on results of receptor transfection studies in other species, 141,142 TLR4 is likely responsible for this phenotypic variation. Additional compounds including lipopolysaccharide derived from nitrogen-fixing plant bacteria of the species Rhizobium are being evaluated to reveal further insight into structural requirements for endotoxin antagonists in horses.","Results: 108 DR-TB included 44 MDR and 34 XDR, as the results of DST, and consisted of 88% lineage 2 (East-Asian) and 12% lineage 4 (Euro-American). To date, we analysed 30 isolates for 6 drugs. The Potential AMR prediction program of TGS-TB site (as of June 28th, 2017) predicted 85% of M/XDR-TB correctly. The proportion of the discrepancy between the phenotypic and the predicted DST was 6.7% in Isoniazid and Kanamycin, 10.0% in Rifampicin and Ethambutol, and 3.3% in Fluoroquinolones.","The MERS-CoV appears to be an entrenched camel virus infecting the URT which may have its origins among bats. Human infection may result from rare zoonotic spillover to humans. Many potential animal, human and environmental sources await further testing but thanks to quick action, the sensitive and rapid molecular diagnostic tools required to achieve this goal have been in place and available since the virus was made known to the public. Serological tools for research application, after a much slower start, are now emerging as robust, validated and ready for wider use, with care. Commercial unavailability remains an issue for more widespread use and so collaboration with the relevant research groups must be the immediate option for future studies. The MERS-CoV can spread from human-to-human but seems to do so only sporadically and sustained chains of transmission have not been evident to date. There is also no evidence that MERS-CoV is a virus of pandemic concern, despite many opportunities for it to become so. But vigilance is key for a virus with a genetic makeup that has only been observed for three years. Of those known to be infected, over a third have died. Central to future vigilance is continued screening, sequencing, analysis and timely data sharing. While whole genome sequencing has been used extensively to study MERS-CoV, it remains a tool for experts and collaborations have again been key for less equipped or experienced researchers to decode the MERS-CoV as it moves through people, over time and across distance in the KSA and beyond its borders. A recently described routine genotyping target which is equivalently informative will also aid this vigilance. Nonetheless, figures modelling efficiency of transmission will need confirmation once better serological and RT-rtPCR-based studies of humans, covering more of the community, have been conducted. It has become very clear that the MERS-CoV may spread poorly from human-to-human, but that spread is at its most effective around a hospital environment. Indications are that this can be traced back to poor IPC practices and protocols (Brown, 2014; Editorial, 2014; Zumla and Hui, 2014) . The virus has its greatest impact on those with underlying diseases and such cases, sometimes suffering multiple comorbidities, are likely to visit hospital for treatment, creating a perfect storm of exposure, transmission and mortality. Educational programmes will be important tools to combat the spread of MERS-CoV. While MERS and SARS have some similarities they also diverge significantly. Characteristics including the higher PFC among MERS cases (above 50% in 2013 and currently at 30-40%; well above the 9.5% of SARS), the association between fatal disease and older males with underlying comorbidities, the very broad tropism of MERS-CoV, its rapid in vitro growth, rapid induction of cytopathogenic change, robust yet distinct transcriptional responses, use of a different receptor, its induction of a more proinflammatory but reduced and delayed innate antiviral response and its sensitivity to external IFN\u2423 or IFN\u2424 all signal differences from SARS-CoV to some degree.","r. If the blood glucose level is~250 mg/dl, decrease the insulin dose to 25% of the initial dose and change the intravenous fluids to a 2.5-5% dextrose solution. 22. Monitor for oliguric renal failure. The minimum volume of urine output should be 1-2 mllkg/h. If oliguria occurs, administer furosemide. 23. Consider the administration of minidose Heparin. 10 units/kg SCq8h, to prevent microthrombi formation and the initiation of DfC. 24. Monitor for cardiac arrhythmias. Provide treatment if necessary. 25. If the patient shows signs of peritonitis, perform abdominocentesis or diagnostic peritoneal lavage and obtain peritoneal fluid samples for cytologic evaluation and culture 8. Sterile technique must be utilized. b. Administer systemic broad-spectrum antibiotics.","In conclusion, this VL-B12 appears to be very promising for CD4 targeting and a valuable mediator of biopharmaceuticals. To fill these gaps, the CCH Fever project proposes to create a multidisciplinary collaborative research environment by bringing together selected competitive advantages such as: operative capacity with appropriate high security research facilities, reference centers and clinical samples from endemic areas and an international network of e xperienced researchers. This multidisciplinary research consortium will facilitate the progress in several key research areas of the field. This program will mainly focus on (i) developing sensitive and biosafe stateof-art diagnostic tools for CCHFV, (ii) gathering the forces and resources in Europe to build a Biobank of clinical samples, (iii) building a comprehensive database consisting in clinical, laboratory and surveillance data, (iv) taking advantage of unique and state-of-art tools to progress towards vaccine candidates and specific antivirals against this bio-treat and (v) disseminating the appropriate knowledge to the health care workers in endemic regions and contributing to capacity building. These achievements will provide tools for local and European public health authorities to prevent or counter future outbreaks and monitor the spread of the disease thanks to the established novel and unique tools and resources. All patients were examined by gynecologist. There was drawn the biopsy material from cervical canal of uterus, the blood serum in each patient. At the time of relapse a PCR method and a fluorescence immunoassay were used for detection of HSV2 in biopsy material and immunoglobulin G to HSV2 in serum respectively. Also urinalysis and urine culture were done. All patients were treated by acyclovir. Results: All patients had a history of more than 4\u22128 episodes of LUTI per year. The most common complaints were frequent and painful urination, itch and burning. There was observed the relationship of LUTI relapses with menstrual cycle in each woman. The gynecological examination was displayed no abnormal changes in each patient. In the urinalysis there was revealed the increased amount of epitheliocytes only. All urine cultures were negative. All biopsy materials from cervical canal of uterus were negative for HSV2, but about 75% (45/60) patients with LUTI symptoms were found high level of immunoglobulin G to HSV2 (1:1600\u22121:3200) in blood serum. Three weeks later the antibody G titer increased twice. The patients were treated by acyclovir 400 mg three times a day for five days. Then the suppressive therapy with acyclovir 400 mg twice a day was given for 3\u22125 months. After the full course of treatment no one patient complained of LUTI.","Good social and planning policy is good climate adaptation policy. We know that long-term investment in Climate change and the people's health: the need to exit the consumptagenic system opinions in this consensus statement Comment represent those of the authors writing in their personal and independent academic roles, without any direction from their governments or institutions.","Muc4 has been detected in synovial sarcomas in humans, thus presenting a novel tissue Screening genetically diverse mouse models provides an opportunity to identify natural 163 variation in novel factors which drive viral disease responses. These studies can also provide 164 therapeutic, prophylactic and molecular insights into emerging pathogens, which are difficult to 165 study during the context of an outbreak. Here, prior phenotypic QTL analysis, bioinformatics, 166","Whether this current COVID-19 epidemic 'frizzles out' or expands into a full-blown pandemic remains to be seen. It might also be desirable to monitor farm animals and pet cats for infection with 2019-nCoV, since their ACE2 receptor responsible for 2019-nCoV binding differs in only a few amino acids from human ACE2. Surveillance might prevent the virus establishing itself in another animal species that is in close contact to humans. In addition, in light of the fact that there are multiple species of CoVs circulating in wildlife species and that these animals are constantly interacting with each other, host-species expansion or interspecies transmission of new CoV to humans seems to be inevitable. Major knowledge gaps regarding the emergence of 2019-nCoV remain exists but worldwide scientists are working with unprecedented speed to investigate the virus, rushing to develop targeted therapeutics (see Outstanding Questions). Notwithstanding, a global surveillance network involving veterinarians and animal biologists is urgently needed to monitor, and possibly to predict, potential sources for the emergence of another highly pathogenic CoV. We propose the concept of 'One Health' to facilitate scientific exchange across disciplines, sharing of data, and coordinated efforts in order to prevent future outbreaks. From surveillance and evolutionary studies on animal viruses, can their zoonotic potential be identified before interspecies transmission occurs?","The brief description of various \"new\" viruses showed that the phenotypic appearance of newly emerged virus epidemics is the result of interactions of at least three different factors.","While significant successes in controlling infectious diseases, such as antibiotics, vaccines, vector reduction, and water purification, have been achieved through highly specific technological interventions, the rate and scale of global change in agriculture, trade, demographics, species translocations and invasions, microbial adaptation, and other complex factors, have evidently outstripped our ability to understand and respond to EIDs, and exposed serious limitations of approaches that fail to engage with the wider contexts from which infectious diseases emerge. In addition to environmental change, social factors such as income inequalities and governance and social power dynamics add another layer of complexity to the task of understanding and responding to emerging infections. Responding to the EID challenges listed in the first paragraph, therefore, demands attention to the interconnections among social and ecological systems at scales ranging from the village to the globe. Accomplishing this will require new forms of knowledge that integrate the natural and human sciences, attend to dynamics at multiple scales\u2014both spatial and temporal\u2014and engage diverse ways of understanding and intervening.","Interferons are potent cytokines of critical importance in controlling viral infections and priming adaptive immune responses [54] . Several studies of antiviral treatments tested against SARS-CoV replication show administration of Type I IFN inhibits SARS-CoV growth in cell culture as well as viral replication in cynomolgus macaques and mouse models [39,40 ,55-59] . Despite the potential importance of IFNs in controlling SARS-CoV replication, infection of mice deficient in Type I, II, or III IFN receptors showed minimal phenotypic difference in weight loss, viral titer, lung pathology, and mortality from wild-type mice in the MA15-SARS-CoV model [60 ] . However, mice deficient in STAT1, a critical transcription factor for IFN signaling (Figure 2b) , were significantly more susceptible to MA15-SARS-CoV infection than wild-type or IFNAR1 \u00c0/\u00c0 mice [60 ] . Transcriptional analysis from these studies showed that ISGs were induced even in the absence of IFNAR1, demonstrating that there may be compensatory mechanisms through other innate immune signaling to protect against severe SARS-CoV disease in the absence of IFN [50 ] . In the SARS-CoV infection model, mice deficient in Type I, [47, 99] . TLR2/6 heterodimers help to activate the innate immune response to RSV, though the viral PAMP recognized has not been determined [100]. TLR1/2 heterodimers have been shown to recognize viral glycoproteins, though their potential role in respiratory virus infection has not been determined [99] . While there are many TLRs that recognize viral PAMPs, they signal through common adaptor molecules, including MyD88, MAL, TRAM, and TRIF. The TLR adaptor molecules signal through the IKKe/TBK1 complex and the IKKa/IKKb/IKKg complex similarly to RLRs, but can also recruit an IRAK-1/IRAK4/TRAF6 complex capable of activating the transcription factors IRF3, IRF7, and NF-kB. Activation of these transcription factors leads to the transcription of Type I IFNs and proinflammatory cytokines. Due to the considerable crosstalk between TLR and RLR signaling, it is difficult to discriminate between transcriptional products generated by the two sensor families, but it is likely that both play an important role in the innate immune response to SARS-CoV infection.","Many viruses, particularly RNA viruses, can rapidly adapt to changes in their hosts due to the high mutation rates of these viruses. Such viral adaptation to hosts imposes huge threats to human health because it allows viruses to emerge from animals to infect humans. Understanding viral adaptation to hosts would allow humans to track viral evolution that may lead to viral epidemics, but current knowledge of the mechanisms of viral adaptation to hosts is limited. This study investigates the mechanisms of host receptor adaptation by the severe acute respiratory syndrome coronavirus (SARS-CoV),3 the agent of the SARS epidemic, and provides a molecular and structural basis for monitoring future SARS-CoV evolution in animals.","After incubation at 37\u00b0C for 1 h the lysed cells were deposited by centrifugation and the optical density of the supernatant fluid determined using a Unicam SP 1750 spectrophotometer to estimate the degree of haemolysis compared with RBC incubated in the absence of virus. pH maximum is the higher pH at which significant haemolysis occurred Influenza B,'HK/73 virus showed a high pH optimum and a high pH maximum of haemolysis, but B/Le/40 was similar to influenza A viruses in its pH profile although neither virus is inhibited by amantadine (Table 7 .9). Finally, as anticipated, high concentrations of added amantadine did not inhibit haemolysis of red blood cells by amantadine-sensitive influenza A viruses. Thus, further work is required to investigate and establish any role for a fusion event, to determine if low pH haemolysis of red blood cells is caused by a fusion process and to look for a correlation between pH optimum for this biological event and amantadine resistance o r susceptibility. Genetic and phenotypic heterogeneity of viruses may be of considerable practical importance in attempts to control certain virus diseases by chemo-or immunoprophylaxis (see Chapter 17). Thus, certain naturally circulating influenza A viruses may be resistant to antiviral drugs such as amantadine by virtue of mutations in gene 7 or may differ in virulence, antigenic or biological properties and thus may be able to circumvent drug induced protection. Studies of influenza A and B viruses circulating in the community have shown that quite extensive genetic and phenotypic variation occurs. Thus, a number of recently isolated influenza viruses of both H3N2 and H IN1 antigenic subtypes have a temperature sensitive ( t s ) phenotype and differ in virulence for volunteers (Chapter 17). Laboratory studies of artificially induced influenza ts mutants have demonstrated clearly that such mutants are attenuated for man and that the shut-off temperature is related to attenuation. The occurrence of non-ts and ts viruses in nature probably indicates that influenza A viruses of varying virulence are circulating in the community. In our studies, even viruses isolated from the same city varied considerably in the phenotypic ts character. Influenza A and B viruses circulating in the community may also differ in respect to the biological property of plaquing in MDCK cells and to the electrophoretic properties of structural and non-structural polypeptides and RNA. Finally, analysis of field isolates of influenza A and B viruses using panels of monoclonal antibodies to virus HA indicates a considerable degree of antigenic heterogeneity even among viruses isolated in rather circumscribed outbreaks in single towns or schools (Chapter 17). It is quite possible that biological techniques currently used to estimate the degree of inhibition of viruses by amantadine are too insensitive and that significant but small differences in drug resistance between viruses may be missed. These differences might, however, have a significant effect on the circulation of viruses in the community. 7.9.5. PHARMACOLOGY OF AMANTADINE In summary, it has been established that, following oral administration of amantadine to man or mice, concentrations of amantadine could be attained in the tissues which would be expected to have influenza A virus inhibitory effects (around 1 pg amantadine/g tissue). However, what does not appear to be clearly established is the exact intracellular localization of administered amantadine and the correlation with antiviral activity (Richman et al., 1981) .","The virulent nature of this virus and its high rate of transmissibility warrants robust, rapid, sensitive, 51 specific, and quantitative diagnostic tools to supplement clinical symptoms aiding clinicians to 52 confidently rule in and rule out patients. Moreover, such a diagnostic tool will help with preventing 53 spread of virus by identifying the infected cases and can monitor the health status of infected patients 54 by quantifying the viral load. Center for Disease Control was the first to develop a quantitative 55 reverse transcriptase real-time PCR (RT-PCR), which later became the gold standard technique 56 (https://www.cdc.gov/coronavirus/2019-ncov/about/testing.html). Subsequently, a Chinese group 57 used a RNA-based metagenomics next generation sequencing (mNGS) to diagnose the viral RNA 58 from the clinical samples of two patients (3). However, the requirement for advanced technology and 59 skilled personnel and long turn-around time (24 hours) are not feasible for local and referral 60 laboratories. Therefore, a colorimetric loop mediated isothermal amplification, also known as LAMP, 61 was developed to obviate the need for expensive technologies, e.g. real-time PCR and NGS, as well 62 as to shorten the turn-around time to up to 40 minutes (4). However, this assay was a qualitative one 63 and also only the swab samples from limited number of patients (n=7) were included for testing (4). 64"],"title":["MERS coronavirus: diagnostics, epidemiology and transmission","Expecting the unexpected: nucleic acid-based diagnosis and discovery of emerging viruses","The species Severe acute respiratory syndrome-related coronavirus: classifying 2019-nCoV and naming it SARS-CoV-2","Advances in respiratory virus therapeutics \u2013 A meeting report from the 6th isirv Antiviral Group conference","Microbial molecular markers and epidemiological surveillance in the era of high throughput sequencing: an update from the IMMEM-10 conference","Angiotensin-Converting Enzyme 2: The First Decade","CHAPTER 21 The Widespread Evolutionary Significance of Viruses","How will this continue? Modelling interactions between the COVID-19 pandemic and policy responses","New regulatory strategies to manage medicines shortages in Europe","Orthogonal genome-wide screenings in bat cells identify MTHFD1 as a target of broad antiviral therapy","Numerical entries and A","Advances in respiratory virus therapeutics \u2013 A meeting report from the 6th isirv Antiviral Group conference","MERS coronavirus: diagnostics, epidemiology and transmission","Changes in nonstructural protein 3 are associated with attenuation in avian coronavirus infectious bronchitis virus","Evolution of Viruses","Microbial molecular markers and epidemiological surveillance in the era of high throughput sequencing: an update from the IMMEM-10 conference","The species Severe acute respiratory syndrome-related coronavirus: classifying 2019-nCoV and naming it SARS-CoV-2","Network-based analysis of comorbidities risk during an infection: SARS and HIV case studies","Emerging Diseases: Overview","Strong evolutionary convergence of receptor-binding protein spike between COVID-19 and SARS-related coronaviruses","Early epidemiological analysis of the coronavirus disease 2019 outbreak based on crowdsourced data: a population-level observational study","Early epidemiological analysis of the coronavirus disease 2019 outbreak based on crowdsourced data: a population-level observational study","Computers and viral diseases. Preliminary bioinformatics studies on the design of a synthetic vaccine and a preventative peptidomimetic antagonist against the SARS-CoV-2 (2019-nCoV, COVID-19) coronavirus","Emerging infections and sensationalism: in Aesop's fable, the boy who cried wolf had a dreadful fate","CHAPTER 21 The Widespread Evolutionary Significance of Viruses","Severe acute respiratory syndrome-related coronavirus: The species and its viruses \u2013 a statement of the Coronavirus Study Group","Middle East respiratory syndrome: An emerging coronavirus infection tracked by the crowd","Acquisition of Cell\u2013Cell Fusion Activity by Amino Acid Substitutions in Spike Protein Determines the Infectivity of a Coronavirus in Cultured Cells","Descriptive review of geographic mapping of severe acute respiratory syndrome (SARS) on the Internet","WeChat, a Chinese social media, may early detect the SARS-CoV-2 outbreak in 2019","Emerging infectious diseases: Focus on infection control issues for novel coronaviruses (Severe Acute Respiratory Syndrome-CoV and Middle East Respiratory Syndrome-CoV), hemorrhagic fever viruses (Lassa and Ebola), and highly pathogenic avian influenza viruses, A(H5N1) and A(H7N9)","Oral presentations","Modeling of COVID-19 Pandemic and Scenarios for Containment","Ecology, evolution and classification of bat coronaviruses in the aftermath of SARS","Acquisition of Cell\u2013Cell Fusion Activity by Amino Acid Substitutions in Spike Protein Determines the Infectivity of a Coronavirus in Cultured Cells","Chapter 13 Disorders of the Gastrointestinal System","Poster Sessions","Middle East respiratory syndrome: An emerging coronavirus infection tracked by the crowd","Chapter 7 Gastrointestinal Emergencies","Abstracts accepted for publication only","Do not violate the International Health Regulations during the COVID-19 outbreak","Mucin 4 Protects Female Mice from Coronavirus Pathogenesis","COVID-19: Epidemiology, Evolution, and Cross-Disciplinary Perspectives","There is nothing permanent except change. The emergence of new virus diseases","All Hands on Deck: Transdisciplinary Approaches to Emerging Infectious Disease","SARS coronavirus pathogenesis: host innate immune responses and viral antagonism of interferon","Mechanisms of Host Receptor Adaptation by Severe Acute Respiratory Syndrome Coronavirus","Chapter 7 Orthomyxovirus infections","Development and validation of a rapid single-step reverse transcriptase loop-mediated isothermal amplification (RT-LAMP) system potentially to be used for reliable and high-throughput screening of COVID-19"],"url":["https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4687373/","https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7103685/","https://doi.org/10.1038/s41564-020-0695-z","https://doi.org/10.1016/j.antiviral.2019.04.006","https://doi.org/10.1016/j.resmic.2014.01.003","https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3216391/","https://doi.org/10.1016/b978-0-12-374153-0.00021-7","https://doi.org/10.1101/2020.03.30.20047597","https://doi.org/10.1016/j.ijpharm.2020.119171","https://doi.org/10.1101/2020.03.29.014209","https://doi.org/10.1016/b978-0-12-384931-1.00001-5","https://doi.org/10.1016/j.antiviral.2019.04.006","https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4687373/","https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7089577/","https://doi.org/10.1016/b978-012374410-4.00706-8","https://doi.org/10.1016/j.resmic.2014.01.003","https://doi.org/10.1038/s41564-020-0695-z","https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4363349/","https://doi.org/10.1016/b978-012373960-5.00453-6","https://doi.org/10.1101/2020.03.04.975995","https://doi.org/10.1016/s2589-7500(20)30026-1","https://doi.org/10.1016/s2589-7500(20)30026-1","https://doi.org/10.1016/j.compbiomed.2020.103670","https://doi.org/10.1111/1469-0691.12606","https://doi.org/10.1016/b978-0-12-374153-0.00021-7","https://doi.org/10.1101/2020.02.07.937862","https://doi.org/10.1016/j.virusres.2015.01.021","https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2700284/","https://www.ncbi.nlm.nih.gov/pmc/articles/PMC343293/","https://doi.org/10.1101/2020.02.24.20026682","https://doi.org/10.1016/j.ajic.2015.11.018","https://doi.org/10.1111/j.1469-0691.2008.02006.x","https://doi.org/10.1101/2020.03.27.20045849","https://doi.org/10.1016/j.antiviral.2013.10.013","https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2700284/","https://doi.org/10.1016/b0-72-169777-1/50015-9","https://doi.org/10.1016/s0924-8579(17)30423-5","https://doi.org/10.1016/j.virusres.2015.01.021","https://doi.org/10.1016/b978-0-7020-2487-0.50011-3","https://doi.org/10.1111/j.1469-0691.2011.03559.x","https://doi.org/10.1016/s0140-6736(20)30373-1","https://doi.org/10.1101/2020.02.19.957118","https://doi.org/10.1016/j.molmed.2020.02.008","https://doi.org/10.1016/0378-1135(95)92531-f","https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7087652/","https://doi.org/10.1016/j.coviro.2012.04.004","http://www.jbc.org/content/287/12/8904.full.pdf","https://doi.org/10.1016/s0168-7069(08)70015-6","https://doi.org/10.1101/2020.03.15.20036376"],"x":[0.5763328892222823,0.5994986632954058,0.6198764193075571,0.5570818263586846,0.5543632526218407,0.7982468127311548,0.7971451394486233,0.8565805958105089,0.43744326953436075,0.5784288976277875,0.4771326256098573,0.4757492342241522,0.8775629426691346,0.7926555541794167,0.6096327655911155,0.5766454097118481,0.6027500040484652,0.32498264217376377,0.4858305397253691,0.7526613038513859,0.5180574691390523,0.5180580009226193,0.7692569570485023,0.5033904897849834,0.5877942271598932,0.6020180855885722,0.8814850017054628,0.4701968809982444,0.5236458774911067,0.5331385887163158,0.4945772111833623,0.38642671817643137,0.5509438993073122,0.8660909189863989,0.5415774611416684,0.47168468323019913,0.4244266098768552,0.8698324720525952,0.18539517422736876,0.2876313261301129,0.47395150232012356,0.566839440910999,0.7610691525642287,0.5875023671128697,0.5144185002275302,0.7451510176700061,0.6073332573346849,0.5595222933135287,0.4868860944535945],"y":[0.44108828414701445,0.5393260711284913,0.4635520488314874,0.5366968023613173,0.4351656014331177,0.25210047356566323,0.3741744966293078,0.40894475567722155,0.6464754657958239,0.3936749564991183,0.23698905698570386,0.4099814653380128,0.5342663447785687,0.3731559610401738,0.4896152576944289,0.39252287188973267,0.46932408214931615,0.6161562293046248,0.6974844223631522,0.30246864207493784,0.8417160682231322,0.8417178527315395,0.5942054798823744,0.6594645136895073,0.4400336665954292,0.4665768753211928,0.47599416483695567,0.3307345673644297,0.6918390053999524,0.8589764815449878,0.6946632342029608,0.40335502733951617,0.7250113418469404,0.3597665651653571,0.23354361987051905,0.2789977905692359,0.4308244786740628,0.5188064306544967,0.3925064623878415,0.6441434821722675,0.7164532183594173,0.4491017577238334,0.5283164049449146,0.485990069739087,0.6811901957273743,0.41583319739333763,0.4470341136297392,0.5405683412038295,0.368878564172971]},"selected":{"id":"3973"},"selection_policy":{"id":"4002"}},"id":"3900","type":"ColumnDataSource"},{"attributes":{"formatter":{"id":"4005"},"ticker":{"id":"3950"}},"id":"3949","type":"LinearAxis"},{"attributes":{"callback":null,"tooltips":[["Title","@title"],["Abstract","@text"],["Published","@date"],["Authors","@authors"],["Journal","@journal"],["URL","@url"]]},"id":"3920","type":"HoverTool"},{"attributes":{},"id":"3999","type":"StringEditor"},{"attributes":{"children":[{"id":"3975"}]},"id":"3978","type":"Row"},{"attributes":{"formatter":{"id":"3997"},"ticker":{"id":"3911"}},"id":"3910","type":"LinearAxis"},{"attributes":{},"id":"3990","type":"StringFormatter"},{"attributes":{},"id":"3911","type":"BasicTicker"},{"attributes":{"active_drag":"auto","active_inspect":"auto","active_multi":null,"active_scroll":"auto","active_tap":"auto","tools":[{"id":"3953"},{"id":"3954"},{"id":"3955"}]},"id":"3956","type":"Toolbar"},{"attributes":{"data":{"authors":["Yonghua Wu","Alba Grifoni; John Sidney; Yun Zhang; Richard H Scheuermann; Bjoern Peters; Alessandro Sette","Grifoni, Alba; Sidney, John; Zhang, Yun; Scheuermann, Richard H.; Peters, Bjoern; Sette, Alessandro","Ziv Klausner; Eyal Fattal; Eitan Hirsch; Shmuel C Shapira","Fang, Shou Guo; Shen, Shuo; Tay, Felicia P.L.; Liu, D.X.","Richard E. Grewelle","Mousavizadeh, Leila; Ghasemi, Sorayya","Domachowske, Joseph B.","Paintsil, E.; Cheng, Y.-C.","Paintsil, E.; Cheng, Yung-Chi","Gao, Fei; Yao, Huochun; Lu, Jiaqi; Wei, Zuzhang; Zheng, Haihong; Zhuang, Jinshan; Tong, Guangzhi; Yuan, Shishan","Goldenberg, Simon","Longxian Lv; Gaolei Li; Jinhui Chen; Xinle Liang; Yudong Li","Stockman, Lauren J; Bellamy, Richard; Garner, Paul","Ethan Fast; Russ B. Altman; Binbin Chen","Tong, Tommy R.","Daughton, Christian","Shi, Weifeng; Li, Juan; Zhou, Hong; Gao, George F.","Honkavuori, Kirsi S.; Briese, Thomas; Krauss, Scott; Sanchez, Maria D.; Jain, Komal; Hutchison, Stephen K.; Webster, Robert G.; Lipkin, W. Ian","Santosh, K. C.","Demurtas, Olivia C.; Massa, Silvia; Illiano, Elena; De Martinis, Domenico; Chan, Paul K. S.; Di Bonito, Paola; Franconi, Rosella","Al-Tawfiq, Jaffar A; Zumla, Alimuddin; Gautret, Philippe; Gray, Gregory C; Hui, David S; Al-Rabeeah, Abdullah A; Memish, Ziad A","Marek Prachar; Sune Justesen; Daniel B Steen-Jensen; Stephan P Thorgrimsen; Erik Jurgons; Ole Winther; Frederik Otzen Bagger","Tomar, Shailly; Mahajan, Supreeti; Kumar, Ravi","Rizzo, Francesca; Edenborough, Kathryn M.; Toffoli, Roberto; Culasso, Paola; Zoppi, Simona; Dondo, Alessandro; Robetto, Serena; Rosati, Sergio; Lander, Angelika; Kurth, Andreas; Orusa, Riccardo; Bertolotti, Luigi; Mandola, Maria Lucia","Fateme Rafiei Nasab; Fakher rahim","Mandary, Madiiha Bibi; Masomian, Malihe; Poh, Chit Laa","Lomongo, Sheila; Alyousef, Khalid; Mustonen, Reetta; Othman, Khalid","Daniel L. Hurdiss; Ieva Drulyte; Yifei Lang; Tatiana M. Shamorkina; Matti F. Pronker; Frank J.M. van Kuppeveld; Joost Snijder; Raoul J. de Groot","Dan Zhang; Rui Guo; Lei Lei; Hongjuan Liu; Yawen Wang; Yili Wang; Tongxin Dai; Tianxiao Zhang; Yanjun Lai; Jingya Wang; Zhiqiang Liu; Aili He; Michael O'Dwyer; Jinsong Hu","Bishajit Sarkar; Md. Asad Ullah; Fatema Tuz Johora; Masuma Afrin Taniya; Yusha Araf","Truyen, Uwe; Parrish, Colin R.; Harder, Timm C.; Kaaden, Oskar-R\u00fcger","Moni, Mohammad Ali; Li\u00f2, Pietro","Villarreal, L.P.","Amna Tariq; Yiseul Lee; Kimberlyn Roosa; Seth Blumberg; Ping Yan; Stefan Ma; Gerardo Chowell","Wu, Kailang; Peng, Guiqing; Wilken, Matthew; Geraghty, Robert J.; Li, Fang","Ming Wang; Aisi Fu; Ben Hu; Yongqing Tong; Ran Liu; Jiashuang Gu; Jianghao Liu; Wen Jiang; Gaigai Shen; Wanxu Zhao; Dong Men; Lilei Yu; Zixin Deng; Yan Li; Tiangang Liu","Tianzhi Wu; Xijin Ge; Guangchuang Yu; Erqiang Hu","Li Xiaoxuan; Wu Qi; Lv Benfu","Bogoch, Isaac I; Watts, Alexander; Thomas-Bachli, Andrea; Huber, Carmen; Kraemer, Moritz U G; Khan, Kamran","Yasuhiko Kamikubo; Atsushi Takahashi","Mayer, J.D.","Husnayain, Atina; Fuad, Anis; Su, Emily Chia-Yu","Gertjan Medema; Leo Heijnen; Goffe Elsinga; Ronald Italiaander; Anke Brouwer","Alexander Lachmann","Schwartz, Stephan A.","Arunachalam Ramaiah; Vaithilingaraja Arumugaswami","Fung, Sin-Yee; Yuen, Kit-San; Ye, Zi-Wei; Chan, Chi-Ping; Jin, Dong-Yan","Zehua Zeng; Zhi Luo; Hongwu Du","Garraud, Olivier"],"color":["#67a9cf","#67a9cf","#67a9cf","#67a9cf","#67a9cf","#67a9cf","#67a9cf","#67a9cf","#67a9cf","#67a9cf","#67a9cf","#67a9cf","#67a9cf","#67a9cf","#67a9cf","#67a9cf","#67a9cf","#67a9cf","#67a9cf","#67a9cf","#67a9cf","#67a9cf","#67a9cf","#67a9cf","#67a9cf","#67a9cf","#67a9cf","#67a9cf","#67a9cf","#67a9cf","#67a9cf","#67a9cf","#67a9cf","#67a9cf","#67a9cf","#67a9cf","#67a9cf","#67a9cf","#67a9cf","#67a9cf","#67a9cf","#67a9cf","#67a9cf","#67a9cf","#67a9cf","#67a9cf","#67a9cf","#67a9cf","#67a9cf","#67a9cf"],"date":["2020-03-04","2020-02-20","2020-03-16","2020-03-30","2005-10-21","2020-03-11","2020-03-31","2004-10-15","2019-12-31","2009-12-31","2013-04-25","2013-11-30","2020-03-02","2006-09-12","2020-02-21","2006-12-31","2020-03-23","2017-11-28","2014-04-03","2020-03-18","2016-02-05","2014-10-31","2020-03-21","2020-12-31","2017-12-22","2020-03-23","2019-09-19","2017-09-30","2020-03-25","2020-03-26","2020-02-11","1995-02-28","2014-10-24","2008-12-31","2020-02-25","2012-01-30","2020-03-06","2020-02-27","2020-02-18","2020-01-27","2020-03-27","2008-12-31","2020-03-12","2020-03-30","2020-03-18","2020-03-12","2020-01-30","2020-03-14","2020-03-02","2020-03-31"],"doc":["4ihv80au","8p1agcm2","bc3108hi","32hjzdum","h8cemq2n","y12cq60a","p3gswh23","4dqz8req","aqlxb6ab","itz0bdrc","s63clavw","n8hlrknq","3h1o0oz3","j35w1vsw","j6y806qu","9qd61n3p","vby3aan7","k0f9fser","7gx4fowz","j97ugs3y","kmpbdvof","k560qnyi","wxagjqbt","mog52dn8","llxqcikz","a7u8xdux","kjqtvcfg","g6xvim0s","5lld270m","nlavfnpt","7vrm081c","19z4ej76","3qgzulgj","01a8er2v","vw3h1m5y","c4q5gtw2","s4d442kx","qtrquuvv","vv1wjen0","9fkrf3tf","7onenxg7","t99l3zii","vnnwnxs2","fb62dwga","gttuxtw6","nirq7mty","xxcx7hg5","89fol3pq","fvig79k3","dvz2lg9e"],"journal":["","","Cell Host &amp; Microbe","","Biochemical and Biophysical Research Communications","","Journal of Microbiology, Immunology and Infection","Clinical Microbiology Newsletter","Encyclopedia of Microbiology","Encyclopedia of Microbiology","Virology","Medicine","","PLoS Med","","Perspectives in Medical Virology","Science of The Total Environment","Sci China Life Sci","PLoS One","J Med Syst","Front Plant Sci","The Lancet Infectious Diseases","","Genomics and Biotechnological Advances in Veterinary, Poultry, and Fisheries","BMC Vet Res","","Int J Mol Sci","Journal of Radiology Nursing","","","","Veterinary Microbiology","BMC Bioinformatics","Encyclopedia of Virology","","Journal of Biological Chemistry","","","","J Travel Med","","International Encyclopedia of Public Health","International Journal of Infectious Diseases","","","EXPLORE","","Emerg Microbes Infect","","Transfusion Clinique et Biologique"],"rad":[0.02,0.01116632803206527,0.010263471791000397,0.010204786673074628,0.009133902771251654,0.008951525232472703,0.00892204417391655,0.007750727897493092,0.007666086474546839,0.007663606780389917,0.0070367909358532545,0.0069308010608784346,0.006426034650195591,0.006391398622831904,0.006376786114523004,0.006228455637170756,0.005698993164230269,0.005401854364888468,0.005199798667576667,0.005083948016447465,0.004947794676601111,0.00472851123366871,0.004701321551927346,0.004575657107908943,0.004518253152012379,0.004471653002087635,0.004341029075596924,0.004329552039647417,0.0041933794266892956,0.004192820658745091,0.004103937447423728,0.003913363339390541,0.0038722145275556052,0.003869125546444469,0.0037892542816531476,0.0036208781727565336,0.003540229529425316,0.003499060281869447,0.0033847164613185774,0.0033838542893895856,0.003318280124185006,0.0032893109759297608,0.003251116771499326,0.0031994917202769397,0.0031765420629111117,0.0031523316727762474,0.0030506722511227146,0.0030206001182225242,0.0030175723937551123,0.003],"scr":[1.0,0.48,0.427,0.424,0.361,0.35,0.348,0.279,0.274,0.274,0.237,0.231,0.202,0.199,0.199,0.19,0.159,0.141,0.129,0.123,0.115,0.102,0.1,0.093,0.089,0.087,0.079,0.078,0.07,0.07,0.065,0.054,0.051,0.051,0.046,0.037,0.032,0.029,0.023,0.023,0.019,0.017,0.015,0.012,0.01,0.009,0.003,0.001,0.001,0.0],"text":["AbstractCoronavirus Disease 2019 (COVID-19) and severe acute respiratory syndrome (SARS)-related coronaviruses (e.g., 2019-nCoV and SARS-CoV) are phylogenetically distantly related, but both are capable of infecting human hosts via the same receptor, angiotensin-converting enzyme 2, and cause similar clinical and pathological features, suggesting their phenotypic convergence. Yet, the molecular basis that underlies their phenotypic convergence remains unknown. Here, we used a recently developed molecular phyloecological approach to examine the molecular basis leading to their phenotypic convergence. Our genome-level analyses show that the spike protein, which is responsible for receptor binding, has undergone significant Darwinian selection along the branches related to 2019-nCoV and SARS-CoV. Further examination shows an unusually high proportion of evolutionary convergent amino acid sites in the receptor binding domain (RBD) of the spike protein between COVID-19 and SARS-related CoV ...","ABSTRACTEffective countermeasures against the recent emergence and rapid expansion of the 2019-Novel Coronavirus (2019-nCoV) require the development of data and tools to understand and monitor viral spread and immune responses. However, little information about the targets of immune responses to 2019-nCoV is available. We used the Immune Epitope Database and Analysis Resource (IEDB) resource to catalog available data related to other coronaviruses, including SARS-CoV, which has high sequence similarity to 2019-nCoV, and is the best-characterized coronavirus in terms of epitope responses. We identified multiple specific regions in 2019-nCoV that have high homology to SARS virus. Parallel bionformatic predictions identified a priori potential B and T cell epitopes for 2019-nCoV. The independent identification of the same regions using two approaches reflects the high probability that these regions are targets for immune recognition of 2019-nCoV.ONE SENTENCE SUMMARYWe identified potential...","Summary Effective countermeasures against the recent emergence and rapid expansion of the 2019 novel coronavirus (SARS-CoV-2) require the development of data and tools to understand and monitor its spread and immune responses to it. However, little information is available about the targets of immune responses to SARS-CoV-2. We used the Immune Epitope Database and Analysis Resource (IEDB) to catalog available data related to other coronaviruses. This includes SARS-CoV, which has high sequence similarity to SARS-CoV-2 and is the best-characterized coronavirus in terms of epitope responses. We identified multiple specific regions in SARS-CoV-2 that have high homology to the SARS-CoV virus. Parallel bioinformatic predictions identified a priori potential B and T cell epitopes for SARS-CoV-2. The independent identification of the same regions using two approaches reflects the high probability that these regions are promising targets for immune recognition of SARS-CoV-2. These predictions c...","Background: The impact of COVID-19 has been profound, and the public health challenge seem to be the most serious regarding respiratory viruses since the 1918 H1N1 influenza pandemic. In the absence of effective vaccine or biomedical treatment, the basic rules of public health measures have not changed, namely public distancing. Methods: We analyzed epidemiological investigation reports during the first month of the outbreak in Israel. In addition, we present a deterministic compartment model and simulations of several scenarios emphasizing quarantine and isolation policies given their efficiency. Results: We identify an abrupt change from controlled epidemic regime to an exponential growth (R_0= 2.19) in light of the actual policy-makers decisions and public behavior in Israel. Our analysis show that before the abrupt change, the new cases trend was due to returning citizens infected abroad. The abrupt change followed a holiday in which social distancing was clearly inefficient and ma...","Abstract An interesting question posed by the current evidence that severe acute respiratory syndrome coronavirus may be originated from an animal coronavirus is how such an animal coronavirus breaks the host species barrier and becomes zoonotic. In this report, we study the chronological order of genotypic changes in the spike protein of avian coronavirus infectious bronchitis virus (IBV) during its adaptation to a primate cell line. Adaptation of the Beaudette strain of IBV from chicken embryo to Vero cells showed the accumulation of 49 amino acid mutations. Among them, 26 (53.06%) substitutions were located in the S protein. Sequencing analysis and comparison of the S gene demonstrated that the majority of the mutations were accumulated and fixed at passage 7 on Vero cells and minor variants were isolated in several passages. Evidence present suggests that the dominant Vero cell-adapted IBV strain may be derived from the chicken embryo passages by selection of and potential recombin...","Emergence of new viral diseases is linked to mutation or recombination events. The likelihood of cross-species transmission is related to phenotypic plasticity of a virus and its capacity to produce genetically variable progeny. Herein a model is described connecting the production of genetic variability with increasing genome size. Comparing all known zoonotic viral genome sizes to known non-zoonotic viral genome sizes demonstrates that zoonotic viruses have significantly larger genomes. These results support the notion that large viral genomes are important in producing new zoonotic disease, and suggest that genome size may be a useful surrogate in screening for potential zoonotic viruses.","Abstract COVID-19 is a novel coronavirus with an outbreak of unusual viral pneumonia in Wuhan, China, and then pandemic. Based on its phylogenetic relationships and genomic structures the COVID-19 belongs to genera Betacoronavirus. Human Betacoronaviruses (SARS-CoV-2, SARS-CoV, and MERS-CoV) have many similarities, but also have differences in their genomic and phenotypic structure that can influence their pathogenesis. COVID-19 is containing single-stranded (positive-sense) RNA associated with a nucleoprotein within a capsid comprised of matrix protein. A typical CoV contains at least six ORFs in its genome. All the structural and accessory proteins are translated from the sgRNAs of CoVs. Four main structural proteins are encoded by ORFs 10, 11 on the one\u2010third of the genome near the 3\u2032\u2010terminus. The genetic and phenotypic structure of COVID-19 in pathogenesis is important. This article highlights the most important of these features compared to other Betacoronaviruses.","Abstract Advances in molecular biology, bioinformatics, and robotics have allowed microarray technology to be used for in-depth, basic science studies in all fields of microbiology. Recently, translation of these basic science applications to clinical microbiology and infectious diseases has also progressed. From a clinical infectious disease perspective, genome-based organism identification, pathogen discovery, and antimicrobial susceptibility testing of problematic organisms offer the potential to yield diagnostic information that may not otherwise become available. Moreover, microarray-based studies have the ability to provide \u201csignatures\u201d of host cell transcriptional responses for individual pathogens and/or groups of pathogens. This type of information has the potential to confirm difficult diagnoses, to monitor responses to therapeutic intervention, or even to predict prognosis and sequelae following an infectious disease. Examples are presented to illustrate ways in which microa...","Abstract Antiviral agents are drugs approved in the USA by the Food and Drug Administration (FDA) for the treatment or control of viral infections. Available antiviral agents mainly target stages in the viral life cycle. The target stages in the viral life cycle are; viral attachment to host cell, uncoating, synthesis of viral mRNA, translation of mRNA, replication of viral RNA and DNA, maturation of new viral proteins, budding, release of newly synthesized virus, and free virus in body fluids. Two important factors that can limit the utility of antiviral drugs are toxicity and the development of resistance to the antiviral agent by the virus. In addition, host phenotypic behaviors toward antiviral drugs because of either genomic or epigenetic factors could limit the efficacy of an antiviral agent in an individual. This article summarizes the most relevant pharmacologic and clinical properties of current antiviral agents, and targets for novel antiviral agents.","Antiviral agents are drugs approved by the Food and Drug Administration (FDA) for the treatment or control of viral infections. Available antiviral agents mainly target stages in the viral life cycle. The target stages in the viral life cycle are; viral attachment to host cell, uncoating, synthesis of viral mRNA, translation of mRNA, replication of viral RNA and DNA, maturation of new viral proteins, budding, release of newly synthesized virus, and free virus in body fluids. At least half of available antiviral agents are for the treatment of human immunodeficiency virus (HIV) infections. The others are used mainly for the management of herpesviruses, hepatitis B virus (HBV), hepatitis C virus (HCV), and respiratory viruses. Antiviral agents can be used for prophylaxis, suppression, preemptive therapy, or treatment of overt disease. Two important factors that can limit the utility of antiviral drugs are toxicity and the development of resistance to the antiviral agent by the virus. In ...","Abstract The 5\u2032 untranslated region (UTR) is believed to be vital for the replication of porcine reproductive and respiratory syndrome virus (PRRSV), yet its functional mechanism remains largely unknown. In this study, to define the cis-acting elements for viral replication and infectivity, The 5\u2032 UTR swapping chimeric clones pTLV8 and pSHSP5 were constructed based on two different genotypes full-length infectious cDNA clone pAPRRS and pSHE backbones. Between them, vTLV8 could be rescued from pTLV8 and had similar virological properties to vAPRRS, including phenotypic characteristic and RNA synthesis level. However, pSHSP5 exhibited no evidence of infectivity. Taken together, the results presented here demonstrate that only the 5\u2032 UTR of type 1 PRRSV did not affect the infectivity and replication of type 2 PRRSV in vitro. The 5\u2032 UTR of type 2 PRRSV could be functionally replaced by its counterpart from type 1.","Abstract Microbiology laboratories have traditionally relied upon phenotypic methods involving culture and biochemical testing to identify and characterize clinically important pathogens. These techniques have several disadvantages including poor sensitivity and long turn-around time. Molecular and mass spectroscopy techniques are rapidly changing infection diagnosis and management. Compared with conventional culture-based techniques, these modern approaches provide substantially more rapid and specific information on organism identification and on the presence of resistance mechanisms. These methods are expected to contribute substantially to enhancing antibiotic stewardship and to improving \u2018time to appropriate antibiotics\u2019, one of the most important factors in improving the prognosis of patients with life-threatening infections. This article gives an overview of some of the practical applications of these newer technologies.","AbstractThe novel coronavirus SARS-CoV-2 (2019-nCoV) is a member of the family coronaviridae and contains a single-stranded RNA genome with positive-polarity. To reveal the evolution mechanism of SARS-CoV-2 genome, we performed comprehensive genomic analysis with newly sequenced SARS-CoV-2 strains and 20 closely related coronavirus strains. Among 98 nucleotide mutations at 93 sites of the genome among different SARS-CoV-2 strains, 58 of them caused amino acid change, indicating a result of neutral evolution. However, the ratio of nucleotide substitutions to amino acid substitutions of spike gene (9.07) between SARS-CoV-2 WIV04 and Bat-SARSr-CoV RaTG13 was extensively higher than those from comparisons between other coronaviruses (range 1.29 - 4.81). The elevated synonymous mutations between SARS-CoV-2 and RaTG13, suggesting they underwent stronger purifying selection. Moreover, their nucleotide substitutions are enriched with T:C transition, which is consistent with the mutation signat...","BACKGROUND: The SARS outbreak of 2002\u20132003 presented clinicians with a new, life-threatening disease for which they had no experience in treating and no research on the effectiveness of treatment options. The World Health Organization (WHO) expert panel on SARS treatment requested a systematic review and comprehensive summary of treatments used for SARS-infected patients in order to guide future treatment and identify priorities for research. METHODS AND FINDINGS: In response to the WHO request we conducted a systematic review of the published literature on ribavirin, corticosteroids, lopinavir and ritonavir (LPV/r), type I interferon (IFN), intravenous immunoglobulin (IVIG), and SARS convalescent plasma from both in vitro studies and in SARS patients. We also searched for clinical trial evidence of treatment for acute respiratory distress syndrome. Sources of data were the literature databases MEDLINE, EMBASE, BIOSIS, and the Cochrane Central Register of Controlled Trials (CENTRAL) up...","As of early March, 2019-nCoV has infected more than one hundred thousand people and claimed thousands of lives. 2019-nCoV is a novel form of coronavirus that causes COVID-19 and has high similarity with SARS-CoV. No approved vaccine yet exists for any form of coronavirus. Here we use computational tools from structural biology and machine learning to identify 2019-nCoV T-cell and B-cell epitopes based on viral protein antigen presentation and antibody binding properties. These epitopes can be used to develop more effective vaccines and identify neutralizing antibodies. We identified 405 viral peptides with good antigen presentation scores for both human MHC-I and MHC-II alleles, and two potential neutralizing B-cell epitopes near the 2019-nCoV spike protein receptor binding domain (440-460 and 494-506). Analyzing mutation profiles of 68 viral genomes from four continents, we identified 96 coding-change mutations. These mutations are more likely to occur in regions with good MHC-I prese...","Publisher Summary Severe acute respiratory syndrome (SARS) is caused by a coronavirus (CoV), SARSCoV. SARS-CoV belongs to the family Coronaviridae, which are enveloped RNA viruses in the order Nidovirales. Global research efforts are continuing to increase the understanding of the virus, the pathogenesis of the disease it causes (SARS), and the \u201cheterogeneity of individual infectiousness\u201d as well as shedding light on how to prepare for other emerging viral diseases. Promising drugs and vaccines have been identified. The milestones achieved have resulted from a truly international effort. Molecular studies dissected the adaptation of this virus as it jumped from an intermediary animal, the civet, to humans, thus providing valuable insights into processes of molecular emergence.","","In the past twenty years, numerous novel zoonotic viral agents with pandemic potential have emerged in China, such as the severe acute respiratory syndrome (SARS) coronavirus and, more recently, the avian-origin influenza A/H7N9 virus, which have caused outbreaks among humans with high morbidity and mortality. In addition, several emerging and re-emerging viral pathogens have also been imported into China from travelers, e.g. the Middle East respiratory syndrome (MERS) coronavirus and Zika virus (ZIKV). Herein, we review these emerging viral pathogens in China and focus on how surveillance by pathogen genomics has been employed to discover and annotate novel pathogenic agents, identify natural reservoirs, monitor the transmission events and delineate their evolution and adaption to the human host. We also highlight the application of genomic sequencing in the recent Ebola epidemics in Western Africa. In summary, genomic sequencing has become a standard research tool in the field of eme...","BACKGROUND: Wild birds are an important but to some extent under-studied reservoir for emerging pathogens. We used unbiased sequencing methods for virus discovery in shorebird samples from the Delaware Bay, USA; an important feeding ground for thousands of migratory birds. FINDINGS: Analysis of shorebird fecal samples indicated the presence of a novel astrovirus and coronavirus. A sanderling sample yielded sequences with distant homology to avian nephritis virus 1, an astrovirus associated with acute nephritis in poultry. A ruddy turnstone sample yielded sequences with homology to deltacoronaviruses. CONCLUSIONS: Our findings highlight shorebirds as a virus reservoir and the need to closely monitor wild bird populations for the emergence of novel virus variants.","The novel coronavirus (COVID-19) outbreak, which was identified in late 2019, requires special attention because of its future epidemics and possible global threats. Beside clinical procedures and treatments, since Artificial Intelligence (AI) promises a new paradigm for healthcare, several different AI tools that are built upon Machine Learning (ML) algorithms are employed for analyzing data and decision-making processes. This means that AI-driven tools help identify COVID-19 outbreaks as well as forecast their nature of spread across the globe. However, unlike other healthcare issues, for COVID-19, to detect COVID-19, AI-driven tools are expected to have active learning-based cross-population train/test models that employs multitudinal and multimodal data, which is the primary purpose of the paper.","Severe acute respiratory syndrome (SARS) is a dangerous infection with pandemic potential. It emerged in 2002 and its aetiological agent, the SARS Coronavirus (SARS-CoV), crossed the species barrier to infect humans, showing high morbidity and mortality rates. No vaccines are currently licensed for SARS-CoV and important efforts have been performed during the first outbreak to develop diagnostic tools. Here we demonstrate the transient expression in Nicotiana benthamiana of two important antigenic determinants of the SARS-CoV, the nucleocapsid protein (N) and the membrane protein (M) using a virus-derived vector or agro-infiltration, respectively. For the M protein, this is the first description of production in plants, while for plant-derived N protein we demonstrate that it is recognized by sera of patients from the SARS outbreak in Hong Kong in 2003. The availability of recombinant N and M proteins from plants opens the way to further evaluation of their potential utility for the de...","Summary Several new viral respiratory tract infectious diseases with epidemic potential that threaten global health security have emerged in the past 15 years. In 2003, WHO issued a worldwide alert for an unknown emerging illness, later named severe acute respiratory syndrome (SARS). The disease caused by a novel coronavirus (SARS-CoV) rapidly spread worldwide, causing more than 8000 cases and 800 deaths in more than 30 countries with a substantial economic impact. Since then, we have witnessed the emergence of several other viral respiratory pathogens including influenza viruses (avian influenza H5N1, H7N9, and H10N8; variant influenza A H3N2 virus), human adenovirus-14, and Middle East respiratory syndrome coronavirus (MERS-CoV). In response, various surveillance systems have been developed to monitor the emergence of respiratory-tract infections. These include systems based on identification of syndromes, web-based systems, systems that gather health data from health facilities (suc...","The recent outbreak of SARS-CoV-2 (2019-nCoV) virus has highlighted the need for fast and efficacious vaccine development. Stimulation of a proper immune response that leads to protection is highly dependent on presentation of epitopes to circulating T-cells via the HLA complex. SARS-CoV-2 is a large RNA virus and testing of all overlapping peptides in vitro to deconvolute an immune response is not feasible. Therefore HLA-binding prediction tools are often used to narrow down the number of peptides to test. We tested 15 epitope-HLA-binding prediction tools, and using an in vitro peptide MHC stability assay, we assessed 777 peptides that were predicted to be good binders across 11 MHC allotypes. In this investigation of potential SARS-CoV-2 epitopes we found that current prediction tools vary in performance when assessing binding stability, and they are highly dependent on the MHC allotype in question. Designing a COVID-19 vaccine where only a few epitope targets are included is therefo...","Abstract The latest advancements in drug development are structure-assisted and computer-aided identification, design, and synthesis of target-specific antiviral. Structural virology of animal viruses has made valuable contribution to our understanding of viruses in general, replication, evolution, and interaction with the host. Structure\u2013function relation studies are definitely the need of the hour for rational design of drugs and vaccines to effectively treat animal viral diseases. These studies are important for economical, veterinary, and human medical perspectives. X-ray crystallography, nuclear magnetic resonance and cryo-electron microscopy techniques elaborate the structure\u2013function relation studies by utilizing various computational approaches for structure-assisted designing of drug molecules and antiviral against viruses. Availability of protein three-dimensional structures, high-performance computing, data management software, and internet are enhancing the modern day drug ...","BACKGROUND: Bat-borne virus surveillance is necessary for determining inter-species transmission risks and is important due to the wide-range of bat species which may harbour potential pathogens. This study aimed to monitor coronaviruses (CoVs) and paramyxoviruses (PMVs) in bats roosting in northwest Italian regions. Our investigation was focused on CoVs and PMVs due to their proven ability to switch host and their zoonotic potential. Here we provide the phylogenetic characterization of the highly conserved polymerase gene fragments. RESULTS: Family-wide PCR screenings were used to test 302 bats belonging to 19 different bat species. Thirty-eight animals from 12 locations were confirmed as PCR positive, with an overall detection rate of 12.6% [95% CI: 9.3\u201316.8]. CoV RNA was found in 36 bats belonging to eight species, while PMV RNA in three Pipistrellus spp. Phylogenetic characterization have been obtained for 15 alpha- CoVs, 5 beta-CoVs and three PMVs; moreover one P. pipistrellus res...","Background and Aim: Since late 2019, an unknown-origin pneumonia outbreak detected in Wuhan city, Hubei Province, China. We aimed to build a model to qualitatively and quantitatively assess publications of research of COVID-19 from 2019 to 2020. Materials and Methods: Data were obtained from the Web of Science (WOS), PubMed, and Scopus Core Collection on March 02, 2020, and updated on March 10. We conducted a qualitative and quantitative analysis of publication outputs, journals, authors, institutions, countries, cited references, keywords, and terms according to bibliometric methods using VOS viewer c software packages. Results: Initially, we identified 227 papers, of which after an exclusion process, 92 studies were selected for statistical analyses. China accounted for the highest proportion of published research (44 papers, 40.48%), followed by the United States (21 papers, 19.32%), and Canada (7 papers, 6.44%). Adjusted by gross domestic product (GDP), ranked first, with 0.003 art...","RNA viruses are known to replicate by low fidelity polymerases and have high mutation rates whereby the resulting virus population tends to exist as a distribution of mutants. In this review, we aim to explore how genetic events such as spontaneous mutations could alter the genomic organization of RNA viruses in such a way that they impact virus replications and plaque morphology. The phenomenon of quasispecies within a viral population is also discussed to reflect virulence and its implications for RNA viruses. An understanding of how such events occur will provide further evidence about whether there are molecular determinants for plaque morphology of RNA viruses or whether different plaque phenotypes arise due to the presence of quasispecies within a population. Ultimately this review gives an insight into whether the intrinsically high error rates due to the low fidelity of RNA polymerases is responsible for the variation in plaque morphology and diversity in virulence. This can be...","Abstract In 2015, an outbreak of Middle East respiratory syndrome coronavirus occurred in Saudi Arabia and necessitated special measures to be implemented to control the spread of the virus. In this article, we will discuss how the outbreak was managed in the vascular and interventional radiology (VIR) suite in a large tertiary care hospital in Saudi Arabia. Various measures were taken to reduce the risk of transmission of infection. Unit-level education played an important role in the care of patients. A hospital-wide educational program was implemented to ensure zero transmission of infection. Special attention was made to monitor staff who acquired the virus. VIR suite was able to handle the situation and control the outbreak.","AbstractThe human betacoronaviruses HKU1 and OC43 (subgenus Embecovirus) arose from separate zoonotic introductions, OC43 relatively recently and HKU1 apparently much longer ago. Embecovirus particles are studded with two types of surface projections called S (for spike) and HE (for haemagglutinin-esterase), with S mediating receptor-binding and membrane fusion and HE acting as a receptor-destroying enzyme. Together, they promote dynamic virion attachment to glycan-based receptors with 9-O-acetylated sialic acid as main constituent. We recently showed that adaptation of HKU1 and OC43 to replication in the human respiratory tract involved loss-of-function mutations in the lectin domain of HE. Here we present the cryo-EM structure of the \u223c80 kDa, heavily glycosylated HKU1 HE at a global resolution of 3.4 \u00c5. Comparison to existing HE structures reveals a drastically truncated lectin domain, incompatible with sialic acid binding, but with the structure and function of the HE esterase domai...","Background: Excessive monocyte/macrophage activation with the development of a cytokine storm and subsequent acute lung injury, leading to acute respiratory distress syndrome (ARDS) is a feared consequence of infection with COVID-19. The ability to recognize and potentially intervene early in those patients at greatest risk of developing this complication could be of great clinical utility. Methods: We performed detailed flow cytometric analysis of peripheral blood samples from 28 COVID-19 patients treated at Xian No.8 Hospital and the First Affiliated Hospital of Xian Jiaotong University in early 2020 in an attempt to identify factors that could help predict severity of disease and patient outcome. Findings: While we did not detect significant differences in the number of monocytes between patients with COVID-19 and normal healthy individuals,we did identify significant morphological and functional differences, which are more pronounced in patients requiring prolonged hospitalization ...","AbstractWuhan Novel Coronavirus disease (COVID-19) outbreak has become a global outbreak which has raised the concern of scientific community to design and discover a definitive cure against this deadly virus which has caused deaths of numerous infected people upon infection and spreading. To date, no antiviral therapy or vaccine is available which can effectively combat the infection caused by this virus. This study was conducted to design possible epitope-based subunit vaccines against the SARS-CoV-2 virus using the approaches of reverse vaccinology and immunoinformatics. Upon continual computational experimentation three possible vaccine constructs were designed and one vaccine construct was selected as the best vaccine based on molecular docking study which is supposed to effectively act against SARS-CoV-2. Later, molecular dynamics simulation and in silico codon adaptation experiments were carried out in order to check biological stability and find effective mass production strate...","Abstract The sudden appearance of apparently new viruses with pathogenic potential is of fundamental importance in medical microbiology and a constant threat to humans and animals. The emergence of a \u201cnew\u201d pathogen is not an isolated event, as for instance the frequent appearance of new influenza virus strains demonstrates. Often the new virus strains co-circulate with the older strains in a susceptible population, but a replacement of the older strains has been also observed. In rare instances the new viruses can cause dramatic epidemics or pandemics, such as those observed with the human immunodeficiency virus, canine parvovirus, or most recently, with the agent of bovine spongiform encephalopathy in the United Kingdom. The mechanisms of the emergence are not always clearly understood, but an altered host range appears to be a common event. Whether a true change in host range occurs, or whether the virus adapted to the host and replicated more efficiently, is often unknown. This revi...","BACKGROUND: Infections are often associated to comorbidity that increases the risk of medical conditions which can lead to further morbidity and mortality. SARS is a threat which is similar to MERS virus, but the comorbidity is the key aspect to underline their different impacts. One UK doctor says \"I\u2019d rather have HIV than diabetes\" as life expectancy among diabetes patients is lower than that of HIV. However, HIV has a comorbidity impact on the diabetes. RESULTS: We present a quantitative framework to compare and explore comorbidity between diseases. By using neighbourhood based benchmark and topological methods, we have built comorbidity relationships network based on the OMIM and our identified significant genes. Then based on the gene expression, PPI and signalling pathways data, we investigate the comorbidity association of these 2 infective pathologies with other 7 diseases (heart failure, kidney disorder, breast cancer, neurodegenerative disorders, bone diseases, Type 1 and Typ...","The study of virus evolution has historically been concerned with disease and its emergence and has not been integrated into the general study of evolution. Yet viruses dominate our planet and their evolution is a broad and applied field that can be studied in the real time (such HIV in human disease) and can also be applied to biotechnological problems. Virus evolution is very similar, but not identical, to host evolution. Darwinian principles apply, such as with Fisher population genetics, but other features, such as reticulated and quasispecies-based evolution, distinguish virus evolution. In RNA viruses, high error rates have led to the quasispecies concept in which collective populations are the basis of evolution. These populations provide viruses with high capacities for adaptation. Yet some RNA viruses show remarkable evolutionary stability. With DNA viruses, especially of bacteria, extensive recombination defines vast dynamic and ancient genetic pools. Temperate viruses can di...","The ongoing COVID-19 epidemic that spread widely in China since December 2019 is now generating local transmission in multiple countries including Singapore as of March 5, 2020. This highlights the need to monitor in real time the transmission potential of COVID-19. In Singapore, four major COVID-19 case clusters have emerged thus far. Here we estimate the effective reproduction number, Rt, of COVID-19 in Singapore from the publicly available daily case series of imported and autochthonous cases by date of symptoms onset, after adjusting the local cases for reporting delays. We also derive the reproduction number from the distribution of cluster sizes using a branching process analysis. The effective reproduction number peaked with a mean value ~1.0 around February 6-12, 2020 and declined thereafter. As of March 5, 2020, our most recent estimate of Rt is at 0.9 (95% CI: 0.7,1.0) while an estimate of the overall R based on cluster size distribution is at 0.7 (95% CI: 0.5, 1.0). The traj...","The severe acute respiratory syndrome coronavirus (SARS-CoV) from palm civets has twice evolved the capacity to infect humans by gaining binding affinity for human receptor angiotensin-converting enzyme 2 (ACE2). Numerous mutations have been identified in the receptor-binding domain (RBD) of different SARS-CoV strains isolated from humans or civets. Why these mutations were naturally selected or how SARS-CoV evolved to adapt to different host receptors has been poorly understood, presenting evolutionary and epidemic conundrums. In this study, we investigated the impact of these mutations on receptor recognition, an important determinant of SARS-CoV infection and pathogenesis. Using a combination of biochemical, functional, and crystallographic approaches, we elucidated the molecular and structural mechanisms of each of these naturally selected RBD mutations. These mutations either strengthen favorable interactions or reduce unfavorable interactions with two virus-binding hot spots on A...","The ongoing novel coronavirus pneumonia COVID-19 outbreak in Wuhan, China, has engendered numerous cases of infection and death. COVID-19 diagnosis relies upon nucleic acid detection; however, current recommended methods exhibit high false-negative rates, low sensitivity, and cannot identify other respiratory virus infections, thereby resulting patient misdiagnosis and impeding epidemic containment. Combining the advantages of target amplification and long-read, real-time nanopore sequencing, we developed nanopore target sequencing (NTS) to detect SARS-CoV-2 and other respiratory viruses simultaneously within 6-10 h. Parallel testing with approved qPCR kits of SARS-CoV-2 and NTS using 61 nucleic acid samples from suspected COVID-19 cases confirmed that NTS identified more infected patients as positive, and could also monitor for mutated nucleic acid sequence or other respiratory virus infection in the test sample. NTS is thus suitable for contemporary COVID-19 diagnosis; moreover, this...","To provide convenient access to epidemiological data on the coronavirus outbreak, we developed an R package, nCov2019 (https://github.com/GuangchuangYu/nCov2019). Besides detailed real-time statistics, it offers access to three data sources with detailed daily statistics from December 1, 2019, for 43 countries and more than 500 Chinese cities. We also developed a web app (http://www.bcloud.org/e/) with interactive plots and simple time-series forecasts. These analytics tools could be useful in informing the public and studying how this and similar viruses spread in populous countries.","Recently the novel coronavirus (2019-nCov) pneumonia outbreak in China then the world, and the Number of infections and death continues to increases. Search Query performs well in forecasting the epidemics. It is still a question whether search engine data can forecast the drift and the inflexion in 2019-nCov pneumonia. Based on the Baidu Search Index, we propose three prediction models: composite Index, composite Index with filtering and suspected NCP(Novel Coronavirus Pneumonia). The result demonstrates that the predictive model of composite index with filtering performs the best while the model of suspected NCP has the highest forecast error. We further predict the out-of-the-set NCP confirmed cases and monitor that the next peak of new diagnoses will occur on February 16th and 17th.","An epidemic of a novel coronavirus emerged from Wuhan, China, in late December 2019 and has since spread to several large Chinese cities. Should a scenario arise where this coronavirus spreads more broadly across China, we evaluate how patterns of international disease transmission could change.","The outbreak of SARS coronavirus 2 (SARS-CoV-2), which occurred in Wuhan, China in December 2019, has caused a worldwide pandemic of coronavirus disease 2019 (COVID-19). However, there is a lack of epidemiological tools to guide effective public policy development. Here we present epidemiological evidence that SARS-CoV-2 S type exited Wuhan or other epicenters in China earlier than L type and conferred partial resistance to the virus on infected populations. Analysis of regional disparities in incidence has revealed that a sharp decline in influenza epidemics is a useful surrogate indicator for the undocumented spread of SARS-CoV-2. The biggest concern in the world is knowing when herd immunity has been achieved and scheduling a time to regain the living activities of each country. This study provides a useful tool to guide the development of local policies to contain the virus.","Emerging infectious diseases are diseases that are either new, are newly recognized, or are increasing in prevalence in new areas. Resurgent diseases are also usually grouped in this category, as is antimicrobial resistance. These diseases have been given formal recognition in the past two decades, although a historical outlook demonstrates that the phenomenon has probably been persistent, although largely undetected, through recorded history. Emergence has accelerated recently, driven by factors such as demographic change, land use change, increased rapidity and frequency of intercontinental transportation, and other mostly social trends. Continued infectious disease emergence poses, and will continue to pose, significant challenges for public health and for basic science.","Abstract Objective An emerging outbreak of COVID-19 has been detected in at least 26 countries worldwide. Given this pandemic situation, robust risk communication is urgently needed particularly in affected countries. Therefore, this study explored the potential use of Google Trends (GT) to monitor public restlessness toward COVID-19 epidemic infection in Taiwan. Methods We retrieved GT data for the specific locations of Taiwan nationwide and subregions using defined search terms related to coronavirus, handwashing, and face masks. Results Searches related to COVID-19 and face masks in Taiwan increased rapidly, following the announcements of Taiwan\u2019 first imported case and reached its peak as local cases were reported. However, searches for handwashing were gradually increased in period of face masks shortage. Moreover, high to moderate correlations between Google relative search volume (RSV) and COVID-19 cases were found in Taipei (lag-3), New Taipei (lag-2), Taoyuan (lag-2), Tainan (...","In the current COVID-19 pandemic, a significant proportion of cases shed SARS-Coronavirus-2 (SARS-CoV-2) with their faeces. To determine if SARS-CoV-2 is present in sewage during the emergence of COVID-19 in the Netherlands, sewage samples of 7 cities and the airport were tested using RT-PCR against three fragments of the nucleocapsid protein gene (N1-3) and one fragment of the envelope protein gene (E). No SARS-CoV-2 was detected in samples of February 6, three weeks before the first case was reported in the Netherlands on February 27. On March 5, the N1 fragment was detected in sewage of five sites. On March 15/16, the N1 fragment was detected in sewage of six sites, and the N3 and E fragment were detected at 5 and 4 sites respectively. This is the first report of detection of SARS-CoV-2 in sewage. The detection of the virus in sewage, even when the COVID-19 incidence is low, indicates that sewage surveillance could be a sensitive tool to monitor the circulation of the virus in the p...","The COVID-19 virus has spread worldwide in a matter of a few months. Healthcare systems struggle to monitor and report current cases. Limited capabilities in testing result in difficult to guide policies and mitigate lack of preparation. Since severe cases, which more likely lead to fatal outcomes, are detected at a higher percentage than mild cases, the reported death rates are likely inflated in most countries. Such under-estimation can be attributed to the under-sampling of infection cases and results in systematic death rate estimation biases. The method proposed here utilizes a benchmark country (South Korea) and its reported death rates in combination with population demographics to correct the reported COVID-19 case numbers. By applying a correction, we predict that the number of cases is highly under-reported in most countries. In the case of China, it is estimated that more than 700.000 cases of COVID-19 actually occurred instead of the confirmed 80,932 cases as of 3/13/2020.","","ABSTRACTNovel Coronavirus (nCoV) outbreak in the city of Wuhan, China during December 2019, has now spread to various countries across the globe triggering a heightened containment effort. This human pathogen is a member of betacoronavirus genus carrying 30 kilobase of single positive-sense RNA genome. Understanding the evolution, zoonotic transmission, and source of this novel virus would help accelerating containment and prevention efforts. The present study reported detailed analysis of 2019-nCoV genome evolution and potential candidate peptides for vaccine development. This nCoV genotype might have been evolved from a bat-CoV by accumulating non-synonymous mutations, indels, and recombination events. Structural proteins Spike (S), and Membrane (M) had extensive mutational changes, whereas Envelope (E) and Nucleocapsid (N) proteins were very conserved suggesting differential selection pressures exerted on 2019-nCoV during evolution. Interestingly, 2019-nCoV Spike protein contains a ...","World Health Organization has declared the ongoing outbreak of coronavirus disease 2019 (COVID-19) a Public Health Emergency of International Concern. The virus was named severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) by the International Committee on Taxonomy of Viruses. Human infection with SARS-CoV-2 leads to a wide range of clinical manifestations ranging from asymptomatic, mild, moderate to severe. The severe cases present with pneumonia, which can progress to acute respiratory distress syndrome. The outbreak provides an opportunity for real-time tracking of an animal coronavirus that has just crossed species barrier to infect humans. The outcome of SARS-CoV-2 infection is largely determined by virus-host interaction. Here, we review the discovery, zoonotic origin, animal hosts, transmissibility and pathogenicity of SARS-CoV-2 in relation to its interplay with host antiviral defense. A comparison with SARS-CoV, Middle East respiratory syndrome coronavirus, community-...","AbstractAn outbreak of new SARS-like viral in Wuhan, China has been named 2019-nCoV. The current state of the epidemic is increasingly serious, and there has been the urgent necessity to develop an effective new drug. In previous studies, it was found that the conformation change in CTD1 was the region where SARS-CoV bound to human ACE2. Although there are mutations of the 2019-nCoV, the binding energy of ACE2 remains high. The surface glycoprotein of 2019-nCoV was coincident with the CTD1 region of the S-protein by comparing the I-TASSER prediction model with the actual SARS model, which suggests that 2019-nCoV may bind to the ACE2 receptor through conformational changes. Furthermore, site prediction on the surface glycoprotein of 2019-nCoV suggests some core amino acid area may be a novel drug target against 2019-nCoV.",""],"title":["Strong evolutionary convergence of receptor-binding protein spike between COVID-19 and SARS-related coronaviruses","Candidate targets for immune responses to 2019-Novel Coronavirus (nCoV): sequence homology- and bioinformatic-based predictions","A Sequence Homology and Bioinformatic Approach Can Predict Candidate Targets for Immune Responses to SARS-CoV-2","A single holiday was the turning point of the COVID-19 policy of Israel","Selection of and recombination between minor variants lead to the adaptation of an avian coronavirus to primate cells","Larger viral genome size facilitates emergence of zoonotic diseases","Genotype and phenotype of COVID-19: Their roles in pathogenesis","Microarrays and gene expression profiling in microbiology and infectious diseases: a clinician's perspective","Antiviral Agents\u2606","Antiviral Agents","Replacement of the heterologous 5\u2032 untranslated region allows preservation of the fully functional activities of type 2 porcine reproductive and respiratory syndrome virus","Molecular-based diagnostics, including future trends","Comparative genomic analysis revealed specific mutation pattern between human coronavirus SARS-CoV-2 and Bat-SARSr-CoV RaTG13","SARS: Systematic Review of Treatment Effects","Potential T-cell and B-cell Epitopes of 2019-nCoV","Severe Acute Respiratory Syndrome Coronavirus (SARS-CoV)","The international imperative to rapidly and inexpensively monitor community-wide Covid-19 infection status and trends","Pathogen genomic surveillance elucidates the origins, transmission and evolution of emerging viral agents in China","Novel Coronavirus and Astrovirus in Delaware Bay Shorebirds","AI-Driven Tools for Coronavirus Outbreak: Need of Active Learning and Cross-Population Train/Test Models on Multitudinal/Multimodal Data","Antigen Production in Plant to Tackle Infectious Diseases Flare Up: The Case of SARS","Surveillance for emerging respiratory viruses","COVID-19 Vaccine Candidates: Prediction and Validation of 174 SARS-CoV-2 Epitopes","Chapter 19 Advances in structure-assisted antiviral discovery for animal viral diseases","Coronavirus and paramyxovirus in bats from Northwest Italy","Bibliometric Analysis of Global Scientific Research on SARSCoV-2 (COVID-19)","Impact of RNA Virus Evolution on Quasispecies Formation and Virulence","Management of Patient Flow in Vascular and Interventional Radiology During MERS-CoV Outbreak","Cryo-EM structure of coronavirus-HKU1 haemagglutinin esterase reveals architectural changes arising from prolonged circulation in humans","COVID-19 infection induces readily detectable morphological and inflammation-related phenotypic changes in peripheral blood monocytes, the severity of which correlate with patient outcome","The Essential Facts of Wuhan Novel Coronavirus Outbreak in China and Epitope-based Vaccine Designing against COVID-19","There is nothing permanent except change. The emergence of new virus diseases","Network-based analysis of comorbidities risk during an infection: SARS and HIV case studies","Evolution of Viruses","Real-time monitoring the transmission potential of COVID-19 in Singapore, February 2020","Mechanisms of Host Receptor Adaptation by Severe Acute Respiratory Syndrome Coronavirus","Nanopore target sequencing for accurate and comprehensive detection of SARS-CoV-2 and other respiratory viruses","Open-source analytics tools for studying the COVID-19 coronavirus outbreak","Can Search Query Forecast successfully in China's 2019-nCov pneumonia?","Potential for global spread of a novel coronavirus from China","Epidemiological Tools that Predict Partial Herd Immunity to SARS Coronavirus 2","Emerging Diseases: Overview","Applications of google search trends for risk communication in infectious disease management: A case study of COVID-19 outbreak in Taiwan","Presence of SARS-Coronavirus-2 in sewage","Correcting under-reported COVID-19 case numbers","Climate Change, Covid-19, Preparedness, and Consciousness","Insights into Cross-species Evolution of Novel Human Coronavirus 2019-nCoV and Defining Immune Determinants for Vaccine Development","A tug-of-war between severe acute respiratory syndrome coronavirus 2 and host antiviral defence: lessons from other pathogenic viruses","Predictions for the binding domain and potential new drug targets of 2019-nCoV","COVID-19: Is a paradigm change to be expected in health care and transfusion medicine?"],"url":["https://doi.org/10.1101/2020.03.04.975995","https://doi.org/10.1101/2020.02.12.946087","https://doi.org/10.1016/j.chom.2020.03.002","https://doi.org/10.1101/2020.03.26.20044412","https://doi.org/10.1016/j.bbrc.2005.08.105","https://doi.org/10.1101/2020.03.10.986109","https://doi.org/10.1016/j.jmii.2020.03.022","https://doi.org/10.1016/j.clinmicnews.2004.09.004","https://doi.org/10.1016/b978-0-12-801238-3.02387-4","https://doi.org/10.1016/b978-012373944-5.00178-4","https://doi.org/10.1016/j.virol.2012.12.013","https://doi.org/10.1016/j.mpmed.2013.08.012","https://doi.org/10.1101/2020.02.27.969006","https://www.ncbi.nlm.nih.gov/pmc/articles/PMC1564166/","https://doi.org/10.1101/2020.02.19.955484","https://doi.org/10.1016/s0168-7069(06)16004-8","https://doi.org/10.1016/j.scitotenv.2020.138149","https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7088571/","https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3974748/","https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7087612/","https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4742786/","https://doi.org/10.1016/s1473-3099(14)70840-0","https://doi.org/10.1101/2020.03.20.000794","https://doi.org/10.1016/b978-0-12-816352-8.00019-9","https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5741894/","https://doi.org/10.1101/2020.03.19.20038752","https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6770471/","https://doi.org/10.1016/j.jradnu.2017.06.003","https://doi.org/10.1101/2020.03.25.998963","https://doi.org/10.1101/2020.03.24.20042655","https://doi.org/10.1101/2020.02.05.935072","https://doi.org/10.1016/0378-1135(95)92531-f","https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4363349/","https://doi.org/10.1016/b978-012374410-4.00706-8","https://doi.org/10.1101/2020.02.21.20026435","http://www.jbc.org/content/287/12/8904.full.pdf","https://doi.org/10.1101/2020.03.04.20029538","https://doi.org/10.1101/2020.02.25.20027433","https://doi.org/10.1101/2020.02.12.20022400","https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7074660/","https://doi.org/10.1101/2020.03.25.20043679","https://doi.org/10.1016/b978-012373960-5.00453-6","https://doi.org/10.1016/j.ijid.2020.03.021","https://doi.org/10.1101/2020.03.29.20045880","https://doi.org/10.1101/2020.03.14.20036178","https://doi.org/10.1016/j.explore.2020.02.022","https://doi.org/10.1101/2020.01.29.925867","https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7103735/","https://doi.org/10.1101/2020.02.26.961938","https://doi.org/10.1016/j.tracli.2020.03.005"],"x":[0.7420616294598316,0.20033918475264556,0.7735956126660515,0.3535372382290728,0.5808247889494748,0.48045093581054843,0.6366834312683033,0.3727181213338299,0.539435393359347,0.5397964114303617,0.5533353860443059,0.40117761935566687,0.6503914978634012,0.6404750018185341,0.8047274935853633,0.6458132502050153,0.50037947951676,0.35665703849095537,0.4470937573521105,0.4542836314809099,0.7223797376224971,0.35655633907622003,0.8054413406990956,0.55220335786974,0.4035522125885926,0.4424115788442467,0.45718353056333927,0.4975035044364394,0.5922990342441411,0.623540565788773,0.8238863770278531,0.41929449462371743,0.38032661624401337,0.41977387183216275,0.30712219806827007,0.749922960277827,0.8396877247815662,0.34280671377607663,0.40926125106836747,0.3262159278781942,0.6717619507012588,0.3316769503327798,0.4295100006428782,0.6863986866467503,0.38900100058581677,0.48582181691577003,0.20616275509671905,0.6682575483486379,0.7188416369957554,0.444894159263511],"y":[0.13682110204300235,0.3805447142440388,0.3034194578514144,0.7539590698711854,0.26839739371567495,0.2213050412035748,0.30305069510096927,0.35812793418313027,0.18267817024923266,0.18277669607100855,0.24727521766909158,0.348294695741071,0.35129787753840036,0.6214981527867098,0.24634011377860437,0.4138862964204373,0.7030750129399217,0.30016772485490034,0.17122496081272798,0.7790704801038173,0.2447236274118415,0.26126026033426514,0.2676032551706223,0.17548366206784144,0.1095302759971803,0.5112980259997472,0.22175676240421907,0.5342213655764798,0.21709264497771188,0.7662882660843546,0.26066984284807276,0.20078371221487534,0.3815697863590165,0.2120709966471966,0.9198773398089709,0.14289900585436088,0.5674766667507029,0.6156819607218764,0.5336494409283605,0.5419948568029936,0.436938178239727,0.3517272971218766,0.8173483534400808,0.5053672642389849,0.9216428547875497,0.7417309116532376,0.37861567298172777,0.39717337375242123,0.013725356864975452,0.5969674192903299]},"selected":{"id":"3971"},"selection_policy":{"id":"3992"}},"id":"3899","type":"ColumnDataSource"},{"attributes":{"data_source":{"id":"3899"},"glyph":{"id":"3926"},"hover_glyph":null,"muted_glyph":null,"nonselection_glyph":{"id":"3927"},"selection_glyph":null,"view":{"id":"3929"}},"id":"3928","type":"GlyphRenderer"},{"attributes":{"axis":{"id":"3945"},"ticker":null},"id":"3948","type":"Grid"},{"attributes":{"editor":{"id":"3989"},"field":"scr","formatter":{"id":"3988"},"title":"Score","width":30},"id":"3932","type":"TableColumn"},{"attributes":{"children":[{"id":"3969"},{"id":"3936"}]},"id":"3982","type":"Row"},{"attributes":{"editor":{"id":"3999"},"field":"scr","formatter":{"id":"3998"},"title":"Score","width":30},"id":"3967","type":"TableColumn"},{"attributes":{"fill_alpha":{"value":0.1},"fill_color":{"field":"color"},"line_alpha":{"value":0.1},"line_color":{"field":"color"},"radius":{"field":"rad","units":"data"},"x":{"field":"x"},"y":{"field":"y"}},"id":"3927","type":"Circle"},{"attributes":{},"id":"3946","type":"BasicTicker"},{"attributes":{"children":[{"id":"3977"}]},"id":"3981","type":"Row"},{"attributes":{"source":{"id":"3899"}},"id":"3929","type":"CDSView"},{"attributes":{"children":[{"id":"3934"},{"id":"3901"}]},"id":"3980","type":"Row"},{"attributes":{"text":""},"id":"3984","type":"Title"},{"attributes":{"active_drag":"auto","active_inspect":"auto","active_multi":null,"active_scroll":"auto","active_tap":"auto","tools":[{"id":"3918"},{"id":"3919"},{"id":"3920"}]},"id":"3921","type":"Toolbar"},{"attributes":{},"id":"3943","type":"LinearScale"},{"attributes":{"formatter":{"id":"4007"},"ticker":{"id":"3946"}},"id":"3945","type":"LinearAxis"},{"attributes":{"text":""},"id":"3986","type":"Title"},{"attributes":{"editor":{"id":"3991"},"field":"title","formatter":{"id":"3990"},"title":"Document title","width":770},"id":"3933","type":"TableColumn"},{"attributes":{"args":{"sd":{"id":"3899"},"sp":{"id":"3900"}},"code":"\n    if (sd.selected.indices.length == 0) {\n      return;\n    }\n    var sel = sd.selected.indices[0];\n    var new_selected = [];\n    for (var i = 0; i &lt; sp.data['doc'].length; i++) {\n      if (sp.data['doc'][i] == sd.data['doc'][sel]) {\n        new_selected.push(i);\n      }\n    }\n    if (JSON.stringify(new_selected) != JSON.stringify(sp.selected.indices) &amp;&amp;\n    !(sp.selected.indices.length == 1 &amp;&amp; new_selected.includes(sp.selected.indices[0]))) {\n      sp.selected.indices = new_selected;\n      sp.change.emit();\n    }\n    "},"id":"3972","type":"CustomJS"},{"attributes":{"axis":{"id":"3949"},"dimension":1,"ticker":null},"id":"3952","type":"Grid"},{"attributes":{},"id":"3992","type":"UnionRenderers"},{"attributes":{},"id":"3950","type":"BasicTicker"},{"attributes":{"fill_color":{"field":"color"},"line_color":{"field":"color"},"radius":{"field":"rad","units":"data"},"x":{"field":"x"},"y":{"field":"y"}},"id":"3961","type":"Circle"},{"attributes":{"columns":[{"id":"3932"},{"id":"3933"}],"height":340,"source":{"id":"3899"},"view":{"id":"3935"},"width":900},"id":"3934","type":"DataTable"},{"attributes":{"source":{"id":"3899"}},"id":"3935","type":"CDSView"},{"attributes":{"below":[{"id":"3945"}],"center":[{"id":"3948"},{"id":"3952"}],"left":[{"id":"3949"}],"plot_height":340,"plot_width":900,"renderers":[{"id":"3963"}],"title":{"id":"3986"},"toolbar":{"id":"3956"},"toolbar_location":"below","x_range":{"id":"3937"},"x_scale":{"id":"3941"},"y_range":{"id":"3939"},"y_scale":{"id":"3943"}},"id":"3936","subtype":"Figure","type":"Plot"},{"attributes":{"callback":null},"id":"3953","type":"TapTool"},{"attributes":{},"id":"3991","type":"StringEditor"},{"attributes":{},"id":"3954","type":"ResetTool"}],"root_ids":["3983"]},"title":"Bokeh Application","version":"2.0.1"}}
        </script>
        <script type="text/javascript">
          (function() {
            var fn = function() {
              Bokeh.safely(function() {
                (function(root) {
                  function embed_document(root) {
                    
                  var docs_json = document.getElementById('4220').textContent;
                  var render_items = [{"docid":"4accd84c-b012-404b-9079-49e6d3180efa","root_ids":["3983"],"roots":{"3983":"8aea6f0b-6974-4dc2-ba1a-4096df311729"}}];
                  root.Bokeh.embed.embed_items(docs_json, render_items);
                
                  }
                  if (root.Bokeh !== undefined) {
                    embed_document(root);
                  } else {
                    var attempts = 0;
                    var timer = setInterval(function(root) {
                      if (root.Bokeh !== undefined) {
                        clearInterval(timer);
                        embed_document(root);
                      } else {
                        attempts++;
                        if (attempts > 100) {
                          clearInterval(timer);
                          console.log("Bokeh: ERROR: Unable to run BokehJS code because BokehJS library is missing");
                        }
                      }
                    }, 10, root)
                  }
                })(window);
              });
            };
            if (document.readyState != "loading") fn();
            else document.addEventListener("DOMContentLoaded", fn);
          })();
        </script>
    
  </body>
  
</html>