



<!DOCTYPE html>
<html lang="en">
  
  <head>
    
      <meta charset="utf-8">
      <title>Bokeh Plot</title>
      
      
        
          
        
        
          
        <script type="text/javascript" src="https://cdn.bokeh.org/bokeh/release/bokeh-2.0.1.min.js" integrity="sha384-JpP8FXbgAZLkfur7LiK3j9AGBhHNIvF742meBJrjO2ShJDhCG2I1uVvW+0DUtrmc" crossorigin="anonymous"></script>
        <script type="text/javascript" src="https://cdn.bokeh.org/bokeh/release/bokeh-widgets-2.0.1.min.js" integrity="sha384-xZlADit0Q04ISQEdKg2k3L4W9AwQBAuDs9nJL9fM/WwzL1tEU9VPNezOFX0nLEAz" crossorigin="anonymous"></script>
        <script type="text/javascript" src="https://cdn.bokeh.org/bokeh/release/bokeh-tables-2.0.1.min.js" integrity="sha384-4BuPRZkdMKSnj3zoxiNrQ86XgNw0rYmBOxe7nshquXwwcauupgBF2DHLVG1WuZlV" crossorigin="anonymous"></script>
        <script type="text/javascript">
            Bokeh.set_log_level("info");
        </script>
        
      
      
    
  </head>
  
  
  <body>
    
      
        
          
          
            
              <div class="bk-root" id="ed53e781-8047-43d9-8641-350b78f819c5" data-root-id="27811"></div>
            
          
        
      
      
        <script type="application/json" id="28048">
          {"69180d8f-ef4c-4732-a67a-3fd39ca57dbd":{"roots":{"references":[{"attributes":{"text":"&lt;h3&gt;Document ranking&lt;/h3&gt;"},"id":"27804","type":"Div"},{"attributes":{"axis":{"id":"27742"},"dimension":1,"ticker":null},"id":"27745","type":"Grid"},{"attributes":{"args":{"sd":{"id":"27727"},"sp":{"id":"27728"}},"code":"\n    if (sd.selected.indices.length == 0) {\n      return;\n    }\n    var sel = sd.selected.indices[0];\n    var new_selected = [];\n    for (var i = 0; i &lt; sp.data['doc'].length; i++) {\n      if (sp.data['doc'][i] == sd.data['doc'][sel]) {\n        new_selected.push(i);\n      }\n    }\n    if (JSON.stringify(new_selected) != JSON.stringify(sp.selected.indices) &amp;&amp;\n    !(sp.selected.indices.length == 1 &amp;&amp; new_selected.includes(sp.selected.indices[0]))) {\n      sp.selected.indices = new_selected;\n      sp.change.emit();\n    }\n    "},"id":"27800","type":"CustomJS"},{"attributes":{"args":{"sd":{"id":"27727"},"sp":{"id":"27728"}},"code":"\n    if (sp.selected.indices.length == 0) {\n      return;\n    }\n    var sel = sp.selected.indices[0];\n    var new_selected = [];\n    for (var i = 0; i &lt; sd.data['doc'].length; i++) {\n      if (sd.data['doc'][i] == sp.data['doc'][sel]) {\n        new_selected.push(i);\n      }\n    }\n    if (JSON.stringify(new_selected) != JSON.stringify(sd.selected.indices) &amp;&amp;\n    !(sd.selected.indices.length == 1 &amp;&amp; new_selected.includes(sd.selected.indices[0]))) {\n      sd.selected.indices = new_selected;\n      sd.change.emit();\n    }\n    "},"id":"27802","type":"CustomJS"},{"attributes":{"js_property_callbacks":{"change:indices":[{"id":"27802"}]}},"id":"27801","type":"Selection"},{"attributes":{"data_source":{"id":"27727"},"glyph":{"id":"27754"},"hover_glyph":null,"muted_glyph":null,"nonselection_glyph":{"id":"27755"},"selection_glyph":null,"view":{"id":"27757"}},"id":"27756","type":"GlyphRenderer"},{"attributes":{"data_source":{"id":"27728"},"glyph":{"id":"27789"},"hover_glyph":null,"muted_glyph":null,"nonselection_glyph":{"id":"27790"},"selection_glyph":null,"view":{"id":"27792"}},"id":"27791","type":"GlyphRenderer"},{"attributes":{},"id":"27820","type":"UnionRenderers"},{"attributes":{},"id":"27835","type":"BasicTickFormatter"},{"attributes":{"text":""},"id":"27814","type":"Title"},{"attributes":{"editor":{"id":"27817"},"field":"scr","formatter":{"id":"27816"},"title":"Score","width":30},"id":"27760","type":"TableColumn"},{"attributes":{"editor":{"id":"27819"},"field":"title","formatter":{"id":"27818"},"title":"Document title","width":770},"id":"27761","type":"TableColumn"},{"attributes":{"fill_alpha":{"value":0.1},"fill_color":{"field":"color"},"line_alpha":{"value":0.1},"line_color":{"field":"color"},"radius":{"field":"rad","units":"data"},"x":{"field":"x"},"y":{"field":"y"}},"id":"27755","type":"Circle"},{"attributes":{"source":{"id":"27727"}},"id":"27757","type":"CDSView"},{"attributes":{},"id":"27829","type":"StringEditor"},{"attributes":{"js_property_callbacks":{"change:indices":[{"id":"27800"}]}},"id":"27799","type":"Selection"},{"attributes":{},"id":"27833","type":"BasicTickFormatter"},{"attributes":{},"id":"27817","type":"StringEditor"},{"attributes":{"sizing_mode":"stretch_width","text":"&lt;h2&gt;Enhance capacity (people, technology, data) for sequencing with advanced analytics for unknown pathogens, and explore capabilities for distinguishing naturally-occurring pathogens from intentional.&lt;/h2&gt;"},"id":"27803","type":"Div"},{"attributes":{"children":[{"id":"27797"},{"id":"27764"}]},"id":"27810","type":"Row"},{"attributes":{"columns":[{"id":"27760"},{"id":"27761"}],"height":340,"source":{"id":"27727"},"view":{"id":"27763"},"width":900},"id":"27762","type":"DataTable"},{"attributes":{"source":{"id":"27727"}},"id":"27763","type":"CDSView"},{"attributes":{},"id":"27823","type":"BasicTickFormatter"},{"attributes":{"children":[{"id":"27804"}]},"id":"27807","type":"Row"},{"attributes":{},"id":"27743","type":"BasicTicker"},{"attributes":{},"id":"27819","type":"StringEditor"},{"attributes":{"callback":null},"id":"27746","type":"TapTool"},{"attributes":{"fill_alpha":{"value":0.1},"fill_color":{"field":"color"},"line_alpha":{"value":0.1},"line_color":{"field":"color"},"radius":{"field":"rad","units":"data"},"x":{"field":"x"},"y":{"field":"y"}},"id":"27790","type":"Circle"},{"attributes":{"below":[{"id":"27773"}],"center":[{"id":"27776"},{"id":"27780"}],"left":[{"id":"27777"}],"plot_height":340,"plot_width":900,"renderers":[{"id":"27791"}],"title":{"id":"27814"},"toolbar":{"id":"27784"},"toolbar_location":"below","x_range":{"id":"27765"},"x_scale":{"id":"27769"},"y_range":{"id":"27767"},"y_scale":{"id":"27771"}},"id":"27764","subtype":"Figure","type":"Plot"},{"attributes":{},"id":"27765","type":"Range1d"},{"attributes":{"children":[{"id":"27803"}]},"id":"27806","type":"Row"},{"attributes":{"fill_color":{"field":"color"},"line_color":{"field":"color"},"radius":{"field":"rad","units":"data"},"x":{"field":"x"},"y":{"field":"y"}},"id":"27789","type":"Circle"},{"attributes":{},"id":"27734","type":"LinearScale"},{"attributes":{"active_drag":"auto","active_inspect":"auto","active_multi":null,"active_scroll":"auto","active_tap":"auto","tools":[{"id":"27781"},{"id":"27782"},{"id":"27783"}]},"id":"27784","type":"Toolbar"},{"attributes":{"data":{"authors":["Song-Kyoo Kim","Albiger, Barbara; Revez, Joana; Leitmeyer, Katrin Claire; Struelens, Marc J.","HIETPAS, JACK; MCMULLAN, LAURA K.; HANSON, HOLLY L.; RICE, CHARLES M.; MINDELL, DAVID P.","Sui, Jianhua; Aird, Daniel R.; Tamin, Azaibi; Murakami, Akikazu; Yan, Meiying; Yammanuru, Anuradha; Jing, Huaiqi; Kan, Biao; Liu, Xin; Zhu, Quan; Yuan, Qing-an; Adams, Gregory P.; Bellini, William J.; Xu, Jianguo; Anderson, Larry J.; Marasco, Wayne A.","Al-Tayib, Omar A.","Jamieson, Denise J.; Ellis, Jane E.; Jernigan, Daniel B.; Treadwell, Tracee A.","Tianzhi Wu; Xijin Ge; Guangchuang Yu; Erqiang Hu","Li Yan; Hai-Tao Zhang; Jorge Goncalves; Yang Xiao; Maolin Wang; Yuqi Guo; Chuan Sun; Xiuchuan Tang; Liang Jin; Mingyang Zhang; Xiang Huang; Ying Xiao; Haosen Cao; Yanyan Chen; Tongxin Ren; Fang Wang; Yaru Xiao; Sufang Huang; Xi Tan; Niannian Huang; Bo Jiao; Yong Zhang; Ailin Luo; Laurent Mombaerts; Junyang Jin; Zhiguo Cao; Shusheng Li; Hui Xu; Ye Yuan","Wu, Kailang; Peng, Guiqing; Wilken, Matthew; Geraghty, Robert J.; Li, Fang","Huang, Weiwei; Yang, Yinhui; Zhang, Xinlei; Zhao, Changan; Yin, Aihua; Zhang, Xiaozhuang; He, Zhengxin; Jiang, Yongqiang; Zhang, Liang","Ganova-Raeva, Lilia M; Khudyakov, Yury E","Alejandro Lopez-Rincon; Alberto Tonda; Lucero Mendoza-Maldonado; Eric Claassen; Johan Garssen; Aletta D. Kraneveld","Chenyu Li; David N. Debruyne; Julia Spencer; Vidushi Kapoor; Lily Y. Liu; Bo Zhou; Lucie Lee; Rounak Feigelman; Grayson Burdon; Jeffrey Liu; Alejandra Oliva; Adam Borcherding; Hongdong Tan; Alexander E. Urban; Guoying Liu; Zhitong Liu","John-Sebastian Eden; Rebecca Rockett; Ian Carter; Hossinur Rahman; Joep de Ligt; James Hadfield; Matthew Storey; Xiaoyun Ren; Rachel Tulloch; Kerri Basile; Jessica Wells; Roy Byun; Nicky Gilroy; Matthew V O\u2019Sullivan; Vitali Sintchenko; Sharon C Chen; Susan Maddocks; Tania C Sorrell; Edward C Holmes; Dominic E Dwyer; Jen Kok","Walter R Hsiang; Howard P Forman; Siddharth Jain; Akshay Khunte; Grace Jin; Laurie Yousman; Alison Mosier-Mills; Michael Najem; Daniel Wiznia","Anton Nekrutenko; Sergei L Kosakovsky Pond","Hong Zhou; Xing Chen; Tao Hu; Juan Li; Hao Song; Yanran Liu; Peihan Wang; Di Liu; Jing Yang; Edward C. Holmes; Alice C. Hughes; Yuhai Bi; Weifeng Shi","Yang, Yang; Islam, Md Sahidul; Wang, Jin; Li, Yuan; Chen, Xin","Domachowske, Joseph B.","Rao, Lang; Tian, Rui; Chen, Xiaoyuan","Yafei Wang; Ying Zhou; Zhen Yang; Dongping Xia; Shuang Geng","Bexfield, Nicholas; Kellam, Paul","Humphries, Linda","Tianmu Chen; Jia Rui; Qiupeng Wang; Zeyu Zhao; Jing-An Cui; Ling Yin","Holmes, Kathryn V.; Dominguez, Samuel R.","Dong, J.; Ismail, N.; Walker, D.H.","Tam, T","Falsey, Ann R.","Butt, Taimur S.; Koutlakis-Barron, Irene; AlJumaah, Suliman; AlThawadi, Sahar; AlMofada, Saleh","Niccolo Alfano; Anisha Dayaram; Jan Axtner; Kyriakos Tsangaras; Marie-Louise Kampmann; Azlan Mohamed; Seth Timothy Wong; M. Thomas P. Gilbert; Andreas Wilting; Alex Daivd Greenwood","Harthill, Michalann","Baudart, Julia; Paniel, Nathalie","Javier A. Jaimes; Nicole M. Andr\u00e9; Jean K. Millet; Gary R. Whittaker","Dongwan Kim; Joo-Yeon Lee; Jeong-Sun Yang; Jun Won Kim; V. Narry Kim; Hyeshik Chang","Kiselev, Daniel; Matsvay, Alina; Abramov, Ivan; Dedkov, Vladimir; Shipulin, German; Khafizov, Kamil","Artika, I Made; Wiyatno, Ageng; Ma'roef, Chairin Nisa","Jung, Sung-Mok; Kinoshita, Ryo; Thompson, Robin N.; Linton, Natalie M.; Yang, Yichi; Akhmetzhanov, Andrei R.; Nishiura, Hiroshi","Ewer, Katie; Sebastian, Sarah; Spencer, Alexandra J.; Gilbert, Sarah; Hill, Adrian V. S.; Lambe, Teresa","Rajan Gupta; Saibal Kumar Pal","Theiler, Regan N.; Rasmussen, Sonja A.; Treadwell, Tracee A.; Jamieson, Denise J.","Bonomi, Lucia; Ghilardi, Laura; Arnoldi, Ermenegildo; Tondini, Carlo Alberto; Bettini, Anna Cecilia","Wenjiao Chang; Yuru Shi; Yingjie Qi; Jiaxing Liu; Ting Liu; Zhaowu Chen; Wenjing Zhang; Mengmeng Wang; Dongfeng Liu; Ming Yin; Jing Xu; Yun Yang; Xiaowu Zhu; Jing Ge; Shu Zhu; Yong Gao; Xiaoling Ma","Villani, Alessia; Fabbrocini, Gabriella; Costa, Claudia; Scalvenzi, Massimiliano","Fausto, James; Hirano, Lianne; Lam, Daniel; Mehta, Amisha; Mills, Blair; Owens, Darrell; Perry, Elizabeth; Curtis, J. Randall","Shi, Weifeng; Li, Juan; Zhou, Hong; Gao, George F.","Kucharski, Adam J.; Edmunds, W. John","DeZure, Adam; Graham, Barney S.","Schwartz, David A.; Graham, Ashley L.","Ming Wang; Aisi Fu; Ben Hu; Yongqing Tong; Ran Liu; Jiashuang Gu; Jianghao Liu; Wen Jiang; Gaigai Shen; Wanxu Zhao; Dong Men; Lilei Yu; Zixin Deng; Yan Li; Tiangang Liu","Ross, Samuel Wade; Lauer, Cynthia W.; Miles, William S.; Green, John M.; Christmas, A Britton; May, Addison K.; Matthews, Brent D."],"color":["#67a9cf","#67a9cf","#67a9cf","#67a9cf","#67a9cf","#67a9cf","#67a9cf","#67a9cf","#67a9cf","#67a9cf","#67a9cf","#67a9cf","#67a9cf","#67a9cf","#67a9cf","#67a9cf","#67a9cf","#67a9cf","#67a9cf","#67a9cf","#67a9cf","#67a9cf","#67a9cf","#67a9cf","#67a9cf","#67a9cf","#67a9cf","#67a9cf","#67a9cf","#67a9cf","#67a9cf","#67a9cf","#67a9cf","#67a9cf","#67a9cf","#67a9cf","#67a9cf","#67a9cf","#67a9cf","#67a9cf","#67a9cf","#67a9cf","#67a9cf","#67a9cf","#67a9cf","#67a9cf","#67a9cf","#67a9cf","#67a9cf","#67a9cf"],"date":["2020-03-20","2018-02-26","2005-12-31","2008-11-07","2019-02-22","2006-06-30","2020-02-27","2020-03-01","2012-01-30","2013-09-20","2013-05-01","2020-03-14","2020-03-14","2020-03-17","2020-03-27","2020-02-22","2020-03-05","2020-03-15","2004-10-15","2020-03-04","2020-03-03","2011-11-30","2005-11-30","2020-01-19","2013-01-08","2017-12-31","2018-05-03","2007-08-31","2016-05-01","2020-03-29","2011-02-12","2014-02-28","2020-02-18","2020-03-14","2020-02-14","2020-07-31","2020","2017-10-30","2020-03-30","2008-12-31","2020-03-31","2020-03-23","2020-03-31","2020-03-31","2017-11-28","2015-04-10","2016-12-31","2020","2020-03-06","2020-03-30"],"doc":["jf36as70","hq8dg87u","qma1lsuw","ttiqxlts","oudj079q","86hc3lbi","qtrquuvv","s9j21zsy","c4q5gtw2","cxqzac4a","qh2kuzdc","c2lljdi7","0hxan9rw","izkz1hz4","600am3mq","v0yzur5h","dbowa5bt","c3geck81","4dqz8req","2r790nix","qlkt5fzp","bxu3sklz","gfcmn5it","v4mbry22","krvei97r","1h87wldg","lnz2laha","v0gqyrn5","1qy6y7f0","nil1vv6h","166su0tn","rtpkd23d","xa6kwguo","wt4pxu08","u2n8so5w","wq5zqzvv","9quf59td","opi8aa5i","w2uqaz8p","nnwytx7u","2wnsfkls","zknngzt2","xrsh3hn2","zawfotcx","k0f9fser","9sj9bgq2","kwavs4mx","iy9vjpuh","s4d442kx","t624zyyc"],"journal":["","Front Public Health","Microbial Forensics","PLoS Pathog","Pathogens","American Journal of Obstetrics and Gynecology","","","Journal of Biological Chemistry","BMC Infect Dis","Expert Review of Molecular Diagnostics","","","","","","","Int J Biol Sci","Clinical Microbiology Newsletter","ACS Nano","","The Veterinary Journal","AORN Journal","","mBio","Diagnostic Molecular Pathology","Canada Communicable Disease Report","Clinics in Geriatric Medicine","American Journal of Infection Control","","Biol Trace Elem Res","Revue Francophone des Laboratoires","","","Viruses","Infection, Genetics and Evolution","J Clin Med","Hum Vaccin Immunother","","Infectious Disease Clinics of North America","Journal of Thoracic Oncology","","Journal of the American Academy of Dermatology","Journal of Pain and Symptom Management","Sci China Life Sci","PLoS Comput Biol","The Vaccine Book","Viruses","","Journal of the American College of Surgeons"],"rad":[0.02,0.018499826099930335,0.014832330142041952,0.014761314725371458,0.01468014918723072,0.01433527730798861,0.014215964440283675,0.013657130381545358,0.013389413897558499,0.013354434465821435,0.012937246104206487,0.011237566682919205,0.011191813179206405,0.010141342302874893,0.009596341960185682,0.009506401575960926,0.009475588332365028,0.00908034407932079,0.009016996044649783,0.008461789468021433,0.008339352837795903,0.007763647773323013,0.007630253173495389,0.006916147926626717,0.006855230411101783,0.006761415852850863,0.0066435567443739445,0.006610473668400965,0.005972695511335393,0.00580358480488345,0.00550225154739601,0.005455092939703341,0.004701671907814784,0.004547117175412936,0.004507477164791816,0.004448594564324825,0.004137824213139508,0.004011368057006914,0.0039455557519462335,0.0039031067977137964,0.003901053129977466,0.0038385876520119246,0.0038196415022899847,0.003625718707695078,0.0035513486963102696,0.0034372104622138132,0.003343784394259477,0.0032183230829957972,0.003088847133149579,0.003],"scr":[1.0,0.912,0.696,0.692,0.687,0.667,0.66,0.627,0.611,0.609,0.585,0.485,0.482,0.42,0.388,0.383,0.381,0.358,0.354,0.321,0.314,0.28,0.272,0.23,0.227,0.221,0.214,0.212,0.175,0.165,0.147,0.144,0.1,0.091,0.089,0.085,0.067,0.059,0.056,0.053,0.053,0.049,0.048,0.037,0.032,0.026,0.02,0.013,0.005,0.0],"text":["This paper deals with an advanced analytical epidemic diffusion model which is capable to predict the status of epidemic impacts. This newly propose model well describes an epidemic growth and it could be widely applied into various topics including pathology, epidemiology, business and data sciences. The Advanced Analytical Epidemic Diffusion Model (AAEDM) is a dynamic diffusion prediction model which is theoretically intuitive and its tractable closed formula could be easily adapted into versatile Bigdata driven analytics including the machine learning system. This dynamic model is still an analytical model but the periods of prediction are segmented for adapting the values from the dataset when the data is available. The epidemiologically vital parameters which effect on the AAEDM are also introduced in this paper. The evaluation of this theoretical model based on the Covid-19 data in Korea has been accomplished with relative fair future prediction accuracies. Although this analytic...","In an era of global health threats caused by epidemics of infectious diseases and rising multidrug resistance, microbiology laboratories provide essential scientific evidence for risk assessment, prevention, and control. Microbiology has been at the core of European infectious disease surveillance networks for decades. Since 2010, these networks have been coordinated by the European Centre for Disease Prevention and Control (ECDC). Activities delivered in these networks include harmonization of laboratory diagnostic, antimicrobial susceptibility and molecular typing methods, multicentre method validation, technical capacity mapping, training of laboratory staff, and continuing quality assessment of laboratory testing. Cooperation among the European laboratory networks in the past 7 years has proved successful in strengthening epidemic preparedness by enabling adaptive capabilities for rapid detection of emerging pathogens across Europe. In partnership with food safety authorities, inte...","Publisher Summary This chapter introduces basic concepts of virology and exemplifies the usage of current technology for identifying and tracking viral pathogens. Virus structure, lifestyle, replication strategies, classication, the varied modes of transmission, infection, and disease manifestations are focussed upon in the chapter. Viruses are extremely simple \u201clife\u201d forms without metabolic capacity, organelles, translational machinery, or autonomous replicative potential; the essential elements of a virus are consequently minimal. A virus enters the host through any opportunistic opening of the body\u2014; such as minute breaks in the skin barrier or mucosal linings of the eye, nose, etc. Antibodies produced during an infection often have the ability to bind the virus and reduce infectivity. An immediate question to solve during a virus outbreak is the mechanism by which the virus spreads. Viruses frequently infect animal or insect vectors that serve to pass the virus to humans. By identi...","Phylogenetic analyses have provided strong evidence that amino acid changes in spike (S) protein of animal and human SARS coronaviruses (SARS-CoVs) during and between two zoonotic transfers (2002/03 and 2003/04) are the result of positive selection. While several studies support that some amino acid changes between animal and human viruses are the result of inter-species adaptation, the role of neutralizing antibodies (nAbs) in driving SARS-CoV evolution, particularly during intra-species transmission, is unknown. A detailed examination of SARS-CoV infected animal and human convalescent sera could provide evidence of nAb pressure which, if found, may lead to strategies to effectively block virus evolution pathways by broadening the activity of nAbs. Here we show, by focusing on a dominant neutralization epitope, that contemporaneous- and cross-strain nAb responses against SARS-CoV spike protein exist during natural infection. In vitro immune pressure on this epitope using 2002/03 strai...","Currently, there has been an increasing socioeconomic impact of zoonotic pathogens transmitted from animals to humans worldwide. Recently, in the Arabian Peninsula, including in Saudi Arabia, epidemiological data indicated an actual increase in the number of emerging and/or reemerging cases of several viral zoonotic diseases. Data presented in this review are very relevant because Saudi Arabia is considered the largest country in the Peninsula. We believe that zoonotic pathogens in Saudi Arabia remain an important public health problem; however, more than 10 million Muslim pilgrims from around 184 Islamic countries arrive yearly at Makkah for the Hajj season and/or for the Umrah. Therefore, for health reasons, several countries recommend vaccinations for various zoonotic diseases among preventive protocols that should be complied with before traveling to Saudi Arabia. However, there is a shortage of epidemiological data focusing on the emerging and reemerging of zoonotic pathogens tran...","Objective The purpose of this study was to summarize 3 recent high-profile infectious disease threats that have affected the United States: severe acute respiratory syndrome, West Nile virus, and anthrax. Study design A systematic review was conducted with the use of Medline searches, searches of the Centers for Disease Control and Prevention website, and review by experts at the Centers for Disease Control and Prevention. Results The 3 emerging infectious diseases pose very different threats: Severe acute respiratory syndrome is a newly identified pathogen that caused an international pandemic; the West Nile virus investigation involved an old pathogen that was identified in a new location; and the anthrax attacks involved the intentional introduction of a pathogen. Conclusion All 3 outbreaks highlight the importance of obstetrician-gynecologists keeping current with new information as it emerges. In this global environment, it is likely that novel disease threats will continue to eme...","To provide convenient access to epidemiological data on the coronavirus outbreak, we developed an R package, nCov2019 (https://github.com/GuangchuangYu/nCov2019). Besides detailed real-time statistics, it offers access to three data sources with detailed daily statistics from December 1, 2019, for 43 countries and more than 500 Chinese cities. We also developed a web app (http://www.bcloud.org/e/) with interactive plots and simple time-series forecasts. These analytics tools could be useful in informing the public and studying how this and similar viruses spread in populous countries.","The sudden increase of COVID-19 cases is putting a high pressure on healthcare services worldwide. At the current stage, fast, accurate and early clinical assessment of the disease severity is vital. To support decision making and logistical planning in healthcare systems, this study leverages a database of blood samples from 404 infected patients in the region of Wuhan, China to identify crucial predictive biomarkers of disease severity. For this purpose, machine learning tools selected three biomarkers that predict the survival of individual patients with more than 90% accuracy: lactic dehydrogenase (LDH), lymphocyte and high-sensitivity C-reactive protein (hs-CRP). In particular, relatively high levels of LDH alone seem to play a crucial role in distinguishing the vast majority of cases that require immediate medical attention. This finding is consistent with current medical knowledge that high LDH levels are associated with tissue breakdown occurring in various diseases, including ...","The severe acute respiratory syndrome coronavirus (SARS-CoV) from palm civets has twice evolved the capacity to infect humans by gaining binding affinity for human receptor angiotensin-converting enzyme 2 (ACE2). Numerous mutations have been identified in the receptor-binding domain (RBD) of different SARS-CoV strains isolated from humans or civets. Why these mutations were naturally selected or how SARS-CoV evolved to adapt to different host receptors has been poorly understood, presenting evolutionary and epidemic conundrums. In this study, we investigated the impact of these mutations on receptor recognition, an important determinant of SARS-CoV infection and pathogenesis. Using a combination of biochemical, functional, and crystallographic approaches, we elucidated the molecular and structural mechanisms of each of these naturally selected RBD mutations. These mutations either strengthen favorable interactions or reduce unfavorable interactions with two virus-binding hot spots on A...","BACKGROUND: Infectious diseases emerge frequently in China, partly because of its large and highly mobile population. Therefore, a rapid and cost-effective pathogen screening method with broad coverage is required for prevention and control of infectious diseases. The availability of a large number of microbial genome sequences generated by conventional Sanger sequencing and next generation sequencing has enabled the development of a high-throughput high-density microarray platform for rapid large-scale screening of vertebrate pathogens. METHODS: An easy operating pathogen microarray (EOPM) was designed to detect almost all known pathogens and related species based on their genomic sequences. For effective identification of pathogens from EOPM data, a statistical enrichment algorithm has been proposed, and further implemented in a user-friendly web-based interface. RESULTS: Using multiple probes designed to specifically detect a microbial genus or species, EOPM can correctly identify k...","Mass spectrometry (MS) has found numerous applications in life sciences. It has high accuracy, sensitivity and wide dynamic range in addition to medium- to high-throughput capabilities. These features make MS a superior platform for analysis of various biomolecules including proteins, lipids, nucleic acids and carbohydrates. Until recently, MS was applied for protein detection and characterization. During the last decade, however, MS has successfully been used for molecular diagnostics of microbial and viral infections with the most notable applications being identification of pathogens, genomic sequencing, mutation detection, DNA methylation analysis, tracking of transmissions, and characterization of genetic heterogeneity. These new developments vastly expand the MS application from experimental research to public health and clinical fields. Matching of molecular techniques with specific requirements of the major MS platforms has produced powerful technologies for molecular diagnosti...","One of the reasons for the fast spread of SARS-CoV-2 is the lack of accuracy in detection tools in the clinical field. Molecular techniques, such as quantitative real-time RT-PCR and nucleic acid sequencing methods, are widely used to identify pathogens. For this particular virus, however, they have an overall unsatisfying detection rate, due to its relatively recent emergence and still not completely understood features. In addition, SARS-CoV-2 is remarkably similar to other Coronaviruses, and it can present with other respiratory infections, making identification even harder. To tackle this issue, we propose an assisted detection test, combining molecular testing with deep learning. The proposed approach employs a state-of-the-art deep convolutional neural network, able to automatically create features starting from the genome sequence of the virus. Experiments on data from the Novel Coronavirus Resource (2019nCoVR) show that the proposed approach is able to correctly classify SARS-C...","AbstractMany detection methods have been used or reported for the diagnosis and/or surveillance of SARS-CoV-2. Among them, reverse transcription polymerase chain reaction (RT-PCR) is the most sensitive, claiming detection of about 5 copies of viruses. However, it has been reported that only 47-59% of the positive cases were identified by RT-PCR, probably due to loss or degradation of virus RNA in the sampling process, or even mutation of the virus genome. Therefore, developing highly sensitive methods is imperative to ensure robust detection capabilities. With the goal of improving sensitivity and accommodate various application settings, we developed a multiplex-PCR-based method comprised of 172 pairs of specific primers, and demonstrated its efficiency to detect SARS-CoV-2 at low copy numbers. The assay produced clean characteristic target peaks of defined sizes, which allowed for direct identification of positives by electrophoresis. In addition, optional sequencing can provide furt...","AbstractThe SARS-CoV-2 epidemic has rapidly spread outside China with major outbreaks occurring in Italy, South Korea and Iran. Phylogenetic analyses of whole genome sequencing data identified a distinct SARS-CoV-2 clade linked to travellers returning from Iran to Australia and New Zealand. This study highlights potential viral diversity driving the epidemic in Iran, and underscores the power of rapid genome sequencing and public data sharing to improve the detection and management of emerging infectious diseases.","As of March 22, 2020, the number of confirmed COVID-19 cases in the U.S. has reached nearly 30,000. While rapid and accessible diagnosis is paramount to monitoring and reducing the spread of disease, COVID-19 testing capabilities across the U.S. remain constrained. For many individuals, urgent care centers (UCCs) may offer the most accessible avenue to be tested. Through a phone survey, we describe the COVID-19 testing capabilities at UCCs and provide a snapshot highlighting the limited COVID-19 testing capabilities at UCCs in states with the greatest disease burden.","The current state of much of the Wuhan pneumonia virus (COVID-19) research shows a regrettable lack of data sharing and considerable analytical obfuscation. This impedes global research cooperation, which is essential for tackling public health emergencies, and requires unimpeded access to data, analysis tools, and computational infrastructure. Here we show that community efforts in developing open analytical software tools over the past ten years, combined with national investments into scientific computational infrastructure, can overcome these deficiencies and provide an accessible platform for tackling global health emergencies in an open and transparent manner. Specifically, we use all COVID-19 genomic data available in the public domain so far to (1) underscore the importance of access to raw data and to (2) demonstrate that existing community efforts in curation and deployment of biomedical software can reliably support rapid, reproducible research during global health crises. A...","SummaryThe unprecedented epidemic of pneumonia caused by a novel coronavirus, HCoV-19, in China and beyond has caused public health concern at a global scale. Although bats are regarded as the most likely natural hosts for HCoV-191,2, the origins of the virus remain unclear. Here, we report a novel bat-derived coronavirus, denoted RmYN02, identified from a metagenomics analysis of samples from 227 bats collected from Yunnan Province in China between May and October, 2019. RmYN02 shared 93.3% nucleotide identity with HCoV-19 at the scale of the complete virus genome and 97.2% identity in the 1ab gene in which it was the closest relative of HCoV-19. In contrast, RmYN02 showed low sequence identity (61.3%) to HCoV-19 in the receptor binding domain (RBD) and might not bind to angiotensin-converting enzyme 2 (ACE2). Critically, however, and in a similar manner to HCoV-19, RmYN02 was characterized by the insertion of multiple amino acids at the junction site of the S1 and S2 subunits of the ...","Currently, Severe Acute Respiratory Syndrome Coronavirus 2 (SARS-CoV-2, formerly known as 2019-nCoV, the causative pathogen of Coronavirus Disease 2019 (COVID-19)) has rapidly spread across China and around the world, causing an outbreak of acute infectious pneumonia. No specific anti-virus drugs or vaccines are available for the treatment of this sudden and lethal disease. The supportive care and non-specific treatment to ameliorate the symptoms of the patient are the only options currently. At the top of these conventional therapies, greater than 85% of SARS-CoV-2 infected patients in China are receiving Traditional Chinese Medicine (TCM) treatment. In this article, relevant published literatures are thoroughly reviewed and current applications of TCM in the treatment of COVID-19 patients are analyzed. Due to the homology in epidemiology, genomics, and pathogenesis of the SARS-CoV-2 and SARS-CoV, and the widely use of TCM in the treatment of SARS-CoV, the clinical evidence showing th...","Abstract Advances in molecular biology, bioinformatics, and robotics have allowed microarray technology to be used for in-depth, basic science studies in all fields of microbiology. Recently, translation of these basic science applications to clinical microbiology and infectious diseases has also progressed. From a clinical infectious disease perspective, genome-based organism identification, pathogen discovery, and antimicrobial susceptibility testing of problematic organisms offer the potential to yield diagnostic information that may not otherwise become available. Moreover, microarray-based studies have the ability to provide \u201csignatures\u201d of host cell transcriptional responses for individual pathogens and/or groups of pathogens. This type of information has the potential to confirm difficult diagnoses, to monitor responses to therapeutic intervention, or even to predict prognosis and sequelae following an infectious disease. Examples are presented to illustrate ways in which microa...","[Image: see text] Infectious diseases are a leading cause of mortality worldwide, with viruses and bacteria in particular having enormous impacts on global healthcare. One major challenge in combatting such diseases is a lack of effective drugs or specific treatments. In addition, drug resistance to currently available therapeutics and adverse effects caused by long-term overuse are both serious public health issues. A promising treatment strategy is to employ cell-membrane mimics as decoys to trap and to detain the pathogens. In this Perspective, we briefly review the infection mechanisms adopted by different pathogens at the cellular membrane interface and highlight the applications of cell-membrane-mimicking nanodecoys for systemic protection against infectious diseases. We also discuss the implication of nanodecoy\u2013pathogen complexes in the development of vaccines. We anticipate this Perspective will provide new insights on design and development of advanced materials against emergi...","Abstract Background: A new virus broke out in Wuhan, Hubei, China, and was later named 2019 novel coronavirus (2019-nCoV). The clinical characteristics of severe pneumonia caused by 2019-nCoV are still not clear. Objectives: The aim of this study was to explore the clinical characteristics and risk factors of the severe pneumonia caused by the 2019-nCoV in Wuhan, China. Method: The study included patients hospitalized at the central hospital of Wuhan who had been diagnosed with a pneumonia caused by the novel coronavirus. Clinical features, chronic co-morbidities, demographic data, laboratory examinations, and chest computed tomography (CT) scans were reviewed through electronic medical records. SPSS was used for data analysis to explore the clinical characteristics and risk factors of the patients with the severe pneumonia. Results: A total of 110 patients diagnosed with 2019 novel coronavirus pneumonia were included in the study, including 38 with severe pneumonia and 72 with non-sev...","Abstract There have been rapid recent developments in establishing methods for identifying and characterising viruses associated with animal and human diseases. These methodologies, commonly based on hybridisation or PCR techniques, are combined with advanced sequencing techniques termed \u2018next generation sequencing\u2019. Allied advances in data analysis, including the use of computational transcriptome subtraction, have also impacted the field of viral pathogen discovery. This review details these molecular detection techniques, discusses their application in viral discovery, and provides an overview of some of the novel viruses discovered. The problems encountered in attributing disease causality to a newly identified virus are also considered.","ABSTRACT \u2022 ADVANCED TECHNOLOGY and improved surgical techniques have led to new therapeutic uses for allografts. \u2022 DISEASE TRANSMISSION via allograft tissue transplants has prompted federal intervention in the tissue banking industry and resulted in federal regulations. \u2022 NEW STANDARDS from the Joint Commission on Accreditation of Healthcare Organizations became effective July 1,2005, and apply to all hospitals that store or implant allograft tissues. These standards include mandatory policies on all aspects of hospital transplantation programs, including tissue ordering, receipt, storage, issuance, and record keeping. AORN J 82 (November 2005) 806\u2013814.","AbstractAs reported by the World Health Organization, a novel coronavirus (2019-nCoV) was identified as the causative virus of Wuhan pneumonia of unknown etiology by Chinese authorities on 7 January, 2020. In this study, we developed a Bats-Hosts-Reservoir-People transmission network model for simulating the potential transmission from the infection source (probable be bats) to the human infection. Since the Bats-Hosts-Reservoir network was hard to explore clearly and public concerns were focusing on the transmission from a seafood market (reservoir) to people, we simplified the model as Reservoir-People transmission network model. The basic reproduction number (R0) was calculated from the RP model to assess the transmissibility of the 2019-nCoV.","A new human betacoronavirus in lineage c, tentatively called HCoV-EMC, was isolated from a patient from the Kingdom of Saudi Arabia who died from acute severe pneumonia and renal failure. The viral RNA has been detected in eight additional cases. Sequencing and bioinformatic analysis of the viral genomic RNA showed that it is a novel virus not previously detected in any other species and that its closest relatives are two Asian bat coronaviruses. HCoV-EMC may represent a sporadic spillover to humans from an unknown animal reservoir. In a recent article, van Boheemen et al. demonstrated how state-of-the-art sequencing technology and bioinformatic analysis can quickly provide critical insight into the viral genome sequence, phylogeny, replication strategy, and potential drug and vaccine targets and generate tools to evaluate the possible epidemic risk associated with this novel human virus.","Abstract It was widely believed in the late 1960s that infectious diseases had been conquered by vaccines and antibiotics and humans were no longer under threat by microbial pathogens. Yet, since that time more than 60 pathogens have been discovered that can cause serious emerging infectious diseases. Molecular methods have played critical roles in the discovery, monitoring, and clinical diagnostics of emerging pathogens. In this chapter, we present well-recognized emerging pathogens. We provide examples of the utility of molecular assays in research and clinical care of emerging infectious diseases. We also discuss some theoretical and practical limitations of molecular tests and the future prospects of expanding molecular diagnostics for emerging pathogens based on new advances of knowledge and technologies.","This year marks the 15th anniversary of Severe Acute Respiratory Syndrome (SARS) in Canada and the 100th anniversary of the 1918 Spanish influenza pandemic. These, and other recent public health events, provide an opportunity for us to review and reflect on the evolution of Canada\u2019s public health emergency response over the past 15 years\u2014from SARS, to the 2009 H1N1 pandemic influenza, to Ebola virus and Zika virus disease. Key lessons have been learned and milestones achieved that have shaped and sharpened our response approach and structures. While SARS was a wake-up-call to strengthen infection prevention and control capacity in health care settings and led to the formation of the Public Health Agency of Canada, it also strengthened our Federal/Provincial/Territorial (FPT) senior-level governance and led to agreements for pan-Canadian mutual aid and infectious disease information sharing. As well, our collective public health laboratory capacity has been strengthened through ongoing ...","Advanced age often is associated with functional and immunologic decline and chronic cardiopulmonary diseases that predispose to pneumonia when viral infection occurs. Influenza virus remains the primary viral pathogen in the elderly, although the impact of the other respiratory viruses remains to be defined. The clinical syndromes associated with respiratory viruses frequently are indistinguishable from one another or bacterial pathogens; often, viral illness in older adults exacerbates underlying conditions, complicating diagnosis. Antiviral therapy is available for influenza A and B; specific viral diagnosis, particularly with the use of rapid antigen detection, may be useful for clinical management. Treatment for other viruses primarily is supportive.","Background Transmission of Middle East respiratory syndrome-coronavirus (MERS-CoV) among health care workers (HCWs) and patients has been documented with mortality rate approximating 36%. We propose advanced infection control measures (A-IC) used in conjunction with basic infection control measures (B-IC) help reduce pathogen transmission. B-IC include standard and transmission-based precautions. A-IC are initiatives implemented within our center to enhance effectiveness of B-IC. Objective Study effectiveness of combining B-IC and A-IC to prevent transmission of MERS-CoV to HCWs. Methods A retrospective observational study was undertaken. A-IC measures include administrative support with daily rounds; infection control risk assessment; timely screening, isolation, and specimen analysis; collaboration; epidemic planning; stockpiling; implementation of contingency plans; full personal protective equipment use for advanced airway management; use of a real-time electronic isolation flaggin...","Environmental DNA (eDNA) and its subdiscipline, invertebrate-derived DNA (iDNA) have been used to survey biodiversity non-invasively [1,2]. Water is ubiquitous in most ecosystems, and, among invertebrates, terrestrial haematophagous leeches are abundant and can be easily collected in many tropical rainforests [3,4]. Such non-invasive nucleic acid sources can mitigate difficulties of obtaining wildlife samples, particularly in remote areas or for rare species. Recently, eDNA/iDNA sources have been applied to monitoring specific wildlife pathogens [5,6]. However, previous studies have focused on known pathogens, whereas most wildlife pathogens are uncharacterized and unknown. Non-invasive approaches to monitoring known and novel pathogens may be of particular benefit in ecosystems prone to viral emergence, many of which occur in areas where invasive sampling is challenging, such as tropical rainforests. Here, we show that both eDNA from natural waterholes, and iDNA from terrestrial haema...","Recently emerged viral infectious diseases (VIDs) include HIV/AIDS, influenzas H5N1 and 2009 H1N1, SARS, and Ebola hemorrhagic fevers. Earlier research determined metabolic oxidative stress in hosts deficient in antioxidant selenium (Se) (&lt;1 \u03bcMol Se/L of blood) induces both impaired human host immunocompetence and rapidly mutated benign variants of RNA viruses to virulence. These viral mutations are consistent, rather than stochastic, and long-lived. When Se-deficient virus-infected hosts were supplemented with dietary Se, viral mutation rates diminished and immunocompetence improved. Herein is described the role of micronutrient Se deficiency on the evolution of some contemporary RNA viruses and their subsequent VIDs. Distinguishing cellular and biomolecular evidence for several VIDs suggests that environmental conditions conducive to chronic dietary Se deprivation could be monitored for bioindicators of incipient viral virulence and subsequent pathogenesis.","R\u00e9sum\u00e9 Un grand nombre d\u2019infections humaines d\u2019origine hydrique sont caus\u00e9es par des micro-organismes qu\u2019il s\u2019agisse de virus, de bact\u00e9ries ou encore de protozoaires. Ces micro-organismes sont soit naturellement pr\u00e9sents dans les environnements aquatiques, soit transf\u00e9r\u00e9s au sein de ces derniers via des sources d\u2019origine f\u00e9cale. Ils s\u00e9journent dans ces environnements pendant un temps plus ou moins long avant de contaminer un nouvel h\u00f4te. Les environnements aquatiques sont soumis \u00e0 une forte variabilit\u00e9 de leurs param\u00e8tres physicochimiques et abritent une vaste communaut\u00e9 microbienne plus ou moins adapt\u00e9e \u00e0 ces changements environnementaux. Ainsi les interactions entre les micro-organismes, qu\u2019ils soient pathog\u00e8nes ou non, sont nombreuses et certaines d\u2019entre elles sont encore aujourd\u2019hui m\u00e9connues. L\u2019objectif de cet article est de pr\u00e9senter les interactions connues entre certains micro-organismes pathog\u00e8nes et le milieu naturel en abordant notamment la diversit\u00e9 des sources de contamin...","AbstractThe 2019 novel coronavirus (2019-nCoV) is currently causing a widespread outbreak centered on Hubei province, China and is a major public health concern. Taxonomically 2019-nCoV is closely related to SARS-CoV and SARS-related bat coronaviruses, and it appears to share a common receptor with SARS-CoV (ACE-2). Here, we perform structural modeling of the 2019-nCoV spike glycoprotein. Our data provide support for the similar receptor utilization between 2019-nCoV and SARS-CoV, despite a relatively low amino acid similarity in the receptor binding module. Compared to SARS-CoV, we identify an extended structural loop containing basic amino acids at the interface of the receptor binding (S1) and fusion (S2) domains, which we predict to be proteolytically-sensitive. We suggest this loop confers fusion activation and entry properties more in line with MERS-CoV and other coronaviruses, and that the presence of this structural loop in 2019-nCoV may affect virus stability and transmission.","AbstractSARS-CoV-2 is a betacoronavirus that is responsible for the COVID-19 pandemic. The genome of SARS-CoV-2 was reported recently, but its transcriptomic architecture is unknown. Utilizing two complementary sequencing techniques, we here present a high-resolution map of the SARS-CoV-2 transcriptome and epitranscriptome. DNA nanoball sequencing shows that the transcriptome is highly complex owing to numerous recombination events, both canonical and noncanonical. In addition to the genomic RNA and subgenomic RNAs common in all coronaviruses, SARS-CoV-2 produces a large number of transcripts encoding unknown ORFs with fusion, deletion, and/or frameshift. Using nanopore direct RNA sequencing, we further find at least 41 RNA modification sites on viral transcripts, with the most frequent motif being AAGAA. Modified RNAs have shorter poly(A) tails than unmodified RNAs, suggesting a link between the internal modification and the 3\u2032 tail. Functional investigation of the unknown ORFs and RN...","Viruses are evolving at an alarming rate, spreading and inconspicuously adapting to cutting-edge therapies. Therefore, the search for rapid, informative and reliable diagnostic methods is becoming urgent as ever. Conventional clinical tests (PCR, serology, etc.) are being continually optimized, yet provide very limited data. Could high throughput sequencing (HTS) become the future gold standard in molecular diagnostics of viral infections? Compared to conventional clinical tests, HTS is universal and more precise at profiling pathogens. Nevertheless, it has not yet been widely accepted as a diagnostic tool, owing primarily to its high cost and the complexity of sample preparation and data analysis. Those obstacles must be tackled to integrate HTS into daily clinical practice. For this, three objectives are to be achieved: (1) designing and assessing universal protocols for library preparation, (2) assembling purpose-specific pipelines, and (3) building computational infrastructure to s...","Abstract Pathogenic viruses are viruses that can infect and replicate within human cells and cause diseases. The continuous emergence and re-emergence of pathogenic viruses has become a major threat to public health. Whenever pathogenic viruses emerge, their rapid detection is critical to enable implementation of specific control measures and the limitation of virus spread. Further molecular characterization to better understand these viruses is required for the development of diagnostic tests and countermeasures. Advances in molecular biology techniques have revolutionized the procedures for detection and characterization of pathogenic viruses. The development of PCR-based techniques together with DNA sequencing technology, have provided highly sensitive and specific methods to determine virus circulation. Pathogenic viruses potentially having global catastrophic consequences may emerge in regions where capacity for their detection and characterization is limited. Development of a loc...","Virological tests have now shown conclusively that a novel coronavirus is causing the 2019-2020 atypical pneumonia outbreak in Wuhan, China. We demonstrate that non-virological descriptive characteristics could have determined that the outbreak is caused by a novel pathogen in advance of virological testing. Characteristics of the ongoing outbreak were collected in real time from two medical social media sites. These were compared against characteristics of eleven pathogens that have previously caused cases of atypical pneumonia. The probability that the current outbreak is due to \"Disease X\" (i.e., previously unknown etiology) as opposed to one of the known pathogens was inferred, and this estimate was updated as the outbreak continued. The probability (expressed as a percentage) that Disease X is driving the outbreak was assessed as over 29% on 31 December 2019, one week before virus identification. After some specific pathogens were ruled out by laboratory tests on 5 January 2020, t...","The 2014\u201315 Ebola outbreak in West Africa highlighted the potential for large disease outbreaks caused by emerging pathogens and has generated considerable focus on preparedness for future epidemics. Here we discuss drivers, strategies and practical considerations for developing vaccines against outbreak pathogens. Chimpanzee adenoviral (ChAd) vectors have been developed as vaccine candidates for multiple infectious diseases and prostate cancer. ChAd vectors are safe and induce antigen-specific cellular and humoral immunity in all age groups, as well as circumventing the problem of pre-existing immunity encountered with human Ad vectors. For these reasons, such viral vectors provide an attractive platform for stockpiling vaccines for emergency deployment in response to a threatened outbreak of an emerging pathogen. Work is already underway to develop vaccines against a number of other outbreak pathogens and we will also review progress on these approaches here, particularly for Lassa f...","COVID-19 is spreading really fast around the world. The current study describes the situation of the outbreak of this disease in India and predicts the number of cases expected to rise in India. The study also discusses the regional analysis of Indian states and presents the preparedness level of India in combating this outbreak. The study uses exploratory data analysis to report the current situation and uses time-series forecasting methods to predict the future trends. The data has been considered from the repository of John Hopkins University and covers up the time period from 30th January 2020 when the first case occurred in India till the end of 24th March 2020 when the Prime Minister of India declared a complete lockdown in the country for 21 days starting 25th March 2020. The major findings show that number of infected cases in India is rising quickly with the average infected cases per day rising from 10 to 73 from the first case to the 300th case. The current mortality rate fo...","Emerging infections, many zoonotic, are caused by a variety of pathogens with global distribution. Previously rare pathogens have emerged; global travel facilitates their rapid spread. Human encroachment on remote areas has brought contact with zoonotic diseases never before characterized. Although systematic study of rare outbreaks can be challenging, knowledge of emerging pathogens and their effects on women is accumulating. This article discusses effects of lymphocytic choriomeningitis virus, West Nile virus, severe acute respiratory syndrome coronavirus, avian influenza A virus, viral hemorrhagic fevers, spirochetal illnesses, and Chagas' disease. The potential impact of candidate bioterror agents and issues of prophylaxis and therapy are discussed.","","Novel coronavirus pneumonia (NCP) is an emerging, highly contagious community acquired pneumonia (CAP) caused by severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2). Highly efficient and accurate microbiological laboratory assay is essential to confirm the SARS-CoV-2 infection, rule out other pathogens that can cause CAP, and monitor secondary infections. Here, we enrolled and provide microbiological analysis for 129 suspected and 52 transferred confirmed NCP patients hospitalized in the First Affiliated Hospital of University of Science and Technology of China (USTC) from Jan 21 to Feb 29, 2020. By analyzing the dual swab samples (sputum and pharyngeal) from 129 suspected patients with realtime RT-PCR, we confirmed 33 SARS-CoV-2 infections, with two co-infection cases with adenovirus or rhinovirus. We also used multiplex PCR to detect 13 common respiratory tract pathogens in 96 non-NCP patients, and found that 30 patients (31.25%) were infected with at least one respiratory t...","","Abstract Introduction The COVID-19 pandemic is stressing healthcare systems throughout the world. Significant numbers of patients are being admitted to the hospital with severe illness, often in the setting of advanced age and underlying co-morbidities. Therefore, palliative care is an important part of the response to this pandemic. The Seattle area and UW Medicine have been on the forefront of the pandemic in the US. Methods UW Medicine developed a strategy to implement a palliative care response for a multi-hospital healthcare system that incorporates conventional capacity, contingency capacity, and crisis capacity. The strategy was developed by our palliative care programs with input from the healthcare system leadership. Results In this publication, we share our multi-faceted strategy to implement high-quality palliative care in the context of the COVID-19 pandemic that incorporates conventional, contingency, and crisis capacity and focuses on the areas of the hospital caring for ...","In the past twenty years, numerous novel zoonotic viral agents with pandemic potential have emerged in China, such as the severe acute respiratory syndrome (SARS) coronavirus and, more recently, the avian-origin influenza A/H7N9 virus, which have caused outbreaks among humans with high morbidity and mortality. In addition, several emerging and re-emerging viral pathogens have also been imported into China from travelers, e.g. the Middle East respiratory syndrome (MERS) coronavirus and Zika virus (ZIKV). Herein, we review these emerging viral pathogens in China and focus on how surveillance by pathogen genomics has been employed to discover and annotate novel pathogenic agents, identify natural reservoirs, monitor the transmission events and delineate their evolution and adaption to the human host. We also highlight the application of genomic sequencing in the recent Ebola epidemics in Western Africa. In summary, genomic sequencing has become a standard research tool in the field of eme...","The transmission potential of a novel infection depends on both the inherent transmissibility of a pathogen, and the level of susceptibility in the host population. However, distinguishing between these pathogen- and population-specific properties typically requires detailed serological studies, which are rarely available in the early stages of an outbreak. Using a simple transmission model that incorporates age-stratified social mixing patterns, we present a novel method for characterizing the transmission potential of subcritical infections, which have effective reproduction number R&lt;1, from readily available data on the size of outbreaks. We show that the model can identify the extent to which outbreaks are driven by inherent pathogen transmissibility and pre-existing population immunity, and can generate unbiased estimates of the effective reproduction number. Applying the method to real-life infections, we obtained accurate estimates for the degree of age-specific immunity against...","Abstract Infectious diseases continue to plague mankind and evolve to keep pace with the efforts to control them. Sir William Osler captured the ongoing fear of infectious pathogens when he said, \u201cHumanity has but three great enemies: fever, famine, and war; of these by far the greatest, by far the most terrible, is fever.\u201d Despite the significant impact of antimicrobials and vaccines on public health, there has only been one major human pathogen eradicated\u2014variola virus, the agent of smallpox. In its place have been a series of new and reemerging microbes responsible for isolated infections, regional outbreaks, and global pandemics. Bacterial, fungal, and parasitic pathogens have the capacity to cause widespread epidemics such as the \u201cBlack Death\u201d caused by Yersinia pestis in 14th century Europe. However, bacteria, fungi, and parasites are less likely to cause widespread human pandemics at this point in history and are less amenable to vaccine strategies than viral diseases. Focusing ...","In early December 2019 a cluster of cases of pneumonia of unknown cause was identified in Wuhan, a city of 11 million persons in the People&amp;rsquo;s Republic of China. Further investigation revealed these cases to result from infection with a newly identified coronavirus, termed the 2019-nCoV. The infection moved rapidly through China, spread to Thailand and Japan, extended into adjacent countries through infected persons travelling by air, eventually reaching multiple countries and continents. Similar to such other coronaviruses as those causing the Middle East respiratory syndrome (MERS) and severe acute respiratory syndrome (SARS), the new coronavirus was reported to spread via natural aerosols from human-to-human. In the early stages of this epidemic the case fatality rate is estimated to be approximately 2%, with the majority of deaths occurring in special populations. Unfortunately, there is limited experience with coronavirus infections during pregnancy, and it now appears certai...","The ongoing novel coronavirus pneumonia COVID-19 outbreak in Wuhan, China, has engendered numerous cases of infection and death. COVID-19 diagnosis relies upon nucleic acid detection; however, current recommended methods exhibit high false-negative rates, low sensitivity, and cannot identify other respiratory virus infections, thereby resulting patient misdiagnosis and impeding epidemic containment. Combining the advantages of target amplification and long-read, real-time nanopore sequencing, we developed nanopore target sequencing (NTS) to detect SARS-CoV-2 and other respiratory viruses simultaneously within 6-10 h. Parallel testing with approved qPCR kits of SARS-CoV-2 and NTS using 61 nucleic acid samples from suspected COVID-19 cases confirmed that NTS identified more infected patients as positive, and could also monitor for mutated nucleic acid sequence or other respiratory virus infection in the test sample. NTS is thus suitable for contemporary COVID-19 diagnosis; moreover, this...","Abstract The novel coronavirus (COVID-19) was first diagnosed in Wuhan, China in December 2019 and has now spread throughout the world, being verified by the World Health Organization as a Pandemic on March 11th. This had led to the calling of a national emergency on March 13th in the United States. Many hospitals, healthcare networks, and specifically Departments of Surgery are asking the same questions of how to cope and plan for surge capacity, personnel attrition, novel infrastructure utilization, and resource exhaustion. Herein, we present a tiered plan for surgical department planning based on incident command levels. This includes Acute Care Surgeon deployment (given their critical care training and vertically integrated position in the hospital), recommended infrastructure and transfer utilization, triage principles, and faculty, resident and advanced care practitioner deployment."],"title":["AAEDM: Theoretical Dynamic Epidemic Diffusion Model and Covid-19 Korea Pandemic Cases","Networking of Public Health Microbiology Laboratories Bolsters Europe\u2019s Defenses against Infectious Diseases","CHAPTER 4 Keeping Track of Viruses","Broadening of Neutralization Activity to Directly Block a Dominant Antibody-Driven SARS-Coronavirus Evolution Pathway","An Overview of the Most Significant Zoonotic Viral Pathogens Transmitted from Animal to Human in Saudi Arabia","Emerging infectious disease outbreaks: Old lessons and new challenges for obstetrician-gynecologists","Open-source analytics tools for studying the COVID-19 coronavirus outbreak","A machine learning-based model for survival prediction in patients with severe COVID-19 infection","Mechanisms of Host Receptor Adaptation by Severe Acute Respiratory Syndrome Coronavirus","An easy operating pathogen microarray (EOPM) platform for rapid screening of vertebrate pathogens","Application of mass spectrometry to molecular diagnostics of viral infections","Accurate Identification of SARS-CoV-2 from Viral Genome Sequences using Deep Learning","High sensitivity detection of coronavirus SARS-CoV-2 using multiplex PCR and a multiplex-PCR-based metagenomic method","An emergent clade of SARS-CoV-2 linked to returned travellers from Iran","Scarce COVID-19 Testing Capabilities at Urgent Care Centers in States with Greatest Disease Burden","No more business as usual: agile and effective responses to emerging pathogen threats require open data and open analytics","A novel bat coronavirus reveals natural insertions at the S1/S2 cleavage site of the Spike protein and a possible recombinant origin of HCoV-19","Traditional Chinese Medicine in the Treatment of Patients Infected with 2019-New Coronavirus (SARS-CoV-2): A Review and Perspective","Microarrays and gene expression profiling in microbiology and infectious diseases: a clinician's perspective","Cell-Membrane-Mimicking Nanodecoys against Infectious Diseases","Clinical Characteristics of Patients with Severe Pneumonia Caused by the 2019 Novel Coronavirus in Wuhan, China","Metagenomics and the molecular identification of novel viruses","Decreasing Latitude and Increasing Regulation in Transplantable Tissue Programs","A mathematical model for simulating the transmission of Wuhan novel Coronavirus","The New Age of Virus Discovery: Genomic Analysis of a Novel Human Betacoronavirus Isolated from a Fatal Case of Pneumonia","Chapter 15 Molecular Testing in Emerging Infectious Diseases","Fifteen years post-SARS: Key milestones in Canada\u2019s public health emergency response","Community-Acquired Viral Pneumonia","Infection control and prevention practices implemented to reduce transmission risk of Middle East respiratory syndrome-coronavirus in a tertiary care institution in Saudi Arabia","Non-invasive surveys of mammalian viruses using environmental DNA","Review: Micronutrient Selenium Deficiency Influences Evolution of Some Viral Infectious Diseases","Sources et devenir des micro-organismes pathog\u00e8nes dans les environnements aquatiques","Structural modeling of 2019-novel coronavirus (nCoV) spike protein reveals a proteolytically-sensitive activation loop as a distinguishing feature compared to SARS-CoV and related SARS-like coronaviruses","The architecture of SARS-CoV-2 transcriptome","Current Trends in Diagnostics of Viral Infections of Unknown Etiology","Pathogenic viruses: Molecular detection and characterization","Epidemiological Identification of A Novel Pathogen in Real Time: Analysis of the Atypical Pneumonia Outbreak in Wuhan, China, 2019-2020","Chimpanzee adenoviral vectors as vaccines for outbreak pathogens","Trend Analysis and Forecasting of COVID-19 outbreak in India","Emerging and Zoonotic Infections in Women","A rapid fatal evolution of Coronavirus Disease-19 (COVID-19) in an advanced lung cancer patient with a long time response to nivolumab","Optimization of Microbiological Laboratory Detection Strategy for Patients in A Designated Hospital Treating Novel Coronavirus Pneumonia in Anhui Province","Patients with advanced basal cell carcinomas in treatment with sonic hedgehog inhibitors during the coronavirus disease 2019 (COVID-19) period: Management and adherence to treatment","Creating a Palliative Care Inpatient Response Plan for COVID19 \u2013 The UW Medicine Experience","Pathogen genomic surveillance elucidates the origins, transmission and evolution of emerging viral agents in China","Characterizing the Transmission Potential of Zoonotic Infections from Minor Outbreaks","Chapter 28 Vaccines for Emerging Viral Diseases","Potential Maternal and Infant Outcomes from (Wuhan) Coronavirus 2019-nCoV Infecting Pregnant Women: Lessons from SARS, MERS, and Other Human Coronavirus Infections","Nanopore target sequencing for accurate and comprehensive detection of SARS-CoV-2 and other respiratory viruses","Maximizing the Calm Before the Storm: Tiered Surgical Response Plan for Novel Coronavirus (COVID-19)"],"url":["https://doi.org/10.1101/2020.03.17.20037838","https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5834927/","https://doi.org/10.1016/b978-012088483-4/50007-x","https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2572002/","https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6471281/","https://doi.org/10.1016/j.ajog.2005.06.062","https://doi.org/10.1101/2020.02.25.20027433","https://doi.org/10.1101/2020.02.27.20028027","http://www.jbc.org/content/287/12/8904.full.pdf","https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3848773/","http://europepmc.org/articles/pmc5831079?pdf=render","https://doi.org/10.1101/2020.03.13.990242","https://doi.org/10.1101/2020.03.12.988246","https://doi.org/10.1101/2020.03.15.992818","https://doi.org/10.1101/2020.03.22.20040923","https://doi.org/10.1101/2020.02.21.959973","https://doi.org/10.1101/2020.03.02.974139","https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7098036/","https://doi.org/10.1016/j.clinmicnews.2004.09.004","https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7094139/","https://doi.org/10.1101/2020.03.02.20029306","https://doi.org/10.1016/j.tvjl.2010.10.014","https://doi.org/10.1016/s0001-2092(06)60273-1","https://doi.org/10.1101/2020.01.19.911669","https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3546555/","https://doi.org/10.1016/b978-0-12-800886-7.00015-7","https://doi.org/10.14745/ccdr.v44i05a01","https://doi.org/10.1016/j.cger.2007.02.002","https://doi.org/10.1016/j.ajic.2016.01.004","https://doi.org/10.1101/2020.03.26.009993","https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7090490/","https://doi.org/10.1016/s1773-035x(14)72362-7","https://doi.org/10.1101/2020.02.10.942185","https://doi.org/10.1101/2020.03.12.988865","https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7077230/","https://doi.org/10.1016/j.meegid.2020.104215","https://doi.org/10.3390/jcm9030637","https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5718829/","https://doi.org/10.1101/2020.03.26.20044511","https://doi.org/10.1016/j.idc.2008.05.007","https://doi.org/10.1016/j.jtho.2020.03.021","https://doi.org/10.1101/2020.03.21.20039065","https://doi.org/10.1016/j.jaad.2020.03.057","https://doi.org/10.1016/j.jpainsymman.2020.03.025","https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7088571/","https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4393285/","https://doi.org/10.1016/b978-0-12-802174-3.00028-x","https://doi.org/10.3390/v12020194","https://doi.org/10.1101/2020.03.04.20029538","https://doi.org/10.1016/j.jamcollsurg.2020.03.019"],"x":[0.2832795953916468,0.23444890777846558,0.41108454380901754,0.7608509908403822,0.3526894774599586,0.3442732193073181,0.34280671377607663,0.6739242509428596,0.749922960277827,0.3717205624305352,0.40374231503120955,0.6357877511852513,0.8559313905047766,0.6174161038754923,0.4812803735265867,0.28848442779096595,0.4914673406594185,0.6429749460865108,0.3727181213338299,0.34078299709027704,0.6507301781304541,0.42998967099835717,0.429657954677696,0.22785258821902063,0.4803501279995843,0.33553717182786097,0.24664668767371944,0.4521993833413539,0.96813512196655,0.44186006701341185,0.49813270520786607,0.06279277633897125,0.22067658744402166,0.6475362866601669,0.40409383241138586,0.4249528186666759,0.3749345037944474,0.4863226006750425,0.33781831933079026,0.3661713436230668,0.4976909168118937,0.6770590766945586,0.5606027975634981,0.4637432292409121,0.35665703849095537,0.3564005143844181,0.35396088341426574,0.5753189490856095,0.8396877247815662,0.4539511386614783],"y":[0.844166745010011,0.5352753114785103,0.21068323151696952,0.2619969997679806,0.309518220806156,0.3852025593407175,0.6156819607218764,0.8661492918389704,0.14289900585436088,0.353804743004535,0.34680773217488753,0.331907869393853,0.5910462507906155,0.3230246944299596,0.7668252472102253,0.5028442398249459,0.0629432350179669,0.6121505823019504,0.35812793418313027,0.34748225884879264,0.754902366688619,0.2186010025371088,0.46719864127397404,0.7060784054356691,0.07476187654915503,0.33560158284107094,0.5489032446365116,0.2711823000557306,0.4323572299618369,0.2098914430696562,0.23662391020396478,0.34573026908789295,0.3748713676028313,0.30443459472560436,0.34453598993568213,0.2246769614399934,0.571415073995299,0.33726118944945827,0.8549119672430735,0.2974218949676004,0.6286066798059821,0.6855867200936266,0.7108463485595437,0.593326165697304,0.30016772485490034,0.8044102550064314,0.3293306225523175,0.9113504801812895,0.5674766667507029,0.5913325818369065]},"selected":{"id":"27799"},"selection_policy":{"id":"27820"}},"id":"27727","type":"ColumnDataSource"},{"attributes":{},"id":"27767","type":"Range1d"},{"attributes":{},"id":"27771","type":"LinearScale"},{"attributes":{},"id":"27827","type":"StringEditor"},{"attributes":{"children":[{"id":"27806"},{"id":"27807"},{"id":"27808"},{"id":"27809"},{"id":"27810"}]},"id":"27811","type":"Column"},{"attributes":{},"id":"27826","type":"StringFormatter"},{"attributes":{},"id":"27828","type":"StringFormatter"},{"attributes":{},"id":"27769","type":"LinearScale"},{"attributes":{},"id":"27830","type":"UnionRenderers"},{"attributes":{},"id":"27774","type":"BasicTicker"},{"attributes":{"formatter":{"id":"27833"},"ticker":{"id":"27778"}},"id":"27777","type":"LinearAxis"},{"attributes":{},"id":"27732","type":"Range1d"},{"attributes":{"formatter":{"id":"27835"},"ticker":{"id":"27774"}},"id":"27773","type":"LinearAxis"},{"attributes":{"axis":{"id":"27773"},"ticker":null},"id":"27776","type":"Grid"},{"attributes":{"editor":{"id":"27827"},"field":"scr","formatter":{"id":"27826"},"title":"Score","width":30},"id":"27795","type":"TableColumn"},{"attributes":{"data":{"authors":["Roux, Simon; Matthijnssens, Jelle; Dutilh, Bas E.","Khabbaz, Rima F; Moseley, Robin R; Steiner, Riley J; Levitt, Alexandra M; Bell, Beth P","Thacker, Stephen B.; Sencer, David J.; Jaffe, Harold W.","Jessica E. Manning; Jennifer A. Bohl; Sreyngim Lay; Sophana Chea; Ly Sovann; Yi Sengdoeurn; Seng Heng; Chan Vuthy; Katrina Kalantar; Vida Ahyong; Michelle Tan; Jonathan Sheu; Cristina M. Tato; Joseph L. DeRisi; Laurence Baril; Veasna Duong; Philippe Dussart; Erik A. Karlsson","Dowling, Nicole F; Gwinn, Marta; Mawle, Alison","Kapata, Nathan; Ihekweazu, Chikwe; Ntoumi, Francine; Raji, Tajudeen; Chanda-Kapata, Pascalina; Mwaba, Peter; Mukonka, Victor; Bates, Matthew; Tembo, John; Corman, Victor; Mfinanga, Sayoki; Asogun, Danny; Elton, Linzy; Arruda, Li\u00e3 B\u00e1rbara; Thomason, Margaret J.; Mboera, Leonard; Yavlinsky, Alexei; Haider, Najmul; Simons, David; Hollmann, Lara; Lule, Swaib A.; Veas, Francisco; Abdel Hamid, Muzamil Mahdi; Dar, Osman; Edwards, Sarah; Vairo, Francesco; McHugh, Timothy D.; Drosten, Christian; Kock, Richard; Ippolito, Giuseppe; Zumla, Alimuddin","Kostoff, Ronald N.; Morse, Stephen A.","Lee, Vernon J; Aguilera, Ximena; Heymann, David; Wilder-Smith, Annelies; Lee, Vernon J.; Aguilera, Ximena; Heymann, David L.; Wilder-Smith, Annelies; Bausch, Daniel G.; Briand, Sylvie; Bruschke, Christianne; Carmo, Eduardo H.; Cleghorn, Sean; Dandona, Lalit; Donnelly, Christl; Fall, Ibrahima Soc\u00e9; Halton, Jane; Hatchett, Richard; Hong, Felicia; Horby, Peter; Ihekweazu, Chikwe; Jacobs, Michael; Khan, Kamran; Lin, Yijun; Leung, Gabriel; Low, Constance; McDonald, Bethan F.; Memish, Ziad A.; Morhard, Ryan; Ng, Deborah HL; Nkengasong, John; Pang, Junxiong; Redd, Stephen C.; Tan, Karen; Yeo, Wen Qing","Jonathan Vu; Benjamin Kaplan; Shomesh Chaudhuri; Monique Mansoura; Andrew Lo","Zlateva, Kalina T.; Coenjaerts, Frank E. J.; Crusio, Kelly M.; Lammens, Christine; Leus, Frank; Viveen, Marco; Ieven, Margareta; Spaan, Willy J. M.; Claas, Eric C. J.; Gorbalenya, Alexander E.","Reusken, Chantal B; Haagmans, Bart; Meijer, Adam; Corman, Victor M; Papa, Anna; Charrel, Remi; Drosten, Christian; Koopmans, Marion","Chenyu Li; David N. Debruyne; Julia Spencer; Vidushi Kapoor; Lily Y. Liu; Bo Zhou; Lucie Lee; Rounak Feigelman; Grayson Burdon; Jeffrey Liu; Alejandra Oliva; Adam Borcherding; Hongdong Tan; Alexander E. Urban; Guoying Liu; Zhitong Liu","Uttamchandani, Mahesh; Neo, Jia Ling; Ong, Brandon Ngiap Zhung; Moochhala, Shabbir","Kostoff, Ronald N.; Morse, Stephen A.","Dong, J.; Ismail, N.; Walker, D.H.","Osterholm, Michael T.; Hedberg, Craig W.","Lai, Chih-Cheng; Wang, Cheng-Yi; Wang, Ya-Hui; Hsueh, Shun-Chung; Ko, Wen-Chien; Hsueh, Po-Ren","Minfeng Xiao; Xiaoqing Liu; Jingkai Ji; Min Li; Jiandong Li; Lin Yang; Wanying Sun; Peidi Ren; Guifang Yang; Jincun Zhao; Tianzhu Liang; Huahui Ren; Tian Chen; Huanzi Zhong; Wenchen Song; Yanqun Wang; Ziqing Deng; Yanping Zhao; Zhihua Ou; Daxi Wang; Jielun Cai; Xinyi Cheng; Taiqing Feng; Honglong Wu; Yanping Gong; Huanming Yang; Jian Wang; Xun Xu; Shida Zhu; Fang Chen; Yanyan Zhang; Weijun Chen; Yimin Li; Junhua Li","Ansari, Issmaeel; Grier, Gareth; Byers, Mark","Huang, Weiwei; Yang, Yinhui; Zhang, Xinlei; Zhao, Changan; Yin, Aihua; Zhang, Xiaozhuang; He, Zhengxin; Jiang, Yongqiang; Zhang, Liang","Hussey, Caroline","Yang, Weizhong; Lan, Yajia; Sun, Qiao; Wang, Jinfeng; Li, Zhongjie","Niu, Song; Xu, Mindan","Uttamchandani, Mahesh; Neo, Jia Ling; Ong, Brandon Ngiap Zhung; Moochhala, Shabbir","Estefania Nunez-Bajo; Michael Kasimatis; Yasin Cotur; Tarek Asfour; Alex Collins; Ugur Tanriverdi; Max Grell; Matti Kaisti; Guglielmo Senesi; Karen Stevenson; Firat Guder","Jiayu Shang; Yanni Sun","Jessica E. Manning; Jennifer A. Bohl; Sreyngim Lay; Sophana Chea; Ly Sovann; Yi Sengdoeurn; Seng Heng; Chan Vuthy; Katrina Kalantar; Vida Ahyong; Michelle Tan; Jonathan Sheu; Cristina M. Tato; Joseph L. DeRisi; Laurence Baril; Veasna Duong; Philippe Dussart; Erik A. Karlsson","Francesco, Cortiula; Pettke, Aleksandra; Michele, Bartoletti; Fabio, Puglisi; Helleday, Thomas","B Shayak; Mohit Manoj Sharma; Richard H Rand; Awadhesh Kumar Singh; Anoop Misra","Song-Kyoo Kim","Ivanov, Dmitry","Mani, Reeta S.; Ravi, V.; Desai, Anita; Madhusudana, S.N.","Brisse, Sylvain; Brehony, Carina; Concei\u00e7\u00e3o, Teresa; Cubero, Meritxell; Glasner, Corinna; Le Gouil, Meriadeg; Renvois\u00e9, Aur\u00e9lie; Sheppard, Sam; Weinert, Lucy A.","Liu, Shan-Lu; Saif, Linda J.; Weiss, Susan R.; Su, Lishan","Rajan Gupta; Saibal Kumar Pal","Ruiyun Li; Sen Pei; Bin Chen; Yimeng Song; Tao Zhang; Wan Yang; Jeffrey Shaman","Clark, David P.; Pazdernik, Nanette J.","Shuai Xia; Meiqin Liu; Chao Wang; Wei Xu; Qiaoshuai Lan; Siliang Feng; Feifei Qi; Linlin Bao; Lanying Du; Shuwen Liu; Chuan Qin; Fei Sun; Zhengli Shi; Yun Zhu; Shibo Jiang; Lu Lu","Ivanov, Dmitry","Henderson, Joan C.","Dong, J.; Ismail, N.; Walker, D.H.","Mani, Reeta S.; Ravi, V.; Desai, Anita; Madhusudana, S.N.","","Huang, Weiwei; Yang, Yinhui; Zhang, Xinlei; Zhao, Changan; Yin, Aihua; Zhang, Xiaozhuang; He, Zhengxin; Jiang, Yongqiang; Zhang, Liang","Chang, Chia-Chien; Yang, Alan H.","Mayer, J.D.","Uttamchandani, Mahesh; Neo, Jia Ling; Ong, Brandon Ngiap Zhung; Moochhala, Shabbir","Yang, Yang; Islam, Md Sahidul; Wang, Jin; Li, Yuan; Chen, Xin","Al-Tayib, Omar A.","Chen, Qi; Gauger, Phillip; Stafne, Molly; Thomas, Joseph; Arruda, Paulo; Burrough, Eric; Madson, Darin; Brodie, Joseph; Magstadt, Drew; Derscheid, Rachel; Welch, Michael; Zhang, Jianqiang"],"color":["#ef8a62","#ef8a62","#ef8a62","#ef8a62","#ef8a62","#ef8a62","#ef8a62","#ef8a62","#ef8a62","#ef8a62","#ef8a62","#ef8a62","#ef8a62","#ef8a62","#ef8a62","#ef8a62","#ef8a62","#ef8a62","#ef8a62","#ef8a62","#ef8a62","#ef8a62","#ef8a62","#ef8a62","#ef8a62","#ef8a62","#ef8a62","#ef8a62","#ef8a62","#ef8a62","#ef8a62","#ef8a62","#ef8a62","#ef8a62","#ef8a62","#ef8a62","#ef8a62","#ef8a62","#ef8a62","#ef8a62","#ef8a62","#ef8a62","#ef8a62","#ef8a62","#ef8a62","#ef8a62","#ef8a62","#ef8a62","#ef8a62","#ef8a62"],"date":["2019-12-31","2014-07-11","2017-12-31","2020-03-05","2009-12-29","2020-04-30","2010-05-30","2020-01-31","2020-03-23","2012-09-28","2020-02-27","2020-03-14","2009-01-31","2010-05-30","2017-12-31","2015-12-31","2020-03-19","2020-03-17","2020-03-30","2013-09-20","1992-12-31","2017-12-31","2018-06-12","2009-01-31","2020-03-25","2020-03-27","2020-03-05","2020-03-19","2020-04-01","2020-03-20","2020-04-30","2012-01-18","2014-03-31","2020-02-26","2020-03-30","2020-02-17","2016-12-31","2020-03-12","2020-04-30","2007-12-31","2017-12-31","2012-01-18","2017-09-30","2013-09-20","2020-03-04","2008-12-31","2009-01-31","2020-03-15","2019-02-22","2015-08-31"],"doc":["1qj9ri18","alnj42iy","9tn0w6n2","fu3cl7lq","03bwk538","caturc6q","2rrlt5z9","55snzm9h","ptgpxpad","2dsesgof","0syudviv","0hxan9rw","vgizgq2y","2rrlt5z9","1h87wldg","6kbfblvl","lsrqko6p","wbyqmvhs","huw7pnzu","cxqzac4a","os6f458c","dgi4ibym","hw0j6szd","vgizgq2y","7a3wdduq","pd0cmefo","fu3cl7lq","g0pes5o3","3ueg2i6w","jf36as70","8d1076ge","hlugf0xc","a4mf2vnp","xy1khp7w","w2uqaz8p","3nipr212","dbuy7vvw","j5bepvvw","8d1076ge","nezgzovk","1h87wldg","hlugf0xc","s5y6gyut","cxqzac4a","60xq8dpg","t99l3zii","vgizgq2y","c3geck81","oudj079q","p5kjnfyf"],"journal":["Reference Module in Life Sciences","The Lancet","International Encyclopedia of Public Health","","Genome Med","International Journal of Infectious Diseases","Scientometrics","The Lancet Infectious Diseases","","Arch Virol","Euro Surveill","","Trends in Biotechnology","Scientometrics","Diagnostic Molecular Pathology","Mandell, Douglas, and Bennett's Principles and Practice of Infectious Diseases","International Journal of Antimicrobial Agents","","Journal of Biosafety and Biosecurity","BMC Infect Dis","Safety in Industrial Microbiology and Biotechnology","Early Warning for Infectious Disease Outbreak","J Relig Health","Trends in Biotechnology","","","","Annals of Oncology","","","Transportation Research Part E: Logistics and Transportation Review","Proc Natl Acad Sci India Sect B Biol Sci","Research in Microbiology","Emerg Microbes Infect","","","Biotechnology","","Transportation Research Part E: Logistics and Transportation Review","Managing Tourism Crises","Diagnostic Molecular Pathology","Proc Natl Acad Sci India Sect B Biol Sci","Disease-a-Month","BMC Infect Dis","Orbis","International Encyclopedia of Public Health","Trends in Biotechnology","Int J Biol Sci","Pathogens","Virology"],"rad":[0.02,0.018647500904300267,0.017307111390617587,0.015671799922827225,0.01498379680536522,0.01496384460096959,0.01287582755107158,0.010226127166904592,0.009934687803760608,0.009749205687557518,0.009675418896836555,0.008669468979796997,0.008438774860947811,0.008438285137201112,0.007937568253094659,0.007708928120923019,0.007583340588112315,0.007171715282979213,0.006467855547673695,0.00644454610670795,0.006429483957297368,0.006087814513498343,0.005981590867862152,0.005937786933599743,0.005233041883506097,0.0052039899268454285,0.005177135375566628,0.00507616017806059,0.005058176394198758,0.00505090818693732,0.004931387756744229,0.004899123975405588,0.004832505024845303,0.004826398639377997,0.004287429699093215,0.004099899371193964,0.003903608326998991,0.0038286952314807035,0.003826114905265057,0.0035781780431592995,0.0035324366581729752,0.003503532419149905,0.003452258505691767,0.0034357314265272307,0.0033760683373552625,0.0032657829893304015,0.0030820807316683345,0.003079390925788658,0.0030670571004645583,0.003],"scr":[1.0,0.92,0.842,0.745,0.705,0.704,0.581,0.425,0.408,0.397,0.393,0.333,0.32,0.32,0.29,0.277,0.27,0.245,0.204,0.203,0.202,0.182,0.175,0.173,0.131,0.13,0.128,0.122,0.121,0.121,0.114,0.112,0.108,0.107,0.076,0.065,0.053,0.049,0.049,0.034,0.031,0.03,0.027,0.026,0.022,0.016,0.005,0.005,0.004,0.0],"text":["Metagenomics bypasses this requirement for cultivation and instead relies on the sequencing of viral genomic material extracted directly from a sample (see Box 1 and Fig. 1 ). Thus far, the history of viral metagenomics has seen two major phases. Initially, entire communities of viruses were assayed by analyzing and comparing short sequencing reads obtained from diverse environments. Because of the fragmented nature of these data, most of these studies had to be conducted at the community scale, and identifying and distinguishing individual viruses in these datasets remained challenging. More recently, bioinformatic advances have enabled the reconstruction of individual viral genome sequences from metagenomes, allowing naturally occurring viruses to be identified and studied at high, genomic resolution. Using a metagenomics approach, entirely new types of viruses can now be discovered, surveyed, and characterized even without cultivation. The unique ability offered by metagenomics to study uncultivated viruses led to the emergence of two parallel and interconnected fields: a clinical one, where metagenomics promises to be a catch-all method for the unbiased surveillance and diagnosis of viral pathogens, and one focused on natural biomes. that aims to describe the diversity of the viral world and understand its ecological and evolutionary drivers and impacts.","For any country, eff ective control of infectious diseases is dependent on focused eff orts to decrease endemic infections in at-risk and susceptible populations, combined with strong epidemiological and laboratory capacity to detect and stop emerging threats. Inherent in these eff orts are well established links between clinical, public health, and research communities working with non-traditional partners such as industry, law enforcement, businesses, and the media. Although at their core these collaborations need to be established locally, such crucial partnerships have to extend across state, national, and international levels. Together, these alliances can form a global infrastructure and system to detect and control infectious diseases. With that aim, in February, 2014, the USA and other nations joined WHO, the Food and Agricultural Organization of the United Nations, and the World Organization for Animal Health in a new global health security agenda designed to enhance strategies to prevent, detect, and respond to threats of infectious disease. 123 Partner countries will work together to meet a series of objectives covering crucial issues aff ecting control of infectious disease such as antimicrobial resistance, biosafety and biosecurity, global disease detection and response networks, and emergency responses. Much emphasis has been placed on the aptly named perfect storm inherent in today's globalised world for rapid emergence and spread of infectious diseases. Despite this persistent challenge, extraordinary achievements in science and technology continue to bolster the capacity to recognise and respond to these threats. Scientifi cally, next-generation sequencing and state-of-the-art bioinformatics capabilities are transforming the practice of microbiology and the speed at which infectious pathogens can be detected and analysed. 85,124 These technologies provide a new level of information about infectious pathogens and therefore off er tremendous opportunities for public health eff orts to detect and control outbreaks, target antimicrobial resistance, and develop and refi ne treatments and vaccines. [125] [126] [127] Additionally, the increasing understanding of the role of human microbiomes in disease development and new links between infectious agents and chronic diseases opens new possibilities for disease prevention and treatment. 128 Advances in electronic communications have also been essential and have enabled rapid sharing of health and safety information and expertise by health-care and public health workers, researchers, law enforcement, the media, and private individuals. These communication capacities can facilitate and enhance research and public health partnerships worldwide, generating broad sharing of research fi ndings and health updates in near real time. The latest surge in health communications has come from social media tools, which are increasingly being used to extend the reach of public health information. To ensure that these rapidly changing scientifi c and technological advances are routinely used in public health is a crucial challenge and priority.","New threats, both man-made and naturally occurring, faced CDC in the early years of the twenty-first century. The first of these threats came in 2001, with the terrorist attacks on the World Trade Center and the Pentagon followed by anthrax attacks. In response to these events, CDC received substantial increases in funding to enhance preparedness and response capabilities within the Agency and at state and local health departments. Both the Strategic National Stockpile (SNS) of vaccines, medications, and medical supplies as well as the Laboratory Response Network were expanded to improve responses to public health emergencies. This enhanced public health capacity proved critical to national responses to infectious disease outbreaks. In 2003 alone, these included the emergence of a novel coronavirus that sparked the SARS pandemic, the introduction of monkeypox associated with the exotic pet trade, and a resurgence of West Nile virus as it moved westward across the United States. These investments were also important in CDC's responses to environmental disasters including Hurricanes Katrina and Rita in 2005, and the Haitian earthquake and Gulf of Mexico (Deepwater Horizon) oil spill, both in 2010.","Overall, agnostic or unbiased metagenomic sequencing capabilities in-country provide the ability to detect and respond to a variety of pathogens, even those that are unanticipated or unknown. Bridging of existing local and global resources for sequencing and analysis allows for better realtime surveillance locally, while also enabling better health pursuits overall, not just during outbreaks. The example described here serves as a call for continued training and infrastructure to support mNGS capacity in developing countries as bioinformatic tools proliferate and the cost of sequencing decreases.","Despite major progress toward understanding infectious agents and controlling their spread, new and evolving infectious diseases - as well as old diseases in new contexts - continue to pose threats to humans worldwide. In 1992, the Institute of Medicine published an influential report calling attention to the emergence and re-emergence of human pathogens as a consequence of such factors as evolutionary changes in infectious agents and their human and non-human hosts; alterations in host behaviors and travel; and naturally occurring and man-made shifts in ecology, geography, and environment [1]. During the following decade, renewed concern about microbial threats to health spurred new investments in scientific research and public health infrastructure. In 2003 the Institute of Medicine published a report entitled Microbial Threats to Health, which highlighted the need for a global approach to preparedness [2]. That same year, the severe acute respiratory syndrome (SARS) epidemic acutely challenged the response capacity of scientists and public health officials across the globe [3,4]. Advances in high-throughput genome sequencing technology played a pivotal role in identifying the novel coronavirus associated with SARS and in facilitating the development of assays for diagnosis and control [5].","An important need remains for ensuring long-term sustainability of what is being built. Africa needs to continue its upward trajectory of activities so as to align public health resources, scientific expertise and experience, and political commitment so that any future infectious disease outbreaks can be stopped before they become an epidemic in Africa. Africa needs more investments into ONE-HEALTH collaborative activities across the continent in order to meet the challenges of current and future public health threats (Kock R,et al 2019; Petersen E et al, 2019; Talisuna AO et al, 2020; Hui DS et al, 2020; Zumla A et al, 2016) . A whole new young generation of enthusiastic, committed and dedicated African public health workers, epidemiologists, researchers, healthcare workers and laboratory personnel have emerged over the past 5 years, and they need to be supported by security of funding to build their careers and sustain their take forward their research and training portfolios. The future of Africa's public health security relies on them. Increased governmental and donor investments are required to advance locally led, world-class public health work with surveillance, data and analytics capabilities and further expanding state-of-the-art laboratory capacities with more J o u r n a l P r e -p r o o f trained personnel to sustain capacity to rapidly respond to outbreaks at their source. A wellplanned long-term strategy from the Africa Union will add major value for consolidating African leadership of public health capacity building, training and research.","How would these numbers change if SARS-CoV had been deliberately released by a bio-terrorist? No advanced technology would have been required. Infected intermediate hosts could have been distributed to multiple locations to serve as multiple sources for the infection. By the time the intentional nature of the outbreak was recognized, it would have spread globally, causing an overwhelming strain on healthcare resources throughout the world.","Failure of host countries to protect the health of refugees and migrants; epidemic of devastating rumours and fake news on social media due to increased digital connectivity; emergence of social influencers exerting influence on politicians and institutions; increased resistance and hesitancy within communities to health interventions because of opposition by local experts Environment Geography affecting biological diversity; planned and unplanned urbanisation; climate change; interaction between humans, animals, and vectors; human development; state of the economy; state of health systems Climate change resulting in increased flooding with failed sanitation and safe water; altered distribution of zoonotic disease reservoirs and vectors; emerging zoonosis with increased agricultural production and human encroachment into animal environments; changing national priorities resulting in sharply reduced investment in health systems Pathogenesis Changing disease biomes; relationship between hosts and pathogens; pathogen evolution and changes; technologies such as synthetic biology, and the risks of manufacturing pathogens and their accidental or deliberate release; characteristics of a population such as underlying disease condition Increased opportunities for mutation or reassortment of infectious agents; increasingly reduced effectiveness of conventional vaccines and therapeutics for prevention and treatment of diseases; failure of conventional control measures to break the chain of transmission of infectious agents To tackle the wide-ranging issues, the Commission has brought together experts from academic, public health, policy making, international, non-governmental, and corporate institutions. They bring local and global knowledge and experience, including policy-making and field response, human and animal health (including One Health) approaches, and novel developments in communications, information technology, analytics, public health, diagnostics, and therapeutics. The Commission aims to deliver the report by 2021 and will include key recommendations for countries and international stakeholders, and monitoring indicators to evaluate the effectiveness of preparedness initiatives over time. The Mok Hing Yiu Charitable Foundation is the sole sponsor for the Lancet Infectious Diseases Commission meetings and travel for some commissioners. The foundation is not involved in the technical discussions, data analysis, and drafting of the report. This Comment was written on the behalf of the Lancet Infectious Diseases Commission on Preparedness for Emerging Epidemic Threats; see appendix (pp 1-2) for the Commissions author list.","S.C. is a co-founder and chief technology officer of QLS Advisors, a healthcare analytics and consulting company.","The main challenge to the identification of new viruses is their unknown characteristics. One of the popular approaches that have succeeded is the use of either degenerate or consensus primers that recognize highly conserved genome regions of already known viruses. These primers can detect new pathogens that are closely related to members of established species. For the identification of distantly related unknown viruses, the combined consensus and degenerate primer design strategy implemented in the consensus-degenerate hybrid oligonucleotide primer (CODEHOP) technique proved to be superior [17\u201320]. Also, other genetic-based techniques that are sequence independent have been developed and used in virus discovery. For instance, the VIDISCA method, based on recognition of restricted enzyme cleavage sites, was introduced [5] and advanced by employing non-rRNA hexamers for random priming, high-throughput genome sequencing [21], and sample pooling using centrifugation [22]. It is highly sensitive and unbiased, but samples with an abundance of background RNA/DNA may not be suitable for VIDISCA, and its processivity remains relatively limited. Thus, CODEHOP- and VIDISCA-based techniques are complementary rather than an alternative for virus discovery at this stage of technology development.","Therefore, we maintain that the urgent implementation and monitoring of diagnostic capacities and capabilities for SARS-CoV-2 in Europe is proportional to the containment and expected mitigation phase of the global public health response and is critical for care of local patients. The implementation of SARS-CoV-2 diagnostics does not replace nor exclude diagnostic capacity for other respiratory viruses. Moreover, it explicitly does not disregard the importance of other seasonal respiratory pathogens for public and patient health. This is clearly illustrated by the authors themselves, who indicate that they have tested thousands of samples of patients suspected of respiratory viral disease not only for common respiratory pathogens but also for SARS-CoV-2. The fact that Colson et al. found a wide range of seasonal respiratory viruses indeed underlines the impact of such pathogens. Furthermore, it clearly shows that diagnosis of respiratory viruses cannot be done syndrome-based and supports the need for the availability of diagnostics for a panel of pathogens including SARS-CoV-2.","Furthermore, we also propose a metagenomic method for the potential identification of unknown pathogens. Metagenomic technology usually requires about 20 to 100 million of sequencing reads in order to detect minuscule numbers of targets embedded in the massive amounts of human background. However, our method appears to selectively amplify target sequences. This bias permits the obtention of around 16% of target bases by sequencing at a depth of about 1 million total reads. It is thus especially suitable for the detection of novel pathogens and highly genetically unstable pathogens, such as influenza viruses. The exact mechanism of deselecting human sequences is currently under investigation. The random hexamer preference, chromosomal structure, sequence composition, target length, circularity, methylation status, telomeric and centromeric regions, as well as the edge effect, may influence such outcome. Ultimately, the observed preferential amplification towards more \"random\" sequences could provide us with an advantageous edge for the continuous improvement of this method.","The applications of microarrays described here are broad-ranging and span from DNA-or protein-based detection of pathogens to epidemic outbreak surveillance and vaccine development, thus showcasing the full spectrum of microarray potential ( Table 1 ). The progress made in the past several years has brought microarrays to the forefront of rapid diagnostics and medical research. Further integration of upstream sample preparation with downstream data processing is expected to transform microarrays into compact, field expedient solutions for analysis and monitoring in the near future. The large assortments of biomolecules now utilized in microarrays provide novel opportunities in molecular forensics and comparative profiling, especially for the newer and less well-understood pathogens. As we equip ourselves with better capabilities and countermeasures against potential biological threats, we hope to become more agile and responsive whenever such threats emerge in the future. Microarrays have improved our confidence in this respect and will continue to play a decisive part in biodefence research and technology.","One of the less tangible weapons in the arsenal to combat both bio-terrorism and natural pandemics is intelligence. This has myriad forms, including the direct use of humans to access information, use of sophisticated computer systems to track infrastructure and resource movements, use of literature-related discovery techniques to help predict the development of engineered biological threat agents (Swanson et al. 2001) and to predict the emergence of naturally occurring pathogens, and development of technologies to detect, neutralize, shield and vaccinate against bio-warfare and naturally occurring agents.","The advances of sequencing technology, nanotechnology, and bioinformatics have driven molecular tests including assays for emerging pathogens to be more comprehensive and precise. For example, the availability of various sequence databases permits quick identification of sequence identity and variations. For example, the HIV database http://www.hiv.lanl.gov/ contains data on HIV genetic sequences and drug resistance associated mutations. It is valuable for HIV epidemiological studies, research, development, and clinical validation studies of HIV clinical assays [38, 71] . It is well known that there are significant variations of clinical phenotypes in the presence of emerging infections ranging from asymptomatic carrier to lethal infection. Recently, assays to examine multiple pathogen panels have been developed [97\u00c0102], which should increase the diagnostic yield for many pathogens. A critical need for emerging pathogen analysis is quicker, easier, cost-effect assays that can be used in a point-of-care setting. New assays that are performed on platforms with a small footprint and detect pathogens quickly (in minutes instead of hours or days) have entered clinical use. For example, the FilmArray (BioFire Diagnostics, Inc.) and Simplexa (Focus Diagnostics, Inc.) molecular assays can generate results in approximately 60 min. The user-friendly Alere i (Alere Inc.) and Cobas Liat (Roche Molecular Systems) platforms are compact and portable, generate rapid molecular results in 15\u00c020 min, can use electricity or rechargeable battery, and therefore are completely mobile and suited for point-of-care testing. It is obvious that the current rapid development of new technologies will further enhance the utility of molecular diagnostics in various emerging infectious diseases.","Several major constraints are confronted in the design of epidemiologic studies of infectious diseases. Time is frequently a problem in the investigation of outbreaks. The need to quickly design and conduct outbreak investigations has increased with the frequency of widespread foodborne outbreaks and concerns over the potential for intentional contamination of the food supply. This necessarily limits the investigator's ability to fully explore the outbreak setting and can result in the loss of information. In any study involving the retrospective collection of data, information may be lost because of difficulty in recalling exposure or in verifying information about the exposure.","Please cite this article as: C.-C. Lai spread. To manage this challenge, infection control measures including prohibiting travel from China, controlling the distribution of face masks, extensive investigation of COVID-19 spread, and a once-daily press conference by the government to inform and educate people were aggressively conducted in Taiwan ( Fig. 3 A) . Under these interventions, Taiwan has a total of 39 cases with a DCI of 1 case per day, which was much lower than that of the nearby countries such as the Republic of Korea and Japan ( Fig. 3 B) . Among the 39 patients, 18 acquired the COVID-19 infection outside of Taiwan, of whom 3 were Chinese visitors from China, whilst the other 21 developed the infection in Taiwan (http://nidss.cdc.gov.tw/ch/SingleDisease.aspx?dc = 1&amp;dt = 5&amp;disease = 19CoV). Four familial cluster transmissions and one nosocomial transmission were reported. To date, 11 patients were discharged ( Fig. 3 C) . Several important actions and responses conducted by the National Health Command Center of Taiwan, including border control, resource re-allocation, case identification (using new data and technology), quarantine of suspicious cases using big data analytics, reassurance and education of the public, mask control and hand hygiene, and formulation of policies toward schools and childcare, have so far successfully alleviated the crisis and contained the epidemic in Taiwan [7] .","sequencing has also allowed us to investigate RNA expression patterns of the overall mi-210 crobiome and host content and thus suitable for discovering new viruses, distinguishing co-211 infections, and dissecting virus-host interactions. To explore the microbiota, we performed 212 further metatranscriptomics analysis of the clinical samples. We were able to identify host 213 nucleic acids in all of the samples, and over 95% of total reads were from the host in sputum, 214 nasal, and throat samples ( Supplementary Fig. 4a ). Virus contributed to less than 5% of 215 reads in anal swab and throat swab while more than 50% of reads in nasal swab (Supple-216 mentary Fig. 4b ). These results suggest nasal swab could be the most ideal sample type for 217 viral detection among the four sample types, which agrees with recent clinical evidence 25 . 218","A deliberate or intentional release of plague is the act of using it as a weapon to infect a healthy population with the intention to cause harm. To understand the likelihood of such an attack, questions, such as, ''who would do such a thing and why?\", ''who would be capable?\", ''how would they go about obtaining plague or transporting it around the world covertly?\", need to be answered. Furthermore, this review will explore how someone could release plague, including the potential use of a ''suicide attacker\", the feasibility of antibiotic resistant plague, the danger of a plague mimic (including a theoretical mechanism to defend against it), explain the public health response and evaluate the likelihood of a deliberate release of plague.","Until now, the available genome-wide technologies to detect unknown pathogens in infectious outbreaks primarily consisted of microarrays and NGS. Although NGS can provide the most in-depth, unbiased information, and can reveal completely novel pathogens, it is time-consuming when the sample contains hundreds of microbial species that require comprehensive data processing. Therefore, NGS cannot meet the short time requirement for infectious disease control. However, the most complicated step in EOPM technology is probe design, which can be undertaken by a core bioinformatics team in the development phase. Once probe design is complete, and the whole microarray procedure is optimized as a standard procedure, pathogen screening results can be interpreted in less than 28 hours. Therefore, EOPM is more suitable for applications requiring detection of unknown pathogens during infectious outbreaks.","Many of the most exciting and potentially rewarding (financially and socially) projects in r-DNA technology involve the use of GMOs in open systems. This has proved to be a much more sensitive issue to scientists, regulatory agencies and the public than contained use. Such projects are classified as deliberate (or intentional) release.","(3) The combined use of multiple early warning technologies: Integrated prediction primarily refers to applying two or more prediction models, combining qualitative and quantitative predictions, and using spatial-temporal technologies for early warning. (4) Developing and utilizing new surveillance resources for early warning: It is important to explore new and available surveillance data for early warning, such as data from hospital emergency departments, OTC, biological media, animal epidemics, environmental meteorology, and to identify better ways to collect information and promote data use through multifactor analyses. (5) Early warning based on lab surveillance: The popularity and refinement of modern laboratory networks with molecular biology and bioinformatics will greatly enhance early identification of the intrinsic relations (e.g., gene fingerprints) among seemingly unrelated sporadic cases for early warning. It will substantially enhance the early identification of the epidemiological significance of pathogens, and provide strong evidence for the early warning of infectious disease outbreaks. (6) Evaluating early warning methods: Evaluating early warning methodology is also very important in current early warning research. This primarily refers to studying evaluation standards for early warning methods, systems, and test data. Sensitivity and specificity are two important indicators of evaluation. Other common indicators include the positive predictive value, receiver operating characteristic curve, activity monitoring operating characteristic, and the probability for successful detection.","With the commencement of large-scale human activities of trans-regional conquest, infectious diseases have become another major threat to individuals and groups besides the war and conflicts. Infectious disease refers to that, after being infected by the pathogenic microorganisms, the human bodies produce infectious factors that can cause epidemic diseases under certain conditions. Viruses, bacteria, fungi, and mycoplasma can cause infectious diseases, but the spread of infectious diseases need to meet three conditions: there must be a source of infection, a path of transmission, and susceptible populations. Source of infection refers to human or animals, which already have the growth and reproduction of the pathogen in their body and have been able to excrete the pathogen in certain ways. This includes a patient and a human or animal that carries the pathogen. The way that pathogens leave the source of infection and then reach another susceptible object is known as the route of transmission, including the transmission through respiratory tract (droplets), digestive tract, insect-borne, blood and other body fluids. People who suffer from a specific immune deficiency toward a certain disease are known as susceptible populations. The spread of infectious disease must meet the three prerequisites mentioned above at the same time.1 In addition to the three basic conditions, there are other factors that will affect the spread of infectious diseases, including natural factors and social factors. Natural factors mainly refer to the temperature, humidity, ecological conditions, animals and plants. Different pathogens have different growth and reproduction in different environment. Their capabilities to adapt to temperature and humidity, and their pH tolerance also vary. Social factors include population density, life habits, economic status and communication degree. First discovered in Guangdong, China, in 2002, the \u201cSevere Acute Respiratory Syndrome\u201d virus (SARS-CoV) is currently considered to spread across species lines from civet cats to human bodies. Nevertheless, we cannot deny that there is a link between the spread of the disease and the local eating habits. Historically, it will take months for worldwide pandemic disease to spread to different countries. However, nowadays, with the advanced aviation industry, the virus can spread from Asia to the Americas overnight, and soon causes worldwide epidemic diseases.","One important application of microarrays is in the resequencing of pathogen genomes, which has been successfully applied to interrogate inter-species and/or interstrain differences (Figure 2b ). This provides a vital alternative to traditional shotgun sequencing approaches because it is quicker and more cost-effective; however, the disadvantage is that the sequence information for the organism of interest must available beforehand. This is not a severe limitation given that many pathogenic organisms have or are being sequenced, and with the advances in de novo sequencing capabilities, new pathogens such as SARS can be readily sequenced in a matter of weeks from the time of discovery [28] . Subsequent resequencing using microarrays would only take several days. Early experiments on resequencing arrays showed high rates of false discovery of up to 12-45% [29] , but experimental improvements and the development of robust algorithms, such as the ABACUS (adaptive background genotype calling scheme) software package [30] , have now greatly improved sequencing quality.","Unlike other biomarkers, such as antibodies or non-genetic molecules originating from pathogens, NA can be chemically amplified, enabling direct detection of low numbers of pathogens (down to single organisms). Hence tests targeting NA tend to be exceptionally sensitive. NA-based reagents (e.g., primers) for NA testing can also be produced synthetically rapidly and on a large scale; DNA-based molecules, in particular, are highly stable, therefore, do not require a cold-chain for storage, making it especially suitable for PoN testing. 1 Ability to produce NA quickly also increases the speed of development and deployment of new test kits to health systems, providing much needed diagnostic capabilities for the detection of novel pathogens (such as SARS-CoV-2). Despite the massive advantages, on-site testing of NA has been limited.","In this section, we will first give details of the hyperparameters of CHEER. Then we present the results of applying CHEER to both simulated and mock sequencing data. For the simulated data by WgSim, we evaluated how sequencing error rate affects the performance of CHEER and compared the taxonomic classification results with state-of-the-art. For the mock data, we tested CHEER's performance in both taxonomic classification and its ability in distinguishing viral reads from others.","Here, we demonstrate that unbiased mNGS is a feasible and efficient means to detect pathogens in the midst of an outbreak of a novel virus. The index Cambodian SARS-CoV-2 sequence data uploaded to public databases represent one of the only sequences generated in a low-to-middle income country close to the outbreak epicenter. Despite major advances in mNGS technologies and significant decreases in costs associated with sequencing, preparation of sequencing libraries and sufficient bioinformatics capabilities for timely analysis still present a challenge in the developing world. 8 As cases of COVID-19 spread globally in mid-January, initial real-time PCR protocols took into account that the genetic diversity of SARS-CoV-2 in humans and animals was not completely known. Non-specificity combined with variable sample quality can hinder ability to detect the pathogen in question. In this scenario, our metagenomics approach demonstrated a first-pass high coverage of the N gene, closely aligning to diagnostic protocols utilizing both the E and N author/funder. All rights reserved. No reuse allowed without permission.","However, in addition to the apparent threats the COVID-19 poses, both to single individuals and health systems, this epidemic hides subtle menaces, like the distraction effect. It is mandatory, especially in a finite resource system to balance the potential benefit of containment measures, such as postponing scheduled procedures, with negative health and social costs. Diverting the attention exclusively to the COVID-19 situation and overshadowing the everyday clinical practice may have substantial negative implications, especially for cancer patients. Reallocating an excessive amount of healthcare personnel, both nurses and doctors, to the COVID-19 triage and management may stretch an already fragile system and potentially leave uncovered some vital activities, such as treatment administration, surgeries and inpatient assistance. It is well established that delayed oncologic surgery may lead to disease progressions and result in tumors no longer resectable, leading to worse survival outcomes. The same goes for neoadjuvant or adjuvant chemotherapy regimens administered with suboptimal timing. It should be emphasized that we are talking about patients potentially cured by oncologic treatments. Therefore, any delay of these fundamental procedures, either intentional or due to shortage of personnel, should be avoided. The same risk is present for the people who have scheduled screening activities (e.g. screening mammography for the early diagnosis of breast cancer). People should be advised to maintain their scheduled appointments, if the procedure is feasible safely, or at least to promptly reschedule their appointment when the epidemic it is expected to slow its pace. For patients with advanced disease, as oncologists we know the enormous negative impact from disease progression in terms of both survival time and quality of life. The inability to deliver palliative care to patients unable to move from their homes and the management of treatment side effects are other significant concerns from a forced quarantine [5] . Patients with advanced disease, and no suggestive symptoms of COVID-19, should keep receiving planned chemotherapy or radiotherapy treatment, without unnecessary delays. Moreover, although postponing follow-up and cancer prevention appointments is a strategy to be considered, an excessive accumulation of visits or examinations risks burdening the national public health system over the next few months.","For impactful epidemic management, societies need to model their strategies, develop an advanced surveillance system for early detection of cases, facilitate implementation of preventive interventions, and enable effective coordination of healthcare professionals, public health authorities, policy makers and government. Subsequent work will have to focus on extending and refining our model. For example, one can take into account the effects of errors in testing and intentional or unintentional violations of quarantine. According to the Institute of Medicine (IoM) categories, drivers of pathogen emergence like human demographics and behaviour, economic development and land use, international travel and commerce, changing ecosystems and human susceptibility should also be taken into consideration. One can also integrate our lumped-parameter model with more detailed analyses which account for heterogeneous spatial mixing and stochasticity in contracting the disease. Such steps will increase even more the predictive power of the model, and further help regional authorities in timing their lockdowns, and estimating the required capacities of quarantine and treatment facilities. * * * * *","\uf03e In the AAEDM, the and factors \uf02c \uf027 are determined as a constant value most based on empirical hunch. But the author strongly believes that and are time dependent stochastic functions. It \uf02c \uf027 \uf061 \uf062 \uf061 \uf062 \uf03e \uf03e could not be easily found analytically but the ML technique could build up the functions based on massive data analytics. Finding the impacts of kappa and zeta factors: Although these two parameters are epidemiologically important and key elements of the AAEDM formula, these factors are coming to the picture by empirical hunch and it is not statistically proven yet. Researchers might be interested in finding statistical relationships (i.e., correlation, p-value, hypothesis testing and so on) between the epidemic growth and these two parameters based on massive amount of data analytics.","Regarding further future directions, we point to the new digital technologies that have a potential to improve the ripple effect control in cases of epidemic outbreaks. Making innovations and data work for the SC resilience in crisis times is a promising future research avenue with a particular focus on data analytics, artificial intelligence, and machine learning. The understanding and progressing the research of how all these technologies can be used to operate the SCs in a resilient way in cases of epidemic outbreaks is an important future research area (Choi et al., 2017 (Choi et al., , 2019 Choi and Lambert, 2017b , Dubey et al., 2019a , Ganasegeran and Abdulrahman, 2020 , Queiroz and Wamba, 2019 , Yoon et al., 2019 . In particular, digital SC twins ) -i.e., the computerized SC models that represent the network state for any given moment in real time -can be used to support the decision-making during the epidemic outbreaks. In the pre-disruption mode, digital twins allow for a visualization of SC risks, assessment of supplier disruption risks, prediction of possible supply interruptions, and computation of alternative supply network topologies and back-up routes with assessment of estimated times of arrival. In the dynamic, reactive mode, i.e., in the case of a real epidemic outbreak the digital twins can be applied using real-time data to simulate disruption impacts on the SC and alternative SC designs that contain non-disrupted network nodes and arcs depending on real-time inventory, demand, and capacity data.","Laboratories in India played a major role in the management of the outbreak investigations during the recent 2009 H1N1 Influenza pandemic. Some laboratories geared their capacity by training manpower to carry out confirmation of index cases and test large number of samples. This activity may have helped build capacity of several state level laboratories to tackle similar emergencies, though, lessons learnt from managing one outbreak may not always suffice for another, as the agent, mode of transmission and susceptible population may differ. Two main factors for containment of an outbreak are isolation and contact tracing [115]. There is a need to move from conventional time consuming laboratory techniques to molecular technologies with rapid and sensitive detection capabilities and increased throughput for responding to the threats posed by emerging and re-emerging viruses in the recent past [115, 116]. There is also a need to create awareness among clinical microbiologists/virologists about existing outbreak reporting systems, laboratories designated for handling and processing samples for outbreak management, and for training and development of skills for testing and identifying new and emerging pathogens [115].","Marc Struelens (ECDC, Stockholm, Sweden) presented current perspectives for integrating molecular typing within European surveillance systems. In 2012, the ECDC initiated molecular typing activities covering up to 16 pathogens and five diseases. In view of the different standards and capabilities of European countries, the methods should be driven by practical needs for surveillance, they should be discriminatory in a meaningful way, they should produce comparable data across laboratories, be cost-efficient and be interpreted and assessed by experts. Collection and coordinated analysis of molecular typing data will be integrated into TESSy. Multidrug-resistant Neisseria gonorrhoeae, carbapenemresistant Enterobacteriaeceae, Acinetobacter baumannii and multidrug-resistant S. aureus are among priority pathogens. Marc Struelens also elaborated on the challenges and opportunities of HTS applications to public health [44] .","Evolution is stepwise and accrues mutations gradually over time, whereas synthetic constructs would typically use a known backbone and introduce logical or targeted changes instead of the randomly occurring mutations that are present in naturally isolated viruses such as bat CoV RaTG13. In our view, there is currently no credible evidence to support the claim that SARS-CoV-2 originated from a laboratory-engineered CoV. It is more likely that SARS-CoV-2 is a recombinant CoV generated in nature between a bat CoV and another coronavirus in an intermediate animal host. More studies are needed to explore this possibility and resolve the natural origin of SARS-CoV-2. We should emphasize that, although SARS-CoV-2 shows no evidence of laboratory origin, viruses with such great public health threats must be handled properly in the laboratory and also properly regulated by the scientific community and governments.","Dr. Rajan Gupta is a Research and Analytics professional and is currently associated with University of Delhi in India as Assistant Professor. He has done his PhD from University of Delhi in the area of Application of Data Science in Improving E-Governance in Developing Nations. He has written two books on E-Governance in India and has published various papers at national and international forums. He is an active consultant to a lot of organizations in the area of Bio-statistics and Data Science.","JS and Columbia University disclose partial ownership of SK Analytics. JS also reports receiving consulting fees from Merck. Table 1 . Best-fit model posterior estimates of key epidemiological parameters for simulation with the full metapopulation model during January 10-23, 2020 ( \\X] = 5000, ! = 10 days).","Naturally occurring infectious diseases have killed far more people. Still, bioterrorism may pose a serious threat today. Even if an attack kills relatively few people, the psychological impact could be enormous.","In April of 2018, the World Health Organization (WHO) established a priority 56 list of pathogens, including Middle East respiratory syndrome (MERS), severe acute 57 respiratory syndrome (SARS) and Disease X, a disease with an epidemic or pandemic 58 potential caused by an unknown pathogen 1,2 (Fig.1a) . 59 In late December 2019, an outbreak of pneumonia with an unknown etiology in 60 Wuhan, China was considered as the first Disease X following the announcement by 61 WHO. Shortly thereafter, a novel coronavirus, 2019-nCoV, as denoted by WHO 3 , 62 was identified as the pathogen causing the coronavirus disease COVID-19 4,5 . 63 2019-nCoV with 79.5% and 96% sequence identity to SARS-CoV and a bat 64 coronavirus, SL-CoV-RaTG13, respectively 6 , was renamed SARS-CoV-2 by the which mediates membrane fusion, has 89.8% sequence identity and 96.9% sequence 83 similarity to those of SARS-CoV, and both of them utilize human 84 angiotensin-converting enzyme 2 (hACE2) as the receptor to infect human cells 6 . 85 Most importantly, the ACE2-binding affinity of the receptor-binding domain (RBD) in 86 S1 subunit of S protein of SARS-CoV-2 is 10-to 20-fold higher than that of 87 SARS-CoV 12 , which may contribute to the higher infectivity and transmissibility of 88 SARS-CoV-2 compared to SARS-CoV. However, it is unclear whether SARS-CoV-2 89 can mediate membrane fusion in a manner that exceeds the capacity of SARS-CoV. 90 After binding of RBD in S1 subunit of S protein on the virion to the ACE2 91 receptor on the target cell, the heptad repeat 1 (HR1) and 2 (HR2) domains in its S2 92 subunit of S protein interact with each other to form a six-helix bundle (6-HB) fusion 93 core, bringing viral and cellular membranes into close proximity for fusion and 94 infection 13 . Therefore, the 6-HB fusion core structure of SARS-CoV-2 and 95 SARS-CoV S proteins should also be compared in order to investigate the structural 96 basis for membrane fusion mediated by their S proteins and thus set the stage for the 97 rational design of coronavirus fusion inhibitors. 98 In our previous studies, we designed a pan-coronavirus fusion inhibitor, EK1, 99 targeting the HR1 domains of HCoV S proteins, which proved to be effective in In this study, we have shown that SARS-CoV-2 exhibits much higher capacity of 108 membrane fusion than SARS-CoV, suggesting that the fusion machinery of SARS-CoV-2 infection, we found a typical syncytium phenomenon naturally formed 139 by infected cells, which is rarely reported in SARS-CoV infection (Fig. 1c) . To 140 further explore the special characteristic of SARS-CoV-2 infection, we cloned the S 141 gene into PAAV-IRES-GFP vector and established the S-mediated cell-cell fusion 142 system, using 293T cells that express SARS-CoV-2 S protein and EGFP 143 (293T/SARS-CoV-2/EGFP) as the effector cells, and ACE2/293T cells expressing 144 human ACE2 receptor as the target cells ( Fig. 1d and Fig. S1a ). After effector cells 145 and target cells were cocultured at 37 \u00b0C for 2 h, the fused cells showed at least 2-fold 146 larger size than normal cells and multiple nuclei, and these cells were observed in the 147 SARS-CoV-2 group, but not the SARS-CoV group. After coincubation for 24 h, 148 hundreds of target cells fused together as one big syncytium, containing multiple 149 nuclei (Fig. 1d) . Another 24h later, the syncytium grew bigger and could be easily Previously, we identified that the 6-HB formed by HR1 and HR2 domains of the 166 author/funder. All rights reserved. No reuse allowed without permission.","The rest of this paper is organized as follows. In Section 2, we analyse literature on SC risks with a focus on epidemic outbreaks and simulation. Section 3 presents our case-study and describes the simulation model. The experimental setup and results are shown in Section 4. This section discusses managerial implications as well. The paper is concluded in Section 5 by summarizing the most important insights and outlining future research agenda, especially in the areas of data analytics and digital twins.","Action is thus required at a company and industry level, with governments and tourists also having a vital contribution to make. The WTO has stressed the significance of health as an aspect of tourist safety and proposes that member states pursue the following program to enhance their capabilities in dealing with diffi culties (WTO, 1991) :","Over the last two decades, sequencing technology has evolved from labor-intensive and time-consuming methodologies to automatic and real-time sequence detection. Recent development and use of NGS has revolutionized the landscape of microbiology and infectious disease. The availability of sequencing data has speeded up pathogen discovery, and also helped improve diagnosis, typing of pathogens, detection of virulence and drug resistance, and development of new vaccines and targeted treatment [103\u00c0106] .","With the adoption of the International Health Regulations [IHR(2005)] by the WHO, India has made considerable progress and continues to demonstrate a strong commitment for establishing and operating a disease surveillance programme. The Union Ministry of Health and Family Welfare launched an Integrated Disease Surveillance Project (IDSP) in 2004 to connect all district hospitals and state-run medical colleges to facilitate tele-education, training of health professionals and monitoring of disease trends. The project is implemented through the Central Surveillance Unit within the National Centre for Disease Control (NCDC) at New Delhi and aims to detect early warning signals of impending disease outbreaks and initiate effective and rapid health actions with the help of information and communication technology at the district, state and national levels. In addition to monitoring diseases like vector borne infections, diarrhoeal diseases, respiratory diseases, and vaccine preventable diseases, the IDSP also has a category called \u2018unusual clinical syndromes,\u2019 which would alert public health professionals to an intentional release of any biological-agent [114].","Regardless which emerging pathogen we discuss -Avian Influenza H5N1, H7N9, MERS CoV, swine flu or other viruses, it is important to recognize that preparedness efforts as a response to a potential pandemic caused by swine or avian influenza, SARS or MERS, can also enhance awareness, promote vaccine use, advances in diagnostic and treatment capabilities toward other significant infectious disease worldwide.","Molecular approaches for microbial surveillance and discovery have emerged as a very promising alternative for early diagnosis of infectious diseases. Currently, molecular approaches include traditional Sanger DNA sequencing, polymerase chain reaction (PCR), oligonucleotide microarrays, and next generation sequencing (NGS). Among these four technologies, the former two can identify a few known pathogens that must then be confirmed individually, and thus cannot cover a wide range of pathogens. The latter two methods cover a broad range of pathogens, and are therefore suitable for identifying unknown or even novel pathogens in infectious outbreaks. Although NGS produces the most in-depth, unbiased information, and can reveal completely novel organisms, it is time-consuming and expensive, especially for the analysis of complex samples [6]. DeRisi and colleagues developed the first generation of microarray platform, called ViroChip, to detect a wide range of viruses [7]. In 2003, the ViroChip helped to characterize SARS as a novel Coronavirus[8]. Since then, ViroChip has also been used to detected a human metapneumovirus [9], a novel influenza virus [10], and a novel adenovirus [11]. More recently, GreeneChip and MDA microarrays have been developed, which are broader spectrum approaches that can detect several thousand pathogenic viruses, bacteria, fungi, and protozoa [12,13]. The aforementioned three platforms all used long oligonucleotide probes and random amplification of nucleic acids.","China has exploited global networks in a number of ways. First, as Hirschman reminded us, great powers regularly seek to convert their economic might into political leverage around the world. China is no exception. With the development of global and regional connectivity, China has used its financial muscles to gain control over major natural resources and strategic locations and to influence other states' stances in disputes. China has provided a substantial amount of aid to African countries to harden its grasp on natural resources. Beijing has also launched comprehensive programs of financial statecraft, known as Belt and Road Initiatives (BRI), to establish its sphere of influence in the Indo-Pacific region and even the whole of Eurasia. China's financial prowess successfully has swayed some states' stances in many regional disputes. Philippine President Rodrigo Duterte's recent concession on the South China Sea dispute is one notable example. Another salient case is China's successful effort to induce the Solomon Islands to end its diplomatic relationship with Taiwan. 15 Second, the fundamental goal for the Chinese leadership is to create a world that is safe for China's authoritarian regime. Such an objective could be achieved if there are more non-democratic regimes around the world, or more countries adopting the \"China model.\" Therefore, China does not only seek to develop its repressive capacities, but also it tries to export its censorship regime-encouraging other authoritarian regimes to censor and silence domestic dissidents with the technologies and programs it supplies. China's strategy is twofold: first, it has exploited global supply chains and cross-border networks to upgrade its infrastructural power for censorship and monitoring domestic citizens. For example, China has engaged in intellectual property theft worldwide and cyber espionage to increase its information and communication technologies. 16 It has also provided various governmental supports to develop Huawei, China's largest telecommunications company and Chinese \"national champion\" of the 5G industry. Huawei's success and control over 5G network technologies have enabled the Chinese government to collect and store the data of almost every Chinese citizen. The government also used similar industrial policies to develop Hikvision Digital Technology and Dahua Technology, Chinese manufacturers and the world's two largest suppliers of video surveillance products. The monitoring capacity provided by Chinese surveillance camera companies, together with Huawei's big data technology, strengthen China's policing capacity and the Communist Party's censorship. 17 The second fold of China's strategy is to export its censorship regime to other countries as well as surveillance hardware and data-analytics tools. China has engaged in social penetration into other societies and wielded its sharp power globally to justify the China model. For example, Beijing is cultivating media elites and government ministers around the world to create a network of countries that will follow its lead on internet policy. Chinese officials have held many all-expensespaid trainings, workshops, seminars, and conferences with representatives from other countries. During those events, China regularly propagates that the Chinese model of censorship is helpful in combating terrorism and fake news, monitoring public opinion, china/diplomacy/article/3027827/us-cancels-solomon-islands-meeting-after-disappointmentit. 16 and ensuring cyber security. Several countries introduced cyber-security laws that closely mimic China's own version after their official participated in those events. 18 Finally, and much more concerning, the Chinese state apparatus has been developing its \"information troops\" to silence or deter dissidents as well as to enforce pro-China views at home and abroad. The great majority of these troops-called by some as the \"50-cent army\"-may not actually be part of the security apparatus, but independent operators including student volunteers at universities, Communist Youth League members, and government bureaucrats. 19 According to political scientists Gary King at Harvard, Jennifer Pan at Stanford, and Margaret E. Roberts at University of California San Diego, the Chinese state apparatus is involved in faking several hundred million social media accounts and posts. 20 The Chinese information troops often use social media to launch campaigns against voices or opinions that criticize the Chinese government or challenge the legitimacy of the CCP. The troops constantly use fake news or specious arguments to manipulate public opinion. With help from Chinese government censorship, they can easily silence dissents and \"manufacture\" some public consensus. Then, backed by this manufactured consensus, the Chinese government is able to resort to coercion against the dissenters.","The evidence affirms this. HIV was probably present in small foci in Central Africa for decades to centuries; Ebola was similarly endemic in West Africa for an unknown period, as was Lassa fever. What is novel about the past few decades is greater interconnection between places, allowing diseases, and news of diseases, to spread; better methods of detection; and changing settlement geographies that have brought people into different forms of contact with animal reservoirs. The root cause of the infectious disease emergence is human action, both intentional and unintentional. Most of this action is the result of cumulative individual acts on a mass scale. For example, the mass urbanization of society in poorer countries is the sum of millions of individuals who move from rural to urban areas. This is largely the result of the perceived economic opportunities in urban areas, and the 'push' factor of lack of opportunity in rural areas. Yet, taken together, millions of individual moves result in urbanization, and this urbanization facilitates the spread of diseases by the respiratory route, the fecal-oral route, and many other modes of transmission.","The CDC defines Category B agents as those that are moderately easy to disseminate. These pathogens result in moderate morbidity and low mortality. These agents would hence require specific improvements of diagnostic capabilities as well as enhanced measures for disease surveillance.","Natural products used in TCM remains to be a wealthy source for the identification of novel therapeutic agents for the treatment of human diseases 52. In the past decade, scientists have made a considerable effort to identify multiple component herbal formulae in TCM with anti-SARS-CoV activity (Table 2). Further identification of chemical entities contained in TCM herbs responsible for the anti-SARS- CoV effect was also pursued (Table 3). Due to the homology of SARS-CoV and SARS-CoV-2, these previous studies may shed light on the naturally occurring compounds with the capacity to inhibit SARS-CoV-2.","More recently, epidemiological data of zoonotic viral pathogens from Saudi Arabia and/or from other neighboring countries after it was confirmed through laboratory test isolation from dromedaries (e.g., rabies, MERS-CoV, RVFV, and WNV) may enhance a high interest in the search for other novel zoonotic viruses in dromedaries [36,52,208,211,226,227]. Furthermore, the habits of ingestion off unpasteurized milk from camels as a rare delicacy by Saudi people need to be checked. Moreover, viral pathogens such as RVFV are acquired through the importation of camels, while the remaining pathogens (e.g., rabies and influenza viruses) are endemic worldwide. Of these (e.g., influenza virus), there is need for a highly preventive zoonotic control in Saudi, due to fact that the isolation and genetic characterization of H5N1 was reported in 2017 among vaccinated meat-turkeys flock in Egypt, a neighboring country, that was previously reported to have more than 100,000 travelers to Saudi Arabia during Hajj pilgrimage seasons, annually [219]. This might be considered as one of such important risk factor of possible introduction or spread of influenza pathogen in Saudi Arabia [218,219].","Although PDCoV has been detected from naturally-infected pigs with diarrhea on farms, experimental infections under controlled conditions are needed to definitely prove PDCoV as an etiological pathogen especially considering that co-infections of PDCoV with other enteric viruses such as PEDV and rotavirus were common in natural infections (Marthaler et al., 2014b) . Clinical materials positive for PDCoV by PCR are not ideal to evaluate viral pathogenicity because, although the clinical materials can be proved negative for the common enteric pathogens such as PEDV, TGEV, rotavirus etc., there is no absolute certainty that they do not contain other unknown pathogens. In contrast, plaque-purified cell culture isolate of PDCoV can be utilized to more precisely investigate PDCoV pathogenicity and clinical outcomes. Ideally, the pathogenicity and pathogenesis of PDCoV should be examined in pigs at different production stages (e.g. gilts, sows, weaned pigs, and neonatal pigs etc.). In the present study, 5-day-old neonatal pigs were chosen as the first step of these investigations because it is generally recognized that neonatal pigs are more sensitive than pigs at other production stages to infection with the swine enteric coronaviruses PEDV and TGEV."],"title":["Metagenomics in Virology","Challenges of infectious diseases in the USA","Centers for Disease Control","Rapid metagenomic characterization of a case of imported COVID-19 in Cambodia","Human genomics and preparedness for infectious threats","Is Africa prepared for tackling the COVID-19 (SARS-CoV-2) epidemic. Lessons from past outbreaks, ongoing pan-African public health efforts, and implications for the future","Structure and infrastructure of infectious agent research literature: SARS","Preparedness for emerging epidemic threats: a Lancet Infectious Diseases Commission","Financing Vaccines for Global Health Security","No novel coronaviruses identified in a large collection of human nasopharyngeal specimens using family-wide CODEHOP-based primers","Authors\u2019 response: Plenty of coronaviruses but no SARS-CoV-2","High sensitivity detection of coronavirus SARS-CoV-2 using multiplex PCR and a multiplex-PCR-based metagenomic method","Applications of microarrays in pathogen detection and biodefence","Structure and infrastructure of infectious agent research literature: SARS","Chapter 15 Molecular Testing in Emerging Infectious Diseases","13 Epidemiologic Principles","Global epidemiology of coronavirus disease 2019 (COVID-19): disease incidence, daily cumulative index, mortality, and their association with country healthcare resources and economic status","Multiple approaches for massively parallel sequencing of HCoV-19 (SARS-CoV-2) genomes directly from clinical samples","Deliberate release: Plague \u2013 A review","An easy operating pathogen microarray (EOPM) platform for rapid screening of vertebrate pathogens","6 Recombinant plasmids","Chapter 1 Introduction","Impact of Hajj on Global Health Security","Applications of microarrays in pathogen detection and biodefence","Ultra-Low-Cost Integrated Silicon-based Transducer for On-Site, Genetic Detection of Pathogens","CHEER: hierarCHical taxonomic classification for viral mEtagEnomic data via deep leaRning","Rapid metagenomic characterization of a case of imported COVID-19 in Cambodia","Managing COVID-19 in the oncology clinic and avoiding the distraction effect","Transmission Dynamics of COVID-19 and Impact on Public Health Policy","AAEDM: Theoretical Dynamic Epidemic Diffusion Model and Covid-19 Korea Pandemic Cases","Predicting the impacts of epidemic outbreaks on global supply chains: A simulation-based analysis on the coronavirus outbreak (COVID-19/SARS-CoV-2) case","Emerging Viral Infections in India","Microbial molecular markers and epidemiological surveillance in the era of high throughput sequencing: an update from the IMMEM-10 conference","No credible evidence supporting claims of the laboratory engineering of SARS-CoV-2","Trend Analysis and Forecasting of COVID-19 outbreak in India","Substantial undocumented infection facilitates the rapid dissemination of novel coronavirus (COVID-19)","Chapter 22 Biological Warfare: Infectious Disease and Bioterrorism","Inhibition of SARS-CoV-2 infection (previously 2019-nCoV) by a highly potent pan-coronavirus fusion inhibitor targeting its spike protein that harbors a high capacity to mediate membrane fusion","Predicting the impacts of epidemic outbreaks on global supply chains: A simulation-based analysis on the coronavirus outbreak (COVID-19/SARS-CoV-2) case","Chapter 7 Tourism and Health Crises","Chapter 15 Molecular Testing in Emerging Infectious Diseases","Emerging Viral Infections in India","SELECTED EPIDEMICS &amp; EMERGING PATHOGENS \u2013 RESPIRATORY ILLNESSES \u2013 AN OVERVIEW","An easy operating pathogen microarray (EOPM) platform for rapid screening of vertebrate pathogens","Weaponized Interdependence: China's Economic Statecraft and Social Penetration against Taiwan","Emerging Diseases: Overview","Applications of microarrays in pathogen detection and biodefence","Traditional Chinese Medicine in the Treatment of Patients Infected with 2019-New Coronavirus (SARS-CoV-2): A Review and Perspective","An Overview of the Most Significant Zoonotic Viral Pathogens Transmitted from Animal to Human in Saudi Arabia","Pathogenicity and pathogenesis of a United States porcine deltacoronavirus cell culture isolate in 5-day-old neonatal piglets"],"url":["https://doi.org/10.1016/b978-0-12-809633-8.20957-6","https://doi.org/10.1016/s0140-6736(14)60890-4","https://doi.org/10.1016/b978-0-12-803678-5.00057-6","https://doi.org/10.1101/2020.03.02.968818","https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2808735/","https://doi.org/10.1016/j.ijid.2020.02.049","https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7088553/","https://doi.org/10.1016/s1473-3099(19)30674-7","https://doi.org/10.1101/2020.03.20.20039966","https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7087030/","https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7055035/","https://doi.org/10.1101/2020.03.12.988246","https://doi.org/10.1016/j.tibtech.2008.09.004","https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7088553/","https://doi.org/10.1016/b978-0-12-800886-7.00015-7","https://doi.org/10.1016/b978-1-4557-4801-3.00013-8","https://doi.org/10.1016/j.ijantimicag.2020.105946","https://doi.org/10.1101/2020.03.16.993584","https://doi.org/10.1016/j.jobb.2020.02.001","https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3848773/","https://doi.org/10.1016/b978-0-7506-1105-3.50010-1","https://doi.org/10.1016/b978-0-12-812343-0.00001-1","https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7088024/","https://doi.org/10.1016/j.tibtech.2008.09.004","https://doi.org/10.1101/2020.03.23.002931","https://doi.org/10.1101/2020.03.26.009001","https://doi.org/10.1101/2020.03.02.968818","https://doi.org/10.1016/j.annonc.2020.03.286","https://doi.org/10.1101/2020.03.29.20047035","https://doi.org/10.1101/2020.03.17.20037838","https://doi.org/10.1016/j.tre.2020.101922","https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7100311/","https://doi.org/10.1016/j.resmic.2014.01.003","https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7054935/","https://doi.org/10.1101/2020.03.26.20044511","https://doi.org/10.1101/2020.02.14.20023127","https://doi.org/10.1016/b978-0-12-385015-7.00022-3","https://doi.org/10.1101/2020.03.09.983247","https://doi.org/10.1016/j.tre.2020.101922","https://doi.org/10.1016/b978-0-7506-7834-6.50008-9","https://doi.org/10.1016/b978-0-12-800886-7.00015-7","https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7100311/","https://doi.org/10.1016/j.disamonth.2017.03.016","https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3848773/","https://doi.org/10.1016/j.orbis.2020.02.002","https://doi.org/10.1016/b978-012373960-5.00453-6","https://doi.org/10.1016/j.tibtech.2008.09.004","https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7098036/","https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6471281/","https://doi.org/10.1016/j.virol.2015.03.024"],"x":[0.5962896549178317,0.5184703284748978,0.5010535323483092,0.48475462089124455,0.518635400703875,0.46440418563540814,0.6784888629849424,0.5179197391981162,0.4212050269710911,0.556778473377167,0.7463317073808745,0.5647075796774272,0.46891109443031015,0.8160737101296647,0.46481774099682654,0.5376419793272793,0.537500240479339,0.5394547403763023,0.47794582183009193,0.5086997448129895,0.4803137388848346,0.5443266089284577,0.5910100455718487,0.5641879339951037,0.47119178395790845,0.5511187669695666,0.9095098454720075,0.36807800189344986,0.7287364012251131,0.7436326103423611,0.8244688801608505,0.5340370245345409,0.5374772335793622,0.7762744170653132,0.5428618907101046,0.7563645504741872,0.47199750316098255,0.7539290918990404,0.7247410341716207,0.47524140295000467,0.5098931502601137,0.5169516327222673,0.5428742188752862,0.5513163320758238,0.7506370011020136,0.5953433405520031,0.3297997800819573,0.7417874114972187,0.5400625059005693,0.40924778433278075],"y":[0.4399153935653039,0.7251682589549399,0.7417215483587272,0.7175799509862334,0.7266586325240295,0.7642901863093572,0.5633754752138411,0.7264438136013377,0.5391335465977065,0.40891073038563314,0.5175065747919325,0.4169369706745552,0.5911897503978182,0.436158877754365,0.415405802645114,0.8241932723494698,0.7895413591077227,0.42239420674000444,0.70943207501531,0.6765516402787554,0.6980853829849635,0.8165060188727017,0.5815849379594716,0.41751156244308446,0.40898272881981174,0.8650027066207265,0.5838952383104804,0.7912889909977733,0.6976980549366795,0.670779852206216,0.41056122121467065,0.6690010960969043,0.8274316765814386,0.47874286706318575,0.8526363225600434,0.7047183807964883,0.6476534706390594,0.31923941221004715,0.6845259304374497,0.7175052209581798,0.6745285560574454,0.7221257419056318,0.5838360504534161,0.4183313596238344,0.7930704886702807,0.5829679718524603,0.6016421445137378,0.4472030208397583,0.5823112528199096,0.4117728326380247]},"selected":{"id":"27801"},"selection_policy":{"id":"27830"}},"id":"27728","type":"ColumnDataSource"},{"attributes":{"source":{"id":"27728"}},"id":"27792","type":"CDSView"},{"attributes":{"source":{"id":"27728"}},"id":"27798","type":"CDSView"},{"attributes":{},"id":"27825","type":"BasicTickFormatter"},{"attributes":{},"id":"27730","type":"Range1d"},{"attributes":{"callback":null,"tooltips":[["Title","@title"],["Passage","@text"],["Published","@date"],["Authors","@authors"],["Journal","@journal"],["URL","@url"]]},"id":"27783","type":"HoverTool"},{"attributes":{"axis":{"id":"27777"},"dimension":1,"ticker":null},"id":"27780","type":"Grid"},{"attributes":{},"id":"27739","type":"BasicTicker"},{"attributes":{},"id":"27778","type":"BasicTicker"},{"attributes":{"below":[{"id":"27738"}],"center":[{"id":"27741"},{"id":"27745"}],"left":[{"id":"27742"}],"plot_height":340,"plot_width":900,"renderers":[{"id":"27756"}],"title":{"id":"27812"},"toolbar":{"id":"27749"},"toolbar_location":"below","x_range":{"id":"27730"},"x_scale":{"id":"27734"},"y_range":{"id":"27732"},"y_scale":{"id":"27736"}},"id":"27729","subtype":"Figure","type":"Plot"},{"attributes":{},"id":"27736","type":"LinearScale"},{"attributes":{"callback":null},"id":"27781","type":"TapTool"},{"attributes":{"columns":[{"id":"27795"},{"id":"27796"}],"height":340,"source":{"id":"27728"},"view":{"id":"27798"},"width":900},"id":"27797","type":"DataTable"},{"attributes":{"children":[{"id":"27762"},{"id":"27729"}]},"id":"27808","type":"Row"},{"attributes":{},"id":"27782","type":"ResetTool"},{"attributes":{"text":"&lt;h3&gt;Paragraph ranking&lt;/h3&gt;"},"id":"27805","type":"Div"},{"attributes":{"fill_color":{"field":"color"},"line_color":{"field":"color"},"radius":{"field":"rad","units":"data"},"x":{"field":"x"},"y":{"field":"y"}},"id":"27754","type":"Circle"},{"attributes":{"axis":{"id":"27738"},"ticker":null},"id":"27741","type":"Grid"},{"attributes":{},"id":"27816","type":"StringFormatter"},{"attributes":{"callback":null,"tooltips":[["Title","@title"],["Abstract","@text"],["Published","@date"],["Authors","@authors"],["Journal","@journal"],["URL","@url"]]},"id":"27748","type":"HoverTool"},{"attributes":{"active_drag":"auto","active_inspect":"auto","active_multi":null,"active_scroll":"auto","active_tap":"auto","tools":[{"id":"27746"},{"id":"27747"},{"id":"27748"}]},"id":"27749","type":"Toolbar"},{"attributes":{"formatter":{"id":"27823"},"ticker":{"id":"27743"}},"id":"27742","type":"LinearAxis"},{"attributes":{"children":[{"id":"27805"}]},"id":"27809","type":"Row"},{"attributes":{},"id":"27747","type":"ResetTool"},{"attributes":{"text":""},"id":"27812","type":"Title"},{"attributes":{"formatter":{"id":"27825"},"ticker":{"id":"27739"}},"id":"27738","type":"LinearAxis"},{"attributes":{},"id":"27818","type":"StringFormatter"},{"attributes":{"editor":{"id":"27829"},"field":"text","formatter":{"id":"27828"},"title":"Passage","width":770},"id":"27796","type":"TableColumn"}],"root_ids":["27811"]},"title":"Bokeh Application","version":"2.0.1"}}
        </script>
        <script type="text/javascript">
          (function() {
            var fn = function() {
              Bokeh.safely(function() {
                (function(root) {
                  function embed_document(root) {
                    
                  var docs_json = document.getElementById('28048').textContent;
                  var render_items = [{"docid":"69180d8f-ef4c-4732-a67a-3fd39ca57dbd","root_ids":["27811"],"roots":{"27811":"ed53e781-8047-43d9-8641-350b78f819c5"}}];
                  root.Bokeh.embed.embed_items(docs_json, render_items);
                
                  }
                  if (root.Bokeh !== undefined) {
                    embed_document(root);
                  } else {
                    var attempts = 0;
                    var timer = setInterval(function(root) {
                      if (root.Bokeh !== undefined) {
                        clearInterval(timer);
                        embed_document(root);
                      } else {
                        attempts++;
                        if (attempts > 100) {
                          clearInterval(timer);
                          console.log("Bokeh: ERROR: Unable to run BokehJS code because BokehJS library is missing");
                        }
                      }
                    }, 10, root)
                  }
                })(window);
              });
            };
            if (document.readyState != "loading") fn();
            else document.addEventListener("DOMContentLoaded", fn);
          })();
        </script>
    
  </body>
  
</html>