



<!DOCTYPE html>
<html lang="en">
  
  <head>
    
      <meta charset="utf-8">
      <title>Bokeh Plot</title>
      
      
        
          
        
        
          
        <script type="text/javascript" src="https://cdn.bokeh.org/bokeh/release/bokeh-2.0.1.min.js" integrity="sha384-JpP8FXbgAZLkfur7LiK3j9AGBhHNIvF742meBJrjO2ShJDhCG2I1uVvW+0DUtrmc" crossorigin="anonymous"></script>
        <script type="text/javascript" src="https://cdn.bokeh.org/bokeh/release/bokeh-widgets-2.0.1.min.js" integrity="sha384-xZlADit0Q04ISQEdKg2k3L4W9AwQBAuDs9nJL9fM/WwzL1tEU9VPNezOFX0nLEAz" crossorigin="anonymous"></script>
        <script type="text/javascript" src="https://cdn.bokeh.org/bokeh/release/bokeh-tables-2.0.1.min.js" integrity="sha384-4BuPRZkdMKSnj3zoxiNrQ86XgNw0rYmBOxe7nshquXwwcauupgBF2DHLVG1WuZlV" crossorigin="anonymous"></script>
        <script type="text/javascript">
            Bokeh.set_log_level("info");
        </script>
        
      
      
    
  </head>
  
  
  <body>
    
      
        
          
          
            
              <div class="bk-root" id="9237dc14-cbc2-48af-adaa-c4793fa375f6" data-root-id="8169"></div>
            
          
        
      
      
        <script type="application/json" id="8406">
          {"eb6e31ac-8a95-4ae7-affb-45523518872a":{"roots":{"references":[{"attributes":{"active_drag":"auto","active_inspect":"auto","active_multi":null,"active_scroll":"auto","active_tap":"auto","tools":[{"id":"8104"},{"id":"8105"},{"id":"8106"}]},"id":"8107","type":"Toolbar"},{"attributes":{"formatter":{"id":"8191"},"ticker":{"id":"8136"}},"id":"8135","type":"LinearAxis"},{"attributes":{"data":{"authors":["Gardy, Jennifer L.; Loman, Nicholas J.","Xu, Bo; Gutierrez, Bernardo; Mekaru, Sumiko; Sewalk, Kara; Goodwin, Lauren; Loskill, Alyssa; Cohn, Emily L.; Hswen, Yulin; Hill, Sarah C.; Cobo, Maria M.; Zarebski, Alexander E.; Li, Sabrina; Wu, Chieh-Hsi; Hulland, Erin; Morgan, Julia D.; Wang, Lin; O\u2019Brien, Katelynn; Scarpino, Samuel\u00a0V.; Brownstein, John S.; Pybus, Oliver G.; Pigott, David M.; Kraemer, Moritz U. G.","Huang, Yi Chao; Xie, Qianyi","HIETPAS, JACK; MCMULLAN, LAURA K.; HANSON, HOLLY L.; RICE, CHARLES M.; MINDELL, DAVID P.","Cleemput, S.; Dumon, W.; Fonseca, V.; Karim, W. A.; Giovanetti, M.; Alcantara, L. C.; Deforche, K.; de Oliveira, T.","Huang, Jyh-Win; Hou, Ting-Wei","Sara Cleemput; Wim Dumon; Vagner Fonseca; Wasim Abdool Karim; Marta Giovanetti; Luiz Carlos Alcantara; Koen Deforche; Tulio de Oliveira","Bitar, Dounia; Tarantola, Arnaud; Capek, Isabelle; Barboza, Philippe; Che, Didier","Pang, Junxiong; Wang, Min Xian; Ang, Ian Yi Han; Tan, Sharon Hui Xuan; Lewis, Ruth Frances; Chen, Jacinta I. Pei; Gutierrez, Ramona A.; Gwee, Sylvia Xiao Wei; Chua, Pearleen Ee Yong; Yang, Qian; Ng, Xian Yi; Yap, Rowena K. S.; Tan, Hao Yi; Teo, Yik Ying; Tan, Chorh Chuan; Cook, Alex R.; Yap, Jason Chin-Huat; Hsu, Li Yang","Petrosillo, Nicola; Puro, Vincenzo; Di Caro, Antonino; Ippolito, Giuseppe","Yam, W.C.; Chan, K.H.; Chow, K.H.; Poon, L.L.M.; Lam, H.Y.; Yuen, K.Y.; Seto, W.H.; Peiris, J.S.M.","Lowenthal, John","Norden E Huang; Fangli Qiao; Ka-Kit Tung","Timokratis Karamitros; Gethsimani Papadopoulou; Maria Bousali; Anastasios Mexias; Sotiris Tsiodras; Andreas Mentis","Ganova-Raeva, Lilia M; Khudyakov, Yury E","Haoxuan Sun; Yumou Qiu; Han Yan; Yaxuan Huang; Yuru Zhu; Song Xi Chen","Byung Mook Weon","Yong Tao","Stephen A. Wells","Huang, Norden E.; Qiao, Fangli","Dahlhausen, Bob","Pedersen, Niels C.","Lau, Lok Ting; Fung, Yin-Wan Wendy; Wong, Freda Pui-Fan; Lin, Selma Sau-Wah; Wang, Chen Ran; Li, Hui Li; Dillon, Natalie; Collins, Richard A; Tam, John Siu-Lun; Chan, Paul K.S; Wang, Chen G; Yu, Albert Cheung-Hoi","Atlani-Duault, La\u00ebtitia; Ward, Jeremy K.; Roy, Melissa; Morin, C\u00e9line; Wilson, Andrew","Poletto, Chiara; Bo\u00eblle, Pierre-Yves; Colizza, Vittoria","Zumla, Alimuddin; Gant, Vanya; Bates, Matthew; Mwaba, Peter; Maeurer, Markus; Memish, Ziad A","Pan, Xingchen; Ojcius, David M.; Gao, Tianyue; Li, Zhongsheng; Pan, Chunhua; Pan, Chungen","Konrad, Regina; Eberle, Ute; Dangel, Alexandra; Treis, Bianca; Berger, Anja; Bengs, Katja; Fingerle, Volker; Liebl, Bernhard; Ackermann, Nikolaus; Sing, Andreas","James Zhu; Jiwoong Kim; Xue Xiao; Yunguan Wang; Danni Luo; Ran Chen; Lin Xu; He Zhang; Guanghua Xiao; John Schoggins; Xiaowei Zhan; Tao Wang; Yang Xie","Zhang, Jiahao; Ma, Kaixiong; Li, Huanan; Liao, Ming; Qi, Wenbao","Tuite, Ashleigh R.; Bogoch, Isaac I.; Sherbo, Ryan; Watts, Alexander; Fisman, David; Khan, Kamran","Goldenberg, Simon","Minfeng Xiao; Xiaoqing Liu; Jingkai Ji; Min Li; Jiandong Li; Lin Yang; Wanying Sun; Peidi Ren; Guifang Yang; Jincun Zhao; Tianzhu Liang; Huahui Ren; Tian Chen; Huanzi Zhong; Wenchen Song; Yanqun Wang; Ziqing Deng; Yanping Zhao; Zhihua Ou; Daxi Wang; Jielun Cai; Xinyi Cheng; Taiqing Feng; Honglong Wu; Yanping Gong; Huanming Yang; Jian Wang; Xun Xu; Shida Zhu; Fang Chen; Yanyan Zhang; Weijun Chen; Yimin Li; Junhua Li","Xiong Ding; Kun Yin; Ziyue Li; Changchun Liu","Roda, Weston C.; Varughese, Marie B.; Han, Donglin; Li, Michael Y.","Lee, Nolan E.; Siriarayapon, Potjaman; Tappero, Jordan; Chen, Kow-Tong; Shuey, Dean; Limpakarnjanarat, Khanchit; Chavavanich, Achara; Dowell, Scott F.; SARS Mobile Response Team Investigators, ","Hanchu Zhou; Jianan Yang; Kaichen Tang; Qingpeng Zhang; zhidong cao; Dirk Pfeiffer; Daniel Dajun Zeng","Antoine Kevorkian; Thierry Grenet; Hubert Gallee","Kevin Dick; Kyle K Biggar; James R Green","Ghosh, Arun K.; Xi, Kai; Johnson, Michael E.; Baker, Susan C.; Mesecar, Andrew D.","Tieying Hou; Weiqi Zeng; Minling Yang; Wenjing Chen; Lili Ren; Jingwen Ai; Ji Wu; Yalong Liao; Xuejing Gou; Yongjun Li; Xiaorui Wang; Hang Su; Jianwei Wang; Bing Gu; Teng Xu","Zhou, Chenghu; Su, Fenzhen; Pei, Tao; Zhang, An; Du, Yunyan; Luo, Bin; Cao, Zhidong; Wang, Juanle; Yuan, Wen; Zhu, Yunqiang; Song, Ci; Chen, Jie; Xu, Jun; Li, Fujia; Ma, Ting; Jiang, Lili; Yan, Fengqin; Yi, Jiawei; Hu, Yunfeng; Liao, Yilan; Xiao, Han","Colb\u00e8re-Garapin, Florence; Blondel, Bruno; Saulnier, Aure; Pelletier, Isabelle; Labadie, Karine","Kim, Jin Hwa; Kang, Minhee; Park, Eunkyoung; Chung, Doo Ryeon; Kim, Jiyeon; Hwang, Eung Soo","Phan, Tung","Julien Riou; Christian L. Althaus","Pengpeng Shi; Shengli Cao; Peihua Feng","Turner, Cameron; Bishay, Hany; Peng, Bo; Merifield, Aaron","de Wit, Emmie; van Doremalen, Neeltje; Falzarano, Darryl; Munster, Vincent J.","Zhai, Junhui; Briese, Thomas; Dai, Erhei; Wang, Xiaoyi; Pang, Xin; Du, Zongmin; Liu, Haihong; Wang, Jin; Wang, Hongxia; Guo, Zhaobiao; Chen, Zeliang; Jiang, Lingxiao; Zhou, Dongsheng; Han, Yanping; Jabado, Omar; Palacios, Gustavo; Lipkin, W. Ian; Yang, Ruifu"],"color":["#67a9cf","#67a9cf","#67a9cf","#67a9cf","#67a9cf","#67a9cf","#67a9cf","#67a9cf","#67a9cf","#67a9cf","#67a9cf","#67a9cf","#67a9cf","#67a9cf","#67a9cf","#67a9cf","#67a9cf","#67a9cf","#67a9cf","#67a9cf","#67a9cf","#67a9cf","#67a9cf","#67a9cf","#67a9cf","#67a9cf","#67a9cf","#67a9cf","#67a9cf","#67a9cf","#67a9cf","#67a9cf","#67a9cf","#67a9cf","#67a9cf","#67a9cf","#67a9cf","#67a9cf","#67a9cf","#67a9cf","#67a9cf","#67a9cf","#67a9cf","#67a9cf","#67a9cf","#67a9cf","#67a9cf","#67a9cf","#67a9cf","#67a9cf"],"date":["2017-11-13","2020-03-24","2013-06-09","2005-12-31","2020","2007-05-31","2020-02-02","2009-11-30","2020","2005-12-31","2005-05-31","2016-06-30","2020-03-30","2020-03-28","2013-05-01","2020-02-20","2020-03-30","2020-03-20","2020-03-12","2020-03-30","2010-04-30","2014-08-31","2003-12-26","2020","2016-08-25","2013-06-30","2020-03-31","2020-03-05","2020-02-12","2020-02-22","2020-03-16","2013-11-30","2020-03-17","2020-03-21","2020-12-31","2004-11-30","2020-02-12","2020-03-17","2020-03-31","2006-12-31","2020-02-23","2020-03-20","2005-04-30","2019-11-11","2020-04-30","2020-01-24","2020-02-20","2006-11-30","2016-06-27","2004-02-03"],"doc":["th1da1bb","l63z3btv","4rulshop","qma1lsuw","st5idleq","rpq6xan7","gxstlzuk","mg779mix","8gncbgot","lk7e2pni","qxlac437","13adnvav","mxen3n0k","jxm69ndw","qh2kuzdc","t2x4yfrv","d950qa2w","7umn0vkv","rzda8x62","ivqbqdh0","o976nab9","j3yb3k57","v2o25ey9","w1hvtiqz","4q5ttptg","vpq4yghn","d13bq4ij","zlhu1jit","xetzg7gp","um2khqhn","pidy7erw","n8hlrknq","wbyqmvhs","vlqd192g","vx2t7jgu","d0ccqrmu","qbemfymp","5u04irwz","dxabs45r","1wae0134","hth8f5sn","5gbkrs73","4uw4frer","3doc40hx","s9mqv4lo","jjkf5gg3","c800ynvc","nl4rhvlu","izn315hb","gk29idya"],"journal":["Nat Rev Genet","Sci Data","J Intell Manuf","Microbial Forensics","Bioinformatics (Oxford, England)","Computer Methods and Programs in Biomedicine","","Bulletin de l'Acad\u00e9mie Nationale de M\u00e9decine","Journal of Clinical Medicine","Archives of Medical Research","Journal of Clinical Virology","Journal of Infection and Public Health","","","Expert Review of Molecular Diagnostics","","","","","Science Bulletin","Journal of Exotic Pet Medicine","The Veterinary Journal","Biochemical and Biophysical Research Communications","The Lancet Public Health","BMC Infect Dis","The Lancet Respiratory Medicine","Microbes and Infection","Euro Surveill","","Journal of Infection","Ann Intern Med","Medicine","","","Infectious Disease Modelling","American Journal of Infection Control","","","","Annual Reports in Medicinal Chemistry","","Geography and Sustainability","Microbes and Infection","Biochip J","Infection, Genetics and Evolution","","","International Journal of Medical Informatics","Nat Rev Microbiol","Emerg Infect Dis"],"rad":[0.02,0.017329789817538202,0.016760334636824123,0.016433010512735418,0.014717850560561237,0.013904684102941225,0.013440877929647078,0.010863559707625193,0.010745554638810937,0.009982989705608278,0.00985336539291238,0.009141371979128881,0.00893579091492365,0.008857575077018134,0.008829329564689178,0.008316070813890103,0.008110739128203539,0.008012505816533,0.00776984096701558,0.007135792630405446,0.0065915890132312634,0.006530756427209201,0.0064930458490408465,0.005996445621621781,0.0059901983381034415,0.005788502169372672,0.005764008948567056,0.005612881073078477,0.005589515453887636,0.0052975629549789575,0.0049637135726947204,0.004762871568070476,0.004709569514420901,0.0046346721247426965,0.004178905914580281,0.003903548554407185,0.003898259385813379,0.003654824376711218,0.0036450601012525553,0.003634719620596662,0.003493744422990965,0.0034809552747399244,0.0034342128423710152,0.003400273104938265,0.003381322097849336,0.0033397428934147863,0.003185879252164895,0.003119766626609654,0.0030426361900179407,0.003],"scr":[1.0,0.843,0.809,0.79,0.689,0.641,0.614,0.463,0.456,0.411,0.403,0.361,0.349,0.345,0.343,0.313,0.301,0.295,0.281,0.243,0.211,0.208,0.205,0.176,0.176,0.164,0.163,0.154,0.152,0.135,0.116,0.104,0.101,0.096,0.069,0.053,0.053,0.039,0.038,0.037,0.029,0.028,0.026,0.024,0.022,0.02,0.011,0.007,0.003,0.0],"text":["The recent Ebola and Zika epidemics demonstrate the need for the continuous surveillance, rapid diagnosis and real-time tracking of emerging infectious diseases. Fast, affordable sequencing of pathogen genomes \u2014 now a staple of the public health microbiology laboratory in well-resourced settings \u2014 can affect each of these areas. Coupling genomic diagnostics and epidemiology to innovative digital disease detection platforms raises the possibility of an open, global, digital pathogen surveillance system. When informed by a One Health approach, in which human, animal and environmental health are considered together, such a genomics-based system has profound potential to improve public health in settings lacking robust laboratory capacity. SUPPLEMENTARY INFORMATION: The online version of this article (doi:10.1038/nrg.2017.88) contains supplementary material, which is available to authorized users.","Cases of a novel coronavirus were first reported in Wuhan, Hubei province, China, in December 2019 and have since spread across the world. Epidemiological studies have indicated human-to-human transmission in China and elsewhere. To aid the analysis and tracking of the COVID-19 epidemic we collected and curated individual-level data from national, provincial, and municipal health reports, as well as additional information from online reports. All data are geo-coded and, where available, include symptoms, key dates (date of onset, admission, and confirmation), and travel history. The generation of detailed, real-time, and robust data for emerging disease outbreaks is important and can help to generate robust evidence that will support and inform public health decision making.","With relatively short latency and rapid propagation, viral diseases could be transmitted through the air to medical personnel or the public during the incubation period. To reduce the possibilities of spread, this research creates an infection probability model based on the settling velocity and concentration distribution of infectious droplets. Then, radio frequency identification (RFID) technology is employed to track the travel history (time, date and place) of the infected patients. A tree structure algorithm and an infection probability model are applied to trace the transmission routes, discover the correlations between carriers and suspected cases, and finally calculate the infection probability on the basis of time interval. In case of an epidemic outbreak or once an infected case is confirmed, the disease tracking and control system could be initiated by accessing RFID logs to plot the carriers\u2019 time of onset and to trace possible routes of transmission via tree diagrams. The ...","Publisher Summary This chapter introduces basic concepts of virology and exemplifies the usage of current technology for identifying and tracking viral pathogens. Virus structure, lifestyle, replication strategies, classication, the varied modes of transmission, infection, and disease manifestations are focussed upon in the chapter. Viruses are extremely simple \u201clife\u201d forms without metabolic capacity, organelles, translational machinery, or autonomous replicative potential; the essential elements of a virus are consequently minimal. A virus enters the host through any opportunistic opening of the body\u2014; such as minute breaks in the skin barrier or mucosal linings of the eye, nose, etc. Antibodies produced during an infection often have the ability to bind the virus and reduce infectivity. An immediate question to solve during a virus outbreak is the mechanism by which the virus spreads. Viruses frequently infect animal or insect vectors that serve to pass the virus to humans. By identi...","SUMMARY: Genome Detective is a web-based, user-friendly software application to quickly and accurately assemble all known virus genomes from next generation sequencing datasets. This application allows the identification of phylogenetic clusters and genotypes from assembled genomes in FASTA format. Since its release in 2019, we have produced a number of typing tools for emergent viruses that have caused large outbreaks, such as Zika and Yellow Fever Virus in Brazil. Here, we present The Genome Detective Coronavirus Typing Tool that can accurately identify the novel severe acute respiratory syndrome (SARS) related coronavirus (SARS-CoV-2) sequences isolated in China and around the world. The tool can accept up to 2,000 sequences per submission and the analysis of a new whole genome sequence will take approximately one minute. The tool has been tested and validated with hundreds of whole genomes from ten coronavirus species, and correctly classified all of the SARS-related coronavirus (S...","Abstract Timeliness is a critical issue in preventing the spread of emerging/notifiable infectious diseases, such as severe acute respiratory syndrome (SARS) or avian influenza (bird flu). Current computerized surveillance systems in many countries have demonstrated their usefulness in detecting specified communicable-diseases. However, the off-line, daily or weekly data reporting mode induces a time lag in data collection, transmission, processing, and responses. This paper proposes an on-line real-time mechanism, named EDICTS, for emerging/notifiable infectious diseases control, tracking and surveillance. It is based on the on-line health IC card system and works at the registration process of primary care practices and emergency departments. Hence, should a disease defined by CDC (Center for disease control) be detected at the registration station, EDICTS responds in real time. Note that EDICTS is a mechanism; it is CDC that determines the policy and activates it. A prototype is des...","ABSTRACTSummaryGenome Detective is a web-based, user-friendly software application to quickly and accurately assemble all known virus genomes from next generation sequencing datasets. This application allows the identification of phylogenetic clusters and genotypes from assembled genomes in FASTA format. Since its release in 2019, we have produced a number of typing tools for emergent viruses that have caused large outbreaks, such as Zika and Yellow Fever Virus in Brazil. Here, we present The Genome Detective Coronavirus Typing Tool that can accurately identify novel coronavirus (2019-nCoV) sequences isolated in China and around the world. The tool can accept up to 2,000 sequences per submission and the analysis of a new whole genome sequence will take approximately one minute. The tool has been tested and validated with hundreds of whole genomes from ten coronavirus species, and correctly classified all of the SARS-related coronavirus (SARSr-CoV) and all of the available public data f...","SUMMARY The French public health institute is responsible for promoting and coordinating threats the detection and assessment of health risks, and for suggesting possible responses. Transmissible diseases affecting both human and animal health are the focus of surveillance networks. Early detection of potential infectious threats is based on the screening of \u201c alert signals \u201c identified through routine surveillance networks and other systems. The quality and accuracy of these signals is first verified, before assessing, through a multidisciplinary approach, the risk of introduction and dissemination. This article examines specific cases illustrating the process of detection, risk analysis and response, with respect to infectious threats that are endemic in tropical regions and have the potential to be imported into metropolitan France. For both novel pathogens and exotic diseases \u2014 which, not being endemic in France, are less well known \u2014 the analysis and response process must regularl...","Rapid diagnostics, vaccines and therapeutics are important interventions for the management of the 2019 novel coronavirus (2019-nCoV) outbreak. It is timely to systematically review the potential of these interventions, including those for Middle East respiratory syndrome-Coronavirus (MERS-CoV) and severe acute respiratory syndrome (SARS)-CoV, to guide policymakers globally on their prioritization of resources for research and development. A systematic search was carried out in three major electronic databases (PubMed, Embase and Cochrane Library) to identify published studies in accordance with the Preferred Reporting Items for Systematic Reviews and Meta-Analyses (PRISMA) guidelines. Supplementary strategies through Google Search and personal communications were used. A total of 27 studies fulfilled the criteria for review. Several laboratory protocols for confirmation of suspected 2019-nCoV cases using real-time reverse transcription polymerase chain reaction (RT-PCR) have been publ...","The last decades have been characterized by the appearance of a substantial number of newly recognized or novel infectious agents and by the re-emergence of infectious diseases with a global impact. The objective of this article is to briefly describe the model of hospital response for early diagnosis and prompt management of patients with highly contagious infectious diseases. We reviewed the main components of hospital preparedness in response to clustering of highly contagious diseases. A model for the initial hospital response to an epidemic in our referral Institute is discussed. Prompt recognition and identification is the initial and indispensable step in facing any communicable diseases, regardless of whether it is a prevalent, a newly emerging one or deliberately released. The importance of developing and implementing nontraditional methods of public health surveillance and a system that allows a wide and immediate dissemination of information and exchange of views on risk ass...","Abstract Background: The protocols of WHO network laboratories facilitated development of rapid diagnosis for SARS coronavirus (CoV) using reverse transcription (RT)-PCR assays. However, several reports have shown that conventional and real-time PCR assays were very specific for SARS CoV but lack sensitivity depending on the assay, specimen, and time course of disease. Objective: To evaluate an automatic nucleic acid extraction system and two standardized real-time PCR assays for rapid diagnosis of SARS CoV during outbreak and post-epidemic periods in Hong Kong. Study design: Specimens from clinically suspected SARS patients collected during outbreak and post-epidemic periods were tested by an automatic nucleic acid extraction system followed by our first generation conventional RT-PCR and two standardized real-time PCR assays (Artus GmbH, Germany and Roche Diagnostics, Germany). Paired serum samples were assayed for increasing titer against SARS CoV. Results: In the SARS epidemic, Art...","Summary Emerging infectious diseases arising from livestock and wildlife pose serious threats to global human health, as shown by a series of continuous outbreaks involving highly pathogenic influenza, SARS, Ebola and MERS. The risk of pandemics and bioterrorism threats is ever present and growing, but our ability to combat them is limited by the lack of available vaccines, therapeutics and rapid diagnostics. The use of high bio-containment facilities, such as the CSIRO Australian Animal Health Laboratory, plays a key role studying these dangerous pathogens and facilitates the development of countermeasures. To combat diseases like MERS, we must take a holistic approach that involves the development of early biomarkers of infection, a suite of treatment options (vaccines, anti-viral drugs and antibody therapeutics) and appropriate animal models to test the safety and efficacy of candidate treatments.","For an emergent disease, such as Covid-19, with no past epidemiological data to guide models, modelers struggle to make predictions of the course of the epidemic (1). Policy decisions depend on such predictions but they vary widely. On the other hand much empirical information is already contained in the data of evolving epidemiological profiles. We show, both with evidence from data, and theoretically, how the ratio of daily infected and recovered cases can be used to track and predict the course of the epidemic. Ability to predict the turning points and the end of the epidemic is of crucial importance for fighting the epidemic and planning for a return to normalcy. The accuracy of the prediction of the peaks of the epidemic is validated using data in different regions in China showing the effects of different levels of quarantine. The validated tool can be applied to other countries where Covid-19 has spread, and generally to future epidemics. A preliminary prediction for South Korea...","In December 2019, an outbreak of atypical pneumonia (Coronavirus disease 2019 -COVID-19) associated with a novel coronavirus (SARS-CoV-2) was reported in Wuhan city, Hubei province, China. The outbreak was traced to a seafood wholesale market and human to human transmission was confirmed. The rapid spread and the death toll of the new epidemic warrants immediate intervention. The intra-host genomic variability of SARS-CoV-2 plays a pivotal role in the development of effective antiviral agents and vaccines, but also in the design of accurate diagnostics. We analyzed NGS data derived from clinical samples of three Chinese patients infected with SARS-CoV-2, in order to identify small- and large-scale intra-host variations in the viral genome. We identified tens of low- or higher-frequency single nucleotide variations (SNVs) with variable density across the viral genome, affecting 7 out of 10 protein-coding viral genes. The majority of these SNVs corresponded to missense changes. The annot...","Mass spectrometry (MS) has found numerous applications in life sciences. It has high accuracy, sensitivity and wide dynamic range in addition to medium- to high-throughput capabilities. These features make MS a superior platform for analysis of various biomolecules including proteins, lipids, nucleic acids and carbohydrates. Until recently, MS was applied for protein detection and characterization. During the last decade, however, MS has successfully been used for molecular diagnostics of microbial and viral infections with the most notable applications being identification of pathogens, genomic sequencing, mutation detection, DNA methylation analysis, tracking of transmissions, and characterization of genetic heterogeneity. These new developments vastly expand the MS application from experimental research to public health and clinical fields. Matching of molecular techniques with specific requirements of the major MS platforms has produced powerful technologies for molecular diagnosti...","By proposing a varying coefficient Susceptible-Infected-Removal model (vSIR), we track the epidemic of COVID-19 in 30 provinces in China and 15 cities in Hubei province, the epicenter of the outbreak. It is found that the spread of COVID-19 has been significantly slowing down within the two weeks from January 27 to February 10th with 87.0% and 84.3% reductions in the reproduction number R0 among the 30 provinces and 15 Hubei cities, respectively. This suggests the extreme control measures implemented since January 23, which include cutting off Wuhan and many other cities and towns, a great public awareness and high level of self isolation at home, have contributed to a substantial decline in the reproductivity of the COVID-19 in China. We predict that Hubei province will reach its peak between February 20 and 22, 2020, and if the removal rate can be increased to 0.1, the epidemic outside Hubei province will end in May 2020, and inside Hubei in early June.","Covid-19 (coronavirus disease 2019) is a rapidly spreading pandemic in many countries. The total confirmed cases of Covid-19 are exponentially increasing and many countries are fighting Covid-19 with all strategies. However, there is still lacking consensus for effective strategies. Here, I demonstrate the time dependence of the doubling time in the Covid-19 exponential growths. Tracking the time-dependent doubling time tells how well Covid-19 prevention works, giving an index for successful fighting.","The key parameter that characterizes the transmissibility of a disease is the reproduction number R. If it exceeds 1, the number of incident cases will inevitably grow over time, and a large epidemic is possible. To prevent the expansion of an epidemic, R must be reduced to a level below 1. To estimate the reproduction number, the probability distribution function of the generation interval of an infectious disease is required to be available; however, this distribution is often unknown. In this letter, given the incomplete information for the generation interval, we propose a maximum entropy method to estimate the reproduction number. Based on this method, given the mean value and variance of the generation interval, we first determine its probability distribution function and in turn estimate the real-time values of reproduction number of COVID-19 in China. By applying these estimated reproduction numbers into the susceptible-infectious-removed epidemic model, we simulate the evoluti...","AbstractThe rigidity and flexibility of two recently reported crystal structures (PDB entries 6Y2E and 6LU7) of a protease from the SARS-CoV-2 virus, the infectious agent of the COVID-19 respiratory disease, has been investigated using pebble-game rigidity analysis, elastic network model normal mode analysis, and all-atom geometric simulations. This computational investigation of the viral protease follows protocols that have been effective in studying other homodimeric enzymes. The protease is predicted to display flexible motions in vivo which directly affect the geometry of a known inhibitor binding site and which open new potential binding sites elsewhere in the structure. A database of generated PDB files representing natural flexible variations on the crystal structures has been produced and made available for download from an institutional data archive. This information may inform structure-based drug design and fragment screening efforts aimed at identifying specific antiviral ...","","Abstract The development of rapid, accurate, and sensitive diagnostic methods for detecting pathogens is the basis for treating, controlling, and eradicating infectious diseases of veterinary importance. Scientific and technological advancements have revolutionized the field of veterinary diagnostics. Genome sequencing has allowed efficient, sensitive, and specific diagnostic assays to be developed based on the detection of nucleic acids. The integration of advances in biochemistry, proteomics, engineering, and medicine offers enormous potential for the rapid and accurate diagnosis of viral, microbial, genetic, and metabolic disease. In the future, polymerase chain reaction assays, microarray testing, genomic analysis, and metabolic profiling will be accomplished in a rapid, portable, sensitive, and cost-efficient manner.","Abstract This review is concerned with what has been learned about feline infectious peritonitis (FIP) diagnostics and therapeutics since the publication of an extensive overview of literature covering the period 1963\u20132009. Although progress has been made in both areas, obtaining a definitive diagnosis of FIP remains a problem for those veterinarians and/or cat owners who require absolute certainty. This review will cover both indirect and direct diagnostic tests for the disease and will emphasize their limitations, as well as their specificity and sensitivity. There is still no effective treatment for FIP, although there are both claims that such therapies exist and glimmers of hope coming from new therapies that are under research. FIP has also been identified in wild felids and FIP-like disease is now a growing problem among pet ferrets.","Abstract An enhanced polymerase chain reaction (PCR) assay to detect the coronavirus associated with severe acute respiratory syndrome (SARS-CoV) was developed in which a target gene pre-amplification step preceded TaqMan real-time fluorescent PCR. Clinical samples were collected from 120 patients diagnosed as suspected or probable SARS cases and analyzed by conventional PCR followed by agarose gel electrophoresis, conventional TaqMan real-time PCR, and our enhanced TaqMan real-time PCR assays. An amplicon of the size expected from SARS-CoV was obtained from 28/120 samples using the enhanced real-time PCR method. Conventional PCR and real-time PCR alone identified fewer SARS-CoV positive cases. Results were confirmed by viral culture in 3/28 cases. The limit of detection of the enhanced real-time PCR method was 102-fold higher than the standard real-time PCR assay and 107-fold higher than conventional PCR methods. The increased sensitivity of the assay may help control the spread of th...","","BACKGROUND: The continuing circulation of MERS in the Middle East makes the international dissemination of the disease a permanent threat. To inform risk assessment, we investigated the spatiotemporal pattern of MERS global dissemination and looked for factors explaining the heterogeneity observed in transmission events following importation. METHODS: We reviewed imported MERS cases worldwide up to July 2015. We modelled importations in time based on air travel combined with incidence in Middle East. We used the detailed history of MERS case management after importation (time to hospitalization and isolation, number of hospitals visited,\u2026) in logistic regression to identify risk factors for secondary transmission. We assessed changes in time to hospitalization and isolation in relation to collective and public health attention to the epidemic, measured by three indicators (Google Trends, ProMED-mail, Disease Outbreak News). RESULTS: Modelled importation events were found to reproduce b...","","Abstract Only a month after the outbreak of pneumonia caused by 2019-nCoV, more than forty-thousand people were infected. This put enormous pressure on the Chinese government, medical healthcare provider, and the general public, but also made the international community deeply nervous. On the 25th day after the outbreak, the Chinese government implemented strict traffic restrictions on the area where the 2019-nCoV had originated\u2014Hubei province, whose capital city is Wuhan. Ten days later, the rate of increase of cases in Hubei showed a significant difference (p = 0.0001) compared with the total rate of increase in other provinces of China. These preliminary data suggest the effectiveness of a traffic restriction policy for this pandemic thus far. At the same time, solid financial support and improved research ability, along with network communication technology, also greatly facilitated the application of epidemic prevention measures. These measures were motivated by the need to provid...","The need for timely establishment of diagnostic assays arose when Germany was confronted with the first travel-associated outbreak of severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) in Europe. We describe our laboratory experiences during a large contact tracing investigation, comparing previously published real-time RT-PCR assays in different PCR systems and a commercial kit. We found that assay performance using the same primers and probes with different PCR systems varied and the commercial kit performed well.","The outbreak of the 2019 Novel Coronavirus (SARS-CoV-2) rapidly spread from Wuhan, China to more than 150 countries, areas or territories, causing staggering number of infections and deaths. A systematic profiling of the immune vulnerability landscape of SARS-CoV-2, which can bring critical insights into the immune clearance mechanism, peptide vaccine development, and antiviral antibody development, is lacking. In this study, we investigated the potential of the SARS-CoV-2 viral proteins to induce class I and II MHC presentation and to form linear antibody epitopes. We created an online database to broadly share the predictions as a resource for the research community. Using this resource, we showed that genetic variations in SARS-CoV-2, though still few for the moment, already follow the pattern of mutations in related coronaviruses, and could alter the immune vulnerability landscape of this virus. Importantly, we discovered evidence that SARS-CoV-2, along with related coronaviruses, ...","","","Abstract Microbiology laboratories have traditionally relied upon phenotypic methods involving culture and biochemical testing to identify and characterize clinically important pathogens. These techniques have several disadvantages including poor sensitivity and long turn-around time. Molecular and mass spectroscopy techniques are rapidly changing infection diagnosis and management. Compared with conventional culture-based techniques, these modern approaches provide substantially more rapid and specific information on organism identification and on the presence of resistance mechanisms. These methods are expected to contribute substantially to enhancing antibiotic stewardship and to improving \u2018time to appropriate antibiotics\u2019, one of the most important factors in improving the prognosis of patients with life-threatening infections. This article gives an overview of some of the practical applications of these newer technologies.","ABSTRACTCOVID-19 has caused a major epidemic worldwide, however, much is yet to be known about the epidemiology and evolution of the virus. One reason is that the challenges underneath sequencing HCoV-19 directly from clinical samples have not been completely tackled. Here we illustrate the application of amplicon and hybrid capture (capture)-based sequencing, as well as ultra-high-throughput metatranscriptomic (meta) sequencing in retrieving complete genomes, inter-individual and intra-individual variations of HCoV-19 from clinical samples covering a range of sample types and viral load. We also examine and compare the bias, sensitivity, accuracy, and other characteristics of these approaches in a comprehensive manner. This is, to date, the first work systematically implements amplicon and capture approaches in sequencing HCoV-19, as well as the first comparative study across methods. Our work offers practical solutions for genome sequencing and analyses of HCoV-19 and other emerging ...","A recent outbreak of novel coronavirus (SARS-CoV-2), the causative agent of COVID-19, has spread rapidly all over the world. Human immunodeficiency virus (HIV) is another deadly virus and causes acquired immunodeficiency syndrome (AIDS). Rapid and early detection of these viruses will facilitate early intervention and reduce disease transmission risk. Here, we present an All-In-One Dual CRISPR-Cas12a (termed \"AIOD-CRISPR\") assay method for simple, rapid, ultrasensitive, one-pot, and visual detection of coronavirus SARS-CoV-2 and HIV virus. In our AIOD CRISPR assay, a pair of crRNAs was introduced to initiate dual CRISPR-Cas12a detection and improve detection sensitivity. The AIOD-CRISPR assay system was successfully utilized to detect nucleic acids (DNA and RNA) of SARS-CoV-2 and HIV with a sensitivity of few copies. Also, it was evaluated by detecting HIV-1 RNA extracted from human plasma samples, achieving a comparable sensitivity with real-time RT-PCR method. Thus, our method has a ...","Abstract Since the COVID-19 outbreak in Wuhan City in December of 2019, numerous model predictions on the COVID-19 epidemics in Wuhan and other parts of China have been reported. These model predictions have shown a wide range of variations. In our study, we demonstrate that nonidentifiability in model calibrations using the confirmed-case data is the main reason for such wide variations. Using the Akaike Information Criterion (AIC) for model selection, we show that an SIR model performs much better than an SEIR model in representing the information contained in the confirmed-case data. This indicates that predictions using more complex models may not be more reliable compared to using a simpler model. We present our model predictions for the COVID-19 epidemic in Wuhan after the lockdown and quarantine of the city on January 23, 2020. We also report our results of modeling the impacts of the strict quarantine measures undertaken in the city after February 7 on the time course of the ep...","Background Despite available recommendations on infection control for severe acute respiratory syndrome (SARS), information is limited on actual practices in Asian hospitals during the epidemic. We describe practices observed by mobile SARS containment teams (mobile teams) during outbreak investigations. Methods We retrospectively summarized infection control practices observed in hospitals visited by mobile teams in the Lao People's Democratic Republic (PDR), Taiwan, and Thailand, during March and April 2003. Results Mobile teams investigated 22 reports of SARS in 20 hospitals (1, 5, and 14 hospitals in Lao PDR, Taiwan, and Thailand, respectively). Facilities ranged from urban hospitals with negative-pressure isolation rooms and high-efficiency particulate air filtration to rural hospitals with patient rooms open to outside air circulation and intermittent running water. At the time of mobile team visits, 5 (25%) hospitals implemented infection control practices consistent with World ...","We integrate the human movement and healthcare resource data to identify cities with high vulnerability towards the 2019-nCoV epidemic with respect to available health resources. The results inform public health responses in multiple ways.","We present a self-synchronizing and robust method for comparing the progression of the Covid-19 epidemics among multiple countries. In their growth phase the epidemics show power law rather than exponential law time dependences. They are similar enough for the earlier China outbreak to guide other countries projections. The delayed reaction of European countries is shown to produce a significantly worse outcome compared to China.","Understanding the disease pathogenesis of the novel coronavirus, denoted SARS-CoV-2, is critical to the development of anti-SARS-CoV-2 therapeutics. The global propagation of the viral disease, denoted COVID-19 (\"coronavirus disease 2019\"), has unified the scientific community in searching for possible inhibitory small molecules or polypeptides. Given the known interaction between the human ACE2 (\"Angiotensin-converting enzyme 2\") protein and the SARS-CoV virus (responsible for the coronavirus outbreak circa. 2003), considerable focus has been directed towards the putative interaction between the SARS-CoV-2 Spike protein and ACE2. However, a more holistic understanding of the SARS-CoV-2 vs. human inter-species interactome promises additional putative protein-protein interactions (PPI) that may be considered targets for the development of inhibitory therapeutics. To that end, we leverage two state-of-the-art, sequence-based PPI predictors (PIPE4 &amp; SPRINT) capable of generating the compr...","Publisher Summary Proteolytic processing of the coronavirus replicase polyproteins is essential for ongoing viral ribonucleic acid (RNA) synthesis. Therefore, the severe acute respiratory syndrome (SARS)-coronaviruses (SARS-CoV) proteases are attractive targets for the development of antiviral drugs to reduce viral replication and pathogenicity. The structure and activity of the coronavirus 3C-like protease (3CLpro) has already been elucidated, and the design of inhibitors to 3CLpro as therapeutics has been proposed. The chapter discusses SARS-CoV 3CLpro inhibitors that include covalent inhibitors, noncovalent inhibitors, and inhibitors from screening. SARS-CoV papain-like protease (PLpro) is considered an equally viable target to 3CLpro for drug design because both are essential for viral replication. However, PLpro has likely not been pursued because of the paucity of structural information. Several compounds have been identified that have shown inhibitory activity against SARS-CoV. ...","Background: The recent outbreak of infections by the 2019 novel coronavirus (2019-nCoV), the third zoonotic CoV has raised great public health concern. The demand for rapid and accurate diagnosis of this novel pathogen brought significant clinical and technological challenges. Currently, metagenomic next-generation sequencing (mNGS) and reverse-transcription PCR (RT-PCR) are the most widely used molecular diagnostics. Methods: 2019-nCoV infections were confirmed in 52 specimens by mNGS. Genomic information was analyzed and used for the design and development of an isothermal, CRISPR-based diagnostic for this novel virus. The diagnostic performance of CRISPR-nCoV was assessed and compared across three technology platforms (mNGS, RT-PCR and CRISPR). Results: 2019-nCoVs sequenced in our study were conserved with the Wuhan strain, and shared certain genetic similarity with SARS-CoV. A high degree of variation in the level of viral RNA was observed in clinical specimens. CRISPR-nCoV demonst...","Abstract The outbreak of the 2019 novel coronavirus disease (COVID-19) has caused more than 100,000 people infected and thousands of deaths. Currently, the number of infections and deaths is still increasing rapidly. COVID-19 seriously threatens human health, production, life, social functioning and international relations. In the fight against COVID-19, Geographic Information Systems (GIS) and big data technologies have played an important role in many aspects, including the rapid aggregation of multi-source big data, rapid visualization of epidemic information, spatial tracking of confirmed cases, prediction of regional transmission, spatial segmentation of the epidemic risk and prevention level, balancing and management of the supply and demand of material resources, and social-emotional guidance and panic elimination, which provided solid spatial information support for decision-making, measures formulation, and effectiveness assessment of COVID-19 prevention and control. GIS has d...","Abstract Post-transcriptional gene silencing (PTGS) makes possible new approaches for studying the various steps of the viral cycle. Plus-strand RNA viruses appear to be attractive targets for small interfering RNAs (siRNAs), as their genome functions as both mRNA and replication template. PTGS creates an alternative to classic reverse genetics for viruses with either negative-strand or double-stranded RNA genomes and for those with a large genome. PTGS allows modification of the expression of a given cellular gene as a means to elucidate its role in the viral cycle and in virus\u2013host cell interactions, and to investigate cellular pathways involved in viral pathogenesis. It also allows the creation of new animal models of human diseases. In addition, PTGS already appears to be a promising new therapeutic tool to fight viral multiplication and dissemination through the host and to prevent inflammation and virus-induced pathogenesis, including virus-induced tumorigenesis.","The current diagnosis of severe acute respiratory syndrome-associated coronavirus (SARS-CoV) relies on laboratory-based tests since its clinical features are nonspecific, unlike other respiratory pathogens. Therefore, the development of a rapid and simple method for on-site detection of SARS-CoV is crucial for the identification and prevention of future SARS outbreaks. In this study, a simple colorimetric and multiplex loop-mediated isothermal amplification (LAMP) assay was developed to rapid screening of severe acute respiratory syndrome-associated coronavirus (SARS-CoV). It can be visually detected based on color change and monitored in real-time with fluorescent signals. The performance of this assay, based on six primers targeting open reading frame (ORF1b) and nucleocapsid (N) genes located in different regions of the SARS-CoV, was compared with real-time RT-PCR assay using various concentrations of target genes. The detection limit of the LAMP assay was comparable to that of real...","Abstract A novel coronavirus designated as 2019-nCoV first appeared in Wuhan, China in late December 2019. Dozens of people died in China, and thousands of people infected as 2019-nCoV continues to spread around the world. We have described the discovery, emergence, genomic characteristics, and clinical diagnostics of 2019-nCoV.","ABSTRACTOn December 31, 2019, the World Health Organization was notified about a cluster of pneumonia of unknown aetiology in the city of Wuhan, China. Chinese authorities later identified a new coronavirus (2019-nCoV) as the causative agent of the outbreak. As of January 23, 2020, 655 cases have been confirmed in China and several other countries. Understanding the transmission characteristics and the potential for sustained human-to-human transmission of 2019-nCoV is critically important for coordinating current screening and containment strategies, and determining whether the outbreak constitutes a public health emergency of international concern (PHEIC). We performed stochastic simulations of early outbreak trajectories that are consistent with the epidemiological findings to date. We found the basic reproduction number, R0, to be around 2.2 (90% high density interval 1.4\u20143.8), indicating the potential for sustained human-to-human transmission. Transmission characteristics appear t...","Pneumonia patients of 2019-ncov in latent period are not easy to be effectively quarantined, but there is evidence that they have strong infectious ability. Here, the infectious ability of patients during the latent period is slightly less than that of the infected patients was assumed. We established a new SEIR propagation dynamics model, that considered the weak transmission ability of the incubation period, the variation of the incubation period length, and the government intervention measures to track and isolate comprehensively. Based on the raw epidemic data of China from January 23, 2020 to February 10, 2020, the dynamic parameters of the new present SEIR model are fitted. Through the Euler integration algorithm to solve the model, the effect of infectious ability of incubation patients on the theoretical estimation of the present SEIR model was analyzed, and the occurrence time of peak number in China was predicted.","Summary Objective Public health surveillance applications are central to the collection, analysis and dissemination of disease and health information. As these applications evolve and mature, it is evident that many of these applications must address similar requirements, such as policies, security and flexibility. It is important a software architecture is created to meet these requirements. Methods We outline the requirements for a public health surveillance application, and define a set of common components to address these requirements. These components are configured to produce services used in the development of public health applications. Results A layered software architecture, the ALPHA architecture, has been developed to support the development of public health applications. The architecture has been used to build eleven surveillance applications for the Public Health Agency of Canada in the areas of disease surveillance, survey, distributed data collection and inventory mana...","The emergence of Middle East respiratory syndrome coronavirus (MERS-CoV) in 2012 marked the second introduction of a highly pathogenic coronavirus into the human population in the twenty-first century. The continuing introductions of MERS-CoV from dromedary camels, the subsequent travel-related viral spread, the unprecedented nosocomial outbreaks and the high case-fatality rates highlight the need for prophylactic and therapeutic measures. Scientific advancements since the 2002\u20132003 severe acute respiratory syndrome coronavirus (SARS-CoV) pandemic allowed for rapid progress in our understanding of the epidemiology and pathogenesis of MERS-CoV and the development of therapeutics. In this Review, we detail our present understanding of the transmission and pathogenesis of SARS-CoV and MERS-CoV, and discuss the current state of development of measures to combat emerging coronaviruses. SUPPLEMENTARY INFORMATION: The online version of this article (doi:10.1038/nrmicro.2016.81) contains suppl...","During the 2003 severe acute respiratory syndrome (SARS) outbreak, a real-time quantitative polymerase chain reaction, which targets the nucleocapsid gene at the 3\u2032-end of the viral genome, was established to detect and identify the SARS-associated coronavirus. We describe the use of this assay to screen &gt;700 clinical samples."],"title":["Towards a genomics-informed, real-time, global pathogen surveillance system","Epidemiological data from the COVID-19 outbreak, real-time case information","The construction of a hospital disease tracking and control system with a disease infection probability model","CHAPTER 4 Keeping Track of Viruses","Genome Detective Coronavirus Typing Tool for rapid identification and characterization of novel coronavirus genomes","Design and prototype of a mechanism for active on-line emerging/notifiable infectious diseases control, tracking and surveillance, based on a national healthcare card system","Genome Detective Coronavirus Typing Tool for rapid identification and characterization of novel coronavirus genomes","Risques d\u2019importation des maladies infectieuses exotiques en France m\u00e9tropolitaine : d\u00e9tection, alerte et r\u00e9ponse","Potential Rapid Diagnostics, Vaccine and Therapeutics for 2019 Novel Coronavirus (2019-nCoV): A Systematic Review","The Initial Hospital Response to an Epidemic","Clinical evaluation of real-time PCR assays for rapid diagnosis of SARS coronavirus during outbreak and post-epidemic periods","Overview of the CSIRO Australian Animal Health Laboratory","A data-driven tool for tracking and predicting the course of COVID-19 epidemic as it evolves","SARS-CoV-2 exhibits intra-host genomic plasticity and low-frequency polymorphic quasispecies","Application of mass spectrometry to molecular diagnostics of viral infections","Tracking and Predicting COVID-19 Epidemic in China Mainland","Doubling time tells how effective Covid-19 prevention works","Maximum entropy method for estimating the reproduction number: An investigation for COVID-19 in China","Rigidity, normal modes and flexible motion of a SARS-CoV-2 (COVID-19) protease structure","A data driven time-dependent transmission rate for tracking an epidemic: a case study of 2019-nCoV","Future Veterinary Diagnostics","An update on feline infectious peritonitis: Diagnostics and therapeutics","A real-time PCR for SARS-coronavirus incorporating target gene pre-amplification","Tracking online heroisation and blame in epidemics","Risk of MERS importation and onward transmission: a systematic review and analysis of cases reported to WHO","Rapid diagnostics urgently needed for killer infections","Lessons learned from the 2019-nCoV epidemic on prevention of future infectious diseases","Rapid establishment of laboratory diagnostics for the novel coronavirus SARS-CoV-2 in Bavaria, Germany, February 2020","The immune vulnerability landscape of the 2019 Novel Coronavirus, SARS-CoV-2","The continuous evolution and dissemination of 2019 novel human coronavirus","Estimation of Coronavirus Disease 2019 (COVID-19) Burden and Potential for International Dissemination of Infection From Iran","Molecular-based diagnostics, including future trends","Multiple approaches for massively parallel sequencing of HCoV-19 (SARS-CoV-2) genomes directly from clinical samples","All-in-One Dual CRISPR-Cas12a (AIOD-CRISPR) Assay: A Case for Rapid, Ultrasensitive and Visual Detection of Novel Coronavirus SARS-CoV-2 and HIV virus","Why is it difficult to accurately predict the COVID-19 epidemic?","Infection control practices for SARS in Lao People's Democratic Republic, Taiwan, and Thailand: Experience from mobile SARS containment teams, 2003","Healthcare-resource-adjusted vulnerabilities towards the 2019-nCoV epidemic across China","Tracking the Covid-19 pandemic : Simple visualization of the epidemic states and trajectories of select European countries &amp; assessing the effects of delays in official response.","Computational Prediction of the Comprehensive SARS-CoV-2 vs. Human Interactome to Guide the Design of Therapeutics","Progress in Anti-SARS Coronavirus Chemistry, Biology and Chemotherapy","Development and Evaluation of A CRISPR-based Diagnostic For 2019-novel Coronavirus","COVID-19: Challenges to GIS with Big Data","Silencing viruses by RNA interference","A Simple and Multiplex Loop-Mediated Isothermal Amplification (LAMP) Assay for Rapid Detection of SARS-CoV","Novel coronavirus: From discovery to clinical diagnostics","Pattern of early human-to-human transmission of Wuhan 2019-nCoV","SEIR Transmission dynamics model of 2019 nCoV coronavirus with considering the weak infectious ability and changes in latency duration","The ALPHA Project: An architecture for leveraging public health applications","SARS and MERS: recent insights into emerging coronaviruses","Real-Time Polymerase Chain Reaction for Detecting SARS Coronavirus, Beijing, 2003"],"url":["https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7097748/","https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7093412/","https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7087954/","https://doi.org/10.1016/b978-012088483-4/50007-x","https://doi.org/10.1093/bioinformatics/btaa145","https://doi.org/10.1016/j.cmpb.2007.02.005","https://doi.org/10.1101/2020.01.31.928796","https://doi.org/10.1016/s0001-4079(19)32418-5","https://doi.org/10.3390/jcm9030623","https://doi.org/10.1016/j.arcmed.2005.05.003","https://doi.org/10.1016/j.jcv.2004.09.029","https://doi.org/10.1016/j.jiph.2016.04.007","https://doi.org/10.1101/2020.03.28.20046177","https://doi.org/10.1101/2020.03.27.009480","http://europepmc.org/articles/pmc5831079?pdf=render","https://doi.org/10.1101/2020.02.17.20024257","https://doi.org/10.1101/2020.03.26.20044644","https://doi.org/10.1101/2020.03.14.20035659","https://doi.org/10.1101/2020.03.10.986190","https://doi.org/10.1016/j.scib.2020.02.005","https://doi.org/10.1053/j.jepm.2010.05.006","https://doi.org/10.1016/j.tvjl.2014.04.016","https://doi.org/10.1016/j.bbrc.2003.11.064","https://doi.org/10.1016/s2468-2667(20)30033-5","https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5000488/","https://doi.org/10.1016/s2213-2600(13)70099-7","https://doi.org/10.1016/j.micinf.2020.02.004","https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7068163/","https://doi.org/10.1101/2020.02.08.939553","https://doi.org/10.1016/j.jinf.2020.02.001","https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7081176/","https://doi.org/10.1016/j.mpmed.2013.08.012","https://doi.org/10.1101/2020.03.16.993584","https://doi.org/10.1101/2020.03.19.998724","https://doi.org/10.1016/j.idm.2020.03.001","https://doi.org/10.1016/j.ajic.2004.03.005","https://doi.org/10.1101/2020.02.11.20022111","https://doi.org/10.1101/2020.03.14.20035964","https://doi.org/10.1101/2020.03.29.014381","https://doi.org/10.1016/s0065-7743(06)41011-3","https://doi.org/10.1101/2020.02.22.20025460","https://doi.org/10.1016/j.geosus.2020.03.005","https://doi.org/10.1016/j.micinf.2005.02.003","https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7097549/","https://doi.org/10.1016/j.meegid.2020.104211","https://doi.org/10.1101/2020.01.23.917351","https://doi.org/10.1101/2020.02.16.20023655","https://doi.org/10.1016/j.ijmedinf.2005.10.006","https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7097822/","https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3322935/"],"x":[0.23278608236054535,0.2889027622426893,0.25410214619760685,0.41108454380901754,0.4867132600015802,0.3284600843159042,0.48635776877475506,0.23339562366929137,0.8456498854988137,0.3295886390389903,0.8676365743055805,0.3485889367536638,0.2779563588602361,0.628689707069077,0.40374231503120955,0.22819717233098313,0.5063220143809753,0.3061395351247071,0.5971790665475807,0.19363517797126845,0.8580373178900748,0.39843134295360083,0.8636849045707552,0.39622572512475274,0.9674091058422362,0.4013455517016946,0.308575430174524,0.8495545562650394,0.7732843791421931,0.3412460944122971,0.5330505282543632,0.40117761935566687,0.48646955051083596,0.8447092107547873,0.2668694289707123,0.5060754218232422,0.2779165044984094,0.3834935177350887,0.727622945013729,0.6191363179382736,0.8450964443514554,0.33695106087525384,0.5252811433443317,0.8618861611898277,0.23535527750218543,0.19629049016706446,0.30477224963828725,0.2498470624656106,0.9475037064351977,0.8442271202319378],"y":[0.5385340137303893,0.6547958367580943,0.6999763280327365,0.21068323151696952,0.1601488286103946,0.37820141558056297,0.16117954095994672,0.531873703834994,0.5378786444049526,0.3670108012031819,0.6160731628538396,0.3125710382884974,0.8039984206370737,0.2929591809173555,0.34680773217488753,0.8138588229674857,0.7979067241831209,0.8961381597501304,0.13906255233272602,0.48032486016938714,0.5567685760619223,0.38832562853789493,0.6113093040377605,0.447822659161038,0.46449143726896663,0.31407714299557865,0.623727358256833,0.6006809789077675,0.30638671556847286,0.5151167743712408,0.7916745324032815,0.348294695741071,0.06769344705958731,0.5758466046796783,0.8325054060554384,0.5511606553313998,0.49156169505353775,0.8709618625085367,0.17131625805800904,0.10507187316071932,0.5414581106868969,0.7500470485515525,0.22487890776987124,0.6066708026766122,0.4602809413807584,0.7109129778823469,0.8136345507090841,0.5651413908346821,0.3615370102751864,0.6056383155980776]},"selected":{"id":"8157"},"selection_policy":{"id":"8178"}},"id":"8085","type":"ColumnDataSource"},{"attributes":{},"id":"8132","type":"BasicTicker"},{"attributes":{"below":[{"id":"8096"}],"center":[{"id":"8099"},{"id":"8103"}],"left":[{"id":"8100"}],"plot_height":340,"plot_width":900,"renderers":[{"id":"8114"}],"title":{"id":"8170"},"toolbar":{"id":"8107"},"toolbar_location":"below","x_range":{"id":"8088"},"x_scale":{"id":"8092"},"y_range":{"id":"8090"},"y_scale":{"id":"8094"}},"id":"8087","subtype":"Figure","type":"Plot"},{"attributes":{},"id":"8136","type":"BasicTicker"},{"attributes":{"axis":{"id":"8135"},"dimension":1,"ticker":null},"id":"8138","type":"Grid"},{"attributes":{},"id":"8178","type":"UnionRenderers"},{"attributes":{},"id":"8088","type":"Range1d"},{"attributes":{},"id":"8177","type":"StringEditor"},{"attributes":{},"id":"8140","type":"ResetTool"},{"attributes":{},"id":"8123","type":"Range1d"},{"attributes":{"callback":null},"id":"8139","type":"TapTool"},{"attributes":{"text":"&lt;h3&gt;Document ranking&lt;/h3&gt;"},"id":"8162","type":"Div"},{"attributes":{"below":[{"id":"8131"}],"center":[{"id":"8134"},{"id":"8138"}],"left":[{"id":"8135"}],"plot_height":340,"plot_width":900,"renderers":[{"id":"8149"}],"title":{"id":"8172"},"toolbar":{"id":"8142"},"toolbar_location":"below","x_range":{"id":"8123"},"x_scale":{"id":"8127"},"y_range":{"id":"8125"},"y_scale":{"id":"8129"}},"id":"8122","subtype":"Figure","type":"Plot"},{"attributes":{},"id":"8176","type":"StringFormatter"},{"attributes":{},"id":"8184","type":"StringFormatter"},{"attributes":{"source":{"id":"8085"}},"id":"8115","type":"CDSView"},{"attributes":{"callback":null,"tooltips":[["Title","@title"],["Abstract","@text"],["Published","@date"],["Authors","@authors"],["Journal","@journal"],["URL","@url"]]},"id":"8106","type":"HoverTool"},{"attributes":{"fill_color":{"field":"color"},"line_color":{"field":"color"},"radius":{"field":"rad","units":"data"},"x":{"field":"x"},"y":{"field":"y"}},"id":"8147","type":"Circle"},{"attributes":{"children":[{"id":"8155"},{"id":"8122"}]},"id":"8168","type":"Row"},{"attributes":{},"id":"8174","type":"StringFormatter"},{"attributes":{"formatter":{"id":"8181"},"ticker":{"id":"8101"}},"id":"8100","type":"LinearAxis"},{"attributes":{"data":{"authors":["McCloskey, Brian; Heymann, David L.","Allam, Zaheer; Jones, David S.","Huang, Yi Chao; Xie, Qianyi","Wilson, Mary E.","Chong Lei; Binxiao Su; Hailong Dong; Andrea Bellavia; Raffaele Di Fenza; Bijan Safaee Fakhr; Stefano Gianni; Luigi Giuseppe Grassi; Robert Kacmarek; Caio Cesar Araujo Morais; Riccardo Pinciroli; Emanuele Vassena; Lorenzo Berra","Canela, Luciana Nascimento Pinto; Magalh\u00e3es-Barbosa, Maria Clara de; Raymundo, Carlos Eduardo; Carney, Sharon; Siqueira, Marilda Mendonca; Prata-Barbosa, Arnaldo; Cunha, Antonio Jos\u00e9 Ledo Alves da","Li, Chun; Yang, Yanling; Ren, Linzhu","Pang, Junxiong; Wang, Min Xian; Ang, Ian Yi Han; Tan, Sharon Hui Xuan; Lewis, Ruth Frances; Chen, Jacinta I. Pei; Gutierrez, Ramona A.; Gwee, Sylvia Xiao Wei; Chua, Pearleen Ee Yong; Yang, Qian; Ng, Xian Yi; Yap, Rowena K. S.; Tan, Hao Yi; Teo, Yik Ying; Tan, Chorh Chuan; Cook, Alex R.; Yap, Jason Chin-Huat; Hsu, Li Yang","Ison, Michael G; Johnston, Sebastian L; Openshaw, Peter; Murphy, Brian; Hayden, Frederick","Zhou, Chenghu; Su, Fenzhen; Pei, Tao; Zhang, An; Du, Yunyan; Luo, Bin; Cao, Zhidong; Wang, Juanle; Yuan, Wen; Zhu, Yunqiang; Song, Ci; Chen, Jie; Xu, Jun; Li, Fujia; Ma, Ting; Jiang, Lili; Yan, Fengqin; Yi, Jiawei; Hu, Yunfeng; Liao, Yilan; Xiao, Han","Xianding Deng; Wei Gu; Scot Federman; Louis Du Plessis; Oliver Pybus; Nuno Faria; Candace Wang; Guixia Yu; Chao-Yang Pan; Hugo Guevara; Alicia Sotomayor-Gonzalez; Kelsey Zorn; Allan Gopez; Venice Servellita; Elaine Hsu; Steve Miller; Trevor Bedford; Alexander Greninger; Pavitra Roychoudhury; Michael Famulare; Helen Y Chu; Jay Shendure; Lea Starita; Catie Anderson; Karthik Gangavarapu; Mark Zeller; Emily Spencer; Kristian Andersen; Duncan MacCannell; Suxiang Tong; Gregory Armstrong; Clinton Paden; Yan Li; Ying Zhang; Scott Morrow; Matthew Willis; Bela Matyas; Sundari Mase; Olivia Kasirye; Maggie Park; Curtis Chan; Alexander Yu; Shua Chai; Elsa Villarino; Brandon Bonin; Debra Wadford; Charles Y Chiu","Li, Jing; Liu, Wenjun","Patricia Rios; Amruta Radhakrishnan; Sonia M. Thomas; Nazia Darvesh; Sharon E. Straus; Andrea C. Tricco","Mao, Kang; Zhang, Hua; Yang, Zhugen","Yang, Yongshi; Peng, Fujun; Wang, Runsheng; Guan, Kai; Jiang, Taijiao; Xu, Guogang; Sun, Jinlyu; Chang, Christopher","Bai, Li; Yang, Dawei; Wang, Xun; Tong, Lin; Zhu, Xiaodan; Zhong, Nanshan; Bai, Chunxue; Powell, Charles A.; Chen, Rongchang; Zhou, Jian; Song, Yuanlin; Zhou, Xin; Zhu, Huili; Han, Baohui; Li, Qiang; Shi, Guochao; Li, Shengqing; Wang, Changhui; Qiu, Zhongmin; Zhang, Yong; Xu, Yu; Liu, Jie; Zhang, Ding; Wu, Chaomin; Li, Jing; Yu, Jinming; Wang, Jiwei; Dong, Chunling; Wang, Yaoli; Wang, Qi; Zhang, Lichuan; Zhang, Min; Ma, Xia; Zhao, Lin; Yu, Wencheng; Xu, Tao; Jin, Yang; Wang, Xiongbiao; Wang, Yuehong; Jiang, Yan; Chen, Hong; Xiao, Kui; Zhang, Xiaoju; Song, Zhenju; Zhang, Ziqiang; Wu, Xueling; Sun, Jiayuan; Shen, Yao; Ye, Maosong; Tu, Chunlin; Jiang, Jinjun; Yu, Hai; Tan, Fei","Dong, J.; Ismail, N.; Walker, D.H.","Jessica E. Manning; Jennifer A. Bohl; Sreyngim Lay; Sophana Chea; Ly Sovann; Yi Sengdoeurn; Seng Heng; Chan Vuthy; Katrina Kalantar; Vida Ahyong; Michelle Tan; Jonathan Sheu; Cristina M. Tato; Joseph L. DeRisi; Laurence Baril; Veasna Duong; Philippe Dussart; Erik A. Karlsson","","Kevin Dick; Kyle K Biggar; James R Green","Dhama, Kuldeep; Sharun, Khan; Tiwari, Ruchi; Dadar, Maryam; Malik, Yashpal Singh; Singh, Karam Pal; Chaicumpa, Wanpen","Mengying Dong; Xiaojun Cao; Mingbiao Liang; Lijuan Li; Huiying Liang; Guangjian Liu","Drifa Belhadi; Nathan Peiffer-Smadja; Yazdan Yazdanpanah; France Mentr\u00e9; C\u00e9dric Laou\u00e9nan","Peng, Philip W.H.; Ho, Pak-Leung; Hota, Susy S.","Timokratis Karamitros; Gethsimani Papadopoulou; Maria Bousali; Anastasios Mexias; Sotiris Tsiodras; Andreas Mentis","Hassan, Md. Zakiul; Sturm-Ramirez, Katharine; Rahman, Mohammad Ziaur; Hossain, Kamal; Aleem, Mohammad Abdul; Bhuiyan, Mejbah Uddin; Islam, Md. Muzahidul; Rahman, Mahmudur; Gurley, Emily S.","Mengying Dong; Xiaojun Cao; Mingbiao Liang; Lijuan Li; Huiying Liang; Guangjian Liu","Beigel, John H.; Nam, Hannah H.; Adams, Peter L.; Krafft, Amy; Ince, William L.; El-Kamary, Samer S.; Sims, Amy C.","Malik, Yashpal Singh; Sircar, Shubhankar; Bhat, Sudipta; Sharun, Khan; Dhama, Kuldeep; Dadar, Maryam; Tiwari, Ruchi; Chaicumpa, Wanpen","Huang, Jyh-Win; Hou, Ting-Wei","Zhidong Cao; Qingpeng Zhang; Xin Lu; Dirk Pfeiffer; Lei Wang; Hongbing Song; Tao Pei; Zhongwei Jia; Daniel Dajun Zeng","Brisse, Sylvain; Brehony, Carina; Concei\u00e7\u00e3o, Teresa; Cubero, Meritxell; Glasner, Corinna; Le Gouil, Meriadeg; Renvois\u00e9, Aur\u00e9lie; Sheppard, Sam; Weinert, Lucy A.","Bleibtreu, A.; Bertine, M.; Bertin, C.; Houhou-Fidouh, N.; Visseaux, B.","Pang, Junxiong; Wang, Min Xian; Ang, Ian Yi Han; Tan, Sharon Hui Xuan; Lewis, Ruth Frances; Chen, Jacinta I. Pei; Gutierrez, Ramona A.; Gwee, Sylvia Xiao Wei; Chua, Pearleen Ee Yong; Yang, Qian; Ng, Xian Yi; Yap, Rowena K. S.; Tan, Hao Yi; Teo, Yik Ying; Tan, Chorh Chuan; Cook, Alex R.; Yap, Jason Chin-Huat; Hsu, Li Yang","","Beigel, John H.; Nam, Hannah H.; Adams, Peter L.; Krafft, Amy; Ince, William L.; El-Kamary, Samer S.; Sims, Amy C.","Beigel, John H.; Nam, Hannah H.; Adams, Peter L.; Krafft, Amy; Ince, William L.; El-Kamary, Samer S.; Sims, Amy C.","Dong, J.; Ismail, N.; Walker, D.H.","Mackey, Tim K.","Huang, Jyh-Win; Hou, Ting-Wei","Mengying Dong; Xiaojun Cao; Mingbiao Liang; Lijuan Li; Huiying Liang; Guangjian Liu","Alexander Ugarov","Richard A Neher; Robert Dyrdak; Valentin Druelle; Emma B Hodcroft; Jan Albert","Peiris, J.S.M.","Peter Byass","Wingfield, Tom; Cuevas, Luis E; MacPherson, Peter; Millington, Kerry A; Squire, S Bertel","Lenard, J.","Zhou, Chenghu; Su, Fenzhen; Pei, Tao; Zhang, An; Du, Yunyan; Luo, Bin; Cao, Zhidong; Wang, Juanle; Yuan, Wen; Zhu, Yunqiang; Song, Ci; Chen, Jie; Xu, Jun; Li, Fujia; Ma, Ting; Jiang, Lili; Yan, Fengqin; Yi, Jiawei; Hu, Yunfeng; Liao, Yilan; Xiao, Han","Alexander Lachmann","McCloskey, Brian; Heymann, David L."],"color":["#ef8a62","#ef8a62","#ef8a62","#ef8a62","#ef8a62","#ef8a62","#ef8a62","#ef8a62","#ef8a62","#ef8a62","#ef8a62","#ef8a62","#ef8a62","#ef8a62","#ef8a62","#ef8a62","#ef8a62","#ef8a62","#ef8a62","#ef8a62","#ef8a62","#ef8a62","#ef8a62","#ef8a62","#ef8a62","#ef8a62","#ef8a62","#ef8a62","#ef8a62","#ef8a62","#ef8a62","#ef8a62","#ef8a62","#ef8a62","#ef8a62","#ef8a62","#ef8a62","#ef8a62","#ef8a62","#ef8a62","#ef8a62","#ef8a62","#ef8a62","#ef8a62","#ef8a62","#ef8a62","#ef8a62","#ef8a62","#ef8a62","#ef8a62"],"date":["2020-02-05","2020","2013-06-09","2020-02-29","2020-03-13","2018-10-31","2020-08-31","2020","2004-06-30","2020-03-20","2020-03-30","2020","2020-03-26","2020-03-23","2020-03-03","2020-12-31","2017-12-31","2020-03-05","2020-02-03","2020-03-31","2020-03-18","2020-03-30","2020-03-20","2020-02-27","2020-03-28","2019-10-28","2020-03-30","2019-07-31","2020-02-08","2007-05-31","2020-02-09","2014-03-31","2019-11-11","2020","2020-03-01","2019-07-31","2019-07-31","2017-12-31","2016-11-24","2007-05-31","2020-03-30","2020-03-30","2020-02-17","2012-12-31","2020-03-31","2020-03-24","2008-12-31","2020-03-20","2020-03-18","2020-02-05"],"doc":["uze6ze8o","94tdt2rv","4rulshop","i6a0u9r1","e99athff","7juagezq","aeogp8c7","8gncbgot","0rrgqukp","5gbkrs73","cbc98t7x","nzvuugls","kwncu3ji","lzuxlzb0","szsb1oan","wyz5jyjh","1h87wldg","fu3cl7lq","r5niszuj","dxabs45r","68sbqdi3","3wuh6k6g","t1wpujpm","4zg3ms5b","jxm69ndw","idh7io9v","3wuh6k6g","h97dui9n","1qkwsh6a","rpq6xan7","q14x0i2c","a4mf2vnp","1ub01j2x","8gncbgot","nb0gkwku","h97dui9n","h97dui9n","1h87wldg","aw8ozs43","rpq6xan7","3wuh6k6g","pd2palu4","3p2dl8yf","b2d7t7ge","r141na6j","yso6x9g7","3wz7k4vy","5gbkrs73","gttuxtw6","uze6ze8o"],"journal":["Epidemiology and Infection","Healthcare","J Intell Manuf","Travel Medicine and Infectious Disease","","The Brazilian Journal of Infectious Diseases","Infection, Genetics and Evolution","Journal of Clinical Medicine","Antiviral Research","Geography and Sustainability","","Protein Cell","","Environ Sci Technol","Journal of Autoimmunity","Clinical eHealth","Diagnostic Molecular Pathology","","Nat Med","","Hum Vaccin Immunother","","","British Journal of Anaesthesia","","PLoS One","","Antiviral Research","Vet Q","Computer Methods and Programs in Biomedicine","","Research in Microbiology","M\u00e9decine et Maladies Infectieuses","Journal of Clinical Medicine","Bull World Health Organ","Antiviral Research","Antiviral Research","Diagnostic Molecular Pathology","BMC Infect Dis","Computer Methods and Programs in Biomedicine","","","","Medical Microbiology","","The Lancet Respiratory Medicine","Encyclopedia of Virology","Geography and Sustainability","","Epidemiology and Infection"],"rad":[0.02,0.01272330626717879,0.012455112866253969,0.011846576473824071,0.011469369250702462,0.010333420876607085,0.010236180350823479,0.010032605208195999,0.009136272155994416,0.009135984637177186,0.008997801503327203,0.00871552040096239,0.008357258154332515,0.0072909622646349795,0.007245598809035412,0.006853670552015261,0.00683413765555157,0.006686470096095095,0.006108937984886549,0.006099199837803229,0.005986795132833361,0.00593539110891728,0.0056385142350239455,0.005628973916453585,0.005540673225016002,0.005394140964535242,0.0053658917681350575,0.0053280231222649745,0.005279828661231391,0.005077684230726519,0.005075976808959802,0.004971786957450265,0.004951946136638312,0.004852909556129149,0.004667536339076724,0.0046538183598232065,0.0045814994211172385,0.0043214463849876985,0.004310705589492498,0.0042581711898418,0.004026303740498484,0.003920808090615323,0.003918655417419405,0.0038736941435112953,0.003510245942049191,0.0034355031488953213,0.0032852595027057035,0.0032830399021222412,0.003154979785665652,0.003],"scr":[1.0,0.572,0.556,0.52,0.498,0.431,0.426,0.414,0.361,0.361,0.353,0.336,0.315,0.252,0.25,0.227,0.226,0.217,0.183,0.182,0.176,0.173,0.155,0.155,0.149,0.141,0.139,0.137,0.134,0.122,0.122,0.116,0.115,0.109,0.098,0.097,0.093,0.078,0.077,0.074,0.06,0.054,0.054,0.051,0.03,0.026,0.017,0.017,0.009,0.0],"text":["One of the critical lessons from the SARS experience was the absolute necessity to be able to coordinate the international resources that are available in an outbreak and to get them focussed on identifying priorities and solving problems. The WHO established the means to do this for SARS and it has since been further developed and integrated into global preparedness, especially after the West Africa Ebola epidemic. Organisations such as the Global Outbreak Alert and Response Network (GOARN), the Coalition for Epidemic Preparedness Innovations (CEPI), the Global Research Collaboration For Infectious Disease Preparedness (GloPID-R) and the Global Initiative on Sharing All Influenza Data (GISAID) have been supported by the WHO Research Blueprint and its Global Coordinating Mechanism to provide a forum where those with the expertise and capacity to contribute to managing new threats can come together both between and during outbreaks to develop innovative solutions to emerging problems. This global coordination has been active in the novel coronavirus outbreak. WHO's response system includes three virtual groups based on those developed for SARS to collate real time information to inform real time guidelines, and a first candidate vaccine is ready for laboratory testing within 4 weeks of the virus being identified.","In addition to these methods, other smart city data sources include the application of terminal tracking systems that are mostly emphasized in Safe City concepts, where, at the point of entry or departure, relevant data is collected and analyzed. Li et al. [32] highlights that sensors installed in such locations have the potential to receive and distribute data in real-time to digital infrastructures within the network, and their interconnectedness in the network renders them extremely efficient in providing real-time updates on different issues. Urban areas are also known to be amassed with numerous Urban Health sensors, some of which are wearable. Though these are not specifically fashioned to track the present case of virus outbreak, they are able to track other related parameters like heartbeat, blood pressure, body temperature and others variables, that when analyzed can offer valuable insights. Loncar-Turukalo et al. [33] hail these devices for their role in transforming the health care sector especially by allowing for Connected Health (CH) care, where data collected from them can be analyzed and provide insightful information on the health scenario in any given area. Vashist et al. [34] further highlight how emerging features such as spatiotemporal mapping, remote monitoring and management, and enhanced cloud computing capabilities can emanate from such endeavours, leading to better urban management potential.","With the increasing emphasis on patient security and medical quality in healthcare, RFID technology has also been broadly implemented in healthcare facilities, such as medical equipment monitoring or real-time locating systems, anti-theft devices, identity tags for patients and medical personnel, drug anti-counterfeiting mechanisms (Wertheimer and Norris 2009). Most of these RFID applications are used for tracking and identity recognition to ensure that patients receive needed medical care and to support disease control (Kumar et al. 2009). Meanwhile, through visual images and RFID tags attached to predefined objects, healthcare personnel could also deliver home care services to dementia patients and have a better understanding of the patients\u2019 behavior (Wherton and Monk 2008). Besides, RFID could help to deal with events involving great complexity, such as monitoring and tracking patients\u2019 body temperatures, so much so that, in case of emergency, medical personnel could make professional judgement decisions immediately based on collected data (Yao et al. 2011). RFID can also be applied to medical inventory management. Without incurring any extra costs, RFID technology is capable of providing continuous review of inventory, which has been proven superior to the periodic review of barcode technology (\u00c7ak\u0131c\u0131 et al. 2011). In medication management, RFID has been used to enhance inpatient medication safety through a wireless network system for automated tracking of prescription drugs (Ohashi et al. 2010). With continuous, real-time information regarding the clinical situations and patients\u2019 location, hospital pharmacy personnel could differentiate batch number, dosage, and usage of drugs to reduce dispensing errors, while nurses could correctly identify patients, administer medication on time, and keep track of drug delivery routes (Peris-Lopez et al. 2011). ","Rapid dissemination of the 2019-nCoV is possible because of massive global travel networks. Based on limited data, the case fatality rate from 2019-nCoV appears to be 3-4%, lower than from SARS (about 10%). Global mortality and impact could be enormous if the virus spreads widely [2] . SARS was contained through intensive screening, isolation of those infected and quarantine of those exposed. This was effective because, for the most part, humans developed fever before they could transmit SARS-CoV. It remains to be determined if 2019-nCoV can be transmitted by people who have mild, pre-symptomatic or asymptomatic infections. Many unanswered questions remain. Diagnostics have been developed and work on therapeutics and vaccines is underway.","Harms. Organ failure, prolonged hospitalization and mortality will be recorded as part of the study defined outcomes. Other adverse events will be monitored by the site PI and research specialist in real time from the start of randomization to hospital discharge or death. Reportable adverse events will include hemodynamic change and deterioration of oxygenation related to NO inhalation and require the adjustment of NO concentration. A serious adverse event in this study will be defined as a reportable adverse event that is fatal, life-threatening, or permanently disabling. All adverse events will be indicated on the data forms for the study and on the specific adverse event report forms and all serious adverse events will be reported to the site IRB within 48 hours of the research team learning about the event followed by more detailed written report to the local IRB. The following information about adverse events will be collected: 1) the onset and resolution of the event, 2) an assessment of the severity or intensity of the event, 3) an assessment of the relationship of the event to the intervention, and 4) any action taken because of event. Sites will inform the principle investigators (CL and LB) at the coordinating centers (Xijing Hospital and Massachusetts General Hospital) for any related adverse events directly caused by inhalation of NO (non-serious) within 14 days of discovery. All fatal, life-threatening, or permanently disabling (regardless of relatedness) events due to inhalation of NO must be reported to the coordinating centers within 48 hours of discovery. The principle investigators will report these events to the DSMB and to the coordinating centers IRBs.","These samples, which had been stored at \u221270 \u2022 C, were retrieved for further testing. Sixty-three samples were found to be viable and were retested for the presence of other infectious agents using one of the following three techniques, depending on availability at the Fiocruz laboratory: RT-PCR for non-influenza respiratory viruses from a CDC-based kit, RespiFinder (RespiFinder 22 kit; Pathofinder B.V., The Netherlands), or the Fast-track (FTDRP 21 plus multiplex real-time RT-PCR assay; Fast-track Diagnostics Ltd, Sliema, Malta).","Notably, although the whole genome sequence of SARS-CoV-2 is almost identical, with a similarity rate of 99.9% (Ceraolo and Giorgi, 2020; Li et al., 2020; Lu et al., 2020; Paraskevis et al., 2020) , the virus begins to mutate in patients . Mutations of different SARS-CoV-2 isolates mainly occurs in five genes, including S, N, ORF8, ORF3a, and ORF1ab, with about 42% of the variations are non-synonymous mutation ). An increased level of viral diversity was found in some SARS-CoV-2 infected patients , suggesting that the virus has begun to adapt to the human environment and its genomes have begun to evolve in the population. Therefore, in-depth detection and tracking of the mutation and evolution of the virus in the population, as well as a cross-species transmission mechanism, are urgent tasks to be carried out at present.","With the possible expansion of 2019-nCoV globally [8] and the declaration of the 2019-nCoV outbreak as a Public Health Emergency of International Concern by the World Health Organization, there is an urgent need for rapid diagnostics, vaccines and therapeutics to detect, prevent and contain 2019-nCoV promptly. There is however currently a lack of understanding of what is available in the early phase of 2019-nCoV outbreak. The systematic review describes and assesses the potential rapid diagnostics, vaccines and therapeutics for 2019-nCoV, based in part on the developments for MERS-CoV and SARS-CoV.","Effective planning for pandemic influenza and the global vaccine supply will require the skills of experts in virology, epidemiology and vaccination. Equally important, however, will be the skills of experts in politics, economics and law. Coordinating their efforts at an international level will be very difficult. That the task must be undertaken cannot be questioned. Because the health benefits of its success are certain, the challenge it presents is extraordinarily compelling. 5.6. How good are the rapid diagnostic tests really? (Peter Wright, Vanderbilt University Medical Center, Nashville, TN and Ruth Karron, Johns Hopkins University, Baltimore, MD) Rapid diagnostic tests for respiratory viral illness has become an increasingly important tool because of the advent of effective antiviral agents that are maximally effective if started early in the course of the illness, the need to isolate patients who could pass their infection to other ill patients, and the need to differentiate between these viral illnesses and other respiratory tract pathogens, including bioterrorism agents. However, currently available rapid testing has many limitations. Areas of improvement that need to be addressed include the ability to test for all or most know agents, improved collection methods, improved integration of diagnosis with use of medications and clinical care, use of diagnostics for epidemiology and vaccine development, increased sensitivity in adults and immunocompromised patients, and improved ease and accuracy of testing.","The comparison of the spatial track of the patient's activities is a critical technical task for virus tracking and transmission chain reconstruction. The comparison of activity tracks between patients and populations provides an important scientific basis for delimiting the potentially infected population. Methods to quickly and automatically extract the patient's spatiotemporal trajectory from text data, establish the spatiotemporal comparison method, find the potential spatiotemporal ex-posure link of patients, support epidemiological investigation and rapid analysis, and realize automatic detection of the cross-regional epidemic infection path are major challenges for GIS. In this research, we 1) developed a reconstruction technology for the progress of the spatiotemporal events in a patient's text track data that can automatically convert the track text into quantitative spatiotemporal events; 2) established a spatiotemporal events database with more than 70000 pieces of patient track text covering the whole country; 3) constructed an exposure calculation model and a patient-node-patient association model that integrates time, space and text similarity, which showed the exposure assessment and site risk assessment of each individual (the exposure analysis of a patient's spatial trajectory is shown in Figure 5 .). Based on the above work, the key epidemic transmission centers were located, including Baodi Mall in Tianjin; Toulong Mall in Harbin, Heilongjiang; and Yintaishimao Mall in Wenzhou, Zhejiang.","Genomic epidemiology of emerging viruses has proven to be a useful tool for outbreak investigation and for tracking virus evolution and spread (6, 7) . During the Ebola virus disease epidemic of 2013-2016 in West Africa, genomic analyses established that the outbreak had a single zoonotic origin (8) , that two major viral lineages were circulating (9) , and that sexual transmission played a role in maintaining some transmission chains (10) . Viral genome sequencing also uncovered the route that Zika virus traveled from northern Brazil (11) to other regions, including Central America and Mexico (12) and the Caribbean and United States (13) . However, real-time genomic epidemiology data of COVID-19 to inform public health interventions in California have been lacking.","At present, complete genomes of different 2019-nCoV have been released, and the latest published information on these genomes provides peer-reviewed information that is urgently needed to improve the risk assessment and response, which occurs in real time (Chen et al., 2020b; Riou and Althaus, 2020) . The possibility of human-to-human transmission of the virus is still under investigation. Understanding the transmission characteristics of 2019-nCoV and the potential for sustained human-to-human transmission is critically important for coordinating current screening and containment strategies, and for determining whether the outbreak constitutes a public health emergency of international concern. The situation is severe and it is urgent for us to need a better understanding towards this virus for further prevention and control. In order to better understand the early transmission pattern of 2019-nCoV, Julien et al. conducted a randomized simulation study of the early outbreak trajectories consistent with the epidemiological findings to date (Riou and Althaus, 2020). They suggested that a rapid establishment of sustained transmission chains from single cases cannot be ruled out, and at this stage, particularly attention should be given to the prevention of superspreading events. Chen et al. reported a mathematical model for simulating the transmission of the novel Wuhan Coronavirus, which is a Bats-Hosts-Reservoir-People transmission network model for simulating the potential transmission from the infection source to the humans (Chen et al., 2020a) . In another study, a data-driven analysis in the early phase of the outbreak was conducted by Zhao et al. and a preliminary estimation of the basic reproduction number of 2019-nCoV was made. They estimated the transmissibility of 2019-nCoV via the basic reproduction number based on only the data from the early stages of the outbreak (Zhao et al., 2020a) .","The line list provides important information for the RCF outbreak control team about the number of affected residents, their symptoms and vaccination status, and can be used to summarise information each day. Additionally, the PHU may use a copy of the line list to track progress and inform advice provided to the RCF on management of the outbreak. An outbreak line list is cumulative; that is, it retains past information and is updated daily. It should include individual information such as:","In summary, the paper-based device has the potential to be used as a small, portable device to detect SARS-CoV-2 in wastewater on site and to track virus carriers in the community. Such an approach could provide near real-time and continuous data and serve as an early warning sensing system to help local governments and agencies make effective interventions to isolate potential virus carriers and prevent the spread of epidemics. We believe that in the case of asymptomatic infections in the community or people are not sure whether they are infected or not, rapid and real-time community sewage detection through paper analytical devices can determine whether there are SARS-CoV-2 carriers in the area in a timely manner to enable rapid screening, quarantine, and prevention. The potentially infected patient will also benefit from paper analytical device tracing SARS-CoV-2 sources with WBE, providing information for the correct and timely treatment of COVID-19.","The strict control of cross-infection in medical institutions is also key to preventing the further spread of the epidemic. In addition, actively mobilizing the population to participate in epidemic prevention and control is critical, and this can be achieved by accurate information dissemination and ongoing updates. Real-time updating the information of the epidemic situation can help to alleviate panic and reduce societal anxiety. A strong public health surveillance system to guide the response to the outbreak is important in the face of a lack of effective therapeutics or vaccines. Big data and artificial intelligence systems are tools that can be used to fully integrate the information, conduct comprehensive research and analyze public health risks. Last but not least, use of the Internet and the media to conduct health education for the people, publicize the correct knowledge of prevention, and offer online consultation to guide patients to seek medical treatment correctly are additional strategies that can be implemented to achieve viral containment.","MIoT can also be used for the prevention and control of COVID-19. We can establish a three-level linkage nCapp system based on the medical theory and technology of the IoT to diagnose and treat COVID-19. The IoT nCapp cloud medical system platform contains the basic functions of the IoT and has a core graphics processing unit (GPU). Cloud computing systems connected to existing electronic medical records, image archiving, and picture archiving and communication can better assist in deep mining and intelligent diagnosis. The ten functions of the IoT (Table 1 ) are considered beneficial time assistance, supervision, and control of medical quality. [15] [16] [17] [18] Among them, the functions of online monitoring, location tracking, alarm linkage, and follow-up scheduling are conducive to online discovery, monitoring, management, and treatment assistance for COVID-19. Plan management, remote maintenance, command management, and statistical decision-making functions can expand the massive information mining of COVID-19 and complete management and timely treatment of COVID-19 by applying preset guidelines or standardized criteria. Security privacy and online upgrade functions are nCapp guarantee that can ensure the normal operation of the IoT cloud plus terminal system. It can also assist in asking questions; registering patients' details; coordinating with patients, community doctors, and experts; and providing safe diagnosis treatment programs and two-way referrals (Fig. 1) .","The advances of sequencing technology, nanotechnology, and bioinformatics have driven molecular tests including assays for emerging pathogens to be more comprehensive and precise. For example, the availability of various sequence databases permits quick identification of sequence identity and variations. For example, the HIV database http://www.hiv.lanl.gov/ contains data on HIV genetic sequences and drug resistance associated mutations. It is valuable for HIV epidemiological studies, research, development, and clinical validation studies of HIV clinical assays [38, 71] . It is well known that there are significant variations of clinical phenotypes in the presence of emerging infections ranging from asymptomatic carrier to lethal infection. Recently, assays to examine multiple pathogen panels have been developed [97\u00c0102], which should increase the diagnostic yield for many pathogens. A critical need for emerging pathogen analysis is quicker, easier, cost-effect assays that can be used in a point-of-care setting. New assays that are performed on platforms with a small footprint and detect pathogens quickly (in minutes instead of hours or days) have entered clinical use. For example, the FilmArray (BioFire Diagnostics, Inc.) and Simplexa (Focus Diagnostics, Inc.) molecular assays can generate results in approximately 60 min. The user-friendly Alere i (Alere Inc.) and Cobas Liat (Roche Molecular Systems) platforms are compact and portable, generate rapid molecular results in 15\u00c020 min, can use electricity or rechargeable battery, and therefore are completely mobile and suited for point-of-care testing. It is obvious that the current rapid development of new technologies will further enhance the utility of molecular diagnostics in various emerging infectious diseases.","The ongoing outbreak of a novel coronavirus in China is now evident in 37 other countries as others employ vigilant screening procedures. 1 As of 24 Feb 2020, 119 SARS-CoV-2 genome sequences from 15 countries are publicly available from over 79,331 confirmed cases. 1,2 These viral genomes differ by only 0 to 5 mutations, likely representing a single introduction in the human population from an unknown reservoir or intermediate host, potentially bats. 3 Subsequently, the virus rapidly spread since December 2019. Currently, incubation period and transmissibility are questions of global concern, but difficult to answer given nonspecific and mild symptoms that could represent a variety of respiratory viruses. 4, 5 Genetic characterization of the virus from geographically diverse patient samples is key to infer the rate of spread. Additionally, rapid and complete results from cases worldwide are important for sequence-dependent countermeasures and accurate diagnostics. However, as the virus reaches resource-limited settings in proximity to the outbreak's epicenter, such as Cambodia, Laos, and Myanmar, there are often logistical challenges in surveillance, contact tracing, sample collection, and biospecimen transport. Information delay is also compounded by a dearth of sequencing and/or bioinformatics expertise in-country, further postponing analysis and dissemination of pathogen genomic information.","Experience gained from viral outbreaks over the past two decades has also better positioned the international response in terms of scope and speed. The SARS outbreak, which killed 774 people worldwide, and the 2012 emergence of the related coronavirus Middle East respiratory syndrome coronavirus (MERS-CoV) stimulated the development of a broad array of research tools, vaccine platforms, diagnostics and potential therapeutics that have yielded extensive scientific insights into coronavirus biology and are now being translated to the new virus. That progress, coupled with lessons learned from the outbreaks of Zika, Ebola and Chikungunya viruses, should enable rapid containment of 2019-nCoV.","Similar efforts are being made to understand the functional and evolutionary characteristics of the SARS-CoV-2 proteome, including the determination of evolutionary conserved functional regions between related viruses to inform the use of anti-viral therapeutics [4] . Given the unique infectivity characteristics of this novel coronavirus, the need for effective anti-viral therapeutics is pressing. The long viral incubation period, during which an individual is simultaneously contagious and asymptomatic, has resulted in rapid global proliferation. Leveraging what is known from the original SARS-CoV outbreak, circa. 2003, and related viral families, this work contributes predicted protein-protein interaction (PPI) networks to guide researchers and form the basis of testable wet laboratory hypotheses.","Suitable animal models for evaluating vaccines for SARS- and MERS-CoV are lacking or highly limited, making the process of vaccine development highly challenging.42,43 Development of an efficient animal model that mimics the clinical disease can inform on pathogenesis as well as to develop vaccines and therapeutics against these CoVs. Several animal models have been evaluated for SARS- and MERS CoVs including mouse, guinea pigs, hamsters, ferrets, rabbits, rhesus macaques, marmosets, and cats.42,44-50","This study specifically shows a holistic picture of the current research in response to the outbreak of COVID-19. We conducted a topic-based bibliometric study to address two questions \"What is the CoVs research interested in?\" and \"Where are we with our research about COVID-19?\". An LDA-based model was used to identify the hotspots and their difference between COVID-19 and other CoV infections in a quantitative way. Generally, eight main research areas are identified, i.e., clinical characterization, pathogenesis research, therapeutics research, epidemiological study, virus transmission, vaccines research, virus diagnostics, and viral genomics. The results also indicate that studies in diagnostics, therapeutics, vaccines, viral genomics, and pathogenesis are urgently needed to minimize the impact of the COVID-19 outbreak. As COVID-19 can be considered a recent emerged disease, we characterize a 'snapshot' of this field at an early stage in its development. Our findings can potentially aid governments and the research community in understanding recent development of the research field, optimizing research topic decision, and monitoring new scientific or technological activities.","Although a number of drugs have been evaluated for SARS-CoV and MERS-CoV there are no approved therapeutic agents available for coronaviruses. Integrating clinical trials of experimental therapeutics is an increasingly recognized part of the response during infectious disease outbreaks. Since the Ebola outbreak in West Africa and subsequent outbreaks in the Democratic Republic of Congo, clinical trials of investigative drugs have been fully integrated in the epidemic response. [5] [6] [7] The COVID-19 pandemic is unique because of its scale, the speed of its spread, the lack of pre-existing scientific data and the importance of media and scientific coverage. 2 To encourage the development of clinical trials that test therapeutics against SARS-CoV-2, the WHO has suggested a number of candidate antiviral agents that have to be tested in clinical trials. 8 Concomitantly, numerous clinical trials have been registered since the beginning of the COVID-19 outbreak numerous to evaluate therapeutic strategies for this disease. In the epidemic context, it is crucial for clinicians and researchers to have access to rapid and quality information on clinical trials that the various teams around the world are setting up. The results will inform about the antiviral agents that are used, their dosing and duration, the inclusion and exclusion criteria for patients, the outcomes that will be evaluated, as well as the design of the clinical trials. We believe that it is essential to carry out a review of these early phase clinical trials before the results are even available in order to best inform the teams wishing to test new therapies, choose therapeutic candidates or to design clinical trials.","Health officials in China exhibited a much higher degree of transparency early in this outbreak and were quicker to notify the WHO after they discovered the cluster of pneumonia of unknown origin. The identification of the aetiological virus occurred promptly in early January, and the full genome sequence data were shared on the Global Initiative on Sharing All Influenza Data platform. 8 The update International Health Regulations (2005) was signed 2 yr after the SARS outbreak by 194 member states, with legal binding on the protection of all people of the world from international spread of disease. 17 Global surveillance and alerting systems developed since SARS enabled rapid dissemination of information on the emerging outbreak. Public health officials and the WHO have held numerous press conferences to inform the public on important developments. Scientific papers have been published with rapid peer review to share the clinical and epidemiological characteristics of the virus within the medical community.","SNVs and quasispecies that are observed at low frequency could represent viral variations of low impact on the functionality of the genome. However, their abundance is largely affected by the population size and the epidemic characteristics. For example, a neutral substitution in a region that represents a primer target for a molecular diagnostic assay can drift to fixation rather quickly in a rapidly spreading virus, jeopardizing the sensitivity of the assay [38], [39] . Here, we highlight three intra-host but also two fixed variants that colocalized with primers or probes of real-time PCR diagnostics assays that are currently in use ( Figure   2 ). Since the alignment of these oligos with their genomic targets is directly linked to the performance of the corresponding diagnostic assays, the community should pay extra attention in the evaluation of these potentially emerging variations and be alerted, in case redesigning of these oligos is needed.","The swab samples were thawed and the cryovials containing the sample were vortexed. About 200 \u03bcL of the swab supernatant was used for nucleic acid extraction using InviMag\u00ae Pathogen Kit/KF 96 (STRATEC Molecular, Berlin, Germany) and the final eluted volume of nucleic acid solution was 200 \u03bcL, as per the manufacturer\u2019s instruction [17]. The real-time multiplex PCR assay was performed as per the manufacturer\u2019s instructions using an AgPath-ID \u2122 One-Step RT-PCR kit (Ambion) with the Fast Track Diagnostic (FTD) respiratory pathogens 33 kit (Fast Track Diagnostics, Luxembourg) for 19 different viruses and 12 bacterial pathogens [18]. Case-patients with at least one pathogen detected in their respiratory swab had corresponding surface swabs tested for those same pathogens. Detection of nucleic acid of at least one similar pathogen on respiratory swab and a nearby surface was defined as contamination of that surface. To investigate wider hospital contamination, we also tested the surface swabs collected near case-patients with no pathogens detected in their respiratory swabs for the most commonly detected pathogens identified on surfaces near patients with detected respiratory pathogens.","The copyright holder for this preprint . https://doi.org/10.1101/2020.03.26.20044164 doi: medRxiv preprint increasing rapidly, therapeutics research will be more and more urgently needed. In addition, virus diagnostics, vaccines and viral genomics show small proportions in all CoV infections research, but even smaller in the field of COVID-19. This shows how little we know about CoVs. It is time to translate research findings into effective measures such as a diagnostics, vaccine or effective therapeutic options, as with other priority diseases [22] .","Respiratory infections such as influenza and RSV continue to be of growing concern and new treatments are urgently needed. Janssen's respiratory program focuses on the development of vaccines, biologics, diagnostics and antiviral therapeutics for RSV and influenza patients.","The emergency pathogens could be counteracted by opting immediate and timely international collaborative efforts, cooperative efforts between human and animal health sectors. Other effective measures include One health approach, implementation of effective prevention and control strategies, rapid communication and networking, and exploring advances in science and technology for developing rapid and confirmatory diagnostics, enhancing disease surveillance and monitoring, implementation of strict biosecurity measures, and timely efforts toward designing appropriate and effective vaccines and therapeutics (Cheng et al. 2020; Cohen 2020; Cyranoski 2020; Lu 2020; Munjal et al. 2017; Singh et al. 2017).","The main question addressed by this study is whether the spread of ENIDs is able to be controlled and traced back quickly and efficiently. With the assistance of EDICTS, the answer is 'yes'. EDICTS suggests that an on-line full-time dedicated network is necessary for real-time data collection and transmission. In addition, a smart card as a medical identifier and storage media to provide a technical solution for accurate data processing and tracking patient pathways is also needed. The benefits from the proposed new outpatient procedure embedded in an on-line surveillance mechanism are:","The copyright holder for this preprint (which was not peer-reviewed) is the . https://doi.org/10.1101/2020.02.07.20021071 doi: medRxiv preprint utilized in this study provides a robust data source for estimating the likely dissemination of 2019-nCoV cases across the country (Fig. 3B) , providing important epidemiological information that is much less affected by delayed-or under-reporting bias. The management of the epidemic over the coming weeks when people travel back from their hometowns to their work places will be critical for being able to bring it under control. We recommend that the national and local authorities process real-time human movement data through mathematical models to forecast the spatio-temporal dynamics of further 2019-nCoV outbreaks across Mainland China. This will allow developing early detection, isolation and quarantine strategies tailored to the very dynamic epidemiological situation, as well as identify potential problems in the implementation of local disease control policies and inform the design of any necessary adjustments.","Matthew W. Gilmour (Diagnostic Services Manitoba, Winnipeg, Canada) discussed \"Real-time genomics for bacterial pathogen surveillance\". He considered the benefits and challenges represented by pathogen genomics to clinical and public health sectors. The speaker also underlined the potential of proteomics to facilitate the investigation of outbreaks. An integrated network coordinating the monitoring, assessment, investigation, outbreak response and recovery with standardized genomic analyses and unified laboratory workflows was envisioned.","The genome amplification and detection methods used (PCR) were initially mostly developed in situ and performed in biosafety level 3 (BSL-3) reference facilities. The time to results is generally relatively long, 24-48 h, due to the usual time required for conventional PCR testing to which must be added the additional preparation and sample neutralization time needed to protect the laboratory staff against this virus. The PCR methods used are generally semi-quantitative and some studies suggest a correlation between the amount of virus detected and the severity of the symptoms [26] . Nevertheless, no consensus has been reached yet regarding a threshold level that could actually predicts clinical severity. Targeting the envelop gene upE is recommended with confirmatory testing for ORF 1A or 1B or the N gene. If results diverge, sequencing is sometimes required to obtain conclusive results [27] . Today, an increasing number of commercial tests are becoming available (Altona Diagnostics, Fast Track Diagnostics, Primerdesing Ltd.) some even with a time to results of less than 1 hour (BioFire-bioM\u00e9rieux). Some of these tests are point-of-care, or can be performed in BSL3 facilities or a standard laboratory following sample neutralization in a BSL3 facility. These commercial tests must always be validated before use to check their sensitivity and compare their performance with reference methods.","Rapid diagnostics, vaccines and therapeutics are key pharmaceutical interventions to limit transmission of respiratory infectious diseases. Many potential developments on these pharmaceutical interventions for 2019-nCoV are ongoing in the containment phase of this outbreak, potentially due to better pandemic preparedness than before. However, lessons from MERS-CoV and SARS-CoV have shown that the journeys for these developments can still be challenging moving ahead.","WHO and the Foundation for Innovative New Diagnostics (FIND) have established a formal collaboration on universal access to essential diagnostics. Announced on 10 February, the aim of the collaboration is to close diagnostic gaps at country level and bolster disease surveillance that will inform public health initiatives and enhance outbreak preparedness and response.","Sepsis is one of the most urgent and costly systemic health threats and is a common pathway for mortality associated with novel pathogens. The current approach is to develop better diagnostics to identify and treat sepsis early, and understand the immune profile (activation versus suppression) in this disease. BARDA's Division of Research, Innovation, and Ventures (DRIVe) is establishing unique public-private partnerships that could accelerate therapeutics development for the identification and treatment of sepsis. Through this three-pronged approach (diagnostics, host targets, and better treatment of sepsis), BARDA hopes to decrease the risk of emerging respiratory viruses in the future.","For any outbreak, the faster the identification, the faster and more robust the response. BARDA is supporting work to improve diagnostics with the goal of recognizing illness before any symptoms develop. This may include wearable diagnostics or biosensors. The high mortality associated with human infections with novel coronaviruses MERS-CoV and SARS-CoV and avian influenza strains (influenza A(H5N1) and A(H7N9)) have raised questions about the possible role of a dysregulated immune response, a so called \"cytokine storm\", in the pathogenesis of severe respiratory disease. As some patients may not respond to typical anti-pathogen therapeutics (e.g. antivirals), BARDA is pursuing therapeutics which target the host in order to modify the immune/host response. More work needs to be done to identify these host targets and develop therapeutics towards the dysregulated immune response. The field also needs better diagnostics to guide treatment as over-suppression of the immune response can be just as dangerous as over-stimulation.","Molecular technologies have been critical in the initial discovery of agents of emerging infectious diseases. These methods have also been routinely used for further characterization of pathogen strains and sequence variations. Molecular data are now widely used in molecular epidemiological studies and phylogenetic analyses, and sequence comparisons have been performed to facilitate the specific detection of genetically diverse strains/sequences and investigate the origin, transmission, distribution, biology, and diversity of these pathogens [12,13,21,30\u00c033] , which are fundamentally important in the prevention and tracking of disease outbreaks. Knowledge of sequence variations is used in the development of accurate diagnostic assays and for the design of effective treatment strategies of diseases caused by these agents. Molecular epidemiological studies are critical for public health surveillance [14,15,34\u00c065] . We provide here examples of how molecular tests contributed to public health surveillance and patient care.","Specific to the need for greater coordination and acceleration of R&amp;D for vaccines, drugs, diagnostics, delivery systems, and other health technologies to avert a future epidemic when no existing medical countermeasures exists, the WHO published its \u201cPlan of Action\u201d for its R&amp;D Blueprint on May 2016 [51]. This document presents the preliminary strategy for the WHO R&amp;D Blueprint, as originally requested at the 68th WHA, and envisions a central convening and coordinating role for WHO in health R&amp;D, a concept supported in different forms by all of the panels [51]. Though the Plan of Action envisions an inclusive and collaborative global approach to tackle the lack of R&amp;D preparedness and access to treatment made evident during the EVD outbreak, it is unclear how this framework will be funded, implemented, and whether it will be aligned with existing governance structures and other related proposals. This includes existing structures such as the UN High-Panel on Access to Medicines and other financing and normative instruments currently being explored (including the WHO TDR Health Product Research &amp; Development Fund and a proposed Global biomedical R&amp;D treaty) [52].","A computer-aided ENID surveillance system is essentially a system to help CDC experts make right decisions by gathering information on ENID occurrences and spread, hence to control, track, and surveillance of ENID. In the last decade, substantial investments have been made in surveillance systems for early detection of natural infectious disease clusters and of intentional acts of bioterrorism. Previous studies demonstrate that surveillance is useful in detecting specified communicable diseases. Such systems include the sentinel provider system (USA) [12] , sentinel physician networks (European countries) [13] , influenza surveillance system (Japan) [14] , Australian sentinel practice research network (Australia) [15] , epidemiological surveillance networks (France) [16] , electronic point-of-sale (EPOS) pharmacy (England) [17] , infectious diseases surveillance system (Japan) [18] , ambulatory-care-based syndromic surveillance system (USA) [19] , real-time syndrome surveillance (Canada) [20] , and infectious disease prevention and control system (Germany) [21] . A categorization of computerized ENID surveillance systems by the timeliness of their response is shown in Fig. 1 . Most of these systems set the types of diseases under surveillance, and report the number of diseases covered, the geographical area covered, and the population covered. However, the time delay allowed is from 24 h to a week, except that [18, 20] , and the proposed EDICTS are on-line. There is generally one-way communication, i.e. from medical institutions to CDC, and only EDICTS is real-time and with automatic reporting, while [20] , for example, is done manually and by telephone. The implementation of an \"All-In-One\" Taiwanese NHI-IC card came into effect on 1st July 2003 in metropolitan and local community hospitals. The infrastructure was fully operational on a national scale and the first phase of implementation went into effect at the beginning of 2004. TBNHI constructed IDC to manage and maintain the NHICS database and a full-time bidirectional service network (the TBNHI intranet) for connecting all PoPs to IDC [22] . Moreover, the Taiwanese experience with SARS in 2003 proved that the health IC cards can play an important role in delivering solutions to healthcare providers and patients. The NHI-IC card carries a visually and electronically readable identifier to speed access to patient information. This leads to reductions in paper work, administrative costs and entry errors. The standard operations of using IC cards at the time of registration, at the doctor's side, and in the information infrastructure serve as a sound basis for better, faster delivery in the ENID reporting system. Moreover, the current practice of a surveillance system needs a real-time mechanism for quick spreading epidemic diseases. EDICTS is thus an appropriate mechanism that utilizes the NHICS infrastructure, in accordance with CDC policy, to deliver real-time response to ENID.","This work suggests that research needs to be focused on pathogenesis, therapeutics, virus diagnostics, vaccines and viral genomics for COVID-19, which could then necessitate nationallevel support for ongoing research, if required. Pathogenesis research is usually important but time-consuming. We believe such studies will be available in the future. The more important and urgent topic is therapeutics research, which accounts for only 9.68% in COVID-19 research, much lower than 18.88% in other CoV infections research. With the number of infection cases . CC-BY-NC-ND 4.0 International license It is made available under a is the author/funder, who has granted medRxiv a license to display the preprint in perpetuity.","Assuming that we test all residents experiencing common cold or flu symptoms which happens about 3 times per year (Arroll, 2011) and using the Medicare reimbursement rate of $51 as a measure of direct testing cost, the total testing costs of testing add up to approximately only $4 bln per month. Case tracking costs would be even smaller as long as infection rate remain in the predicted range. For example, hiring additional 1000 specialists to track cases with wages of $100,000 per year would cost less than $10 mln per month.","is the (which was not peer-reviewed) The copyright holder for this preprint . https://doi.org/10. 1101 compatible within the limits of the current knowledge about the outbreak. The implications of our work are that: 1) reductions in prevalence need not be attributable to successful interventions, but could be due to seasonal variation in transmissibility, 2) sub-population dynamics can differ greatly, meaning that case count trajectories in one country should be used cautiously to inform projections in a second country, even in the same climate zone, 3) seasonal variation might slow down a pandemic and thereby provide a window of opportunity for better preparation of health care systems world-wide by scaling up capacity for care and diagnostics, and potentially through rapid development of antivirals and vaccine, and 4) after several years SARS-CoV-2 could develop into an endemic seasonal CoV similar to the transition of the 2009 A/H1N1 pandemic influenza virus into a seasonal influenza virus.","This experience demonstrates the importance of 'classical' virological methods (cell culture, electron microscopy), which are 'catch-all' methods indispensible for detecting novel pathogens. Such methods should not be completely replaced by newer PCR based molecular diagnostics. The sharing of information in 'real-time' within the WHO laboratory network allowed rapid progress to be made in identifying the new pathogen, in establishing consensus, in validating reliable diagnostic tests to diagnose SARS and in disseminating credible information about the disease and its diagnosis.","In infectious outbreak situations, much epidemiological effort rightly goes into case-finding and follow-up in order to track epidemics. Population-based analyses of ecological factors are however also important, not least to inform models and prognostics for future spread of the same infection elsewhere. Where a new disease is involved, such as COVID-19, it is particularly important to chart an unknown infectious agent's interactions with environments in which transmission has occurred.","Tuberculosis disproportionately affects men and boys compared with women and girls. 9 Early data show that more men are dying from COVID-19, potentially due to sex-based immunological differences or gender-based factors such as prevalence of smoking. 10 The association between COVID-19 and poverty is also unclear but, as more data become available, we will be able to better understand the differential effects of COVID-19 according to socioeconomic position. COVID-19, like tuberculosis, will almost certainly be associated with the medical poverty trap, in which poorer people have a higher likelihood of infection, disease, and adverse outcomes. Moreover, unemployed populations and informal or so-called zero-hours contract workers will experience further impoverishment, which increases risk of tuberculosis. 5 Amid the expanding COVID-19 pandemic, our plea on World Tuberculosis Day is that we do not forget the tuberculosis pandemic, which, at present, is still the leading cause of infectious disease mortality. We need to continue to mobilise funding for research for better tuberculosis diagnostics, vaccine development, novel therapeutics, equitable access to care, and innovative social protection interventions for tuberculosis-affected households. 5 We should drastically increase and sustain investment in health systems that are responsive to the needs of the poor and resilient to the threat of infections, especially those that are air-borne and require isolation facilities. We need to continue to inform, advocate for, and empower local communities and to lobby governments and policymakers to ensure that tuberculosis, as well as COVID-19, remain high on the global agenda. These two pandemics, one old and one new, remind us of the need to be proactive and longsighted, to plan ahead, and to not become complacent.","Viral pathogenesis deals with the interaction between a virus and its host. Included within the scope of pathogenesis are the stepwise progression of infection from virus entry through dissemination to shedding, the defensive responses of the host, and the mechanisms of virus clearance or persistence. Pathogenesis also encompasses the disease processes that result from infection, variations in viral pathogenicity, and the genetic basis of host resistance to infection or disease. A subject this broad cannot be treated in a single entry, and this article focuses on the dissemination of viruses and their pathogenicity.","In this study, we analyzed the spatial representation of the disease, material, population and social psychology at three scales: individual, JID: GEOSUS [m5GeSdc; March 28, 2020; 14:39 ] group and regional. At the individual scale, the comparison between spatial epidemic tracking and the spatiotemporal trajectories of patients was carried out. At the group scale, the estimation of population flow and the spatial distribution was carried out. At the regional scale, the segmentation of spatial risk, the analysis of balance between the supply and demand of medical resources, and the spatial differentiation analysis of material transportation capacity and social sentiment were carried out. From the perspective of GIS technology, this study revolutionized the data acquisition methods through big data technology, which achieved rapid data acquisition and integration from traditional data to big data of various organizations. The analysis platform was quickly constructed through an innovative construction technology system, which provided the technical platform for timely epidemic analysis. The production of epidemic maps was quickly completed through multi-scale dynamic template technology, which allowed timely dissemination of epidemic dynamic information. From the perspective of spatial simulation and analysis, this study simulated the spatial transmission process of the epidemic scientifically by increasing detailed regional variables and changing the population flow and R 0 (basic reproduction number) according to the prevention and control measures. The calculation of the overlap between the spatial tracking of the virus and that of patient trajectories was realized based on the exposure index of fused spatial and text information via text spatialization technology. The estimation of population flow and spatial distribution was realized via the combination of big data and traditional geographic data, which identified the problems in the key risk areas and the spatial mismatching of medical resources in a timely manner, to bring about a rapid supply of information about the delimitation of prevention areas and resource deployment in epidemic control. The monitoring of social sentiment from social media was realized through the construction of a knowledge database on public opinion, which provided important foundational information on public opinion to guide the government.","This study suggests that the current reporting of COVID-19 cases is significantly underestimating the true scale of the pandemic. The lack of testing makes the estimation of the true death rate difficult and causes a significant misinformation. This study tries to leverage the information derived from a well-tested sub-population (South Korea). With testing capacities of 20,000 tests daily, it has the largest and most accurate coverage compared to all other countries as of writing. The low false-negative rate in detecting COVID-19 infections leads to the lowest death rate compared to all other countries (0.84) with major case count. By applying the parameters, estimated from this benchmark country, the proposed method can adjust global COVID-19 case numbers. This method is limited in its ability to predict the exact number of cases accurately. The method relies on the assumption that deaths by COVID-19 are detected and reported reliably. False-negative rates can have a distorting effect on the case adjustment. This is especially true if the benchmark country does not adequately report deaths from COVID-19. Germany, as an example, only reports eight deaths from with 3,675 reported cases. This could be due to the very recent increase in actual cases leaving not enough time for fatal disease progression. Over time, when more data is available, death rates will most likely increase in Germany. Additionally, the assumption of a globally similar death rate is untested. Improvements in this method could look at the case number of other viral diseases to see if there are significant differences between countries. This method explains the observed fluctuations in death rate over time by country. It is unlikely that the death rate in the same country can fluctuate by multiple percent points over a period of a few days. This method suggests that due to the fast exponential growth of true case counts, most modern healthcare systems are not able to track the changes adequately. In addition, the method suggests that computational tools can be used to impute missing information based on regions where testing and tracking is more advanced. It also highlights the importance of publicly accessible real time data and the relevance of combining global healthcare efforts. . CC-BY 4.0 International license It is made available under a author/funder, who has granted medRxiv a license to display the preprint in perpetuity.","On 29 December 2019 clinicians in a hospital in Wuhan City, China noticed a clustering of cases of unusual pneumonia (with the first case identified at that time on 12 December) with an apparent link to a market that sells live fish, poultry and animals to the public. This event was reported to the World Health Organisation (WHO) on 31 December [1]. Within 4 weeks, by 26 January 2020, the causative organism had been identified as a novel coronavirus, the genome of the virus had been sequenced and published, reverse transcription polymerase chain reaction tests had been developed, the WHO R&amp;D Blueprint had been activated to accelerate diagnostics, therapeutics and vaccine development and a candidate vaccine was ready for initial laboratory testing. Currently Chinese health authorities are building a 1000 bed hospital in Wuhan in 10 days."],"title":["SARS to novel coronavirus \u2013 old lessons and new lessons","On the Coronavirus (COVID-19) Outbreak and the Smart City Network: Universal Data Sharing Standards Coupled with Artificial Intelligence (AI) to Benefit Urban Health Monitoring and Management","The construction of a hospital disease tracking and control system with a disease infection probability model","What goes on board aircraft? Passengers include Aedes, Anopheles, 2019-nCoV, dengue, Salmonella, Zika, et al","Protocol of a randomized controlled trial testing inhaled Nitric Oxide in mechanically ventilated patients with severe acute respiratory syndrome in COVID-19 (SARS-CoV-2).","Viral detection profile in children with severe acute respiratory infection","Genetic evolution analysis of 2019 novel coronavirus and coronavirus from other species","Potential Rapid Diagnostics, Vaccine and Therapeutics for 2019 Novel Coronavirus (2019-nCoV): A Systematic Review","Current research on respiratory viral infections: Fifth International Symposium","COVID-19: Challenges to GIS with Big Data","A Genomic Survey of SARS-CoV-2 Reveals Multiple Introductions into Northern California without a Predominant Lineage","Puzzle of highly pathogenic human coronaviruses (2019-nCoV)","Guidelines for preventing respiratory illness in older adults aged 60 years and above living in long-term care: A rapid review of clinical practice guidelines","Can a Paper-Based Device Trace COVID-19 Sources with Wastewater-Based Epidemiology?","The deadly coronaviruses: The 2003 SARS pandemic and the 2020 novel coronavirus epidemic in China","Chinese experts\u2019 consensus on the Internet of Things-aided diagnosis and treatment of coronavirus disease 2019 (COVID-19)","Chapter 15 Molecular Testing in Emerging Infectious Diseases","Rapid metagenomic characterization of a case of imported COVID-19 in Cambodia","Communication, collaboration and cooperation can stop the 2019 coronavirus","Computational Prediction of the Comprehensive SARS-CoV-2 vs. Human Interactome to Guide the Design of Therapeutics","COVID-19, an emerging coronavirus infection: advances and prospects in designing and developing vaccines, immunotherapeutics, and therapeutics","Understand Research Hotspots Surrounding COVID-19 and Other Coronavirus Infections Using Topic Modeling","A brief review of antiviral drugs evaluated in registered clinical trials for COVID-19","Outbreak of a new coronavirus: what anaesthetists should know","SARS-CoV-2 exhibits intra-host genomic plasticity and low-frequency polymorphic quasispecies","Contamination of hospital surfaces with respiratory pathogens in Bangladesh","Understand Research Hotspots Surrounding COVID-19 and Other Coronavirus Infections Using Topic Modeling","Advances in respiratory virus therapeutics \u2013 A meeting report from the 6th isirv Antiviral Group conference","Emerging novel coronavirus (2019-nCoV)\u2014current scenario, evolutionary perspective based on genome analysis and recent developments","Design and prototype of a mechanism for active on-line emerging/notifiable infectious diseases control, tracking and surveillance, based on a national healthcare card system","Incorporating Human Movement Data to Improve Epidemiological Estimates for 2019-nCoV","Microbial molecular markers and epidemiological surveillance in the era of high throughput sequencing: an update from the IMMEM-10 conference","Focus on Middle East respiratory syndrome coronavirus (MERS-CoV)","Potential Rapid Diagnostics, Vaccine and Therapeutics for 2019 Novel Coronavirus (2019-nCoV): A Systematic Review","Public health round-up","Advances in respiratory virus therapeutics \u2013 A meeting report from the 6th isirv Antiviral Group conference","Advances in respiratory virus therapeutics \u2013 A meeting report from the 6th isirv Antiviral Group conference","Chapter 15 Molecular Testing in Emerging Infectious Diseases","The Ebola Outbreak: Catalyzing a \u201cShift\u201d in Global Health Governance?","Design and prototype of a mechanism for active on-line emerging/notifiable infectious diseases control, tracking and surveillance, based on a national healthcare card system","Understand Research Hotspots Surrounding COVID-19 and Other Coronavirus Infections Using Topic Modeling","Inclusive Costs of NPI Measures for COVID-19 Pandemic: Three Approaches","Potential impact of seasonal forcing on a SARS-CoV-2 pandemic","57 Coronaviruses","Eco-epidemiological assessment of the COVID-19 epidemic in China, January-February 2020","Tackling two pandemics: a plea on World Tuberculosis Day","Viral Membranes","COVID-19: Challenges to GIS with Big Data","Correcting under-reported COVID-19 case numbers","SARS to novel coronavirus \u2013 old lessons and new lessons"],"url":["https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7026896/","https://doi.org/10.3390/healthcare8010046","https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7087954/","https://doi.org/10.1016/j.tmaid.2020.101572","https://doi.org/10.1101/2020.03.09.20033530","https://doi.org/10.1016/j.bjid.2018.09.001","https://doi.org/10.1016/j.meegid.2020.104285","https://doi.org/10.3390/jcm9030623","https://doi.org/10.1016/j.antiviral.2003.12.006","https://doi.org/10.1016/j.geosus.2020.03.005","https://doi.org/10.1101/2020.03.27.20044925","https://doi.org/10.1007/s13238-020-00693-y","https://doi.org/10.1101/2020.03.19.20039180","https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7099676/","https://doi.org/10.1016/j.jaut.2020.102434","https://doi.org/10.1016/j.ceh.2020.03.001","https://doi.org/10.1016/b978-0-12-800886-7.00015-7","https://doi.org/10.1101/2020.03.02.968818","https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7095972/","https://doi.org/10.1101/2020.03.29.014381","https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7103671/","https://doi.org/10.1101/2020.03.26.20044164","https://doi.org/10.1101/2020.03.18.20038190","https://doi.org/10.1016/j.bja.2020.02.008","https://doi.org/10.1101/2020.03.27.009480","https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6816543/","https://doi.org/10.1101/2020.03.26.20044164","https://doi.org/10.1016/j.antiviral.2019.04.006","https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7054940/","https://doi.org/10.1016/j.cmpb.2007.02.005","https://doi.org/10.1101/2020.02.07.20021071","https://doi.org/10.1016/j.resmic.2014.01.003","https://doi.org/10.1016/j.medmal.2019.10.004","https://doi.org/10.3390/jcm9030623","https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7047037/","https://doi.org/10.1016/j.antiviral.2019.04.006","https://doi.org/10.1016/j.antiviral.2019.04.006","https://doi.org/10.1016/b978-0-12-800886-7.00015-7","https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5121963/","https://doi.org/10.1016/j.cmpb.2007.02.005","https://doi.org/10.1101/2020.03.26.20044164","https://doi.org/10.1101/2020.03.26.20044552","https://doi.org/10.1101/2020.02.13.20022806","https://doi.org/10.1016/b978-0-7020-4089-4.00072-x","https://doi.org/10.1101/2020.03.29.20046565","https://doi.org/10.1016/s2213-2600(20)30151-x","https://doi.org/10.1016/b978-012374410-4.00530-6","https://doi.org/10.1016/j.geosus.2020.03.005","https://doi.org/10.1101/2020.03.14.20036178","https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7026896/"],"x":[0.6869758483031994,0.5285839986645121,0.8588000961777149,0.6868574238082588,0.425487723395792,0.4989790406470221,0.8600602229366905,0.7725159791197534,0.43560679354208043,0.45481593563668254,0.620158504952292,0.9076873600944783,0.5298172738585122,0.6682206330561893,0.533574168530104,0.3772619742011261,0.46481774099682654,0.6153333151799181,0.7678947999508836,0.6668632029236118,0.8286141280592714,0.43546754135816496,0.262173665030212,0.523052017456161,0.48041440220922316,0.4984739484087618,0.667771701569111,0.5359600710859429,0.8765844008170718,0.5455175237056545,0.6801459847040818,0.4830481396423187,0.48554487630889476,0.8001311091854514,0.5034827827759368,0.3276360619081397,0.7227371014368541,0.5027251300974446,0.49782054764233447,0.5102179448351762,0.39936777643754734,0.6215295351241192,0.5416241839863783,0.47529404551724824,0.5547966742556154,0.4536640643889551,0.5990735569139058,0.5467829582855449,0.6306702937577432,0.775749403473536],"y":[0.5664521431681786,0.8135955654148591,0.44767942476859984,0.601581241933511,0.5983108611053601,0.36622289664455293,0.3274188510503432,0.5415403549578632,0.626891941545845,0.6484562003427512,0.5491264945993148,0.4123550409675776,0.659997274188331,0.5646763619041669,0.7499216436315376,0.8367791908215415,0.415405802645114,0.4422854290890841,0.624853763772978,0.3410258273496889,0.49194485635739144,0.8301877451387424,0.5671280641093518,0.7816584178347079,0.4012271854961696,0.3960100605200574,0.15386317607783334,0.536320623004385,0.40954263577893296,0.8490662789052851,0.1867113401232449,0.722549627452965,0.3766140392540208,0.5320259682460985,0.726613621041017,0.5164992477100744,0.3990915974906302,0.7078001570307583,0.7407572957208673,0.696060803365547,0.16742095377946764,0.6929047158146933,0.5833734041871566,0.4064537314803543,0.7020392056098324,0.8288735494052666,0.4886493401800102,0.8147566224128853,0.774820430850713,0.8007762053290066]},"selected":{"id":"8159"},"selection_policy":{"id":"8188"}},"id":"8086","type":"ColumnDataSource"},{"attributes":{"axis":{"id":"8131"},"ticker":null},"id":"8134","type":"Grid"},{"attributes":{},"id":"8105","type":"ResetTool"},{"attributes":{},"id":"8188","type":"UnionRenderers"},{"attributes":{"source":{"id":"8085"}},"id":"8121","type":"CDSView"},{"attributes":{"children":[{"id":"8163"}]},"id":"8167","type":"Row"},{"attributes":{},"id":"8092","type":"LinearScale"},{"attributes":{},"id":"8097","type":"BasicTicker"},{"attributes":{"fill_alpha":{"value":0.1},"fill_color":{"field":"color"},"line_alpha":{"value":0.1},"line_color":{"field":"color"},"radius":{"field":"rad","units":"data"},"x":{"field":"x"},"y":{"field":"y"}},"id":"8148","type":"Circle"},{"attributes":{},"id":"8129","type":"LinearScale"},{"attributes":{"formatter":{"id":"8193"},"ticker":{"id":"8132"}},"id":"8131","type":"LinearAxis"},{"attributes":{},"id":"8094","type":"LinearScale"},{"attributes":{"data_source":{"id":"8085"},"glyph":{"id":"8112"},"hover_glyph":null,"muted_glyph":null,"nonselection_glyph":{"id":"8113"},"selection_glyph":null,"view":{"id":"8115"}},"id":"8114","type":"GlyphRenderer"},{"attributes":{},"id":"8187","type":"StringEditor"},{"attributes":{"sizing_mode":"stretch_width","text":"&lt;h2&gt;Real-time tracking of whole genomes and a mechanism for coordinating the rapid dissemination of that information to inform the development of diagnostics and therapeutics and to track variations of the virus over time.&lt;/h2&gt;"},"id":"8161","type":"Div"},{"attributes":{"editor":{"id":"8177"},"field":"title","formatter":{"id":"8176"},"title":"Document title","width":770},"id":"8119","type":"TableColumn"},{"attributes":{"callback":null,"tooltips":[["Title","@title"],["Passage","@text"],["Published","@date"],["Authors","@authors"],["Journal","@journal"],["URL","@url"]]},"id":"8141","type":"HoverTool"},{"attributes":{},"id":"8127","type":"LinearScale"},{"attributes":{"children":[{"id":"8162"}]},"id":"8165","type":"Row"},{"attributes":{"args":{"sd":{"id":"8085"},"sp":{"id":"8086"}},"code":"\n    if (sd.selected.indices.length == 0) {\n      return;\n    }\n    var sel = sd.selected.indices[0];\n    var new_selected = [];\n    for (var i = 0; i &lt; sp.data['doc'].length; i++) {\n      if (sp.data['doc'][i] == sd.data['doc'][sel]) {\n        new_selected.push(i);\n      }\n    }\n    if (JSON.stringify(new_selected) != JSON.stringify(sp.selected.indices) &amp;&amp;\n    !(sp.selected.indices.length == 1 &amp;&amp; new_selected.includes(sp.selected.indices[0]))) {\n      sp.selected.indices = new_selected;\n      sp.change.emit();\n    }\n    "},"id":"8158","type":"CustomJS"},{"attributes":{"children":[{"id":"8120"},{"id":"8087"}]},"id":"8166","type":"Row"},{"attributes":{"fill_alpha":{"value":0.1},"fill_color":{"field":"color"},"line_alpha":{"value":0.1},"line_color":{"field":"color"},"radius":{"field":"rad","units":"data"},"x":{"field":"x"},"y":{"field":"y"}},"id":"8113","type":"Circle"},{"attributes":{},"id":"8125","type":"Range1d"},{"attributes":{},"id":"8175","type":"StringEditor"},{"attributes":{"js_property_callbacks":{"change:indices":[{"id":"8158"}]}},"id":"8157","type":"Selection"},{"attributes":{"fill_color":{"field":"color"},"line_color":{"field":"color"},"radius":{"field":"rad","units":"data"},"x":{"field":"x"},"y":{"field":"y"}},"id":"8112","type":"Circle"},{"attributes":{},"id":"8186","type":"StringFormatter"},{"attributes":{"js_property_callbacks":{"change:indices":[{"id":"8160"}]}},"id":"8159","type":"Selection"},{"attributes":{"editor":{"id":"8185"},"field":"scr","formatter":{"id":"8184"},"title":"Score","width":30},"id":"8153","type":"TableColumn"},{"attributes":{"text":""},"id":"8172","type":"Title"},{"attributes":{"source":{"id":"8086"}},"id":"8156","type":"CDSView"},{"attributes":{"columns":[{"id":"8153"},{"id":"8154"}],"height":340,"source":{"id":"8086"},"view":{"id":"8156"},"width":900},"id":"8155","type":"DataTable"},{"attributes":{},"id":"8185","type":"StringEditor"},{"attributes":{},"id":"8183","type":"BasicTickFormatter"},{"attributes":{"editor":{"id":"8187"},"field":"text","formatter":{"id":"8186"},"title":"Passage","width":770},"id":"8154","type":"TableColumn"},{"attributes":{"data_source":{"id":"8086"},"glyph":{"id":"8147"},"hover_glyph":null,"muted_glyph":null,"nonselection_glyph":{"id":"8148"},"selection_glyph":null,"view":{"id":"8150"}},"id":"8149","type":"GlyphRenderer"},{"attributes":{},"id":"8181","type":"BasicTickFormatter"},{"attributes":{"source":{"id":"8086"}},"id":"8150","type":"CDSView"},{"attributes":{},"id":"8193","type":"BasicTickFormatter"},{"attributes":{"active_drag":"auto","active_inspect":"auto","active_multi":null,"active_scroll":"auto","active_tap":"auto","tools":[{"id":"8139"},{"id":"8140"},{"id":"8141"}]},"id":"8142","type":"Toolbar"},{"attributes":{"callback":null},"id":"8104","type":"TapTool"},{"attributes":{"text":""},"id":"8170","type":"Title"},{"attributes":{"axis":{"id":"8096"},"ticker":null},"id":"8099","type":"Grid"},{"attributes":{"args":{"sd":{"id":"8085"},"sp":{"id":"8086"}},"code":"\n    if (sp.selected.indices.length == 0) {\n      return;\n    }\n    var sel = sp.selected.indices[0];\n    var new_selected = [];\n    for (var i = 0; i &lt; sd.data['doc'].length; i++) {\n      if (sd.data['doc'][i] == sp.data['doc'][sel]) {\n        new_selected.push(i);\n      }\n    }\n    if (JSON.stringify(new_selected) != JSON.stringify(sd.selected.indices) &amp;&amp;\n    !(sd.selected.indices.length == 1 &amp;&amp; new_selected.includes(sd.selected.indices[0]))) {\n      sd.selected.indices = new_selected;\n      sd.change.emit();\n    }\n    "},"id":"8160","type":"CustomJS"},{"attributes":{"text":"&lt;h3&gt;Paragraph ranking&lt;/h3&gt;"},"id":"8163","type":"Div"},{"attributes":{"editor":{"id":"8175"},"field":"scr","formatter":{"id":"8174"},"title":"Score","width":30},"id":"8118","type":"TableColumn"},{"attributes":{"formatter":{"id":"8183"},"ticker":{"id":"8097"}},"id":"8096","type":"LinearAxis"},{"attributes":{"children":[{"id":"8164"},{"id":"8165"},{"id":"8166"},{"id":"8167"},{"id":"8168"}]},"id":"8169","type":"Column"},{"attributes":{"axis":{"id":"8100"},"dimension":1,"ticker":null},"id":"8103","type":"Grid"},{"attributes":{},"id":"8191","type":"BasicTickFormatter"},{"attributes":{"children":[{"id":"8161"}]},"id":"8164","type":"Row"},{"attributes":{},"id":"8090","type":"Range1d"},{"attributes":{"columns":[{"id":"8118"},{"id":"8119"}],"height":340,"source":{"id":"8085"},"view":{"id":"8121"},"width":900},"id":"8120","type":"DataTable"},{"attributes":{},"id":"8101","type":"BasicTicker"}],"root_ids":["8169"]},"title":"Bokeh Application","version":"2.0.1"}}
        </script>
        <script type="text/javascript">
          (function() {
            var fn = function() {
              Bokeh.safely(function() {
                (function(root) {
                  function embed_document(root) {
                    
                  var docs_json = document.getElementById('8406').textContent;
                  var render_items = [{"docid":"eb6e31ac-8a95-4ae7-affb-45523518872a","root_ids":["8169"],"roots":{"8169":"9237dc14-cbc2-48af-adaa-c4793fa375f6"}}];
                  root.Bokeh.embed.embed_items(docs_json, render_items);
                
                  }
                  if (root.Bokeh !== undefined) {
                    embed_document(root);
                  } else {
                    var attempts = 0;
                    var timer = setInterval(function(root) {
                      if (root.Bokeh !== undefined) {
                        clearInterval(timer);
                        embed_document(root);
                      } else {
                        attempts++;
                        if (attempts > 100) {
                          clearInterval(timer);
                          console.log("Bokeh: ERROR: Unable to run BokehJS code because BokehJS library is missing");
                        }
                      }
                    }, 10, root)
                  }
                })(window);
              });
            };
            if (document.readyState != "loading") fn();
            else document.addEventListener("DOMContentLoaded", fn);
          })();
        </script>
    
  </body>
  
</html>