



<!DOCTYPE html>
<html lang="en">
  
  <head>
    
      <meta charset="utf-8">
      <title>Bokeh Plot</title>
      
      
        
          
        
        
          
        <script type="text/javascript" src="https://cdn.bokeh.org/bokeh/release/bokeh-2.0.1.min.js" integrity="sha384-JpP8FXbgAZLkfur7LiK3j9AGBhHNIvF742meBJrjO2ShJDhCG2I1uVvW+0DUtrmc" crossorigin="anonymous"></script>
        <script type="text/javascript" src="https://cdn.bokeh.org/bokeh/release/bokeh-widgets-2.0.1.min.js" integrity="sha384-xZlADit0Q04ISQEdKg2k3L4W9AwQBAuDs9nJL9fM/WwzL1tEU9VPNezOFX0nLEAz" crossorigin="anonymous"></script>
        <script type="text/javascript" src="https://cdn.bokeh.org/bokeh/release/bokeh-tables-2.0.1.min.js" integrity="sha384-4BuPRZkdMKSnj3zoxiNrQ86XgNw0rYmBOxe7nshquXwwcauupgBF2DHLVG1WuZlV" crossorigin="anonymous"></script>
        <script type="text/javascript">
            Bokeh.set_log_level("info");
        </script>
        
      
      
    
  </head>
  
  
  <body>
    
      
        
          
          
            
              <div class="bk-root" id="9115fd91-1a0d-4e5e-b323-f38bf5e9d594" data-root-id="6881"></div>
            
          
        
      
      
        <script type="application/json" id="7118">
          {"48122c88-4510-4592-b8fb-bd0fc99b6c17":{"roots":{"references":[{"attributes":{"data":{"authors":["Lauren Tindale; Michelle Coombe; Jessica E Stockdale; Emma Garlock; Wing Yin Venus Lau; Manu Saraswat; Yen-Hsiang Brian Lee; Louxin Zhang; Dongxuan Chen; Jacco Wallinga; Caroline Colijn","Shujuan Ma; Jiayue Zhang; Minyan Zeng; Qingping Yun; Wei Guo; Yixiang Zheng; Shi Zhao; Maggie H Wang; Zuyao Yang","Cowling, Benjamin J; Leung, Gabriel M","Tapiwa Ganyani; Cecile Kremer; Dongxuan Chen; Andrea Torneri; Christel Faes; Jacco Wallinga; Niel Hens","Pung, Rachael; Chiew, Calvin J; Young, Barnaby E; Chin, Sarah; Chen, Mark I-C; Clapham, Hannah E; Cook, Alex R; Maurer-Stroh, Sebastian; Toh, Matthias P H S; Poh, Cuiqin; Low, Mabel; Lum, Joshua; Koh, Valerie T J; Mak, Tze M; Cui, Lin; Lin, Raymond V T P; Heng, Derrick; Leo, Yee-Sin; Lye, David C; Lee, Vernon J M; Kam, Kai-qian; Kalimuddin, Shirin; Tan, Seow Yen; Loh, Jiashen; Thoon, Koh Cheng; Vasoo, Shawn; Khong, Wei Xin; Suhaimi, Nur-Afidah; Chan, Sherlynn JH; Zhang, Emma; Oh, Olivia; Ty, Albert; Tow, Charlene; Chua, Yi Xian; Chaw, Wei Liang; Ng, Yixiang; Abdul-Rahman, Farid; Sahib, Shafiq; Zhao, Zheng; Tang, Cheryl; Low, Constance; Goh, Ee Hui; Lim, Georgina; Hou, Yan'an; Roshan, Imran; Tan, James; Foo, Kelly; Nandar, Khine; Kurupatham, Lalitha; Chan, Pei Pei; Raj, Pream; Lin, Yijun; Said, Zubaidah; Lee, Anne; See, Cherie; Markose, Jessey; Tan, Joanna; Chan, Guanhao; See, Wanhan; Peh, Xinyi; Cai, Vincent; Chen, Wen Kai; Li, Zongbin; Soo, Roy; Chow, Angela LP; Wei, Wycliffe; Farwin, Aysha; Ang, Li Wei","Wei Xia; Jiaqiang Liao; Chunhui Li; Yuanyuan Li; Xi Qian; Xiaojie Sun; Hongbo Xu; Gaga Mahai; Xin Zhao; Lisha Shi; Juan Liu; Ling Yu; Meng Wang; Qianqian Wang; Asmagvl Namat; Ying Li; Jingyu Qu; Qi Liu; Xiaofang Lin; Shuting Cao; Shu Huan; Jiying Xiao; Fengyu Ruan; Hanjin Wang; Qing Xu; Xingjuan Ding; Xingjie Fang; Feng Qiu; Jiaolong Ma; Yu Zhang; Aizhen Wang; Yuling Xing; Shunqing Xu","Hiroshi Nishiura; Natalie M Linton; Andrei R. Akhmetzhanov","Nishiura, Hiroshi; Linton, Natalie M.; Akhmetzhanov, Andrei R.","Kaike Ping","Christian Alvin H Buhat; Monica C Torres; Yancee H Olave; Maica Krizna A Gavina; Edd Francis O Felix; Gimelle B Gamilla; Kyrell Vann B Verano; Ariel L Babierra; Jomar Fajardo Rabajante","DJ Darwin R Bandoy; Bart C Weimer","Kim, Yunhwan; Lee, Sunmi; Chu, Chaeshin; Choe, Seoyun; Hong, Saeme; Shin, Youngseo","Juanjuan Zhang; Maria Litvinova; Wei Wang; Yan Wang; Xiaowei Deng; Xinghui Chen; Mei Li; Wen Zheng; Lan Yi; Xinhua Chen; Qianhui Wu; Yuxia Liang; Xiling Wang; Juan Yang; Kaiyuan Sun; Ira M. Longini; M. Elizabeth Halloran; Peng Wu; Benjamin J. Cowling; Stefano Merler; Cecile Viboud; Alessandro Vespignani; Marco Ajelli; Hongjie Yu","Sang Woo Park; Benjamin M. Bolker; David Champredon; David J.D. Earn; Michael Li; Joshua S. Weitz; Bryan T. Grenfell; Jonathan Dushoff","Qingyuan Zhao; Yang Chen; Dylan S Small","Zhanwei Du; Xiaoke Xu; Ye Wu; Lin Wang; Benjamin J Cowling; Lauren Ancel Meyers","Zhang, Sheng; Diao, MengYuan; Yu, Wenbo; Pei, Lei; Lin, Zhaofen; Chen, Dechang","Pengpeng Shi; Shengli Cao; Peihua Feng","Yuke Wang; Peter F.M. Teunis","Qifang Bi; Yongsheng Wu; Shujiang Mei; Chenfei Ye; Xuan Zou; Zhen Zhang; Xiaojian Liu; Lan Wei; Shaun A Truelove; Tong Zhang; Wei Gao; Cong Cheng; Xiujuan Tang; Xiaoliang Wu; Yu Wu; Binbin Sun; Suli Huang; Yu Sun; Juncen Zhang; Ting Ma; Justin Lessler; Teijian Feng","Sang Woo Park; Daniel M Cornforth; Jonathan Dushoff; Joshua S Weitz","Steven Sanche; Yen Ting Lin; Chonggang Xu; Ethan Romero-Severson; Nick Hengartner; Ruian Ke","Joe Hilton; Matt J Keeling","Xie, Mingxuan; Chen, Qiong","Can Zhou","HIETPAS, JACK; MCMULLAN, LAURA K.; HANSON, HOLLY L.; RICE, CHARLES M.; MINDELL, DAVID P.","Liu, Ying; Gayle, Albert A; Wilder-Smith, Annelies; Rockl\u00f6v, Joacim","Tao Liu; Jianxiong Hu; Jianpeng Xiao; Guanhao He; Min Kang; Zuhua Rong; Lifeng Lin; Haojie Zhong; Qiong Huang; Aiping Deng; Weilin Zeng; Xiaohua Tan; Siqing Zeng; Zhihua Zhu; Jiansen Li; Dexin Gong; Donghua Wan; Shaowei Chen; Lingchuan Guo; Yan Li; Limei Sun; Wenjia Liang; Tie Song; Jianfeng He; Wenjun Ma","Chentong Li; Jinhu Xu; Jiawei Liu; Yicang Zhou","Biao Tang; Fan Xia; Nicola Luigi Bragazzi; Xia Wang; Sha He; Xiaodan Sun; Sanyi Tang; Yanni Xiao; Jianhong Wu","Gautret, Philippe; Gray, Gregory C; Charrel, Remi N; Odezulu, Nnanyelugo G; Al-Tawfiq, Jaffar A; Zumla, Alimuddin; Memish, Ziad A","Polack, Bruno; Boulouis, Henri-Jean; Guillot, Jacques; Chermette, Ren\u00e9","Qin Jing; Chong You; Qiushi Lin; Taojun Hu; Shicheng Yu; Xiao-Hua Zhou","Wu, Joseph T; Leung, Kathy; Leung, Gabriel M","McBryde, Emma","Wangping Jia; Ke Han; Yang Song; Wenzhe Cao; Shengshu Wang; Shanshan Yang; Jianwei Wang; Fuyin Kou; Penggang Tai; Jing Li; Miao Liu; Yao He","Li, Peng; Fu, Ji-Bo; Li, Ke-Feng; Chen, Yan; Wang, Hong-Ling; Liu, Lei-Jie; Liu, Jie-Nan; Zhang, Yong-Li; Liu, She-Lan; Tang, An; Tong, Zhen-Dong; Yan, Jian-Bo","Meili Li; Pian Chen; Qianqian Yuan; Baojun Song; Junling Ma","Tang, Biao; Bragazzi, Nicola Luigi; Li, Qian; Tang, Sanyi; Xiao, Yanni; Wu, Jianhong","Corey M Peak; Rebecca Kahn; Yonatan H Grad; Lauren M Childs; Ruoran Li; Marc Lipsitch; Caroline O Buckee","de Sousa, Rita; Reusken, Chantal; Koopmans, Marion","Chong You; Yuhao Deng; Wenjie Hu; Jiarui Sun; Qiushi Lin; Feng Zhou; Cheng Heng Pang; Yuan Zhang; Zhengchao Chen; Xiao-Hua Zhou","Wenjun Zhang; Zeliang Chen; Yi Lu; Zhongmin Guo; Yanhong Qi; Guoling Wang; Jiahai Lu","Nian Shao; Jin Cheng; Wenbin Chen","Virlogeux, Victor; Fang, Vicky J.; Wu, Joseph T.; Ho, Lai-Ming; Malik Peiris, J. S.; Leung, Gabriel M.; Cowling, Benjamin J.","Stephen M Kissler; Christine Tedijanto; Edward Goldstein; Yonatan H. Grad; Marc Lipsitch","Linton, M. Natalie; Kobayashi, Tetsuro; Yang, Yichi; Hayashi, Katsuma; Akhmetzhanov, R. Andrei; Jung, Sung-mok; Yuan, Baoyin; Kinoshita, Ryo; Nishiura, Hiroshi","Natalie M. Linton; Tetsuro Kobayashi; Yichi Yang; Katsuma Hayashi; Andrei R. Akhmetzhanov; Sung-mok Jung; Baoyin Yuan; Ryo Kinoshita; Hiroshi Nishiura","Zhou Tang; Xianbin Li; Houqiang Li","Ke K. Zhang; Linglin Xie; Lauren Lawless; Huijuan Zhou; Guannan Gao; Chengbin Xue"],"color":["#67a9cf","#67a9cf","#67a9cf","#67a9cf","#67a9cf","#67a9cf","#67a9cf","#67a9cf","#67a9cf","#67a9cf","#67a9cf","#67a9cf","#67a9cf","#67a9cf","#67a9cf","#67a9cf","#67a9cf","#67a9cf","#67a9cf","#67a9cf","#67a9cf","#67a9cf","#67a9cf","#67a9cf","#67a9cf","#67a9cf","#67a9cf","#67a9cf","#67a9cf","#67a9cf","#67a9cf","#67a9cf","#67a9cf","#67a9cf","#67a9cf","#67a9cf","#67a9cf","#67a9cf","#67a9cf","#67a9cf","#67a9cf","#67a9cf","#67a9cf","#67a9cf","#67a9cf","#67a9cf","#67a9cf","#67a9cf","#67a9cf","#67a9cf"],"date":["2020-03-06","2020-03-24","2020-02-13","2020-03-08","2020-03-17","2020-03-08","2020-02-13","2020","2020-03-06","2020-03-30","2020-03-20","2016-01-18","2020-02-23","2020-02-02","2020-02-09","2020-02-23","2020","2020-02-20","2020-03-16","2020-03-04","2020-03-13","2020-02-11","2020-02-27","2020-04-01","2020-02-18","2005-12-31","2020-02-13","2020-01-26","2020-03-02","2020-03-13","2014-11-30","2015-12-31","2020-03-10","2020-03-06","2020-03-11","2020-03-20","2020-03-16","2020-03-01","2020-02-11","2020-03-08","2014-01-31","2020-02-11","2020-02-12","2020-02-20","2015-09-01","2020-03-06","2020","2020-01-28","2020-03-06","2020-02-25"],"doc":["66ulqu11","56zhxd6e","belin2jq","cq6mivr9","0hnh4n9e","v3gww4iv","vspnuxz9","8n4vklsf","tne83uu0","xdjgjeb9","sz26gchn","u5qt9es8","ragcpbl6","p5aj5k2g","ra3t6kmm","st5vs6gq","yuppix3r","c800ynvc","j181i5pr","hnx213kp","50zrnau8","45g12waw","96wkqutc","juz9jnfk","ehm4fgos","qma1lsuw","n11fcwmw","3e2soc6w","1yka7ej9","z7r45291","a559unhv","vm3bhmlh","it4ka7v0","32v44sw9","jsz2lg6l","y7giefdw","aul6ahww","px0twvhs","thu861hj","e2p46wa8","yiy2y2x3","juiomzb3","cv36vc8i","q6pz6l2b","kzp0sgo6","f4hj35dr","4vrqzbof","ykofrn9i","1mu1z4xd","rr7r7ef0"],"journal":["","","Euro Surveill","","The Lancet","","","International Journal of Infectious Diseases","","","","Osong Public Health Res Perspect","","","","","International Journal of Infectious Diseases","","","","","","","International Journal of Infectious Diseases","","Microbial Forensics","J Travel Med","","","","The Lancet Infectious Diseases","Revue Francophone des Laboratoires","","The Lancet","The Lancet Infectious Diseases","","International Journal of Infectious Diseases","","Infect Dis Model","","Journal of Clinical Virology","","","","Epidemiology","","Journal of Clinical Medicine","","",""],"rad":[0.02,0.01696022216702704,0.01603181950412539,0.01598478307488338,0.013826740053171669,0.01304893030552666,0.012142302217015427,0.012078644722439219,0.011768152035955715,0.011744689635788536,0.011429760103496452,0.010060162629109045,0.009714990317351931,0.00960782231292767,0.008285592853321106,0.007091506916669049,0.006638817289457468,0.006343904048729438,0.006340108811399655,0.00595575394213484,0.00590592762509678,0.005643143221781279,0.005589381094040696,0.005489674340010204,0.005398606342398856,0.005328040253699404,0.0050285055255833225,0.004735494414989312,0.00469162190603065,0.0045177285142693,0.004440930088273242,0.004278833276651017,0.004211085254097782,0.004155104407320256,0.004112383605100385,0.004068741664066616,0.004037083999389877,0.003951979746226891,0.0039265430189366285,0.0038207404584430235,0.0038114771912521085,0.003719307138166465,0.0035740815833394755,0.0034990750224174506,0.003444818278741893,0.0032341505697706056,0.003192330483729359,0.003192330483729359,0.003110755018990174,0.003],"scr":[1.0,0.821,0.767,0.764,0.637,0.591,0.538,0.534,0.516,0.514,0.496,0.415,0.395,0.389,0.311,0.241,0.214,0.197,0.196,0.174,0.171,0.155,0.152,0.146,0.141,0.137,0.119,0.102,0.1,0.089,0.085,0.075,0.071,0.068,0.065,0.063,0.061,0.056,0.055,0.048,0.048,0.042,0.034,0.029,0.026,0.014,0.011,0.011,0.007,0.0],"text":["Background: As the COVID-19 epidemic is spreading, incoming data allows us to quantify values of key variables that determine the transmission and the effort required to control the epidemic. We determine the incubation period and serial interval distribution for transmission clusters in Singapore and in Tianjin. We infer the basic reproduction number and identify the extent of pre-symptomatic transmission. Methods: We collected outbreak information from Singapore and Tianjin, China, reported from Jan.19-Feb.26 and Jan.21-Feb.27, respectively. We estimated incubation periods and serial intervals in both populations. Results: The mean incubation period was 7.1 (6.13, 8.25) days for Singapore and 9 (7.92, 10.2)days for Tianjin. Both datasets had shorter incubation periods for earlier-occurring cases. The mean serial interval was 4.56 (2.69, 6.42) days for Singapore and 4.22 (3.43, 5.01) for Tianjin. We inferred that early in the outbreaks, infection was transmitted on average 2.55 and 2....","Background The outbreak of coronavirus disease 2019 (COVID-19) has been declared a pandemic by the World Health Organization, while several key epidemiological parameters of the disease remain to be clarified. This study aimed to obtain robust estimates of the incubation period, upper limit of latent period (interval between exposure of infector and infectee), serial interval, time point of exposure and basic reproduction number (R0) of COVID-19. Methods Between late February and early March of 2020, the individual data of laboratory confirmed cases of COVID-19 were retrieved from 10728 publicly available reports released by the health authorities of and outside China and from 1790 publications identified in PubMed and CNKI. To be eligible, a report had to contain the data that allowed for estimation of at least one parameter. As relevant data mainly came from clustering cases, the clusters for which no evidence was available to establish transmission order were all excluded to ensure ...","Infections with 2019-nCoV can spread from person to person, and in the earliest phase of the outbreak the basic reproductive number was estimated to be around 2.2, assuming a mean serial interval of 7.5 days [2]. The serial interval was not precisely estimated, and a potentially shorter mean serial interval would have corresponded to a slightly lower basic reproductive number. Control measures and changes in population behaviour later in January should have reduced the effective reproductive number. However, it is too early to estimate whether the effective reproductive number has been reduced to below the critical threshold of 1 because cases currently being detected and reported would have mostly been infected in mid- to late-January. Average delays between infection and illness onset have been estimated at around 5\u20136 days, with an upper limit of around 11-14 days [2,5], and delays from illness onset to laboratory confirmation added a further 10 days on average [2].","Background: Estimating key infectious disease parameters from the COVID-19 outbreak is quintessential for modelling studies and guiding intervention strategies. Whereas different estimates for the incubation period distribution and the serial interval distribution have been reported, estimates of the generation interval for COVID-19 have not been provided. Methods: We used outbreak data from clusters in Singapore and Tianjin, China to estimate the generation interval from symptom onset data while acknowledging uncertainty about the incubation period distribution and the underlying transmission network. From those estimates we obtained the proportions pre-symptomatic transmission and reproduction numbers. Results: The mean generation interval was 5.20 (95%CI 3.78-6.78) days for Singapore and 3.95 (95%CI 3.01-4.91) days for Tianjin, China when relying on a previously reported incubation period with mean 5.2 and SD 2.8 days. The proportion of pre-symptomatic transmission was 48% (95%CI 32...","Summary Background Three clusters of coronavirus disease 2019 (COVID-19) linked to a tour group from China, a company conference, and a church were identified in Singapore in February, 2020. Methods We gathered epidemiological and clinical data from individuals with confirmed COVID-19, via interviews and inpatient medical records, and we did field investigations to assess interactions and possible modes of transmission of severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2). Open source reports were obtained for overseas cases. We reported the median (IQR) incubation period of SARS-CoV-2. Findings As of Feb 15, 2020, 36 cases of COVID-19 were linked epidemiologically to the first three clusters of circumscribed local transmission in Singapore. 425 close contacts were quarantined. Direct or prolonged close contact was reported among affected individuals, although indirect transmission (eg, via fomites and shared food) could not be excluded. The median incubation period of SARS-C...","Background: The ongoing outbreak of novel corona virus disease 2019 (COVID-19) in Wuhan, China, is arousing international concern. This study evaluated whether and when the infected but asymptomatic cases during the incubation period could infect others. Methods: We collected data on demographic characteristics, exposure history, and symptom onset day of the confirmed cases, which had been announced by the Chinese local authorities. We evaluated the potential of transmission during the incubation period in 50 infection clusters, including 124 cases. All the secondary cases had a history of contact with their first-generation cases prior to symptom onset. Results: The estimated mean incubation period for COVID-19 was 4.9 days (95% confidence interval [CI], 4.4 to 5.4) days, ranging from 0.8 to 11.1 days (2.5th to 97.5th percentile). The observed mean and standard deviation (SD) of serial interval was 4.1\u00b13.3 days, with the 2.5th and 97.5th percentiles at -1 and 13 days. The infectious c...","Objective: To estimate the serial interval of novel coronavirus (COVID-19) from information on 28 infector-infectee pairs. Methods: We collected dates of illness onset for primary cases (infectors) and secondary cases (infectees) from published research articles and case investigation reports. We subjectively ranked the credibility of the data and performed analyses on both the full dataset (n=28) and a subset of pairs with highest certainty in reporting (n=18). In addition, we adjusting for right truncation of the data as the epidemic is still in its growth phase. Results: Accounting for right truncation and analyzing all pairs, we estimated the median serial interval at 4.0 days (95% credible interval [CrI]: 3.1, 4.9). Limiting our data to only the most certain pairs, the median serial interval was estimated at 4.6 days (95% CrI: 3.5, 5.9). Conclusions: The serial interval of COVID-19 is shorter than its median incubation period. This suggests that a substantial proportion of seconda...","Objective To estimate the serial interval of novel coronavirus (COVID-19) from information on 28 infector-infectee pairs. Methods We collected dates of illness onset for primary cases (infectors) and secondary cases (infectees) from published research articles and case investigation reports. We subjectively ranked the credibility of the data and performed analyses on both the full dataset (n = 28) and a subset of pairs with highest certainty in reporting (n = 18). In addition, we adjust for right truncation of the data as the epidemic is still in its growth phase. Results Accounting for right truncation and analyzing all pairs, we estimated the median serial interval at 4.0 days (95% credible interval [CrI]: 3.1, 4.9). Limiting our data to only the most certain pairs, the median serial interval was estimated at 4.6 days (95% CrI: 3.5, 5.9). Conclusions The serial interval of COVID-19 is close to or shorter than its median incubation period. This suggests that a substantial proportion o...","At the end of 2019, a coronavirus disease 2019 (COVID-19) outbroke in Wuhan, China, and spread to Guizhou province on January of 2020. To acquire the epidemiologic characteristics of COVID-19 in Guizhou, China, we collected data on 162 laboratory-confirmed cases related to COVID-19. We described the demographic characteristics of the cases and estimated the incubation period, serial interval and basic reproduction number. We also presented two representative case studies in Guizhou province -- Case Study 1 was an example of asymptomatic carrier; and Case Study 2 was an example of a large and complex infection chain that involved four different districts spanning three provinces and eight families. With an estimation of 8 days incubation period and 6 days serial interval, our results indicate that there may exist infectiousness during the incubation period for 2019-nCoV. This increases the difficulty of screening or identifying cases related to COVID-19.","The number of COVID-19 cases is continuously increasing in different countries (as of March 2020) including the Philippines. It is estimated that the basic reproductive number of COVID-19 is around 1.5 to 4. The basic reproductive number characterizes the average number of persons that a primary case can directly infect in a population full of susceptible individuals. However, there can be superspreaders that can infect more than this estimated basic reproductive number. In this study, we formulate a conceptual mathematical model on the transmission dynamics of COVID-19 between the frontliners and the general public. We assume that the general public has a reproductive number between 1.5 to 4, and frontliners (e.g. healthcare workers, customer service and retail personnel, food service crews, and transport or delivery workers) have a higher reproduction number. Our simulations show that both the frontliners and the general public should be protected or resilient against the disease. Pr...","Background: Global spread of COVID-19 created an unprecedented infectious disease crisis that progressed to a pandemic with &gt;180,000 cases in &gt;100 countries. Reproductive number (R) is an outbreak metric estimating the transmission of a pathogen. Initial R values were published based on the early outbreak in China with limited number of cases with whole genome sequencing. Initial comparisons failed to show a direct relationship viral genomic diversity and epidemic severity was not established for SARS-Cov-2. Methods: Each country's COVID-19 outbreak status was classified according to epicurve stage (index, takeoff, exponential, decline). Instantaneous R estimates (Wallinga and Teunis method) with a short and standard serial interval examined asymptomatic spread. Whole genome sequences were used to quantify the pathogen genome identity score that were used to estimate transmission time and epicurve stage. Transmission time was estimated based on evolutionary rate of 2 mutations/month. F...","OBJECTIVES: The outbreak of Middle Eastern respiratory syndrome coronavirus (MERS-CoV) was one of the major events in South Korea in 2015. In particular, this study pays attention to formulating a mathematical model for MERS transmission dynamics and estimating transmission rates. METHODS: Incidence data of MERS-CoV from the government authority was analyzed for the first aim and a mathematical model was built and analyzed for the second aim of the study. A mathematical model for MERS-CoV transmission dynamics is used to estimate the transmission rates in two periods due to the implementation of intensive interventions. RESULTS: Using the estimates of the transmission rates, the basic reproduction number was estimated in two periods. Due to the superspreader, the basic reproduction number was very large in the first period; however, the basic reproduction number of the second period has reduced significantly after intensive interventions. CONCLUSION: It turned out to be the intensive i...","Background The COVID-19 epidemic originated in Wuhan City of Hubei Province in December 2019 and has spread throughout China. Understanding the fast evolving epidemiology and transmission dynamics of the outbreak beyond Hubei would provide timely information to guide intervention policy. Methods We collected individual information on 8,579 laboratory-confirmed cases from official publically sources reported outside Hubei in mainland China, as of February 17, 2020. We estimated the temporal variation of the demographic characteristics of cases and key time-to-event intervals. We used a Bayesian approach to estimate the dynamics of the net reproduction number (Rt) at the provincial level. Results The median age of the cases was 44 years, with an increasing of cases in younger age groups and the elderly as the epidemic progressed. The delay from symptom onset to hospital admission decreased from 4.4 days (95%CI: 0.0-14.0) until January 27 to 2.6 days (0.0-9.0) from January 28 to February ...","A novel coronavirus (SARS-CoV-2) has recently emerged as a global threat. As the epidemic progresses, many disease modelers have focused on estimating the basic reproductive number Ro -- the average number of secondary cases caused by a primary case in an otherwise susceptible population. The modeling approaches and resulting estimates of Ro vary widely, despite relying on similar data sources. Here, we present a novel statistical framework for comparing and combining different estimates of Ro across a wide range of models by decomposing the basic reproductive number into three key quantities: the exponential growth rate $r$, the mean generation interval $\\bar G$, and the generation-interval dispersion $\\kappa$. We then apply our framework to early estimates of Ro for the SARS-CoV-2 outbreak. We show that many early Ro estimates are overly confident. Our results emphasize the importance of propagating uncertainties in all components of Ro, including the shape of the generation-interval...","Background: On January 23, 2020, a quarantine was imposed on travel in and out of Wuhan, where the 2019 novel coronavirus (2019-nCoV) outbreak originated from. Previous analyses estimated the basic epidemiological parameters using symptom onset dates of the confirmed cases in Wuhan and outside China. Methods: We obtained information on the 46 coronavirus cases who traveled from Wuhan before January 23 and have been subsequently confirmed in Hong Kong, Japan, Korea, Macau, Singapore, and Taiwan as of February 5, 2020. Most cases have detailed travel history and disease progress. Compared to previous analyses, an important distinction is that we used this data to informatively simulate the infection time of each case using the symptom onset time, previously reported incubation interval, and travel history. We then fitted a simple exponential growth model with adjustment for the January 23 travel ban to the distribution of the simulated infection time. We used a Bayesian analysis with dif...","As a novel coronavirus (COVID-19) continues to emerge throughout China and threaten the globe, its transmission characteristics remain uncertain. Here, we analyze the serial intervals-the time period between the onset of symptoms in an index (infector) case and the onset of symptoms in a secondary (infectee) case-of 468 infector-infectee pairs with confirmed COVID-19 cases reported by health departments in 18 Chinese provinces between January 21, 2020, and February 8, 2020. The reported serial intervals range from -11 days to 20 days, with a mean of 3.96 days (95% confidence interval: 3.53-4.39), a standard deviation of 4.75 days (95% confidence interval: 4.46-5.07), and 12.6% of reports indicating pre-symptomatic transmission.","Backgrounds Up to February 16, 2020, 355 cases have been confirmed as having COVID-19 infection on the Diamond Princess cruise ship. It is of crucial importance to estimate the reproductive number (R0) of the novel virus in the early stage of outbreak and make a prediction of daily new cases on the ship. Method We fitted the reported serial interval (Mean and standard deviation) with a gamma distribution and applied \u201cearlyR\u201d package in R to estimate the R0 in the early stage of COVID-19 outbreak. We applied \u201cprojections\u201d package in R to simulate the plausible cumulative epidemic trajectories and future daily incidence by fitting the data of existing daily incidence, a serial interval distribution, and the estimated R0 into a model based on the assumption that daily incidence obeys approximately Poisson distribution determined by daily infectiousness. Results The Maximum-Likelihood (ML) value of R0 was 2.28 for COVID-19 outbreak at early stage on the ship. The median with 95% confidence...","Pneumonia patients of 2019-ncov in latent period are not easy to be effectively quarantined, but there is evidence that they have strong infectious ability. Here, the infectious ability of patients during the latent period is slightly less than that of the infected patients was assumed. We established a new SEIR propagation dynamics model, that considered the weak transmission ability of the incubation period, the variation of the incubation period length, and the government intervention measures to track and isolate comprehensively. Based on the raw epidemic data of China from January 23, 2020 to February 10, 2020, the dynamic parameters of the new present SEIR model are fitted. Through the Euler integration algorithm to solve the model, the effect of infectious ability of incubation patients on the theoretical estimation of the present SEIR model was analyzed, and the occurrence time of peak number in China was predicted.","Background The outbreak of novel coronavirus disease 2019 (COVID-19) started in the city of Wuhan, China, with a period of rapid initial spread. Transmission on a regional and then national scale was promoted by intense travel during the holiday period of the Chinese New Year. We studied the variation in transmission of COVID-19, locally in Wuhan, as well as on a larger spatial scale, among different cities and even among provinces in mainland China. Methods In addition to reported numbers of new cases, we have been able to assemble detailed contact data for some of the initial clusters of COVID-19. This enabled estimation of the serial interval for clinical cases, as well as reproduction numbers for small and large regions. Findings We estimated the average serial interval was 4.8 days. For early transmission in Wuhan, any infectious case produced as many as four new cases, transmission outside Wuhan was less intense, with reproduction numbers below two. During the rapid growth phase ...","Background Rapid spread of SARS-CoV-2 in Wuhan prompted heightened surveillance in Shenzhen and elsewhere in China. The resulting data provide a rare opportunity to measure key metrics of disease course, transmission, and the impact of control. Methods The Shenzhen CDC identified 391 SARS-CoV-2 cases from January 14 to February 12, 2020 and 1286 close contacts. We compare cases identified through symptomatic surveillance and contact tracing, and estimate the time from symptom onset to confirmation, isolation, and hospitalization. We estimate metrics of disease transmission and analyze factors influencing transmission risk. Findings Cases were older than the general population (mean age 45) and balanced between males (187) and females (204). Ninety-one percent had mild or moderate clinical severity at initial assessment. Three have died, 225 have recovered (median time to recovery is 21 days). Cases were isolated on average 4.6 days after developing symptoms; contact tracing reduced thi...","We assess the impact of asymptomatic transmission on epidemic potential of novel respiratory pathogens (like COVID-19) -- as measured both by the basic reproduction number (i.e., the expected number of secondary cases generated by an average primary case in a fully susceptible population) and the fraction of new secondary cases attributable to asymptomatic individuals. We show that the impact of asymptomatic transmission depends on generation intervals (i.e., time between when an individual is infected and when that individual infects another person). If the generation-interval distribution of asymptomatic transmission differs from that of symptomatic transmission, then estimates of the basic reproduction number which do not explicitly account for asymptomatic cases may be systematically biased. Specifically, if asymptomatic cases have a shorter generation interval than symptomatic cases, R_0 will be over-estimated, and if they have a longer generation interval, R_0 will be under-estim...","The novel coronavirus (2019-nCoV) is a recently emerged human pathogen that has spread widely since January 2020. Initially, the basic reproductive number, R0, was estimated to be 2.2 to 2.7. Here we provide a new estimate of this quantity. We collected extensive individual case reports and estimated key epidemiology parameters, including the incubation period. Integrating these estimates and high-resolution real-time human travel and infection data with mathematical models, we estimated that the number of infected individuals during early epidemic double every 2.4 days, and the R0 value is likely to be between 4.7 and 6.6. We further show that quarantine and contact tracing of symptomatic individuals alone may not be effective and early, strong control measures are needed to stop transmission of the virus.","The outbreak of novel coronavirus (COVID-19) has the potential for global spread, infecting large numbers in all countries. In this case, estimating the country-specific basic reproductive ratio is a vital first step in public-health planning. The basic reproductive ratio (R0) is determined by both the nature of pathogen and the network of contacts through which the disease can spread - with this network determined by socio-demographics including age-structure and household composition. Here we focus on the age-structured transmission within the population, using data from China to inform age-dependent susceptibility and synthetic age-mixing matrices to inform the contact network. This allows us to determine the country-specific basic reproductive ratio as a multiplicative scaling of the value from China. We predict that R0 will be highest across Eastern Europe and Japan, and lowest across Africa, Central America and South-Western Asia. This pattern is largely driven by the ratio of ch...","Abstract Background The rapid spread of the coronavirus disease 2019 (COVID-19), caused by a zoonotic beta-coronavirus entitled 2019 novel coronavirus (2019-nCoV), has become a global threat. Awareness of the biological features of 2019-nCoV should be updated in time and needs to be comprehensively summarized to help optimize control measures and make therapeutic decisions. Methods Based on recently published literatures, official documents and selected up-to-date preprint studies, we reviewed the virology and origin, epidemiology, clinical manifestations, pathology and treatment of 2019-nCoV infection, in comparison with severe acute respiratory syndrome coronavirus (SARS-CoV) and middle east respiratory syndrome coronavirus (MERS-CoV) infection. Results The genome of 2019-nCoV partially resembled SARS-CoV and MERS-CoV, and indicating a bat origin. The COVID-19 generally had a high reproductive number, a long incubation period, a short serial interval and a low case fatality rate (muc...","The novel coronavirus (COVID-19), first detected in Wuhan, China in December 2019, has spread to 28 countries/regions with over 43,000 confirmed cases. Much about this outbreak is still unknown. At this early stage of the epidemic, it is important to investigate alternative sources of information to understand its dynamics and spread. With updated real time domestic traffic, this study aims to integrate recent evidence of international evacuees extracted from Wuhan between Jan. 29 and Feb. 2, 2020 to infer the dynamics of the COVD-19 outbreak in Wuhan. In addition, a modified SEIR model was used to evaluate the empirical support for the presence of asymptomatic transmissions. Based on the data examined, this study found little evidence for the presence of asymptomatic transmissions. However, it is still too early to rule out its presence conclusively due to sample size and other limitations. The updated basic reproductive number was found to be 2.12 on average with a 95% credible inter...","Publisher Summary This chapter introduces basic concepts of virology and exemplifies the usage of current technology for identifying and tracking viral pathogens. Virus structure, lifestyle, replication strategies, classication, the varied modes of transmission, infection, and disease manifestations are focussed upon in the chapter. Viruses are extremely simple \u201clife\u201d forms without metabolic capacity, organelles, translational machinery, or autonomous replicative potential; the essential elements of a virus are consequently minimal. A virus enters the host through any opportunistic opening of the body\u2014; such as minute breaks in the skin barrier or mucosal linings of the eye, nose, etc. Antibodies produced during an infection often have the ability to bind the virus and reduce infectivity. An immediate question to solve during a virus outbreak is the mechanism by which the virus spreads. Viruses frequently infect animal or insect vectors that serve to pass the virus to humans. By identi...","Teaser: Our review found the average R0 for 2019-nCoV to be 3.28, which exceeds WHO estimates of 1.4 to 2.5.Here we review the basic reproduction number (R0) of the 2019-nCoV virus. R0 is an indication of the transmissibility of a virus, representing the average number of new infections generated by an infectious person in a totally na\u00efve population. For R0 greater than one the number infected is likely to increase, and for R0 less than one transmission is likely to die out. The basic reproduction number is a central concept in infectious disease epidemiology, indicating the risk of an infectious agent with respect to epidemic spread.","ABSTRACTRationaleSeveral studies have estimated basic production number of novel coronavirus pneumonia (NCP). However, the time-varying transmission dynamics of NCP during the outbreak remain unclear.ObjectivesWe aimed to estimate the basic and time-varying transmission dynamics of NCP across China, and compared them with SARS.MethodsData on NCP cases by February 7, 2020 were collected from epidemiological investigations or official websites. Data on severe acute respiratory syndrome (SARS) cases in Guangdong Province, Beijing and Hong Kong during 2002-2003 were also obtained. We estimated the doubling time, basic reproduction number (R0) and time-varying reproduction number (Rt) of NCP and SARS.Measurements and main resultsAs of February 7, 2020, 34,598 NCP cases were identified in China, and daily confirmed cases decreased after February 4. The doubling time of NCP nationwide was 2.4 days which was shorter than that of SARS in Guangdong (14.3 days), Hong Kong (5.7 days) and Beijing (...","AbstractIn this work, we use a within-host viral dynamic model to describe the SARS-CoV-2 kinetics in host. Chest radiograph score data are used to estimate the parameters of that model. Our result shows that the basic reproductive number of SARS-CoV-2 in host growth is around 3.79. Using the same method we also estimate the basic reproductive number of MERS virus is 8.16 which is higher than SARS-CoV-2. The PRCC method is used to analyze the sensitivities of model parameters and the drug effects on virus growth are also implemented to analyze the model.","We conducted a comparative study of COVID-19 epidemic in three different settings: mainland China, the Guangdong province of China and South Korea, by formulating two disease transmission dynamics models incorporating epidemic characteristics and setting-specific interventions, and fitting the models to multi-source data to identify initial and effective reproduction numbers and evaluate effectiveness of interventions. We estimated the initial basic reproduction number for South Korea, the Guangdong province and mainland China as 2.6 (95% confidence interval (CI): (2.5, 2.7)), 3.0 (95%CI: (2.6, 3.3)) and 3.8 (95%CI: (3.5,4.2)), respectively, given a serial interval with mean of 5 days with standard deviation of 3 days. We found that the effective reproduction number for the Guangdong province and mainland China has fallen below the threshold 1 since February 8th and 18th respectively, while the effective reproduction number for South Korea remains high, suggesting that the intervention...","Summary The past decade has seen the emergence of several novel viruses that cause respiratory tract infections in human beings, including Middle East respiratory syndrome coronavirus (MERS-CoV) in Saudi Arabia, an H7N9 influenza A virus in eastern China, a swine-like influenza H3N2 variant virus in the USA, and a human adenovirus 14p1 also in the USA. MERS-CoV and H7N9 viruses are still a major worldwide public health concern. The pathogenesis and mode of transmission of MERS-CoV and H7N9 influenza A virus are poorly understood, making it more difficult to implement intervention and preventive measures. A united and coordinated global response is needed to tackle emerging viruses that can cause fatal respiratory tract infections and to fill major gaps in the understanding of the epidemiology and transmission dynamics of these viruses.","R\u00e9sum\u00e9 Les animaux r\u00e9servoirs d\u2019agents pathog\u00e8nes pour l\u2019homme sont d\u2019une importance primordiale \u00e0 conna\u00eetre dans le but de comprendre l\u2019\u00e9pid\u00e9mio-logie des zoonoses correspondantes et de mettre en place des mesures de lutte judicieuses. Cet article r\u00e9capitule, sous forme de diff\u00e9rents tableaux, les principales zoonoses dues \u00e0 des virus, des bact\u00e9ries, des protozoaires, des helminthes, des arthropodes et des champignons. Chaque tableau liste les principaux agents pathog\u00e8nes, les animaux r\u00e9servoirs et les modes de transmission aux humains. Summary The knowledge of animals which act as reservoirs for human pathogenic agents is essential for the understanding of the epidemiology of zoonotic diseases and for the application of relevant methods of control. In the present article, synthetic tables present the main zoonoses due to viruses, bacteria, protozoa, helminths, arthropods and fungi. A list of the main pathogenic agents, the animal reservoirs and the way of transmission to man is detai...","Background: The current outbreak of coronavirus disease 2019 (COVID-19) has quickly spread across countries and become a global crisis. However, one of the most important clinical characteristics in epidemiology, the distribution of the incubation period, remains unclear. Different estimates of the incubation period of COVID-19 were reported in recent published studies, but all have their own limitations. In this study, we propose a novel low-cost and accurate method to estimate the incubation distribution. Methods: We have conducted a cross-sectional and forward follow-up study by identifying those asymptomatic individuals at their time of departure from Wuhan and then following them until their symptoms developed. The renewal process is hence adopted by considering the incubation period as a renewal and the duration between departure and symptom onset as a forward recurrence time. Under mild assumptions, the observations of selected forward times can be used to consistently estimate ...","Summary Background Since Dec 31, 2019, the Chinese city of Wuhan has reported an outbreak of atypical pneumonia caused by the 2019 novel coronavirus (2019-nCoV). Cases have been exported to other Chinese cities, as well as internationally, threatening to trigger a global outbreak. Here, we provide an estimate of the size of the epidemic in Wuhan on the basis of the number of cases exported from Wuhan to cities outside mainland China and forecast the extent of the domestic and global public health risks of epidemics, accounting for social and non-pharmaceutical prevention interventions. Methods We used data from Dec 31, 2019, to Jan 28, 2020, on the number of cases exported from Wuhan internationally (known days of symptom onset from Dec 25, 2019, to Jan 19, 2020) to infer the number of infections in Wuhan from Dec 1, 2019, to Jan 25, 2020. Cases exported domestically were then estimated. We forecasted the national and global spread of 2019-nCoV, accounting for the effect of the metropo...","The world is braced for a public health emergency of international concern caused by a novel emerging infectious disease, a coronavirus with similarities to severe acute respiratory syndrome coronavirus (SARS-CoV). Person-to-person transmission of SARS-CoV-2, the causative agent of coronavirus disease 2019 (COVID-19), started in December, 2019, in Wuhan, China and has spread to become a global pandemic, with, as of Feb 26, 2020, community transmission in Italy, Iran, and South Korea. Modelling studies have aided understanding of COVID-19 dynamics from the first announcement of the epidemic and publication of the genetic sequence of the causative virus. Initial phylogenetic analysis of closely related viruses suggested highly linked person-to-person spread of SARS-CoV-2 originating from mid-November to early December, 2019.1, 2 Following this, modellers provided simple calculations that identified a mismatch between reported cases in China and reported importations of cases from travell...","Background: Coronavirus Disease 2019 (COVID19) is currently a global public health threat. Outside of China, Italy is one of the most suffering countries with the COVID19 epidemic. It is important to predict the epidemics trend of COVID19 epidemic in Italy to help develop public health strategies. Methods: We used time series data of COVID 19 from Jan 22,2020 to Mar 16,2020. An infectious disease dynamic extended susceptible infected removed (eSIR) model, which covers the effects of different intervention measures in dissimilar periods, was applied to estimate the epidemic trend in Italy. The basic reproductive number was estimated using Markov Chain Monte Carlo methods and presented using the resulting posterior mean and 95% credible interval (CI). Hunan, with similar total number of populations in Italy, was used as a comparative item. Results: In the eSIR model, we estimated that the basic reproductive number for COVID 19 was respectively 4.10 (95% CI: 2.15 to 6.77) in Italy and 3.1...","Abstract We report a familial cluster of 2019 novel coronavirus disease (COVID-19) to evidence that a potential transmission of the COVID-19 during the incubation period. The first patient in this familial cluster was identified in presymptomatic period, as a close contact of a confirmed patient. Five family members had close contact with the first patient during his incubation period, four of them were confirmed to the severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) infection in the subsequent sampling test.","The COVID-19 outbreak has been a serious public health threat worldwide. We use individually documented case descriptions of COVID-19 from China (excluding Hubei Province) to estimate the distributions of the generation time, incubation period, and periods from symptom onset to isolation and to diagnosis. The recommended 14-day quarantine period may lead to a 6.7% failure for quarantine. We recommend a 22-day quarantine period. The mean generation time is 3.3 days and the mean incubation period is 7.2 days. It took 3.7 days to isolate and 6.6 days to diagnose a patient after his/her symptom onset. Patients may become infectious on average 3.9 days before showing major symptoms. This makes contact tracing and quarantine ineffective. The basic reproduction number is estimated to be 1.54 with contact tracing, quarantine and isolation, mostly driven by super spreaders.","The basic reproduction number of an infectious agent is the average number of infections one case can generate over the course of the infectious period, in a na\u00efve, uninfected population. It is well-known that the estimation of this number may vary due to several methodological issues, including different assumptions and choice of parameters, utilized models, used datasets and estimation period. With the spreading of the novel coronavirus (2019-nCoV) infection, the reproduction number has been found to vary, reflecting the dynamics of transmission of the coronavirus outbreak as well as the case reporting rate. Due to significant variations in the control strategies, which have been changing over time, and thanks to the introduction of detection technologies that have been rapidly improved, enabling to shorten the time from infection/symptoms onset to diagnosis, leading to faster confirmation of the new coronavirus cases, our previous estimations on the transmission risk of the 2019-nCo...","Individual quarantine and active monitoring of contacts are core disease control strategies, particularly for emerging infectious diseases such as Coronavirus Disease 2019 (COVID-19). To estimate the comparative efficacy of these interventions to control COVID-19, we fit a stochastic branching model, comparing two sets of reported parameters for the dynamics of the disease. Our results suggest that individual quarantine may contain an outbreak of COVID-19 with a short serial interval (4.8 days) only in settings with high intervention performance where at least three-quarters of infected contacts are individually quarantined. However, in settings where this performance is unrealistically high and the outbreak of COVID-19 continues to grow, so too will the burden of the number of contacts traced for active monitoring or quarantine. In such circumstances where resources are prioritized for scalable interventions such as social distancing, we show active monitoring or individual quarantine...","Abstract Since the emergence of Middle East Respiratory Syndrome Coronavirus (MERS-CoV) in 2012, many questions remain on modes of transmission and sources of virus. In outbreak situations, especially with emerging organisms causing severe human disease, it is important to understand the full spectrum of disease, and shedding kinetics in relation to infectivity and the ability to transmit the microorganism. Laboratory response capacity during the early stages of an outbreak focuses on development of virological and immunological methods for patient diagnosis, for contact tracing, and for epidemiological studies into sources, modes of transmission, identification of risk groups, and animal reservoirs. However, optimal use of this core public health laboratory capacity requires a fundamental understanding of kinetics of viral shedding and antibody response, of assay validation and of interpretation of test outcomes. We reviewed available data from MERS-CoV case reports, and compared this...","Background: The 2019-nCoV outbreak in Wuhan, China has attracted world-wide attention. As of February 11, 2020, a total of 44730 cases of novel coronavirus-infected pneumonia associated with COVID-19 were confirmed by the National Health Commission of China. Methods: Three approaches, namely Poisson likelihood-based method (ML), exponential growth rate-based method (EGR) and stochastic Susceptible-Infected-Removed dynamic model-based method (SIR), were implemented to estimate the basic and controlled reproduction numbers. Results: A total of 71 chains of transmission together with dates of symptoms onset and 67 dates of infections were identified among 5405 confirmed cases outside Hubei as reported by February 2, 2020. Based on this information, we find the serial interval having an average of 4.41 days with a standard deviation of 3.17 days and the infectious period having an average of 10.91 days with a standard deviation of 3.95 days. Conclusions: The controlled reproduction number ...","AbstractA discrete dynamic model for human epidemics was developed in present study. The model included major parameters as transmission strength and its decline parameters, mean incubation period, hospitalization time, non-hospitalization daily mortality, non-hospitalization daily recovery rate, and hospitalization proportion, etc. Sensitivity analysis of the model indicated the total cumulative cases significantly increased with initial transmission strength, hospitalization time. The total cumulative cases significantly decreased with transmission strength\u2019s decline and hospitalization proportion, and linearly decreased with non-hospitalization daily mortality and non-hospitalization daily recovery rate. In a certain range, the total cumulative cases significantly increased with mean incubation period. Sensitivity analysis demonstrated that dynamic change of transmission strength is one of the most important and controllable factors. In addition, reducing the delay for hospitalizati...","In this paper, we estimate the reproductive number R0 of COVID-19 based on Wallinga and Lipsitch framework {11} and a novel statistical time delay dynamic system. We use the observed data reported in CCDC's paper to estimate distribution of the generation interval of the infection and apply the simulation results from the time delay dynamic system as well as released data from CCDC to fit the growth rate. The conclusion is: Based our Fudan-CCDC model, the growth rate r of COVID-19 is almost in [0.30, 0.32] which is larger than the growth rate 0.1 estimated by CCDC {9}, and the reproductive number R0 of COVID-19 is estimated by 3.25\u2264 R0 \u22643.4 if we simply use R=1+r*Tc with Tc=7.5, which is bigger than that of SARS. Some evolutions and predictions are listed.","BACKGROUND: Few previous studies have investigated the association between the severity of an infectious disease and the length of incubation period. METHODS: We estimated the association between the length of the incubation period and the severity of infection with the severe acute respiratory syndrome (SARS) coronavirus, using data from the epidemic in 2003 in Hong Kong. RESULTS: We estimated the incubation period of SARS based on a subset of patients with available data on exposure periods and a separate subset of patients in a putative common source outbreak, and we found significant associations between shorter incubation period and greater severity in both groups after adjusting for potential confounders. CONCLUSIONS: Our findings suggest that patients with a shorter incubation period proceeded to have more severe disease. Further studies are needed to investigate potential biological mechanisms for this association.","There is an urgent need to project how transmission of the novel betacoronavirus SARS-CoV-2 will unfold in coming years. These dynamics will depend on seasonality, the duration of immunity, and the strength of cross-immunity to/from the other human coronaviruses. Using data from the United States, we measured how these factors affect transmission of human betacoronaviruses HCoV-OC43 and HCoV-HKU1. We then built a mathematical model to simulate transmission of SARS-CoV-2 through the year 2025. We project that recurrent wintertime outbreaks of SARS-CoV-2 will probably occur after an initial pandemic wave. We summarize the full range of plausible transmission scenarios and identify key data still needed to distinguish between them, most importantly longitudinal serological studies to determine the duration of immunity to SARS-CoV-2.","The geographic spread of 2019 novel coronavirus (COVID-19) infections from the epicenter of Wuhan, China, has provided an opportunity to study the natural history of the recently emerged virus. Using publicly available event-date data from the ongoing epidemic, the present study investigated the incubation period and other time intervals that govern the epidemiological dynamics of COVID-19 infections. Our results show that the incubation period falls within the range of 2&amp;ndash;14 days with 95% confidence and has a mean of around 5 days when approximated using the best-fit lognormal distribution. The mean time from illness onset to hospital admission (for treatment and/or isolation) was estimated at 3&amp;ndash;4 days without truncation and at 5&amp;ndash;9 days when right truncated. Based on the 95th percentile estimate of the incubation period, we recommend that the length of quarantine should be at least 14 days. The median time delay of 13 days from illness onset to death (17 days with rig...","The geographic spread of 2019 novel coronavirus (COVID-19) infections from the epicenter of Wuhan, China, has provided an opportunity to study the natural history of the recently emerged virus. Using publicly available event-date data from the ongoing epidemic, the present study investigated the incubation period and other time intervals that govern the epidemiological dynamics of COVID-19 infections. Our results show that the incubation period falls within the range of 2-14 days with 95% confidence and has a mean of around 5 days when approximated using the best-fit lognormal distribution. The mean time from illness onset to hospital admission (for treatment and/or isolation) was estimated at 3-4 days without truncation and at 5-9 days when right truncated. Based on the 95th percentile estimate of the incubation period, we recommend that the length of quarantine should be at least 14 days. The median time delay of 13 days from illness onset to death (17 days with right truncation) sho...","BACKGROUND The outbreak of the new coronavirus infection in Wuhan City, Hubei Province in December 2019, poses a huge threat to China and even global public health security. Respiratory droplets and contact transmission are the main routes of transmission of new coronaviruses. Compared with SARS and Ebola viruses, new coronavirus infections are infectious during the incubation period. Traditional SEIR (susceptibility-exposure-infection-Removal) There are some differences in conditions for the prediction of the epidemic trend of new coronavirus infection. The outbreak of the new coronavirus infection coincided with the Spring Festival before and after the Chinese Spring Festival.It is necessary to make appropriate optimization and amendments to the traditional model to meet the actual evolution of the epidemic situation. METHODS The traditional SEIR model assumes that the virus-infected person is not infectious during the incubation period and that the infected person did not take isola...","The outbreak of the novel coronavirus disease, COVID-19, originating from Wuhan, China in early December, has infected more than 70,000 people in China and other countries and has caused more than 2,000 deaths. As the disease continues to spread, the biomedical society urgently began identifying effective approaches to prevent further outbreaks. Through rigorous epidemiological analysis, we characterized the fast transmission of COVID-19 with a basic reproductive number 5.6 and proved a sole zoonotic source to originate in Wuhan. No changes in transmission have been noted across generations. By evaluating different control strategies through predictive modeling and Monte carlo simulations, a comprehensive quarantine in hospitals and quarantine stations has been found to be the most effective approach. Government action to immediately enforce this quarantine is highly recommended."],"title":["Transmission interval estimates suggest pre-symptomatic spread of COVID-19","Epidemiological parameters of coronavirus disease 2019: a pooled analysis of publicly reported individual data of 1155 cases from seven countries","Epidemiological research priorities for public health control of the ongoing global novel coronavirus (2019-nCoV) outbreak","Estimating the generation interval for COVID-19 based on symptom onset data","Investigation of three clusters of COVID-19 in Singapore: implications for surveillance and response measures","Transmission of corona virus disease 2019 during the incubation period may lead to a quarantine loophole","Serial interval of novel coronavirus (2019-nCoV) infections","Serial interval of novel coronavirus (COVID-19) infections","Epidemiologic Characteristics of COVID-19 in Guizhou, China","A mathematical model of COVID-19 transmission between frontliners and the general public","Pandemic dynamics of COVID-19 using epidemic stage, instantaneous reproductive number and pathogen genome identity (GENI) score: modeling molecular epidemiology","The Characteristics of Middle Eastern Respiratory Syndrome Coronavirus Transmission Dynamics in South Korea","Evolving epidemiology of novel coronavirus diseases 2019 and possible interruption of local transmission outside Hubei Province in China: a descriptive and modeling study","Reconciling early-outbreak estimates of the basic reproductive number and its uncertainty: framework and applications to the novel coronavirus (SARS-CoV-2) outbreak","Analysis of the epidemic growth of the early 2019-nCoV outbreak using internationally confirmed cases","The serial interval of COVID-19 from publicly reported confirmed cases","Estimation of the reproductive number of Novel Coronavirus (COVID-19) and the probable outbreak size on the Diamond Princess cruise ship: A data-driven analysis","SEIR Transmission dynamics model of 2019 nCoV coronavirus with considering the weak infectious ability and changes in latency duration","Strongly heterogeneous transmission of COVID-19 in mainland China: local and regional variation","Epidemiology and Transmission of COVID-19 in Shenzhen China: Analysis of 391 cases and 1,286 of their close contacts","The time scale of asymptomatic transmission affects estimates of epidemic potential in the COVID-19 outbreak","The Novel Coronavirus, 2019-nCoV, is Highly Contagious and More Infectious Than Initially Estimated","Estimation of country-level basic reproductive ratios for novel Coronavirus (COVID-19) using synthetic contact matrices","Insight into 2019 novel coronavirus \u2014 an updated intrim review and lessons from SARS-CoV and MERS-CoV","Evaluating new evidence in the early dynamics of the novel coronavirus COVID-19 outbreak in Wuhan, China with real time domestic traffic and potential asymptomatic transmissions","CHAPTER 4 Keeping Track of Viruses","The reproductive number of COVID-19 is higher compared to SARS coronavirus","Time-varying transmission dynamics of Novel Coronavirus Pneumonia in China","The within-host viral kinetics of SARS-CoV-2","Lessons drawn from China and South Korea for managing COVID-19 epidemic: insights from a comparative modeling study","Emerging viral respiratory tract infections\u2014environmental risk factors and transmission","Les zoonoses (tableaux synth\u00e9tiques: animaux r\u00e9servoirs de pathog\u00e8nes et modes de transmission)","Estimation of incubation period distribution of COVID-19 using disease onset forward time: a novel cross-sectional and forward follow-up study","Nowcasting and forecasting the potential domestic and international spread of the 2019-nCoV outbreak originating in Wuhan, China: a modelling study","The value of early transmission dynamic studies in emerging infectious diseases","Extended SIR prediction of the epidemics trend of COVID-19 in Italy and compared with Hunan, China","Transmission of COVID-19 in the terminal stage of incubation period: a familial cluster","Transmission characteristics of the COVID-19 outbreak in China: a study driven by data","An updated estimation of the risk of transmission of the novel coronavirus (2019-nCov)","Modeling the Comparative Impact of Individual Quarantine vs. Active Monitoring of Contacts for the Mitigation of COVID-19","MERS coronavirus: Data gaps for laboratory preparedness","Estimation of the Time-Varying Reproduction Number of COVID-19 Outbreak in China","A Generalized Discrete Dynamic Model for Human Epidemics","The reproductive number R0 of COVID-19 based on estimate of a statistical time delay dynamical system","Incubation Period Duration and Severity of Clinical Disease Following Severe Acute Respiratory Syndrome Coronavirus Infection","Projecting the transmission dynamics of SARS-CoV-2 through the post-pandemic period","Incubation Period and Other Epidemiological Characteristics of 2019 Novel Coronavirus Infections with Right Truncation: A Statistical Analysis of Publicly Available Case Data","Incubation Period and Other Epidemiological Characteristics of 2019 Novel Coronavirus Infections with Right Truncation: A Statistical Analysis of Publicly Available Case Data","Prediction of New Coronavirus Infection Based on a Modified SEIR Model","Characterizing the transmission and identifying the control strategy for COVID-19 through epidemiological modeling"],"url":["https://doi.org/10.1101/2020.03.03.20029983","https://doi.org/10.1101/2020.03.21.20040329","https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7029449/","https://doi.org/10.1101/2020.03.05.20031815","https://doi.org/10.1016/s0140-6736(20)30528-6","https://doi.org/10.1101/2020.03.06.20031955","https://doi.org/10.1101/2020.02.03.20019497","https://doi.org/10.1016/j.ijid.2020.02.060","https://doi.org/10.1101/2020.03.01.20028944","https://doi.org/10.1101/2020.03.27.20045195","https://doi.org/10.1101/2020.03.17.20037481","https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4776270/","https://doi.org/10.1101/2020.02.21.20026328","https://doi.org/10.1101/2020.01.30.20019877","https://doi.org/10.1101/2020.02.06.20020941","https://doi.org/10.1101/2020.02.19.20025452","https://doi.org/10.1016/j.ijid.2020.02.033","https://doi.org/10.1101/2020.02.16.20023655","https://doi.org/10.1101/2020.03.10.20033852","https://doi.org/10.1101/2020.03.03.20028423","https://doi.org/10.1101/2020.03.09.20033514","https://doi.org/10.1101/2020.02.07.20021154","https://doi.org/10.1101/2020.02.26.20028167","https://doi.org/10.1016/j.ijid.2020.03.071","https://doi.org/10.1101/2020.02.15.20023440","https://doi.org/10.1016/b978-012088483-4/50007-x","https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7074654/","https://doi.org/10.1101/2020.01.25.919787","https://doi.org/10.1101/2020.02.29.965418","https://doi.org/10.1101/2020.03.09.20033464","https://doi.org/10.1016/s1473-3099(14)70831-x","https://doi.org/10.1016/s1773-035x(15)30318-x","https://doi.org/10.1101/2020.03.06.20032417","https://doi.org/10.1016/s0140-6736(20)30260-9","https://doi.org/10.1016/s1473-3099(20)30161-4","https://doi.org/10.1101/2020.03.18.20038570","https://doi.org/10.1016/j.ijid.2020.03.027","https://doi.org/10.1101/2020.02.26.20028431","https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7029158/","https://doi.org/10.1101/2020.03.05.20031088","https://doi.org/10.1016/j.jcv.2013.10.030","https://doi.org/10.1101/2020.02.08.20021253","https://doi.org/10.1101/2020.02.11.944728","https://doi.org/10.1101/2020.02.17.20023747","http://europepmc.org/articles/pmc4889459?pdf=render","https://doi.org/10.1101/2020.03.04.20031112","https://doi.org/10.3390/jcm9020538","https://doi.org/10.1101/2020.01.26.20018754","https://doi.org/10.1101/2020.03.03.20030858","https://doi.org/10.1101/2020.02.24.20026773"],"x":[0.27138272751771886,0.269049345303822,0.26848711656282787,0.2697908845861721,0.2906716659825433,0.2685666957563967,0.27324864929095566,0.2734658780602419,0.2743730795034525,0.3289544392925557,0.4074411011366128,0.2843681525880758,0.20787451561713546,0.28716744769899005,0.2584923298078653,0.2742667284866039,0.2891747345161611,0.30477224963828725,0.20251369490906543,0.293761483438522,0.27621588356527055,0.2635676798441711,0.3498942523206507,0.8789018194000207,0.24996368044600883,0.41108454380901754,0.27741187574083537,0.22314928948999027,0.7543962612981764,0.3132180406502219,0.3477648595379171,0.06211669747096601,0.26289689807488975,0.20204879780060642,0.39744204670623057,0.32123498482146007,0.2809182210447256,0.2641786187143566,0.2631797985970801,0.3258444696823588,0.955876053889151,0.22324704379964366,0.3124283229937648,0.3068656671443599,0.2618591984801076,0.7099356460452726,0.25872560667289907,0.25872875629916725,0.24349995878898095,0.28584793154430466],"y":[0.9763730006411795,0.9878447029157573,0.9273217038989815,0.9763532111974572,0.9856579774882194,0.9926592428051535,0.9806596194366796,0.9809265581247284,0.9900269388462348,0.831678763227515,0.8437807617157862,0.8891361894038744,0.819490945652831,0.9049662775966406,0.9674491770901809,0.986758957760178,0.920094545566584,0.8136345507090841,0.8153426341611845,0.9849446336826883,0.9522270672863696,0.9176286933578937,0.7887458487042894,0.38837210374663683,0.7279153191388564,0.21068323151696952,0.9079127734157397,0.8525237411938494,0.4773529468276691,0.9059820627736552,0.2549474196830394,0.3388185505986132,0.9932828047732559,0.7743872458477683,0.8598920674663141,0.8971353386360776,0.9858066252221257,0.9879283744990635,0.9163946003084641,0.7747425968335285,0.46576555005176173,0.8489620140979446,0.8221307456131597,0.8803459081309679,1.0,0.45847068135160085,0.99070870220042,0.9907087913420584,0.844421349736821,0.6994082769939944]},"selected":{"id":"6869"},"selection_policy":{"id":"6890"}},"id":"6797","type":"ColumnDataSource"},{"attributes":{},"id":"6841","type":"LinearScale"},{"attributes":{"source":{"id":"6798"}},"id":"6862","type":"CDSView"},{"attributes":{},"id":"6800","type":"Range1d"},{"attributes":{},"id":"6809","type":"BasicTicker"},{"attributes":{},"id":"6844","type":"BasicTicker"},{"attributes":{"source":{"id":"6798"}},"id":"6868","type":"CDSView"},{"attributes":{},"id":"6813","type":"BasicTicker"},{"attributes":{},"id":"6804","type":"LinearScale"},{"attributes":{"data_source":{"id":"6797"},"glyph":{"id":"6824"},"hover_glyph":null,"muted_glyph":null,"nonselection_glyph":{"id":"6825"},"selection_glyph":null,"view":{"id":"6827"}},"id":"6826","type":"GlyphRenderer"},{"attributes":{},"id":"6893","type":"BasicTickFormatter"},{"attributes":{},"id":"6848","type":"BasicTicker"},{"attributes":{"source":{"id":"6797"}},"id":"6833","type":"CDSView"},{"attributes":{"children":[{"id":"6832"},{"id":"6799"}]},"id":"6878","type":"Row"},{"attributes":{"axis":{"id":"6847"},"dimension":1,"ticker":null},"id":"6850","type":"Grid"},{"attributes":{"axis":{"id":"6812"},"dimension":1,"ticker":null},"id":"6815","type":"Grid"},{"attributes":{"children":[{"id":"6873"}]},"id":"6876","type":"Row"},{"attributes":{"children":[{"id":"6876"},{"id":"6877"},{"id":"6878"},{"id":"6879"},{"id":"6880"}]},"id":"6881","type":"Column"},{"attributes":{"children":[{"id":"6867"},{"id":"6834"}]},"id":"6880","type":"Row"},{"attributes":{"below":[{"id":"6843"}],"center":[{"id":"6846"},{"id":"6850"}],"left":[{"id":"6847"}],"plot_height":340,"plot_width":900,"renderers":[{"id":"6861"}],"title":{"id":"6884"},"toolbar":{"id":"6854"},"toolbar_location":"below","x_range":{"id":"6835"},"x_scale":{"id":"6839"},"y_range":{"id":"6837"},"y_scale":{"id":"6841"}},"id":"6834","subtype":"Figure","type":"Plot"},{"attributes":{"callback":null},"id":"6851","type":"TapTool"},{"attributes":{"axis":{"id":"6808"},"ticker":null},"id":"6811","type":"Grid"},{"attributes":{"below":[{"id":"6808"}],"center":[{"id":"6811"},{"id":"6815"}],"left":[{"id":"6812"}],"plot_height":340,"plot_width":900,"renderers":[{"id":"6826"}],"title":{"id":"6882"},"toolbar":{"id":"6819"},"toolbar_location":"below","x_range":{"id":"6800"},"x_scale":{"id":"6804"},"y_range":{"id":"6802"},"y_scale":{"id":"6806"}},"id":"6799","subtype":"Figure","type":"Plot"},{"attributes":{"formatter":{"id":"6893"},"ticker":{"id":"6813"}},"id":"6812","type":"LinearAxis"},{"attributes":{},"id":"6852","type":"ResetTool"},{"attributes":{"children":[{"id":"6874"}]},"id":"6877","type":"Row"},{"attributes":{},"id":"6806","type":"LinearScale"},{"attributes":{},"id":"6802","type":"Range1d"},{"attributes":{"axis":{"id":"6843"},"ticker":null},"id":"6846","type":"Grid"},{"attributes":{"formatter":{"id":"6905"},"ticker":{"id":"6844"}},"id":"6843","type":"LinearAxis"},{"attributes":{"fill_color":{"field":"color"},"line_color":{"field":"color"},"radius":{"field":"rad","units":"data"},"x":{"field":"x"},"y":{"field":"y"}},"id":"6824","type":"Circle"},{"attributes":{"data_source":{"id":"6798"},"glyph":{"id":"6859"},"hover_glyph":null,"muted_glyph":null,"nonselection_glyph":{"id":"6860"},"selection_glyph":null,"view":{"id":"6862"}},"id":"6861","type":"GlyphRenderer"},{"attributes":{"formatter":{"id":"6895"},"ticker":{"id":"6809"}},"id":"6808","type":"LinearAxis"},{"attributes":{},"id":"6900","type":"UnionRenderers"},{"attributes":{"fill_alpha":{"value":0.1},"fill_color":{"field":"color"},"line_alpha":{"value":0.1},"line_color":{"field":"color"},"radius":{"field":"rad","units":"data"},"x":{"field":"x"},"y":{"field":"y"}},"id":"6860","type":"Circle"},{"attributes":{"callback":null,"tooltips":[["Title","@title"],["Abstract","@text"],["Published","@date"],["Authors","@authors"],["Journal","@journal"],["URL","@url"]]},"id":"6818","type":"HoverTool"},{"attributes":{"active_drag":"auto","active_inspect":"auto","active_multi":null,"active_scroll":"auto","active_tap":"auto","tools":[{"id":"6851"},{"id":"6852"},{"id":"6853"}]},"id":"6854","type":"Toolbar"},{"attributes":{"source":{"id":"6797"}},"id":"6827","type":"CDSView"},{"attributes":{},"id":"6839","type":"LinearScale"},{"attributes":{},"id":"6887","type":"StringEditor"},{"attributes":{},"id":"6888","type":"StringFormatter"},{"attributes":{"data":{"authors":["Shujuan Ma; Jiayue Zhang; Minyan Zeng; Qingping Yun; Wei Guo; Yixiang Zheng; Shi Zhao; Maggie H Wang; Zuyao Yang","Yuke Wang; Peter F.M. Teunis","Cowling, Benjamin J; Leung, Gabriel M","Lauren Tindale; Michelle Coombe; Jessica E Stockdale; Emma Garlock; Wing Yin Venus Lau; Manu Saraswat; Yen-Hsiang Brian Lee; Louxin Zhang; Dongxuan Chen; Jacco Wallinga; Caroline Colijn","Ying Wen; Lan Wei; Yuan Li; Xiujuan Tang; Shuo Feng; Kathy Leung; Xiaoliang Wu; Xiong-Fei Pan; Cong Chen; Junjie Xia; Xuan Zou; Tiejian Feng; Shujiang Mei","Wenlei Xiao; Qiang Liu; J Huan; Pengpeng Sun; Liuquan Wang; Chenxin Zang; Sanying Zhu; Liansheng Gao","Tang, Biao; Bragazzi, Nicola Luigi; Li, Qian; Tang, Sanyi; Xiao, Yanni; Wu, Jianhong","Wu, Di; Wu, Tiantian; Liu, Qun; Yang, Zhicong","Joel Hellewell; Sam Abbott; Amy Gimma; Nikos I Bosse; Christopher I Jarvis; Timothy W Russell; James D Munday; Adam J Kucharski; W John Edmunds; CMMID nCoV working group; Sebastian Funk; Rosalind M Eggo","Mackenzie, John S; Smith, David W","Joshua L Santarpia; Danielle N Rivera; Vicki Herrera; M. Jane Morwitzer; Hannah Creager; George W. Santarpia; Kevin K Crown; David Brett-Major; Elizabeth Schnaubelt; M. Jana Broadhurst; James V. Lawler; St. Patrick Reid; John J. Lowe","Chong You; Yuhao Deng; Wenjie Hu; Jiarui Sun; Qiushi Lin; Feng Zhou; Cheng Heng Pang; Yuan Zhang; Zhengchao Chen; Xiao-Hua Zhou","Corey M Peak; Rebecca Kahn; Yonatan H Grad; Lauren M Childs; Ruoran Li; Marc Lipsitch; Caroline O Buckee","Lauren Tindale; Michelle Coombe; Jessica E Stockdale; Emma Garlock; Wing Yin Venus Lau; Manu Saraswat; Yen-Hsiang Brian Lee; Louxin Zhang; Dongxuan Chen; Jacco Wallinga; Caroline Colijn","Wu, Joseph T; Leung, Kathy; Leung, Gabriel M","Shaun A Truelove; Orit Abrahim; Chiara Altare; Andrew Azman; Paul B Spiegel","Bushra Zareie; Amin Roshani; Mohammad Ali Mansournia; Mohammad Aziz Rasouli; Ghobad Moradi","Hellewell, Joel; Abbott, Sam; Gimma, Amy; Bosse, Nikos I; Jarvis, Christopher I; Russell, Timothy W; Munday, James D; Kucharski, Adam J; Edmunds, W John; Funk, Sebastian; Eggo, Rosalind M","Shaw, Karen","Wilder-Smith, Annelies; Chiew, Calvin J; Lee, Vernon J","Shujuan Ma; Jiayue Zhang; Minyan Zeng; Qingping Yun; Wei Guo; Yixiang Zheng; Shi Zhao; Maggie H Wang; Zuyao Yang","Joel Hellewell; Sam Abbott; Amy Gimma; Nikos I Bosse; Christopher I Jarvis; Timothy W Russell; James D Munday; Adam J Kucharski; W John Edmunds; CMMID nCoV working group; Sebastian Funk; Rosalind M Eggo","Wei Luo; Maimuna S Majumder; Dianbo Liu; Canelle Poirier; Kenneth D Mandl; Marc Lipsitch; Mauricio Santillana","Nishiura, Hiroshi; Linton, Natalie M.; Akhmetzhanov, Andrei R.","Nishiura, Hiroshi; Linton, Natalie M.; Akhmetzhanov, Andrei R.","Sun, Jiumeng; He, Wan-Ting; Wang, Lifang; Lai, Alexander; Ji, Xiang; Zhai, Xiaofeng; Li, Gairu; Suchard, Marc A.; Tian, Jin; Zhou, Jiyong; Veit, Michael; Su, Shuo","Jonathan M Read; Jessica RE Bridgen; Derek AT Cummings; Antonia Ho; Chris P Jewell","Maltezou, Helena C.; Tsiodras, Sotirios","Xi He; Eric HY Lau; Peng Wu; Xilong Deng; Jian Wang; Xinxin Hao; Yiu Chung Lau; Jessica Y Wong; Yujuan Guan; Xinghua Tan; Xiaoneng Mo; Yanqing Chen; Baolin Liao; Weilie Chen; Fengyu Hu; Qing Zhang; Mingqiu Zhong; Yanrong Wu; Lingzhai Zhao; Fuchun Zhang; Benjamin J Cowling; Fang Li; Gabriel M Leung","Shujuan Ma; Jiayue Zhang; Minyan Zeng; Qingping Yun; Wei Guo; Yixiang Zheng; Shi Zhao; Maggie H Wang; Zuyao Yang","Corey M Peak; Rebecca Kahn; Yonatan H Grad; Lauren M Childs; Ruoran Li; Marc Lipsitch; Caroline O Buckee","Qihui Yang; Chunlin Yi; Aram Vajdi; Lee W Cohnstaedt; Hongyu Wu; Xiaolong Guo; Caterina M Scoglio","Corey M Peak; Rebecca Kahn; Yonatan H Grad; Lauren M Childs; Ruoran Li; Marc Lipsitch; Caroline O Buckee","Poon, Leo L. M.; Peiris, Malik","Lauren Tindale; Michelle Coombe; Jessica E Stockdale; Emma Garlock; Wing Yin Venus Lau; Manu Saraswat; Yen-Hsiang Brian Lee; Louxin Zhang; Dongxuan Chen; Jacco Wallinga; Caroline Colijn","Hiroshi Nishiura; Natalie M Linton; Andrei R. Akhmetzhanov","Marek Kochanczyk; Frederic Grabowski; Tomasz Lipniacki","Shujuan Ma; Jiayue Zhang; Minyan Zeng; Qingping Yun; Wei Guo; Yixiang Zheng; Shi Zhao; Maggie H Wang; Zuyao Yang","Christian Alvin H Buhat; Monica C Torres; Yancee H Olave; Maica Krizna A Gavina; Edd Francis O Felix; Gimelle B Gamilla; Kyrell Vann B Verano; Ariel L Babierra; Jomar Fajardo Rabajante","Corey M Peak; Rebecca Kahn; Yonatan H Grad; Lauren M Childs; Ruoran Li; Marc Lipsitch; Caroline O Buckee","Pengpeng Shi; Shengli Cao; Peihua Feng","Shujuan Ma; Jiayue Zhang; Minyan Zeng; Qingping Yun; Wei Guo; Yixiang Zheng; Shi Zhao; Maggie H Wang; Zuyao Yang","Xie, Mingxuan; Chen, Qiong","Wei Xia; Jiaqiang Liao; Chunhui Li; Yuanyuan Li; Xi Qian; Xiaojie Sun; Hongbo Xu; Gaga Mahai; Xin Zhao; Lisha Shi; Juan Liu; Ling Yu; Meng Wang; Qianqian Wang; Asmagvl Namat; Ying Li; Jingyu Qu; Qi Liu; Xiaofang Lin; Shuting Cao; Shu Huan; Jiying Xiao; Fengyu Ruan; Hanjin Wang; Qing Xu; Xingjuan Ding; Xingjie Fang; Feng Qiu; Jiaolong Ma; Yu Zhang; Aizhen Wang; Yuling Xing; Shunqing Xu","Peng, Philip W.H.; Ho, Pak-Leung; Hota, Susy S.","Manuel Adrian Acuna-Zegarra; Andreu Comas-Garcia; Esteban Hernandez-Vargas; Mario Santana-Cibrian; Jorge X. Velasco-Hernandez","Jonathan M Read; Jessica RE Bridgen; Derek AT Cummings; Antonia Ho; Chris P Jewell","Zhai, Pan; Ding, Yanbing; Wu, Xia; Long, Junke; Zhong, Yanjun; Li, Yiming","Lauren Tindale; Michelle Coombe; Jessica E Stockdale; Emma Garlock; Wing Yin Venus Lau; Manu Saraswat; Yen-Hsiang Brian Lee; Louxin Zhang; Dongxuan Chen; Jacco Wallinga; Caroline Colijn","Sang Woo Park; Daniel M Cornforth; Jonathan Dushoff; Joshua S Weitz"],"color":["#ef8a62","#ef8a62","#ef8a62","#ef8a62","#ef8a62","#ef8a62","#ef8a62","#ef8a62","#ef8a62","#ef8a62","#ef8a62","#ef8a62","#ef8a62","#ef8a62","#ef8a62","#ef8a62","#ef8a62","#ef8a62","#ef8a62","#ef8a62","#ef8a62","#ef8a62","#ef8a62","#ef8a62","#ef8a62","#ef8a62","#ef8a62","#ef8a62","#ef8a62","#ef8a62","#ef8a62","#ef8a62","#ef8a62","#ef8a62","#ef8a62","#ef8a62","#ef8a62","#ef8a62","#ef8a62","#ef8a62","#ef8a62","#ef8a62","#ef8a62","#ef8a62","#ef8a62","#ef8a62","#ef8a62","#ef8a62","#ef8a62","#ef8a62"],"date":["2020-03-24","2020-03-16","2020-02-13","2020-03-06","2020-03-23","2020-03-17","2020-02-11","2020-03-12","2020-02-11","2020-03-17","2020-03-26","2020-02-11","2020-03-08","2020-03-06","2020-03-06","2020-03-30","2020-03-23","2020-02-28","2006-01-31","2020-03-05","2020-03-24","2020-02-11","2020-02-17","2020","2020","2020-03-21","2020-01-24","2014-12-31","2020-03-18","2020-03-24","2020-03-08","2020-03-30","2020-03-08","2020-02-27","2020-03-06","2020-02-13","2020-03-17","2020-03-24","2020-03-30","2020-03-08","2020-02-20","2020-03-24","2020-04-01","2020-03-08","2020-02-27","2020-03-31","2020-01-24","2020-03-28","2020-03-06","2020-03-13"],"doc":["56zhxd6e","j181i5pr","belin2jq","66ulqu11","ddq2q1pg","2v5wkjrq","thu861hj","5fg87lvu","a8ig607t","n8e2n30b","xs7vkm19","juiomzb3","e2p46wa8","66ulqu11","32v44sw9","6njag0dq","p42cgpf0","ueb7mjnv","s00gvgwx","l5ogbl5p","56zhxd6e","a8ig607t","zvngy7zz","8n4vklsf","8n4vklsf","9il7coyk","mt8h4rk5","08a1oin2","eflwztji","56zhxd6e","e2p46wa8","kcb68hue","e2p46wa8","nugginc9","66ulqu11","vspnuxz9","s956fh59","56zhxd6e","xdjgjeb9","e2p46wa8","c800ynvc","56zhxd6e","juz9jnfk","v3gww4iv","4zg3ms5b","aiq6ejcq","mt8h4rk5","u7zxlgxz","66ulqu11","50zrnau8"],"journal":["","","Euro Surveill","","","","Infect Dis Model","International Journal of Infectious Diseases","","Microbiol Aust","","","","","The Lancet","","","Lancet Glob Health","Public Health","The Lancet Infectious Diseases","","","","International Journal of Infectious Diseases","International Journal of Infectious Diseases","Trends in Molecular Medicine","","American Journal of Infection Control","","","","","","Nat Med","","","","","","","","","International Journal of Infectious Diseases","","British Journal of Anaesthesia","","","International Journal of Antimicrobial Agents","",""],"rad":[0.02,0.014973359213288021,0.01296170997580378,0.012947964716738234,0.011447080933326807,0.011147898951945647,0.01015021177937185,0.009610807140297602,0.009324944751130392,0.008711864653731718,0.00865228750462969,0.00839590384578998,0.008057554357583884,0.007669467961178546,0.007418356226846287,0.006983775520355545,0.006952332470262076,0.005641354007407692,0.005619325553396403,0.005364539974318396,0.005185850218367019,0.00513809314217425,0.0049381653770171514,0.00490218996451058,0.00490218996451058,0.004810502743128447,0.004763693311068975,0.004510823056596332,0.004368088347276645,0.004318140345321523,0.004307982814870298,0.004068004426615527,0.004035551991247995,0.0038927169762173687,0.0037632611052254373,0.0036272600819031507,0.0035143708818074952,0.0035007978731657315,0.0034107343577056836,0.003405212831720866,0.003400259656050416,0.0033963607174182795,0.00325371692596329,0.0032265605561528315,0.0032174771542762895,0.003215526316047799,0.0031971193464555743,0.003160507472160798,0.003052976145770414,0.003],"scr":[1.0,0.704,0.586,0.585,0.497,0.479,0.421,0.389,0.372,0.336,0.332,0.317,0.298,0.275,0.26,0.234,0.232,0.155,0.154,0.139,0.129,0.126,0.114,0.112,0.112,0.107,0.104,0.089,0.08,0.078,0.077,0.063,0.061,0.053,0.045,0.037,0.03,0.029,0.024,0.024,0.024,0.023,0.015,0.013,0.013,0.013,0.012,0.009,0.003,0.0],"text":["Incubation period and serial interval are also key parameters for epidemiological modelling in predicting the transmission dynamics, including the basic reproduction number (R0). R0 is defined as the average number of secondary cases caused by a single infectious individual in a totally susceptible population. 18 Estimation of R0 may require the knowledge of latent period and infectious period, 19, 20 which are both difficult to measure directly. For a disease that is not infectious until onset of symptoms, latent period is equivalent to incubation period, and infectious period can commonly be approximated by the difference of serial interval minus incubation period. 18 However, for COVID-19, the latent period appears to be shorter than incubation period as pre-symptomatic transmission might occur, 16, 17 and this should be accounted for when estimating R0. To our best knowledge, no published studies estimated the latent period of COVID-19, or how often and approximately at which time point the disease could be transmitted prior to the symptomatic onset.","Spreading patterns of COVID-19 at different spatial scales, from local (railway department and department store) to regional (cities within Guangdong Province) to nationwide (Provinces within China), appear similar: new cases arise from contacts with a reservoir, spreading to remote locations through travel from the origin outbreak region. There likely is some secondary spread locally, from cases infected from the reservoir, but local reproduction numbers are low, often insufficient to support sustained transmission. Knowledge of the serial interval, for symptomatic cases in the outbreak, is essential for analysis of the transmission probabilities [14] . Previous studies have estimated the distribution of serial intervals from a set of confirmed transmission pairs, where both the ancestor (who caused infection) and the descendant (who became infected) were known [19, 20] . As such information is mostly lacking [6] (and may be hard to obtain) in this COVID-19 outbreak, we have estimated the serial interval distribution from a curated network, using whatever contact information was available. Existing knowledge of contacts (and timing of those contacts) between cases was used to construct a matrix of prior probabilities, to restrict the transmission network to only those links that are possible. When all transmission links are known, the serial interval distribution can be easily estimated. However, at early stages of an outbreak, information on transmission links is usually incomplete. When a sufficiently large proportion of the transmission links is known, it is possible to estimate both the serial interval distribution, and the transmission probability matrix [14] . Starting from a plausible serial interval distribution, the transmission probability matrix V is updated until convergence; then V is frozen and the serial interval parameters are updated, again until obtaining a new optimum; then the serial interval distribution is frozen and the transmission probability matrix is updated again, and so on, until no further improvement can be found. The estimated serial interval distribution, Gamma(3\u00b716,1\u00b752), has a mean of 4\u00b78 days and good spread out (i.e. large scale parameter) to long serial interval. The variation of serial interval could caused by highly varied incubation period [21] . Such a short serial interval, compared to SARS (mean: 8\u00b74 days) [22] and MERS (mean: 6\u00b78 days) [23], gives COVID-19 ability to spread more rapidly. The rapid spread of COVID-19 in South Korea (from 31 cases on 18 February to 2,022 cases on 28 February) and Italy (from 20 cases on 21 February to 650 cases on 28 February), shows how missed infectious subjects may cause rapid transmission within a very short period, due to the combination of a short serial interval and an occasionally high reproduction number [24] . As the incubation period seems to be highly variable, it may be possible that appearance of symptoms in any case precedes symptom onset in its ancestor. When that happens, the serial interval for symptom onset is negative. To check whether negative serial intervals would adversely affect analysis we used an alternative distribution, where the serial intervals were shifted leftward by a small amount. A shift of one or two days had no destructive effect on estimation of V , and the resulting estimates of the effective reproduction numbers did not change substantially.","Infections with 2019-nCoV can spread from person to person, and in the earliest phase of the outbreak the basic reproductive number was estimated to be around 2.2, assuming a mean serial interval of 7.5 days [2]. The serial interval was not precisely estimated, and a potentially shorter mean serial interval would have corresponded to a slightly lower basic reproductive number. Control measures and changes in population behaviour later in January should have reduced the effective reproductive number. However, it is too early to estimate whether the effective reproductive number has been reduced to below the critical threshold of 1 because cases currently being detected and reported would have mostly been infected in mid- to late-January. Average delays between infection and illness onset have been estimated at around 5\u20136 days, with an upper limit of around 11-14 days [2,5], and delays from illness onset to laboratory confirmation added a further 10 days on average [2].","In conclusion, in both the Singapore and Tianjin COVID-19 clusters we identified both the incubation period and the serial interval. Our results suggest that there is substantial pre-symptomatic transmission, with the serial interval shorter than incubation period by 2-4 days. We find differences in estimated incubation period between early and later cases; this may be due to presymptomatic transmission or differences in reporting and/or in perceived exposure as the outbreak progressed, in the context of social distancing measures. Finally, our shorter serial intervals lead to an estimate of the basic reproduction number of approximately 2 in both datasets, suggesting that stopping half the transmission events may be sufficient to control outbreaks.","Demographic and clinical characteristics as categorical variables were presented as numbers and percentages, while continuous variables were presented as means and standard deviations or 95% confidence intervals (CI), or medians and interquartile ranges (IQR) if appropriate. Inter-group differences in the characteristics were tested by using Pearson's \u03c7 2 test or Fisher's exact test for categorical variables, and by using Student's t-test or analysis of variance for continuous variables showing a normal distribution, and Kruskal-Wallis and Wilcoxon tests for continuous variables with non-parametric distribution. The incubation period was estimated by using a previously described parametric accelerated failure time model (10). Patients with detailed information on the time of exposure, the date of illness onset, or the first time of presentation were included for this analysis. We fitted lognormal, gamma, and Weilbull distributions using Markov Chain Monto Carlo in a Bayesian framework (11). We estimated the serial interval by using the time difference of illness onset between the infector and infectee. Initial reproductive number was estimated by using the best fit model based on date of illness onset of the early (Jan 10-23) local exposed cases without relation to the imported cases and the estimated serial interval of COVID-19. Logistic regression models were applied to identify factors associated with the clinical severity of COVID-19. All statistical tests and analyses of the incubation period, serial interval, and initial reproductive number were performed in R software (R foundation for Statistical Computing).","At the beginning, when the novel coronavirus from Wuhan causes concern in public, delayed and missed detection may exist in the reported number of cases. It brings troubles in estimating the epidemiological parameters and epidemic predictions [4, 5] . Most estimated R 0 ranges from 1.5 to 4.0 [1, 6, 7, 8] . As of Jan 29, 2020, the first investigated incubation period from patients was reported, which had a mean of 5.2 days(95% confidence interval [CI], 4.1 to 7.0) and followed a Poisson distribution, and the basic reproductive number was estimated to be 2.2 (95% CI, 1.4 to 3.9) [6] . However, the early sampled 425 patients had a median age of 59 years and 56% were male. Given those bias on the samples, the estimated epidemiological parameters may have deviations that might lead to great errors in the simulation. In the view of this, we used the data from Shanghai, a relatively well controlled city, to identify and calibrate the key parameters of the incubation period and the basic reproductive number. Subsequently, those parameters were used to evaluate the status of other cities (except for those cities in Hubei province). In Shanghai Model, there is no worry about the shortage of medical supplies, so a negative summation channel performs a direct control effect on the positive feedback infection loop, which is thus of paramount importance in reducing the number of total infectious cases. Two factors in the system can be regulated by the administration, C 0 and C 1 . The confirmation delay does not affect the infecting process, but brings a direct hinder in inspecting the real number of confirmed cases. The input of the Shanghai Model mainly comes from the inspected cases from Wuhan, so the number of daily imported cases should be given by R 1 x 1 , where R 1 is the infection rate of immigration, and x 1 is the daily immigrated population from Wuhan. On Jan 23, 2020, Wuhan went into lockdown to contain the outbreak of the epidemic. Since then, the input from immigration was switched off. Those sudden events changed the system dynamics dramatically, and is difficult to be estimated or approximated by other transmission models. Other similar cities can reuse the Shanghai Model with some modification on the parameters C 0 and C 1 . Nonetheless, the 4 . CC-BY-NC-ND 4.0 International license It is made available under a is the author/funder, who has granted medRxiv a license to display the preprint in perpetuity.","There are three further models incorporating data from international travels: the models of Imai and coauthors (Imai et al., 2020), of Kucharski et al. (Kucharski et al., 2020) and of Wu and collaborators (Wu, Leung, &amp; Leung, 2020). In particular, Imai and coworkers (Imai et al., 2020) estimated a reproduction number of 2.6 (uncertainty range: 1.5\u20133.5). Depending on the different scenarios and levels of zoonotic exposure, the reproduction number was found to vary from 1.7 to 2.6 to 1.9\u20134.2. Depending on the different estimates of generation time, the reproduction number oscillated from 1.3 to 2.7 to 1.7\u20134.3. Based on the level of infectiousness, the reproduction number varied in the range of 1.6\u20132.9. Finally, assuming that the novel virus would cause more mild-to-moderate cases than the SARS virus, the reproduction number would be 2.0 (uncertainty 1.4\u20132.3). Moreover, authors found that only public health interventions blocking over 60% of transmission would be really effective in controlling and containing the coronavirus outbreak. Partially based on the findings of Imai and coworkers (Imai et al., 2020) and building on a SIR model, Yu (Yu, 2020) has computed a basic reproduction number of 3.5 and has estimated that only a quarantine rate of infectious population higher than 90% would enable to effectively control the coronavirus outbreak. Kucharski and colleagues (Kucharski et al., 2020) designed a stochastic SEIR model, based on the Euler-Maruyama algorithm with a 6-h time-step and with the transmission rate following a geometric Brownian motion. Time-varying reproduction number was estimated using the sequential Monte-Carlo approach. Authors utilized several datasets to overcome the issue of unreliability of some data sources and to provide real-time estimates, relying on the Poisson probability calculation. Transmission was modeled as a random process, fluctuating and varying over the time. Similar to the model of Imai and coworkers (Imai et al., 2020), the risk of transmission and the risk of causing a large outbreak were modeled based on a negative binomial offspring distribution, with incubation and infectious period being Erlang distributed. Median reproduction number was found to oscillate between 1.6 and 2.9 before the introduction and implementation of travel restriction. The study by Wu and collaborators (Wu et al., 2020), based on nowcasting and forecasting approach, estimated a basic reproductive number of 2.68 (95% credible interval or CrI 2.47\u20132.86) with 75,815 individuals (95% CrI 37,304\u2013130,330) being infected in Wuhan as of January 25th, 2020. The epidemics doubling time was found to be 6.4 days (95% CrI 5.8\u20137.1). The dynamics transmission model by Shen and coworkers (Shen, Peng, Xiao, &amp; Zhang, 2020) predicted 8042 (95% CI 4199\u201311,884) infections and 898 (95% CI 368\u20131429) deaths, with a fatality rate of 11.02% (95% CI 9.26\u201312.78%). Authors computed a basic reproduction number of 4.71 (95% CI 4.50\u20134.92), which decreased to 2.08 (95% CI 1.99\u20132.18) on January 22nd, 2020. Based on these estimates, the pandemics outbreak is expected to significantly decrease within 77 [95% CI 75\u201380] days from its beginning. Furthermore, authors found that every one-day reduction in the duration of the period from illness/symptom onset to isolation would reduce the peak population size by 72\u201384% and the cumulative infected cases and deaths both by 68\u201380%. The study by Majumder and Mandl (Majumder &amp; Mandl, 2020) utilized the \u201cIncidence Decay and Exponential Adjustment\u201d (IDEA) model and led to an estimate of the reproduction number in the range of 2.0\u20133.1. Finally, Riou and Althaus (Riou &amp; Althaus, 2020), using a stochastic model simulating epidemics trajectories, computed a reproduction number of 2.2 (90% high density interval 1.4\u20133.8). Using statistical approaches, namely exponential growth and maximum likelihood techniques, Liu and colleagues (Liu et al., 2020) estimated the value of the reproduction number ranging from 2.90 (95% CI 2.32\u20133.63) to 2.92 (95% CI 2.28\u20133.67). Zhang and Wang (Zhang &amp; Wang, 2020), employing a Bayesian framework to infer time-calibrated phylogeny from 33 available genomic sequences, found that the time of the most recent common ancestor (MRCA) was December 17th, 2019 (95% highest posterior density interval from December 7th, 2019 to December 23rd, 2019) and that the value of the reproduction number oscillated between 1.1 and 1.6.","In recent publications, the mean incubation period of CoVID-19 was a little bit different. Wang et al, with 138 cases, reported that the median durations from first symptoms to dyspnea, hospital admission, and Acute severe respiratory syndrome (ARDS) were 5 days (range, 1-10), 7 days (range, 4-8), and 8 days (range, 6-12), respectively [35] . And Li et al, with 425 confirmed cases, instructed that the mean incubation period was 5.2 days (95% confidence interval [CI], 4.1 to 7.0), with the 95th percentile of the distribution at 12.5 days. In its early stages, the epidemic doubled in size every 7.4 days. With a mean serial interval of 7.5 days (95% CI, 5.3 to 19), the basic reproductive number was estimated to be 2.2 (95% CI, 1.4 to 3.9) [7] . While Guan et al [36] , with 1099 patients, reported that the estimated mean incubation period of SARS-CoV-2 infection was 3.0 days (range, 0 to 24.0), which was shorter than the two researches above (3.0 days vs 5 days and 5.2 days ). The median incubation period of CoVID-19 ARD was 3.0 days and it had a relatively lower fatality rate than SARS and MERS [35] . While the estimated mean incubation period of SARS-CoV infection was 4.6 days (95% CI, 3.8-5.8 days) [37] and 95% of illness onset occurred within 10 days [38] . The mean time from symptom onset to hospitalization was between 2 and 8 days, but was shorter toward the later phase of the epidemic. The mean time from symptom onset to need for invasive mechanical ventilation (IMV) and to death was 11 and 23.7 days, respectively [39] .","The copyright holder for this preprint . https://doi.org/10.1101/2020.02.08.20021162 doi: medRxiv preprint short delay to isolation, 15% of transmission before symptom onset, and 0% subclinical infection. Table 1 . B) The incubation distribution estimate fitted to data from the Wuhan outbreak by Backer et al. 24 . C) An example of the method used to sample the serial interval for a case that has an incubation period of 5 days. Each case has an incubation period drawn from the distribution in B, their serial interval is then drawn from a skewed normal distribution with the mean set to the incubation period of the case. In C, the incubation period was 5 days. The skew parameter of the skewed normal distribution controls the proportion of transmission that occurs before symptom onset, the three scenarios explored are &lt;1% of transmission before onset (grey), 15% of transmission before onset (gold), and 30% of transmission before onset (pink).","There are a number of important questions still to be answered about the transmission dynamics. These include information about the infectivity during the incubation period; the length of time and virus load during incubation and during the symptomatic period of virus shedding; the incidence and infectiousness of asymptomatic cases, the risk of vertical transmission from mother to fetus35, and other modes of transmission, such as from faeces, saliva and urine. There is some evidence that virus can be isolated from saliva36 and while the initial family cluster found no evidence of virus in stools or urine8, it has since been detected in faeces by PCR in other patients30,37,38 and cultured in one patient, but there has been no evidence of virus in urine. The number of mild or asymptomatic cases has not been determined and relatively few cases have been recorded, but it is probable that current figures only see the tip of the iceberg, and many cases remain undiagnosed19. They pose the greatest threat for increased virus spread27. Information is also needed on the stability of the virus in the environment to better determine transmission risks, and especially the survival of the virus in aerosols and on hard surfaces under different conditions of temperature and humidity.","Understanding the modes of transmission of emerging infectious disease is a key factor in protecting healthcare workers and implementing effective public health measures. The lack of evidence on SARS-CoV-2 transmission dynamics has led to shifting isolation guidelines 5 between airborne and droplet isolation precautions by the WHO, U.S. CDC and other public health authorities. Other emerging coronaviruses (e.g. SARS and MERS) have been suggested to have airborne transmission potential (5, 6) in addition to more direct contact and droplet transmission. At least one study suggests that MERS-CoV has the possibility of transmission from mildly ill or asymptomatic individuals (7). Surface samples taken in patient care areas for 10 MERS and SARS have shown positive PCR results (6); however, experts question the possibility of transmission through contact with surfaces that have been contaminated by an infected person, either by the direct contact of the infected person or the settling of virus-laden particles onto the surface(8). Nonetheless, coronaviruses have been implicated in nosocomial outbreaks with reports of transmission related to environmental contamination (9,10). Nosocomial transmission 15 of SARS-CoV-2 has been reported, but the role of aerosol transmission and environmental contamination remains unclear (11).","In this study, serial interval is defined as the time difference between dates of infection of successive cases in a chain of transmission (different textbooks may have different definitions). Infectious period is the duration of which an infected individual can transmit pathogens to a susceptible host. In this study, infectious period is defined as the time difference between date of infection and date of diagnosis as there is strong evidence showing that a diseased individual remains contagious even during the incubation period, and would be immediately isolated upon positive diagnosis hence losing the transmissibility. Both are key quantities that depict an epidemic and are essential to estimate the basic/controlled reproductive number, . CC-BY-NC-ND 4.0 International license It is made available under a author/funder, who has granted medRxiv a license to display the preprint in perpetuity.","Under both serial interval scenarios, in a low feasibility setting, the reproductive number was rarely brought below one under either individual quarantine or active monitoring, though a median reduction in the reproductive number of 21.0% and 13.6%, respectively for serial interval scenario 2, can still meaningfully slow the growth of an epidemic; for serial interval scenario 1, active monitoring resulted in only a 3% reduction in the reproductive number while individual quarantine resulted in a 17% reduction. If the epidemic continues to grow, however, the feasibility and social acceptability of quarantining individuals becomes a crucial consideration. In these circumstances, complementary interventions such as social distancing and pharmaceutical interventions may be needed if efficient contact tracing and rapid isolation are not readily achievable, regardless of the extent of presymptomatic transmission.","is the (which was not peer-reviewed) The copyright holder for this preprint . https://doi.org/10.1101/2020.03.03.20029983 doi: medRxiv preprint Our serial intervals are notably shorter than our incubation period estimates, suggesting that there is pre-symptomatic transmission, with infection occurring on average 2.89 and 2.55 days before symptom onset of the infector (Tianjin, Singapore). Because the incubation period is different for early-and late-occurring cases in our data, on average transmission for early-occurring cases is 2.79 and 1.2 days before symptom onset (Tianjin, Singapore) and 8.2, 3.3 days before (Tianjin, Singapore) for late-occurring cases. The fact that serial intervals are shorter than incubation periods is robust in our sensitivity analysis (Table S3 ). These estimates are strengthened by the fact that we have estimated both incubation period and serial interval in the same population and by the fact that we obtain the same result in two distinct datasets. In both sets of estimates, samples of the incubation period minus serial interval are negative with probability 0.8 or higher (Tianjin) and 0.7 or higher (Singapore), suggesting that a substantial portion of transmission may occur before symptom onset (see Supplementary Information and Figure S2 ), consistent with the clinical observations reported by Rothe et al. [21] and Bai et al. [22] .","where S(t), E(t), I(t), and R(t) were the number of susceptible, latent, infectious, and removed individuals at time t; D E and D I were the mean latent (assumed to be the same as incubation) and infectious period (equal to the serial interval minus the mean latent period 4 ); R 0 was the basic reproductive number; z(t) was the zoonotic The case definition of 2019-nCoV differs depending on the context in which it is used.","We used a stochastic Susceptible Exposed Infectious Recovered (SEIR) model to simulate transmission in this population. We simulated epidemics under three potential scenarios with different values of the basic reproductive number, R 0 : 1) a low transmission scenario based on transmission levels in many of the Chinese provinces with elevated isolation and control practices and an R 0 similar to influenza (R 0 =1.5-2.0) 14 ; 2) a moderate transmission scenario that mirrors estimates in early stages of the outbreak in Wuhan, China (R 0 =2.0-3.0) 15 ; and, 3) a high transmission scenario where we assume that R 0 is increased by a factor of 1.65 (R 0 =3.3-5.0) compared to estimates from open community settings, as was observed during the 2017 diphtheria outbreak. 16 The R 0 in each of these scenarios falls within the 95% confidence interval of the current range of estimates for COVID-19. 17 We assumed an Erlang distributed serial interval (time between the onset of symptoms in infector-infectee pairs) with a mean of 6 days (standard deviation = 4.2). 18 The proportion of infections that result in severe disease was estimated using data from a closed population of contacts in Shenzhen, China, for which complete data on contact, age, test results, symptoms, and initial severity were available. 18 Using a Bayesian binomial model, we estimated the proportion of infections that were severe by 10-year age groups and then applied those estimates to the Kutupalong-Balukhali population to get an overall proportion of infections that will develop into severe disease in the specific population. These estimates do not account for differences in comorbidities, differing attack rates by age due to population mixing characteristics, or various other factors.","March 2020 because of the rapid outbreak of the disease worldwide (3) . The course of each epidemic depends on a number of important factors such as the Basic Reproductive Rate, the doubling time or serial interval, and the fatality rate. Estimating the epidemiological factors of COVID-19 and modeling-based prediction are useful in assessing epidemic transmission rates, predicting epidemic trends, and designing control measures (4, 5) .","The incubation period for each case was drawn from a Weibull distribution. A corresponding serial interval for each case was then drawn from a skewed normal distribution with the mean parameter of the distribution set to the incubation period for that case, an SD of 2, and a skew parameter chosen such that a set proportion of serial intervals were shorter than the incubation period (meaning that a set proportion of transmission happened before symptom onset; figure 2). This sampling approach ensured that the serial interval and incubation period for each case was correlated, and prevented biologically implausible scenarios where a case could develop symptoms soon after exposure, but not become infectious until very late after exposure and vice versa.","The causative agent of SARS was identified as a novel coronavirus, the SARS coronavirus (SARS-CoV) . 8, 9 Epidemiological data indicate that the virus is largely spread by direct or indirect contact with respiratory droplets, airborne transmission has also been reported. 10, 11 Initially, as a result of early reports of high infection rates among HCWs, SARS was thought to be highly contagious. This was reinforced with the reporting of 'super-spreading events'. 2, 5, [12] [13] [14] [15] The concept of super-spreading events has been proposed to explain incidents where a SARS patient infects many more persons than would be expected. Such events contributed to large-scale nosocomial outbreaks and have been attributed to the lack of stringent IC measures in hospitals. However, this does not fully explain these events and further research is required to understand the underlying risk factors. Aerosol-generating procedures have been associated with increased risk of transmission through the aerosolisation of virus-containing particles. 16 In addition patients with unrecognised SARS, particularly those with atypical presentations were the source of a number of nosocomial outbreaks. 13, 15 As the outbreak evolved SARS was shown to be only moderately transmissible with an estimated basic reproductive number of approximately three. 17, 18 The basic reproductive number refers to the average number of secondary cases generated by one primary case in a susceptible population. For SARS this is much lower than for influenza, and unlike influenza and other viral respiratory diseases, SARS-CoV has a long incubation period (up to 14 days), with the viral load peaking during the second week of illness and appearing to be highest in patients with more severe disease. 2 This may explain why transmission has not been identified in asymptomatic patients or after symptoms have resolved, and why transmission mainly occurs in hospitals.","In November, 2002, the severe acute respiratory syn drome coronavirus (SARSCoV) emerged in China causing global anxiety as the outbreak rapidly spread, and by July, 2003, had resulted in over 8000 cases in 26 countries. In December, 2019, a novel coronavirus, named SARSCoV2, emerged in Wuhan, China, and led to a rapidly spreading outbreak of coronavirus disease 2019 (COVID19). By Jan 30, 2020, COVID19 was declared a public health emergency of international concern. The similarities between SARSCoV and SARS CoV2 are striking, not only in name. The whole genome of SARSCoV2 has a 86% similarity with SARSCoV. 1 Both viruses share high degrees of homology to SARS like coronaviruses isolated in bats, suggesting that bats are the probable origin of both SARSCoV and SARS CoV2. Live animal markets selling multiple species of wild and domestic animals in proximity to large populations of densely housed humans are thought to be the source of both outbreaks. Even in terms of disease dynamics there are apparent similarities. The main transmission route is thought to be respiratory droplets, although viral shedding via faeces has also been reported for both viruses. The angiotensinconverting enzyme 2 (ACE2), found in the lower respiratory tract of humans, has been identified as the receptor used for cell entry for both SARSCoV and SARSCoV2. 2,3 COVID19 and SARS have a median incubation time of about 5 days. The mean serial interval of COVID19 is 7\u00b75 days (95% CI 5\u00b73-19\u00b70) and the initial estimate for the basic number (R\u2080) was 2\u00b72 (95% CI 1\u00b74-3\u00b79), 4 similar to that reported for SARS (mean serial interval 8\u00b74 days, and basic R\u2080 range 2\u00b72-3\u00b76 for serial intervals of 8-12 days). 5 Risk factors for severe disease outcomes are old age and comorbidities. The progression for patients with severe disease follows a similar pattern in both viruses, with progression to acute respiratory distress syndrome approximately 8-20 days after onset of first symptoms, whereby lung abnormalities on chest CT show greatest severity approximately 10 days after initial onset of symptoms. [6] [7] [8] [9] However, the similarities end here. successful and included active case detection, isolation of cases, contact tracing and quarantine of all contacts, social distancing, and community quarantine. \u2022 Whether these measures will also be successful for COVID-19 will not depend on the similarities but the differences between SARS and COVID-19. \u2022 Clear differences are emerging, such as in transmissibility and severity pyramids; COVID-19 has a higher transmissibility than SARS, and many more patients with COVID-19 rather than SARS have mild symptoms that contribute to spread because these patients are often missed and not isolated. \u2022 Because of the extent of community spread, traditional public health measures might not be able to halt all human-to-human transmission, and we need to consider moving from containment to mitigation.","The negative values of serial interval and time point of exposure provide evidence of presymptomatic transmission, which are consistent with the much shorter latent period than incubation period. The percentages of negative serial interval (3.92%) and negative time point of exposure (43.78%) can be viewed as the lower and upper limits of probability of presymptomatic transmission, respectively. Asymptomatic transmission was also observed in this study (Table 3 ). These findings are in line with those of previous studies. 8,9,14-17 Stratified analysis in this study found a trend towards longer incubation period of less severe cases as compared with that of more severe cases. This implies that severe cases could be detected and 'removed' from the pool of transmission sources more quickly, while less severe cases would All rights reserved. No reuse allowed without permission. the author/funder, who has granted medRxiv a license to display the preprint in perpetuity.","The copyright holder for this preprint . https://doi.org/10.1101/2020.02.08.20021162 doi: medRxiv preprint The incubation period for each case was drawn from a Weibull distribution. A corresponding serial interval for each case was then drawn from a skewed normal distribution with the mean parameter of the distribution set to the incubation period for that case, a standard deviation of 2, and a skew parameter chosen such that a set proportion of serial intervals were shorter than the incubation period (meaning that a set proportion of transmission happened before symptom onset) ( Figure 2 ). This sampling approach ensured that the serial interval and incubation period for each case was correlated, and prevents biologically implausible scenarios where a case can develop symptoms very soon after exposure but not become infectious until very late after exposure and vice versa.","where the values of d range from [5 to 7] , consistent with reported estimates of serial intervals (7). Our measure is considered only a proxy for R because it does not use details of the (so far poorly defined) serial interval distribution, but instead, simply calculates the multiplicative increase in the number of incident cases over approximately one serial interval. Such proxies are at least approximately monotonically related to the true reproductive number and cross 1 when the true reproductive number crosses 1 (8) . After computing these proxy values over a variety of subsequent moving time windows, a mean value was obtained and utilized as our estimated reproductive number R for each province. To minimize potential inclusion of imported cases in our analysis, R estimates were calculated after the closure of Wuhan on January 23rd, 2020.","More importantly, the estimated median serial interval is shorter than the preliminary estimates of the mean incubation period (approximately 5 days) [2, 8] . As illustrated in Figure 2 , when the serial interval is shorter than the incubation period, pre-symptomatic transmission is likely to have taken place and may even occur more frequently than symptomatic transmission. A substantial proportion of secondary transmission occurring before illness onset indicates that many transmissions cannot be prevented solely through isolation of symptomatic cases, as by the time contacts are traced they may have already become infectious themselves and generated secondary cases [9] . It is possible that serial intervals were shortened due to case isolation, especially outside of China, but even the subset of data in China alone can indicate that the mean is shorter than 7.5 days.","More importantly, the estimated median serial interval is shorter than the preliminary estimates of the mean incubation period (approximately 5 days) [2, 8] . As illustrated in Figure 2 , when the serial interval is shorter than the incubation period, pre-symptomatic transmission is likely to have taken place and may even occur more frequently than symptomatic transmission. A substantial proportion of secondary transmission occurring before illness onset indicates that many transmissions cannot be prevented solely through isolation of symptomatic cases, as by the time contacts are traced they may have already become infectious themselves and generated secondary cases [9] . It is possible that serial intervals were shortened due to case isolation, especially outside of China, but even the subset of data in China alone can indicate that the mean is shorter than 7.5 days.","Initial evaluation of COVID-19 transmission dynamics showed that the basic reproductive number (R 0 ) of 2019-nCoV is estimated to be 1.4-3.9 [12] . The R 0 of SARS-CoV in the absence of interventions was 2.3-3.7 [22, 23] . Breban et al. estimated MERS-CoV R 0 to be 0.50-0.92 by analysis of 55 of the first 64 laboratory-confirmed cases [24] . With the implementation of rapid diagnosis, coupled with effective isolation of patients, the R 0 of SARS-CoV dropped to less than 1, explaining why the SARS-CoV outbreak could eventually be controlled [25] [26] [27] . However, it is worth noting that R 0 estimates may vary upon numerous biologic, socio-behavioral, and environmental factors, and must be interpreted with caution [28] .","We estimated the transmission rate within Wuhan, , to be 1.94 d -1 (95%CI, 1.25-6.71), while we found the infectious period to be 1.61 days (95%CI, 0.35-3.23). We calculated the basic reproductive number, 0 , of the infection to be 3.11 (95%CI, 2.39-4.13), comparable to the range for SARS estimated from outbreaks during the 2003 epidemic (Lipsitch et al., 2003; World Health Organization, 2003) . We highlight that this number is highly uncertain and that a large range of parameters are consistent with the data given the assumptions of our model. This estimate reflects both the dynamics of transmission and, potentially, the dynamics of case reporting, with increases as reporting over time potentially increasing our estimate. The estimate we give here represents an update from the report of 23 January in which we estimated a higher 0 , and highlights the sensitivity of our findings to additional data at this early period. The current estimate of 0 is significantly greater than 1, the epidemic threshold, providing evidence that sustained human-to-human transmission is occurring in China. Further, it suggests that a concerted effort will be required to control the outbreak, requiring between 38% and 80% of transmission to be averted to control the epidemic.","From the very first events of the MERS-CoV epidemic, the virus showed its health careeassociated dynamic. 6 Apart of sporadic community cases and family clusters, health careeassociated transmission has been reported on several occasions during the last 2 years, indicating human-to-human, although inconsistent, transmission (Table 1) . 2, 3, 6, 8, [22] [23] [24] [25] [26] [27] Gaps in infection control were the common denominator in the events of health care associatede transmission. 2, 3, 6, 8, 22, 24 During the largest so farepublished outbreak of MERS-CoV that occurred in Al-Hasa, Saudi Arabia, in 2013, 4 health care facilities were affected through transfer of patients but also possibly because of repeated introductions of cases from the community. 3 The outbreak extended for almost 2 months and involved 34 cases, including 2 HCWs. Most cases were confined in the hemodialysis unit with rapid transmission and high attack rates. 3 This outbreak gave the opportunity to elucidate several epidemiologic parameters of secondary MERS-CoV infection, such as the incubation period (5.2 days; 95% confidence interval, 1.92-14.7 days), serial interval (7.6 days; 95% confidence interval, 2.5-23.1 days), and heterogeneity in transmission, with many infected patients not transmitting the infection at all and 1 infected patient transmitting the infection to 7 others. 3 Moreover, this outbreak raised the possibility of transmission through direct or indirect contact and between rooms in the same ward. 3 A recent study showed that MERS-CoV remained viable for up to 48 hours under specific environmental conditions, which mimic the hospital environment (20 C with 40% relative humidity), whereas its stability was not reduced during aerosolization. 28 These data show that MERS-CoV has the potential to spread through contact or fomites caused by prolonged survival. A model-based study found that the virus structural characteristics render it very likely to remain viable in the environment for a long period and support fecal-oral transmission. 29 Vomiting and diarrhea are common in patients with MERS-CoV 1,22,27 and may contribute to transmission. The MERS-CoV case imported in France shared his bathroom with the secondary hospital-acquired case, which raises the possibility of spread through stools. 22 MERS-CoV is predominantly shed through respiratory secretions during cough. MERS-CoV has been detected through polymerase chain reaction for up to 16 days in respiratory specimens and stools and up to 13 days in urine. 22, 30, 31 Our knowledge about virus shedding and viral load kinetics throughout the clinical course of ill patients is scarce and therefore can provide limited guidance about the duration of implementation of infection control measures. 31 The possibility of prolonged shedding under an immunocompromised status should also be investigated and considered for infection control purposes.","variation can be summarized by the incubation period distribution and the serial interval distribution respectively. If the observed mean serial interval is shorter than the observed mean incubation period, this indicates that a significant portion of transmission may have occurred before infected persons have developed symptoms. A large proportion of presymptomatic transmission would likely reduce the effectiveness of control or preventive measures that are directly or indirectly initiated by symptom onset, such as isolation, contact tracing for quarantine, and enhanced hygiene or use of face masks for symptomatic persons. Failing to consider pre-symptomatic transmission would likely overestimate the effect of such measures when devising control strategies.","In conclusion, this study obtained robust estimates of several key epidemiological parameters of COVID-19. It provides additional evidence on the mean incubation period of COVID-19, which supports current practice of 14-day quarantine of persons with potential exposure but also suggests that longer monitoring periods might be needed for selected groups. The estimates of serial interval, time point of exposure and latent period provide consistent evidence on pre-symptomatic transmission. Asymptomatic transmission was also observed. This together with asymptomatic transmission and the generally longer incubation and serial interval of less severe cases suggest a high risk of long-term epidemic in the absence of appropriate control measures.","We fit the model assuming short (mean of 4.8 days; scenario 1) versus long (mean of 7.5 days; scenario 2) serial interval estimates. Model fitting by SMC for serial interval scenario 1 (mean = 4.8 days) resulted in mean duration of infectiousness 2.4 days [95% CI 1.0, 6.7]; mean time of peak relative infectiousness at 43% of the duration of infectiousness [0%, 97%]; and a mean time of infectiousness onset 0.77 days before symptom onset [1.98 days before, 0.29 days after] (Table 2, Figure S1 ). The longer serial interval in scenario 2 (mean = 7.5 days) resulted in slower disease dynamics: mean duration of infectiousness 4.8 days [95% CI 1.12, 10.5]; mean time of peak relative infectiousness at 38% of the duration of infectiousness [0%, 97%]; and a mean time of infectiousness onset 0.51 days after symptom onset [0.77 days before, 1.50 after] (Table 2, Figure S2 ). Therefore, given the same incubation period distribution with mean = 5.2 days, a serial interval with mean = 4.8 days is best fit by substantial presymptomatic infectiousness (mean = 20.5% [0%, 91.4%]) while a longer serial interval with mean = 7.5 days is best fit by limited presymptomatic infectiousness (mean = 0.065% [0%, 0.88%]).","Understanding the impact of non-pharmaceutical interventions on the epidemic dynamics is crucial for decision-makers to take proper actions to contain the COVID-19. Since the beginning of the outbreak, scientists have been actively retrieving the epidemiological characteristics such as the time from infection to illness onset (incubation period) and the time from illness onset to hospital admission or isolation (infectious period) (Chen et al., 2020; Guan et al., 2020; Huang et al., 2020; Linton et al., 2020) . Researchers have used the reported cases during the early stage of the epidemic and predicted the epidemic trajectory, the basic reproductive number 0 from which ranges from 2.2 to 6.6 (Read, Bridgen, Cummings, Ho, &amp; Jewell, 2020; Sanche et al., 2020; Zhao et al., 2020) . Several studies explored the effects of the quarantine of Wuhan (X. Shen, Peng, Guo, Xiao, &amp; Zhang, 2020) , travel restrictions (Chinazzi et al., 2020) , and other non-pharmaceutical intervention strategies (Lai et al., 2020; Zhu et al., 2020) on the future transmission dynamics at different geographical scales.","The effectiveness of individual quarantine versus active monitoring, targeted by contact tracing, heavily depends on the assumptions regarding the serial interval, the amount of transmission that occurs prior to symptom onset, and the feasibility setting. Under our fitted disease natural history parameters for serial interval scenario 1, with a shorter mean serial interval and hence substantial presymptomatic infectiousness, individual quarantine was considerably more effective than active monitoring at reducing onward transmission by an infected contact. To offset this relative benefit of individual quarantine compared to active monitoring, perverse incentives to avoid quarantine would have to be correspondingly larger. Both scenarios were fit using an incubation period with mean = 5.2 days among 451 lab-confirmed cases from Wuhan; 12 other recent estimates of the incubation period of COVID-19 include a mean of 6.4 7 . CC-BY 4.0 International license It is made available under a author/funder, who has granted medRxiv a license to display the preprint in perpetuity.","Apart from the early cases2,5, subsequent human infections were caused by sustainable human-to-human transmission. Using the first 425 confirmed cases in Wuhan, Li et al. estimated that the mean incubation period of infection with SARS-CoV-2 was 5.2 days (95% confidence interval (CI), 4.1\u20137.0), with about 95% of the cases developing symptoms within 12.5 days of an exposure5, justifying the current recommendations of a 14-day period for medical observation or quarantine. The reproductive number (R0; the number of secondary cases expected in a completely susceptible population) and the epidemic doubling time were estimated to be 2.2 (95% CI, 1.4\u20133.9) and 7.4 days (95% CI, 4.2\u201314), respectively. Studies from others also have led to broadly similar figures13. These are comparable to those observed during the SARS epidemic. However, transmission of SARS-CoV-2 can occur from patients with mild disease9,10. Whether transmission can occur during the late incubation period remains controversial10. This is in sharp contrast to the transmission pattern observed during SARS, for which transmission rarely occurred until after the 4\u20135 days after symptom onset. Taken together, these findings suggest that the public-health interventions that successfully interrupted the spread of SARS-CoV are unlikely to be as effective in the current outbreak.","We sampled the estimated serial interval and incubation period distributions to estimate the probability that the incubation period minus the serial interval is negative. This could indicate the portion of transmission that occurs before symptom onset, but it makes the strong assumption that incubation period and onset of symptoms occur independently and it does not take the growth curve of 3 . CC-BY-NC 4.0 International license It is made available under a author/funder, who has granted medRxiv a license to display the preprint in perpetuity.","More importantly, the estimated median serial interval is shorter than the 20 preliminary estimates of the mean incubation period (approximately 5 days) 21 [3, 6] . As illustrated in Figure 2 , when the serial interval is shorter than the 22 incubation period, pre-symptomatic transmission is likely to have taken place 23 and may even occur more frequently than symptomatic transmission. A 24 All rights reserved. No reuse allowed without permission. author/funder, who has granted medRxiv a license to display the preprint in perpetuity.","We consider a deterministic Susceptible-Infected-Infectious-Excluded (SIIE) model shown in figure 1. Our model is similar to a commonly used SIR model extension, SEIR [3, 4] ; key differences and their consequences are explained in Discussion. We assume that an individual can be in one of the following states: susceptible (S), infected (i 1 , . . . , i 5 ), infectious (I), or excluded (E). Susceptible individuals get infected by contact with infectious individuals at the rate proportional to the contact rate, r C . Multiple infected states are introduced to account for the average incubation period, \u03c4. With Figure 1 . Scheme of the model with notation guide and default parameter values. The incubation period, resulting from inclusion of k = 5 intermediate infected states, follows the Erlang distribution with average \u03c4 = 5 days, in agreement with data [1, 2] . The serial interval follows the hypoexponential distribution with rate parameters \u03bb = {5/\u03c4, 5/\u03c4, 5/\u03c4, 5/\u03c4, 5/\u03c4, r E }. The model is fully characterized by three parameters: r C , r E , and \u03c4. k = 5 infected states such representation is equivalent to the assumption that the incubation period is distributed according to the Erlang distribution with k = 5 and the rate parameter \u03bb = k/\u03c4 = 1/day. Infectious individuals may become excluded with the exclusion rate r E . As a consequence, the serial interval is distributed according to the hypoexponential distribution that is a series of k = 5 exponential distributions, each originating from one step of the incubation process, having mean times \u03c4/k = 1 day, and one exponential distribution, originating from the \"exclusion\" process, with mean time 1/r E = 3 days. The serial interval distribution implicated by the model agrees perfectly with the distribution obtained from epidemic data (see figure 2B in Ref. [5] ). Excluded individuals comprise the confirmed cases currently handled by the healthcare, those recovered with acquired resistance, and the deceased. Excluded individuals cannot infect. We assume that excluded individuals maintain resistance to the disease. The dynamics is governed by a system of 8 ordinary differential equations provided as equations (8a-8e) and analysed in Methods.","Despite the explosive growth of the number of studies on COVID-19, several key epidemiological parameters of the disease remain to be clarified, among which are incubation period and serial interval. The two parameters have important implications for the monitoring, surveillance and control of infectious disease. The mean or median incubation period and serial interval estimated by previous studies were mostly 4 to 5 days, 2-7 and 4 to 4.5 days, [8] [9] [10] [11] [12] respectively. While some of the studies included only a limited number of cases (around or less than 100), 2, 4, 6, 7, [9] [10] [11] [12] others might have suffered from inaccuracy of original data. For example, in some studies, most cases had a long or even unclear interval of exposure, making it difficult to determine the exact exposure date 3, 5, 13 and giving rise to error. Besides, the order of transmission (i.e., who is infector and who is infectee) in clustering cases, which is crucial to estimation of both parameters, is easy to be mistaken given the possibility of pre-symptomatic and asymptomatic transmission, [14] [15] [16] [17] but previous studies rarely described how this issue was handled.","In models of the spread of COVID-19, the key parameters are the basic reproduction number which refers to the average number of secondary cases generated from a contagious person, and a dispersion parameter that can provide further information about outbreak dynamics and potential for superspreading events [Riou and Althaus 2020] . It is estimated that the basic reproductive number of COVID-19 is around 1.5 to 4 [Rabajante 2020]. Parameters in models of disease spread are usually considered constant for the entire population with time-variation [Liu, et.al. 2020] . However, these parameters vary considering the heterogeneity of the population, location of virus transmission, and socio-economic and political factors [Rabajante, 2020] .","is the (which was not peer-reviewed) The copyright holder for this preprint . https://doi.org/10.1101/2020.03.05.20031088 doi: medRxiv preprint Figure 2 . The effective reproductive number under active monitoring (x axis) and individual quarantine (y axis) increases with the basic reproductive number (colors) and in low feasibility settings (circles) compared to high feasibility settings (triangles) in serial interval scenario 1. Equivalent control under individual quarantine and active monitoring would follow the y=x identify line.","The rapid spread of 2019-noV pneumonia around the world poses a major threat to the closely connected and interdependent world today. The reproductive number R refers to the average number of secondary cases generated from primary cases, and has become a key quantity to determine the intensity of interventions needed to control epidemics [2] . On January 29, 2020, Li et al conducted a study of the first 425 confirmed cases in Wuhan, showing that the R of 2019-nCoV was 2.2, and revealed that person-to-person transmission occurred between close contacts [3] . On January 26, 2020, new research shows that the reproductive ratio of 2019-nCoV is 2.90, which is higher than the 1.77 SARS epidemic [4] \u3002In the absence of comprehensive treatments or vaccines, China has adopted the most effective isolation prevention and control measures to isolate patients diagnosed with 2019-nCoV to control the spread of infection, but the number of 2019-nCoV infections has far exceeded that of SARS epidemics. Existing basic research results and actual situation of epidemic spread have shown that 2019-nCoV has a higher pandemic risk than the outbreak of SARS in 2003 [4] \u3002 This paper discusses the feasibility of using the deterministic model of the transmission dynamics model of infectious diseases to assess the development of the 2019-nCoV epidemic in China. Some researchers have tried to study and evaluate the development trend of the 2019-nCoV pneumonia epidemic through the transmission dynamics. On Jan 24, 2020, British scholars Read et al [5] estimated that the number of 2019-nCoV infection cases in Wuhan would reach 190000 on Feb 4, which obviously overestimated the development trend of the epidemic. On Jan 31, 2020, Wu et al [6] , scholars from Hong Kong, China, predicted that the number of infections in Wuhan on Jan 25 exceeded 75815, which also obviously overestimated the spread of the epidemic. Chinese scholars Shen et al. [7] used dynamics models to predict the peak time and scale of the epidemic, and estimated that the peak number of infections was less than 20,000, which was lower than the raw data released on Feb 10. Chinese scholars Xiao et al. [8] established a transmission dynamic model with considering intervention strategies such as close tracking and quarantine. Based on their model, scholars predicted that the epidemic would reach its peak around Feb 5 [8] , which made an early estimation of the peak time of the epidemic. In summary, although some researchers have carried out research on the transmission dynamics of the 2019-nCoV pneumonia epidemic, the real epidemic development has far deviated from the prediction results of previous studies.","In total, 1155 cases from China, Japan, Singapore, South Korea, Vietnam, Germany and Malaysia were included for the final analysis. The mean and standard deviation were 7.44 days and 4.39 days for incubation period, 2.52 days and 3.95 days for the upper limit of latent period, 6 .70 days and 5.20 days for serial interval, and -0.19 day (i.e., 0.19 day before infector's symptom onset) and 3.32 days for time point of exposure. R0 was estimated to be 1.70 and 1.78 based on two different formulas. For 39 (6.64%) cases, the incubation periods were longer than 14 days. In 102 (43.78%) infector-infectee pairs, transmission occurred before infectors' symptom onsets. In 27 (3.92%) infector-infectee pairs, infectees' symptom onsets occurred before those of infectors. Stratified analysis showed that incubation period and serial interval were consistently longer for those with less severe disease and for those whose primary cases had less severe disease. Asymptomatic transmission was also observed.","Serial interval is the interval from illness onset in a primary case to illness onset in the secondary case. The mean serial interval was estimated at 7.5 days(95% CI, 5.3-19days) using contact tracing data from early Wuhan cases in 2019-nCoV pandemic, which was shorter than the 8.4-day mean serial interval reported for SARS (Lipsitch et al., 2003) and 12.6-day for MERS (Cowling et al., 2015) . Another estimation of the mean serial interval from 26 infector-infectee pairs was surprisingly 2.6 days, which was shorter than the median incubation period, suggesting a substantial proportion of secondary transmission before illness onset (Nishiura et al., 2020) .","We observed the mean serial interval was 4.1 days for COVID-19, which is shorter than the 8.4-day and the 7.6-day mean serial interval reported for SARS and MERS, respectively. 5,20 More importantly, the serial interval is shorter than the estimates of the mean incubation period (4.9 days), suggesting that a proportion of secondary transmission may occur before symptom onset of the first-generation cases. Li et al. 17 estimated the mean serial interval at 7.5 days based on the cases reported in Wuhan city early in the epidemic (mostly in December 2019). However, there were only six pairs first-generation and secondary cases in this study, and the small sampling bias may have been introduced to the variance and mean. Our observed serial interval has been consistently demonstrated by a recent study that the estimated mean serial interval was 4.2 days based on 20 pairs of first-generation and secondary cases. 21 . CC-BY-ND 4.0 International license It is made available under a author/funder, who has granted medRxiv a license to display the preprint in perpetuity.","The 2019-nCoV or SARS-CoV-2 is an enveloped RNA virus that belongs to the Coronaviridae, a family of viruses that can infect humans and several animal species. To date, seven human coronaviruses have been identified (HCoV-229E, HCoV-NL63, HCoV-OC43, HCoV-HKU1, SARS-CoV, MERS-CoV, and SARS-CoV-2). 7 The SARS-CoV-2 belongs to the subgenus Sarbecovirus and most closely resembles a bat coronavirus, with which it shares 96.2% genetic sequence homology. 2 8 9 The current belief is that COVID-19 was introduced into humans through an as yet unidentified intermediary animal, and it has since propagated through human-to-human spread. Human-to-human transmission occurs primarily through respiratory droplets that travel up to 2 m and may enter the respiratory tract of individuals within range or contaminated surfaces, leading to infection through contact transmission. 10 The average incubation period is 5 days, but it ranges from 1 to 14 days. 10 11 The basic reproductive number (the number of cases one infected individual generates), R 0 , is estimated to be 2.68 (95% confidence interval: 2.47e2.86). 12 Whilst asymptomatic transmission (during the incubation period, before onset of symptoms in infected individuals) has been suggested, it remains controversial and currently does not appear to be a major factor for infection transmission. 10 Infections in humans result in a spectrum of clinical disease, from mild upper respiratory tract infection, mostly commonly characterised by fever (82%) and cough (81%), to severe acute respiratory distress syndrome (ARDS) and sepsis. The median age of infected individuals is between 49 and 56 yr. 1 13 Children are rarely diagnosed with 2019-nCoV, a phenomenon that has not yet been explained. Characterisation of the first 99 cases of 2019-nCoV within Wuhan demonstrated that 33% had severe illness, with 17% developing ARDS, 4% requiring mechanical ventilation, and 4% having sepsis. 13 The case fatality rate is currently estimated at 2% based on the confirmed cases reported for the ongoing outbreak. 3 However, as mild and subclinical infections are currently under-reported, the true case fatality is likely lower than this. 14 People with co-morbidities, such as diabetes mellitus and cardiovascular disease, appear to be at higher risk for death. 1 13 ","The incubation and infectious periods have been a most important issue in the present epidemic and estimates have varied as more information is available on the dynamics of the disease. We will use incubation period 1/\u03b3 = 6 days [3, [7] [8] [9] . The same situation occurs with the infectious period that we take 1/\u03b7 = 10 days [7, 10, 11] . One of the first articles to calculate the basic reproduction number R 0 of the new coronavirus in Wuhan China, estimated (Jan 24, 2020) its value as 2.68(2.47 \u2212 2.86) with an epidemic doubling time of 6.4 days (5.8 \u2212 7.1) [3] . Other authors have estimated the basic reproductive number to be around 2.5, value that we use here for the Wuhan original outbreak but again there exist some variability and uncertainity in the different estimates [9] [10] [11] [12] [13] [14] . In what follows some simple models are described related to three specific situations described above and occurring during this epidemic: disease dispersion due to long range population movement, behavioral change imposed on individuals as an strategy of containment, and the effect of social distancing on the general fate of the epidemic.","Our estimates of the basic reproductive number for this novel coronavirus are comparable to most estimates reported for SARS and MERS-CoV, but similar to some estimates from subsets of data in the early period of SARS. For the SARS coronavirus, estimates of the mean reproductive number ranged from 1.1 to 4.2 with most estimates between 2 and 3 (Bauch et al., 2005) . These estimates represent a range of methods and settings. Some estimates come from data that mixes time periods before and after control. Estimates of 0 also varied based on assumed serial intervals (e.g. (Lipsitch et al., 2003) estimated 0 ranging from 2.2 to 3.6 for serial intervals of 8 to 12 days (9). Another study (Bauch et al., 2005) reviewed sources of variation in basic reproductive numbers of SARS and noted that those locations in which outbreaks occurred, 0 was approximately 3. Estimates from MERS-CoV were uniformly lower, with estimates from Saudi Arabia having a mean of less than 1 (~0.5) but exhibited large temporal variability with increases in some periods of time particularly in healthcare settings (Cauchemez et al., 2016) .","The basic reproduction number is model-based and largely depends on the epidemiological setting and the most important parameter to determine the intrinsic transmissibility. The early outbreak data largely follow exponential growth. Different models based on the clinical progression of the disease were devised to estimate the basic reproductive number. In the early stages of COVID-19, the pandemic doubled in size every 7.4 days, and the basic reproductive number was estimated to be 2.2 [7] . Another study estimated a similar reproductive number, which ranged from 2.24 to 3.58 [11] . However, a deterministic compartmental model devised based on the likelihood and a model analysis showed that the control reproduction number may be as high as 6.47 [12] . As noted in the paper, this basic reproductive number is higher because the estimation accounts for three to four generations of viral transmission and intensive social contacts. The basic reproductive number estimated by the majority of studies ranges from 2.24 to 3.58 [13] , which is slightly higher than that of SARS.","In Singapore and Tianjin we estimated relatively short serial intervals. Of particular note, early estimates of R0 for COVID-19 used the SARS serial interval of 8.4 days [7, 8, 26] . Our serial interval findings from two populations mirror those of and Zhao et al [27] and Nishiura et al [13] , who estimated a serial interval of 4.4 and 4.0 days. Furthermore, we estimate the serial interval to be shorter than the incubation period in both clusters, which suggests pre-symptomatic transmission. This indicates that spread of SARS-CoV-2 is likely to be difficult to stop by isolation of detected cases alone. However, shorter serial intervals also lead to lower estimates of R0, and 13 . CC-BY-NC 4.0 International license It is made available under a author/funder, who has granted medRxiv a license to display the preprint in perpetuity.","Much is still unknown about the time scale and effectiveness of asymptomatic transmission in COVID-19. Here we highlight the need to characterize the generation-interval distribution for asymptomatic transmission, and its consequences not only for contact tracing but for estimation of the basic reproduction number of the ongoing COVID-19 outbreak [11] and of the effective proportion of asymptomatic transmission during the exponential-growth phase. Our reproductive number findings fit into a broader framework linking epidemic speed, strength, and generation intervals -for a given observed speed increases in the mean generation interval imply larger reproduction number [9, 10, [13] [14] [15] ."],"title":["Epidemiological parameters of coronavirus disease 2019: a pooled analysis of publicly reported individual data of 1155 cases from seven countries","Strongly heterogeneous transmission of COVID-19 in mainland China: local and regional variation","Epidemiological research priorities for public health control of the ongoing global novel coronavirus (2019-nCoV) outbreak","Transmission interval estimates suggest pre-symptomatic spread of COVID-19","Epidemiological and clinical characteristics of COVID-19 in Shenzhen, the largest migrant city of China","A Cybernetics-based Dynamic Infection Model for Analyzing SARS-COV-2 Infection Stability and Predicting Uncontrollable Risks","An updated estimation of the risk of transmission of the novel coronavirus (2019-nCov)","The SARS-CoV-2 outbreak: what we know","Feasibility of controlling 2019-nCoV outbreaks by isolation of cases and contacts","COVID-19: a novel zoonotic disease caused by a coronavirus from China: what we know and what we don\u2019t","Transmission Potential of SARS-CoV-2 in Viral Shedding Observed at the University of Nebraska Medical Center","Estimation of the Time-Varying Reproduction Number of COVID-19 Outbreak in China","Modeling the Comparative Impact of Individual Quarantine vs. Active Monitoring of Contacts for the Mitigation of COVID-19","Transmission interval estimates suggest pre-symptomatic spread of COVID-19","Nowcasting and forecasting the potential domestic and international spread of the 2019-nCoV outbreak originating in Wuhan, China: a modelling study","COVID-19: Projecting the impact in Rohingya refugee camps and beyond","A model for COVID-19 prediction in Iran based on China parameters","Feasibility of controlling COVID-19 outbreaks by isolation of cases and contacts","The 2003 SARS outbreak and its impact on infection control practices","Can we contain the COVID-19 outbreak with the same measures as for SARS?","Epidemiological parameters of coronavirus disease 2019: a pooled analysis of publicly reported individual data of 1155 cases from seven countries","Feasibility of controlling 2019-nCoV outbreaks by isolation of cases and contacts","The role of absolute humidity on transmission rates of the COVID-19 outbreak","Serial interval of novel coronavirus (COVID-19) infections","Serial interval of novel coronavirus (COVID-19) infections","COVID-19: Epidemiology, Evolution, and Cross-Disciplinary Perspectives","Novel coronavirus 2019-nCoV: early estimation of epidemiological parameters and epidemic predictions","Middle East respiratory syndrome coronavirus: Implications for health care facilities","Temporal dynamics in viral shedding and transmissibility of COVID-19","Epidemiological parameters of coronavirus disease 2019: a pooled analysis of publicly reported individual data of 1155 cases from seven countries","Modeling the Comparative Impact of Individual Quarantine vs. Active Monitoring of Contacts for the Mitigation of COVID-19","Short-term forecasts and long-term mitigation evaluations for the COVID-19 epidemic in Hubei Province, China","Modeling the Comparative Impact of Individual Quarantine vs. Active Monitoring of Contacts for the Mitigation of COVID-19","Emergence of a novel human coronavirus threatening human health","Transmission interval estimates suggest pre-symptomatic spread of COVID-19","Serial interval of novel coronavirus (2019-nCoV) infections","Impact of the contact and exclusion rates on the spread of COVID-19 pandemic","Epidemiological parameters of coronavirus disease 2019: a pooled analysis of publicly reported individual data of 1155 cases from seven countries","A mathematical model of COVID-19 transmission between frontliners and the general public","Modeling the Comparative Impact of Individual Quarantine vs. Active Monitoring of Contacts for the Mitigation of COVID-19","SEIR Transmission dynamics model of 2019 nCoV coronavirus with considering the weak infectious ability and changes in latency duration","Epidemiological parameters of coronavirus disease 2019: a pooled analysis of publicly reported individual data of 1155 cases from seven countries","Insight into 2019 novel coronavirus \u2014 an updated intrim review and lessons from SARS-CoV and MERS-CoV","Transmission of corona virus disease 2019 during the incubation period may lead to a quarantine loophole","Outbreak of a new coronavirus: what anaesthetists should know","The SARS-CoV-2 epidemic outbreak: a review of plausible scenarios of containment and mitigation for Mexico","Novel coronavirus 2019-nCoV: early estimation of epidemiological parameters and epidemic predictions","The epidemiology, diagnosis and treatment of COVID-19","Transmission interval estimates suggest pre-symptomatic spread of COVID-19","The time scale of asymptomatic transmission affects estimates of epidemic potential in the COVID-19 outbreak"],"url":["https://doi.org/10.1101/2020.03.21.20040329","https://doi.org/10.1101/2020.03.10.20033852","https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7029449/","https://doi.org/10.1101/2020.03.03.20029983","https://doi.org/10.1101/2020.03.22.20035246","https://doi.org/10.1101/2020.03.13.20034082","https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7029158/","https://doi.org/10.1016/j.ijid.2020.03.004","https://doi.org/10.1101/2020.02.08.20021162","https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7086482/","https://doi.org/10.1101/2020.03.23.20039446","https://doi.org/10.1101/2020.02.08.20021253","https://doi.org/10.1101/2020.03.05.20031088","https://doi.org/10.1101/2020.03.03.20029983","https://doi.org/10.1016/s0140-6736(20)30260-9","https://doi.org/10.1101/2020.03.27.20045500","https://doi.org/10.1101/2020.03.19.20038950","https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7097845/","https://doi.org/10.1016/j.puhe.2005.10.002","https://doi.org/10.1016/s1473-3099(20)30129-8","https://doi.org/10.1101/2020.03.21.20040329","https://doi.org/10.1101/2020.02.08.20021162","https://doi.org/10.1101/2020.02.12.20022467","https://doi.org/10.1016/j.ijid.2020.02.060","https://doi.org/10.1016/j.ijid.2020.02.060","https://doi.org/10.1016/j.molmed.2020.02.008","https://doi.org/10.1101/2020.01.23.20018549","https://doi.org/10.1016/j.ajic.2014.06.019","https://doi.org/10.1101/2020.03.15.20036707","https://doi.org/10.1101/2020.03.21.20040329","https://doi.org/10.1101/2020.03.05.20031088","https://doi.org/10.1101/2020.03.27.20045625","https://doi.org/10.1101/2020.03.05.20031088","https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7095779/","https://doi.org/10.1101/2020.03.03.20029983","https://doi.org/10.1101/2020.02.03.20019497","https://doi.org/10.1101/2020.03.13.20035485","https://doi.org/10.1101/2020.03.21.20040329","https://doi.org/10.1101/2020.03.27.20045195","https://doi.org/10.1101/2020.03.05.20031088","https://doi.org/10.1101/2020.02.16.20023655","https://doi.org/10.1101/2020.03.21.20040329","https://doi.org/10.1016/j.ijid.2020.03.071","https://doi.org/10.1101/2020.03.06.20031955","https://doi.org/10.1016/j.bja.2020.02.008","https://doi.org/10.1101/2020.03.28.20046276","https://doi.org/10.1101/2020.01.23.20018549","https://doi.org/10.1016/j.ijantimicag.2020.105955","https://doi.org/10.1101/2020.03.03.20029983","https://doi.org/10.1101/2020.03.09.20033514"],"x":[0.682087645683285,0.6773443866333726,0.6842901409940504,0.6762585166365084,0.862051997757135,0.6805561090099428,0.7031178230186833,0.695870043633192,0.7359765693957074,0.6235194872588914,0.8839252373194331,0.42338079890576363,0.6258876366559833,0.7356163185769334,0.6776252204335521,0.5923058649227877,0.5904016090588999,0.6757789099504445,0.7436576872924948,0.7238958287868424,0.879816212795067,0.7345801918753224,0.6643649818698357,0.6790883663351438,0.6790883663351438,0.7441291519815703,0.6266716168272464,0.8731465652695277,0.6692709306381673,0.6740433626735985,0.6908597711469918,0.9130443334339848,0.43362345114656087,0.6930087375845686,0.43274760761739595,0.8828273095558111,0.6950491941360836,0.6796703939353518,0.6786397853645411,0.6954693331983878,0.7370093912276923,0.6944427705891799,0.7007006213445325,0.434094778633473,0.7812157780124279,0.6804051565860748,0.8974817424230079,0.6826190787339421,0.43310468676280434,0.6291945192482767],"y":[0.6507809362701503,0.6472697714235144,0.669721648689971,0.6467964280761314,0.728081314405488,0.7996128155099341,0.6744398490417544,0.6370668894663629,0.1958195507889885,0.5387801142398417,0.5337002877455006,0.14669601209326455,0.739645357565987,0.19543558240178419,0.6753263908246678,0.6968355890632842,0.7593391425030355,0.6481963300597638,0.5146156213254018,0.5737694939723343,0.6635240011445976,0.19427674856227686,0.7747942713531659,0.6462992281442855,0.6462992281442855,0.4853340944178613,0.7374555763330599,0.5301063207118311,0.6456298480744311,0.645892220656917,0.6388674684187122,0.35729675568296393,0.1462031488159283,0.6291368630916278,0.14442718411641567,0.6639080692250089,0.7086002648031029,0.6498327641465982,0.7445283806970941,0.1929512177111522,0.7664218464025362,0.6373514537567525,0.6566203960916792,0.14376200113058082,0.5660886756920854,0.6644438304893668,0.4166981655662314,0.7465472478957431,0.1439845761847862,0.7403468787258155]},"selected":{"id":"6871"},"selection_policy":{"id":"6900"}},"id":"6798","type":"ColumnDataSource"},{"attributes":{"fill_color":{"field":"color"},"line_color":{"field":"color"},"radius":{"field":"rad","units":"data"},"x":{"field":"x"},"y":{"field":"y"}},"id":"6859","type":"Circle"},{"attributes":{"editor":{"id":"6897"},"field":"scr","formatter":{"id":"6896"},"title":"Score","width":30},"id":"6865","type":"TableColumn"},{"attributes":{"text":"&lt;h3&gt;Document ranking&lt;/h3&gt;"},"id":"6874","type":"Div"},{"attributes":{"callback":null,"tooltips":[["Title","@title"],["Passage","@text"],["Published","@date"],["Authors","@authors"],["Journal","@journal"],["URL","@url"]]},"id":"6853","type":"HoverTool"},{"attributes":{"formatter":{"id":"6903"},"ticker":{"id":"6848"}},"id":"6847","type":"LinearAxis"},{"attributes":{"text":""},"id":"6882","type":"Title"},{"attributes":{},"id":"6897","type":"StringEditor"},{"attributes":{},"id":"6895","type":"BasicTickFormatter"},{"attributes":{"columns":[{"id":"6830"},{"id":"6831"}],"height":340,"source":{"id":"6797"},"view":{"id":"6833"},"width":900},"id":"6832","type":"DataTable"},{"attributes":{},"id":"6898","type":"StringFormatter"},{"attributes":{"text":"&lt;h3&gt;Paragraph ranking&lt;/h3&gt;"},"id":"6875","type":"Div"},{"attributes":{"callback":null},"id":"6816","type":"TapTool"},{"attributes":{"editor":{"id":"6887"},"field":"scr","formatter":{"id":"6886"},"title":"Score","width":30},"id":"6830","type":"TableColumn"},{"attributes":{"args":{"sd":{"id":"6797"},"sp":{"id":"6798"}},"code":"\n    if (sd.selected.indices.length == 0) {\n      return;\n    }\n    var sel = sd.selected.indices[0];\n    var new_selected = [];\n    for (var i = 0; i &lt; sp.data['doc'].length; i++) {\n      if (sp.data['doc'][i] == sd.data['doc'][sel]) {\n        new_selected.push(i);\n      }\n    }\n    if (JSON.stringify(new_selected) != JSON.stringify(sp.selected.indices) &amp;&amp;\n    !(sp.selected.indices.length == 1 &amp;&amp; new_selected.includes(sp.selected.indices[0]))) {\n      sp.selected.indices = new_selected;\n      sp.change.emit();\n    }\n    "},"id":"6870","type":"CustomJS"},{"attributes":{"editor":{"id":"6899"},"field":"text","formatter":{"id":"6898"},"title":"Passage","width":770},"id":"6866","type":"TableColumn"},{"attributes":{},"id":"6886","type":"StringFormatter"},{"attributes":{},"id":"6835","type":"Range1d"},{"attributes":{},"id":"6889","type":"StringEditor"},{"attributes":{"active_drag":"auto","active_inspect":"auto","active_multi":null,"active_scroll":"auto","active_tap":"auto","tools":[{"id":"6816"},{"id":"6817"},{"id":"6818"}]},"id":"6819","type":"Toolbar"},{"attributes":{},"id":"6890","type":"UnionRenderers"},{"attributes":{"editor":{"id":"6889"},"field":"title","formatter":{"id":"6888"},"title":"Document title","width":770},"id":"6831","type":"TableColumn"},{"attributes":{},"id":"6899","type":"StringEditor"},{"attributes":{"columns":[{"id":"6865"},{"id":"6866"}],"height":340,"source":{"id":"6798"},"view":{"id":"6868"},"width":900},"id":"6867","type":"DataTable"},{"attributes":{},"id":"6817","type":"ResetTool"},{"attributes":{"children":[{"id":"6875"}]},"id":"6879","type":"Row"},{"attributes":{"text":""},"id":"6884","type":"Title"},{"attributes":{},"id":"6896","type":"StringFormatter"},{"attributes":{},"id":"6905","type":"BasicTickFormatter"},{"attributes":{"js_property_callbacks":{"change:indices":[{"id":"6872"}]}},"id":"6871","type":"Selection"},{"attributes":{},"id":"6837","type":"Range1d"},{"attributes":{"fill_alpha":{"value":0.1},"fill_color":{"field":"color"},"line_alpha":{"value":0.1},"line_color":{"field":"color"},"radius":{"field":"rad","units":"data"},"x":{"field":"x"},"y":{"field":"y"}},"id":"6825","type":"Circle"},{"attributes":{"args":{"sd":{"id":"6797"},"sp":{"id":"6798"}},"code":"\n    if (sp.selected.indices.length == 0) {\n      return;\n    }\n    var sel = sp.selected.indices[0];\n    var new_selected = [];\n    for (var i = 0; i &lt; sd.data['doc'].length; i++) {\n      if (sd.data['doc'][i] == sp.data['doc'][sel]) {\n        new_selected.push(i);\n      }\n    }\n    if (JSON.stringify(new_selected) != JSON.stringify(sd.selected.indices) &amp;&amp;\n    !(sd.selected.indices.length == 1 &amp;&amp; new_selected.includes(sd.selected.indices[0]))) {\n      sd.selected.indices = new_selected;\n      sd.change.emit();\n    }\n    "},"id":"6872","type":"CustomJS"},{"attributes":{},"id":"6903","type":"BasicTickFormatter"},{"attributes":{"js_property_callbacks":{"change:indices":[{"id":"6870"}]}},"id":"6869","type":"Selection"},{"attributes":{"sizing_mode":"stretch_width","text":"&lt;h2&gt;Transmission dynamics of the virus, including the basic reproductive number, incubation period, serial interval, modes of transmission and environmental factors&lt;/h2&gt;"},"id":"6873","type":"Div"}],"root_ids":["6881"]},"title":"Bokeh Application","version":"2.0.1"}}
        </script>
        <script type="text/javascript">
          (function() {
            var fn = function() {
              Bokeh.safely(function() {
                (function(root) {
                  function embed_document(root) {
                    
                  var docs_json = document.getElementById('7118').textContent;
                  var render_items = [{"docid":"48122c88-4510-4592-b8fb-bd0fc99b6c17","root_ids":["6881"],"roots":{"6881":"9115fd91-1a0d-4e5e-b323-f38bf5e9d594"}}];
                  root.Bokeh.embed.embed_items(docs_json, render_items);
                
                  }
                  if (root.Bokeh !== undefined) {
                    embed_document(root);
                  } else {
                    var attempts = 0;
                    var timer = setInterval(function(root) {
                      if (root.Bokeh !== undefined) {
                        clearInterval(timer);
                        embed_document(root);
                      } else {
                        attempts++;
                        if (attempts > 100) {
                          clearInterval(timer);
                          console.log("Bokeh: ERROR: Unable to run BokehJS code because BokehJS library is missing");
                        }
                      }
                    }, 10, root)
                  }
                })(window);
              });
            };
            if (document.readyState != "loading") fn();
            else document.addEventListener("DOMContentLoaded", fn);
          })();
        </script>
    
  </body>
  
</html>