



<!DOCTYPE html>
<html lang="en">
  
  <head>
    
      <meta charset="utf-8">
      <title>Bokeh Plot</title>
      
      
        
          
        
        
          
        <script type="text/javascript" src="https://cdn.bokeh.org/bokeh/release/bokeh-2.0.1.min.js" integrity="sha384-JpP8FXbgAZLkfur7LiK3j9AGBhHNIvF742meBJrjO2ShJDhCG2I1uVvW+0DUtrmc" crossorigin="anonymous"></script>
        <script type="text/javascript" src="https://cdn.bokeh.org/bokeh/release/bokeh-widgets-2.0.1.min.js" integrity="sha384-xZlADit0Q04ISQEdKg2k3L4W9AwQBAuDs9nJL9fM/WwzL1tEU9VPNezOFX0nLEAz" crossorigin="anonymous"></script>
        <script type="text/javascript" src="https://cdn.bokeh.org/bokeh/release/bokeh-tables-2.0.1.min.js" integrity="sha384-4BuPRZkdMKSnj3zoxiNrQ86XgNw0rYmBOxe7nshquXwwcauupgBF2DHLVG1WuZlV" crossorigin="anonymous"></script>
        <script type="text/javascript">
            Bokeh.set_log_level("info");
        </script>
        
      
      
    
  </head>
  
  
  <body>
    
      
        
          
          
            
              <div class="bk-root" id="8db81f8e-2eae-4194-b0a2-815d55c2b746" data-root-id="27489"></div>
            
          
        
      
      
        <script type="application/json" id="27726">
          {"9451a660-16c8-4c1a-b08d-331d5094d081":{"roots":{"references":[{"attributes":{"editor":{"id":"27505"},"field":"scr","formatter":{"id":"27504"},"title":"Score","width":30},"id":"27473","type":"TableColumn"},{"attributes":{"active_drag":"auto","active_inspect":"auto","active_multi":null,"active_scroll":"auto","active_tap":"auto","tools":[{"id":"27424"},{"id":"27425"},{"id":"27426"}]},"id":"27427","type":"Toolbar"},{"attributes":{"fill_alpha":{"value":0.1},"fill_color":{"field":"color"},"line_alpha":{"value":0.1},"line_color":{"field":"color"},"radius":{"field":"rad","units":"data"},"x":{"field":"x"},"y":{"field":"y"}},"id":"27468","type":"Circle"},{"attributes":{"source":{"id":"27405"}},"id":"27435","type":"CDSView"},{"attributes":{"sizing_mode":"stretch_width","text":"&lt;h2&gt;Enhance capabilities for rapid sequencing and bioinformatics to target regions of the genome that will allow specificity for a particular variant.&lt;/h2&gt;"},"id":"27481","type":"Div"},{"attributes":{},"id":"27500","type":"BasicTickFormatter"},{"attributes":{"columns":[{"id":"27438"},{"id":"27439"}],"height":340,"source":{"id":"27405"},"view":{"id":"27441"},"width":900},"id":"27440","type":"DataTable"},{"attributes":{"text":""},"id":"27490","type":"Title"},{"attributes":{"text":"&lt;h3&gt;Document ranking&lt;/h3&gt;"},"id":"27482","type":"Div"},{"attributes":{"fill_alpha":{"value":0.1},"fill_color":{"field":"color"},"line_alpha":{"value":0.1},"line_color":{"field":"color"},"radius":{"field":"rad","units":"data"},"x":{"field":"x"},"y":{"field":"y"}},"id":"27433","type":"Circle"},{"attributes":{"source":{"id":"27406"}},"id":"27476","type":"CDSView"},{"attributes":{"children":[{"id":"27482"}]},"id":"27485","type":"Row"},{"attributes":{"args":{"sd":{"id":"27405"},"sp":{"id":"27406"}},"code":"\n    if (sp.selected.indices.length == 0) {\n      return;\n    }\n    var sel = sp.selected.indices[0];\n    var new_selected = [];\n    for (var i = 0; i &lt; sd.data['doc'].length; i++) {\n      if (sd.data['doc'][i] == sp.data['doc'][sel]) {\n        new_selected.push(i);\n      }\n    }\n    if (JSON.stringify(new_selected) != JSON.stringify(sd.selected.indices) &amp;&amp;\n    !(sd.selected.indices.length == 1 &amp;&amp; new_selected.includes(sd.selected.indices[0]))) {\n      sd.selected.indices = new_selected;\n      sd.change.emit();\n    }\n    "},"id":"27480","type":"CustomJS"},{"attributes":{},"id":"27414","type":"LinearScale"},{"attributes":{},"id":"27443","type":"Range1d"},{"attributes":{"data":{"authors":["Jessica E. Manning; Jennifer A. Bohl; Sreyngim Lay; Sophana Chea; Ly Sovann; Yi Sengdoeurn; Seng Heng; Chan Vuthy; Katrina Kalantar; Vida Ahyong; Michelle Tan; Jonathan Sheu; Cristina M. Tato; Joseph L. DeRisi; Laurence Baril; Veasna Duong; Philippe Dussart; Erik A. Karlsson","Andrew D. Davidson; Maia Kavangh Williamson; Sebastian Lewis; Deborah Shoemark; Miles W Carroll; Kate Heesom; Maria Zambon; Joanna Ellis; Phillip A Lewis; Julian A Hiscox; David A Matthews","Walter R Hsiang; Howard P Forman; Siddharth Jain; Akshay Khunte; Grace Jin; Laurie Yousman; Alison Mosier-Mills; Michael Najem; Daniel Wiznia","Chenyu Li; David N. Debruyne; Julia Spencer; Vidushi Kapoor; Lily Y. Liu; Bo Zhou; Lucie Lee; Rounak Feigelman; Grayson Burdon; Jeffrey Liu; Alejandra Oliva; Adam Borcherding; Hongdong Tan; Alexander E. Urban; Guoying Liu; Zhitong Liu","Cleemput, S.; Dumon, W.; Fonseca, V.; Karim, W. A.; Giovanetti, M.; Alcantara, L. C.; Deforche, K.; de Oliveira, T.","Zhang, Chengxin; Zheng, Wei; Huang, Xiaoqiang; Bell, Eric W.; Zhou, Xiaogen; Zhang, Yang","Albiger, Barbara; Revez, Joana; Leitmeyer, Katrin Claire; Struelens, Marc J.","Bourog\u00e2a, Hager; Larbi, Imen; Miled, Khaled; Hellal, Ymene Kort; Hassen, Jihene; Behi, Imen; Nsiri, Jihene; Ghram, Abdeljelil","Jakub M Bartoszewicz; Anja Seidel; Bernhard Y Renard","Laura E Lamb; Sarah N Bartolone; Elijah Ward; Michael B Chancellor","Alejandro Lopez-Rincon; Alberto Tonda; Lucero Mendoza-Maldonado; Eric Claassen; Johan Garssen; Aletta D. Kraneveld","Kenneth N. Hass; Mengdi Bao; Qian He; Myeongkee Park; Peiwu Qin; Ke Du","Fredricks, David N; Relman, David A","Segondy, Michel","Dowling, Nicole F; Gwinn, Marta; Mawle, Alison","Timokratis Karamitros; Gethsimani Papadopoulou; Maria Bousali; Anastasios Mexias; Sotiris Tsiodras; Andreas Mentis","Mitsuki, Yu-ya; Ohnishi, Kazuo; Takagi, Hirotaka; Oshima, Masamichi; Yamamoto, Takuya; Mizukoshi, Fuminori; Terahara, Kazutaka; Kobayashi, Kazuo; Yamamoto, Naoki; Yamaoka, Shoji; Tsunetsugu-Yokota, Yasuko","Jonas Pfab; Dong Si","Ganova-Raeva, Lilia M; Khudyakov, Yury E","Ryu, Wang-Shick","Paraskevis, D.; Kostaki, E.G.; Magiorkinis, G.; Panayiotakopoulos, G.; Sourvinos, G.; Tsiodras, S.","John-Sebastian Eden; Rebecca Rockett; Ian Carter; Hossinur Rahman; Joep de Ligt; James Hadfield; Matthew Storey; Xiaoyun Ren; Rachel Tulloch; Kerri Basile; Jessica Wells; Roy Byun; Nicky Gilroy; Matthew V O\u2019Sullivan; Vitali Sintchenko; Sharon C Chen; Susan Maddocks; Tania C Sorrell; Edward C Holmes; Dominic E Dwyer; Jen Kok","Chang, Chung-ke; Sue, Shih-Che; Yu, Tsan-hung; Hsieh, Chiu-Min; Tsai, Cheng-Kun; Chiang, Yen-Chieh; Lee, Shin-jye; Hsiao, Hsin-hao; Wu, Wen-Jin; Chang, Wei-Lun; Lin, Chun-Hung; Huang, Tai-huang","Kim, Jin Hwa; Kang, Minhee; Park, Eunkyoung; Chung, Doo Ryeon; Kim, Jiyeon; Hwang, Eung Soo","Yi Chen Chong","Lamia Wahba; Nimit Jain; Andrew Z. Fire; Massa J. Shoura; Karen L. Artiles; Matthew J. McCoy; Dae-Eun Jeong","Ramya Rangan; Ivan N. Zheludev; Rhiju Das","Erik Procko","Ming Wang; Aisi Fu; Ben Hu; Yongqing Tong; Ran Liu; Jiashuang Gu; Jianghao Liu; Wen Jiang; Gaigai Shen; Wanxu Zhao; Dong Men; Lilei Yu; Zixin Deng; Yan Li; Tiangang Liu","Tu, Xinyi; Chong, Wai Po; Zhai, Yun; Zhang, Hongxing; Zhang, Fang; Wang, Shixin; Liu, Wei; Wei, Maoti; Siu, Nora Ho On; Yang, Hao; Yang, Wanling; Cao, Wuchun; Lau, Yu Lung; He, Fuchu; Zhou, Gangqiao","Dahlhausen, Bob","Ibrahim, Ibrahim M.; Abdelmalek, Doaa H.; Elshahat, Mohammed E.; Elfiky, Abdo A.","Roberts, Lisa O.; Jopling, Catherine L.; Jackson, Richard J.; Willis, Anne E.","Shuo Jin; Bo Wang; Haibo Xu; Chuan Luo; Lai Wei; Wei Zhao; Xuexue Hou; Wenshuo Ma; Zhengqing Xu; Zhuozhao Zheng; Wenbo Sun; Lan Lan; Wei Zhang; Xiangdong Mu; Chenxi Shi; Zhongxiao Wang; Jihae Lee; Zijian Jin; Minggui Lin; Hongbo Jin; Liang Zhang; Jun Guo; Benqi Zhao; Zhizhong Ren; Shuhao Wang; Zheng You; Jiahong Dong; Xinghuan Wang; Jianming Wang; Wei Xu","Phillippy, Adam M; Deng, Xiangyu; Zhang, Wei; Salzberg, Steven L","Al-Tawfiq, Jaffar A; Zumla, Alimuddin; Gautret, Philippe; Gray, Gregory C; Hui, David S; Al-Rabeeah, Abdullah A; Memish, Ziad A","Deng, Xin; Gumm, Jordan; Karki, Suman; Eickholt, Jesse; Cheng, Jianlin","Gautret, Philippe; Gray, Gregory C; Charrel, Remi N; Odezulu, Nnanyelugo G; Al-Tawfiq, Jaffar A; Zumla, Alimuddin; Memish, Ziad A","Intikhab alam; Allan Kamau; Maxat Kulmanov; Stefan T. Arold; Arnab Pain; Takashi Gojobori; Carlos M. Duarte","Minfeng Xiao; Xiaoqing Liu; Jingkai Ji; Min Li; Jiandong Li; Lin Yang; Wanying Sun; Peidi Ren; Guifang Yang; Jincun Zhao; Tianzhu Liang; Huahui Ren; Tian Chen; Huanzi Zhong; Wenchen Song; Yanqun Wang; Ziqing Deng; Yanping Zhao; Zhihua Ou; Daxi Wang; Jielun Cai; Xinyi Cheng; Taiqing Feng; Honglong Wu; Yanping Gong; Huanming Yang; Jian Wang; Xun Xu; Shida Zhu; Fang Chen; Yanyan Zhang; Weijun Chen; Yimin Li; Junhua Li","Robson, B.","Rishikesh Magar; Prakarsh Yadav; Amir Barati Farimani","Yao Yu Yeo; Yao-Rui Yeo; Wan-Jin Yeo","Domachowske, Joseph B.","Lumangtad, Liezel A.; Bell, Thomas W.","Agut, H; Fillet, A.M; Gautheret-Dejean, A; Boutolleau, D","Gentile Francesco Ficetola; Diego Rubolini","Huang, Weiwei; Yang, Yinhui; Zhang, Xinlei; Zhao, Changan; Yin, Aihua; Zhang, Xiaozhuang; He, Zhengxin; Jiang, Yongqiang; Zhang, Liang","Petrosillo, Nicola; Puro, Vincenzo; Di Caro, Antonino; Ippolito, Giuseppe","Danielle E Anderson; Jin Cui; Qian Ye; Baoying Huang; Wenhong Zu; Jing Gong; Weiqiang Liu; So Young Kim; Biao Guo Yan; Kristmundur Sigmundsson; Xiao Fang Lim; Fei Ye; Peihua Niu; Xuming Zhou; Wenjie Tan; Lin-Fa Wang; Xu Tan"],"color":["#67a9cf","#67a9cf","#67a9cf","#67a9cf","#67a9cf","#67a9cf","#67a9cf","#67a9cf","#67a9cf","#67a9cf","#67a9cf","#67a9cf","#67a9cf","#67a9cf","#67a9cf","#67a9cf","#67a9cf","#67a9cf","#67a9cf","#67a9cf","#67a9cf","#67a9cf","#67a9cf","#67a9cf","#67a9cf","#67a9cf","#67a9cf","#67a9cf","#67a9cf","#67a9cf","#67a9cf","#67a9cf","#67a9cf","#67a9cf","#67a9cf","#67a9cf","#67a9cf","#67a9cf","#67a9cf","#67a9cf","#67a9cf","#67a9cf","#67a9cf","#67a9cf","#67a9cf","#67a9cf","#67a9cf","#67a9cf","#67a9cf","#67a9cf"],"date":["2020-03-05","2020-03-24","2020-03-27","2020-03-14","2020","2020-03-22","2018-02-26","2014-06-01","2020-01-30","2020-02-24","2020-03-14","2020-03-20","2005-03-01","2015-08-31","2009-12-29","2020-03-28","2008-07-31","2020-02-13","2013-05-01","2017-12-31","2020-04-30","2020-03-17","2005-10-14","2019-11-11","2020-02-25","2020-02-14","2020-03-28","2020-03-17","2020-03-06","2015-07-31","2010-04-30","2020-03-10","2009-12-31","2020-03-23","2009-09-16","2014-10-31","2015-07-07","2014-11-30","2020-02-21","2020-03-17","2020-02-26","2020-03-20","2020-03-27","2004-10-15","2020-03-17","2003-09-30","2020-03-27","2013-09-20","2005-12-31","2020-03-30"],"doc":["fu3cl7lq","txuthqgo","600am3mq","0hxan9rw","st5idleq","3r8jbhhq","hq8dg87u","sb55ybtb","ac00tai9","6kpgt70s","c2lljdi7","d840uu3e","cisrvghj","ifutobng","03bwk538","jxm69ndw","ai6jfviy","er6lz09f","qh2kuzdc","wl5sjbbm","qed8hayx","izkz1hz4","1ujmjjwb","3doc40hx","o3hytzwu","emr0eh0i","kjeqdse5","jalijjmg","s4d442kx","p6jl9gxf","o976nab9","j10308i3","b37c8aye","e6q92shw","g4sjyytr","k560qnyi","ypy56yxr","a559unhv","p4q64ksa","wbyqmvhs","13jupb26","fn7l93wh","8g64u3ux","4dqz8req","oh85yi49","fwhhsnmq","fcaeoyxd","cxqzac4a","lk7e2pni","0l33i6s4"],"journal":["","","","","Bioinformatics (Oxford, England)","J Proteome Res","Front Public Health","Journal of Applied Poultry Research","","","","","Medicine","Revue Francophone des Laboratoires","Genome Med","","Microbes and Infection","","Expert Review of Molecular Diagnostics","Molecular Virology of Human Pathogenic Viruses","Infection, Genetics and Evolution","","J Biomed Sci","Biochip J","","","","","","Journal of Infection","Journal of Exotic Pet Medicine","Journal of Infection","Progress in Molecular Biology and Translational Science","","BMC Bioinformatics","The Lancet Infectious Diseases","Int J Mol Sci","The Lancet Infectious Diseases","","","Computers in Biology and Medicine","","","Clinical Microbiology Newsletter","Bioorganic &amp; Medicinal Chemistry Letters","Immuno-analyse &amp; Biologie Sp\u00e9cialis\u00e9e","","BMC Infect Dis","Archives of Medical Research",""],"rad":[0.02,0.014971979523490046,0.014378661507618799,0.013174942104202187,0.011849280312815357,0.010791375917442045,0.010691136503434137,0.010661447825081724,0.01028726825202436,0.0098727632006307,0.009251652687629383,0.009059792646563758,0.008996983836155211,0.008365235148543267,0.007999367627126662,0.007880720230147216,0.007734759746448332,0.00766424466816813,0.00758079876841944,0.00744405020065882,0.006659869040228779,0.0065994393296113,0.0062629110135363165,0.005967544302901568,0.005674376583766124,0.005645670335247386,0.005606040514269741,0.005422768272039377,0.0054051388723860404,0.005336040480107156,0.004901781862251285,0.00489285277012704,0.004675050904584789,0.004663729640361732,0.0046604321196786055,0.004570621655832686,0.004064887846841732,0.0039506626167298275,0.003853897856613466,0.003835508475977756,0.003800371606595306,0.0037235039425643204,0.003656193051251758,0.0035723610373511964,0.003472703718532853,0.003414134205978221,0.0033946572956224585,0.0031734801314795363,0.0031011885617607824,0.003],"scr":[1.0,0.704,0.669,0.599,0.521,0.458,0.452,0.451,0.429,0.404,0.368,0.356,0.353,0.316,0.294,0.287,0.279,0.274,0.269,0.261,0.215,0.212,0.192,0.175,0.157,0.156,0.153,0.143,0.141,0.137,0.112,0.111,0.099,0.098,0.098,0.092,0.063,0.056,0.05,0.049,0.047,0.043,0.039,0.034,0.028,0.024,0.023,0.01,0.006,0.0],"text":["AbstractRapid production and publication of pathogen genome sequences during emerging disease outbreaks provide crucial public health information. In resource-limited settings, especially near an outbreak epicenter, conventional deep sequencing or bioinformatics are often challenging. Here we successfully used metagenomic next generation sequencing on an iSeq100 Illumina platform paired with an open-source bioinformatics pipeline to quickly characterize Cambodia\u2019s first case of COVID-2019.","Direct RNA sequencing using an Oxford Nanopore MinION characterised the transcriptome of SARS-CoV-2 grown in Vero E6 cells. This cell line is being widely used to propagate the novel coronavirus. The viral transcriptome was analysed using a recently developed ORF-centric pipeline. This revealed the pattern of viral transcripts, (i.e. subgenomic mRNAs), generally fitted the predicted replication and transcription model for coronaviruses. A 24 nt in-frame deletion was detected in subgenomic mRNAs encoding the spike (S) glycoprotein. This feature was identified in over half of the mapped transcripts and was predicted to remove a proposed furin cleavage site from the S glycoprotein. This motif directs cleavage of the S glycoprotein into functional subunits during virus entry or exit. Cleavage of the S glycoprotein can be a barrier to zoonotic coronavirus transmission and affect viral pathogenicity. Allied to this transcriptome analysis, tandem mass spectrometry was used to identify over 50...","As of March 22, 2020, the number of confirmed COVID-19 cases in the U.S. has reached nearly 30,000. While rapid and accessible diagnosis is paramount to monitoring and reducing the spread of disease, COVID-19 testing capabilities across the U.S. remain constrained. For many individuals, urgent care centers (UCCs) may offer the most accessible avenue to be tested. Through a phone survey, we describe the COVID-19 testing capabilities at UCCs and provide a snapshot highlighting the limited COVID-19 testing capabilities at UCCs in states with the greatest disease burden.","AbstractMany detection methods have been used or reported for the diagnosis and/or surveillance of SARS-CoV-2. Among them, reverse transcription polymerase chain reaction (RT-PCR) is the most sensitive, claiming detection of about 5 copies of viruses. However, it has been reported that only 47-59% of the positive cases were identified by RT-PCR, probably due to loss or degradation of virus RNA in the sampling process, or even mutation of the virus genome. Therefore, developing highly sensitive methods is imperative to ensure robust detection capabilities. With the goal of improving sensitivity and accommodate various application settings, we developed a multiplex-PCR-based method comprised of 172 pairs of specific primers, and demonstrated its efficiency to detect SARS-CoV-2 at low copy numbers. The assay produced clean characteristic target peaks of defined sizes, which allowed for direct identification of positives by electrophoresis. In addition, optional sequencing can provide furt...","SUMMARY: Genome Detective is a web-based, user-friendly software application to quickly and accurately assemble all known virus genomes from next generation sequencing datasets. This application allows the identification of phylogenetic clusters and genotypes from assembled genomes in FASTA format. Since its release in 2019, we have produced a number of typing tools for emergent viruses that have caused large outbreaks, such as Zika and Yellow Fever Virus in Brazil. Here, we present The Genome Detective Coronavirus Typing Tool that can accurately identify the novel severe acute respiratory syndrome (SARS) related coronavirus (SARS-CoV-2) sequences isolated in China and around the world. The tool can accept up to 2,000 sequences per submission and the analysis of a new whole genome sequence will take approximately one minute. The tool has been tested and validated with hundreds of whole genomes from ten coronavirus species, and correctly classified all of the SARS-related coronavirus (S...","[Image: see text] As the infection of 2019-nCoV coronavirus is quickly developing into a global pneumonia epidemic, the careful analysis of its transmission and cellular mechanisms is sorely needed. In this Communication, we first analyzed two recent studies that concluded that snakes are the intermediate hosts of 2019-nCoV and that the 2019-nCoV spike protein insertions share a unique similarity to HIV-1. However, the reimplementation of the analyses, built on larger scale data sets using state-of-the-art bioinformatics methods and databases, presents clear evidence that rebuts these conclusions. Next, using metagenomic samples from Manis javanica, we assembled a draft genome of the 2019-nCoV-like coronavirus, which shows 73% coverage and 91% sequence identity to the 2019-nCoV genome. In particular, the alignments of the spike surface glycoprotein receptor binding domain revealed four times more variations in the bat coronavirus RaTG13 than in the Manis coronavirus compared with 2019-...","In an era of global health threats caused by epidemics of infectious diseases and rising multidrug resistance, microbiology laboratories provide essential scientific evidence for risk assessment, prevention, and control. Microbiology has been at the core of European infectious disease surveillance networks for decades. Since 2010, these networks have been coordinated by the European Centre for Disease Prevention and Control (ECDC). Activities delivered in these networks include harmonization of laboratory diagnostic, antimicrobial susceptibility and molecular typing methods, multicentre method validation, technical capacity mapping, training of laboratory staff, and continuing quality assessment of laboratory testing. Cooperation among the European laboratory networks in the past 7 years has proved successful in strengthening epidemic preparedness by enabling adaptive capabilities for rapid detection of emerging pathogens across Europe. In partnership with food safety authorities, inte...","Abstract Due to serotype variations among different avian infectious bronchitis viruses isolated in Tunisia since 2000, protection of chicks, especially broiler flocks, with Mass H120 vaccine often fails. Therefore, association of CR88 (793B type) with H120 vaccines was used for better response. Challenge experiments were then conducted to evaluate tracheal and renal cross-protection in chickens immunized via nasal and eye drops. Conferred protection was measured by clinical signs and macroscopic lesions observed, based on scores attributed according to their severities. The results showed a low protection conferred by H120 alone, as vaccination did not reduce tracheal and kidney lesions (70% scored as 3) after TN20/00 virus challenge, which also led to 10% mortality. Conversely, the challenge results indicated that the combination of the 2 strains (H120/CR88) allow high protection. Based on the results of the challenge experiments, a vaccination protocol coupling CR88 to H120 was appl...","Viruses evolve extremely quickly, so reliable methods for viral host prediction are necessary to safeguard biosecurity and biosafety alike. Novel human-infecting viruses are difficult to detect with standard bioinformatics workflows. Here, we predict whether a virus can infect humans directly from next-generation sequencing reads. We show that deep neural architectures significantly outperform both shallow machine learning and standard, homology-based algorithms, cutting the error rates in half and generalizing to taxonomic units distant from those presented during training. We propose a new approach for convolutional filter visualization to disentangle the information content of each nucleotide from its contribution to the final classification decision. Nucleotide-resolution maps of the learned associations between pathogen genomes and the infectious phenotype can be used to detect virulence-related genes in novel agents, as we show here for the SARS-CoV-2 coronavirus, unknown before ...","Novel Corona virus (COVID-19 or 2019-nCoV) is an emerging global health concern that requires a rapid diagnostic test. Quantitative reverse transcription PCR (qRT-PCR) is currently the standard for COVID-19 detection; however, Reverse Transcription Loop-Mediated Isothermal Amplification (RT-LAMP) may allow for faster and cheaper field based testing at point-of-risk. The objective of this study was to develop a rapid screening diagnostic test that could be completed in under 30 minutes. Simulated patient samples were generated by spiking serum, urine, saliva, oropharyngeal swabs, and nasopharyngeal swabs with a portion of the COVID-19 nucleic sequence. The samples were tested using RT-LAMP as well as by conventional qRT-PCR. Specificity of the RT-LAMP was evaluated by also testing against other related coronaviruses. RT-LAMP specifically detected COVID-19 in simulated patient samples. This test was performed in under 30 minutes. This approach could be used for monitoring of exposed indi...","One of the reasons for the fast spread of SARS-CoV-2 is the lack of accuracy in detection tools in the clinical field. Molecular techniques, such as quantitative real-time RT-PCR and nucleic acid sequencing methods, are widely used to identify pathogens. For this particular virus, however, they have an overall unsatisfying detection rate, due to its relatively recent emergence and still not completely understood features. In addition, SARS-CoV-2 is remarkably similar to other Coronaviruses, and it can present with other respiratory infections, making identification even harder. To tackle this issue, we propose an assisted detection test, combining molecular testing with deep learning. The proposed approach employs a state-of-the-art deep convolutional neural network, able to automatically create features starting from the genome sequence of the virus. Experiments on data from the Novel Coronavirus Resource (2019nCoVR) show that the proposed approach is able to correctly classify SARS-C...","ABSTRACTA fully Integrated Micropillar Polydimethylsiloxane Accurate CRISPR Detection (IMPACT) system is developed for viral DNA detection. This powerful system is patterned with high-aspect ratio micropillars to enhance reporter probe binding. After surface modification and probe immobilization, CRISPR Cas12a/crRNA complex is injected into the fully enclosed system. With the presence of double-stranded DNA target, the CRISPR enzyme is activated and non-specifically cleaves the ssDNA reporters initially immobilized on the micropillars. This collateral cleavage releases fluorescence dyes into the assay, and the intensity is linearly proportional to the target DNA concentration ranging from 0.1 to 10 nM. Importantly, this system does not rely on traditional dye-quencher labeled probe thus eliminating the fluorescence background presented in the assay. Furthermore, our one-step detection protocol is performed at isothermal conditions (37\u00b0C) without using complicated and time-consuming off...","Abstract There are many diseases for which a microbial aetiology is suspected. The hypothesis that a disease has an infectious cause is supported by: clinical features (similar to those of known infectious diseases, e.g. fever, leucocytosis), epidemiology (case clustering in time or location), histology (inflammation of affected tissues, e.g. granulomata) or characteristic microbial structures, treatment (clinical response to antimicrobial treatment), and prevention of disease by vaccines targeting microbial antigens. Proof that a microbe causes a disease requires more rigorous evidence. Future attempts to identify novel microbes associated with human disease may use sequence-based approaches. High-throughput sequencing may allow identification of unique microbial nucleic acid sequences in a background of host DNA. The complete sequencing of the human genome and multiple microbial genomes make this approach more feasible. DNA microarrays are also likely to be used in the search for nov...","R\u00e9sum\u00e9 Les tests de d\u00e9pistage rapide (TDR) bas\u00e9s sur la technique d\u2019immunochromatographie sont tr\u00e8s r\u00e9pandus pour le diagnostic des maladies infectieuses. Dans le cadre des infections respiratoires virales, il s\u2019agit essentiellement de tests pour le diagnostic de la grippe et des infections \u00e0 virus respiratoire syncytial (VRS). Pour le diagnostic des gastroent\u00e9rites virales, ce sont des tests de d\u00e9tection des rotavirus, ad\u00e9novirus, norovirus et astrovirus qui peuvent \u00eatre utilis\u00e9s. Les TDR pour les infections \u00e0 VRS et les gastroent\u00e9rites sont utilis\u00e9s essentiellement dans le cadre de la pathologie p\u00e9diatrique. Ces tests pr\u00e9sentent g\u00e9n\u00e9ralement une sp\u00e9cificit\u00e9 \u00e9lev\u00e9e mais une sensibilit\u00e9 modeste par rapport aux techniques mol\u00e9culaires. Ce sont donc surtout des tests de d\u00e9pistages et leur n\u00e9gativit\u00e9 ne permet pas d\u2019exclure le diagnostic. Summary Rapid diagnostic tests (RDT), based on immunochromatography, are widely used for the diagnosis of infectious diseases. For viral respiratory inf...","Public health preparedness requires effective surveillance of and rapid response to infectious disease outbreaks. Inclusion of research activities within the outbreak setting provides important opportunities to maximize limited resources, to enhance gains in scientific knowledge, and ultimately to increase levels of preparedness. With rapid advances in laboratory technologies, banking and analysis of human genomic specimens can be conducted as part of public health investigations, enabling valuable research well into the future.","In December 2019, an outbreak of atypical pneumonia (Coronavirus disease 2019 -COVID-19) associated with a novel coronavirus (SARS-CoV-2) was reported in Wuhan city, Hubei province, China. The outbreak was traced to a seafood wholesale market and human to human transmission was confirmed. The rapid spread and the death toll of the new epidemic warrants immediate intervention. The intra-host genomic variability of SARS-CoV-2 plays a pivotal role in the development of effective antiviral agents and vaccines, but also in the design of accurate diagnostics. We analyzed NGS data derived from clinical samples of three Chinese patients infected with SARS-CoV-2, in order to identify small- and large-scale intra-host variations in the viral genome. We identified tens of low- or higher-frequency single nucleotide variations (SNVs) with variable density across the viral genome, affecting 7 out of 10 protein-coding viral genes. The majority of these SNVs corresponded to missense changes. The annot...","Abstract In response to SARS-CoV infection, neutralizing antibodies are generated against the Spike (S) protein. Determination of the active regions that allow viral escape from neutralization would enable the use of these antibodies for future passive immunotherapy. We immunized mice with UV-inactivated SARS-CoV to generate three anti-S monoclonal antibodies, and established several neutralization escape mutants with S protein. We identified several amino acid substitutions, including Y442F and V601G in the S1 domain and D757N and A834V in the S2 region. In the presence of each neutralizing antibody, double mutants with substitutions in both domains exhibited a greater growth advantage than those with only one substitution. Importantly, combining two monoclonal antibodies that target different epitopes effected almost complete suppression of wild type virus replication. Thus, for effective passive immunotherapy, it is important to use neutralizing antibodies that recognize both the S1...","AbstractMotivationAccurately determining the atomic structure of proteins represents a fundamental problem in the field of structural bioinformatics. A solution would be significant as protein structure information could be utilized in the medical field, e.g. in the development of vaccines for new viruses. This paper focuses on predicting the protein structure based on 3D images of the proteins captured through cryogenic electron microscopes (cryo-EM). A fully automated computationally efficient protein structure prediction method would be particularly beneficial in the field of cryo-EM as the technology allows researchers to photograph multiple large protein complexes in a single study, which means that a fast prediction method could allow for a high throughput of derived protein structures. We present a deep learning approach, DeepTracer, for predicting locations of the backbone atoms, secondary structure elements, and the amino acid types. In order to connect the predicted amino aci...","Mass spectrometry (MS) has found numerous applications in life sciences. It has high accuracy, sensitivity and wide dynamic range in addition to medium- to high-throughput capabilities. These features make MS a superior platform for analysis of various biomolecules including proteins, lipids, nucleic acids and carbohydrates. Until recently, MS was applied for protein detection and characterization. During the last decade, however, MS has successfully been used for molecular diagnostics of microbial and viral infections with the most notable applications being identification of pathogens, genomic sequencing, mutation detection, DNA methylation analysis, tracking of transmissions, and characterization of genetic heterogeneity. These new developments vastly expand the MS application from experimental research to public health and clinical fields. Matching of molecular techniques with specific requirements of the major MS platforms has produced powerful technologies for molecular diagnosti...","Abstract Newly emerging viruses such as the Ebola virus, severe acute respiratory syndrome (SARS)-, Middle East respiratory syndrome (MERS)-coronavirus, and the avian influenza virus are serious threats to public health and have become a global concern. The swine flu pandemic which occurred in 2009 reminded us of the Spanish flu that killed over 40 million people. Newly emerging viruses could be either a novel previously undescribed virus or a variant of a previously known virus. Examples of the novel viruses include HIV, the Ebola virus, and SARS viruses. Variants of previously described viruses are also called \u201cremerging viruses.\u201d They are mutants of known viruses but cause new epidemics with considerable virulence. The influenza virus that caused the 2009 swine flu pandemic was a variant of an existing virus.","Abstract Background A novel coronavirus (2019-nCoV) associated with human to human transmission and severe human infection has been recently reported from the city of Wuhan in China. Our objectives were to characterize the genetic relationships of the 2019-nCoV and to search for putative recombination within the subgenus of sarbecovirus. Methods Putative recombination was investigated by RDP4 and Simplot v3.5.1 and discordant phylogenetic clustering in individual genomic fragments was confirmed by phylogenetic analysis using maximum likelihood and Bayesian methods. Results Our analysis suggests that the 2019-nCoV although closely related to BatCoV RaTG13 sequence throughout the genome (sequence similarity 96.3%), shows discordant clustering with the Bat_SARS-like coronavirus sequences. Specifically, in the 5\u2032-part spanning the first 11,498 nucleotides and the last 3\u2032-part spanning 24,341\u201330,696 positions, 2019-nCoV and RaTG13 formed a single cluster with Bat_SARS-like coronavirus seque...","AbstractThe SARS-CoV-2 epidemic has rapidly spread outside China with major outbreaks occurring in Italy, South Korea and Iran. Phylogenetic analyses of whole genome sequencing data identified a distinct SARS-CoV-2 clade linked to travellers returning from Iran to Australia and New Zealand. This study highlights potential viral diversity driving the epidemic in Iran, and underscores the power of rapid genome sequencing and public data sharing to improve the detection and management of emerging infectious diseases.","The SARS-CoV nucleocapsid (N) protein is a major antigen in severe acute respiratory syndrome. It binds to the viral RNA genome and forms the ribonucleoprotein core. The SARS-CoV N protein has also been suggested to be involved in other important functions in the viral life cycle. Here we show that the N protein consists of two non-interacting structural domains, the N-terminal RNA-binding domain (RBD) (residues 45\u2013181) and the C-terminal dimerization domain (residues 248\u2013365) (DD), surrounded by flexible linkers. The C-terminal domain exists exclusively as a dimer in solution. The flexible linkers are intrinsically disordered and represent potential interaction sites with other protein and protein-RNA partners. Bioinformatics reveal that other coronavirus N proteins could share the same modular organization. This study provides information on the domain structure partition of SARS-CoV N protein and insights into the differing roles of structured and disordered regions in coronavirus n...","The current diagnosis of severe acute respiratory syndrome-associated coronavirus (SARS-CoV) relies on laboratory-based tests since its clinical features are nonspecific, unlike other respiratory pathogens. Therefore, the development of a rapid and simple method for on-site detection of SARS-CoV is crucial for the identification and prevention of future SARS outbreaks. In this study, a simple colorimetric and multiplex loop-mediated isothermal amplification (LAMP) assay was developed to rapid screening of severe acute respiratory syndrome-associated coronavirus (SARS-CoV). It can be visually detected based on color change and monitored in real-time with fluorescent signals. The performance of this assay, based on six primers targeting open reading frame (ORF1b) and nucleocapsid (N) genes located in different regions of the SARS-CoV, was compared with real-time RT-PCR assay using various concentrations of target genes. The detection limit of the LAMP assay was comparable to that of real...","The CoViD-19 outbreak has escalated to a pandemic in the last few weeks, with no signs of stopping. Pharmaceutical solutions based upon virologic studies, at this point, remain inconclusive. In contrast, this paper looks towards epidemiological models during this phase of viral growth, in particular, by providing a responsive, timely model of the R value based on the previous few days' results. Such an R value, although bearing less statistical precision due to limited sampling, could allow R to become a more effective, responsive standalone measure of infectious transmission. It demonstrates that the R value can be used as a dynamic, time-dependent indicator without the use of curve-fitting, and also estimates the most recent R-value of the CoViD-19 outbreak to be about 4.29, based on the data from the previous 3 days.","AbstractIn numerous instances, tracking the biological significance of a nucleic acid sequence can be augmented through the identification of environmental niches in which the sequence of interest is present. Many metagenomic datasets are now available, with deep sequencing of samples from diverse biological niches. While any individual metagenomic dataset can be readily queried using web-based tools, meta-searches through all such datasets are less accessible. In this brief communication, we demonstrate such a meta-meta-genomic approach, examining close matches to the Wuhan coronavirus 2019-nCoV in all high-throughput sequencing datasets in the NCBI Sequence Read Archive accessible with the keyword \u201cvirome\u201d. In addition to the homology to bat coronaviruses observed in descriptions of the 2019-nCoV sequence (F. Wu et al. 2020, Nature, doi.org/10.1038/s41586-020-2008-3; P. Zhou et al. 2020, Nature, doi.org/10.1038/s41586-020-2012-7), we note a strong homology to numerous sequence reads ...","As the COVID-19 outbreak spreads, there is a growing need for a compilation of conserved RNA genome regions in the SARS-CoV-2 virus along with their structural propensities to guide development of antivirals and diagnostics. Using sequence alignments spanning a range of betacoronaviruses, we rank genomic regions by RNA sequence conservation, identifying 79 regions of length at least 15 nucleotides as exactly conserved over SARS-related complete genome sequences available near the beginning of the COVID-19 outbreak. We then confirm the conservation of the majority of these genome regions across 739 SARS-CoV-2 sequences reported to date from the current COVID-19 outbreak, and we present a curated list of 30 \u2032SARS-related-conserved\u2032 regions. We find that known RNA structured elements curated as Rfam families and in prior literature are enriched in these conserved genome regions, and we predict additional conserved, stable secondary structures across the viral genome. We provide 106 \u2032SARS-...","SUMMARYThe rapid and escalating spread of SARS coronavirus 2 (SARS-CoV-2) poses an immediate public health emergency, and no approved therapeutics or vaccines are currently available. The viral spike protein S binds ACE2 on host cells to initiate molecular events that release the viral genome intracellularly. Soluble ACE2 inhibits entry of both SARS and SARS-2 coronaviruses by acting as a decoy for S binding sites, and is a candidate for therapeutic and prophylactic development. Using deep mutagenesis, variants of ACE2 are identified with increased binding to the receptor binding domain of S at a cell surface. Mutations are found across the interface and also at buried sites where they are predicted to enhance folding and presentation of the interaction epitope. The N90-glycan on ACE2 hinders association. The mutational landscape offers a blueprint for engineering high affinity ACE2 receptors to meet this unprecedented challenge.","The ongoing novel coronavirus pneumonia COVID-19 outbreak in Wuhan, China, has engendered numerous cases of infection and death. COVID-19 diagnosis relies upon nucleic acid detection; however, current recommended methods exhibit high false-negative rates, low sensitivity, and cannot identify other respiratory virus infections, thereby resulting patient misdiagnosis and impeding epidemic containment. Combining the advantages of target amplification and long-read, real-time nanopore sequencing, we developed nanopore target sequencing (NTS) to detect SARS-CoV-2 and other respiratory viruses simultaneously within 6-10 h. Parallel testing with approved qPCR kits of SARS-CoV-2 and NTS using 61 nucleic acid samples from suspected COVID-19 cases confirmed that NTS identified more infected patients as positive, and could also monitor for mutated nucleic acid sequence or other respiratory virus infection in the test sample. NTS is thus suitable for contemporary COVID-19 diagnosis; moreover, this...","Summary Objectives To assess associations between the functional polymorphisms G-2518A at the chemokine (C\u2013C motif) ligand 2 gene (CCL2) and mannose binding lectin (MBL) codon 54 variant (A/B) and susceptibility to SARS. Methods We genotyped the CCL2 G-2518A and MBL codon 54 variant (A/B) in 4 case\u2013control populations of Chinese descent, totally consisting of 932 patients with SARS and 982 control subjects. Results Both the high-CCL2-producing GG genotype and the low-MBL-producing B allele were consistently associated with increased risks of SARS-CoV infection in all 4 case\u2013control populations (joint P = 1.6 \u00d7 10\u22124 and 4.9 \u00d7 10\u22128, for CCL2 and MBL respectively), with no interaction between polymorphisms could be detected. Furthermore, all the 4 case\u2013control studies demonstrated a cumulative effect on risk of SARS-CoV infection for the combination of polymorphisms (joint P = 1.3 \u00d7 10\u221210). However, tests using the area under the curve (AUC) indicated that at this stage, the polymorphisms...","Abstract The development of rapid, accurate, and sensitive diagnostic methods for detecting pathogens is the basis for treating, controlling, and eradicating infectious diseases of veterinary importance. Scientific and technological advancements have revolutionized the field of veterinary diagnostics. Genome sequencing has allowed efficient, sensitive, and specific diagnostic assays to be developed based on the detection of nucleic acids. The integration of advances in biochemistry, proteomics, engineering, and medicine offers enormous potential for the rapid and accurate diagnosis of viral, microbial, genetic, and metabolic disease. In the future, polymerase chain reaction assays, microarray testing, genomic analysis, and metabolic profiling will be accomplished in a rapid, portable, sensitive, and cost-efficient manner.","Summary Objectives Understanding the novel coronavirus (COVID-19) mode of host cell recognition may help to fight the disease and save lives. The spike protein of coronaviruses is the main driving force for host cell recognition. Methods In this study, the COVID-19 spike binding site to the cell-surface receptor (Glucose Regulated Protein 78 (GRP78)) is predicted using combined molecular modeling docking and structural bioinformatics. The COVID-19 spike protein is modeled using its counterpart, the SARS spike. Results Sequence and structural alignments show that four regions, in addition to its cyclic nature have sequence and physicochemical similarities to the cyclic Pep42. Protein-protein docking was performed to test the four regions of the spike that fit tightly in the GRP78 Substrate Binding Domain \u03b2 (SBD\u03b2). The docking pose revealed the involvement of the SBD\u03b2 of GRP78 and the receptor-binding domain of the coronavirus spike protein in recognition of the host cell receptor. Concl...","Abstract Viruses do not carry their own protein biosynthesis machinery and the translation of viral proteins therefore requires that the virus usurps the machinery of the host cell. To allow optimal translation of viral proteins at the expense of cellular proteins, virus families have evolved a variety of methods to repress the host translation machinery, while allowing effective viral protein synthesis. Many viruses use noncanonical mechanisms that permit translation of their own RNAs under these conditions. Viruses have also developed mechanisms to evade host innate immune responses that would repress translation under conditions of viral infection, in particular PKR activation in response to double-stranded RNA (dsRNA). Importantly, the study of viral translation mechanisms has enormously enhanced our understanding of many aspects of the cellular protein biosynthesis pathway and its components. A number of unusual mechanisms of translation initiation that were first discovered in vi...","The sudden outbreak of novel coronavirus 2019 (COVID-19) increased the diagnostic burden of radiologists. In the time of an epidemic crisis, we hoped artificial intelligence (AI) to help reduce physician workload in regions with the outbreak, and improve the diagnosis accuracy for physicians before they could acquire enough experience with the new disease. Here, we present our experience in building and deploying an AI system that automatically analyzes CT images to detect COVID-19 pneumonia features. Different from conventional medical AI, we were dealing with an epidemic crisis. Working in an interdisciplinary team of over 30 people with medical and / or AI background, geographically distributed in Beijing and Wuhan, we were able to overcome a series of challenges in this particular situation and deploy the system in four weeks. Using 1,136 training cases (723 positives for COVID-19) from five hospitals, we were able to achieve a sensitivity of 0.974 and specificity of 0.922 on the t...","BACKGROUND: Array comparative genomic hybridization is a fast and cost-effective method for detecting, genotyping, and comparing the genomic sequence of unknown bacterial isolates. This method, as with all microarray applications, requires adequate coverage of probes targeting the regions of interest. An unbiased tiling of probes across the entire length of the genome is the most flexible design approach. However, such a whole-genome tiling requires that the genome sequence is known in advance. For the accurate analysis of uncharacterized bacteria, an array must query a fully representative set of sequences from the species' pan-genome. Prior microarrays have included only a single strain per array or the conserved sequences of gene families. These arrays omit potentially important genes and sequence variants from the pan-genome. RESULTS: This paper presents a new probe selection algorithm (PanArray) that can tile multiple whole genomes using a minimal number of probes. Unlike arrays b...","Summary Several new viral respiratory tract infectious diseases with epidemic potential that threaten global health security have emerged in the past 15 years. In 2003, WHO issued a worldwide alert for an unknown emerging illness, later named severe acute respiratory syndrome (SARS). The disease caused by a novel coronavirus (SARS-CoV) rapidly spread worldwide, causing more than 8000 cases and 800 deaths in more than 30 countries with a substantial economic impact. Since then, we have witnessed the emergence of several other viral respiratory pathogens including influenza viruses (avian influenza H5N1, H7N9, and H10N8; variant influenza A H3N2 virus), human adenovirus-14, and Middle East respiratory syndrome coronavirus (MERS-CoV). In response, various surveillance systems have been developed to monitor the emergence of respiratory-tract infections. These include systems based on identification of syndromes, web-based systems, systems that gather health data from health facilities (suc...","Protein disordered regions are segments of a protein chain that do not adopt a stable structure. Thus far, a variety of protein disorder prediction methods have been developed and have been widely used, not only in traditional bioinformatics domains, including protein structure prediction, protein structure determination and function annotation, but also in many other biomedical fields. The relationship between intrinsically-disordered proteins and some human diseases has played a significant role in disorder prediction in disease identification and epidemiological investigations. Disordered proteins can also serve as potential targets for drug discovery with an emphasis on the disordered-to-ordered transition in the disordered binding regions, and this has led to substantial research in drug discovery or design based on protein disordered region prediction. Furthermore, protein disorder prediction has also been applied to healthcare by predicting the disease risk of mutations in patie...","Summary The past decade has seen the emergence of several novel viruses that cause respiratory tract infections in human beings, including Middle East respiratory syndrome coronavirus (MERS-CoV) in Saudi Arabia, an H7N9 influenza A virus in eastern China, a swine-like influenza H3N2 variant virus in the USA, and a human adenovirus 14p1 also in the USA. MERS-CoV and H7N9 viruses are still a major worldwide public health concern. The pathogenesis and mode of transmission of MERS-CoV and H7N9 influenza A virus are poorly understood, making it more difficult to implement intervention and preventive measures. A united and coordinated global response is needed to tackle emerging viruses that can cause fatal respiratory tract infections and to fill major gaps in the understanding of the epidemiology and transmission dynamics of these viruses.","AbstractThe spread of the novel coronavirus (SARS-CoV-2) has triggered a global emergency, that demands urgent solutions for detection and therapy to prevent escalating health, social and economic impacts. The spike protein (S) of this virus enables binding to the human receptor ACE2, and hence presents a prime target for vaccines preventing viral entry into host cells1. The S proteins from SARS-CoV-1 and SARS-CoV-2 are similar2, but structural differences in the receptor binding domain (RBD) preclude the use of SARS-CoV-1\u2013specific neutralizing antibodies to inhibit SARS-CoV-23. Here we used comparative pangenomic analysis of all sequenced Betacoronaviruses to reveal that, among all core gene clusters present in these viruses, the envelope protein E shows a variant shared by SARS and SARS-Cov2 with two completely-conserved key functional features, an ion-channel and a PDZ-binding Motif (PBM). These features trigger a cytokine storm that activates the inflammasome, leading to increased ...","ABSTRACTCOVID-19 has caused a major epidemic worldwide, however, much is yet to be known about the epidemiology and evolution of the virus. One reason is that the challenges underneath sequencing HCoV-19 directly from clinical samples have not been completely tackled. Here we illustrate the application of amplicon and hybrid capture (capture)-based sequencing, as well as ultra-high-throughput metatranscriptomic (meta) sequencing in retrieving complete genomes, inter-individual and intra-individual variations of HCoV-19 from clinical samples covering a range of sample types and viral load. We also examine and compare the bias, sensitivity, accuracy, and other characteristics of these approaches in a comprehensive manner. This is, to date, the first work systematically implements amplicon and capture approaches in sequencing HCoV-19, as well as the first comparative study across methods. Our work offers practical solutions for genome sequencing and analyses of HCoV-19 and other emerging ...","Abstract This paper concerns study of the genome of the Wuhan Seafood Market isolate believed to represent the causative agent of the disease COVID-19. This is to find a short section or sections of viral protein sequence suitable for preliminary design proposal for a peptide synthetic vaccine and a peptidomimetic therapeutic, and to explore some design possibilities. The project was originally directed towards a use case for the Q-UEL language and its implementation in a knowledge management and automated inference system for medicine called the BioIngine, but focus here remains mostly on the virus itself. However, using Q-UEL systems to access relevant and emerging literature, and to interact with standard publically available bioinformatics tools on the Internet, did help quickly identify sequences of amino acids that are well conserved across many coronaviruses including 2019-nCoV. KRSFIEDLLFNKV was found to be particularly well conserved in this study and corresponds to the region...","AbstractThe fast and untraceable virus mutations take lives of thousands of people before the immune system can produce the inhibitory antibody. Recent outbreak of novel coronavirus infected and killed thousands of people in the world. Rapid methods in finding peptides or antibody sequences that can inhibit the viral epitopes of COVID-19 will save the life of thousands. In this paper, we devised a machine learning (ML) model to predict the possible inhibitory synthetic antibodies for Corona virus. We collected 1933 virus-antibody sequences and their clinical patient neutralization response and trained an ML model to predict the antibody response. Using graph featurization with variety of ML methods, we screened thousands of hypothetical antibody sequences and found 8 stable antibodies that potentially inhibit COVID-19. We combined bioinformatics, structural biology, and Molecular Dynamics (MD) simulations to verify the stability of the candidate antibodies that can inhibit the Corona v...","The ongoing coronavirus disease 2019 (COVID-19) pandemic is of global concern and has recently emerged in the US. In this paper, we construct a stochastic variant of the SEIR model to make a quasi-worst-case scenario prediction of the COVID-19 outbreak in the US West and East Coasts. The model is then fitted to current data and implemented using Runge-Kutta methods. Our computation results predict that the number of new cases would peak around mid-April and begin to abate by July, and that the number of cases of COVID-19 might be significantly mitigated by having greater numbers of functional testing kits available for screening. The model also showed how small changes in variables can make large differences in outcomes and highlights the importance of healthcare preparedness during pandemics.","Abstract Advances in molecular biology, bioinformatics, and robotics have allowed microarray technology to be used for in-depth, basic science studies in all fields of microbiology. Recently, translation of these basic science applications to clinical microbiology and infectious diseases has also progressed. From a clinical infectious disease perspective, genome-based organism identification, pathogen discovery, and antimicrobial susceptibility testing of problematic organisms offer the potential to yield diagnostic information that may not otherwise become available. Moreover, microarray-based studies have the ability to provide \u201csignatures\u201d of host cell transcriptional responses for individual pathogens and/or groups of pathogens. This type of information has the potential to confirm difficult diagnoses, to monitor responses to therapeutic intervention, or even to predict prognosis and sequelae following an infectious disease. Examples are presented to illustrate ways in which microa...","Abstract Many current and potential drug targets are membrane-bound or secreted proteins that are expressed and transported via the Sec61 secretory pathway. They are targeted to translocon channels across the membrane of the endoplasmic reticulum (ER) by signal peptides (SPs), which are temporary structures on the N-termini of their nascent chains. During translation, such proteins enter the lumen and membrane of the ER by a process known as co-translational translocation. Small molecules have been found that interfere with this process, decreasing protein expression by recognizing the unique structures of the SPs of particular proteins. The SP may thus become a validated target for designing drugs for numerous disorders, including certain hereditary diseases.","R\u00e9sum\u00e9 La d\u00e9tection, la quantification et l\u2019analyse g\u00e9n\u00e9tique des g\u00e9nomes viraux ont \u00e9t\u00e9 progressivement introduites dans la pratique quotidienne de la virologie m\u00e9dicale. Dans le cas des infections \u00e0 herp\u00e8svirus humains 6 et 7 (HHV-6, HHV-7), la d\u00e9tection de l\u2019ADN viral a permis d\u2019impliquer ces infections dans des maladies au-del\u00e0 de ce qui avait \u00e9t\u00e9 mis en \u00e9vidence par les m\u00e9thodes conventionnelles (culture, s\u00e9rologie) notamment chez les sujets immunod\u00e9prim\u00e9s. La quantification, en particulier gr\u00e2ce \u00e0 l\u2019apport r\u00e9cent de la PCR en temps r\u00e9el, est susceptible d\u2019\u00e9tablir la distinction entre latence et r\u00e9activation virales, de pr\u00e9dire la survenue d\u2019\u00e9v\u00e8nements cliniques, de suivre les effets d\u2019une th\u00e9rapeutique sp\u00e9cifique. L\u2019analyse g\u00e9n\u00e9tique a conduit \u00e0 l\u2019identification de variants et contribu\u00e9 \u00e0 caract\u00e9riser les mutations associ\u00e9es \u00e0 la r\u00e9sistance aux antiviraux. Comme le montre l\u2019\u00e9tude des herp\u00e8svirus, ces techniques mol\u00e9culaires ont aussi des limites qu\u2019il faut conna\u00eetre mais constitu...","Environmental factors, including seasonal climatic variability, can strongly impact on spatio-temporal patterns of infectious disease outbreaks. We assessed the effects of temperature and humidity on the global patterns of COVID-19 early outbreak dynamics during January-March 2020. Climatic variables were the best drivers of global variation of confirmed COVID-19 cases growth rates. Growth rates peaked in temperate regions of the Northern Hemisphere with mean temperature of ~5 degrees, and humidity of approx 0.6-1 kPa during the outbreak month, while they decreased in warmer and colder regions. The strong relationship between local climate and COVID-19 growth rates suggests the possibility of seasonal variation in the spatial pattern of outbreaks, with temperate regions of the Southern Hemisphere becoming at particular risk of severe outbreaks during the next months.","BACKGROUND: Infectious diseases emerge frequently in China, partly because of its large and highly mobile population. Therefore, a rapid and cost-effective pathogen screening method with broad coverage is required for prevention and control of infectious diseases. The availability of a large number of microbial genome sequences generated by conventional Sanger sequencing and next generation sequencing has enabled the development of a high-throughput high-density microarray platform for rapid large-scale screening of vertebrate pathogens. METHODS: An easy operating pathogen microarray (EOPM) was designed to detect almost all known pathogens and related species based on their genomic sequences. For effective identification of pathogens from EOPM data, a statistical enrichment algorithm has been proposed, and further implemented in a user-friendly web-based interface. RESULTS: Using multiple probes designed to specifically detect a microbial genus or species, EOPM can correctly identify k...","The last decades have been characterized by the appearance of a substantial number of newly recognized or novel infectious agents and by the re-emergence of infectious diseases with a global impact. The objective of this article is to briefly describe the model of hospital response for early diagnosis and prompt management of patients with highly contagious infectious diseases. We reviewed the main components of hospital preparedness in response to clustering of highly contagious diseases. A model for the initial hospital response to an epidemic in our referral Institute is discussed. Prompt recognition and identification is the initial and indispensable step in facing any communicable diseases, regardless of whether it is a prevalent, a newly emerging one or deliberately released. The importance of developing and implementing nontraditional methods of public health surveillance and a system that allows a wide and immediate dissemination of information and exchange of views on risk ass...","Bats are responsible for the zoonotic transmission of several major viral diseases including the 2003 SARS outbreak and the ongoing COVID-19 pandemic. While bat genomic sequencing studies have revealed characteristic adaptations of the innate immune system, functional genomic studies are urgently needed to provide a foundation for the molecular dissection of the tolerance of viral infections in bats. Here we report the establishment and screening of genome-wide RNAi library and CRISPR library for the model megabat, Pteropus Alecto. We used the complementary RNAi and CRISPR libraries to interrogate Pteropus Alecto cells for infection with two different viruses, mumps virus and Influenza A virus, respectively. Screening results converged on the endocytosis pathway and the protein secretory pathway as required for both viral infections. Additionally, we revealed a general dependence of the C-1-tetrahydrofolate synthase gene, MTHFD1, for viral replication in bat cells as well as in human c..."],"title":["Rapid metagenomic characterization of a case of imported COVID-19 in Cambodia","Characterisation of the transcriptome and proteome of SARS-CoV-2 using direct RNA sequencing and tandem mass spectrometry reveals evidence for a cell passage induced in-frame deletion in the spike glycoprotein that removes the furin-like cleavage site.","Scarce COVID-19 Testing Capabilities at Urgent Care Centers in States with Greatest Disease Burden","High sensitivity detection of coronavirus SARS-CoV-2 using multiplex PCR and a multiplex-PCR-based metagenomic method","Genome Detective Coronavirus Typing Tool for rapid identification and characterization of novel coronavirus genomes","Protein Structure and Sequence Reanalysis of 2019-nCoV Genome Refutes Snakes as Its Intermediate Host and the Unique Similarity between Its Spike Protein Insertions and HIV-1","Networking of Public Health Microbiology Laboratories Bolsters Europe\u2019s Defenses against Infectious Diseases","Evaluation of protection conferred by a vaccination program based on the H120 and CR88 commercial vaccines against a field variant of avian infectious bronchitis virus","Interpretable detection of novel human viruses from genome sequencing data","Rapid Detection of Novel Coronavirus (COVID-19) by Reverse Transcription-Loop-Mediated Isothermal Amplification","Accurate Identification of SARS-CoV-2 from Viral Genome Sequences using Deep Learning","Integrated Micropillar Polydimethylsiloxane Accurate CRISPR Detection (IMPACT) System for Rapid Viral DNA Sensing","The infectious aetiology of disease: the search for new agents","Les tests de diagnostic rapide des viroses respiratoires et des gastroent\u00e9rites virales: int\u00e9r\u00eats et limites","Human genomics and preparedness for infectious threats","SARS-CoV-2 exhibits intra-host genomic plasticity and low-frequency polymorphic quasispecies","A single amino acid substitution in the S1 and S2 Spike protein domains determines the neutralization escape phenotype of SARS-CoV","DeepTracer: Predicting Backbone Atomic Structure from High Resolution Cryo-EM Density Maps of Protein Complexes","Application of mass spectrometry to molecular diagnostics of viral infections","Chapter 21 New Emerging Viruses","Full-genome evolutionary analysis of the novel corona virus (2019-nCoV) rejects the hypothesis of emergence as a result of a recent recombination event","An emergent clade of SARS-CoV-2 linked to returned travellers from Iran","Modular organization of SARS coronavirus nucleocapsid protein","A Simple and Multiplex Loop-Mediated Isothermal Amplification (LAMP) Assay for Rapid Detection of SARS-CoV","A Novel Method for the Estimation of a Dynamic Effective Reproduction Number (Dynamic-R) in the CoViD-19 Outbreak","Identification of a Pangolin Niche for a 2019-nCoV-like Coronavirus via an Extensive Meta-metagenomic Search","RNA genome conservation and secondary structure in SARS-CoV-2 and SARS-related viruses","The sequence of human ACE2 is suboptimal for binding the S spike protein of SARS coronavirus 2","Nanopore target sequencing for accurate and comprehensive detection of SARS-CoV-2 and other respiratory viruses","Functional polymorphisms of the CCL2 and MBL genes cumulatively increase susceptibility to severe acute respiratory syndrome coronavirus infection","Future Veterinary Diagnostics","COVID-19 spike-host cell receptor GRP78 binding site prediction","Chapter 9 Viral Strategies to Subvert the Mammalian Translation Machinery","AI-assisted CT imaging analysis for COVID-19 screening: Building and deploying a medical AI system in four weeks","Efficient oligonucleotide probe selection for pan-genomic tiling arrays","Surveillance for emerging respiratory viruses","An Overview of Practical Applications of Protein Disorder Prediction and Drive for Faster, More Accurate Predictions","Emerging viral respiratory tract infections\u2014environmental risk factors and transmission","Functional pangenome analysis suggests inhibition of the protein E as a readily available therapy for COVID-2019","Multiple approaches for massively parallel sequencing of HCoV-19 (SARS-CoV-2) genomes directly from clinical samples","Computers and viral diseases. Preliminary bioinformatics studies on the design of a synthetic vaccine and a preventative peptidomimetic antagonist against the SARS-CoV-2 (2019-nCoV, COVID-19) coronavirus","Potential Neutralizing Antibodies Discovered for Novel Corona Virus Using Machine Learning","A Computational Model for Estimating the Progression of COVID-19 Cases in the US West and East Coasts","Microarrays and gene expression profiling in microbiology and infectious diseases: a clinician's perspective","The signal peptide as a new target for drug design","D\u00e9tection, quantification et analyse des g\u00e9nomes viraux dans les infections \u00e0 herp\u00e8svirus humains 6 et 7 (HHV-6, HHV-7)","Climate affects global patterns of COVID-19 early outbreak dynamics","An easy operating pathogen microarray (EOPM) platform for rapid screening of vertebrate pathogens","The Initial Hospital Response to an Epidemic","Orthogonal genome-wide screenings in bat cells identify MTHFD1 as a target of broad antiviral therapy"],"url":["https://doi.org/10.1101/2020.03.02.968818","https://doi.org/10.1101/2020.03.22.002204","https://doi.org/10.1101/2020.03.22.20040923","https://doi.org/10.1101/2020.03.12.988246","https://doi.org/10.1093/bioinformatics/btaa145","https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7099673/","https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5834927/","https://doi.org/10.3382/japr.2013-00828","https://doi.org/10.1101/2020.01.29.925354","https://doi.org/10.1101/2020.02.19.20025155","https://doi.org/10.1101/2020.03.13.990242","https://doi.org/10.1101/2020.03.17.994137","https://doi.org/10.1383/medc.33.3.37.61122","https://doi.org/10.1016/s1773-035x(15)30200-8","https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2808735/","https://doi.org/10.1101/2020.03.27.009480","https://doi.org/10.1016/j.micinf.2008.05.009","https://doi.org/10.1101/2020.02.12.946772","http://europepmc.org/articles/pmc5831079?pdf=render","https://doi.org/10.1016/b978-0-12-800838-6.00021-7","https://doi.org/10.1016/j.meegid.2020.104212","https://doi.org/10.1101/2020.03.15.992818","https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7089556/","https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7097549/","https://doi.org/10.1101/2020.02.22.20023267","https://doi.org/10.1101/2020.02.08.939660","https://doi.org/10.1101/2020.03.27.012906","https://doi.org/10.1101/2020.03.16.994236","https://doi.org/10.1101/2020.03.04.20029538","https://doi.org/10.1016/j.jinf.2015.03.006","https://doi.org/10.1053/j.jepm.2010.05.006","https://doi.org/10.1016/j.jinf.2020.02.026","https://doi.org/10.1016/s1877-1173(09)90009-6","https://doi.org/10.1101/2020.03.19.20039354","https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2753849/","https://doi.org/10.1016/s1473-3099(14)70840-0","https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4519904/","https://doi.org/10.1016/s1473-3099(14)70831-x","https://doi.org/10.1101/2020.02.17.952895","https://doi.org/10.1101/2020.03.16.993584","https://doi.org/10.1016/j.compbiomed.2020.103670","https://doi.org/10.1101/2020.03.14.992156","https://doi.org/10.1101/2020.03.24.20043026","https://doi.org/10.1016/j.clinmicnews.2004.09.004","https://doi.org/10.1016/j.bmcl.2020.127115","https://doi.org/10.1016/s0923-2532(03)00065-6","https://doi.org/10.1101/2020.03.23.20040501","https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3848773/","https://doi.org/10.1016/j.arcmed.2005.05.003","https://doi.org/10.1101/2020.03.29.014209"],"x":[0.3325246459869446,0.2712074127448634,0.8444387805692368,0.347194952994359,0.3398907779611516,0.3408994750037282,0.9366216014881423,0.41416015331024925,0.29816880529192247,0.3488854141573288,0.3174125130022033,0.33925857074739807,0.27881110516815705,0.028477208484666918,0.9386200604830588,0.36097379047751993,0.3505099411457334,0.2846437830385081,0.27469082809158685,0.5795480303478483,0.3438245861886615,0.5893031186453876,0.25834012549238256,0.35767167789930127,0.7918332595551703,0.3135090383857653,0.3466161214078789,0.3144287736478418,0.37260842352995355,0.45129191570069,0.2730343103114578,0.2902518264688611,0.3704969592248106,0.834412981724943,0.2973373112970192,0.6192237077013465,0.2668490276290778,0.5589698872839037,0.34413179145594824,0.32126803025158773,0.3185440071311221,0.3708216513505467,0.7513883823480055,0.2762997127356595,0.23776673205229032,0.02483407694871943,0.7570520638174792,0.29618298410076377,0.9125766595558245,0.38608769318709535],"y":[0.50204336047393,0.18198865876739698,0.6481705182525053,0.5562927563719667,0.43854486928266034,0.41972387179424303,0.6671789651656425,0.4988400201788137,0.5124285670211735,0.5651563141343305,0.48715838105840104,0.5508396033158974,0.5496438328773356,0.47028420642551,0.6648066702723477,0.3455005818544789,0.12721577119966945,0.31502900504375747,0.5166303218328695,0.33228909621514807,0.40960073897301535,0.6920559727782476,0.16103692231033007,0.5554393024964405,0.868528410601168,0.4654838924212808,0.3543428289646162,0.24597088602906744,0.5062581463447147,0.16535242488845514,0.5295894358685618,0.26302795753833896,0.30774401815540225,0.5939511346289676,0.4858586586589791,0.4177651017850595,0.46980043006224126,0.34532511232705626,0.18779798986067728,0.48605663398148025,0.5039602886591966,0.11527546642041508,0.8154643044483253,0.5408135053616254,0.14172186759243363,0.4704692207239298,0.8472248528300728,0.5468476078988428,0.6396167607997801,0.311450516319903]},"selected":{"id":"27477"},"selection_policy":{"id":"27499"}},"id":"27405","type":"ColumnDataSource"},{"attributes":{"fill_color":{"field":"color"},"line_color":{"field":"color"},"radius":{"field":"rad","units":"data"},"x":{"field":"x"},"y":{"field":"y"}},"id":"27467","type":"Circle"},{"attributes":{"source":{"id":"27405"}},"id":"27441","type":"CDSView"},{"attributes":{},"id":"27512","type":"BasicTickFormatter"},{"attributes":{"formatter":{"id":"27510"},"ticker":{"id":"27452"}},"id":"27451","type":"LinearAxis"},{"attributes":{"active_drag":"auto","active_inspect":"auto","active_multi":null,"active_scroll":"auto","active_tap":"auto","tools":[{"id":"27459"},{"id":"27460"},{"id":"27461"}]},"id":"27462","type":"Toolbar"},{"attributes":{},"id":"27421","type":"BasicTicker"},{"attributes":{"below":[{"id":"27451"}],"center":[{"id":"27454"},{"id":"27458"}],"left":[{"id":"27455"}],"plot_height":340,"plot_width":900,"renderers":[{"id":"27469"}],"title":{"id":"27492"},"toolbar":{"id":"27462"},"toolbar_location":"below","x_range":{"id":"27443"},"x_scale":{"id":"27447"},"y_range":{"id":"27445"},"y_scale":{"id":"27449"}},"id":"27442","subtype":"Figure","type":"Plot"},{"attributes":{"text":"&lt;h3&gt;Paragraph ranking&lt;/h3&gt;"},"id":"27483","type":"Div"},{"attributes":{"axis":{"id":"27451"},"ticker":null},"id":"27454","type":"Grid"},{"attributes":{},"id":"27494","type":"StringFormatter"},{"attributes":{},"id":"27509","type":"UnionRenderers"},{"attributes":{"js_property_callbacks":{"change:indices":[{"id":"27478"}]}},"id":"27477","type":"Selection"},{"attributes":{"children":[{"id":"27481"}]},"id":"27484","type":"Row"},{"attributes":{},"id":"27412","type":"LinearScale"},{"attributes":{},"id":"27496","type":"StringFormatter"},{"attributes":{"children":[{"id":"27440"},{"id":"27407"}]},"id":"27486","type":"Row"},{"attributes":{},"id":"27425","type":"ResetTool"},{"attributes":{"columns":[{"id":"27473"},{"id":"27474"}],"height":340,"source":{"id":"27406"},"view":{"id":"27476"},"width":900},"id":"27475","type":"DataTable"},{"attributes":{"children":[{"id":"27483"}]},"id":"27487","type":"Row"},{"attributes":{"below":[{"id":"27416"}],"center":[{"id":"27419"},{"id":"27423"}],"left":[{"id":"27420"}],"plot_height":340,"plot_width":900,"renderers":[{"id":"27434"}],"title":{"id":"27490"},"toolbar":{"id":"27427"},"toolbar_location":"below","x_range":{"id":"27408"},"x_scale":{"id":"27412"},"y_range":{"id":"27410"},"y_scale":{"id":"27414"}},"id":"27407","subtype":"Figure","type":"Plot"},{"attributes":{},"id":"27502","type":"BasicTickFormatter"},{"attributes":{},"id":"27408","type":"Range1d"},{"attributes":{"formatter":{"id":"27502"},"ticker":{"id":"27421"}},"id":"27420","type":"LinearAxis"},{"attributes":{"callback":null},"id":"27424","type":"TapTool"},{"attributes":{},"id":"27507","type":"StringEditor"},{"attributes":{"data_source":{"id":"27406"},"glyph":{"id":"27467"},"hover_glyph":null,"muted_glyph":null,"nonselection_glyph":{"id":"27468"},"selection_glyph":null,"view":{"id":"27470"}},"id":"27469","type":"GlyphRenderer"},{"attributes":{"callback":null,"tooltips":[["Title","@title"],["Abstract","@text"],["Published","@date"],["Authors","@authors"],["Journal","@journal"],["URL","@url"]]},"id":"27426","type":"HoverTool"},{"attributes":{},"id":"27504","type":"StringFormatter"},{"attributes":{"editor":{"id":"27507"},"field":"text","formatter":{"id":"27506"},"title":"Passage","width":770},"id":"27474","type":"TableColumn"},{"attributes":{},"id":"27447","type":"LinearScale"},{"attributes":{},"id":"27505","type":"StringEditor"},{"attributes":{"children":[{"id":"27475"},{"id":"27442"}]},"id":"27488","type":"Row"},{"attributes":{"editor":{"id":"27495"},"field":"scr","formatter":{"id":"27494"},"title":"Score","width":30},"id":"27438","type":"TableColumn"},{"attributes":{},"id":"27495","type":"StringEditor"},{"attributes":{},"id":"27499","type":"UnionRenderers"},{"attributes":{"args":{"sd":{"id":"27405"},"sp":{"id":"27406"}},"code":"\n    if (sd.selected.indices.length == 0) {\n      return;\n    }\n    var sel = sd.selected.indices[0];\n    var new_selected = [];\n    for (var i = 0; i &lt; sp.data['doc'].length; i++) {\n      if (sp.data['doc'][i] == sd.data['doc'][sel]) {\n        new_selected.push(i);\n      }\n    }\n    if (JSON.stringify(new_selected) != JSON.stringify(sp.selected.indices) &amp;&amp;\n    !(sp.selected.indices.length == 1 &amp;&amp; new_selected.includes(sp.selected.indices[0]))) {\n      sp.selected.indices = new_selected;\n      sp.change.emit();\n    }\n    "},"id":"27478","type":"CustomJS"},{"attributes":{"data_source":{"id":"27405"},"glyph":{"id":"27432"},"hover_glyph":null,"muted_glyph":null,"nonselection_glyph":{"id":"27433"},"selection_glyph":null,"view":{"id":"27435"}},"id":"27434","type":"GlyphRenderer"},{"attributes":{"axis":{"id":"27416"},"ticker":null},"id":"27419","type":"Grid"},{"attributes":{"axis":{"id":"27455"},"dimension":1,"ticker":null},"id":"27458","type":"Grid"},{"attributes":{},"id":"27497","type":"StringEditor"},{"attributes":{"axis":{"id":"27420"},"dimension":1,"ticker":null},"id":"27423","type":"Grid"},{"attributes":{"fill_color":{"field":"color"},"line_color":{"field":"color"},"radius":{"field":"rad","units":"data"},"x":{"field":"x"},"y":{"field":"y"}},"id":"27432","type":"Circle"},{"attributes":{"callback":null,"tooltips":[["Title","@title"],["Passage","@text"],["Published","@date"],["Authors","@authors"],["Journal","@journal"],["URL","@url"]]},"id":"27461","type":"HoverTool"},{"attributes":{},"id":"27456","type":"BasicTicker"},{"attributes":{"editor":{"id":"27497"},"field":"title","formatter":{"id":"27496"},"title":"Document title","width":770},"id":"27439","type":"TableColumn"},{"attributes":{},"id":"27417","type":"BasicTicker"},{"attributes":{"children":[{"id":"27484"},{"id":"27485"},{"id":"27486"},{"id":"27487"},{"id":"27488"}]},"id":"27489","type":"Column"},{"attributes":{},"id":"27410","type":"Range1d"},{"attributes":{"text":""},"id":"27492","type":"Title"},{"attributes":{},"id":"27506","type":"StringFormatter"},{"attributes":{"formatter":{"id":"27512"},"ticker":{"id":"27456"}},"id":"27455","type":"LinearAxis"},{"attributes":{},"id":"27449","type":"LinearScale"},{"attributes":{"data":{"authors":["Khabbaz, Rima F; Moseley, Robin R; Steiner, Riley J; Levitt, Alexandra M; Bell, Beth P","Khabbaz, Rima; Bell, Beth P.; Schuchat, Anne; Ostroff, Stephen M.; Moseley, Robin; Levitt, Alexandra; Hughes, James M.","Jessica E. Manning; Jennifer A. Bohl; Sreyngim Lay; Sophana Chea; Ly Sovann; Yi Sengdoeurn; Seng Heng; Chan Vuthy; Katrina Kalantar; Vida Ahyong; Michelle Tan; Jonathan Sheu; Cristina M. Tato; Joseph L. DeRisi; Laurence Baril; Veasna Duong; Philippe Dussart; Erik A. Karlsson","Brisse, Sylvain; Brehony, Carina; Concei\u00e7\u00e3o, Teresa; Cubero, Meritxell; Glasner, Corinna; Le Gouil, Meriadeg; Renvois\u00e9, Aur\u00e9lie; Sheppard, Sam; Weinert, Lucy A.","Richard E. Grewelle","Verdaguer, Nuria; Ferrero, Diego; Murthy, Mathur R. N.","Jessica E. Manning; Jennifer A. Bohl; Sreyngim Lay; Sophana Chea; Ly Sovann; Yi Sengdoeurn; Seng Heng; Chan Vuthy; Katrina Kalantar; Vida Ahyong; Michelle Tan; Jonathan Sheu; Cristina M. Tato; Joseph L. DeRisi; Laurence Baril; Veasna Duong; Philippe Dussart; Erik A. Karlsson","Robinson, Esther R; Walker, Timothy M; Pallen, Mark J","Roux, Simon; Matthijnssens, Jelle; Dutilh, Bas E.","Dong, J.; Ismail, N.; Walker, D.H.","Yang, Weizhong; Lan, Yajia; Sun, Qiao; Wang, Jinfeng; Li, Zhongjie","Colb\u00e8re-Garapin, Florence; Blondel, Bruno; Saulnier, Aure; Pelletier, Isabelle; Labadie, Karine","Brisse, Sylvain; Brehony, Carina; Concei\u00e7\u00e3o, Teresa; Cubero, Meritxell; Glasner, Corinna; Le Gouil, Meriadeg; Renvois\u00e9, Aur\u00e9lie; Sheppard, Sam; Weinert, Lucy A.","Chan, Jasper Fuk-Woo; Yuan, Shuofeng; Kok, Kin-Hang; To, Kelvin Kai-Wang; Chu, Hin; Yang, Jin; Xing, Fanfan; Liu, Jieling; Yip, Cyril Chik-Yan; Poon, Rosana Wing-Shan; Tsoi, Hoi-Wah; Lo, Simon Kam-Fai; Chan, Kwok-Hung; Poon, Vincent Kwok-Man; Chan, Wan-Mui; Ip, Jonathan Daniel; Cai, Jian-Piao; Cheng, Vincent Chi-Chung; Chen, Honglin; Hui, Christopher Kim-Ming; Yuen, Kwok-Yung","McLoughlin, Kevin S.","Timothy R. Abbott; Girija Dhamdhere; Yanxia Liu; Xueqiu Lin; Laine Goudy; Leiping Zeng; Augustine Chemparathy; Stephen Chmura; Nicholas S. Heaton; Robert Debs; Tara Pande; Drew Endy; Marie La Russa; David B. Lewis; Lei S. Qi","Uzoma, Ijeoma; Zhu, Heng","Kenneth Siu-Sing Leung; Timothy Ting-Leung Ng; Alan Ka-Lun Wu; Miranda Chong-Yee Yau; Hiu-Yin Lao; Ming-Pan Choi; Kingsley King-Gee Tam; Lam-Kwong Lee; Barry Kin-Chung Wong; Alex Yat-Man Ho; Kam-Tong Yip; Kwok-Cheung Lung; Raymond Wai-To Liu; Eugene Yuk-Keung Tso; Wai-Shing Leung; Man-Chun Chan; Yuk-Yung Ng; Kit-Man Sin; Kitty Sau-Chun Fung; Sandy Ka-Yee Chau; Wing-Kin To; Tak-Lun Que; David Ho-Keung Shum; Shea Ping Yip; Wing-Cheong Yam; Gilman Kit Hang Siu","Cleemput, S.; Dumon, W.; Fonseca, V.; Karim, W. A.; Giovanetti, M.; Alcantara, L. C.; Deforche, K.; de Oliveira, T.","Yinhua Zhang; Nelson Odiwuor; Jin Xiong; Luo Sun; Raphael Ohuru Nyaruaba; Hongping Wei; Nathan A Tanner","Henderson, Joan C.","Edison Ong; Mei U Wong; Anthony Huffman; Yongqun He","","Cook, Jane K.A.","Rovnak, Joel; Perera, Rushika; Hopken, Matthew W.; Read, Jenna; Waller, Derrick M.; Cohrs, Randall J.","Lu, Roujian; Zhao, Xiang; Li, Juan; Niu, Peihua; Yang, Bo; Wu, Honglong; Wang, Wenling; Song, Hao; Huang, Baoying; Zhu, Na; Bi, Yuhai; Ma, Xuejun; Zhan, Faxian; Wang, Liang; Hu, Tao; Zhou, Hong; Hu, Zhenhong; Zhou, Weimin; Zhao, Li; Chen, Jing; Meng, Yao; Wang, Ji; Lin, Yang; Yuan, Jianying; Xie, Zhihao; Ma, Jinmin; Liu, William J; Wang, Dayan; Xu, Wenbo; Holmes, Edward C; Gao, George F; Wu, Guizhen; Chen, Weijun; Shi, Weifeng; Tan, Wenjie","Hsin, Wei-Chen; Chang, Chan-Hua; Chang, Chi-You; Peng, Wei-Hao; Chien, Chung-Liang; Chang, Ming-Fu; Chang, Shin C.","Minfeng Xiao; Xiaoqing Liu; Jingkai Ji; Min Li; Jiandong Li; Lin Yang; Wanying Sun; Peidi Ren; Guifang Yang; Jincun Zhao; Tianzhu Liang; Huahui Ren; Tian Chen; Huanzi Zhong; Wenchen Song; Yanqun Wang; Ziqing Deng; Yanping Zhao; Zhihua Ou; Daxi Wang; Jielun Cai; Xinyi Cheng; Taiqing Feng; Honglong Wu; Yanping Gong; Huanming Yang; Jian Wang; Xun Xu; Shida Zhu; Fang Chen; Yanyan Zhang; Weijun Chen; Yimin Li; Junhua Li","","Roux, Simon; Matthijnssens, Jelle; Dutilh, Bas E.","Jessica E. Manning; Jennifer A. Bohl; Sreyngim Lay; Sophana Chea; Ly Sovann; Yi Sengdoeurn; Seng Heng; Chan Vuthy; Katrina Kalantar; Vida Ahyong; Michelle Tan; Jonathan Sheu; Cristina M. Tato; Joseph L. DeRisi; Laurence Baril; Veasna Duong; Philippe Dussart; Erik A. Karlsson","Kiselev, Daniel; Matsvay, Alina; Abramov, Ivan; Dedkov, Vladimir; Shipulin, German; Khafizov, Kamil","Hussey, Caroline","PUMFERY, ANNE; BERRO, REEM; KASHANCHI, FATAH","Shen, Minzhe; Zhou, Ying; Ye, Jiawei; Abdullah AL-maskri, Abdu Ahmed; Kang, Yu; Zeng, Su; Cai, Sheng","H\u00e4fner, Sophia","Ming Wang; Aisi Fu; Ben Hu; Yongqing Tong; Ran Liu; Jiashuang Gu; Jianghao Liu; Wen Jiang; Gaigai Shen; Wanxu Zhao; Dong Men; Lilei Yu; Zixin Deng; Yan Li; Tiangang Liu","Chung, Grace TY; Chiu, Rossa WK; Cheung, Jo LK; Jin, Yongjie; Chim, Stephen SC; Chan, Paul KS; Lo, YM Dennis","SHAO, HONGWEI; LAN, DONGMING; DUAN, ZHAOHUI; LIU, ZEHUAN; MIN, JUN; ZHANG, LICHUN; HUANG, JIAN; SU, JING; CHEN, SHANGWU; XU, ANLONG","Nina H Fefferman; Eric Lofgren; Nianpeng Li; Pieter Blue; David Weber; Abdul-Aziz Yakubu","Zhou, Peng; Yang, Xing-Lou; Wang, Xian-Guang; Hu, Ben; Zhang, Lei; Zhang, Wei; Si, Hao-Rui; Zhu, Yan; Li, Bei; Huang, Chao-Lin; Chen, Hui-Dong; Chen, Jing; Luo, Yun; Guo, Hua; Jiang, Ren-Di; Liu, Mei-Qin; Chen, Ying; Shen, Xu-Rui; Wang, Xi; Zheng, Xiao-Shuang; Zhao, Kai; Chen, Quan-Jiao; Deng, Fei; Liu, Lin-Lin; Yan, Bing; Zhan, Fa-Xian; Wang, Yan-Yi; Xiao, Geng-Fu; Shi, Zheng-Li","Sara Cleemput; Wim Dumon; Vagner Fonseca; Wasim Abdool Karim; Marta Giovanetti; Luiz Carlos Alcantara; Koen Deforche; Tulio de Oliveira","Jessica E. Manning; Jennifer A. Bohl; Sreyngim Lay; Sophana Chea; Ly Sovann; Yi Sengdoeurn; Seng Heng; Chan Vuthy; Katrina Kalantar; Vida Ahyong; Michelle Tan; Jonathan Sheu; Cristina M. Tato; Joseph L. DeRisi; Laurence Baril; Veasna Duong; Philippe Dussart; Erik A. Karlsson","Timokratis Karamitros; Gethsimani Papadopoulou; Maria Bousali; Anastasios Mexias; Sotiris Tsiodras; Andreas Mentis","Wei Lin; Longfei Hu; Yan Zhang; Joshua D. Ooi; Ting Meng; Peng Jin; Xiang Ding; Longkai Peng; Lei Song; Zhou Xiao; Xiang Ao; Xiangcheng Xiao; Qiaoling Zhou; Ping Xiao; Jue Fan; Yong Zhong","Miyssa I. Abdelmageed; Abdelrahman H. Abdelmoneim; Mujahed I. Mustafa; Nafisa M. Elfadol; Naseem S. Murshed; Shaza W. Shantier; Abdelrafie M. Makhawi","Joseph R Fauver; Mary E. Petrone; Emma B Hodcroft; Kayoko Shioda; Hanna Y Ehrlich; Alexander G. Watts; Chantal B.F. Vogels; Anderson F. Brito; Tara Alpert; Anthony Muyombwe; Jafar Razeq; Randy Downing; Nagarjuna R. Cheemarla; Anne L Wyllie; Chaney C. Kalinich; Isabel Ott; Josh Quick; Nicholas J. Loman; Karla M. Neugebauer; Alexander L. Greninger; Keith R. Jerome; Pavitra Roychoundhury; Hong Xie; Lasata Shrestha; Meei-Li Huang; Virginia E. Pitzer; Akiko Iwasaki; Saad B. Omer; Kamran Khan; Isaac Bogoch; Richard A. Martinello; Ellen F. Foxman; Marie-Louise Landry; Richard A Neher; Albert I Ko; Nathan D. Grubaugh","Jessica E. Manning; Jennifer A. Bohl; Sreyngim Lay; Sophana Chea; Ly Sovann; Yi Sengdoeurn; Seng Heng; Chan Vuthy; Katrina Kalantar; Vida Ahyong; Michelle Tan; Jonathan Sheu; Cristina M. Tato; Joseph L. DeRisi; Laurence Baril; Veasna Duong; Philippe Dussart; Erik A. Karlsson","Xiaoqiang Huang; Robin Pearce; Yang Zhang","Xiang He; Lei Zhang; Qin Ran; Anying Xiong; Junyi Wang; Dehong Wu; Feng Chen; Guoping Li"],"color":["#ef8a62","#ef8a62","#ef8a62","#ef8a62","#ef8a62","#ef8a62","#ef8a62","#ef8a62","#ef8a62","#ef8a62","#ef8a62","#ef8a62","#ef8a62","#ef8a62","#ef8a62","#ef8a62","#ef8a62","#ef8a62","#ef8a62","#ef8a62","#ef8a62","#ef8a62","#ef8a62","#ef8a62","#ef8a62","#ef8a62","#ef8a62","#ef8a62","#ef8a62","#ef8a62","#ef8a62","#ef8a62","#ef8a62","#ef8a62","#ef8a62","#ef8a62","#ef8a62","#ef8a62","#ef8a62","#ef8a62","#ef8a62","#ef8a62","#ef8a62","#ef8a62","#ef8a62","#ef8a62","#ef8a62","#ef8a62","#ef8a62","#ef8a62"],"date":["2014-07-11","2015-12-31","2020-03-05","2014-03-31","2020-03-11","2014-10-14","2020-03-05","2013-04-29","2019-12-31","2017-12-31","2017-12-31","2005-04-30","2014-03-31","2020-02-21","2011-09-19","2020-03-14","2013-01-17","2020-03-31","2020","2020-02-29","2007-12-31","2020-03-21","2017-09-30","2008-12-31","2017-11-08","2020-02-28","2018-05-24","2020-03-17","2008-05-31","2019-12-31","2020-03-05","2020-02-14","1992-12-31","2008-12-31","2020-03-01","2013-02-28","2020-03-06","2005-10-18","2006-10-06","2020-03-20","2020-02-03","2020-02-02","2020-03-05","2020-03-28","2020-02-18","2020-02-11","2020-03-26","2020-03-05","2020-03-31","2020-02-05"],"doc":["alnj42iy","p2ahklyl","fu3cl7lq","a4mf2vnp","y12cq60a","uirxpvwv","fu3cl7lq","vuupgg7i","1qj9ri18","1h87wldg","dgi4ibym","4uw4frer","a4mf2vnp","j3avpu1y","9ntgqyhw","xieqswct","t96j8qt0","uz91cd6h","st5idleq","cv3qgno3","nezgzovk","ld0vo1rl","s5y6gyut","ovxmy1as","8s7qw9gv","m6akijzn","lcu1mgno","wbyqmvhs","39vjafky","1qj9ri18","fu3cl7lq","u2n8so5w","os6f458c","9dyh8bj2","mjdqrgjr","29bihd2b","s4d442kx","i948aq4b","aevfqe48","fcnkwwpw","o877uul1","gxstlzuk","fu3cl7lq","jxm69ndw","mm2sp48e","6ojmmmuj","8m06zdho","fu3cl7lq","imkeghfd","mjhez5im"],"journal":["The Lancet","Mandell, Douglas, and Bennett's Principles and Practice of Infectious Diseases","","Research in Microbiology","","IUCrJ","","Genome Med","Reference Module in Life Sciences","Diagnostic Molecular Pathology","Early Warning for Infectious Disease Outbreak","Microbes and Infection","Research in Microbiology","The Lancet","Brief Funct Genomics","","Genomics Proteomics Bioinformatics","","Bioinformatics (Oxford, England)","","Managing Tourism Crises","","Disease-a-Month","Poultry Diseases","Viruses","The Lancet","J Biomed Sci","","Clinical Microbiology and Infection","Reference Module in Life Sciences","","Viruses","Safety in Industrial Microbiology and Biotechnology","Medical Applications of Mass Spectrometry","Journal of Pharmaceutical Analysis","Microbes and Infection","","BMC Infect Dis","J Clin Immunol","","Nature","","","","","","","","",""],"rad":[0.02,0.019955886442877498,0.016441588714313567,0.01497131919636921,0.013641135304624727,0.011699369484444288,0.009967936629536926,0.008575187949020225,0.008497580972839593,0.008023326660803402,0.007681513994983761,0.007658224643813495,0.007277455855431494,0.007265950875141835,0.007154904407877384,0.007084809876946651,0.006827045368695957,0.006591493326940869,0.005920767246317463,0.0057390142283621735,0.005581619288107211,0.005536743797632353,0.005450044010946512,0.005334716724784637,0.005332019106167385,0.0052315176718071895,0.0051660741500694635,0.005150838049139019,0.005118782356981951,0.0046360896134145044,0.004555881808373478,0.004429474785194792,0.004323860554921039,0.00423536025851833,0.004182538022709248,0.0040826976437642664,0.004078888895073996,0.004012049930365141,0.003979699019429717,0.003968145819838853,0.003961540445326956,0.003952205156345231,0.0038683470027714552,0.0037147387855201042,0.0036225383124077804,0.003444761531789772,0.0033711148556873557,0.0030922744475389615,0.003066893726723651,0.003],"scr":[1.0,0.997,0.791,0.704,0.626,0.512,0.41,0.328,0.323,0.295,0.275,0.274,0.252,0.251,0.244,0.24,0.225,0.211,0.172,0.161,0.152,0.149,0.144,0.137,0.137,0.131,0.127,0.127,0.125,0.096,0.092,0.084,0.078,0.073,0.07,0.064,0.063,0.06,0.058,0.057,0.057,0.056,0.051,0.042,0.037,0.026,0.022,0.005,0.004,0.0],"text":["For any country, eff ective control of infectious diseases is dependent on focused eff orts to decrease endemic infections in at-risk and susceptible populations, combined with strong epidemiological and laboratory capacity to detect and stop emerging threats. Inherent in these eff orts are well established links between clinical, public health, and research communities working with non-traditional partners such as industry, law enforcement, businesses, and the media. Although at their core these collaborations need to be established locally, such crucial partnerships have to extend across state, national, and international levels. Together, these alliances can form a global infrastructure and system to detect and control infectious diseases. With that aim, in February, 2014, the USA and other nations joined WHO, the Food and Agricultural Organization of the United Nations, and the World Organization for Animal Health in a new global health security agenda designed to enhance strategies to prevent, detect, and respond to threats of infectious disease. 123 Partner countries will work together to meet a series of objectives covering crucial issues aff ecting control of infectious disease such as antimicrobial resistance, biosafety and biosecurity, global disease detection and response networks, and emergency responses. Much emphasis has been placed on the aptly named perfect storm inherent in today's globalised world for rapid emergence and spread of infectious diseases. Despite this persistent challenge, extraordinary achievements in science and technology continue to bolster the capacity to recognise and respond to these threats. Scientifi cally, next-generation sequencing and state-of-the-art bioinformatics capabilities are transforming the practice of microbiology and the speed at which infectious pathogens can be detected and analysed. 85,124 These technologies provide a new level of information about infectious pathogens and therefore off er tremendous opportunities for public health eff orts to detect and control outbreaks, target antimicrobial resistance, and develop and refi ne treatments and vaccines. [125] [126] [127] Additionally, the increasing understanding of the role of human microbiomes in disease development and new links between infectious agents and chronic diseases opens new possibilities for disease prevention and treatment. 128 Advances in electronic communications have also been essential and have enabled rapid sharing of health and safety information and expertise by health-care and public health workers, researchers, law enforcement, the media, and private individuals. These communication capacities can facilitate and enhance research and public health partnerships worldwide, generating broad sharing of research fi ndings and health updates in near real time. The latest surge in health communications has come from social media tools, which are increasingly being used to extend the reach of public health information. To ensure that these rapidly changing scientifi c and technological advances are routinely used in public health is a crucial challenge and priority.","More recent efforts have focused on the use of new technologies and host targets to better understand and reduce antimicrobial resistance. Whole-genome sequencing techniques have been used to track bacterial transmission from person to person and to better define and determine transmission linkages in outbreaks of resistant HAIs. [496] [497] [498] As these technologies continue to advance, rapid, point-of-care diagnostic tests could be used to quickly identify resistant infections and target treatment. Efforts to prevent and control infectious diseases by altering the host-microbe interaction as opposed to targeting microbes FIGURE parasitic disease drug service regarding requests from physicians in New York and California for pentamidine isethionate for treatment of Pneumocystis carinii (now Pneumocystis jirovecii) pneumonia in patients with no known cause of immunodeficiency hinted at the first U.S. cases of AIDS. 510 Recognition of unusually severe respiratory disease by observant clinicians signaled hantavirus pulmonary syndrome in 1993, and suspicion of anthrax by alert clinical and laboratory staff in Florida in 2001 suggested a possible bioterrorist event. A February 10, 2003, email posted by a physician on ProMED, an informal online infectious disease reporting program of the International Society for Infectious Diseases, is widely regarded as the first notification of the global outbreak of SARS. 511 Such observations have not been limited to the medical and scientific community, however. In the mid-1970s, two Connecticut mothers questioning what was believed to be an unusually large number of juvenile rheumatoid arthritis cases in their community led researchers to the discovery of Lyme disease and a concerned American Legion official provided the first indication to health authorities of the emergence of legionnaires' disease. 512 Although today's globalized world has created a perfect environment for rapid emergence and spread of infectious diseases, it has also brought significant scientific, technologic, and communication advances for their control. Next-generation sequencing technologies and expanded bioinformatics capacities are revolutionizing the field of microbiology, reducing the amount of time needed for pathogen detection and analysis and generating data for a more detailed understanding of infectious agents. 503, 513 These tools offer new opportunities to improve public health efforts to detect and control outbreaks, determine antimicrobial susceptibility, and develop and target vaccines. 313, 314, 497, 498 Scientists are also gaining new understanding of the role of the microbiome and microbial sensors in infectious diseases, 514, 515 offering new insight into pathogen-host complexities and disease treatment and prevention. Continued advances in electronic communications are facilitating earlier recognition of emerging problems and rapid exchange of information. In particular, early warning systems such as ProMED-mail, the Public Health Agency of Canada's Global Public Health Intelligence Network (GPHIN), and HealthMap, collect, categorize, and display outbreak and disease information from a variety of formal and informal sources-enhancing disease surveillance and tracking capabilities. [516] [517] [518] Expansions in Internet access and use and far-reaching social media networks have also increased the exchange of health information and broadened public health partnerships to include nontraditional partners such as law enforcement, the media, and members of the public at local, national, and global levels.","Here, we demonstrate that unbiased mNGS is a feasible and efficient means to detect pathogens in the midst of an outbreak of a novel virus. The index Cambodian SARS-CoV-2 sequence data uploaded to public databases represent one of the only sequences generated in a low-to-middle income country close to the outbreak epicenter. Despite major advances in mNGS technologies and significant decreases in costs associated with sequencing, preparation of sequencing libraries and sufficient bioinformatics capabilities for timely analysis still present a challenge in the developing world. 8 As cases of COVID-19 spread globally in mid-January, initial real-time PCR protocols took into account that the genetic diversity of SARS-CoV-2 in humans and animals was not completely known. Non-specificity combined with variable sample quality can hinder ability to detect the pathogen in question. In this scenario, our metagenomics approach demonstrated a first-pass high coverage of the N gene, closely aligning to diagnostic protocols utilizing both the E and N author/funder. All rights reserved. No reuse allowed without permission.","We are in a transition phase of clinical microbiology, with genomic sequencing being increasingly used for diagnosis and epidemiology. There is a pressing need for tools enabling automated detection of clusters and for follow-up of epidemiological trends of clones, virulence or resistance markers. Ren\u00e9 Hendriksen (DTU, Copenhagen, Denmark) presented a talk envisioning global surveillance in the genomic era. Bioinformatics web-based tools were developed at Copenhagen's Center for Genomic Epidemiology (www. genomicepidemiology.org) to allow global epidemiology based on whole-genome sequencing. Among these tools, \"ResFinder\" is intended to allow the detection of acquired antimicrobial resistance genes, whereas the \"Pathogen-Finder\" and \"VirulenceFinder\" tools can be used to detect bacterial pathogenicity-associated genes. Automated solutions for extracting typing data, plasmid sequences, prediction of bacterial species using a k-mer approach or 16S ribosomal RNA sequences and SNP-based phylogenies were also presented. It was concluded that whole-genome sequencing combined with automated bioinformatics tools has the potential to replace many steps in conventional clinical bacteriology analyses.","Among the viral groups there exists much variation in replication strategy, thus there is potential for differences in the predominant strategy for producing genetic variability leading to 7 . CC-BY-ND 4.0 International license author/funder. It is made available under a The copyright holder for this preprint (which was not peer-reviewed) is the . https://doi.org/10.1101/2020.03.10.986109 doi: bioRxiv preprint zoonotic transmission events [15] . Though mutation rate is 100 fold lower per nucleotide for dsDNA viruses than for RNA viruses, their much larger genomes enable a comparable likelihood of accumulating at least one mutation during replication. A negative correlation between mutation rate and genome size has been noted for a small set of viruses for which mutation rate has been directly measured [14] . Much of the correlation for RNA viruses results from measurement of bacteriophage mutation rate, and this interesting relationship should be further explored. Coronaviruses have evolved 3 \u2212 \u2192 5 proofreading capabilities to accommodate mutagenesis associated with a large genome [24] . As recombination is generally associated with lower reduction in fitness compared to mutation, recombination may be the preferred mechanism to generate genetic variability for the viruses with the largest genomes. dsDNA viruses are known to have high per nucleotide rates of non-homologous recombination. Because of the large genome sizes of dsDNA viruses, the likelihood of at least one recombination event occurring during replication of the genome is high [15, 27] . The many coded genes of ds-DNA viruses allow more easily incorporation of recombinant DNA without deleterious effects and may reduce host specificity through phenotypic plasticity [15, 17] . In contrast, Papillomaviruses are dsDNA viruses that are characteristically species-specific [28] . These viruses have small genomes with little plasticity to accommodate recombinant DNA or expand host range. dsRNA viruses are highly segmented, and the number of segments is highly correlated with the size of the genome (R 2 = 0.84, Fig. S2) , which allows for a high degree of recombination via reassortment of segments during a coinfection [15] . Gene expression is regulated by translation for ssRNA(+) viruses. This group includes Coronaviruses and is known for high levels of homologous recombination because of frequent template switching during discontinuous transcription. Having the largest RNA viral genomes, Coronaviruses can produce over 25% recombinant progeny with their high rates of recombination [11, 15] . ssRNA(-) viruses are not known to recombine with high frequency, so their genetic variation relies on high mutation 8 . CC-BY-ND 4.0 International license author/funder. It is made available under a The copyright holder for this preprint (which was not peer-reviewed) is the . https://doi.org/10.1101/2020.03.10.986109 doi: bioRxiv preprint rates alone [29] . ssRNA-RT viruses are known for maintaining high mutation rates characteristic of RNA viruses as well as high recombination rates for a subset of these viruses, including HIV. Pseudodiploidy in retroviruses can promote recombination due to the physical proximity of RNA strands in the virion as well as partial masking of deleterious mutations [30] . Large genomes are more likely to contain intergenic, non-coding regions that assist in gene regulation but also serve as hotspots for recombination and mutation due to weaker selection to conserve these regions [17] . Viral zoonoses can emerge as a result of large genetic changes associated with recombination or significant mutation-both more likely to occur in large genomes. Phenotypic plasticity is hypothesized to be positively related to genome size and may facilitate cross-species transmission, though this notion requires future exploration. The relationship between likelihood of zoonotic transmission and genome size has implications for predicting future zoonoses and for understanding general principles about viral life history. Host specificity may be considered a function of genome reduction. Guiding principles like the relationship between genome size and host range should be explored to broaden scientific understanding of the viral world and its role in ecosystems. Predicting future epidemics requires knowledge of potential zoonotic viruses. Far less than 1% of viruses have been discovered, and the potential host interactions of each viral species are even less well-characterized, making zoonotic risks difficult to estimate [31] . These data demonstrate that viral genome size is an effective indicator of zoonotic risk across viral genome subtypes. In an age of high-throughput genomics and rapid discovery of new viruses, characterization of recombination and mutation rates is far slower and more laborintensive than genome sequencing, making genome size an appealing first step in zoonotic risk evaluation. Analysis of viral genome size and structure has the potential to provide deep insights in predicting future pandemics.","The initial event in the life cycle of a virus is its contact with the target cell. This encounter is mediated by interactions between components on the viral surface (capsid proteins or viral membrane anchored proteins) and several elements on the cellular membrane that can get attached to the virus. Most viruses establish contacts with cellular receptors which could be sialic acid bound molecules or proteins on the cell surface. Frequently, the first virus\u2013cell contact is mediated by adhesion receptors that allow reversible attachment of the virus to particular cells or organs, mainly by means of electrostatic interactions. This initial interaction could lack specificity and is employed to enhance the encounter between virus and entry receptors, which irreversibly bind the virus, allowing them access to the intracellular milieu either by direct endocytosis/pinocytosis or by inducing fusion or penetration of the viral genome into the cell. ","The copyright holder for this preprint (which was not peer-reviewed) is the . https://doi.org/10.1101/2020.03.02.968818 doi: bioRxiv preprint genes for primary detection. This detection directly increased on-the-ground certainty in diagnostic sensitivity and specificity of available assays. 7 mNGS in field settings proved critical in development of countermeasures for the 2014-2016 Ebola virus epidemic in West Africa, just as ongoing sequencing of COVID-19 cases from lesser developed areas of Southeast Asia will contribute to overall understanding of pathogen transmission, origin, and evolution. 9,10 While COVID-19 cases are not limited to one remote region, as is often the case with viral hemorrhagic fever outbreaks, sequenced samples from all countries will be important for global disease containment. Instead of being limited to consensus Sanger sequencing, which may not detect high risk quasispecies, metagenomics is a powerful approach to detect variants of both novel and known species as epidemics evolve. 11 The clear limitation in an mNGS approach is that low viral titers or high levels of host material demand greater read depth than may be available on instruments such as the iSeq100. To overcome this barrier, our follow-up steps included a target enrichment of SARS-CoV-2 while keeping the comprehensiveness of our mNGS pipeline intact. 6 For an emerging threat, this strategy offers the flexibility to successfully recover the pathogen genome in question for subsequent phylogenetic analyses without compromising discovery. However, the other key factor in mNGS success is the accessibility to open-access, cloud-based metagenomics bioinformatics pipelines, such as IDseq which automates the process of separating host sequence characterizing the remaining non-host sequences. 12 As recent as the Ebola epidemic, bioinformatics analyses were still mainly completed in the Global North. 10 This newly available combination -more rugged, deployable sequencers plus user-friendly, globally accessible bioinformatics -represents an opportunity for responders in limited-resource settings; however, further proof-of-principle during outbreaks remains necessary.","Case definition within an outbreak usually involves a combination of clinical and laboratory criteria; for instance, a complex of symptoms and an associated organism. This definition can then be used for active case finding to identify additional patients in the cluster. During the German STEC outbreak, rapid genome sequencing together with crowd-sourced bioinformatics analyses led to the development of a set of diagnostic reagents that could then be used in defining cases within the outbreak [3]. Similarly, during new outbreaks of viral infection, genome-scale sequencing can act as a precursor to the development of simpler specific tests that can be used in case definition [31,32].","Overall, the different methods developed over the last decade to sequence genomes from uncultivated viruses are mostly complementary and can be individually tailored for specific applications. Virus discovery can be achieved through bulk metagenomes or viromes, while viral ecology studies will tend to rely more on viromes as a reflection of virus activity and transport, and metagenomics used as a diagnostic tool in the clinic would be the most likely to use sequence capture. Nevertheless, all these complementary approaches will be needed for achieving a comprehensive picture of viral diversity. specific assays for each pathogen. Metagenomics instead offers the possibility to detect known and novel viruses without prior knowledge from a single analysis, and is thus well suited and already applied to study emerging and/or rare viruses, as well as cases which remain negative using the available diagnostic tests (see below). A number of challenges remain however for viral metagenomics to become a standard clinical procedure. First, given the current cost associated with sample processing and sequencing, metagenomes are still more expensive and slower than ELISA's or qPCR assays. Second, there is no generally validated bioinformatics pipelines than can perform a rapid, sensitive, and specific analysis of the obtained data on a bench top computer. Finally, physicians will have to be trained and guided to deal with the obtained breadth of data. Specifically, it is becoming clear that each individual is chronically infected with a dozen or more eukaryotic viruses (many of which have not been associated with any disease, e.g., anelloviruses), and that many known viral pathogens can also cause asymptomatic infections. Therefore, a physician might get a list of viruses (and other potentially pathogenic or unknown organisms), and it will be a challenge to identify the actual cause of a particular disease. Nevertheless, the price of sample preparation and high throughput sequencing has declined enormously in the last decade including with the development of smaller and faster machines, while automatic virome Fig. 1 Overview of the viral metagenomics workflow. The overall process used to generate and analyze viral metagenomes can be divided into four major steps: (i) Collection of environmental and/or clinical sample, (ii) sample preparation, (iii) library preparation, and (iv) sequence analysis. The sample preparation step can target either the cellular fraction (left) or the viral fraction (right) in which case viral particles are often further concentrated and purified to remove free nucleic acids. *Targeted sequence capture can be applied to the extracted DNA/RNA to enrich for a specific virus. **While whole genome amplification was initially used routinely for viral metagenomes, it has now been supplanted by methods enabling the preparation of more quantitative libraries from low input (B 1 ng), hence whole genome amplification is now primarily used in single-cell or single-virus-particle experiments. ***The genome assembly can be bypassed if using long-read sequencing technologies, although these long-read datasets require a more careful error correction. ****Genome binning, i.e., the identification of multiple contigs assembled from a metagenome and corresponding to the same genome, is typically only used for large genomes (e.g., 4500 kb), and individual contigs are directly analyzed instead for most viruses.","The advances of sequencing technology, nanotechnology, and bioinformatics have driven molecular tests including assays for emerging pathogens to be more comprehensive and precise. For example, the availability of various sequence databases permits quick identification of sequence identity and variations. For example, the HIV database http://www.hiv.lanl.gov/ contains data on HIV genetic sequences and drug resistance associated mutations. It is valuable for HIV epidemiological studies, research, development, and clinical validation studies of HIV clinical assays [38, 71] . It is well known that there are significant variations of clinical phenotypes in the presence of emerging infections ranging from asymptomatic carrier to lethal infection. Recently, assays to examine multiple pathogen panels have been developed [97\u00c0102], which should increase the diagnostic yield for many pathogens. A critical need for emerging pathogen analysis is quicker, easier, cost-effect assays that can be used in a point-of-care setting. New assays that are performed on platforms with a small footprint and detect pathogens quickly (in minutes instead of hours or days) have entered clinical use. For example, the FilmArray (BioFire Diagnostics, Inc.) and Simplexa (Focus Diagnostics, Inc.) molecular assays can generate results in approximately 60 min. The user-friendly Alere i (Alere Inc.) and Cobas Liat (Roche Molecular Systems) platforms are compact and portable, generate rapid molecular results in 15\u00c020 min, can use electricity or rechargeable battery, and therefore are completely mobile and suited for point-of-care testing. It is obvious that the current rapid development of new technologies will further enhance the utility of molecular diagnostics in various emerging infectious diseases.","(3) The combined use of multiple early warning technologies: Integrated prediction primarily refers to applying two or more prediction models, combining qualitative and quantitative predictions, and using spatial-temporal technologies for early warning. (4) Developing and utilizing new surveillance resources for early warning: It is important to explore new and available surveillance data for early warning, such as data from hospital emergency departments, OTC, biological media, animal epidemics, environmental meteorology, and to identify better ways to collect information and promote data use through multifactor analyses. (5) Early warning based on lab surveillance: The popularity and refinement of modern laboratory networks with molecular biology and bioinformatics will greatly enhance early identification of the intrinsic relations (e.g., gene fingerprints) among seemingly unrelated sporadic cases for early warning. It will substantially enhance the early identification of the epidemiological significance of pathogens, and provide strong evidence for the early warning of infectious disease outbreaks. (6) Evaluating early warning methods: Evaluating early warning methodology is also very important in current early warning research. This primarily refers to studying evaluation standards for early warning methods, systems, and test data. Sensitivity and specificity are two important indicators of evaluation. Other common indicators include the positive predictive value, receiver operating characteristic curve, activity monitoring operating characteristic, and the probability for successful detection.","Although RNA interference is an extremely powerful tool, this new technology has several limits. RNA viruses show a high degree of sequence diversity between different genotypes; and there is rapid evolution of quasi-species, by mutation and recombination, and the generation of reassortant genomes in the case of segmented genomes. These are major problems for the development of siRNA-based gene therapies [33] . In addition, large sections of highly structured, untranslated regions of viral RNA genomes are resistant to PTGS [33] . A few siRNAs even enhance expression instead of having a silencing effect. Others have an off-target effect and therefore silence the wrong genes [6, 63, 64] . SiRNAs may also cross-react with targets of limited similarity [63] . Escape mutants have emerged after treatment of cells with a single virus-specific siRNA, in particular when cells are infected at high MOI [15, 65, 66] . The design of multiple siRNAs targeting different regions of the viral genome, in particular wellconserved regions, reduces the probability of generating escape mutants. In addition, some studies have shown that transfection with siRNAs results in IFN-mediated activation of the Jak-Stat pathway and general upregulation of IFNstimulated genes [67, 68] . The effect was mediated by dsRNAdependent protein kinase, PKR, and depended on the concentration of siRNAs [68] . Bridge et al. [67] also noted that some shRNAs produced by a lentiviral vector induced an IFN response, whereas homologous chemically synthesized siR-NAs did not. However, some synthetic siRNAs do induce an IFN response [68] . SiRNAs caused Toll-like receptor 3 to induce type 1 IFN and sequence-independent mRNA degradation in the course of acute HSV-1 infection of keratinocytes [69] . It is therefore recommended to use the lowest effective dose of siRNAs or siRNA-encoding vectors [67] [68] [69] .","In the last talk of the session, \"Rapid taxonomic assignment from high throughput sequencing reads\", Anthony Underwood (Bioinformatics Unit, PHE, Colindale, UK) described the bioinformatics component of the PHE project to replace existing phenotypic or molecular typing tests by HTS. Salmonella, S. pneumoniae and S. aureus were chosen in a pilot study to extract MLST, predict serotype or determine virulence gene profile. One identified limitation was the lack of accuracy in species identification and the issue of mixed cultures. Two identification methods, ribosomal MLST [23] and k-mer quantification in the sequencing reads, were described and compared. Although both methods were able to predict species correctly, the k-mer approach was faster.","Whole-genome sequencing was done using Oxford Nanopore MinION device (Oxford Nanopore Technologies, Oxford, UK) supplemented by Sanger sequencing. RNA was extracted from host cell-depleted nasopharyngeal and sputum samples using a QIAamp Viral RNA Mini Kit, as described previously. [10] [11] [12] Whole-genome amplification of the coronavirus was done using a sequence-independent single-primer amplification ap proach, as described previously. 13 Bioinformatics analyses were done using an inhouse pipeline. Details on the library preparation and bioinformatics analysis are described in the appendix (pp 1-2). The con sensus sequence of HKU-SZ-002a (accession number MN938384) and HKU-SZ-005b (accession number MN975262) have been deposited into GenBank. Raw reads, after excluding human reads, have been deposited into BioProject (accession number PRJNA601630).","Microarrays occupy a middle ground with respect to cost, processing time, sensitivity, specificity and ability to detect novel organisms. The high-density arrays available at present are able to test for the presence of thousands of different organisms simultaneously, at a cost less than US$ 100 per sample. Arrays can be designed with a combination of high-specificity probes and probes designed against conserved regions, so that they can be used in both detection and discovery modes. While most array designs select probes from fully sequenced genomes in GenBank and other databases, cross-hybridization between probes and similar but non-identical sequences allows detection of novel species, provided that they are closely related to those that were used for probe design. A limitation of microarrays is that, except for so-called universal arrays, probe designs must be updated periodically to include the ever-increasing number of microbial genome sequences being added to GenBank. Nevertheless, for many applications, microarrays offer an ideal balance of capabilities for broad-spectrum microbial surveillance.","In this work, we developed a Prophylactic Antiviral CRISPR in huMAN cells (PAC-MAN) strategy as a form of genetic intervention to target SARS-CoV-2, IAV, and potentially all sequenced coronaviruses. We created a bioinformatic pipeline to define conserved regions across the SARS-CoV-2 genomes and target these regions using CRISPR-Cas13d for viral genome degradation and viral gene inhibition. At the time of paper submission, there is no widely available laboratory strains of SARS-CoV-2. Therefore, we tested our approach using synthesized fragments of SARS-CoV-2, as well as with live H1N1 IAV. We designed and screened a panel of crRNA pools targeting conserved viral regions and defined the most effective crRNAs. We demonstrated the ability of our approach to cleave SARS-CoV-2 fragments and to reduce the replication of IAV in human lung epithelial cells. Our bioinformatics author/funder. All rights reserved. No reuse allowed without permission.","Since the completion of major whole genome sequencing efforts, the scientific community has been faced with the challenge of identifying and characterizing the expressed gene products of given organisms [1]. The post-genomics era gave birth to the field of proteomics that aimed to systematically chart the biochemical properties and functions of all expressed proteins [2]. With a global view in mind, we now strive to integrate complex \u201comics\u201d-data from all molecular ranks. The scope of proteomics is not limited to identifying protein\u2013protein interactions, but also includes identification of protein posttranslational modifications (PTMs) and of interactions with DNA and RNA sequences, lipids and glycans. Weaving these layers together will allow us to construct the carefully tuned network that exists within live cells. Improvements in high throughput proteomic technologies coupled with advances in genomics and bioinformatics have laid a framework to enable this level of research.","Nanopore sequencing data were analysed using a modified Artic Network nCoV-2019 novel coronavirus bioinformatics protocol (Supplementary Appendix). 13 Illumina sequencing reads were mapped with reference to the respective draft genome of each sample constructed from the Nanopore data. Variants were called using freebayes (v1.0.0) with haploid decoding and minimum base quality set at Q30. Consensus genomes were constructed by GATK (v4.1.4.1) based on the VCF file 14 . SPAdes genome assembler (v3.14.0) and minimap2 (v2.17) were used to combine Nanopore and Illumina sequencing All rights reserved. No reuse allowed without permission. author/funder, who has granted medRxiv a license to display the preprint in perpetuity.","We are currently faced with a potential global epidemic of a new coronavirus that has infected thousands of people in China and is spreading rapidly around the world. In the end of January 2020, the WHO has declared it a global emergency (WHO, 2020) . The novel coronavirus (SARS-CoV-2), first isolated in Wuhan China, has already caused more infections than the previous severe acute respiratory syndrome (SARS) outbreak of 2002 and 2003. The virus is a SARS related coronavirus (SARSr-CoV), and it is genetically associated with SARSr-CoV strains that infect bats in China . It causes severe respiratory illness, which the WHO recently named COVID-19 disease. It has high fatality rate , can be transmitted from person to person, has infected over 70,000 individuals and has spread to over 30 countries in less than two months (WHO, 2020) . This coronavirus outbreak has been unprecedented; so too is the way that the scientific community has responded to it. They have openly and rapidly shared genomic and clinical data as never seen before allowing research results to be released almost instantaneously. This has helped the understanding of the transmission dynamics, the development of rapid diagnostic and has informed public health response. Here, we present a new contribution that can speed up this communal effort. The Genome Detective Coronavirus Typing Tool is a free-of-charge web-based bioinformatics pipeline that can accurately and quickly identify, assemble and classify coronaviruses genomes. The tool also identifies changes at nucleotides, coding regions and proteins using a novel dynamic aligner to allow tracking new viral mutations (Figure 1 ). Figure 1 : Genome Detective Coronavirus Typing Tool assembles genomes from next generation sequencing (NGS) in FASTAQ format or assembled genomes in FASTA format. A user can submit up to 1Gb of NGS data or 2,000 assembled genomic sequences. For each assembled genomic sequence, the tool identifies the virus species, constructs a phylogenetic tree and identifies phylogenetic clusters, which includes the novel coronavirus identified in Wuhan China in 2019 (SARS-CoV-2). The tool identifies changes at nucleotides, coding regions and proteins using a novel dynamic aligner and display all of the mutations in detailed tables and reports. . To this reference dataset, we added 47 whole genomes of the current Coronavirus 2019 (SARS-CoV-2) outbreak that originated in Wuhan, China, in December 2019. The SARS-CoV-2 sequences were downloaded from the GISAID database (https://www.gisaid.org) together with annotation of its original location, collection date and originating and submitting laboratory. The SARS-CoV-2 data generators are properly acknowledged in the acknowledgements section of this paper and detailed information is provided in Supplementary Table 1. The 431 reference WGS were aligned with MUSCLE (Edgar 2004) . The alignment was manually edited until a codon alignment was attained in all coding sequences (CDS). A Maximum likelihood phylogenetic tree, 1000 bootstrap replicates was constructed in PhyML (Guidon &amp; Gascuel 2003; Lemoine et al., 2018 ) and a Bayesian tree using MrBayes (Ronquist &amp; Huelsenbeck 2003) were constructed. The trees were visualized in Figtree (Rambaut 2018). We selected 25 reference sequences that represent the diversity of each well-defined phylogenetic cluster (with bootstrap support of 100% and posterior probability of 1). We identified five well supported phylogenetic clusters with more than two sequences of SARSr-CoV and used them to set up our automated phylogenetic classification tool. Cluster 1 included SARS strains from the 2002 and 2003 Asian outbreaks. In our tool, we named this cluster SARS-CoV Outbreak 2000s but may rename it as SARS-A if a new proposed naming system for SARSr-Cov is adopted in the near future (Rambaut 2020). Cluster 2 (provisionally named as SARS related CoV) includes 7 sequences from bats which did not cause large human outbreaks. Cluster 3 (named as Bat SARS-CoV HKU3) includes three WGS sampled from Rhinolophus sinicus (i.e. Chinese rufous horseshoe bats). Cluster 4 (Bat SARS-CoV ZXC21/ZC45) includes two SARSr-CoV sampled from Rhinolophus sinicus bats in Zhoushan, China. Cluster 5 (virus named SARS-CoV-2 by the ICTV committee and disease named COVID-19 by the WHO) includes three public sequences from the outbreak. We identified this cluster with many sequences from GISAID but kept only three ones as these were the first GenBank sequences. The first whole genome of SARS-CoV-2 was kindly shared by Prof. Yong-Zhen Zhang and colleagues in the virological.org website. Detailed information about the phylogenetic reference datasets are available in Supplementary Table 2. The phylogenetic reference dataset was used to create an automated Coronavirus Typing Tool using the Genome Detective framework (Vilsker et al., 2019 , Fonseca et al., 2019 . To determine the accuracy of this tool, each of the 431 test WGS was considered for evaluation (i.e. 384 reference sequences from VIPR and 47 public SARS-CoV-2 sequences). The sensitivity, specificity and accuracy of our method was calculated for both species assignment and phylogenetic clustering of Classifying query sequences in an automated fashion involves two steps. The first step enables virus species assignments and the second, which is restricted to SARSr-CoV, includes phylogenetic analysis. The first classification analysis subjects a query sequence to BLAST and AGA analysi. AGA is a novel alignment method for nucleic acid sequences against annotated genomes from NCBI RefSeq Virus Database. AGA (Deforche 2017 ) expands the optimal alignment algorithms of Smith-Waterman (Smith &amp; Waterman 1981) and Gotoh (Gotoh 1982 ) based on an induction state with additional parameters. The result is a more accurate aligner, as it takes into account both nucleotide and protein scores and identifies all of the polymorphisms at nucleotide and amino acid levels. In the second step, a query sequence is aligned against the phylogenetic reference dataset using -add alignment option in the MAFFT software (Katoh &amp; Standley 2013) . In addition, a Neighbor Joining phylogenetic tree is constructed using the HKY distance metric with gamma among-site rate variation with 1,000 bootstrap replicates using PAUP* (Swofford) . The query sequence is assigned to a particular phylogenetic cluster if it clusters monophyletically with that clade or a subset of it with bootstrap support &gt;70%. If the bootstrap support is &lt;70%, the genotype is reported to be unassigned.","is the (which was not peer-reviewed) The copyright holder for this preprint . https://doi.org /10.1101 /10. /2020 In conclusion, colorimetric LAMP provides a simple, rapid method for SARS-CoV-2 RNA detection. Not only purified RNA can be used as the sample input, but also direct tissue or cell lysate may be used without an RNA purification step. This combination of a quick sample preparation method with an easy detection process may allow the development of portable, field detection in addition to a rapid screening for point-of-need testing applications. This virus represents an emerging significant public health concern and expanding the scope of diagnostic utility to applications outside of traditional laboratories will enable greater prevention and surveillance approaches. The efforts made here will serve as a model for inevitable future outbreaks where the use of next generation portable diagnostics will dramatically expand the reach of our testing capabilities for better healthcare outcomes.","Action is thus required at a company and industry level, with governments and tourists also having a vital contribution to make. The WTO has stressed the significance of health as an aspect of tourist safety and proposes that member states pursue the following program to enhance their capabilities in dealing with diffi culties (WTO, 1991) :","In recent years, the development of vaccine design has been revolutionized by the reverse 90 vaccinology (RV), which aims to first identify promising vaccine candidate through 91 bioinformatics analysis of the pathogen genome. RV has been successfully applied to vaccine 92 discovery for pathogens such as Group B meningococcus and led to the license Bexsero 93 vaccine 12 . Among current RV prediction tools 13,14 , Vaxign is the first web-based RV program 15 94 and has been used to successfully predict vaccine candidates against different bacterial and viral 95 pathogens [16] [17] [18] . Recently we have also developed a machine learning approach called Vaxign-ML 96 to enhance prediction accuracy 19 . 97","Regardless which emerging pathogen we discuss -Avian Influenza H5N1, H7N9, MERS CoV, swine flu or other viruses, it is important to recognize that preparedness efforts as a response to a potential pandemic caused by swine or avian influenza, SARS or MERS, can also enhance awareness, promote vaccine use, advances in diagnostic and treatment capabilities toward other significant infectious disease worldwide.","In situations where it is not essential to actually obtain an isolate of the virus, IBV may be detected by immunochemical assays (immunofl uorescence or immunoperoxidase using an anti-IBV serum) and increasingly by molecular methods. Th ese involve the reverse-transcription polymerase chain reaction (RT-PCR) together with restriction enzyme fragment length polymorphism (RFLP) analysis, DNA probes or nucleic acid sequencing. Th ey are increasingly used for the detection of IBVs generally, as well as for the diff erentiation of specifi c variants. Th ese techniques have also facilitated molecular epidemiological studies to monitor the distribution of specifi c variants. While direct sequencing of the genome is the most accurate method for identifying new variants, RFLP, which identifi es sites unique to a particular serotype, is acceptable and has provided a rapid method for variant identifi cation. Th e RNA required for these techniques can be extracted directly from either swab material or appropriate tissues of infected birds, or from allantoic fl uid or TOC supernatant of samples from which virus isolation is being attempted. Since viral nucleic acid is what is being detected, it is not necessary for live virus to be present and RT-PCR probably detects the presence of IBV for longer than is possible by virus isolation methods. While, for many purposes, the lack of a live virus is not a problem, for epidemiological studies and for further research, the availability of an isolate could be necessary.","Justin Lee (Colorado State University, Fort Collins, CO, USA) described new advances in next-generation nucleic acid sequencing through the introduction of MinION; a platform especially useful for assembling RNA virus genomes in remote research stations. The MinION by Oxford Nanopore Technologies Inc. is the first portable next-generation sequencing (NGS) instrument and is capable of producing long-reads of single molecules from DNA and RNA libraries. The NGS Facility in the Department of Microbiology, Immunology and Pathology at Colorado State University has a MinION instrument and has recently performed pilot testing of this instrument to better understand its capabilities and limitations. They obtained sequence data from bacterial and eukaryotic genomic DNA on the MinION and on an Illumina MiSeq. The MinION runs produced reads ranging from less than 1 kb to over 500 kb in length with an error rate of approximately 25%. Output from a single run was sufficient to assemble a complete draft 6 Mbp bacterial genome sequence in one contig, with adequate depth to minimize the effect of the high error rate. A hybrid assembly incorporating long and short reads produced a near perfect assembly of the bacterial genome. Output from a single MinION run produced only ~0.1\u00d7 coverage over a bat genome, and thus was not sufficient to assemble the genome. They conclude that the MinION is a valuable tool for small genome sequencing and for producing long reads to span areas of low complexity that don\u2019t assemble well with short-read data alone. The low upfront cost and portability of the instrument also make it a possibly useful tool for field and surveillance applications.","The raw fastQ files for each virus sample were filtered using previously described criteria, 18 then subjected to de novo assembly with the CLCBio software version 11.0.1. Mapped assemblies were also done using the batderived SARS-like coronavirus isolate bat-SL-CoVZC45 (accession number MG772933.1) as a reference. Variant calling, genome alignments, and sequence illustrations were generated with CLCBio software, and the assembled genome sequences were confirmed by Sanger sequencing. Rapid amplification of cDNA ends (RACE) was done to obtain the sequences of the 5\u02b9 and 3\u02b9 termini, using the Invitrogen 5\u02b9 RACE System and 3\u02b9 RACE System (Invitrogen, Carlsbad, CA, USA), according to the manufacturer's instructions. Gene-specific primers (appendix p 1) for 5\u02b9 and 3\u02b9 RACE PCR amplification were designed to obtain a fragment of approximately 400-500 bp for the two regions. Purified PCR products were cloned into the pMD18-T Simple Vector (TaKaRa, Takara Biotechnology, Dalian, China) and chemically com petent Escherichia coli (DH5\u03b1 cells; TaKaRa), according to the manufacturer's instructions. PCR products were sequenced with use of M13 forward and reverse primers.","Bioinformatics and functional analysis have previously identified the minimum size packaging signal (approximately 60 to 95 nt) for various coronaviruses. However, the packaging signal of lineage C betacoronavirus MERS-CoV has not been determined. Based on the observations that a stable stem\u2013loop structure is often a prerequisite for an RNA fragment to be a packaging signal and that unique sequences may determine the packaging specificity of different coronavirus lineages, we proposed the analysis of the genome sequence and the structure of different lineage betacoronaviruses for the prediction of the potential packaging signal of MERS-CoV. The whole genome sequence of MERS-CoV was first compared with lineage B betacoronavirus SARS-CoV by alignment with NCBI blastn. This revealed a highly variable region from nt 19,757 to 20,434 of the MERS-CoV genome located at the 3\u2032 end of ORF 1b and between two regions that have the highest conservation (the alignment score was higher than 200; Fig. 1a, top). Interestingly, this variable region overlaps the PS580 RNA fragment spanning nt 19,715 to 20,294 of the SARS-CoV genome, which has been identified as a functional packaging signal [19]. In addition, when the MERS-CoV RNA sequence was compared with that of the lineage A betacoronavirus MHV, the divergent sequence was localized to a similar but shorter fragment spanning nt 19,756 to 20,182 of the MERS-CoV genome (Fig. 1a, bottom). These results located a highly variable region of the genome sequences among the different lineage betacoronaviruses. Secondary structures of the MERS-CoV RNA(19757\u201320434) and RNA(19756\u201320182) were then analyzed (Fig. 1b). Two RNA subdomains of 94-nt (nt 19,801 to 19,894) and 152-nt (nt 20,022 to 20,173) that form stable structures in both RNA fragments were identified as potential packaging signals of MERS-CoV.","To enrich adequate viral content for whole-genome sequencing in a convenient manner, we 117 pursued two other methods: multiplex PCR amplification (amplicon) and hybrid capture 118 (capture) (Fig. 1) . We designed a systematic study to comprehensively validate the bias, 119 sensitivity, inter-individual (variant) and intra-individual (iSNV) accuracy of multiple ap-120 proaches by sequencing serial dilutions of a cultured isolate (unpublished), as well as the 121 eight clinical samples (Fig. 2) . We performed qRT-PCR of 10-fold serial dilutions (D1-D7) of 122 the cultured isolate, and the Ct was 17.3, 20.8, 24.5 for, 28.7, 31.8, 35, and 39.9, respec-123 tively, indicating the undiluted RNA (D0) of the cultured isolate contained ~1E+08 genome 124 copies per mL. For amplicon sequencing, we utilized two kits comprising of two set of pri-125 mers generating PCR products of 300-400 bp and 100-200 bp, respectively. The ~400 bp 126 amplicon-based sequencing was implemented in all samples and analyzed throughout the 127 study, while the ~200 bp amplicon-based sequencing was only applied in the cultured isolate 128 for coverage analysis. 129 to inadequate depth in amplicon sequencing (Fig. 3a) . Another pitfall is that amplification 142 across the genome can hardly be unbiased, causing difficulties in complete genome assem-143 bly. Indeed, amplicon sequencing exhibited a higher level of bias compared with meta-144 transcriptomic sequencing, in terms of coverage across the viral genomes from the cultural 145","Background: Insertion sequence ISAba1 belonging to the IS4 family has been identified repeatedly in Acinetobacter baumannii, being often associated with antibiotic resistance genes. In particular, it has been shown to provide promoter sequences enhancing expression of the natural cephalosporinase AmpC and oxacillinase OXA-51 of A. baumannii. In addition, two copies of ISAba1 bracketing blaOXA-23 may form composite transposon Tn2006 at the origin of acquisition of this carbapenem-hydrolysing oxacillinase gene. The transposase of ISAba1 is formed by two open reading frames Orf1 and Orf2 likely giving rise to a functional transposase when a frameshift occurs. Our study was aimed to evaluate the transposition ability of ISAba1. Methods: We evaluated experimentally the ability of ISAba1 to transpose in Escherichia coli. In a first step, the b-lactamase gene blaTEM-1 was inserted into ISAba1 to provide an ampicillin resistance marker allowing its tracing. This ISAba1::blaTEM-1 structure was cloned into pTOPO vector and then transformed into recA(\u2212) E. coli RZ201 harboring conjugative plasmid pOX38-Gen used as a target for transposition events. Transposition events onto pOX38-Gen were selected by conjugating this plasmid into azide-resistant E. coli J53. The same technique was used to evaluate the transposition ability of Tn2006. Also the role of the frameshift in the transposase expression was evaluated by eliminating the frameshift in the transposase encoding gene by site-directed mutagenesis. Results: Transposition of ISAba1 was obtained at a frequency of 1\u00d710 5 (\u00b10.7\u00d710 5 ) per E. coli donor. Sequencing of the target sites of transposition revealed i) a systematic 9-bp duplication upon transposition, and an adenine-rich hotspot of target transposition. Transposition frequency of the mutated ISAba1 with a transposase gene made of a unique frame was 2\u00d710 3 (\u00b10.2\u00d710 3 ). The mobility of Tn2006 was demonstrated and its frequency of transposition was estimated to be 1.6\u00d710 8 (\u00b12.5\u00d710 2 ). Conclusion: This study iss the very first demonstration of the functionality of the widespread ISAba1 as a mobile element. The frameshift-mediated down-regulation of transposition has been assessed, as well as the functionality of the ISAba1-made composite transposon. This in-vivo model will allow further experiments able to evaluate the possible selective factors such as antibiotics for ISAba1-transposition enhance","Eventually, a fully developed virus metagenomics toolkit will enable the accurate identification of viruses in natural, clinical, and biotechnological samples for monitoring and diagnostics purposes. Moreover, as bioinformatics analysis tools advance, the reconstruction of full viral genome sequences will allow predictions to be made for the most important viruses in different environments, leading to the reconstruction of environmental virus-host networks and, when combined with other 'omics' approaches, the comprehensive evaluation of viral activity across an entire ecosystem. Collectively, these studies should lead to a","Overall, agnostic or unbiased metagenomic sequencing capabilities in-country provide the ability to detect and respond to a variety of pathogens, even those that are unanticipated or unknown. Bridging of existing local and global resources for sequencing and analysis allows for better realtime surveillance locally, while also enabling better health pursuits overall, not just during outbreaks. The example described here serves as a call for continued training and infrastructure to support mNGS capacity in developing countries as bioinformatic tools proliferate and the cost of sequencing decreases.","Target PCR is an effective and simple alternative to metagenomics, working well for both individual genes and small viral genomes [127]. PCR-based amplification is often used for whole viral genome sequencing of samples with low viral load [54], e.g., during the investigation of the measles outbreak during the Olympics in 2010 [128] and epidemics of Ebola [56] and Zika diseases [129]. Sequencing of long (2.5\u20133.0 kb) amplified fragments has clarified the variability of a Norovirus genome and its spreading among patients in a few hospitals in Vietnam [130,131]. A similar approach has been utilized to measure the specificity and sensitivity of the Illumina platform for detection of minor polymorphisms in mixed HIV populations [132]. Deep sequencing of PCR-amplified viral genomes yielded complete genome sequences for the influenza virus [133], Dengue virus [134] and HCV [135].","However, although these methods are highly specific for detecting the gene of interest, even if transferred to indigenous species, they do not distinguish between living and dead cells and they require enrichment to develop sufficient DNA for hybridization (Colwell et ai, 1988) . If the DNA sequences in GMOs are to be detected by DNA probes, a high concentration of these sequences must be present in the sample, otherwise the target sequence concentration will be below the detection threshold of conventional probes. Somerville et al. (1988) developed a method for enhancing sensitivity, based on chain elongation. In this method, specificity is determined by probe hybridization, while sensitivity is enhanced by polymerization into adjacent genes. Steffan and Atlas (1988) also used target DNA amplification to enhance the detection of genetically engineered bacteria. They used the polymerase chain reaction (PCR) technique of Saiki et al. (1988) which involves melting the DNA, annealing short oligomer primers to regions flanking a target sequence and using taq polymerase to extend the DNA from the primers across the target region. The new duplexes are melted by heating and the process is repeated. The result is an exponential increase of the target sequence, such that the it can be amplified by a factor of more than 10 7 within a few hours. Applying PCR to bacterial DNA isolated from sediment samples permitted the detection of as few as 100 cells of Pseudomonas cepacia per 100 g of sediment against a background of 10 11 diverse non-target organisms. Bej et al. (1990) have used a similar technique to detect E. coli and other coliform bacteria in water samples.","The SELDI ProteinChip is unique in that it allows for differential separation of complex protein mixtures based on chemical characteristics such as hydrophobicity or charge, resulting in a decrease in the complexity of the sample analyzed. However, SELDI is considered a soft-ionization method and the results obtained are patterns of protein peaks and not the identification of peptide masses. To ensure the specificity of peak profile for a particular disease state, a robust algorithm is needed. Lastly, protein microarrays have been developed to determine the immune response to a viral infection. The method requires the expression and printing of all ORFs of a pathogen and cross-linking them to a solid support. Protein microarrays would allow for the rapid diagnosis of a particular viral infection. However, expression of a complete proteome is a challenging task. As the proteins are expressed in bacteria, potentially important PTM are lost and proteins may not be properly folded.","With the advance in molecular biology technology, nucleic acid detection methods have developed rapidly and become a revolutionary technology for virus detection. Especially, the method based on polymerase chain reaction (PCR) is characterized by rapid detection, high sensitivity and specificity, which has been regarded as the \"gold standard\" for virus detection. Novel PCR-based methods also have been reported to enhance both the specificity and sensitivity of the assay. Besides, several molecular tests which employ non-PCR-based methods, such as isothermal nucleic acid amplification (loop mediated isothermal amplification (LAMP) and nucleic acid sequence-based amplification), were developed for the detection of coronavirus RNA. In this work, we reviewed various approaches currently available for detection of coronavirus. It is anticipated that this will assist researchers to develop rapid and accurate detection techniques.","My background comprises microbes, immunology, genomics and bioinformatics. I took part in the work of SARS genome sequencing in the Beijing Genomics Institute et the Chinese Academy of Sciences in 2003 [9] . Now I am majoring in the study on the PAMP and TLRs. Given this, I am in general interested in the topic.","By mapping output reads on the SARS-CoV-2 genome, all reads with high identity were calculated 109 for each plasmid concentration. For 10 min and 1 h sequencing data, reads mapped to SARS-CoV-2 110 significantly differed from those of negative controls in all replicates at concentrations ranging from 111 3000 to 500 (Fig. 2a) , and 3000 and 10 ( Fig. 2b ) copies/mL, respectively. This result confirmed that 112 high-copy samples could rapidly yield sufficient valid sequencing data for diagnosis, and by 113 extending the sequencing time, valid sequencing data could also be obtained from low-copy 114 samples. Notably, as more sequencing data could be achieved with additional sequencing time 115 Evaluation of the target distribution of these valid data revealed that in higher copies samples 119 (1000 and 3000 copies/mL), all targeted regions could be detected (Fig. 2c, d) . However, in lower 120 viral concentration samples (from 10 to 500 copies/mL), some targeted regions were lost (i.e., no 121 reads mapped; Fig. 2c , d), indicating that for low-quality or low-abundance samples, comprehensive 122 fragment amplification is difficult. Therefore, for accurate results, NTS cannot label a sample as 123 positive for infection by monitoring only one or two sites, as is customary for qPCR; rather, the 124 results from all target regions should be considered. 125","We have evaluated a rapid approach for characterizing SARS-CoV genotypes. The assay is simple, easy to perform and reproducible. It can therefore be used as an efficient means to screen for virus genotypes and track the transmission of a particular viral strain in times of epidemics. Incidentally, we identified a previously reported deletion variant of the SARS-CoV in a new cohort of patients and traced the emergence of this variant to an earlier date than previously reported.","Sequencing of transformants from subtractive cDNA was performed on ABI3730 sequencer (Applied Biosystem). Sequences were jointly analyzed for homology with BLASTN program at NCBI (http://www.ncbi.nlm.nih.gov/BLAST) and BLAT program at UCSC Genome Bioinformatics (http://www.ucsc.edu). Genes were classified by gene annotation given by Gene Ontology (GO) Consortium (http://www.geneontology.org/).","itself, but we provide this model in the meanwhile to allow others to work in parallel and 370 improve our real-time decision-support capabilities. 371 372 . CC-BY-NC-ND 4.0 International license It is made available under a author/funder, who has granted medRxiv a license to display the preprint in perpetuity.","Samples from patient BALF or from the supernatant of virus cultures were used for RNA extraction and next-generation sequencing (NGS) using BGI MGISEQ2000 and Illumina MiSeq 3000 sequencers. Metagenomic analysis was carried out mainly based on the bioinformatics platform MGmapper (PE_2.24 and SE_2.24). The raw NGS reads were first processed by Cutadapt (v.1.18) with minimum read length of 30 base pairs. BWA (v.0.7.12-r1039) was used to align reads to a local database with a filter hits parameter of 0.8 FMM ((match + mismatch)/read length \u2265 fraction] value and minimum alignment score of 30. Parameters for post-processing of assigned reads were set to a minimum size normalized abundance of 0.01, minimum read count of 20 and were otherwise set to default parameters. A local nucleic acid database for human and mammals was used to filter reads of host genomes before mapping reads to the virus database. The results of the metagenomic analysis were displayed as pie charts using Microsoft Office 2010. NGS reads were assembled into genomes using Geneious (v.11.0.3) and MEGAHIT (v.1.2.9). PCR and Sanger sequencing was performed to fill gaps in the genome. 5\u2032-rapid amplification of cDNA ends (RACE) was performed to determine the 5\u2032-end of the genomes using a SMARTer RACE 5\u2032/3\u2032 kit (Takara). Genomes were annotated using the Clone Manager Professional Suite 8 (Sci-Ed Software).","This coronavirus outbreak has been unprecedented; so too is the way that the scientific community has responded to it. They have openly and rapidly shared genomic and clinical data as never seen before allowing research results to be released almost instantaneously. This has helped the understanding of the transmission dynamics, the development of rapid diagnostic and has informed public health response. Here, we present a new contribution that can speed up this communal effort. The Genome Detective Coronavirus Typing Tool is a free-of-charge web-based bioinformatics pipeline that can accurately and quickly identify, assemble and classify coronaviruses genomes. The tool also identifies changes at nucleotides, coding regions and proteins using a novel dynamic aligner to allow tracking new viral mutations (Figure 1 ). To this reference dataset, we added 47 whole genomes of the current Coronavirus 2019 (2019-nCoV) outbreak that originated in Wuhan, China, in December 2019. The 2019-nCoV sequences were downloaded from the GISAID database (https://www.gisaid.org) together with annotation of its original location, collection date and originating and submitting laboratory. The 2019-nCoV data generators are properly acknowledged in the acknowledgements section of this paper and detailed information is provided in Supplementary Table 1. The 431 reference WGS were aligned with MUSCLE (Edgar 2004) . The alignment was manually edited until a codon alignment was attained in all coding sequences (CDS). A Maximum likelihood phylogenetic tree, 1000 bootstrap replicates was constructed in PhyML (Guidon &amp; Gascuel 2003; Lemoine et al., 2018) and a Bayesian tree using MrBayes (Ronquist &amp; Huelsenbeck 2003) were constructed. The trees were visualized in Figtree (Rambaut 2018). We selected 25 reference sequences that represent the diversity of each well-defined phylogenetic cluster (with bootstrap support of 100% and posterior probability of 1). We identified five well supported phylogenetic clusters with more than two sequences of SARSr-CoV and used them to set up our automated phylogenetic classification tool. Cluster 1 included SARS strains from the 2002 and 2003 Asian outbreaks. In our tool, we named this cluster SARS-CoV Outbreak 2000s but may rename it as SARS-A if a new proposed naming system for SARSr-Cov is adopted in the near future (Rambaut 2020). Cluster 2 (provisionally named as SARS related CoV) includes 7 sequences from bats which did not cause large human outbreaks. Cluster 3 (named as Bat SARS-CoV HKU3) includes three WGS sampled from Rhinolophus sinicus (i.e. Chinese rufous horseshoe bats). Cluster 4 (Bat SARS-CoV ZXC21/ZC45) includes two SARSr-CoV sampled from Rhinolophus sinicus bats in Zhoushan, China. Cluster 5 (provisionally named as Wuhan 2019-nCoV, which may be renamed as SARS-B) includes one public sequence from the outbreak in Wuhan, China. We identified this cluster with many sequences from GISAID but kept only this one as this is the only GenBank sequence, accession number MN908947, which was kindly shared by Prof. Yong-Zhen Zhang and colleagues in the virological.org website. Detailed information about the phylogenetic reference datasets are available in Supplementary Table 2. The phylogenetic reference dataset was used to create an automated Coronavirus Typing Tool using the Genome Detective framework (Vilsker et al., 2019 . To determine the accuracy of this tool, each of the 431 test WGS was considered for evaluation (i.e. 384 reference sequences from VIPR and 47 public 2019-nCoV sequences). The sensitivity, specificity and accuracy of our method was calculated for both species assignment and phylogenetic clustering of SARSr-CoV. Sensitivity was computed by the formula Classifying query sequences in an automated fashion involves two steps. The first step enables virus species assignments and the second, which is restricted to SARSr-CoV, includes phylogenetic analysis. The first classification analysis subjects a query sequence to BLAST and AGA analysi. AGA is a novel alignment method for nucleic acid sequences against annotated genomes from NCBI RefSeq Virus Database. AGA (Deforche 2017) expands the optimal alignment algorithms of Smith-Waterman (Smith &amp; Waterman 1981) and Gotoh (Gotoh 1982) based on an induction state with additional parameters. The result is a more accurate aligner, as it takes into account both nucleotide and protein scores and identifies all of the polymorphisms at nucleotide and amino acid levels. In the second step, a query sequence is aligned against the phylogenetic reference dataset using -add alignment option in the MAFFT software (Katoh &amp; Standley 2013) . In addition, a Neighbor Joining phylogenetic tree is constructed using the HKY distance metric with gamma among-site rate variation with 1,000 bootstrap replicates using PAUP* (Swofford) . The query sequence is assigned to a particular phylogenetic cluster if it clusters monophyletically with that clade or a subset of it with bootstrap support &gt;70%. If the bootstrap support is &lt;70%, the genotype is reported to be unassigned.","By February 1, the SARS-CoV-2 genome was correctly identified in the index COVID-19 patient sample using mNGS and globally accessible bioinformatics. An mNGS library was prepared, multiplexed, and sequenced. Notably, the sample generated 4,773,430 reads with 1,903,800 of those being non-host reads. The resulting non-host fastq files were processed using the IDseq platform which utilizes open-source components. When processed on IDseq pipeline version 3.18 with a February 2020 NCBI index, which included 54 SARS-CoV-2 sequences deposited by the international research community, sequences from the index patient aligned to the SARS-CoV-2 reference genome ( Figure 1A , 582 reads, at 100% identity to the reference sequence; NCBI accession MN985325.1). Further analysis resulted in 33.2% coverage of the SARS-CoV-2 reference at an average depth of 2.2x ( Figure 1B ). Assembly and alignment on Geneious resulted in 50.5% coverage at 2.6x of the SARS-CoV-2 genome (NCBI accession MN908947.1). Re-sequencing the index case to a total sequencing depth of 8,786,916 single reads resulted in a greater number of SARS-CoV-2 alignments (819 reads) but had no impact on overall coverage -32.3% coverage breadth at an average depth of 2.7x.","In this study we analysed NGS data derived from clinical specimens (oral swabs) from three Chinese patients infected by SARS-CoV-2 (SRA projects PRJNA601736 and PRJNA603194). We aligned the raw read data on reference strain MN975262.1 using bowtie2 [18] , after quality check with FastQC (www.bioinformatics.bbsrc.ac.uk/projects/fastqc). The resulting alignments were visualized with the Integrated Genomics Viewer (IGV) [19] . After removing PCR duplicates, SNVs were called with a Bonferroni-corrected P-value threshold of 0.05 using samtools [20] and LoFreq [21] . Variants supported by absolute read concordance (&gt;98%) were filtered-out from intra-host variant frequency calculations. We annotated the variations to the reference strain using snpEff [22] , SNVs effects were further filtered with snpSift [23] and we estimated the average mutation rate per gene across the viral genome using R scripts. We compared the localization of the intra-host SNVs with all available SNVs observed at population level up to February 18 th 2020 (retrieved from www.GISAID.org). We also compared all intra-host and population level SNPs with all primers and probes coordinates to investigate for potential interferences with currently available molecular diagnostic assays [24] (www.who.int/docs/default-source/coronaviruse/peiris-protocol-16-1-20.pdf).","Based on bioinformatics analyses, it has been shown that 2019-nCoV genome is 88% identical with two bat-derived severe acute respiratory syndrome (SARS)-like coronaviruses (bat-SL-CoVZC45 and bat-SL-CoVZXC21), 79% with SARS-CoV, and 50% with MERS- Single-cell RNA sequencing (scRNA-Seq) has been extensively applied in 2019-nCoV research, due to its capability to profile gene expressions for all cell types in multiple tissues unbiasedly at a high resolution. In addition to well-known alveolar type 2 cells in lung, expression of ACE2 has also been investigated in liver cholangiocytes, esophagus epithelial cells and absorptive enterocytes of ileum and colon by scRNA-Seq [11] [12] [13] . These results demonstrated the potential utility of scRNA-Seq to unmask the potential target cell types of 2019-nCoV. Since the urinary system infection and its potential aftermath could be essential to the patient care during and after the infection, here we used two scRNA-Seq transcriptome data in healthy kidneys and one dataset in heathy bladders to investigate the expression patterns of cell types in the urinary system.","Designing of a novel vaccine is very crucial to defending the rapid endless of global burden of disease [56] [57] [58] [59] . In the last few decades, biotechnology has advanced rapidly; alongside with the understanding of immunology which assisted the rise of new approaches towards rational vaccines design [60] . Peptide-based vaccines are designed to elicit immunity particular pathogens by selectively stimulating antigen specific for B and T cells [61] .Applying the advanced bioinformatics tools and databases, various peptide-based vaccines could be designed where the peptides act as ligands [62] [63] [64] . This approach has been used frequently in Saint Louis encephalitis virus [65] , dengue virus [66] , chikungunya virus [67] proposing promising peptides for designing vaccines.","We sequenced SARS-CoV-2 genomes from nine of the first COVID-19 cases reported in Connecticut (CT), with sample collection dating from March 6-14, 2020 ( Data S1 ). These individuals are residents of eight different cities in Connecticut. According to the Connecticut State Department of Public Health, none of the cases were associated with international travel. Using our amplicon sequencing approach, 'PrimalSeq' (Grubaugh et al., 2019b; Quick et al., 2017) , with the portable Oxford Nanopore Technologies (ONT) MinION platform, we generated the first SARS-CoV-2 genome approximately 14 hours after receiving the sample (CT-Yale-006), demonstrating our ability to perform near real-time clinical sequencing and bioinformatics. Our complete workflow included RNA extraction, PCR testing, validation of PCR results, library preparation, sequencing, and live base calling and read mapping. We shared the genomes of these viruses publicly as we generated them (GISAID EPI_ISL_416416-416424). We combined our genomes with other publicly available sequences for a final dataset of 168 SARS-CoV-2 genomes ( Figure 1, Data S2 ). The dataset can be visualized on our 'community' Nextstrain page ( https://nextstrain.org/community/grubaughlab/CT-SARS-CoV-2 ).","The ongoing outbreak of a novel coronavirus in China is now evident in 37 other countries as others employ vigilant screening procedures. 1 As of 24 Feb 2020, 119 SARS-CoV-2 genome sequences from 15 countries are publicly available from over 79,331 confirmed cases. 1,2 These viral genomes differ by only 0 to 5 mutations, likely representing a single introduction in the human population from an unknown reservoir or intermediate host, potentially bats. 3 Subsequently, the virus rapidly spread since December 2019. Currently, incubation period and transmissibility are questions of global concern, but difficult to answer given nonspecific and mild symptoms that could represent a variety of respiratory viruses. 4, 5 Genetic characterization of the virus from geographically diverse patient samples is key to infer the rate of spread. Additionally, rapid and complete results from cases worldwide are important for sequence-dependent countermeasures and accurate diagnostics. However, as the virus reaches resource-limited settings in proximity to the outbreak's epicenter, such as Cambodia, Laos, and Myanmar, there are often logistical challenges in surveillance, contact tracing, sample collection, and biospecimen transport. Information delay is also compounded by a dearth of sequencing and/or bioinformatics expertise in-country, further postponing analysis and dissemination of pathogen genomic information.","We designed a series of peptide binders that showed enhanced binding affinity to the SARS-CoV-2 RBD in computational experiments. Structural bioinformatics and sequence logo analyses indicated that the good binders to a high extent recapitulated the critical residues that contributed significantly to binding. Detailed inspection confirmed this point and also revealed that some extra favorable interactions were introduced to enhance binding in the top designed binders. Moreover, the designs were predicted to maintain the secondary structure of the motif, which was important for facilitating the protein-peptide binding interaction. Although our experimental results are not available yet, we would like to share these useful data to the community so that people can work together to test them, which should help to develop new antiviral drugs to combat COVID-19.","In early December 2019, several cases of pneumonia with unknown causes were reported in Wuhan, Hubei, China (1) . Later, a novel coronavirus, 2019-nCoV was identified based on sequencing of the patient respiratory tract samples. As of February 1, 2020, more than 10,000 confirmed patients and more than 15,000 suspected cases were reported in China, with additional patients being identified in a rapidly growing number internationally. Analyzing of the genome of 2019-nCoV showed that this new virus shared around 80~90% sequence identity to SARS-CoV (2) . Both bioinformatics modeling and in vitro experiments indicated that 2019-nCoV was more likely to use the angiotensin-converting enzyme 2 (ACE2) as the entry receptor (3, 4) . ACE2 is previously known as the receptor for SARS-CoV and HCoV-NL63 (5, 6) . Considering the sequence of 2019-nCoV is similar to the SRAS-CoV and they are reported to share the same molecular as the entry point, analyses of ACE2 expression and distribution in lung and related biological processes may help us understand the pathogenesis and design therapeutic strategies for 2019-nCoV. Recent advances in bioinformatics enable researchers to reveal the underlying mechanisms of various diseases. In this study, we used bioinformatics approaches to identify the ACE2 expression features in lung and mine associated regulating networks. These results may help us understand the pathogenesis and design therapeutic strategies for 2019-nCoV."],"title":["Challenges of infectious diseases in the USA","14 Emerging and Reemerging Infectious Disease Threats","Rapid metagenomic characterization of a case of imported COVID-19 in Cambodia","Microbial molecular markers and epidemiological surveillance in the era of high throughput sequencing: an update from the IMMEM-10 conference","Larger viral genome size facilitates emergence of zoonotic diseases","Viruses and viral proteins","Rapid metagenomic characterization of a case of imported COVID-19 in Cambodia","Genomics and outbreak investigation: from sequence to consequence","Metagenomics in Virology","Chapter 15 Molecular Testing in Emerging Infectious Diseases","Chapter 1 Introduction","Silencing viruses by RNA interference","Microbial molecular markers and epidemiological surveillance in the era of high throughput sequencing: an update from the IMMEM-10 conference","A familial cluster of pneumonia associated with the 2019 novel coronavirus indicating person-to-person transmission: a study of a family cluster","Microarrays for Pathogen Detection and Analysis","Development of CRISPR as a prophylactic strategy to combat novel coronavirus and influenza","Interactome Mapping: Using Protein Microarray Technology to Reconstruct Diverse Protein Networks","A Territory-wide study of COVID-19 cases and clusters with unknown source in Hong Kong community: A clinical, epidemiological and phylogenomic investigation","Genome Detective Coronavirus Typing Tool for rapid identification and characterization of novel coronavirus genomes","Rapid Molecular Detection of SARS-CoV-2 (COVID-19) Virus RNA Using Colorimetric LAMP","Chapter 7 Tourism and Health Crises","COVID-19 coronavirus vaccine design using reverse vaccinology and machine learning","SELECTED EPIDEMICS &amp; EMERGING PATHOGENS \u2013 RESPIRATORY ILLNESSES \u2013 AN OVERVIEW","Chapter 28 Coronaviridae","The 17th Rocky Mountain Virology Association Meeting","Genomic characterisation and epidemiology of 2019 novel coronavirus: implications for virus origins and receptor binding","Nucleocapsid protein-dependent assembly of the RNA packaging signal of Middle East respiratory syndrome coronavirus","Multiple approaches for massively parallel sequencing of HCoV-19 (SARS-CoV-2) genomes directly from clinical samples","Oral presentations","Metagenomics in Virology","Rapid metagenomic characterization of a case of imported COVID-19 in Cambodia","Current Trends in Diagnostics of Viral Infections of Unknown Etiology","6 Recombinant plasmids","Chapter 15 Proteomics of viruses","Recent advances and perspectives of nucleic acid detection for coronavirus","SARS attacks!","Nanopore target sequencing for accurate and comprehensive detection of SARS-CoV-2 and other respiratory viruses","A simple and rapid approach for screening of SARS-coronavirus genotypes: an evaluation study","Upregulation of Mitochondrial Gene Expression in PBMC from Convalescent SARS Patients","Fear, Access, and the Real-Time Estimation of Etiological Parameters for Outbreaks of Novel Pathogens","A pneumonia outbreak associated with a new coronavirus of probable bat origin","Genome Detective Coronavirus Typing Tool for rapid identification and characterization of novel coronavirus genomes","Rapid metagenomic characterization of a case of imported COVID-19 in Cambodia","SARS-CoV-2 exhibits intra-host genomic plasticity and low-frequency polymorphic quasispecies","Single-cell Analysis of ACE2 Expression in Human Kidneys and Bladders Reveals a Potential Route of 2019-nCoV Infection","Design of multi epitope-based peptide vaccine against E protein of human COVID-19: An immunoinformatics approach","Coast-to-coast spread of SARS-CoV-2 in the United States revealed by genomic epidemiology","Rapid metagenomic characterization of a case of imported COVID-19 in Cambodia","Computational Design of Peptides to Block Binding of the SARS-CoV-2 Spike Protein to Human ACE2","Integrative Bioinformatics Analysis Provides Insight into the Molecular Mechanisms of 2019-nCoV"],"url":["https://doi.org/10.1016/s0140-6736(14)60890-4","https://doi.org/10.1016/b978-1-4557-4801-3.00014-x","https://doi.org/10.1101/2020.03.02.968818","https://doi.org/10.1016/j.resmic.2014.01.003","https://doi.org/10.1101/2020.03.10.986109","https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4224467/","https://doi.org/10.1101/2020.03.02.968818","https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3706975/","https://doi.org/10.1016/b978-0-12-809633-8.20957-6","https://doi.org/10.1016/b978-0-12-800886-7.00015-7","https://doi.org/10.1016/b978-0-12-812343-0.00001-1","https://doi.org/10.1016/j.micinf.2005.02.003","https://doi.org/10.1016/j.resmic.2014.01.003","https://doi.org/10.1016/s0140-6736(20)30154-9","https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3245552/","https://doi.org/10.1101/2020.03.13.991307","https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3968920/","https://doi.org/10.1101/2020.03.30.20045740","https://doi.org/10.1093/bioinformatics/btaa145","https://doi.org/10.1101/2020.02.26.20028373","https://doi.org/10.1016/b978-0-7506-7834-6.50008-9","https://doi.org/10.1101/2020.03.20.000141","https://doi.org/10.1016/j.disamonth.2017.03.016","https://doi.org/10.1016/b978-0-7020-2862-5.50033-7","https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5707540/","https://doi.org/10.1016/s0140-6736(20)30251-8","https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5966903/","https://doi.org/10.1101/2020.03.16.993584","https://doi.org/10.1111/j.1469-0691.2008.02006.x","https://doi.org/10.1016/b978-0-12-809633-8.20957-6","https://doi.org/10.1101/2020.03.02.968818","https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7077230/","https://doi.org/10.1016/b978-0-7506-1105-3.50010-1","https://doi.org/10.1016/b978-044451980-1.50017-3","https://doi.org/10.1016/j.jpha.2020.02.010","https://doi.org/10.1016/j.micinf.2012.11.011","https://doi.org/10.1101/2020.03.04.20029538","https://www.ncbi.nlm.nih.gov/pmc/articles/PMC1276795/","https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7086694/","https://doi.org/10.1101/2020.03.19.20038729","https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7095418/","https://doi.org/10.1101/2020.01.31.928796","https://doi.org/10.1101/2020.03.02.968818","https://doi.org/10.1101/2020.03.27.009480","https://doi.org/10.1101/2020.02.08.939892","https://doi.org/10.1101/2020.02.04.934232","https://doi.org/10.1101/2020.03.25.20043828","https://doi.org/10.1101/2020.03.02.968818","https://doi.org/10.1101/2020.03.28.013607","https://doi.org/10.1101/2020.02.03.20020206"],"x":[0.693166069306275,0.6736912731763915,0.6283647379366628,0.625335061409079,0.5355860148377537,0.30122758219398144,0.6279506935159028,0.6229314145194241,0.6172429798003286,0.619456831533147,0.6959961521925269,0.416125036483945,0.5856364100801368,0.5751071297429022,0.6137474076525782,0.4002360818372826,0.6318068647714769,0.5862259408946914,0.5831365116783057,0.6150696929431273,0.7721410466549844,0.3915486838446074,0.49982219789061805,0.5743040652955703,0.5909721960669521,0.5666944187662275,0.4995482452295834,0.5775537719999358,0.480098925159388,0.6275313778930717,0.667752286320134,0.5295358321863509,0.5710169118016518,0.6239078303111634,0.6029838011284283,0.588182550527235,0.5855530974561094,0.5430460304320098,0.604592195789789,0.8038060395732946,0.5922392506374116,0.5867771534709633,0.58566598938805,0.5910249590992979,0.26417617530589765,0.38382582018416395,0.5789197115581223,0.6090017210474763,0.6441497882634646,0.5204043472156108],"y":[0.6961544848576958,0.6846830284656804,0.37166974380409473,0.3567086705473729,0.3715657752218061,0.36510765748851753,0.371780020463632,0.37338869485434434,0.35137802413380903,0.3762730574416611,0.705137920890541,0.31968507417513203,0.32363323090965657,0.23767807429678617,0.3447516871841467,0.3846923334530837,0.3565463350466874,0.24864053767022595,0.28848783586511456,0.3727816169691683,0.777056401355934,0.4395382356704772,0.5758208979052493,0.3472785538299371,0.3140349064450784,0.2349077732153558,0.2880281376029291,0.3350564935914925,0.30491779652071926,0.3584517299219436,0.6980055499839942,0.37958437363562747,0.3388408582329486,0.3458793076038791,0.3612515126625877,0.31406780020062214,0.35017368602550203,0.4128753912398539,0.16928424919790794,0.22758042537566983,0.26562271887740635,0.2848975535567515,0.2622727857814471,0.2529572581545404,0.43278142818562004,0.43274498814373946,0.254288812977105,0.6029720379795339,0.3295888586785512,0.42392336690847593]},"selected":{"id":"27479"},"selection_policy":{"id":"27509"}},"id":"27406","type":"ColumnDataSource"},{"attributes":{"source":{"id":"27406"}},"id":"27470","type":"CDSView"},{"attributes":{},"id":"27460","type":"ResetTool"},{"attributes":{},"id":"27445","type":"Range1d"},{"attributes":{"formatter":{"id":"27500"},"ticker":{"id":"27417"}},"id":"27416","type":"LinearAxis"},{"attributes":{},"id":"27452","type":"BasicTicker"},{"attributes":{"callback":null},"id":"27459","type":"TapTool"},{"attributes":{"js_property_callbacks":{"change:indices":[{"id":"27480"}]}},"id":"27479","type":"Selection"},{"attributes":{},"id":"27510","type":"BasicTickFormatter"}],"root_ids":["27489"]},"title":"Bokeh Application","version":"2.0.1"}}
        </script>
        <script type="text/javascript">
          (function() {
            var fn = function() {
              Bokeh.safely(function() {
                (function(root) {
                  function embed_document(root) {
                    
                  var docs_json = document.getElementById('27726').textContent;
                  var render_items = [{"docid":"9451a660-16c8-4c1a-b08d-331d5094d081","root_ids":["27489"],"roots":{"27489":"8db81f8e-2eae-4194-b0a2-815d55c2b746"}}];
                  root.Bokeh.embed.embed_items(docs_json, render_items);
                
                  }
                  if (root.Bokeh !== undefined) {
                    embed_document(root);
                  } else {
                    var attempts = 0;
                    var timer = setInterval(function(root) {
                      if (root.Bokeh !== undefined) {
                        clearInterval(timer);
                        embed_document(root);
                      } else {
                        attempts++;
                        if (attempts > 100) {
                          clearInterval(timer);
                          console.log("Bokeh: ERROR: Unable to run BokehJS code because BokehJS library is missing");
                        }
                      }
                    }, 10, root)
                  }
                })(window);
              });
            };
            if (document.readyState != "loading") fn();
            else document.addEventListener("DOMContentLoaded", fn);
          })();
        </script>
    
  </body>
  
</html>