



<!DOCTYPE html>
<html lang="en">
  
  <head>
    
      <meta charset="utf-8">
      <title>Bokeh Plot</title>
      
      
        
          
        
        
          
        <script type="text/javascript" src="https://cdn.bokeh.org/bokeh/release/bokeh-2.0.1.min.js" integrity="sha384-JpP8FXbgAZLkfur7LiK3j9AGBhHNIvF742meBJrjO2ShJDhCG2I1uVvW+0DUtrmc" crossorigin="anonymous"></script>
        <script type="text/javascript" src="https://cdn.bokeh.org/bokeh/release/bokeh-widgets-2.0.1.min.js" integrity="sha384-xZlADit0Q04ISQEdKg2k3L4W9AwQBAuDs9nJL9fM/WwzL1tEU9VPNezOFX0nLEAz" crossorigin="anonymous"></script>
        <script type="text/javascript" src="https://cdn.bokeh.org/bokeh/release/bokeh-tables-2.0.1.min.js" integrity="sha384-4BuPRZkdMKSnj3zoxiNrQ86XgNw0rYmBOxe7nshquXwwcauupgBF2DHLVG1WuZlV" crossorigin="anonymous"></script>
        <script type="text/javascript">
            Bokeh.set_log_level("info");
        </script>
        
      
      
    
  </head>
  
  
  <body>
    
      
        
          
          
            
              <div class="bk-root" id="a1ec7714-fb14-4a34-bb19-24d9fcdf1986" data-root-id="9457"></div>
            
          
        
      
      
        <script type="application/json" id="9694">
          {"980948ec-e35e-411c-9dc9-8bd4d847085d":{"roots":{"references":[{"attributes":{"active_drag":"auto","active_inspect":"auto","active_multi":null,"active_scroll":"auto","active_tap":"auto","tools":[{"id":"9392"},{"id":"9393"},{"id":"9394"}]},"id":"9395","type":"Toolbar"},{"attributes":{},"id":"9463","type":"StringEditor"},{"attributes":{"columns":[{"id":"9406"},{"id":"9407"}],"height":340,"source":{"id":"9373"},"view":{"id":"9409"},"width":900},"id":"9408","type":"DataTable"},{"attributes":{"data":{"authors":["Fung, Sin-Yee; Yuen, Kit-San; Ye, Zi-Wei; Chan, Chi-Ping; Jin, Dong-Yan","Zhao, Shan; Li, Wentao; Schuurman, Nancy; van Kuppeveld, Frank; Bosch, Berend-Jan; Egberink, Herman","Mishra, Nischay; Fagbo, Shamsudeen F.; Alagaili, Abdulaziz N.; Nitido, Adam; Williams, Simon H.; Ng, James; Lee, Bohyun; Durosinlorun, Abdulkareem; Garcia, Joel A.; Jain, Komal; Kapoor, Vishal; Epstein, Jonathan H.; Briese, Thomas; Memish, Ziad A.; Olival, Kevin J.; Lipkin, W. Ian","Mackenzie, John S; Jeggo, Martyn","Boncristiani, H.F.; Criado, M.F.; Arruda, E.","","Kiselev, Daniel; Matsvay, Alina; Abramov, Ivan; Dedkov, Vladimir; Shipulin, German; Khafizov, Kamil","Al-Ahmadi, Khalid; Alahmadi, Sabah; Al-Zahrani, Ali","Turner, Anthony J.","Hu, Ben; Ge, Xingyi; Wang, Lin-Fa; Shi, Zhengli","Fung, Sin-Yee; Yuen, Kit-San; Ye, Zi-Wei; Chan, Chi-Ping; Jin, Dong-Yan","Li, Hong-Ying; Zhu, Guang-Jian; Zhang, Yun-Zhi; Zhang, Li-Biao; Hagan, Emily A; Martinez, Stephanie; Chmura, Aleksei A; Francisco, Leilani; Tai, Hina; Miller, Maureen; Daszak, Peter","Kiselev, Daniel; Matsvay, Alina; Abramov, Ivan; Dedkov, Vladimir; Shipulin, German; Khafizov, Kamil","Weber, David J.; Rutala, William A.; Fischer, William A.; Kanamori, Hajime; Sickbert-Bennett, Emily E.","Desforges, Marc; Le Coupanec, Alain; Stodola, Jenny K.; Meessen-Pinard, Mathieu; Talbot, Pierre J.","Martin, R.R.; Tzanetakis, I.E.","DeZure, Adam; Graham, Barney S.","Falsey, Ann Regina","Dawson, Patrick; Malik, Mamunur Rahman; Parvez, Faruque; Morse, Stephen S.","","Biscayart, Cristian; Angeleri, Patricia; Lloveras, Susana; Chaves, T\u00e2nia do Socorro Souza; Schlagenhauf, Patricia; Rodr\u00edguez-Morales, Alfonso J.","Bosch, Berend Jan; Raj, V Stalin; Haagmans, Bart L","Peiris, J.S.M.; Poon, L.L.M.","Jiang, Shibo; Du, Lanying; Shi, Zhengli","Kiselev, Daniel; Matsvay, Alina; Abramov, Ivan; Dedkov, Vladimir; Shipulin, German; Khafizov, Kamil","Kun Sun; Liuqi Gu; Li Ma; Yunfeng Duan","Shi, Z.; Wang, L.-F.","Shipley, Rebecca; Wright, Edward; Selden, David; Wu, Guanghui; Aegerter, James; Fooks, Anthony R; Banyard, Ashley C","Meyer, Benjamin; Drosten, Christian; M\u00fcller, Marcel A.","Brisse, Sylvain; Brehony, Carina; Concei\u00e7\u00e3o, Teresa; Cubero, Meritxell; Glasner, Corinna; Le Gouil, Meriadeg; Renvois\u00e9, Aur\u00e9lie; Sheppard, Sam; Weinert, Lucy A.","Meyer, R.F.; Morse, S.A.","Reed, Kurt D.","Jessica E. Manning; Jennifer A. Bohl; Sreyngim Lay; Sophana Chea; Ly Sovann; Yi Sengdoeurn; Seng Heng; Chan Vuthy; Katrina Kalantar; Vida Ahyong; Michelle Tan; Jonathan Sheu; Cristina M. Tato; Joseph L. DeRisi; Laurence Baril; Veasna Duong; Philippe Dussart; Erik A. Karlsson","Brisse, Sylvain; Brehony, Carina; Concei\u00e7\u00e3o, Teresa; Cubero, Meritxell; Glasner, Corinna; Le Gouil, Meriadeg; Renvois\u00e9, Aur\u00e9lie; Sheppard, Sam; Weinert, Lucy A.","Sun, Jiumeng; He, Wan-Ting; Wang, Lifang; Lai, Alexander; Ji, Xiang; Zhai, Xiaofeng; Li, Gairu; Suchard, Marc A.; Tian, Jin; Zhou, Jiyong; Veit, Michael; Su, Shuo","Lu, Dan; Liu, Kefang; Zhang, Di; Yue, Can; Lu, Qiong; Cheng, Hao; Wang, Liang; Chai, Yan; Qi, Jianxun; Wang, Lin-Fa; Gao, George F.; Liu, William J.","Yang, Weizhong; Lan, Yajia; Sun, Qiao; Wang, Jinfeng; Li, Zhongjie","Zhao, Shan; Li, Wentao; Schuurman, Nancy; van Kuppeveld, Frank; Bosch, Berend-Jan; Egberink, Herman","Sun, Lei; Zhang, Gui-hong; Jiang, Jing-wei; Fu, Jia-dong; Ren, Tao; Cao, Wei-sheng; Xin, Chao-an; Liao, Ming; Liu, Wen-jun","Gurjit S. Randhawa; Maximillian P.M. Soltysiak; Hadi El Roz; Camila P.E. de Souza; Kathleen A. Hill; Lila Kari","Mi\u0142ek, Justyna; Blicharz-Doma\u0144ska, Katarzyna","Niccolo Alfano; Anisha Dayaram; Jan Axtner; Kyriakos Tsangaras; Marie-Louise Kampmann; Azlan Mohamed; Seth Timothy Wong; M. Thomas P. Gilbert; Andreas Wilting; Alex Daivd Greenwood","Tomar, Shailly; Mahajan, Supreeti; Kumar, Ravi","Earp, Laurie J; Hernandez, Lorraine D; Delos, Sue E; White, Judith M","Cauchemez, Simon; Epperson, Scott; Biggerstaff, Matthew; Swerdlow, David; Finelli, Lyn; Ferguson, Neil M.","Coleman, Christopher M.; Frieman, Matthew B.","Chappell, James D.; Dermody, Terence S.","Smith, Ina; Wang, Lin-Fa","Wang, Lin-Fa; Shi, Zhengli; Zhang, Shuyi; Field, Hume; Daszak, Peter; Eaton, Bryan T.","Danielle E Anderson; Jin Cui; Qian Ye; Baoying Huang; Wenhong Zu; Jing Gong; Weiqiang Liu; So Young Kim; Biao Guo Yan; Kristmundur Sigmundsson; Xiao Fang Lim; Fei Ye; Peihua Niu; Xuming Zhou; Wenjie Tan; Lin-Fa Wang; Xu Tan"],"color":["#ef8a62","#ef8a62","#ef8a62","#ef8a62","#ef8a62","#ef8a62","#ef8a62","#ef8a62","#ef8a62","#ef8a62","#ef8a62","#ef8a62","#ef8a62","#ef8a62","#ef8a62","#ef8a62","#ef8a62","#ef8a62","#ef8a62","#ef8a62","#ef8a62","#ef8a62","#ef8a62","#ef8a62","#ef8a62","#ef8a62","#ef8a62","#ef8a62","#ef8a62","#ef8a62","#ef8a62","#ef8a62","#ef8a62","#ef8a62","#ef8a62","#ef8a62","#ef8a62","#ef8a62","#ef8a62","#ef8a62","#ef8a62","#ef8a62","#ef8a62","#ef8a62","#ef8a62","#ef8a62","#ef8a62","#ef8a62","#ef8a62","#ef8a62"],"date":["2020-03-14","2019-08-13","2019-04-10","2013-04-30","2009-12-31","2008-12-31","2020-02-14","2019-07-15","2004-12-31","2015-12-22","2020-03-14","2020-02-10","2020-02-14","2016-05-02","2014-12-19","2014-12-31","2016-12-31","2019-12-31","2019-03-01","2003-08-31","2020-02-29","2013-08-13","2008-12-31","2020-01-31","2020-02-14","2020-03-31","2017-12-31","2019-02-07","2014-12-19","2014-03-31","2008-12-31","2018-12-31","2020-03-05","2014-03-31","2020-03-21","2019-09-09","2017-12-31","2019-08-13","2007-12-31","2020-02-04","2018-12-10","2020-03-29","2020-12-31","2003-12-31","2013-03-05","2013-09-05","2015-12-31","2013-02-28","2006-12-03","2020-03-30"],"doc":["89fol3pq","eiobmxp2","4kvkioa3","0g5079y1","e5frjae8","obgjbl6k","u2n8so5w","3nfukrfl","76jrm77c","d2knbzhl","89fol3pq","dqpujzo8","u2n8so5w","ljggnelt","m4jwq0nm","z0bxsjug","kwavs4mx","b5l6epy3","yb54i1ne","z9i5mk33","zpiaka80","zfhpij93","a6a422uv","6p23qxb8","u2n8so5w","89sg0cpk","apc0lm5e","hibzy1oh","nna0s5mf","a4mf2vnp","jizitnfl","fgrwwzn7","fu3cl7lq","a4mf2vnp","9il7coyk","r2os3kn9","dgi4ibym","eiobmxp2","dwf5vydu","cetdqgff","ur8g68vp","nil1vv6h","mog52dn8","xpl6tjps","16c8dwfq","k38we345","apy0nc1m","v8uzd2f3","v08cs51n","0l33i6s4"],"journal":["Emerg Microbes Infect","Viruses","PLoS One","Current Opinion in Virology","Encyclopedia of Microbiology","Travel Medicine","Viruses","Int J Environ Res Public Health","Handbook of Proteolytic Enzymes","Virol J","Emerg Microbes Infect","Int Health","Viruses","American Journal of Infection Control","Virus Research","Encyclopedia of Agriculture and Food Systems","The Vaccine Book","Genomic and Precision Medicine","Vector Borne Zoonotic Dis","Journal of Allergy and Clinical Immunology","Travel Medicine and Infectious Disease","Cell Research","Encyclopedia of Virology","Emerg Microbes Infect","Viruses","","Genetics and Evolution of Infectious Diseases","Trop Med Infect Dis","Virus Research","Research in Microbiology","Encyclopedia of Virology","Reference Module in Biomedical Sciences","","Research in Microbiology","Trends in Molecular Medicine","PLoS Biol","Early Warning for Infectious Disease Outbreak","Viruses","Virus Research","","J Vet Res","","Genomics and Biotechnological Advances in Veterinary, Poultry, and Fisheries","Methods in Enzymology","PLoS Med","PLoS Pathog","Mandell, Douglas, and Bennett's Principles and Practice of Infectious Diseases","Current Opinion in Virology","Emerg Infect Dis",""],"rad":[0.02,0.017274167730724297,0.012441188635449952,0.010507631935880397,0.009727073297666613,0.009615437447580753,0.009480365321490748,0.009391731114179267,0.009377546333749385,0.00890123425520983,0.008730634373288595,0.007978728219939602,0.0079609477576137,0.007729115797286722,0.007659857698649107,0.0068325804824439255,0.006759380481240591,0.006590783252651439,0.006049194617122984,0.006020898899248265,0.0059544737383213486,0.005836106971716985,0.005799868628274902,0.005561587547696504,0.005277915881044251,0.0051993380647060335,0.005114322452577649,0.004911133350883711,0.004841656280031955,0.0047473711771526175,0.004724323300904344,0.0046845887119378905,0.004600504410649091,0.004419402805806617,0.004241045857610445,0.0042288206026279695,0.00418401128784277,0.003902318811844747,0.0038878851080174598,0.003699740639163428,0.0036304331648486076,0.0035911058162349323,0.0035145251026062778,0.0034424097527639697,0.0033922360316212883,0.0033751089694378186,0.0033257631916445898,0.0031610902573465257,0.0031424490885575643,0.003],"scr":[1.0,0.84,0.555,0.442,0.396,0.389,0.381,0.376,0.375,0.347,0.337,0.293,0.292,0.278,0.274,0.225,0.221,0.211,0.179,0.178,0.174,0.167,0.165,0.151,0.134,0.129,0.124,0.112,0.108,0.103,0.101,0.099,0.094,0.083,0.073,0.072,0.07,0.053,0.052,0.041,0.037,0.035,0.03,0.026,0.023,0.022,0.019,0.009,0.008,0.0],"text":["The genomic sequence of MERS-CoV was closely related to bat CoVs HKU4 and HKU5 [18]. Bat CoVs that are evolutionarily closer to MERS-CoV, sharing \u223c75% nucleotide sequence homology and using the same DPP4 receptor, were also identified [32]. Although bats are the evolutionary reservoir host and bat CoVs serve as the gene pool of MERS-CoV, humans acquire MERS-CoV from diseased dromedary camels, but not directly from bats. These camels are the natural reservoir host of MERS-CoV. MERS-CoVs isolated from dromedaries are identical to those found in humans. Experimental infection of dromedary camels with MERS-CoV results in mild disease, shedding large quantities of the virus from the upper respiratory tract [33]. In addition, other non-camelid domestic animals in close contact with infected camels, including sheep, goats, a cow, and donkeys, are also infected by MERS-CoV [34]. These domestic animals could also pose a risk to humans and should, therefore, be included in the MERS-CoV surveillance programme.","Cats live in close contact with humans and often roam around freely in the environment. Hence cats are an interesting species to study for infections with coronaviruses. Infections with feline coronaviruses (FCoVs) are recognized and widespread [15,16]. FCoVs are classified into two types, type 1 and type 2, based on the genetic and antigenic difference of their spike (S) protein [17]. In the field, the majority of FCoV infections are caused by FCoV type 1, while FCoV type 2, derived from recombination events of type 1 FCoVs and CCoVs obtaining the S gene and some flanking regions of CCoVs, is less prevalent [18,19]. Depending on the virulence of the FCoV strain and the immune response of the cat, the clinical presentation can range from apparently asymptomatic, through diarrhea, to full-blown feline infectious peritonitis [20]. FCoVs are members of the genus alphacoronavirus, to which also HCoV-229E, porcine transmissible gastroenteritis virus (TGEV), and CCoV belong. The latter three viruses and FCoV type 2 have been proven to use feline aminopeptidase N (fAPN) as a functional receptor in vitro [21]. The receptor for type 1 FCoV has still not been identified [22]. Notably, previous studies have shown that HCoV-229E and CCoV could infect cats after experimental inoculation, causing an asymptomatic infection [23,24]. Thus, cats might potentially become naturally infected with CoVs of other species which may lead to virus-host adaptation e.g., mutation or recombination, resulting in emergence of novel coronaviruses and potentially new diseases [19,25]. The extent to which infections with CoVs of other species occur in the field, has not been explored in previous epidemiological studies of CoV infections in cats [15,26,27,28].","Bats (order Chiroptera) are the second most diverse mammalian order after rodents, comprising more than 1400 species that represent approximately 20% of all mammalian species [1]. Bats are beneficial in arthropod suppression, seed dispersal, and pollination; and serve as a bushmeat food source for some human populations [2, 3]. Bats are also natural reservoirs for a wide range of emerging viruses that may cause infectious diseases in humans, livestock, domestic animals and wildlife [4], and as a group bats also harbor a higher proportion of zoonotic viruses than other mammals [5]. As with most groups of mammals, bats are predicted to have a large diversity of viruses, which are currently undescribed [5, 6]. The development of more powerful high throughput sequencing (HTS) technology and bioinformatic tools now allow us to better characterize this viral diversity, including analysis of samples from geographic regions that have received little wildlife surveillance attention to date. Recently several bat virome studies have been published using high-throughput viral metagenomic approaches, revealing a wide range of viruses and expanding our knowledge of viral diversity. A coronavirus similar to Middle East respiratory syndrome coronavirus (MERS-CoV), adenoviruses, herpesviruses and circoviruses were reported in Neoromicia capensis bats from South Africa. Yinda et. al reported diverse rotaviruses, bastroviruses and sapoviruses using HTS analysis of Eidolon helvum and Epomophorus gambianus bats from Cameroon [7\u201310].","A vector can be defined as an organism that transmits a pathogen or disease-causing organism from a reservoir to a host. In the context of this review, vectors are restricted to arthropods, and particularly mosquitoes, ticks, sand flies and Culicoides or biting midges, as transmitters of pathogenic threats to humans or livestock. The role of mosquitoes in pathogen emergence is largely one of major geographic spread due to incursions of mosquitoes into new habitats. There are a number of major mosquito species that have jumped continents over the past three decades (e.g. [110 ] ), but there is little doubt that the most important ongoing threats come from extensive tropical urbanization and the colonization of this expanding habitat by Aedes (Stegomyia) aegypti [111] , and the global expansion in the geographic distribution of Aedes (Stegomyia) albopictus [112] . The latter has expanded to establish in at least 26 new countries in Africa, Europe and the Americas from its original home in tropical forests of south-eastern Asia. Thus there is a significantly increased risk of transmission of arthropod-borne (arbo)viral diseases, especially dengue and chikungunya [110 ,113-115] .","Until recently, the recognized 100 HRV serotypes were classified according to receptor specificity into three groups: The major group with 90 serotypes, whose receptor is intercellular adhesion molecule-1 (ICAM-1); the minor group with 10 serotypes, whose receptor is the low-density lipoprotein receptor (LDLR); and HRV-87 that shares properties with human enterovirus 68 and utilizes sialic acid residues on cell proteins as receptor. The range of HRV serotypes and species by the currently accepted classification system, which is based on sequencing and phylogenetic comparisons rather than on receptor usage or antiviral susceptibility, is expected to expand as more sequence information is obtained from field strains. In fact, a new genetic clade has been recently detected in Hong Kong, consisting of strains associated mostly with LRTI in children with wheezing that contained important deletions in the VP1 region, probably representing a new species of HRV, named HRV C.","Scrub typhus (tsutsugamushi fever) is caused by the rickettsia Orienta tsutsugamushi, which is transmitted by the bite of larval mites (chiggers). The term tick typhus encompasses a number of tick-borne infections caused by various rickettsias in different parts of the world (e.g. Rickettsia conorii in southern Europe, Africa, and Asia, and R. africae in East and Southern Africa). The reservoir of infection includes domestic dogs, rodents, and livestock. People become infected in grassy or shrub-covered areas while on safari, treks, or camping trips.","Experiments that compare modern clinical assays with HTS-based tests conclude that data derived from sequencing might be helpful in several ways. It can serve for pathogen detection and identification and also supply clinically valuable information on the presence of resistance genes or resistance-yielding mutations in a pathogen\u2019s genome [110]. It could help healthcare professionals choose the correct therapy without having to wait for microbiological tests and prescribing broad-spectrum drugs that are constantly shown to have adverse side effects [111]. Of note is that NGS finds pathogens beyond standard testing panels, unlike traditional tests. However, genetic libraries must be created, often from samples with low concentrations of a pathogen\u2019s nucleic acids and high content of nucleic acids from a host and other microorganisms. An elegant solution has been proposed by two independent teams of researchers, resulting in the development of a novel commercially available enrichment tool [112] based on hybridization.","The risk assessment of MERS-CoV infection, transmission and severity is crucial in predicting and preventing further outbreaks of human infections and in enhancing control measures. Recent studies have advised that dromedary camels (Camelus dromedarius) serve as a reservoir host for MERS-CoV, and camel-to-human transmission can occur through sporadic zoonotic infections associated with exposure to infected dromedary camels and their products [3,4,5]. The risk factors were identified for primary MERS-CoV infection in persons with either direct or indirect exposure to camels. In particular, these included persons who handle live camels (e.g., camel workers, camel salesmen, camel shepherds, slaughterhouse workers, animal and waste transporters) and persons who administer medications to camels or clean camel medical equipment [6,7,8]. Nevertheless, most patients report no exposure to camels [9]. The global camel population is estimated at approximately 35 million camels [10]; of these, 95% are dromedary camels [11]. In Saudi Arabia, the camel is considered a national, socio-cultural heritage animal and is central in the livestock economy, as it is used for milk or meat and in a traditional race festival (locally known as Mazain al-Ibal) [12]. The number of camels in Saudi Arabia is estimated at more than 1.39 million heads [13] and has been growing annually by 5.2% since 1961 [10]. Information is lacking regarding the distribution of camels in Saudi Arabia. In June 2019, the Saudi Ministry of Environment, Water and Agriculture launched an electronic system for numbering and tracking camels in order to create a database on them through the use of microchip implants [14]. This endeavor is a part of a wider initiative to control animal diseases and to create a database on each camel and its owner. The new project will help improve camel health and control the spread of zoonotic diseases.","A novel feature of mAAP is its ability to serve as a receptor for certain viruses, especially coronavirus 229E, an RNA virus that causes upper respiratory tract infections in humans (Yeager et aL, 1992) . Mutagenesis studies suggest that the virus-binding site lies close to the activesite region, although enzyme activity is not essential for virus binding. Human mAAP also appears to mediate human cytomegalovirus infection although, again, enzyme activity is not essential for infection (Soderberg et aL, 1993) . Another coronavirus, transmissible gastroenteritis virus, which causes a fatal diarrhea in newborn pigs, uses intestinal mAAP as its receptor (Delmas et aL, 1992) . mAAP appears to be the major receptor for the CryLAc toxin of Bacillus thuringiensis in Lymantria dispar (gypsy moth) (Lee et aL, 1996; de Maagd et aL 2001) . mAAP is synthesized in a fully active form. Substance P and bradykinin, which are not substrates for mAAP, have been reported as natural inhibitors of the enzyme with K[ values in the low micromolar range (Xu et aL, 1995) . However, it is unlikely that they play any physiological role in regulating enzyme activity and the enzyme is therefore probably essentially unregulated at the surface of cells. A recent study has, however, shown that oxidoreductasemediated modulation of the thiol status of the cell surface markedly affects the activity of mAAP (Firla et aL, 2002) and the enzyme is upregulated in response to hypoxia (Bhagwat et aL, 2001) . Expression of CD13 may vary during cell growth and differentiation and certain cytokines, e.g. interleukin 4 and interferon \u03b3, can upregulate levels of mAAP mRNA and protein (Riemann et aL 1999) .","With human activity increasingly overlapping the habitats of bats, diseases outbreaks resulted from spillover of bat coronaviruses will continue to occur in the future despite the fact that direct transmission of bat coronaviruses to humans appears to be rare. To better prepare ourselves in predicting and preventing the next emergence of a coronavirus disease, it is necessary to maintain our vigilance in long-term coronavirus surveillance studies in bats as well as in other wildlife and livestock. Combined with other laboratory-based studies such as receptor specificity, pathogenesis and animal infection, a focus on continued surveillance will help us to improve risk assessment as well as to reveal the potential intermediate hosts that may play an important role in the interspecies transmission of various known and as yet unknown bat coronaviruses.","In an effort to identify the direct animal source of SARS-CoV, SARS-CoV-related CoVs (SARS-rCoV), which share 99.8% sequence homology at the nucleotide level with SARS-CoV, were isolated in 2003 from workers working in a live animal market where animal meats were sold and from animals in the same market [28], including Himalayan palm civets and a raccoon dog. Palm civets were once thought to be the natural host of SARS-CoV as anti-SARS-CoV antibody was detected in civets in the market. In experimental infection, civets were equally susceptible to SARS-CoV and SARS-rCoV. Infected animals displayed clinical symptoms. However, no anti-SARS-CoV antibodies were detected in any wild or farmed civets [29], raising the possibility that they are not a natural host of SARS-CoV and SARS-rCoVs. In 2005, horseshoe bats were identified as a natural host of SARS-rCoVs [30,31]. These bat SARS-rCoVs serve as the gene pool and an evolutionary origin of SARS-CoV. It is particularly noteworthy that a SARS-rCoV using the same ACE2 receptor as SARS-CoV was also found in bats [32]. Their genomes share 95% nucleotide sequence homology. Presumably, palm civets and other mammals in the market were transiently infected, and they transmitted the virus to humans. It remains to be clarified whether another stable and natural reservoir host of SARS-CoV, exactly like dromedary camels for MERS-CoV, might exist.","Enforcement of current wildlife protection policy and continued community infrastructure development appears to significantly reduce high-risk contact between humans, wildlife and livestock. Closer collaboration between local animal and human health authorities within the current epidemic disease prevention programmes will provide educational and training opportunities to promote risk-mitigation knowledge, skills and best practice in local communities. For example, cave monitoring and management is a low-cost and efficient method to help restrict human activities (e.g. recreation and mining) that lead to contact with bats in caves. This is of particular importance given the emergence of 2019-nCoV, which appears likely to be a bat-origin coronavirus.10,11","Metagenomics allows for identification and in-depth studies of unknown viral pathogens by using shotgun sequencing [42,43,44], which has been extensively applied in clinical and environmental research [45,46]. Nucleic acids from a sample undergo virtually unbiased sequencing, i.e., with minimum prejudice towards specific organisms [47,48]; in theory, the method can be used to analyze a potentially unlimited range of targets [49,50]. Nevertheless, there is evidence that metagenomic sequencing is restricted by multiple pitfalls and biases, based on the pathogen\u2019s structure, extraction method, GC content, and other factors [51,52,53]. Therefore, there are five major limitations: (1) sequences of interest should share at least low identity with the analogous sequences in a reference genome, ensuring correct mapping; (2) the analytical complexity of obtained data often requires employment of a qualified bioinformatician and the use of a specialized computational infrastructure (including both hardware and software; (3) hidden experimental and methodological prejudices towards certain taxa; (4) defining the method\u2019s sensitivity and properly measuring it against a relevant reference; (5) dealing with the abundance of host-cell, bacterial and fungal nucleic acids.","The natural reservoir host of Ebola virus remains unknown. However, the detection of antibodies against Ebola and Ebola virus fragments in fruit and insectivore bats are highly suggestive that these animals serve as a reservoir.","Human primary monocytes are activated following HCoV-229E infection . Since they eventually become macrophages as they invade tissues, this activation suggests that HCoV-229E-infected monocytes would serve to facilitate their passage toward other tissues including the CNS, especially in immune-compromised individuals, as this was observed for murine cytomegalovirus (MCMV) (Reuter et al., 2004) . The fact that HCoV-229E could only infect partially immune-compromised transgenic mice (Lassnig et al., 2005) suggests that, being an opportunistic pathogen, HCoV-229E could take advantage of an immune-suppressed environment and disseminate to the CNS within susceptible individuals. The establishment of a persistent infection in a human leukocytic cell line is also consistent with the possibility that monocytes/macrophages serve as a reservoir and vector for this neuroinvasive HCoV (Arbour et al., 2000) . The SARS-CoV was also shown to infect monocytesmacrophages (Gu et al., 2005; Nicholls et al., 2006) and dendritic cells, where it modulates innate immunity (Spiegel et al., 2006) . These cells could also serve as a reservoir for virus to reach and maintain itself in the CNS. Our results indicate that HCoV are able to infect human endothelial cells of the BBB in culture (unpublished data). It has also been speculated that SARS-CoV could do the same after viremia (Guo et al., 2008) . Therefore, neuroinvasive coronaviruses that infect humans could use the hematogenous route to penetrate into the CNS.","Certification standards often have tolerance levels of some small percentage of infection, defined requirements for the site where the crop (seed or vegetative) is grown, required field inspection(s) during the production cycle that include cleanliness in terms of weeds, other crops that may serve as contaminants in seed lots or as hosts for targeted pathogens and freedom from pathogen vectors. Tolerances for pathogens depend on the rate of pathogen spread for annual crops, but tolerances are much lower for perennial crops. Crops that are only grown for a single year or a few years in production fields, such as most seed crops and potatoes, strawberries, sweet potatoes etc., may have higher tolerances than most of the fruit and nut crops, such as citrus, tree fruits, grapevines, or berries, which are expected to be productive for many years or decades in the same field. To set tolerances for various pathogens requires sampling and testing, and a major concern is how many samples to test. The number of samples that need to be tested per lot to achieve a confidence level (i.e., 95% or 99%) that a pathogen is not present is outlined in ISPM No. 31 (Anonymous, 2008) . In some countries, certification schemes are managed at the national level and in others at the state or provincial level.","The complete genome sequence of the implicated coronavirus was rapidly identified and made available in GenBank. Genomic sequencing enabled phylogenetic and taxonomic analysis and MERS CoV was identified as a lineage C betacoronavirus, distinct from previously known human coronaviruses and most closely related to HKU4 and HKU5 coronaviruses previously isolated from bats. 50, 51 Clinical presentation of MERS CoV can range from asymptomatic to a severe respiratory illness that culminates in death. Following an incubation period of 5.2 days (1.9-14.7 days), presenting symptoms can include fever, cough, and dyspnea. Rapid deterioration of respiratory status can occur within just a few days of symptom onset and fatality rates are estimated to be about 35%. Since 2012, there have been 1595 lab-confirmed cases of MERS CoV infection and at least 571 deaths globally. 52 This includes an outbreak of 186 individuals in South Korea initiated by a single index case, a man returning from a trip to Saudi Arabia. The outbreak in Seoul was primarily restricted to hospital settings where 5 individual \"super-spreaders\" accounted for the majority of infections of other patients and health care providers. 53 MERS CoV expresses a membrane-anchored trimeric spike (S) protein that consists of a S1 subunit that engages the receptor on the host cell and a S2 subunit that mediates membrane fusion. 54 The receptor binding domain on S1 has served as the primary antigenic target for vaccine development. Similar to the angiotensin-converting enzyme 2 (ACE2) peptidase receptor used by SARS CoV, the MERS CoV target cell receptor is CD26, dipeptidyl peptidase 4 (DPP4), which is a 766 amino acid, type II transmembrane glycoprotein expressed on epithelial and endothelial cells of several organs including lung and kidney. DPP4 has been shown to be a biomarker of IL-13 expression in asthma and is associated with glycemic homeostasis and microvascular complications of diabetes. 55, 56 Transfection of human DPP4 into otherwise nonsusceptible cells from feline, murine and canine species enables MERS CoV infection. 57 Lu et al. subsequently solved the crystal structure of the receptor binding domain of S1 in complex with CD26. 58 Comparative serologic studies emerged shortly after MERS CoV was first identified to look for zoonotic reservoirs. Evaluation of serum-specific IgG targeting the receptor-binding S1 subunit was performed by protein microarray and confirmed by virus neutralization testing and revealed high-titer neutralizing antibodies were prevalent in camels. 59 Experimental support for the camel as a zoonotic reservoir was performed by Adney and coworkers who inoculated camels with MERS CoV. The camels subsequently developed upper respiratory symptoms and showed evidence of upper respiratory tract viral shedding in nasal secretions for 7 days postinoculation. 60 Evidence of zoonotic transmission from camel to humans was further supported from a patient in 2013 who died of MERS CoV following exposure to camels with rhinorrhea. Nasal swabs for both the patient and camels were positive for MERS CoV RNA and subsequent genomic sequencing revealed the isolates to be identical 61 However, not all cases of MERS CoV infection have been linked to camel exposure. Additional studies are needed to further define transmission dynamics and intermediate hosts.","The genetic and antigenic evolution of error prone RNA respiratory viruses, particularly influenza, has been of interest for several decades [37, 38] . Understanding the selective pressure exerted by pre-existing immunity on viral evolution may help design more effective influenza vaccines and surveillance of animal populations can be critical for early identification of emerging influenza viruses [39] . Advances in deep sequencing make it possible to measure low frequency within host viral diversity and factors such as antigenic diversity, antiviral resistance, and tissue specificity can now be studied to understand the complexities of viral evolution [40] . Influenza evolution at a population level has been studied years, yet, new antigenic variants are initially generated and selected at the level of the individual infected host. Within a host, influenza viruses exist as a \"swarm\" of genetically distinct viruses [41] . Sanger sequencing defines consensus sequences and cannot resolve minority variants below 20% of the viral population. Deep sequencing has been used in natural infection and human challenge studies to characterize between and within host genetic diversity [41, 42] . The identification of low frequency mutations in the hemagglutinin (HA) antigenic sites or near the receptor-binding domain in vaccinated and unvaccinated influenza infected persons highlight viral evolution within a host due to selective immune pressure [41] . Similarly, NGS can reveal the rapid evolution of drug resistant variants during therapy [43] . Using samples collected over time, the mutational spectrum of H3N2 influenza A virus in an immunocompromised child was delineated [44] . Individual resistance mutations appeared weeks before they became dominant, evolved independently on cocirculating lineages. The within host evolution of antiviral resistance reflected a combination of frequent mutation, natural selection, and a complex pattern of segment linkage and reassortment. Within host sequencing diversity has also been examined in an infant with severe combined immune deficiency with persistent RSV infection [45] . NGS was performed on 26 samples obtained before and after bone marrow transplantation. The viral population appeared to diversify after engraftment with most variation occurring in the attachment protein (G). In addition, minority viral populations with palivizumab resistance mutations emerged after its administration. Deep sequencing of HRV during human challenge studies has shown that HRV generates new variants rapidly during the course of infection with accumulation of changes in \"hot spots\" in the capsid, 2C, and 3C genes [46] .","As an emerging infection of zoonotic origin (Morse 1995), a unified One Health approach that engages the human health, livestock and companion animal, wildlife, and environment sectors will be crucial in dealing with MERS (Morse 2012, Morse et al. 2012). It could serve as an excellent case study for epidemiology and response to a new zoonotic, or emerging, infection.","SARS appears to be due to an infection by a novel coronavirus (Cv). 1, 2 The usual human-infecting Cv strains cause up to 30% of \"colds\" but only rarely cause pneumonia. However, animal-strain Cv can cause extensive epidemics of lower respiratory tract diseases in livestock and poultry. Current evidence suggests that the SARS-associated Cv is a previously unknown Cv strain, probably originating in nonhuman hosts, which has somehow acquired the capacity to infect human beings. Airborne transmission by both large droplets and droplet nuclei appears to allow this Cv to reach the alveoli, leading to a high incidence of pneumonia.","Regarding 2019-nCoV investigations, to date, could already reveal that its genome (GenBank accession MN908947) has the highest similarity (89%) to a SARS-related member of the Sarbecoviruses (GenBank accession MG772933), a subgenus within the Betacoronavirus genus [16] . This fact could perhaps be part of the explanation about the behavior of this novel coronavirus concerning human infection. Preliminary genome sequencing suggested the snake as a likely wildlife reservoir responsible for the current outbreak of 2019-nCoV infection and origin-unknown homologous recombination identified within the spike glycoprotein of the 2019-nCoV may explain snake-to-human cross-species transmission [2] . However, this theory has now been largely discounted and some researchers point towards bats as a reservoir and doubt that the coronavirus could have originated in animals other than birds or mammals [2] ; they are skeptical that the animal host or hosts of 2019-nCoV can be identified without further field and laboratory research. Many hope that genetic tests of animals or environmental sources, such as cages and containers, from the Wuhan market will reveal clues [17] regarding the definite origin of the virus. The outbreak of viral pneumonia in Wuhan is associated with exposures at the Huanan Seafood Wholesale Market, suggesting a possible zoonosis.","In conclusion, knowing the molecular details of the coronavirus-receptor interface will be highly instrumental in predicting interactions between MERS-CoV and orthologous receptors and mutation-driven host range expansion, and may help to identify susceptible host species and hence the host reservoir(s) for MERS-CoV. In addition, the structural information of the interface between the S protein and the receptor may provide novel strategies for developing effective antibodies or drugs that target the spike-receptor interface.","Other cell-surface molecules such as L-SIGN, DC-SIGNR, DC-SIGN (CD209), and L-SECtin may serve as binding receptors but do not appear to be functional viral receptors in the absence of ACE-2. They may, however, promote cell-mediated transfer of the virus to other susceptible target cells. On the other hand, binding to L-SIGN appears to lead to proteasome-dependent viral degradation and it may function as a scavenger receptor (see below).","The rapid identification of this novel coronavirus is attributed to recent advances in the detection of respiratory virus infection, including reverse transcription PCR (RT-PCR), real-time reverse transcription PCR (rRT-PCR), reverse transcription loop-mediated isothermal amplification (RT-LAMP), and real-time RT-LAMP as well as multiplex nucleic acid amplification and microarray-based assays [4]. These methods are useful for detecting novel coronaviruses not only in humans, but also in animals for identification of animal reservoir or intermediate host of 2019-nCoV. WHO recommended that if there is no clue about the putative pathogen from the pneumonia outbreak, a pan-coronavirus assay should be used for amplification followed by sequencing of the amplicon for characterization and confirmation (https://apps.who.int/iris/bitstream/handle/10665/330374/WHO-2019-nCoV-laboratory-2020.1-eng.pdf). By aligning 2019-nCoV S protein sequence with those of SARS-CoV and several bat-SL-CoVs, we predicted that the cleavage site for generating S1 and S2 subunits is located at R694/S695 (Figure 1). S1 subunit contains two functional domains, the N-terminal domain (NTD) and a receptor-binding domain (RBD), both of which are responsible for the binding of the virion to the receptor on the host cell. They also contain several conformational neutralizing epitopes, serving as a target for developing neutralizing antibodies and vaccines [5]. S2 subunit contains three functional domains, fusion peptide (FP), and heptad repeat (HR) 1 and 2. After binding of RBD in S1 to the receptor, the S2 subunit changes conformation by inserting the FP into the host cell membrane and association between HR1 and HR2 to form six-helical bundle (6-HB), resulting in the fusion between viral and cellular membranes. The viral genetic materials enter into the host cell through the fusion pore for replication in the cell [5]. A peptide derived from the HR2 domain of SARS-CoV S protein (SC-1) can interact with HR1 region in viral S protein to form heterologous 6-HB, resulting in the inhibition of homologous 6-HB formation between HR1 and HR2 domains in viral S protein and thus blocking the viral fusion with the host cell [6]. Since 2019-nCoV S-HR2 sequence is 100% identical to that of SARS-CoV, while there are only a few mutations of non-critical amino acids in S-HR1 region, SC-1 peptide is expected to be also effective against 2019-nCoV infection.","The most notable feature of this approach is that molecular probes, instead of targeting selected viral species or families, bind to certain sequences in all known viral genera infecting vertebrates, thus supposedly creating the most comprehensive tool for viral screening. This enrichment technique greatly increases sequencing depth, while also raising the fraction of viral reads by 100\u201310,000 times compared to standard library preparation protocols [113,114]. A diagnostic tool of such sensitivity could serve as an alternative to traditional methods when they fail to detect the pathogen. In contrast, total sample sequencing could at least partly detect a new pathogen\u2019s sequences, revealing its identity.","Besides pets, livestock are another category of mammals that are frequently in direct contact with humans. We profiled the ACE2 expression in cows, pigs, rabbits (which can also serve as pets), and goats. In these animals, ACE2 was highly expressed in multiple tissues including kidney, liver, and heart, but not in muscles ( Figures 5D-5F and S2B in Supplementary data). ACE2 was also highly expressed in the fat of pigs.","Natural reservoir refers to the long-term host of the pathogen of an infectious disease. It is often the case that hosts do not get the disease carried by the pathogen, or the infection in the reservoir host is subclinical, asymptomatic, and nonlethal. Once discovered, natural reservoirs elucidate the complete life cycle of infectious diseases, which in turn will help to provide effective prevention and control strategies.","Surveillance programmes for bat lyssaviruses commonly report a diverse range of wild bat species apparently participating in the epizootiology of lyssaviruses. Spill over between ecologically distinct species appears to be common, with the divergence dates for viral clades often measured as evolutionarily recent events (i.e., &lt;1000 years) [88,138,139,140]. The sheer diversity of bat species associated as potential reservoir hosts to different lyssavirus species suggests historic spill over events and the subsequent maintenance of disease in novel hosts. This would have ultimately promoted new disease reservoirs and the speciation of lyssaviruses. From a host perspective, factors such as the fundamental ecology of roost choice by individual temperate bats, which may fall within geographical locations consistent with what is now defined as the EU, have mediated intraspecific and interspecific transmission events. ","Serological assays have been used successfully as a prescreening tool for virus discovery (Burbelo et al., 2012; Drexler et al., 2013; Muller et al., 2007; Peel et al., 2013) . Antibody detection assays are particularly valuable because short phases of virus shedding or poor sample quality decrease the chances of detecting nucleic acids. In addition, they allow the distribution of pathogens in various species and the antigenic relatedness with already known pathogens to be defined (Drexler et al., , 2012 . Importantly, testing of livestock and wild animals can help to trace the source of infection during an outbreak of a zoonotic virus . The identification of the animal reservoir allows the pathogen to be contained by interrupting the chain of transmission; for example, by shutting down animal markets or culling infected animals. This has been a successful approach with outbreaks of avian Influenza and SARS-CoV (Kan et al., 2005; Tam, 2002; Tu et al., 2004; Watts, 2004) .","Furthering the topic of adaptation of bacterial pathogens to different host species, Nathalie van der Mee-Marquet (Centre Hospitalier Universitaire, Tours, France) summarized the host switching process and genomic changes of S. aureus clonal complex 398 (CC398). CC398 is primarily a livestockassociated clone found in pigs, but is also able to successfully infect humans and is increasingly being found in humans with no associations with livestock [47] . Based on characterization of the prophages of CC398, it was shown how these elements may play a role in host adaptation. Interestingly, the phi-MR11-like prophage appears to mediate expression of virulence genes encoded on a second prophage, thereby conferring increased invasion of human cells.","Viral agents were part of the biological weapons arsenal of both the Soviet Union and the United States ( Table 2) . VEE virus was stockpiled by both countries as an incapacitating agent; Variola major and Marburg viruses were stockpiled as lethal agents by the Soviet Union. The Soviet Union reportedly conducted a live field test of Variola major virus on Vozrozhdeniye Island in the Aral Sea in the 1970s, in which 400 g of the virus was released into the atmosphere by explosion. Unfortunately, a laboratory technician who was collecting plankton samples from an oceanographic research vessel 15 km from the island became infected. It was reported that after returning home to Aralsk, she transmitted the infection to several people including children. All those infected died. A number of other viruses that infect humans (e.g., Ebola virus, Lassa fever virus, enterovirus 70) or livestock (e.g., foot and mouth disease virus, rinderpest, Newcastle disease virus) have also been studied for their offensive capabilities or for the development of medical and veterinary countermeasures. Today, with the increased level of concern, a number of viruses have been cited as possible weapons for use against humans or animals ( Table 2 ). The requirements for an ideal biological warfare agent include availability, ease of production, stability after production, a susceptible population, absence of specific treatment, ability to incapacitate or kill the host, appropriate particle size in aerosol so that the virus can be carried long distances by prevailing winds and inhaled deeply into the lungs of unsuspecting victims, ability to be disseminated via food or water, and the availability of a vaccine to protect certain groups. Other factors such as the economic and psychological impact of an attack on animal agriculture with a viral agent must also be considered.","Africa cause serious disease in small ruminant animals and humans. RVF disease has expanded its historical geographic range in the livestock-raising areas of eastern and southern Africa and into the Middle East (Saudi Arabia and Yemen) over the past 25 years, causing massive epidemics in Egypt, along the Mauritania-Senegal border and in Madagascar. A major outbreak in East Africa began in 2006 in northeastern Kenya, and spread into southern Somalia and Tanzania. Cattle, sheep, and humans are affected. Abortion storms with febrile disease and bloody diarrhea occur in ruminant animals, and mortality may be heavy in young stock. Most infected humans develop febrile disease, with prolonged convalescence. A few individuals develop more severe disease, with liver necrosis, hemorrhagic pneumonia, meningoencephalitis, and retinitis with vision loss. The human case-fatality rate is &lt;1%. RVF virus is in the genus Phlebovirus of the Bunyaviridae. In sub-Saharan Africa, RVF virus is closely tied to its Aedes mosquito vectors. RVF vectors transmit the virus transovarially and horizontally. The virus persists in mosquito eggs laid around seasonally flooded pools and depressions. When these pools flood, the eggs hatch and infected mosquitoes emerge and begin transmission. The vertebrate reservoir hosts of RVF virus are unknown. Field and laboratory workers need to exercise caution to avoid becoming infected by exposure to the virus during postmortem examination of animals or processing materials in the laboratory. Both live, attenuated and inactivated vaccines are available for animals, but the unpredictability of scattered, sporadic RVF outbreaks across sub-Saharan Africa has been a major obstacle for triggering implementation of extensive, cost-effective vaccination programs. More recently, predictive ecologic models using climate data from earth-orbiting satellites along with geographic information systems has made it more feasible to anticipate RVF outbreaks in time to intervene with vaccination programs.","The ongoing outbreak of a novel coronavirus in China is now evident in 37 other countries as others employ vigilant screening procedures. 1 As of 24 Feb 2020, 119 SARS-CoV-2 genome sequences from 15 countries are publicly available from over 79,331 confirmed cases. 1,2 These viral genomes differ by only 0 to 5 mutations, likely representing a single introduction in the human population from an unknown reservoir or intermediate host, potentially bats. 3 Subsequently, the virus rapidly spread since December 2019. Currently, incubation period and transmissibility are questions of global concern, but difficult to answer given nonspecific and mild symptoms that could represent a variety of respiratory viruses. 4, 5 Genetic characterization of the virus from geographically diverse patient samples is key to infer the rate of spread. Additionally, rapid and complete results from cases worldwide are important for sequence-dependent countermeasures and accurate diagnostics. However, as the virus reaches resource-limited settings in proximity to the outbreak's epicenter, such as Cambodia, Laos, and Myanmar, there are often logistical challenges in surveillance, contact tracing, sample collection, and biospecimen transport. Information delay is also compounded by a dearth of sequencing and/or bioinformatics expertise in-country, further postponing analysis and dissemination of pathogen genomic information.","The use of molecular markers to track bacterial, viral and other pathogenic agents as they spread and evolve is a key component of infectious disease surveillance and control. The International Meetings on Microbial Epidemiological Markers (IMMEM), initiated by the European Study Group on Epidemiological Markers (ESGEM) of the European Society for Clinical Microbiology and Infectious Diseases (ESCMID), are organized every three years to discuss progress in the field. IMMEM-10 was held on October 2e5, 2013 at the Institut Pasteur in Paris and gathered 400 participants from nearly 40 countries. The program included 190 posters and 72 oral communications, six of which were presented by industry. IMMEM-10 featured the most recent advances in the exploding field of pathogen genome sequencing and applications to public health for better prevention and control of communicable diseases.","The amino acid sequence in the ACE2 receptor responsible for 2019-nCoV binding in farm animals and cats has only a few exchanges compared with the human receptor, suggesting that the species barrier for virus transmission is small. As major natural reservoir species of Alphacoronavirus and Betacoronavirus, bats carry highly diverse SARS-like-CoVs. These bats are distributed in many provinces of China. The genetic diversity of these SARS-like-CoVs and their molecular evolution within their natural host species have been studied intensively [2, [8] [9] [10] [11] . Here, we review the recent but still very limited facts about the current epidemiology of COVID-19 and discuss viral characteristics of 2019-nCoV on the backdrop of our knowledge about the previous epidemic of SARS and MERS.","In recent years, emerging and re-emerging viral diseases with high mortality have continuously posed serious threats to human health [1\u20133]. In etiologic studies with convincing evidence, a series of such fatal diseases in humans have been confirmed or hypothesized to be caused by bat-borne viruses such as Hendra virus (HeV), Ebola virus (EBOV), Marburg virus, and coronaviruses, including severe acute respiratory syndrome coronavirus (SARS-CoV) and Middle East respiratory syndrome coronavirus (MERS-CoV) [4\u201310]. Similarly, fetal diseases in livestock have also been associated with emerging viruses of bat origin, such as the fatal disease outbreak of pigs in China, which was found to be caused by a novel coronavirus, swine acute diarrhea syndrome coronavirus (SADS-CoV), from bats [11,12]. Furthermore, it was shown that influenza-like viruses, termed H17N10 and H18N11 circulating among bats in Central America, may act as an ancient influenza reservoir [13,14]. It is well accepted now that bats harbor an exceptionally high proportion of zoonotic viruses with interspecies transmission potential [15] that have the potential to become virulent pathogens for humans. However, studies from wild or experimental bats indicate that most of these lethal viruses in humans and other mammals cause only asymptomatic infection in bats, suggesting a potentially special immune system in bats that is different from most other mammals [16,17].","Laboratory surveillance is a very important component of the global epidemic outbreak surveillance and early warning network and of early warning networks for infectious diseases that are established in some countries. Laboratory surveillance allows the monitoring of pathogens, vectors, reservoir hosts, and human immunity and provides strong evidence for the early detection of infectious disease outbreaks. Moreover, it facilitates verifying the outbreaks, tracking the sources of infection, and identifying the paths of transmission. The current early warning laboratory surveillance includes the following:","Coronavirus infections are endemic and ubiquitous in feline populations. Two viral types, type 1 and 2, are distinguished and both of them could well sustain themselves in the cat reservoir [15,43]. Both have been shown to have worldwide distribution, with the seropositivity rate up to 90% among animal shelter populations and in multi-cat households [20,44]. The majority of natural infections are caused by type 1 FCoVs, while in the field type 2 FCoVs are less common and mainly occur in Asia [15,28,45,46,47]. CoVs are generally considered to be host-specific; however, cross-species transmission does occur which may lead to incidental infections like the spillover of MERS-CoV from dromedary camel to humans, where humans function as an incidental and ultimately dead-end host [4]. But CoVs might also adapt to the new host exemplified by the animal origin of all four endemic human CoVs (HCoV-OC43, HCoV-NL63, HCoV-229E and HCoV-HKU1) [8,9,10,11]. Whereas in cats infections with FCoV are well recognized, studies regarding possible natural infections with other animal and human coronaviruses are lacking to the best of our knowledge. Knowing the genetic variability of coronaviruses and the use of orthologous receptors by non-feline CoVs, studies on cross-species transmission are desirable. This may provide insight regarding whether cross-species transmission does occur. In the present study we used the highly immunogenic S1 antigens to screen cat sera for the presence of antibodies against feline and non-feline coronaviruses, as a first indication of possible infections with these viruses. ","Since coronavirus can break the host species barrier, causes trans-species infection and becomes zoonotic, it is necessary to pay attention to the coronavirus IBV infection in other species. KQ6 was pathogenic to chickens under experimental conditions, so the peafowl IBV had the potential ability to infect chickens in nature. Our investigations in peafowls indicate that KQ6 is a Massachusetts prototype like coronavirus strain which undergoes few genetic changes and peafowl might have acted as a natural reservoir of IBV for very long time.","Additional mammalian mechanisms for inhibiting viral RNA have been highlighted for 424 retroviruses with the actions of zinc-finger antiviral protein (ZAP) [81] . ZAP targets 425 CG dinucleotide sequences, and in vertebrate host cells with the CG suppression in host 426 genomes, this can serve as a mechanism for the distinction of self vs non-self RNA and 427 inhibitory consequences [81] . Coronaviruses have A/U rich and C/G poor genomes, 428 which over time may have been, in part, a product of cytidine deamination and 429 selection against CG dinucleotides [88] [89] [90] . This is consistent with the fact that bats 430 serve as a reservoir for many coronaviruses and that bats have been observed to have 431 some of the largest and most diverse arrays of APOBEC genes in mammals [66, 67] . The 432 Spearman's rank correlation data and the patterns observed in the CGR images from 433 Figure 4 , of the coronavirus genomes, including COVID-19 identify patterns such as CG 434 underepresentation, also present in vertebrate and, importantly, bat host genomes.","Wild bird species serve as a natural reservoir of many emerging zoonotic pathogens and thus have a significant impact on public health. They are also the source of pathogens dangerous to domestic animals, and such infections could have socio-economic consequences. This is the reason why over the years wild birds have been under epidemiological surveillance. Among the viruses transmitted by wild birds, the most well-known are influenza A viruses (9). Wild birds are also implicated in the spread of the West Nile virus, Borrelia burgdorferi, and other bacterial infections such as Salmonella or Campylobacter, and with them also resistance genes to antibiotics (35). Studies from the last 10 years have also shown the presence of CoVs in wild birds (22, 24, 25, 53). Among factors which make birds an excellent reservoir of various pathogens and also a bioreactor contributing to their variability, there are the high biodiversity of bird species, their ecological traits such gathering/grouping during feeding and roosting, but most importantly their capability to fly long distances (8).","Several novel viruses were identified with our short RNA bait approach, which is not unexpected as little is known about the virology of wildlife in Southeast Asia, where the leeches were collected. Several viral contigs were phylogenetically distinct from known viruses and may represent new genera. For example, the novel coronavirus identified in leech bloodmeals did not cluster with any of the known Coronaviridae clades. This finding highlights the ability of this method to detect unknown viruses. We could also associate the novel corona-and rhabdoviruses with mammal bloodmeals with limited evidence of a cervid association for both. Cervids are regularly sold as bushmeat in wildlife markets [25] and both recent coronavirus epidemics (SARS-CoV [18] and SARS-Cov-2 [16] ) spilled over from wildlife. This suggests that eDNA/iDNA-based pathogen surveillance approaches may complement efforts to proactively identify viruses that could potentially spillover to humans or livestock.","The structure provides the basis for the structure-based design of inhibitors for a range of paramyxovirus-induced diseases (Crennell et al., 2000) . In 2005, the crystal structure of the secreted, uncleaved ectodomain of the paramyxovirus was presented (Yin et al., 2005) . In 2012, combined X-ray crystallography and cryoelectron tomography were done to show the structure of matrix protein of Newcastle disease virus, a paramyxovirus. Structure and sequence conservation imply that other paramyxovirus matrix proteins function similarly (Habchi and Longhi, 2012) . In 2012, it was established that favipiravir may serve as antiviral against these viruses. Hendra virus (HeV) and NiV are bat-born paramyxoviruses. In 2013, experimental findings in animals have demonstrated that a human monoclonal antibody targeting the viral G glycoprotein is an effective postexposure treatment against Hendra and NiV infection. Also, a subunit vaccine based on the G glycoprotein of HeV affords protection against Hendra and NiV. The vaccine has been developed for use in horses in Australia. It is the first vaccine against a biosafety level-4 (BSL-4) agent to be licensed and commercially deployed. HeV is one of the members of the Henipavirus genus of paramyxoviruses, which are designated BSL-4 organisms because of the high mortality rate of NiV and HeV in humans. Paramyxovirus cell entry is mediated by the fusion protein, F, and this is in response to binding of a host receptor by the attachment protein. During posttranslational processing, the fusion peptide of F is released. Upon receptor-induced triggering, it is inserted into the host-cell membrane. F undergoes a dramatic refolding from its prefusion to postfusion conformation. This brings the host and viral membranes together, allowing entry of the viral RNA. In 2015, the crystal structure of the prefusion form of the HeV F ectodomain was reported. The structure shows great similarity with the structure of prefusion parainfluenzavirus 5 fusion protein (Wong et al., 2016) . In 2018, first successful treatment of henipavirus infection in vivo with a small-molecule drug suggests that favipiravir should be evaluated as an antiviral treatment option for henipavirus infections (Dawes et al., 2018) . Inhibitor binding pocket on the surface of neuraminidase active site. Active site residues in the protein that are involved in interaction are A125, A182, S181, A242 and T243. These are shown in the circle on the surface of the protein. (PDB ID 1E8T) (Crennell et al., 2000) .","In summary, the key principle that underlies the operation of enveloped virus fusion proteins is conversion of the fusion protein ectodomain from a hydrophilic to a hydrophobic entity by virtue of an exposed fusion peptide (Fig. 1, step 2) . In many cases (Table I ) this event appears to be induced by binding to the host cell receptor under fusion-permissive conditions (e.g., physiologic temperature). Derivatives of the assays that we have described in this chapter should prove useful in monitoring this critical unifying event of receptor-induced fusion: receptor-induced binding of the fusion protein to target membranes. It is possible to do this with either the free (oligomeric) fusion protein ectodomain or with intact viral particles or subviral particles containing the viral fusion protein. Once a target membrane binding assay is established, it should provide a powerful tool with which to dissect further requirements for a key step in the fusion cascade: hydrophobic binding of the fusion protein to the host cell bilayer (Fig. 1, step 2) .","We first consider scenario 1 where each case has the same probability of being detected. In a typical chain of transmission of average length  there is, by definition, one reservoir-to-human transmission event and  human-to-human infections. The probability G that a case randomly picked up by surveillance was infected by the reservoir is therefore . However, for subcritical outbreaks (0&lt;R&lt;1), branching process theory tells us that the average length of a chain is  [14]. We therefore obtain .","The closest phylogenetic neighbors to MERS-CoV are putative bat coronaviruses in China (BtCoV-HKU4 and BtCoV-HKU5) [3], the Netherlands (BtCoV/VM314/2008} [2], and a recently discovered isolate from South Africa [6]. All four of these bat coronaviruses have been sequenced only from bat samples and have never been isolated as live viruses from either bats or the environment. The natural reservoir of MERS-CoV has not been identified, although its similarity to these other four viruses suggests that it is of bat origin. Importantly, SARS-CoV emerged from bats as well [7]. Anecdotal evidence suggests that MERS-CoV may have been transmitted to humans via livestock (camels or goats); however, there is no scientific data yet to support this theory.","The genetic basis of host susceptibility to viral infections is complex. Studies with inbred strains of mice indicate that genetic variation can alter susceptibility to viral disease by a variety of mechanisms. 198 These can involve differences in immune responses, variability in the ability to produce antiviral mediators such as IFN, and differential expression of functional virus receptors. Polymorphisms in the expression of chemokine receptor CCR5, which serves as a co-receptor for HIV, [55] [56] [57] are associated with alterations in susceptibility to HIV infection. 199, 200 Persistent Infections Several viruses encode proteins that directly attenuate the host immune response (e.g., the adenovirus E3/19K protein 208 and CMV US11 gene product 209 block cell surface expression of MHC class I proteins, resulting in diminished presentation of viral antigens to cytotoxic T lymphocytes [CTLs]). The poxviruses encode a variety of immunomodulatory molecules including CrmA, which blocks T-cellmediated apoptosis of virus-infected cells. 210 In some cases (e.g., the CNS), preferential sites for persistent viral infections are not readily accessible by the immune system, 211 which may favor establishment of persistence.","Ina Smith 1 and Lin-Fa Wang 1, 2 Bats are being increasingly recognized as an important reservoir of zoonotic viruses of different families, including SARS coronavirus, Nipah virus, Hendra virus and Ebola virus. Several recent studies hypothesized that bats, an ancient group of flying mammals, are the major reservoir of several important RNA virus families from which other mammalian viruses of livestock and humans were derived. Although this hypothesis needs further investigation, the premise that bats carry a large number of viruses is commonly accepted. The question of whether bats have unique biological features making them ideal reservoir hosts has been the subject of several recent reviews. In this review, we will focus on the public health implications of bat derived zoonotic viral disease outbreaks, examine the drivers and risk factors of past disease outbreaks and outline research directions for better control of future disease events.","Although at this stage we cannot rule out the possibility of direct transmission from the natural reservoir host to humans, molecular epidemiologic studies (2,10) and studies of the receptor-S protein interaction (35) indicate that the progenitor viruses are unlikely to be able to infect humans and that a rapid viral evolution in an intermediate host (such as civets) seems to be necessary to adapt the virus for human infection. Ability to efficiently use the receptor molecules (ACE2 for human and civet) seems to be a major limiting factor for animal-to-human and human-to-human transmission (35). This also explains why the SARS-CoV was able to cause the human pandemic but the closely related bat SARS-like\u2013CoVs were not. For the SARS-like\u2013CoVs to infect humans, substantial genetic changes in the S1 receptor-binding domain will be necessary. These changes may be achieved in 1 of 2 possible ways. They could be achieved by genetic recombination, as coronaviruses are known to be able to recombine. For example, bat SARS-like\u2013CoVs and another yet unknown coronavirus could coinfect an intermediate host, and the bat viruses would gain the ACE2 binding site in the S1 domain by recombination. The alternative is continuous evolution independent of recombination. Coronaviruses in bats could have a spectrum sufficiently diverse to encompass the progenitor virus for SARS-CoVs. The progenitor virus's ability to bind human ACE2 may be acquired or improved by adaptation (i.e., point mutations) in &gt;1 intermediate host before it could efficiently infect humans. The existence of at least 3 discontinuous highly variable genomic regions between SARS-CoV and SARS-like\u2013CoV indicates that the second mechanism is more likely.","Bats are widely considered the largest reservoir of numerous virus species but are resilient to the 21 diseases associated with many of these viruses 7, 8 . How bats tolerate the infections with these 22 viruses is of intense interest due to the scientific and public health relevance. Genomic 23 sequencing studies have revealed several genetic adaptions underlying this tolerance of viral 24"],"title":["A tug-of-war between severe acute respiratory syndrome coronavirus 2 and host antiviral defence: lessons from other pathogenic viruses","Serological Screening for Coronavirus Infections in Cats","A viral metagenomic survey identifies known and novel mammalian viruses in bats from Saudi Arabia","Reservoirs and vectors of emerging viruses","Respiratory Viruses","","Current Trends in Diagnostics of Viral Infections of Unknown Etiology","Spatiotemporal Clustering of Middle East Respiratory Syndrome Coronavirus (MERS-CoV) Incidence in Saudi Arabia, 2012\u20132019","69 Membrane alanyl aminopeptidase","Bat origin of human coronaviruses","A tug-of-war between severe acute respiratory syndrome coronavirus 2 and host antiviral defence: lessons from other pathogenic viruses","A qualitative study of zoonotic risk factors among rural communities in southern China","Current Trends in Diagnostics of Viral Infections of Unknown Etiology","Emerging infectious diseases: Focus on infection control issues for novel coronaviruses (Severe Acute Respiratory Syndrome-CoV and Middle East Respiratory Syndrome-CoV), hemorrhagic fever viruses (Lassa and Ebola), and highly pathogenic avian influenza viruses, A(H5N1) and A(H7N9)","Human coronaviruses: Viral and cellular factors involved in neuroinvasiveness and neuropathogenesis","Pathogen-Tested Planting Material","Chapter 28 Vaccines for Emerging Viral Diseases","Chapter 9 Respiratory viral infections","What Have We Learned About Middle East Respiratory Syndrome Coronavirus Emergence in Humans? A Systematic Literature Review","Beyond our pages","The next big threat to global health? 2019 novel coronavirus (2019-nCoV): What advice can we give to travellers? \u2013 Interim recommendations January 2020, from the Latin-American society for Travel Medicine (SLAMVI)","Spiking the MERS-coronavirus receptor","Severe Acute Respiratory Syndrome (SARS)","An emerging coronavirus causing pneumonia outbreak in Wuhan, China: calling for developing therapeutic and prophylactic strategies","Current Trends in Diagnostics of Viral Infections of Unknown Etiology","Atlas of ACE2 gene expression in mammals reveals novel insights in transmission of SARS-Cov-2","26 Evolution of SARS Coronavirus and the Relevance of Modern Molecular Epidemiology","Bats and Viruses: Emergence of Novel Lyssaviruses and Association of Bats with Viral Zoonoses in the EU","Serological assays for emerging coronaviruses: Challenges and pitfalls","Microbial molecular markers and epidemiological surveillance in the era of high throughput sequencing: an update from the IMMEM-10 conference","Viruses and Bioterrorism","Viral Zoonoses","Rapid metagenomic characterization of a case of imported COVID-19 in Cambodia","Microbial molecular markers and epidemiological surveillance in the era of high throughput sequencing: an update from the IMMEM-10 conference","COVID-19: Epidemiology, Evolution, and Cross-Disciplinary Perspectives","Peptide presentation by bat MHC class I provides new insight into the antiviral immunity of bats","Chapter 1 Introduction","Serological Screening for Coronavirus Infections in Cats","A Massachusetts prototype like coronavirus isolated from wild peafowls is pathogenic to chickens","Machine learning using intrinsic genomic signatures for rapid classification of novel pathogens: COVID-19 case study","Coronaviruses in Avian Species \u2013 Review with Focus on Epidemiology and Diagnosis in Wild Birds","Non-invasive surveys of mammalian viruses using environmental DNA","Chapter 19 Advances in structure-assisted antiviral discovery for animal viral diseases","Receptor-Activated Binding of Viral Fusion Proteins to Target Membranes","Using Routine Surveillance Data to Estimate the Epidemic Potential of Emerging Zoonoses: Application to the Emergence of US Swine Origin Influenza A H3N2v Virus","Emergence of the Middle East Respiratory Syndrome Coronavirus","134 Biology of Viruses and Viral Diseases","Bats and their virome: an important source of emerging viruses capable of infecting humans","Review of Bats and SARS","Orthogonal genome-wide screenings in bat cells identify MTHFD1 as a target of broad antiviral therapy"],"url":["https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7103735/","https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6723642/","https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6457491/","https://doi.org/10.1016/j.coviro.2013.02.002","https://doi.org/10.1016/b978-012373944-5.00314-x","https://doi.org/10.1016/b978-0-323-03453-1.10063-x","https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7077230/","https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6678379/","https://doi.org/10.1016/b978-0-12-079611-3.50077-x","https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4687304/","https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7103735/","https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7017878/","https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7077230/","https://doi.org/10.1016/j.ajic.2015.11.018","https://doi.org/10.1016/j.virusres.2014.09.011","https://doi.org/10.1016/b978-0-444-52512-3.00173-x","https://doi.org/10.1016/b978-0-12-802174-3.00028-x","https://doi.org/10.1016/b978-0-12-801496-7.00009-5","https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6396572/","https://doi.org/10.1016/s0091-6749(04)70038-x","https://doi.org/10.1016/j.tmaid.2020.101567","https://www.nature.com/articles/cr2013108.pdf","https://doi.org/10.1016/b978-012374410-4.00780-9","https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7033706/","https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7077230/","https://doi.org/10.1101/2020.03.30.015644","https://doi.org/10.1016/b978-0-12-799942-5.00026-3","https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6473451/","https://doi.org/10.1016/j.virusres.2014.03.018","https://doi.org/10.1016/j.resmic.2014.01.003","https://doi.org/10.1016/b978-012374410-4.00549-5","https://doi.org/10.1016/b978-0-12-801238-3.95729-5","https://doi.org/10.1101/2020.03.02.968818","https://doi.org/10.1016/j.resmic.2014.01.003","https://doi.org/10.1016/j.molmed.2020.02.008","https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6752855/","https://doi.org/10.1016/b978-0-12-812343-0.00001-1","https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6723642/","https://doi.org/10.1016/j.virusres.2007.06.003","https://doi.org/10.1101/2020.02.03.932350","https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6296008/","https://doi.org/10.1101/2020.03.26.009993","https://doi.org/10.1016/b978-0-12-816352-8.00019-9","https://doi.org/10.1016/s0076-6879(03)72026-6","https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3589342/","https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3764217/","https://doi.org/10.1016/b978-1-4557-4801-3.00134-x","https://doi.org/10.1016/j.coviro.2012.11.006","https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3291347/","https://doi.org/10.1101/2020.03.29.014209"],"x":[0.5596453900829815,0.5605450524964966,0.5690511583339023,0.6023936316057196,0.33802728918316177,0.6046017232749843,0.6203057501247873,0.605896111578765,0.43874906234242195,0.5633974699760407,0.5567204244967043,0.697639725751259,0.6234442727241735,0.5750629908647518,0.29595200162421037,0.6274168992867979,0.5000470550431674,0.5329109903117857,0.6140493327550882,0.5755348165872487,0.5622695177521758,0.3265183740076006,0.302177716367329,0.4614894053514434,0.6175561699717812,0.5863101738535965,0.5836199202297088,0.567444732934122,0.5020213402065274,0.5119168363937054,0.596222318643655,0.5993188841749417,0.6090017210474763,0.6495541785201406,0.5653232611430874,0.5557382071944598,0.6683829867107045,0.5647076081214941,0.5544051882061445,0.5172516842978457,0.5919246172618435,0.5589488836231659,0.3642455681305844,0.2834225881041042,0.8632301815767577,0.5616022696520828,0.3076281286816052,0.5757215244775495,0.5553095479408602,0.5396376626707837],"y":[0.4503768623121414,0.4588976467557256,0.45400291141850413,0.4917102922393412,0.33686924575042904,0.5059220511353157,0.37117111897066957,0.4742771588617344,0.34989576361115915,0.4735917839748253,0.4545012598434536,0.6932897231867752,0.3539563188130227,0.5110701399582275,0.44454488432655104,0.4200702327557784,0.46890378841442715,0.3844665731029711,0.5937032130627298,0.5803942401474229,0.4359565942810215,0.34940025267404845,0.35882419321039627,0.286681133725721,0.35508042964795855,0.4511667597452138,0.6036106333124904,0.4368819628341623,0.40264009777932636,0.35516956851409726,0.557734120760294,0.5087868766898551,0.6029720379795339,0.6386683231448138,0.4399602770053347,0.5034637843708349,0.6822454268475737,0.458308647819944,0.47488956879257016,0.32334356400239983,0.48636653729717777,0.4612411674159,0.3449396564169243,0.3447725267775299,0.601328313617782,0.44935674299795686,0.38903140746932097,0.4852085348835698,0.43291482681558136,0.4192255273068005]},"selected":{"id":"9447"},"selection_policy":{"id":"9477"}},"id":"9374","type":"ColumnDataSource"},{"attributes":{},"id":"9378","type":"Range1d"},{"attributes":{"formatter":{"id":"9468"},"ticker":{"id":"9385"}},"id":"9384","type":"LinearAxis"},{"attributes":{},"id":"9467","type":"UnionRenderers"},{"attributes":{"active_drag":"auto","active_inspect":"auto","active_multi":null,"active_scroll":"auto","active_tap":"auto","tools":[{"id":"9427"},{"id":"9428"},{"id":"9429"}]},"id":"9430","type":"Toolbar"},{"attributes":{"source":{"id":"9373"}},"id":"9409","type":"CDSView"},{"attributes":{},"id":"9470","type":"BasicTickFormatter"},{"attributes":{"sizing_mode":"stretch_width","text":"&lt;h2&gt;Evidence that livestock could be infected (e.g., field surveillance, genetic sequencing, receptor binding) and serve as a reservoir after the epidemic appears to be over: Experimental infections to test host range for this pathogen.&lt;/h2&gt;"},"id":"9449","type":"Div"},{"attributes":{},"id":"9415","type":"LinearScale"},{"attributes":{"args":{"sd":{"id":"9373"},"sp":{"id":"9374"}},"code":"\n    if (sp.selected.indices.length == 0) {\n      return;\n    }\n    var sel = sp.selected.indices[0];\n    var new_selected = [];\n    for (var i = 0; i &lt; sd.data['doc'].length; i++) {\n      if (sd.data['doc'][i] == sp.data['doc'][sel]) {\n        new_selected.push(i);\n      }\n    }\n    if (JSON.stringify(new_selected) != JSON.stringify(sd.selected.indices) &amp;&amp;\n    !(sd.selected.indices.length == 1 &amp;&amp; new_selected.includes(sd.selected.indices[0]))) {\n      sd.selected.indices = new_selected;\n      sd.change.emit();\n    }\n    "},"id":"9448","type":"CustomJS"},{"attributes":{"formatter":{"id":"9480"},"ticker":{"id":"9424"}},"id":"9423","type":"LinearAxis"},{"attributes":{},"id":"9376","type":"Range1d"},{"attributes":{},"id":"9424","type":"BasicTicker"},{"attributes":{"below":[{"id":"9419"}],"center":[{"id":"9422"},{"id":"9426"}],"left":[{"id":"9423"}],"plot_height":340,"plot_width":900,"renderers":[{"id":"9437"}],"title":{"id":"9460"},"toolbar":{"id":"9430"},"toolbar_location":"below","x_range":{"id":"9411"},"x_scale":{"id":"9415"},"y_range":{"id":"9413"},"y_scale":{"id":"9417"}},"id":"9410","subtype":"Figure","type":"Plot"},{"attributes":{},"id":"9462","type":"StringFormatter"},{"attributes":{"callback":null},"id":"9427","type":"TapTool"},{"attributes":{"below":[{"id":"9384"}],"center":[{"id":"9387"},{"id":"9391"}],"left":[{"id":"9388"}],"plot_height":340,"plot_width":900,"renderers":[{"id":"9402"}],"title":{"id":"9458"},"toolbar":{"id":"9395"},"toolbar_location":"below","x_range":{"id":"9376"},"x_scale":{"id":"9380"},"y_range":{"id":"9378"},"y_scale":{"id":"9382"}},"id":"9375","subtype":"Figure","type":"Plot"},{"attributes":{},"id":"9464","type":"StringFormatter"},{"attributes":{"axis":{"id":"9419"},"ticker":null},"id":"9422","type":"Grid"},{"attributes":{},"id":"9478","type":"BasicTickFormatter"},{"attributes":{"editor":{"id":"9465"},"field":"title","formatter":{"id":"9464"},"title":"Document title","width":770},"id":"9407","type":"TableColumn"},{"attributes":{"fill_color":{"field":"color"},"line_color":{"field":"color"},"radius":{"field":"rad","units":"data"},"x":{"field":"x"},"y":{"field":"y"}},"id":"9400","type":"Circle"},{"attributes":{"callback":null,"tooltips":[["Title","@title"],["Abstract","@text"],["Published","@date"],["Authors","@authors"],["Journal","@journal"],["URL","@url"]]},"id":"9394","type":"HoverTool"},{"attributes":{},"id":"9473","type":"StringEditor"},{"attributes":{"children":[{"id":"9451"}]},"id":"9455","type":"Row"},{"attributes":{"editor":{"id":"9463"},"field":"scr","formatter":{"id":"9462"},"title":"Score","width":30},"id":"9406","type":"TableColumn"},{"attributes":{"axis":{"id":"9423"},"dimension":1,"ticker":null},"id":"9426","type":"Grid"},{"attributes":{},"id":"9393","type":"ResetTool"},{"attributes":{},"id":"9480","type":"BasicTickFormatter"},{"attributes":{},"id":"9472","type":"StringFormatter"},{"attributes":{"children":[{"id":"9408"},{"id":"9375"}]},"id":"9454","type":"Row"},{"attributes":{"text":""},"id":"9460","type":"Title"},{"attributes":{},"id":"9428","type":"ResetTool"},{"attributes":{},"id":"9477","type":"UnionRenderers"},{"attributes":{"callback":null,"tooltips":[["Title","@title"],["Passage","@text"],["Published","@date"],["Authors","@authors"],["Journal","@journal"],["URL","@url"]]},"id":"9429","type":"HoverTool"},{"attributes":{"text":"&lt;h3&gt;Paragraph ranking&lt;/h3&gt;"},"id":"9451","type":"Div"},{"attributes":{},"id":"9385","type":"BasicTicker"},{"attributes":{"data":{"authors":["Shi, Weifeng; Li, Juan; Zhou, Hong; Gao, George F.","Liu, Quan; Cao, Lili; Zhu, Xing-Quan","Smith, Ina; Wang, Lin-Fa","Fredricks, David N; Relman, David A","Villarreal, L.P.","Weicheng Liang; Zhijie Feng; Shitao Rao; Cuicui Xiao; Zexiao Lin; Qi Zhang; Wei Qi","Mohd, Hamzah A.; Al-Tawfiq, Jaffar A.; Memish, Ziad A.","Truyen, Uwe; Parrish, Colin R.; Harder, Timm C.; Kaaden, Oskar-R\u00fcger","Bexfield, Nicholas; Kellam, Paul","Muhammet Uslupehlivan; Ecem Sener","Ganova-Raeva, Lilia M; Khudyakov, Yury E","Li, Fang","Wang, Liang; Su, Shuo; Bi, Yuhai; Wong, Gary; Gao, George F.","Dahlhausen, Bob","Wang, Lin-Fa; Shi, Zhengli; Zhang, Shuyi; Field, Hume; Daszak, Peter; Eaton, Bryan T.","Gardy, Jennifer L.; Loman, Nicholas J.","Qiang Huang; Andreas Herrmann","Grubaugh, Nathan D.; Saraf, Sharada; Gangavarapu, Karthik; Watts, Alexander; Tan, Amanda L.; Oidtman, Rachel J.; Ladner, Jason T.; Oliveira, Glenn; Matteson, Nathaniel L.; Kraemer, Moritz U.G.; Vogels, Chantal B.F.; Hentoff, Aaron; Bhatia, Deepit; Stanek, Danielle; Scott, Blake; Landis, Vanessa; Stryker, Ian; Cone, Marshall R.; Kopp, Edgar W.; Cannons, Andrew C.; Heberlein-Larson, Lea; White, Stephen; Gillis, Leah D.; Ricciardi, Michael J.; Kwal, Jaclyn; Lichtenberger, Paola K.; Magnani, Diogo M.; Watkins, David I.; Palacios, Gustavo; Hamer, Davidson H.; Gardner, Lauren M.; Perkins, T. Alex; Baele, Guy; Khan, Kamran; Morrison, Andrea; Isern, Sharon; Michael, Scott F.; Andersen, Kristian G.","Agol, Vadim I.","Sutton, Troy C.; Subbarao, Kanta","Ibrahim, Ibrahim M.; Abdelmalek, Doaa H.; Elshahat, Mohammed E.; Elfiky, Abdo A.","Jonas Pfab; Dong Si","Vinicio Armijos-Jaramillo; Justin Yeager; Claire Muslin; Yunierkis Perez-Castillo","Yang, Weizhong; Lan, Yajia; Sun, Qiao; Wang, Jinfeng; Li, Zhongjie","Falsey, Ann Regina","El-Duah, Philip; Meyer, Benjamin; Sylverken, Augustina; Owusu, Michael; Gottula, Lina Theresa; Yeboah, Richmond; Lamptey, Jones; Frimpong, Yaw Oppong; Burimuah, Vitus; Folitse, Raphael; Agbenyega, Olivia; Oppong, Samuel; Adu-Sarkodie, Yaw; Drosten, Christian","Lipkin, W. Ian","Earp, Laurie J; Hernandez, Lorraine D; Delos, Sue E; White, Judith M","Jackwood, Mark W.; Boynton, Tye O.; Hilt, Deborah A.; McKinley, Enid T.; Kissinger, Jessica C.; Paterson, Andrew H.; Robertson, Jon; Lemke, Conelia; McCall, Amber W.; Williams, Susan M.; Jackwood, Joshua W.; Byrd, Lauren A.","Lu, Yanning; Liu, Ding Xiang; Tam, James P.","Zhang, Yongjiang; Yin, Jun; Li, Guifen; Li, Mingfu; Huang, Xin; Chen, Hongjun; Zhao, Wenjun; Zhu, Shuifang","Sun, Lei; Zhang, Gui-hong; Jiang, Jing-wei; Fu, Jia-dong; Ren, Tao; Cao, Wei-sheng; Xin, Chao-an; Liao, Ming; Liu, Wen-jun","Lowenthal, John","Cauchemez, Simon; Epperson, Scott; Biggerstaff, Matthew; Swerdlow, David; Finelli, Lyn; Ferguson, Neil M.","Tommy Tsan-Yuk Lam; Marcus Ho-Hin Shum; Hua-Chen Zhu; Yi-Gang Tong; Xue-Bing Ni; Yun-Shi Liao; Wei Wei; William Yiu-Man Cheung; Wen-Juan Li; Lian-Feng Li; Gabriel M Leung; Edward C. Holmes; Yan-Ling Hu; Yi Guan","Brisse, Sylvain; Brehony, Carina; Concei\u00e7\u00e3o, Teresa; Cubero, Meritxell; Glasner, Corinna; Le Gouil, Meriadeg; Renvois\u00e9, Aur\u00e9lie; Sheppard, Sam; Weinert, Lucy A.","Ralph, R.; Lew, J.; Zeng, T.; Francis, M.; Xue, B.; Roux, M.; Ostadgavahi, A. T.; Rubino, S.; Dawe, N. J.; Al-Ahdal, M. N.; Kelvin, D. J.; Richardson, C. D.; Kindrachuk, J.; Falzarano, D.; Kelvin, A. A.","Mingwang Shen; Zhihang Peng; Yuming Guo; Yanni Xiao; Lei Zhang","Chambon, M.; Laveran, H.; L'Haridon, R.; Peigue-Lafeuille, H.; Trimolet, M.; Beytout, D.","Yinhua Zhang; Nelson Odiwuor; Jin Xiong; Luo Sun; Raphael Ohuru Nyaruaba; Hongping Wei; Nathan A Tanner","Xiaoqiang Huang; Robin Pearce; Yang Zhang","Valentine, Helen; Daugherity, Erin K.; Singh, Bhupinder; Maurer, Kirk J.","Cao, Yanan; Li, Lin; Feng, Zhimin; Wan, Shengqing; Huang, Peide; Sun, Xiaohui; Wen, Fang; Huang, Xuanlin; Ning, Guang; Wang, Weiqing","Phan, Tung","Jessica E. Manning; Jennifer A. Bohl; Sreyngim Lay; Sophana Chea; Ly Sovann; Yi Sengdoeurn; Seng Heng; Chan Vuthy; Katrina Kalantar; Vida Ahyong; Michelle Tan; Jonathan Sheu; Cristina M. Tato; Joseph L. DeRisi; Laurence Baril; Veasna Duong; Philippe Dussart; Erik A. Karlsson","Crossley, Beate M.; Mock, Richard E.; Callison, Scott A.; Hietala, Sharon K.","Tu, Wenxiao; Jin, Lianmei; Ni, Daxin","Honkavuori, Kirsi S.; Briese, Thomas; Krauss, Scott; Sanchez, Maria D.; Jain, Komal; Hutchison, Stephen K.; Webster, Robert G.; Lipkin, W. Ian","Zhao, Shan; Li, Wentao; Schuurman, Nancy; van Kuppeveld, Frank; Bosch, Berend-Jan; Egberink, Herman","Yonghua Wu"],"color":["#67a9cf","#67a9cf","#67a9cf","#67a9cf","#67a9cf","#67a9cf","#67a9cf","#67a9cf","#67a9cf","#67a9cf","#67a9cf","#67a9cf","#67a9cf","#67a9cf","#67a9cf","#67a9cf","#67a9cf","#67a9cf","#67a9cf","#67a9cf","#67a9cf","#67a9cf","#67a9cf","#67a9cf","#67a9cf","#67a9cf","#67a9cf","#67a9cf","#67a9cf","#67a9cf","#67a9cf","#67a9cf","#67a9cf","#67a9cf","#67a9cf","#67a9cf","#67a9cf","#67a9cf","#67a9cf","#67a9cf","#67a9cf","#67a9cf","#67a9cf","#67a9cf","#67a9cf","#67a9cf","#67a9cf","#67a9cf","#67a9cf","#67a9cf"],"date":["2017-11-28","2014-08-31","2013-02-28","2005-03-01","2008-12-31","2020-02-11","2016-06-03","1995-02-28","2011-11-30","2020-03-25","2013-05-01","2013-10-31","2018-06-30","2010-04-30","2006-12-03","2017-11-13","2020-02-03","2019-08-22","2012-12-31","2015-05-31","2020-03-10","2020-02-13","2020-03-23","2017-12-31","2019-12-31","2019-01-09","2013-01-03","2003-12-31","2010-03-01","2008-05-02","2010-07-31","2007-12-31","2016-06-30","2013-03-05","2020-02-18","2014-03-31","2020","2020-02-13","1987-06-30","2020-02-29","2020-03-31","2012-12-31","2020","2020-07-31","2020-03-05","2012-12-12","2017-12-31","2014-04-03","2019-08-13","2020-03-04"],"doc":["k0f9fser","p1jbb1wa","v8uzd2f3","cisrvghj","01a8er2v","bnv38la3","yzp9wjuk","19z4ej76","bxu3sklz","x8jgcqts","qh2kuzdc","wm6yey1v","wuegn0jg","o976nab9","v08cs51n","th1da1bb","eecyduzq","m1o3g2bz","7icslv2r","1k5h662u","j10308i3","er6lz09f","q0yqnlyl","dgi4ibym","b5l6epy3","tjp33y35","0i3ewil9","xpl6tjps","9ojile3q","6kjjqsxu","j8fqan9x","dwf5vydu","13adnvav","16c8dwfq","5f42du0b","a4mf2vnp","rxrlbw60","tzdgc3dw","olhh6pjg","cv3qgno3","imkeghfd","7yse6kjd","tucolje2","0nh58odf","fu3cl7lq","vexo81k5","bjub9j3c","7gx4fowz","eiobmxp2","4ihv80au"],"journal":["Sci China Life Sci","International Journal of Infectious Diseases","Current Opinion in Virology","Medicine","Encyclopedia of Virology","","Virol J","Veterinary Microbiology","The Veterinary Journal","","Expert Review of Molecular Diagnostics","Antiviral Research","Trends in Microbiology","Journal of Exotic Pet Medicine","Emerg Infect Dis","Nat Rev Genet","","Cell","Trends in Microbiology","Virology","Journal of Infection","","","Early Warning for Infectious Disease Outbreak","Genomic and Precision Medicine","Viruses","Nature Reviews Microbiology","Methods in Enzymology","Virology","Biochemical and Biophysical Research Communications","Journal of Virological Methods","Virus Research","Journal of Infection and Public Health","PLoS Med","","Research in Microbiology","Journal of Infection in Developing Countries","","Annales de l'Institut Pasteur / Virologie","","","The Laboratory Rabbit, Guinea Pig, Hamster, and Other Rodents","Cell Discovery","Infection, Genetics and Evolution","","Viruses","Early Warning for Infectious Disease Outbreak","PLoS One","Viruses",""],"rad":[0.02,0.010102425659302267,0.007917363795514103,0.007891346298589688,0.007065837075167039,0.00705107172774611,0.0068506092806840425,0.0065339572678104075,0.006493918902695276,0.006366045105308774,0.006296373557191873,0.005933566507030478,0.00559807523185964,0.005485952921223582,0.0054732390932844596,0.005385654247689412,0.004989931282619491,0.0049200395720763596,0.004894899041043967,0.004755117350791225,0.004514963348137974,0.004481202230405857,0.00430744121324362,0.0041452908464761715,0.004141768796312594,0.004122885210699272,0.004114921585750784,0.004038869224434363,0.0040317732535997295,0.003945232360095979,0.003859463468765658,0.003851721975579446,0.0038221526164909027,0.0036024670659916807,0.003516177899745339,0.0035115720356622094,0.0034718797291823917,0.003413752389303387,0.0034008373072349275,0.003396722533002109,0.003285287874048789,0.003284313121397665,0.003147750966658331,0.0031272171326000263,0.003085956906747375,0.003061804960703018,0.0030564354668812937,0.003048020616888056,0.0030309298297995202,0.003],"scr":[1.0,0.418,0.289,0.288,0.239,0.238,0.227,0.208,0.206,0.198,0.194,0.173,0.153,0.146,0.145,0.14,0.117,0.113,0.111,0.103,0.089,0.087,0.077,0.067,0.067,0.066,0.066,0.061,0.061,0.056,0.051,0.05,0.048,0.035,0.03,0.03,0.028,0.024,0.024,0.023,0.017,0.017,0.009,0.007,0.005,0.004,0.003,0.003,0.002,0.0],"text":["In the past twenty years, numerous novel zoonotic viral agents with pandemic potential have emerged in China, such as the severe acute respiratory syndrome (SARS) coronavirus and, more recently, the avian-origin influenza A/H7N9 virus, which have caused outbreaks among humans with high morbidity and mortality. In addition, several emerging and re-emerging viral pathogens have also been imported into China from travelers, e.g. the Middle East respiratory syndrome (MERS) coronavirus and Zika virus (ZIKV). Herein, we review these emerging viral pathogens in China and focus on how surveillance by pathogen genomics has been employed to discover and annotate novel pathogenic agents, identify natural reservoirs, monitor the transmission events and delineate their evolution and adaption to the human host. We also highlight the application of genomic sequencing in the recent Ebola epidemics in Western Africa. In summary, genomic sequencing has become a standard research tool in the field of eme...","Summary Emerging and re-emerging zoonoses are a significant public health concern and cause considerable socioeconomic problems globally. The emergence of severe acute respiratory syndrome (SARS), highly pathogenic avian influenza (HPAI) H5N1, avian influenza H7N9, and severe fever with thrombocytopenia syndrome (SFTS), and the re-emergence of rabies, brucellosis, and other zoonoses have had a significant effect on the national economy and public health in China, and have affected other countries. Contributing factors that continue to affect emerging and re-emerging zoonoses in China include social and environmental factors and microbial evolution, such as population growth, urbanization, deforestation, livestock production, food safety, climate change, and pathogen mutation. The Chinese government has devised new strategies and has taken measures to deal with the challenges of these diseases, including the issuing of laws and regulations, establishment of disease reporting systems, im...","Bats are being increasingly recognized as an important reservoir of zoonotic viruses of different families, including SARS coronavirus, Nipah virus, Hendra virus and Ebola virus. Several recent studies hypothesized that bats, an ancient group of flying mammals, are the major reservoir of several important RNA virus families from which other mammalian viruses of livestock and humans were derived. Although this hypothesis needs further investigation, the premise that bats carry a large number of viruses is commonly accepted. The question of whether bats have unique biological features making them ideal reservoir hosts has been the subject of several recent reviews. In this review, we will focus on the public health implications of bat derived zoonotic viral disease outbreaks, examine the drivers and risk factors of past disease outbreaks and outline research directions for better control of future disease events.","Abstract There are many diseases for which a microbial aetiology is suspected. The hypothesis that a disease has an infectious cause is supported by: clinical features (similar to those of known infectious diseases, e.g. fever, leucocytosis), epidemiology (case clustering in time or location), histology (inflammation of affected tissues, e.g. granulomata) or characteristic microbial structures, treatment (clinical response to antimicrobial treatment), and prevention of disease by vaccines targeting microbial antigens. Proof that a microbe causes a disease requires more rigorous evidence. Future attempts to identify novel microbes associated with human disease may use sequence-based approaches. High-throughput sequencing may allow identification of unique microbial nucleic acid sequences in a background of host DNA. The complete sequencing of the human genome and multiple microbial genomes make this approach more feasible. DNA microarrays are also likely to be used in the search for nov...","The study of virus evolution has historically been concerned with disease and its emergence and has not been integrated into the general study of evolution. Yet viruses dominate our planet and their evolution is a broad and applied field that can be studied in the real time (such HIV in human disease) and can also be applied to biotechnological problems. Virus evolution is very similar, but not identical, to host evolution. Darwinian principles apply, such as with Fisher population genetics, but other features, such as reticulated and quasispecies-based evolution, distinguish virus evolution. In RNA viruses, high error rates have led to the quasispecies concept in which collective populations are the basis of evolution. These populations provide viruses with high capacities for adaptation. Yet some RNA viruses show remarkable evolutionary stability. With DNA viruses, especially of bacteria, extensive recombination defines vast dynamic and ancient genetic pools. Temperate viruses can di...","The outbreak of pneumonia caused by the 2019 Novel Coronavirus (2019-nCoV) was reported in Wuhan City, China. However, the clinical symptoms varied in different reports. Based on the results of inter-group difference test, we found that the incidence of diarrhea differed in three recent reports. As 2019-nCoV utilizes the same cell entry receptor ACE2 as severe acute respiratory syndrome coronavirus (SARS-CoV) and ACE2 tightly controls intestinal inflammation, to trace the route of infection mediated by 2019-nCoV, we used the single-cell RNA sequencing data for analysis. We found that the ACE2 mRNA was highly expressed in the healthy human small intestine rather than the lung. Besides, single-cell RNA sequencing data showed that ACE2 was significantly elevated in the proximal and distal enterocytes, where the small intestinal epithelium is exposed to the foreign pathogen. Thus, we suspect that ACE2-expressing small intestinal epithelium cells might be vulnerable to 2019-nCoV infection w...","Middle East Respiratory Syndrome-Coronavirus (MERS-CoV) is a novel coronavirus discovered in 2012 and is responsible for acute respiratory syndrome in humans. Though not confirmed yet, multiple surveillance and phylogenetic studies suggest a bat origin. The disease is heavily endemic in dromedary camel populations of East Africa and the Middle East. It is unclear as to when the virus was introduced to dromedary camels, but data from studies that investigated stored dromedary camel sera and geographical distribution of involved dromedary camel populations suggested that the virus was present in dromedary camels several decades ago. Though bats and alpacas can serve as potential reservoirs for MERS-CoV, dromedary camels seem to be the only animal host responsible for the spill over human infections.","Abstract The sudden appearance of apparently new viruses with pathogenic potential is of fundamental importance in medical microbiology and a constant threat to humans and animals. The emergence of a \u201cnew\u201d pathogen is not an isolated event, as for instance the frequent appearance of new influenza virus strains demonstrates. Often the new virus strains co-circulate with the older strains in a susceptible population, but a replacement of the older strains has been also observed. In rare instances the new viruses can cause dramatic epidemics or pandemics, such as those observed with the human immunodeficiency virus, canine parvovirus, or most recently, with the agent of bovine spongiform encephalopathy in the United Kingdom. The mechanisms of the emergence are not always clearly understood, but an altered host range appears to be a common event. Whether a true change in host range occurs, or whether the virus adapted to the host and replicated more efficiently, is often unknown. This revi...","Abstract There have been rapid recent developments in establishing methods for identifying and characterising viruses associated with animal and human diseases. These methodologies, commonly based on hybridisation or PCR techniques, are combined with advanced sequencing techniques termed \u2018next generation sequencing\u2019. Allied advances in data analysis, including the use of computational transcriptome subtraction, have also impacted the field of viral pathogen discovery. This review details these molecular detection techniques, discusses their application in viral discovery, and provides an overview of some of the novel viruses discovered. The problems encountered in attributing disease causality to a newly identified virus are also considered.","COVID-19 outbreak is still threatening the public health. Therefore, in the middle of the pandemic, all kind of knowledge on SARS-CoV-2 may help us to find the solution. Determining the 3D structures of the proteins involved in host-pathogen interactions are of great importance in the fight against infection. Besides, post-translational modifications of the protein on 3D structure should be revealed in order to understand the protein function since these modifications are responsible for the host-pathogen interaction. Based on these, we predicted O-glycosylation and phosphorylation positions using full amino acid sequence of S1 protein. Candidate positions were further analyzed with enzyme binding activity, solvent accessibility, surface area parameters and the positions determined with high accuracy rate were used to design full 3D glycoprotein structure of the S1 protein using carbohydrate force field. In addition, the interaction between the C-type lectin CD209L and \u03b1-mannose residu...","Mass spectrometry (MS) has found numerous applications in life sciences. It has high accuracy, sensitivity and wide dynamic range in addition to medium- to high-throughput capabilities. These features make MS a superior platform for analysis of various biomolecules including proteins, lipids, nucleic acids and carbohydrates. Until recently, MS was applied for protein detection and characterization. During the last decade, however, MS has successfully been used for molecular diagnostics of microbial and viral infections with the most notable applications being identification of pathogens, genomic sequencing, mutation detection, DNA methylation analysis, tracking of transmissions, and characterization of genetic heterogeneity. These new developments vastly expand the MS application from experimental research to public health and clinical fields. Matching of molecular techniques with specific requirements of the major MS platforms has produced powerful technologies for molecular diagnosti...","Abstract Receptor recognition is a major determinant of the host range, cross-species infections, and pathogenesis of the severe acute respiratory syndrome coronavirus (SARS-CoV). A defined receptor-binding domain (RBD) in the SARS-CoV spike protein specifically recognizes its host receptor, angiotensin-converting enzyme 2 (ACE2). This article reviews the latest knowledge about how RBDs from different SARS-CoV strains interact with ACE2 from several animal species. Detailed research on these RBD/ACE2 interactions has established important principles on host receptor adaptations, cross-species infections, and future evolution of SARS-CoV. These principles may apply to other emerging animal viruses, including the recently emerged Middle East respiratory syndrome coronavirus (MERS-CoV). This paper forms part of a series of invited articles in Antiviral Research on \u201cFrom SARS to MERS: 10years of research on highly pathogenic human coronaviruses\u201d.","Infections with bat-origin coronaviruses have caused severe illness in humans by \u2018host jump\u2019. Recently, novel bat-origin coronaviruses were found in pigs. The large number of mutations on the receptor-binding domain allowed the viruses to infect the new host, posing a potential threat to both agriculture and public health.","Abstract The development of rapid, accurate, and sensitive diagnostic methods for detecting pathogens is the basis for treating, controlling, and eradicating infectious diseases of veterinary importance. Scientific and technological advancements have revolutionized the field of veterinary diagnostics. Genome sequencing has allowed efficient, sensitive, and specific diagnostic assays to be developed based on the detection of nucleic acids. The integration of advances in biochemistry, proteomics, engineering, and medicine offers enormous potential for the rapid and accurate diagnosis of viral, microbial, genetic, and metabolic disease. In the future, polymerase chain reaction assays, microarray testing, genomic analysis, and metabolic profiling will be accomplished in a rapid, portable, sensitive, and cost-efficient manner.","Bats have been identified as a natural reservoir for an increasing number of emerging zoonotic viruses, including henipaviruses and variants of rabies viruses. Recently, we and another group independently identified several horseshoe bat species (genus Rhinolophus) as the reservoir host for a large number of viruses that have a close genetic relationship with the coronavirus associated with severe acute respiratory syndrome (SARS). Our current research focused on the identification of the reservoir species for the progenitor virus of the SARS coronaviruses responsible for outbreaks during 2002\u20132003 and 2003\u20132004. In addition to SARS-like coronaviruses, many other novel bat coronaviruses, which belong to groups 1 and 2 of the 3 existing coronavirus groups, have been detected by PCR. The discovery of bat SARS-like coronaviruses and the great genetic diversity of coronaviruses in bats have shed new light on the origin and transmission of SARS coronaviruses.","The recent Ebola and Zika epidemics demonstrate the need for the continuous surveillance, rapid diagnosis and real-time tracking of emerging infectious diseases. Fast, affordable sequencing of pathogen genomes \u2014 now a staple of the public health microbiology laboratory in well-resourced settings \u2014 can affect each of these areas. Coupling genomic diagnostics and epidemiology to innovative digital disease detection platforms raises the possibility of an open, global, digital pathogen surveillance system. When informed by a One Health approach, in which human, animal and environmental health are considered together, such a genomics-based system has profound potential to improve public health in settings lacking robust laboratory capacity. SUPPLEMENTARY INFORMATION: The online version of this article (doi:10.1038/nrg.2017.88) contains supplementary material, which is available to authorized users.","AbstractThe outbreaks of 2002/2003 SARS, 2012/2015 MERS and 2019/2020 Wuhan respiratory syndrome clearly indicate that genome evolution of an animal coronavirus (CoV) may enable it to acquire human transmission ability, and thereby to cause serious threats to global public health. It is widely accepted that CoV human transmission is driven by the interactions of its spike protein (S-protein) with human receptor on host cell surface; so, quantitative evaluation of these interactions may be used to assess the human transmission capability of CoVs. However, quantitative methods directly using viral genome data are still lacking. Here, we perform large-scale protein-protein docking to quantify the interactions of 2019-nCoV S-protein receptor-binding domain (S-RBD) with human receptor ACE2, based on experimental SARS-CoV S-RBD-ACE2 complex structure. By sampling a large number of thermodynamically probable binding conformations with Monte Carlo algorithm, this approach successfully identifi...","Summary The Zika epidemic in the Americas has challenged surveillance and control. As the epidemic appears to be waning, it is unclear whether transmission is still ongoing, which is exacerbated by discrepancies in reporting. To uncover locations with lingering outbreaks, we investigated travel-associated Zika cases to identify transmission not captured by reporting. We uncovered an unreported outbreak in Cuba during 2017, a year after peak transmission in neighboring islands. By sequencing Zika virus, we show that the establishment of the virus was delayed by a year and that the ensuing outbreak was sparked by long-lived lineages of Zika virus from other Caribbean islands. Our data suggest that, although mosquito control in Cuba may initially have been effective at mitigating Zika virus transmission, such measures need to be maintained to be effective. Our study highlights how Zika virus may still be \u201csilently\u201d spreading and provides a framework for understanding outbreak dynamics. Vi...","The capacity to injure infected cells is a widespread property of viruses. Usually, this cytopathic effect (CPE) is ascribed to viral hijacking of cellular resources to fulfill viral needs. However, evidence is accumulating that CPE is not necessarily directly coupled to viral reproduction but may largely be due to host defensive and viral antidefensive activities. A major part in this virus\u2013cell interaction appears to be played by a putative host-encoded program with multiple competing branches, leading to necrotic, apoptotic, and, possibly, other types of cell suicide. Manifestations of this program are controlled and modulated by host, viral, and environmental factors.","Abstract Two novel coronaviruses have emerged to cause severe disease in humans. While bats may be the primary reservoir for both viruses, SARS coronavirus (SARS-CoV) likely crossed into humans from civets in China, and MERS coronavirus (MERS-CoV) has been transmitted from camels in the Middle East. Unlike SARS-CoV that resolved within a year, continued introductions of MERS-CoV present an on-going public health threat. Animal models are needed to evaluate countermeasures against emerging viruses. With SARS-CoV, several animal species were permissive to infection. In contrast, most laboratory animals are refractory or only semi-permissive to infection with MERS-CoV. This host-range restriction is largely determined by sequence heterogeneity in the MERS-CoV receptor. We describe animal models developed to study coronaviruses, with a focus on host-range restriction at the level of the viral receptor and discuss approaches to consider in developing a model to evaluate countermeasures agai...","Summary Objectives Understanding the novel coronavirus (COVID-19) mode of host cell recognition may help to fight the disease and save lives. The spike protein of coronaviruses is the main driving force for host cell recognition. Methods In this study, the COVID-19 spike binding site to the cell-surface receptor (Glucose Regulated Protein 78 (GRP78)) is predicted using combined molecular modeling docking and structural bioinformatics. The COVID-19 spike protein is modeled using its counterpart, the SARS spike. Results Sequence and structural alignments show that four regions, in addition to its cyclic nature have sequence and physicochemical similarities to the cyclic Pep42. Protein-protein docking was performed to test the four regions of the spike that fit tightly in the GRP78 Substrate Binding Domain \u03b2 (SBD\u03b2). The docking pose revealed the involvement of the SBD\u03b2 of GRP78 and the receptor-binding domain of the coronavirus spike protein in recognition of the host cell receptor. Concl...","AbstractMotivationAccurately determining the atomic structure of proteins represents a fundamental problem in the field of structural bioinformatics. A solution would be significant as protein structure information could be utilized in the medical field, e.g. in the development of vaccines for new viruses. This paper focuses on predicting the protein structure based on 3D images of the proteins captured through cryogenic electron microscopes (cryo-EM). A fully automated computationally efficient protein structure prediction method would be particularly beneficial in the field of cryo-EM as the technology allows researchers to photograph multiple large protein complexes in a single study, which means that a fast prediction method could allow for a high throughput of derived protein structures. We present a deep learning approach, DeepTracer, for predicting locations of the backbone atoms, secondary structure elements, and the amino acid types. In order to connect the predicted amino aci...","The emergence of SARS-CoV-2 has resulted in more than 200,000 infections and nearly 9,000 deaths globally so far. This novel virus is thought to have originated from an animal reservoir, and acquired the ability to infect human cells using the SARS-CoV cell receptor hACE2. In the wake of a global pandemic it is essential to improve our understanding of the evolutionary dynamics surrounding the origin and spread of a novel infectious disease. One way theory predicts selection pressures should shape viral evolution is to enhance binding with host cells. We first assessed evolutionary dynamics in select betacoronavirus spike protein genes to predict where these genomic regions are under directional or purifying selection between divergent viral lineages at various scales of relatedness. With this analysis, we determine a region inside the receptor-binding domain with putative sites under positive selection interspersed among highly conserved sites, which are implicated in structural stabi...","Abstract This chapter provides an overview of the basic theory of early warning for infectious diseases, including definitions of surveillance, early warning and prediction; the characteristics of early warning (surveillance, information for action, timeliness, insufficiency, and the uncertainty of information); a conceptual framework of an early warning system for infectious diseases (the target setting, data collection and analysis, disseminating early warning signals, response actions, and evaluation); classifying early warning based on the data sources (case-based, event-based, laboratory-based, or syndromic surveillance). The significance of early warning for infectious diseases, the status quo and research and application trends in the field are also highlighted.","Abstract Molecular analysis of respiratory viruses and the host response to both infection and vaccination have transformed our understanding of these ubiquitous pathogens. Polymerase chain reaction for the rapid and accurate diagnosis of viral infections has led to a better understanding of the epidemiology and impact of many common respiratory viruses and resulted in better patient care. Over the past decade a number of new respiratory viruses including human metapneumovirus and new coronaviruses have been discovered using molecular techniques such as random primer amplification, pan-viral array and next generation sequencing. Analysis of the host transcriptional response during respiratory viral infection using in-vitro, animal models and natural and experimental human challenge have furthered the understanding of the mechanisms and predictors of severe disease and may identify potential therapeutic targets to prevent and ameliorate illness.","Known human coronaviruses are believed to have originated in animals and made use of intermediate hosts for transmission to humans. The intermediate hosts of most of the human coronaviruses are known, but not for HCoV-NL63. This study aims to assess the possible role of some major domestic livestock species as intermediate hosts of HCoV-NL63. We developed a testing algorithm for high throughput screening of livestock sera with ELISA and confirmation with recombinant immunofluorescence assay testing for antibodies against HCoV-NL63 in livestock. Optimization of the ELISA showed a capability of the assay to significantly distinguish HCoV-NL63 from HCoV-229E (U = 27.50, p &lt; 0.001) and HCoV-OC43 (U = 55.50, p &lt; 0.001) in coronavirus-characterized sera. Evaluation of the assay with collected human samples showed no significant difference in mean optical density values of immunofluorescence-classified HCoV-NL63-positive and HCoV-NL63-negative samples (F (1, 215) = 0.437, p = 0.509). All the ...","The pace of pathogen discovery is increasing dramatically. This reflects not only factors that enable the appearance and globalization of new microbial infections but also improvements in methods for ascertainment. New molecular diagnostic platforms; investments in pathogen surveillance in wildlife, domestic animals and humans; and the advent of social media tools that mine the world wide web for clues to outbreaks of infectious disease are proving invaluable in early recognition of threats to public health. Additionally, models of microbial pathogenesis are becoming more complex, providing insights into the mechanisms by which microorganisms can contribute to chronic illnesses like cancer, peptic ulcer disease and mental illness. Here we review the contributions of each of these elements to infectious disease emergence and strategies for addressing the challenges of pathogen surveillance and discovery.","Publisher Summary This chapter describes three assays to monitor receptor-induced association of the envelope glycoprotein (EnvA) of avian sarcoma/leukosis virus (ASLV) with target bilayers: (1) the original assay for monitoring binding of the EnvA ectodomain (EnvA-PI) to target membranes (liposomes), (2) a modified and miniaturized EnvA-PI-liposome binding assay, and (3) an assay to measure binding of intact sarcoma/leukosis virus subtype A (ASLV-A) virus particles to target membranes. These assays are also useful for studying other receptor-activated viral fusion proteins. When one viral glycoprotein and one \u201csimple\u201d host cell receptor are involved, it should be possible to develop assays directly analogous to those described above for studying Tva-induced binding of the EnvA ectodomain (EnvA-PI) to target membranes. A general prerequisite for a fusion protein/target membrane binding assay is a soluble and correctly oligomeric form of the viral fusion protein ectodomain. The simplest...","Abstract Analyses of turkey coronavirus (TCoV), an enteric disease virus that is highly similar to infectious bronchitis virus (IBV) an upper-respiratory tract disease virus in chickens, were conducted to determine the adaptive potential, and genetic changes associated with emergence of this group 3 coronavirus. Strains of TCoV that were pathogenic in poults and nonpathogenic in chickens did not adapt to cause disease in chickens. Comparative genomics revealed two recombination sites that replaced the spike gene in IBV with an unidentified sequence likely from another coronavirus, resulting in cross-species transmission and a pathogenicity shift. Following emergence in turkeys, TCoV diverged to different serotypes through the accumulation of mutations within spike. This is the first evidence that recombination can directly lead to the emergence of new coronaviruses and new coronaviral diseases, emphasizing the importance of limiting exposure to reservoirs of coronaviruses that can serv...","Abstract Lipid rafts often serve as an entry site for certain viruses. Here, we report that lipid rafts in Vero E6 cells are involved in the entry of severe acute respiratory syndrome coronavirus (SARS-CoV). Infectivity assay showed the integrity of lipid rafts was required for productive infection of pseudotyped SARS-CoV. Depletion of plasma membrane cholesterol with M\u03b2CD relocalized raft-resident marker caveolin-1 as well as SARS-CoV receptor ACE2 to a nonraft environment, but did not significantly change the surface expression of ACE2. M\u03b2CD-treatment inhibited infectivity of pseudotyped SARS-CoV by 90%. Biochemical fractionation and confocal imaging confirmed that ACE2 colocalized with raft-resident markers. Furthermore, an ectodomain of SARS-CoV S protein (S1188HA) could associate with lipid rafts after binding to its receptor, and colocalize with raft-resident marker ganglioside GM1. The binding of S1188HA was not affected by depleting plasma membrane cholesterol. Taken together, ...","Abstract A major challenge facing agriculture at present is the development of techniques that can screen field samples and other plant materials simultaneously for the presence of many viruses. Microarray techniques show promise in this regard, as their high throughput nature can potentially detect a range of viruses using a single test. In this paper we present an array that can detect a wide spectrum of 169 plant virus species from 13 different genera. The array was constructed using an automated probe design protocol which generated a minimal number of probes to detect viruses at the genus level. The designed arrays showed a high specificity and sensitivity when tested with a set of standard virus samples. Field samples collected from a severe disease outbreak of Panax notoginseng farms in Yunnan, China, in 2001 were screened, where a potyvirus infection was identified associated with the disease.","Abstract Coronavirus infection was investigated in apparently healthy wild peafowls in Guangdong province of China in 2003, while severe acute respiratory syndrome (SARS) broke out there. No SARS-like coronavirus had been isolated but a novel avian coronavirus strain, Peafowl/GD/KQ6/2003 (KQ6), was identified. Sequence analysis revealed that KQ6 was an avian coronavirus infectious bronchitis virus (IBV), a member of coronavirus in group 3. The genome sequence of KQ6 had extremely high degree of identity with that of a Massachusetts prototype IBV M41. KQ6 was pathogenic to chickens but non-pathogenic to peafowls under experimental conditions. Seventeen out of fifty-four (31.48%) peafowl serum samples were tested positive for specific antibodies against IBV. Present results indicate that the peafowl isolate KQ6 is a Massachusetts prototype like coronavirus strain which undergoes few genetic changes and peafowl might have acted as a natural reservoir of IBV for very long time.","Summary Emerging infectious diseases arising from livestock and wildlife pose serious threats to global human health, as shown by a series of continuous outbreaks involving highly pathogenic influenza, SARS, Ebola and MERS. The risk of pandemics and bioterrorism threats is ever present and growing, but our ability to combat them is limited by the lack of available vaccines, therapeutics and rapid diagnostics. The use of high bio-containment facilities, such as the CSIRO Australian Animal Health Laboratory, plays a key role studying these dangerous pathogens and facilitates the development of countermeasures. To combat diseases like MERS, we must take a holistic approach that involves the development of early biomarkers of infection, a suite of treatment options (vaccines, anti-viral drugs and antibody therapeutics) and appropriate animal models to test the safety and efficacy of candidate treatments.","BACKGROUND: Prior to emergence in human populations, zoonoses such as SARS cause occasional infections in human populations exposed to reservoir species. The risk of widespread epidemics in humans can be assessed by monitoring the reproduction number R (average number of persons infected by a human case). However, until now, estimating R required detailed outbreak investigations of human clusters, for which resources and expertise are not always available. Additionally, existing methods do not correct for important selection and under-ascertainment biases. Here, we present simple estimation methods that overcome many of these limitations. METHODS AND FINDINGS: Our approach is based on a parsimonious mathematical model of disease transmission and only requires data collected through routine surveillance and standard case investigations. We apply it to assess the transmissibility of swine-origin influenza A H3N2v-M virus in the US, Nipah virus in Malaysia and Bangladesh, and also present...","AbstractThe ongoing outbreak of viral pneumonia in China and beyond is associated with a novel coronavirus, provisionally termed 2019-nCoV. This outbreak has been tentatively associated with a seafood market in Wuhan, China, where the sale of wild animals may be the source of zoonotic infection. Although bats are likely reservoir hosts for 2019-nCoV, the identity of any intermediate host facilitating transfer to humans is unknown. Here, we report the identification of 2019-nCoV related coronaviruses in pangolins (Manis javanica) seized in anti-smuggling operations in southern China. Metagenomic sequencing identified pangolin associated CoVs that belong to two sub-lineages of 2019-nCoV related coronaviruses, including one very closely related to 2019-nCoV in the receptor-binding domain. The discovery of multiple lineages of pangolin coronavirus and their similarity to 2019-nCoV suggests that pangolins should be considered as possible intermediate hosts for this novel human virus and sho...","","On 31 December 2019 the Wuhan Health Commission reported a cluster of atypical pneumonia cases that was linked to a wet market in the city of Wuhan, China. The first patients began experiencing symptoms of illness in mid-December 2019. Clinical isolates were found to contain a novel coronavirus with similarity to bat coronaviruses. As of 28 January 2020, there are in excess of 4,500 laboratory-confirmed cases, with &gt; 100 known deaths. As with the SARS-CoV, infections in children appear to be rare. Travel-related cases have been confirmed in multiple countries and regions outside mainland China including Germany, France, Thailand, Japan, South Korea, Vietnam, Canada, and the United States, as well as Hong Kong and Taiwan. Domestically in China, the virus has also been noted in several cities and provinces with cases in all but one provinence. While zoonotic transmission appears to be the original source of infections, the most alarming development is that human-to-human transmission is ...","We present a timely evaluation of the impact of lockdown on the 2019-nCov epidemic in Hubei province, China. The implementation appears to be effective in reducing about 60% of new infections and deaths, and its effect also appears to be sustainable even after its removal. Delaying its implementation reduces its effectiveness. However, the direct economic cost of such a lockdown remains to be seen and whether the model is replicable in other Chinese regions remains a matter of further investigation.","Summary Four-hundred seventy-two sera were examined by the indirect immunofluorescence test against Coronavirus strain OC43 adapted to the HRT18 cell culture. Sera were from children and adults hospitalized over a 4-year period, from 1979 to 1982, at the CHRU of Clermont-Ferrand (France) for different clinical reasons. Of the 446 sera tested, 61 % showed antibodies against OC43; 90 % of adult sera were positive. Coronavirus infections are very common in France. Immunity appears early in life, and infection is frequent in children from 12\u201336 months of age.","The ability to detect an infectious agent in a widespread epidemic is crucial to the success of quarantine efforts in addition to sensitive and accurate screening of potential cases of infection from patients in a clinical setting. Enabling testing outside of sophisticated laboratories broadens the scope of control and surveillance efforts, but also requires robust and simple methods that can be used without expensive instrumentation. Here we report a method to identify SARS-CoV-2 (COVID-19) virus RNA from purified RNA or cell lysis using loop-mediated isothermal amplification (LAMP) using a visual, colorimetric detection. This test was additionally verified using RNA samples purified from respiratory swabs collected from COVID-19 patients in Wuhan, China with equivalent performance to a commercial RT-qPCR test while requiring only heating and visual inspection. This simple and sensitive method provides an opportunity to facilitate virus detection in the field without a requirement for...","The outbreak of COVID-19 has now become a global pandemic and it continues to spread rapidly worldwide, severely threatening lives and economic stability. Making the problem worse, there is no specific antiviral drug that can be used to treat COVID-19 to date. SARS-CoV-2 initiates its entry into human cells by binding to angiotensin-converting enzyme 2 (hACE2) via the receptor binding domain (RBD) of its spike protein. Therefore, molecules that can block SARS-CoV-2 from binding to hACE2 may potentially prevent the virus from entering human cells and serve as an effective antiviral drug. Based on this idea, we designed a series of peptides that can strongly bind to SARS-CoV-2 RBD in computational experiments. Specifically, we first constructed a 31-mer peptidic scaffold by linking two fragments grafted from hACE2 (a.a. 22-44 and 351-357) with a linker glycine, and then redesigned the peptide sequence to enhance its binding affinity to SARS-CoV-2 RBD. Compare with several computational s...","The Syrian hamster (Mesocricetus auratus) possesses a variety of unique features which make them desirable research models. Despite these features, experimental use of the hamster has declined over the past 20\u201330 years. The current chapter will focus on research and literature describing the experimental use of the hamster with a particular emphasis on research done since the previous edition of this text (1987\u2013present).","","Abstract COVID-19 is a viral respiratory illness caused by a new coronavirus called SARS-CoV-2. The World Health Organization declared the SARS-CoV-2 outbreak a global public health emergency. We performed genetic analyses of eighty-six complete or near-complete genomes of SARS-CoV-2 and revealed many mutations and deletions on coding and non-coding regions. These observations provided evidence of the genetic diversity and rapid evolution of this novel coronavirus.","AbstractRapid production and publication of pathogen genome sequences during emerging disease outbreaks provide crucial public health information. In resource-limited settings, especially near an outbreak epicenter, conventional deep sequencing or bioinformatics are often challenging. Here we successfully used metagenomic next generation sequencing on an iSeq100 Illumina platform paired with an open-source bioinformatics pipeline to quickly characterize Cambodia\u2019s first case of COVID-2019.","In 2007, a novel coronavirus associated with an acute respiratory disease in alpacas (Alpaca Coronavirus, ACoV) was isolated. Full-length genomic sequencing of the ACoV demonstrated the genome to be consistent with other Alphacoronaviruses. A putative additional open-reading frame was identified between the nucleocapsid gene and 3'UTR. The ACoV was genetically most similar to the common human coronavirus (HCoV) 229E with 92.2% nucleotide identity over the entire genome. A comparison of spike gene sequences from ACoV and from HCoV-229E isolates recovered over a span of five decades showed the ACoV to be most similar to viruses isolated in the 1960\u2019s to early 1980\u2019s. The true origin of the ACoV is unknown, however a common ancestor between the ACoV and HCoV-229E appears to have existed prior to the 1960\u2019s, suggesting virus transmission, either as a zoonosis or anthroponosis, has occurred between alpacas and humans.","Abstract On the Internet, there is a lot of open access information related to infectious diseases, e.g., official announcement, media news, bulletin boards, blogs, and other social media. If proper data retrieval and screening methods were applied on these Internet-based data, it would be very useful to improve the timeliness of early detection for infectious diseases. In China, we practice early warning using Internet-based data with routine intelligence screening assessment every day, and respond to any unusual event in a timely manner. Through this mechanism, the surveillance and early warning of Internet-based data for infectious disease and public health emergency plays an important supplemental role to the traditional surveillance systems.","BACKGROUND: Wild birds are an important but to some extent under-studied reservoir for emerging pathogens. We used unbiased sequencing methods for virus discovery in shorebird samples from the Delaware Bay, USA; an important feeding ground for thousands of migratory birds. FINDINGS: Analysis of shorebird fecal samples indicated the presence of a novel astrovirus and coronavirus. A sanderling sample yielded sequences with distant homology to avian nephritis virus 1, an astrovirus associated with acute nephritis in poultry. A ruddy turnstone sample yielded sequences with homology to deltacoronaviruses. CONCLUSIONS: Our findings highlight shorebirds as a virus reservoir and the need to closely monitor wild bird populations for the emergence of novel virus variants.","Coronaviruses (CoVs) are widespread among mammals and birds and known for their potential for cross-species transmission. In cats, infections with feline coronaviruses (FCoVs) are common. Several non-feline coronaviruses have been reported to infect feline cells as well as cats after experimental infection, supported by their ability to engage the feline receptor ortholog for cell entry. However, whether cats might become naturally infected with CoVs of other species is unknown. We analyzed coronavirus infections in cats by serological monitoring. In total 137 cat serum samples and 25 FCoV type 1 or type 2-specific antisera were screened for the presence of antibodies against the S1 receptor binding subunit of the CoV spike protein, which is immunogenic and possesses low amino acid sequence identity among coronavirus species. Seventy-eight sera were positive for antibodies that recognized one or more coronavirus S1s whereas 1 serum exclusively reacted with human coronavirus 229E (HCoV-...","AbstractCoronavirus Disease 2019 (COVID-19) and severe acute respiratory syndrome (SARS)-related coronaviruses (e.g., 2019-nCoV and SARS-CoV) are phylogenetically distantly related, but both are capable of infecting human hosts via the same receptor, angiotensin-converting enzyme 2, and cause similar clinical and pathological features, suggesting their phenotypic convergence. Yet, the molecular basis that underlies their phenotypic convergence remains unknown. Here, we used a recently developed molecular phyloecological approach to examine the molecular basis leading to their phenotypic convergence. Our genome-level analyses show that the spike protein, which is responsible for receptor binding, has undergone significant Darwinian selection along the branches related to 2019-nCoV and SARS-CoV. Further examination shows an unusually high proportion of evolutionary convergent amino acid sites in the receptor binding domain (RBD) of the spike protein between COVID-19 and SARS-related CoV ..."],"title":["Pathogen genomic surveillance elucidates the origins, transmission and evolution of emerging viral agents in China","Major emerging and re-emerging zoonoses in China: a matter of global health and socioeconomic development for 1.3 billion","Bats and their virome: an important source of emerging viruses capable of infecting humans","The infectious aetiology of disease: the search for new agents","Evolution of Viruses","Diarrhea may be underestimated: a missing link in 2019 novel coronavirus","Middle East Respiratory Syndrome Coronavirus (MERS-CoV) origin and animal reservoir","There is nothing permanent except change. The emergence of new virus diseases","Metagenomics and the molecular identification of novel viruses","Glycoinformatics approach for identifying target positions to inhibit initial binding of SARS-CoV-2 S1 protein to the host cell","Application of mass spectrometry to molecular diagnostics of viral infections","Receptor recognition and cross-species infections of SARS coronavirus","Bat-Origin Coronaviruses Expand Their Host Range to Pigs","Future Veterinary Diagnostics","Review of Bats and SARS","Towards a genomics-informed, real-time, global pathogen surveillance system","Fast assessment of human receptor-binding capability of 2019 novel coronavirus (2019-nCoV)","Travel Surveillance and Genomics Uncover a Hidden Zika Outbreak during the Waning Epidemic","Cytopathic effects: virus-modulated manifestations of innate immunity?","Development of animal models against emerging coronaviruses: From SARS to MERS coronavirus","COVID-19 spike-host cell receptor GRP78 binding site prediction","DeepTracer: Predicting Backbone Atomic Structure from High Resolution Cryo-EM Density Maps of Protein Complexes","SARS-CoV-2, an evolutionary perspective of interaction with human ACE2 reveals undiscovered amino acids necessary for complex stability","Chapter 1 Introduction","Chapter 9 Respiratory viral infections","Development of a Whole-Virus ELISA for Serological Evaluation of Domestic Livestock as Possible Hosts of Human Coronavirus NL63","The changing face of pathogen discovery and surveillance","Receptor-Activated Binding of Viral Fusion Proteins to Target Membranes","Emergence of a group 3 coronavirus through recombination","Lipid rafts are involved in SARS-CoV entry into Vero E6 cells","Oligonucleotide microarray with a minimal number of probes for the detection and identification of thirteen genera of plant viruses","A Massachusetts prototype like coronavirus isolated from wild peafowls is pathogenic to chickens","Overview of the CSIRO Australian Animal Health Laboratory","Using Routine Surveillance Data to Estimate the Epidemic Potential of Emerging Zoonoses: Application to the Emergence of US Swine Origin Influenza A H3N2v Virus","Identification of 2019-nCoV related coronaviruses in Malayan pangolins in southern China","Microbial molecular markers and epidemiological surveillance in the era of high throughput sequencing: an update from the IMMEM-10 conference","2019-nCoV (Wuhan virus), a novel Coronavirus: Human-to-human transmission, travel-related cases, and vaccine readiness","Lockdown may partially halt the spread of 2019 novel coronavirus in Hubei province, China","Enqu\u00eate s\u00e9rologique par immunofluorescence indirecte, sur les infections \u00e0 coronavirus OC43","Rapid Molecular Detection of SARS-CoV-2 (COVID-19) Virus RNA Using Colorimetric LAMP","Computational Design of Peptides to Block Binding of the SARS-CoV-2 Spike Protein to Human ACE2","Chapter 34 The Experimental Use of Syrian Hamsters","Comparative genetic analysis of the novel coronavirus (2019-nCoV/SARS-CoV-2) receptor ACE2 in different populations","Genetic diversity and evolution of SARS-CoV-2","Rapid metagenomic characterization of a case of imported COVID-19 in Cambodia","Identification and Characterization of a Novel Alpaca Respiratory Coronavirus Most Closely Related to the Human Coronavirus 229E","Chapter 12 Early Warning Practice Using Internet-Based Data","Novel Coronavirus and Astrovirus in Delaware Bay Shorebirds","Serological Screening for Coronavirus Infections in Cats","Strong evolutionary convergence of receptor-binding protein spike between COVID-19 and SARS-related coronaviruses"],"url":["https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7088571/","https://doi.org/10.1016/j.ijid.2014.04.003","https://doi.org/10.1016/j.coviro.2012.11.006","https://doi.org/10.1383/medc.33.3.37.61122","https://doi.org/10.1016/b978-012374410-4.00706-8","https://doi.org/10.1101/2020.02.03.20020289","https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4891877/","https://doi.org/10.1016/0378-1135(95)92531-f","https://doi.org/10.1016/j.tvjl.2010.10.014","https://doi.org/10.1101/2020.03.25.007898","http://europepmc.org/articles/pmc5831079?pdf=render","https://doi.org/10.1016/j.antiviral.2013.08.014","https://doi.org/10.1016/j.tim.2018.03.001","https://doi.org/10.1053/j.jepm.2010.05.006","https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3291347/","https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7097748/","https://doi.org/10.1101/2020.02.01.930537","https://doi.org/10.1016/j.cell.2019.07.018","https://doi.org/10.1016/j.tim.2012.09.003","https://doi.org/10.1016/j.virol.2015.02.030","https://doi.org/10.1016/j.jinf.2020.02.026","https://doi.org/10.1101/2020.02.12.946772","https://doi.org/10.1101/2020.03.21.001933","https://doi.org/10.1016/b978-0-12-812343-0.00001-1","https://doi.org/10.1016/b978-0-12-801496-7.00009-5","https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6356407/","http://europepmc.org/articles/pmc4098826?pdf=render","https://doi.org/10.1016/s0076-6879(03)72026-6","https://doi.org/10.1016/j.virol.2009.11.044","https://doi.org/10.1016/j.bbrc.2008.02.023","https://doi.org/10.1016/j.jviromet.2010.03.010","https://doi.org/10.1016/j.virusres.2007.06.003","https://doi.org/10.1016/j.jiph.2016.04.007","https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3589342/","https://doi.org/10.1101/2020.02.13.945485","https://doi.org/10.1016/j.resmic.2014.01.003","https://doi.org/10.3855/jidc.12425","https://doi.org/10.1101/2020.02.11.20022236","https://doi.org/10.1016/s0769-2617(87)80010-2","https://doi.org/10.1101/2020.02.26.20028373","https://doi.org/10.1101/2020.03.28.013607","https://doi.org/10.1016/b978-0-12-380920-9.00034-1","https://doi.org/10.1038/s41421-020-0147-1","https://doi.org/10.1016/j.meegid.2020.104260","https://doi.org/10.1101/2020.03.02.968818","https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3528286/","https://doi.org/10.1016/b978-0-12-812343-0.00012-6","https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3974748/","https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6723642/","https://doi.org/10.1101/2020.03.04.975995"],"x":[0.6214775569681427,0.6626559016115418,0.5023235942363486,0.27881110516815705,0.49401776517833734,0.5161050147597874,0.4617019570972068,0.5037305370076611,0.29342847606142736,0.2868492959482408,0.27469082809158685,0.4134494533746387,0.4962803499487116,0.2730343103114578,0.46229133699879593,0.9355116218871715,0.3087385052365971,0.663329260943974,0.3702625474189634,0.4622119508404131,0.2902518264688611,0.2846437830385081,0.3417597390957836,0.9228727733850561,0.5935581677397966,0.38936994121288526,0.6645972563227168,0.2452622033738497,0.4836253043070023,0.3238422082154111,0.30491162839196595,0.421226984048368,0.6565056719025264,0.799048328705212,0.3811608806972735,0.26466857417286255,0.6081551909044011,0.7795166822540117,0.44424386153987616,0.3471103503194878,0.2991610962460405,0.7510326216876039,0.4025593483585243,0.4130746423418677,0.3325246459869446,0.3367027586682832,0.9231020047000812,0.4510592259216946,0.3221552703626108,0.3210243670546203],"y":[0.38168706983498,0.4338016431777134,0.3971169552606186,0.5496438328773356,0.3603293304354969,0.15479614828561214,0.42633028494897846,0.3870681505017016,0.5301040749314184,0.3045906055441385,0.5166303218328695,0.3527528705803271,0.4024507395587627,0.5295894358685618,0.3870927300606332,0.6669917806915412,0.29215719064518575,0.5532446236580655,0.3028279426508974,0.36206722640342326,0.26302795753833896,0.31502900504375747,0.31674178845906026,0.6949220872367693,0.16915455267203744,0.5222384341642189,0.4152191155258915,0.13690281331170115,0.3750658356355144,0.162156527404571,0.5397391547062033,0.41319306370376985,0.42575463084202014,0.8438172404286156,0.4140974548587086,0.5214583624236317,0.5904488112494339,0.7578639581079327,0.48341914834957134,0.563147946606329,0.29170367304326006,0.6193729873121305,0.38155164759810484,0.3691952466494706,0.50204336047393,0.4027406608408697,0.6992217182450544,0.4017128689125844,0.11245639365771701,0.30981928275857445]},"selected":{"id":"9445"},"selection_policy":{"id":"9467"}},"id":"9373","type":"ColumnDataSource"},{"attributes":{},"id":"9420","type":"BasicTicker"},{"attributes":{"source":{"id":"9374"}},"id":"9438","type":"CDSView"},{"attributes":{"formatter":{"id":"9470"},"ticker":{"id":"9389"}},"id":"9388","type":"LinearAxis"},{"attributes":{"children":[{"id":"9443"},{"id":"9410"}]},"id":"9456","type":"Row"},{"attributes":{"children":[{"id":"9449"}]},"id":"9452","type":"Row"},{"attributes":{"data_source":{"id":"9373"},"glyph":{"id":"9400"},"hover_glyph":null,"muted_glyph":null,"nonselection_glyph":{"id":"9401"},"selection_glyph":null,"view":{"id":"9403"}},"id":"9402","type":"GlyphRenderer"},{"attributes":{"fill_alpha":{"value":0.1},"fill_color":{"field":"color"},"line_alpha":{"value":0.1},"line_color":{"field":"color"},"radius":{"field":"rad","units":"data"},"x":{"field":"x"},"y":{"field":"y"}},"id":"9436","type":"Circle"},{"attributes":{"fill_alpha":{"value":0.1},"fill_color":{"field":"color"},"line_alpha":{"value":0.1},"line_color":{"field":"color"},"radius":{"field":"rad","units":"data"},"x":{"field":"x"},"y":{"field":"y"}},"id":"9401","type":"Circle"},{"attributes":{},"id":"9468","type":"BasicTickFormatter"},{"attributes":{},"id":"9411","type":"Range1d"},{"attributes":{"source":{"id":"9373"}},"id":"9403","type":"CDSView"},{"attributes":{},"id":"9474","type":"StringFormatter"},{"attributes":{"text":"&lt;h3&gt;Document ranking&lt;/h3&gt;"},"id":"9450","type":"Div"},{"attributes":{"callback":null},"id":"9392","type":"TapTool"},{"attributes":{"editor":{"id":"9475"},"field":"text","formatter":{"id":"9474"},"title":"Passage","width":770},"id":"9442","type":"TableColumn"},{"attributes":{},"id":"9389","type":"BasicTicker"},{"attributes":{"axis":{"id":"9388"},"dimension":1,"ticker":null},"id":"9391","type":"Grid"},{"attributes":{"args":{"sd":{"id":"9373"},"sp":{"id":"9374"}},"code":"\n    if (sd.selected.indices.length == 0) {\n      return;\n    }\n    var sel = sd.selected.indices[0];\n    var new_selected = [];\n    for (var i = 0; i &lt; sp.data['doc'].length; i++) {\n      if (sp.data['doc'][i] == sd.data['doc'][sel]) {\n        new_selected.push(i);\n      }\n    }\n    if (JSON.stringify(new_selected) != JSON.stringify(sp.selected.indices) &amp;&amp;\n    !(sp.selected.indices.length == 1 &amp;&amp; new_selected.includes(sp.selected.indices[0]))) {\n      sp.selected.indices = new_selected;\n      sp.change.emit();\n    }\n    "},"id":"9446","type":"CustomJS"},{"attributes":{"data_source":{"id":"9374"},"glyph":{"id":"9435"},"hover_glyph":null,"muted_glyph":null,"nonselection_glyph":{"id":"9436"},"selection_glyph":null,"view":{"id":"9438"}},"id":"9437","type":"GlyphRenderer"},{"attributes":{},"id":"9413","type":"Range1d"},{"attributes":{"children":[{"id":"9452"},{"id":"9453"},{"id":"9454"},{"id":"9455"},{"id":"9456"}]},"id":"9457","type":"Column"},{"attributes":{"columns":[{"id":"9441"},{"id":"9442"}],"height":340,"source":{"id":"9374"},"view":{"id":"9444"},"width":900},"id":"9443","type":"DataTable"},{"attributes":{"children":[{"id":"9450"}]},"id":"9453","type":"Row"},{"attributes":{},"id":"9380","type":"LinearScale"},{"attributes":{"source":{"id":"9374"}},"id":"9444","type":"CDSView"},{"attributes":{},"id":"9465","type":"StringEditor"},{"attributes":{},"id":"9475","type":"StringEditor"},{"attributes":{"axis":{"id":"9384"},"ticker":null},"id":"9387","type":"Grid"},{"attributes":{"fill_color":{"field":"color"},"line_color":{"field":"color"},"radius":{"field":"rad","units":"data"},"x":{"field":"x"},"y":{"field":"y"}},"id":"9435","type":"Circle"},{"attributes":{"editor":{"id":"9473"},"field":"scr","formatter":{"id":"9472"},"title":"Score","width":30},"id":"9441","type":"TableColumn"},{"attributes":{"formatter":{"id":"9478"},"ticker":{"id":"9420"}},"id":"9419","type":"LinearAxis"},{"attributes":{"js_property_callbacks":{"change:indices":[{"id":"9446"}]}},"id":"9445","type":"Selection"},{"attributes":{"text":""},"id":"9458","type":"Title"},{"attributes":{"js_property_callbacks":{"change:indices":[{"id":"9448"}]}},"id":"9447","type":"Selection"},{"attributes":{},"id":"9417","type":"LinearScale"},{"attributes":{},"id":"9382","type":"LinearScale"}],"root_ids":["9457"]},"title":"Bokeh Application","version":"2.0.1"}}
        </script>
        <script type="text/javascript">
          (function() {
            var fn = function() {
              Bokeh.safely(function() {
                (function(root) {
                  function embed_document(root) {
                    
                  var docs_json = document.getElementById('9694').textContent;
                  var render_items = [{"docid":"980948ec-e35e-411c-9dc9-8bd4d847085d","root_ids":["9457"],"roots":{"9457":"a1ec7714-fb14-4a34-bb19-24d9fcdf1986"}}];
                  root.Bokeh.embed.embed_items(docs_json, render_items);
                
                  }
                  if (root.Bokeh !== undefined) {
                    embed_document(root);
                  } else {
                    var attempts = 0;
                    var timer = setInterval(function(root) {
                      if (root.Bokeh !== undefined) {
                        clearInterval(timer);
                        embed_document(root);
                      } else {
                        attempts++;
                        if (attempts > 100) {
                          clearInterval(timer);
                          console.log("Bokeh: ERROR: Unable to run BokehJS code because BokehJS library is missing");
                        }
                      }
                    }, 10, root)
                  }
                })(window);
              });
            };
            if (document.readyState != "loading") fn();
            else document.addEventListener("DOMContentLoaded", fn);
          })();
        </script>
    
  </body>
  
</html>