



<!DOCTYPE html>
<html lang="en">
  
  <head>
    
      <meta charset="utf-8">
      <title>Bokeh Plot</title>
      
      
        
          
        
        
          
        <script type="text/javascript" src="https://cdn.bokeh.org/bokeh/release/bokeh-2.0.1.min.js" integrity="sha384-JpP8FXbgAZLkfur7LiK3j9AGBhHNIvF742meBJrjO2ShJDhCG2I1uVvW+0DUtrmc" crossorigin="anonymous"></script>
        <script type="text/javascript" src="https://cdn.bokeh.org/bokeh/release/bokeh-widgets-2.0.1.min.js" integrity="sha384-xZlADit0Q04ISQEdKg2k3L4W9AwQBAuDs9nJL9fM/WwzL1tEU9VPNezOFX0nLEAz" crossorigin="anonymous"></script>
        <script type="text/javascript" src="https://cdn.bokeh.org/bokeh/release/bokeh-tables-2.0.1.min.js" integrity="sha384-4BuPRZkdMKSnj3zoxiNrQ86XgNw0rYmBOxe7nshquXwwcauupgBF2DHLVG1WuZlV" crossorigin="anonymous"></script>
        <script type="text/javascript">
            Bokeh.set_log_level("info");
        </script>
        
      
      
    
  </head>
  
  
  <body>
    
      
        
          
          
            
              <div class="bk-root" id="8c050973-10f9-464f-8f41-c65363121638" data-root-id="28133"></div>
            
          
        
      
      
        <script type="application/json" id="28370">
          {"0eec7a5d-3b52-49ed-9d41-90a7d26934bb":{"roots":{"references":[{"attributes":{"children":[{"id":"28128"},{"id":"28129"},{"id":"28130"},{"id":"28131"},{"id":"28132"}]},"id":"28133","type":"Column"},{"attributes":{"axis":{"id":"28064"},"dimension":1,"ticker":null},"id":"28067","type":"Grid"},{"attributes":{},"id":"28096","type":"BasicTicker"},{"attributes":{"text":""},"id":"28134","type":"Title"},{"attributes":{},"id":"28151","type":"StringEditor"},{"attributes":{"callback":null,"tooltips":[["Title","@title"],["Passage","@text"],["Published","@date"],["Authors","@authors"],["Journal","@journal"],["URL","@url"]]},"id":"28105","type":"HoverTool"},{"attributes":{"active_drag":"auto","active_inspect":"auto","active_multi":null,"active_scroll":"auto","active_tap":"auto","tools":[{"id":"28103"},{"id":"28104"},{"id":"28105"}]},"id":"28106","type":"Toolbar"},{"attributes":{},"id":"28052","type":"Range1d"},{"attributes":{"active_drag":"auto","active_inspect":"auto","active_multi":null,"active_scroll":"auto","active_tap":"auto","tools":[{"id":"28068"},{"id":"28069"},{"id":"28070"}]},"id":"28071","type":"Toolbar"},{"attributes":{"children":[{"id":"28127"}]},"id":"28131","type":"Row"},{"attributes":{},"id":"28149","type":"StringEditor"},{"attributes":{},"id":"28054","type":"Range1d"},{"attributes":{"callback":null,"tooltips":[["Title","@title"],["Abstract","@text"],["Published","@date"],["Authors","@authors"],["Journal","@journal"],["URL","@url"]]},"id":"28070","type":"HoverTool"},{"attributes":{},"id":"28091","type":"LinearScale"},{"attributes":{"axis":{"id":"28060"},"ticker":null},"id":"28063","type":"Grid"},{"attributes":{},"id":"28100","type":"BasicTicker"},{"attributes":{"children":[{"id":"28119"},{"id":"28086"}]},"id":"28132","type":"Row"},{"attributes":{"js_property_callbacks":{"change:indices":[{"id":"28122"}]}},"id":"28121","type":"Selection"},{"attributes":{},"id":"28061","type":"BasicTicker"},{"attributes":{},"id":"28093","type":"LinearScale"},{"attributes":{"source":{"id":"28049"}},"id":"28085","type":"CDSView"},{"attributes":{},"id":"28148","type":"StringFormatter"},{"attributes":{"text":""},"id":"28136","type":"Title"},{"attributes":{"fill_alpha":{"value":0.1},"fill_color":{"field":"color"},"line_alpha":{"value":0.1},"line_color":{"field":"color"},"radius":{"field":"rad","units":"data"},"x":{"field":"x"},"y":{"field":"y"}},"id":"28112","type":"Circle"},{"attributes":{},"id":"28143","type":"UnionRenderers"},{"attributes":{"fill_alpha":{"value":0.1},"fill_color":{"field":"color"},"line_alpha":{"value":0.1},"line_color":{"field":"color"},"radius":{"field":"rad","units":"data"},"x":{"field":"x"},"y":{"field":"y"}},"id":"28077","type":"Circle"},{"attributes":{},"id":"28140","type":"StringFormatter"},{"attributes":{"columns":[{"id":"28082"},{"id":"28083"}],"height":340,"source":{"id":"28049"},"view":{"id":"28085"},"width":900},"id":"28084","type":"DataTable"},{"attributes":{},"id":"28153","type":"UnionRenderers"},{"attributes":{"children":[{"id":"28084"},{"id":"28051"}]},"id":"28130","type":"Row"},{"attributes":{"callback":null},"id":"28068","type":"TapTool"},{"attributes":{"children":[{"id":"28125"}]},"id":"28128","type":"Row"},{"attributes":{"text":"&lt;h3&gt;Paragraph ranking&lt;/h3&gt;"},"id":"28127","type":"Div"},{"attributes":{},"id":"28141","type":"StringEditor"},{"attributes":{},"id":"28139","type":"StringEditor"},{"attributes":{"args":{"sd":{"id":"28049"},"sp":{"id":"28050"}},"code":"\n    if (sp.selected.indices.length == 0) {\n      return;\n    }\n    var sel = sp.selected.indices[0];\n    var new_selected = [];\n    for (var i = 0; i &lt; sd.data['doc'].length; i++) {\n      if (sd.data['doc'][i] == sp.data['doc'][sel]) {\n        new_selected.push(i);\n      }\n    }\n    if (JSON.stringify(new_selected) != JSON.stringify(sd.selected.indices) &amp;&amp;\n    !(sd.selected.indices.length == 1 &amp;&amp; new_selected.includes(sd.selected.indices[0]))) {\n      sd.selected.indices = new_selected;\n      sd.change.emit();\n    }\n    "},"id":"28124","type":"CustomJS"},{"attributes":{"formatter":{"id":"28144"},"ticker":{"id":"28061"}},"id":"28060","type":"LinearAxis"},{"attributes":{},"id":"28150","type":"StringFormatter"},{"attributes":{"source":{"id":"28050"}},"id":"28114","type":"CDSView"},{"attributes":{"source":{"id":"28050"}},"id":"28120","type":"CDSView"},{"attributes":{"axis":{"id":"28095"},"ticker":null},"id":"28098","type":"Grid"},{"attributes":{},"id":"28069","type":"ResetTool"},{"attributes":{},"id":"28087","type":"Range1d"},{"attributes":{},"id":"28089","type":"Range1d"},{"attributes":{"formatter":{"id":"28156"},"ticker":{"id":"28100"}},"id":"28099","type":"LinearAxis"},{"attributes":{"data_source":{"id":"28049"},"glyph":{"id":"28076"},"hover_glyph":null,"muted_glyph":null,"nonselection_glyph":{"id":"28077"},"selection_glyph":null,"view":{"id":"28079"}},"id":"28078","type":"GlyphRenderer"},{"attributes":{"columns":[{"id":"28117"},{"id":"28118"}],"height":340,"source":{"id":"28050"},"view":{"id":"28120"},"width":900},"id":"28119","type":"DataTable"},{"attributes":{},"id":"28056","type":"LinearScale"},{"attributes":{},"id":"28138","type":"StringFormatter"},{"attributes":{},"id":"28154","type":"BasicTickFormatter"},{"attributes":{"editor":{"id":"28141"},"field":"title","formatter":{"id":"28140"},"title":"Document title","width":770},"id":"28083","type":"TableColumn"},{"attributes":{"sizing_mode":"stretch_width","text":"&lt;h2&gt;One Health surveillance of humans and potential sources of future spillover or ongoing exposure for this organism and future pathogens, including both evolutionary hosts (e.g., bats) and transmission hosts (e.g., heavily trafficked and farmed wildlife and domestic food and companion species), inclusive of environmental, demographic, and occupational risk factors.&lt;/h2&gt;"},"id":"28125","type":"Div"},{"attributes":{"callback":null},"id":"28103","type":"TapTool"},{"attributes":{"formatter":{"id":"28146"},"ticker":{"id":"28065"}},"id":"28064","type":"LinearAxis"},{"attributes":{"editor":{"id":"28151"},"field":"text","formatter":{"id":"28150"},"title":"Passage","width":770},"id":"28118","type":"TableColumn"},{"attributes":{"js_property_callbacks":{"change:indices":[{"id":"28124"}]}},"id":"28123","type":"Selection"},{"attributes":{"children":[{"id":"28126"}]},"id":"28129","type":"Row"},{"attributes":{"below":[{"id":"28095"}],"center":[{"id":"28098"},{"id":"28102"}],"left":[{"id":"28099"}],"plot_height":340,"plot_width":900,"renderers":[{"id":"28113"}],"title":{"id":"28136"},"toolbar":{"id":"28106"},"toolbar_location":"below","x_range":{"id":"28087"},"x_scale":{"id":"28091"},"y_range":{"id":"28089"},"y_scale":{"id":"28093"}},"id":"28086","subtype":"Figure","type":"Plot"},{"attributes":{},"id":"28144","type":"BasicTickFormatter"},{"attributes":{},"id":"28065","type":"BasicTicker"},{"attributes":{"source":{"id":"28049"}},"id":"28079","type":"CDSView"},{"attributes":{"editor":{"id":"28139"},"field":"scr","formatter":{"id":"28138"},"title":"Score","width":30},"id":"28082","type":"TableColumn"},{"attributes":{"editor":{"id":"28149"},"field":"scr","formatter":{"id":"28148"},"title":"Score","width":30},"id":"28117","type":"TableColumn"},{"attributes":{"args":{"sd":{"id":"28049"},"sp":{"id":"28050"}},"code":"\n    if (sd.selected.indices.length == 0) {\n      return;\n    }\n    var sel = sd.selected.indices[0];\n    var new_selected = [];\n    for (var i = 0; i &lt; sp.data['doc'].length; i++) {\n      if (sp.data['doc'][i] == sd.data['doc'][sel]) {\n        new_selected.push(i);\n      }\n    }\n    if (JSON.stringify(new_selected) != JSON.stringify(sp.selected.indices) &amp;&amp;\n    !(sp.selected.indices.length == 1 &amp;&amp; new_selected.includes(sp.selected.indices[0]))) {\n      sp.selected.indices = new_selected;\n      sp.change.emit();\n    }\n    "},"id":"28122","type":"CustomJS"},{"attributes":{"formatter":{"id":"28154"},"ticker":{"id":"28096"}},"id":"28095","type":"LinearAxis"},{"attributes":{},"id":"28146","type":"BasicTickFormatter"},{"attributes":{"below":[{"id":"28060"}],"center":[{"id":"28063"},{"id":"28067"}],"left":[{"id":"28064"}],"plot_height":340,"plot_width":900,"renderers":[{"id":"28078"}],"title":{"id":"28134"},"toolbar":{"id":"28071"},"toolbar_location":"below","x_range":{"id":"28052"},"x_scale":{"id":"28056"},"y_range":{"id":"28054"},"y_scale":{"id":"28058"}},"id":"28051","subtype":"Figure","type":"Plot"},{"attributes":{},"id":"28104","type":"ResetTool"},{"attributes":{},"id":"28058","type":"LinearScale"},{"attributes":{"fill_color":{"field":"color"},"line_color":{"field":"color"},"radius":{"field":"rad","units":"data"},"x":{"field":"x"},"y":{"field":"y"}},"id":"28111","type":"Circle"},{"attributes":{"text":"&lt;h3&gt;Document ranking&lt;/h3&gt;"},"id":"28126","type":"Div"},{"attributes":{"data_source":{"id":"28050"},"glyph":{"id":"28111"},"hover_glyph":null,"muted_glyph":null,"nonselection_glyph":{"id":"28112"},"selection_glyph":null,"view":{"id":"28114"}},"id":"28113","type":"GlyphRenderer"},{"attributes":{},"id":"28156","type":"BasicTickFormatter"},{"attributes":{"data":{"authors":["Mackenzie, John S; Jeggo, Martyn","Yonghua Wu","Niccolo Alfano; Anisha Dayaram; Jan Axtner; Kyriakos Tsangaras; Marie-Louise Kampmann; Azlan Mohamed; Seth Timothy Wong; M. Thomas P. Gilbert; Andreas Wilting; Alex Daivd Greenwood","Ping Liu; Jing-Zhe Jiang; Xiu-Feng Wan; Yan Hua; Xiaohu Wang; Fanghui Hou; Jing Chen; Jiejian Zou; Jinping Chen","Shipley, Rebecca; Wright, Edward; Selden, David; Wu, Guanghui; Aegerter, James; Fooks, Anthony R; Banyard, Ashley C","Daszak, Peter; Olival, Kevin J.; Li, Hongying","Tang, Qin; Song, Yulong; Shi, Mijuan; Cheng, Yingyin; Zhang, Wanting; Xia, Xiao-Qin","Razanajatovo, Norosoa H; Nomenjanahary, Lalaina A; Wilkinson, David A; Razafimanahaka, Julie H; Goodman, Steven M; Jenkins, Richard K; Jones, Julia PG; Heraud, Jean-Michel","Rizzo, Francesca; Edenborough, Kathryn M.; Toffoli, Roberto; Culasso, Paola; Zoppi, Simona; Dondo, Alessandro; Robetto, Serena; Rosati, Sergio; Lander, Angelika; Kurth, Andreas; Orusa, Riccardo; Bertolotti, Luigi; Mandola, Maria Lucia","Bean, Andrew G. D.; Baker, Michelle L.; Stewart, Cameron R.; Cowled, Christopher; Deffrasnes, Celine; Wang, Lin-Fa; Lowenthal, John W.","Han, Hui-Ju; Wen, Hong-ling; Zhou, Chuan-Min; Chen, Fang-Fang; Luo, Li-Mei; Liu, Jian-wei; Yu, Xue-Jie","El-Duah, Philip; Meyer, Benjamin; Sylverken, Augustina; Owusu, Michael; Gottula, Lina Theresa; Yeboah, Richmond; Lamptey, Jones; Frimpong, Yaw Oppong; Burimuah, Vitus; Folitse, Raphael; Agbenyega, Olivia; Oppong, Samuel; Adu-Sarkodie, Yaw; Drosten, Christian","Louie Florendo Dy; Jomar Fajardo Rabajante","Zhu, Sophie; Zimmerman, Dawn; Deem, Sharon L.","Tang, Xianchun; Li, Gang; Vasilakis, Nikos; Zhang, Yuan; Shi, Zhengli; Zhong, Yang; Wang, Lin-Fa; Zhang, Shuyi","Menachery, Vineet D; Graham, Rachel L; Baric, Ralph S","Wray, Amy K.; Olival, Kevin J.; Mor\u00e1n, David; Lopez, Maria Renee; Alvarez, Danilo; Navarrete-Macias, Isamara; Liang, Eliza; Simmons, Nancy B.; Lipkin, W. Ian; Daszak, Peter; Anthony, Simon J.","Chu, Daniel K. W.; Peiris, J. S. Malik; Poon, Leo L. M.","Mi\u0142ek, Justyna; Blicharz-Doma\u0144ska, Katarzyna","Shi, Weifeng; Li, Juan; Zhou, Hong; Gao, George F.","Perlman, Stanley; Netland, Jason","Smith, Ina; Wang, Lin-Fa","Hoe GAN, Wee; Wah LIM, John; KOH, David","Malik, Yashpal Singh; Sircar, Shubhankar; Bhat, Sudipta; Sharun, Khan; Dhama, Kuldeep; Dadar, Maryam; Tiwari, Ruchi; Chaicumpa, Wanpen","Sun, Zhong; Thilakavathy, Karuppiah; Kumar, S. S.; He, Guozhong; Liu, V. Shi","Fredricks, David N; Relman, David A","Hu, Ben; Ge, Xingyi; Wang, Lin-Fa; Shi, Zhengli","Lipkin, W. Ian","Urisman, Anatoly; Fischer, Kael F; Chiu, Charles Y; Kistler, Amy L; Beck, Shoshannah; Wang, David; DeRisi, Joseph L","Tianmu Chen; Jia Rui; Qiupeng Wang; Zeyu Zhao; Jing-An Cui; Ling Yin","Qiu, Ye; Zhao, Yuan-Bo; Wang, Qiong; Li, Jin-Yan; Zhou, Zhi-Jian; Liao, Ce-Heng; Ge, Xing-Yi","Baker, Kate S.; Leggett, Richard M.; Bexfield, Nicholas H.; Alston, Mark; Daly, Gordon; Todd, Shawn; Tachedjian, Mary; Holmes, Clare E.G.; Crameri, Sandra; Wang, Lin-Fa; Heeney, Jonathan L.; Suu-Ire, Richard; Kellam, Paul; Cunningham, Andrew A.; Wood, James L.N.; Caccamo, Mario; Murcia, Pablo R.","Mirkena, Tadele; Walelign, Elias; Tewolde, Nega; Gari, Getachew; Abebe, Getachew; Newman, Scott","Kohl, Claudia; Kurth, Andreas","Cui, Jie; Li, Fang; Shi, Zheng-Li","Attrey, D.P.","Goldenberg, Simon","Donno, Dario; Mellano, Maria Gabriella; Cerutti, Alessandro Kim; Beccaro, Gabriele Loris","Lin, Qianying; Zhao, Shi; Gao, Daozhou; Lou, Yijun; Yang, Shu; Musa, Salihu S.; Wang, Maggie H.; Cai, Yongli; Wang, Weiming; Yang, Lin; He, Daihai","Alexander Ugarov","Sun, Jiumeng; He, Wan-Ting; Wang, Lifang; Lai, Alexander; Ji, Xiang; Zhai, Xiaofeng; Li, Gairu; Suchard, Marc A.; Tian, Jin; Zhou, Jiyong; Veit, Michael; Su, Shuo","Tommy Tsan-Yuk Lam; Marcus Ho-Hin Shum; Hua-Chen Zhu; Yi-Gang Tong; Xue-Bing Ni; Yun-Shi Liao; Wei Wei; William Yiu-Man Cheung; Wen-Juan Li; Lian-Feng Li; Gabriel M Leung; Edward C. Holmes; Yan-Ling Hu; Yi Guan","Elston, Dirk M.","Berger, A.; Preiser, W.","Kun Sun; Liuqi Gu; Li Ma; Yunfeng Duan","Musazzi, Umberto M.; Di Giorgio, Domenico; Minghetti, Paola","Berger, A.; Preiser, W.","Qiang, Xiao-Li; Xu, Peng; Fang, Gang; Liu, Wen-Bin; Kou, Zheng","Poon, L. L. M.; Chu, D. K. W.; Chan, K. H.; Wong, O. K.; Ellis, T. M.; Leung, Y. H. C.; Lau, S. K. P.; Woo, P. C. Y.; Suen, K. Y.; Yuen, K. Y.; Guan, Y.; Peiris, J. S. M.","Mohd, Hamzah A.; Al-Tawfiq, Jaffar A.; Memish, Ziad A."],"color":["#67a9cf","#67a9cf","#67a9cf","#67a9cf","#67a9cf","#67a9cf","#67a9cf","#67a9cf","#67a9cf","#67a9cf","#67a9cf","#67a9cf","#67a9cf","#67a9cf","#67a9cf","#67a9cf","#67a9cf","#67a9cf","#67a9cf","#67a9cf","#67a9cf","#67a9cf","#67a9cf","#67a9cf","#67a9cf","#67a9cf","#67a9cf","#67a9cf","#67a9cf","#67a9cf","#67a9cf","#67a9cf","#67a9cf","#67a9cf","#67a9cf","#67a9cf","#67a9cf","#67a9cf","#67a9cf","#67a9cf","#67a9cf","#67a9cf","#67a9cf","#67a9cf","#67a9cf","#67a9cf","#67a9cf","#67a9cf","#67a9cf","#67a9cf"],"date":["2013-04-30","2020-03-04","2020-03-29","2020-02-20","2019-02-07","2020","2015-11-26","2015-03-12","2017-12-22","2013-10-25","2015-07-02","2019-01-09","2020-03-30","2019-05-28","2009-03-05","2017-04-30","2016-09-22","2009-04-14","2018-12-10","2017-11-28","2009-05-11","2013-02-28","2020-03-24","2020-02-08","2020","2005-03-01","2015-12-22","2013-01-03","2005-08-30","2020-01-19","2020-03-19","2013-07-05","2018-12-21","2014-08-13","2019","2017-12-31","2013-11-30","2018-12-31","2020","2020-03-30","2020-03-21","2020-02-18","2020-03-18","2011-12-31","2020-03-31","2020-04-15","2011-12-31","2020-03-25","2005-01-28","2016-06-03"],"doc":["0g5079y1","4ihv80au","nil1vv6h","2inlyd0t","hibzy1oh","h5sox8bq","gl65i3s0","ghcwfcx2","llxqcikz","exe1exm1","3lg1bzsi","tjp33y35","axijpn4y","ysxvl327","i5hkucrn","cyetlmgt","9ojdhxwk","g5fqxdqh","ur8g68vp","k0f9fser","ixytcvge","v8uzd2f3","froywddm","1qkwsh6a","6dbt99h0","cisrvghj","d2knbzhl","0i3ewil9","kqithgfo","v4mbry22","yaspd6l7","rwv5ibg3","puv3mn42","4rxradwz","ne5r4d4b","i5k60y63","n8hlrknq","nclkfw4h","zq2h0avw","pd2palu4","9il7coyk","5f42du0b","t49h7bxx","kromgrnu","89sg0cpk","k0kha22e","yvsyo4l9","llv3cvdr","b6abkv7p","yzp9wjuk"],"journal":["Current Opinion in Virology","","","","Trop Med Infect Dis","Biosafety and Health","Sci Rep","Virol J","BMC Vet Res","Nat Rev Immunol","Virus Research","Viruses","","Ecohealth","BMC Evol Biol","Current Opinion in Virology","Ecohealth","Virol Sin","J Vet Res","Sci China Life Sci","Nature Reviews Microbiology","Current Opinion in Virology","Safety and Health at Work","Vet Q","International Journal of Environmental Research and Public Health","Medicine","Virol J","Nature Reviews Microbiology","Genome Biol","","Microbes and Infection","Virology","Pastoralism","Viruses","Nat Rev Microbiol","Food Safety in the 21st Century","Medicine","Alternative and Replacement Foods","International Journal of Infectious Diseases","","Trends in Molecular Medicine","","Journal of the American Academy of Dermatology","Encyclopedia of Environmental Health","","International Journal of Pharmaceutics","Encyclopedia of Environmental Health","Infect Dis Poverty","Journal of Virology","Virol J"],"rad":[0.02,0.019849436895887165,0.01885002282724155,0.018359842174422276,0.01590170184698876,0.015294601677644928,0.014142633443742909,0.01411855006469765,0.0136611147962061,0.013158480248823084,0.012955505995065935,0.012480868484421176,0.012160375662188517,0.01203184411484657,0.011978882108564615,0.011963540236086239,0.011529260264824308,0.01083222513688619,0.010775566477869314,0.010364305046374016,0.010332454314153468,0.009670149266666282,0.00953326278885909,0.009197052790479578,0.009047151737469887,0.00871729869007597,0.007195011048512403,0.007143616611384441,0.006981759755186591,0.006904988919053305,0.006360957844621042,0.006348735108441951,0.006057025532859299,0.004882103170911565,0.004843750217862713,0.004679312678848885,0.004546820093748913,0.004495694224532806,0.004192422152864407,0.0041757544494888865,0.004119027682299102,0.0038265395815736563,0.0037249219722230676,0.0037202229025201424,0.0037030179480421624,0.0036633289202193694,0.0036626705318826037,0.0035332016547907795,0.0034798756573422816,0.003],"scr":[1.0,0.991,0.932,0.904,0.759,0.723,0.655,0.654,0.627,0.598,0.586,0.558,0.539,0.531,0.528,0.527,0.502,0.461,0.457,0.433,0.431,0.392,0.384,0.365,0.356,0.336,0.247,0.244,0.234,0.23,0.198,0.197,0.18,0.111,0.108,0.099,0.091,0.088,0.07,0.069,0.066,0.049,0.043,0.042,0.041,0.039,0.039,0.031,0.028,0.0],"text":["Wildlife, especially mammals and birds, are hosts to an enormous number of viruses, most of which we have absolutely no knowledge about even though we know these viruses circulate readily in their specific niches. More often than not, these viruses are silent or asymptomatic in their natural hosts. In some instances, they can infect other species, and in rare cases, this cross-species transmission might lead to human infection. There are also instances where we know the reservoir hosts of zoonotic viruses that can and do infect humans. Studies of these animal hosts, the reservoirs of the viruses, provide us with the knowledge of the types of virus circulating in wildlife species, their incidence, pathogenicity for their host, and in some instances, the potential for transmission to other hosts. This paper describes examples of some of the viruses that have been detected in wildlife, and the reservoir hosts from which they have been detected. It also briefly explores the spread of arthr...","AbstractCoronavirus Disease 2019 (COVID-19) and severe acute respiratory syndrome (SARS)-related coronaviruses (e.g., 2019-nCoV and SARS-CoV) are phylogenetically distantly related, but both are capable of infecting human hosts via the same receptor, angiotensin-converting enzyme 2, and cause similar clinical and pathological features, suggesting their phenotypic convergence. Yet, the molecular basis that underlies their phenotypic convergence remains unknown. Here, we used a recently developed molecular phyloecological approach to examine the molecular basis leading to their phenotypic convergence. Our genome-level analyses show that the spike protein, which is responsible for receptor binding, has undergone significant Darwinian selection along the branches related to 2019-nCoV and SARS-CoV. Further examination shows an unusually high proportion of evolutionary convergent amino acid sites in the receptor binding domain (RBD) of the spike protein between COVID-19 and SARS-related CoV ...","Environmental DNA (eDNA) and its subdiscipline, invertebrate-derived DNA (iDNA) have been used to survey biodiversity non-invasively [1,2]. Water is ubiquitous in most ecosystems, and, among invertebrates, terrestrial haematophagous leeches are abundant and can be easily collected in many tropical rainforests [3,4]. Such non-invasive nucleic acid sources can mitigate difficulties of obtaining wildlife samples, particularly in remote areas or for rare species. Recently, eDNA/iDNA sources have been applied to monitoring specific wildlife pathogens [5,6]. However, previous studies have focused on known pathogens, whereas most wildlife pathogens are uncharacterized and unknown. Non-invasive approaches to monitoring known and novel pathogens may be of particular benefit in ecosystems prone to viral emergence, many of which occur in areas where invasive sampling is challenging, such as tropical rainforests. Here, we show that both eDNA from natural waterholes, and iDNA from terrestrial haema...","AbstractThe outbreak of 2019-nCoV pneumonia (COVID-19) in the city of Wuhan, China has resulted in more than 70,000 laboratory confirmed cases, and recent studies showed that 2019-nCoV (SARS-CoV-2) could be of bat origin but involve other potential intermediate hosts. In this study, we assembled the genomes of coronaviruses identified in sick pangolins. The molecular and phylogenetic analyses showed that pangolin Coronaviruses (pangolin-CoV) are genetically related to both the 2019-nCoV and bat Coronaviruses but do not support the 2019-nCoV arose directly from the pangolin-CoV. Our study also suggested that pangolin be natural host of Betacoronavirus, with a potential to infect humans. Large surveillance of coronaviruses in pangolins could improve our understanding of the spectrum of coronaviruses in pangolins. Conservation of wildlife and limits of the exposures of humans to wildlife will be important to minimize the spillover risks of coronaviruses from wild animals to humans.","Bats in the EU have been associated with several zoonotic viral pathogens of significance to both human and animal health. Virus discovery continues to expand the existing understating of virus classification, and the increased interest in bats globally as reservoirs or carriers of zoonotic agents has fuelled the continued detection and characterisation of new lyssaviruses and other viral zoonoses. Although the transmission of lyssaviruses from bat species to humans or terrestrial species appears rare, interest in these viruses remains, through their ability to cause the invariably fatal encephalitis\u2014rabies. The association of bats with other viral zoonoses is also of great interest. Much of the EU is free of terrestrial rabies, but several bat species harbor lyssaviruses that remain a risk to human and animal health. Whilst the rabies virus is the main cause of rabies globally, novel related viruses continue to be discovered, predominantly in bat populations, that are of interest pure...","A novel bat-origin coronavirus emerged in Wuhan, China in December 2019 and continues to spread across China and the world. At the time of writing, a massive global response has been implemented to control the disease as it spreads from person to person. Yet the high-risk human-wildlife interactions and interfaces that led to the emergence of SARS-CoV and of 2019-nCoV continue to exist in emerging disease hotspots globally. To prevent the next pandemic related to these interfaces, we call for research and investment in three areas: 1) surveillance among wildlife to identify the high-risk pathogens they carry; 2) surveillance among people who have contact with wildlife to identify early spillover events; and 3) improvement of market biosecurity regarding the wildlife trade. As the emergence of a novel virus anywhere can impact the furthest reaches of our connected world, international collaboration among scientists is essential to address these risks and prevent the next pandemic.","Many coronaviruses are capable of interspecies transmission. Some of them have caused worldwide panic as emerging human pathogens in recent years, e.g., severe acute respiratory syndrome coronavirus (SARS-CoV) and Middle East respiratory syndrome coronavirus (MERS-CoV). In order to assess their threat to humans, we explored to infer the potential hosts of coronaviruses using a dual-model approach based on nineteen parameters computed from spike genes of coronaviruses. Both the support vector machine (SVM) model and the Mahalanobis distance (MD) discriminant model achieved high accuracies in leave-one-out cross-validation of training data consisting of 730 representative coronaviruses (99.86% and 98.08% respectively). Predictions on 47 additional coronaviruses precisely conformed to conclusions or speculations by other researchers. Our approach is implemented as a web server that can be accessed at http://bioinfo.ihb.ac.cn/seq2hosts.","BACKGROUND: Bats are amongst the natural reservoirs of many coronaviruses (CoVs) of which some can lead to severe infection in human. African bats are known to harbor a range of pathogens (e.g., Ebola and Marburg viruses) that can infect humans and cause disease outbreaks. A recent study in South Africa isolated a genetic variant closely related to MERS-CoV from an insectivorous bat. Though Madagascar is home to 44 bat species (41 insectivorous and 3 frugivorous) of which 34 are endemic, no data exists concerning the circulation of CoVs in the island\u2019s chiropteran fauna. Certain Malagasy bats can be frequently found in close contact with humans and frugivorous bats feed in the same trees where people collect and consume fruits and are hunted and consumed as bush meat. The purpose of our study is to detect and identify CoVs from frugivorous bats in Madagascar to evaluate the risk of human infection from infected bats. METHODS: Frugivorous bats belonging to three species were captured in...","BACKGROUND: Bat-borne virus surveillance is necessary for determining inter-species transmission risks and is important due to the wide-range of bat species which may harbour potential pathogens. This study aimed to monitor coronaviruses (CoVs) and paramyxoviruses (PMVs) in bats roosting in northwest Italian regions. Our investigation was focused on CoVs and PMVs due to their proven ability to switch host and their zoonotic potential. Here we provide the phylogenetic characterization of the highly conserved polymerase gene fragments. RESULTS: Family-wide PCR screenings were used to test 302 bats belonging to 19 different bat species. Thirty-eight animals from 12 locations were confirmed as PCR positive, with an overall detection rate of 12.6% [95% CI: 9.3\u201316.8]. CoV RNA was found in 36 bats belonging to eight species, while PMV RNA in three Pipistrellus spp. Phylogenetic characterization have been obtained for 15 alpha- CoVs, 5 beta-CoVs and three PMVs; moreover one P. pipistrellus res...","Zoonotic viruses that emerge from wildlife and domesticated animals pose a serious threat to human and animal health. In many instances, mouse models have improved our understanding of the human immune response to infection; however, when dealing with emerging zoonotic diseases, they may be of limited use. This is particularly the case when the model fails to reproduce the disease status that is seen in the natural reservoir, transmission species or human host. In this Review, we discuss how researchers are placing more emphasis on the study of the immune response to zoonotic infections in the natural reservoir hosts and spillover species. Such studies will not only lead to a greater understanding of how these infections induce variable disease and immune responses in distinct species but also offer important insights into the evolution of mammalian immune systems. SUPPLEMENTARY INFORMATION: The online version of this article (doi:10.1038/nri3551) contains supplementary material, which...","Abstract In recent years severe infectious diseases have been constantly emerging, causing panic in the world. Now we know that many of these terrible diseases are caused by viruses originated from bats (Table 1), such as Ebola virus, Marburg, SARS coronavirus (SARS-CoV), MERS coronavirus (MERS-CoV), Nipah virus (NiV) and Hendra virus (HeV). These viruses have co-evolved with bats due to bats\u2019 special social, biological and immunological features. Although bats are not in close contact with humans, spillover of viruses from bats to intermediate animal hosts, such as horses, pigs, civets, or non-human primates, is thought to be the most likely mode to cause human infection. Humans may also become infected with viruses through aerosol by intruding into bat roosting caves or via direct contact with bats, such as catching bats or been bitten by bats.","Known human coronaviruses are believed to have originated in animals and made use of intermediate hosts for transmission to humans. The intermediate hosts of most of the human coronaviruses are known, but not for HCoV-NL63. This study aims to assess the possible role of some major domestic livestock species as intermediate hosts of HCoV-NL63. We developed a testing algorithm for high throughput screening of livestock sera with ELISA and confirmation with recombinant immunofluorescence assay testing for antibodies against HCoV-NL63 in livestock. Optimization of the ELISA showed a capability of the assay to significantly distinguish HCoV-NL63 from HCoV-229E (U = 27.50, p &lt; 0.001) and HCoV-OC43 (U = 55.50, p &lt; 0.001) in coronavirus-characterized sera. Evaluation of the assay with collected human samples showed no significant difference in mean optical density values of immunofluorescence-classified HCoV-NL63-positive and HCoV-NL63-negative samples (F (1, 215) = 0.437, p = 0.509). All the ...","The number of confirmed COVID-19 cases admitted in hospitals is continuously increasing in the Philippines. Frontline health care workers are faced with imminent risks of getting infected. In this study, we formulate a theoretical model to calculate the risk of being infected in health care facilities considering the following factors: the average number of encounters with a suspected COVID-19 patient per hour; interaction time for each encounter; work shift duration or exposure time; crowd density, which may depend on the amount of space available in a given location; and availability and effectiveness of protective gears and facilities provided for the frontline health care workers. Based on the simulation results, we recommend the following: (i) decrease the rate of patient encounter per frontline health care worker, e.g., maximum of three encounters per hour in a 12-hour work shift duration; (ii) decrease the interaction time between the frontline health care worker and the patient...","Dromedary, or one-humped, camels Camelus dromedarius are an almost exclusively domesticated species that are common in arid areas as both beasts of burden and production animals for meat and milk. Currently, there are approximately 30 million dromedary camels, with highest numbers in Africa and the Middle East. The hardiness of camels in arid regions has made humans more dependent on them, especially as a stable protein source. Camels also carry and may transmit disease-causing agents to humans and other animals. The ability for camels to act as a point source or vector for disease is a concern due to increasing human demands for meat, lack of biosafety and biosecurity protocols in many regions, and a growth in the interface with wildlife as camel herds become sympatric with non-domestic species. We conducted a literature review of camel-borne zoonotic diseases and found that the majority of publications (65%) focused on Middle East respiratory syndrome (MERS), brucellosis, Echinococcu...","BACKGROUND: SARS coronavirus (SARS-CoV) was identified as the etiological agent of SARS, and extensive investigations indicated that it originated from an animal source (probably bats) and was recently introduced into the human population via wildlife animals from wet markets in southern China. Previous studies revealed that the spike (S) protein of SARS had experienced adaptive evolution, but whether other functional proteins of SARS have undergone adaptive evolution is not known. RESULTS: We employed several methods to investigate selective pressure among different SARS-CoV groups representing different epidemic periods and hosts. Our results suggest that most functional proteins of SARS-CoV have experienced a stepwise adaptive evolutionary pathway. Similar to previous studies, the spike protein underwent strong positive selection in the early and middle phases, and became stabilized in the late phase. In addition, the replicase experienced positive selection only in human patients, ...","Zoonotic transmission of novel viruses represents a significant threat to global public health and is fueled by globalization, the loss of natural habitats, and exposure to new hosts. For coronaviruses (CoVs), broad diversity exists within bat populations and uniquely positions them to seed future emergence events. In this review, we explore the host and viral dynamics that shape these CoV populations for survival, amplification, and possible emergence in novel hosts.","Certain bat species serve as natural reservoirs for pathogens in several key viral families including henipa-, lyssa-, corona-, and filoviruses, which may pose serious threats to human health. The Common Vampire Bat (Desmodus rotundus), due to its abundance, sanguivorous feeding habit involving humans and domestic animals, and highly social behavioral ecology, may have an unusually high potential for interspecies disease transmission. Previous studies have investigated rabies dynamics in D. rotundus, yet the diversity of other viruses, bacteria, and other microbes that these bats may carry remains largely unknown. We screened 396 blood, urine, saliva, and fecal samples from D. rotundus captured in Guatemala for 13 viral families and genera. Positive results were found for rhabdovirus, adenovirus, and herpesvirus assays. We also screened these samples for Bartonella spp. and found that 38% of individuals tested positive. To characterize potential for interspecies transmission associated...","Zoonotic transmissions of emerging pathogens from wildlife to human have shaped the history of mankind. These events have also highlighted our poor understanding of microorganisms circulated in wild animals. Coronaviruses and astroviruses, which can be found from a wide range of mammals, were recently detected in bats. Strikingly, these bat viruses are genetically highly diverse and these interesting findings might help to better understand the evolution and ecology of these viruses. The discoveries of these novel bats viruses not only suggested that bats are important hosts for these virus families, but also reiterated the role of bats as a reservoir of viruses that might pose a zoonotic threat to human health.","Coronaviruses (CoVs) are a large group of enveloped viruses with a single-strand RNA genome, which continuously circulate in mammals and birds and pose a threat to livestock, companion animals, and humans. CoVs harboured by avian species are classified to the genera gamma- and deltacoronaviruses. Within the gamma-CoVs the main representative is avian coronavirus, a taxonomic name which includes the highly contagious infectious bronchitis viruses (IBVs) in chickens and similar viruses infecting other domestic birds such as turkeys, guinea fowls, or quails. Additionally, IBVs have been detected in healthy wild birds, demonstrating that they may act as the vector between domestic and free-living birds. Moreover, CoVs other than IBVs, are identified in wild birds, which suggests that wild birds play a key role in the epidemiology of other gammaCoVs and deltaCoVs. Development of molecular techniques has significantly improved knowledge of the prevalence of CoVs in avian species. The methods...","In the past twenty years, numerous novel zoonotic viral agents with pandemic potential have emerged in China, such as the severe acute respiratory syndrome (SARS) coronavirus and, more recently, the avian-origin influenza A/H7N9 virus, which have caused outbreaks among humans with high morbidity and mortality. In addition, several emerging and re-emerging viral pathogens have also been imported into China from travelers, e.g. the Middle East respiratory syndrome (MERS) coronavirus and Zika virus (ZIKV). Herein, we review these emerging viral pathogens in China and focus on how surveillance by pathogen genomics has been employed to discover and annotate novel pathogenic agents, identify natural reservoirs, monitor the transmission events and delineate their evolution and adaption to the human host. We also highlight the application of genomic sequencing in the recent Ebola epidemics in Western Africa. In summary, genomic sequencing has become a standard research tool in the field of eme...","While coronaviruses were first identified nearly 60 years ago, they received notoriety in 2003 when one of their members was identified as the etiological agent of the Severe Acute Respiratory Syndrome (SARS-CoV) 1. Previously these viruses were known to be important agents of respiratory and enteric infections of domestic and companion animals and to cause approximately 15% of all cases of the common cold. This Review focuses on recent advances in our understanding of the mechanisms of coronavirus replication, interactions with the host immune response and disease pathogenesis and highlights the recent identification of numerous novel coronaviruses and the propensity of this virus family to cross species barriers.","Bats are being increasingly recognized as an important reservoir of zoonotic viruses of different families, including SARS coronavirus, Nipah virus, Hendra virus and Ebola virus. Several recent studies hypothesized that bats, an ancient group of flying mammals, are the major reservoir of several important RNA virus families from which other mammalian viruses of livestock and humans were derived. Although this hypothesis needs further investigation, the premise that bats carry a large number of viruses is commonly accepted. The question of whether bats have unique biological features making them ideal reservoir hosts has been the subject of several recent reviews. In this review, we will focus on the public health implications of bat derived zoonotic viral disease outbreaks, examine the drivers and risk factors of past disease outbreaks and outline research directions for better control of future disease events.","Abstract Coronavirus disease 2019 (COVID-19) poses an occupational health risk to healthcare workers. Several thousand healthcare workers have already been infected, mainly in China. Preventing intra-hospital transmission of the communicable disease is therefore a priority. Based on the Systems Engineering Initiative for Patient Safety model, the strategies and measures to protect healthcare workers in an acute tertiary hospital are described along the domains of work task, technologies and tools, work environmental factors, and organizational conditions. The principle of zero occupational infection remains an achievable goal that all healthcare systems need to strive for in the face of a potential pandemic.","Coronaviruses are the well-known cause of severe respiratory, enteric and systemic infections in a wide range of hosts including man, mammals, fish, and avian. The scientific interest on coronaviruses increased after the emergence of Severe Acute Respiratory Syndrome coronavirus (SARS-CoV) outbreaks in 2002-2003 followed by Middle East Respiratory Syndrome CoV (MERS-CoV). This decade\u2019s first CoV, named 2019-nCoV, emerged from Wuhan, China, and declared as \u2018Public Health Emergency of International Concern\u2019 on January 30(th), 2020 by the World Health Organization (WHO). As on February 4, 2020, 425 deaths reported in China only and one death outside China (Philippines). In a short span of time, the virus spread has been noted in 24 countries. The zoonotic transmission (animal-to-human) is suspected as the route of disease origin. The genetic analyses predict bats as the most probable source of 2019-nCoV though further investigations needed to confirm the origin of the novel virus. The ong...","Within last 17 years two widespread epidemics of severe acute respiratory syndrome (SARS) occurred in China, which were caused by related coronaviruses (CoVs): SARS-CoV and SARS-CoV-2. Although the origin(s) of these viruses are still unknown and their occurrences in nature are mysterious, some general patterns of their pathogenesis and epidemics are noticeable. Both viruses utilize the same receptor&amp;mdash;angiotensin-converting enzyme 2 (ACE2)&amp;mdash;for invading human bodies. Both epidemics occurred in cold dry winter seasons celebrated with major holidays, and started in regions where dietary consumption of wildlife is a fashion. Thus, if bats were the natural hosts of SARS-CoVs, cold temperature and low humidity in these times might provide conducive environmental conditions for prolonged viral survival in these regions concentrated with bats. The widespread existence of these bat-carried or -released viruses might have an easier time in breaking through human defenses when harsh wi...","Abstract There are many diseases for which a microbial aetiology is suspected. The hypothesis that a disease has an infectious cause is supported by: clinical features (similar to those of known infectious diseases, e.g. fever, leucocytosis), epidemiology (case clustering in time or location), histology (inflammation of affected tissues, e.g. granulomata) or characteristic microbial structures, treatment (clinical response to antimicrobial treatment), and prevention of disease by vaccines targeting microbial antigens. Proof that a microbe causes a disease requires more rigorous evidence. Future attempts to identify novel microbes associated with human disease may use sequence-based approaches. High-throughput sequencing may allow identification of unique microbial nucleic acid sequences in a background of host DNA. The complete sequencing of the human genome and multiple microbial genomes make this approach more feasible. DNA microarrays are also likely to be used in the search for nov...","Bats have been recognized as the natural reservoirs of a large variety of viruses. Special attention has been paid to bat coronaviruses as the two emerging coronaviruses which have caused unexpected human disease outbreaks in the 21st century, Severe Acute Respiratory Syndrome Coronavirus (SARS-CoV) and Middle East Respiratory Syndrome Coronavirus (MERS-CoV), are suggested to be originated from bats. Various species of horseshoe bats in China have been found to harbor genetically diverse SARS-like coronaviruses. Some strains are highly similar to SARS-CoV even in the spike protein and are able to use the same receptor as SARS-CoV for cell entry. On the other hand, diverse coronaviruses phylogenetically related to MERS-CoV have been discovered worldwide in a wide range of bat species, some of which can be classified to the same coronavirus species as MERS-CoV. Coronaviruses genetically related to human coronavirus 229E and NL63 have been detected in bats as well. Moreover, intermediate ...","The pace of pathogen discovery is increasing dramatically. This reflects not only factors that enable the appearance and globalization of new microbial infections but also improvements in methods for ascertainment. New molecular diagnostic platforms; investments in pathogen surveillance in wildlife, domestic animals and humans; and the advent of social media tools that mine the world wide web for clues to outbreaks of infectious disease are proving invaluable in early recognition of threats to public health. Additionally, models of microbial pathogenesis are becoming more complex, providing insights into the mechanisms by which microorganisms can contribute to chronic illnesses like cancer, peptic ulcer disease and mental illness. Here we review the contributions of each of these elements to infectious disease emergence and strategies for addressing the challenges of pathogen surveillance and discovery.","DNA microarrays may be used to identify microbial species present in environmental and clinical samples. However, automated tools for reliable species identification based on observed microarray hybridization patterns are lacking. We present an algorithm, E-Predict, for microarray-based species identification. E-Predict compares observed hybridization patterns with theoretical energy profiles representing different species. We demonstrate the application of the algorithm to viral detection in a set of clinical samples and discuss its relevance to other metagenomic applications.","AbstractAs reported by the World Health Organization, a novel coronavirus (2019-nCoV) was identified as the causative virus of Wuhan pneumonia of unknown etiology by Chinese authorities on 7 January, 2020. In this study, we developed a Bats-Hosts-Reservoir-People transmission network model for simulating the potential transmission from the infection source (probable be bats) to the human infection. Since the Bats-Hosts-Reservoir network was hard to explore clearly and public concerns were focusing on the transmission from a seafood market (reservoir) to people, we simplified the model as Reservoir-People transmission network model. The basic reproduction number (R0) was calculated from the RP model to assess the transmissibility of the 2019-nCoV.","Abstract SARS-CoV-2, the newly identified human coronavirus causing severe pneumonia pandemic, was probably originated from Chinese horseshoe bats. However, direct transmission of the virus from bats to humans is unlikely due to lack of direct contact, implying the existence of unknown intermediate hosts. Angiotensin converting enzyme 2 (ACE2) is the receptor of SARS-CoV-2, but only ACE2s of certain species can be utilized by SARS-CoV-2. Here, we evaluated and ranked the receptor-utilizing capability of ACE2s from various species by phylogenetic clustering and sequence alignment with the currently known ACE2s utilized by SARS-CoV-2. As a result, we predicted that SARS-CoV-2 tends to utilize ACE2s of various mammals, except murines, and some birds, such as pigeon. This prediction may help to screen the intermediate hosts of SARS-CoV-2.","Viral emergence as a result of zoonotic transmission constitutes a continuous public health threat. Emerging viruses such as SARS coronavirus, hantaviruses and henipaviruses have wildlife reservoirs. Characterising the viruses of candidate reservoir species in geographical hot spots for viral emergence is a sensible approach to develop tools to predict, prevent, or contain emergence events. Here, we explore the viruses of Eidolon helvum, an Old World fruit bat species widely distributed in Africa that lives in close proximity to humans. We identified a great abundance and diversity of novel herpes and papillomaviruses, described the isolation of a novel adenovirus, and detected, for the first time, sequences of a chiropteran poxvirus closely related with Molluscum contagiosum. In sum, E. helvum display a wide variety of mammalian viruses, some of them genetically similar to known human pathogens, highlighting the possibility of zoonotic transmission.","Camels are the most adapted species to the harsh conditions of arid/semi-arid rangelands of Ethiopia where pastoralism is the dominant mode of life and mobility is an inherent strategy to efficiently utilize the spatially and temporally distributed pasture and water resources. Usually, large numbers of camels and other domestic animals from many different herds/flocks congregate at watering sites, and this may create a perfect condition for disease transmission and spread among animals. The same water sources are also shared by multitudes of wild animals. Camel herd sizes per household range from few heads (five to ten) to several hundreds. Female camels account for more than 75% of the herd. Male camels are usually sold early as pack animals or for slaughter. Female camels may remain fertile up to 25 years, during which time they produce eight to ten calves. Camels are herded during daytime on communal rangelands. During night, they are kept in traditional kraals around homesteads. Br...","Bats are being increasingly recognized as reservoir hosts of highly pathogenic and zoonotic emerging viruses (Marburg virus, Nipah virus, Hendra virus, Rabies virus, and coronaviruses). While numerous studies have focused on the mentioned highly human-pathogenic bat viruses in tropical regions, little is known on similar human-pathogenic viruses that may be present in European bats. Although novel viruses are being detected, their zoonotic potential remains unclear unless further studies are conducted. At present, it is assumed that the risk posed by bats to the general public is rather low. In this review, selected viruses detected and isolated in Europe are discussed from our point of view in regard to their human-pathogenic potential. All European bat species and their roosts are legally protected and some European species are even endangered. Nevertheless, the increasing public fear of bats and their viruses is an obstacle to their protection. Educating the public regarding bat lys...","Severe acute respiratory syndrome coronavirus (SARS-CoV) and Middle East respiratory syndrome coronavirus (MERS-CoV) are two highly transmissible and pathogenic viruses that emerged in humans at the beginning of the 21st century. Both viruses likely originated in bats, and genetically diverse coronaviruses that are related to SARS-CoV and MERS-CoV were discovered in bats worldwide. In this Review, we summarize the current knowledge on the origin and evolution of these two pathogenic coronaviruses and discuss their receptor usage; we also highlight the diversity and potential of spillover of bat-borne coronaviruses, as evidenced by the recent spillover of swine acute diarrhoea syndrome coronavirus (SADS-CoV) to pigs.","Abstract Concept of \u201crisk analysis\u201d in food safety management needs to be understood to assess, manage, and communicate risks for proper implementation of food safety policies, food laws, and standards, in a country. Science-based approaches, which are an integral part of risk analysis to improve modern food safety systems, are intimately linked to processes such as good quality practices. Risk analysis strengthens the ability of traditional food safety systems to meet current challenges. Risk analysis is just one part of an effective food safety system. Other components of food safety system like food safety policies, food legislation, food inspection, laboratory analysis, epidemiological surveillance of food-borne diseases, monitoring systems for chemical and microbiological contamination in foods, and information, education and communication, also need to be improved continuously. \u201cFood safety controls\u201d are now being replaced by \u201crisk-based controls\u201d in all stages of food production...","Abstract Microbiology laboratories have traditionally relied upon phenotypic methods involving culture and biochemical testing to identify and characterize clinically important pathogens. These techniques have several disadvantages including poor sensitivity and long turn-around time. Molecular and mass spectroscopy techniques are rapidly changing infection diagnosis and management. Compared with conventional culture-based techniques, these modern approaches provide substantially more rapid and specific information on organism identification and on the presence of resistance mechanisms. These methods are expected to contribute substantially to enhancing antibiotic stewardship and to improving \u2018time to appropriate antibiotics\u2019, one of the most important factors in improving the prognosis of patients with life-threatening infections. This article gives an overview of some of the practical applications of these newer technologies.","Abstract A diet containing high levels of fruit has been associated with a lowered risk of chronic diseases as, in addition to their vitamin and mineral content, they also contain various compounds with health-protective effects, in particular antioxidant and antiinflammatory compounds. Wild plant species are of interest to the food industry because of their ability to replace synthetic chemicals and nutraceuticals; however, the nutritional, economical, and sociocultural values of some neglected and underutilized natural resources have not yet been fully exploited. Some of these less well-known and underutilized fruits, which have the potential to provide novel sources of health-promoting agents, are presented in this chapter (i.e., Asimina triloba (L.) Dunal, Crataegus azarolus L., Lycium barbarum L., Morus nigra L., and Amelanchier canadensis (L.) Medicus). Underutilized fruits could represent an opportunity for growers to gain access to these special markets where consumers place em...","The ongoing Coronavirus Disease 2019 (COVID-19) outbreak, originated in the end of 2019 in Wuhan, China, has claimed more than 2200 lives and posed a huge threat to global public health. The Chinese government has implemented control measures including setting up special hospitals and travel restriction to mitigate the spread. We propose conceptual models for the outbreak in Wuhan with the consideration of individual behavioural reaction and governmental actions, e.g., holiday extension, travel restriction, hospitalisation and quarantine. We employed the estimates of these two key components from the 1918 influenza pandemic in London, United Kingdom, incorporated zoonotic introductions and the emigration, then computed future trends and the reporting ratio. The model is concise in structure, and it successfully captures the course of the COVID-19 outbreak, and thus sheds light on understanding the trends of the outbreak.","The paper evaluates total inclusive costs of three public health approaches to address the COVID-19 epidemic in the US based on epidemiological projections in Ferguson et al (2020). We calculate and add costs of lost productivity and costs of mortality measured through the value of statistical life. We find that the aggressive approach which involves strict suppression measures and a drastic reduction of economic activity for three months with extensive testing and case tracking afterwards results in the lowest total costs for the society. The approach of doing no non-pharmaceutical measures results in the lowest total costs if the infection fatality rate falls below 0.15%.","The recent outbreak of COVID-19 in Wuhan turned into a public health emergency of international concern. With no antiviral drugs nor vaccines, and the presence of carriers without obvious symptoms, traditional public health intervention measures are significantly less effective. Here, we report the epidemiological and virological characteristics of the COVID-19 outbreak. Originated in bats, 2019-nCoV/ severe acute respiratory syndrome coronavirus (SARS-CoV)-2 likely experienced adaptive evolution in intermediate hosts before transfer to humans at a concentrated source of transmission. Similarities of receptor sequence binding to 2019-nCoV between humans and animals suggest a low species barrier for transmission of the virus to farm animals. We propose, based on the One Health model, that veterinarians and animal specialists should be involved in a cross-disciplinary collaboration in the fight against this epidemic.","AbstractThe ongoing outbreak of viral pneumonia in China and beyond is associated with a novel coronavirus, provisionally termed 2019-nCoV. This outbreak has been tentatively associated with a seafood market in Wuhan, China, where the sale of wild animals may be the source of zoonotic infection. Although bats are likely reservoir hosts for 2019-nCoV, the identity of any intermediate host facilitating transfer to humans is unknown. Here, we report the identification of 2019-nCoV related coronaviruses in pangolins (Manis javanica) seized in anti-smuggling operations in southern China. Metagenomic sequencing identified pangolin associated CoVs that belong to two sub-lineages of 2019-nCoV related coronaviruses, including one very closely related to 2019-nCoV in the receptor-binding domain. The discovery of multiple lineages of pangolin coronavirus and their similarity to 2019-nCoV suggests that pangolins should be considered as possible intermediate hosts for this novel human virus and sho...","","Severe acute respiratory syndrome (SARS) emerged in southern China in late 2002. It first spread within Guangdong Province and then to other parts of China. Via air travelers, it quickly reached various countries around the globe, causing several major hospital outbreaks. Within weeks, the causative agent, a previously unknown coronavirus (SARS-CoV), was identified, thanks to an unprecedented international effort led by the World Health Organization (WHO). Its origin was quickly traced to wild animals traded locally for culinary purposes. Masked palm civet and some other species seem to have acted as intermediate hosts. Since then, SARS-like coronaviruses were found in different bat species in China and elsewhere, and bats are now regarded as the wildlife reservoir for SARS-CoV. Fortunately, the SARS outbreak could be contained within months. Until July 2003, it had caused 8096 cases, with 774 deaths. Once adequate measures such as isolating patients and quarantining their contacts wer...","Background: COVID-19 has become a worldwide pandemic. It is caused by a novel coronavirus named SARS-CoV-2 with elusive origin. SARS-CoV-2 infects mammalian cells by binding to ACE2, a transmembrane protein. Therefore, the conservation of ACE2 and its expression pattern across mammalian species, which are yet to be comprehensively investigated, may provide valuable insights into tracing potential hosts of SARS-CoV-2. Methods: We analyzed gene conservation of ACE2 across mammals and collected more than 140 transcriptome datasets from human and common mammalian species, including presumed hosts of SARS-CoV-2 and other animals in close contact with humans. In order to enable comparisons across species and tissues, we used a unified pipeline to quantify and normalize ACE2 expression levels. Results: We first found high conservation of ACE2 genes among common mammals at both DNA and peptide levels, suggesting that a broad range of mammalian species can potentially be the hosts of SARS-CoV-2...","Abstract Medicine shortages have been spreading in European countries. In many cases, the unavailability of medicinal products has a substantial impact on the capability of National Healthcare Systems in ensuring the continuity of care. Shortages originate from multifactorial causes. In particular, they can be due to supply-related factors (e.g., manufacturing issues, regulatory issues, logistics, distribution) and demand-related ones (e.g., fluctuating drug demand, parallel market, tendering, price and reimbursement policies). However, some extraordinary geopolitical events (e.g., Brexit) may also affect medicines\u2019 availability. The capability of European Regulatory Authorities and other stakeholders, which are involved in the pharmaceutical distribution chain and the healthcare assistance services, to define suitable problem-solving strategies has been limited for years by the fragmentation of the European regulatory framework, starting from the lack of a univocal definition of a med...","Abstract Severe acute respiratory syndrome (SARS) emerged in southern China in late 2002. It first spread within Guangdong Province and then to other parts of China. Via air travelers, it quickly reached various countries around the globe, causing several major hospital outbreaks. Within weeks, the causative agent, a previously unknown coronavirus (SARS-CoV), was identified, thanks to an unprecedented international effort led by the World Health Organization (WHO). Its origin was quickly traced to wild animals traded locally for culinary purposes. Masked palm civet and some other species seem to have acted as intermediate hosts. Since then, SARS-like coronaviruses were found in different bat species in China and elsewhere, and bats are now regarded as the wildlife reservoir for SARS-CoV. Fortunately, the SARS outbreak could be contained within months. Until July 2003, it had caused 8096 cases, with 774 deaths. Once adequate measures such as isolating patients and quarantining their con...","BACKGROUND: Coronavirus can cross the species barrier and infect humans with a severe respiratory syndrome. SARS-CoV-2 with potential origin of bat is still circulating in China. In this study, a prediction model is proposed to evaluate the infection risk of non-human-origin coronavirus for early warning. METHODS: The spike protein sequences of 2666 coronaviruses were collected from 2019 Novel Coronavirus Resource (2019nCoVR) Database of China National Genomics Data Center on Jan 29, 2020. A total of 507 human-origin viruses were regarded as positive samples, whereas 2159 non-human-origin viruses were regarded as negative. To capture the key information of the spike protein, three feature encoding algorithms (amino acid composition, AAC; parallel correlation-based pseudo-amino-acid composition, PC-PseAAC and G-gap dipeptide composition, GGAP) were used to train 41 random forest models. The optimal feature with the best performance was identified by the multidimensional scaling method, ...","Exotic wildlife can act as reservoirs of diseases that are endemic in the area or can be the source of new emerging diseases through interspecies transmission. The recent emergence of severe acute respiratory syndrome-associated coronavirus (SARS-CoV) highlights the importance of virus surveillance in wild animals. Here, we report the identification of a novel bat coronavirus through surveillance of coronaviruses in wildlife. Analyses of the RNA sequence from the ORF1b and S-gene regions indicated that the virus is a group 1 coronavirus. The virus was detected in fecal and respiratory samples from three bat species (Miniopterus spp.). In particular, 63% (12 of 19) of fecal samples from Miniopterus pusillus were positive for the virus. These findings suggest that this virus might be commonly circulating in M. pusillus in Hong Kong.","Middle East Respiratory Syndrome-Coronavirus (MERS-CoV) is a novel coronavirus discovered in 2012 and is responsible for acute respiratory syndrome in humans. Though not confirmed yet, multiple surveillance and phylogenetic studies suggest a bat origin. The disease is heavily endemic in dromedary camel populations of East Africa and the Middle East. It is unclear as to when the virus was introduced to dromedary camels, but data from studies that investigated stored dromedary camel sera and geographical distribution of involved dromedary camel populations suggested that the virus was present in dromedary camels several decades ago. Though bats and alpacas can serve as potential reservoirs for MERS-CoV, dromedary camels seem to be the only animal host responsible for the spill over human infections."],"title":["Reservoirs and vectors of emerging viruses","Strong evolutionary convergence of receptor-binding protein spike between COVID-19 and SARS-related coronaviruses","Non-invasive surveys of mammalian viruses using environmental DNA","Are pangolins the intermediate host of the 2019 novel coronavirus (2019-nCoV) ?","Bats and Viruses: Emergence of Novel Lyssaviruses and Association of Bats with Viral Zoonoses in the EU","A strategy to prevent future pandemics similar to the 2019-nCoV outbreak","Inferring the hosts of coronavirus using dual statistical models based on nucleotide composition","Detection of new genetic variants of Betacoronaviruses in Endemic Frugivorous Bats of Madagascar","Coronavirus and paramyxovirus in bats from Northwest Italy","Studying immunity to zoonotic diseases in the natural host \u2014 keeping it real","Bats as reservoirs of severe emerging infectious diseases","Development of a Whole-Virus ELISA for Serological Evaluation of Domestic Livestock as Possible Hosts of Human Coronavirus NL63","A COVID-19 Infection Risk Model for Frontline Health Care Workers","A Review of Zoonotic Pathogens of Dromedary Camels","Differential stepwise evolution of SARS coronavirus functional proteins in different host species","Jumping species\u2014a mechanism for coronavirus persistence and survival","Viral Diversity, Prey Preference, and Bartonella Prevalence in Desmodus rotundus in Guatemala","Novel coronaviruses and astroviruses in bats","Coronaviruses in Avian Species \u2013 Review with Focus on Epidemiology and Diagnosis in Wild Birds","Pathogen genomic surveillance elucidates the origins, transmission and evolution of emerging viral agents in China","Coronaviruses post-SARS: Update on replication and pathogenesis","Bats and their virome: an important source of emerging viruses capable of infecting humans","Preventing intra-hospital infection and transmission of COVID-19 in healthcare workers","Emerging novel coronavirus (2019-nCoV)\u2014current scenario, evolutionary perspective based on genome analysis and recent developments","Potential Factors Influencing Repeated SARS Outbreaks in China","The infectious aetiology of disease: the search for new agents","Bat origin of human coronaviruses","The changing face of pathogen discovery and surveillance","E-Predict: a computational strategy for species identification based on observed DNA microarray hybridization patterns","A mathematical model for simulating the transmission of Wuhan novel Coronavirus","Predicting the angiotensin converting enzyme 2 (ACE2) utilizing capability as the receptor of SARS-CoV-2","Metagenomic study of the viruses of African straw-coloured fruit bats: Detection of a chiropteran poxvirus and isolation of a novel adenovirus","Camel production systems in Ethiopia: a review of literature with notes on MERS-CoV risk factors","European Bats as Carriers of Viruses with Zoonotic Potential","Origin and evolution of pathogenic coronaviruses","Chapter 5 Role of risk analysis and risk communication in food safety management","Molecular-based diagnostics, including future trends","Chapter 9 Nutraceuticals in Alternative and Underutilized Fruits as Functional Food Ingredients: Ancient Species for New Health Needs","A conceptual model for the outbreak of Coronavirus disease 2019 (COVID-19) in Wuhan, China with individual reaction and governmental action","Inclusive Costs of NPI Measures for COVID-19 Pandemic: Three Approaches","COVID-19: Epidemiology, Evolution, and Cross-Disciplinary Perspectives","Identification of 2019-nCoV related coronaviruses in Malayan pangolins in southern China","Letter from the Editor: Occupational skin disease among healthcare workers during the Coronavirus (COVID-19) epidemic","SARS","Atlas of ACE2 gene expression in mammals reveals novel insights in transmission of SARS-Cov-2","New regulatory strategies to manage medicines shortages in Europe","SARS","Using the spike protein feature to predict infection risk and monitor the evolutionary dynamic of coronavirus","Identification of a Novel Coronavirus in Bats","Middle East Respiratory Syndrome Coronavirus (MERS-CoV) origin and animal reservoir"],"url":["https://doi.org/10.1016/j.coviro.2013.02.002","https://doi.org/10.1101/2020.03.04.975995","https://doi.org/10.1101/2020.03.26.009993","https://doi.org/10.1101/2020.02.18.954628","https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6473451/","https://doi.org/10.1016/j.bsheal.2020.01.003","https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4660426/","https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4404003/","https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5741894/","https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7098194/","https://doi.org/10.1016/j.virusres.2015.05.006","https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6356407/","https://doi.org/10.1101/2020.03.27.20045336","https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7087575/","https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2676248/","https://doi.org/10.1016/j.coviro.2017.01.002","https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5164864/","https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7091142/","https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6296008/","https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7088571/","https://www.nature.com/articles/nrmicro2147.pdf","https://doi.org/10.1016/j.coviro.2012.11.006","https://doi.org/10.1016/j.shaw.2020.03.001","https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7054940/","https://doi.org/10.3390/ijerph17051633","https://doi.org/10.1383/medc.33.3.37.61122","https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4687304/","http://europepmc.org/articles/pmc4098826?pdf=render","https://www.ncbi.nlm.nih.gov/pmc/articles/PMC1242213/","https://doi.org/10.1101/2020.01.19.911669","https://doi.org/10.1016/j.micinf.2020.03.003","https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3667569/","https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7099212/","https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4147689/","https://doi.org/10.1038/s41579-018-0118-9","https://doi.org/10.1016/b978-0-12-801773-9.00005-4","https://doi.org/10.1016/j.mpmed.2013.08.012","https://doi.org/10.1016/b978-0-12-811446-9.00009-5","https://doi.org/10.1016/j.ijid.2020.02.058","https://doi.org/10.1101/2020.03.26.20044552","https://doi.org/10.1016/j.molmed.2020.02.008","https://doi.org/10.1101/2020.02.13.945485","https://doi.org/10.1016/j.jaad.2020.03.012","https://doi.org/10.1016/b978-0-444-52272-6.00624-3","https://doi.org/10.1101/2020.03.30.015644","https://doi.org/10.1016/j.ijpharm.2020.119171","https://doi.org/10.1016/b978-0-444-63951-6.00624-0","https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7093988/","https://jvi.asm.org/content/jvi/79/4/2001.full.pdf","https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4891877/"],"x":[0.5051993322455406,0.3210243670546203,0.3828295319218313,0.38215487206284354,0.5029554736165383,0.7404585049962339,0.3563613523625898,0.3747959452219079,0.36001790706182163,0.513509423139882,0.4828280779259371,0.38936994121288526,0.9754658655442296,0.4843439875302776,0.35992613527934564,0.5084409577299471,0.38321398342291924,0.5023667998180098,0.4914884319639943,0.6214775569681427,0.5239228725644361,0.5023235942363486,0.9056913981761415,0.6333575709213155,0.6417060817477993,0.27881110516815705,0.46269695718143483,0.6645972563227168,0.29159873246719076,0.5925834673980472,0.31095546710309785,0.48942423333311086,0.4783273986331775,0.5019683523455425,0.4639033846963747,0.958434213004692,0.3051396996805344,0.4810365791791044,0.7878813376226163,0.8334624589283985,0.6709070365028911,0.3811608806972735,0.556963452206456,0.6412343523058565,0.3592880630871093,0.8951805281446267,0.641293117372817,0.35549947432068163,0.45547836472122344,0.4617019570972068],"y":[0.4073498520387681,0.30981928275857445,0.4416432432917195,0.41777174146757134,0.3981034028864027,0.7091001742179927,0.38444397515936507,0.4266225073813259,0.4131857282189126,0.39935744245105204,0.40579181324599556,0.5222384341642189,0.653409555190751,0.6473387752292473,0.38904228637339955,0.3707151099964367,0.4400970314106825,0.4048070992140289,0.3940406266346807,0.38168706983498,0.3633963998877402,0.3971169552606186,0.5134122769148535,0.537970543385068,0.5358522299910348,0.5496438328773356,0.38412104737880975,0.4152191155258915,0.49607300070791743,0.5205079418970661,0.3281528650648429,0.38023562250425247,0.6550567694673854,0.40018810857237946,0.38264396632585873,0.6567617398864173,0.562071644975703,0.6899147206454541,0.6989858912749624,0.7575834087831684,0.4662117562473227,0.4140974548587086,0.5108667499761881,0.5559528046273602,0.3973327085604316,0.6857397701411039,0.5558710001006321,0.3854865553887408,0.3987633827803526,0.42633028494897846]},"selected":{"id":"28121"},"selection_policy":{"id":"28143"}},"id":"28049","type":"ColumnDataSource"},{"attributes":{"data":{"authors":["Marty, Aileen Maria; Jones, Malcolm K.","Hu, Ben; Ge, Xingyi; Wang, Lin-Fa; Shi, Zhengli","Ye, Zi-Wei; Yuan, Shuofeng; Yuen, Kit-San; Fung, Sin-Yee; Chan, Chi-Ping; Jin, Dong-Yan","Smith, Ina; Wang, Lin-Fa","Markotter, Wanda; Geldenhuys, Marike; Jansen van Vuren, Petrus; Kemp, Alan; Mortlock, Marinda; Mudakikwa, Antoine; Nel, Louis; Nziza, Julius; Paweska, Janusz; Weyer, Jacqueline","Daszak, Peter; Olival, Kevin J.; Li, Hongying","Daszak, Peter; Olival, Kevin J.; Li, Hongying","Mackenzie, John S; Jeggo, Martyn","Ye, Zi-Wei; Yuan, Shuofeng; Yuen, Kit-San; Fung, Sin-Yee; Chan, Chi-Ping; Jin, Dong-Yan","El-Duah, Philip; Meyer, Benjamin; Sylverken, Augustina; Owusu, Michael; Gottula, Lina Theresa; Yeboah, Richmond; Lamptey, Jones; Frimpong, Yaw Oppong; Burimuah, Vitus; Folitse, Raphael; Agbenyega, Olivia; Oppong, Samuel; Adu-Sarkodie, Yaw; Drosten, Christian","Kock, Richard A.; Karesh, William B.; Veas, Francisco; Velavan, Thirumalaisamy P.; Simons, David; Mboera, Leonard E. G.; Dar, Osman; Arruda, Li\u00e3 B\u00e1rbara; Zumla, Alimuddin","Kock, Richard A.; Karesh, William B.; Veas, Francisco; Velavan, Thirumalaisamy P.; Simons, David; Mboera, Leonard E. G.; Dar, Osman; Arruda, Li\u00e3 B\u00e1rbara; Zumla, Alimuddin","Li, Jin-Yan; You, Zhi; Wang, Qiong; Zhou, Zhi-Jian; Qiu, Ye; Luo, Rui; Ge, Xing-Yi","Wang, Liang; Su, Shuo; Bi, Yuhai; Wong, Gary; Gao, George F.","Blacklaws, B.A.; Daly, J.M.","Sun, Jiumeng; He, Wan-Ting; Wang, Lifang; Lai, Alexander; Ji, Xiang; Zhai, Xiaofeng; Li, Gairu; Suchard, Marc A.; Tian, Jin; Zhou, Jiyong; Veit, Michael; Su, Shuo","Ye, Zi-Wei; Yuan, Shuofeng; Yuen, Kit-San; Fung, Sin-Yee; Chan, Chi-Ping; Jin, Dong-Yan","Han, Hui-Ju; Wen, Hong-ling; Zhou, Chuan-Min; Chen, Fang-Fang; Luo, Li-Mei; Liu, Jian-wei; Yu, Xue-Jie","Xiaojun Li; Elena E. Giorgi; Manukumar Honnayakanahalli Marichann; Brian Foley; Chuan Xiao; Xiang-peng Kong; Yue Chen; Bette Korber; Feng Gao","Fung, Sin-Yee; Yuen, Kit-San; Ye, Zi-Wei; Chan, Chi-Ping; Jin, Dong-Yan","Kostoff, Ronald N.; Morse, Stephen A.","Walls, Alexandra C.; Park, Young-Jun; Tortorici, M. Alejandra; Wall, Abigail; McGuire, Andrew T.; Veesler, David","Mackenzie, John S; Smith, David W","Berger, A.; Preiser, W.","Huang, Canping; Liu, William J.; Xu, Wen; Jin, Tao; Zhao, Yingze; Song, Jingdong; Shi, Yi; Ji, Wei; Jia, Hao; Zhou, Yongming; Wen, Honghua; Zhao, Honglan; Liu, Huaxing; Li, Hong; Wang, Qihui; Wu, Ying; Wang, Liang; Liu, Di; Liu, Guang; Yu, Hongjie; Holmes, Edward C.; Lu, Lin; Gao, George F.","Daszak, Peter; Olival, Kevin J.; Li, Hongying","Daszak, Peter; Olival, Kevin J.; Li, Hongying","Shi, Z.; Wang, L.-F.","Smith, Ina; Wang, Lin-Fa","Burdet, C.; Gu\u00e9gan, J.-F.; Duval, X.; Le Tyrant, M.; Bergeron, H.; Manuguerra, J.-C.; Raude, J.; Leport, C.; Zylberman, P.","Ralph, R.; Lew, J.; Zeng, T.; Francis, M.; Xue, B.; Roux, M.; Ostadgavahi, A. T.; Rubino, S.; Dawe, N. J.; Al-Ahdal, M. N.; Kelvin, D. J.; Richardson, C. D.; Kindrachuk, J.; Falzarano, D.; Kelvin, A. A.","Rizzo, Francesca; Edenborough, Kathryn M.; Toffoli, Roberto; Culasso, Paola; Zoppi, Simona; Dondo, Alessandro; Robetto, Serena; Rosati, Sergio; Lander, Angelika; Kurth, Andreas; Orusa, Riccardo; Bertolotti, Luigi; Mandola, Maria Lucia","Kohl, Claudia; Kurth, Andreas","Wray, Amy K.; Olival, Kevin J.; Mor\u00e1n, David; Lopez, Maria Renee; Alvarez, Danilo; Navarrete-Macias, Isamara; Liang, Eliza; Simmons, Nancy B.; Lipkin, W. Ian; Daszak, Peter; Anthony, Simon J.","Peters, Alexandra; Vetter, Pauline; Guitart, Chlo\u00e9; Lotfinejad, Nasim; Pittet, Didier","Jacobsen, Kathryn H","El-Duah, Philip; Meyer, Benjamin; Sylverken, Augustina; Owusu, Michael; Gottula, Lina Theresa; Yeboah, Richmond; Lamptey, Jones; Frimpong, Yaw Oppong; Burimuah, Vitus; Folitse, Raphael; Agbenyega, Olivia; Oppong, Samuel; Adu-Sarkodie, Yaw; Drosten, Christian","Louten, Jennifer","Mackenzie, John S; Jeggo, Martyn","Niccolo Alfano; Anisha Dayaram; Jan Axtner; Kyriakos Tsangaras; Marie-Louise Kampmann; Azlan Mohamed; Seth Timothy Wong; M. Thomas P. Gilbert; Andreas Wilting; Alex Daivd Greenwood","Rizzo, Francesca; Edenborough, Kathryn M.; Toffoli, Roberto; Culasso, Paola; Zoppi, Simona; Dondo, Alessandro; Robetto, Serena; Rosati, Sergio; Lander, Angelika; Kurth, Andreas; Orusa, Riccardo; Bertolotti, Luigi; Mandola, Maria Lucia","Kohl, Claudia; Kurth, Andreas","Berger, A.; Preiser, W.","Kohl, Claudia; Kurth, Andreas","Shi, Z.; Wang, L.-F.","Koopmans, Marion","Markotter, Wanda; Geldenhuys, Marike; Jansen van Vuren, Petrus; Kemp, Alan; Mortlock, Marinda; Mudakikwa, Antoine; Nel, Louis; Nziza, Julius; Paweska, Janusz; Weyer, Jacqueline","Mackenzie, John S; Smith, David W","Maciej F Boni; Philippe Lemey; Xiaowei Jiang; Tommy Tsan-Yuk Lam; Blair Perry; Todd Castoe; Andrew Rambaut; David L Robertson","Mackenzie, John S; Jeggo, Martyn"],"color":["#ef8a62","#ef8a62","#ef8a62","#ef8a62","#ef8a62","#ef8a62","#ef8a62","#ef8a62","#ef8a62","#ef8a62","#ef8a62","#ef8a62","#ef8a62","#ef8a62","#ef8a62","#ef8a62","#ef8a62","#ef8a62","#ef8a62","#ef8a62","#ef8a62","#ef8a62","#ef8a62","#ef8a62","#ef8a62","#ef8a62","#ef8a62","#ef8a62","#ef8a62","#ef8a62","#ef8a62","#ef8a62","#ef8a62","#ef8a62","#ef8a62","#ef8a62","#ef8a62","#ef8a62","#ef8a62","#ef8a62","#ef8a62","#ef8a62","#ef8a62","#ef8a62","#ef8a62","#ef8a62","#ef8a62","#ef8a62","#ef8a62","#ef8a62"],"date":["2020-02-14","2015-12-22","2020-03-15","2013-02-28","2019-07-02","2020","2020","2013-04-30","2020-03-15","2019-01-09","2020","2020","2020-03-31","2018-06-30","2018-03-31","2020-03-21","2020-03-15","2015-07-02","2020-03-22","2020-03-14","2010-05-30","2020-03-09","2020-03-17","2011-12-31","2016-09-27","2020","2020","2017-12-31","2013-02-28","2018-02-28","2020","2017-12-22","2014-08-13","2016-09-22","2020-03-05","2020-03-18","2019-01-09","2016-12-31","2013-04-30","2020-03-29","2017-12-22","2014-08-13","2011-12-31","2014-08-13","2017-12-31","2013-02-28","2019-07-02","2020-03-17","2020-03-31","2013-04-30"],"doc":["ork04nx8","d2knbzhl","lfndq85x","v8uzd2f3","muxrxvyo","h5sox8bq","h5sox8bq","0g5079y1","lfndq85x","tjp33y35","oxs4o9xe","oxs4o9xe","epcel2ez","wuegn0jg","kdlcaium","9il7coyk","lfndq85x","3lg1bzsi","7v5aln90","89fol3pq","2rrlt5z9","trlni6iq","n8e2n30b","kromgrnu","zqf351sv","h5sox8bq","h5sox8bq","apc0lm5e","v8uzd2f3","vov6o1jx","rxrlbw60","llxqcikz","4rxradwz","9ojdhxwk","0be4wta5","mnhoyz35","tjp33y35","3p542sqs","0g5079y1","nil1vv6h","llxqcikz","4rxradwz","yvsyo4l9","4rxradwz","apc0lm5e","ulsk50kb","muxrxvyo","n8e2n30b","h2uc7ria","0g5079y1"],"journal":["One Health","Virol J","Int J Biol Sci","Current Opinion in Virology","Trop Med Infect Dis","Biosafety and Health","Biosafety and Health","Current Opinion in Virology","Int J Biol Sci","Viruses","The Lancet Planetary Health","The Lancet Planetary Health","Microbes and Infection","Trends in Microbiology","The Veterinary Journal","Trends in Molecular Medicine","Int J Biol Sci","Virus Research","","Emerg Microbes Infect","Scientometrics","Cell","Microbiol Aust","Encyclopedia of Environmental Health","PLoS Pathog","Biosafety and Health","Biosafety and Health","Genetics and Evolution of Infectious Diseases","Current Opinion in Virology","Revue d'\u00c9pid\u00e9miologie et de Sant\u00e9 Publique","Journal of Infection in Developing Countries","BMC Vet Res","Viruses","Ecohealth","Journal of Hospital Infection","The Lancet","Viruses","Essential Human Virology","Current Opinion in Virology","","BMC Vet Res","Viruses","Encyclopedia of Environmental Health","Viruses","Genetics and Evolution of Infectious Diseases","Current Opinion in Virology","Trop Med Infect Dis","Microbiol Aust","","Current Opinion in Virology"],"rad":[0.02,0.018634495237975324,0.018185997346867988,0.017318460032747247,0.016525119253661832,0.015101808214533306,0.015101808214533306,0.013415811938753752,0.011159376158057885,0.01066855763132626,0.010513775378799614,0.010513775378799614,0.009924761672861774,0.009443014866748443,0.009209726296673701,0.009115032345159636,0.008861345517156307,0.008182196201591604,0.007448051521408278,0.007053714841634818,0.007033037535578176,0.006953072170089342,0.006664072921591528,0.006610878386572429,0.006583334525331117,0.006503419820860379,0.006503419820860379,0.006398485386394025,0.006339368162897459,0.006329979297850716,0.006311242290194396,0.006289643998849788,0.005822150346727147,0.005585376645458268,0.005380251317499349,0.005228937809570394,0.005126813388431073,0.004853255938392314,0.004553978947204032,0.004279701339492212,0.004153818023664921,0.003982918738398669,0.003888234396321008,0.0038239437392204832,0.0036746659055472484,0.0035205615469087064,0.0034796748927091886,0.00327198484352,0.0030921362256819147,0.003],"scr":[1.0,0.92,0.893,0.842,0.796,0.712,0.712,0.613,0.48,0.451,0.442,0.442,0.407,0.379,0.365,0.36,0.345,0.305,0.262,0.238,0.237,0.233,0.216,0.212,0.211,0.206,0.206,0.2,0.196,0.196,0.195,0.194,0.166,0.152,0.14,0.131,0.125,0.109,0.091,0.075,0.068,0.058,0.052,0.048,0.04,0.031,0.028,0.016,0.005,0.0],"text":["One Health approaches attempt to strategize the coordinated efforts of multiple overlapping disciplines [19], including environmental surveillance and environmental health. Primary components of the approach lie in animal health and environmental aspects. At the time of writing, the host from which the SARS-CoV-2 entered the human population is unknown although the suspicion is that food markets are likely sources for the original spillover. While the search for the natural host highly implicates bats [21], search for the intermediary host, if any, is ongoing with the suggestions of the pangolin as a host far from certain. While it is premature to implicate any one particular urban source or natural host, the ensuing search will give insight into pathogens with potential to cross over into human transmission. This approach of environmental surveillance forms part of the PREDICT strategy [20] for detecting viruses with potential for spillover into human.","With human activity increasingly overlapping the habitats of bats, diseases outbreaks resulted from spillover of bat coronaviruses will continue to occur in the future despite the fact that direct transmission of bat coronaviruses to humans appears to be rare. To better prepare ourselves in predicting and preventing the next emergence of a coronavirus disease, it is necessary to maintain our vigilance in long-term coronavirus surveillance studies in bats as well as in other wildlife and livestock. Combined with other laboratory-based studies such as receptor specificity, pathogenesis and animal infection, a focus on continued surveillance will help us to improve risk assessment as well as to reveal the potential intermediate hosts that may play an important role in the interspecies transmission of various known and as yet unknown bat coronaviruses.","A variety of zoonotic CoVs are circulating in the wild. Particularly, bat CoVs with zoonotic potential are so diverse. There are plenty of opportunities that these zoonotic CoVs evolve and recombine, resulting in the emergence of new CoVs that are more transmissible and/or deadly in humans in future. The culture of eating wild animals in some places of China should be abandoned to reduce unnecessary contact between humans and animals. With the ordeals of SARS, MERS and COVID-19, a better preparedness and response plan should be in place. In fact, many viruses have existed in the planet for a very long time. They stay in their own natural reservoirs until there is a chance for spillover. Although bats have many features that favours the spreading of viruses, the chance for humans to be in contact with bats and other wildlife species can be minimized if people are educated to stay away from them. Continuous surveillance in mammals is necessary for better understanding of the ecology of CoVs and their natural hosts, which will prove useful in preventing animal-to-human transmission and future outbreaks. To conclude, the most effective way to prevent viral zoonosis is for humans to stay away from the ecological niches of the natural reservoirs of the zoonotic viruses.","There are many potential drivers that can contribute to a spillover of zoonotic bat-borne disease (Table 1 and Figure 1 ). These factors can be extrinsic and/or intrinsic. Extrinsic factors such as environmental and anthropogenic stressors can have an effect on the ecology of disease in bats. Environmental stressors such as climatic events (typhoons/cyclones and droughts) that destroy habitat and food resources have been hypothesized to have an impact on the health of bats [40] [41] [42] [43] 44 ]. In addition, human activities are selecting for some species of bats that are synanthrophic, and so are benefiting from living close to humans, thereby increasing their numbers and the risk of transmission of disease to humans [45] . Human activities are artificially increasing animal densities by changing the land use and this is increasing the contact between humans, domesticated animals and bats [46] . In the cases of Hendra virus, Nipah virus and Menangle virus, domesticated animals are the amplifying hosts of these zoonotic viruses [15, 47, 48] . Habitat change such as deforestation force changes in roosting sites and can lead to alterations in the population density and the migratory patterns of bats. These anthropogenic activities may be impacting our ecosystem in such a way that the equilibrium is disturbed and spillover of zoonotic viruses readily occur [49] [50] [51] .","The detection of highly similar viral sequences from bats in Rwanda and Kenya, which are more than 1000 km apart, can be explained by either the phenomena of metapopulations or the hypothesis of co-evolution of paramyxoviruses with their bat hosts [3,52,53]. RNA viruses have an exceptionally high mutation rate commonly associated with quasi-species populations and with the potential to cross the species barrier, among others. This is evident in the high diversity of paramyxoviruses described to date and the wide host range associated with these viruses [3,28,36,54]. Additionally, due to the emergence of SARS, MERS and SADS, it is widely accepted that coronaviruses are capable of readily adapting to new hosts [5]. The Rwandan caves are considered an ecotourism site and guano is also mined on a small scale, providing an ideal bat-human interface. Several insectivorous bat species co-roost with the Egyptian fruit bats in these caves and future studies should investigate viral sharing. The Egyptian fruit bat also uses these caves as a maternity roost and studies have shown that increased viral shedding is linked to reproductive cycles [37]. Longitudinal biosurveillance studies can therefore identify high risk periods in the future. As such, the detection of a SARS-related bat coronavirus potentially circulating within the Rhinolophus population and a Henipavirus-related paramyxovirus in R. aegyptiacus in the Ruhengeri region may merit further investigation to determine exposure, and the potential for spill-over events to occur. To date, emphasis of paramyxovirus surveillance has mostly been placed on fruit bats, the Henipavirus genus and related viruses due to the association of other henipavirus species with zoonotic events [30,31]. However, research regarding the zoonotic potential of the insectivorous bat-associated viruses is still lacking. One major aim regarding surveillance of wildlife populations is to identify potential zoonotic agents and to evaluate any threat to the public as well as domestic animal health. Though these bat-borne viruses are unlikely to pose a significant threat, it still merits continued monitoring of the chiropteran species within these caves as well as mammalian species that inhabit the surrounding area.","The wildlife trade has clearly played a role in the emergence of 2019-nCoV, as well as previous diseases in China (SARS) and across the world (e.g. monkeypox in the USA, Ebola in Africa, salmonellosis in the USA and Europe). China's response in the current outbreak was swift and broad: The index market was closed down immediately and once the virus spread, the wildlife trade was banned temporarily in certain provinces, then nationally. Our recent behavioral risk investigations in China identified low levels of environmental biosecurity and high levels of human-animal contact as key risk factors for zoonotic disease emergence, particularly in local wet and animal markets [23] . While the current wildlife trade bans may help disease control at this moment, to prevent future disease emergence, market biosecurity needs to be improved regarding the hygiene and sanitation facilities and regulations, and the source of animals traded at the market. From a viral emergence perspective, farmed animals are likely a lower risk than wild-caught animals. However, given the size of the wildlife farming industry in China, routine disease surveillance and veterinary care at farms and in transport to markets would need to be improved. The connection between this outbreak and wildlife trade have triggered a strong public opinion against wildlife consumption, and scientists have jointly called for an urgent amendment of the Wildlife Protection Law to standardize and manage wildlife trade as a public health and security issue. It requires collaboration among the State Forestry and Grassland Administration, Ministry of Agriculture and Rural Affairs, State Administration for Market Regulation and public health authorities to tackle this issue as a long-term goal.","The wildlife trade has clearly played a role in the emergence of 2019-nCoV, as well as previous diseases in China (SARS) and across the world (e.g. monkeypox in the USA, Ebola in Africa, salmonellosis in the USA and Europe). China's response in the current outbreak was swift and broad: The index market was closed down immediately and once the virus spread, the wildlife trade was banned temporarily in certain provinces, then nationally. Our recent behavioral risk investigations in China identified low levels of environmental biosecurity and high levels of human-animal contact as key risk factors for zoonotic disease emergence, particularly in local wet and animal markets [23] . While the current wildlife trade bans may help disease control at this moment, to prevent future disease emergence, market biosecurity needs to be improved regarding the hygiene and sanitation facilities and regulations, and the source of animals traded at the market. From a viral emergence perspective, farmed animals are likely a lower risk than wild-caught animals. However, given the size of the wildlife farming industry in China, routine disease surveillance and veterinary care at farms and in transport to markets would need to be improved. The connection between this outbreak and wildlife trade have triggered a strong public opinion against wildlife consumption, and scientists have jointly called for an urgent amendment of the Wildlife Protection Law to standardize and manage wildlife trade as a public health and security issue. It requires collaboration among the State Forestry and Grassland Administration, Ministry of Agriculture and Rural Affairs, State Administration for Market Regulation and public health authorities to tackle this issue as a long-term goal.","Before the emergence of SARS, there had been a growing awareness of the importance of bats as reservoirs or hosts of novel diseases [14 ,15] . This is not unexpected given that they constitute 20% of known mammalian species, have unique and diverse lifestyles including the ability to fly, they often have gregarious social structures achieving incredible abundance and densities, some cave-dwelling bats reaching up to 500 individuals per square foot, and they have long life spans. Importantly they also frequently live in very close proximity to humans, often interact closely with livestock and other domestic animals that are potential intermediate hosts for human pathogens, and with habitat loss due to land changes, are therefore very much at the wildlife-human interface [16] . Bats are grouped into two suborders, Megachiroptera and Microchiroptera; the former comprises a single family, Pteropididae, containing 42 genera and 166 species of fruit bats and flying foxes which do not use echolocation; and the latter comprises 18 families of 135 genera and 917 species, most of which use echolocation. Thus bats provide a rich and diverse source of potential reservoirs. Prior to SARS, Pteropid fruit bats had been found to be the reservoirs of a number of novel viruses able to infect humans, including two new paramyxoviruses, Hendra [17, 18] . This latter virus, which is closely related to classical rabies virus, was also found in at least one species of insectivorous bat [24,25], but has not crossed into terrestrial wildlife or domestic animal hosts. Thus, it is clear that RNA viruses associated with Old World fruit bats pose zoonotic disease threats of high public health significance. Insectivorous bats have also been strong candidates as potential reservoirs, as demonstrated by their role as reservoirs of rabies and rabies-like Lyssaviruses in Europe, Africa and elsewhere in the Old World, and in the Americas. It was therefore not surprising that bats were the major initial target in the search for the natural reservoirs of SARS coronavirus (SARS-CoV).","Several pieces in the puzzle of the zoonotic origin of SARS-CoV-2 are still missing. First, if bats transmit an ancestral virus of SARS-CoV-2 to pangolins, it will be of interest to see under what circumstances bats and pangolins could share the same ecological niche. Second, if bats play a more direct role in human transmission, how humans get into contact with bats should be determined. Third, if a third mammal acts as the true intermediate host, how it interacts with the different species including humans, bats and pangolins has to be clarified. Finally, since many mammals including domestic animals might be susceptible to SARS-CoV-2, both surveillance and experimental infection should be conducted. Should it be a bat, a pangolin or another mammal, it is expected that SARS-CoV-2 or its parental viruses that are almost identical will be identified in its natural hosts in future. Continued investigations in this area will elucidate the evolutionary pathway of SARS-CoV-2 in animals, with important implications in the prevention and control of COVID-19 in humans.","The importance of cronaviruses as emerging zoonotic viruses became evident after the international public health threat caused by severe acute respiratory syndrome coronavirus (SARS-CoV) in 2002/2003 [1]. Thereafter, there have been several studies that looked for novel coronaviruses aimed at assessing their zoonotic potential [2,3,4,5]. Coronaviruses are members of the order Nidovirales and family Coronaviridae which are made up of single-stranded positive sense RNA genomes and infect both mammalian and avian hosts. They are divided into four genera namely Alphacoronavirus, Betacoronavirus, Gammacoronavirus, and Deltacoronavirus [6,7]. In 2003, a coronavirus belonging to the Alphacoronavirus genus was discovered in an infant in the Netherlands and was designated human coronavirus NL63 (HCoV-NL63) [8]. This, among other coronaviruses, namely human coronavirus 229E (HCoV-229E), human coronavirus OC43 (HCoV-OC43), human coronavirus HKU1 (HCoV-HKU1), Middle East respiratory syndrome coronavirus (MERS-CoV), and SARS-CoV predominantly cause respiratory disease [1,9,10]. Human coronavirus NL63 has a worldwide distribution and is known to be associated with both upper and lower respiratory tract infections in both adults and children with seroconversion occurring at a very early age [11,12]. Most of the known human coronaviruses are believed to have originated from mammalian reservoirs such as bats and used other mammalian hosts as intermediate hosts before ending up in the human population. Some of these, like HCoV-229E and MERS-CoV, used camelid species [13,14,15,16], while SARS-CoV went through Himalayan palm civets as intermediate hosts [17,18]. Further, HCoV-OC43 is reported to have originated directly from cattle [19]. Unlike these groups of coronaviruses, HCoV-NL63 and HCoV-HKU1 have no known intermediate mammalian hosts. Human coronavirus NL63 is known to use the same receptor as SARS-CoV [20], and may therefore, like some SARS-CoV-related viruses, be capable of infecting swine [21]. This assertion is, however, yet to be explored through surveillance data. Different serological studies have mainly employed enzyme-linked immunosorbent assay (ELISA) and immunofluorescence assay (IFA) approaches for investigating HCoV-NL63 [22,23,24,25]. Most of these assays are designed for specific purposes ranging from seroprevalence studies to studies of the HCoV-NL63 genome [12,26], and would therefore vary in parameters like sensitivity and specificity. There is no single assay that is widely accepted as the standard for serological detection of HCoV-NL63, and this presents a challenge in the general validation of new assays. Coronaviruses have the potential to recombine to produce new viruses [27], and as such, knowledge of potential hosts other than humans that can be infected by two human coronaviruses is important to provide information on potential sources of novel human coronaviruses that may later spillover into human populations and cause disease. Knowledge of potential intermediate hosts of human coronaviruses will also provide information on the evolution of coronaviruses in general and interspecies transmission events that lead to emergence. The purpose of this study was therefore to assess the potential of domestic livestock species as intermediate hosts for HCoV-NL63.","The emergence of a new coronavirus (2019-nCoV) in Wuhan creates a sense of d\u00e9j\u00e0 vu with the severe acute respiratory syndrome coronavirus (SARS-CoV) epidemic in China in 2003. Coronaviruses are enveloped, positivestranded RNA viruses of mammals and birds. These viruses have high mutation and gene recombination rates, making them ideal for pathogen evolution. 1 In humans, coronavirus is usually associated with mild disease, the common cold. Previous emerging novel coronaviruses, such as SARS-CoV and Middle East respiratory syndrome coronavirus (MERS-CoV), which emerged in the Middle East in 2012, were associated with severe and sometimes fatal disease. MERS-CoV was less pathogenic than SARS-CoV, with the most severe infections mainly in individuals with underlying illnesses. Clinically and epidemiologically, the contemporary 2019-nCoV in China seems to resemble SARS-CoV. The genome of 2019-nCoV also appears most closely related to SARS-CoV and related bat coronaviruses. 2 The infection has now spread widely, with phylogenetic analysis of the emerging viruses suggesting an initial single-locus zoonotic spillover event in November, 2019, 3 There is an increasing focus on the human-animalenvironment disease interface, as encompassed in the One Health concept. Mortalities, disability-adjusted life-years, and billions of dollars of economic losses from these infections demand action and investment in prevention to face novel challenges to human and animal health. Research has led to better understanding of the nature and drivers of cross-species viral jumps, but the detail is still elusive. No reservoir population of bats for SARS and MERS-CoV or Ebola virus have been definitively identified, despite considerable searching, possibly because of the source virus circulating in small and isolated populations. Forensic examination has clarified the human infection sources and multispecies involvement in these diseases, with some species confirmed as competent hosts (eg, camels for MERS-CoV 4 ), bridge (or amplifying) hosts (eg, pigs for Nipah virus, non-human primates for Ebola virus 5 ), or deadend hosts. The crucial checkpoint is the jump and bridging of the viruses to humans, which occurs most frequently through animal-based food systems. In the case of SARS, markets with live and dead animals of wild and domestic origins were the crucible for virus evolution and emergence in the human population. Once the viruses' functional proteins enabled cell entry in civets (Paguma larvata) and racoon dogs (Nyctereutes procyonoides), the bridge was established and it was only a matter of time before the jump to humans occurred. 6 Sequence comparison of civet viruses suggested evolution was ongoing; this was further supported by high seroprevalence of antibodies against SARS-CoV among civet sellers, suggesting previous cross-species transmission events without necessarily human-tohuman transmission. 7, 8 Similarly, early Ebola virus was mostly associated with bushmeat and its consumption in Africa; Nipah virus is associated with date palm sap, fruit, and domestic pig farms; MERS is associated with the camel livestock industry; and H5N1 arose from viral evolution in domestic and wild birds, to ultimately bring all these cases to humans. The 2019-nCoV is another virus in the pipeline that originated from contact with animals, in this case a seafood and animal market in Wuhan, China.","The emergence of a new coronavirus (2019-nCoV) in Wuhan creates a sense of d\u00e9j\u00e0 vu with the severe acute respiratory syndrome coronavirus (SARS-CoV) epidemic in China in 2003. Coronaviruses are enveloped, positivestranded RNA viruses of mammals and birds. These viruses have high mutation and gene recombination rates, making them ideal for pathogen evolution. 1 In humans, coronavirus is usually associated with mild disease, the common cold. Previous emerging novel coronaviruses, such as SARS-CoV and Middle East respiratory syndrome coronavirus (MERS-CoV), which emerged in the Middle East in 2012, were associated with severe and sometimes fatal disease. MERS-CoV was less pathogenic than SARS-CoV, with the most severe infections mainly in individuals with underlying illnesses. Clinically and epidemiologically, the contemporary 2019-nCoV in China seems to resemble SARS-CoV. The genome of 2019-nCoV also appears most closely related to SARS-CoV and related bat coronaviruses. 2 The infection has now spread widely, with phylogenetic analysis of the emerging viruses suggesting an initial single-locus zoonotic spillover event in November, 2019, 3 There is an increasing focus on the human-animalenvironment disease interface, as encompassed in the One Health concept. Mortalities, disability-adjusted life-years, and billions of dollars of economic losses from these infections demand action and investment in prevention to face novel challenges to human and animal health. Research has led to better understanding of the nature and drivers of cross-species viral jumps, but the detail is still elusive. No reservoir population of bats for SARS and MERS-CoV or Ebola virus have been definitively identified, despite considerable searching, possibly because of the source virus circulating in small and isolated populations. Forensic examination has clarified the human infection sources and multispecies involvement in these diseases, with some species confirmed as competent hosts (eg, camels for MERS-CoV 4 ), bridge (or amplifying) hosts (eg, pigs for Nipah virus, non-human primates for Ebola virus 5 ), or deadend hosts. The crucial checkpoint is the jump and bridging of the viruses to humans, which occurs most frequently through animal-based food systems. In the case of SARS, markets with live and dead animals of wild and domestic origins were the crucible for virus evolution and emergence in the human population. Once the viruses' functional proteins enabled cell entry in civets (Paguma larvata) and racoon dogs (Nyctereutes procyonoides), the bridge was established and it was only a matter of time before the jump to humans occurred. 6 Sequence comparison of civet viruses suggested evolution was ongoing; this was further supported by high seroprevalence of antibodies against SARS-CoV among civet sellers, suggesting previous cross-species transmission events without necessarily human-tohuman transmission. 7, 8 Similarly, early Ebola virus was mostly associated with bushmeat and its consumption in Africa; Nipah virus is associated with date palm sap, fruit, and domestic pig farms; MERS is associated with the camel livestock industry; and H5N1 arose from viral evolution in domestic and wild birds, to ultimately bring all these cases to humans. The 2019-nCoV is another virus in the pipeline that originated from contact with animals, in this case a seafood and animal market in Wuhan, China.","The 2019-nCoV epidemic continues. During the past month, huge progress has been made in pathogen monitoring, identifying sources, basic etiology, clinical treatment, drug testing, and vaccine development. With the efforts of China and the helps of other countries in the world, we hope that the epidemic will subside soon. Future research on viral replication, pathogenesis, antiviral drugs and other aspects of 2019-nCoV will contribute to the treatment and prevention of the virus. However, given the emergence of 2019-nCoV pneumonia as a new infectious disease with interspecies transmission from animals, we should reflect on the origin of the human pathogen and learn from our experience. With changes of ecology and human activities, including unlimited invasion of natural habitats of animals, consumption of some of these animals, and modern agricultural practices, the spillover of viruses Table 2 . Bootstrap values above 50 were shown. from natural hosts to humans continues to appear and may even be escalating. Although we have built an unparalleled human civilization and a highly developed information society, invisible viruses can still have devastating effects on human beings. The high lethality viruses such as SARS-CoV, MERS-CoV, H5N1, H7N9, Ebola and emerging 2019-nCoV should serve as an alarm to the world. We should strive to reduce the probability of their occurrence and harm. Strengthening the investigation of animal etiology, normalizing the storage of high risk pathogens, reducing direct contact with wildlife, maintaining the barriers between natural reservoirs and human society and completely eradicating wildlife trading will help to achieve these goals.","Pigs are well established as mixing vessels and as intermediate hosts for IAVs, and coronaviruses have already been shown to possess potential for recombination in animals [9] . Given that pigs are in frequent contact with human and multiple wildlife species, and that pork is one of the most commonly consumed meats in non-Muslim countries, it is important to assess whether pigs could be mixing vessels for the emergence of novel coronaviruses with high agricultural impact and risks to public health. It has already been reported that pigs are susceptible to infection with human SARS-CoV [10] and MERS-CoV [11] . Additionally, the CD26 receptor sequence alignment of pigs and humans shows 94.5% similarity, which is sufficient for potential cross-species transmission [7] . In southern China, the unique climate, the high density of domestic as well as wild pigs, and extensive bat distribution, together with bats carrying large numbers of recombinant novel coronaviruses [12] , could lead to the emergence of more novel coronaviruses in the future.","In this Special Issue, we have papers focussing on the emergence and re-emergence of Schmallenberg virus (McGowan et al., 2018; Stavrou et al., 2017) and African swine fever virus (Sanchez-Cordon et al., 2018) . The continued surveillance of wildlife (Bodewes, 2018) and domestic animals, including livestock (Bourret, 2018) and companion animals (Caddy, 2018), to identify novel viruses is particularly important in understanding the sources and risks of viruses affecting the health of animals and humans. A greater understanding of virus replication, pathogenesis and transmission allows intervention strategies to be formulated, including development of vaccines and antiviral treatments. Understanding of replication of Zika virus is exemplified by the paper by Royle et al. (2017) , whilst the review by Pusterla et al. (2018) summarises what is known about a novel coronavirus of horses.","Whether this current COVID-19 epidemic 'frizzles out' or expands into a full-blown pandemic remains to be seen. It might also be desirable to monitor farm animals and pet cats for infection with 2019-nCoV, since their ACE2 receptor responsible for 2019-nCoV binding differs in only a few amino acids from human ACE2. Surveillance might prevent the virus establishing itself in another animal species that is in close contact to humans. In addition, in light of the fact that there are multiple species of CoVs circulating in wildlife species and that these animals are constantly interacting with each other, host-species expansion or interspecies transmission of new CoV to humans seems to be inevitable. Major knowledge gaps regarding the emergence of 2019-nCoV remain exists but worldwide scientists are working with unprecedented speed to investigate the virus, rushing to develop targeted therapeutics (see Outstanding Questions). Notwithstanding, a global surveillance network involving veterinarians and animal biologists is urgently needed to monitor, and possibly to predict, potential sources for the emergence of another highly pathogenic CoV. We propose the concept of 'One Health' to facilitate scientific exchange across disciplines, sharing of data, and coordinated efforts in order to prevent future outbreaks. From surveillance and evolutionary studies on animal viruses, can their zoonotic potential be identified before interspecies transmission occurs?","All seven HCoVs have a zoonotic origin from bats, mice or domestic animals 2,20. Multiple lines of evidence support an evolutionary origin of all HCoVs from bats, where viruses are well adapted and non-pathogenic but show great genetic diversity. The COVID-19 epidemic has presented enormous medical, scientific, social and moral challenges to China and the world. Tracing the zoonotic origins of HCoVs provides a framework to understand the natural history, driving force and restriction factors of species jumping. This might also guide or facilitate the search for the reservoir, intermediate and amplifying animal host(s) of SARS-CoV-2, with important implications in the prevention of future spillovers. In this review we present an overview of the zoonotic origins, interspecies transmission and pathogenesis of HCoVs. Particularly, we highlight and discuss the common theme that parental viruses of HCoVs are typically non-pathogenic in their natural reservoir hosts but become pathogenic after interspecies transmission to a new host. We also review the trend of HCoV evolution in which the increase in transmissibility often comes with the decrease in pathogenicity. The outcome of the ongoing SARS-CoV-2 outbreak is also discussed in this context.","A popular theory for bats to transmit viruses to humans is through intermediate hosts, which are in close contact with humans and can amplify viruses. When fruit bats eat fruit, they contaminate the fruit with viruses in their saliva and urine; discarded fruit fall to the ground and is consumed by intermediate hosts, such as pigs, horses, and nonhuman primates; humans become infected by contact with or consuming products of the intermediate animal hosts. Flying foxes are thought to transmit NiV/HeV in Southeast Asia and tropical Australia. During the 1998 NiV outbreak in Malaysia, it is thought that pigs were infected by NiV by consuming mangoes contaminated by flying foxes. Mangoes were a food source for flying foxes, and discarded mangoes contaminated by the saliva and urine of the bats fell into the pigsties and were consumed by pigs, resulting in the cross-species infection of pigs and subsequently humans (Chua et al., 2002a) . HeV is endemic in flying-foxes in Australia. HeV spills over from bats to domestic animals, primarily horses, which amplify the virus and subsequently infect humans (Murray et al., 1995) . Bats contaminate a drip zone around trees where they feed or roost by excreting urine, feces and saliva (with partially eaten fruit). Horses may be exposed to HeV when consuming contaminated grass, fruit, feed or water; or when browsing or sniffing contaminated surfaces within this drip zone (Plowright et al., 2015) . Fruit contaminated by bats is also thought to transmit Ebola virus to apes in Central Africa (Leroy et al., 2005) . In the case of the 2003 SARS outbreak in China, the transmission of SARS-CoV from bats to humans was made possible by the special taste for wildlife cuisine, including civets, which acted as intermediate hosts for SARS-CoV. In South China, civets are thought to strengthen health in winter, and the demand was so high that farming of this wildlife species was widespread. Trade of civets in live wet markets exposed other susceptible animals. Therefore, bat SARS-CoV was likely introduced into humans through civets and other animals (Liu, 2003) . As for MERS in Middle East, dromedary camels are hypothesized as intermediate animals to transmit MERS-CoVs from bats to humans (Memish et al., 2014) . Camels are popular in Middle East countries both for entertainment and transportation. People may be infected through direct contact with infected camels, which can shed virus in respiratory secretions (Azhar et al., 2014) . MERS-CoVs were also detected in camel milk samples from mares infected with the virus (Reusken et al., 2014) . Whether the viruses is excreted in the milk or is introduced as contaminant during the milking process, it poses a risk for people to become infected by consumption of unpasteurized camel milk.","So far efforts have failed to find the original pathway of SARS-CoV-2 into humans by identifying a coronavirus that is nearly identical to SARS-CoV-2, as those found for SARS and MERS in civets and domestic camels respectively (12, 13) . However, if the new SARS-CoV-2 strain did not cause widespread infections in its natural or intermediate hosts, such a strain may never be identified. The close proximity of animals of different species in a wet market setting may increase the potential for cross-species spillover infections, by enabling recombination between more distant coronaviruses and the emergence of recombinants with novel phenotypes. While the direct reservoir of SARS-CoV-2 is still being sought, one thing is clear: reducing or eliminating direct human contact with wild animals is critical to preventing new coronavirus zoonosis in the future.","All HCoVs have a zoonotic origin. Whereas bats are the evolutionary reservoir host of 229E, NL63, SARS-CoV, MERS-CoV, and SARS-CoV-2, parental viruses of OC43 and HKU1 have been found in rodents. Intermediate and amplifying hosts of HCoVs were also found in domestic and wild mammals. Ancestors of OC43 were identified in domestic animals such as cattle and swine. The switch of hosts from cattle or pigs to humans might have occurred in the context of a pandemic of respiratory disease recorded around 1890 in human history [26]. Similar to MERS-CoV, 229E could be acquired by humans from dromedary camels. However, the direction of this cross-species transmission remains to be determined and the possibility cannot be excluded that both humans and camels might have acquired 229E from an unidentified host including bats [27].","The first pandemic of the twenty first century was the outbreak of Severe Acute Respiratory Syndrome (SARS) caused by the SARS-CoV coronavirus. As far as is known, this outbreak was not due to the deliberate release of the SARS-CoV coronavirus, but rather was a naturally occurring event. The appearance of SARS seems to have involved: (1) a zoonotic origin for SARS-CoV [e.g., horseshoe bats and/or Chiroptera as one wildlife reservoir (Janies et al. 2008)]; (2) transmission to intermediate hosts [e.g., civet cats, racoon dogs (Feng and Gao 2007)]; (3) human contact with these intermediate hosts in Southern China (Guangdong Province, November 2002) and subsequent cross-species transmission of the coronavirus to humans (Feng and Gao 2007); (4) transmission of the virus through both non-hospital personal contact and hospital staff contact (Zhang 2007); and (5) global transmission of the virus via travelers from affected regions in Asia to other countries. SARS was eventually controlled through increased hygienic measures (e.g., face mask protection, frequent hand washing, living quarter disinfection), travel restrictions, and quarantine.","MERS-CoV was suggested to originate from bats, but the reservoir host fueling spillover to humans is unequivocally dromedary camels (Haagmans et al., 2014; Memish et al., 2013) . Both SARS-CoV and SARS-CoV-2 are closely related and originated in bats, who most likely serve as reservoir host for these two viruses (Ge et al., 2013; Hu et al., 2017; Li et al., 2005b; Yang et al., 2015a; Zhou et al., 2020) . Whereas palm civets and racoon dogs have been recognized as intermediate hosts for zoonotic transmission of SARS-CoV between bats and humans (Guan et al., 2003; Kan et al., 2005; Wang et al., 2005) , the SARS-CoV-2 intermediate host remains unknown. The recurrent spillovers of coronaviruses in humans along with detection of numerous coronaviruses in bats, including many SARS-related coronaviruses (SARSr-CoVs), suggest that future zoonotic transmission events may continue (Anthony et al., 2017; Ge et al., 2013; Hu et al., 2017; Li et al., 2005b; Menachery et al., 2015; Menachery et al., 2016; Yang et al., 2015a; Zhou et al., 2020) . In addition to the highly pathogenic zoonotic pathogens SARS-CoV, MERS-CoV, and SARS-CoV-2, all belonging to the b-coronavirus genus, four low-pathogenicity coronaviruses are endemic in humans: HCoV-OC43, HCoV-HKU1, HCoV-NL63, and HCoV-229E. To date, no therapeutics or vaccines are approved against any human-infecting coronaviruses.","SARS-CoV-2 is the seventh coronavirus known to infect humans, and the third zoonotic virus after SARS-CoV and MERS-CoV. Bats are the reservoir hosts of a number of additional novel coronaviruses, particularly Chinese horseshoe bats, and a number of these novel coronaviruses can efficiently use multiple orthologs of the SARS receptor, human ACE2, and replicate efficiently in primary human airway cells and achieve in vitro titres equivalent to epidemic strains of SARS-CoV48,49. This indicates that other potential cross-species events could occur in the future. There is therefore a strong reason to ban unregulated wild animal sales in Chinese wet markets, particularly exotic species, both from a public health perspective and for ecological reasons. Such a ban would be difficult to instigate for cultural reasons, but China\u2019s top legislative committee on 24 February 2020, passed a proposal to ban all trade and consumption of wild animals. If this is legislated as a permanent ban, it might help reduce the risk of another novel virus emerging from wildlife in China in the future.","Studies by different groups demonstrate that SARS was the result of spillover from a wildlife reservoir -most likely bats -into an intermediate host (or hosts; most importantly Paguma larvata) and from there to the human population. Rapid viral evolution was demonstrated and most likely was the pivotal factor that allowed SARS-CoV to rapidly adapt to nonreservoir species. Table 1 shows the distribution of several different coronaviruses in humans and animals and their classification into different groups.","As coronaviruses were previously known to cause only mild respiratory illnesses in humans they were not a major concern of the public health community. However, the emergence of severe acute respiratory syndrome coronavirus (SARS-CoV) [7\u20139] and its high infectivity and fatality generated considerable interest in the biodiversity, genomics, evolution, natural hosts and potential inter-species transmission of coronaviruses [10]. To date, at least 90 types of coronavirus have been isolated or genome-identified from humans and a wide variety of animals, including domestic animals, wild birds and bats. Bats are particularly notable in this respect because they are known to harbor a diverse range of pathogens, and are known to be the reservoir hosts of both human coronavirus 229E [11] and SARS-CoV [12], and are closely related to MERS-CoV [13, 14]. As a consequence bats have been prioritized for the surveillance of emerging zoonotic diseases [15\u201317].","The finding of people in a small sample of rural communities in southern China seropositive for a bat SARSr-CoV suggests that bat-origin coronaviruses commonly spillover in the region [18] . Single cases or small clusters of human infection may evade surveillanceparticularly in regions and countries that border China with less healthcare capacity or rural areas where people don't seek diagnosis or treatment in a timely fashion. Surveillance programs can be designed by local public health authorities to identify communities living in regions with high wildlife diversity and likely high diversity of novel viruses [21] . People with frequent contact with wild or domestic animals related to their livelihood and occupation, and patients presenting acute respiratory infection (ARI) or influenza-like illness (ILI) symptoms with unknown etiology can be included into the surveillance as a cost-effective method to identify novel virus spillovers. This 'pre-outbreak surveillance' strategy can be coordinated with different sectors of public health, healthcare, agriculture and forestry to implement sample collection and testing of wildlife, domestic animals, and people in collaboration with research institutions. These efforts will help identify and characterize viral genetic sequence, identify high-risk human populations with antibodies and cell-mediated immunity responses to wildlife-origin CoVs [22] , as well as the risk factors in human behaviors and living environment through interviews. Evidencebased strategies to reduce risk can then be designed and implemented in the communities where viral spillover is identified.","The finding of people in a small sample of rural communities in southern China seropositive for a bat SARSr-CoV suggests that bat-origin coronaviruses commonly spillover in the region [18] . Single cases or small clusters of human infection may evade surveillanceparticularly in regions and countries that border China with less healthcare capacity or rural areas where people don't seek diagnosis or treatment in a timely fashion. Surveillance programs can be designed by local public health authorities to identify communities living in regions with high wildlife diversity and likely high diversity of novel viruses [21] . People with frequent contact with wild or domestic animals related to their livelihood and occupation, and patients presenting acute respiratory infection (ARI) or influenza-like illness (ILI) symptoms with unknown etiology can be included into the surveillance as a cost-effective method to identify novel virus spillovers. This 'pre-outbreak surveillance' strategy can be coordinated with different sectors of public health, healthcare, agriculture and forestry to implement sample collection and testing of wildlife, domestic animals, and people in collaboration with research institutions. These efforts will help identify and characterize viral genetic sequence, identify high-risk human populations with antibodies and cell-mediated immunity responses to wildlife-origin CoVs [22] , as well as the risk factors in human behaviors and living environment through interviews. Evidencebased strategies to reduce risk can then be designed and implemented in the communities where viral spillover is identified.","With the demonstration of an increasing number of spillover events that led to severe disease outbreaks in human and domestic animals, we believe it is paramount that from now on we include active surveillance of wildlife animals as part of an integrated infectious disease prevention and control strategy. Surveillance of wildlife animals has also been made more feasible and productive, thanks to the advance in modern molecular techniques including PCR with virus group-specific primers, virus discovery using next generation high-throughput sequencing technologies, and high density virus microarrays. 56e63 Since the SARS outbreaks, especially after the discovery of SL-CoVs in bats, there is a significant surge in international effort for surveillance of coronaviruses in wildlife animals. Before the SARS outbreak, there were only 10 coronaviruses with complete genomes sequenced. This number has increased more than sixfold as a result of the active surveillance works conducted around the world. 27e29, 31,32,34,40,55,64e73 Although this only marks the beginning of our understanding of coronaviruses in wildlife animals, it is fair to say that we have learnt a lot more about coronaviruses in general than the past 50 years or so; during that period studying of viruses was only possible and called for in response to disease outbreaks. Based on phylogenetic analysis of the large number of bat coronavirus sequences available presently, it is postulated that all known disease-causing coronaviruses previously identified in humans or animals originated from bat strains. 31, 34, 43, 55 This hypothesis was unfortunately proved by the outbreak of another SARS-like disease, Middle East Respiratory Syndrome (MERS), which was caused by a novel coronavirus (previously named HCoV-EMC, now MERS-CoV) and supposed to originate from bats. 74 Even though the MERS-CoV-like viruses found in bats are not the direct progenitor of the MERS-CoV, the highly genetic diversity of these bat viruses is likely the gene sources for the deadly pathogen in humans, just like that for SARS-CoV. 71,72,75e77 ","Approximately 75% of emerging infectious diseases are zoonoses [1, 2] . The rate of emergence of zoonotic viruses appears to be increasing and/or our ability to detect new viruses is improving. Viruses are well adapted to their reservoir hosts and therefore exhibit stability within their host's cellular and ecological environments and display little or no clinical disease in these species. However, when a virus jumps the species barrier and spills over into humans, the effects can be devastating. Only when these viruses from wildlife spillover to humans or their domesticated animals and cause mortalities, does this become a significant concern for public health. High mortality rates often characterize these spillover events as do high economic losses. This is particularly true for the bat borne viruses that have emerged in the last 20 years such as Severe Acute Respiratory Syndrome coronavirus (SARS-CoV), Nipah virus (NiV) Hendra virus (HeV) and Ebola virus (EBOV). International travel has contributed to the transmission of many infectious diseases and the bat borne zoonotic viruses are no exception with the most notable being the transmission of SARS-CoV from the key port in Hong Kong to Canada, the Americas, Europe, Asia and Australasia resulting in 8422 cases with 916 deaths (10.9% case fatality) [3] .","In order to perfect pandemic response, it is necessary to improve the coordination and interconnection between individual and institutional participants, such as healthcare providers and public health systems. A global response approach (OneResponse) should be reflected upon and developed. On an institutional level, a first step in bringing together the fields of environment and animal health occurred in 2010 in France with the creation of a national French Agency for Food, Environmental and Occupational Health &amp; Safety (ANSES), originating in the French Agency for Food Safety (AFSSA, which also includes the National Agency for Veterinary Drugs) and the French Agency for Environmental and Occupational Health &amp; Safety (AFSSET). Another case of bringing together the areas of surveillance, prevention and human health intervention occurred in 2016 with the creation of the French Public Health Agency, with the merger of the Health Surveillance Institute (InVS), the National Institute for Prevention and Health Education (INPES), and the Organization for Preparedness and Response to Health Emergencies (EPRUS). The relations between these two new institutions should be developed in order to provide a better-coordinated response to future health crises.","Determining the animal reservoir and incidental hosts of a virus with evidence of zoonosis such as 2019-nCoV is important for controlling spillover events and limiting human infections. Although the ecological reservoir for both MERS-CoV and SARS-CoV remain undefined, evidence of viral presence in a variety of animal species has been found. Serologic evidence and the isolation of viral genomic material for SARS-CoVrelated viruses have provided evidence for bats as the putative reservoir for SARS-CoV. While masked palm civets and raccoon dogs were involved in transmission to humans and initially considered potential reservoirs, they are now considered as incidental or spillover hosts [39] [40] [41] . In contrast, dromedary camels are considered to act a reservoir host for MERS-CoV, and over half of primary human infections report contact with camels, however, it is suspected that bats are likely the ancestral host for MERS-CoV or a MERS-CoV-like virus [42, 43] . Zoonotic transmission of SARS-CoV likely results from direct contact with intermediate hosts [44] , while human-to-human transmission of both viruses occurs primarily through close contact and nosocomial transmission via respiratory secretions [45, 46] . Considering the animals implicated as reservoir hosts for SARS-CoV and MERS-CoV, along with the conclusions from phylogenetic analyses from us and other groups, these potential reservoirs will be important starting points for investigation of the virus source.","Recently, emerging disease surveillance programs have intensified to investigate the role of bats in the evolution and spillover of zoonotic pathogens from wildlife. Our study involved three years of active and passive surveillance to characterize the viral diversity of the Northwestern Italian bat population. Using viral family-wide PCRs we identified and phylogenetically characterized 20 new CoVs and 3 PMVs strains. To date, studies on bat CoVs phylogeny are mainly based on datasets of short sequences (i.e. 440 bp) (e.g. [9, 10, 13, 14, 24, 25, 44]) due to the difficulties of obtaining isolates and good quality viral RNA from bats, but ideally long sequence fragments would be beneficial to infer more reliable phylogenies.","The effect of decline in biodiversity on the emergence of diseases is subject of numerous publications [108,109,110,111,112,113,114]. Basically, there are arguments in favor of two controversial theories; reduced biodiversity could either increase (dilution effect) or decrease the risk of disease transmission. For almost half of the zoonotic diseases that have newly emerged by spillover since 1940, a preceding change in land-use, agriculture, and wildlife hunting was reported [108]. All of the above-mentioned effects contribute to changes in biodiversity and increased contact situations between human and animal hosts, also in Europe. Once spillover in novel hosts has occurred, a high density of the novel host population eventually facilitates the establishment in the novel niche. Thus, human overpopulation and a decreased biodiversity might be mutual factors promoting the establishment of emerging infectious diseases.","Multi-disciplinary microbial ecology studies, such as our efforts to characterize the microbe diversity hosted by Desmodus rotundus in Guatemala combined with analysis of ecological feeding preference, can answer questions about human\u2013bat public health interfaces and have additional value for conservation and ecology. The results of our study demonstrate that D. rotundus carries both novel and known microbes, and that the populations in this region appear to display an overwhelming preference for feeding on domestic livestock species. As populations of D. rotundus continue to expand in Latin America with the increasing production of livestock, better understanding of their infectious disease ecology and diversity will become even more critical. Subsequent studies are necessary to further resolve the phylogeny and further investigate the actual pathogenicity of the novel viruses detected in this study. Additional research efforts involving microbial discovery and surveillance among domestic animals may also prove useful for investigating whether microbe sharing occurs between D. rotundus and their prey, as well as how viral and bacterial diversity may change seasonally and geographically. Gaining a better understanding of the role of D. rotundus as a potential source and reservoir of infectious diseases will help better inform public health and wildlife management policies aimed at preventing or mitigating future disease spillover events in the face of ecosystem changes.","Global disease outbreaks and pandemics have been increasing exponentially over the last 40 years, and experts have long been warning of the potentially devastating effects of a severe pandemic, though it is important to note that the improvement in diagnostic capabilities undoubtedly plays a role (Figures 1 and 2 ). 2, 3 By analyzing the causes of this and similar pandemics, as well as the physiopathological, human, and political influences on its spread, we hope to shed some light on the current health risks, decision-making dynamics, and future implications of the current coronavirus pandemic. 4 It originates from bats, with dromedary camel as the intermediate host, causes pneumonia in humans and has a mortality rate of over 30%. Nosocomial spread has also been described. No specific antiviral treatment is available and treatment is mainly supportive and symptomatic. 5 A novel 2019 coronavirus (SARS-CoV-2) belongs to the Betacoronavirus genus of the Coronaviridae family. It shares more than 79% homology with SARS-CoV, 6 and causes mild to severe disease, with fatal bilateral viral pneumonia described in humans. 7 ,8 It is referred to as a novel coronavirus because it has not previously been observed in the human population. Its likely reservoir is bats, with pangolins identified as the likely intermediate host. 9 It is worth mentioning that pangolins are endangered and the most trafficked animal in the world. 10 Because the pangolin trade is illegal, this would explain why they were not listed in the official registers of the Wuhan market, making it difficult for researchers to identify the source of the virus. Some researchers are suggesting that the virus may have been circulating previously, and the market may have been an amplification setting and not the source of the spillover. 11","Several specific actions can mitigate future threats to the health of the global population. First, we need to fully implement the International Health Regulations (IHR) in countries of all income levels. Reports from WHO and the Global Preparedness Monitoring Board have shown that responses to urgent crises have improved under the IHR (2005), but important deficiencies in coverage still exist. 3, 4 We must scale up laboratory capacity and other surveillance capabilities, augment the readiness of health systems to care for large numbers of seriously ill patients while safe guarding the health-care workforce, and improve communication and coordination strategies and implementation. 5 Procurement and supply chain management must also be enhanced to ensure adequate stocks of personal protective equipment and essential medications and to enable equitable access to new diagnostics, therapeutics, and vaccines during health emergencies. 6 Second, we need to accelerate progress towards achieving the priorities and targets of the Sendai Framework for Disaster Risk Reduction, 7 which was adopted by UN member nations in 2015. The Sendai Framework applies an all-hazards approach to increasing international, regional, national, and local resilience to withstand a broad spectrum of disasters, including epidemics, pandemics, and epizootics. Preparedness for emerging infectious diseases requires strengthening health systems and developing response plans that account for the social, environmental, political, and institutional factors that can either support or disrupt emergency management efforts. 8 The Sendai Framework complements the IHR by integrating infrastructure, climate change, and economic considerations into disaster management plans and promoting inclusive policies that protect vulnerable populations during all phases of mitigation, preparedness, response, and recovery. 7 Third, plans for preventing, detecting, and controlling outbreaks of emerging infectious diseases need to be built on a One Health foundation that emphasises the interconnectedness of humans, animals, and ecosystems. 9 Transdisciplinary, multisectoral strategies are necessary when seeking to solve complex problems that threaten global public health and safety. 10 More specifically in relation to diseases such as COVID-19, One Health focuses on the danger existing at the human-wildlife interface. The ability to detect viruses in livestock and wildlife is a crucial component of early warning systems for human pandemics. 11 ","Although several bat species have been found to harbor several alpha and betacoronaviruses believed to be the ancestors of endemic human coronaviruses including HCoV-NL63 [13,23], bats may not have been a direct source of virus transmission to humans given that, CoVs such as SARS-CoV and MERS-CoV both make use of terrestrial mammals which are more likely to have contact with humans instead of bats as transmission hosts. Again, HCoV-229E is more closely related to their relatives in camels as compared to those in bats, indicating a probability of camels being intermediate hosts between bats and humans [14,57]. Human coronavirus NL63 uses the angiotensin-converting enzyme (ACE) 2 receptor for infection of target cells similar to SARS-CoV [20] and has been found to be able to replicate in swine cells in vitro [58]. No antibodies to HCoV-NL63 were found in any of the pigs tested in the present study as evidence of the fact that the ability to replicate in swine cells does not imply capability to infect an actual animal since several other barriers need to be surmounted for this to happen. Based on the results of the present study it may be concluded that cattle, sheep, goats, donkeys, and swine may not be intermediate hosts for HCoV-NL63. However, there have not been any reports of HCoV-NL63-related viruses circulating in Ghanaian bats, and as such, a spillover opportunity may not be present, and hence no likely infection. Surveillance of local livestock populations can also be performed for antibodies in areas where such HCoV-NL63-related viruses have been detected like in Kenya to confirm this [57]. ","Nearly every emerging viral disease can be associated with an environmental or ecological component. The consequences of urbanization and globalization-including but not limited to deforestation, habitat modification, and the use of wild land for farming-impact the emergence and reemergence of potential zoonoses. Increases in the human population have led to the encroachment of humans into animal habitats formerly unoccupied by humans. These circumstances contribute to humans coming in contact with animals with which they would not normally interact, putting them at risk of being exposed to zoonotic viruses that are found in the local wildlife populations. The transfer of zoonotic viruses to humans can occur directly or indirectly. For example, during an outbreak of 276 cases of Nipah virus in Malaysia and Singapore in 1998-99, most infected individuals had contact with sick pigs that were being farmed commercially as livestock and then slaughtered for meat. The pigs contracted Nipah virus from the local bats, which were thought to have spread the virus through urine or by dropping partially eaten fruit, laden with bat saliva, into the pig stalls. The bats may have acquired the fruit from the fruit trees on the farms. Human infection with Nipah virus causes encephalitis, fever, headache, and reduced consciousness. Forty percent of infected individuals died during this outbreak. Interestingly, Nipah virus infection is asymptomatic in bats.","Since the SARS outbreak there has been an explosion in our knowledge of novel viruses in a variety of hosts, but perhaps more in bats than other animal orders for reasons relating to their ecology and to their association with novel viruses in the preceding decade. Some of the virus isolates can be cultured, and their biology explored for possible cross-species transmission and other factors associated with assessing their pathogenic potential. Many others are known only from short genomic sequences, and it is less obvious how they can be used for determining future risk potential. Nevertheless, having sequence data from viruses in wildlife niches can be useful when tracking the origins of novel diseases, as demonstrated recently with the SARS-like virus infection in Saudi Arabia, and also in seeking information on genetic diversity and perhaps indications of host range.","Environmental DNA (eDNA) and its subdiscipline, invertebrate-derived DNA (iDNA) have been used to survey biodiversity non-invasively [1, 2] . Water is ubiquitous in most ecosystems, and, among invertebrates, terrestrial haematophagous leeches are abundant and can be easily collected in many tropical rainforests [3, 4] . Such non-invasive nucleic acid sources can mitigate difficulties of obtaining wildlife samples, particularly in remote areas or for rare species. Recently, eDNA/iDNA sources have been applied to monitoring specific wildlife pathogens [5, 6] . However, previous studies have focused on known pathogens, whereas most wildlife pathogens are uncharacterized and unknown. Non-invasive approaches to monitoring known and novel pathogens may be of particular benefit in ecosystems prone to viral emergence, many of which occur in areas where invasive sampling is challenging, such as tropical rainforests. Here, we show that both eDNA from natural waterholes, and iDNA from terrestrial haematophagous leeches, can be used to detect unknown viruses circulating in mammalian hosts (Figure 1 ). Using a curated set of RNA oligonucleotides based on the ViroChip microarray assay [7] as baits in a hybridization capture system, multiple mammalian RNA and DNA viruses were detected from both eDNA and iDNA samples. Congruence was found between host DNA assignment and viruses identified in leeches, and between animals observed visiting the waterholes and the viruses detected. Our results demonstrate that eDNA/iDNA samples may represent an effective non-invasive resource for studying wildlife viral diversity. Several of the detected viruses were novel, highlighting the potential of eDNA/iDNA for epidemiological analysis of emerging viruses prior to their emergence.","Considering the amount of novel bat-borne PMVs associated with the emergence of zoonotic infections in animals and humans in the last years define the virus diversity within European bat species is needed. Performing surveillance studies within a specific geographic area can provide awareness of viral burden where bat roosts are in close proximity to spillover hosts, and can form the basis for the appropriate control measures to curb potential threats for public health and optimal management of bats and their habitats.","Overall, lyssaviruses prevalent in European bats pose a risk to public health, and preventive measures have already been implemented by many European countries for decades (e.g., surveillance, vaccination plans, and post exposure prophylaxis) [87]. Especially the high-risk occupational groups (i.e., bat workers, bat carers in bat bat hospitals) are at increased risk. However, lyssavirus prevalence in European bats is very difficult to determine and results are very heterogenic [40]. The lyssavirus prevalences are considerably low, but changes of behavior as a result of a lyssavirus infection may be more likely to bring bats into contact with humans. However, it is necessary to balance the risk with the total number of fatal human cases during the last 35 years (five cases in 590 million people living in greater Europe) [87]. Accordingly, the risk is relatively low and would probably fall to zero if people were educated appropriately. Direct contact (bites and scratches) with certain bat species might be risky and require post exposure prophylaxis. Only few of the European bat species are known to be reservoirs of EBLV-1 and EBLV-2, but all of the European species are endangered or close to extinction. Relocation or culling of bat colonies, in spite of being an obvious solution from the viewpoint of the general public, increases the risk of lyssavirus exposure and transmission and should not be considered [95]. Only education can channel public fear to avoid further threats to the bats and the general public.","Studies by different groups demonstrate that SARS was the result of spillover from a wildlife reservoirmost likely batsinto an intermediate host (or hosts; most importantly Paguma larvata) and from there to the human population. Rapid viral evolution was demonstrated and most likely was the pivotal factor that allowed SARS-CoV to rapidly adapt to nonreservoir species. Table 1 shows the distribution of several different coronaviruses in humans and animals and their classification into different groups.","Rhabdoviruses of the genus Lyssavirus that have been detected in Europe are considerably harmful and truly zoonotic agents, inevitably causing the death of unvaccinated humans if not treated in time before onset of the rabies disease [86]. Even though bat-transmitted lyssaviruses have a fatality rate of virtually 100% and are suspected to be transmissible by bat biting and scratching, the reported total number of human fatalities in Europe is low (n = 2\u20135 since 1963) [86,87,88]. All described hosts of European bat lyssaviruses (EBLV-1 and EBLV-2) are synanthropic, hence sharing their habitats with humans [87]. EBLV-1 has been predominantly detected in Eptesicus serotinus and E. isabellinus in Europe, both living in buildings, roofs, and attics usually in the southern regions of Europe (E. serotinus until 55\u00b0 North, E. isabellinus in southern Portugal\u2014E. isabellinus is a North African population of E. serotinus that is controversially but not concludingly discussed as a novel species [1]), and male bats are reported to co-roost with multiple bat species [90]. EBLV-1 was also detected in V. murinus, M. schreibersii, M. myotis, M. nattereri, R. ferrumequinum, and T. teniotis. Whether these bat species constitute accidental hosts infected by spillover from co-roosting E. serotinus species, or whether they are additional reservoirs, has not yet been determined [38,39,40,41,91].","The emergence of SARS-CoV has had a huge impact on the global health and economy. It served as a warning to what may come out of a seemingly harmless virusereservoir equilibrium in bats or any other wildlife species. At the same time, the experience gained from the SARS outbreaks and the following in-depth studies on SARS-like coronaviruses has provided and will continue to provide invaluable knowledge and guideline to our future fight against new and emerging infectious diseases. One of the major lessons is that we need to pay much more attention to the reservoir species in understanding the genetic diversity of different viruses, the intricate interplay at the virusehost interface, and the major factors responsible for the disturbance of viruse host equilibrium, which in turn trigger spillover events leading to disease outbreaks.","Kris Murray and Peter Daszak discuss how the recent new science is changing the way ecologists view the interaction between wildlife habitats and humans, and present two hypotheses that are currently studied addressing the potential effects of land use change on disease emergence. These hypotheses are not mutually exclusive, and it is conceivable that the evidence for either one may differ for different diseases. The perturbation hypothesis assumes that changes in the environment may disturb an equilibrium that is associated with lower risks of transmission of viruses from wildlife to humans. What exactly that equilibrium is may differ for each ecosystem and its components, including the animals and pathogens in there. The second hypothesis, the pathogen pool hypothesis, assumes that disease emergence related to change of land use is explained by increased exposure of humans (or animals, as intermediate hosts) to a rich pool of pathogens from which they were separated before the change occurred. Understanding the complex processes that drive disease emergence can inform risk-based surveillance strategies and might eventually lead to interventions.","Bats (Order Chiroptera) account for 20% of all mammalian species and are distributed worldwide. With the advancement in detection techniques and increased surveillance, bats are being increasingly recognized as hosts for many zoonotic viruses [1], including filo-, paramyxo-, corona- and lyssaviruses [2,3,4,5]. Regions in Africa are considered a hotspot for emerging infectious diseases with more than 50% of recently emerging diseases originating from wildlife species on this continent [6,7]. Although several surveillance studies have been implemented to detect potential zoonotic viruses in bats, including from countries in the Congo basin and East Africa, limited information is available for Rwanda. Importantly, in the bordering Democratic Republic of Congo and Uganda, Marburg and Ebola disease outbreaks in humans have occurred [8], and corona- and paramyxoviruses have been reported to circulate in bats [3,9,10,11]. ","Angiotensin-converting enzyme II (ACE2) was known to be the cell receptor for SARS-CoV, and also for some SARS-like bat coronaviruses16. Sequence studies found that the receptor-binding domain of the SARS-CoV-2 virus was sufficiently similar to that of SARS-CoV to indicate it could efficiently use the human ACE2 receptor for entry to human cells6,7. Infectivity experiments were undertaken with HeLa cells expressing or not expressing ACE2 from humans, bats, civets, pigs and mice, and the results confirmed that SARS-CoV-2 virus was able to use entry receptors on all ACE2-expressing cells other than mice5. Molecular modelling has indicated that the binding affinity of SARS-CoV-2 to ACE-2 may be even higher than that of SARS-CoV and it may therefore be more efficient at infecting human cells17. Evidence from the sequence analyses clearly indicates that the reservoir host of the virus was a bat, probably a Chinese or Intermediate horseshoe bat, and it is probable that, like SARS-CoV, an intermediate host was the source of the outbreak. To ensure that future cross-species transmission events of this new virus don\u2019t occur again in the future, it is important to identify the reservoir and intermediate wildlife hosts. The closest known wildlife sequence to SARS-CoV-2 remains the sequence from the virus isolated from an Intermediate horseshoe bat, but there were significant differences in the receptor-binding domain between the two viruses. Malayan pangolins (Manis javanica) have been suggested as potential intermediate hosts, and SARS-like viruses have been identified in pangolins seized in anti-smuggling operations in southern China, but they only shared about 85\u201392%% homology with SARS-CoV-218. No other possible intermediate wildlife host has been proposed at this time.","Using the most conservative approach to identifying a non-recombinant genomic region (NRR1), SARS-CoV-2 forms a sister lineage with RaTG13, with genetically related cousin lineages of coronavirus sampled in pangolins in Guangdong and Guangxi provinces ( Figure 2 ). Given that these pangolin viruses are ancestral to the progenitor of the RaTG13/SARS-CoV-2 lineage, it is more likely that they are also picking up viruses from bats. While pangolins could be acting as intermediate hosts for bat viruses to get into humansthey develop severe respiratory disease (Lie et al., 2019) and commonly come into contact with people as they are trafficked in large numbers for consumption and use in Chinese medicine -pangolin infection is not a requirement for bat viruses to cross into humans.","Studies conducted on animals sampled from live animal markets in Guangdong, China, during and immediately after the SARS pandemic indicated that masked palm civets (Paguma larvata) and two other species had been infected by SARS-CoV [27 ] , but no evidence of infection was detected in wild or farmed civets [28] [29] [30] indicating they were probably spill-over hosts rather than natural hosts of the virus. The finding of SARS-CoV-like viruses in Chinese horseshoe bats from the genus Rhinolophus [31 ,32 ,33,34] , however, clearly suggested that bats could be a potential reservoir of SARS-CoV, and possibly even the natural hosts for all presently known coronavirus lineages [35] . A large number of studies have since demonstrated further SARS-CoV-like viruses and an astonishing diversity of other coronaviruses belonging to alpha-coronavirus and beta-coronavirus genera in the subfamily Coronavirinae occurring widely in bat species in most parts of the world including Africa [36] [37] [38] , Europe [39] [40] [41] [42] [43] , the Americas [44] [45] [46] [47] [48] [49] and Asia [50] [51] [52] . Interestingly, an analysis of viruses isolated from bats in Mexico showed that host species was a strong selective driver in coronavirus evolution, and that a single species of bat can maintain multiple coronaviruses. Furthermore, phylogenetic association of CoVs with host species/ genus was particularly evident in allopatric populations separated by significant geographical distances [49] . A similar diversity of coronaviruses has also been found in birds, comprising the gamma-coronaviruses [53], and a new genus, the delta-coronaviruses recently described with viruses from birds and pigs [54] ."],"title":["The novel Coronavirus (SARS-CoV-2) is a one health issue","Bat origin of human coronaviruses","Zoonotic origins of human coronaviruses","Bats and their virome: an important source of emerging viruses capable of infecting humans","Paramyxo- and Coronaviruses in Rwandan Bats","A strategy to prevent future pandemics similar to the 2019-nCoV outbreak","A strategy to prevent future pandemics similar to the 2019-nCoV outbreak","Reservoirs and vectors of emerging viruses","Zoonotic origins of human coronaviruses","Development of a Whole-Virus ELISA for Serological Evaluation of Domestic Livestock as Possible Hosts of Human Coronavirus NL63","2019-nCoV in context: lessons learned?","2019-nCoV in context: lessons learned?","The epidemic of 2019-novel-coronavirus (2019-nCoV) pneumonia and insights for emerging infectious diseases in the future","Bat-Origin Coronaviruses Expand Their Host Range to Pigs","Emerging viruses of zoonotic and veterinary importance","COVID-19: Epidemiology, Evolution, and Cross-Disciplinary Perspectives","Zoonotic origins of human coronaviruses","Bats as reservoirs of severe emerging infectious diseases","Emergence of SARS-CoV-2 through Recombination and Strong Purifying Selection","A tug-of-war between severe acute respiratory syndrome coronavirus 2 and host antiviral defence: lessons from other pathogenic viruses","Structure and infrastructure of infectious agent research literature: SARS","Structure, Function, and Antigenicity of the SARS-CoV-2 Spike Glycoprotein","COVID-19: a novel zoonotic disease caused by a coronavirus from China: what we know and what we don\u2019t","SARS","A Bat-Derived Putative Cross-Family Recombinant Coronavirus with a Reovirus Gene","A strategy to prevent future pandemics similar to the 2019-nCoV outbreak","A strategy to prevent future pandemics similar to the 2019-nCoV outbreak","26 Evolution of SARS Coronavirus and the Relevance of Modern Molecular Epidemiology","Bats and their virome: an important source of emerging viruses capable of infecting humans","Need for integrative thinking to fight against emerging infectious diseases. Proceedings of the 5th seminar on emerging infectious diseases, March 22, 2016 \u2013 current trends and proposals","2019-nCoV (Wuhan virus), a novel Coronavirus: Human-to-human transmission, travel-related cases, and vaccine readiness","Coronavirus and paramyxovirus in bats from Northwest Italy","European Bats as Carriers of Viruses with Zoonotic Potential","Viral Diversity, Prey Preference, and Bartonella Prevalence in Desmodus rotundus in Guatemala","Understanding the emerging coronavirus: what it means for health security and infection prevention","Will COVID-19 generate global preparedness?","Development of a Whole-Virus ELISA for Serological Evaluation of Domestic Livestock as Possible Hosts of Human Coronavirus NL63","Chapter 16 Emerging and Reemerging Viral Diseases","Reservoirs and vectors of emerging viruses","Non-invasive surveys of mammalian viruses using environmental DNA","Coronavirus and paramyxovirus in bats from Northwest Italy","European Bats as Carriers of Viruses with Zoonotic Potential","SARS","European Bats as Carriers of Viruses with Zoonotic Potential","26 Evolution of SARS Coronavirus and the Relevance of Modern Molecular Epidemiology","Viral abundance and its public health implications","Paramyxo- and Coronaviruses in Rwandan Bats","COVID-19: a novel zoonotic disease caused by a coronavirus from China: what we know and what we don\u2019t","Evolutionary origins of the SARS-CoV-2 sarbecovirus lineage responsible for the COVID-19 pandemic","Reservoirs and vectors of emerging viruses"],"url":["https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7049657/","https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4687304/","https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7098031/","https://doi.org/10.1016/j.coviro.2012.11.006","https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6789848/","https://doi.org/10.1016/j.bsheal.2020.01.003","https://doi.org/10.1016/j.bsheal.2020.01.003","https://doi.org/10.1016/j.coviro.2013.02.002","https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7098031/","https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6356407/","https://doi.org/10.1016/s2542-5196(20)30035-8","https://doi.org/10.1016/s2542-5196(20)30035-8","https://doi.org/10.1016/j.micinf.2020.02.002","https://doi.org/10.1016/j.tim.2018.03.001","https://doi.org/10.1016/j.tvjl.2017.12.022","https://doi.org/10.1016/j.molmed.2020.02.008","https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7098031/","https://doi.org/10.1016/j.virusres.2015.05.006","https://doi.org/10.1101/2020.03.20.000885","https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7103735/","https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7088553/","https://doi.org/10.1016/j.cell.2020.02.058","https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7086482/","https://doi.org/10.1016/b978-0-444-52272-6.00624-3","https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5038965/","https://doi.org/10.1016/j.bsheal.2020.01.003","https://doi.org/10.1016/j.bsheal.2020.01.003","https://doi.org/10.1016/b978-0-12-799942-5.00026-3","https://doi.org/10.1016/j.coviro.2012.11.006","https://doi.org/10.1016/j.respe.2017.08.001","https://doi.org/10.3855/jidc.12425","https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5741894/","https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4147689/","https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5164864/","https://doi.org/10.1016/j.jhin.2020.02.023","https://doi.org/10.1016/s0140-6736(20)30559-6","https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6356407/","https://doi.org/10.1016/b978-0-12-800947-5.00016-8","https://doi.org/10.1016/j.coviro.2013.02.002","https://doi.org/10.1101/2020.03.26.009993","https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5741894/","https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4147689/","https://doi.org/10.1016/b978-0-444-63951-6.00624-0","https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4147689/","https://doi.org/10.1016/b978-0-12-799942-5.00026-3","https://doi.org/10.1016/j.coviro.2013.01.004","https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6789848/","https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7086482/","https://doi.org/10.1101/2020.03.30.015008","https://doi.org/10.1016/j.coviro.2013.02.002"],"x":[0.7081211922472861,0.5633974699760407,0.5911961467002743,0.6009103365977094,0.5674278565855551,0.6951215896279067,0.6951216108827537,0.5823707579895058,0.57318816031902,0.5395821035633072,0.5659921341762383,0.5660013268975924,0.6091996833792963,0.5807955559600589,0.5004096999300206,0.610447252568712,0.5549688136131005,0.5923122132136623,0.5661937151457704,0.5713895050196722,0.5702364295665805,0.47888575162622965,0.5712736979854571,0.5677094992502566,0.5553704855873858,0.6708794163214951,0.6708850276011193,0.5647000520233637,0.5770088721132411,0.7231260940401155,0.5679327761049269,0.5547526592576805,0.6140145598252954,0.6113957288639889,0.5830591557251674,0.7314026783487841,0.5699275052429331,0.5901263817457252,0.5475223139760906,0.5734743345171149,0.6132641893320563,0.6399990142001932,0.5677603036484455,0.5868159276257154,0.6146529810390611,0.6123594555120323,0.5994494016888082,0.5551573037223274,0.5643847501828555,0.5523728921886992],"y":[0.6849639558186399,0.4735917839748253,0.4831288443931648,0.4867030548676216,0.4416227420911306,0.6921708079136458,0.6921710717059671,0.4651630102929126,0.4632092037920812,0.4660291746737673,0.48367297454893843,0.4836681769694601,0.596599595404513,0.4661632224622236,0.408746993240643,0.5927505477389495,0.4304925331805724,0.47509655630479075,0.4665700062786014,0.46023301280387147,0.4852842375451689,0.38252586446435405,0.472135844191226,0.43792057743083973,0.49789677602314986,0.6684145775261031,0.6684172957339358,0.476810221186306,0.50752687232306,0.8019664020107125,0.4710232289668835,0.4219309090979575,0.5941074117143587,0.5953397118087241,0.5717998943477346,0.7810906690795177,0.45907392310713047,0.49183143106382365,0.42218608802651136,0.4256565822417694,0.5967089315870787,0.6454337130554985,0.4372420232958211,0.4576430332717471,0.5945501183102827,0.5947965404806923,0.503684949688095,0.4488712963113657,0.46243908275464474,0.4648399988893197]},"selected":{"id":"28123"},"selection_policy":{"id":"28153"}},"id":"28050","type":"ColumnDataSource"},{"attributes":{"axis":{"id":"28099"},"dimension":1,"ticker":null},"id":"28102","type":"Grid"},{"attributes":{"fill_color":{"field":"color"},"line_color":{"field":"color"},"radius":{"field":"rad","units":"data"},"x":{"field":"x"},"y":{"field":"y"}},"id":"28076","type":"Circle"}],"root_ids":["28133"]},"title":"Bokeh Application","version":"2.0.1"}}
        </script>
        <script type="text/javascript">
          (function() {
            var fn = function() {
              Bokeh.safely(function() {
                (function(root) {
                  function embed_document(root) {
                    
                  var docs_json = document.getElementById('28370').textContent;
                  var render_items = [{"docid":"0eec7a5d-3b52-49ed-9d41-90a7d26934bb","root_ids":["28133"],"roots":{"28133":"8c050973-10f9-464f-8f41-c65363121638"}}];
                  root.Bokeh.embed.embed_items(docs_json, render_items);
                
                  }
                  if (root.Bokeh !== undefined) {
                    embed_document(root);
                  } else {
                    var attempts = 0;
                    var timer = setInterval(function(root) {
                      if (root.Bokeh !== undefined) {
                        clearInterval(timer);
                        embed_document(root);
                      } else {
                        attempts++;
                        if (attempts > 100) {
                          clearInterval(timer);
                          console.log("Bokeh: ERROR: Unable to run BokehJS code because BokehJS library is missing");
                        }
                      }
                    }, 10, root)
                  }
                })(window);
              });
            };
            if (document.readyState != "loading") fn();
            else document.addEventListener("DOMContentLoaded", fn);
          })();
        </script>
    
  </body>
  
</html>