



<!DOCTYPE html>
<html lang="en">
  
  <head>
    
      <meta charset="utf-8">
      <title>Bokeh Plot</title>
      
      
        
          
        
        
          
        <script type="text/javascript" src="https://cdn.bokeh.org/bokeh/release/bokeh-2.0.1.min.js" integrity="sha384-JpP8FXbgAZLkfur7LiK3j9AGBhHNIvF742meBJrjO2ShJDhCG2I1uVvW+0DUtrmc" crossorigin="anonymous"></script>
        <script type="text/javascript" src="https://cdn.bokeh.org/bokeh/release/bokeh-widgets-2.0.1.min.js" integrity="sha384-xZlADit0Q04ISQEdKg2k3L4W9AwQBAuDs9nJL9fM/WwzL1tEU9VPNezOFX0nLEAz" crossorigin="anonymous"></script>
        <script type="text/javascript" src="https://cdn.bokeh.org/bokeh/release/bokeh-tables-2.0.1.min.js" integrity="sha384-4BuPRZkdMKSnj3zoxiNrQ86XgNw0rYmBOxe7nshquXwwcauupgBF2DHLVG1WuZlV" crossorigin="anonymous"></script>
        <script type="text/javascript">
            Bokeh.set_log_level("info");
        </script>
        
      
      
    
  </head>
  
  
  <body>
    
      
        
          
          
            
              <div class="bk-root" id="b0e05c20-e9ea-414e-b8bc-85991ec0d789" data-root-id="8491"></div>
            
          
        
      
      
        <script type="application/json" id="8728">
          {"ec36bda8-e37f-4504-ae0f-16c12589fe11":{"roots":{"references":[{"attributes":{"fill_alpha":{"value":0.1},"fill_color":{"field":"color"},"line_alpha":{"value":0.1},"line_color":{"field":"color"},"radius":{"field":"rad","units":"data"},"x":{"field":"x"},"y":{"field":"y"}},"id":"8435","type":"Circle"},{"attributes":{},"id":"8513","type":"BasicTickFormatter"},{"attributes":{},"id":"8496","type":"UnionRenderers"},{"attributes":{},"id":"8419","type":"BasicTicker"},{"attributes":{"data":{"authors":["Doysabas, Karla Cristine C.; Oba, Mami; Furuta, Masaya; Iida, Keisuke; Omatsu, Tsutomu; Furuya, Tetsuya; Okada, Takashi; Sutummaporn, Kripitch; Shimoda, Hiroshi; Wong, Min-Liang; Wu, Chung-Hsin; Ohmori, Yasushige; Kobayashi, Ryosuke; Hengjan, Yupadee; Yonemitsu, Kenzo; Kuwata, Ryusei; Kim, Yoo-Kyung; Han, Sang-Hyun; Sohn, Joon-Hyuk; Han, Sang-Hoon; Suzuki, Kazuo; Kimura, Junpei; Maeda, Ken; Oh, Hong-Shik; Endoh, Daiji; Mizutani, Tetsuya; Hondo, Eiichi","Min, Jae; Cella, Eleonora; Ciccozzi, Massimo; Pelosi, Antonello; Salemi, Marco; Prosperi, Mattia","Reza S. Abhari; Marcello Marini; Ndaona Chokani","Casey M Zipfel; Shweta Bansal","Xu, Bo; Gutierrez, Bernardo; Mekaru, Sumiko; Sewalk, Kara; Goodwin, Lauren; Loskill, Alyssa; Cohn, Emily L.; Hswen, Yulin; Hill, Sarah C.; Cobo, Maria M.; Zarebski, Alexander E.; Li, Sabrina; Wu, Chieh-Hsi; Hulland, Erin; Morgan, Julia D.; Wang, Lin; O\u2019Brien, Katelynn; Scarpino, Samuel\u00a0V.; Brownstein, John S.; Pybus, Oliver G.; Pigott, David M.; Kraemer, Moritz U. G.","Min, Jae; Cella, Eleonora; Ciccozzi, Massimo; Pelosi, Antonello; Salemi, Marco; Prosperi, Mattia","Mishra, Nischay; Fagbo, Shamsudeen F.; Alagaili, Abdulaziz N.; Nitido, Adam; Williams, Simon H.; Ng, James; Lee, Bohyun; Durosinlorun, Abdulkareem; Garcia, Joel A.; Jain, Komal; Kapoor, Vishal; Epstein, Jonathan H.; Briese, Thomas; Memish, Ziad A.; Olival, Kevin J.; Lipkin, W. Ian","Lin, Xian-Dan; Wang, Wen; Hao, Zong-Yu; Wang, Zhao-Xiao; Guo, Wen-Ping; Guan, Xiao-Qing; Wang, Miao-Ruo; Wang, Hong-Wei; Zhou, Run-Hong; Li, Ming-Hui; Tang, Guang-Peng; Wu, Jun; Holmes, Edward C.; Zhang, Yong-Zhen","Pollack, Marjorie P.; Pringle, Craig; Madoff, Lawrence C.; Memish, Ziad A.","Roux, Simon; Matthijnssens, Jelle; Dutilh, Bas E.","Bamias, Giorgos; Boletis, John","Wang, Lin-Fa; Shi, Zhengli; Zhang, Shuyi; Field, Hume; Daszak, Peter; Eaton, Bryan T.","Xi Zhang; Hua-Xiang Rao; Yuwan Wu; Yubei Huang; Hongji Dai","Shi, Z.; Wang, L.-F.","Cao, Wuchun; Fang, Liqun; Xiao, Dan","Xu, Bo; Gutierrez, Bernardo; Mekaru, Sumiko; Sewalk, Kara; Goodwin, Lauren; Loskill, Alyssa; Cohn, Emily L.; Hswen, Yulin; Hill, Sarah C.; Cobo, Maria M.; Zarebski, Alexander E.; Li, Sabrina; Wu, Chieh-Hsi; Hulland, Erin; Morgan, Julia D.; Wang, Lin; O\u2019Brien, Katelynn; Scarpino, Samuel\u00a0V.; Brownstein, John S.; Pybus, Oliver G.; Pigott, David M.; Kraemer, Moritz U. G.","Zhou, Chenghu; Su, Fenzhen; Pei, Tao; Zhang, An; Du, Yunyan; Luo, Bin; Cao, Zhidong; Wang, Juanle; Yuan, Wen; Zhu, Yunqiang; Song, Ci; Chen, Jie; Xu, Jun; Li, Fujia; Ma, Ting; Jiang, Lili; Yan, Fengqin; Yi, Jiawei; Hu, Yunfeng; Liao, Yilan; Xiao, Han","Gorbalenya, A.E.","Lin, Xian-Dan; Wang, Wen; Hao, Zong-Yu; Wang, Zhao-Xiao; Guo, Wen-Ping; Guan, Xiao-Qing; Wang, Miao-Ruo; Wang, Hong-Wei; Zhou, Run-Hong; Li, Ming-Hui; Tang, Guang-Peng; Wu, Jun; Holmes, Edward C.; Zhang, Yong-Zhen","Young June Choe","Vazquez, Arni S.R. Srinivasa Rao; Jose A.","Ivanov, Dmitry","Pavel Skums; Alexander Kirpich; Pelin Icer Baykal; Alex Zelikovsky; Gerardo Chowell","Boulos, Maged N Kamel","Hagai Rossman; Ayya Keshet; Smadar Shilo; Amir Gavrieli; Tal Bauman; Ori Cohen; Ran Balicer; Benjamin Geiger; Yuval Dor; Eran Segal","Carla Mavian; Simone Marini; Costanza Manes; Ilaria Capua; Mattia Prosperi; Marco Salemi","Liang, Wannian; Zhu, Zonghan; Guo, Jiyong; Liu, Zejun; He, Xiong; Zhou, Weigong; Chin, Daniel P.; Schuchat, Anne","Boulos, Maged N Kamel","Boulos, Maged N Kamel","Xu, Bo; Gutierrez, Bernardo; Mekaru, Sumiko; Sewalk, Kara; Goodwin, Lauren; Loskill, Alyssa; Cohn, Emily L.; Hswen, Yulin; Hill, Sarah C.; Cobo, Maria M.; Zarebski, Alexander E.; Li, Sabrina; Wu, Chieh-Hsi; Hulland, Erin; Morgan, Julia D.; Wang, Lin; O\u2019Brien, Katelynn; Scarpino, Samuel\u00a0V.; Brownstein, John S.; Pybus, Oliver G.; Pigott, David M.; Kraemer, Moritz U. G.","James Zhu; Jiwoong Kim; Xue Xiao; Yunguan Wang; Danni Luo; Ran Chen; Lin Xu; He Zhang; Guanghua Xiao; John Schoggins; Xiaowei Zhan; Tao Wang; Yang Xie","Mengying Dong; Xiaojun Cao; Mingbiao Liang; Lijuan Li; Huiying Liang; Guangjian Liu","Aramini, Jeffery J.; Muchaal, Pia K.; Pollari, Frank","Arabi, Yaseen M.; Al-Enezi, Farhan; Longuere, Kajsa-Stina; Balkhy, Hanan H.; Al-Sultan, Mohamed; Al-Omari, Awad; Al-Hameed, Fahad M.; Carson, Gail; Shindo, Nahoko; Fowler, Robert","Berto, A.; Anh, P. H.; Carrique\u2010Mas, J. J.; Simmonds, P.; Van Cuong, N.; Tue, N. T.; Van Dung, N.; Woolhouse, M. E.; Smith, I.; Marsh, G. A.; Bryant, J. E.; Thwaites, G. E.; Baker, S.; Rabaa, M. A.; Kiet, Bach Tuan; Boni, Maciej F.; Phu, Bui Duc; Campbell, James I; Hung, Dang Manh; Huong, Dang Thao; Oanh, Dang Tram; Day, Jeremy N.; Van Tan, Dinh; van Doorn, H. Rogier; Han, Duong An; Farrar, Jeremy J; Trang, Hau Thi Thu; Nghia, Ho Dang Trung; Long, Hoang Bao; Van Duong, Hoang; Thu, Huynh Thi Kim; Cuong, Lam Chi; Hung, Manh; Phuong, Thanh; Phuc, Thi; Phuong, Thi; Luat, Xuan; Ha, Luu Thi Thu; Van Chuong, Ly; Loan, Mai Thi Phuoc; Nadjm, Behzad; Bao, Ngo Thanh; Tu, Nguyen Canh; Thuan, Nguyen Dac; Dong, Nguyen; Chuyen, Nguyen Khac; An, Nguyen Ngoc; Vinh, Nguyen Ngoc; Hung, Nguyen Quoc; Dung, Nguyen Thanh; Minh, Nguyen Thanh; Binh, Nguyen Thi; Tham, Nguyen Thi Hong; Tien, Nguyen Thi Hong; Chuc, Nguyen Thi Kim; Le Ngoc, Nguyen Thi; Ha, Nguyen Thi Lien; Lien, Nguyen Thi Nam; Diep, Nguyen Thi Ngoc; Nhung, Nguyen Thi; Chau, Nguyen Thi Song; Chi, Nguyen Thi Yen; Trinh, Nguyen Thieu; Van, Nguyen Thu; Van Hung, Nguyen; Van Kinh, Nguyen; Van Minh Hoang, Nguyen; Van My, Nguyen; Van Thang, Nguyen; Van Thanh, Nguyen; Van Vinh Chau, Nguyen; Van Xang, Nguyen; My, Pham Ha; Khoa, Pham Thi Minh; Tam, Pham Thi Thanh; Van Lao, Pham; Van Minh, Pham; Van Be Bay, Phan; Rahman, Motiur; Thompson, Corinne; Ngan, Ta Thi Dieu; Nhu, Tran Do Hoang; Chau, Tran Hoang Minh; Toan, Tran Khanh; Phuc, Tran My; Hong, Tran Thi Kim; Dung, Tran Thi Ngoc; Thanh, Tran Thi Thanh; Minh, Tran Thi Thuy; Nguyen, Tran Thua; Hien, Tran Tinh; Tri, Trinh Quang; Hien, Vo Be; Tai, Vo Nhut; Cuong, Vo Quoc; Phat, Voong Vinh; Huong, Vu Thi Lan; Hang, Vu Thi Ty; Wertheim, Heiman; Bogaardt, Carlijn; Brierley, Liam; Chase\u2010Topping, Margo; Ivens, Al; Lu, Lu; Rambaut, Andrew; Woolhouse, Mark; Cotten, Matthew; Oude Munnink, Bas B.; Kellam, Paul; Phan, My Vu Tra; van der Hoek, Lia; Deijs, Martin; Jebbink, Maarten F.; Farsani, Seyed Mohammad Jazaeri; Saylors, Karen; Wolfe, Nathan","Wray, Amy K.; Olival, Kevin J.; Mor\u00e1n, David; Lopez, Maria Renee; Alvarez, Danilo; Navarrete-Macias, Isamara; Liang, Eliza; Simmons, Nancy B.; Lipkin, W. Ian; Daszak, Peter; Anthony, Simon J.","Carr, Anthony P.","Phillippy, Adam M; Deng, Xiangyu; Zhang, Wei; Salzberg, Steven L","Roux, Simon; Matthijnssens, Jelle; Dutilh, Bas E.","Huipeng Liao; Gifty Marley; Yafei Si; Zaisheng Wang; Yewei Xie; Cheng Wang; Weiming Tang","Al-Ahmadi, Khalid; Alahmadi, Sabah; Al-Zahrani, Ali","","Dima Kagan; Jacob Moran-Gilad; Michael Fire","Boulos, Maged N Kamel","Moritz U.G. Kraemer; Chia-Hung Yang; Bernardo Gutierrez; Chieh-Hsi Wu; Brennan Klein; David M. Pigott; Louis du Plessis; Nuno R Faria; Ruoran Li; William P. Hanage; John S Brownstein; Maylis Layan; Alessandro Vespignani; Huaiyu Tian; Christopher Dye; Simon Cauchemez; Oliver Pybus; Samuel V Scarpino","Mariano Supino; Alberto d'Onofrio; Federico Luongo; Giovanni Occhipinti; Alma Dal Co","Kofstad, Tone; Jonassen, Christine M.","Reed, Kurt D.","Peter Byass","Yasuhiko Kamikubo; Atsushi Takahashi"],"color":["#ef8a62","#ef8a62","#ef8a62","#ef8a62","#ef8a62","#ef8a62","#ef8a62","#ef8a62","#ef8a62","#ef8a62","#ef8a62","#ef8a62","#ef8a62","#ef8a62","#ef8a62","#ef8a62","#ef8a62","#ef8a62","#ef8a62","#ef8a62","#ef8a62","#ef8a62","#ef8a62","#ef8a62","#ef8a62","#ef8a62","#ef8a62","#ef8a62","#ef8a62","#ef8a62","#ef8a62","#ef8a62","#ef8a62","#ef8a62","#ef8a62","#ef8a62","#ef8a62","#ef8a62","#ef8a62","#ef8a62","#ef8a62","#ef8a62","#ef8a62","#ef8a62","#ef8a62","#ef8a62","#ef8a62","#ef8a62","#ef8a62","#ef8a62"],"date":["2019-01-02","2016-09-28","2020-04-01","2020-04-01","2020-03-24","2016-09-28","2019-04-10","2017-07-31","2013-02-28","2019-12-31","2008-09-30","2006-12-03","2020-03-26","2017-12-31","2019-09-30","2020-03-24","2020-03-20","2008-12-31","2017-07-31","2020-03-18","2020","2020-04-30","2020-03-27","2004-01-28","2020-03-20","2020-03-20","2004-01-03","2004-01-28","2004-01-28","2020-03-24","2020-02-12","2020-03-30","2013-05-24","2016-07-12","2017-04-18","2016-09-22","2007-12-31","2009-09-16","2019-12-31","2020-03-23","2019-07-15","2006-12-31","2020-03-20","2004-01-28","2020-03-06","2020-03-27","2011-10-17","2018-12-31","2020-03-31","2020-03-27"],"doc":["c9kpohz5","sm8kgh68","elmagpu4","f3ds1rq6","l63z3btv","sm8kgh68","4kvkioa3","xutbbxpr","9uvr4592","1qj9ri18","pbxd67tt","v08cs51n","brwmpm5a","apc0lm5e","73tilrw4","l63z3btv","5gbkrs73","5yk1j4ms","xutbbxpr","grx4gx95","47goscew","8d1076ge","ry9wpcxo","kuybfc1y","lioj0tkn","szg12wfa","0ynor3i5","kuybfc1y","kuybfc1y","l63z3btv","xetzg7gp","3wuh6k6g","wgu21rf7","7hjsux4m","tcim3wrz","9ojdhxwk","jp0ui9cu","g4sjyytr","1qj9ri18","fb0nwixw","3nfukrfl","cgcvfftf","kh9whqzd","kuybfc1y","yu89n6rq","5war06j2","t9lvgwmf","fgrwwzn7","r141na6j","7onenxg7"],"journal":["Virus Research","Glob Health Res Policy","","","Sci Data","Glob Health Res Policy","PLoS One","Virology","International Journal of Infectious Diseases","Reference Module in Life Sciences","American Journal of Kidney Diseases","Emerg Infect Dis","","Genetics and Evolution of Infectious Diseases","Global Health Journal","Sci Data","Geography and Sustainability","Encyclopedia of Virology","Virology","","Infection Control &amp; Hospital Epidemiology","Transportation Research Part E: Logistics and Transportation Review","","Int J Health Geogr","","","Emerg Infect Dis","Int J Health Geogr","Int J Health Geogr","Sci Data","","","MMWR Morb Mortal Wkly Rep","BMC Anesthesiol","Zoonoses Public Health","Ecohealth","Canine Internal Medicine Secrets","BMC Bioinformatics","Reference Module in Life Sciences","","Int J Environ Res Public Health","Clinical Microbiology and Infection","","Int J Health Geogr","","","PLoS One","Reference Module in Biomedical Sciences","",""],"rad":[0.01660000394168358,0.012123096786808351,0.011619653685222928,0.011364274813625345,0.010414146080036383,0.009587907980701044,0.009218508312868797,0.009180404871096885,0.009074261380311806,0.007846094020216153,0.007591357264649945,0.007339155101127844,0.0071352939641652355,0.007004886723071273,0.006757466930187739,0.006698782963770203,0.006519782348772618,0.006448210388740714,0.006299665391819902,0.006174249670764334,0.00606643083549352,0.006036314100535383,0.005667806460276331,0.005667791565370351,0.00555839861168146,0.005331706350313627,0.005118284552426744,0.0049786767784184005,0.004974956926906072,0.004957801915846077,0.004788664024663209,0.004628248291141125,0.004584288008792273,0.004538916685271737,0.004463009040004154,0.0040651496173912186,0.003913161537037957,0.003649297579707451,0.003623161419612658,0.003437436345625855,0.003437024822033255,0.003278569630261869,0.0032773236114187077,0.0032195988851640234,0.003114605036985777,0.0031112229950895826,0.0030393128848952428,0.003038906228891843,0.0030316850215761394,0.00301891856424],"scr":[0.8,0.537,0.507,0.492,0.436,0.388,0.366,0.364,0.357,0.285,0.27,0.255,0.243,0.236,0.221,0.218,0.207,0.203,0.194,0.187,0.18,0.179,0.157,0.157,0.15,0.137,0.125,0.116,0.116,0.115,0.105,0.096,0.093,0.091,0.086,0.063,0.054,0.038,0.037,0.026,0.026,0.016,0.016,0.013,0.007,0.007,0.002,0.002,0.002,0.001],"text":["A NGS library was first constructed according to the manufacturer's protocol using the NEBNext \u00ae Ultra\u2122 RNA Library Prep Kit for Illumina \u00ae (New England Bio Labs, Massachusets, USA). Sequencing of paired ends was performed with the MiSeq Reagent Kit v2 (300 cycles) (Illumina \u00ae , San Diego, CA, USA) wherein each read was output as a FASTQ format file from the MiSeq reporter and analyzed using CLC genomics Workbench 6.5.1 (Filgen, Nagoya, Japan). Commands for trimming sequences with low quality and commands for removing 20 bases corresponding to primers from the 5'-end of each read were executed. Trimmed reads were mapped to each sequence of the prepared data sets for viral identification (Supplementary Table 1 ) with a 50% length fraction and 80% similarity fraction. Reads with a minimum consensus length of 200 and a minimum mapping read number of 100 were selected, and the reference sequence that was mapped at the forward and reverse end of the predicted fragment between the primers' binding site was selected and judged as positive. For animal species identification, NGS data that showed less than 50% similarity to the COI reference in consecutive full-length sequence of 5' and 3\u2032 ends of NGS data (less than 151 bp each sequence) were eliminated. Since only one COI genomic sequence per species was included in the reference despite the intraspecies variation, the maximum variation in COI barcoding was computed with the formula: similarity + [(100-similarity) \u00d7 0.11]. The maximum similarity was used as the threshold to determine the species.","In this paper we reviewed epidemiological contact tracing information from public health agencies and peer-reviewed literature, in order to see geographic and temporal distribution of MERS cases around the globe. Concurrently, we used genetic sequences to infer transmission dynamics and inter-host evolution of MERS-CoV. The combined analysis was used to present a phylodynamic picture, detailing international, zoonotic and healthcare associated transmissions at genetic and population levels. These analyses can be used to understand pathogen spread and to implement public health measures to curb a pandemic [23].","Additionally, limitations on the movement of people and the closing of national and international transportation have been implemented. The primary goal of public authorities is to manage the outbreak so as to avoid overwhelming the health care system, in addition to buy time in order to bring online more capacity and resources that can handle the current and potential future peaks in the demand for emergency services. As such, there is an acute need to better understand the temporal growth of the outbreak (both symptomatic and asymptomatic), the geographic distribution of the outbreak, and the evolution of hot spots in the outbreak.","Our results suggest that the dynamics of the 2019-2020 influenza season were impacted directly through transmission-reducing behaviors resulting from COVID-19 risk perception or indirectly through ecological interference between the two infections. While COVID-19 testing remains limited, researchers have advocated using existing sources of ILI tracking for COVID-19 surveillance. In addition to this strategy, we suggest that the interaction between a novel and circulating pathogen due to shared preventative behaviors or a shared susceptible pool can be leveraged to understand circulation of the former.","While every effort has been made to standardize the geographic representation of cases, with a common source of reference shapefiles outlined, when considering geographic analysis of the data a few limitations must be acknowledged. The first is that, while native speakers were consulted wherever possible, there is the possibility of transliteration errors occurring when extracting data from native language into English-script analogues. This is most likely to happen when looking at point data. We have provided sources consulted so that users may cross-reference the original source wherever this is may be an issue. While administrative units are supplied with shapefiles sources and unique identifiers with these files so that users can understand the corresponding geographic scale which the row represents, with points, different settlements cover different spatial extents. Should users wish to incorporate this information into spatially-dependent models, they should exercise caution in possible misrepresentation of geographic specificity. We recommend that sensitivity of results could be evaluated by using buffers around point latitude and longitudes, or cross-referencing city-gazettes.","There were three aims for this paper: (1) to examine case incidence trends over time and geographic area using epidemiological information, (2) to trace evolutionary history of the virus in circulation using genetic data and (3) to explore ways in which these two analyses can be combined to design public health interventions.","In summary, as in other geographic catchment areas, bats endemic in the KSA have richly diverse viromes. The most remarkable single finding was the presence of Hepe-Astroviruses, that are similar to Bastroviruses and bastro-like viruses but with a distinctive hepevirus-like genomic organization. Further molecular and serological studies are required to test whether these viruses are capable or infecting humans, domestic animals, or other wildlife.","At least 120 bat species are found in China, mainly distributed in the eastern, central, and southern regions of that country (Zhang et al., 1997) . Herein we report novel and diverse CoVs and SARS-related CoVs identified in Rhinolophus, Miniopterus, Murina and Myotis spp. bats sampled from several geographic regions of China. In addition, we inferred their genomic characteristics and evolutionary relationships with known viruses and their hosts.","It is clearly very early in our body of knowledge on the behavior of this nCoV. Information on the epidemiology of this organism including the host/reservoir, the geographic distribution and the clinical spectrum of disease is presently being identified -a sample of nine confirmed cases is too small to feel confident that we fully understand this novel virus.","Assembling genomes of uncultivated viruses can not only identify some of the most abundant and widespread viruses in an ecosystem, but these sequences also represent foundational data for targeted follow-up experiments aimed at further characterizing these novel viruses. In the case of crAssphage, two major studies leveraged this initial genome sequence to better understand the diversity and host of these phages. First, predicted proteins from the original crAssphage genome were used as \"bait\" to identify related phages in a broad range of metagenomes. This revealed an extensive and diverse group of \"crAss-like\" phages predicted to represent a new family within the Caudovirales order, that may be related to Podoviridae. Genome comparisons within this new family also enabled the identification of conserved structural and replication gene modules. Meanwhile, another study was able to isolate a member of the crAssphage-like family through broth enrichment on Bacteroides intestinalis strains isolated from human gut samples, confirming the computational predictions from bioinformatic analyses that these phages were likely infecting Bacteroidetes hosts and had a Podoviridae-like morphology.","No matter what the obstacles were, research about the causative factor(s) of BEN started almost immediately after its recognition as a discrete nosological entity. For a disease with such a narrow, defined, and well-preserved geographic distribution, it was a logical assumption that the causative factor should be an environmental one endemic to the afflicted regions. Nonetheless, along with this ecological influence, other factors had to be taken into consideration. For example, an obvious question was how could such an environmental trigger selectively affect certain villages and households and leave the ones just next to them undisturbed? Could these differences be explained by the existence of genetic variability between diseased and healthy families? Or could it be an infectious agent prevalent to the inflicted foci? Another pivotal question related to the long incubation period of the disease, which pointed to a long exposure to a low-dose harming agent that existed in the environment for many years. If that is the case, why then did BEN appear only during the second half of the century in the endemic areas and not before? Such a temporal association requires a major local environmental change before the first appearance of the disease, and such an alteration is not easy to identify in the history of the region. In addition, any etiologic hypothesis should take into consideration not only the nephropathy itself, but also the increased incidence of upper urothelial neoplasms in the patients. Finally, if somebody were to solve the mystery of BEN, he or she should use information from both animal models of interstitial nephropathy and the recognized effects of the candidate environmental factors in human health and disease. Given all these varied questions, it is not surprising that throughout the years, the effort to solve the riddle of BEN involved not only nephrologists, but also scientists from such diverse fields as occupational medicine, animal models of human disease, environmental sciences and epidemiology, oncology, genetics, and geology.","The discovery of SARS-CoV has boosted the search for novel coronaviruses of human and animal origin. Bats have been chosen as the main target because of their species diversity, large population size, broad geographic distribution, ability for long-distance migration, and habit of roosting in large groups. In addition to the SARS-like\u2013CoVs described above, many other coronaviruses have been detected by PCR among diverse bat populations in Hong Kong (Table 2).","The results of spatiotemporal scans are sensitive to various parameters, such as the maximum spatial and temporal cluster sizes. Thus, the selection of the maximum radius of the spatial scanning window and the maximum length of the temporal scanning window were very important. In this study, we selected the maximum spatial cluster sizes as 10% and 30% of the total population at risk and the maximum temporal cluster sizes as 50% of the total study period. Based on the optimal spatiotemporal parameters, retrospective space-time scanning analysis was applied to identify the geographic areas and time periods of potential clusters with significantly higher COVID-19 and SARS incidence than those of nearby areas.","In this context, we and other groups continued the search for the direct progenitor of SARS-CoV and made great progress in the last 10 years following the initial discovery of SL-CoVs in horseshoe bats. First, highly diverse SL-CoVs have been found not only in Chinese but also in European and African bats, indicating a much wider geographic distribution and long evolutionary history of SL-CoVs in different bat populations The region of nsp3 is highly variable and was calculated alone.","During the 2002-2003 SARS outbreak, mainland China reported 5,327 probable cases, of whom 343 died, amounting to a case fatality ratio (CFR) of 6.4%. The epidemic spanned a large geographical extent (170 counties of 22 provinces) but clustered in two areas: first in Guangdong Province, and about 3 months later in Beijing and its surrounding areas Shanxi, Inner Mongolia, Hebei, and Tianjin. Fig. 1 shows the temporal distribution of SARS in the six most seriously affected geographic areas of mainland China. Spatiotemporal analyses indicated that the spread of SARS occurred in two different patterns. 13 In the early stage of the epidemic, especially before strict control measures were taken, SARS spread to new areas randomly through certain index cases. Thereafter, human travel along transportation routes influenced the transmission of SARS, as illustrated by the spread of SARS in middle North China and South China. The epidemic period in middle North China was shorter than in South China, but the geographic spread was wider. SARS not only spread locally, but also diffused quickly and resulted in several outbreaks in areas of middle North China close to Beijing. In contrast, the SARS epidemic in South China was mainly limited to Guangdong Province.","Geographical information came in two forms: references to specific settlements, and references to areas, typically administrative units. All geographic metadata was standardized via the use of a common geographic reference table. New unique distinct locations were added to the reference table, and all subsequent entries had geographic information populated from this table. Location names are often duplicated within a country, so contextual information was used to ensure the correct site was selected. When the site name was not found, information from the text was also used to scan sites in the approximate area to check for alternate spellings of the site name. We had curators skilled with the following languages: English, mandarin Chinese, Cantonese, Spanish, and Portuguese. To add new geographies to the database, Google Maps (https://www.maps.google.co.uk) and Google Earth (http://www.google.co.uk/intl/en_uk/earth) were used to determine latitude and longitudes, and relevant administrative metadata was extracted by querying the relevant country reference shapefile. For locations that are administrative units, information was populated by referring to the country reference shapefile, sourced primarily from GADM (https://gadm.org/) with the admin_id field to allow for easier polygon selection.","The decisions and actions of large-scale epidemic prevention and control depend on data support. The development and application of big data will undoubtedly contribute to quickly identifying the spatiotemporal process of epidemic development, prevention and control measures, and the resulting effectiveness. Strategies for gathering and integrating massive geographic and social-spatial information in the face of prevention and control of an epidemic emergency is the most basic problem for subsequent temporal and spatial mining and analysis. Based on the unified geographical framework, this research quickly absorbed and integrated geographical big data, including the WHO data released internationally, daily domestic health and disease control data, professional population health platform data, Tencent location request data, Baidu migration data, microblog text data, patient spatiotemporal trajectory data, international airline data, census data, education enrollment data, land cover data, remote sensing imagery and other multi-source data through statistical data collection, network data mining, API interface connection, and international and domestic data platform exchange. With the support of unified spatial registration, raster-vector transformation, statistical normalization, format transformation, etc., these data were rapidly merged and applied to the spatiotemporal analysis and visualization systems for the COVID-19 epidemic. In this process, due to the weakness of current research on the classification and security management of spatiotemporal big data, it is clear that GIS needs to be developed in the aspects of automatic correlation aggregation of data, historical data tracing, adaptive conversion of heterogeneous data, and standardized storage of multi-source data.","Geographic distribution of places of virus isolation is another important characteristic relative to which virus phylogeny may be evaluated. This field of study belongs to phylogeography. The evolution of human JC polyomavirus provides an example of confinement of circulation of virus clusters to geographically isolated areas, represented by three continents. Recent identification of West Nile virus in the USA illustrates geographical expansion of an Old World virus into the New World. Analysis of phylogenies of field isolates of rabies virus of the family Rhabdoviridae sampled from different animals over Europe led to the recognition that interspecies virus expansion is occuring faster when compared to geographical expansion.","Since the discovery of SARSr-Rh-BatCoVs in bats in 2005 Li et al., 2005) , intense effort has focused on characterizing additional CoVs in bats, leading to the identification of diverse SARSlike viruses and other bat CoVs worldwide Hu et al., 2015; Huang et a, 2016; Woo et al., 2006) . More importantly, recent studies provide more evidence that human CoVs including MERS and SARS viruses may have their ultimate ancestry in bats (Annan et al., 2013; Corman et al., 2014 Corman et al., , 2015 Ge et al., 2013; Huynh et al., 2012; Hu et al., 2015; Ithete et al., 2013; Tao et al., 2017) . These results highlight the potential significance of detecting and characterizing of CoVs in bats before their emergence in humans. Herein we describe diverse SARSr-Rh-BatCoVs and alphacoronaviruses (including novel species) in 13 species of bats sampled from three provinces in China, with an overall prevalence of 6.84%. As such, these data reveal both the high genetic diversity and the wide geographic distribution of CoVs in diverse bats in China.","The analysis of the first 7,755 cases of COVID-19 in Korea showed a first increase of cases in specific geographic region comparable to the epidemiological situation in China and Italy (4, 5) . In Korea, despite a relevant number of cases per week in late February and early March, the peak seemed to have levelled off, yet the first wave did not go away. With descriptive analysis of interim surveillance data, the data suggest that the patient outcome differed markedly between the geographic regions, highlighting the magnitude of surge capacity on access to hospitalitzation.","Comparison of test criteria results of (iii) and (iv) with their corresponding geographic and socio-demographic details will yield further investigations of signs and symptoms Suppose we define two events and using the sets and as below:","We model a global SC of a company selling the lightning equipment, in total five different products. This is a multi-stage SC with suppliers, factory, distributions centres (DC), and customers located in different geographic zones (Fig. 1) .","Our results presented here summarize the up-to-date picture of the spread of SARS-CoV-2 between and within countries and geographic regions. The structural properties of the inferred network, transmission clusters and pathways as well as virus introduction routes emphasize the extent of the global epidemiological transmission network. It also demonstrates the importance of internationally coordinated public health measures and highlights how epidemiological and molecular surveillance analyses complement each other to characterize the spatial-temporal spread of epidemics.","SarsNet is unique in that it permits users to examine the temporal dimension of SARS spread by creating animated series (dynamic maps \u2013 see [19]) based on successive static maps of daily or weekly SARS occurrences. The animated series are built using Macromedia Flash or as animated GIFs. Maps included in an animated series must cover exactly the same geographic area, be of the same scale and use the same classifications (if any). SarsNet automatically takes cares of all these conditions.","As testing the entire population for the presence of 2019-nCoVis is currently not feasible, we developed a simple one-minute online questionnaire with the goal of early and temporal detection of geographic clusters in which the virus is spreading. The survey was posted online as a Google Form ( http://predict-corona.org/ ) on March 14th. This additional information provides real-time analysis of symptoms over the course of days as the virus spreads through the population, and renders an informative situation snapshot.","where the corners represent distinct tree topologies and the center star-like trees. Quartets are considered \"resolved\" when the three likelihoods are significantly different (phylogenetic signal, most dots equally distributed in the corners, i.e. data are suitable for robust phylogeny inference), unresolved or partially resolved, when all three likelihood values or two of them are not significantly different (phylogenetic noise, most dots in sides or center areas, i.e. data may not be sufficient for robust phylogeny inference). Extensive simulation studies have shown that, for sequences to be considered robust in terms of phylogenetic signal, side/center areas of the likelihood mapping must include &lt;40% of the unresolved quartets 12 , way lower than the 52.9% unresolved quartets detected in the SARS-CoV-2 full genomes alignment ( Figure 3b ). It is important to keep in mind that such a lack of phylogenetic signal makes the overall topology of any SARS-CoV-2 tree unreliable, and even clades with high bootstrap values should be interpreted with extreme caution. Finally, the mean genetic distance of each sequence from the root of a ML tree indicated that the available genomic data lack temporal signal (no significant linear correlation between accumulated genetic distance and sampling time, Figure 3c ). This is, again, expected in genomes obtained over a very short period of time (~ three months) since the beginning of the outbreak. Bayesian analysis 13 , which infers phylogenetic and phylogeographic patterns from a posterior distribution of trees, might facilitate comparisons about different evolutionary scenarios, help in retrieving the correct topology, and estimate an accurate evolutionary rate. With more genome sequences sampled from different time points and geographic areas made available, literally, every day, Bayesian analysis may soon be a viable option. However, as long the new data do not increase phylogenetic and temporal signal, Bayesian phylogenetic and phylogeography results would remain equally questionable.","In the response to SARS, opportunities exist for ensuring broad public health benefits. A stronger public health infrastructure capable of improved preparedness and response to SARS will also improve control of other diseases. Strengthening infection control practices to prevent repeated introductions of SARS epidemics is likely to reduce other healthcare-associated infections. Like other emerging infectious diseases, SARS has demonstrated the importance of enhanced communication between disparate geographic regions and diverse sectors of society.","It is noteworthy that during June and July 2003, the HKU Department of Geography offered full-day \"Mapping SARS Workshops\" to introduce GIS for SARS mapping, which is also applicable to disease mapping in general . Specific workshop highlights included methods to examine: (1) spatio-temporal disease progress; (2) disease diffusion within a vertical building (this could prove helpful when reviewing current buildings/building regulations to achieve healthier living conditions and minimise disease spread in case of infectious disease outbreaks); (3) possible disease correlation with demographic or environmental data; and (4) identification of clusters by infection sources or geographic proximity.","Also worth mentioning in this context is the recent news that remote sensing satellite data (provided by the European Space Agency \u2013 ESA) and GIS (Geographic Information Systems) are currently being used to better understand, predict and help combat Ebola haemorrhagic fever outbreaks in central Africa .","city \u2013 Initial generic geographic metadata is reported here. Subsequently standardized via lookup with geographic reference table.","The Immune Viewer is a dynamic website. It is developed using HTML (HyperText Markup Language), JavaScript and CSS (Cascading Style Sheets). Specifically, we used the D3.js library to allow users to interactively explore the mutation rates or immunogenic scores across the viral genomic regions. We also used the D3.phylogram.js to visualize the phylogenetic tree and the Select2 library to facilitate users' query for different SARS-CoV-2 strains across multiple geographic regions. The data packet downloaded for each viral strain can be directly visualized in the desktop JBrowse software, which can be downloaded from https://jbrowse.org/blog.","Understanding of COVID-19 is evolving rapidly. It is essential to understand the emerging scientific knowledge to coordinate COVID-19 research globally. Recently, the WHO has published a Global Research Roadmap with immediate, mid-term and longer-term priorities to enable the implementation of priority research [8] . Bonilla-Aldana DK et al. [9] and Md Mahbub Hossain MBBS [10] have performed bibliometric analysis to evaluate the scientific literature on coronavirus infection and COVID-19, basing on indicators such as the number of articles, the productivity of authors, geographic distribution of articles and prominent keywords. However, to the best of our knowledge, there is still no quantitative thematic analysis available to date that focuses on CoVs.","The findings in this report are subject to at least three limitations. First, although analysis results were similar regardless of whether the 44% of cases with estimated onset dates were considered, the validity of estimating onset dates based on specimen or reported date cannot be assessed. Second, the proportion of prescriptions administered for prophylaxis versus treatment is not known, neither is the effect this might have had on the temporal association between onset dates and prescription dispensing dates. Finally, this study focused on one event, the 2009 influenza A(H1N1) pandemic. Additional investigation involving more years of data and more geographic locations are required before any findings can be generalized.","A majority 146/179 (82 %) of respondents said that they had access to registry data for patients receiving care at their sites, and more than 50 % of respondents reported that their institutions had a research administration infrastructure in place to aid with data-sharing agreements (52 %), research contract (49 %), ethics reviews (74 %), and administrative support for research projects (60 %). The average time for the ethical review process (from submission to approval to enroll patients) was approximately 1 to 3 months.","In conclusion, we have detected the presence of bat paramyxoviruses, bat coronaviruses and rodent\u2010associated coronaviruses within a limited geographic area and time frame in Viet Nam. Consequently, we suggest that larger\u2010scale surveillance is needed to fully understand the role played by rodents and bats in the evolution, emergence and dissemination of paramyxoviruses and coronaviruses in Viet Nam."," D. rotundus feeding preference was characterized by detecting cytochrome B sequences from digested blood meals via fresh fecal samples. Positive PCR products were cloned in order to obtain separate D. rotundus and prey sequences, as well as to detect the presence of sequences from different prey species. Half of the fecal specimens we tested (52/103) yielded CytB sequences from prey, and demonstrated preferential feeding on Bos taurus, with occasional feeding on Bos indicus and Equus spp. These results are consistent with previous evidence based on stable isotope analyses, which also demonstrated preferential feeding by Desmodus rotundus on livestock over native mammals (Voigt and Kelm 2006). However, other recent studies have shown variations in D. rotundus feeding behavior based on different prey availability, which included preferential feeding on chickens and pigs in the absence of high density cattle ranching (Bobrowiec et al. 2015). We found no evidence of human blood meals from the bats that we sampled. While we recognize that there may be geographic and temporal differences in vampire feeding behavior based on prey availability, our results suggest that frequent contact between D. rotundus and domestic species, particularly cattle and horses, may increase the probability of viral and bacterial transmission.","The geographic distribution of this infection is generally limited to areas near the Gulf of Mexico.","Tiling arrays have traditionally been constructed based on the genome of a single reference strain and used to locate genomic differences contained in the experimental strains. However, single-genome arrays can only detect and analyze sequences similar to those included on the array, and cannot discover or analyze sequences absent in the reference strain. After the introduction of the pan-genome concept [14,15], it has become increasingly clear that some microbial species contain significant genetic diversity, and it is not suitable to compare against only a single reference strain. The pan-genome hypothesis states that any given species has two sets of genes. First, a set of core genes present in all strains that define the species; and second, a set of dispensable genes present in only one or a few of the strains that presumably mediate adaptation. A single genome describes the genomic material for a particular strain, but the pan-genome describes the genomic makeup for an entire species. Single reference tiling arrays cannot survey this full diversity. Ideally, an array for analyzing new strains should cover the genomic diversity of the entire pan-genome.","Metagenomic analysis can also be leveraged in response to viral outbreaks, for example to rapidly determine viral subtypes in a novel infection source. This has been applied to cases of influenza infections as well as for a novel wild type Ebola virus outbreak, for which metagenomic approaches could correctly identify the causative agent, even in cases where traditional methods were unsuccessful because the wild type virus was too distantly related to known Ebola viruses. A correct and rapid identification of these viruses could enable the application of the correct therapeutics and guide preventive efforts against potential epidemics.","The research studied the COVID-19 pandemic outbreak at the aspects of the global epidemic trend, geographic transmission patterns, and the changes in disease distribution from 13 th Jan.","We analyzed the spatiotemporal clustering of the MERS-CoV incidence in Saudi Arabia between 2012 and 2019 at the city level by using Kulldorff\u2019s spatial scan statistics via SaTScan 9.6 [39]. We used purely temporal, seasonal, purely spatial and spatiotemporal retrospective analyses to scan, detect and evaluate the periods and geographical areas with the highest MERS-CoV risk incidence clusters. The purely spatial scan statistic is defined by a circular window on the map. The window is sequentially centered on each of several possible cities that are positioned throughout the study area. The spatiotemporal scan statistic imposes a cylindrical window with a circular geographic base and height corresponding to time. The temporal scan statistic uses a window that moves in one dimension, time, defined in the same way as the height of the cylinder used by the spatiotemporal scan statistic. The key feature that distinguishes the seasonal scan statistic from the purely temporal scan statistic is that the former ignores the year of the observation and retains only the day and month [40]. The number of MERS-CoV cases by city was used as the case file, the city population estimated from the 2010 census [38] was used as the population file and the latitude and longitude of each city were used in the coordinates file.","The increasing incidence of imported Methicillin resistant Staphylococcus aureus (MRSA) cases in acute-care facilities in Belgium led us to investigate the prevalence of MRSA carriers among nursing home (NH) residents. Objectives: To study the national prevalence of MRSA colonisation in Belgian NH-residents, to identify risk factors for MRSA carriage. Methods: In 2005, a prevalence survey was performed in a random stratified sample of 60 NH, representative of the geographic regions of the country and patient care profile. For each NH, maximum 50 residents were randomly selected for screening (nose, throat, skin wound/ urinary meatus) during a single day survey. Risk factors for MRSA carriage were collected by questionnaire, at resident -and at NH level. Results: The mean size of the NH was 106 beds, 46% of which were high-care beds. Most of these facilities (68%) were private, 36 were located in the Flanders region, 18 in the Walloon region and 6 in the Brussels region. From all screened residents, 50.7% were S. aureus carriers and 19.0% (95%CI, were MRSA carriers. There were no statistically significant differences in MRSA prevalence by region, by proportion of high-care beds in the NH's or by NH size. MRSA carriership was more frequent in private (20.9%) than in public NH's (17.6%) (p = 0.004). In NH with an infection control protocol the MRSA prevalence is lower (17%) than in NH without such protocol (22.2%) (p = 0.046). Among S. aureus carriers the mean resistance proportion was 37.8% (95% CI, 33. 4-42.1) . A linear relation was observed between resistance proportion and prevalence of MRSA carriership (Pearson: 0.88, p \u00a3 0.001).","WikiData -WikiData is a dataset holding a vast knowledge about the world, containing data on over 78,252,808 items [40] . WikiData stores metadata about items, and each item has an identifier and can be associated with other items. We utilized the WikiData dataset to extract geographic information for academic institutions in order to match a paper with its authors' geographic locations.","SuperMap GIS Technologies, Inc., a software company affiliated to the Geographic Information Industrial Development Centre of the Chinese Academy of Sciences, has produced an online interactive SARS distribution map of China with a Chinese interface .","We extract human mobility data from the Baidu Qianxi web platform, which presents daily population travels between cities or provinces tracked through the Baidu Huiyan system. The data do not represent numbers of individual travelers but rather an index of relative movements constructed by Baidu's proprietary methods which are correlated with human mobility (32) (http://qianxi.baidu.com/). In particular, two pieces of information are collected. First, we extract a series of migration scale indices for traveling out of Wuhan, from January 1st to February 10, both in 2019 and 2020. Second, we obtain the proportion of human movement from Wuhan were bound for each of 31 provinces in China. These proportions are available for January 1st -February 10, 2020. Based on this data we had access to both changes in mobility volume and changes in mobility direction. See more detailed descriptions of the human movement data here: (33, 34) Review of interventions and reporting shifts: We reviewed the literature and online social media to understand the key timings of interventions and announcements that are relevant for disease transmission across China. We collated information about the type (e.g., announcement of outbreak, travel restrictions, isolation of patients, etc.), geographic location (e.g., city where available, province), and timing (specific date or date range).","The copyright holder for this preprint (which was not peer-reviewed) is . https://doi.org/10.1101/2020.03.25.20042713 doi: medRxiv preprint https://www.ecdc.europa.eu/en/publications-data/download-todays-data-geographic-distribution-covid-","The three astrovirus isolates from feral pigeon which were characterized were genetically diverse, although the host pigeons were captured in the same area on two subsequent days. It has previously been reported that the amino acid sequence identities of the RdRp gene between four groups of avastrovirus, i.e. TAstV-1, TAstV-2, ANV and chicken-origin astroviruses, detected in different regions also were highly diverse, ranging from 50,3% to 73,8% identity. There were also multiple phylogenetic subgroups within each group. Phylogenetic analysis revealed no clear assortment by geographic region or isolation date [51], [54]. The distances between both the polymerase and the capsid sequences of the feral pigeons were comparable to the distances between the HAstV-1 to HAstV-8 types. The sequence identity of the entire capsid protein precursors between different serotypes of HAstVs was 66\u201376% at the nucleotide level [46], while the sequence identity between the feral pigeons at the nucleotide level was 66\u201368% in the same region, suggesting a variety of astrovirus genogroups within feral pigeons, analogous to HAstV genogroups.","Since then the geographic distribution and number of cases has expanded dramatically, with many thousands of cases being documented in the Americas ranging from the southern United States into South America. No vaccine or specific antiviral therapy is currently available.","This assessment is based on the open-access COVID-19 incident case data maintained by the Open COVID-19 Data Curation Group. [7] A total of 18,069 geo-located COVID-19 incident confirmed cases during January and February 2020 were extracted for the whole of China, excluding Wuhan City. No case confirmations were reported for the first two weeks of January. All cases were mapped by week of confirmation onto a 0.25\u00b0 latitude/longitude grid (approximately 25 x 25 km squares) for the whole of China, which included 15,539 cells. This grid was also filled with population data from the NASA Socioeconomic Data and Applications Center. [8] Gridded weather data for the whole of China during January and February 2020 were sourced from the Copernicus Climate Change Service ERA-5T model, specifically temperature at 2m, total precipitation and total sky direct solar radiation at surface. [9] Weekly averages/totals of daily weather data for each geographic cell were calculated and mapped on the geographic grid. These data were used as the basis for generating maps of cases and ecological factors.","Until March 9, 2020, the Japanese government did not restrict entry of tourists from areas of China outside Hubei Province which might have unintentionally allowed selective influx of S form to Japan. S type influx into Ireland and the Balkans may be due to travel restrictions in those countries restricted to infected individuals. In contrast to other areas in Japan, epidemic is ongoing in Nagoya City in Aichi prefecture and Sapporo City in Hokkaido, where the spread of the S form may have been incomplete as discussed above. In the United States of America and most European countries, travel from across China was restricted from the beginning of February, blocking the inflow of the S form from China. There is deep concern that serious outbreaks of SARS-CoV-2 in the United States and Europe may have been caused by the spread of the L form preceding the S form, by which up to 55% of the population may be infected as calculate above."],"title":["Encephalomyocarditis virus is potentially derived from eastern bent-wing bats living in East Asian countries","The global spread of Middle East respiratory syndrome: an analysis fusing traditional epidemiological tracing and molecular phylodynamics","COVID-19 Epidemic in Switzerland: Growth Prediction and Containment Strategy Using Artificial Intelligence and Big Data","Assessing the interactions between COVID-19 and influenza in the United States","Epidemiological data from the COVID-19 outbreak, real-time case information","The global spread of Middle East respiratory syndrome: an analysis fusing traditional epidemiological tracing and molecular phylodynamics","A viral metagenomic survey identifies known and novel mammalian viruses in bats from Saudi Arabia","Extensive diversity of coronaviruses in bats from China","Latest outbreak news from ProMED-mail Novel coronavirus \u2013 Middle East","Metagenomics in Virology","Balkan Nephropathy: Evolution of Our Knowledge","Review of Bats and SARS","Comparison of the spatiotemporal characteristics of the COVID-19 and SARS outbreaks in mainland China","26 Evolution of SARS Coronavirus and the Relevance of Modern Molecular Epidemiology","What we have learnt from the SARS epdemics in mainland China?","Epidemiological data from the COVID-19 outbreak, real-time case information","COVID-19: Challenges to GIS with Big Data","Phylogeny of Viruses","Extensive diversity of coronaviruses in bats from China","Coronavirus disease-19: The First 7,755 Cases in the Republic of Korea","Identification of COVID-19 Can be Quicker through Artificial Intelligence framework using a Mobile Phone-Based Survey in the Populations when Cities/Towns Are Under Quarantine","Predicting the impacts of epidemic outbreaks on global supply chains: A simulation-based analysis on the coronavirus outbreak (COVID-19/SARS-CoV-2) case","Global transmission network of SARS-CoV-2: from outbreak to pandemic","Descriptive review of geographic mapping of severe acute respiratory syndrome (SARS) on the Internet","A framework for identifying regional outbreak and spread of COVID-19 from one-minute population-wide surveys","Regaining perspective on SARS-CoV-2 molecular tracing and its implications","Severe Acute Respiratory Syndrome, Beijing, 2003","Descriptive review of geographic mapping of severe acute respiratory syndrome (SARS) on the Internet","Descriptive review of geographic mapping of severe acute respiratory syndrome (SARS) on the Internet","Epidemiological data from the COVID-19 outbreak, real-time case information","The immune vulnerability landscape of the 2019 Novel Coronavirus, SARS-CoV-2","Understand Research Hotspots Surrounding COVID-19 and Other Coronavirus Infections Using Topic Modeling","Value of Pharmacy-Based Influenza Surveillance \u2014 Ontario, Canada, 2009","Feasibility of a randomized controlled trial to assess treatment of Middle East Respiratory Syndrome Coronavirus (MERS-CoV) infection in Saudi Arabia: a survey of physicians","Detection of potentially novel paramyxovirus and coronavirus viral RNA in bats and rats in the Mekong Delta region of southern Viet Nam","Viral Diversity, Prey Preference, and Bartonella Prevalence in Desmodus rotundus in Guatemala","35 The Approach to Diarrhea","Efficient oligonucleotide probe selection for pan-genomic tiling arrays","Metagenomics in Virology","A Tempo-geographic Analysis of Global COVID-19 Epidemic Outside of China","Spatiotemporal Clustering of Middle East Respiratory Syndrome Coronavirus (MERS-CoV) Incidence in Saudi Arabia, 2012\u20132019","S8 \u2013 S47 and O50 \u2013 O440","Scientometric Trends for Coronaviruses and Other Emerging Viral Infections","Descriptive review of geographic mapping of severe acute respiratory syndrome (SARS) on the Internet","The effect of human mobility and control measures on the COVID-19 epidemic in China","World governments should protect their population from COVID-19 pandemic using Italy and Lombardy as precursor","Screening of Feral and Wood Pigeons for Viruses Harbouring a Conserved Mobile Viral Element: Characterization of Novel Astroviruses and Picornaviruses","Viral Zoonoses","Eco-epidemiological assessment of the COVID-19 epidemic in China, January-February 2020","Epidemiological Tools that Predict Partial Herd Immunity to SARS Coronavirus 2"],"url":["https://doi.org/10.1016/j.virusres.2018.10.020","https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5693564/","https://doi.org/10.1101/2020.03.30.20047472","https://doi.org/10.1101/2020.03.30.20047993","https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7093412/","https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5693564/","https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6457491/","https://doi.org/10.1016/j.virol.2017.03.019","https://doi.org/10.1016/j.ijid.2012.12.001","https://doi.org/10.1016/b978-0-12-809633-8.20957-6","https://doi.org/10.1053/j.ajkd.2008.05.024","https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3291347/","https://doi.org/10.1101/2020.03.23.20034058","https://doi.org/10.1016/b978-0-12-799942-5.00026-3","https://doi.org/10.1016/j.glohj.2019.09.003","https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7093412/","https://doi.org/10.1016/j.geosus.2020.03.005","https://doi.org/10.1016/b978-012374410-4.00712-3","https://doi.org/10.1016/j.virol.2017.03.019","https://doi.org/10.1101/2020.03.15.20036368","https://doi.org/10.1017/ice.2020.61","https://doi.org/10.1016/j.tre.2020.101922","https://doi.org/10.1101/2020.03.22.20041145","https://www.ncbi.nlm.nih.gov/pmc/articles/PMC343293/","https://doi.org/10.1101/2020.03.19.20038844","https://doi.org/10.1101/2020.03.16.20034470","https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3092360/","https://www.ncbi.nlm.nih.gov/pmc/articles/PMC343293/","https://www.ncbi.nlm.nih.gov/pmc/articles/PMC343293/","https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7093412/","https://doi.org/10.1101/2020.02.08.939553","https://doi.org/10.1101/2020.03.26.20044164","https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4604937/","https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4942900/","https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5811810/","https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5164864/","https://doi.org/10.1016/b978-1-56053-629-1.50038-8","https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2753849/","https://doi.org/10.1016/b978-0-12-809633-8.20957-6","https://doi.org/10.1101/2020.03.20.20039602","https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6678379/","https://doi.org/10.1111/j.1470-9465.2006.12_4_1426.x","https://doi.org/10.1101/2020.03.17.995795","https://www.ncbi.nlm.nih.gov/pmc/articles/PMC343293/","https://doi.org/10.1101/2020.03.02.20026708","https://doi.org/10.1101/2020.03.25.20042713","https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3197151/","https://doi.org/10.1016/b978-0-12-801238-3.95729-5","https://doi.org/10.1101/2020.03.29.20046565","https://doi.org/10.1101/2020.03.25.20043679"],"x":[0.2022651190613739,0.441217344097657,0.6307364891309861,0.25176865435735674,0.7467482860506223,0.2951144968534787,0.7499336939959206,0.12977022416936213,0.940185801081553,0.4122863420366495,0.5729556556926659,0.9001543603501351,0.3534812291981938,0.11407935173430372,0.9878957958441723,0.34539483658620307,0.7770745743005536,0.38069785362416064,0.5350793522144468,0.18398838179754107,0.7610272967966372,0.36834013757713147,0.7786236095678041,0.9094522763295457,0.9107153548448874,0.6460646535106567,0.22218106668105442,0.48126896955719867,0.38369882110805076,0.22069896720907523,0.3747862500295046,0.4052278811297988,0.5630470588243267,0.8356070497997357,0.4024231412196704,0.43931333555060414,0.8776464477059892,0.6327220508964423,0.5232221067831986,0.5255774537661878,0.1793518179972794,0.7337939678232601,0.29351096505840646,0.11988243352092276,0.1999277390891716,0.8240587679837562,0.09654625834021291,0.9992572746061484,0.3930869827410479,0.8353474763256702],"y":[0.6953096726882106,0.7014243411455087,0.5275243617346205,0.177178696994572,0.22758998260957652,0.29588443714848633,0.4178564056719092,0.6149183352722678,0.11582323072825196,0.8197600681822399,0.8259934891746189,0.6852878516614251,0.6230271054191029,0.9830491526386681,0.2169966283912983,0.6501648153880695,0.16314307320327548,0.17752706698172482,0.59085568984427,0.5035642602196758,0.2158836983968605,0.7438682566241059,0.37015903060594957,0.7719140624037539,0.018742907968832268,0.9531037038798366,0.6426229588316209,0.764107632593752,0.8878690210423615,0.7436570203002569,0.7332201967247761,0.8049616519536573,0.5649699431926732,0.5986909371552942,0.23180879874251448,0.9683836585975862,0.6056254471039378,0.8761139136509417,0.9534338897088771,0.6719964648159441,0.416590783547952,0.5993274676873278,0.5983134999418949,0.4300860477158819,0.7849647434107149,0.4644099425889451,0.9303215981747706,0.024582709306997574,0.05873733329894559,0.8554619587710961]},"selected":{"id":"8481"},"selection_policy":{"id":"8506"}},"id":"8408","type":"ColumnDataSource"},{"attributes":{"editor":{"id":"8500"},"field":"title","formatter":{"id":"8501"},"title":"Document title","width":770},"id":"8441","type":"TableColumn"},{"attributes":{"columns":[{"id":"8475"},{"id":"8476"}],"height":340,"source":{"id":"8408"},"view":{"id":"8478"},"width":900},"id":"8477","type":"DataTable"},{"attributes":{"data_source":{"id":"8407"},"glyph":{"id":"8434"},"hover_glyph":null,"muted_glyph":null,"nonselection_glyph":{"id":"8435"},"selection_glyph":null,"view":{"id":"8437"}},"id":"8436","type":"GlyphRenderer"},{"attributes":{"children":[{"id":"8485"}]},"id":"8489","type":"Row"},{"attributes":{"source":{"id":"8407"}},"id":"8437","type":"CDSView"},{"attributes":{"children":[{"id":"8442"},{"id":"8409"}]},"id":"8488","type":"Row"},{"attributes":{"children":[{"id":"8484"}]},"id":"8487","type":"Row"},{"attributes":{},"id":"8454","type":"BasicTicker"},{"attributes":{"formatter":{"id":"8515"},"ticker":{"id":"8454"}},"id":"8453","type":"LinearAxis"},{"attributes":{"editor":{"id":"8510"},"field":"text","formatter":{"id":"8511"},"title":"Passage","width":770},"id":"8476","type":"TableColumn"},{"attributes":{},"id":"8427","type":"ResetTool"},{"attributes":{"sizing_mode":"stretch_width","text":"&lt;h2&gt;Access to geographic and temporal diverse sample sets to understand geographic distribution and genomic differences, and determine whether there is more than one strain in circulation. Multi-lateral agreements such as the Nagoya Protocol could be leveraged.&lt;/h2&gt;"},"id":"8483","type":"Div"},{"attributes":{"axis":{"id":"8453"},"ticker":null},"id":"8456","type":"Grid"},{"attributes":{"args":{"sd":{"id":"8407"},"sp":{"id":"8408"}},"code":"\n    if (sd.selected.indices.length == 0) {\n      return;\n    }\n    var sel = sd.selected.indices[0];\n    var new_selected = [];\n    for (var i = 0; i &lt; sp.data['doc'].length; i++) {\n      if (sp.data['doc'][i] == sd.data['doc'][sel]) {\n        new_selected.push(i);\n      }\n    }\n    if (JSON.stringify(new_selected) != JSON.stringify(sp.selected.indices) &amp;&amp;\n    !(sp.selected.indices.length == 1 &amp;&amp; new_selected.includes(sp.selected.indices[0]))) {\n      sp.selected.indices = new_selected;\n      sp.change.emit();\n    }\n    "},"id":"8480","type":"CustomJS"},{"attributes":{"formatter":{"id":"8513"},"ticker":{"id":"8458"}},"id":"8457","type":"LinearAxis"},{"attributes":{},"id":"8447","type":"Range1d"},{"attributes":{},"id":"8451","type":"LinearScale"},{"attributes":{},"id":"8506","type":"UnionRenderers"},{"attributes":{"editor":{"id":"8508"},"field":"scr","formatter":{"id":"8509"},"title":"Score","width":30},"id":"8475","type":"TableColumn"},{"attributes":{},"id":"8445","type":"Range1d"},{"attributes":{},"id":"8498","type":"StringEditor"},{"attributes":{},"id":"8449","type":"LinearScale"},{"attributes":{"active_drag":"auto","active_inspect":"auto","active_multi":null,"active_scroll":"auto","active_tap":"auto","tools":[{"id":"8426"},{"id":"8427"},{"id":"8428"}]},"id":"8429","type":"Toolbar"},{"attributes":{},"id":"8499","type":"StringFormatter"},{"attributes":{"axis":{"id":"8422"},"dimension":1,"ticker":null},"id":"8425","type":"Grid"},{"attributes":{},"id":"8510","type":"StringEditor"},{"attributes":{"formatter":{"id":"8503"},"ticker":{"id":"8423"}},"id":"8422","type":"LinearAxis"},{"attributes":{"editor":{"id":"8498"},"field":"scr","formatter":{"id":"8499"},"title":"Score","width":30},"id":"8440","type":"TableColumn"},{"attributes":{},"id":"8508","type":"StringEditor"},{"attributes":{"formatter":{"id":"8505"},"ticker":{"id":"8419"}},"id":"8418","type":"LinearAxis"},{"attributes":{},"id":"8414","type":"LinearScale"},{"attributes":{},"id":"8509","type":"StringFormatter"},{"attributes":{"text":"&lt;h3&gt;Document ranking&lt;/h3&gt;"},"id":"8484","type":"Div"},{"attributes":{"fill_color":{"field":"color"},"line_color":{"field":"color"},"radius":{"field":"rad","units":"data"},"x":{"field":"x"},"y":{"field":"y"}},"id":"8434","type":"Circle"},{"attributes":{"below":[{"id":"8418"}],"center":[{"id":"8421"},{"id":"8425"}],"left":[{"id":"8422"}],"plot_height":340,"plot_width":900,"renderers":[{"id":"8436"}],"title":{"id":"8493"},"toolbar":{"id":"8429"},"toolbar_location":"below","x_range":{"id":"8410"},"x_scale":{"id":"8414"},"y_range":{"id":"8412"},"y_scale":{"id":"8416"}},"id":"8409","subtype":"Figure","type":"Plot"},{"attributes":{},"id":"8458","type":"BasicTicker"},{"attributes":{"data_source":{"id":"8408"},"glyph":{"id":"8469"},"hover_glyph":null,"muted_glyph":null,"nonselection_glyph":{"id":"8470"},"selection_glyph":null,"view":{"id":"8472"}},"id":"8471","type":"GlyphRenderer"},{"attributes":{},"id":"8423","type":"BasicTicker"},{"attributes":{},"id":"8511","type":"StringFormatter"},{"attributes":{"source":{"id":"8407"}},"id":"8443","type":"CDSView"},{"attributes":{},"id":"8416","type":"LinearScale"},{"attributes":{"text":"&lt;h3&gt;Paragraph ranking&lt;/h3&gt;"},"id":"8485","type":"Div"},{"attributes":{"columns":[{"id":"8440"},{"id":"8441"}],"height":340,"source":{"id":"8407"},"view":{"id":"8443"},"width":900},"id":"8442","type":"DataTable"},{"attributes":{"children":[{"id":"8483"}]},"id":"8486","type":"Row"},{"attributes":{"fill_alpha":{"value":0.1},"fill_color":{"field":"color"},"line_alpha":{"value":0.1},"line_color":{"field":"color"},"radius":{"field":"rad","units":"data"},"x":{"field":"x"},"y":{"field":"y"}},"id":"8470","type":"Circle"},{"attributes":{"below":[{"id":"8453"}],"center":[{"id":"8456"},{"id":"8460"}],"left":[{"id":"8457"}],"plot_height":340,"plot_width":900,"renderers":[{"id":"8471"}],"title":{"id":"8495"},"toolbar":{"id":"8464"},"toolbar_location":"below","x_range":{"id":"8445"},"x_scale":{"id":"8449"},"y_range":{"id":"8447"},"y_scale":{"id":"8451"}},"id":"8444","subtype":"Figure","type":"Plot"},{"attributes":{"children":[{"id":"8486"},{"id":"8487"},{"id":"8488"},{"id":"8489"},{"id":"8490"}]},"id":"8491","type":"Column"},{"attributes":{},"id":"8503","type":"BasicTickFormatter"},{"attributes":{},"id":"8501","type":"StringFormatter"},{"attributes":{"callback":null,"tooltips":[["Title","@title"],["Abstract","@text"],["Published","@date"],["Authors","@authors"],["Journal","@journal"],["URL","@url"]]},"id":"8428","type":"HoverTool"},{"attributes":{"axis":{"id":"8457"},"dimension":1,"ticker":null},"id":"8460","type":"Grid"},{"attributes":{},"id":"8505","type":"BasicTickFormatter"},{"attributes":{"fill_color":{"field":"color"},"line_color":{"field":"color"},"radius":{"field":"rad","units":"data"},"x":{"field":"x"},"y":{"field":"y"}},"id":"8469","type":"Circle"},{"attributes":{},"id":"8412","type":"Range1d"},{"attributes":{"source":{"id":"8408"}},"id":"8478","type":"CDSView"},{"attributes":{"text":""},"id":"8493","type":"Title"},{"attributes":{},"id":"8462","type":"ResetTool"},{"attributes":{"js_property_callbacks":{"change:indices":[{"id":"8482"}]}},"id":"8481","type":"Selection"},{"attributes":{},"id":"8515","type":"BasicTickFormatter"},{"attributes":{"callback":null},"id":"8461","type":"TapTool"},{"attributes":{"args":{"sd":{"id":"8407"},"sp":{"id":"8408"}},"code":"\n    if (sp.selected.indices.length == 0) {\n      return;\n    }\n    var sel = sp.selected.indices[0];\n    var new_selected = [];\n    for (var i = 0; i &lt; sd.data['doc'].length; i++) {\n      if (sd.data['doc'][i] == sp.data['doc'][sel]) {\n        new_selected.push(i);\n      }\n    }\n    if (JSON.stringify(new_selected) != JSON.stringify(sd.selected.indices) &amp;&amp;\n    !(sd.selected.indices.length == 1 &amp;&amp; new_selected.includes(sd.selected.indices[0]))) {\n      sd.selected.indices = new_selected;\n      sd.change.emit();\n    }\n    "},"id":"8482","type":"CustomJS"},{"attributes":{"source":{"id":"8408"}},"id":"8472","type":"CDSView"},{"attributes":{"active_drag":"auto","active_inspect":"auto","active_multi":null,"active_scroll":"auto","active_tap":"auto","tools":[{"id":"8461"},{"id":"8462"},{"id":"8463"}]},"id":"8464","type":"Toolbar"},{"attributes":{},"id":"8410","type":"Range1d"},{"attributes":{"text":""},"id":"8495","type":"Title"},{"attributes":{"data":{"authors":["Huipeng Liao; Gifty Marley; Yafei Si; Zaisheng Wang; Yewei Xie; Cheng Wang; Weiming Tang","Boulos, Maged N Kamel","Pavel Skums; Alexander Kirpich; Pelin Icer Baykal; Alex Zelikovsky; Gerardo Chowell","Markus Luczak-Roesch","Cordes, Anne K.; Heim, Albert","Hayden C. Metsky; Catherine A. Freije; Tinna-Solveig F. Kosoko-Thoroddsen; Pardis C. Sabeti; Cameron Myhrvold","Michael Letko; Vincent Munster","Hotez, Peter J.; Bottazzi, Maria Elena; Tseng, Chien-Te K.; Zhan, Bin; Lustigman, Sara; Du, Lanying; Jiang, Shibo","Lekana-Douki, Sonia Etenna; Nkoghe, Dieudonn\u00e9; Drosten, Christian; Ngoungou, Edgar Brice; Drexler, Jan Felix; Leroy, Eric M","Zhou, Chenghu; Su, Fenzhen; Pei, Tao; Zhang, An; Du, Yunyan; Luo, Bin; Cao, Zhidong; Wang, Juanle; Yuan, Wen; Zhu, Yunqiang; Song, Ci; Chen, Jie; Xu, Jun; Li, Fujia; Ma, Ting; Jiang, Lili; Yan, Fengqin; Yi, Jiawei; Hu, Yunfeng; Liao, Yilan; Xiao, Han","Miguel Fudolig; Reka Howard","Muradrasoli, Shaman; B\u00e1lint, \u00c1d\u00e1m; Wahlgren, John; Waldenstr\u00f6m, Jonas; Bel\u00e1k, S\u00e1ndor; Blomberg, Jonas; Olsen, Bj\u00f6rn","James P Broughton; Xianding Deng; Guixia Yu; Clare L Fasching; Jasmeet Singh; Jessica Streithorst; Andrea Granados; Alicia Sotomayor-Gonzalez; Kelsey Zorn; Allan Gopez; Elaine Hsu; Wei Gu; Steven Miller; Chao-Yang Pan; Hugo Guevara; Debra Wadford; Janice Chen; Charles Y Chiu","Reed, Kurt D.","Tetro, Jason A.","Xi He; Eric HY Lau; Peng Wu; Xilong Deng; Jian Wang; Xinxin Hao; Yiu Chung Lau; Jessica Y Wong; Yujuan Guan; Xinghua Tan; Xiaoneng Mo; Yanqing Chen; Baolin Liao; Weilie Chen; Fengyu Hu; Qing Zhang; Mingqiu Zhong; Yanrong Wu; Lingzhai Zhao; Fuchun Zhang; Benjamin J Cowling; Fang Li; Gabriel M Leung","Kumari G. Lokugamage; Adam Hage; Craig Schindewolf; Ricardo Rajsbaum; Vineet D. Menachery","Maslow, Joel N.","Li, Mingzhi; Lei, Pinggui; Zeng, Bingliang; Li, Zongliang; Yu, Peng; Fan, Bing; Wang, Chuanhong; Li, Zicong; Zhou, Jian; Hu, Shaobo; Liu, Hao","Linton, M. Natalie; Kobayashi, Tetsuro; Yang, Yichi; Hayashi, Katsuma; Akhmetzhanov, R. Andrei; Jung, Sung-mok; Yuan, Baoyin; Kinoshita, Ryo; Nishiura, Hiroshi","Natalie M. Linton; Tetsuro Kobayashi; Yichi Yang; Katsuma Hayashi; Andrei R. Akhmetzhanov; Sung-mok Jung; Baoyin Yuan; Ryo Kinoshita; Hiroshi Nishiura","Chantal B.F. Vogels; Anderson F. Brito; Anne Louise Wyllie; Joseph R Fauver; Isabel M. Ott; Chaney C. Kalinich; Mary E. Petrone; Marie-Louise Landry; Ellen F. Foxman; Nathan D. Grubaugh","Parashar, Arun; Udayabanu, Malairaman","MacIntyre, C Raina","Louten, Jennifer","Allam, Zaheer; Jones, David S.","Nesteruk, Igor","Miyssa I. Abdelmageed; Abdelrahman H. Abdelmoneim; Mujahed I. Mustafa; Nafisa M. Elfadol; Naseem S. Murshed; Shaza W. Shantier; Abdelrafie M. Makhawi","Mani, Reeta S.; Ravi, V.; Desai, Anita; Madhusudana, S.N.","Eunha Shim; Kenji Mizumoto; Wongyeong Choi; Gerardo Chowell","Chui, Linda; Drebot, Michael; Andonov, Anton; Petrich, Astrid; Glushek, Martin; Mahony, James","Yun Qiu; Xi Chen; Wei Shi","Lin, Xian-Dan; Wang, Wen; Hao, Zong-Yu; Wang, Zhao-Xiao; Guo, Wen-Ping; Guan, Xiao-Qing; Wang, Miao-Ruo; Wang, Hong-Wei; Zhou, Run-Hong; Li, Ming-Hui; Tang, Guang-Peng; Wu, Jun; Holmes, Edward C.; Zhang, Yong-Zhen","Tam, T","Yamey, Gavin; Sch\u00e4ferhoff, Marco; Hatchett, Richard; Pate, Muhammad; Zhao, Feng; McDade, Kaci Kennedy","Chu, Daniel K. W.; Peiris, J. S. Malik; Poon, Leo L. M.","Amariles, Pedro; Granados, Johan; Ceballos, Mauricio; Montoya, Carlos Julio","Vabret, Astrid; Mourez, Thomas; Dina, Julia; van der Hoek, Lia; Gouarin, St\u00e9phanie; Petitjean, Jo\u00eblle; Brouard, Jacques; Freymuth, Fran\u00e7ois","Artika, I Made; Wiyatno, Ageng; Ma'roef, Chairin Nisa","Foddai, Alessandro; Lubroth, Juan; Ellis-Iversen, Johanne","Du, Mulong; Cai, Guoshuai; Chen, Feng; Christiani, David C.; Zhang, Zhengdong; Wang, Meilin","","Drexler, Jan Felix; Corman, Victor Max; Drosten, Christian","Ward, Kenneth","Porzio, Giampiero; Cortellini, Alessio; Bruera, Eduardo; Verna, Lucilla; Ravoni, Giulio; Peris, Flaminia; Spinelli, Giuseppe","Pierre-Yves BOELLE","Idan Yelin; Noga Aharony; Einat Shaer-Tamar; Amir Argoetti; Esther Messer; Dina Berenbaum; Einat Shafran; Areen Kuzli; Nagam Gandali; Tamar Hashimshony; Yael Mandel-Gutfreund; Michael Halberthal; Yuval Geffen; Moran Szwarcwort-Cohen; Roy Kishony","Bijlenga, G.","Wayne Gao; Mattia Sanna; Chi Pang Wen","Anton Nekrutenko; Sergei L Kosakovsky Pond"],"color":["#67a9cf","#67a9cf","#67a9cf","#67a9cf","#67a9cf","#67a9cf","#67a9cf","#67a9cf","#67a9cf","#67a9cf","#67a9cf","#67a9cf","#67a9cf","#67a9cf","#67a9cf","#67a9cf","#67a9cf","#67a9cf","#67a9cf","#67a9cf","#67a9cf","#67a9cf","#67a9cf","#67a9cf","#67a9cf","#67a9cf","#67a9cf","#67a9cf","#67a9cf","#67a9cf","#67a9cf","#67a9cf","#67a9cf","#67a9cf","#67a9cf","#67a9cf","#67a9cf","#67a9cf","#67a9cf","#67a9cf","#67a9cf","#67a9cf","#67a9cf","#67a9cf","#67a9cf","#67a9cf","#67a9cf","#67a9cf","#67a9cf","#67a9cf"],"date":["2020-03-23","2004-01-28","2020-03-27","2020-02-11","2020-04-30","2020-03-02","2020-01-22","2014-07-31","2014-07-07","2020-03-20","2020-03-23","2010-10-29","2020-03-10","2018-12-31","2020-03-31","2020-03-18","2020-03-09","2017-08-28","2020-03-20","2020","2020-01-28","2020-04-01","2017-05-31","2020","2016-12-31","2020","2020","2020-02-11","2012-01-18","2020-04-01","2005-09-30","2020-03-17","2017-07-31","2018-05-03","2020-03-31","2009-04-14","2020-03-31","2005-08-03","2020-07-31","2020-03-28","2020-03-28","2020-03-13","2014-01-31","2006-08-31","2020-03-31","2020-03-17","2020-03-27","2005-10-31","2020-03-23","2020-02-22"],"doc":["fb0nwixw","kuybfc1y","ry9wpcxo","kevrp8rg","m81hfdpe","7tkvl894","tyhwtt7j","lm2ixz8n","6eptw6io","5gbkrs73","cycc3fvl","9le7uk7m","wuvry51z","fgrwwzn7","ctd9sutv","eflwztji","2w0zr9c0","huan94tf","tl4pumj7","4vrqzbof","ykofrn9i","6mdimxnk","8d7tv6yl","zehnqsp7","3p542sqs","94tdt2rv","3iz5x8et","6ojmmmuj","hlugf0xc","p8aa2y6c","6h094fqh","8ozauxlk","xutbbxpr","lnz2laha","b4zibmi1","g5fqxdqh","lgqseo8t","q6501gmd","wq5zqzvv","1n8brl09","gmbqmlb3","2qgtvuex","sxbmd0df","jia6dqld","mf3utolt","ba3we97o","su7f5ges","4hcs0z65","hypyxzk2","v0yzur5h"],"journal":["","Int J Health Geogr","","","Journal of Clinical Virology","","","Microbes and Infection","BMC Infect Dis","Geography and Sustainability","","PLoS One","","Reference Module in Biomedical Sciences","Microbes and Infection","","","Hum Vaccin Immunother","Academic Radiology","Journal of Clinical Medicine","","","Parkinsonism &amp; Related Disorders","Global Biosecurity","Essential Human Virology","Healthcare","Innovative Biosystems and Bioengineering","","Proc Natl Acad Sci India Sect B Biol Sci","","Diagnostic Microbiology and Infectious Disease","","Virology","Canada Communicable Disease Report","The Lancet","Virol Sin","Research in Social and Administrative Pharmacy","Emerg Infect Dis","Infection, Genetics and Evolution","One Health","Gastroenterology","Nat Plants","Antiviral Research","American Journal of Obstetrics and Gynecology","Journal of Pain and Symptom Management","","","Journal of Infection","",""],"rad":[0.016600025907195563,0.013596178207288957,0.009970572447145413,0.00931514387023621,0.0082075498214827,0.008084701132958734,0.007585872886359876,0.007382255536499092,0.007375874140107694,0.0071098120782663995,0.006461233441457565,0.0064573815344089935,0.006240637551420994,0.006152782748163865,0.0059694724415551206,0.005923445102232449,0.005794477467732247,0.005735247083948147,0.005723061252725344,0.0057228440458747375,0.0057228440458747375,0.0055827911765256024,0.005460259616834329,0.005393232381737698,0.005348359679079123,0.005256276452709725,0.005073310243827614,0.004997892981700267,0.00486457888447505,0.0048322239105066285,0.004828756512463192,0.004713229267272047,0.0046880717498907236,0.00441321401388407,0.0043350258616712736,0.0042427733415914,0.0041274956741200845,0.004006173802647349,0.003889250517929736,0.003888291141459966,0.003794065463554686,0.003675543114101412,0.0036318590846018744,0.0034746204535463445,0.003388669482359286,0.0031999095320987517,0.0031814407902650706,0.0031794245922424453,0.003174409207136767,0.00300000957245418],"scr":[0.8,0.623,0.41,0.371,0.306,0.299,0.27,0.258,0.257,0.242,0.204,0.203,0.191,0.185,0.175,0.172,0.164,0.161,0.16,0.16,0.16,0.152,0.145,0.141,0.138,0.133,0.122,0.118,0.11,0.108,0.108,0.101,0.099,0.083,0.079,0.073,0.066,0.059,0.052,0.052,0.047,0.04,0.037,0.028,0.023,0.012,0.011,0.011,0.01,0.0],"text":["Background: Understanding the global epidemic trends, geographic distribution, and transmission patterns of COVID-19 contribute to providing timely information for the global response of the epidemic. This study aims to understand the global pandemic geospatial patterns and trends and identify new epicenters requiring urgent attention. Methods: Data on COVID-19 between 31st Dec. 2019 and 8th Mar. 2020 was included. The epidemic trend was analyzed using joinpoint regressions; the growth of affected countries was by descriptive analysis; and the global distribution and transmission trend by spatial analysis. Findings: The number of new cases in the regions outside of China slowly increased before 24th Feb. and rapidly accelerated after 24th Feb. Compared to China, other affected countries experienced a longer duration of a slow increase at the early stage and rapid growth at the latter stages. The first apparent increase in the number of affected countries occurred from 23rd Jan to 1st F...","From geographic mapping at different scales to location-based alerting services, geoinformatics plays an important role in the study and control of global outbreaks like severe acute respiratory syndrome (SARS). This paper reviews several geographic mapping efforts of SARS on the Internet that employ a variety of techniques like choropleth rendering, graduated circles, graduated pie charts, buffering, overlay analysis and animation. The aim of these mapping services is to educate the public (especially travellers to potentially at-risk areas) and assist public health authorities in analysing the spatial and temporal trends and patterns of SARS and in assessing/revising current control measures.","Background: The COVID-19 pandemic caused by the severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) is straining health systems around the world. Although the Chinese government implemented a number of severe restrictions on people's movement in an attempt to contain its local and international spread, the virus had already reached many areas of the world in part due to its potent transmissibility and the fact that a substantial fraction of infected individuals develop little or no symptoms at all. Following its emergence, the virus started to generate sustained transmission in neighboring countries in Asia, Western Europe, Australia, Canada and the United States, and finally in South America and Africa. As the virus continues its global spread, a clear and evidence-based understanding of properties and dynamics of the global transmission network of SARS-CoV-2 is essential to design and put in place efficient and globally coordinated interventions. Methods: We employ molecular...","Profiling the genetic evolution and dynamic spreading of viruses is a crucial task when responding to epidemic outbreaks. We aim to devise novel ways to model, visualise and analyse the temporal dynamics of epidemic outbreaks in order to help researchers and other people involved in crisis response to make well-informed and targeted decisions about from which geographical locations and time periods more genetic samples may be required to fully understand the outbreak. Our approach relies on the application of Transcendental Information Cascades to a set of temporally ordered nucleotide sequences and we apply it to real-world data that was collected during the currently ongoing outbreak of the novel 2019-nCoV coronavirus. We assess information-theoretic and network-theoretic measures that characterise the resulting complex network and suggest touching points and temporal pathways that are of interest for deeper investigation by geneticists and epidemiologists.","","AbstractThe emergence and outbreak of SARS-CoV-2, the causative agent of COVID-19, has rapidly become a global concern and has highlighted the need for fast, sensitive, and specific tools to surveil circulating viruses. Here we provide assay designs and experimental resources, for use with CRISPR-based nucleic acid detection, that could be valuable for ongoing surveillance. We provide assay designs for detection of 67 viral species and subspecies, including: SARS-CoV-2, phylogenetically-related viruses, and viruses with similar clinical presentation. The designs are outputs of algorithms that we are developing for rapidly designing nucleic acid detection assays that are comprehensive across genomic diversity and predicted to be highly sensitive and specific. Of our design set, we experimentally screened 4 SARS-CoV-2 designs with a CRISPR-Cas13 detection system and then extensively tested the highest-performing SARS-CoV-2 assay. We demonstrate the sensitivity and speed of this assay usi...","AbstractOver the past 20 years, several coronaviruses have crossed the species barrier into humans, causing outbreaks of severe, and often fatal, respiratory illness. Since SARS- CoV was first identified in animal markets, global viromics projects have discovered thousands of coronavirus sequences in diverse animals and geographic regions. Unfortunately, there are few tools available to functionally test these novel viruses for their ability to infect humans, which has severely hampered efforts to predict the next zoonotic viral outbreak. Here we developed an approach to rapidly screen lineage B betacoronaviruses, such as SARS-CoV and the recent 2019-nCoV, for receptor usage and their ability to infect cell types from different species. We show that host protease processing during viral entry is a significant barrier for several lineage B viruses and that bypassing this barrier allows several lineage B viruses to enter human cells through an unknown receptor. We also demonstrate how di...","Abstract The geographic spread and rapid increase in the cases of Middle East respiratory syndrome (MERS) caused by a novel coronavirus (MERS-CoV) during the past two months have raised concern about its pandemic potential. Here we call for the rapid development of an effective and safe MERS vaccine to control the spread of MERS-CoV.","BACKGROUND: Surveillance of influenza-like illness (ILI) in Central Africa began only recently, and few data are therefore available on the circulation of influenza virus and other respiratory viruses. In Gabon, a Central African country, we established a surveillance network in four major towns in order to analyze cases of ILI among patients who visited health centers between March 2010 and June 2011, and to determine the viral etiology. METHODS: Nasal swabs were sent for analysis to the Centre International de Recherches M\u00e9dicales de Franceville, where they were screened for 17 respiratory viruses in a multiplex real-time reverse transcription polymerase chain reaction for all pathogens according the following pairs: adenovirus/parainfluenza virus 4, respiratory syncytial virus/human metapneumovirus, parainfluenza virus 1/parainfluenza virus 2, pandemic influenza virus A/seasonal influenza virus A (H1N1, H3N2)/seasonal influenza virus B, human coronaviruses 229E/OC43, human coronavir...","Abstract The outbreak of the 2019 novel coronavirus disease (COVID-19) has caused more than 100,000 people infected and thousands of deaths. Currently, the number of infections and deaths is still increasing rapidly. COVID-19 seriously threatens human health, production, life, social functioning and international relations. In the fight against COVID-19, Geographic Information Systems (GIS) and big data technologies have played an important role in many aspects, including the rapid aggregation of multi-source big data, rapid visualization of epidemic information, spatial tracking of confirmed cases, prediction of regional transmission, spatial segmentation of the epidemic risk and prevention level, balancing and management of the supply and demand of material resources, and social-emotional guidance and panic elimination, which provided solid spatial information support for decision-making, measures formulation, and effectiveness assessment of COVID-19 prevention and control. GIS has d...","We study a novel multi-strain SIR epidemic model with selective immunity by vaccination. A newer strain is made to emerge in the population when a preexisting strain has reached equilibrium. We assume that this newer strain does not exhibit cross-immunity with the original strain, hence those who are vaccinated and recovered from the original strain become susceptible to the newer strain. Recent events involving the COVID-19 virus demonstrates that it is possible for a viral strain to emerge from a population at a time when the influenza virus, a well-known virus with a vaccine readily available for some of its strains, is active in a population. We solved for four different equilibrium points and investigated the conditions for existence and local stability. The reproduction number was also determined for the epidemiological model and found to be consistent with the local stability condition for the disease-free equilibrium.","Coronaviruses (CoVs) can cause mild to severe disease in humans and animals, their host range and environmental spread seem to have been largely underestimated, and they are currently being investigated for their potential medical relevance. Infectious bronchitis virus (IBV) belongs to gamma-coronaviruses and causes a costly respiratory viral disease in chickens. The role of wild birds in the epidemiology of IBV is poorly understood. In the present study, we examined 1,002 cloacal and faecal samples collected from 26 wild bird species in the Beringia area for the presence of CoVs, and then we performed statistical and phylogenetic analyses. We detected diverse CoVs by RT-PCR in wild birds in the Beringia area. Sequence analysis showed that the detected viruses are gamma-coronaviruses related to IBV. These findings suggest that wild birds are able to carry gamma-coronaviruses asymptomatically. We concluded that CoVs are widespread among wild birds in Beringia, and their geographic sprea...","An outbreak of novel betacoronavirus, SARS-CoV-2 (formerly named 2019-nCoV), began in Wuhan, China in December 2019 and the COVID-19 disease associated with infection has since spread rapidly to multiple countries. Here we report the development of SARS-CoV-2 DETECTR, a rapid (~30 min), low-cost, and accurate CRISPR-Cas12 based lateral flow assay for detection of SARS-CoV-2. We validated this method using contrived reference samples and clinical samples from infected US patients and demonstrated comparable performance to the US CDC SARS-CoV-2 real-time RT-PCR assay.","Abstract Zoonotic viral infections represent an important public health problem across the globe. Unlike infectious agents that are limited to humans, the presence of domestic and wild animal reservoirs and insect and arthropod vectors greatly complicate transmission dynamics and make control measures very difficult to implement. Some viral zoonoses are categorized as \u201cemerging infectious diseases\u201d because they are newly recognized or have shown significant changes in their geographic range and/or epidemiologic characteristics. This article provides updated information on the current status of the most important viral zoonotic diseases. The recent upsurge in Ebola virus infections in West Africa and the emergence of Zika virus in the Western Hemisphere are highlighted.","Abstract One of the most perplexing questions regarding the current COVID-19 coronavirus epidemic is the discrepancy between the severity of cases observed in the Hubei province of China and those occurring elsewhere in the world. One possible answer is antibody dependent enhancement (ADE) of SARS-CoV-2 due to prior exposure to other coronaviruses. ADE modulates the immune response and can elicit sustained inflammation, lymphopenia, and/or cytokine storm, one or all of which have been documented in severe cases and deaths. ADE also requires prior exposure to similar antigenic epitopes, presumably circulating in local viruses, making it a possible explanation for the observed geographic limitation of severe cases and deaths.","We report temporal patterns of viral shedding in 94 laboratory-confirmed COVID-19 patients and modelled COVID-19 infectiousness profile from a separate sample of 77 infector-infectee transmission pairs. We observed the highest viral load in throat swabs at the time of symptom onset, and inferred that infectiousness peaked on or before symptom onset. We estimated that 44% of transmission could occur before first symptoms of the index. Disease control measures should be adjusted to account for probable substantial pre-symptomatic transmission.","AbstractSARS-CoV-2, a novel coronavirus (CoV), has recently emerged causing an ongoing outbreak of viral pneumonia around the world. While genetically distinct from the original SARS-CoV, both group 2B CoVs share similar genome organization and origins to coronaviruses harbored in bats. Importantly, initial guidance has used insights from SARS-CoV infection to inform treatment and public health strategies. In this report, we evaluate type-I Interferon (IFN-I) sensitivity of SARS-CoV-2 relative to the original SARS-CoV. Our results indicate that while SARS-CoV-2 maintains similar viral replication kinetics to SARS-CoV in Vero cell, the novel CoV is much more sensitive to IFN-I pretreatment. Examining transcriptional factor activation and interferon stimulated gene (ISG) induction, SARS-CoV-2 in the context of type I IFN induces phosphorylation of STAT1 and increased ISG proteins. In contrast, the original SARS-CoV has no evidence for STAT1 phosphorylation or ISG protein increases even i...","The past decade and a half has been characterized by numerous emerging infectious diseases. With each new threat, there has been a call for rapid vaccine development. Pathogens such as the Middle East Respiratory Syndrome coronavirus (MERS-CoV) and the Zika virus represent either new viral entities or viruses emergent in new geographic locales and characterized by novel complications. Both serve as paradigms for the global spread that can accompany new pathogens. In this paper, we review the epidemiology and pathogenesis of MERS-CoV and Zika virus with respect to vaccine development. The challenges in vaccine development and the approach to clinical trial design to test vaccine candidates for disease entities with a changing epidemiology are discussed.","Coronavirus disease is an emerging infection caused by a novel coronavirus that is moving rapidly. High resolution computed tomography (CT) allows objective evaluation of the lung lesions, thus enabling us to better understand the pathogenesis of the disease. With serial CT examinations, the occurrence, development, and prognosis of the disease can be better understood. The imaging can be sorted into four phases: early phase, progressive phase, severe phase, and dissipative phase. The CT appearance of each phase and temporal progression of the imaging findings are demonstrated.","The geographic spread of 2019 novel coronavirus (COVID-19) infections from the epicenter of Wuhan, China, has provided an opportunity to study the natural history of the recently emerged virus. Using publicly available event-date data from the ongoing epidemic, the present study investigated the incubation period and other time intervals that govern the epidemiological dynamics of COVID-19 infections. Our results show that the incubation period falls within the range of 2&amp;ndash;14 days with 95% confidence and has a mean of around 5 days when approximated using the best-fit lognormal distribution. The mean time from illness onset to hospital admission (for treatment and/or isolation) was estimated at 3&amp;ndash;4 days without truncation and at 5&amp;ndash;9 days when right truncated. Based on the 95th percentile estimate of the incubation period, we recommend that the length of quarantine should be at least 14 days. The median time delay of 13 days from illness onset to death (17 days with rig...","The geographic spread of 2019 novel coronavirus (COVID-19) infections from the epicenter of Wuhan, China, has provided an opportunity to study the natural history of the recently emerged virus. Using publicly available event-date data from the ongoing epidemic, the present study investigated the incubation period and other time intervals that govern the epidemiological dynamics of COVID-19 infections. Our results show that the incubation period falls within the range of 2-14 days with 95% confidence and has a mean of around 5 days when approximated using the best-fit lognormal distribution. The mean time from illness onset to hospital admission (for treatment and/or isolation) was estimated at 3-4 days without truncation and at 5-9 days when right truncated. Based on the 95th percentile estimate of the incubation period, we recommend that the length of quarantine should be at least 14 days. The median time delay of 13 days from illness onset to death (17 days with right truncation) sho...","The recent spread of severe acute respiratory syndrome coronavirus (SARS-CoV-2) exemplifies the critical need for accurate and rapid diagnostic assays to prompt public health actions. Currently, several quantitative reverse-transcription polymerase chain reaction (qRT-PCR) assays are being used by clinical, research, and public health laboratories for rapid detection of the virus. However, it is currently unclear if results from different tests are comparable. Our goal was to evaluate the primer-probe sets used in four common diagnostic assays available on the World Health Organization (WHO) website. To facilitate this effort, we generated RNA transcripts to create standards and distributed them to other laboratories for internal validation. We then used these RNA transcript standards, full-length SARS-CoV-2 RNA, and RNA-spiked mock samples to determine analytical efficiency and sensitivity of nine primer-probe sets. We show that all primer-probe sets can be used to detect SARS-CoV-2, ...","Abstract Gut microbiota (GM) can influence various neurological outcomes, like cognition, learning, and memory. Commensal GM modulates brain development and behavior and has been implicated in several neurological disorders like Alzheimer's disease, multiple sclerosis, amyotrophic lateral sclerosis, anxiety, stress and much more. A recent study has shown that Parkinson's disease patients suffer from GM dysbiosis, but whether it is a cause or an effect is yet to be understood. In this review, we try to connect the dots between GM and PD pathology using direct and indirect evidence.","Information about the open-access article 'Wuhan novel coronavirus 2019nCoV' in DOAJ. DOAJ is an online directory that indexes and provides access to quality open access, peer-reviewed journals.","Abstract An emerging infectious disease (EID) is defined as a disease caused by a pathogen that has not been observed previously within a population or geographic location. Viruses are a major cause of EIDs, particularly \u2212ssRNA viruses. Many variables are involved in the emergence or reemergence of viruses. These can be classified into human factors, environmental/ecological factors, and viral factors and include urbanization, globalization, weather and climate change, and the genetic composition of the virus. The great majority of emerging viral diseases are zoonoses, notably transmitted by arthropods and nonhuman mammals. Flaviviruses include several notable vector-transmitted viruses, while rodents and bats are thought to be the natural reservoirs of arenaviruses and filoviruses, respectively. This chapter discusses several notable outbreaks of emerging and reemerging viruses, including the 2014\u201315 outbreak of Ebolavirus in West Africa.","As the Coronavirus (COVID-19) expands its impact from China, expanding its catchment into surrounding regions and other countries, increased national and international measures are being taken to contain the outbreak. The placing of entire cities in \u2018lockdown\u2019 directly affects urban economies on a multi-lateral level, including from social and economic standpoints. This is being emphasised as the outbreak gains ground in other countries, leading towards a global health emergency, and as global collaboration is sought in numerous quarters. However, while effective protocols in regard to the sharing of health data is emphasised, urban data, on the other hand, specifically relating to urban health and safe city concepts, is still viewed from a nationalist perspective as solely benefiting a nation\u2019s economy and its economic and political influence. This perspective paper, written one month after detection and during the outbreak, surveys the virus outbreak from an urban standpoint and adva...","Information about the open-access article 'Statistics-Based Predictions of Coronavirus Epidemic Spreading in Mainland China' in DOAJ. DOAJ is an online directory that indexes and provides access to quality open access, peer-reviewed journals.","AbstractBackgroundNew endemic disease has been spread across Wuhan City, China on December 2019. Within few weeks, the World Health Organization (WHO) announced a novel coronavirus designated as coronavirus disease 2019 (COVID-19). In late January 2020, WHO declared the outbreak of a \u201cpublic-health emergency of international concern\u201d due to the rapid and increasing spread of the disease worldwide. Currently, there is no vaccine or approved treatment for this emerging infection; thus the objective of this study is to design a multi epitope peptide vaccine against COVID-19 using immunoinformatics approach.MethodSeveral techniques facilitating the combination of immunoinformatics approach and comparative genomic approach were used in order to determine the potential peptides for designing the T cell epitopes-based peptide vaccine using the envelope protein of 2019-nCoV as a target.ResultsExtensive mutations, insertion and deletion were discovered with comparative sequencing in COVID-19 st...","Despite an elaborate armamentarium to tackle microbes, emerging infectious diseases remain a crucial global challenge. Emerging infections can be defined as \u201cinfections that have newly appeared in a population or have existed previously but are rapidly increasing in incidence or geographic range.\u201d Several factors like increase in international travel and trade, human encroachment on wild-life habitats, changes in agricultural practices and wild-life trade have contributed to the emergence of pathogens. Emergence/re-emergence of several viral infections has been reported from India in the past few decades; some of the important emerging viral infections are discussed in this review. They include infection due to Nipah, Hantaviruses, Chikungunya, Human Enterovirus-71, Influenza, Chandipura, Crimean Congo, SARS Coronavirus, Buffalopox, Dengue and Japanese Encephalitis viruses. Creating increased awareness and training of clinical microbiologists/virologists for identification of new/emerg...","Background: In Korea, a total of 8,799 confirmed cases of COVID-19 including 102 deaths have been recorded as of Mar 21, 2020. The time-delay adjusted case fatality risk of COVID-19 in Korea is yet to be estimated. Methods: We obtained the daily series of confirmed cases and deaths in Korea reported prior to March 21,2020. Using statistical methods, we estimated the time-delay adjusted risk for death from COVID-19 in the city of Daegu, Gyeongsangbuk-do, other regions in Korea, as well as for the entire country. Results: Our model-based crude CFR fitted the observed data well throughout the course of the epidemic except for the very early stage in Gyeongsangbuk-do, partially due to the reporting delay. Our estimates of the risk for death in Gyeongsangbuk-do reached 2.4% (95% CrI: 1.6-3.4%), 1.3% (95% CrI: 1.0-1.6%) in Daegu and 0.7% (95% CrI: 0.3-1.4%) in other regions, whereas the national estimate of the risk for death was estimated at 1.4% (95% CrI: 1.2-1.7%) in Korea. Conclusions: T...","Abstract The sensitivity and specificity of various severe acute respiratory syndrome coronavirus (SARS-CoV) PCR primer and probe sets were evaluated through the use of commercial kits and in-house amplification formats. Conventional and real-time PCR assays were performed using a heat-block thermocycler ABI 9600, the Roche LightCycler\u2122 version 1.2, or the ABI 7000 Sequence Detection System. The sensitivity of all primers was between 0.0004 and 0.04 PFU with viral cell lysate and between 0.004 and 0.4 PFU in spiked stool specimen per PCR assay. The primer sets for real-time PCR assays were at one least 1 log more sensitive than the primer sets used in the conventional PCR. A panel of viruses including swine gastroenteritis virus, bovine coronavirus, avian bronchitis virus (Connecticut strain), avian bronchitis virus (Massachusetts strain), human coronaviruses 229E and OC43, parainfluenza virus (type III), human metapneumovirus, adenovirus, respiratory syncytial virus, and influenza A w...","This paper examines the role of various socioeconomic factors in mediating the local and cross-city transmissions of the novel coronavirus 2019 (COVID-19) in China. We implement a machine learning approach to select instrumental variables that strongly predict virus transmission among the rich exogenous weather characteristics. Our 2SLS estimates show that the stringent quarantine, massive lockdown and other public health measures imposed in late January significantly reduced the transmission rate of COVID-19. By early February, the virus spread had been contained. While many socioeconomic factors mediate the virus spread, a robust government response since late January played a determinant role in the containment of the virus. We also demonstrate that the actual population flow from the outbreak source poses a higher risk to the destination than other factors such as geographic proximity and similarity in economic conditions. The results have rich implications for ongoing global effor...","Abstract To help reveal the diversity and evolution of bat coronaviruses we collected 1067 bats from 21 species in China. A total of 73 coronaviruses (32 alphacoronaviruses and 41 betacoronaviruses) were identified in these bats, with an overall prevalence of 6.84%. All newly-identified betacoronaviruses were SARS-related Rhinolophus bat coronaviruses (SARSr-Rh-BatCoV). Importantly, with the exception of the S gene, the genome sequences of the SARSr-Rh-BatCoVs sampled in Guizhou province were closely related to SARS-related human coronavirus. Additionally, the newly-identified alphacoronaviruses exhibited high genetic diversity and some may represent novel species. Our phylogenetic analyses also provided insights into the transmission of these viruses among bat species, revealing a general clustering by geographic location rather than by bat species. Inter-species transmission among bats from the same genus was also commonplace in both the alphacoronaviruses and betacoronaviruses. Over...","This year marks the 15th anniversary of Severe Acute Respiratory Syndrome (SARS) in Canada and the 100th anniversary of the 1918 Spanish influenza pandemic. These, and other recent public health events, provide an opportunity for us to review and reflect on the evolution of Canada\u2019s public health emergency response over the past 15 years\u2014from SARS, to the 2009 H1N1 pandemic influenza, to Ebola virus and Zika virus disease. Key lessons have been learned and milestones achieved that have shaped and sharpened our response approach and structures. While SARS was a wake-up-call to strengthen infection prevention and control capacity in health care settings and led to the formation of the Public Health Agency of Canada, it also strengthened our Federal/Provincial/Territorial (FPT) senior-level governance and led to agreements for pan-Canadian mutual aid and infectious disease information sharing. As well, our collective public health laboratory capacity has been strengthened through ongoing ...","","Zoonotic transmissions of emerging pathogens from wildlife to human have shaped the history of mankind. These events have also highlighted our poor understanding of microorganisms circulated in wild animals. Coronaviruses and astroviruses, which can be found from a wide range of mammals, were recently detected in bats. Strikingly, these bat viruses are genetically highly diverse and these interesting findings might help to better understand the evolution and ecology of these viruses. The discoveries of these novel bats viruses not only suggested that bats are important hosts for these virus families, but also reiterated the role of bats as a reservoir of viruses that might pose a zoonotic threat to human health.","Abstract The infection by the new coronavirus (SARS-CoV-2) has taken the dimension of a pandemic, affecting more than 160 countries in a few weeks. In Colombia, despite the implementation of the rules established by the national government, exists an elevate concern both for mortality and for the limited capacity of the health system to respond effectively to the needs of patients infected. For Colombia, assuming a case fatality rate among people infected with SARS-CoV-2 of 0.6% (average data from the information reported for Latin American countries for March 18) (Table 1), the number of deaths, in one or two weeks, could be 16 and 243, respectively. These estimates differ markedly from those documented in countries such as Spain and Italy, in which COVID-19 case fatality rates exceed 8% (case of Italy) and from the percentage of patients who have required intensive care, which has ranged from 9% to 11% of patients in Mediterranean European countries. These differences could be explai...","The human coronavirus NL63 (HCoV-NL63) was first identified in the Netherlands, and its circulation in France has not been investigated. We studied HCoV-NL63 infection in hospitalized children diagnosed with respiratory tract infections. From November 2002 to April 2003, we evaluated 300 respiratory specimens for HCoV-NL63. Of the 300 samples, 28 (9.3%) were positive for HCoV-NL63. The highest prevalence was found in February (18%). The main symptoms were fever (61%), rhinitis (39%), bronchiolitis (39%), digestive problems (33%), otitis (28%), pharyngitis (22%), and conjunctivitis (17%). A fragment of the spike protein gene was sequenced to determine the variety of circulating HCoV-NL63. Phylogenetic analysis indicated that strains with different genetic markers cocirculate in France.","Abstract Pathogenic viruses are viruses that can infect and replicate within human cells and cause diseases. The continuous emergence and re-emergence of pathogenic viruses has become a major threat to public health. Whenever pathogenic viruses emerge, their rapid detection is critical to enable implementation of specific control measures and the limitation of virus spread. Further molecular characterization to better understand these viruses is required for the development of diagnostic tests and countermeasures. Advances in molecular biology techniques have revolutionized the procedures for detection and characterization of pathogenic viruses. The development of PCR-based techniques together with DNA sequencing technology, have provided highly sensitive and specific methods to determine virus circulation. Pathogenic viruses potentially having global catastrophic consequences may emerge in regions where capacity for their detection and characterization is limited. Development of a loc...","Abstract The pandemic of new coronavirus disease COVID-19 is threatening our health, economy and life style. Collaborations across countries and sectors as a One Health World could be a milestone. We propose a general protocol, for setting timely active random surveillance of COVID-19, at the human community level, with systematic repeated detection efforts. Strengths and limitations are discussed. If considered applicable by public health, the protocol could evaluate the status of COVID-19 epidemics consistently and objectively.","","Epidemic diseases are not a new phenomenon, but easy access to transport in the modern world has accelerated their spread. Perhaps some botanical understanding can help slow them down.","Abstract In 2002/2003, a novel coronavirus (CoV) caused a pandemic, infecting more than 8000 people, of whom nearly 10% died. This virus, termed severe acute respiratory syndrome-CoV was linked to a zoonotic origin from rhinolophid bats in 2005. Since then, numerous studies have described novel bat CoVs, including close relatives of the newly emerging Middle East respiratory syndrome (MERS)-CoV. In this paper we discuss CoV genomic properties and compare different taxonomic approaches in light of the technical difficulties of obtaining full genomic sequences directly from bat specimens. We first present an overview of the available studies on bat CoVs, with details on their chiropteran hosts, then comparatively analyze the increase in bat CoV studies and novel genomic sequences obtained since the SARS pandemic. We then conduct a comprehensive phylogenetic analysis of the genera Alpha- and Betacoronavirus, to show that bats harbour more CoV diversity than other mammalian hosts and are w...","Microarrays can be constructed with dozens to millions of probes on their surface to allow high-throughput analyses of many biologic processes to be performed simultaneously on the same sample. Microarrays are now widely used for gene expression analysis, deoxyribonucleic acid resequencing, single-nucleotide polymorphism genotyping, and comparative genomic hybridization. Microarray technology is accelerating research in many fields and now microarrays are moving into clinical application. This review discusses the emerging role of microarrays in molecular diagnostics, pathogen detection, oncology, and pharmacogenomics.","Abstract Cancer patients have an increased risk of developing severe forms of COVID-19, and advanced cancer patients who are followed at home, represent a particularly frail population. Although with substantial differences, the challenges which cancer care professionals have to face with during a pandemic are quite similar to those posed by natural disasters. Some of us have already managed the oncological home care service after the 2009 earthquake in L\u2019Aquila (middle Italy). With this letter, we want to share with the medical community the procedures and tools that we have started using at the home care service of the Tuscany Tumor Association during the COVID-19 pandemic.","Several French regions where COVID19 has been reported currently show a renewed increase in ILI cases in the general practice based Sentinelles network. Here we computed the number of excess cases by region and found correlation with the number of reported COVID19 cases so far. These data suggest larger circulation of SARS-CoV-2 in the French population than apparent from confirmed cases.","The recent emergence of SARS-CoV-2 lead to a current pandemic of unprecedented levels. Though diagnostic tests are fundamental to the ability to detect and respond, many health systems are already experiencing shortages of reagents associated with this test. Here, testing a pooling approach for the standard RT-qPCR test, we find that a single positive sample can be detected even in pools of up to 32 samples, with an estimated false negative rate of 10%. Detection of positive samples diluted in even up to 64 samples may also be attainable, though may require additional amplification cycles. As it uses the standard protocols, reagents and equipment, this pooling method can be applied immediately in current clinical testing laboratories. We hope that such implementation of a pool test for COVID-19 would allow expanding current screening capacities thereby enabling the expansion of detection in the community, as well as in close integral groups, such as hospital departments, army units, or...","","Introduction The COVID 19 outbreak is posing an unprecedented challenge to healthcare workers. This study analyzes the geo temporal effects on disease severity for the 1,688 Chinese healthcare workers infected with COVID 19. Method Using the descriptive results recently reported by the Chinese CDC, we compare the percentage of infected healthcare workers in severe conditions over time and across three areas in China, and the fatality rate of infected healthcare workers with all the infected individuals in China aged 22 to 59 years. Results Among the infected Chinese healthcare workers whose symptoms onset appeared during the same ten day period, the percentage of those in severe conditions decreased statistical significantly from 19.7% (Jan 11 to 20) to 14.4% (Jan 21 to 31) to 8.7% (Feb 1 to 11). Across the country, there was also a significant difference in the disease severity among patients symptoms onset during the same period, with Wuhan being the most severe (17%), followed by Hu...","The current state of much of the Wuhan pneumonia virus (COVID-19) research shows a regrettable lack of data sharing and considerable analytical obfuscation. This impedes global research cooperation, which is essential for tackling public health emergencies, and requires unimpeded access to data, analysis tools, and computational infrastructure. Here we show that community efforts in developing open analytical software tools over the past ten years, combined with national investments into scientific computational infrastructure, can overcome these deficiencies and provide an accessible platform for tackling global health emergencies in an open and transparent manner. Specifically, we use all COVID-19 genomic data available in the public domain so far to (1) underscore the importance of access to raw data and to (2) demonstrate that existing community efforts in curation and deployment of biomedical software can reliably support rapid, reproducible research during global health crises. A..."],"title":["A Tempo-geographic Analysis of Global COVID-19 Epidemic Outside of China","Descriptive review of geographic mapping of severe acute respiratory syndrome (SARS) on the Internet","Global transmission network of SARS-CoV-2: from outbreak to pandemic","Networks of information token recurrences derived from genomic sequences may reveal hidden patterns in epidemic outbreaks: A case study of the 2019-nCoV coronavirus.","Rapid random access detection of the novel SARS-coronavirus-2 (SARS-CoV-2, previously 2019-nCoV) using an open access protocol for the Panther Fusion","CRISPR-based surveillance for COVID-19 using genomically-comprehensive machine learning design","Functional assessment of cell entry and receptor usage for lineage B \u03b2-coronaviruses, including 2019-nCoV","Calling for rapid development of a safe and effective MERS vaccine","Viral etiology and seasonality of influenza-like illness in Gabon, March 2010 to June 2011","COVID-19: Challenges to GIS with Big Data","The local stability of a modified multi-strain SIR model for emerging viral strains","Prevalence and Phylogeny of Coronaviruses in Wild Birds from the Bering Strait Area (Beringia)","Rapid Detection of 2019 Novel Coronavirus SARS-CoV-2 Using a CRISPR-based DETECTR Lateral Flow Assay","Viral Zoonoses","Is COVID-19 receiving ADE from other coronaviruses?","Temporal dynamics in viral shedding and transmissibility of COVID-19","SARS-CoV-2 is sensitive to type I interferon pretreatment","Vaccines for emerging infectious diseases: Lessons from MERS coronavirus and Zika virus","Coronavirus Disease (COVID-19): Spectrum of CT Findings and Temporal Progression of the Disease","Incubation Period and Other Epidemiological Characteristics of 2019 Novel Coronavirus Infections with Right Truncation: A Statistical Analysis of Publicly Available Case Data","Incubation Period and Other Epidemiological Characteristics of 2019 Novel Coronavirus Infections with Right Truncation: A Statistical Analysis of Publicly Available Case Data","Analytical sensitivity and efficiency comparisons of SARS-COV-2 qRT-PCR assays","Gut microbiota: Implications in Parkinson's disease","Wuhan novel coronavirus 2019nCoV","Chapter 16 Emerging and Reemerging Viral Diseases","On the Coronavirus (COVID-19) Outbreak and the Smart City Network: Universal Data Sharing Standards Coupled with Artificial Intelligence (AI) to Benefit Urban Health Monitoring and Management","Statistics-Based Predictions of Coronavirus Epidemic Spreading in Mainland China","Design of multi epitope-based peptide vaccine against E protein of human COVID-19: An immunoinformatics approach","Emerging Viral Infections in India","Estimating the risk of COVID-19 death during the course of the outbreak in Korea, February-March, 2020","Comparison of 9 different PCR primers for the rapid detection of severe acute respiratory syndrome coronavirus using 2 RNA extraction methods","Impacts of social and economic factors on the transmission of coronavirus disease (COVID-19) in China","Extensive diversity of coronaviruses in bats from China","Fifteen years post-SARS: Key milestones in Canada\u2019s public health emergency response","Ensuring global access to COVID-19 vaccines","Novel coronaviruses and astroviruses in bats","COVID-19 in Colombia endpoints. Are we different, like Europe?","Human Coronavirus NL63, France","Pathogenic viruses: Molecular detection and characterization","Base protocol for real time active random surveillance of coronavirus disease (COVID-19) \u2013 Adapting veterinary methodology to public health","Multi-omics Evaluation of Gastrointestinal and Other Clinical Characteristics of SARS-CoV-2 and COVID-19","Redeploying plant defences","Ecology, evolution and classification of bat coronaviruses in the aftermath of SARS","Microarray technology in obstetrics and gynecology: A guide for clinicians","Home care for cancer patients during COVID-19 pandemic: the \u201cdouble triage\u201d protocol.","Excess cases of Influenza like illnesses in France synchronous with COVID19 invasion.","Evaluation of COVID-19 RT-qPCR test in multi-sample pools","Proposal for vaccination against SARS coronavirus using avian infectious bronchitis virus strain H from The Netherlands","Geo temporal distribution of 1,688 Chinese healthcare workers infected with COVID-19 in severe conditions, a secondary data analysis","No more business as usual: agile and effective responses to emerging pathogen threats require open data and open analytics"],"url":["https://doi.org/10.1101/2020.03.20.20039602","https://www.ncbi.nlm.nih.gov/pmc/articles/PMC343293/","https://doi.org/10.1101/2020.03.22.20041145","https://doi.org/10.1101/2020.02.07.20021139","https://doi.org/10.1016/j.jcv.2020.104305","https://doi.org/10.1101/2020.02.26.967026","https://doi.org/10.1101/2020.01.22.915660","https://doi.org/10.1016/j.micinf.2014.05.002","https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4107952/","https://doi.org/10.1016/j.geosus.2020.03.005","https://doi.org/10.1101/2020.03.19.20039198","https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2966397/","https://doi.org/10.1101/2020.03.06.20032334","https://doi.org/10.1016/b978-0-12-801238-3.95729-5","https://doi.org/10.1016/j.micinf.2020.02.006","https://doi.org/10.1101/2020.03.15.20036707","https://doi.org/10.1101/2020.03.07.982264","https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5718785/","https://doi.org/10.1016/j.acra.2020.03.003","https://doi.org/10.3390/jcm9020538","https://doi.org/10.1101/2020.01.26.20018754","https://doi.org/10.1101/2020.03.30.20048108","https://doi.org/10.1016/j.parkreldis.2017.02.002","https://doi.org/10.31646/gbio.50","https://doi.org/10.1016/b978-0-12-800947-5.00016-8","https://doi.org/10.3390/healthcare8010046","https://doi.org/10.20535/ibb.2020.4.1.195074","https://doi.org/10.1101/2020.02.04.934232","https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7100311/","https://doi.org/10.1101/2020.03.30.20048264","https://doi.org/10.1016/j.diagmicrobio.2005.03.007","https://doi.org/10.1101/2020.03.13.20035238","https://doi.org/10.1016/j.virol.2017.03.019","https://doi.org/10.14745/ccdr.v44i05a01","https://doi.org/10.1016/s0140-6736(20)30763-7","https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7091142/","https://doi.org/10.1016/j.sapharm.2020.03.013","https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3320486/","https://doi.org/10.1016/j.meegid.2020.104215","https://doi.org/10.1016/j.onehlt.2020.100129","https://doi.org/10.1053/j.gastro.2020.03.045","https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7091691/","https://doi.org/10.1016/j.antiviral.2013.10.013","https://doi.org/10.1016/j.ajog.2005.12.014","https://doi.org/10.1016/j.jpainsymman.2020.03.021","https://doi.org/10.1101/2020.03.14.20035741","https://doi.org/10.1101/2020.03.26.20039438","https://doi.org/10.1016/j.jinf.2005.04.010","https://doi.org/10.1101/2020.03.19.20032532","https://doi.org/10.1101/2020.02.21.959973"],"x":[0.4006417395758378,0.453518014882648,0.9031251325456979,0.3105781194351065,0.19919850810870354,0.02622979600423392,0.3791155801872569,0.15869351110573782,0.43319871027123524,0.12592099071810436,0.08866678099142278,0.026550618933006476,0.07628253418545305,0.31013921830376134,0.9574260258416804,0.6870436033414467,0.7759541226326784,0.4328172388239244,0.0832319405797699,0.7360404589649053,0.0019869529317871892,0.29834884994035527,0.8597355686870862,0.08470597638017963,0.48029199957156377,0.655586689655212,0.7004108780824132,0.1472135535111062,0.14859439914158867,0.01729228259740978,0.2610382078971365,0.12865563663338964,0.7876005704660581,0.4043613542609521,0.20629676353129522,0.17155783967517346,0.09204153821153471,0.1398990008978055,0.014444433458907668,0.3366910337474023,0.6249708680459749,0.6537855860336537,0.4294372396277951,0.44359467299521804,0.7623726689132058,0.5423903733143626,0.015771684520813123,0.9529009263986938,0.5050245895358885,0.3392994313457969],"y":[0.794387490623152,0.014026212216725065,0.3753767048109051,0.7964116795225247,0.45580208671506783,0.9449233241834861,0.6916052343710982,0.502201637502859,0.1710280911165244,0.18525196442402037,0.9661173791925576,0.5850212939756524,0.16328168975627244,0.7274315533502483,0.2555337641591994,0.402286039972074,0.2627602132058795,0.40213784307659617,0.8436439943940329,0.26213637364390563,0.36422940737619014,0.6625561384599761,0.9395535037245101,0.8703883410864772,0.862697132442346,0.7694011394748437,0.07428099578828395,0.5885565001921809,0.38589609247857104,0.09838657889068847,0.13341947598931003,0.5357852478231127,0.266419116420892,0.7634111724712684,0.2137676853085878,0.6397209463084131,0.8409849107703695,0.8326345390350905,0.681686233750882,0.14430746559630336,0.97762462905116,0.005795631378498478,0.8456545606815545,0.5736022185391242,0.6567082926605806,0.34123342498127374,0.29979113509370203,0.8257889753844365,0.6753526884607782,0.05415015396152856]},"selected":{"id":"8479"},"selection_policy":{"id":"8496"}},"id":"8407","type":"ColumnDataSource"},{"attributes":{},"id":"8500","type":"StringEditor"},{"attributes":{"callback":null,"tooltips":[["Title","@title"],["Passage","@text"],["Published","@date"],["Authors","@authors"],["Journal","@journal"],["URL","@url"]]},"id":"8463","type":"HoverTool"},{"attributes":{"callback":null},"id":"8426","type":"TapTool"},{"attributes":{"js_property_callbacks":{"change:indices":[{"id":"8480"}]}},"id":"8479","type":"Selection"},{"attributes":{"axis":{"id":"8418"},"ticker":null},"id":"8421","type":"Grid"},{"attributes":{"children":[{"id":"8477"},{"id":"8444"}]},"id":"8490","type":"Row"}],"root_ids":["8491"]},"title":"Bokeh Application","version":"2.0.1"}}
        </script>
        <script type="text/javascript">
          (function() {
            var fn = function() {
              Bokeh.safely(function() {
                (function(root) {
                  function embed_document(root) {
                    
                  var docs_json = document.getElementById('8728').textContent;
                  var render_items = [{"docid":"ec36bda8-e37f-4504-ae0f-16c12589fe11","root_ids":["8491"],"roots":{"8491":"b0e05c20-e9ea-414e-b8bc-85991ec0d789"}}];
                  root.Bokeh.embed.embed_items(docs_json, render_items);
                
                  }
                  if (root.Bokeh !== undefined) {
                    embed_document(root);
                  } else {
                    var attempts = 0;
                    var timer = setInterval(function(root) {
                      if (root.Bokeh !== undefined) {
                        clearInterval(timer);
                        embed_document(root);
                      } else {
                        attempts++;
                        if (attempts > 100) {
                          clearInterval(timer);
                          console.log("Bokeh: ERROR: Unable to run BokehJS code because BokehJS library is missing");
                        }
                      }
                    }, 10, root)
                  }
                })(window);
              });
            };
            if (document.readyState != "loading") fn();
            else document.addEventListener("DOMContentLoaded", fn);
          })();
        </script>
    
  </body>
  
</html>