



<!DOCTYPE html>
<html lang="en">
  
  <head>
    
      <meta charset="utf-8">
      <title>Bokeh Plot</title>
      
      
        
          
        
        
          
        <script type="text/javascript" src="https://cdn.bokeh.org/bokeh/release/bokeh-2.0.1.min.js" integrity="sha384-JpP8FXbgAZLkfur7LiK3j9AGBhHNIvF742meBJrjO2ShJDhCG2I1uVvW+0DUtrmc" crossorigin="anonymous"></script>
        <script type="text/javascript" src="https://cdn.bokeh.org/bokeh/release/bokeh-widgets-2.0.1.min.js" integrity="sha384-xZlADit0Q04ISQEdKg2k3L4W9AwQBAuDs9nJL9fM/WwzL1tEU9VPNezOFX0nLEAz" crossorigin="anonymous"></script>
        <script type="text/javascript" src="https://cdn.bokeh.org/bokeh/release/bokeh-tables-2.0.1.min.js" integrity="sha384-4BuPRZkdMKSnj3zoxiNrQ86XgNw0rYmBOxe7nshquXwwcauupgBF2DHLVG1WuZlV" crossorigin="anonymous"></script>
        <script type="text/javascript">
            Bokeh.set_log_level("info");
        </script>
        
      
      
    
  </head>
  
  
  <body>
    
      
        
          
          
            
              <div class="bk-root" id="6b01c24d-4af5-4bc1-b7d9-84c1216de234" data-root-id="24591"></div>
            
          
        
      
      
        <script type="application/json" id="24828">
          {"73736b34-131c-4147-ac8e-d78b70400a90":{"roots":{"references":[{"attributes":{"columns":[{"id":"24575"},{"id":"24576"}],"height":340,"source":{"id":"24508"},"view":{"id":"24578"},"width":900},"id":"24577","type":"DataTable"},{"attributes":{},"id":"24609","type":"StringFormatter"},{"attributes":{"axis":{"id":"24557"},"dimension":1,"ticker":null},"id":"24560","type":"Grid"},{"attributes":{},"id":"24611","type":"StringFormatter"},{"attributes":{"fill_color":{"field":"color"},"line_color":{"field":"color"},"radius":{"field":"rad","units":"data"},"x":{"field":"x"},"y":{"field":"y"}},"id":"24569","type":"Circle"},{"attributes":{},"id":"24549","type":"LinearScale"},{"attributes":{},"id":"24551","type":"LinearScale"},{"attributes":{},"id":"24606","type":"UnionRenderers"},{"attributes":{"data_source":{"id":"24508"},"glyph":{"id":"24569"},"hover_glyph":null,"muted_glyph":null,"nonselection_glyph":{"id":"24570"},"selection_glyph":null,"view":{"id":"24572"}},"id":"24571","type":"GlyphRenderer"},{"attributes":{"text":""},"id":"24595","type":"Title"},{"attributes":{"children":[{"id":"24585"}]},"id":"24589","type":"Row"},{"attributes":{"editor":{"id":"24608"},"field":"scr","formatter":{"id":"24609"},"title":"Score","width":30},"id":"24575","type":"TableColumn"},{"attributes":{"editor":{"id":"24598"},"field":"scr","formatter":{"id":"24599"},"title":"Score","width":30},"id":"24540","type":"TableColumn"},{"attributes":{"editor":{"id":"24610"},"field":"text","formatter":{"id":"24611"},"title":"Passage","width":770},"id":"24576","type":"TableColumn"},{"attributes":{"text":"&lt;h3&gt;Paragraph ranking&lt;/h3&gt;"},"id":"24585","type":"Div"},{"attributes":{"js_property_callbacks":{"change:indices":[{"id":"24580"}]}},"id":"24579","type":"Selection"},{"attributes":{"children":[{"id":"24583"}]},"id":"24586","type":"Row"},{"attributes":{"source":{"id":"24507"}},"id":"24537","type":"CDSView"},{"attributes":{},"id":"24610","type":"StringEditor"},{"attributes":{},"id":"24598","type":"StringEditor"},{"attributes":{"axis":{"id":"24518"},"ticker":null},"id":"24521","type":"Grid"},{"attributes":{},"id":"24562","type":"ResetTool"},{"attributes":{"source":{"id":"24508"}},"id":"24578","type":"CDSView"},{"attributes":{"text":""},"id":"24593","type":"Title"},{"attributes":{"source":{"id":"24507"}},"id":"24543","type":"CDSView"},{"attributes":{"children":[{"id":"24584"}]},"id":"24587","type":"Row"},{"attributes":{"children":[{"id":"24577"},{"id":"24544"}]},"id":"24590","type":"Row"},{"attributes":{"formatter":{"id":"24603"},"ticker":{"id":"24523"}},"id":"24522","type":"LinearAxis"},{"attributes":{"below":[{"id":"24553"}],"center":[{"id":"24556"},{"id":"24560"}],"left":[{"id":"24557"}],"plot_height":340,"plot_width":900,"renderers":[{"id":"24571"}],"title":{"id":"24595"},"toolbar":{"id":"24564"},"toolbar_location":"below","x_range":{"id":"24545"},"x_scale":{"id":"24549"},"y_range":{"id":"24547"},"y_scale":{"id":"24551"}},"id":"24544","subtype":"Figure","type":"Plot"},{"attributes":{"formatter":{"id":"24615"},"ticker":{"id":"24554"}},"id":"24553","type":"LinearAxis"},{"attributes":{"callback":null},"id":"24561","type":"TapTool"},{"attributes":{"fill_alpha":{"value":0.1},"fill_color":{"field":"color"},"line_alpha":{"value":0.1},"line_color":{"field":"color"},"radius":{"field":"rad","units":"data"},"x":{"field":"x"},"y":{"field":"y"}},"id":"24570","type":"Circle"},{"attributes":{"children":[{"id":"24542"},{"id":"24509"}]},"id":"24588","type":"Row"},{"attributes":{},"id":"24547","type":"Range1d"},{"attributes":{},"id":"24613","type":"BasicTickFormatter"},{"attributes":{},"id":"24615","type":"BasicTickFormatter"},{"attributes":{"columns":[{"id":"24540"},{"id":"24541"}],"height":340,"source":{"id":"24507"},"view":{"id":"24543"},"width":900},"id":"24542","type":"DataTable"},{"attributes":{},"id":"24512","type":"Range1d"},{"attributes":{"js_property_callbacks":{"change:indices":[{"id":"24582"}]}},"id":"24581","type":"Selection"},{"attributes":{},"id":"24519","type":"BasicTicker"},{"attributes":{"callback":null,"tooltips":[["Title","@title"],["Passage","@text"],["Published","@date"],["Authors","@authors"],["Journal","@journal"],["URL","@url"]]},"id":"24563","type":"HoverTool"},{"attributes":{"below":[{"id":"24518"}],"center":[{"id":"24521"},{"id":"24525"}],"left":[{"id":"24522"}],"plot_height":340,"plot_width":900,"renderers":[{"id":"24536"}],"title":{"id":"24593"},"toolbar":{"id":"24529"},"toolbar_location":"below","x_range":{"id":"24510"},"x_scale":{"id":"24514"},"y_range":{"id":"24512"},"y_scale":{"id":"24516"}},"id":"24509","subtype":"Figure","type":"Plot"},{"attributes":{"editor":{"id":"24600"},"field":"title","formatter":{"id":"24601"},"title":"Document title","width":770},"id":"24541","type":"TableColumn"},{"attributes":{},"id":"24600","type":"StringEditor"},{"attributes":{},"id":"24599","type":"StringFormatter"},{"attributes":{"fill_color":{"field":"color"},"line_color":{"field":"color"},"radius":{"field":"rad","units":"data"},"x":{"field":"x"},"y":{"field":"y"}},"id":"24534","type":"Circle"},{"attributes":{},"id":"24510","type":"Range1d"},{"attributes":{},"id":"24596","type":"UnionRenderers"},{"attributes":{},"id":"24514","type":"LinearScale"},{"attributes":{"args":{"sd":{"id":"24507"},"sp":{"id":"24508"}},"code":"\n    if (sp.selected.indices.length == 0) {\n      return;\n    }\n    var sel = sp.selected.indices[0];\n    var new_selected = [];\n    for (var i = 0; i &lt; sd.data['doc'].length; i++) {\n      if (sd.data['doc'][i] == sp.data['doc'][sel]) {\n        new_selected.push(i);\n      }\n    }\n    if (JSON.stringify(new_selected) != JSON.stringify(sd.selected.indices) &amp;&amp;\n    !(sd.selected.indices.length == 1 &amp;&amp; new_selected.includes(sd.selected.indices[0]))) {\n      sd.selected.indices = new_selected;\n      sd.change.emit();\n    }\n    "},"id":"24582","type":"CustomJS"},{"attributes":{"fill_alpha":{"value":0.1},"fill_color":{"field":"color"},"line_alpha":{"value":0.1},"line_color":{"field":"color"},"radius":{"field":"rad","units":"data"},"x":{"field":"x"},"y":{"field":"y"}},"id":"24535","type":"Circle"},{"attributes":{"args":{"sd":{"id":"24507"},"sp":{"id":"24508"}},"code":"\n    if (sd.selected.indices.length == 0) {\n      return;\n    }\n    var sel = sd.selected.indices[0];\n    var new_selected = [];\n    for (var i = 0; i &lt; sp.data['doc'].length; i++) {\n      if (sp.data['doc'][i] == sd.data['doc'][sel]) {\n        new_selected.push(i);\n      }\n    }\n    if (JSON.stringify(new_selected) != JSON.stringify(sp.selected.indices) &amp;&amp;\n    !(sp.selected.indices.length == 1 &amp;&amp; new_selected.includes(sp.selected.indices[0]))) {\n      sp.selected.indices = new_selected;\n      sp.change.emit();\n    }\n    "},"id":"24580","type":"CustomJS"},{"attributes":{"data":{"authors":["Aiping Wu; Peihua Niu; Lulan Wang; Hangyu Zhou; Xiang Zhao; Wenling Wang; Jingfeng Wang; Chengyang Ji; Xiao Ding; Xianyue Wang; Roujian Lu; Sarah Gold; Saba Aliyari; Shilei Zhang; Ellee Vikram; Angela Zou; Emily Lenh; Janet Chen; Fei Ye; Na Han; Yousong Peng; Haitao Guo; Guizhen Wu; Taijiao Jiang; Wenjie Tan; Genhong Cheng","Jing-Wen Ai; Hao-Cheng Zhang; Teng Xu; Jing Wu; Mengqi Zhu; Yi-Qi Yu; Han-Yue Zhang; Zhongliang Shen; Yang Li; Xian Zhou; Guo-Qing Zang; Jie Xu; Wen-Jing Chen; Yong-Jun Li; De-Sheng Xie; Ming-Zhe Zhou; Jing-Ying Sun; Jia-Zhen Chen; Wen-Hong Zhang","Longxian Lv; Gaolei Li; Jinhui Chen; Xinle Liang; Yudong Li","FENNER, FRANK; McAUSLAN, B.R.; MIMS, C.A.; SAMBROOK, J.; WHITE, DAVID O.","Dowling, Nicole F; Gwinn, Marta; Mawle, Alison","Ison, Michael G; Johnston, Sebastian L; Openshaw, Peter; Murphy, Brian; Hayden, Frederick","Su, John R.","Wat, Dennis","Mandary, Madiiha Bibi; Masomian, Malihe; Poh, Chit Laa","Ming Wang; Aisi Fu; Ben Hu; Yongqing Tong; Ran Liu; Jiashuang Gu; Jianghao Liu; Wen Jiang; Gaigai Shen; Wanxu Zhao; Dong Men; Lilei Yu; Zixin Deng; Yan Li; Tiangang Liu","Hu, Tingsong; Qiu, Wei; He, Biao; Zhang, Yan; Yu, Jing; Liang, Xiu; Zhang, Wendong; Chen, Gang; Zhang, Yingguo; Wang, Yiyin; Zheng, Ying; Feng, Ziliang; Hu, Yonghe; Zhou, Weiguo; Tu, Changchun; Fan, Quanshui; Zhang, Fuqiang","Villarreal, Luis P.","Falsey, Ann Regina","Villarreal, L.P.","Broadbent, Andrew J.; Boonnak, Kobporn; Subbarao, Kanta","Mandary, Madiiha Bibi; Masomian, Malihe; Poh, Chit Laa","Stannard, Daphne","Pratelli, Annamaria","Huang, Jyh-Win; Hou, Ting-Wei","Hon, Kam-Lun; Leung, Alexander K.C.","Villarreal, L.P.","Min Zhou; Yong Chen; Dexiang Wang; Yanping Xu; Weiwu Yao; Jingwen Huang; Xiaoyan Jin; Zilai Pan; Jingwen Tan; Lan Wang; Yihan Xia; Longkuan Zou; Xin Xu; Jingqi Wei; Mingxin Guan; Jianxing Feng; Huan Zhang; Jieming Qu","McCloskey, Brian; Zumla, Alimuddin; Ippolito, Giuseppe; Blumberg, Lucille; Arbon, Paul; Cicero, Anita; Endericks, Tina; Lim, Poh Lian; Borodina, Maya","Villarreal, L.P.","Barnard, Ross Thomas; Hall, Roy A; Gould, Ernest A","Wang, Wei; Cavailler, Philippe; Ren, Peijun; Zhang, Jing; Dong, Wei; Yan, Huajie; Mardy, Sek; Cailhol, Johann; Buchy, Philippe; Sheng, Jun; Fontanet, Arnaud; Deubel, Vincent","Guojun He; Yuhang Pan; Takanao Tanaka","van Doorn, H. Rogier; Yu, Hongjie","Yam, R.; Yuen, P.L.; Yung, R.; Choy, T.","Gorbalenya, A.E.","Yuzhen Zhang; Bin Jiang; Jiamin Yuan; Yanyun Tao","Dong, J.; Ismail, N.; Walker, D.H.","Pavel Skums; Alexander Kirpich; Pelin Icer Baykal; Alex Zelikovsky; Gerardo Chowell","Hudson, James B.","Chang Ha Woo; Sungho Jang; Giyoung Shin; Gyoo Yeol Jung; Jeong Wook Lee","Nor, Mohd Basri Mat; Richards, Guy A.; McGloughlin, Steve; Amin, Pravin R.","Nicola F Fletcher; Luke W Meredith; Emma L Tidswell; Steven R Bryden; Daniel Gon\u00e7alves-Carneiro; Yasmin Chaudhry; Claire Shannon-Lowe; Michael A Folan; Daniella A Lefteri; Marieke Pingen; Dalan Bailey; Clive S McKimmie; Alan W Baird","KWOK, Kin On; LAI, Florence; WEI, Wan In; WONG, Samuel Yeung Shan; TANG, Julian","MacIntyre, C Raina","Guojun He; Yuhang Pan; Takanao Tanaka","Joseph R Fauver; Mary E. Petrone; Emma B Hodcroft; Kayoko Shioda; Hanna Y Ehrlich; Alexander G. Watts; Chantal B.F. Vogels; Anderson F. Brito; Tara Alpert; Anthony Muyombwe; Jafar Razeq; Randy Downing; Nagarjuna R. Cheemarla; Anne L Wyllie; Chaney C. Kalinich; Isabel Ott; Josh Quick; Nicholas J. Loman; Karla M. Neugebauer; Alexander L. Greninger; Keith R. Jerome; Pavitra Roychoundhury; Hong Xie; Lasata Shrestha; Meei-Li Huang; Virginia E. Pitzer; Akiko Iwasaki; Saad B. Omer; Kamran Khan; Isaac Bogoch; Richard A. Martinello; Ellen F. Foxman; Marie-Louise Landry; Richard A Neher; Albert I Ko; Nathan D. Grubaugh","Oliva Bendtsen Cano; Sabrina Cano Morales; Claus Bendtsen","HANMING FANG; LONG WANG; YANG YANG","Kandel, Nirmal; Chungong, Stella; Omaar, Abbas; Xing, Jun","Hudson, James; Vimalanathan, Selvarani","Memish, Z.A.; Al-Tawfiq, J.A.; Makhdoom, H.Q.; Al-Rabeeah, A.A.; Assiri, A.; Alhakeem, R.F.; AlRabiah, F.A.; Al Hajjar, S.; Albarrak, A.; Flemban, H.; Balkhy, H.; Barry, M.; Alhassan, S.; Alsubaie, S.; Zumla, A.","Xie, Mingxuan; Chen, Qiong","Roman, Laura M.; Garoff, Henrik","Ye, Zi-Wei; Yuan, Shuofeng; Yuen, Kit-San; Fung, Sin-Yee; Chan, Chi-Ping; Jin, Dong-Yan","Xinhai Li; Xumao Zhao; Yingqiang Lou; Yuehua Sun"],"color":["#ef8a62","#ef8a62","#ef8a62","#ef8a62","#ef8a62","#ef8a62","#ef8a62","#ef8a62","#ef8a62","#ef8a62","#ef8a62","#ef8a62","#ef8a62","#ef8a62","#ef8a62","#ef8a62","#ef8a62","#ef8a62","#ef8a62","#ef8a62","#ef8a62","#ef8a62","#ef8a62","#ef8a62","#ef8a62","#ef8a62","#ef8a62","#ef8a62","#ef8a62","#ef8a62","#ef8a62","#ef8a62","#ef8a62","#ef8a62","#ef8a62","#ef8a62","#ef8a62","#ef8a62","#ef8a62","#ef8a62","#ef8a62","#ef8a62","#ef8a62","#ef8a62","#ef8a62","#ef8a62","#ef8a62","#ef8a62","#ef8a62","#ef8a62"],"date":["2020-03-02","2020-02-17","2020-03-02","1974-12-31","2009-12-29","2004-06-30","2004-09-30","2004-04-30","2019-09-19","2020-03-06","2014-11-30","2008-12-31","2019-12-31","2008-12-31","2015-12-31","2019-09-19","2009-06-30","2011-07-07","2007-05-31","2009-12-31","2008-12-31","2020-03-30","2020-03-20","2008-12-31","2014-01-09","2010-11-30","2020-04-01","2020-12-31","2011-01-31","2008-12-31","2020-03-06","2017-12-31","2020-03-27","2011-10-26","2020-03-09","2017-12-31","2020-03-30","2020-03-21","2020","2020-04-01","2020-03-26","2020-03-30","2020-03-26","2020-03-18","2011-07-13","2014-05-31","2020-04-01","1985-11-30","2020-03-15","2020-02-18"],"doc":["jmrg4oeb","nqxo3f45","3h1o0oz3","ggv87nhn","03bwk538","0rrgqukp","8z4aqfff","gzk9xbzk","kjqtvcfg","s4d442kx","1fw9ng2p","ydkigome","b5l6epy3","01a8er2v","dcui85lw","kjqtvcfg","28q5vpfc","z122v1uz","rpq6xan7","1nu4p8oz","01a8er2v","ilc2bzkx","s96xufes","01a8er2v","59gmu16g","lbx1plqx","if7av1x8","9koytpba","pz2us1xx","5yk1j4ms","utlponna","1h87wldg","ry9wpcxo","ckzn5otr","1uqjmeic","4hn55wui","nly9vojr","42h6uj3a","zehnqsp7","if7av1x8","8m06zdho","cc05jqb6","ucwuptgg","4so0jlrx","osdkejhz","fbqqfot9","juz9jnfk","fsfer9ue","lfndq85x","z5i8hyja"],"journal":["","","","The Biology of Animal Viruses","Genome Med","Antiviral Research","Clinics in Laboratory Medicine","European Journal of Internal Medicine","Int J Mol Sci","","BMC Microbiol","Origin and Evolution of Viruses","Genomic and Precision Medicine","Encyclopedia of Virology","Mucosal Immunology","Int J Mol Sci","Journal of PeriAnesthesia Nursing","Adv Virol","Computer Methods and Programs in Biomedicine","Advances in Pediatrics","Encyclopedia of Virology","","The Lancet","Encyclopedia of Virology","Expert Rev Mol Diagn","Journal of Clinical Virology","","Hunter's Tropical Medicine and Emerging Infectious Diseases","Journal of Hospital Infection","Encyclopedia of Virology","","Diagnostic Molecular Pathology","","J Biomed Biotechnol","","Journal of Critical Care","","Journal of Infection","Global Biosecurity","","","","","The Lancet","Pharmaceuticals (Basel)","Clinical Microbiology and Infection","International Journal of Infectious Diseases","Trends in Biochemical Sciences","Int J Biol Sci",""],"rad":[0.01936617029145928,0.016602266561040004,0.015281830429821744,0.013305545240847951,0.013076812782885847,0.01256925441043667,0.012275832472270178,0.011617094306015617,0.01158411413782378,0.010561604773489972,0.010286690546346943,0.008909662609276461,0.008327112869754908,0.008093242034911696,0.008019906308409517,0.007885092797831731,0.0077336399175717396,0.007637349132028905,0.007572775255329788,0.007357860937277358,0.006908508530114209,0.006883430801536288,0.006756747584777081,0.006588506190499527,0.006516503468464065,0.006243017838067336,0.006200253741281841,0.006049414304243223,0.005531514206792135,0.005351573466296684,0.005255581480987414,0.004995430131328273,0.0047337206420182566,0.004656919332334655,0.004622195054821679,0.004618788168368675,0.0046177555180040766,0.004488680373609815,0.004439673057346642,0.004255848848569664,0.004040185044613703,0.004025063417506195,0.0040181210623576324,0.0037034498239581866,0.0035176715178435705,0.003486513434311413,0.0031869371227048774,0.0031624740348302474,0.0030732372205854895,0.00300000460192584],"scr":[0.963,0.8,0.722,0.606,0.593,0.563,0.546,0.507,0.505,0.445,0.429,0.348,0.313,0.3,0.295,0.287,0.278,0.273,0.269,0.256,0.23,0.228,0.221,0.211,0.207,0.191,0.188,0.179,0.149,0.138,0.133,0.117,0.102,0.097,0.095,0.095,0.095,0.088,0.085,0.074,0.061,0.06,0.06,0.041,0.03,0.029,0.011,0.01,0.004,0.0],"text":["The start and stop codons of each predicted gene were manually checked to ensure the completeness. We tried to infer the possible evolution pathway and track the cotransmission of 2019-nCoV. To avoid random mutations that may mislead the inference, we removed the substitution that appeared only once among all the sample genomes.","Metagenomic sequencing was the method by which SARS-COV-2 was initially identified. Despite its longer turnaround time and higher cost, mNGS demonstrated a satisfying level of sensitivity when compared to PCR assays in our study. Moreover, the nature of unbiased sequencing offers higher tolerance for genetic drift in the target pathogens, whereas certain changes in sequence can result in critical impact on the assay efficiency 21, 22 . Along with targeted assays, mNGS should serve as an indispensable tool for sensitive and close monitoring of the molecular evolution for SARS-COV-2. Such genetic information will offer valuable insights not only for public health management, but also for contentious optimization of diagnostic assays 23, 24 . Therefore, as the SARS-COV-2 epidemic is currently entering into a critical stage, both within China and over the global 25 , according to the results of our study, the following improvements in molecular diagnosis are recommended in the optimized diagnostic flow diagram [ Figure  5 ]. First of all, repeated sampling could increase the positivity, and especially during clinical situations where epidemiological history and clinical manifestations highly indicate NCP, a third or fourth sampling may be helpful when resources are available. Second, our study showed that mNGS showed reliable and stable detection ability in the NCP diagnosis, and may facilitate to solve diagnostic difficulties during important or critical clinical cases and low positive RT-PCR results. What's more, in order to circumvent possible mutations in RT-PCR primers region, more than 1 gene target was recommended to amplify when designing RT-PCR assays [25] . Whole Genome Sequencing (WGS) method can overcome the mutation problems which cause false-negative result in RT-PCR. Therefore, we used this method in all enrolled patients and should be continue using for mutation surveillance. Lastly, lower respiratory tract specimens might provide higher positivity, due to that higher viral loads may be detected there. Therefore, physicians might collect such samples through bronchoscopy or trachea cannula when under through medical protection.","According to random drift hypothesis (15) \uff0c these nucleotide mutations among different SARS-CoV-2 strains now available are still determined by neutral evolution. In short, there has no powerful factor to force SARS-CoV-2 to evolve in a certain direction by far. However, we should take strict precautions against the strong factors that may cause directional variation of SARS-CoV-2 both in natural environment and infection treatment.","The virus neutralization test is the most sensitive and the most specific serological procedure available. Only antibodies directed against surface antigens of the virion, particularly those involved in adsorption to the host cell, will register in this test. These superficial antigens are those that have been most subject to the selective pressures of evolution (e.g., by antigenic drift, see Chapter 17), and they are therefore specific to the viral type or strain. Hence, neutralizing antibodies against a given serotype usually show little or no cross-reaction with other viruses within the same group.","Public health efforts to control and prevent infectious diseases are based on epidemiologic and laboratory surveillance systems that detect and monitor disease incidence, define pathogen characteristics, and track cases 'by person, place and time' [25]. Molecular epidemiology has long been a mainstay of public health surveillance; now, increasingly powerful molecular methods, including sequencing of whole pathogen genomes, help investigators to identify epidemiologically related cases, describe patterns of transmission, pinpoint sources of infection, and explain antimicrobial resistance [26-33]. Combining the methods of molecular pathogenomics with population genetics and epidemiology can provide new insights into the episodic behavior of epidemics of familiar pathogens, such as group A streptococcus [7]. Archived biological samples can also provide new insights into the emergence and evolution of infectious threats, from HIV [34] to influenza [35]. Building the capacity for human biological sample collection into existing surveillance networks has the potential to facilitate a more comprehensive, population-based evaluation of genomic and environmental determinants of health outcomes. For example, a meta-analysis of surveillance cohorts from Arizona, Colorado, California, and Illinois demonstrated that human homozygosity for CCR5delta32 (a non-functional variant of chemokine receptor CCR5) is consistently associated with symptomatic West Nile virus infection [36]. When disease surveillance is conducted independently in multiple jurisdictions, the capacity to share biological samples and epidemiologic and clinical data provides important infrastructure for research.","The NAs of approximately 100 isolates, both control viruses and clinical isolates including those lying above the 95% confidence limits, were sequenced. The sequences revealed variation in previously conserved residues. One of 10 N1, 7 of 38 N2, and 1 of 42 influenza B isolates had D151 to N, G, E, or V. Originally the D151 was thought to be the proton donor for the catalytic reaction mediated by the NA. Since none of these isolates showed decreased enzyme activity, the D151 likely does not provide this role. In addition to changes in non-conserved residues associated with natural antigen drift, there was also a change in the conserved residue T225I in 1 B virus, and co-evolution of I203L and E375G in 5/42 B isolates. Molecular modeling shows these latter two residues lie on opposite sides of the NA head and hence are more likely to be random rather than compensatory mutations. Importantly, despite sequence variations, there was no naturally occurring resistance to either zanamivir or oseltamivir. 9.9. Effect of mutations in the hemagglutinin gene of influenza A viruses on the resistance phenotype of neuraminidase inhibitors (Guy Boivin, Laval University, Qu\u00e9bec, Canada) Previous studies have shown that resistance to neuraminidase inhibitors (NAI) generated after in vitro passages may result in mutations in the neuraminidase (NA) and/or hemagglutinin (HA) genes. However, the effect of HA mutations alone on the resistance phenotype remains to be confirmed in clinical isolates. In influenza A/H1N9 the HA mutation S186F results in 10-fold reduced susceptibility to zanamivir, while S186F + K222T results in 1000-fold resistance to all known NAIs (Blick et al., 1998) . The S186F, K222T, and both mutations were incorporated in the HA transcription plasmid using the Quick change site-directed mutagenesis kit (Stratagene). Recombinant influenza A/WSN33 (H1N1) viruses were then generated by reverse genetics (Fodor et al., 1999) . In the NA inhibition assay, the wild type and HA mutants had similar low IC 50 values for zanamivir (0.67-1.06 nM) and peramivir (0.18-0.26 nM). The K222T virus had a three-to four-fold reduction in susceptibility by plaque reduction assay, while the other two mutants were no different than the wild type virus. This data suggests that in vitro generated resistance to NAIs may not be solely attributable to HA mutations as previously reported. In addition, the combined effects of HA and NA mutations needs to be investigated as potential mechanisms for NAI resistance (Nedyalkova et al., 2002) . This study also documents the utility of reverse genetics to study various HA substitutions in the same genetic background and the impact of these variant HAs on antiviral susceptibility.","Like other bunyaviruses, the hantavirus genome consists of three segments of single-stranded, negative-sense RNA housed in a helical nucleocapsid. This segmented genome has raised the question of genetic variation in hantaviruses. Genetic drift clearly plays a role [42, 43] . The role of genetic shift is less clear. Studies have demonstrated that diploid strains are unstable, but reassorted virus has been reported with genetic variants of SNV and Dobrava virus [44, 45] . Notably, homologous recombination-the first instance of such recombination in negative-sense RNA viruses-was first described in Tula virus, and later in Dobrava virus [45, 46] . Nonetheless, human infections appear to be more due to encroachment upon hantavirus host habitats (or vice versa) than to fundamental changes in the viruses themselves [41, 47] .","The existence of diverse viral serotypes in causing the common cold has made vaccine preparation very difficult. Frequent mutations of viral proteins of RNA viruses (for example, genetic drift and shift of influenza) have further hampered prevention of the illness.","Another challenge to overcome is to accurately determine the origin and spread of a founder population of the virus. Hence, the discrepancies in the evolution of HBV was investigated. Eight related patients with acquired chronic HBV through mother-to-infant transmission were selected and the viral genomes isolated were analyzed. Sequence analysis indicated that the samples originated from a single source of HBV genotype B2 (HBV-B2) which diverged from a tiny common ancestral pool regardless of the route of acquisition. Between individuals, viral strains obtained from a time point showed evidence that they originated from a small pool of the previous time point. This conferred the strain an advantage over other strains with regards to the recovery of the founder state shortly after transmission to the new host and the adaptation to the local environment within the host. Natural selection rather than genetic drift was hypothesized to be the root cause for the evolution of HBV, due to the observed varying patterns of divergence at synonymous and non-synonymous sites. This was in line with the higher rate of substitutions within the host rather than between hosts. Approximately 85/88 amino acid residues changed from common to rare residues. Since these changes were shown not to be a random process, it is concluded that the HBV was able to evolve and change but was limited to a defined range of phenotypes. It can be argued that the mechanism observed thus far suggest that the adaptive mutations accumulated in one individual would not be maintained in another individual and might revert after transmission. Hence, within the host, substitutions were higher than between hosts [156].","can detect variations directly from clinical samples. Whereas the detection throughput of NTS is not 274 high at present, the NTS method can be integrated into widely used automated or semi-automated 275 platforms to improve the detection throughput in the future 21-23 . In addition, because PCR is 276 included in NTS, processes involving opening the lid of the PCR tubes may cause mutual 277 contamination between samples 24, 25 . However, this situation also is inevitable in current nucleic 278 acid detection methods (e.g., qPCR) or other nucleic acid detection schemes (e.g., SHERLOCK 11, 12 279 or toehold switch biosensor 9, 10 ) that also involve PCR. The introduction of integration systems or 280 sealed devices such as microfluidics may avoid this situation 26, 27 . At present, our processes of 281 sequencing data analysis and interpretation of results are not mature; nevertheless, as the number of 282 test samples increases, additional test results will be collected and the process continuously 283 optimized to obtain more accurate results. 284","In the present study we isolated a novel reovirus from intestinal contents taken from one fruit bat ( Rousettus leschenaultia) in Yunnan province, China. In the absence of targeted sequencing protocols for a novel virus, we applied the VIDISCR (Virus-Discovery-cDNA RAPD) virus discovery strategy to confirm and identify a novel Melaka-like reovirus, the \u201cCangyuan virus\u201d. To track virus evolution and to provide evidence of genetic reassortment PCR sequencing was conducted on each of the 10 genome segments, and phylogenetic analysis performed to determine genetic relatedness with other bat-borne fusogenic orthoreoviruses.","Another major paradigm for the high rates of negative-strand virus evolution is found with infl uenza virus. Due to its history and potential for initiating great human epidemics, it has long held the special interest of evolutionary virologists (see Chapter 5; Nelson and Holmes, 2007 ) . However, this research has not much emphasized the quasispecies character of infl uenza virus evolution. Instead, it concentrates on the evolution of the master template or clades of template for the purposes of vaccine development ( Webster and Govorkova, 2006 ) . The views stemming from this type of evolution have lent themselves well to master template-based phylogenetic analysis and have dominated how many researchers think of virus-host evolution. Thus it is curious, given the above emphasis, that the quasispecies character of infl uenza populations often seems of low relevance to issues of acute disease and vaccination, other then to provide a source of diversity. In some situations, viral competitive interference may contribute to drift variation and displacement in antigenic epitopes ( Levin et al. , 2004 ). Yet outcomes of individual human and bird infections do not seem much affected by specifi c quasispecies structures, as we saw with HIV-1 and HCV.","The genetic and antigenic evolution of error prone RNA respiratory viruses, particularly influenza, has been of interest for several decades [37, 38] . Understanding the selective pressure exerted by pre-existing immunity on viral evolution may help design more effective influenza vaccines and surveillance of animal populations can be critical for early identification of emerging influenza viruses [39] . Advances in deep sequencing make it possible to measure low frequency within host viral diversity and factors such as antigenic diversity, antiviral resistance, and tissue specificity can now be studied to understand the complexities of viral evolution [40] . Influenza evolution at a population level has been studied years, yet, new antigenic variants are initially generated and selected at the level of the individual infected host. Within a host, influenza viruses exist as a \"swarm\" of genetically distinct viruses [41] . Sanger sequencing defines consensus sequences and cannot resolve minority variants below 20% of the viral population. Deep sequencing has been used in natural infection and human challenge studies to characterize between and within host genetic diversity [41, 42] . The identification of low frequency mutations in the hemagglutinin (HA) antigenic sites or near the receptor-binding domain in vaccinated and unvaccinated influenza infected persons highlight viral evolution within a host due to selective immune pressure [41] . Similarly, NGS can reveal the rapid evolution of drug resistant variants during therapy [43] . Using samples collected over time, the mutational spectrum of H3N2 influenza A virus in an immunocompromised child was delineated [44] . Individual resistance mutations appeared weeks before they became dominant, evolved independently on cocirculating lineages. The within host evolution of antiviral resistance reflected a combination of frequent mutation, natural selection, and a complex pattern of segment linkage and reassortment. Within host sequencing diversity has also been examined in an infant with severe combined immune deficiency with persistent RSV infection [45] . NGS was performed on 26 samples obtained before and after bone marrow transplantation. The viral population appeared to diversify after engraftment with most variation occurring in the attachment protein (G). In addition, minority viral populations with palivizumab resistance mutations emerged after its administration. Deep sequencing of HRV during human challenge studies has shown that HRV generates new variants rapidly during the course of infection with accumulation of changes in \"hot spots\" in the capsid, 2C, and 3C genes [46] .","collective. Since random, even lethal, errors and drift are inherent in a quasispecies, lethal valleys can be readily crossed by such variable genomes, allowing the master genome to adapt by natural selection to a new fitness peak. Thus, error-prone replication and the generation of mutant clouds allows for much better exploration of sequence space and eventual adaptability. When viruses are transmitted to new hosts, they can experience a genetic bottleneck since a relatively small number of viral genomes could be involved (aka low multiplicity passage). If this process is serially repeated, a phenomenon known as 'Muller's Ratchet' can result in lost competitiveness as the essentially clonal RNA virus accumulates deleterious mutations (sometimes measured as pfu/plaque). However, in lab studies, virus extinction from serial passage does not occur, presumably due to plaque selection for a restored phenotype. Even a single plaque is in reality a small population (due to nonideal particle/pfu ratios and ID 50 ). However, lost competitiveness with other viruses is seen with clonal laboratory passage. However, if a quasispecies population is passed, this generally results in increased competitive fitness. Such passage can produce a seemingly never-ending better version of the virus that outcompetes all prior versions of the same virus (although virion yields and absolute replication are not necessarily improved). This has been likened to the Red Queen hypothesis in that the viruses are evolving at high rates, simply to maintain their competitive position, so as not to be displaced as the dominant viral type. Virus-virus competition is thus a crucial selection.","Viruses that can cause repeated infection are typically characterized either by a failure to induce robust immunity or by significant antigenic diversity in the face of protective immune responses. Influenza viruses can reinfect hosts because the antigenic sites evolve and drift to avoid neutralization by prior immunity. Infection induces a strong homosubtypic neutralizing antibody response in healthy individuals that contributes to recovery and protection from repeat influenza virus infection with homologous virus or an antigenically similar virus (Wrammert et al., 2008) . Moreover, natural infection can lead to long-lasting immunity to the infecting virus. For example, when the influenza A H1N1 subtype reemerged in 1977, the most susceptible members of the population were those born after the time when similar H1N1 viruses had previously circulated, the 1950s (Shortridge et al., 1979) . However, because influenza viruses undergo antigenic drift and shift, the effective period of protection may last only until an antigenic variant emerges.","The epidemiology and molecular characterization of low and highly pathogenic avian influenza virus strains (LPAIV &amp; HPAIV, respectively) isolated from Germany were investigated. The complete genome analysis of the two strains showed that both LPAIV and HPAIV had high nucleotide similarity with only ten mutations outside the hemagglutinin cleavage site (HACS) which were spread along the six genome segments of the HPAIV. Of the ten mutations, five were previously identified as minor variants in the quasispecies population of the progenitor virus, LPAIV, with 18\u201342% significant variable frequency [144]. However, studies focusing on the diversity of quasispecies of avian influenza in the human host are few. Watanabe et al. successfully demonstrated that infections caused by a single-virus in vitro produced an evident spectra of mutants in the H5N1 progeny viruses. Analysis of the genetic diversity of the hemagglutinin (HA) revealed that variants with mutated HA had lower thermostability leading to higher binding specificity. Both traits were deemed beneficial for viral infection. On the other hand, other variants with higher thermostability also emerged but were unable to thrive against mutants with lower thermostability [145]. The quasispecies population of influenza A virus was also reported to be in a state of continuous genetic drift in a given subtype population. A viral single nucleotide polymorphism (vSNP) was reported to be important and was shared by more than 15% of the variants within the quasispecies population of the subtype strain in a given season. However, between the season 2010\u20132011, various vSNPs in the PB2, PA, HA, NP, NA, M and NS segments were shared among variants with more than 58\u201380% of the sample population and less than 50% of the shared vSNPs were located within the PB1 segment [146].","Influenza viruses can change in two ways. One way of changing is called antigenic drift, which is a gradual process whereby the virus adaptively mutates to increase the capability of binding to human cells during subsequent human infections. These small, but important, mutations in the surface proteins of the influenza virus require the flu vaccine to be re-formulated on an annual basis. In a typical year, seasonal influenza causes widespread illness and approximately 30,000 deaths in the United States alone. These seasonal influenza epidemics are caused by slight mutations in the influenza virus. During these outbreaks, people may have some residual immunity from exposure to previously circulating influenza strains or cross-subtype immunity from seasonal influenza vaccinations. 6 Another way the influenza virus can change is called antigenic shift or reassortment, which is an abrupt, major change in the influenza virus. Dramatic changes in the surface proteins of the influenza virus, through mutation of nonhuman (e.g., avian or swine) viruses or reassortment of human and nonhuman viruses, result in the creation of a new human subtype or a protein combination that has not been seen in humans for many years. Pandemic influenza can cause a public health crisis because most people would be immunologically na\u00efve to the new virus. 7 In order for a pandemic to occur, three elements must be present. First, an influenza virus must emerge from an animal reservoir and engage in a reassortment event with a human influenza virus. Genetic material can be exchanged between human and avian or swine viruses during coinfection of a human, bird, or pig. Second, the novel virus must actually make humans sick. Reassortment events occur frequently, yet most do not yield pathogenic viruses. Third, the virus must be fully transmissible from human to human-spread easily through coughing, sneezing, or a handshake. If all three elements are present, the public is at great risk in facing a new and potentially deadly virus that is easily transmissible.","In 2001, by sequence analysis of CCoVs detected in fecal samples collected from diarrheic dogs in the South of Italy, multiple nt substitutions accumulating over a fragment of the M gene were observed [28]. The point mutations affecting the M gene were later observed in the sequences of CCoVs detected in the faeces of two naturally infected pups during the latter stages of long-term viral shedding. These CCoVs are dissimilar from the 1971-like CCoVs, showing an evident genetic drift toward FCoVs type 2, and it was postulated that the two dogs might have been infected by a mixed population of genetically different CCoVs, or that the viruses detected in both pups were the result of mutation/recombination events [29].","EDICTS checks two kinds of information. One is the most recent medical treatment records stored in smart cards. The other is the current medical record that is to be stored into the card. In the Taiwanese national healthcare insurance IC (NHI-IC) card system (NHICS), each NHI-IC card carries the six most recent patient visits and medical treatment history. A card reader, associated with a PoP, reads the history data and sends it to EDICTS. EDICTS checks this data with CDC edicts. If there is a match, EDICTS writes a suspect case mark into the card and sends an alarm signal immediately and automatically to the PoP's user interface. Hence clinicians are advised to refer to the suspect case description for further details of the symptoms and signs of ENID. Meanwhile the abnormal case is uploaded to IDC via the network. CDC thus has fast first-hand responses and is to able to set in motion the standard operating procedures. CDC could use the reports from EDICTS to know the contact information of the suspect cases, and therefore control and track the distribution of the ENIDs. Some computerized surveillance schemes have been established to detect the outbreak of ENIDs worldwide. Their goals are to collect information, to identify the causative viral strains, to rapidly assess related morbidity and mortality, to predict the situation, and to redistribute certified information [7] [8] [9] [10] . EDICTS was designed neither to detect nor to predict the outbreak of infectious diseases, though such extension is possible. EDICTS could be regarded as the mechanism to invoke after CDC has analyzed the information and determines that an emerging/notifiable disease outbreak is possible.","The influenza viruses are RNA viruses of the family Orthomyxoviridae. Influenzavirus A, Influenzavirus B, and Influenzavirus C are the genera that affect humans [18] . Influenzavirus A has one species, influenza A virus. The type A viruses are the most virulent human pathogens among the 3 influenza types and cause the most severe disease [19] . The influenza A virus has different serotypes based on the antibody response to these viruses [20] . Influenzavirus B also has only one species. The virus almost exclusively infects humans [20] , and is less common than influenza A. This virus mutates at a lower rate than type A and consequently is less genetically diverse, with only one influenza B serotype [20, 21] . This reduced rate of antigenic change, combined with its limited host range, ensures that pandemics of influenza B do not occur [18, 22] . Influenzavirus C has only one species. Influenza C virus causes mild disease in children [23, 24] . Indeed, seroepidemiological studies have revealed that influenza C virus is widely distributed globally. Nevertheless, because the isolation of this virus is difficult, there have been few reports on its clinical features [23] . Severe illness and local epidemics from influenza C have been reported [19, 23, 25] . Hay and colleagues [20] summarized the evolution of influenza viruses, which results in recurrent annual epidemics of disease that are caused by progressive antigenic drift of influenza A and B viruses due to the mutability of the RNA genome, and infrequent but severe pandemics caused by the emergence of novel influenza A subtypes to which the population has little immunity. The latter characteristic is a consequence of the wide antigenic diversity and peculiar host range of influenza A viruses, and the ability of their segmented RNA genomes to undergo frequent genetic reassortment (recombination) during mixed infections. Contrasting features of the evolution of recently circulating influenza AH1N1, AH3N2, and B viruses include the rapid drift of AH3N2 viruses as a single lineage, the slow replacement of successive antigenic variants of AH1N1 viruses, and the cocirculation over some 25 years of antigenically and genetically distinct lineages of influenza B viruses. Constant monitoring of changes in the circulating viruses is important for maintaining the efficacy of influenza vaccines in combating disease.","For example, measles virus shows much less antigenic drift in human infections compared to influenza virus A. Hepatitis G virus (a human prevalent and distant relative of HCV) shows little variation in even isolated human populations. The filoviruses (Ebola virus and Marburg virus) have shown no genetic variation in Zaire isolates from 20 years apart. Hendra virus, isolated from Australian fruit bats, and Nipah virus from Malaysian fruit bats also show little genetic diversity. Arenaviruses and hantavirus are also genetically stable in their natural rodent host. The reasons for such population stability have not been well evaluated. In some cases (measles), purifying selection would seem likely. In other cases, persistence and low replication rates seem to apply. For example, simian foamy virus (SFV) and human T-lymphotropic virus II (HTLV-II) generate only about 10 \u00c08 substitutions per site per year, probably due to low replication rates.","The diagnostic efficiency of nucleic acid detection (4) is constrained by following limitations: 1) high false negative rate owing to low virus load at early infection stage (5) or possible genetic mutations (6) ; 2) shortage of detection reagents; and 3) long waiting time. It is found that some early-onset NCP patients who had already presented abnormal chest CT findings still got negative results on the initial nucleic acid test. As a result, the category of clinically diagnosed NCP was added in the fifth version of diagnosis and treatment scheme released by Chinese National Health . CC-BY-NC-ND 4.0 International license It is made available under a is the author/funder, who has granted medRxiv a license to display the preprint in perpetuity.","At present there is scant evidence on the effectiveness of individual mitigation actions for COVID-19. As better epidemiology about COVID-19 and evidence on the effectiveness of different mitigation strategies become available, the COVID-19 Risk Assessment for MGs tool will be continuously refined to reflect changing knowledge. This rigorous process can inform risk assessment and decision making about MGs during the COVID-19 pandemic. Such MG risk assessments should be reviewed regularly during planning and updated immediately before the MG operational phase, especially in light of the evolving national and international epidemiological situations. There is no specific evidence base yet specific to planning and implementing a MG during the COVID-19 pandemic. Detection and monitoring of MGevent-related COVID-19 should be considered in the context of surveillance schemes that are already in place and if new or enhanced surveillance is deemed necessary. In collaboration with local health authorities, organisers should agree in advance the circumstances in which riskmitigation measures would need to be enhanced or the event postponed or cancelled.","The initial study of virus evolution sought to explain how virus variation affects viral and host survival and to understand viral disease. However, we now realize that virus evolution is a basic issue, impacting all life in some way. In general, the principles of virus evolution are very much the same Darwinian principles of evolution for all life, involving genetic variation, natural selection, and survival of favorable types. However, virus evolution also entails features such as high error rates, quasispecies populations, and genetic exchange across vast reticulated gene pools that extend the traditional concepts of evolution. Evolution simply means a noncyclic change in the genetic characteristics of a virus; and viruses are the most rapidly evolving genetic agents for all biological entities. Principles of virus evolution provide an integrating framework for understanding the diversity of viruses and the relationships with host as well as providing explanations for the emergence of new viral disease. Most emerging viral diseases are due to species jumps from persistently infected hosts that have long-term virus-host evolutionary histories (called natural reservoirs). Since early human populations (i.e., small bands of hunter gatherers) could not have supported the great human viral plagues of civilization (e.g., smallpox virus/variola and measles), these viruses must also have originated from species jumps that adapted to humans. In recent history, the emergence of HIV demonstrates that virus evolution continues to impact human populations. Early observation established that viruses often show significant variation in virulence. Such variation was used as an early, yet risky, form of vaccination (i.e., variolation against smallpox). The variation that occurred with passage into alternate tissue and host was also used to make various vaccines (rabies in 1880s) or attenuate viral virulence, such as live yellow fever vaccine. But variation also allows some viruses, such as influenza A, to escape neutralization by vaccination. However, variation in viral disease or virulence does not provide a quantitative basis to study evolution.","Viruses exist as quasispecies [32]. Infected humans and animals host a diverse population of viral nucleic acid sequences; the raw material for rapid evolution. RNA viruses, including retroviruses, have much higher mutation rates than DNA viruses [33,34], increasing the complexity of generating effective probes for nucleic acid amplification tests. This can result in assay \u2018drop out\u2019. This has been a problem with influenza A diagnostics targeting hemagglutinin (HA) and neuraminidase (NA) segments. It is exemplified by the necessity for ad hoc test development for the 2009 H1N1 human influenza of swine origin, the \u2018swine influenza virus real-time detection panel\u2019, and its authorization for emergency use by the US FDA. This test was subsequently replaced by an improved version developed by the CDC, as more H1N1 specimens became available [203]. The problem of drop out continues to plague commercial and in-house assays [35]. Whiley and colleagues cite several examples of PCR or reverse-transcriptase (RT)-PCR-based virus diagnostic assays that are prone to false-negative results caused by sequence variation. This list includes commercial assays for HIV-1, human parvovirus B19, and in-house methods for rabies and cytomegalovirus [35]. This calls for astute primer design or the use of multiple gene targets within a single organism (e.g., combining PCRs for the por A and ctr A genes in Neisseria meningitidis[36] or for the HA and NA genes for influenza A detection) [37]. The challenge to the designer of an influenza diagnostic test is the same as the challenge faced by the designer of influenza vaccines, where the vaccine needs to be reformulated annually to account for antigenic drift and also when antigenic shift is detected.","HMPV is another recently identified respiratory virus 27 and has been found worldwide. 28 It is grouped into four distinct genetic lineages based on the F gene: A1, A2, B1, and B2. 29 In this study, two strains of HMPV-A1, 11 strains of HMPV-B1 and two strains of HMPV-B2 were detected in the first season, whereas one strain of HMPV-A1, three strains of HMPV-B1 and 12 strains of HMPV-B2 were identified in the second season. No HMPV-A2 lineage was found. Hence, a change of predominant lineage in the seasons was observed, but no association between the severity of infection and genetic drift of HMPV was found, as shown in other previous studies. 30, 31 Besides, studies showed more sequence diversity in G and SH genes but not in F gene, 30 which could explain the constant incidence of HMPV infection in the population. HCoV-HKU1 (in group II with HCoV-OC43) and HCoV-NL63 (in group I with HCoV-229E) are two novel coronaviruses. 32, 33 During the present 2-year study, HCoV-NL63 and HCoV-229E were the major HCoV circulating in Shanghai, whereas only one strain of HCoV-OC43 and one of HCoV-HKU1 were detected, indicating a sporadic introduction of group II HCoV to the region. In context that the recent emerged HCoV, like HKU-1 and SARS whose sequence is more homologous to group II virus, could cause severe respiratory infection, the surveillance for emergence of new species of HCoV is necessary.","To contain the virus, many countries adopted dramatic measures to reduce human interaction, including enforcing strict quarantine, prohibiting large-scale private and public gatherings, restricting private and public transportations, encouraging social-distancing, imposing a curfew, and even locking down an entire city.","Yearly epidemics of influenza are caused by influenza A and B viruses with mutations in the regions of the HA and NA genes that encode antigenicity, allowing them to escape the hosts' immunity against parent strains (antigenic drift).","The US Centers for Disease and Control Prevention (CDC) specifies the following two key ventilation design requirements for dedicated isolation facilities: e increased dilution efficiency through higher air ventilation rates; e the avoidance of air flow from less clean to clean areas, by creating a pressure gradient and providing an air-locking ante room. 6 No such requirements are stipulated for general wards. Most hospital general wards have corridor return airflow without air return duct design, which has the effect of potentially driving air from less clean to clean areas.","The tree may be characterized by topology, length of branches, shape, and the position of the root. The topology is determined by relative position of internal and terminal nodes; it defines branching events leading to contemporary species diversity. If two or more trees obtained for different data sets feature a common topology, these trees are called congruent. The branch length of a tree may define either the amount of change fixed or the time passed between two nodes connected in a tree, and is known as 'additive' or 'ultrametric', respectively. The tree shape may be linked to particulars of evolutionary process and reflect changes in the population size and diversity due to genetic drift and natural selection. The position of the root at the tree defines the direction of evolution. Species that descend from an internal node in a rooted tree form a cluster and the node is called most recent common ancestor (MRCA) of the cluster that thus has a monophyletic origin. The branch lengths and the root position may be left undefined for a tree that is then called 'cladogram' and 'unrooted tree', respectively.","Non-pharmaceutical interventions to interrupt transmission could be implemented immediately, gaining time for pharmaceutical development. Briefly, there were three steps of non-pharmaceutical interventions for reducing contact rates between susceptible individuals and infected individuals. First, quarantine management, i.e. quarantining the infected, the suspicious and their close contacts; second, social distancing to confine within-population contact; third, locking down the epicenter to prevent further exportation of infected and latently infected individuals to other regions.","Continuing challenges in influenza include the sporadic human cases of highly pathogenic avian H5N1 influenza, emergence of pandemic H1N1 influenza in 2009 [62, 69] , and human infections with avian H7N9 influenza in 2013 [11] . Influenza A virus undergoes continuous antigenic drift and sporadic antigenic shifts in the viral surface glycoproteins, hemagglutinin (H) and neuraminidase (N). Influenza A has 15 H and 9 N subtypes. Antigenic H and N subtypes to which humans lack immunity are introduced by reassortment of virus genes and cause pandemics, whereas H and N antigenic variants determined by point mutations cause seasonal influenza epidemics [66, 67] . Molecular assays are the preferred method for identification and surveillance of new strains of influenza A infections [11, 62, 67] . Influenza A has no pathognomonic symptoms, and diagnosis based on clinical signs is correct in only two-thirds of patients [68, 70] . Therefore, sensitive and rapid laboratory tests are required to diagnose and guide antiviral treatment. Recently, multiplex molecular assays for respiratory viruses including influenza viruses have been developed, and several have received approval/clearance by the US Food and Drug Administration (FDA) for routine clinical use (Table 15 .2, listed under pandemic H1N1 influenza virus). These assays provide rapid and sensitive tests for respiratory viral infections.","In this study, we used a network-based approach to analyze the global SARS-CoV-2 transmission patterns by constructing the transmission network based on genomic compositions of viral sequences sampled around the world and before the pandemic declaration on March 11, 2020 . Here we sought to characterize the transmission pathways that facilitated the virus to establish itself as a pandemic. Such analyses rely on viral genomic data that continues to accumulate in near real time as next-generation sequencing technologies are now more widely used. The richness and accessibility of the genomic data distinguishes this SARS-CoV-2 pandemic from previous large-scale epidemics or pandemics including the 2009/AH1N1 influenza pandemic and the 2003 SARS outbreaks. This provides an unprecedented opportunity to gain insights into the epidemiological and evolutionary dynamics of this emerging virus spreading in an essentially naive population. Another feature of the scope of the genomic data for COVID-19 is the high sampling density available to investigate the early transmission stages. Indeed, although the virus genetic diversity gradually increases as the virus spreads, particular genomic variants have been repeatedly sequenced at different time points and geographical locations. This indicates that the available sequencing data cover a significant part of the evolutionary space explored from the onset of the epidemics. In this setting, viral evolution could be accurately modeled, reconstructed and visualized by genetic networks models [8, 52] , which have been successfully used for the analysis of different viral epidemics [52, 62, 9] . However, in emerging outbreak settings, when all circulating viral genomes are relatively close to each other, methods that allow the analysis of viral population structures at finer resolution than provided by state-of-the-art network models are needed. We achieve such resolution by using mutation trees [34] (related to character-based phylogenies [24] ) that keep track of the accumulation of mutations in viral populations. Moreover, as the SARS-CoV-2 population contain relatively few 4-gamete rule violations, it is feasible to efficiently generate and analyze all plausible mutation trees. These trees, in turn, allow for accurate inferences of transmission networks.","Clearly various families of viruses, with different structures and replication schemes, and consequently bearing different potential molecular targets, are involved in respiratory symptoms, and many of them are susceptible to Echinacea extracts, as indicated in Table 2. Among the possible viral targets are: (i) the virion itself (membrane components); (ii) cellular attachment or entry; (iii) one or more of the many stages in virus replication and development, particularly those that involve virus-specific enzymes; (iv) egress of progeny virus from infected cells. However, because of the variety of replication schemes among these viruses the chances of success for a single therapeutic drug are low, especially considering that in the majority of respiratory infections specific virus information is lacking. ","The simple enzyme composition is another reason to enable one-step and one-pot 311 detection. The fewer the enzymes, the easier it is to optimize in terms of temperature and 312 buffer composition. In designing the detection scheme, we deliberately tried to reduce the 313 number of enzymes, thus creating one of the simplest isothermal detection schemes based on 314 two enzymes: SplintR ligase for target detection and T7 RNA polymerase for amplification. 315","Influenza viruses have been associated with annual epidemics and intermittent pandemics throughout the world. Despite the absence of a winter season in the tropics, consistent seasons of infection have nevertheless been observed. The composition of the antigenic surface glycoproteins of the influenza virus, hemagglutinin (H) and neuraminidase (N) are used for subtyping, resulting in names like H3N2 and H1N1. Antigenic drift represents the minor changes of H and N side chains and is responsible for seasonal epidemics. Influenza pandemics occur less frequently and these result from major changes in antigenic structure in the envelope glycoproteins (antigenic shift) resulting from reassortment of various viruses such as swine, equine and human varieties. Contemporary geographic distributions show that East, South and Southeast Asia influence unduly the evolution of seasonal influenza A (H3N2), exporting most of the evolutionarily strains that ultimately spread globally. The obvious role of Asia in H3N2's evolution has been ascribed to the seasonal nature of influenza in temperate climates [7] .","6-8-week-old female C57bl/6J mice were derived from a locally bred colony maintained in a pathogen-free facility, in filter-topped cages, and maintained in accordance with local and governmental regulations. To prevent genetic drift, mice have been rederived using externally supplied mice (Charles River).","Previous workers have attempted to predict the cumulative number of cases of Coronavirus Disease 2019 in China [1] . However, since then, the epidemic has rapidly evolved into a pandemic affecting multiple countries worlwide [2] . There have been serious debates about how to react to the spread of this disease, particularly by European countries, such as Italy, Spain, Germany, France and the UK, e.g. from closing schools and universities to locking down entire cities and countries. An alternative strategy would be to allow the causal virus (SARS-CoV-2) to spread to increase the population herd immunity, but at the same time protecting the elderly and those with multiple comorbidities, who are the most vulnerable to this virus [3] .","China took the extraordinary measure of locking down Wuhan and other cities on January 23 rd 2020, thereby reducing travel out of the disease epicentre. Given the timing of this epidemic around the Chinese Lunar New Year (Spring festival), when travel is at a peak, this would reduce the risk of travel-related importations of cases to other parts of China and the world.","While the costs of enforcing these preventative measures are undoubtedly enormous, there are also speculations that these measures could unintendedly bring about substantial social benefits. Among them, some argue that locking down cities could have significantly improved environmental quality, which would be beneficial to ordinary people and partially offset the costs of these counter-COVID-19 measures. For example, satellite images caught a sharp drop in air pollution in several counties that have taken aggressive measures on the transmission of the virus. 2","We argue that, though simplistic, our model demonstrates the urgent need to focus control efforts in the U.S. on preventing further domestic virus spread. In China, local outbreak dynamics were highly correlated with travel between Wuhan and the outbreak dynamics therein during the early months of the epidemic . Similarly, if interstate introductions are not curtailed in the U.S. with improved surveillance measures, more robust diagnostic capabilities, and proper clinical care, quelling local transmission within states will be a Sisyphean task. We therefore propose that a unified effort to detect and prevent new COVID-19 cases will be essential for mitigating the risk of future domestic outbreaks. This effort must ensure that states have sufficient personal protective equipment, sample collection materials, and testing reagents, as these supplies enable effective surveillance. Finally, stateand local-level policymakers must recognize that the health and well-being of their constituents are contingent on that of the nation. If spread between states is now common, as our results indicate, the U.S. will struggle to control COVID-19 in the absence of a unified surveillance strategy.","As we know, different countries have taken different approaches to try and keep the pandemic at bay; with China going on complete lockdown, the UK only going on semi-lockdown with people still going to work, and Sweden not taking any particular approach to locking down, the most efficient method is unclear. It is hard for everyone to attain the necessary information, and the information is usually rather difficult for non-experts to understand.","Locking down Wuhan, a city of 11 million residents, was an unprecedented measure to contain the spread of the Novel Coronavirus. An important policy-relevant question is, then, how many COVID-19 cases were actually prevented by the Wuhan lockdown in China? To answer this question, we must estimate the counterfactual number of COVID-19 cases that would have occurred in other cities in the absence of Wuhan lockdown, which would, in turn,","Many countries have made substantial progress in developing effective levels of disease detection, which involves strengthening surveillance and laboratory capacities. The application of lessons learned from previous infectious disease emergencies including the 2002 SARS-CoV, 2009 H1N1 influenza pandemic, MERS-CoV, Ebola virus, and Zika virus outbreaks, have helped to strengthen countries' capacities to effectively detect and verify suspect cases. 5, 8, 28, 29 The early detection of COVID-19 in China and the development of laboratory reagents for testing and genetically sequencing the novel virus are key steps that have supported the early response. 29 We showed that 76% of countries have robust detection capacities in place, which will enable early detection and verification of potential outbreaks when they occur.","Acute respiratory infections in humans are usually ascribed to one or more of a group of well known viruses, including more than 100 rhinoviruses (\u201ccommon cold\u201d viruses), influenza viruses A and B, parainfluenza viruses, corona viruses, respiratory syncytial virus, and certain adenoviruses [1-4]. In addition the recent application of more sensitive molecular detection techniques has revealed the presence of other viruses, such as metapneumoviruses and bocaviruses, which might also be involved in the generation of respiratory symptoms. However we do not know if these newly recognized viruses are really pathogenic, or are simply \u201cpassengers\u201d that eluded previous diagnostic techniques. Nevertheless various families of viruses, with different structures and replication schemes, and consequently bearing different potential molecular targets, are clearly involved in respiratory symptoms, as indicated in Table 1. Among the possible targets are: (i) the virion itself; (ii) cellular attachment or entry; (iii) one or more of the many stages in virus replication and development, particularly those that involve virus-specific enzymes; (iv) egress of progeny virus from infected cells. However the variety of replication schemes indicated in Table 1 reduces the chances that a single antiviral drug could target many of these viruses [3,5]. In addition, in the majority of respiratory infections specific virus information is lacking; consequently it is difficult to conceive of a single therapeutic agent or regimen that could control the \u201ccausative agent\u201d. Nevertheless, in spite of this limitation, considerable time and money has been spent trying to find the \u201csilver bullet\u201d for specific virus \u201cinfections\u201d, so far without much success [6].","The KSA-MoH has implemented a pro-active early case detection and surveillance system for MERS-CoV. It recommends sending respiratory tract samples from critically ill patients admitted to hospitals with fever and lower respiratory tract infection symptoms. Screening for MERS-CoV is also recommended for family and healthcare worker (HCW) contacts of proven cases of MERS-CoV infection. All samples are transported to and are processed by the KSA-MoH virology laboratory in Jeddah, which is accredited and regulated by the Central Board for Accreditation of Health Care Institute. Quality assurance and control for all diagnostic tests is monitored through Internal Policy Procedures and by external quality assurance schemes.","In conclusion, it still remains a challenging task to fight the 2019-nCoV of unknown origin and mysterious biological features, and to control an outbreak of COVID-19 with such a high R0, a long incubation period and a short serial interval, by limited treatment and prevention measures. Lessons learned from the MERS and SARS outbreaks can provide valuable insight into how to handle the current pandemic. The successful public health outbreak response tactics of Chinese government, such as hand hygiene, wearing masks, isolation, quarantine, social distancing, and community containment, can be copied by other countries according to their national situation. As the pandemic is still ongoing and expanding, experiences and research literatures from China and other countries will increase. The 2019-nCoV should be monitored of any possible gene variation of antigenic drift or antigenic conversion, to avoid another round of outbreak. Another lessons from this pandemic will be awe for nature and love for life. Funding Source: This research did not receive any specific grant from funding agencies in the public, commercial, or not-for-profit sectors.","Eukaryotic cells possess a variety of membrane-bound organelles. The assembly of these specialized structures requires the transport of specific sets of proteins to particular cellular locations. Transport events include: the insertion of newly synthesized proteins into the nucleus, the mitochondria, the plasma membrane, and the iysosome (in yeast, the vacuole); secretion of proteins into the extracellular space; and the endocytic retrieval of surface proteins and external compounds. While cells in the specialized organs of higher eukaryotes may use variations of the basic transport schemes, the transport of specific proteins to their target organelles is probably accomplished by similar mechanisms in all eukaryotes. Yeast offers an ease of genetic manipulation unequalled by other eukaryotes. This article will summarize what has been learned through the isolation of yeast mutants defective in various aspects of intracellular transport. Nuclear and mitochondrial transport will not be covered.","Apart from different types of the animal hosts, three major factors on the viral side are also important in facilitating CoVs to cross species barriers 70. First of all, their relatively high mutation rates in RNA replication. In comparison to other single-stranded RNA viruses, the estimated mutation rates of CoVs could be regarded as \u201cmoderate\u201d to \u201chigh\u201d with an average substitution rate being ~10-4 substitution per year per site 2, depending on the phase of CoV adaptation to novel hosts. CoVs have a proof-reading exoribonuclease, deletion of which results in exceedingly high mutability and attenuation or even inviability. Interestingly, the nucleotide analogue Remdesivir is known to suppress CoV replication through inhibition of this exoribonuclease and the RNA-dependent RNA polymerase 70. Remdesivir is one of most promising anti-SARS-CoV-2 agents to be tested in clinical trials. Nevertheless, mutation rates of CoVs are about a million times higher than those of their hosts. In addition, mutation rate is often high when CoVs are not well adapted to the host 71. Compared to SARS-CoV with a high mutation rate 72, the mutation rate of SARS-CoV-2 is apparently lower, suggestive of a higher level of adaptation to humans. Presumably, it has already been adapted to another host close to humans. In addition to SARS-CoV-2, this also applies to MERS-CoV, which is well adapted to dromedary camels. Theoretically, it is unlikely that genetic drift would render vaccines and antivirals against SARS-CoV-2 ineffective quickly. Second, the large RNA genome in CoVs exerts extra plasticity in genome modification for mutations and recombination, thereby increasing the probability for interspecies co-evolution, which is advantageous for the emergence of novel CoVs when the conditions become appropriate. This is supported by the copious unique open reading frames and protein functions encoded towards the 3\u2032 end of the genome. Third, CoVs randomly and frequently switch templates during RNA replication through a unique \u201ccopy-choice\u201d mechanism. In a host that serves as the mixing vessel, strand switching occurs frequently during CoV RNA transcription. Highly homologous full-length and subgenomic RNAs could recombine to generate new CoVs. Phylogenetic evidence of natural recombination has been found in both HCoV-HKU1 and HCoV-OC43, as well as animal CoVs such as bat SL-CoV and batCoV-HKU9 73.","On the contrary, China is running a national level antivirus campaign. All of the 31 provinces and province-level municipalities in mainland China have initiated the first-level response to public health emergencies after the city closure of Wuhan on Jan. 23, 2020. Very strict control measures are enforced all over the country, such as tourism ban, school closure, meeting cancelation, etc. Compared with the prevention activities against SARS 17 years ago, traffic control and community locking are much more stricter, let alone the unprecedented province lockdown that constraining over 58 million people locally [6] ."],"title":["Mutations, Recombination and Insertion in the Evolution of 2019-nCoV","Optimizing diagnostic strategy for novel coronavirus pneumonia, a multi-center study in Eastern China","Comparative genomic analysis revealed specific mutation pattern between human coronavirus SARS-CoV-2 and Bat-SARSr-CoV RaTG13","CHAPTER 2 Cultivation, Assay, and Analysis of Viruses","Human genomics and preparedness for infectious threats","Current research on respiratory viral infections: Fifth International Symposium","Emerging viral infections","The common cold: a review of the literature","Impact of RNA Virus Evolution on Quasispecies Formation and Virulence","Nanopore target sequencing for accurate and comprehensive detection of SARS-CoV-2 and other respiratory viruses","Characterization of a novel orthoreovirus isolated from fruit bat, China","CHAPTER 21 The Widespread Evolutionary Significance of Viruses","Chapter 9 Respiratory viral infections","Evolution of Viruses","Chapter 59 Respiratory Virus Vaccines","Impact of RNA Virus Evolution on Quasispecies Formation and Virulence","Pandemic Influenza: Impact on Perianesthesia Nursing Areas","The Evolutionary Processes of Canine Coronaviruses","Design and prototype of a mechanism for active on-line emerging/notifiable infectious diseases control, tracking and surveillance, based on a national healthcare card system","Severe Childhood Respiratory Viral Infections","Evolution of Viruses","Improved deep learning model for differentiating novel coronavirus pneumonia and influenza pneumonia","Mass gathering events and reducing further global spread of COVID-19: a political and public health dilemma","Evolution of Viruses","Expecting the unexpected: nucleic acid-based diagnosis and discovery of emerging viruses","Molecular monitoring of causative viruses in child acute respiratory infection in endemo-epidemic situations in Shanghai","COVID-19, City Lockdown, and Air Pollution: Evidence from China","33 Viral Respiratory Infections","Rethinking hospital general ward ventilation design using computational fluid dynamics","Phylogeny of Viruses","The impact of social distancing and epicenter lockdown on the COVID-19 epidemic in mainland China: A data-driven SEIQR model study","Chapter 15 Molecular Testing in Emerging Infectious Diseases","Global transmission network of SARS-CoV-2: from outbreak to pandemic","Applications of the Phytomedicine Echinacea purpurea (Purple Coneflower) in Infectious Diseases","Sensitive one-step isothermal detection of pathogen-derived RNAs","Pneumonia in the tropics: Report from the Task Force on tropical diseases by the World Federation of Societies of Intensive and Critical Care Medicine","A novel antiviral formulation inhibits a range of enveloped viruses.","Herd immunity \u2013 estimating the level required to halt the COVID-19 epidemics in affected countries","Wuhan novel coronavirus 2019nCoV","COVID-19, City Lockdown, and Air Pollution: Evidence from China","Coast-to-coast spread of SARS-CoV-2 in the United States revealed by genomic epidemiology","COVID-19 Modelling: the Effects of Social Distancing","Human Mobility Restrictions and the Spread of the Novel Coronavirus (2019-nCoV) in China","Health security capacities in the context of COVID-19 outbreak: an analysis of International Health Regulations annual report data from 182 countries","Echinacea\u2014A Source of Potent Antivirals for Respiratory Virus Infections","Screening for Middle East respiratory syndrome coronavirus infection in hospital patients and their healthcare worker and family contacts: a prospective descriptive study","Insight into 2019 novel coronavirus \u2014 an updated intrim review and lessons from SARS-CoV and MERS-CoV","Revelation through exploitation: the viral model for intracellular traffic","Zoonotic origins of human coronaviruses","Risk map of the novel coronavirus (2019-nCoV) in China: proportionate control is needed"],"url":["https://doi.org/10.1101/2020.02.29.971101","https://doi.org/10.1101/2020.02.13.20022673","https://doi.org/10.1101/2020.02.27.969006","https://doi.org/10.1016/b978-0-12-253040-1.50007-5","https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2808735/","https://doi.org/10.1016/j.antiviral.2003.12.006","https://doi.org/10.1016/j.cll.2004.05.002","https://doi.org/10.1016/j.ejim.2004.01.006","https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6770471/","https://doi.org/10.1101/2020.03.04.20029538","https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4264558/","https://doi.org/10.1016/b978-0-12-374153-0.00021-7","https://doi.org/10.1016/b978-0-12-801496-7.00009-5","https://doi.org/10.1016/b978-012374410-4.00706-8","https://doi.org/10.1016/b978-0-12-415847-4.00059-8","https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6770471/","https://doi.org/10.1016/j.jopan.2009.04.002","https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3265307/","https://doi.org/10.1016/j.cmpb.2007.02.005","https://doi.org/10.1016/j.yapd.2009.08.019","https://doi.org/10.1016/b978-012374410-4.00706-8","https://doi.org/10.1101/2020.03.24.20043117","https://doi.org/10.1016/s0140-6736(20)30681-4","https://doi.org/10.1016/b978-012374410-4.00706-8","https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7103685/","https://doi.org/10.1016/j.jcv.2010.08.005","https://doi.org/10.1101/2020.03.29.20046649","https://doi.org/10.1016/b978-0-323-55512-8.00033-8","https://doi.org/10.1016/j.jhin.2010.08.010","https://doi.org/10.1016/b978-012374410-4.00712-3","https://doi.org/10.1101/2020.03.04.20031187","https://doi.org/10.1016/b978-0-12-800886-7.00015-7","https://doi.org/10.1101/2020.03.22.20041145","https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3205674/","https://doi.org/10.1101/2020.03.05.20031971","https://doi.org/10.1016/j.jcrc.2017.11.004","https://doi.org/10.1101/2020.03.29.009464","https://doi.org/10.1016/j.jinf.2020.03.027","https://doi.org/10.31646/gbio.50","https://doi.org/10.1101/2020.03.29.20046649","https://doi.org/10.1101/2020.03.25.20043828","https://doi.org/10.1101/2020.03.29.20046870","https://doi.org/10.1101/2020.03.24.20042424","https://doi.org/10.1016/s0140-6736(20)30553-5","https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4058675/","https://doi.org/10.1111/1469-0691.12562","https://doi.org/10.1016/j.ijid.2020.03.071","https://doi.org/10.1016/0968-0004(85)90024-6","https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7098031/","https://doi.org/10.1101/2020.02.16.20023838"],"x":[0.43482209017219,0.36127297790557755,0.45637208429003495,0.06318997543467564,0.7520866812353981,0.0449603836607364,0.02957454063842091,0.7434532654253501,0.9376664244531229,0.13114438011090257,0.22726857172005088,0.9447811705161321,0.4867960661646531,0.26723667524171035,0.7646664374552737,0.12814054870568425,0.06816201545877665,0.4692581986013252,0.3415413784271857,0.10490946836818738,0.9686806142702006,0.7464808335813956,0.8227942377615004,0.5256291848228055,0.026378436066026856,0.04861965080370123,0.46460301267299464,0.3785723543008286,0.027990099377845357,0.7326335226030274,0.2923227820116907,0.6097895872611149,0.02260071209139658,0.498170827682602,0.15837670681432414,0.46462389680668814,0.25962329139701756,0.24670927355628924,0.8449444684172441,0.7924162205655683,0.032931150207961335,0.8474275466117145,0.3062061340586607,0.9817906726482439,0.6938622045104935,0.24265580459437308,0.8296189948725574,0.5218628105392413,0.6515355881210121,0.26756878613504653],"y":[0.26258622048417113,0.1662402388534957,0.4684139787261683,0.49336703825482997,0.500361065057841,0.654680070945067,0.3562758914547475,0.44482188919890864,0.22443641835139694,0.12758034613978053,0.475356828898192,0.3435291558143372,0.40487677736106265,0.14984319614281183,0.7536533467854336,0.9390008914307887,0.8858212015097295,0.09350682069809191,0.7011653626481054,0.2475338765405204,0.14208357721030007,0.22635892266722824,0.8691028969150342,0.6015299732619466,0.7621150965283744,0.8062079214439861,0.3519099168283144,0.6227224963512442,0.6670231375083563,0.014418623272964393,0.35058017262292474,0.998231406791236,0.8526767916150387,0.12729299873536282,0.7160983766153668,0.5100575983041202,0.1656978532654656,0.6822325789588862,0.5713056393610934,0.3381117028002284,0.0630087278881789,0.8171615513208801,0.14837508273895084,0.8246917527489063,0.4121228133150774,0.7081973057097579,0.9453812063803546,0.9387793116496695,0.8954749117873958,0.9254269448445158]},"selected":{"id":"24581"},"selection_policy":{"id":"24606"}},"id":"24508","type":"ColumnDataSource"},{"attributes":{"data_source":{"id":"24507"},"glyph":{"id":"24534"},"hover_glyph":null,"muted_glyph":null,"nonselection_glyph":{"id":"24535"},"selection_glyph":null,"view":{"id":"24537"}},"id":"24536","type":"GlyphRenderer"},{"attributes":{},"id":"24516","type":"LinearScale"},{"attributes":{"callback":null,"tooltips":[["Title","@title"],["Abstract","@text"],["Published","@date"],["Authors","@authors"],["Journal","@journal"],["URL","@url"]]},"id":"24528","type":"HoverTool"},{"attributes":{"axis":{"id":"24522"},"dimension":1,"ticker":null},"id":"24525","type":"Grid"},{"attributes":{},"id":"24545","type":"Range1d"},{"attributes":{"sizing_mode":"stretch_width","text":"&lt;h2&gt;Efforts to track the evolution of the virus (i.e., genetic drift or mutations) and avoid locking into specific reagents and surveillance/detection schemes.&lt;/h2&gt;"},"id":"24583","type":"Div"},{"attributes":{},"id":"24605","type":"BasicTickFormatter"},{"attributes":{"source":{"id":"24508"}},"id":"24572","type":"CDSView"},{"attributes":{"text":"&lt;h3&gt;Document ranking&lt;/h3&gt;"},"id":"24584","type":"Div"},{"attributes":{},"id":"24527","type":"ResetTool"},{"attributes":{"active_drag":"auto","active_inspect":"auto","active_multi":null,"active_scroll":"auto","active_tap":"auto","tools":[{"id":"24526"},{"id":"24527"},{"id":"24528"}]},"id":"24529","type":"Toolbar"},{"attributes":{},"id":"24601","type":"StringFormatter"},{"attributes":{},"id":"24558","type":"BasicTicker"},{"attributes":{"active_drag":"auto","active_inspect":"auto","active_multi":null,"active_scroll":"auto","active_tap":"auto","tools":[{"id":"24561"},{"id":"24562"},{"id":"24563"}]},"id":"24564","type":"Toolbar"},{"attributes":{},"id":"24603","type":"BasicTickFormatter"},{"attributes":{"formatter":{"id":"24605"},"ticker":{"id":"24519"}},"id":"24518","type":"LinearAxis"},{"attributes":{"formatter":{"id":"24613"},"ticker":{"id":"24558"}},"id":"24557","type":"LinearAxis"},{"attributes":{"children":[{"id":"24586"},{"id":"24587"},{"id":"24588"},{"id":"24589"},{"id":"24590"}]},"id":"24591","type":"Column"},{"attributes":{},"id":"24608","type":"StringEditor"},{"attributes":{},"id":"24523","type":"BasicTicker"},{"attributes":{"data":{"authors":["Reina, J.","Phan, Tung","Li Xiaoxuan; Wu Qi; Lv Benfu","Arunachalam Ramaiah; Vaithilingaraja Arumugaswami","Villarreal, L.P.","Anna S Fomsgaard; Maiken W Rosenstierne","Idan Yelin; Noga Aharony; Einat Shaer-Tamar; Amir Argoetti; Esther Messer; Dina Berenbaum; Einat Shafran; Areen Kuzli; Nagam Gandali; Tamar Hashimshony; Yael Mandel-Gutfreund; Michael Halberthal; Yuval Geffen; Moran Szwarcwort-Cohen; Roy Kishony","Braden, Christopher R.; Dowell, Scott F.; Jernigan, Daniel B.; Hughes, James M.","Jimenez-Guarde\u00f1o, Jose M.; Regla-Nava, Jose A.; Nieto-Torres, Jose L.; DeDiego, Marta L.; Casta\u00f1o-Rodriguez, Carlos; Fernandez-Delgado, Raul; Perlman, Stanley; Enjuanes, Luis","Benito-Pe\u00f1a, Elena; Vald\u00e9s, Mayra Granda; Glahn-Mart\u00ednez, Bettina; Moreno-Bondi, Maria C.","Ghosh, Arun K.; Brindisi, Margherita","Mandary, Madiiha Bibi; Masomian, Malihe; Poh, Chit Laa","Livio Fenga","Douglas, Madeline G.; Kocher, Jacob F.; Scobey, Trevor; Baric, Ralph S.; Cockrell, Adam S.","Aiping Wu; Peihua Niu; Lulan Wang; Hangyu Zhou; Xiang Zhao; Wenling Wang; Jingfeng Wang; Chengyang Ji; Xiao Ding; Xianyue Wang; Roujian Lu; Sarah Gold; Saba Aliyari; Shilei Zhang; Ellee Vikram; Angela Zou; Emily Lenh; Janet Chen; Fei Ye; Na Han; Yousong Peng; Haitao Guo; Guizhen Wu; Taijiao Jiang; Wenjie Tan; Genhong Cheng","tianyi qiu; Han Xiao","Earp, Laurie J; Hernandez, Lorraine D; Delos, Sue E; White, Judith M","Min Zhou; Yong Chen; Dexiang Wang; Yanping Xu; Weiwu Yao; Jingwen Huang; Xiaoyan Jin; Zilai Pan; Jingwen Tan; Lan Wang; Yihan Xia; Longkuan Zou; Xin Xu; Jingqi Wei; Mingxin Guan; Jianxing Feng; Huan Zhang; Jieming Qu","Paterson, Beverley J.; Durrheim, David N.","Yujuan Xiong; Zhen-Zhen Li; Qizhen Zhuang; Yan Chao; Fei Li; Yi-Yuan Ge; Yi Wang; Pei-Feng Ke; Xianzhang Huang","Ying Diao; Xiaoyun Liu; Tao Wang; Xiaofei Zeng; Chen Dong; Changlong Zhou; Yuanming Zhang; Xuan She; Dingfu Liu; Zhongli Hu","Nieto-Torres, Jose L.; DeDiego, Marta L.; Verdi\u00e1-B\u00e1guena, Carmina; Jimenez-Guarde\u00f1o, Jose M.; Regla-Nava, Jose A.; Fernandez-Delgado, Raul; Casta\u00f1o-Rodriguez, Carlos; Alcaraz, Antonio; Torres, Jaume; Aguilella, Vicente M.; Enjuanes, Luis","S. Banach, Bridget; Orenstein, Jan M.; Fox, Linda M.; Randell, Scott H.; Rowley, Anne H.; Baker, Susan C.","Kruse, Robert L.","Yinhua Zhang; Nelson Odiwuor; Jin Xiong; Luo Sun; Raphael Ohuru Nyaruaba; Hongping Wei; Nathan A Tanner","Bin Fang; Linlin Liu; Xiao Yu; Xiang Li; Guojun Ye; Juan Xu; Ling Zhang; Faxian Zhan; Guiming Liu; Tao Pan; Yilin Shu; Yongzhong Jiang","Obameso, Joseph O.; Li, Hong; Jia, Hao; Han, Min; Zhu, Shiyan; Huang, Canping; Zhao, Yuhui; Zhao, Min; Bai, Yu; Yuan, Fei; Zhao, Honglan; Peng, Xia; Xu, Wen; Tan, Wenjie; Zhao, Yingze; Yuen, Kwok-Yung; Liu, William J.; Lu, Lin; Gao, George F.","Jackwood, Mark W.; Boynton, Tye O.; Hilt, Deborah A.; McKinley, Enid T.; Kissinger, Jessica C.; Paterson, Andrew H.; Robertson, Jon; Lemke, Conelia; McCall, Amber W.; Williams, Susan M.; Jackwood, Joshua W.; Byrd, Lauren A.","Roux, Simon; Matthijnssens, Jelle; Dutilh, Bas E.","Agut, H; Fillet, A.M; Gautheret-Dejean, A; Boutolleau, D","Foddai, Alessandro; Lubroth, Juan; Ellis-Iversen, Johanne","Al-Tawfiq, Jaffar A.; Memish, Ziad A.","Shi, Weifeng; Li, Juan; Zhou, Hong; Gao, George F.","Quilty, Billy J; Clifford, Sam; Flasche, Stefan; Eggo, Rosalind M","Villarreal, Luis P.","Bellika, Johan Gustav; Hasvold, Toralf; Hartvigsen, Gunnar","De Groot, Anne S.","Harthill, Michalann","Pengpeng Shi; Shengli Cao; Peihua Feng","Longxian Lv; Gaolei Li; Jinhui Chen; Xinle Liang; Yudong Li","Jonathan Dushoff; Sang Woo Park","Alagaili, Abdulaziz N.; Briese, Thomas; Amor, Nabil M.S.; Mohammed, Osama B.; Lipkin, W. Ian","Katelyn Gostic; Ana C. R. Gomez; Riley O. Mummah; Adam J. Kucharski; James O. Lloyd-Smith","Haoxuan Sun; Yumou Qiu; Han Yan; Yaxuan Huang; Yuru Zhu; Song Xi Chen","Wang, Pengfei; Chen, Kaiyu; Zhu, Shengqiang; Wang, Peng; Zhang, Hongliang","Gupta, Ritesh; Misra, Anoop","Yang, Chu-Wen; Chen, Mei-Fang","James Zhu; Jiwoong Kim; Xue Xiao; Yunguan Wang; Danni Luo; Ran Chen; Lin Xu; He Zhang; Guanghua Xiao; John Schoggins; Xiaowei Zhan; Tao Wang; Yang Xie","Kolawole, Olatunji; Oguntoye, Michael; Dam, Tina; Chunara, Rumi","Wu, Kailang; Peng, Guiqing; Wilken, Matthew; Geraghty, Robert J.; Li, Fang"],"color":["#67a9cf","#67a9cf","#67a9cf","#67a9cf","#67a9cf","#67a9cf","#67a9cf","#67a9cf","#67a9cf","#67a9cf","#67a9cf","#67a9cf","#67a9cf","#67a9cf","#67a9cf","#67a9cf","#67a9cf","#67a9cf","#67a9cf","#67a9cf","#67a9cf","#67a9cf","#67a9cf","#67a9cf","#67a9cf","#67a9cf","#67a9cf","#67a9cf","#67a9cf","#67a9cf","#67a9cf","#67a9cf","#67a9cf","#67a9cf","#67a9cf","#67a9cf","#67a9cf","#67a9cf","#67a9cf","#67a9cf","#67a9cf","#67a9cf","#67a9cf","#67a9cf","#67a9cf","#67a9cf","#67a9cf","#67a9cf","#67a9cf","#67a9cf"],"date":["2012-06-30","2020-07-31","2020-02-18","2020-01-30","2008-12-31","2020-03-30","2020-03-27","2013-06-03","2015-10-29","2016-11-02","2015-01-07","2019-09-19","2020-03-18","2018-04-30","2020-03-02","2020-03-12","2003-12-31","2020-03-30","2013-03-31","2020-03-29","2020-02-20","2014-05-01","2009-03-31","2020-02-07","2020-02-29","2020-03-07","2017-12-01","2010-03-01","2019-12-31","2003-09-30","2020-03-28","2014-10-31","2017-11-28","2020-02-06","2008-12-31","2007-04-30","2003-10-01","2011-02-12","2020-02-20","2020-03-02","2020-03-03","2018-05-03","2020-01-30","2020-02-20","2020-07-31","2020-03-25","2020-03-10","2020-02-12","2017-12-07","2012-01-30"],"doc":["zsr5xjma","0nh58odf","vv1wjen0","xxcx7hg5","01a8er2v","vou46eie","su7f5ges","oobydik2","7u11aj7g","xgraws2v","o8rlqsq5","kjqtvcfg","hhj7zte1","bbnhmruf","jmrg4oeb","9zs68dnn","xpl6tjps","ilc2bzkx","yybatog9","8m6793vd","od8s0zhm","7qk98wku","ea8kz8fh","2jq626ye","cv3qgno3","fnj57b0l","d0u3hb3h","9ojile3q","1qj9ri18","fwhhsnmq","1n8brl09","6ml0puh8","k0f9fser","626ch774","ydkigome","6xq5rs0u","0plznmwi","166su0tn","c800ynvc","3h1o0oz3","fhqbw32a","q7ux2r53","x22rc60j","t2x4yfrv","ubm8j1xe","3egv50vb","ietu4wqk","xetzg7gp","okb61ueg","c4q5gtw2"],"journal":["Vacunas","Infection, Genetics and Evolution","","","Encyclopedia of Virology","","","Emerg Infect Dis","PLoS Pathog","Analytica Chimica Acta","J Med Chem","Int J Mol Sci","","Virology","","","Methods in Enzymology","","Journal of Epidemiology and Global Health","","","PLoS Pathog","Journal of Virological Methods","F1000Res","","","Sci China Life Sci","Virology","Reference Module in Life Sciences","Immuno-analyse &amp; Biologie Sp\u00e9cialis\u00e9e","One Health","Trends in Microbiology","Sci China Life Sci","Euro Surveill","Origin and Evolution of Viruses","International Journal of Medical Informatics","Vaccine","Biol Trace Elem Res","","","","Saudi J Biol Sci","","","Resources, Conservation and Recycling","Diabetes &amp; Metabolic Syndrome: Clinical Research &amp; Reviews","Journal of Microbiology, Immunology and Infection","","BMC Res Notes","Journal of Biological Chemistry"],"rad":[0.01660000396366016,0.016576531208326594,0.015612863480803815,0.015419929944040827,0.015204880853998463,0.013145000449019097,0.012396913689573606,0.01233647303428725,0.010862992566744317,0.010264476609106038,0.00975391348864137,0.009397154556737855,0.00918059460015233,0.008638126904727404,0.00840583020527346,0.008075551862560506,0.0079623947298346,0.007817482257311315,0.007509382810718289,0.007258730606183881,0.007255067372295266,0.00695164776779909,0.006880378071322819,0.006332085527432724,0.006262364948118522,0.005629323213995017,0.005623501201357225,0.005417005537530351,0.005247757983071906,0.005246130681089917,0.005000021426026551,0.004694273612853229,0.004660209749384749,0.004446136849565576,0.00438548056202231,0.004253859747801196,0.004144672790229382,0.004128476976356995,0.003857214082218503,0.0036956067361986217,0.003424323861480865,0.003397338702263294,0.0033905089245219854,0.003344031753227427,0.0032222587789218567,0.003112156050067016,0.003101464972041827,0.003066581878168805,0.0030364974396374414,0.00300000514395316],"scr":[0.8,0.799,0.742,0.731,0.718,0.597,0.553,0.549,0.463,0.427,0.397,0.376,0.364,0.332,0.318,0.299,0.292,0.283,0.265,0.251,0.25,0.232,0.228,0.196,0.192,0.155,0.154,0.142,0.132,0.132,0.118,0.1,0.098,0.085,0.081,0.074,0.067,0.066,0.05,0.041,0.025,0.023,0.023,0.02,0.013,0.007,0.006,0.004,0.002,0.0],"text":["Abstract Virus-like particles (VLPs) are multimeric protein complexes mimicking the organization of native viruses but lack the viral genome. VLPs therefore constitute a safe and effective approach for the induction of neutralizing antibodies to surface proteins. Influenza VLPs have recently been developed as a new generation of non-egg based cell culture-derived vaccine candidates against influenza infection. FluBock, a recombinant trivalent hemagglutinin (rHA) vaccine produced in insect cell culture using the baculovirus expression system, provides an attractive alternative to classical inactivated influenza vaccines. The highly purified protein vaccine, administered with a three-fold higher antigen content (135 \u03bcg), is well tolerated and results in stronger immunogenicity, a long lasting immune response and provides cross-protection against drift influenza viruses.","Abstract COVID-19 is a viral respiratory illness caused by a new coronavirus called SARS-CoV-2. The World Health Organization declared the SARS-CoV-2 outbreak a global public health emergency. We performed genetic analyses of eighty-six complete or near-complete genomes of SARS-CoV-2 and revealed many mutations and deletions on coding and non-coding regions. These observations provided evidence of the genetic diversity and rapid evolution of this novel coronavirus.","Recently the novel coronavirus (2019-nCov) pneumonia outbreak in China then the world, and the Number of infections and death continues to increases. Search Query performs well in forecasting the epidemics. It is still a question whether search engine data can forecast the drift and the inflexion in 2019-nCov pneumonia. Based on the Baidu Search Index, we propose three prediction models: composite Index, composite Index with filtering and suspected NCP(Novel Coronavirus Pneumonia). The result demonstrates that the predictive model of composite index with filtering performs the best while the model of suspected NCP has the highest forecast error. We further predict the out-of-the-set NCP confirmed cases and monitor that the next peak of new diagnoses will occur on February 16th and 17th.","ABSTRACTNovel Coronavirus (nCoV) outbreak in the city of Wuhan, China during December 2019, has now spread to various countries across the globe triggering a heightened containment effort. This human pathogen is a member of betacoronavirus genus carrying 30 kilobase of single positive-sense RNA genome. Understanding the evolution, zoonotic transmission, and source of this novel virus would help accelerating containment and prevention efforts. The present study reported detailed analysis of 2019-nCoV genome evolution and potential candidate peptides for vaccine development. This nCoV genotype might have been evolved from a bat-CoV by accumulating non-synonymous mutations, indels, and recombination events. Structural proteins Spike (S), and Membrane (M) had extensive mutational changes, whereas Envelope (E) and Nucleocapsid (N) proteins were very conserved suggesting differential selection pressures exerted on 2019-nCoV during evolution. Interestingly, 2019-nCoV Spike protein contains a ...","The study of virus evolution has historically been concerned with disease and its emergence and has not been integrated into the general study of evolution. Yet viruses dominate our planet and their evolution is a broad and applied field that can be studied in the real time (such HIV in human disease) and can also be applied to biotechnological problems. Virus evolution is very similar, but not identical, to host evolution. Darwinian principles apply, such as with Fisher population genetics, but other features, such as reticulated and quasispecies-based evolution, distinguish virus evolution. In RNA viruses, high error rates have led to the quasispecies concept in which collective populations are the basis of evolution. These populations provide viruses with high capacities for adaptation. Yet some RNA viruses show remarkable evolutionary stability. With DNA viruses, especially of bacteria, extensive recombination defines vast dynamic and ancient genetic pools. Temperate viruses can di...","Abstract The World Health Organisation has declared a pandemic caused by the newly discovered SARS-CoV-2. Due to growing demand for reagents used for SARS-CoV-2 RNA extraction for subsequent molecular diagnostics, there is a worldwide risk of kit- and/or reagent-shortages for extraction. With a detection sensitivity of 97.4% (95% CI=86.2-99.9%), we describe a simple, fast, alternative workflow for molecular detection of SARS-CoV-2, where samples are simply heat-processed for 5 minutes at 98\u00b0C prior to the RT-qPCR reaction.","The recent emergence of SARS-CoV-2 lead to a current pandemic of unprecedented levels. Though diagnostic tests are fundamental to the ability to detect and respond, many health systems are already experiencing shortages of reagents associated with this test. Here, testing a pooling approach for the standard RT-qPCR test, we find that a single positive sample can be detected even in pools of up to 32 samples, with an estimated false negative rate of 10%. Detection of positive samples diluted in even up to 64 samples may also be attainable, though may require additional amplification cycles. As it uses the standard protocols, reagents and equipment, this pooling method can be applied immediately in current clinical testing laboratories. We hope that such implementation of a pool test for COVID-19 would allow expanding current screening capacities thereby enabling the expansion of detection in the community, as well as in close integral groups, such as hospital departments, army units, or...","Ten years have elapsed since the World Health Organization issued its first global alert for an unexplained illness named severe acute respiratory syndrome (SARS). The anniversary provides an opportunity to reflect on the international response to this new global microbial threat. While global surveillance and response capacity for public health threats have been strengthened, critical gaps remain. Of 194 World Health Organization member states that signed on to the International Health Regulations (2005), &lt;20% had achieved compliance with the core capacities required by the deadline in June 2012. Lessons learned from the global SARS outbreak highlight the need to avoid complacency, strengthen efforts to improve global capacity to address the next pandemic using all available 21st century tools, and support research to develop new treatment options, countermeasures, and insights while striving to address the global inequities that are the root cause of many of these challenges.","A SARS-CoV lacking the full-length E gene (SARS-CoV-\u2206E) was attenuated and an effective vaccine. Here, we show that this mutant virus regained fitness after serial passages in cell culture or in vivo, resulting in the partial duplication of the membrane gene or in the insertion of a new sequence in gene 8a, respectively. The chimeric proteins generated in cell culture increased virus fitness in vitro but remained attenuated in mice. In contrast, during SARS-CoV-\u2206E passage in mice, the virus incorporated a mutated variant of 8a protein, resulting in reversion to a virulent phenotype. When the full-length E protein was deleted or its PDZ-binding motif (PBM) was mutated, the revertant viruses either incorporated a novel chimeric protein with a PBM or restored the sequence of the PBM on the E protein, respectively. Similarly, after passage in mice, SARS-CoV-\u2206E protein 8a mutated, to now encode a PBM, and also regained virulence. These data indicated that the virus requires a PBM on a trans...","Abstract The application of optical biosensors, specifically those that use optical fibers and planar waveguides, has escalated throughout the years in many fields, including environmental analysis, food safety and clinical diagnosis. Fluorescence is, without doubt, the most popular transducer signal used in these devices because of its higher selectivity and sensitivity, but most of all due to its wide versatility. This paper focuses on the working principles and configurations of fluorescence-based fiber optic and planar waveguide biosensors and will review biological recognition elements, sensing schemes, as well as some major and recent applications, published in the last ten years. The main goal is to provide the reader a general overview of a field that requires the joint collaboration of researchers of many different areas, including chemistry, physics, biology, engineering, and material science.","[Image: see text] The carbamate group is a key structural motif in many approved drugs and prodrugs. There is an increasing use of carbamates in medicinal chemistry and many derivatives are specifically designed to make drug\u2013target interactions through their carbamate moiety. In this Perspective, we present properties and stabilities of carbamates, reagents and chemical methodologies for the synthesis of carbamates, and recent applications of carbamates in drug design and medicinal chemistry.","RNA viruses are known to replicate by low fidelity polymerases and have high mutation rates whereby the resulting virus population tends to exist as a distribution of mutants. In this review, we aim to explore how genetic events such as spontaneous mutations could alter the genomic organization of RNA viruses in such a way that they impact virus replications and plaque morphology. The phenomenon of quasispecies within a viral population is also discussed to reflect virulence and its implications for RNA viruses. An understanding of how such events occur will provide further evidence about whether there are molecular determinants for plaque morphology of RNA viruses or whether different plaque phenotypes arise due to the presence of quasispecies within a population. Ultimately this review gives an insight into whether the intrinsically high error rates due to the low fidelity of RNA polymerases is responsible for the variation in plaque morphology and diversity in virulence. This can be...","To date, official data on the number of people infected with the SARS-CoV-2 , responsible for the CoViD19 , have been released by the Italian Government just on the basis of a non representative sample of population which tested positive for the swab. However a reliable estimation of the number of infected, including asymptomatic people, turns out to be crucial in the preparation of operational schemes and to estimate the future number of people, who will require, to different extents, medical attentions. In order to overcome the current data shortcoming, this paper proposes a bootstrap driven, estimation procedure for the number of people infected with the SARSCoV2. This method is designed to be robust, automatic and suitable to generate estimations at regional level. Obtained results show that, while official data at March the 12th report 12.839 cases in Italy, people infected wiyh the SARSCoV2 could be as high as 105.789.","Abstract We recently established a mouse model (288\u2013330+/+) that developed acute respiratory disease resembling human pathology following infection with a high dose (5 \u00d7 106 PFU) of mouse-adapted MERS-CoV (icMERSma1). Although this high dose conferred fatal respiratory disease in mice, achieving similar pathology at lower viral doses may more closely reflect naturally acquired infections. Through continued adaptive evolution of icMERSma1 we generated a novel mouse-adapted MERS-CoV (maM35c4) capable of achieving severe respiratory disease at doses between 103 and 105 PFU. Novel mutations were identified in the maM35c4 genome that may be responsible for eliciting etiologies of acute respiratory distress syndrome at 10\u20131000 fold lower viral doses. Importantly, comparative genetics of the two mouse-adapted MERS strains allowed us to identify specific mutations that remained fixed through an additional 20 cycles of adaptive evolution. Our data indicate that the extent of MERS-CoV adaptation...","AbstractBackgroundThe 2019 novel coronavirus (2019-nCoV or SARS-CoV-2) has spread more rapidly than any other betacoronavirus including SARS-CoV and MERS-CoV. However, the mechanisms responsible for infection and molecular evolution of this virus remained unclear.MethodsWe collected and analyzed 120 genomic sequences of 2019-nCoV including 11 novel genomes from patients in China. Through comprehensive analysis of the available genome sequences of 2019-nCoV strains, we have tracked multiple inheritable SNPs and determined the evolution of 2019-nCoV relative to other coronaviruses.ResultsSystematic analysis of 120 genomic sequences of 2019-nCoV revealed co-circulation of two genetic subgroups with distinct SNPs markers, which can be used to trace the 2019-nCoV spreading pathways to different regions and countries. Although 2019-nCoV, human and bat SARS-CoV share high homologous in overall genome structures, they evolved into two distinct groups with different receptor entry specificities...","Background The epidemic caused by SARS-CoV-2 was first reported in Wuhan, China, and now is spreading worldwide. The Chinese government responded to this epidemic with multiple public health policies including locking down the city of Wuhan, establishing multiple temporary hospitals, and prohibiting public gathering events. Here, we constructed a new real-time status dynamic model of SEIO (MH) to reveal the influence of national public health policies and to model the epidemic in Wuhan. Methods A real-time status dynamic model was proposed to model the population of Wuhan in status Susceptible (S), Exposed (E), Infected with symptoms (I), with Medical care (M), and Out of the system (O) daily. Model parameters were fitted according to the daily report of new infections from Jan. 27th, 2020 to Feb. 2nd, 2020. Using the fitted parameters, the epidemic under different conditions was simulated and compared with the current situation. Finding According to our study, the first patient is mos...","Publisher Summary This chapter describes three assays to monitor receptor-induced association of the envelope glycoprotein (EnvA) of avian sarcoma/leukosis virus (ASLV) with target bilayers: (1) the original assay for monitoring binding of the EnvA ectodomain (EnvA-PI) to target membranes (liposomes), (2) a modified and miniaturized EnvA-PI-liposome binding assay, and (3) an assay to measure binding of intact sarcoma/leukosis virus subtype A (ASLV-A) virus particles to target membranes. These assays are also useful for studying other receptor-activated viral fusion proteins. When one viral glycoprotein and one \u201csimple\u201d host cell receptor are involved, it should be possible to develop assays directly analogous to those described above for studying Tva-induced binding of the EnvA ectodomain (EnvA-PI) to target membranes. A general prerequisite for a fusion protein/target membrane binding assay is a soluble and correctly oligomeric form of the viral fusion protein ectodomain. The simplest...","Background: Chest CT had high sensitivity in diagnosing novel coronavirus pneumonia (NCP) at early stage, giving it an advantage over nucleic acid detection in time of crisis. Deep learning was reported to discover intricate structures from clinical images and achieve expert-level performance in medical image analysis. To develop and validate an integrated deep learning framework on chest CT images for auto-detection of NCP, particularly focusing on differentiating NCP from influenza pneumonia (IP). Methods: 35 confirmed NCP cases were consecutively enrolled as training set from 1138 suspected patients in three NCP designated hospitals together with 361 confirmed viral pneumonia patients from center one including 156 IP patients, from May, 2015 to February, 2020. The external validation set enrolled 57 NCP patients and 50 IP patients from eight centers. Results: 96.6% of NCP lesions were larger than 1 cm and 76.8% were with intensity below -500 Hu, indicating less consolidation than IP...","Abstract The goal of syndromic surveillance is the earlier detection of epidemics, allowing a timelier public health response than is possible using traditional surveillance methods. Syndromic surveillance application for public health purposes has changed over time and reflects a dynamic evolution from the collection, interpretation of data with dissemination of data to those who need to act, to a more holistic approach that incorporates response as a core component of the surveillance system. Recent infectious disease threats, such as severe acute respiratory syndrome (SARS), avian influenza (H5N1) and pandemic influenza (H1N1), have all highlighted the need for countries to be rapidly aware of the spread of infectious diseases within a region and across the globe. The International Health Regulations (IHR) obligation to report public health emergencies of international concern has raised the importance of early outbreak detection and response. The emphasis in syndromic surveillance ...","Coronavirus disease 2019 (COVID-19) can be screened and diagnosed through the detection of severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) by real-time reverse transcription polymerase chain reaction. SARS-CoV-2 nucleic acid amplification tests (NAATs) have been rapidly developed and quickly applied to clinical testing during the pandemic. However, studies evaluating the performance of these NAAT assays are limited. We evaluated the performance of four NAATs, which were marked by the Conformit\u00e9 Europ\u00e9enne and widely used in China during the pandemic. Results showed that the analytical sensitivity of the four assays was significantly lower than that claimed by the NAAT manufacturers. The limit of detection (LOD) of Daan, Sansure, and Hybribio NAATs was 3000 copies/mL, whereas the LOD of Bioperfectus NAATs was 4000 copies/mL. The results of the consistency test using 46 samples showed that Daan, Sansure, and Hybribio NAATs could detect the samples with a specificity of 100% ...","The epidemic caused by the novel coronavirus COVID-19 in Wuhan at the end of 2019 has become an urgent public event of worldwide concern. However, due to the changing data of the epidemic, there is no scientific estimate of the cure rate and case fatality rate of the epidemic. This study proposes a method to estimate the cure rate and case fatality rate of COVID-19. The ratio of cumulative discharges on a given day to the sum of cumulative discharges on a given day and cumulative deaths before j days is used to estimate the cure rate. Moreover, the case fatality ratio can also be estimated. After simulation calculations, j is statistically appropriate when it is 8-10, and it is also clinically appropriate. When j is 9, based on the available data, it is inferred that the cure rate of this epidemic is about 93% and the case fatality rate is about 7%. This method of estimating the cure rate can be used to evaluate the effectiveness of treatment in different medical schemes and different ...","Deletion of Severe Acute Respiratory Syndrome Coronavirus (SARS-CoV) envelope (E) gene attenuates the virus. E gene encodes a small multifunctional protein that possesses ion channel (IC) activity, an important function in virus-host interaction. To test the contribution of E protein IC activity in virus pathogenesis, two recombinant mouse-adapted SARS-CoVs, each containing one single amino acid mutation that suppressed ion conductivity, were engineered. After serial infections, mutant viruses, in general, incorporated compensatory mutations within E gene that rendered active ion channels. Furthermore, IC activity conferred better fitness in competition assays, suggesting that ion conductivity represents an advantage for the virus. Interestingly, mice infected with viruses displaying E protein IC activity, either with the wild-type E protein sequence or with the revertants that restored ion transport, rapidly lost weight and died. In contrast, mice infected with mutants lacking IC acti...","Abstract Propagation of new human respiratory virus pathogens in established cell lines is hampered by a lack of predictability regarding cell line permissivity and by availability of suitable antibody reagents to detect infection in cell lines that do not exhibit significant cytopathic effect. Recently, molecular methods have been used to amplify and identify novel nucleic acid sequences directly from clinical samples, but these methods may be hampered by the quantity of virus present in respiratory secretions at different time points following the onset of infection. Human airway epithelial (HAE) cultures, which effectively mimic the human bronchial environment, allow for cultivation of a wide variety of human respiratory viral pathogens. The goal of the experiments described here was to determine if propagation and identification of a human respiratory virus may be achieved through inoculation of HAE cultures followed by whole transcriptome amplification (WTA) and sequence analysis....","A novel coronavirus (2019-nCoV) originating in Wuhan, China presents a potential respiratory viral pandemic to the world population. Current efforts are focused on containment and quarantine of infected individuals. Ultimately, the outbreak could be controlled with a protective vaccine to prevent 2019-nCoV infection. While vaccine research should be pursued intensely, there exists today no therapy to treat 2019-nCoV upon infection, despite an urgent need to find options to help these patients and preclude potential death. Herein, I review the potential options to treat 2019-nCoV in patients, with an emphasis on the necessity for speed and timeliness in developing new and effective therapies in this outbreak. I consider the options of drug repurposing, developing neutralizing monoclonal antibody therapy, and an oligonucleotide strategy targeting the viral RNA genome, emphasizing the promise and pitfalls of these approaches. Finally, I advocate for the fastest strategy to develop a treat...","The ability to detect an infectious agent in a widespread epidemic is crucial to the success of quarantine efforts in addition to sensitive and accurate screening of potential cases of infection from patients in a clinical setting. Enabling testing outside of sophisticated laboratories broadens the scope of control and surveillance efforts, but also requires robust and simple methods that can be used without expensive instrumentation. Here we report a method to identify SARS-CoV-2 (COVID-19) virus RNA from purified RNA or cell lysis using loop-mediated isothermal amplification (LAMP) using a visual, colorimetric detection. This test was additionally verified using RNA samples purified from respiratory swabs collected from COVID-19 patients in Wuhan, China with equivalent performance to a commercial RT-qPCR test while requiring only heating and visual inspection. This simple and sensitive method provides an opportunity to facilitate virus detection in the field without a requirement for...","AbstractSince December 2019, coronavirus disease 2019 (COVID-19) emerged in Wuhan, Central China and rapidly spread throughout China. Up to March 3, 2020, SARS-CoV-2 has infected more than 89,000 people in China and other 66 countries across six continents. In this study, we used 10 new sequenced genomes of SARS-CoV-2 and combined 136 genomes from GISAID database to investigate the genetic variation and population demography through different analysis approaches (e.g. Network, EBSP, Mismatch, and neutrality tests). The results showed that 80 haplotypes had 183 substitution sites, including 27 parsimony-informative and 156 singletons. Sliding window analyses of genetic diversity suggested a certain mutations abundance in the genomes of SARS-CoV-2, which may be explaining the existing widespread and high adaptation of the deadly virus. Phylogenetic analysis showed that the view, pangolin acted as an intermediate host, may be controversial. The network indicated that, in the original hapl...","Bats are connected with the increasing numbers of emerging and re-emerging viruses that may break the species barrier and spread into the human population. Coronaviruses are one of the most common viruses discovered in bats, which were considered as the natural source of recent human-susceptible coronaviruses, i.e. SARS-COV and MERS-CoV. Our previous study reported the discovery of a bat-derived putative cross-family recombinant coronavirus with a reovirus gene p10, named as Ro-BatCoV GCCDC1. In this report, through a two-year follow-up of a special bat population in one specific cave of south China, we illustrate that Ro-BatCoV GCCDC1 persistently circulates among bats. Notably, through the longitudinal observation, we identified the dynamic evolution of Ro-BatCoV GCCDC1 in bats represented by continuously recombination events. Our study provides the first glimpse of the virus evolution in one longitudinally observed bat population cohort and underlines the surveillance and pre-warnin...","Abstract Analyses of turkey coronavirus (TCoV), an enteric disease virus that is highly similar to infectious bronchitis virus (IBV) an upper-respiratory tract disease virus in chickens, were conducted to determine the adaptive potential, and genetic changes associated with emergence of this group 3 coronavirus. Strains of TCoV that were pathogenic in poults and nonpathogenic in chickens did not adapt to cause disease in chickens. Comparative genomics revealed two recombination sites that replaced the spike gene in IBV with an unidentified sequence likely from another coronavirus, resulting in cross-species transmission and a pathogenicity shift. Following emergence in turkeys, TCoV diverged to different serotypes through the accumulation of mutations within spike. This is the first evidence that recombination can directly lead to the emergence of new coronaviruses and new coronaviral diseases, emphasizing the importance of limiting exposure to reservoirs of coronaviruses that can serv...","Abstract Metagenomics, i.e., the sequencing and analysis of genomic information extracted directly from clinical or environmental samples, has become a fundamental tool to explore the viral world. Against the background of an extensive viral diversity revealed by metagenomics across many environments, new sequence assembly approaches that reconstruct complete genome sequences from metagenomes have recently revealed surprisingly cosmopolitan viruses in specific ecological niches. Metagenomics is also applied to clinical samples as a non-targeted diagnostic and surveillance tool. By enabling the study of these uncultivated viruses, metagenomics provides invaluable insights into the virus-host interactions, epidemiology, ecology, and evolution of viruses across all ecosystems.","R\u00e9sum\u00e9 La d\u00e9tection, la quantification et l\u2019analyse g\u00e9n\u00e9tique des g\u00e9nomes viraux ont \u00e9t\u00e9 progressivement introduites dans la pratique quotidienne de la virologie m\u00e9dicale. Dans le cas des infections \u00e0 herp\u00e8svirus humains 6 et 7 (HHV-6, HHV-7), la d\u00e9tection de l\u2019ADN viral a permis d\u2019impliquer ces infections dans des maladies au-del\u00e0 de ce qui avait \u00e9t\u00e9 mis en \u00e9vidence par les m\u00e9thodes conventionnelles (culture, s\u00e9rologie) notamment chez les sujets immunod\u00e9prim\u00e9s. La quantification, en particulier gr\u00e2ce \u00e0 l\u2019apport r\u00e9cent de la PCR en temps r\u00e9el, est susceptible d\u2019\u00e9tablir la distinction entre latence et r\u00e9activation virales, de pr\u00e9dire la survenue d\u2019\u00e9v\u00e8nements cliniques, de suivre les effets d\u2019une th\u00e9rapeutique sp\u00e9cifique. L\u2019analyse g\u00e9n\u00e9tique a conduit \u00e0 l\u2019identification de variants et contribu\u00e9 \u00e0 caract\u00e9riser les mutations associ\u00e9es \u00e0 la r\u00e9sistance aux antiviraux. Comme le montre l\u2019\u00e9tude des herp\u00e8svirus, ces techniques mol\u00e9culaires ont aussi des limites qu\u2019il faut conna\u00eetre mais constitu...","Abstract The pandemic of new coronavirus disease COVID-19 is threatening our health, economy and life style. Collaborations across countries and sectors as a One Health World could be a milestone. We propose a general protocol, for setting timely active random surveillance of COVID-19, at the human community level, with systematic repeated detection efforts. Strengths and limitations are discussed. If considered applicable by public health, the protocol could evaluate the status of COVID-19 epidemics consistently and objectively.","The Middle East respiratory syndrome coronavirus (MERS-CoV) was first described in 2012 and, subsequently, many cases were reported with a lower case fatality rate than initial cases. Humans can become infected within their communities and transmission can then be amplified in the healthcare setting. Contact investigation among cases shows a variable amount of spread among family members and healthcare workers. So far, circulating virus strains remain similar under continuous monitoring, with no genetic changes. Here, we discuss the transmission pattern, phylogenetic evolution, and pathogenesis of MERS-CoV infection.","In the past twenty years, numerous novel zoonotic viral agents with pandemic potential have emerged in China, such as the severe acute respiratory syndrome (SARS) coronavirus and, more recently, the avian-origin influenza A/H7N9 virus, which have caused outbreaks among humans with high morbidity and mortality. In addition, several emerging and re-emerging viral pathogens have also been imported into China from travelers, e.g. the Middle East respiratory syndrome (MERS) coronavirus and Zika virus (ZIKV). Herein, we review these emerging viral pathogens in China and focus on how surveillance by pathogen genomics has been employed to discover and annotate novel pathogenic agents, identify natural reservoirs, monitor the transmission events and delineate their evolution and adaption to the human host. We also highlight the application of genomic sequencing in the recent Ebola epidemics in Western Africa. In summary, genomic sequencing has become a standard research tool in the field of eme...","We evaluated effectiveness of thermal passenger screening for 2019-nCoV infection at airport exit and entry to inform public health decision-making. In our baseline scenario, we estimated that 46% (95% confidence interval: 36 to 58) of infected travellers would not be detected, depending on incubation period, sensitivity of exit and entry screening, and proportion of asymptomatic cases. Airport screening is unlikely to detect a sufficient proportion of 2019-nCoV infected travellers to avoid entry of infected travellers.","ABSTRACT In the last 30 years, the study of virus evolution has undergone a transformation. Originally concerned with disease and its emergence, virus evolution had not been well integrated into the general study of evolution. This chapter reviews the developments that have brought us to this new appreciation for the general significance of virus evolution to all life. We now know that viruses numerically dominate all habitats of life, especially the oceans. Theoretical developments in the 1970s regarding quasispecies, error rates, and error thresholds have yielded many practical insights into virus\u2013host dynamics. The human diseases of HIV-1 and hepatitis C virus cannot be understood without this evolutionary framework. Yet recent developments with poliovirus demonstrate that viral fitness can be the result of a consortia, not one fittest type, a basic Darwinian concept in evolutionary biology. Darwinian principles do apply to viruses, such as with Fisher population genetics, but other...","Abstract Purpose The purpose of the study was (1) to identify the requirements for syndromic, disease surveillance and epidemiology systems arising from events such as the SARS outbreak in March 2003, and the deliberate spread of Bacillus anthracis, or anthrax, in the US in 2001; and (2) to use these specifications as input to the construction of a system intended to meet these requirements. An important goal was to provide information about the diffusion of a communicable disease without being dependent on centralised storage of information about individual patients or revealing patient-identifiable information. Methods The method applied is rooted in the engineering paradigm involving phases of analysis, system specification, design, implementation, and testing. The requirements were established from earlier projects\u2019 conclusions and analysis of disease outbreaks. The requirements were validated by a literature study of syndromic and disease surveillance systems. The system was teste...","Abstract A remarkable collaborative effort coordinated by the severe acute respiratory syndrome (SARS) team at WHO resulted in discovery of the etiologic agent of severe acute respiratory syndrome less than 2 months after the announcement of global alert. The development of a vaccine to prevent SARS should be pursued with the same urgency and cooperative spirit, as SARS is highly lethal and, if not controlled during the first few generations of transmission, is likely to become endemic in regions of the world where health-care infrastructure is underdeveloped and epidemiological control measures are weak. The scientific community already learned many important lessons from HIV vaccine development; these should be heeded. For example, consideration should be given to the development of a vaccine that will protect across regional strains of SARS, as the newly emergent coronavirus SARS-coronavirus (SARS-CoV) is proving to be variable and may be mutating in response to immune pressure. SAR...","Recently emerged viral infectious diseases (VIDs) include HIV/AIDS, influenzas H5N1 and 2009 H1N1, SARS, and Ebola hemorrhagic fevers. Earlier research determined metabolic oxidative stress in hosts deficient in antioxidant selenium (Se) (&lt;1 \u03bcMol Se/L of blood) induces both impaired human host immunocompetence and rapidly mutated benign variants of RNA viruses to virulence. These viral mutations are consistent, rather than stochastic, and long-lived. When Se-deficient virus-infected hosts were supplemented with dietary Se, viral mutation rates diminished and immunocompetence improved. Herein is described the role of micronutrient Se deficiency on the evolution of some contemporary RNA viruses and their subsequent VIDs. Distinguishing cellular and biomolecular evidence for several VIDs suggests that environmental conditions conducive to chronic dietary Se deprivation could be monitored for bioindicators of incipient viral virulence and subsequent pathogenesis.","Pneumonia patients of 2019-ncov in latent period are not easy to be effectively quarantined, but there is evidence that they have strong infectious ability. Here, the infectious ability of patients during the latent period is slightly less than that of the infected patients was assumed. We established a new SEIR propagation dynamics model, that considered the weak transmission ability of the incubation period, the variation of the incubation period length, and the government intervention measures to track and isolate comprehensively. Based on the raw epidemic data of China from January 23, 2020 to February 10, 2020, the dynamic parameters of the new present SEIR model are fitted. Through the Euler integration algorithm to solve the model, the effect of infectious ability of incubation patients on the theoretical estimation of the present SEIR model was analyzed, and the occurrence time of peak number in China was predicted.","AbstractThe novel coronavirus SARS-CoV-2 (2019-nCoV) is a member of the family coronaviridae and contains a single-stranded RNA genome with positive-polarity. To reveal the evolution mechanism of SARS-CoV-2 genome, we performed comprehensive genomic analysis with newly sequenced SARS-CoV-2 strains and 20 closely related coronavirus strains. Among 98 nucleotide mutations at 93 sites of the genome among different SARS-CoV-2 strains, 58 of them caused amino acid change, indicating a result of neutral evolution. However, the ratio of nucleotide substitutions to amino acid substitutions of spike gene (9.07) between SARS-CoV-2 WIV04 and Bat-SARSr-CoV RaTG13 was extensively higher than those from comparisons between other coronaviruses (range 1.29 - 4.81). The elevated synonymous mutations between SARS-CoV-2 and RaTG13, suggesting they underwent stronger purifying selection. Moreover, their nucleotide substitutions are enriched with T:C transition, which is consistent with the mutation signat...","AbstractAn epidemic can be characterized by its speed (i.e., the exponential growth rate r) and strength (i.e., the reproductive number \u211b). Disease modelers have historically placed much more emphasis on strength, in part because the effectiveness of an intervention strategy is typically evaluated on this scale. Here, we develop a mathematical framework for this classic, strength-based paradigm and show that there is a corresponding speed-based paradigm which can provide complementary insights. In particular, we note that r = 0 is a threshold for disease spread, just like \u211b = 1, and show that we can measure the speed and strength of an intervention on the same scale as the speed and strength of an epidemic, respectively. We argue that, just as the strength-based paradigm provides the clearest insight into certain questions, the speed-based paradigm provides the clearest view in other cases. As an example, we show that evaluating the prospects of \u201ctest-and-treat\u201d interventions against t...","The Middle East Respiratory Syndrome (MERS-CoV) emerged in the Kingdom of Saudi Arabia in 2012 causing a critical challenge to public health. The epidemiology of MERS-CoV remain enigmatic as human-to-human transmission is not fully understood. One possible scenario that might play a role in the virus transmission is the cultural waterpipe smoking. Caf\u00e9s providing waterpipe smoking in cities within Saudi Arabia have been moved to areas outside city limits that frequently place them close to camels markets. We report results of a surveillance study wherein waterpipe hoses throughout several regions in Saudi Arabia were tested for the presence of MERS-CoV. A total of 2489 waterpipe samples were collected from cities where MERS-CoV cases were continuously recorded. MERS-CoV RNA wasn\u2019t detected in collected samples. Irrespective of the negative results of our survey, the public health risk of waterpipe smoking should not be underestimated. To avoid a possible transmission within country whe...","Traveller screening is being used to limit further global spread of 2019 novel coronavirus (nCoV) following its recent emergence. Here, we analyze the expected impact of different travel screening programs given remaining uncertainty around the values of key nCoV life history and epidemiological parameters. Even under best-case assumptions, we estimate that screening will miss around half of infected travellers. Breaking down the factors leading to screening successes and failures, we find that most cases missed by screening are fundamentally undetectable, because they have not yet developed symptoms and are unaware they were exposed. These findings emphasize the need for measures to track travellers who become ill after being missed by a travel screening program. We make our model available for interactive use so stakeholders can explore scenarios of interest using the most up-to-date information. We hope these findings contribute to evidence-based policy to combat the spread of nCoV,...","By proposing a varying coefficient Susceptible-Infected-Removal model (vSIR), we track the epidemic of COVID-19 in 30 provinces in China and 15 cities in Hubei province, the epicenter of the outbreak. It is found that the spread of COVID-19 has been significantly slowing down within the two weeks from January 27 to February 10th with 87.0% and 84.3% reductions in the reproduction number R0 among the 30 provinces and 15 Hubei cities, respectively. This suggests the extreme control measures implemented since January 23, which include cutting off Wuhan and many other cities and towns, a great public awareness and high level of self isolation at home, have contributed to a substantial decline in the reproductivity of the COVID-19 in China. We predict that Hubei province will reach its peak between February 20 and 22, 2020, and if the removal rate can be increased to 0.1, the epidemic outside Hubei province will end in May 2020, and inside Hubei in early June.","Abstract Due to the pandemic of coronavirus disease 2019 in China, almost all avoidable activities in China are prohibited since Wuhan announced lockdown on January 23, 2020. With reduced activities, severe air pollution events still occurred in the North China Plain, causing discussions regarding why severe air pollution was not avoided. The Community Multi-scale Air Quality model was applied during January 01 to February 12, 2020 to study PM2.5 changes under emission reduction scenarios. The estimated emission reduction case (Case 3) better reproduced PM2.5. Compared with the case without emission change (Case 1), Case 3 predicted that PM2.5 concentrations decreased by up to 20% with absolute decreases of 5.35, 6.37, 9.23, 10.25, 10.30, 12.14, 12.75, 14.41, 18.00 and 30.79 \u03bcg/m3 in Guangzhou, Shanghai, Beijing, Shijiazhuang, Tianjin, Jinan, Taiyuan, Xi'an, Zhengzhou, Wuhan, respectively. In high-pollution days with PM2.5 greater than 75 \u03bcg/m3, the reductions of PM2.5 in Case 3 were 7...","Abstract Background and Aims Multiple issues in management of COVID have emerged, but confusion persists regarding rational interpretation. Aim of this brief review is to review these issues based on current literature. Methods This is a narrative review with Pubmed and Google Scholar search till 23 March 2020. Search terms were, COVID-19, treatment of coronavirus, COVID 19 and following terms; chloroquine, hydroxychloroquine, ibuprofen, ACE-inhibitors or angiotensin receptor blockers, cardiovascular disease, diarrhoea, liver, testis and gastrointestinal disease. Results We discuss evidence regarding role of chloroquine and hydroxychloroquine in treatment and prophylaxis, use of inhibitors of the renin angiotensin system, safety of ibuprofen, unusual clinical features like gastrointestinal symptoms and interpretation of tests for cardiac enzymes and biomarkers. Conclusions While our conclusions on management of COVID-19 patients with co-morbidities are based on current evidence, howeve...","Abstract Translation of a genetic codon without a cognate tRNA gene is affected by both the cognate tRNA availability and the interaction with non-cognate isoacceptor tRNAs. Moreover, two consecutive slow codons (slow di-codons) lead to a much slower translation rate. Calculating the composition of host specific slow codons and slow di-codons in the viral protein coding sequences can predict the order of viral protein synthesis rates between different virus strains. Comparison of human-specific slow codon and slow di-codon compositions in the genomes of 590 coronaviruses infect humans revealed that the protein synthetic rates of 2019 novel coronavirus (2019-nCoV) and severe acute respiratory syndrome-related coronavirus (SARS-CoV) may be much faster than other coronaviruses infect humans. Analysis of host-specific slow codon and di-codon compositions provides links between viral genomic sequences and capability of virus replication in host cells that may be useful for surveillance of t...","The outbreak of the 2019 Novel Coronavirus (SARS-CoV-2) rapidly spread from Wuhan, China to more than 150 countries, areas or territories, causing staggering number of infections and deaths. A systematic profiling of the immune vulnerability landscape of SARS-CoV-2, which can bring critical insights into the immune clearance mechanism, peptide vaccine development, and antiviral antibody development, is lacking. In this study, we investigated the potential of the SARS-CoV-2 viral proteins to induce class I and II MHC presentation and to form linear antibody epitopes. We created an online database to broadly share the predictions as a resource for the research community. Using this resource, we showed that genetic variations in SARS-CoV-2, though still few for the moment, already follow the pattern of mutations in related coronaviruses, and could alter the immune vulnerability landscape of this virus. Importantly, we discovered evidence that SARS-CoV-2, along with related coronaviruses, ...","OBJECTIVE: Recognizing increasing interest in community disease surveillance globally, the goal of this study was to investigate whether respiratory viruses circulating in the community may be represented through clinical (hospital) surveillance in Nigeria. RESULTS: Children were selected via convenience sampling from communities and a tertiary care center (n = 91) during spring 2017 in Ilorin, Nigeria. Nasal swabs were collected and tested using polymerase chain reaction. The majority (79.1%) of subjects were under 6 years old, of whom 46 were infected (63.9%). A total of 33 of the 91 subjects had one or more respiratory tract virus; there were 10 cases of triple infection and 5 of quadruple. Parainfluenza virus 4, respiratory syncytial virus B and enterovirus were the most common viruses in the clinical sample; present in 93.8% (15/16) of clinical subjects, and 6.7% (5/75) of community subjects (significant difference, p &lt; 0.001). Coronavirus OC43 was the most common virus detected i...","The severe acute respiratory syndrome coronavirus (SARS-CoV) from palm civets has twice evolved the capacity to infect humans by gaining binding affinity for human receptor angiotensin-converting enzyme 2 (ACE2). Numerous mutations have been identified in the receptor-binding domain (RBD) of different SARS-CoV strains isolated from humans or civets. Why these mutations were naturally selected or how SARS-CoV evolved to adapt to different host receptors has been poorly understood, presenting evolutionary and epidemic conundrums. In this study, we investigated the impact of these mutations on receptor recognition, an important determinant of SARS-CoV infection and pathogenesis. Using a combination of biochemical, functional, and crystallographic approaches, we elucidated the molecular and structural mechanisms of each of these naturally selected RBD mutations. These mutations either strengthen favorable interactions or reduce unfavorable interactions with two virus-binding hot spots on A..."],"title":["Las vacunas de la gripe basadas en las part\u00edculas virus-like obtenidas mediante sistemas de expresi\u00f3n g\u00e9nica en c\u00e9lulas de insectos","Genetic diversity and evolution of SARS-CoV-2","Can Search Query Forecast successfully in China's 2019-nCov pneumonia?","Insights into Cross-species Evolution of Novel Human Coronavirus 2019-nCoV and Defining Immune Determinants for Vaccine Development","Evolution of Viruses","An alternative workflow for molecular detection of SARS-CoV-2 - escape from the NA extraction kit-shortage","Evaluation of COVID-19 RT-qPCR test in multi-sample pools","Progress in Global Surveillance and Response Capacity 10 Years after Severe Acute Respiratory Syndrome","Identification of the Mechanisms Causing Reversion to Virulence in an Attenuated SARS-CoV for the Design of a Genetically Stable Vaccine","Fluorescence based fiber optic and planar waveguide biosensors. A review","Organic Carbamates in Drug Design and Medicinal Chemistry","Impact of RNA Virus Evolution on Quasispecies Formation and Virulence","CoViD--19: An Automatic, Semiparametric Estimation Method for the Population Infected in Italy","Adaptive evolution influences the infectious dose of MERS-CoV necessary to achieve severe respiratory disease","Mutations, Recombination and Insertion in the Evolution of 2019-nCoV","Revealing the influence of national public health policies for the outbreak of the SARS-CoV-2 epidemic in Wuhan, China through status dynamic modeling","Receptor-Activated Binding of Viral Fusion Proteins to Target Membranes","Improved deep learning model for differentiating novel coronavirus pneumonia and influenza pneumonia","The remarkable adaptability of syndromic surveillance to meet public health needs","Comparative performance of four nucleic acid amplification tests for SARS-CoV-2 virus","Estimating the cure rate and case fatality rate of the ongoing epidemic COVID-19","Severe Acute Respiratory Syndrome Coronavirus Envelope Protein Ion Channel Activity Promotes Virus Fitness and Pathogenesis","Human airway epithelial cell culture to identify new respiratory viruses: Coronavirus NL63 as a model","Therapeutic strategies in an outbreak scenario to treat the novel coronavirus originating in Wuhan, China","Rapid Molecular Detection of SARS-CoV-2 (COVID-19) Virus RNA Using Colorimetric LAMP","Genome-wide data inferring the evolution and population demography of the novel pneumonia coronavirus (SARS-CoV-2)","The persistent prevalence and evolution of cross-family recombinant coronavirus GCCDC1 among a bat population: a two-year follow-up","Emergence of a group 3 coronavirus through recombination","Metagenomics in Virology","D\u00e9tection, quantification et analyse des g\u00e9nomes viraux dans les infections \u00e0 herp\u00e8svirus humains 6 et 7 (HHV-6, HHV-7)","Base protocol for real time active random surveillance of coronavirus disease (COVID-19) \u2013 Adapting veterinary methodology to public health","Middle East respiratory syndrome coronavirus: transmission and phylogenetic evolution","Pathogen genomic surveillance elucidates the origins, transmission and evolution of emerging viral agents in China","Effectiveness of airport screening at detecting travellers infected with novel coronavirus (2019-nCoV)","CHAPTER 21 The Widespread Evolutionary Significance of Viruses","Propagation of program control: A tool for distributed disease surveillance","How the SARS vaccine effort can learn from HIV\u2014speeding towards the future, learning from the past","Review: Micronutrient Selenium Deficiency Influences Evolution of Some Viral Infectious Diseases","SEIR Transmission dynamics model of 2019 nCoV coronavirus with considering the weak infectious ability and changes in latency duration","Comparative genomic analysis revealed specific mutation pattern between human coronavirus SARS-CoV-2 and Bat-SARSr-CoV RaTG13","Speed and strength of an epidemic intervention","Waterpipe smoking as a public health risk: Potential risk for transmission of MERS-CoV","Estimated effectiveness of traveller screening to prevent international spread of 2019 novel coronavirus (2019-nCoV)","Tracking and Predicting COVID-19 Epidemic in China Mainland","Severe air pollution events not avoided by reduced anthropogenic activities during COVID-19 outbreak","Contentious Issues and Evolving Concepts in the Clinical Presentation and Management of Patients with COVID-19 Infectionwith Reference to Use of Therapeutic and Other Drugs used in Co-morbid Diseases (Hypertension, Diabetes etc)","Composition of human-specific slow codons and slow di-codons in SARS-CoV and 2019-nCoV are lower than other coronaviruses suggesting a faster protein synthesis rate of SARS-CoV and 2019-nCoV","The immune vulnerability landscape of the 2019 Novel Coronavirus, SARS-CoV-2","Etiology of respiratory tract infections in the community and clinic in Ilorin, Nigeria","Mechanisms of Host Receptor Adaptation by Severe Acute Respiratory Syndrome Coronavirus"],"url":["https://doi.org/10.1016/s1576-9887(12)70039-0","https://doi.org/10.1016/j.meegid.2020.104260","https://doi.org/10.1101/2020.02.12.20022400","https://doi.org/10.1101/2020.01.29.925867","https://doi.org/10.1016/b978-012374410-4.00706-8","https://doi.org/10.1101/2020.03.27.20044495","https://doi.org/10.1101/2020.03.26.20039438","https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3713843/","https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4626112/","https://doi.org/10.1016/j.aca.2016.08.049","https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4393377/","https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6770471/","https://doi.org/10.1101/2020.03.14.20036103","https://doi.org/10.1016/j.virol.2017.12.006","https://doi.org/10.1101/2020.02.29.971101","https://doi.org/10.1101/2020.03.10.20032995","https://doi.org/10.1016/s0076-6879(03)72026-6","https://doi.org/10.1101/2020.03.24.20043117","https://doi.org/10.1016/j.jegh.2012.12.005","https://doi.org/10.1101/2020.03.26.010975","https://doi.org/10.1101/2020.02.18.20024513","https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4006877/","https://doi.org/10.1016/j.jviromet.2008.10.022","https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7029759/","https://doi.org/10.1101/2020.02.26.20028373","https://doi.org/10.1101/2020.03.04.976662","https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7088801/","https://doi.org/10.1016/j.virol.2009.11.044","https://doi.org/10.1016/b978-0-12-809633-8.20957-6","https://doi.org/10.1016/s0923-2532(03)00065-6","https://doi.org/10.1016/j.onehlt.2020.100129","https://doi.org/10.1016/j.tim.2014.08.001","https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7088571/","https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7014668/","https://doi.org/10.1016/b978-0-12-374153-0.00021-7","https://doi.org/10.1016/j.ijmedinf.2006.02.007","https://doi.org/10.1016/s0264-410x(03)00489-4","https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7090490/","https://doi.org/10.1101/2020.02.16.20023655","https://doi.org/10.1101/2020.02.27.969006","https://doi.org/10.1101/2020.03.02.974048","https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6600605/","https://doi.org/10.1101/2020.01.28.20019224","https://doi.org/10.1101/2020.02.17.20024257","https://doi.org/10.1016/j.resconrec.2020.104814","https://doi.org/10.1016/j.dsx.2020.03.012","https://doi.org/10.1016/j.jmii.2020.03.002","https://doi.org/10.1101/2020.02.08.939553","https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5719735/","http://www.jbc.org/content/287/12/8904.full.pdf"],"x":[0.887070552758065,0.99344602619614,0.39772339901942744,0.6627293031640831,0.9478486085480471,0.2784936811512788,0.5871715752588084,0.3772559009059101,0.3384629798635477,0.872158589956428,0.903922091402023,0.1432000724432677,0.4747086616298666,0.9238885416408443,0.8138051671250631,0.331579756541169,0.17466857902454602,0.6926749315114029,0.9892841587752126,0.2872459159033932,0.4493914210925246,0.8553241673617354,0.47689568968752016,0.8926284972608866,0.9821341777439611,0.2417827065564523,0.35707636885385485,0.560911890409537,0.30641879284936213,0.10689061214057816,0.15681305211705943,0.44256297990060234,0.3550860335959828,0.2085647260774396,0.09592552831850654,0.7748979139100404,0.5462942551332349,0.8889225652966373,0.9788073838742105,0.9317872676497827,0.395369504443571,0.28785980473684103,0.9912604358424435,0.8352928495552661,0.30482308162034044,0.09690579364997409,0.10775373732773075,0.35750830901446873,0.3275159707248222,0.45818561351320786],"y":[0.3570798359016426,0.761406325063259,0.8639872605127387,0.8257904976745563,0.9218167899681453,0.8223845057351666,0.6387037988446457,0.9178096142794768,0.8574846459171918,0.6344009186544378,0.7355841194913642,0.6582070503666914,0.5271894038545479,0.9111256494895732,0.2722936637432182,0.24033359904820784,0.46708928952716466,0.975409479740455,0.5760694105529968,0.41944254720865604,0.132656164313792,0.11756036140350157,0.6679143838321242,0.9291650800959559,0.67422385923251,0.636453221040939,0.6592320821018,0.8579068820107533,0.6358607988088959,0.49937490633948656,0.9734416160483854,0.9894031626177758,0.2164490400451171,0.330402492630149,0.5684802965086004,0.7285793555297794,0.19850218695310606,0.8237852450693551,0.9265532702411479,0.15689613525578394,0.2599213522423178,0.5005122626000685,0.45786852231643216,0.25443392462876147,0.33343340480780337,0.09278572373751104,0.014380254193779196,0.4443815497893878,0.4841217281728313,0.5341591234495388]},"selected":{"id":"24579"},"selection_policy":{"id":"24596"}},"id":"24507","type":"ColumnDataSource"},{"attributes":{"axis":{"id":"24553"},"ticker":null},"id":"24556","type":"Grid"},{"attributes":{},"id":"24554","type":"BasicTicker"},{"attributes":{"callback":null},"id":"24526","type":"TapTool"}],"root_ids":["24591"]},"title":"Bokeh Application","version":"2.0.1"}}
        </script>
        <script type="text/javascript">
          (function() {
            var fn = function() {
              Bokeh.safely(function() {
                (function(root) {
                  function embed_document(root) {
                    
                  var docs_json = document.getElementById('24828').textContent;
                  var render_items = [{"docid":"73736b34-131c-4147-ac8e-d78b70400a90","root_ids":["24591"],"roots":{"24591":"6b01c24d-4af5-4bc1-b7d9-84c1216de234"}}];
                  root.Bokeh.embed.embed_items(docs_json, render_items);
                
                  }
                  if (root.Bokeh !== undefined) {
                    embed_document(root);
                  } else {
                    var attempts = 0;
                    var timer = setInterval(function(root) {
                      if (root.Bokeh !== undefined) {
                        clearInterval(timer);
                        embed_document(root);
                      } else {
                        attempts++;
                        if (attempts > 100) {
                          clearInterval(timer);
                          console.log("Bokeh: ERROR: Unable to run BokehJS code because BokehJS library is missing");
                        }
                      }
                    }, 10, root)
                  }
                })(window);
              });
            };
            if (document.readyState != "loading") fn();
            else document.addEventListener("DOMContentLoaded", fn);
          })();
        </script>
    
  </body>
  
</html>