



<!DOCTYPE html>
<html lang="en">
  
  <head>
    
      <meta charset="utf-8">
      <title>Bokeh Plot</title>
      
      
        
          
        
        
          
        <script type="text/javascript" src="https://cdn.bokeh.org/bokeh/release/bokeh-2.0.1.min.js" integrity="sha384-JpP8FXbgAZLkfur7LiK3j9AGBhHNIvF742meBJrjO2ShJDhCG2I1uVvW+0DUtrmc" crossorigin="anonymous"></script>
        <script type="text/javascript" src="https://cdn.bokeh.org/bokeh/release/bokeh-widgets-2.0.1.min.js" integrity="sha384-xZlADit0Q04ISQEdKg2k3L4W9AwQBAuDs9nJL9fM/WwzL1tEU9VPNezOFX0nLEAz" crossorigin="anonymous"></script>
        <script type="text/javascript" src="https://cdn.bokeh.org/bokeh/release/bokeh-tables-2.0.1.min.js" integrity="sha384-4BuPRZkdMKSnj3zoxiNrQ86XgNw0rYmBOxe7nshquXwwcauupgBF2DHLVG1WuZlV" crossorigin="anonymous"></script>
        <script type="text/javascript">
            Bokeh.set_log_level("info");
        </script>
        
      
      
    
  </head>
  
  
  <body>
    
      
        
          
          
            
              <div class="bk-root" id="782a74a2-695d-4689-b93d-48aa92951136" data-root-id="9135"></div>
            
          
        
      
      
        <script type="application/json" id="9372">
          {"68bcf5ce-f2f1-4881-8178-050bfeaa41c5":{"roots":{"references":[{"attributes":{"formatter":{"id":"9149"},"ticker":{"id":"9063"}},"id":"9062","type":"LinearAxis"},{"attributes":{"fill_color":{"field":"color"},"line_color":{"field":"color"},"radius":{"field":"rad","units":"data"},"x":{"field":"x"},"y":{"field":"y"}},"id":"9078","type":"Circle"},{"attributes":{},"id":"9098","type":"BasicTicker"},{"attributes":{"children":[{"id":"9086"},{"id":"9053"}]},"id":"9132","type":"Row"},{"attributes":{"data":{"authors":["Liu, Quan; Cao, Lili; Zhu, Xing-Quan","El-Duah, Philip; Meyer, Benjamin; Sylverken, Augustina; Owusu, Michael; Gottula, Lina Theresa; Yeboah, Richmond; Lamptey, Jones; Frimpong, Yaw Oppong; Burimuah, Vitus; Folitse, Raphael; Agbenyega, Olivia; Oppong, Samuel; Adu-Sarkodie, Yaw; Drosten, Christian","Shi, Weifeng; Li, Juan; Zhou, Hong; Gao, George F.","Smith, Ina; Wang, Lin-Fa","Lowenthal, John","H\u00f6rmansdorfer, Stefan; Campe, H.; Sing, A.","Poon, L. L. M.; Chu, D. K. W.; Chan, K. H.; Wong, O. K.; Ellis, T. M.; Leung, Y. H. C.; Lau, S. K. P.; Woo, P. C. Y.; Suen, K. Y.; Yuen, K. Y.; Guan, Y.; Peiris, J. S. M.","Ping Liu; Jing-Zhe Jiang; Xiu-Feng Wan; Yan Hua; Xiaohu Wang; Fanghui Hou; Jing Chen; Jiejian Zou; Jinping Chen","Huss, Anne; Cochrane, Roger; Jones, Cassie; Atungulu, Griffiths G.","Hu, Tingsong; Qiu, Wei; He, Biao; Zhang, Yan; Yu, Jing; Liang, Xiu; Zhang, Wendong; Chen, Gang; Zhang, Yingguo; Wang, Yiyin; Zheng, Ying; Feng, Ziliang; Hu, Yonghe; Zhou, Weiguo; Tu, Changchun; Fan, Quanshui; Zhang, Fuqiang","Reusken, Chantal BEM; Haagmans, Bart L; M\u00fcller, Marcel A; Gutierrez, Carlos; Godeke, Gert-Jan; Meyer, Benjamin; Muth, Doreen; Raj, V Stalin; Vries, Laura Smits-De; Corman, Victor M; Drexler, Jan-Felix; Smits, Saskia L; El Tahir, Yasmin E; De Sousa, Rita; van Beek, Janko; Nowotny, Norbert; van Maanen, Kees; Hidalgo-Hermoso, Ezequiel; Bosch, Berend-Jan; Rottier, Peter; Osterhaus, Albert; Gort\u00e1zar-Schmidt, Christian; Drosten, Christian; Koopmans, Marion PG","Daszak, Peter; Olival, Kevin J.; Li, Hongying","Deng, X.; Baker, S.C.","","Wang, Lin-Fa; Shi, Zhengli; Zhang, Shuyi; Field, Hume; Daszak, Peter; Eaton, Bryan T.","Yang, Weizhong; Lan, Yajia; Sun, Qiao; Wang, Jinfeng; Li, Zhongjie","Baker, Kate S.; Leggett, Richard M.; Bexfield, Nicholas H.; Alston, Mark; Daly, Gordon; Todd, Shawn; Tachedjian, Mary; Holmes, Clare E.G.; Crameri, Sandra; Wang, Lin-Fa; Heeney, Jonathan L.; Suu-Ire, Richard; Kellam, Paul; Cunningham, Andrew A.; Wood, James L.N.; Caccamo, Mario; Murcia, Pablo R.","Lipkin, W. Ian","Grubaugh, Nathan D.; Saraf, Sharada; Gangavarapu, Karthik; Watts, Alexander; Tan, Amanda L.; Oidtman, Rachel J.; Ladner, Jason T.; Oliveira, Glenn; Matteson, Nathaniel L.; Kraemer, Moritz U.G.; Vogels, Chantal B.F.; Hentoff, Aaron; Bhatia, Deepit; Stanek, Danielle; Scott, Blake; Landis, Vanessa; Stryker, Ian; Cone, Marshall R.; Kopp, Edgar W.; Cannons, Andrew C.; Heberlein-Larson, Lea; White, Stephen; Gillis, Leah D.; Ricciardi, Michael J.; Kwal, Jaclyn; Lichtenberger, Paola K.; Magnani, Diogo M.; Watkins, David I.; Palacios, Gustavo; Hamer, Davidson H.; Gardner, Lauren M.; Perkins, T. Alex; Baele, Guy; Khan, Kamran; Morrison, Andrea; Isern, Sharon; Michael, Scott F.; Andersen, Kristian G.","Qiang, Xiao-Li; Xu, Peng; Fang, Gang; Liu, Wen-Bin; Kou, Zheng","Berger, A.; Preiser, W.","Mi\u0142ek, Justyna; Blicharz-Doma\u0144ska, Katarzyna","Berger, A.; Preiser, W.","Kun Sun; Liuqi Gu; Li Ma; Yunfeng Duan","Zhang, Yongjiang; Yin, Jun; Li, Guifen; Li, Mingfu; Huang, Xin; Chen, Hongjun; Zhao, Wenjun; Zhu, Shuifang","Mohd, Hamzah A.; Al-Tawfiq, Jaffar A.; Memish, Ziad A.","Pavli, Androula; Tsiodras, Sotirios; Maltezou, Helena C.","Foddai, Alessandro; Lindberg, Ann; Lubroth, Juan; Ellis-Iversen, Johanne","Qinghe Liu; Zhicheng Liu; Junkai Zhu; Yuhao Zhu; Deqiang Li; Zefei Gao; Liuling Zhou; Junyan Yang; Qiao Wang","Fredricks, David N; Relman, David A","Chu, Daniel K. W.; Peiris, J. S. Malik; Poon, Leo L. M.","Mackenzie, John S; Jeggo, Martyn","Muradrasoli, Shaman; B\u00e1lint, \u00c1d\u00e1m; Wahlgren, John; Waldenstr\u00f6m, Jonas; Bel\u00e1k, S\u00e1ndor; Blomberg, Jonas; Olsen, Bj\u00f6rn","Brisse, Sylvain; Brehony, Carina; Concei\u00e7\u00e3o, Teresa; Cubero, Meritxell; Glasner, Corinna; Le Gouil, Meriadeg; Renvois\u00e9, Aur\u00e9lie; Sheppard, Sam; Weinert, Lucy A.","Javier A. Jaimes; Nicole M. Andr\u00e9; Jean K. Millet; Gary R. Whittaker","Tu, Wenxiao; Jin, Lianmei; Ni, Daxin","Khalili Bagaloy, Heydar; Sakhaee, Ehsanollah; Khalili, Mohammad","Dahlhausen, Bob","Jackwood, Mark W.; Boynton, Tye O.; Hilt, Deborah A.; McKinley, Enid T.; Kissinger, Jessica C.; Paterson, Andrew H.; Robertson, Jon; Lemke, Conelia; McCall, Amber W.; Williams, Susan M.; Jackwood, Joshua W.; Byrd, Lauren A.","Chang, Yoon-Seok","Lu, Yanning; Liu, Ding Xiang; Tam, James P.","Liangde Xu; Jian Yuan; Yaru Zhang; Guosi Zhang; Fan Lu; Jianzhong Su; Jia Qu","Lau, Susanna K.P.; Woo, Patrick C.Y.; Li, Kenneth S.M.; Huang, Yi; Wang, Ming; Lam, Carol S.F.; Xu, Huifang; Guo, Rongtong; Chan, Kwok-hung; Zheng, Bo-jian; Yuen, Kwok-yung","Phan, Tung","Elachola, Habida; Ebrahim, Shahul H.; Gozzer, Ernesto","Gardy, Jennifer L.; Loman, Nicholas J.","Cauchemez, Simon; Epperson, Scott; Biggerstaff, Matthew; Swerdlow, David; Finelli, Lyn; Ferguson, Neil M.","Kim, Yongkwan; Son, Kidong; Kim, Young-Sik; Lee, Sook-Young; Jheong, Weonhwa; Oem, Jae-Ku","Buckee, Caroline","Arunachalam Ramaiah; Vaithilingaraja Arumugaswami"],"color":["#67a9cf","#67a9cf","#67a9cf","#67a9cf","#67a9cf","#67a9cf","#67a9cf","#67a9cf","#67a9cf","#67a9cf","#67a9cf","#67a9cf","#67a9cf","#67a9cf","#67a9cf","#67a9cf","#67a9cf","#67a9cf","#67a9cf","#67a9cf","#67a9cf","#67a9cf","#67a9cf","#67a9cf","#67a9cf","#67a9cf","#67a9cf","#67a9cf","#67a9cf","#67a9cf","#67a9cf","#67a9cf","#67a9cf","#67a9cf","#67a9cf","#67a9cf","#67a9cf","#67a9cf","#67a9cf","#67a9cf","#67a9cf","#67a9cf","#67a9cf","#67a9cf","#67a9cf","#67a9cf","#67a9cf","#67a9cf","#67a9cf","#67a9cf"],"date":["2014-08-31","2019-01-09","2017-11-28","2013-02-28","2016-06-30","2008-11-14","2005-01-28","2020-02-20","2018-12-31","2014-11-30","2013-10-31","2020","2014-12-31","2020-02-20","2006-12-03","2017-12-31","2013-07-05","2013-01-03","2019-08-22","2020-03-25","2011-12-31","2018-12-10","2011-12-31","2020-03-31","2010-07-31","2016-06-03","2014-12-31","2020-03-27","2020-03-20","2005-03-01","2009-04-14","2013-04-30","2010-10-29","2014-03-31","2020-02-18","2017-12-31","2016-11-18","2010-04-30","2010-03-01","2020-01-31","2008-05-02","2020-02-29","2007-10-25","2020-07-31","2020-03-20","2017-11-13","2013-03-05","2019-05-10","2020-03-17","2020-01-30"],"doc":["p1jbb1wa","tjp33y35","k0f9fser","v8uzd2f3","13adnvav","8fxst74b","b6abkv7p","2inlyd0t","hqm3p9za","1fw9ng2p","1w8epdht","h5sox8bq","eqfz0wpm","x64cwmqf","v08cs51n","dgi4ibym","rwv5ibg3","0i3ewil9","m1o3g2bz","llv3cvdr","kromgrnu","ur8g68vp","yvsyo4l9","89sg0cpk","j8fqan9x","yzp9wjuk","x1f4aqme","5j4nt2qs","adcptyfj","cisrvghj","g5fqxdqh","0g5079y1","9le7uk7m","a4mf2vnp","xa6kwguo","bjub9j3c","haci3hzm","o976nab9","9ojile3q","emeb5ge7","6kjjqsxu","acmfhbtm","z2kgcjtp","0nh58odf","hvmw7g5q","th1da1bb","16c8dwfq","jojtdu3a","yxms5vqh","xxcx7hg5"],"journal":["International Journal of Infectious Diseases","Viruses","Sci China Life Sci","Current Opinion in Virology","Journal of Infection and Public Health","J Verbrauch Lebensm","Journal of Virology","","Food and Feed Safety Systems and Analysis","BMC Microbiol","The Lancet Infectious Diseases","Biosafety and Health","Reference Module in Biomedical Sciences","Nat Ecol Evol","Emerg Infect Dis","Early Warning for Infectious Disease Outbreak","Virology","Nature Reviews Microbiology","Cell","Infect Dis Poverty","Encyclopedia of Environmental Health","J Vet Res","Encyclopedia of Environmental Health","","Journal of Virological Methods","Virol J","Travel Medicine and Infectious Disease","One Health","","Medicine","Virol Sin","Current Opinion in Virology","PLoS One","Research in Microbiology","","Early Warning for Infectious Disease Outbreak","Comp Clin Path","Journal of Exotic Pet Medicine","Virology","Asia Pac Allergy","Biochemical and Biophysical Research Communications","","Virology","Infection, Genetics and Evolution","Travel Medicine and Infectious Disease","Nat Rev Genet","PLoS Med","Virus Genes","The Lancet Digital Health",""],"rad":[0.016600006834374,0.016278873111013625,0.015473177712343134,0.013956397269234824,0.013953267456889556,0.011113216042598517,0.011058796447567878,0.010356509448185879,0.0090221722248848,0.008846586694482123,0.008069241177753818,0.007592328088600159,0.0072550657634032475,0.0071234047128067345,0.006953413830690094,0.006548915804464923,0.006541873508663518,0.006412417632021007,0.0061271708792181075,0.006076191681982476,0.006047795818220025,0.006023848183901495,0.006006255283364651,0.005947846472871747,0.005928064291179236,0.005733074650758953,0.005470310757589289,0.005072483837378799,0.0050071172872898344,0.004972097631854265,0.004966722933554454,0.004794037302602889,0.004753658476771216,0.004718262080312499,0.004685057068115852,0.004611452314698166,0.0042897846677907555,0.004128861679889308,0.003923691861801002,0.0038251142961320053,0.0037509885946124194,0.0037302099319789727,0.003613235862131119,0.003560570138,0.0034369234050083243,0.003369395563934243,0.003299918082021375,0.00326792362864211,0.003018073712173335,0.0030146767695757755],"scr":[0.8,0.781,0.734,0.644,0.644,0.477,0.474,0.433,0.354,0.344,0.298,0.27,0.25,0.243,0.233,0.209,0.208,0.201,0.184,0.181,0.179,0.178,0.177,0.173,0.172,0.161,0.145,0.122,0.118,0.116,0.116,0.106,0.103,0.101,0.099,0.095,0.076,0.066,0.054,0.049,0.044,0.043,0.036,0.033,0.026,0.022,0.018,0.016,0.001,0.001],"text":["Summary Emerging and re-emerging zoonoses are a significant public health concern and cause considerable socioeconomic problems globally. The emergence of severe acute respiratory syndrome (SARS), highly pathogenic avian influenza (HPAI) H5N1, avian influenza H7N9, and severe fever with thrombocytopenia syndrome (SFTS), and the re-emergence of rabies, brucellosis, and other zoonoses have had a significant effect on the national economy and public health in China, and have affected other countries. Contributing factors that continue to affect emerging and re-emerging zoonoses in China include social and environmental factors and microbial evolution, such as population growth, urbanization, deforestation, livestock production, food safety, climate change, and pathogen mutation. The Chinese government has devised new strategies and has taken measures to deal with the challenges of these diseases, including the issuing of laws and regulations, establishment of disease reporting systems, im...","Known human coronaviruses are believed to have originated in animals and made use of intermediate hosts for transmission to humans. The intermediate hosts of most of the human coronaviruses are known, but not for HCoV-NL63. This study aims to assess the possible role of some major domestic livestock species as intermediate hosts of HCoV-NL63. We developed a testing algorithm for high throughput screening of livestock sera with ELISA and confirmation with recombinant immunofluorescence assay testing for antibodies against HCoV-NL63 in livestock. Optimization of the ELISA showed a capability of the assay to significantly distinguish HCoV-NL63 from HCoV-229E (U = 27.50, p &lt; 0.001) and HCoV-OC43 (U = 55.50, p &lt; 0.001) in coronavirus-characterized sera. Evaluation of the assay with collected human samples showed no significant difference in mean optical density values of immunofluorescence-classified HCoV-NL63-positive and HCoV-NL63-negative samples (F (1, 215) = 0.437, p = 0.509). All the ...","In the past twenty years, numerous novel zoonotic viral agents with pandemic potential have emerged in China, such as the severe acute respiratory syndrome (SARS) coronavirus and, more recently, the avian-origin influenza A/H7N9 virus, which have caused outbreaks among humans with high morbidity and mortality. In addition, several emerging and re-emerging viral pathogens have also been imported into China from travelers, e.g. the Middle East respiratory syndrome (MERS) coronavirus and Zika virus (ZIKV). Herein, we review these emerging viral pathogens in China and focus on how surveillance by pathogen genomics has been employed to discover and annotate novel pathogenic agents, identify natural reservoirs, monitor the transmission events and delineate their evolution and adaption to the human host. We also highlight the application of genomic sequencing in the recent Ebola epidemics in Western Africa. In summary, genomic sequencing has become a standard research tool in the field of eme...","Bats are being increasingly recognized as an important reservoir of zoonotic viruses of different families, including SARS coronavirus, Nipah virus, Hendra virus and Ebola virus. Several recent studies hypothesized that bats, an ancient group of flying mammals, are the major reservoir of several important RNA virus families from which other mammalian viruses of livestock and humans were derived. Although this hypothesis needs further investigation, the premise that bats carry a large number of viruses is commonly accepted. The question of whether bats have unique biological features making them ideal reservoir hosts has been the subject of several recent reviews. In this review, we will focus on the public health implications of bat derived zoonotic viral disease outbreaks, examine the drivers and risk factors of past disease outbreaks and outline research directions for better control of future disease events.","Summary Emerging infectious diseases arising from livestock and wildlife pose serious threats to global human health, as shown by a series of continuous outbreaks involving highly pathogenic influenza, SARS, Ebola and MERS. The risk of pandemics and bioterrorism threats is ever present and growing, but our ability to combat them is limited by the lack of available vaccines, therapeutics and rapid diagnostics. The use of high bio-containment facilities, such as the CSIRO Australian Animal Health Laboratory, plays a key role studying these dangerous pathogens and facilitates the development of countermeasures. To combat diseases like MERS, we must take a holistic approach that involves the development of early biomarkers of infection, a suite of treatment options (vaccines, anti-viral drugs and antibody therapeutics) and appropriate animal models to test the safety and efficacy of candidate treatments.","The SARS-epidemic of 2002/2003 with worldwide 8.096 cases and 774 fatalities was the first pandemia of the 21(st) century. SARS, the severe acute respiratory syndrome, arose in southern China and spread from Southeast-Asia finally over all five continents. It caused heavy pneumonia with pulmonal failure and enteric involvement in man. The causative agent was a novel coronavirus (SARS-CoV), which was transmitted from bats to small carnivores and from them to man. The mutations of the viral receptor gene thus allowed the infection of man and the transmission from man to man. The SARS-pandemia can therefore be regarded as a model of an emerging disease.","Exotic wildlife can act as reservoirs of diseases that are endemic in the area or can be the source of new emerging diseases through interspecies transmission. The recent emergence of severe acute respiratory syndrome-associated coronavirus (SARS-CoV) highlights the importance of virus surveillance in wild animals. Here, we report the identification of a novel bat coronavirus through surveillance of coronaviruses in wildlife. Analyses of the RNA sequence from the ORF1b and S-gene regions indicated that the virus is a group 1 coronavirus. The virus was detected in fecal and respiratory samples from three bat species (Miniopterus spp.). In particular, 63% (12 of 19) of fecal samples from Miniopterus pusillus were positive for the virus. These findings suggest that this virus might be commonly circulating in M. pusillus in Hong Kong.","AbstractThe outbreak of 2019-nCoV pneumonia (COVID-19) in the city of Wuhan, China has resulted in more than 70,000 laboratory confirmed cases, and recent studies showed that 2019-nCoV (SARS-CoV-2) could be of bat origin but involve other potential intermediate hosts. In this study, we assembled the genomes of coronaviruses identified in sick pangolins. The molecular and phylogenetic analyses showed that pangolin Coronaviruses (pangolin-CoV) are genetically related to both the 2019-nCoV and bat Coronaviruses but do not support the 2019-nCoV arose directly from the pangolin-CoV. Our study also suggested that pangolin be natural host of Betacoronavirus, with a potential to infect humans. Large surveillance of coronaviruses in pangolins could improve our understanding of the spectrum of coronaviruses in pangolins. Conservation of wildlife and limits of the exposures of humans to wildlife will be important to minimize the spillover risks of coronaviruses from wild animals to humans.","Abstract Feed for livestock animals have been under increased scrutiny as a vector for pathogenic bacteria leading to human illnesses. Control of these pathogenic bacteria in animal feeds can be controlled via physical and chemical means. Physical methods can include thermal processing, including irradiation, pelleting, and extrusion. Chemical mitigation can be accomplished through the inclusion of various additives, including formaldehyde, organic acids, essential oils, or medium chain fatty acids. While physical and chemical interventions can help mitigate risk of pathogens contaminating finished animal feeds, the implementation of a biosecurity plan at the feed mill can help prevent pathogens from entering or spreading throughout the facility.","BACKGROUND: In recent years novel human respiratory disease agents have been described for Southeast Asia and Australia. The causative pathogens were classified as pteropine orthoreoviruses with a strong phylogenetic relationship to orthoreoviruses of bat origin. RESULTS: In this report, we isolated a novel Melaka-like reovirus (named \u201cCangyuan virus\u201d) from intestinal content samples of one fruit bat residing in China\u2019s Yunnan province. Phylogenetic analysis of the whole Cangyuan virus genome sequences of segments L, M and S demonstrated the genetic diversity of the Cangyuan virus. In contrast to the L and M segments, the phylogenetic trees for the S segments of Cangyuan virus demonstrated a greater degree of heterogeneity. CONCLUSIONS: Phylogenetic analysis indicated that the Cangyuan virus was a novel orthoreovirus and substantially different from currently known members of Pteropine orthoreovirus (PRV) species group. ELECTRONIC SUPPLEMENTARY MATERIAL: The online version of this arti...","Summary Background A new betacoronavirus\u2014Middle East respiratory syndrome coronavirus (MERS-CoV)\u2014has been identified in patients with severe acute respiratory infection. Although related viruses infect bats, molecular clock analyses have been unable to identify direct ancestors of MERS-CoV. Anecdotal exposure histories suggest that patients had been in contact with dromedary camels or goats. We investigated possible animal reservoirs of MERS-CoV by assessing specific serum antibodies in livestock. Methods We took sera from animals in the Middle East (Oman) and from elsewhere (Spain, Netherlands, Chile). Cattle (n=80), sheep (n=40), goats (n=40), dromedary camels (n=155), and various other camelid species (n=34) were tested for specific serum IgG by protein microarray using the receptor-binding S1 subunits of spike proteins of MERS-CoV, severe acute respiratory syndrome coronavirus, and human coronavirus OC43. Results were confirmed by virus neutralisation tests for MERS-CoV and bovine ...","A novel bat-origin coronavirus emerged in Wuhan, China in December 2019 and continues to spread across China and the world. At the time of writing, a massive global response has been implemented to control the disease as it spreads from person to person. Yet the high-risk human-wildlife interactions and interfaces that led to the emergence of SARS-CoV and of 2019-nCoV continue to exist in emerging disease hotspots globally. To prevent the next pandemic related to these interfaces, we call for research and investment in three areas: 1) surveillance among wildlife to identify the high-risk pathogens they carry; 2) surveillance among people who have contact with wildlife to identify early spillover events; and 3) improvement of market biosecurity regarding the wildlife trade. As the emergence of a novel virus anywhere can impact the furthest reaches of our connected world, international collaboration among scientists is essential to address these risks and prevent the next pandemic.","Abstract Coronaviruses (CoVs) are enveloped, positive-strand RNA viruses with characteristic spike glycoproteins that project outward like the rays of the sun (corona \u2013 Latin for \u2018crown\u2019), when visualized by electron microscopy. CoV are classified, together with the toroviruses, in the family Coronaviridae and the order Nidovirales. All nidoviruses have a common genome organization and generate a nested set (nido \u2013 Latin for \u2018nest\u2019) of 3\u2032 co-terminal mRNAs. CoVs have been isolated from a variety of species, including birds, livestock, domestic animals, and humans. CoV infections can cause respiratory, gastrointestinal, and neurologic disease, depending on the strain of the virus and the site of infection. Importantly, CoVs have been shown to cross species barriers and have emerged from animal reservoirs to infect humans and cause severe disease. The CoV responsible for an outbreak of severe acute respiratory disease (SARS-CoV) in 2002\u201303 likely originated as a bat coronavirus which, du...","As the COVID-19 outbreak continues, the next pandemic could be prevented by ending the wildlife trade and reinvesting in the monitoring of potential zoonoses.","Bats have been identified as a natural reservoir for an increasing number of emerging zoonotic viruses, including henipaviruses and variants of rabies viruses. Recently, we and another group independently identified several horseshoe bat species (genus Rhinolophus) as the reservoir host for a large number of viruses that have a close genetic relationship with the coronavirus associated with severe acute respiratory syndrome (SARS). Our current research focused on the identification of the reservoir species for the progenitor virus of the SARS coronaviruses responsible for outbreaks during 2002\u20132003 and 2003\u20132004. In addition to SARS-like coronaviruses, many other novel bat coronaviruses, which belong to groups 1 and 2 of the 3 existing coronavirus groups, have been detected by PCR. The discovery of bat SARS-like coronaviruses and the great genetic diversity of coronaviruses in bats have shed new light on the origin and transmission of SARS coronaviruses.","Abstract This chapter provides an overview of the basic theory of early warning for infectious diseases, including definitions of surveillance, early warning and prediction; the characteristics of early warning (surveillance, information for action, timeliness, insufficiency, and the uncertainty of information); a conceptual framework of an early warning system for infectious diseases (the target setting, data collection and analysis, disseminating early warning signals, response actions, and evaluation); classifying early warning based on the data sources (case-based, event-based, laboratory-based, or syndromic surveillance). The significance of early warning for infectious diseases, the status quo and research and application trends in the field are also highlighted.","Viral emergence as a result of zoonotic transmission constitutes a continuous public health threat. Emerging viruses such as SARS coronavirus, hantaviruses and henipaviruses have wildlife reservoirs. Characterising the viruses of candidate reservoir species in geographical hot spots for viral emergence is a sensible approach to develop tools to predict, prevent, or contain emergence events. Here, we explore the viruses of Eidolon helvum, an Old World fruit bat species widely distributed in Africa that lives in close proximity to humans. We identified a great abundance and diversity of novel herpes and papillomaviruses, described the isolation of a novel adenovirus, and detected, for the first time, sequences of a chiropteran poxvirus closely related with Molluscum contagiosum. In sum, E. helvum display a wide variety of mammalian viruses, some of them genetically similar to known human pathogens, highlighting the possibility of zoonotic transmission.","The pace of pathogen discovery is increasing dramatically. This reflects not only factors that enable the appearance and globalization of new microbial infections but also improvements in methods for ascertainment. New molecular diagnostic platforms; investments in pathogen surveillance in wildlife, domestic animals and humans; and the advent of social media tools that mine the world wide web for clues to outbreaks of infectious disease are proving invaluable in early recognition of threats to public health. Additionally, models of microbial pathogenesis are becoming more complex, providing insights into the mechanisms by which microorganisms can contribute to chronic illnesses like cancer, peptic ulcer disease and mental illness. Here we review the contributions of each of these elements to infectious disease emergence and strategies for addressing the challenges of pathogen surveillance and discovery.","Summary The Zika epidemic in the Americas has challenged surveillance and control. As the epidemic appears to be waning, it is unclear whether transmission is still ongoing, which is exacerbated by discrepancies in reporting. To uncover locations with lingering outbreaks, we investigated travel-associated Zika cases to identify transmission not captured by reporting. We uncovered an unreported outbreak in Cuba during 2017, a year after peak transmission in neighboring islands. By sequencing Zika virus, we show that the establishment of the virus was delayed by a year and that the ensuing outbreak was sparked by long-lived lineages of Zika virus from other Caribbean islands. Our data suggest that, although mosquito control in Cuba may initially have been effective at mitigating Zika virus transmission, such measures need to be maintained to be effective. Our study highlights how Zika virus may still be \u201csilently\u201d spreading and provides a framework for understanding outbreak dynamics. Vi...","BACKGROUND: Coronavirus can cross the species barrier and infect humans with a severe respiratory syndrome. SARS-CoV-2 with potential origin of bat is still circulating in China. In this study, a prediction model is proposed to evaluate the infection risk of non-human-origin coronavirus for early warning. METHODS: The spike protein sequences of 2666 coronaviruses were collected from 2019 Novel Coronavirus Resource (2019nCoVR) Database of China National Genomics Data Center on Jan 29, 2020. A total of 507 human-origin viruses were regarded as positive samples, whereas 2159 non-human-origin viruses were regarded as negative. To capture the key information of the spike protein, three feature encoding algorithms (amino acid composition, AAC; parallel correlation-based pseudo-amino-acid composition, PC-PseAAC and G-gap dipeptide composition, GGAP) were used to train 41 random forest models. The optimal feature with the best performance was identified by the multidimensional scaling method, ...","Severe acute respiratory syndrome (SARS) emerged in southern China in late 2002. It first spread within Guangdong Province and then to other parts of China. Via air travelers, it quickly reached various countries around the globe, causing several major hospital outbreaks. Within weeks, the causative agent, a previously unknown coronavirus (SARS-CoV), was identified, thanks to an unprecedented international effort led by the World Health Organization (WHO). Its origin was quickly traced to wild animals traded locally for culinary purposes. Masked palm civet and some other species seem to have acted as intermediate hosts. Since then, SARS-like coronaviruses were found in different bat species in China and elsewhere, and bats are now regarded as the wildlife reservoir for SARS-CoV. Fortunately, the SARS outbreak could be contained within months. Until July 2003, it had caused 8096 cases, with 774 deaths. Once adequate measures such as isolating patients and quarantining their contacts wer...","Coronaviruses (CoVs) are a large group of enveloped viruses with a single-strand RNA genome, which continuously circulate in mammals and birds and pose a threat to livestock, companion animals, and humans. CoVs harboured by avian species are classified to the genera gamma- and deltacoronaviruses. Within the gamma-CoVs the main representative is avian coronavirus, a taxonomic name which includes the highly contagious infectious bronchitis viruses (IBVs) in chickens and similar viruses infecting other domestic birds such as turkeys, guinea fowls, or quails. Additionally, IBVs have been detected in healthy wild birds, demonstrating that they may act as the vector between domestic and free-living birds. Moreover, CoVs other than IBVs, are identified in wild birds, which suggests that wild birds play a key role in the epidemiology of other gammaCoVs and deltaCoVs. Development of molecular techniques has significantly improved knowledge of the prevalence of CoVs in avian species. The methods...","Abstract Severe acute respiratory syndrome (SARS) emerged in southern China in late 2002. It first spread within Guangdong Province and then to other parts of China. Via air travelers, it quickly reached various countries around the globe, causing several major hospital outbreaks. Within weeks, the causative agent, a previously unknown coronavirus (SARS-CoV), was identified, thanks to an unprecedented international effort led by the World Health Organization (WHO). Its origin was quickly traced to wild animals traded locally for culinary purposes. Masked palm civet and some other species seem to have acted as intermediate hosts. Since then, SARS-like coronaviruses were found in different bat species in China and elsewhere, and bats are now regarded as the wildlife reservoir for SARS-CoV. Fortunately, the SARS outbreak could be contained within months. Until July 2003, it had caused 8096 cases, with 774 deaths. Once adequate measures such as isolating patients and quarantining their con...","Background: COVID-19 has become a worldwide pandemic. It is caused by a novel coronavirus named SARS-CoV-2 with elusive origin. SARS-CoV-2 infects mammalian cells by binding to ACE2, a transmembrane protein. Therefore, the conservation of ACE2 and its expression pattern across mammalian species, which are yet to be comprehensively investigated, may provide valuable insights into tracing potential hosts of SARS-CoV-2. Methods: We analyzed gene conservation of ACE2 across mammals and collected more than 140 transcriptome datasets from human and common mammalian species, including presumed hosts of SARS-CoV-2 and other animals in close contact with humans. In order to enable comparisons across species and tissues, we used a unified pipeline to quantify and normalize ACE2 expression levels. Results: We first found high conservation of ACE2 genes among common mammals at both DNA and peptide levels, suggesting that a broad range of mammalian species can potentially be the hosts of SARS-CoV-2...","Abstract A major challenge facing agriculture at present is the development of techniques that can screen field samples and other plant materials simultaneously for the presence of many viruses. Microarray techniques show promise in this regard, as their high throughput nature can potentially detect a range of viruses using a single test. In this paper we present an array that can detect a wide spectrum of 169 plant virus species from 13 different genera. The array was constructed using an automated probe design protocol which generated a minimal number of probes to detect viruses at the genus level. The designed arrays showed a high specificity and sensitivity when tested with a set of standard virus samples. Field samples collected from a severe disease outbreak of Panax notoginseng farms in Yunnan, China, in 2001 were screened, where a potyvirus infection was identified associated with the disease.","Middle East Respiratory Syndrome-Coronavirus (MERS-CoV) is a novel coronavirus discovered in 2012 and is responsible for acute respiratory syndrome in humans. Though not confirmed yet, multiple surveillance and phylogenetic studies suggest a bat origin. The disease is heavily endemic in dromedary camel populations of East Africa and the Middle East. It is unclear as to when the virus was introduced to dromedary camels, but data from studies that investigated stored dromedary camel sera and geographical distribution of involved dromedary camel populations suggested that the virus was present in dromedary camels several decades ago. Though bats and alpacas can serve as potential reservoirs for MERS-CoV, dromedary camels seem to be the only animal host responsible for the spill over human infections.","Summary Middle East respiratory syndrome coronavirus (MERS-CoV), a novel coronavirus that causes a severe lower respiratory tract infection in humans, emerged in the Middle East in 2012. Since then, MERS-CoV has caused an ongoing epidemic in the Arabian Peninsula with sporadic cases imported in Europe, North Africa, Southeast Asia, and the United States of America. As of 28th May 2014, 636 laboratory-confirmed cases of infection with MERS-CoV have been reported to World Health Organization including 14 cases imported by travelers. The epicenter of the current MERS-CoV epidemic is located in Saudi Arabia, where millions of pilgrims travel for two mass gatherings annually. In this review we summarize MERS-CoV cases in relation to travel with focus on the epidemiology and prevention in travelers. It is important to increase awareness of travelers about the risks and appropriate preventive measures and for health professionals to be on alert if a patient with severe respiratory symptoms re...","Abstract During the first few months of 2020, the COVID-19 pandemic has reached Europe. Health systems all over the world are trying to control the outbreak in the shortest possible time. Exotic disease outbreaks are not uncommon in animal health and randomised surveillance is frequently used as support for decision-making. This editorial discusses the possibilities of practicing One Health, by using methods from animal health to enhance surveillance for COVID-19 to provide an evidence base fort decision-making in communities and countries.","COVID-19 is now widely spreading around the world as a global pandemic, whereas it had been effecitvely under control in China by March 15. In this report, we estimate the global tendency of COVID-19 and analyze the associated global epidemic risk, given that the status quo is continued without further measures being taken. The results show that the global R_0, excluding China, is estimated to be 2.63 (95% CI: 1.67 - 4.69). Seven countries, including Italy and Republic of Korea, have peak values over 10,000. And the peak arrival time of all these countries is after May 12, 2020 (95 % CI: April 1, 2020 - June 24, 2020). We predict that there are four regional clusters of the outbreak: Southeast Asia extending southward to Oceania, the Middle East, Western Europe and North America. Among them, Western Europe will become the major center of the outbreak. The peak values in Spain and Italy are estimated to be 232,661 and 108,062, respectively. The total number of peak values in Europe will...","Abstract There are many diseases for which a microbial aetiology is suspected. The hypothesis that a disease has an infectious cause is supported by: clinical features (similar to those of known infectious diseases, e.g. fever, leucocytosis), epidemiology (case clustering in time or location), histology (inflammation of affected tissues, e.g. granulomata) or characteristic microbial structures, treatment (clinical response to antimicrobial treatment), and prevention of disease by vaccines targeting microbial antigens. Proof that a microbe causes a disease requires more rigorous evidence. Future attempts to identify novel microbes associated with human disease may use sequence-based approaches. High-throughput sequencing may allow identification of unique microbial nucleic acid sequences in a background of host DNA. The complete sequencing of the human genome and multiple microbial genomes make this approach more feasible. DNA microarrays are also likely to be used in the search for nov...","Zoonotic transmissions of emerging pathogens from wildlife to human have shaped the history of mankind. These events have also highlighted our poor understanding of microorganisms circulated in wild animals. Coronaviruses and astroviruses, which can be found from a wide range of mammals, were recently detected in bats. Strikingly, these bat viruses are genetically highly diverse and these interesting findings might help to better understand the evolution and ecology of these viruses. The discoveries of these novel bats viruses not only suggested that bats are important hosts for these virus families, but also reiterated the role of bats as a reservoir of viruses that might pose a zoonotic threat to human health.","Wildlife, especially mammals and birds, are hosts to an enormous number of viruses, most of which we have absolutely no knowledge about even though we know these viruses circulate readily in their specific niches. More often than not, these viruses are silent or asymptomatic in their natural hosts. In some instances, they can infect other species, and in rare cases, this cross-species transmission might lead to human infection. There are also instances where we know the reservoir hosts of zoonotic viruses that can and do infect humans. Studies of these animal hosts, the reservoirs of the viruses, provide us with the knowledge of the types of virus circulating in wildlife species, their incidence, pathogenicity for their host, and in some instances, the potential for transmission to other hosts. This paper describes examples of some of the viruses that have been detected in wildlife, and the reservoir hosts from which they have been detected. It also briefly explores the spread of arthr...","Coronaviruses (CoVs) can cause mild to severe disease in humans and animals, their host range and environmental spread seem to have been largely underestimated, and they are currently being investigated for their potential medical relevance. Infectious bronchitis virus (IBV) belongs to gamma-coronaviruses and causes a costly respiratory viral disease in chickens. The role of wild birds in the epidemiology of IBV is poorly understood. In the present study, we examined 1,002 cloacal and faecal samples collected from 26 wild bird species in the Beringia area for the presence of CoVs, and then we performed statistical and phylogenetic analyses. We detected diverse CoVs by RT-PCR in wild birds in the Beringia area. Sequence analysis showed that the detected viruses are gamma-coronaviruses related to IBV. These findings suggest that wild birds are able to carry gamma-coronaviruses asymptomatically. We concluded that CoVs are widespread among wild birds in Beringia, and their geographic sprea...","","AbstractThe 2019 novel coronavirus (2019-nCoV) is currently causing a widespread outbreak centered on Hubei province, China and is a major public health concern. Taxonomically 2019-nCoV is closely related to SARS-CoV and SARS-related bat coronaviruses, and it appears to share a common receptor with SARS-CoV (ACE-2). Here, we perform structural modeling of the 2019-nCoV spike glycoprotein. Our data provide support for the similar receptor utilization between 2019-nCoV and SARS-CoV, despite a relatively low amino acid similarity in the receptor binding module. Compared to SARS-CoV, we identify an extended structural loop containing basic amino acids at the interface of the receptor binding (S1) and fusion (S2) domains, which we predict to be proteolytically-sensitive. We suggest this loop confers fusion activation and entry properties more in line with MERS-CoV and other coronaviruses, and that the presence of this structural loop in 2019-nCoV may affect virus stability and transmission.","Abstract On the Internet, there is a lot of open access information related to infectious diseases, e.g., official announcement, media news, bulletin boards, blogs, and other social media. If proper data retrieval and screening methods were applied on these Internet-based data, it would be very useful to improve the timeliness of early detection for infectious diseases. In China, we practice early warning using Internet-based data with routine intelligence screening assessment every day, and respond to any unusual event in a timely manner. Through this mechanism, the surveillance and early warning of Internet-based data for infectious disease and public health emergency plays an important supplemental role to the traditional surveillance systems.","Coronaviride is a colossal family of viruses that cause a variety of diseases in humans and other animals. As of late, a novel coronavirus, not anterior-optically discerned in humans, has been identified in a denizen of the Middle East. There is growing evidence that the Camelus dromedarius is host species for the virus and plays an important role of a source of human infection. Along these lines, the authors decided to detect coronaviruses in dromedary camels in two high-risk areas of Iran by employing an reverse transcription polymerase chain reaction (RT-PCR) assay. In the present study, nasal swab specimens were collected from 98 camels (C. dromedarius) traditionally reared in southeast and northwest of Iran. The detection of pancoronavirus was carried out, using RT-PCR. Pancoronavirus RNA was observed in seven cases among 98 nasal swab samples. Among these, 4 positive samples belonged to Azerbaijan province located in northwest of Iran and 3 positive samples were taken from southe...","Abstract The development of rapid, accurate, and sensitive diagnostic methods for detecting pathogens is the basis for treating, controlling, and eradicating infectious diseases of veterinary importance. Scientific and technological advancements have revolutionized the field of veterinary diagnostics. Genome sequencing has allowed efficient, sensitive, and specific diagnostic assays to be developed based on the detection of nucleic acids. The integration of advances in biochemistry, proteomics, engineering, and medicine offers enormous potential for the rapid and accurate diagnosis of viral, microbial, genetic, and metabolic disease. In the future, polymerase chain reaction assays, microarray testing, genomic analysis, and metabolic profiling will be accomplished in a rapid, portable, sensitive, and cost-efficient manner.","Abstract Analyses of turkey coronavirus (TCoV), an enteric disease virus that is highly similar to infectious bronchitis virus (IBV) an upper-respiratory tract disease virus in chickens, were conducted to determine the adaptive potential, and genetic changes associated with emergence of this group 3 coronavirus. Strains of TCoV that were pathogenic in poults and nonpathogenic in chickens did not adapt to cause disease in chickens. Comparative genomics revealed two recombination sites that replaced the spike gene in IBV with an unidentified sequence likely from another coronavirus, resulting in cross-species transmission and a pathogenicity shift. Following emergence in turkeys, TCoV diverged to different serotypes through the accumulation of mutations within spike. This is the first evidence that recombination can directly lead to the emergence of new coronaviruses and new coronaviral diseases, emphasizing the importance of limiting exposure to reservoirs of coronaviruses that can serv...","","Abstract Lipid rafts often serve as an entry site for certain viruses. Here, we report that lipid rafts in Vero E6 cells are involved in the entry of severe acute respiratory syndrome coronavirus (SARS-CoV). Infectivity assay showed the integrity of lipid rafts was required for productive infection of pseudotyped SARS-CoV. Depletion of plasma membrane cholesterol with M\u03b2CD relocalized raft-resident marker caveolin-1 as well as SARS-CoV receptor ACE2 to a nonraft environment, but did not significantly change the surface expression of ACE2. M\u03b2CD-treatment inhibited infectivity of pseudotyped SARS-CoV by 90%. Biochemical fractionation and confocal imaging confirmed that ACE2 colocalized with raft-resident markers. Furthermore, an ectodomain of SARS-CoV S protein (S1188HA) could associate with lipid rafts after binding to its receptor, and colocalize with raft-resident marker ganglioside GM1. The binding of S1188HA was not affected by depleting plasma membrane cholesterol. Taken together, ...","In late December 2019, Chinese authorities reported a cluster of pneumonia cases of unknown aetiology in Wuhan, China. Wenzhou, as a southeast coastal city with the most cases outside Hubei Province, its policy control and epidemic projections have certain references for national and worldwide epidemic prevention and control. The maximum number of illness onset in Wenzhou appeared on January 26 (38 cases). Since January 27, the incidence per day in Wenzhou has continued to decline is likely to be due to isolation of people return to Wenzhou, implementation of region quarantine and suspension of public transportation system. Under the current control efforts, we estimate that the daily incidence by March 3-9, 2020 will drop to 0 in Wenzhou using SEIR model. The total number of affected people is 538.","Abstract Apart from bat-SARS-CoV, we have identified a novel group 1 coronavirus, bat-CoV HKU2, in Rhinolophus sinicus (Chinese horseshoe bats). Since it has been suggested that the receptor-binding motif (RBM) of SARS-CoV may have been acquired from a group 1 coronavirus, we conducted a surveillance study and identified bat-SARS-CoV and bat-CoV HKU2 in 8.7% and 7.5% respectively of R. sinicus in Hong Kong and Guangdong. Complete genome sequencing of four strains of bat-CoV HKU2 revealed the smallest coronavirus genome (27164 nucleotides) and a unique spike protein evolutionarily distinct from the rest of the genome. This spike protein, sharing similar deletions with other group 2 coronaviruses in its C-terminus, also contained a 15-amino acid peptide homologous to a corresponding peptide within the RBM of spike protein of SARS-CoV, which was absent in other coronaviruses except bat-SARS-CoV. These suggest a common evolutionary origin in the spike protein of bat-CoV HKU2, bat-SARS-CoV,...","Abstract COVID-19 is a viral respiratory illness caused by a new coronavirus called SARS-CoV-2. The World Health Organization declared the SARS-CoV-2 outbreak a global public health emergency. We performed genetic analyses of eighty-six complete or near-complete genomes of SARS-CoV-2 and revealed many mutations and deletions on coding and non-coding regions. These observations provided evidence of the genetic diversity and rapid evolution of this novel coronavirus.","","The recent Ebola and Zika epidemics demonstrate the need for the continuous surveillance, rapid diagnosis and real-time tracking of emerging infectious diseases. Fast, affordable sequencing of pathogen genomes \u2014 now a staple of the public health microbiology laboratory in well-resourced settings \u2014 can affect each of these areas. Coupling genomic diagnostics and epidemiology to innovative digital disease detection platforms raises the possibility of an open, global, digital pathogen surveillance system. When informed by a One Health approach, in which human, animal and environmental health are considered together, such a genomics-based system has profound potential to improve public health in settings lacking robust laboratory capacity. SUPPLEMENTARY INFORMATION: The online version of this article (doi:10.1038/nrg.2017.88) contains supplementary material, which is available to authorized users.","BACKGROUND: Prior to emergence in human populations, zoonoses such as SARS cause occasional infections in human populations exposed to reservoir species. The risk of widespread epidemics in humans can be assessed by monitoring the reproduction number R (average number of persons infected by a human case). However, until now, estimating R required detailed outbreak investigations of human clusters, for which resources and expertise are not always available. Additionally, existing methods do not correct for important selection and under-ascertainment biases. Here, we present simple estimation methods that overcome many of these limitations. METHODS AND FINDINGS: Our approach is based on a parsimonious mathematical model of disease transmission and only requires data collected through routine surveillance and standard case investigations. We apply it to assess the transmissibility of swine-origin influenza A H3N2v-M virus in the US, Nipah virus in Malaysia and Bangladesh, and also present...","Bats have been widely known as natural reservoir hosts of zoonotic diseases, such as severe acute respiratory syndrome (SARS) and Middle East respiratory syndrome (MERS) caused by coronaviruses (CoVs). In the present study, we investigated the whole genomic sequence of a SARS-like bat CoV (16BO133) and found it to be 29,075 nt in length with a 40.9% G+C content. Phylogenetic analysis using amino acid sequences of the ORF 1ab and the spike gene showed that the bat coronavirus strain 16BO133 was grouped with the Beta-CoV lineage B and was closely related to the JTMC15 strain isolated from Rhinolophus ferrumequinum in China. However, 16BO133 was distinctly located in the phylogenetic topology of the human SARS CoV strain (Tor2). Interestingly, 16BO133 showed complete elimination of ORF8 regions induced by a frame shift of the stop codon in ORF7b. The lowest amino acid identity of 16BO133 was identified at the spike region among various ORFs. The spike region of 16BO133 showed 84.7% and 75...","","ABSTRACTNovel Coronavirus (nCoV) outbreak in the city of Wuhan, China during December 2019, has now spread to various countries across the globe triggering a heightened containment effort. This human pathogen is a member of betacoronavirus genus carrying 30 kilobase of single positive-sense RNA genome. Understanding the evolution, zoonotic transmission, and source of this novel virus would help accelerating containment and prevention efforts. The present study reported detailed analysis of 2019-nCoV genome evolution and potential candidate peptides for vaccine development. This nCoV genotype might have been evolved from a bat-CoV by accumulating non-synonymous mutations, indels, and recombination events. Structural proteins Spike (S), and Membrane (M) had extensive mutational changes, whereas Envelope (E) and Nucleocapsid (N) proteins were very conserved suggesting differential selection pressures exerted on 2019-nCoV during evolution. Interestingly, 2019-nCoV Spike protein contains a ..."],"title":["Major emerging and re-emerging zoonoses in China: a matter of global health and socioeconomic development for 1.3 billion","Development of a Whole-Virus ELISA for Serological Evaluation of Domestic Livestock as Possible Hosts of Human Coronavirus NL63","Pathogen genomic surveillance elucidates the origins, transmission and evolution of emerging viral agents in China","Bats and their virome: an important source of emerging viruses capable of infecting humans","Overview of the CSIRO Australian Animal Health Laboratory","SARS \u2013 Pandemie und Emerging Disease","Identification of a Novel Coronavirus in Bats","Are pangolins the intermediate host of the 2019 novel coronavirus (2019-nCoV) ?","Chapter 5 Physical and Chemical Methods for the Reduction of Biological Hazards in Animal Feeds","Characterization of a novel orthoreovirus isolated from fruit bat, China","Middle East respiratory syndrome coronavirus neutralising serum antibodies in dromedary camels: a comparative serological study","A strategy to prevent future pandemics similar to the 2019-nCoV outbreak","Coronaviruses: Molecular Biology\u2606","Prevent and predict","Review of Bats and SARS","Chapter 1 Introduction","Metagenomic study of the viruses of African straw-coloured fruit bats: Detection of a chiropteran poxvirus and isolation of a novel adenovirus","The changing face of pathogen discovery and surveillance","Travel Surveillance and Genomics Uncover a Hidden Zika Outbreak during the Waning Epidemic","Using the spike protein feature to predict infection risk and monitor the evolutionary dynamic of coronavirus","SARS","Coronaviruses in Avian Species \u2013 Review with Focus on Epidemiology and Diagnosis in Wild Birds","SARS","Atlas of ACE2 gene expression in mammals reveals novel insights in transmission of SARS-Cov-2","Oligonucleotide microarray with a minimal number of probes for the detection and identification of thirteen genera of plant viruses","Middle East Respiratory Syndrome Coronavirus (MERS-CoV) origin and animal reservoir","Middle East respiratory syndrome coronavirus (MERS-CoV): Prevention in travelers","Surveillance to improve evidence for community control decisions during the COVID-19 pandemic \u2013 Opening the animal epidemic toolbox for public health","Assessing the Global Tendency of COVID-19 Outbreak","The infectious aetiology of disease: the search for new agents","Novel coronaviruses and astroviruses in bats","Reservoirs and vectors of emerging viruses","Prevalence and Phylogeny of Coronaviruses in Wild Birds from the Bering Strait Area (Beringia)","Microbial molecular markers and epidemiological surveillance in the era of high throughput sequencing: an update from the IMMEM-10 conference","Structural modeling of 2019-novel coronavirus (nCoV) spike protein reveals a proteolytically-sensitive activation loop as a distinguishing feature compared to SARS-CoV and related SARS-like coronaviruses","Chapter 12 Early Warning Practice Using Internet-Based Data","Detection of pancoronavirus using PCR in Camelus dromedarius in Iran (first report)","Future Veterinary Diagnostics","Emergence of a group 3 coronavirus through recombination","The 10th Anniversary of Asia Pacific Allergy","Lipid rafts are involved in SARS-CoV entry into Vero E6 cells","Highland of COVID-19 outside Hubei: epidemic characteristics, control and projections of Wenzhou, China","Complete genome sequence of bat coronavirus HKU2 from Chinese horseshoe bats revealed a much smaller spike gene with a different evolutionary lineage from the rest of the genome","Genetic diversity and evolution of SARS-CoV-2","COVID-19: Facemask use prevalence in international airports in Asia, Europe and the Americas, March 2020","Towards a genomics-informed, real-time, global pathogen surveillance system","Using Routine Surveillance Data to Estimate the Epidemic Potential of Emerging Zoonoses: Application to the Emergence of US Swine Origin Influenza A H3N2v Virus","Complete genome analysis of a SARS-like bat coronavirus identified in the Republic of Korea","Improving epidemic surveillance and response: big data is dead, long live big data","Insights into Cross-species Evolution of Novel Human Coronavirus 2019-nCoV and Defining Immune Determinants for Vaccine Development"],"url":["https://doi.org/10.1016/j.ijid.2014.04.003","https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6356407/","https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7088571/","https://doi.org/10.1016/j.coviro.2012.11.006","https://doi.org/10.1016/j.jiph.2016.04.007","https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7079748/","https://jvi.asm.org/content/jvi/79/4/2001.full.pdf","https://doi.org/10.1101/2020.02.18.954628","https://doi.org/10.1016/b978-0-12-811835-1.00005-1","https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4264558/","https://doi.org/10.1016/s1473-3099(13)70164-6","https://doi.org/10.1016/j.bsheal.2020.01.003","https://doi.org/10.1016/b978-0-12-801238-3.02550-2","https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7097101/","https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3291347/","https://doi.org/10.1016/b978-0-12-812343-0.00001-1","https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3667569/","http://europepmc.org/articles/pmc4098826?pdf=render","https://doi.org/10.1016/j.cell.2019.07.018","https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7093988/","https://doi.org/10.1016/b978-0-444-52272-6.00624-3","https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6296008/","https://doi.org/10.1016/b978-0-444-63951-6.00624-0","https://doi.org/10.1101/2020.03.30.015644","https://doi.org/10.1016/j.jviromet.2010.03.010","https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4891877/","https://doi.org/10.1016/j.tmaid.2014.10.006","https://doi.org/10.1016/j.onehlt.2020.100130","https://doi.org/10.1101/2020.03.18.20038224","https://doi.org/10.1383/medc.33.3.37.61122","https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7091142/","https://doi.org/10.1016/j.coviro.2013.02.002","https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2966397/","https://doi.org/10.1016/j.resmic.2014.01.003","https://doi.org/10.1101/2020.02.10.942185","https://doi.org/10.1016/b978-0-12-812343-0.00012-6","https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7087877/","https://doi.org/10.1053/j.jepm.2010.05.006","https://doi.org/10.1016/j.virol.2009.11.044","https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7016318/","https://doi.org/10.1016/j.bbrc.2008.02.023","https://doi.org/10.1101/2020.02.25.20024398","https://doi.org/10.1016/j.virol.2007.06.009","https://doi.org/10.1016/j.meegid.2020.104260","https://doi.org/10.1016/j.tmaid.2020.101637","https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7097748/","https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3589342/","https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7089380/","https://doi.org/10.1016/s2589-7500(20)30059-5","https://doi.org/10.1101/2020.01.29.925867"],"x":[0.10224856487921452,0.6701049877079406,0.79321930112039,0.9229620253686314,0.5116446266212868,0.7018752673846673,0.9773777828961879,0.8249405291665973,0.45940901125220124,0.5005302076902071,0.05928792825727647,0.7210928147849259,0.8062005977738611,0.4444876906679477,0.1518401203112162,0.24928664188835326,1.645624517987443e-06,0.5159019144684748,0.7562881654174679,0.8290736640938858,0.498870379425577,0.40328007183889136,0.6652349965937016,0.18110214814649928,0.7546761990315597,0.14631984788053065,0.2265531574997527,0.7477819232596223,0.8890402945350695,0.19795185056832565,0.5992734562817306,0.19461304040457905,0.4040868279531077,0.9438158767772256,0.27220817790320295,0.3688817307278206,0.7183632264393447,0.8818172278109428,0.16396522811316272,0.5403606787407245,0.6533492844822972,0.8661785500077667,0.06535984645884696,0.7299300974065328,0.28221890385328985,0.9620414969746939,0.8948166419778373,0.18827206189822343,0.14927264788383687,0.8992248507824999],"y":[0.877647758277609,0.9334618640948332,0.8554296191503148,0.5595632902003417,0.06329756985258184,0.005740104512574673,0.4050491730048513,0.2729158813364091,0.37685442458447427,0.27805778528547,0.7891903178540984,0.5448178566822135,0.4222064527994024,0.5025098006351202,0.6322975870385408,0.10488026279771623,0.025520132257707773,0.3679862589816292,0.11623318782710512,0.15638318151879294,0.29175445471305483,0.911525460202477,0.34096708241746954,0.2945564401944487,0.9372378330599493,0.29214171923910703,0.10844587440023656,0.25098625545769837,0.6167324590810271,0.3542979573829994,0.562235210327421,0.17042595314747688,0.7103517863153913,0.8755287520342332,0.38226411349403233,0.15778447734035916,0.6400843132492788,0.2460458589655985,0.41396273735621947,0.1296980040340182,0.6420125911846778,0.1659784636627002,0.5983158313342101,0.9805507160239104,0.8098189151030957,0.7463507054033253,0.9368957632832998,0.8147620726557768,0.41156353494767495,0.15304081886616494]},"selected":{"id":"9123"},"selection_policy":{"id":"9140"}},"id":"9051","type":"ColumnDataSource"},{"attributes":{"editor":{"id":"9142"},"field":"scr","formatter":{"id":"9143"},"title":"Score","width":30},"id":"9084","type":"TableColumn"},{"attributes":{},"id":"9063","type":"BasicTicker"},{"attributes":{},"id":"9056","type":"Range1d"},{"attributes":{"axis":{"id":"9062"},"ticker":null},"id":"9065","type":"Grid"},{"attributes":{"data_source":{"id":"9051"},"glyph":{"id":"9078"},"hover_glyph":null,"muted_glyph":null,"nonselection_glyph":{"id":"9079"},"selection_glyph":null,"view":{"id":"9081"}},"id":"9080","type":"GlyphRenderer"},{"attributes":{"js_property_callbacks":{"change:indices":[{"id":"9124"}]}},"id":"9123","type":"Selection"},{"attributes":{"columns":[{"id":"9084"},{"id":"9085"}],"height":340,"source":{"id":"9051"},"view":{"id":"9087"},"width":900},"id":"9086","type":"DataTable"},{"attributes":{},"id":"9058","type":"LinearScale"},{"attributes":{"axis":{"id":"9097"},"ticker":null},"id":"9100","type":"Grid"},{"attributes":{"args":{"sd":{"id":"9051"},"sp":{"id":"9052"}},"code":"\n    if (sd.selected.indices.length == 0) {\n      return;\n    }\n    var sel = sd.selected.indices[0];\n    var new_selected = [];\n    for (var i = 0; i &lt; sp.data['doc'].length; i++) {\n      if (sp.data['doc'][i] == sd.data['doc'][sel]) {\n        new_selected.push(i);\n      }\n    }\n    if (JSON.stringify(new_selected) != JSON.stringify(sp.selected.indices) &amp;&amp;\n    !(sp.selected.indices.length == 1 &amp;&amp; new_selected.includes(sp.selected.indices[0]))) {\n      sp.selected.indices = new_selected;\n      sp.change.emit();\n    }\n    "},"id":"9124","type":"CustomJS"},{"attributes":{},"id":"9093","type":"LinearScale"},{"attributes":{},"id":"9159","type":"BasicTickFormatter"},{"attributes":{"source":{"id":"9052"}},"id":"9122","type":"CDSView"},{"attributes":{"fill_alpha":{"value":0.1},"fill_color":{"field":"color"},"line_alpha":{"value":0.1},"line_color":{"field":"color"},"radius":{"field":"rad","units":"data"},"x":{"field":"x"},"y":{"field":"y"}},"id":"9079","type":"Circle"},{"attributes":{},"id":"9106","type":"ResetTool"},{"attributes":{"children":[{"id":"9128"}]},"id":"9131","type":"Row"},{"attributes":{},"id":"9067","type":"BasicTicker"},{"attributes":{},"id":"9154","type":"StringEditor"},{"attributes":{"source":{"id":"9051"}},"id":"9081","type":"CDSView"},{"attributes":{"children":[{"id":"9121"},{"id":"9088"}]},"id":"9134","type":"Row"},{"attributes":{"fill_color":{"field":"color"},"line_color":{"field":"color"},"radius":{"field":"rad","units":"data"},"x":{"field":"x"},"y":{"field":"y"}},"id":"9113","type":"Circle"},{"attributes":{"columns":[{"id":"9119"},{"id":"9120"}],"height":340,"source":{"id":"9052"},"view":{"id":"9122"},"width":900},"id":"9121","type":"DataTable"},{"attributes":{"callback":null,"tooltips":[["Title","@title"],["Passage","@text"],["Published","@date"],["Authors","@authors"],["Journal","@journal"],["URL","@url"]]},"id":"9107","type":"HoverTool"},{"attributes":{},"id":"9095","type":"LinearScale"},{"attributes":{},"id":"9145","type":"StringFormatter"},{"attributes":{},"id":"9157","type":"BasicTickFormatter"},{"attributes":{"children":[{"id":"9127"}]},"id":"9130","type":"Row"},{"attributes":{},"id":"9149","type":"BasicTickFormatter"},{"attributes":{},"id":"9140","type":"UnionRenderers"},{"attributes":{},"id":"9147","type":"BasicTickFormatter"},{"attributes":{},"id":"9143","type":"StringFormatter"},{"attributes":{"editor":{"id":"9144"},"field":"title","formatter":{"id":"9145"},"title":"Document title","width":770},"id":"9085","type":"TableColumn"},{"attributes":{"callback":null},"id":"9105","type":"TapTool"},{"attributes":{"text":""},"id":"9139","type":"Title"},{"attributes":{"data":{"authors":["Niccolo Alfano; Anisha Dayaram; Jan Axtner; Kyriakos Tsangaras; Marie-Louise Kampmann; Azlan Mohamed; Seth Timothy Wong; M. Thomas P. Gilbert; Andreas Wilting; Alex Daivd Greenwood","Li, Hong-Ying; Zhu, Guang-Jian; Zhang, Yun-Zhi; Zhang, Li-Biao; Hagan, Emily A; Martinez, Stephanie; Chmura, Aleksei A; Francisco, Leilani; Tai, Hina; Miller, Maureen; Daszak, Peter","Hu, Ben; Ge, Xingyi; Wang, Lin-Fa; Shi, Zhengli","Daszak, Peter; Olival, Kevin J.; Li, Hongying","Daszak, Peter; Olival, Kevin J.; Li, Hongying","Daszak, Peter; Olival, Kevin J.; Li, Hongying","Daszak, Peter; Olival, Kevin J.; Li, Hongying","Mishra, Nischay; Fagbo, Shamsudeen F.; Alagaili, Abdulaziz N.; Nitido, Adam; Williams, Simon H.; Ng, James; Lee, Bohyun; Durosinlorun, Abdulkareem; Garcia, Joel A.; Jain, Komal; Kapoor, Vishal; Epstein, Jonathan H.; Briese, Thomas; Memish, Ziad A.; Olival, Kevin J.; Lipkin, W. Ian","Li, Hong-Ying; Zhu, Guang-Jian; Zhang, Yun-Zhi; Zhang, Li-Biao; Hagan, Emily A; Martinez, Stephanie; Chmura, Aleksei A; Francisco, Leilani; Tai, Hina; Miller, Maureen; Daszak, Peter","Wray, Amy K.; Olival, Kevin J.; Mor\u00e1n, David; Lopez, Maria Renee; Alvarez, Danilo; Navarrete-Macias, Isamara; Liang, Eliza; Simmons, Nancy B.; Lipkin, W. Ian; Daszak, Peter; Anthony, Simon J.","Ewer, Katie; Sebastian, Sarah; Spencer, Alexandra J.; Gilbert, Sarah; Hill, Adrian V. S.; Lambe, Teresa","El-Duah, Philip; Meyer, Benjamin; Sylverken, Augustina; Owusu, Michael; Gottula, Lina Theresa; Yeboah, Richmond; Lamptey, Jones; Frimpong, Yaw Oppong; Burimuah, Vitus; Folitse, Raphael; Agbenyega, Olivia; Oppong, Samuel; Adu-Sarkodie, Yaw; Drosten, Christian","Shi, Z.; Wang, L.-F.","Mirkena, Tadele; Walelign, Elias; Tewolde, Nega; Gari, Getachew; Abebe, Getachew; Newman, Scott","Dawson, Patrick; Malik, Mamunur Rahman; Parvez, Faruque; Morse, Stephen S.","Baker, Kimberlee L.; Thomas, Paul R.; Karriker, Locke A.; Ramirez, Alejandro; Zhang, Jianqiang; Wang, Chong; Holtkamp, Derald J.","Smith, Ina; Wang, Lin-Fa","Mirkena, Tadele; Walelign, Elias; Tewolde, Nega; Gari, Getachew; Abebe, Getachew; Newman, Scott","Goh, Gerard Kian-Meng; Dunker, A. Keith; Foster, James A.; Uversky, Vladimir N.","","Al-Tayib, Omar A.","Mackenzie, John S; Jeggo, Martyn","Zhu, Sophie; Zimmerman, Dawn; Deem, Sharon L.","Blacklaws, B.A.; Daly, J.M.","Louten, Jennifer","Yusof, Mohammed Farouk; Queen, Krista; Eltahir, Yassir Mohammed; Paden, Clinton R; Al Hammadi, Zulaikha Mohamed Abdel Hameed; Tao, Ying; Li, Yan; Khalafalla, Abdelmalik Ibrahim; Shi, Mang; Zhang, Jing; Mohamed, Muzammil Sayed Ahmed Elhaj; Abd Elaal Ahmed, Mahmud Hamed; Azeez, Ihsaan Abdulwahab; Bensalah, Oum Keltoum; Eldahab, Ziyada Swar; Al Hosani, Farida Ismail; Gerber, Susan I; Hall, Aron J; Tong, Suxiang; Al Muhairi, Salama Suhail","El-Duah, Philip; Meyer, Benjamin; Sylverken, Augustina; Owusu, Michael; Gottula, Lina Theresa; Yeboah, Richmond; Lamptey, Jones; Frimpong, Yaw Oppong; Burimuah, Vitus; Folitse, Raphael; Agbenyega, Olivia; Oppong, Samuel; Adu-Sarkodie, Yaw; Drosten, Christian","Coleman, Christopher M.; Frieman, Matthew B.","Zhu, Sophie; Zimmerman, Dawn; Deem, Sharon L.","Miguel, Eve; Chevalier, V\u00e9ronique; Ayelet, Gelagay; Ben Bencheikh, Med Nadir; Boussini, Hiver; Chu, Daniel KW; El Berbri, Ikhlass; Fassi-Fihri, Ouaffa; Faye, Bernard; Fekadu, Getnet; Grosbois, Vladimir; Ng, Bryan CY; Perera, Ranawaka APM; So, TY; Traore, Amadou; Roger, Fran\u00e7ois; Peiris, Malik","Hanscheid, Thomas; Valadas, Em\u00edlia; Grobusch, Martin P.","Al-Tayib, Omar A.","Kock, Richard A.; Karesh, William B.; Veas, Francisco; Velavan, Thirumalaisamy P.; Simons, David; Mboera, Leonard E. G.; Dar, Osman; Arruda, Li\u00e3 B\u00e1rbara; Zumla, Alimuddin","Kock, Richard A.; Karesh, William B.; Veas, Francisco; Velavan, Thirumalaisamy P.; Simons, David; Mboera, Leonard E. G.; Dar, Osman; Arruda, Li\u00e3 B\u00e1rbara; Zumla, Alimuddin","Yusof, Mohammed Farouk; Queen, Krista; Eltahir, Yassir Mohammed; Paden, Clinton R; Al Hammadi, Zulaikha Mohamed Abdel Hameed; Tao, Ying; Li, Yan; Khalafalla, Abdelmalik Ibrahim; Shi, Mang; Zhang, Jing; Mohamed, Muzammil Sayed Ahmed Elhaj; Abd Elaal Ahmed, Mahmud Hamed; Azeez, Ihsaan Abdulwahab; Bensalah, Oum Keltoum; Eldahab, Ziyada Swar; Al Hosani, Farida Ismail; Gerber, Susan I; Hall, Aron J; Tong, Suxiang; Al Muhairi, Salama Suhail","Mani, Reeta S.; Ravi, V.; Desai, Anita; Madhusudana, S.N.","Liu, Quan; Cao, Lili; Zhu, Xing-Quan","Kumar, A.; Chaterjee, Souranshu","Keyaerts, Els; Vijgen, Leen; Pannecouque, Christophe; Van Damme, Els; Peumans, Willy; Egberink, Herman; Balzarini, Jan; Van Ranst, Marc","","Kolawole, Olatunji; Oguntoye, Michael; Dam, Tina; Chunara, Rumi","Niccolo Alfano; Anisha Dayaram; Jan Axtner; Kyriakos Tsangaras; Marie-Louise Kampmann; Azlan Mohamed; Seth Timothy Wong; M. Thomas P. Gilbert; Andreas Wilting; Alex Daivd Greenwood","Mackenzie, John S; Jeggo, Martyn","Liu, Quan; Cao, Lili; Zhu, Xing-Quan","Kock, Richard A.; Karesh, William B.; Veas, Francisco; Velavan, Thirumalaisamy P.; Simons, David; Mboera, Leonard E. G.; Dar, Osman; Arruda, Li\u00e3 B\u00e1rbara; Zumla, Alimuddin","Kock, Richard A.; Karesh, William B.; Veas, Francisco; Velavan, Thirumalaisamy P.; Simons, David; Mboera, Leonard E. G.; Dar, Osman; Arruda, Li\u00e3 B\u00e1rbara; Zumla, Alimuddin","Torres-V\u00e9lez, Fernando; Brown, Corrie","Kun Sun; Liuqi Gu; Li Ma; Yunfeng Duan","Henderson, Joan C.","Yusof, Mohammed Farouk; Queen, Krista; Eltahir, Yassir Mohammed; Paden, Clinton R; Al Hammadi, Zulaikha Mohamed Abdel Hameed; Tao, Ying; Li, Yan; Khalafalla, Abdelmalik Ibrahim; Shi, Mang; Zhang, Jing; Mohamed, Muzammil Sayed Ahmed Elhaj; Abd Elaal Ahmed, Mahmud Hamed; Azeez, Ihsaan Abdulwahab; Bensalah, Oum Keltoum; Eldahab, Ziyada Swar; Al Hosani, Farida Ismail; Gerber, Susan I; Hall, Aron J; Tong, Suxiang; Al Muhairi, Salama Suhail"],"color":["#ef8a62","#ef8a62","#ef8a62","#ef8a62","#ef8a62","#ef8a62","#ef8a62","#ef8a62","#ef8a62","#ef8a62","#ef8a62","#ef8a62","#ef8a62","#ef8a62","#ef8a62","#ef8a62","#ef8a62","#ef8a62","#ef8a62","#ef8a62","#ef8a62","#ef8a62","#ef8a62","#ef8a62","#ef8a62","#ef8a62","#ef8a62","#ef8a62","#ef8a62","#ef8a62","#ef8a62","#ef8a62","#ef8a62","#ef8a62","#ef8a62","#ef8a62","#ef8a62","#ef8a62","#ef8a62","#ef8a62","#ef8a62","#ef8a62","#ef8a62","#ef8a62","#ef8a62","#ef8a62","#ef8a62","#ef8a62","#ef8a62","#ef8a62"],"date":["2020-03-29","2020-02-10","2015-12-22","2020","2020","2020","2020","2019-04-10","2020-02-10","2016-09-22","2017-10-30","2019-01-09","2017-12-31","2018-12-21","2019-03-01","2017-12-01","2013-02-28","2018-12-21","2020-03-31","2008-12-31","2019-02-22","2013-04-30","2019-05-28","2018-03-31","2016-12-31","2017-11-08","2019-01-09","2013-09-05","2019-05-28","2017-03-30","2020-02-07","2019-02-22","2020","2020","2017-11-08","2012-01-18","2014-08-31","2017-08-31","2007-09-30","2003-08-31","2017-12-07","2020-03-29","2013-04-30","2014-08-31","2020","2020","2004-09-30","2020-03-31","2007-12-31","2017-11-08"],"doc":["nil1vv6h","dqpujzo8","d2knbzhl","h5sox8bq","h5sox8bq","h5sox8bq","h5sox8bq","4kvkioa3","dqpujzo8","9ojdhxwk","opi8aa5i","tjp33y35","apc0lm5e","puv3mn42","yb54i1ne","ycy3ge18","v8uzd2f3","puv3mn42","75u57fw1","obgjbl6k","oudj079q","0g5079y1","ysxvl327","kdlcaium","3p542sqs","g0gh28hu","tjp33y35","k38we345","ysxvl327","hrz293mb","5sfwyn9g","oudj079q","oxs4o9xe","oxs4o9xe","g0gh28hu","hlugf0xc","p1jbb1wa","m4kdkmey","6qmckoo8","z9i5mk33","okb61ueg","nil1vv6h","0g5079y1","p1jbb1wa","oxs4o9xe","oxs4o9xe","z4egpcld","89sg0cpk","nezgzovk","g0gh28hu"],"journal":["","Int Health","Virol J","Biosafety and Health","Biosafety and Health","Biosafety and Health","Biosafety and Health","PLoS One","Int Health","Ecohealth","Hum Vaccin Immunother","Viruses","Genetics and Evolution of Infectious Diseases","Pastoralism","Vector Borne Zoonotic Dis","BMC Vet Res","Current Opinion in Virology","Pastoralism","Microbial Pathogenesis","Travel Medicine","Pathogens","Current Opinion in Virology","Ecohealth","The Veterinary Journal","Essential Human Virology","Emerg Microbes Infect","Viruses","PLoS Pathog","Ecohealth","Euro Surveill","Travel Medicine and Infectious Disease","Pathogens","The Lancet Planetary Health","The Lancet Planetary Health","Emerg Microbes Infect","Proc Natl Acad Sci India Sect B Biol Sci","International Journal of Infectious Diseases","Current Medicine Research and Practice","Antiviral Research","Journal of Allergy and Clinical Immunology","BMC Res Notes","","Current Opinion in Virology","International Journal of Infectious Diseases","The Lancet Planetary Health","The Lancet Planetary Health","Clinics in Laboratory Medicine","","Managing Tourism Crises","Emerg Microbes Infect"],"rad":[0.019997965338000002,0.013050749987696605,0.012365976543630558,0.01057670493156853,0.01057670493156853,0.009883137271119086,0.009883137271119086,0.009550443286490296,0.008792212330757582,0.008043116786786049,0.008042953799545841,0.007840816550152178,0.007791207898443293,0.007501766643132372,0.007392253013170994,0.007339550500202697,0.007246826622761889,0.007056899866948867,0.0068898135436833916,0.006483104752740782,0.006395086218000001,0.005887894076385111,0.005587304589562605,0.005415359164865083,0.005274987015503418,0.005211168074360572,0.005062086220724618,0.004858228003313176,0.0047329006622838805,0.004686737420789729,0.00455133505451451,0.004319071997593202,0.004250365803987747,0.004250365803987747,0.0042462832716223045,0.004032479057322706,0.004014803478060319,0.004005662837467729,0.003982074762501389,0.00380051740962889,0.0036808198894323248,0.0036332770485029107,0.003614610356255126,0.003461178805073609,0.003414656280297113,0.003414656280297113,0.0033972835770819986,0.003223460737777123,0.003156085753525208,0.0030342587034452835],"scr":[1.0,0.591,0.551,0.446,0.446,0.405,0.405,0.385,0.341,0.297,0.297,0.285,0.282,0.265,0.258,0.255,0.25,0.239,0.229,0.205,0.2,0.17,0.152,0.142,0.134,0.13,0.121,0.109,0.102,0.099,0.091,0.078,0.074,0.074,0.073,0.061,0.06,0.059,0.058,0.047,0.04,0.037,0.036,0.027,0.024,0.024,0.023,0.013,0.009,0.002],"text":["Several novel viruses were identified with our short RNA bait approach, which is not unexpected as little is known about the virology of wildlife in Southeast Asia, where the leeches were collected. Several viral contigs were phylogenetically distinct from known viruses and may represent new genera. For example, the novel coronavirus identified in leech bloodmeals did not cluster with any of the known Coronaviridae clades. This finding highlights the ability of this method to detect unknown viruses. We could also associate the novel corona-and rhabdoviruses with mammal bloodmeals with limited evidence of a cervid association for both. Cervids are regularly sold as bushmeat in wildlife markets [25] and both recent coronavirus epidemics (SARS-CoV [18] and SARS-Cov-2 [16] ) spilled over from wildlife. This suggests that eDNA/iDNA-based pathogen surveillance approaches may complement efforts to proactively identify viruses that could potentially spillover to humans or livestock.","Emerging and re-emerging zoonotic diseases are key contributors to morbidity and mortality in southern China.1,2 This region, considered a \u2018hotspot\u2019 for emerging zoonotic diseases, harbours abundant wildlife while also undergoing land use change and natural resource overexploitation leading to intensified human\u2013animal interactions that favour the emergence of zoonotic diseases.3 People living in the rural areas of southern China primarily cultivate rice and fruits, raise swine and poultry in households or on small farms,4 but also traditionally hunt wild animals as an alternative income source.5 The mixed landscape has abundant crops, which attracts wild animals into the communities, and livestock rearing is common.6 This brings humans and animals into close contact in dense populations, creating a wildlife\u2013livestock\u2013human interface for zoonotic disease emergence.7","With human activity increasingly overlapping the habitats of bats, diseases outbreaks resulted from spillover of bat coronaviruses will continue to occur in the future despite the fact that direct transmission of bat coronaviruses to humans appears to be rare. To better prepare ourselves in predicting and preventing the next emergence of a coronavirus disease, it is necessary to maintain our vigilance in long-term coronavirus surveillance studies in bats as well as in other wildlife and livestock. Combined with other laboratory-based studies such as receptor specificity, pathogenesis and animal infection, a focus on continued surveillance will help us to improve risk assessment as well as to reveal the potential intermediate hosts that may play an important role in the interspecies transmission of various known and as yet unknown bat coronaviruses.","Here, we propose three key steps to helping reduce the risk of a future epidemic similar to that caused by the novel coronavirus 2019-nCoV: 1) Surveillance of wildlife for high-risk pathogens Over the past 10 years collaborating with scientists in China, we collected samples from over 10,000 bats and~2000 other mammals across South China and discovered 52 novel SARSr-CoVs, 122 other \u03b2-CoVs, more than 350 novel \u03b1-CoVs (including the new Swine Acute Diarrheal Syndrome Coronavirus SADS-CoV), and a new \"lineage E\" \u03b2-CoV clade [11, 12] . We found SARS-related CoVs that can bind to human cells, and that cause SARS-like disease in humanized mouse models that was not prevented with a vaccine candidate against SARS-CoV, and were not treatable with almost all of the monoclonal therapies being developed for SARS [8, [13] [14] [15] [16] . Finally, we showed serological evidence that people living at the wildlife-human interface in rural China were being exposed to SARSrelated coronaviruses, perhaps even the same virus as nCoV-2019, between the emergence of SARS and the current outbreak [17, 18] . Together these data mark wildlife-origin coronaviruses as a 'clear and present danger'. They also highlight exactly the issue of key concern in the current outbreakthat there is a large diversity of viral strains in wildlife in China with significant potential for emergence in people. Further, we estimate that there are thousands of other CoVs in bats across Southeast Asia, many of which will have pandemic potential. We strongly urge that scientists in these countries work to discover all of these viruses so that we can catalog them, develop a reference library for rapid pathogen identification and risk assessment, and test vaccines and therapies against them [19, 20] .","Here, we propose three key steps to helping reduce the risk of a future epidemic similar to that caused by the novel coronavirus 2019-nCoV: 1) Surveillance of wildlife for high-risk pathogens Over the past 10 years collaborating with scientists in China, we collected samples from over 10,000 bats and~2000 other mammals across South China and discovered 52 novel SARSr-CoVs, 122 other \u03b2-CoVs, more than 350 novel \u03b1-CoVs (including the new Swine Acute Diarrheal Syndrome Coronavirus SADS-CoV), and a new \"lineage E\" \u03b2-CoV clade [11, 12] . We found SARS-related CoVs that can bind to human cells, and that cause SARS-like disease in humanized mouse models that was not prevented with a vaccine candidate against SARS-CoV, and were not treatable with almost all of the monoclonal therapies being developed for SARS [8, [13] [14] [15] [16] . Finally, we showed serological evidence that people living at the wildlife-human interface in rural China were being exposed to SARSrelated coronaviruses, perhaps even the same virus as nCoV-2019, between the emergence of SARS and the current outbreak [17, 18] . Together these data mark wildlife-origin coronaviruses as a 'clear and present danger'. They also highlight exactly the issue of key concern in the current outbreakthat there is a large diversity of viral strains in wildlife in China with significant potential for emergence in people. Further, we estimate that there are thousands of other CoVs in bats across Southeast Asia, many of which will have pandemic potential. We strongly urge that scientists in these countries work to discover all of these viruses so that we can catalog them, develop a reference library for rapid pathogen identification and risk assessment, and test vaccines and therapies against them [19, 20] .","The second question is what is the source of this virus? The evidence is fairly clear: Most of the initial cases were linked to a seafood market that also sold butchered livestock meat and some wildlife. The virus itself appears to have a wildlife (bat) origin similar to SARS-CoV, which also emerged in a wildlife market through the interactions of humans and other animals that acted as the intermediate hosts of the virus [3, 8, 9] . With two disease outbreaks originating in China significantly linked to wildlife markets, this is an obvious target for control programs to prevent future epedemics and pandemics. Indeed, there have already been calls from Chinese conservationists, public health leaders and policy makers to reduce the wildlife consumption. However, banning or even reducing the sale of wild game may not be straightforward and it is challenging to change behaviors that are influenced by Chinese culture and traditions. In addition to a strong belief in the purported curative power of wildlife and their by-products, the consumption of the rare and expensive wildlife has become a social statement of wealth boosted by economic growth. Changing these cultural habits will take time, however, recent behavioral questionnaire data suggests a generational transformation with reduced wildlife consumption in the younger generation [10] . There is no doubt, however, that the wildlife trade has an inherent risk of bringing people close to pathogens that wildlife carry, that we have not yet encountered, and that have the potential of leading to the next outbreak.","The second question is what is the source of this virus? The evidence is fairly clear: Most of the initial cases were linked to a seafood market that also sold butchered livestock meat and some wildlife. The virus itself appears to have a wildlife (bat) origin similar to SARS-CoV, which also emerged in a wildlife market through the interactions of humans and other animals that acted as the intermediate hosts of the virus [3, 8, 9] . With two disease outbreaks originating in China significantly linked to wildlife markets, this is an obvious target for control programs to prevent future epedemics and pandemics. Indeed, there have already been calls from Chinese conservationists, public health leaders and policy makers to reduce the wildlife consumption. However, banning or even reducing the sale of wild game may not be straightforward and it is challenging to change behaviors that are influenced by Chinese culture and traditions. In addition to a strong belief in the purported curative power of wildlife and their by-products, the consumption of the rare and expensive wildlife has become a social statement of wealth boosted by economic growth. Changing these cultural habits will take time, however, recent behavioral questionnaire data suggests a generational transformation with reduced wildlife consumption in the younger generation [10] . There is no doubt, however, that the wildlife trade has an inherent risk of bringing people close to pathogens that wildlife carry, that we have not yet encountered, and that have the potential of leading to the next outbreak.","Bats (order Chiroptera) are the second most diverse mammalian order after rodents, comprising more than 1400 species that represent approximately 20% of all mammalian species [1]. Bats are beneficial in arthropod suppression, seed dispersal, and pollination; and serve as a bushmeat food source for some human populations [2, 3]. Bats are also natural reservoirs for a wide range of emerging viruses that may cause infectious diseases in humans, livestock, domestic animals and wildlife [4], and as a group bats also harbor a higher proportion of zoonotic viruses than other mammals [5]. As with most groups of mammals, bats are predicted to have a large diversity of viruses, which are currently undescribed [5, 6]. The development of more powerful high throughput sequencing (HTS) technology and bioinformatic tools now allow us to better characterize this viral diversity, including analysis of samples from geographic regions that have received little wildlife surveillance attention to date. Recently several bat virome studies have been published using high-throughput viral metagenomic approaches, revealing a wide range of viruses and expanding our knowledge of viral diversity. A coronavirus similar to Middle East respiratory syndrome coronavirus (MERS-CoV), adenoviruses, herpesviruses and circoviruses were reported in Neoromicia capensis bats from South Africa. Yinda et. al reported diverse rotaviruses, bastroviruses and sapoviruses using HTS analysis of Eidolon helvum and Epomophorus gambianus bats from Cameroon [7\u201310].","Enforcement of current wildlife protection policy and continued community infrastructure development appears to significantly reduce high-risk contact between humans, wildlife and livestock. Closer collaboration between local animal and human health authorities within the current epidemic disease prevention programmes will provide educational and training opportunities to promote risk-mitigation knowledge, skills and best practice in local communities. For example, cave monitoring and management is a low-cost and efficient method to help restrict human activities (e.g. recreation and mining) that lead to contact with bats in caves. This is of particular importance given the emergence of 2019-nCoV, which appears likely to be a bat-origin coronavirus.10,11","Multi-disciplinary microbial ecology studies, such as our efforts to characterize the microbe diversity hosted by Desmodus rotundus in Guatemala combined with analysis of ecological feeding preference, can answer questions about human\u2013bat public health interfaces and have additional value for conservation and ecology. The results of our study demonstrate that D. rotundus carries both novel and known microbes, and that the populations in this region appear to display an overwhelming preference for feeding on domestic livestock species. As populations of D. rotundus continue to expand in Latin America with the increasing production of livestock, better understanding of their infectious disease ecology and diversity will become even more critical. Subsequent studies are necessary to further resolve the phylogeny and further investigate the actual pathogenicity of the novel viruses detected in this study. Additional research efforts involving microbial discovery and surveillance among domestic animals may also prove useful for investigating whether microbe sharing occurs between D. rotundus and their prey, as well as how viral and bacterial diversity may change seasonally and geographically. Gaining a better understanding of the role of D. rotundus as a potential source and reservoir of infectious diseases will help better inform public health and wildlife management policies aimed at preventing or mitigating future disease spillover events in the face of ecosystem changes.","The key bottlenecks for this approach are the identification of vaccine antigens and the availability of appropriate animal models of disease. For preparations to be made to counter future threats, some knowledge of emerging pathogens is required, and yet detailed epidemiological surveillance for many infectious diseases remains limited in regions where incidence is greatest.110 Recent data suggests that around 60% of emerging infectious diseases are zoonotic with the majority originating in wildlife, requiring surveillance among livestock animals and wildlife species, as well as in humans.111 Although Ebola outbreaks have occurred sporadically since 1976, the pace of vaccine development for Ebola has been slow with most vaccines undergoing preclinical evaluation for more than 5 years before the start of Phase I clinical trials. The 2014\u201315 outbreak provided much needed momentum for public health experts and the research community to improve preparedness for future epidemics.112 In order to continue to improve our preparedness for future outbreaks, epidemiological surveillance and vaccine development will need to accelerate substantially.","Ethical approval for the study was obtained from the committee on human research, publications and ethics of the school of medical sciences, Kwame Nkrumah University of Science and Technology (Protocol number CHRPE49/09). Permission for livestock sampling was also obtained from the wildlife division of the Ghana Forestry Commission (Approval Number: AO4957).","With the demonstration of an increasing number of spillover events that led to severe disease outbreaks in human and domestic animals, we believe it is paramount that from now on we include active surveillance of wildlife animals as part of an integrated infectious disease prevention and control strategy. Surveillance of wildlife animals has also been made more feasible and productive, thanks to the advance in modern molecular techniques including PCR with virus group-specific primers, virus discovery using next generation high-throughput sequencing technologies, and high density virus microarrays. 56e63 Since the SARS outbreaks, especially after the discovery of SL-CoVs in bats, there is a significant surge in international effort for surveillance of coronaviruses in wildlife animals. Before the SARS outbreak, there were only 10 coronaviruses with complete genomes sequenced. This number has increased more than sixfold as a result of the active surveillance works conducted around the world. 27e29, 31,32,34,40,55,64e73 Although this only marks the beginning of our understanding of coronaviruses in wildlife animals, it is fair to say that we have learnt a lot more about coronaviruses in general than the past 50 years or so; during that period studying of viruses was only possible and called for in response to disease outbreaks. Based on phylogenetic analysis of the large number of bat coronavirus sequences available presently, it is postulated that all known disease-causing coronaviruses previously identified in humans or animals originated from bat strains. 31, 34, 43, 55 This hypothesis was unfortunately proved by the outbreak of another SARS-like disease, Middle East Respiratory Syndrome (MERS), which was caused by a novel coronavirus (previously named HCoV-EMC, now MERS-CoV) and supposed to originate from bats. 74 Even though the MERS-CoV-like viruses found in bats are not the direct progenitor of the MERS-CoV, the highly genetic diversity of these bat viruses is likely the gene sources for the deadly pathogen in humans, just like that for SARS-CoV. 71,72,75e77 ","Pastoralists keep indigenous breeds/types and obtain more than 50% of household income from livestock and livestock products. The system is much simpler than the mixed crop-livestock systems of the highlands. There are few inputs other than labour. Herd and flock composition is regulated to some extent (only few breeding males are maintained). Grazing management and herd movement are determined by the seasonal patterns of rainfall and availability of water. There is little to no interaction with crop agriculture, and although a range of livestock species is managed to reduce risk, one or two species dominate. For example, camels and goats are the main species in Afar and Somali, while in Borana zone, cattle are still the main species. Production is mainly for subsistence, but surplus animals are sold. Generally, camel populations have been increasing in the pastoral areas during the past 20 years by at least 10, 20, 25, 15, 25 and more than 200% in Gode, Jijiga, Shinille, Mille, Amibara districts and Borana zone, respectively. On the contrary, cattle populations decreased by 50 to 70% in these districts during the same time (Tadesse et al. 2013). According to these authors, about 14, 25, 10 and 8% of the households studied in Gode, Jijiga, Shinille and Borana, respectively, do not possess cattle at present. In the agro-pastoral production system, crop agriculture is combined to a limited extent with livestock rearing. It is practised in semi-arid areas and may take the form of either sedentary or transhumance way of living. Indigenous breeds/types are reared and livestock contribute between 10 and 50% of household incomes. Mixed crop-livestock production systems prevail in sub-humid and humid central highland parts of Ethiopia. The system is sedentary and livestock is secondary to crop production. It is characterized by smallholdings of about 1 to 3 ha of land and two to four heads of cattle (MoARD 2007).","As an emerging infection of zoonotic origin (Morse 1995), a unified One Health approach that engages the human health, livestock and companion animal, wildlife, and environment sectors will be crucial in dealing with MERS (Morse 2012, Morse et al. 2012). It could serve as an excellent case study for epidemiology and response to a new zoonotic, or emerging, infection.","Porcine epidemic diarrhea (PED) was first described in 1971 as cases of profuse, watery diarrhea affecting all ages of pigs in England [1]. In 1978, a novel coronavirus, porcine epidemic diarrhea virus (PEDV), was determined to be the causative agent [2, 3]. The first cases of PEDV in the United States were confirmed during May 2013 from farms in Iowa and Indiana [4]. PEDV spread rapidly throughout the United States after its introduction mainly due to the ease of transmission via fecal-oral and pig-to-pig contact [5]. Livestock trailers that haul pigs to and from collection points such as livestock auctions or harvest facilities have been implicated as mechanical vectors for PEDV [6]. Contaminated livestock trailers likely pose a significant risk for PEDV transmission and movement throughout the country.","The introduction of biocontainment measures within piggeries, including surveillance of pigs being transferred between farms and sent to abattoirs, is another effective approach which can be applied at the farming and trade level [87] [88] [89] . Since some of these zoonotic agents involve an intermediate host of livestock importance, it is highly important that veterinarians wear appropriate PPE when performing procedures on animals to minimize the risk of transmission of zoonotic diseases.","Livestock production systems in Ethiopia can be broadly classified into two as the traditional production systems (pastoral nomadic, pastoral transhumant, agro-pastoral and smallholder mixed crop-livestock) and the modern production systems (ranching, intensive/semi-intensive peri-urban/urban, feedlot and commercial production). Camels are predominantly kept in the pastoral and agro-pastoral production systems. Only few male camels are to be found in the mixed crop-livestock system.","Before the SARS-CoV outbreak, coronaviruses were not considered medically important, as, in the past, they had been primarily associated with cold viruses that cause minor sniffles. This is, however, not true in the field of veterinary medicine, where coronaviruses have posed a menace to livestock. This is especially so in the case of TGEV (transmissible gastroenteritis virus) and PEDV (porcine epidemic diarrhea virus). TGEV could move rapidly among farm pigs to quickly devastate the farming community if not controlled early 5-7 . Our analysis reveals that TGEV is of category C (higher fecal-oral, lower respiratory transmission potentials) with M and N PIDs at 14% and 43% respectively. The reason that it is able to move rapidly is that its high fecal-oral transmission potential represents a more efficient mode of transmission among the farm animals.","Scrub typhus (tsutsugamushi fever) is caused by the rickettsia Orienta tsutsugamushi, which is transmitted by the bite of larval mites (chiggers). The term tick typhus encompasses a number of tick-borne infections caused by various rickettsias in different parts of the world (e.g. Rickettsia conorii in southern Europe, Africa, and Asia, and R. africae in East and Southern Africa). The reservoir of infection includes domestic dogs, rodents, and livestock. People become infected in grassy or shrub-covered areas while on safari, treks, or camping trips.","By enhancing biosecurity and management in animal farms, the risk of reemerging pathogens particularly responsible for zoonotic diseases caused by viruses, can be reduced. This is a matter of economic importance; in view of the large livestock trade existing or that existed between countries in the Indian Ocean and Eastern Africa countries where several zoonotic diseases are endemic. However, a phylogenetic study strongly suggests that some zoonotic infections have been introduced into Saudi Arabia through ruminant trade [212,213]. Furthermore, following the adoption of the recommended guidelines of the World Organization for Animal Health through its Office International des Epizooties (OIE) Code, if such policies regarding the exportation and/or importation of animals are exactly followed, these would greatly limit the extent of this risk [214].","Since 2000, increasing BTV types have been found in southern Europe with a variety of Culicoides species being incriminated [111, 130, 135] . In the summer of 2006 however, BTV serotype 8 (BTV-8) emerged for the first time in northern Europe, resulting in over 2000 infected farms by the end of the year [136] . This was probably due in large part to climatic changes permitting its major vector, Culicoidesi micola, to move northwards, and to the ability of some northern Culicoides species to become competent to transmit the virus [111, 128 ] . Interestingly, the initial spread from the Netherlands indicated a single point introduction of the disease into Europe, not typical of a spread by competent vectors from southern Europe. The virus subsequently overwintered and spread across much of Europe, causing tens of thousands of livestock deaths. In August 2007, BTV-8 reached the United Kingdom (UK), threatening the large and valuable livestock industry. A voluntary vaccination scheme was launched in UK in May 2008 and, in contrast with elsewhere in Europe, there were no reported cases in the UK during 2008. Thus whilst the global range of BTV has historically been assumed to be restricted by regional differences in vector competence amongst Culicoides species as well as by the temperature requirements of the virus for replication, this outbreak did not follow this pattern. It has been postulated that the use of a live attenuated BTC 8 vaccine may have been the initial cause of this outbreak [137] . Importantly on 15th January 2013, Spain proceeded to declared itself free of serotype 8 of bluetongue virus. As with other parts of Europe BTV serotype 8 appeared for the first time in Spain in January 2008 but the system of disease surveillance implemented in Spain allowed for early detection and the implementation of rapid and effective control measures based on vaccination and movement control limited spread and enabled eradication of BTV from this region [138] . This still leaves the question of the underlying reservoir of BTV 8 and the process for emergence into Europe.","Lastly, a limitation not necessary related to methodology but rather the epidemiology of diseases relates to missing diseases at the camel\u2013other livestock/wildlife\u2013human interface that we would not have captured in our review. Although we did not include them as part of this review there were papers found in our initial searches that described camel\u2013livestock and camel\u2013wildlife transmission. These shared pathogens may be zoonotic and include camels as part of the transmission cycle but are not necessarily identified as a camel-borne zoonosis. For example, other animals may be the original or primary source of many pathogens, such as the case of MERS in bats, with the camel being an intermediate host in the transmission from animal to human (Corman et al. 2014a, b).","In this Special Issue, we have papers focussing on the emergence and re-emergence of Schmallenberg virus (McGowan et al., 2018; Stavrou et al., 2017) and African swine fever virus (Sanchez-Cordon et al., 2018) . The continued surveillance of wildlife (Bodewes, 2018) and domestic animals, including livestock (Bourret, 2018) and companion animals (Caddy, 2018), to identify novel viruses is particularly important in understanding the sources and risks of viruses affecting the health of animals and humans. A greater understanding of virus replication, pathogenesis and transmission allows intervention strategies to be formulated, including development of vaccines and antiviral treatments. Understanding of replication of Zika virus is exemplified by the paper by Royle et al. (2017) , whilst the review by Pusterla et al. (2018) summarises what is known about a novel coronavirus of horses.","Nearly every emerging viral disease can be associated with an environmental or ecological component. The consequences of urbanization and globalization-including but not limited to deforestation, habitat modification, and the use of wild land for farming-impact the emergence and reemergence of potential zoonoses. Increases in the human population have led to the encroachment of humans into animal habitats formerly unoccupied by humans. These circumstances contribute to humans coming in contact with animals with which they would not normally interact, putting them at risk of being exposed to zoonotic viruses that are found in the local wildlife populations. The transfer of zoonotic viruses to humans can occur directly or indirectly. For example, during an outbreak of 276 cases of Nipah virus in Malaysia and Singapore in 1998-99, most infected individuals had contact with sick pigs that were being farmed commercially as livestock and then slaughtered for meat. The pigs contracted Nipah virus from the local bats, which were thought to have spread the virus through urine or by dropping partially eaten fruit, laden with bat saliva, into the pig stalls. The bats may have acquired the fruit from the fruit trees on the farms. Human infection with Nipah virus causes encephalitis, fever, headache, and reduced consciousness. Forty percent of infected individuals died during this outbreak. Interestingly, Nipah virus infection is asymptomatic in bats.","This study was carried out in a livestock market located within the eastern region of Abu Dhabi Emirate, UAE, during the period of March\u2013April 2015 at Abu Dhabi Food Control Authority Veterinary Laboratory Abu Dhabi, UAE in response to the MERS-CoV notification from the Health Authority of Abu Dhabi of a German citizen who visited the same livestock market during his travels in UAE. Collection of samples from camels performed in this study was approved by the Centers for Disease Control and Prevention (CDC) Institutional Animal Care and Use Committee protocol number 2702HALMULX. A total of 376 nasal swabs were collected during week 1 from a livestock market and screened for MERS-CoV by PCR. These included 210 dromedaries that originated from local farms within Abu Dhabi Emirate, 106 camels from Oman and 60 camels of unknown origin. Camels sampled were predominantly male (n=269, 71.5%) and aged &lt;1 year (n=255, 67.8%). After week 1, positive camels from pens 17, 19 and 20 were microchipped and sampled in weeks 3, 4, 5 and 6.","Commercial and household livestock farms across Ghana were targeted and a purposive sampling strategy was adopted. Target farms were shortlisted and visited to engage and sensitize the farm owners, family, and workers as well as the entire community. During the sensitization visits, the objectives of the study, the study design, and information on use of data was provided to potential participants. Participants were allowed to ask questions and were further encouraged to seek clarification on issues they were not convinced about.","The closest phylogenetic neighbors to MERS-CoV are putative bat coronaviruses in China (BtCoV-HKU4 and BtCoV-HKU5) [3], the Netherlands (BtCoV/VM314/2008} [2], and a recently discovered isolate from South Africa [6]. All four of these bat coronaviruses have been sequenced only from bat samples and have never been isolated as live viruses from either bats or the environment. The natural reservoir of MERS-CoV has not been identified, although its similarity to these other four viruses suggests that it is of bat origin. Importantly, SARS-CoV emerged from bats as well [7]. Anecdotal evidence suggests that MERS-CoV may have been transmitted to humans via livestock (camels or goats); however, there is no scientific data yet to support this theory.","Worldwide there are roughly 30 million dromedary camels, with highest numbers found in Africa and the Middle East (Figure 1). Due to increased consumption and contact with camel meat and milk, camels represent a significant point source for zoonotic disease transmission to humans. Pastoralist camel production, in particular, is associated with a risk of disease spillover from wildlife into camel populations, and from camels to wildlife, as well as less opportunities for disease surveillance and control. By 2050 the human population is projected to grow by 2.5 billion, with nearly half of all population growth occurring in the African continent (Roser 2018). Along with the increase in human population in Africa and the Middle East, the camel population has also experienced steady growth as the demand and production of camel meat and milk escalates (FAOSTAT 2016). More and more, nomadic herders are switching to an agricultural or urban setting to raise camels and this change in camel production brings potential risks of disease emergence and transmission. In addition, with the realities of climate change and the increase in drought conditions, camel hardiness has led to a shift in livestock choice in many regions of the world from cattle to camels, making camels even more abundant (Watson et al. 2016)","Nomadic, sedentary, mixed lifestyles and extensive, semi-extensive and intensive camel breeding systems occur in African ecosystems. Extensive system/nomadic lifestyle are characterised by the use of natural resources, low inputs, and herd mobility [24]. However, camel husbandry practices and the use of camels have changed in the last five decades in the following ways: (i) increasing camel populations in settled livestock farming systems, (ii) use of camels in agriculture-related work, (iii) camel trade being more closely market integrated and (iv) increasing importance of camels for the sustainability and resilience of farms which traditionally relied on cattle [25]. These changing camel herding practices lead to sedentary or mixed lifestyles with intensive or semi-intensive camel production systems (milk, meat, skin etc.) [26]. Usually, camel calves are suckled by their mother during the first year of life. Camels are considered as young and sexually immature until 2\u20134 years-old. Males represent 20 to 40% of the herd [27]. Adult males are separated from females and young camels in non-extensive systems because of their aggressiveness associated with sexual behaviour. In extensive systems, the male is let with non-lactating females for reproduction only and during the rutting season. The contacts of adult males with young (less than 4 years-old) camels is not common.","Once again, a virus has jumped the species barrier. Coronavirus 2019-nCoV emerged apparently in a wet market in China, and after a few weeks the number of cases already exceeds those of SARS in 2002/ 03 ( Fig. 1 ) [1] in terms of both morbidity and mortality. Excellent publications have addressed many aspects of a possible novel endemic/ pandemic zoonosis, often with a focus on MERS-CoV (2-4), including prevention [2] , drug and vaccine-development for coronaviruses [3] , or the importance of \"a 'One Health' approach to control \u2026 \u2026zoonotic pathogens with epidemic potential\" [4] . The hugely increased appetite for meat-products worldwide, but also in China [5] , is likely to increase livestock production and sale, as well as scavenging of remaining wildlife resources, primarily the latter with consequent increases in the risk of exposure to novel infectious agents.","Interestingly, some of these zoonotic viral pathogens were first exotic to Saudi Arabia (e.g., MERS-CoV and AHFV) and should be of more concern when reported in prevalence studies, and whenever they are detected by Saudi authorities. Epidemiological data should be focused more on both the trade routes and wildlife migration across the region, since these are potential risks for Saudi Arabia (e.g., from Yemen, Egypt, Gulf areas, and Sudan). Fortunately, there are many ways and/or approaches to improve the control of such different zoonotic pathogens in animals and humans in Saudi Arabia. However, the control measures of these viral zoonotic pathogens will not only benefit Saudi Arabia or Arabian Peninsula but will also be of high benefit to other countries, especially those with low prevalence, by stopping or controlling the spread of the epidemic worldwide. Prevention, control, and management of several zoonotic diseases usually require several important measures including the following. Having vaccination protocols for all suspected animal species by the use of up to date vaccines and compliance with the standards needed for all animals. Taking into account the highly needed and important investigation for these zoonotic viral diseases vectors, including vector breeding control (including vectors, hosts, and arthropods), and control of the animals (livestock) movements, with respect to trade and export [212,213]. Because an intensive livestock trade exists between Saudi Arabia and its neighboring countries, there may be increased risk of reemerging viral diseases of all kinds [214,215]. This is supported by several previous studies concerned with the route of livestock trade between Saudi Arabia and the neighboring countries (e.g., rabies through Yemen and/or Oman [36,216]; RVF through Kenya, Djibouti, and/or Egypt [127,149,212]; CCHF through Sudan [121]; influenza through Oman and Egypt [71,87,121,217,218,219]; WNV through Emirates, Egypt, Jordan, and Israel [196,197,199,203,208]; and DHFV through Egypt [190]; as well as MERS-CoV and AHFV viral infections, which originated and are transmitted globally from Saudi Arabia) [34,52,53,97,98]. ","In conclusion, have we learned lessons? Yes and no. These events are of global public health and economic importance and need collective societal response. But governments and civil society are not heeding these warnings, as the 2019-nCoV attests. 9 Concerns have been repeatedly raised and voiced since the idea of One Health was first expressed in around 2000. 10 What we need to learn and communicate is that the zoonotic or agricultural bridging of novel pathogens from domestic and captive wildlife needs urgent attention, along with attention to the human appetite for meat. This approach is easily achieved for coronavirus threatseg, by substantially reducing the trade of risky species of wild caught animals for food or other purposes, and a culturally sensitive ban on the sale of these animals in wet markets. Vaccines and therapeutic alternatives might be possible and are needed, but they are a response, because the emerging strain is unpredictable and a vaccine is unlikely to prevent the initial events. In some parts of Africa, prevention of Ebola virus and future coronavirus threats require shifts in food habits, a transition from bushmeat being a cultural norm or primary source of protein, and by discouraging agricultural development that brings bats into increased contact with humans or livestock. In the Middle East, re-evaluating and improving infection prevention and control measures for camel farms, a recent introduction coincident with the emergence of MERS-CoV, would be a positive step forward.","In conclusion, have we learned lessons? Yes and no. These events are of global public health and economic importance and need collective societal response. But governments and civil society are not heeding these warnings, as the 2019-nCoV attests. 9 Concerns have been repeatedly raised and voiced since the idea of One Health was first expressed in around 2000. 10 What we need to learn and communicate is that the zoonotic or agricultural bridging of novel pathogens from domestic and captive wildlife needs urgent attention, along with attention to the human appetite for meat. This approach is easily achieved for coronavirus threatseg, by substantially reducing the trade of risky species of wild caught animals for food or other purposes, and a culturally sensitive ban on the sale of these animals in wet markets. Vaccines and therapeutic alternatives might be possible and are needed, but they are a response, because the emerging strain is unpredictable and a vaccine is unlikely to prevent the initial events. In some parts of Africa, prevention of Ebola virus and future coronavirus threats require shifts in food habits, a transition from bushmeat being a cultural norm or primary source of protein, and by discouraging agricultural development that brings bats into increased contact with humans or livestock. In the Middle East, re-evaluating and improving infection prevention and control measures for camel farms, a recent introduction coincident with the emergence of MERS-CoV, would be a positive step forward.","Despite mounting evidence that camels can transmit MERS-CoV to humans, there have only been a few studies focused on full-genome sequencing analysis of MERS-CoVs from camels. In all, 129 full or nearly full-genome MERS-CoV sequences were obtained from 109 unique camels in this study. These camel MERS-CoV genomes sequenced were placed throughout the MERS-CoV phylogenetic tree and represented four lineages previously identified by Sabir et al.21 as well as two new lineages (lineages 6 and 7). The majority of samples fell within the established lineage 5 and the lineage 7. Lineage 5 MERS-CoV sequences include human MERS-CoVs from the Republic of Korea outbreak as well as human cases from Saudi Arabia.52 Previously, Lau et al.20 identified two human MERS-CoV sequences, KF600612 and KF600620, both from Saudi Arabia from 2012, as a potentially new lineage. Sabir et al.21 placed these two sequences as an outlier to lineage 2 and our analysis places them within lineage 2. Lineage 7 is almost entirely comprised of MERS-CoVs genomes from camels, with the notable exception of the travel-related MERS-CoV infection in a German citizen who visited a livestock market in UAE before becoming ill. We also detected smaller numbers of camel MERS-CoVs within two known lineages (2 and 3) and a new lineage, 6. Interestingly, in three separate pens there were three separate instances of highly related MERS-CoV genomes identified indicating some of the camels may have been infected at the market by their pen mates. Most of the different lineages of MERS-CoV identified were throughout the pens and of the 32 pens sampled, 16 had more than one lineage of MERS-CoV present. This large diversity of camel MERS-CoVs was detected in camels sampled from a single livestock market. This is in contrast to a large sequencing study, which sequenced camels from several cities and various locations camels were housed.21 In the previous study, camels were sampled from slaughterhouses, farms and wholesale markets in Jeddah, Riyadh and Taif within Saudi Arabia. Despite a large sample size, our study did not detect any camel MERS-CoV sequences belonging to clade A, which has been identified in a camel from UAE in a previous study.20 A recent study of MERS-CoV in dromedaries in Egypt identified a potentially new MERS-CoV lineage, however only a limited amount of the Spike gene (600 bp) was sequenced and used for comparisons.53","Severe acute respiratory syndrome (SARS) is a form of atypical pneumonia that apparently originated in Guangdong Province of the People\u2019s Republic of China in late 2002. A novel coronavirus, the SARS-associated coronavirus (SARS-CoV) was identified as the agent responsible for this first pandemic of the new millennium. After its recognition in early 2003, the infection was reported worldwide. As of July 31, 2003, 8,096 cases of \u2018probable SARS\u2019, have been reported in more than 30 countries world-wide with 774 (9.6%) deaths [85]. Human-to-human transmission of the virus, the lack of awareness in hospital infection control, and international air travel facilitated the rapid global dissemination of this agent. The findings that horseshoe bats are the natural reservoir for SARS-CoV-like virus and that civets are the amplification host emphasizes the importance of wildlife and biosecurity in farms and wet markets, which can serve as the source and amplification centers for emerging infections [86].","Livestock production in China has expanded at an unprecedented rate over the past three decades. The consumption of pigs increased 3.7-fold from 160 million in 1978 to 610 million in 2008, of cattle increased 1.5-fold from 70 million to 106 million, of sheep and goats increased 1.65-fold from 170 million to 281 million, and of poultry increased 7.3-fold from 170 million to 9700 million. 57 Large-scale farms and scattered small-scale individual farms coexist in China, and the latter account for the majority. 58 Almost every household in rural areas is engaged in livestock production, including pigs, chickens, bovines, sheep, ducks, etc. Different domestic animals live in the same environment, which provides a dynamic micro-environment that contributes to interspecies transmission, even from animals to humans. 59, 60 Another example is human echinococcosis in Tibetan pastoral communities in western China, in which dogs, domestic animals (yaks, sheep, and goats), and wild animals living in the same environment are involved in the transmission cycle of Echinococcus granulosus and Echinococcus multilocularis, contributing to the high prevalence of human echinococcosis. 61 Due to the practice of scattered small-scale individual farming and the lack of meat inspection at such small operations, farmers often slaughter their animals for their own consumption. Compared with professionals in modern abattoirs, the local farmers do not wear protective clothing or gloves, resulting in high incidences of brucellosis infection in this population and other livestock-related occupations, such as herdsmen, coat and leatherworkers, and veterinarians. 37 On the other hand, the largescale farms, also termed concentrated animal feeding operations (CAFO), are increasing at an unprecedented rate in China, which will increase the risk of disease transmission.","The emergence of MERS-CoV nearly 10 years after SARS-CoV disappearance has shown that pathogenic coronaviruses may continue to spill over from zoonotic sources into the human population. Surveillance studies of viruses in animal species, including bats, rodents, and livestock, will help us to understand the potential human pathogens that exist in the environment before they can spill over human population. There are no antiviral medications available to treat MERS-CoV infections and the patient management is purely symptomatic. There is a need to develop vaccines and therapeutic strategies to prepare for the emerging coronaviruses. The sooner we understand the dynamics of current threats, the more we will be better equipped to save people from infection and disease.","Before the emergence of SARS-CoV, coronavirus research was focused on the veterinary field, where coronaviruses cause devastating epizootics of respiratory or enteric diseases in livestock and poultry. In cats, two coronaviruses are described, the feline infectious peritonitis virus (FIPV) and feline enteric coronavirus (FECV). These feline coronaviruses are spread worldwide and infect domestic cats and other members of the Felidae family. An infection with FECV is usually subclinical, except in young kittens where it may cause mild to severe diarrhea (Pedersen et al., 1981) . In contrast, FIPV infection causes a chronic and very often fatal peritonitis which is the most important cause of death of infectious origin in cats.","SARS appears to be due to an infection by a novel coronavirus (Cv). 1, 2 The usual human-infecting Cv strains cause up to 30% of \"colds\" but only rarely cause pneumonia. However, animal-strain Cv can cause extensive epidemics of lower respiratory tract diseases in livestock and poultry. Current evidence suggests that the SARS-associated Cv is a previously unknown Cv strain, probably originating in nonhuman hosts, which has somehow acquired the capacity to infect human beings. Airborne transmission by both large droplets and droplet nuclei appears to allow this Cv to reach the alveoli, leading to a high incidence of pneumonia.","Disease Surveillance and Notification Officers, who are employed by the State Ministry of Health and familiar with the communities in this study, performed specimen and data collection. Symptoms considered were derived in accordance with other ARI surveillance efforts: sore throat, fever, couch, running nose, vomiting, body ache, leg pain, nausea, chills, shortness of breath [10, 26]. Gender and age, type of residential area (rural/urban), education level, proximity of residence to livestock, proximity to an untarred road and number of people who sleep in same room, were all recorded. The general difference between the two settings was that those from the hospital had severe illnesses, while those from the community were generally \u201chealthy\u201d but exhibiting ARI symptoms (i.e. mild illness).","Emerging infectious viruses increasingly threaten human, domestic animal and wildlife health [12] . Sixty percent of emerging infectious diseases in humans are of zoonotic origin [13] . Wildlife trade and consumption of bushmeat, especially in Africa and Asia, have increasingly played a role in pathogen spillovers into human populations [14, 15] . Wildlife markets have recently facilitated the spillover of SARS-CoV-2 to humans [16] resulting in a pandemic [17] . The 2002-2003 SARS-CoV outbreak [18] , the Ebola outbreak in West Africa [19] and the global emergence of HIV [20] have all been linked to wildlife trade and bushmeat consumption. Early detection of novel infectious agents in wildlife represent a key factor to prevent their emergence. However, identification, surveillance and monitoring of emerging viruses using currently broadly applied approaches based on direct sampling of wildlife requires enormous investment in sampling, particularly for viruses that have low prevalence [21] . For example, 25,000 wild birds were sampled in Germany to detect avian influenza prevalence below 1% [22] . Similarly, sampling of over 8,157 animals in Poland was required to determine an 0.12% prevalence of African swine fever virus (ASF) [23] . Sampling under remote field conditions or in developing countries present additional challenges.","A vector can be defined as an organism that transmits a pathogen or disease-causing organism from a reservoir to a host. In the context of this review, vectors are restricted to arthropods, and particularly mosquitoes, ticks, sand flies and Culicoides or biting midges, as transmitters of pathogenic threats to humans or livestock. The role of mosquitoes in pathogen emergence is largely one of major geographic spread due to incursions of mosquitoes into new habitats. There are a number of major mosquito species that have jumped continents over the past three decades (e.g. [110 ] ), but there is little doubt that the most important ongoing threats come from extensive tropical urbanization and the colonization of this expanding habitat by Aedes (Stegomyia) aegypti [111] , and the global expansion in the geographic distribution of Aedes (Stegomyia) albopictus [112] . The latter has expanded to establish in at least 26 new countries in Africa, Europe and the Americas from its original home in tropical forests of south-eastern Asia. Thus there is a significantly increased risk of transmission of arthropod-borne (arbo)viral diseases, especially dengue and chikungunya [110 ,113-115] .","Human population growth and over-exploitation of the land, such as logging, mining, road construction, and agricultural production, has resulted in deforestation. These changes in turn cause a cascade of factors that aggravate the emergence of zoonotic diseases, such as forest fragmentation, population migration, pathogen introduction, and environmental pollution. 52 Deforestation forces wildlife to leave their habitat, increasing interactions among humans, wildlife, and livestock, and thus the potential for the transmission of pathogens directly or indirectly to human beings. 53, 54 Some zoonotic diseases, including mountain and desert subtypes of leishmaniasis in western China, Lyme disease in northern China, and SFTS in central and northeast China caused by a novel bunyavirus, are related to deforestation, and humans are infected by the bites of wild vectors, Phlebotomus chinensis for leishmaniasis, Ixodes persulcatus for Lyme disease, and Haemaphysalis longicornis for SFTS during agricultural activities on the land changed from a forest. 9,55,56","The emergence of a new coronavirus (2019-nCoV) in Wuhan creates a sense of d\u00e9j\u00e0 vu with the severe acute respiratory syndrome coronavirus (SARS-CoV) epidemic in China in 2003. Coronaviruses are enveloped, positivestranded RNA viruses of mammals and birds. These viruses have high mutation and gene recombination rates, making them ideal for pathogen evolution. 1 In humans, coronavirus is usually associated with mild disease, the common cold. Previous emerging novel coronaviruses, such as SARS-CoV and Middle East respiratory syndrome coronavirus (MERS-CoV), which emerged in the Middle East in 2012, were associated with severe and sometimes fatal disease. MERS-CoV was less pathogenic than SARS-CoV, with the most severe infections mainly in individuals with underlying illnesses. Clinically and epidemiologically, the contemporary 2019-nCoV in China seems to resemble SARS-CoV. The genome of 2019-nCoV also appears most closely related to SARS-CoV and related bat coronaviruses. 2 The infection has now spread widely, with phylogenetic analysis of the emerging viruses suggesting an initial single-locus zoonotic spillover event in November, 2019, 3 There is an increasing focus on the human-animalenvironment disease interface, as encompassed in the One Health concept. Mortalities, disability-adjusted life-years, and billions of dollars of economic losses from these infections demand action and investment in prevention to face novel challenges to human and animal health. Research has led to better understanding of the nature and drivers of cross-species viral jumps, but the detail is still elusive. No reservoir population of bats for SARS and MERS-CoV or Ebola virus have been definitively identified, despite considerable searching, possibly because of the source virus circulating in small and isolated populations. Forensic examination has clarified the human infection sources and multispecies involvement in these diseases, with some species confirmed as competent hosts (eg, camels for MERS-CoV 4 ), bridge (or amplifying) hosts (eg, pigs for Nipah virus, non-human primates for Ebola virus 5 ), or deadend hosts. The crucial checkpoint is the jump and bridging of the viruses to humans, which occurs most frequently through animal-based food systems. In the case of SARS, markets with live and dead animals of wild and domestic origins were the crucible for virus evolution and emergence in the human population. Once the viruses' functional proteins enabled cell entry in civets (Paguma larvata) and racoon dogs (Nyctereutes procyonoides), the bridge was established and it was only a matter of time before the jump to humans occurred. 6 Sequence comparison of civet viruses suggested evolution was ongoing; this was further supported by high seroprevalence of antibodies against SARS-CoV among civet sellers, suggesting previous cross-species transmission events without necessarily human-tohuman transmission. 7, 8 Similarly, early Ebola virus was mostly associated with bushmeat and its consumption in Africa; Nipah virus is associated with date palm sap, fruit, and domestic pig farms; MERS is associated with the camel livestock industry; and H5N1 arose from viral evolution in domestic and wild birds, to ultimately bring all these cases to humans. The 2019-nCoV is another virus in the pipeline that originated from contact with animals, in this case a seafood and animal market in Wuhan, China.","The emergence of a new coronavirus (2019-nCoV) in Wuhan creates a sense of d\u00e9j\u00e0 vu with the severe acute respiratory syndrome coronavirus (SARS-CoV) epidemic in China in 2003. Coronaviruses are enveloped, positivestranded RNA viruses of mammals and birds. These viruses have high mutation and gene recombination rates, making them ideal for pathogen evolution. 1 In humans, coronavirus is usually associated with mild disease, the common cold. Previous emerging novel coronaviruses, such as SARS-CoV and Middle East respiratory syndrome coronavirus (MERS-CoV), which emerged in the Middle East in 2012, were associated with severe and sometimes fatal disease. MERS-CoV was less pathogenic than SARS-CoV, with the most severe infections mainly in individuals with underlying illnesses. Clinically and epidemiologically, the contemporary 2019-nCoV in China seems to resemble SARS-CoV. The genome of 2019-nCoV also appears most closely related to SARS-CoV and related bat coronaviruses. 2 The infection has now spread widely, with phylogenetic analysis of the emerging viruses suggesting an initial single-locus zoonotic spillover event in November, 2019, 3 There is an increasing focus on the human-animalenvironment disease interface, as encompassed in the One Health concept. Mortalities, disability-adjusted life-years, and billions of dollars of economic losses from these infections demand action and investment in prevention to face novel challenges to human and animal health. Research has led to better understanding of the nature and drivers of cross-species viral jumps, but the detail is still elusive. No reservoir population of bats for SARS and MERS-CoV or Ebola virus have been definitively identified, despite considerable searching, possibly because of the source virus circulating in small and isolated populations. Forensic examination has clarified the human infection sources and multispecies involvement in these diseases, with some species confirmed as competent hosts (eg, camels for MERS-CoV 4 ), bridge (or amplifying) hosts (eg, pigs for Nipah virus, non-human primates for Ebola virus 5 ), or deadend hosts. The crucial checkpoint is the jump and bridging of the viruses to humans, which occurs most frequently through animal-based food systems. In the case of SARS, markets with live and dead animals of wild and domestic origins were the crucible for virus evolution and emergence in the human population. Once the viruses' functional proteins enabled cell entry in civets (Paguma larvata) and racoon dogs (Nyctereutes procyonoides), the bridge was established and it was only a matter of time before the jump to humans occurred. 6 Sequence comparison of civet viruses suggested evolution was ongoing; this was further supported by high seroprevalence of antibodies against SARS-CoV among civet sellers, suggesting previous cross-species transmission events without necessarily human-tohuman transmission. 7, 8 Similarly, early Ebola virus was mostly associated with bushmeat and its consumption in Africa; Nipah virus is associated with date palm sap, fruit, and domestic pig farms; MERS is associated with the camel livestock industry; and H5N1 arose from viral evolution in domestic and wild birds, to ultimately bring all these cases to humans. The 2019-nCoV is another virus in the pipeline that originated from contact with animals, in this case a seafood and animal market in Wuhan, China.","Because fruit bats had been identified as the natural reservoir for Hendra virus, several ecological surveys were undertaken to investigate the possibility that Nipah virus also had originated from fruit bats. Early studies were able to detect antibody titers against Nipah virus in five species of fruit bats [39] , and eventually the virus was isolated from the urine of an Island flying fox and from a partially eaten fruit regurgitated from one of these bats [40] . These findings implicated bats of the Pteropus species as the natural reservoir for Nipah virus. The critical link with the human population was made when the virus moved from fruit bats to swine. The susceptibility of swine and the marked respiratory involvement created an outbreak situation, with resulting spread to humans and a major public health crisis. The proposed chain of events that enabled this interaction began around 1997 when extensive slash-and-burn deforestation produced a haze that extended all over Southeast Asia [41] . This deforestation and haze, aggravated by an El Nin\u02dco Southern Oscillation drought, decreased the already scarce quantities of fruiting forest in the region. Impelled by these anthropogenic factors, Pteropus bats invaded areas of fruit cultivation, like the index farms, that were also used as piggeries.","Besides pets, livestock are another category of mammals that are frequently in direct contact with humans. We profiled the ACE2 expression in cows, pigs, rabbits (which can also serve as pets), and goats. In these animals, ACE2 was highly expressed in multiple tissues including kidney, liver, and heart, but not in muscles ( Figures 5D-5F and S2B in Supplementary data). ACE2 was also highly expressed in the fat of pigs.","The ramifi cations of foot and mouth were felt elsewhere due to anxieties about its being unwittingly exported. Busch Gardens in Tampa Bay, Florida, directed foreigners away from susceptible animals like giraffes and gazelles. Visitors with a history of travel to infected areas were requested to desist from joining the optional tours, the highlight of which was close proximity to wildlife, and offered alternatives such as half-price entry to the water park (The Business Journal, 2001). Immigration procedures were also modifi ed in an attempt to prevent the disease invading countries such as the Irish Republic where livestock farming is a key economic sector. International arrivals were asked to complete declaration forms concerning their travels and walk across disinfected mats when entering countries.","In the present study, MERS-CoV RNA was detected in 109 (29%) out of the 376 camels screened at a livestock market in Abu Dhabi Emirate, UAE sampled between March and April 2015. Such prevalence is higher compared to the previously reported 1.6\u201315% molecular prevalence detected in camels at the borders with Saudi Arabia and Oman, slaughterhouses and farms, suggesting that livestock markets may represent a location for greater potential for human and camel infections.23, 38, 51 A total of 129 full or nearly full MERS-CoV genomes were obtained from camels sampled in this study, including camel MERS-CoV genomes that are more closely related to human MERS cases than previously available camel MERS-CoV genomes. Within lineage 5, two camel MERS-CoV genomes, MF598617 and MF598618, were more related to human MERS cases (KT806051 and KT225476, respectively) than previous camel MERS-CoVs, with only 15 nucleotide differences between MF598617 and KT806051 (differences were throughout the genome, with 1 difference in the spike gene) and 30 nucleotide differences between MF598618 and KT225476 (differences were primarily in ORF1AB and spike genes). Interestingly, while the camels sampled in this study came from UAE and Oman, the human MERS-CoV genome (KT806051) related to MF598617 was from a case in Riyadh, Saudi Arabia. In addition, a camel MERS-CoV that is closely related to the 2015 Germany imported case from UAE was identified with only 7 nucleotide differences, 2 of which were in the spike gene. A previously sequenced camel MERS-CoV genome was also only 7 non-overlapping nucleotides different than the 2015 Germany imported case, but that sequence was obtained from a camel in a closed dairy herd in Dubai (KT751244). This study demonstrates that livestock markets may actively participate in the spread of MERS-CoV among camels and/or humans. Unlike the severe acute respiratory syndrom-CoV outbreak, which originated from a limited number of zoonotic events and then acquired characteristic genetic changes associated with more efficient person-to-person spread, MERS-CoV cases and outbreaks appear to result at least in part from multiple ongoing zoonotic transmissions. The great diversity of MERS-CoVs in humans is mirrored in the camel population as shown in this study as well as previous studies.8, 20, 21 This supports the hypothesis that some human MERS-CoV infections are a result of multiple independent transmissions from camels.8, 15"],"title":["Non-invasive surveys of mammalian viruses using environmental DNA","A qualitative study of zoonotic risk factors among rural communities in southern China","Bat origin of human coronaviruses","A strategy to prevent future pandemics similar to the 2019-nCoV outbreak","A strategy to prevent future pandemics similar to the 2019-nCoV outbreak","A strategy to prevent future pandemics similar to the 2019-nCoV outbreak","A strategy to prevent future pandemics similar to the 2019-nCoV outbreak","A viral metagenomic survey identifies known and novel mammalian viruses in bats from Saudi Arabia","A qualitative study of zoonotic risk factors among rural communities in southern China","Viral Diversity, Prey Preference, and Bartonella Prevalence in Desmodus rotundus in Guatemala","Chimpanzee adenoviral vectors as vaccines for outbreak pathogens","Development of a Whole-Virus ELISA for Serological Evaluation of Domestic Livestock as Possible Hosts of Human Coronavirus NL63","26 Evolution of SARS Coronavirus and the Relevance of Modern Molecular Epidemiology","Camel production systems in Ethiopia: a review of literature with notes on MERS-CoV risk factors","What Have We Learned About Middle East Respiratory Syndrome Coronavirus Emergence in Humans? A Systematic Literature Review","Evaluation of an accelerated hydrogen peroxide disinfectant to inactivate porcine epidemic diarrhea virus in swine feces on aluminum surfaces under freezing conditions","Bats and their virome: an important source of emerging viruses capable of infecting humans","Camel production systems in Ethiopia: a review of literature with notes on MERS-CoV risk factors","Shell disorder analysis predicts greater resilience of the SARS-CoV-2 (COVID-19) outside the body and in body fluids","","An Overview of the Most Significant Zoonotic Viral Pathogens Transmitted from Animal to Human in Saudi Arabia","Reservoirs and vectors of emerging viruses","A Review of Zoonotic Pathogens of Dromedary Camels","Emerging viruses of zoonotic and veterinary importance","Chapter 16 Emerging and Reemerging Viral Diseases","Diversity of Middle East respiratory syndrome coronaviruses in 109 dromedary camels based on full-genome sequencing, Abu Dhabi, United Arab Emirates","Development of a Whole-Virus ELISA for Serological Evaluation of Domestic Livestock as Possible Hosts of Human Coronavirus NL63","Emergence of the Middle East Respiratory Syndrome Coronavirus","A Review of Zoonotic Pathogens of Dromedary Camels","Risk factors for MERS coronavirus infection in dromedary camels in Burkina Faso, Ethiopia, and Morocco, 2015","Coronavirus 2019-nCoV: Is the genie already out of the bottle?","An Overview of the Most Significant Zoonotic Viral Pathogens Transmitted from Animal to Human in Saudi Arabia","2019-nCoV in context: lessons learned?","2019-nCoV in context: lessons learned?","Diversity of Middle East respiratory syndrome coronaviruses in 109 dromedary camels based on full-genome sequencing, Abu Dhabi, United Arab Emirates","Emerging Viral Infections in India","Major emerging and re-emerging zoonoses in China: a matter of global health and socioeconomic development for 1.3 billion","Outbreak of Middle East respiratory syndrome coronavirus, Saudi Arabian experience","Plant lectins are potent inhibitors of coronaviruses by interfering with two targets in the viral replication cycle","Beyond our pages","Etiology of respiratory tract infections in the community and clinic in Ilorin, Nigeria","Non-invasive surveys of mammalian viruses using environmental DNA","Reservoirs and vectors of emerging viruses","Major emerging and re-emerging zoonoses in China: a matter of global health and socioeconomic development for 1.3 billion","2019-nCoV in context: lessons learned?","2019-nCoV in context: lessons learned?","Emerging infections in animals\u2014potential new zoonoses?","Atlas of ACE2 gene expression in mammals reveals novel insights in transmission of SARS-Cov-2","Chapter 7 Tourism and Health Crises","Diversity of Middle East respiratory syndrome coronaviruses in 109 dromedary camels based on full-genome sequencing, Abu Dhabi, United Arab Emirates"],"url":["https://doi.org/10.1101/2020.03.26.009993","https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7017878/","https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4687304/","https://doi.org/10.1016/j.bsheal.2020.01.003","https://doi.org/10.1016/j.bsheal.2020.01.003","https://doi.org/10.1016/j.bsheal.2020.01.003","https://doi.org/10.1016/j.bsheal.2020.01.003","https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6457491/","https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7017878/","https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5164864/","https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5718829/","https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6356407/","https://doi.org/10.1016/b978-0-12-799942-5.00026-3","https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7099212/","https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6396572/","https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5709983/","https://doi.org/10.1016/j.coviro.2012.11.006","https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7099212/","https://doi.org/10.1016/j.micpath.2020.104177","https://doi.org/10.1016/b978-0-323-03453-1.10063-x","https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6471281/","https://doi.org/10.1016/j.coviro.2013.02.002","https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7087575/","https://doi.org/10.1016/j.tvjl.2017.12.022","https://doi.org/10.1016/b978-0-12-800947-5.00016-8","https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5717090/","https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6356407/","https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3764217/","https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7087575/","https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5388105/","https://doi.org/10.1016/j.tmaid.2020.101577","https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6471281/","https://doi.org/10.1016/s2542-5196(20)30035-8","https://doi.org/10.1016/s2542-5196(20)30035-8","https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5717090/","https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7100311/","https://doi.org/10.1016/j.ijid.2014.04.003","https://doi.org/10.1016/j.cmrp.2017.07.006","https://doi.org/10.1016/j.antiviral.2007.03.003","https://doi.org/10.1016/s0091-6749(04)70038-x","https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5719735/","https://doi.org/10.1101/2020.03.26.009993","https://doi.org/10.1016/j.coviro.2013.02.002","https://doi.org/10.1016/j.ijid.2014.04.003","https://doi.org/10.1016/s2542-5196(20)30035-8","https://doi.org/10.1016/s2542-5196(20)30035-8","https://doi.org/10.1016/j.cll.2004.05.001","https://doi.org/10.1101/2020.03.30.015644","https://doi.org/10.1016/b978-0-7506-7834-6.50008-9","https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5717090/"],"x":[0.317012812275689,0.8347286466570475,0.6585056771205677,0.09269732287491717,0.21380007527644784,0.9228188155139474,0.12303480586470095,0.8832476341317514,0.8289235435615104,0.23917171068174314,0.8624865495985639,0.3791682203898479,0.658212909516946,0.4877285625377341,0.9766682942311626,0.700819628036892,0.20054343928904705,0.23727117456911606,0.8381186487195957,0.8582003111773309,0.4905086617532811,0.04233381738479369,0.6405269245707216,0.1407644417554248,0.9706652807746176,0.3054557299741494,0.46107993399921865,0.6871954160520367,0.3120341807251634,0.5694918673523202,0.42274665662133604,0.9699889766581149,0.6312313883437487,0.6211330348251433,0.37079338828517683,0.49638540415883814,0.7145782931037793,0.5937950354533049,0.26568167426059897,0.5642122076018491,0.6461864816508786,0.8465752543631205,0.20974702206470597,0.34716747042293217,0.07290716724115043,0.26337385037013594,0.7311051825813741,0.2347383284484783,0.16068076097711403,0.7364071048170827],"y":[0.5497491683441528,0.948646044337226,0.9933915998680656,0.3868660372692635,0.8992619754853375,0.27654471939047054,0.363507079935532,0.48712735689932773,0.647538558835275,0.6204765402495697,0.15597152838225037,0.5031376885903406,0.4388667131720726,0.2651627721862446,0.7288740106453787,0.7540886732819316,0.32388745701594135,0.1156708650460313,0.9522505574507784,0.7439462493623764,0.14049794793270265,0.044122932948982396,0.8724902528589883,0.20744959130048024,0.0730192674466893,0.47196580392896703,0.9605277554755504,0.5122626164328249,0.4382488734250699,0.6010140237043542,0.8447771581078684,0.9553322317704596,0.5320716314394681,0.7488843163331846,0.7056531741233723,0.7725435679003388,0.33343998755626714,0.13607508004749747,0.6794699566393396,0.8500534845349074,0.9562678584126407,0.9455742642730225,0.3231897603526781,0.7631569186087754,0.508763307496151,0.385135319253174,0.15734367763935075,0.9551926806399507,0.7214133844714803,0.08137969083514895]},"selected":{"id":"9125"},"selection_policy":{"id":"9150"}},"id":"9052","type":"ColumnDataSource"},{"attributes":{"callback":null},"id":"9070","type":"TapTool"},{"attributes":{"source":{"id":"9052"}},"id":"9116","type":"CDSView"},{"attributes":{},"id":"9089","type":"Range1d"},{"attributes":{"children":[{"id":"9129"}]},"id":"9133","type":"Row"},{"attributes":{"below":[{"id":"9062"}],"center":[{"id":"9065"},{"id":"9069"}],"left":[{"id":"9066"}],"plot_height":340,"plot_width":900,"renderers":[{"id":"9080"}],"title":{"id":"9137"},"toolbar":{"id":"9073"},"toolbar_location":"below","x_range":{"id":"9054"},"x_scale":{"id":"9058"},"y_range":{"id":"9056"},"y_scale":{"id":"9060"}},"id":"9053","subtype":"Figure","type":"Plot"},{"attributes":{"callback":null,"tooltips":[["Title","@title"],["Abstract","@text"],["Published","@date"],["Authors","@authors"],["Journal","@journal"],["URL","@url"]]},"id":"9072","type":"HoverTool"},{"attributes":{"formatter":{"id":"9157"},"ticker":{"id":"9102"}},"id":"9101","type":"LinearAxis"},{"attributes":{},"id":"9144","type":"StringEditor"},{"attributes":{},"id":"9102","type":"BasicTicker"},{"attributes":{},"id":"9142","type":"StringEditor"},{"attributes":{},"id":"9091","type":"Range1d"},{"attributes":{"args":{"sd":{"id":"9051"},"sp":{"id":"9052"}},"code":"\n    if (sp.selected.indices.length == 0) {\n      return;\n    }\n    var sel = sp.selected.indices[0];\n    var new_selected = [];\n    for (var i = 0; i &lt; sd.data['doc'].length; i++) {\n      if (sd.data['doc'][i] == sp.data['doc'][sel]) {\n        new_selected.push(i);\n      }\n    }\n    if (JSON.stringify(new_selected) != JSON.stringify(sd.selected.indices) &amp;&amp;\n    !(sd.selected.indices.length == 1 &amp;&amp; new_selected.includes(sd.selected.indices[0]))) {\n      sd.selected.indices = new_selected;\n      sd.change.emit();\n    }\n    "},"id":"9126","type":"CustomJS"},{"attributes":{"axis":{"id":"9101"},"dimension":1,"ticker":null},"id":"9104","type":"Grid"},{"attributes":{"data_source":{"id":"9052"},"glyph":{"id":"9113"},"hover_glyph":null,"muted_glyph":null,"nonselection_glyph":{"id":"9114"},"selection_glyph":null,"view":{"id":"9116"}},"id":"9115","type":"GlyphRenderer"},{"attributes":{"formatter":{"id":"9147"},"ticker":{"id":"9067"}},"id":"9066","type":"LinearAxis"},{"attributes":{"editor":{"id":"9154"},"field":"text","formatter":{"id":"9155"},"title":"Passage","width":770},"id":"9120","type":"TableColumn"},{"attributes":{"text":""},"id":"9137","type":"Title"},{"attributes":{},"id":"9152","type":"StringEditor"},{"attributes":{"children":[{"id":"9130"},{"id":"9131"},{"id":"9132"},{"id":"9133"},{"id":"9134"}]},"id":"9135","type":"Column"},{"attributes":{"active_drag":"auto","active_inspect":"auto","active_multi":null,"active_scroll":"auto","active_tap":"auto","tools":[{"id":"9070"},{"id":"9071"},{"id":"9072"}]},"id":"9073","type":"Toolbar"},{"attributes":{"js_property_callbacks":{"change:indices":[{"id":"9126"}]}},"id":"9125","type":"Selection"},{"attributes":{"axis":{"id":"9066"},"dimension":1,"ticker":null},"id":"9069","type":"Grid"},{"attributes":{"text":"&lt;h3&gt;Document ranking&lt;/h3&gt;"},"id":"9128","type":"Div"},{"attributes":{},"id":"9071","type":"ResetTool"},{"attributes":{"below":[{"id":"9097"}],"center":[{"id":"9100"},{"id":"9104"}],"left":[{"id":"9101"}],"plot_height":340,"plot_width":900,"renderers":[{"id":"9115"}],"title":{"id":"9139"},"toolbar":{"id":"9108"},"toolbar_location":"below","x_range":{"id":"9089"},"x_scale":{"id":"9093"},"y_range":{"id":"9091"},"y_scale":{"id":"9095"}},"id":"9088","subtype":"Figure","type":"Plot"},{"attributes":{"editor":{"id":"9152"},"field":"scr","formatter":{"id":"9153"},"title":"Score","width":30},"id":"9119","type":"TableColumn"},{"attributes":{"active_drag":"auto","active_inspect":"auto","active_multi":null,"active_scroll":"auto","active_tap":"auto","tools":[{"id":"9105"},{"id":"9106"},{"id":"9107"}]},"id":"9108","type":"Toolbar"},{"attributes":{"formatter":{"id":"9159"},"ticker":{"id":"9098"}},"id":"9097","type":"LinearAxis"},{"attributes":{},"id":"9155","type":"StringFormatter"},{"attributes":{},"id":"9150","type":"UnionRenderers"},{"attributes":{"source":{"id":"9051"}},"id":"9087","type":"CDSView"},{"attributes":{"sizing_mode":"stretch_width","text":"&lt;h2&gt;Evidence that livestock could be infected (e.g., field surveillance, genetic sequencing, receptor binding) and serve as a reservoir after the epidemic appears to be over: Surveillance of mixed wildlife- livestock farms for SARS-CoV-2 and other coronaviruses in Southeast Asia.&lt;/h2&gt;"},"id":"9127","type":"Div"},{"attributes":{"text":"&lt;h3&gt;Paragraph ranking&lt;/h3&gt;"},"id":"9129","type":"Div"},{"attributes":{},"id":"9054","type":"Range1d"},{"attributes":{},"id":"9153","type":"StringFormatter"},{"attributes":{"fill_alpha":{"value":0.1},"fill_color":{"field":"color"},"line_alpha":{"value":0.1},"line_color":{"field":"color"},"radius":{"field":"rad","units":"data"},"x":{"field":"x"},"y":{"field":"y"}},"id":"9114","type":"Circle"},{"attributes":{},"id":"9060","type":"LinearScale"}],"root_ids":["9135"]},"title":"Bokeh Application","version":"2.0.1"}}
        </script>
        <script type="text/javascript">
          (function() {
            var fn = function() {
              Bokeh.safely(function() {
                (function(root) {
                  function embed_document(root) {
                    
                  var docs_json = document.getElementById('9372').textContent;
                  var render_items = [{"docid":"68bcf5ce-f2f1-4881-8178-050bfeaa41c5","root_ids":["9135"],"roots":{"9135":"782a74a2-695d-4689-b93d-48aa92951136"}}];
                  root.Bokeh.embed.embed_items(docs_json, render_items);
                
                  }
                  if (root.Bokeh !== undefined) {
                    embed_document(root);
                  } else {
                    var attempts = 0;
                    var timer = setInterval(function(root) {
                      if (root.Bokeh !== undefined) {
                        clearInterval(timer);
                        embed_document(root);
                      } else {
                        attempts++;
                        if (attempts > 100) {
                          clearInterval(timer);
                          console.log("Bokeh: ERROR: Unable to run BokehJS code because BokehJS library is missing");
                        }
                      }
                    }, 10, root)
                  }
                })(window);
              });
            };
            if (document.readyState != "loading") fn();
            else document.addEventListener("DOMContentLoaded", fn);
          })();
        </script>
    
  </body>
  
</html>