



<!DOCTYPE html>
<html lang="en">
  
  <head>
    
      <meta charset="utf-8">
      <title>Bokeh Plot</title>
      
      
        
          
        
        
          
        <script type="text/javascript" src="https://cdn.bokeh.org/bokeh/release/bokeh-2.0.1.min.js" integrity="sha384-JpP8FXbgAZLkfur7LiK3j9AGBhHNIvF742meBJrjO2ShJDhCG2I1uVvW+0DUtrmc" crossorigin="anonymous"></script>
        <script type="text/javascript" src="https://cdn.bokeh.org/bokeh/release/bokeh-widgets-2.0.1.min.js" integrity="sha384-xZlADit0Q04ISQEdKg2k3L4W9AwQBAuDs9nJL9fM/WwzL1tEU9VPNezOFX0nLEAz" crossorigin="anonymous"></script>
        <script type="text/javascript" src="https://cdn.bokeh.org/bokeh/release/bokeh-tables-2.0.1.min.js" integrity="sha384-4BuPRZkdMKSnj3zoxiNrQ86XgNw0rYmBOxe7nshquXwwcauupgBF2DHLVG1WuZlV" crossorigin="anonymous"></script>
        <script type="text/javascript">
            Bokeh.set_log_level("info");
        </script>
        
      
      
    
  </head>
  
  
  <body>
    
      
        
          
          
            
              <div class="bk-root" id="9a11312b-3c05-48d3-a79f-d30641580089" data-root-id="9779"></div>
            
          
        
      
      
        <script type="application/json" id="10016">
          {"c4d480b4-eb05-4f75-b928-38f6da1f92dc":{"roots":{"references":[{"attributes":{"source":{"id":"9695"}},"id":"9731","type":"CDSView"},{"attributes":{},"id":"9733","type":"Range1d"},{"attributes":{"below":[{"id":"9741"}],"center":[{"id":"9744"},{"id":"9748"}],"left":[{"id":"9745"}],"plot_height":340,"plot_width":900,"renderers":[{"id":"9759"}],"title":{"id":"9783"},"toolbar":{"id":"9752"},"toolbar_location":"below","x_range":{"id":"9733"},"x_scale":{"id":"9737"},"y_range":{"id":"9735"},"y_scale":{"id":"9739"}},"id":"9732","subtype":"Figure","type":"Plot"},{"attributes":{},"id":"9704","type":"LinearScale"},{"attributes":{"axis":{"id":"9741"},"ticker":null},"id":"9744","type":"Grid"},{"attributes":{"formatter":{"id":"9791"},"ticker":{"id":"9711"}},"id":"9710","type":"LinearAxis"},{"attributes":{},"id":"9739","type":"LinearScale"},{"attributes":{},"id":"9742","type":"BasicTicker"},{"attributes":{},"id":"9735","type":"Range1d"},{"attributes":{"formatter":{"id":"9803"},"ticker":{"id":"9742"}},"id":"9741","type":"LinearAxis"},{"attributes":{"source":{"id":"9695"}},"id":"9725","type":"CDSView"},{"attributes":{"active_drag":"auto","active_inspect":"auto","active_multi":null,"active_scroll":"auto","active_tap":"auto","tools":[{"id":"9714"},{"id":"9715"},{"id":"9716"}]},"id":"9717","type":"Toolbar"},{"attributes":{"callback":null},"id":"9714","type":"TapTool"},{"attributes":{},"id":"9798","type":"StringEditor"},{"attributes":{},"id":"9737","type":"LinearScale"},{"attributes":{},"id":"9702","type":"LinearScale"},{"attributes":{},"id":"9787","type":"StringFormatter"},{"attributes":{"text":"&lt;h3&gt;Document ranking&lt;/h3&gt;"},"id":"9772","type":"Div"},{"attributes":{},"id":"9788","type":"StringEditor"},{"attributes":{"axis":{"id":"9706"},"ticker":null},"id":"9709","type":"Grid"},{"attributes":{"axis":{"id":"9745"},"dimension":1,"ticker":null},"id":"9748","type":"Grid"},{"attributes":{},"id":"9700","type":"Range1d"},{"attributes":{},"id":"9750","type":"ResetTool"},{"attributes":{"formatter":{"id":"9801"},"ticker":{"id":"9746"}},"id":"9745","type":"LinearAxis"},{"attributes":{"args":{"sd":{"id":"9695"},"sp":{"id":"9696"}},"code":"\n    if (sp.selected.indices.length == 0) {\n      return;\n    }\n    var sel = sp.selected.indices[0];\n    var new_selected = [];\n    for (var i = 0; i &lt; sd.data['doc'].length; i++) {\n      if (sd.data['doc'][i] == sp.data['doc'][sel]) {\n        new_selected.push(i);\n      }\n    }\n    if (JSON.stringify(new_selected) != JSON.stringify(sd.selected.indices) &amp;&amp;\n    !(sd.selected.indices.length == 1 &amp;&amp; new_selected.includes(sd.selected.indices[0]))) {\n      sd.selected.indices = new_selected;\n      sd.change.emit();\n    }\n    "},"id":"9770","type":"CustomJS"},{"attributes":{},"id":"9711","type":"BasicTicker"},{"attributes":{"source":{"id":"9696"}},"id":"9760","type":"CDSView"},{"attributes":{},"id":"9793","type":"BasicTickFormatter"},{"attributes":{"below":[{"id":"9706"}],"center":[{"id":"9709"},{"id":"9713"}],"left":[{"id":"9710"}],"plot_height":340,"plot_width":900,"renderers":[{"id":"9724"}],"title":{"id":"9781"},"toolbar":{"id":"9717"},"toolbar_location":"below","x_range":{"id":"9698"},"x_scale":{"id":"9702"},"y_range":{"id":"9700"},"y_scale":{"id":"9704"}},"id":"9697","subtype":"Figure","type":"Plot"},{"attributes":{},"id":"9746","type":"BasicTicker"},{"attributes":{"data_source":{"id":"9696"},"glyph":{"id":"9757"},"hover_glyph":null,"muted_glyph":null,"nonselection_glyph":{"id":"9758"},"selection_glyph":null,"view":{"id":"9760"}},"id":"9759","type":"GlyphRenderer"},{"attributes":{"callback":null},"id":"9749","type":"TapTool"},{"attributes":{"formatter":{"id":"9793"},"ticker":{"id":"9707"}},"id":"9706","type":"LinearAxis"},{"attributes":{"callback":null,"tooltips":[["Title","@title"],["Passage","@text"],["Published","@date"],["Authors","@authors"],["Journal","@journal"],["URL","@url"]]},"id":"9751","type":"HoverTool"},{"attributes":{},"id":"9784","type":"UnionRenderers"},{"attributes":{"fill_color":{"field":"color"},"line_color":{"field":"color"},"radius":{"field":"rad","units":"data"},"x":{"field":"x"},"y":{"field":"y"}},"id":"9722","type":"Circle"},{"attributes":{"editor":{"id":"9788"},"field":"title","formatter":{"id":"9789"},"title":"Document title","width":770},"id":"9729","type":"TableColumn"},{"attributes":{},"id":"9698","type":"Range1d"},{"attributes":{"active_drag":"auto","active_inspect":"auto","active_multi":null,"active_scroll":"auto","active_tap":"auto","tools":[{"id":"9749"},{"id":"9750"},{"id":"9751"}]},"id":"9752","type":"Toolbar"},{"attributes":{},"id":"9786","type":"StringEditor"},{"attributes":{},"id":"9791","type":"BasicTickFormatter"},{"attributes":{},"id":"9715","type":"ResetTool"},{"attributes":{"editor":{"id":"9796"},"field":"scr","formatter":{"id":"9797"},"title":"Score","width":30},"id":"9763","type":"TableColumn"},{"attributes":{},"id":"9796","type":"StringEditor"},{"attributes":{},"id":"9803","type":"BasicTickFormatter"},{"attributes":{},"id":"9799","type":"StringFormatter"},{"attributes":{"fill_color":{"field":"color"},"line_color":{"field":"color"},"radius":{"field":"rad","units":"data"},"x":{"field":"x"},"y":{"field":"y"}},"id":"9757","type":"Circle"},{"attributes":{"text":""},"id":"9783","type":"Title"},{"attributes":{},"id":"9707","type":"BasicTicker"},{"attributes":{"children":[{"id":"9773"}]},"id":"9777","type":"Row"},{"attributes":{},"id":"9789","type":"StringFormatter"},{"attributes":{"fill_alpha":{"value":0.1},"fill_color":{"field":"color"},"line_alpha":{"value":0.1},"line_color":{"field":"color"},"radius":{"field":"rad","units":"data"},"x":{"field":"x"},"y":{"field":"y"}},"id":"9723","type":"Circle"},{"attributes":{"editor":{"id":"9798"},"field":"text","formatter":{"id":"9799"},"title":"Passage","width":770},"id":"9764","type":"TableColumn"},{"attributes":{"children":[{"id":"9730"},{"id":"9697"}]},"id":"9776","type":"Row"},{"attributes":{"children":[{"id":"9771"}]},"id":"9774","type":"Row"},{"attributes":{"source":{"id":"9696"}},"id":"9766","type":"CDSView"},{"attributes":{"columns":[{"id":"9728"},{"id":"9729"}],"height":340,"source":{"id":"9695"},"view":{"id":"9731"},"width":900},"id":"9730","type":"DataTable"},{"attributes":{"axis":{"id":"9710"},"dimension":1,"ticker":null},"id":"9713","type":"Grid"},{"attributes":{},"id":"9797","type":"StringFormatter"},{"attributes":{"editor":{"id":"9786"},"field":"scr","formatter":{"id":"9787"},"title":"Score","width":30},"id":"9728","type":"TableColumn"},{"attributes":{"args":{"sd":{"id":"9695"},"sp":{"id":"9696"}},"code":"\n    if (sd.selected.indices.length == 0) {\n      return;\n    }\n    var sel = sd.selected.indices[0];\n    var new_selected = [];\n    for (var i = 0; i &lt; sp.data['doc'].length; i++) {\n      if (sp.data['doc'][i] == sd.data['doc'][sel]) {\n        new_selected.push(i);\n      }\n    }\n    if (JSON.stringify(new_selected) != JSON.stringify(sp.selected.indices) &amp;&amp;\n    !(sp.selected.indices.length == 1 &amp;&amp; new_selected.includes(sp.selected.indices[0]))) {\n      sp.selected.indices = new_selected;\n      sp.change.emit();\n    }\n    "},"id":"9768","type":"CustomJS"},{"attributes":{"children":[{"id":"9772"}]},"id":"9775","type":"Row"},{"attributes":{"data":{"authors":["Shipley, Rebecca; Wright, Edward; Selden, David; Wu, Guanghui; Aegerter, James; Fooks, Anthony R; Banyard, Ashley C","Sutton, Troy C.; Subbarao, Kanta","Mohd, Hamzah A.; Al-Tawfiq, Jaffar A.; Memish, Ziad A.","Tang, Xianchun; Li, Gang; Vasilakis, Nikos; Zhang, Yuan; Shi, Zhengli; Zhong, Yang; Wang, Lin-Fa; Zhang, Shuyi","Wu, Fan; Zhao, Su; Yu, Bin; Chen, Yan-Mei; Wang, Wen; Song, Zhi-Gang; Hu, Yi; Tao, Zhao-Wu; Tian, Jun-Hua; Pei, Yuan-Yuan; Yuan, Ming-Li; Zhang, Yu-Ling; Dai, Fa-Hui; Liu, Yi; Wang, Qi-Min; Zheng, Jiao-Jiao; Xu, Lin; Holmes, Edward C.; Zhang, Yong-Zhen","Fan Wu; Su Zhao; Bin Yu; Yan-Mei Chen; Wen Wang; Yi Hu; Zhi-Gang Song; Zhao-Wu Tao; Jun-Hua Tian; Yuan-Yuan Pei; Ming-Li Yuan; Yu-Ling Zhang; Fa-Hui Dai; Yi Liu; Qi-Min Wang; Jiao-Jiao Zheng; Lin Xu; Edward C. Holmes; Yong-Zhen Zhang","Alexander E. Gorbalenya; Susan C. Baker; Ralph S. Baric; Raoul J. de Groot; Christian Drosten; Anastasia A. Gulyaeva; Bart L. Haagmans; Chris Lauber; Andrey M Leontovich; Benjamin W. Neuman; Dmitry Penzar; Stanley Perlman; Leo L.M. Poon; Dmitry Samborskiy; Igor A. Sidorov; Isabel Sola; John Ziebuhr","Yusof, Mohammed Farouk; Queen, Krista; Eltahir, Yassir Mohammed; Paden, Clinton R; Al Hammadi, Zulaikha Mohamed Abdel Hameed; Tao, Ying; Li, Yan; Khalafalla, Abdelmalik Ibrahim; Shi, Mang; Zhang, Jing; Mohamed, Muzammil Sayed Ahmed Elhaj; Abd Elaal Ahmed, Mahmud Hamed; Azeez, Ihsaan Abdulwahab; Bensalah, Oum Keltoum; Eldahab, Ziyada Swar; Al Hosani, Farida Ismail; Gerber, Susan I; Hall, Aron J; Tong, Suxiang; Al Muhairi, Salama Suhail","Gorbalenya, Alexander E.; Baker, Susan C.; Baric, Ralph S.; de Groot, Raoul J.; Drosten, Christian; Gulyaeva, Anastasia A.; Haagmans, Bart L.; Lauber, Chris; Leontovich, Andrey M.; Neuman, Benjamin W.; Penzar, Dmitry; Perlman, Stanley; Poon, Leo L. M.; Samborskiy, Dmitry V.; Sidorov, Igor A.; Sola, Isabel; Ziebuhr, John; Coronaviridae Study Group of the International Committee on Taxonomy of, Viruses","Fang, Shou Guo; Shen, Shuo; Tay, Felicia P.L.; Liu, D.X.","Safronetz, David; Geisbert, Thomas W; Feldmann, Heinz","Shi, Z.; Wang, L.-F.","Gretebeck, Lisa M; Subbarao, Kanta","Fung, Sin-Yee; Yuen, Kit-San; Ye, Zi-Wei; Chan, Chi-Ping; Jin, Dong-Yan","Song, Zhiqi; Xu, Yanfeng; Bao, Linlin; Zhang, Ling; Yu, Pin; Qu, Yajin; Zhu, Hua; Zhao, Wenjie; Han, Yunlin; Qin, Chuan","Yu, Meng; Stevens, Vicky; Berry, Jody D.; Crameri, Gary; McEachern, Jennifer; Tu, Changchun; Shi, Zhengli; Liang, Guodong; Weingartl, Hana; Cardosa, Jane; Eaton, Bryan T.; Wang, Lin-Fa","Deng, X.; Baker, S.C.","Simmons, Graham; Bertram, Stephanie; Glowacka, Ilona; Steffen, Imke; Chaipan, Chawaree; Agudelo, Juliet; Lu, Kai; Rennekamp, Andrew J.; Hofmann, Heike; Bates, Paul; P\u00f6hlmann, Stefan","Nicholls, John M; Butany, Jagdish; Poon, Leo L. M; Chan, Kwok H; Beh, Swan Lip; Poutanen, Susan; Peiris, J. S. Malik; Wong, Maria","Yamada, Yoshiyuki; Liu, Xiao Bo; Fang, Shou Guo; Tay, Felicia P. L.; Liu, Ding Xiang","Kirchdoerfer, Robert N.; Wang, Nianshuang; Pallesen, Jesper; Wrapp, Daniel; Turner, Hannah L.; Cottrell, Christopher A.; Corbett, Kizzmekia S.; Graham, Barney S.; McLellan, Jason S.; Ward, Andrew B.","Xiaojun Li; Elena E. Giorgi; Manukumar Honnayakanahalli Marichann; Brian Foley; Chuan Xiao; Xiang-peng Kong; Yue Chen; Bette Korber; Feng Gao","Zhong, Yanxin; Tan, Yong Wah; Liu, Ding Xiang","Raj, V Stalin; Osterhaus, Albert DME; Fouchier, Ron AM; Haagmans, Bart L","Foddai, Alessandro; Lindberg, Ann; Lubroth, Juan; Ellis-Iversen, Johanne","Papachristofilou, Alexandros; Finazzi, Tobias; Kohler, Goetz; Dott, Christian; Zimmermann, Frank","Simmons, Graham; Zmora, Pawel; Gierer, Stefanie; Heurich, Adeline; P\u00f6hlmann, Stefan","","Lane, Thomas E.; Buchmeier, Michael J.","Cao, Zhiliang; Liu, Lifeng; Du, Lanying; Zhang, Chao; Jiang, Shibo; Li, Taisheng; He, Yuxian","Mayer, J.D.","Ba, Lei; Yi, Christopher E.; Zhang, Linqi; Ho, David D.; Chen, Zhiwei","Jonsdottir, Hulda R.; Dijkman, Ronald","Pei-Hui Wang; Yun Cheng","Lau, Susanna K. P.; Chan, Jasper F. W.","van den Brand, J.M.A.; Haagmans, B.L.; van Riel, D.; Osterhaus, A.D.M.E.; Kuiken, T.","Liangde Xu; Jian Yuan; Yaru Zhang; Guosi Zhang; Fan Lu; Jianzhong Su; Jia Qu","Hofmann, Heike; P\u00f6hlmann, Stefan","Li, Fang","Du, Ning Wang; Jian, Shang; Shibo, Jiang; Lanying","Tomar, Shailly; Mahajan, Supreeti; Kumar, Ravi","Mallick, Bibekanand; Ghosh, Zhumur; Chakrabarti, Jayprokas","Wang, Liang; Su, Shuo; Bi, Yuhai; Wong, Gary; Gao, George F.","Leong, Hoe Nam; Chan, Kwai Peng; Khan, Ali S.; Oon, Lynette; Se-Thoe, Su Yun; Bai, Xin Lai; Yeo, Daniel; Leo, Yee Sin; Ang, Brenda; Ksiazek, Thomas G.; Ling, Ai Ee","Douglas, Madeline G.; Kocher, Jacob F.; Scobey, Trevor; Baric, Ralph S.; Cockrell, Adam S.","Khabbaz, Rima F; Moseley, Robin R; Steiner, Riley J; Levitt, Alexandra M; Bell, Beth P","Hualan Chen","Ping Liu; Jing-Zhe Jiang; Xiu-Feng Wan; Yan Hua; Xiaohu Wang; Fanghui Hou; Jing Chen; Jiejian Zou; Jinping Chen","Sui, Jianhua; Aird, Daniel R.; Tamin, Azaibi; Murakami, Akikazu; Yan, Meiying; Yammanuru, Anuradha; Jing, Huaiqi; Kan, Biao; Liu, Xin; Zhu, Quan; Yuan, Qing-an; Adams, Gregory P.; Bellini, William J.; Xu, Jianguo; Anderson, Larry J.; Marasco, Wayne A.","Michael Letko; Vincent Munster"],"color":["#67a9cf","#67a9cf","#67a9cf","#67a9cf","#67a9cf","#67a9cf","#67a9cf","#67a9cf","#67a9cf","#67a9cf","#67a9cf","#67a9cf","#67a9cf","#67a9cf","#67a9cf","#67a9cf","#67a9cf","#67a9cf","#67a9cf","#67a9cf","#67a9cf","#67a9cf","#67a9cf","#67a9cf","#67a9cf","#67a9cf","#67a9cf","#67a9cf","#67a9cf","#67a9cf","#67a9cf","#67a9cf","#67a9cf","#67a9cf","#67a9cf","#67a9cf","#67a9cf","#67a9cf","#67a9cf","#67a9cf","#67a9cf","#67a9cf","#67a9cf","#67a9cf","#67a9cf","#67a9cf","#67a9cf","#67a9cf","#67a9cf","#67a9cf"],"date":["2019-02-07","2015-05-31","2016-06-03","2009-03-05","2020-02-03","2020-01-25","2020-02-11","2017-11-08","2020","2005-10-21","2013-04-30","2017-12-31","2015-08-31","2020-03-14","2019-01-14","2008-02-29","2014-12-31","2011-05-10","2006-01-03","2009-07-02","2018-10-24","2020-03-22","2012-06-19","2014-04-30","2020-03-27","2020-03-31","2013-12-31","2013-12-05","1997-01-31","2010-11-04","2008-12-31","2007-06-21","2016-02-06","2020-02-27","2015-12-22","2014-07-31","2020-02-29","2004-10-31","2013-10-31","2020","2020-12-31","2009-11-13","2018-06-30","2004-10-03","2018-04-30","2014-07-11","2020-03-31","2020-02-20","2008-11-07","2020-01-22"],"doc":["hibzy1oh","1k5h662u","yzp9wjuk","i5hkucrn","qele28zk","5jai745w","ptbk69rk","g0gh28hu","vvhhpchm","h8cemq2n","guz3am41","apc0lm5e","v5sya5to","89fol3pq","7aj3fx5u","4hqhty5o","eqfz0wpm","uzch2hwd","do5c85ad","gkm7i62s","gz38qibg","7v5aln90","i5korxut","8aj97x4n","5j4nt2qs","zlxqt7qy","ormyf4w2","fw9k7tn3","8x49bdqy","bd2ndoug","t99l3zii","zciyvo29","40zyv1h8","epy7774j","zhxrlopp","aa08u19x","acmfhbtm","cil5czqp","wm6yey1v","hq4jb2wy","mog52dn8","dff7y6dr","wuegn0jg","fk7qb4ui","bbnhmruf","alnj42iy","mo4luyx6","2inlyd0t","ttiqxlts","tyhwtt7j"],"journal":["Trop Med Infect Dis","Virology","Virol J","BMC Evol Biol","Nature","","","Emerg Microbes Infect","Nature Microbiology","Biochemical and Biophysical Research Communications","Current Opinion in Virology","Genetics and Evolution of Infectious Diseases","Current Opinion in Virology","Emerg Microbes Infect","Viruses","Journal of Immunological Methods","Reference Module in Biomedical Sciences","Virology","PLoS Med","PLoS One","Sci Rep","","Viruses","Current Opinion in Virology","One Health","Advances in Radiation Oncology","Antiviral Research","Nat Med","Trends in Microbiology","Virol J","International Encyclopedia of Public Health","Appl Microbiol Biotechnol","Virol J","","Virol J","Journal of Comparative Pathology","","Trends in Microbiology","Antiviral Research","Frontiers in Microbiology","Genomics and Biotechnological Advances in Veterinary, Poultry, and Fisheries","PLoS One","Trends in Microbiology","Emerg Infect Dis","Virology","The Lancet","","","PLoS Pathog",""],"rad":[0.017052233966000002,0.013803415837849733,0.012436687517859712,0.010890823737117576,0.007542126990040958,0.007462437707977017,0.006643622128924145,0.006557455276044452,0.006542889506859509,0.005217254087515494,0.005090217414075145,0.004997441999310794,0.004842822060851578,0.00483832497341788,0.004703818316059687,0.004652853395227315,0.004547995763293007,0.0045188219013606885,0.004458333613435722,0.004412010617694856,0.004372322589567672,0.004157580467363205,0.004156505769720997,0.004097921751979778,0.004080080167411733,0.004007767588703767,0.004000852891695332,0.003998498536472787,0.003904805003755133,0.00375081132716677,0.003669524304802994,0.003562371640704395,0.0035622418472324806,0.003514279466963141,0.003470087799280656,0.0033846071623004712,0.003375826976409221,0.00326492331952263,0.0032534734900590808,0.003249510826933828,0.0032411188922746074,0.0032273041224074013,0.0031884843406629762,0.0031581667376335103,0.0031065366096114434,0.0031065351336139833,0.0030990962857733073,0.003063700888411768,0.0030426072536223992,0.0030001212445542],"scr":[0.827,0.635,0.555,0.464,0.267,0.262,0.214,0.209,0.208,0.13,0.123,0.117,0.108,0.108,0.1,0.097,0.091,0.089,0.086,0.083,0.081,0.068,0.068,0.065,0.064,0.059,0.059,0.059,0.053,0.044,0.039,0.033,0.033,0.03,0.028,0.023,0.022,0.016,0.015,0.015,0.014,0.013,0.011,0.009,0.006,0.006,0.006,0.004,0.003,0.0],"text":["Bats in the EU have been associated with several zoonotic viral pathogens of significance to both human and animal health. Virus discovery continues to expand the existing understating of virus classification, and the increased interest in bats globally as reservoirs or carriers of zoonotic agents has fuelled the continued detection and characterisation of new lyssaviruses and other viral zoonoses. Although the transmission of lyssaviruses from bat species to humans or terrestrial species appears rare, interest in these viruses remains, through their ability to cause the invariably fatal encephalitis\u2014rabies. The association of bats with other viral zoonoses is also of great interest. Much of the EU is free of terrestrial rabies, but several bat species harbor lyssaviruses that remain a risk to human and animal health. Whilst the rabies virus is the main cause of rabies globally, novel related viruses continue to be discovered, predominantly in bat populations, that are of interest pure...","Abstract Two novel coronaviruses have emerged to cause severe disease in humans. While bats may be the primary reservoir for both viruses, SARS coronavirus (SARS-CoV) likely crossed into humans from civets in China, and MERS coronavirus (MERS-CoV) has been transmitted from camels in the Middle East. Unlike SARS-CoV that resolved within a year, continued introductions of MERS-CoV present an on-going public health threat. Animal models are needed to evaluate countermeasures against emerging viruses. With SARS-CoV, several animal species were permissive to infection. In contrast, most laboratory animals are refractory or only semi-permissive to infection with MERS-CoV. This host-range restriction is largely determined by sequence heterogeneity in the MERS-CoV receptor. We describe animal models developed to study coronaviruses, with a focus on host-range restriction at the level of the viral receptor and discuss approaches to consider in developing a model to evaluate countermeasures agai...","Middle East Respiratory Syndrome-Coronavirus (MERS-CoV) is a novel coronavirus discovered in 2012 and is responsible for acute respiratory syndrome in humans. Though not confirmed yet, multiple surveillance and phylogenetic studies suggest a bat origin. The disease is heavily endemic in dromedary camel populations of East Africa and the Middle East. It is unclear as to when the virus was introduced to dromedary camels, but data from studies that investigated stored dromedary camel sera and geographical distribution of involved dromedary camel populations suggested that the virus was present in dromedary camels several decades ago. Though bats and alpacas can serve as potential reservoirs for MERS-CoV, dromedary camels seem to be the only animal host responsible for the spill over human infections.","BACKGROUND: SARS coronavirus (SARS-CoV) was identified as the etiological agent of SARS, and extensive investigations indicated that it originated from an animal source (probably bats) and was recently introduced into the human population via wildlife animals from wet markets in southern China. Previous studies revealed that the spike (S) protein of SARS had experienced adaptive evolution, but whether other functional proteins of SARS have undergone adaptive evolution is not known. RESULTS: We employed several methods to investigate selective pressure among different SARS-CoV groups representing different epidemic periods and hosts. Our results suggest that most functional proteins of SARS-CoV have experienced a stepwise adaptive evolutionary pathway. Similar to previous studies, the spike protein underwent strong positive selection in the early and middle phases, and became stabilized in the late phase. In addition, the replicase experienced positive selection only in human patients, ...","Emerging infectious diseases, such as severe acute respiratory syndrome (SARS) and Zika virus disease, present a major threat to public health(1\u20133). Despite intense research efforts, how, when and where new diseases appear are still a source of considerable uncertainty. A severe respiratory disease was recently reported in Wuhan, Hubei province, China. As of 25 January 2020, at least 1,975 cases had been reported since the first patient was hospitalized on 12 December 2019. Epidemiological investigations have suggested that the outbreak was associated with a seafood market in Wuhan. Here we study a single patient who was a worker at the market and who was admitted to the Central Hospital of Wuhan on 26 December 2019 while experiencing a severe respiratory syndrome that included fever, dizziness and a cough. Metagenomic RNA sequencing(4) of a sample of bronchoalveolar lavage fluid from the patient identified a new RNA virus strain from the family Coronaviridae, which is designated here ...","Emerging and re-emerging infectious diseases, such as SARS, MERS, Zika and highly pathogenic influenza present a major threat to public health1\u20133. Despite intense research effort, how, when and where novel diseases appear are still the source of considerable uncertainly. A severe respiratory disease was recently reported in the city of Wuhan, Hubei province, China. At the time of writing, at least 62 suspected cases have been reported since the first patient was hospitalized on December 12nd 2019. Epidemiological investigation by the local Center for Disease Control and Prevention (CDC) suggested that the outbreak was associated with a sea food market in Wuhan. We studied seven patients who were workers at the market, and collected bronchoalveolar lavage fluid (BALF) from one patient who exhibited a severe respiratory syndrome including fever, dizziness and cough, and who was admitted to Wuhan Central Hospital on December 26th 2019. Next generation metagenomic RNA sequencing4 identifie...","AbstractThe present outbreak of lower respiratory tract infections, including respiratory distress syndrome, is the third spillover, in only two decades, of an animal coronavirus to humans resulting in a major epidemic. Here, the Coronavirus Study Group (CSG) of the International Committee on Taxonomy of Viruses, which is responsible for developing the official classification of viruses and taxa naming (taxonomy) of the Coronaviridae family, assessed the novelty of the human pathogen tentatively named 2019-nCoV. Based on phylogeny, taxonomy and established practice, the CSG formally recognizes this virus as a sister to severe acute respiratory syndrome coronaviruses (SARS-CoVs) of the species Severe acute respiratory syndrome-related coronavirus and designates it as severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2). To facilitate communication, the CSG further proposes to use the following naming convention for individual isolates: SARS-CoV-2/Isolate/Host/Date/Location. The ...","Middle East respiratory syndrome coronavirus (MERS-CoV) was identified on the Arabian Peninsula in 2012 and is still causing cases and outbreaks in the Middle East. When MERS-CoV was first identified, the closest related virus was in bats; however, it has since been recognized that dromedary camels serve as a virus reservoir and potential source for human infections. A total of 376 camels were screened for MERS-Cov at a live animal market in the Eastern Region of the Emirate of Abu Dhabi, UAE. In all, 109 MERS-CoV-positive camels were detected in week 1, and a subset of positive camels were sampled again weeks 3 through 6. A total of 126 full and 3 nearly full genomes were obtained from 139 samples. Spike gene sequences were obtained from 5 of the 10 remaining samples. The camel MERS-CoV genomes from this study represent 3 known and 2 potentially new lineages within clade B. Within lineages, diversity of camel and human MERS-CoV sequences are intermixed. We identified sequences from ma...","The present outbreak of a coronavirus-associated acute respiratory disease called coronavirus disease 19 (COVID-19) is the third documented spillover of an animal coronavirus to humans in only two decades that has resulted in a major epidemic. The Coronaviridae Study Group (CSG) of the International Committee on Taxonomy of Viruses, which is responsible for developing the classification of viruses and taxon nomenclature of the family Coronaviridae, has assessed the placement of the human pathogen, tentatively named 2019-nCoV, within the Coronaviridae. Based on phylogeny, taxonomy and established practice, the CSG recognizes this virus as forming a sister clade to the prototype human and bat severe acute respiratory syndrome coronaviruses (SARS-CoVs) of the species Severe acute respiratory syndrome-related coronavirus, and designates it as SARS-CoV-2. In order to facilitate communication, the CSG proposes to use the following naming convention for individual isolates: SARS-CoV-2/host/lo...","Abstract An interesting question posed by the current evidence that severe acute respiratory syndrome coronavirus may be originated from an animal coronavirus is how such an animal coronavirus breaks the host species barrier and becomes zoonotic. In this report, we study the chronological order of genotypic changes in the spike protein of avian coronavirus infectious bronchitis virus (IBV) during its adaptation to a primate cell line. Adaptation of the Beaudette strain of IBV from chicken embryo to Vero cells showed the accumulation of 49 amino acid mutations. Among them, 26 (53.06%) substitutions were located in the S protein. Sequencing analysis and comparison of the S gene demonstrated that the majority of the mutations were accumulated and fixed at passage 7 on Vero cells and minor variants were isolated in several passages. Evidence present suggests that the dominant Vero cell-adapted IBV strain may be derived from the chicken embryo passages by selection of and potential recombin...","Exotic and emerging viral pathogens associated with high morbidity and mortality in humans are being identified annually with recent examples including Lujo virus in southern Africa, Severe Fever with Thrombocytopenia Syndrome virus in China and a SARS-like coronavirus in the Middle East. The sporadic nature of these infections hampers our understanding of these diseases and limits the opportunities to design appropriate medical countermeasures against them. Because of this, animal models are utilized to gain insight into the pathogenesis of disease with the overall goal of identifying potential targets for intervention and evaluating specific therapeutics and vaccines. For these reasons it is imperative that animal models of disease recapitulate the human condition as closely as possible in order to provide the best predictive data with respect to the potential efficacy in humans. In this article we review the current status of disease models for highly pathogenic and emerging viral p...","Abstract The severe acute respiratory syndrome (SARS) is the first major zoonotic infectious disease of the 21st Century. The disease, originally termed \u201catypical pneumonia,\u201d was first recognized in November 2002 in Guangdong Province, China, which spread rapidly to 26 countries within 5months and eventually led to 8096 human infection cases and 774 deaths. A previously unrecognized coronavirus, named SARS coronavirus (SARS-CoV), was later identified as the causative agent of SARS. Subsequent investigation indicated that the source of the human infections originated most likely from infected wildlife animals traded in the live animal markets, with masked palm civets considered the most prominent and important carrier. Large-scale culling of civets appeared to be effective in preventing further outbreaks in the region. Subsequent epidemiological studies showed that civets are intermediate host of SARS-CoV and bats are the likely natural reservoir of SARS-CoV and a large number of SARS-l...","The emergence of Severe Acute Respiratory Syndrome coronavirus (SARS-CoV) and Middle East Respiratory Syndrome coronavirus (MERS-CoV), two strains of animal coronaviruses that crossed the species barrier to infect and cause severe respiratory infections in humans within the last 12 years, have taught us that coronaviruses represent a global threat that does not recognize international borders. We can expect to see other novel coronaviruses emerge in the future. An ideal animal model should reflect the clinical signs, viral replication and pathology seen in humans. In this review, we present factors to consider in establishing an animal model for the study of novel coronaviruses and compare the different animal models that have been employed to study SARS-CoV and MERS-CoV.","World Health Organization has declared the ongoing outbreak of coronavirus disease 2019 (COVID-19) a Public Health Emergency of International Concern. The virus was named severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) by the International Committee on Taxonomy of Viruses. Human infection with SARS-CoV-2 leads to a wide range of clinical manifestations ranging from asymptomatic, mild, moderate to severe. The severe cases present with pneumonia, which can progress to acute respiratory distress syndrome. The outbreak provides an opportunity for real-time tracking of an animal coronavirus that has just crossed species barrier to infect humans. The outcome of SARS-CoV-2 infection is largely determined by virus-host interaction. Here, we review the discovery, zoonotic origin, animal hosts, transmissibility and pathogenicity of SARS-CoV-2 in relation to its interplay with host antiviral defense. A comparison with SARS-CoV, Middle East respiratory syndrome coronavirus, community-...","Coronaviruses (CoVs) have formerly been regarded as relatively harmless respiratory pathogens to humans. However, two outbreaks of severe respiratory tract infection, caused by the severe acute respiratory syndrome coronavirus (SARS-CoV) and the Middle East respiratory syndrome coronavirus (MERS-CoV), as a result of zoonotic CoVs crossing the species barrier, caused high pathogenicity and mortality rates in human populations. This brought CoVs global attention and highlighted the importance of controlling infectious pathogens at international borders. In this review, we focus on our current understanding of the epidemiology, pathogenesis, prevention, and treatment of SARS-CoV and MERS-CoV, as well as provides details on the pivotal structure and function of the spike proteins (S proteins) on the surface of each of these viruses. For building up more suitable animal models, we compare the current animal models recapitulating pathogenesis and summarize the potential role of host receptor...","Abstract Knowledge of immunodominant regions in major viral antigens is important for rational design of effective vaccines and diagnostic tests. Although there have been many reports of such work done for SARS\u2013CoV, these were mainly focused on the immune responses of humans and mice. In this study, we aim to search for and compare immunodominant regions of the spike (S) and nucleocapsid (N) proteins which are recognized by sera from different animal species, including mouse, rat, rabbit, civet, pig and horse. Twelve overlapping recombinant protein fragments were produced in Escherichia coli, six each for the S and N proteins, which covered the entire coding region of the two proteins. Using a membrane-strip based Western blot approach, the reactivity of each antigen fragment against a panel of animal sera was determined. Immunodominant regions containing linear epitopes, which reacted with sera from all the species tested, were identified for both proteins. The S3 fragment (aa 402\u2013622...","Abstract Coronaviruses (CoVs) are enveloped, positive-strand RNA viruses with characteristic spike glycoproteins that project outward like the rays of the sun (corona \u2013 Latin for \u2018crown\u2019), when visualized by electron microscopy. CoV are classified, together with the toroviruses, in the family Coronaviridae and the order Nidovirales. All nidoviruses have a common genome organization and generate a nested set (nido \u2013 Latin for \u2018nest\u2019) of 3\u2032 co-terminal mRNAs. CoVs have been isolated from a variety of species, including birds, livestock, domestic animals, and humans. CoV infections can cause respiratory, gastrointestinal, and neurologic disease, depending on the strain of the virus and the site of infection. Importantly, CoVs have been shown to cross species barriers and have emerged from animal reservoirs to infect humans and cause severe disease. The CoV responsible for an outbreak of severe acute respiratory disease (SARS-CoV) in 2002\u201303 likely originated as a bat coronavirus which, du...","Abstract Severe acute respiratory syndrome coronavirus (SARS-CoV) poses a considerable threat to human health. Activation of the viral spike (S)-protein by host cell proteases is essential for viral infectivity. However, the cleavage sites in SARS-S and the protease(s) activating SARS-S are incompletely defined. We found that R667 was dispensable for SARS-S-driven virus\u2013cell fusion and for SARS-S-activation by trypsin and cathepsin L in a virus\u2013virus fusion assay. Mutation T760R, which optimizes the minimal furin consensus motif 758-RXXR-762, and furin overexpression augmented SARS-S activity, but did not result in detectable SARS-S cleavage. Finally, SARS-S-driven cell\u2013cell fusion was independent of cathepsin L, a protease essential for virus\u2013cell fusion. Instead, a so far unknown leupeptin-sensitive host cell protease activated cellular SARS-S for fusion with target cells expressing high levels of ACE2. Thus, different host cell proteases activate SARS-S for virus\u2013cell and cell\u2013cell ...","BACKGROUND: Cellular localization of severe acute respiratory syndrome coronavirus (SARS-CoV) in the lungs of patients with SARS is important in confirming the etiological association of the virus with disease as well as in understanding the pathogenesis of the disease. To our knowledge, there have been no comprehensive studies investigating viral infection at the cellular level in humans. METHODS AND FINDINGS: We collected the largest series of fatal cases of SARS with autopsy material to date by merging the pathological material from two regions involved in the 2003 worldwide SARS outbreak in Hong Kong, China, and Toronto, Canada. We developed a monoclonal antibody against the SARS-CoV nucleoprotein and used it together with in situ hybridization (ISH) to analyze the autopsy lung tissues of 32 patients with SARS from Hong Kong and Toronto. We compared the results of these assays with the pulmonary pathologies and the clinical course of illness for each patient. SARS-CoV nucleoprotein...","Coronavirus host and cell specificities are determined by specific interactions between the viral spike (S) protein and host cell receptor(s). Avian coronavirus infectious bronchitis (IBV) has been adapted to embryonated chicken eggs, primary chicken kidney (CK) cells, monkey kidney cell line Vero, and other human and animal cells. Here we report that acquisition of the cell\u2013cell fusion activity by amino acid mutations in the S protein determines the infectivity of IBV in cultured cells. Expression of S protein derived from Vero- and CK-adapted strains showed efficient induction of membrane fusion. However, expression of S protein cloned from the third passage of IBV in chicken embryo (EP3) did not show apparent syncytia formation. By construction of chimeric S constructs and site-directed mutagenesis, a point mutation (L857-F) at amino acid position 857 in the heptad repeat 1 region of S protein was shown to be responsible for its acquisition of the cell\u2013cell fusion activity. Furtherm...","Severe acute respiratory syndrome coronavirus (SARS-CoV) emerged in 2002 as a highly transmissible pathogenic human betacoronavirus. The viral spike glycoprotein (S) utilizes angiotensin-converting enzyme 2 (ACE2) as a host protein receptor and mediates fusion of the viral and host membranes, making S essential to viral entry into host cells and host species tropism. As SARS-CoV enters host cells, the viral S is believed to undergo a number of conformational transitions as it is cleaved by host proteases and binds to host receptors. We recently developed stabilizing mutations for coronavirus spikes that prevent the transition from the pre-fusion to post-fusion states. Here, we present cryo-EM analyses of a stabilized trimeric SARS-CoV S, as well as the trypsin-cleaved, stabilized S, and its interactions with ACE2. Neither binding to ACE2 nor cleavage by trypsin at the S1/S2 cleavage site impart large conformational changes within stabilized SARS-CoV S or expose the secondary cleavage s...","COVID-19 has become a global pandemic caused by a novel coronavirus SARS-CoV-2. Understanding the origins of SARS-CoV-2 is critical for deterring future zoonosis and for drug discovery and vaccine development. We show evidence of strong purifying selection around the receptor binding motif (RBM) in the spike gene and in other genes among bat, pangolin and human coronaviruses, indicating similar strong evolutionary constraints in different host species. We also demonstrate that SARS-CoV-2's entire RBM was introduced through recombination with coronaviruses from pangolins, possibly a critical step in the evolution of SARS-CoV-2's ability to infect humans. Similar purifying selection in different host species and frequent recombination among coronaviruses suggest a common evolutionary mechanism that could lead to new emerging human coronaviruses.","Animal coronaviruses, such as infectious bronchitis virus (IBV), and arteriviruses, such as porcine reproductive and respiratory syndrome virus (PRRSV), are able to manifest highly contagious infections in their specific native hosts, thereby arising in critical economic damage to animal industries. This review discusses recent progress in studies of virus-host interactions during animal and human coronavirus and arterivirus infections, with emphasis on IBV-host cell interactions. These interactions may be directly involved in viral replication or lead to the alteration of certain signaling pathways, such as cell stress response and innate immunity, to facilitate viral replication and pathogenesis.","A novel coronavirus (CoV) that causes a severe lower respiratory tract infection in humans, emerged in the Middle East region in 2012. This virus, named Middle East respiratory syndrome (MERS)-CoV, is phylogenetically related to bat CoVs, but other animal species like dromedary camels may potentially act as intermediate hosts by spreading the virus to humans. Although human to human transmission has been demonstrated, analysis of human MERS clusters indicated that chains of transmission were not self-sustaining, especially when infection control was implemented. Thus, timely identification of new MERS cases followed by their quarantine, combined with measures to limit spread of the virus from the (intermediate) host to humans, may be crucial in controlling the outbreak of this emerging CoV.","Abstract During the first few months of 2020, the COVID-19 pandemic has reached Europe. Health systems all over the world are trying to control the outbreak in the shortest possible time. Exotic disease outbreaks are not uncommon in animal health and randomised surveillance is frequently used as support for decision-making. This editorial discusses the possibilities of practicing One Health, by using methods from animal health to enhance surveillance for COVID-19 to provide an evidence base fort decision-making in communities and countries.","Abstract The ongoing novel coronavirus (2019-nCoV) pandemic is expected to develop into an unprecedented stress test for health care systems worldwide. This brief report, written from a radiation oncology perspective during the developing outbreak of 2019-nCoV in Switzerland, highlights the challenges identified and measures taken in our department to mitigate risks and ensure continued operations during the outbreak.","Abstract The severe acute respiratory syndrome (SARS) pandemic revealed that zoonotic transmission of animal coronaviruses (CoV) to humans poses a significant threat to public health and warrants surveillance and the development of countermeasures. The activity of host cell proteases, which cleave and activate the SARS-CoV spike (S) protein, is essential for viral infectivity and constitutes a target for intervention. However, the identities of the proteases involved have been unclear. Pioneer studies identified cathepsins and type II transmembrane serine proteases as cellular activators of SARS-CoV and demonstrated that several emerging viruses might exploit these enzymes to promote their spread. Here, we will review the proteolytic systems hijacked by SARS-CoV for S protein activation, we will discuss their contribution to viral spread in the host and we will outline antiviral strategies targeting these enzymes. This paper forms part of a series of invited articles in Antiviral Resea...","Biomedical research in 2013 saw some dramatic developments, with unprecedented government action in the US ranging from the budget sequester in the spring to a dramatic government shutdown in autumn. But throughout the year, bright spots in science around the globe continued to dazzle, including multimillion-dollar partnerships to advance drug discovery and the go-ahead for highly anticipated trials of regenerative medicine.","A variety of neurological diseases in humans, including multiple sclerosis (MS), have been postulated to have a viral etiology. The use of animal models provides insights into potential mechanism(s) involved in the disease process. The murine coronavirus-induced demyelinating disease in rodents is one such model for demyelinating disease in humans.","BACKGROUND: The spike (S) protein of SARS-CoV not only mediates receptor-binding but also induces neutralizing antibodies. We previously identified the receptor-binding domain (RBD) of S protein as a major target of neutralizing antibodies in animal models and thus proposed a RBD-based vaccine. However, the antigenicity and immunogenicity of RBD in humans need to be characterized. RESULTS: Two panels of serum samples from recovered SARS patients were included and the antibody responses against the RBD were measured by ELISA and micro-neutralization assays. We found that the RBD of S protein induced potent antibody responses in the recovered SARS patients and RBD-specific antibodies could persist at high titers over three year follow-up. Furthermore, affinity purified anti-RBD antibodies possessed robust neutralizing activity. CONCLUSION: The RBD of SARS-CoV is highly immunogenic in humans and mediates protective responses and RBD-based vaccines and diagnostic approaches can be further ...","Emerging infectious diseases are diseases that are either new, are newly recognized, or are increasing in prevalence in new areas. Resurgent diseases are also usually grouped in this category, as is antimicrobial resistance. These diseases have been given formal recognition in the past two decades, although a historical outlook demonstrates that the phenomenon has probably been persistent, although largely undetected, through recorded history. Emergence has accelerated recently, driven by factors such as demographic change, land use change, increased rapidity and frequency of intercontinental transportation, and other mostly social trends. Continued infectious disease emergence poses, and will continue to pose, significant challenges for public health and for basic science.","Severe acute respiratory syndrome (SARS) is caused by a novel coronavirus (CoV), SARS-CoV. In previous studies, we showed that a SARS-CoV spike (S) glycoprotein-based modified vaccinia Ankara (MVA-S) vaccine could induce strong neutralizing antibody (Nab) response which might have played a critical role in protecting Chinese rhesus monkeys from the pathogenic viral challenge. To date, however, it remains unknown what the minimal level of Nab is required to achieve sterile immunity in humans. It is therefore important to explore techniques to maximize the level of Nab response in vivo. Here, we evaluate various vaccination regimens using combinations of DNA-S, MVA-S, and adenovirus type 5 (Ad5-S) vaccines. We show that in vaccinated mice and rabbits, a heterologous MVA-S prime with Ad5-S boost regimen induces the highest and most persistent level of Nab response when compared with other combinations. Interestingly, the initial level of Nab after prime does not necessarily predict the ma...","Human coronaviruses (HCoVs) are large RNA viruses that infect the human respiratory tract. The emergence of both Severe Acute Respiratory Syndrome and Middle East Respiratory syndrome CoVs as well as the yearly circulation of four common CoVs highlights the importance of elucidating the different mechanisms employed by these viruses to evade the host immune response, determine their tropism and identify antiviral compounds. Various animal models have been established to investigate HCoV infection, including mice and non-human primates. To establish a link between the research conducted in animal models and humans, an organotypic human airway culture system, that recapitulates the human airway epithelium, has been developed. Currently, different cell culture systems are available to recapitulate the human airways, including the Air-Liquid Interface (ALI) human airway epithelium (HAE) model. Tracheobronchial HAE cultures recapitulate the primary entry point of human respiratory viruses w...","AbstractThe ongoing outbreak of a new coronavirus (2019-nCoV) causes an epidemic of acute respiratory syndrome in humans. 2019-nCoV rapidly spread to national regions and multiple other countries, thus, pose a serious threat to public health. Recent studies show that spike (S) proteins of 2019-nCoV and SARS-CoV may use the same host cell receptor called angiotensin-converting enzyme 2 (ACE2) for entering into host cells. The affinity between ACE2 and 2019-nCoV S is much higher than ACE2 binding to SARS-CoV S protein, explaining that why 2019-nCoV seems to be more readily transmitted from the human to human. Here, we reported that ACE2 can be significantly upregulated after infection of various viruses including SARS-CoV and MERS-CoV. Basing on findings here, we propose that coronavirus infection can positively induce its cellular entry receptor to accelerate their replication and spread, thus drugs targeting ACE2 expression may be prepared for the future emerging infectious diseases ca...","The severe acute respiratory syndrome coronavirus (SARS-CoV) and recently emerged Middle East respiratory syndrome coronavirus (MERS-CoV) epidemics have proven the ability of coronaviruses to cross species barrier and emerge rapidly in humans. Other coronaviruses such as porcine epidemic diarrhea virus (PEDV) are also known to cause major disease epidemics in animals wiith huge economic loss. This special issue in Virology Journal aims to highlight the advances and key discoveries in the animal origin, viral evolution, epidemiology, diagnostics and pathogenesis of the emerging and re-emerging coronaviruses in both humans and animals.","Summary Respiratory viruses that emerge in the human population may cause high morbidity and mortality, as well as concern about pandemic spread. Examples are severe acute respiratory syndrome coronavirus (SARS-CoV) and novel variants of influenza A virus, such as H5N1 and pandemic H1N1. Different animal models are used to develop therapeutic and preventive measures against such viruses, but it is not clear which are most suitable. Therefore, this review compares animal models of SARS and influenza, with an emphasis on non-human primates, ferrets and cats. Firstly, the pathology and pathogenesis of SARS and influenza are compared. Both diseases are similar in that they affect mainly the respiratory tract and cause inflammation and necrosis centred on the pulmonary alveoli and bronchioles. Important differences are the presence of multinucleated giant cells and intra-alveolar fibrosis in SARS and more fulminant necrotizing and haemorrhagic pneumonia in H5N1 influenza. Secondly, the path...","In late December 2019, Chinese authorities reported a cluster of pneumonia cases of unknown aetiology in Wuhan, China. Wenzhou, as a southeast coastal city with the most cases outside Hubei Province, its policy control and epidemic projections have certain references for national and worldwide epidemic prevention and control. The maximum number of illness onset in Wenzhou appeared on January 26 (38 cases). Since January 27, the incidence per day in Wenzhou has continued to decline is likely to be due to isolation of people return to Wenzhou, implementation of region quarantine and suspension of public transportation system. Under the current control efforts, we estimate that the daily incidence by March 3-9, 2020 will drop to 0 in Wenzhou using SEIR model. The total number of affected people is 538.","Enveloped viruses have evolved membrane glycoproteins (GPs) that mediate entry into host cells. These proteins are important targets for antiviral therapies and vaccines. Several efforts to understand and combat infection by severe acute respiratory syndrome coronavirus (SARS-CoV) have therefore focused on the viral GP, known as spike (S). In a short period of time, important aspects of SARS-CoV S-protein function were unraveled. The identification of angiotensin-converting enzyme 2 (ACE2) as a receptor for SARS-CoV provided an insight into viral tropism and pathogenesis, whereas mapping of functional domains in the S-protein enabled inhibitors to be generated. Vaccines designed on the basis of SARS-CoV S-protein were shown to be effective in animals and consequently are attractive candidates for vaccine trials in humans. Here, we discuss how SARS-CoV S facilitates viral entry into target cells and illustrate current approaches that are used to inhibit this process.","Abstract Receptor recognition is a major determinant of the host range, cross-species infections, and pathogenesis of the severe acute respiratory syndrome coronavirus (SARS-CoV). A defined receptor-binding domain (RBD) in the SARS-CoV spike protein specifically recognizes its host receptor, angiotensin-converting enzyme 2 (ACE2). This article reviews the latest knowledge about how RBDs from different SARS-CoV strains interact with ACE2 from several animal species. Detailed research on these RBD/ACE2 interactions has established important principles on host receptor adaptations, cross-species infections, and future evolution of SARS-CoV. These principles may apply to other emerging animal viruses, including the recently emerged Middle East respiratory syndrome coronavirus (MERS-CoV). This paper forms part of a series of invited articles in Antiviral Research on \u201cFrom SARS to MERS: 10years of research on highly pathogenic human coronaviruses\u201d.","Seven coronaviruses (CoVs) have been isolated from humans so far. Among them, three emerging pathogenic CoVs, including severe acute respiratory syndrome coronavirus (SARS-CoV), Middle East respiratory syndrome coronavirus (MERS-CoV), and a newly identified CoV (2019-nCoV), once caused or continue to cause severe infections in humans, posing significant threats to global public health. SARS-CoV infection in humans (with about 10% case fatality rate) was first reported from China in 2002, while MERS-CoV infection in humans (with about 34.4% case fatality rate) was first reported from Saudi Arabia in June 2012. 2019-nCoV was first reported from China in December 2019, and is currently infecting more than 70000 people (with about 2.7% case fatality rate). Both SARS-CoV and MERS-CoV are zoonotic viruses, using bats as their natural reservoirs, and then transmitting through intermediate hosts, leading to human infections. Nevertheless, the intermediate host for 2019-nCoV is still under inve...","Abstract The latest advancements in drug development are structure-assisted and computer-aided identification, design, and synthesis of target-specific antiviral. Structural virology of animal viruses has made valuable contribution to our understanding of viruses in general, replication, evolution, and interaction with the host. Structure\u2013function relation studies are definitely the need of the hour for rational design of drugs and vaccines to effectively treat animal viral diseases. These studies are important for economical, veterinary, and human medical perspectives. X-ray crystallography, nuclear magnetic resonance and cryo-electron microscopy techniques elaborate the structure\u2013function relation studies by utilizing various computational approaches for structure-assisted designing of drug molecules and antiviral against viruses. Availability of protein three-dimensional structures, high-performance computing, data management software, and internet are enhancing the modern day drug ...","Severe acute respiratory syndrome (SARS), caused by the coronavirus SARS-CoV, is an acute infectious disease with significant mortality. A typical clinical feature associated with SARS is pulmonary fibrosis and associated lung failure. In the aftermath of the SARS epidemic, although significant progress towards understanding the underlying molecular mechanism of the infection has been made, a large gap still remains in our knowledge regarding how SARS-CoV interacts with the host cell at the onset of infection. The rapidly changing viral genome adds another variable to this equation. We have focused on a novel concept of microRNA (miRNA)\u2013mediated host\u2013virus interactions in bronchoalveolar stem cells (BASCs) at the onset of infection by correlating the \u201cBASC\u2013microRNome\u201d with their targets within BASCs and viral genome. This work encompasses miRNA array data analysis, target prediction, and miRNA\u2013mRNA enrichment analysis and develops a complex interaction map among disease-related factors...","Infections with bat-origin coronaviruses have caused severe illness in humans by \u2018host jump\u2019. Recently, novel bat-origin coronaviruses were found in pigs. The large number of mutations on the receptor-binding domain allowed the viruses to infect the new host, posing a potential threat to both agriculture and public health.","Severe acute respiratory syndrome (SARS) is caused by a novel coronavirus (SARS-CoV). In a longitudinal cross-sectional study, we determined the prevalence of virus in bodily excretions and time of seroconversion in discharged patients with SARS. Conjunctival, throat, stool, and urine specimens were collected weekly from 64 patients and tested for SARS-CoV RNA by real-time polymerase chain reaction; serum samples were collected weekly and tested for SARS-CoV antibody with indirect enzyme immunoassay and immunofluorescence assay. In total, 126 conjunctival, 124 throat swab, 116 stool, and 124 urine specimens were analyzed. Five patients had positive stool samples, collected in weeks 5\u20139. Two patients seroconverted in weeks 7 and 8; the others were seropositive at the first serum sample collection. In this study, 5 (7.8%) of 64 patients continued to shed viral RNA in stool samples only, for up to week 8 of illness. Most seroconversions occurred by week 6 of illness.","Abstract We recently established a mouse model (288\u2013330+/+) that developed acute respiratory disease resembling human pathology following infection with a high dose (5 \u00d7 106 PFU) of mouse-adapted MERS-CoV (icMERSma1). Although this high dose conferred fatal respiratory disease in mice, achieving similar pathology at lower viral doses may more closely reflect naturally acquired infections. Through continued adaptive evolution of icMERSma1 we generated a novel mouse-adapted MERS-CoV (maM35c4) capable of achieving severe respiratory disease at doses between 103 and 105 PFU. Novel mutations were identified in the maM35c4 genome that may be responsible for eliciting etiologies of acute respiratory distress syndrome at 10\u20131000 fold lower viral doses. Importantly, comparative genetics of the two mouse-adapted MERS strains allowed us to identify specific mutations that remained fixed through an additional 20 cycles of adaptive evolution. Our data indicate that the extent of MERS-CoV adaptation...","Summary In the USA, infectious diseases continue to exact a substantial toll on health and health-care resources. Endemic diseases such as chronic hepatitis, HIV, and other sexually transmitted infections affect millions of individuals and widen health disparities. Additional concerns include health-care-associated and foodborne infections\u2014both of which have been targets of broad prevention efforts, with success in some areas, yet major challenges remain. Although substantial progress in reduction of the burden of vaccine-preventable diseases has been made, continued cases and outbreaks of these diseases persist, driven by various contributing factors. Worldwide, emerging and reemerging infections continue to challenge prevention and control strategies while the growing problem of antimicrobial resistance needs urgent action. An important priority for control of infectious disease is to ensure that scientific and technological advances in molecular diagnostics and bioinformatics are we...","Severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) causes the infectious disease COVID-19, which was first reported in Wuhan, China in December, 2019. Despite the tremendous efforts to control the disease, COVID-19 has now spread to over 100 countries and caused a global pandemic. SARS-CoV-2 is thought to have originated in bats; however, the intermediate animal sources of the virus are completely unknown. Here, we investigated the susceptibility of ferrets and animals in close contact with humans to SARS-CoV-2. We found that SARS-CoV-2 replicates poorly in dogs, pigs, chickens, and ducks, but efficiently in ferrets and cats. We found that the virus transmits in cats via respiratory droplets. Our study provides important insights into the animal reservoirs of SARS-CoV-2 and animal management for COVID-19 control.","AbstractThe outbreak of 2019-nCoV pneumonia (COVID-19) in the city of Wuhan, China has resulted in more than 70,000 laboratory confirmed cases, and recent studies showed that 2019-nCoV (SARS-CoV-2) could be of bat origin but involve other potential intermediate hosts. In this study, we assembled the genomes of coronaviruses identified in sick pangolins. The molecular and phylogenetic analyses showed that pangolin Coronaviruses (pangolin-CoV) are genetically related to both the 2019-nCoV and bat Coronaviruses but do not support the 2019-nCoV arose directly from the pangolin-CoV. Our study also suggested that pangolin be natural host of Betacoronavirus, with a potential to infect humans. Large surveillance of coronaviruses in pangolins could improve our understanding of the spectrum of coronaviruses in pangolins. Conservation of wildlife and limits of the exposures of humans to wildlife will be important to minimize the spillover risks of coronaviruses from wild animals to humans.","Phylogenetic analyses have provided strong evidence that amino acid changes in spike (S) protein of animal and human SARS coronaviruses (SARS-CoVs) during and between two zoonotic transfers (2002/03 and 2003/04) are the result of positive selection. While several studies support that some amino acid changes between animal and human viruses are the result of inter-species adaptation, the role of neutralizing antibodies (nAbs) in driving SARS-CoV evolution, particularly during intra-species transmission, is unknown. A detailed examination of SARS-CoV infected animal and human convalescent sera could provide evidence of nAb pressure which, if found, may lead to strategies to effectively block virus evolution pathways by broadening the activity of nAbs. Here we show, by focusing on a dominant neutralization epitope, that contemporaneous- and cross-strain nAb responses against SARS-CoV spike protein exist during natural infection. In vitro immune pressure on this epitope using 2002/03 strai...","AbstractOver the past 20 years, several coronaviruses have crossed the species barrier into humans, causing outbreaks of severe, and often fatal, respiratory illness. Since SARS- CoV was first identified in animal markets, global viromics projects have discovered thousands of coronavirus sequences in diverse animals and geographic regions. Unfortunately, there are few tools available to functionally test these novel viruses for their ability to infect humans, which has severely hampered efforts to predict the next zoonotic viral outbreak. Here we developed an approach to rapidly screen lineage B betacoronaviruses, such as SARS-CoV and the recent 2019-nCoV, for receptor usage and their ability to infect cell types from different species. We show that host protease processing during viral entry is a significant barrier for several lineage B viruses and that bypassing this barrier allows several lineage B viruses to enter human cells through an unknown receptor. We also demonstrate how di..."],"title":["Bats and Viruses: Emergence of Novel Lyssaviruses and Association of Bats with Viral Zoonoses in the EU","Development of animal models against emerging coronaviruses: From SARS to MERS coronavirus","Middle East Respiratory Syndrome Coronavirus (MERS-CoV) origin and animal reservoir","Differential stepwise evolution of SARS coronavirus functional proteins in different host species","A new coronavirus associated with human respiratory disease in China","Complete genome characterisation of a novel coronavirus associated with severe human respiratory disease in Wuhan, China","Severe acute respiratory syndrome-related coronavirus: The species and its viruses \u2013 a statement of the Coronavirus Study Group","Diversity of Middle East respiratory syndrome coronaviruses in 109 dromedary camels based on full-genome sequencing, Abu Dhabi, United Arab Emirates","The species Severe acute respiratory syndrome-related coronavirus: classifying 2019-nCoV and naming it SARS-CoV-2","Selection of and recombination between minor variants lead to the adaptation of an avian coronavirus to primate cells","Animal models for highly pathogenic emerging viruses","26 Evolution of SARS Coronavirus and the Relevance of Modern Molecular Epidemiology","Animal models for SARS and MERS coronaviruses","A tug-of-war between severe acute respiratory syndrome coronavirus 2 and host antiviral defence: lessons from other pathogenic viruses","From SARS to MERS, Thrusting Coronaviruses into the Spotlight","Determination and application of immunodominant regions of SARS coronavirus spike and nucleocapsid proteins recognized by sera from different animal species","Coronaviruses: Molecular Biology\u2606","Different host cell proteases activate the SARS-coronavirus spike-protein for cell\u2013cell and virus\u2013cell fusion","Time Course and Cellular Localization of SARS-CoV Nucleoprotein and RNA in Lungs from Fatal Cases of SARS","Acquisition of Cell\u2013Cell Fusion Activity by Amino Acid Substitutions in Spike Protein Determines the Infectivity of a Coronavirus in Cultured Cells","Stabilized coronavirus spikes are resistant to conformational changes induced by receptor recognition or proteolysis","Emergence of SARS-CoV-2 through Recombination and Strong Purifying Selection","Recent Progress in Studies of Arterivirus- and Coronavirus-Host Interactions","MERS: emergence of a novel human coronavirus","Surveillance to improve evidence for community control decisions during the COVID-19 pandemic \u2013 Opening the animal epidemic toolbox for public health","Contingency plans in a radiation oncology department amid the 2019-nCoV outbreak in Switzerland","Proteolytic activation of the SARS-coronavirus spike protein: Cutting enzymes at the cutting edge of antiviral research","Timeline of events: A brief history of what made news this year","Murine coronavirus infection: a paradigm for virus-induced demyelinating disease","Potent and persistent antibody responses against the receptor-binding domain of SARS-CoV spike protein in recovered patients","Emerging Diseases: Overview","Heterologous MVA-S prime Ad5-S boost regimen induces high and persistent levels of neutralizing antibody response against SARS coronavirus","Coronaviruses and the human airway: a universal system for virus-host interaction studies","Increasing Host Cellular Receptor\u2014Angiotensin-Converting Enzyme 2 (ACE2) Expression by Coronavirus may Facilitate 2019-nCoV Infection","Coronaviruses: emerging and re-emerging pathogens in humans and animals","The Pathology and Pathogenesis of Experimental Severe Acute Respiratory Syndrome and Influenza in Animal Models","Highland of COVID-19 outside Hubei: epidemic characteristics, control and projections of Wenzhou, China","Cellular entry of the SARS coronavirus","Receptor recognition and cross-species infections of SARS coronavirus","Subunit Vaccines Against Emerging Pathogenic Human Coronaviruses","Chapter 19 Advances in structure-assisted antiviral discovery for animal viral diseases","MicroRNome Analysis Unravels the Molecular Basis of SARS Infection in Bronchoalveolar Stem Cells","Bat-Origin Coronaviruses Expand Their Host Range to Pigs","Virus-specific RNA and Antibody from Convalescent-phase SARS Patients Discharged from Hospital","Adaptive evolution influences the infectious dose of MERS-CoV necessary to achieve severe respiratory disease","Challenges of infectious diseases in the USA","Susceptibility of ferrets, cats, dogs, and different domestic animals to SARS-coronavirus-2","Are pangolins the intermediate host of the 2019 novel coronavirus (2019-nCoV) ?","Broadening of Neutralization Activity to Directly Block a Dominant Antibody-Driven SARS-Coronavirus Evolution Pathway","Functional assessment of cell entry and receptor usage for lineage B \u03b2-coronaviruses, including 2019-nCoV"],"url":["https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6473451/","https://doi.org/10.1016/j.virol.2015.02.030","https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4891877/","https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2676248/","https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7094943/","https://doi.org/10.1101/2020.01.24.919183","https://doi.org/10.1101/2020.02.07.937862","https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5717090/","https://doi.org/10.1038/s41564-020-0695-z","https://doi.org/10.1016/j.bbrc.2005.08.105","https://doi.org/10.1016/j.coviro.2013.01.001","https://doi.org/10.1016/b978-0-12-799942-5.00026-3","https://doi.org/10.1016/j.coviro.2015.06.009","https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7103735/","https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6357155/","https://doi.org/10.1016/j.jim.2007.11.009","https://doi.org/10.1016/b978-0-12-801238-3.02550-2","https://doi.org/10.1016/j.virol.2011.02.020","https://www.ncbi.nlm.nih.gov/pmc/articles/PMC1324951/","https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2700284/","https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6200764/","https://doi.org/10.1101/2020.03.20.000885","https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3397358/","https://doi.org/10.1016/j.coviro.2014.01.010","https://doi.org/10.1016/j.onehlt.2020.100130","https://doi.org/10.1016/j.adro.2020.03.012","https://doi.org/10.1016/j.antiviral.2013.09.028","https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7095981/","https://doi.org/10.1016/s0966-842x(97)81768-4","https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2988023/","https://doi.org/10.1016/b978-012373960-5.00453-6","https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7079952/","https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4744394/","https://doi.org/10.1101/2020.02.24.963348","https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4687117/","https://doi.org/10.1016/j.jcpa.2014.01.004","https://doi.org/10.1101/2020.02.25.20024398","https://doi.org/10.1016/j.tim.2004.08.008","https://doi.org/10.1016/j.antiviral.2013.08.014","https://doi.org/10.3389/fmicb.2020.00298","https://doi.org/10.1016/b978-0-12-816352-8.00019-9","https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2773932/","https://doi.org/10.1016/j.tim.2018.03.001","https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3323266/","https://doi.org/10.1016/j.virol.2017.12.006","https://doi.org/10.1016/s0140-6736(14)60890-4","https://doi.org/10.1101/2020.03.30.015347","https://doi.org/10.1101/2020.02.18.954628","https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2572002/","https://doi.org/10.1101/2020.01.22.915660"],"x":[0.6252574738079936,0.45045282312968427,0.33820015753894916,0.9230722983793721,0.386143040559496,0.645819321063631,0.16507563898825184,0.6317652452337363,0.4680263228195113,0.836927604947233,0.2460892102834561,0.3286133599238469,0.04701590641155795,0.7250002098155808,0.15823167554979367,0.3160721184220312,0.04894671721600474,0.8492648275870158,0.4313728271950771,0.9611131094206182,0.26287677576669055,0.29648682614080546,0.5435023905148105,0.2361748008474054,0.7279497810507898,0.899095063621512,0.6098088318574298,0.04740790449065979,0.9194185457437721,0.6231620267932713,0.709894914435745,0.6639404873299696,0.09576237090866047,0.7022534720391804,0.03746997620580139,0.8699353175237723,0.5229159313660628,0.42338594382342987,0.1996808549203004,0.8108240213833493,0.30880834000781265,0.5390040195124592,0.5476435010907278,0.7456571199059382,0.9332350257786045,0.2725155401810002,0.38833026120508907,0.718210801091095,0.5771798013177888,0.9954262869938136],"y":[0.5704564767832538,0.5711087910349902,0.9492455230992576,0.5886846498756969,0.5195778825630241,0.18613399819361287,0.9987459913253852,0.38445228459589187,0.0617348053780693,0.6344493466370055,0.08942206736976577,0.4369366258181271,0.5547393223350697,0.09731857913127162,0.7437079313359684,0.9091346891782437,0.3178603378576629,0.045698324936057966,0.6154380152008445,0.49396971778717147,0.45539978235598266,0.5826508092934806,0.5657469267614208,0.4038455972486489,0.9521256594488701,0.3420866555907409,0.5425706352229008,0.7144594023925948,0.44943066894146866,0.06249344500326237,0.3959433265998793,0.8222902626349136,0.16627811396171766,0.38254723133265756,0.045800280742735566,0.2068447840356007,0.998756581302013,0.46834984888595155,0.10793538858858831,0.411996946760145,0.1344066014140881,0.8833558574678709,0.8617817485616972,0.7796468303115887,0.7520587529780984,0.05405637754739934,0.5562495764591555,0.481953433132851,0.7054868201583702,0.8840548324114493]},"selected":{"id":"9767"},"selection_policy":{"id":"9784"}},"id":"9695","type":"ColumnDataSource"},{"attributes":{"data_source":{"id":"9695"},"glyph":{"id":"9722"},"hover_glyph":null,"muted_glyph":null,"nonselection_glyph":{"id":"9723"},"selection_glyph":null,"view":{"id":"9725"}},"id":"9724","type":"GlyphRenderer"},{"attributes":{"js_property_callbacks":{"change:indices":[{"id":"9770"}]}},"id":"9769","type":"Selection"},{"attributes":{"text":"&lt;h3&gt;Paragraph ranking&lt;/h3&gt;"},"id":"9773","type":"Div"},{"attributes":{"data":{"authors":["Shipley, Rebecca; Wright, Edward; Selden, David; Wu, Guanghui; Aegerter, James; Fooks, Anthony R; Banyard, Ashley C","Shipley, Rebecca; Wright, Edward; Selden, David; Wu, Guanghui; Aegerter, James; Fooks, Anthony R; Banyard, Ashley C","Tang, Xianchun; Li, Gang; Vasilakis, Nikos; Zhang, Yuan; Shi, Zhengli; Zhong, Yang; Wang, Lin-Fa; Zhang, Shuyi","Kucharski, Adam J.; Edmunds, W. John","Mackay, Ian M.; Arden, Katherine E.","Mackenzie, John S; Jeggo, Martyn","Shipley, Rebecca; Wright, Edward; Selden, David; Wu, Guanghui; Aegerter, James; Fooks, Anthony R; Banyard, Ashley C","Mackenzie, John S; Jeggo, Martyn","Kumar, A.; Chaterjee, Souranshu","Chu, Daniel K. W.; Pan, Yang; Cheng, Samuel M. S.; Hui, Kenrie P. Y.; Krishnan, Pavithra; Liu, Yingzhi; Ng, Daisy Y. M.; Wan, Carrie K. C.; Yang, Peng; Wang, Quanyi; Peiris, Malik; Poon, Leo L. M.","Berry, Michael; Gamieldien, Junaid; Fielding, Burtram C.","Li, Hong-Ying; Zhu, Guang-Jian; Zhang, Yun-Zhi; Zhang, Li-Biao; Hagan, Emily A; Martinez, Stephanie; Chmura, Aleksei A; Francisco, Leilani; Tai, Hina; Miller, Maureen; Daszak, Peter","Shipley, Rebecca; Wright, Edward; Selden, David; Wu, Guanghui; Aegerter, James; Fooks, Anthony R; Banyard, Ashley C","Cauchemez, Simon; Epperson, Scott; Biggerstaff, Matthew; Swerdlow, David; Finelli, Lyn; Ferguson, Neil M.","Markotter, Wanda; Geldenhuys, Marike; Jansen van Vuren, Petrus; Kemp, Alan; Mortlock, Marinda; Mudakikwa, Antoine; Nel, Louis; Nziza, Julius; Paweska, Janusz; Weyer, Jacqueline","Shipley, Rebecca; Wright, Edward; Selden, David; Wu, Guanghui; Aegerter, James; Fooks, Anthony R; Banyard, Ashley C","Shi, Z.; Wang, L.-F.","Shipley, Rebecca; Wright, Edward; Selden, David; Wu, Guanghui; Aegerter, James; Fooks, Anthony R; Banyard, Ashley C","Phillip Davis; John Bagnoli; David Yarmosh; Alan Shteyman; Lance Presser; Sharon Altmann; Shelton Bradrick; Joseph A. Russell","Blacklaws, B.A.; Daly, J.M.","Parkes, Margot W.; Bienen, Leslie; Breilh, Jaime; Hsu, Lee-Nah; McDonald, Marian; Patz, Jonathan A.; Rosenthal, Joshua P.; Sahani, Mazrura; Sleigh, Adrian; Waltner-Toews, David; Yassi, Annalee","Zhao, Shan; Li, Wentao; Schuurman, Nancy; van Kuppeveld, Frank; Bosch, Berend-Jan; Egberink, Herman","Torres-V\u00e9lez, Fernando; Brown, Corrie","Ralph, R.; Lew, J.; Zeng, T.; Francis, M.; Xue, B.; Roux, M.; Ostadgavahi, A. T.; Rubino, S.; Dawe, N. J.; Al-Ahdal, M. N.; Kelvin, D. J.; Richardson, C. D.; Kindrachuk, J.; Falzarano, D.; Kelvin, A. A.","Heymann, David L; Shindo, Nahoko","Shanlang Lin; Junpei Huang; Ziwen He; Dandan Zhan","Razanajatovo, Norosoa H; Nomenjanahary, Lalaina A; Wilkinson, David A; Razafimanahaka, Julie H; Goodman, Steven M; Jenkins, Richard K; Jones, Julia PG; Heraud, Jean-Michel","Kock, Richard A.; Karesh, William B.; Veas, Francisco; Velavan, Thirumalaisamy P.; Simons, David; Mboera, Leonard E. G.; Dar, Osman; Arruda, Li\u00e3 B\u00e1rbara; Zumla, Alimuddin","Kock, Richard A.; Karesh, William B.; Veas, Francisco; Velavan, Thirumalaisamy P.; Simons, David; Mboera, Leonard E. G.; Dar, Osman; Arruda, Li\u00e3 B\u00e1rbara; Zumla, Alimuddin","Tartari, Ermira; Allegranzi, Benedetta; Ang, Brenda; Calleja, Neville; Collignon, Peter; Hopman, Joost; Lang, Lily; Lee, Lai Chee; Ling, Moi Lin; Mehtar, Shaheen; Tambyah, Paul A.; Widmer, Andreas; Voss, Andreas","Brisse, Sylvain; Brehony, Carina; Concei\u00e7\u00e3o, Teresa; Cubero, Meritxell; Glasner, Corinna; Le Gouil, Meriadeg; Renvois\u00e9, Aur\u00e9lie; Sheppard, Sam; Weinert, Lucy A.","Shi, Z.; Wang, L.-F.","Chu, Daniel K. W.; Pan, Yang; Cheng, Samuel M. S.; Hui, Kenrie P. Y.; Krishnan, Pavithra; Liu, Yingzhi; Ng, Daisy Y. M.; Wan, Carrie K. C.; Yang, Peng; Wang, Quanyi; Peiris, Malik; Poon, Leo L. M.","","","Morris, Gayti B.; Ridgway, Elisabeth J.; Suvarna, S. Kim","Lestari, Fatma; Bowolaksono, Anom; Yuniautami, Sri; Wulandari, Tia Retno; Andani, Saraswati","Rovnak, Joel; Perera, Rushika; Hopken, Matthew W.; Read, Jenna; Waller, Derrick M.; Cohrs, Randall J.","Phillip Davis; John Bagnoli; David Yarmosh; Alan Shteyman; Lance Presser; Sharon Altmann; Shelton Bradrick; Joseph A. Russell","Mackay, Ian M.; Arden, Katherine E.","Shipley, Rebecca; Wright, Edward; Selden, David; Wu, Guanghui; Aegerter, James; Fooks, Anthony R; Banyard, Ashley C","Udwadia, Zarir F.; Raju, Reyma Sara","Peiris, J.S.M","Wong, S. F.; Chow, K. M.; Shek, C. C.; Leung, Y. P.; Chiu, A.; Lam, P. W. Y.; Ho, L. C.","Agnew, Dalen","Song, Zhiqi; Xu, Yanfeng; Bao, Linlin; Zhang, Ling; Yu, Pin; Qu, Yajin; Zhu, Hua; Zhao, Wenjie; Han, Yunlin; Qin, Chuan","Mackenzie, John S; Jeggo, Martyn","Thacker, Stephen B.; Sencer, David J.; Jaffe, Harold W.","Gruber, Marion F.; Marshall, Valerie B.","Chow, Ken Yan Ching; Hon, Chung Chau; Hui, Raymond Kin Hi; Wong, Raymond Tsz Yeung; Yip, Chi Wai; Zeng, Fanya; Leung, Frederick Chi Ching"],"color":["#ef8a62","#ef8a62","#ef8a62","#ef8a62","#ef8a62","#ef8a62","#ef8a62","#ef8a62","#ef8a62","#ef8a62","#ef8a62","#ef8a62","#ef8a62","#ef8a62","#ef8a62","#ef8a62","#ef8a62","#ef8a62","#ef8a62","#ef8a62","#ef8a62","#ef8a62","#ef8a62","#ef8a62","#ef8a62","#ef8a62","#ef8a62","#ef8a62","#ef8a62","#ef8a62","#ef8a62","#ef8a62","#ef8a62","#ef8a62","#ef8a62","#ef8a62","#ef8a62","#ef8a62","#ef8a62","#ef8a62","#ef8a62","#ef8a62","#ef8a62","#ef8a62","#ef8a62","#ef8a62","#ef8a62","#ef8a62","#ef8a62","#ef8a62"],"date":["2019-02-07","2019-02-07","2009-03-05","2015-04-10","2015-12-22","2013-04-30","2019-02-07","2013-04-30","2017-08-31","2020","2015-03-06","2020-02-10","2019-02-07","2013-03-05","2019-07-02","2019-02-07","2017-12-31","2019-02-07","2020-03-02","2018-03-31","2005-11-10","2019-08-13","2004-09-30","2020","2020-02-28","2020-03-30","2015-03-12","2020","2020","2015-06-08","2014-03-31","2017-12-31","2020","2017-09-30","2007-12-31","2019-12-31","2019-10-31","2017-11-08","2020-03-02","2015-12-22","2019-02-07","2020-03-31","2003-12-31","2004-01-08","2018-12-31","2019-01-14","2013-04-30","2017-12-31","2018-12-31","2016-11-28"],"doc":["hibzy1oh","hibzy1oh","i5hkucrn","9sj9bgq2","mwj2qby4","0g5079y1","hibzy1oh","0g5079y1","m4kdkmey","6mef38eo","ssnvr6nj","dqpujzo8","hibzy1oh","16c8dwfq","muxrxvyo","hibzy1oh","apc0lm5e","hibzy1oh","48mtdwuv","kdlcaium","7z10l0m1","eiobmxp2","z4egpcld","rxrlbw60","1g2mup0k","z470rn66","ghcwfcx2","oxs4o9xe","oxs4o9xe","w0o8qt5k","a4mf2vnp","apc0lm5e","6mef38eo","d7eishmm","achvui93","j90f8ytz","0p44d2qc","8s7qw9gv","48mtdwuv","mwj2qby4","hibzy1oh","yxm0f6zn","arhj9ujv","kox3uah2","j6g0au0b","7aj3fx5u","0g5079y1","9tn0w6n2","y0l943xg","xuj4yymz"],"journal":["Trop Med Infect Dis","Trop Med Infect Dis","BMC Evol Biol","PLoS Comput Biol","Virol J","Current Opinion in Virology","Trop Med Infect Dis","Current Opinion in Virology","Current Medicine Research and Practice","Clin Chem","Viruses","Int Health","Trop Med Infect Dis","PLoS Med","Trop Med Infect Dis","Trop Med Infect Dis","Genetics and Evolution of Infectious Diseases","Trop Med Infect Dis","","The Veterinary Journal","Ecohealth","Viruses","Clinics in Laboratory Medicine","Journal of Infection in Developing Countries","The Lancet","","Virol J","The Lancet Planetary Health","The Lancet Planetary Health","Antimicrob Resist Infect Control","Research in Microbiology","Genetics and Evolution of Infectious Diseases","Clin Chem","Disease-a-Month","Jubb, Kennedy &amp; Palmer's Pathology of Domestic Animals","Bancroft's Theory and Practice of Histological Techniques","Journal of Chemical Health and Safety","Viruses","","Virol J","Trop Med Infect Dis","Medical Journal Armed Forces India","Journal of Clinical Virology","Eur J Clin Microbiol Infect Dis","Pathology of Wildlife and Zoo Animals","Viruses","Current Opinion in Virology","International Encyclopedia of Public Health","Plotkin's Vaccines","Genomics Proteomics Bioinformatics"],"rad":[0.016600118577845803,0.01486368990159664,0.014770284831087298,0.01429364217152283,0.01355575480814519,0.012989670136246155,0.012659229793033133,0.0124259291038963,0.011763436554472241,0.011581321896848795,0.01135498659302231,0.011229924948965363,0.010925444972776654,0.010879753631815457,0.010572949276606833,0.01025273550981312,0.009660618133220786,0.009595049262376336,0.0091602169547632,0.008272013169501826,0.007879761805823077,0.00769218376448985,0.007060258235357069,0.006741599464021647,0.0066164231664328065,0.0063645254270477195,0.006284665375414156,0.005942908209357193,0.005942908209357193,0.005840949273460686,0.0056887369373595535,0.005525418921216689,0.005449991297968226,0.005431556256093279,0.005280114131966184,0.005162389612009021,0.005068949011773318,0.00503053962563715,0.004918135485454442,0.004755577665825846,0.004742783578289562,0.00450944682497098,0.004373041683469515,0.004330047510251567,0.004208966193450025,0.004015476573426956,0.00391107187884034,0.003828786532362425,0.0030525263624149655,0.003000056842271],"scr":[0.8,0.698,0.692,0.664,0.621,0.588,0.568,0.554,0.515,0.505,0.491,0.484,0.466,0.464,0.445,0.427,0.392,0.388,0.362,0.31,0.287,0.276,0.239,0.22,0.213,0.198,0.193,0.173,0.173,0.167,0.158,0.149,0.144,0.143,0.134,0.127,0.122,0.119,0.113,0.103,0.103,0.089,0.081,0.078,0.071,0.06,0.054,0.049,0.003,0.0],"text":["As well as lyssa- and filoviruses, bats within the EU have been also linked with coronavirus infection. Severe Acute Respiratory Syndrome Coronavirus (SARS-CoV) and Middle East Respiratory Syndrome Coronavirus (MERS-CoV) are coronaviruses capable of infecting humans resulting in a clinical disease of respiratory and gastrointestinal systems [123]. In animal reservoirs, coronavirus infection can result in respiratory, neurological or hepatic disease [124]. Since 2003, numerous novel coronavirus species have been isolated. Within the Alphacoronavirus and Betacoronavirus genera, nine of the 18 recognised viral species have been identified in bats [125]. Moreover, two high profile human disease outbreaks, Severe Acute Respiratory Syndrome (SARS) and Middle East Respiratory Syndrome (MERS) have been phylogenetically linked to a zoonotic viral origin in bats [126,127]. Most coronaviruses are associated with insectivorous bats (Rhinolophus sinicus, Rhinolophus macrotis, Rhinolophus ferrumequinum, Chaerephon plicata, Rhinolophus pusillus, Rhinolophus blasii, Tylonycteris pachypus, Pipistrellus abramus, Neoromica capensis, Vespertilio superans) [126]. Bat-to-human transmission of coronaviruses is likely very rare, if it occurs at all, and it is more common for bats to infect another terrestrial animal which subsequently infects humans. After the discovery and characterisation of SARS-CoV circulating in masked palm civets sold at Chinese markets and then later in horseshoe bat populations (Rhinolophus) in China, it was suggested that bats were the primary reservoir and the civet infection was a result of a spill over event [128,129,130,131,132]. In addition to this, the initial detection of MERS-CoV in dromedary camels in Saudi Arabia suggested that camelids may be a reservoir host for this pathogen but subsequent studies have reported a single isolation from fecal matter from a Taphozous perforates bat and a \u03b2-coronavirus, with 96.5% amino acid identity to the MERS-CoV, has been isolated from a Nyctinomops laticaudatus bat [127,133]. This warrants the speculation that either the MERS-CoV reservoir resides in a bat species and remains to be discovered, or that dromedary camels are the primary reservoir for MERS-CoV that originated following cross species transmission of a viral ancestor that once resided in a bat populations. The coronvirus surface-located trimeric spike glycoprotein (S) dictates the coronavirus host range as it specifically binds certain receptors for infection [134,135] and so studies surrounding receptor utilisation are warranted in determining any host restriction for these viruses. Rhinolophid bats in China have been described as hosts to many SARS-like coronaviruses, some of which are proposed to be the direct ancestors of SARS-CoV [136]. Each of these pathogens has the potential to cross the species barrier and cause disease outbreaks in terrestrial species, a process though to be driven by adaptation to the new host through genetic mutation [137]. Whilst many of the exact mechanisms required for a spill over event to occur are largely undefined, the S protein and host receptors are the logical starting point and key binding sites of the S protein and potential host receptors remain to be characterised for multiple pathogens. Certainly, identification of receptor binding domains may reveal host tropism patterns and enable evaluation of virus spill over potential.","The severity of disease caused by lyssaviruses means that the potential for cross species transmission events (CSTs) is of significance to human and animal populations [6]. For the rabies virus, spill over events are considered as those that result in dead-end infection, whilst CSTs result in the sustained onward transmission of the virus in the new host. Spill over from bats species appears common for RABV in the Americas, whilst events involving the other lyssaviruses across the Old World appear to be rare. Whilst spill over events for lyssaviruses have been reported, host switching events are far rarer and have only been described for RABV in the Americas [5,7,8,9]. The factors involved in CSTs with the sustained onward transmission of the virus remain undefined, and endeavours to identify specific amino acid substitutions facilitating virus adaptation to new host species have been, on the most part, unsuccessful. Kuzmin et al. (2012) observed that for sustainability within a bat population, a Serine at position 242 in the viral G protein appeared to predominate, and that contrastingly, an Alanine/Threonine substitution at position 242 appears to facilitate RABV sustainability within the carnivore population [10,11]. Intensive characterisation of the genetics within viral populations, including quasispecies, may elucidate the molecular mechanisms that facilitate lyssavirus adaption, however opportunities to genetically characterise such events are rare. ","The S protein is a structural protein of coronavirus. It has a crucial role in the binding of virus to host receptor and subsequent fusion between the viral and host membranes, both processes being important for virus entry into host cell. In the case of several mammalian and avian coronaviruses, the S protein is cleaved into S1 and S2. The former contains receptor attachment sites and the later is involved in the fusion of CoV onto host cell. The S1 subunit, which usually has high divergence, contains a receptor binding domain (RBD); the S2 subunit, which is comparatively more conserved, contains two heptad repeat (HR) domains [51]. Several studies revealed that the S gene of SARS experienced noticeable positive selection during the SARS epidemics, especially in the early and middle phases [20-22]. However, our analyses indicated that the S protein of SARS-CoV underwent a stepwise adaptive process subsequent to its spillover into the civet and human populations. In the BSL group, our analyses suggest that the bat SARS-like-CoVs experienced purifying selection, indicating that the S gene is relatively stable in bats. In palm civet, SARS-CoV experienced strong positive selection as indicated by the results of PCHP04 group. The failure to identify significant positive selection in PC03 group was most likely due to the limited number of sequences available for analyses (only four sequences for PC03 group and two of them have identical S gene sequences). During the early and middle phases of the 2002\u20132003 SARS epidemic in human population, a small fraction of sites among the S protein were under strong positive selection. In contrast, isolates from the late phase showed no sign of positive selection, implying the S protein became stable again after two stages of adaptive evolution. Our analysis using the HYPHY package accessed through Datamonkey facility also revealed similar evolutionary patterns for the S protein (Table 3). Taken together, these results support the hypothesis that SARS-CoVs originated in bats, that the spill over into civets and humans were recent events and that the two SARS epidemics that took place one year apart, were results of independent animal-to-human transmissions [7,20]. The major sequence difference of the S genes between bat SARS-like-CoVs and civet/human SARS-CoVs suggests that there may be other more closely related SARS-CoVs in bats or there may be other unknown intermediate animal host(s) in the transmission of the virus(es) from bats to civets [49].","Infections that spill over into humans from an external reservoir have the potential to cause epidemics with substantial morbidity and mortality, particularly if there is limited pre-existing immunity in the host population [1, 2]. However, novel pathogens do not always transmit efficiently when first introduced into human populations. Outbreaks of infections such as Middle East respiratory syndrome coronavirus (MERS-CoV) [3, 4] and monkeypox [5] have generally occurred as \u2018stuttering chains\u2019 of transmission [6], generating a relatively small number of linked clusters of cases without evidence of sustained transmission. Infections such as influenza A(H5N1) [7] and A(H7N9) [8] also appear to be subcritical at present, having so far failed to transmit efficiently between humans.","In humans, overt disease was given the name Middle East respiratory syndrome, with the acronym MERS. From intermittent animal-to-human spill-over events, the MERS-CoV spreads sporadically among people, causing more severe disease among older adults, especially males, with pre-existing diseases. The spread of MERS-CoV among humans has often been associated with outbreaks in hospitals, with around 20 % of all cases to date involving healthcare workers (HCWs).","Most new viruses that have the potential to cause pandemics are zoonoses, that is, they originate in animals, and then with assistance from various drivers of emergence such as ecological, behavioural or socioeconomic changes, spill over to infect humans. This is the start of the first of three stages in disease emergence described by Morse et al. [13 ] , and it is at this stage that surveillance of potential reservoirs at known hot spots [8 ] might provide the first enigmatic indication of the potential to spill over to infect humans and thus lead to that early crosstransmission event.","Numerous viruses exist in European bat species, some of which currently no effective human or animal vaccines are available. Vaccination of bats, through their protected status in the EU is unlikely to ever be a viable option although developments in vaccine applications for chiroptera may have future applications in preventing the disease in wild bat populations [162]. For lyssaviruses, the OIE and WHO have targeted 2030 for the elimination of dog-mediated human rabies. It is possible that should this be achieved, the spill over of bat lyssaviruses may become more evident and host switching events may occur with pathogens for which there is no vaccine protection. Certainly, the recent detection of highly divergent lyssaviruses (LLEBV) and filoviruses (LLOV) in bats across the EU poses potential the risk to human populations, especially as vaccines or antiviral drugs against these viruses do not exist. Further studies are required to understand: the mechanisms of both maintenance and transmission of viral pathogens within bat populations; the zoonotic potential of viral pathogens detected in bats; and the risk of host switching events that may impact on human and animal health.","Studies conducted on animals sampled from live animal markets in Guangdong, China, during and immediately after the SARS pandemic indicated that masked palm civets (Paguma larvata) and two other species had been infected by SARS-CoV [27 ] , but no evidence of infection was detected in wild or farmed civets [28] [29] [30] indicating they were probably spill-over hosts rather than natural hosts of the virus. The finding of SARS-CoV-like viruses in Chinese horseshoe bats from the genus Rhinolophus [31 ,32 ,33,34] , however, clearly suggested that bats could be a potential reservoir of SARS-CoV, and possibly even the natural hosts for all presently known coronavirus lineages [35] . A large number of studies have since demonstrated further SARS-CoV-like viruses and an astonishing diversity of other coronaviruses belonging to alpha-coronavirus and beta-coronavirus genera in the subfamily Coronavirinae occurring widely in bat species in most parts of the world including Africa [36] [37] [38] , Europe [39] [40] [41] [42] [43] , the Americas [44] [45] [46] [47] [48] [49] and Asia [50] [51] [52] . Interestingly, an analysis of viruses isolated from bats in Mexico showed that host species was a strong selective driver in coronavirus evolution, and that a single species of bat can maintain multiple coronaviruses. Furthermore, phylogenetic association of CoVs with host species/ genus was particularly evident in allopatric populations separated by significant geographical distances [49] . A similar diversity of coronaviruses has also been found in birds, comprising the gamma-coronaviruses [53], and a new genus, the delta-coronaviruses recently described with viruses from birds and pigs [54] .","The emergence of MERS-CoV nearly 10 years after SARS-CoV disappearance has shown that pathogenic coronaviruses may continue to spill over from zoonotic sources into the human population. Surveillance studies of viruses in animal species, including bats, rodents, and livestock, will help us to understand the potential human pathogens that exist in the environment before they can spill over human population. There are no antiviral medications available to treat MERS-CoV infections and the patient management is purely symptomatic. There is a need to develop vaccines and therapeutic strategies to prepare for the emerging coronaviruses. The sooner we understand the dynamics of current threats, the more we will be better equipped to save people from infection and disease.","Coronaviruses can be classified into 4 genera (a, b, c, and d) and these viruses are detected in a wide range of animal species, including humans (1) . There are 6 previously known human coronaviruses that can be transmitted between humans. Human alphacoronaviruses, 229E and NL63, and betacoronaviruses, OC43 and HKU1, are common respiratory viruses usually causing mild upper respiratory diseases. In contrast to these, the other 2 human beta-coronaviruses, SARS and MERS coronaviruses, are highly pathogenic in humans. The case fatality rates of SARS and MERS are about 10% and 35%, respectively (2, 3) . SARS and MERS can be transmitted between humans and may cause major outbreaks in health care settings or in community settings. Both SARS and MERS coronaviruses are of zoonotic origins. MERS coronavirus is widely circulated in dromedary camels. Over 90% of dromedary camels in Middle East countries are seropositive for MERS coronavirus (4, 5) . Because of such a high prevalence in this animal species, sporadic zoonotic transmissions of MERS coronavirus from camels to humans have been repeatedly detected in the affected region since its discovery in 2012. For the SARS outbreak in 2003, exotic mammals in wet markets, such as palm civets and raccoon dogs, are believed to be the animal sources of the virus (6) . The detection of SARS coronavirus in these species led to a temporary ban of selling of these animals in 2004, thereby preventing additional spill-over events from animals to humans. But further surveillance studies indicated that these animals are rarely positive for SARS coronavirus in non-market settings, suggesting that they only served as intermediate hosts in the SARS event (1, 7) . Subsequent virus surveillances in wild animals identified a vast number of coronaviruses in bats (8) . Interestingly, several bat coronaviruses that are genetically similar to human SARS coronavirus were detected in horseshoe bats. This group of SARS-like bat coronaviruses, together with SARS coronavirus, form a unique clade under the subgenus Sarbecovirus. It is now generally believed that SARS coronavirus is a recombinant virus between several bat SARS-like beta-coronaviruses (9) . Some of these bat coronaviruses are experimentally capable of infecting human cells in cultures (9), suggesting that these animal viruses and their recombinants can pose threats to human health.","The outbreak of SARS-CoV in 2003 and MERS-CoV less than 10 years later highlights the significant threat of coronaviruses to humans and confirms that the SARS outbreak was not an isolated incident. With the ever increasing diversity of animal coronavirus species, especially within bats, the likelihood of recombination leading to future outbreaks is high and the threat of potential pandemics is real as highly pathogenic coronaviruses continue to spill over from zoonotic sources into the human population. Misdiagnosis of these outbreaks pose a further substantial threat to healthcare workers with nosocomial spread to other patients putting further pressure on an already strained healthcare system. Understanding the dynamics and molecular characteristics of human coronaviruses currently in circulation and how they emerge, infect and cause disease, we can be better prepared for future pandemics allowing for improved response, management and treatment of related conditions. Table 1 summarizes the human coronaviruses discovered since 2003.","In spite of these positive changes over the long term, there is little understanding within enrolled participants of the transmission mechanisms and ecology of zoonotic pathogens that currently circulate in animal populations in the region. This is of particular concern in rural communities where close contact with bats and rodents was reported, and zoonotic pathogens have been detected in the widely distributed animal populations with the potential to spill over into the human population.20,32\u201335 In addition, rural residents may face a higher risk because of their limited access to quality healthcare facilities for proper diagnosis and treatment compared with urban residents.36","Surveillance programmes for bat lyssaviruses commonly report a diverse range of wild bat species apparently participating in the epizootiology of lyssaviruses. Spill over between ecologically distinct species appears to be common, with the divergence dates for viral clades often measured as evolutionarily recent events (i.e., &lt;1000 years) [88,138,139,140]. The sheer diversity of bat species associated as potential reservoir hosts to different lyssavirus species suggests historic spill over events and the subsequent maintenance of disease in novel hosts. This would have ultimately promoted new disease reservoirs and the speciation of lyssaviruses. From a host perspective, factors such as the fundamental ecology of roost choice by individual temperate bats, which may fall within geographical locations consistent with what is now defined as the EU, have mediated intraspecific and interspecific transmission events. ","The 2009 A(H1N1)pdm09 influenza pandemic [1], the SARS epidemic in 2003 [2], and the recent emergence of a novel coronavirus [3] are recent reminders of the global health threat posed by zoonotic viruses. Prior to widespread emergence in human populations, such pathogens can cause occasional infections in sub-populations that have been exposed to reservoir species (common reservoir species include for example bats, birds, swine, non-human primates). Whilst viruses causing such \u201cspill-over\u201d infections are usually poorly adapted for sustained human-to-human transmission, they are under strong selection pressure to increase transmissibility once in humans [4]. If the reproduction number R (i.e., the average number of persons infected by a case) evolves to exceed 1, a large scale epidemic in humans may result. Over the last decade, particular concerns were raised regarding highly pathogenic H5N1 avian influenza, due to the high mortality rate seen in humans and the virus's rapid spread in avian populations. However, as the A(H1N1)pdm09 influenza pandemic demonstrated, H5N1 is not the only influenza virus that may pose a pandemic risk. Recently, a swine-origin triple reassortant influenza A(H3N2) variant virus has emerged in the United States, carrying the matrix gene (M) from the H1N1pdm09 virus (H3N2v-M) [2]\u2013[4]. Studies in animal models have suggested that the presence of the H1N1pdm09 M gene may increase transmissibility of the virus [5],[6]. From January 2012 to September 2012, 307 laboratory-confirmed H3N2v-M human infections were reported to Centers for Disease Control and Prevention (CDC) [5] as opposed to 12 throughout 2011 [6]. The majority of cases have been associated with agricultural fairs but there are documented events of human-to-human transmission [7]. The surge in cases observed in summer 2012 raised public health concerns [8]. Threats from zoonoses are not limited to influenza: more than half of all recent emerging infectious disease events were zoonotic [9].","The detection of highly similar viral sequences from bats in Rwanda and Kenya, which are more than 1000 km apart, can be explained by either the phenomena of metapopulations or the hypothesis of co-evolution of paramyxoviruses with their bat hosts [3,52,53]. RNA viruses have an exceptionally high mutation rate commonly associated with quasi-species populations and with the potential to cross the species barrier, among others. This is evident in the high diversity of paramyxoviruses described to date and the wide host range associated with these viruses [3,28,36,54]. Additionally, due to the emergence of SARS, MERS and SADS, it is widely accepted that coronaviruses are capable of readily adapting to new hosts [5]. The Rwandan caves are considered an ecotourism site and guano is also mined on a small scale, providing an ideal bat-human interface. Several insectivorous bat species co-roost with the Egyptian fruit bats in these caves and future studies should investigate viral sharing. The Egyptian fruit bat also uses these caves as a maternity roost and studies have shown that increased viral shedding is linked to reproductive cycles [37]. Longitudinal biosurveillance studies can therefore identify high risk periods in the future. As such, the detection of a SARS-related bat coronavirus potentially circulating within the Rhinolophus population and a Henipavirus-related paramyxovirus in R. aegyptiacus in the Ruhengeri region may merit further investigation to determine exposure, and the potential for spill-over events to occur. To date, emphasis of paramyxovirus surveillance has mostly been placed on fruit bats, the Henipavirus genus and related viruses due to the association of other henipavirus species with zoonotic events [30,31]. However, research regarding the zoonotic potential of the insectivorous bat-associated viruses is still lacking. One major aim regarding surveillance of wildlife populations is to identify potential zoonotic agents and to evaluate any threat to the public as well as domestic animal health. Though these bat-borne viruses are unlikely to pose a significant threat, it still merits continued monitoring of the chiropteran species within these caves as well as mammalian species that inhabit the surrounding area.","The diversity observed in bat species across Europe also extends to the range of viruses that they have been shown to harbour. Many species have been reported antigen or antibody positive for lyssaviruses (Table 1). However the association of bats with other zoonoses is also of interest. The 2014 outbreak of Ebola in West Africa highlights the potential for bat pathogens to spill over into human populations. It is generally accepted that this outbreak initiated through human-bat interaction, although some controversy surrounds this conclusion [116]. Regardless, the threat of bat viruses crossing the species barrier and entering human populations exists and must be considered a significant threat to public health in areas where bats and humans live in close proximity. Alongside the risk from lyssaviruses several other viruses of note have been identified in European bat species. Filoviruses represent one of the most feared viral families from the perspective of human health as infection has been sensationalised through films and books and explosive outbreaks have been recorded although the outbreaks themselves have often been self-limiting. Filoviruses can cause a lethal hemorrhagic fever in humans and nonhuman primates and the association with bats and other potential reservoir hosts remains undefined [117]. As with the lyssaviruses, the filovirus family is continually expanding with the discovery of novel isolates in different parts of the world [118]. Although primarily associated with primate infection, several bat species have been associated with filoviruses; with both the ebolavirus and marburgvirus genera reported as being associated with different bat species. For the ebolavirus genus the association is predominantly through the large fruit bat species (Hypsignathus monstrous, Epomops franqueti and Myonycteris torquata) whilst Marburgvirus infection has been linked to both fruit bats (Rousettus aegyptiacus) and insectivorous bats (Rhinolophus eloquens, Miniopterus inflatus and Miniopteris schreibersii). The association of the latter species in Spain and Hungary with a novel ebolavirus, Lloviu virus (LLOV), led to a heightened awareness of bats as potential reservoirs of zoonoses [119]. Large insectivorous bat die off events have been reported historically in colonies of Schreiber\u2019s bats (Miniopterus schreibersii) in Spain, France and Portugal [120]. In Spain, these mortality events prompted investigation into the potential cause of the mortalities and although no causal relationship between filoviruses and the die off were established, the detection of LLOV in Miniopteris schreibersii is of potential concern to human and animal health. As described above, this species of bat has also been associated with the highly divergent Lleida bat lyssavirus and as such represents an important possible threat to human health as vaccines or antivirals specific for either infection do not exist; therefore, the consequences of any infection would be very serious. A further concern of filovirus infection of bats is the apparently asymptomatic nature of natural infection, which makes it difficult to identify the infected bats. Whilst bats are considered to be the reservoir host for lyssaviruses, infection will often result in the development of clinical disease and death. Lyssavirus natural infection is hard to define in terms of disease presentation but where experimental studies have been performed in bats, lyssavirus infection typically results in the development of rabies where intracranial inoculation is concerned but with either the development of disease or survivorship and seroconversion following inoculation by peripheral routes. Contrastingly, numerous experimental studies have demonstrated that bats experimentally infected with filoviruses remain healthy post infection but can shed viral products in fecal matter for several weeks [115,121,122]. Interestingly, LLOV was only detected in association with diseased bats with no virus, or vial products, being associated with healthy Schreiber\u2019s bats or co-roosting species. This is considered unusual as filovirus infection of other bat species is generally asymptomatic, so the isolation of virus from only diseased bats may indicate differences in pathogenicity for LLOV compared to other filovirus infections of bat. Certainly, the relationship between filovirus and bats requires further investigation as these viral pathogens pose potential threats to humans where interactions with bats occurs. ","Zoonosis contributes to the majority of emerging disease in the last 30 years, many of them originated from wildlife animals. 51e55 The story of SARS is just one of such examples that spectacularly demonstrated the seamless evolution of a bat virus into a human pathogen responsible for one of the most severe global pandemic outbreaks in modern history of mankind. In general, pathogens carried by wildlife reservoir animals usually do not cause clinical symptoms and they lie dormant until they spill over into and cause diseases in domestic animals or humans. Classical outbreak response measures, such as those deployed during the SARS outbreaks, are still useful, but no longer sufficient for early detection and prevention of major infectious disease outbreaks in the 21st century.","The lyssavirus species have a distinct and unique epidemiology through their association with bats [28]. Classical RABV is present globally, being reported in terrestrial carnivores, herbivores, and across the New World within multiple bat species. Whilst terrestrial rabies has been largely eliminated in the Americas, it is still associated with the infection of insectivorous; hematophagous; and, to a lesser extent, frugivorous bats. Interestingly, of the 16 classified lyssaviruses, only classical RABV has been reported in the Americas, and the current bat population represents an omnipresent source of RABV infection, for which elimination options are very limited. Certainly, the potential for host switching events to occur into both animal and human populations persists with any resulting human fatalities being reported. In contrast to the situation across the Americas, classical RABV has never been detected in bats in the Old World [29], yet it exists in terrestrial carnivore populations globally. From a bat infection perspective, a further contrasting feature of RABV infection is the association with different bat hosts. Bat rabies in the New World has been detected in over 40 different bat species, although infection is most typically associated with a handful of chiropteran species. In contrast, the Old-World lyssaviruses appear to be most commonly associated with a single or restricted host bat reservoir species. For example, EBLV-1 is predominantly associated with Eptesicus serotinus; EBLV-2 with Myotis daubentoniid, and so on. This species specific detection virus\u2013host relationship has led to the suggestion of host restriction or co-evolution of pathogens with certain bat species, although evidence for either is scant. Further occasional cases of presumed spill over infection are reported in other species, although this appears to be rare (Table 2). EBLV-1 has been reported in sheep, cats, and a stone marten, although onward transmission within the new host has not been demonstrated. With several bat lyssavirus species, the detection of only a handful of cases of each isolate, or in some cases, only a single isolation, precludes a thorough and accurate assessment of the viral epidemiology. The basis for the apparent abundance of bat lyssaviruses in the Old World, but only bat RABV in the New World, remains an enigma. ","human pathogen in advance of a spill-over event. This is observed in the data. The 449","With globalisation of human movement, increased density of human populations, climate change and loss of natural habitats, there is increased risk of interaction between species, and within species. This allows the emergence of infectious diseases by increased contact and cross-species transmission, as well as movement of infectious agents into geographic regions where they have not been found before. These types of drivers are discussed in the review by Heffernan (2018) , whilst the potential for spill-over between different hosts is highlighted in the systematic literature search by Glennon et al. (2018) .","Several theories of Nipah virus\u2019 emergence have been posited (Davies and Unam, 1999; Mohd Shahwahid and Othman, 1999; Field et al., 2001; Chua et al., 2002; Motavalli, 2004). Ecological theories of emergence include hypotheses that fire haze from slash-and-burn agriculture negatively affected fruiting trees and led bats to seek out fruit trees planted around pig farms. Though the ecosystem-scale drivers of bat foraging behavior are poorly understood, certainly the proximity of fruit trees and intensive pig farms allowed bats and pigs to come into close contact, which allowed spill-over of the Nipah virus from its reservoir host into domestic pigs, and then to humans.","Considering the fact that cats play an important role in human society and have constant interaction with humans, it is of interest to conduct serological surveys for possible reverse zoonosis of human pathogens. In our study S1 antigens of several human coronaviruses were included and this led us to identify HCoV-229E seropositive feline samples in our ELISA survey (Table 1); one serum in particular reacted solely with HCoV-229E S1 but not with any other coronavirus. Of the HCoV-229E S1 reactive feline sera one showed low neutralizing activity against HCoV-229E infection. This might suggest that positive cats were indeed exposed to HCoV-229E or related viruses. MERS seropositivity is also seen in other species besides the dromedary host [52]. Rare cases of seropositivity might be considered as spill-over infections from the dromedary camel reservoir. Similar (perhaps dead-end) spill-over infections of 229E from the human reservoir to cats might also occur. A similar principle could also apply for PDCoV, a porcine pathogen that emerged rather recently. Both HCoV-229E and PDCoV use APN as their receptor and have been reported to also be able to use feline APN for cellular entry [21,37]. Although reports are lacking regarding the natural infection of these two viruses in cats, HCoV-229E was shown to cause a priming effect of FCoV antibody in experimentally FCoV infected cats suggesting that infection occurred [24]. Therefore, the detection of antibodies against S1 of HCoV-229E in a portion of the cats might be specific and due to the exposure to HCoV-229E through daily interaction with humans. Eight cats were seropositive for PDCoV of which two cats were seropositive only for PDCoV and not for any other CoVs. This could be caused by infection with PDCoV or PDCoV-related viruses through avian sources, considering the fact that cats are natural avian predators and the presumed avian origin of PDCoV [2,37]. Our findings emphasize the potential role of cats as incidental hosts for non-feline coronaviruses and the need of in-depth study of naturally infected pathogens in cats. Besides serological studies, efforts should also focus on isolation and identification of these viruses in cats.","It is well known that 75% of all emerging infectious diseases of humans occur as a result of an animal pathogen's moving into a human host. A lessrecognized possibility is that of an infectious agent in one animal species moving into a second animal species to create an emerging disease of animals. What is the total number of potential pathogens currently present in animals? How many are capable of moving from animals to humans, or from one animal species to another? A recent paper by Dr. Sarah Cleaveland et al [18] catalogs and categorizes all known pathogens of humans, domestic livestock, and domestic carnivores based on their ability to move from one species to another. Surprisingly, of the 1415 known pathogens of humans, 61.6% have an animal origin. A total of 616 pathogens were documented for domestic livestock, with 77.3% considered ''multiple species'' (ie, capable of infecting more than one type of animal). For domestic carnivores the total was 374 pathogens, with 90% classified as ''multiple species.'' So it is apparent that there is considerable promiscuity among animal pathogens. As unusual species are grouped together, swapping of flora can easily occur.","Determining the animal reservoir and incidental hosts of a virus with evidence of zoonosis such as 2019-nCoV is important for controlling spillover events and limiting human infections. Although the ecological reservoir for both MERS-CoV and SARS-CoV remain undefined, evidence of viral presence in a variety of animal species has been found. Serologic evidence and the isolation of viral genomic material for SARS-CoVrelated viruses have provided evidence for bats as the putative reservoir for SARS-CoV. While masked palm civets and raccoon dogs were involved in transmission to humans and initially considered potential reservoirs, they are now considered as incidental or spillover hosts [39] [40] [41] . In contrast, dromedary camels are considered to act a reservoir host for MERS-CoV, and over half of primary human infections report contact with camels, however, it is suspected that bats are likely the ancestral host for MERS-CoV or a MERS-CoV-like virus [42, 43] . Zoonotic transmission of SARS-CoV likely results from direct contact with intermediate hosts [44] , while human-to-human transmission of both viruses occurs primarily through close contact and nosocomial transmission via respiratory secretions [45, 46] . Considering the animals implicated as reservoir hosts for SARS-CoV and MERS-CoV, along with the conclusions from phylogenetic analyses from us and other groups, these potential reservoirs will be important starting points for investigation of the virus source.","Finally, continued research is important to understand the source of the outbreak by study of animals and animal handlers in markets to provide evidence necessary for prevention of future coronavirus outbreaks.","With the progress of globalization, emerging infectious diseases continue to occur and are more likely to cause a global pandemic. Most of them originate from animals. According to statistics, zoonoses account for up to 60.3% of the new infectious diseases, of which 71.8% originate in wildlife (Jones et. al., 2008) . The situation of prevention and control of emerging infectious diseases is very serious (Han et al., 2016) . Since the 21st century, emerging infectious (Zheng et al., 2019) , which have posed a serious threat to human life and health and socio-economic development. There are three stages in the pandemic zoonoses emergence: the first stage is the pre-emergence, under which the pathogen is transmitted between their animal reservoirs. The second stage is localized emergence, during which self-limiting spill events or large-scale spills can lead to the person-to-person transmission for a few pathogen generations.","We observed that all Malagasy bat CoVs detected in the present study belong to a SARS-related subgroup of the Betacoronaviruses, with relatively close homology to BtCoV-HKU9 [36]. Our strains displayed 3 distinct clusters: 2 clusters associated with P. rufus and 1 cluster associated with E. dupreanum. It can be inferred from the results that multiple clusters of CoVs occurring within Malagasy bat populations co-circulate and possibly in a syntopic manner. The high nucleotide and amino acid divergence between clusters and compared to the reference virus BtCoV-HKU9 suggests previously undescribed genetic lineages. Given the mobility of bats, and the especially long distances that can be travelled by colonies of fruit bats [37], these coronaviruses may be spread over a large region. However, host-genus-specific phylogenetic clustering (Figure 2, inset) suggests likely host-specificity which may limit viral spill-over. Thus, further molecular epidemiology studies would be required to fully understand the dispersal potential of CoVs amongst Malagasy bats species.","Inevitably, the health sectors are primarily reactive to these events, acting to save lives as well as undertake surveillance and control. The drama and panic typically fade into history with the substantial costs being absorbed by ordinary people, international financial systems, and tax bases, making life go on as normal, but not quite. The frequency, severity, and financial impacts of these events are growing, and the world can no longer afford to just wait and see, especially because prevention of these threats is in theory relatively simple and where addressed has resulted in a cessation of risk. The best example being Nipah virus, where separation of pig farms from fruit agriculture, and by the same measure, fruit bats, has substantially reduced the potential for Nipah virus spill over. Bats have always had viral populations, and despite close association with humans for millennia, this has not resulted in pandemics until recent times.","Inevitably, the health sectors are primarily reactive to these events, acting to save lives as well as undertake surveillance and control. The drama and panic typically fade into history with the substantial costs being absorbed by ordinary people, international financial systems, and tax bases, making life go on as normal, but not quite. The frequency, severity, and financial impacts of these events are growing, and the world can no longer afford to just wait and see, especially because prevention of these threats is in theory relatively simple and where addressed has resulted in a cessation of risk. The best example being Nipah virus, where separation of pig farms from fruit agriculture, and by the same measure, fruit bats, has substantially reduced the potential for Nipah virus spill over. Bats have always had viral populations, and despite close association with humans for millennia, this has not resulted in pandemics until recent times.","The WHO guidelines stress on safeguarding the wellbeing of healthcare workers by making optimal provisions for hand hygiene, PPE availability and ease of access. One of the primary responsibilities of the employer is allowing all the necessary equipment provisions so as to protect the health and safety of the employees [13]. Scientific evidence indicates alcohol hand rub is more effective, it saves time and provides better hand hygiene compliance when compared with hand washing with soap and water [14, 15]. Nevertheless, in our survey, alcohol-based hand rub availability at point of care was lacking in 25 % of EVD admitting hospitals and 16 % of hospitals who would transfer EVD suspected cases to a specialized centre. Moreover, at the time of this survey both admitting and non-admitting hospitals lacked spill kits in considerable amounts (44 and 41 % respectively). This is a concern given the global efforts to disseminate the WHO hand hygiene formulation to even very low resource settings.","To date, WHO has been informed of a total of 77 laboratoryconfirmed cases of infection with MERS-CoV, including 40 deaths. Meriadeg Le Gouil (Environment and Infectious Risks Unit, Institut Pasteur, Paris, France) described the complete sequencing of the MERS-CoV genome using a combined HTS-Sanger approach and phylogenetic analyses of new strains from two cases involving nosocomial infection. Six genomes from clinical samples or virus isolates of the two MERS-CoV FRA-IPasteur cases described in France [19] were obtained. Sample preparation and HTS were depicted as pivotal for rapid exploration, directly from the clinical specimen, of such a large viral RNA genome. This strategy avoids cell culture and polymeraseinduced biases. In addition, based on phylogenetic analysis of all available MERS-CoV strains, the date of emergence of MERS-CoV in humans was estimated at around July 2010. The monophyly of the human MERS-CoV was confirmed and the observed phylodynamic pattern supported a model of repeated spill-over from infected animals rather than a sustained human to human transmission model.","Rapid identification of causative agent in an outbreak caused by unknown pathogen is the key for an effective response. However, in the case of SARS outbreak, this was not the case. Due to the association of nonspecific clinical symptoms associated with SARS patients, several pathogens were initially \"identified\" as the potential causes of SARS, which included Chlamydia, influenza virus, and paramyxovirus. 5 The confusion continued until March, 2003 when three laboratories independently confirmed that a previously unknown coronavirus was the most likely etiological agent of SARS. 6e8 Coronaviruses are enveloped viruses with the largest single-stranded, positivesense RNA genomes currently known, ranging in size from 27 to nearly 32 kb in length. Coronaviruses can infect and cause disease in a broad array of avian and mammal species, including humans. The name \"coronavirus\" is derived from the Greek word, meaning crown, as the virus envelope appears under electron microscopy to be crowned by a characteristic ring of small bulbous structures. Within the virion, the ssRNA genome is encased in a helical nucleocapsid composed of many copies of the nucleocapsid (N) protein. The lipid bilayer envelope contains three proteins: envelope (E) and membrane (M) protein, which coordinate virion assembly and release, and the large spike (S) protein, which confers the virus's characteristic corona shape as well as serves as the principal mediator of host cell attachment and entry via virusand host-specific cell receptors. The size of the SARS-CoV viral particle is approximately 80e90 nm and its genomic size is around 29.7 kb. 9,10 The SARS-CoV genome contains 14 open reading frames (ORFs) flanked by 5 0 -and 3 0 -untranslated regions of 265 and 342 nucleotides, respectively. While all CoVs carry strain-specific accessory proteins encoded by their downstream ORFs, the order of essential genesdthe replicase/transcriptase gene, S gene, E gene, M gene, and N genedis highly conserved. 11 Similar to other known coronaviruses, the SARS-CoV genome expression starts with two long open reading frames (ORFs), ORF1a and ORF1b, which account for twothirds of the genomic capacity, followed by ORFs encoding S, E, M, and N proteins ( Fig. 26.1 ). In addition to these conserved core genes in coronaviruses, the SARS-CoV genome contains several accessory genes that are specific to SARS-CoV and their encoded products have no homologue to known proteins. Phylogenetic analysis based on the most conserved gene ORF1b indicated that SARS-CoV is distantly related to the group 2 coronaviruses (now the genus Betacoronavirus) in the family The phylogenetic tree is generated based on full-length genome sequences of selected SARS-CoVs and bat SL-CoVs using the Neighbor-Joining algorithm in the MEGA4 program 78 with a bootstrap of 1000 replicates. A bat coronavirus BtCoV HKU9 is used as an outgroup. 79 outbreaks, and it was therefore impossible to trace the origin of the virus. However, through retrospective investigation, it emerged that the majority of the early index cases were limited in several cities of the Guangdong province and most of them have history of contact directly or indirectly with wildlife animals, including handling, killing, and selling wildlife animals as well as preparing and serving wildlife animal meat in restaurants. 14e16 As these epidemic regions have a unique dietary tradition favoring freshly slaughtered game meat, there is a huge trafficking and trading industry dedicated to live animal trading in specialized market, the \"wet market.\" Immediately after SARS-CoV was identified as the etiological agent of SARS, studies were conducted in those markets for evidence of SARS-CoV in market animals. One of the earliest and most important studies was conducted by a joint team from Hong Kong and Shenzhen in mainland China. 14 In this investigation, out of 25 samples collected from market animals, SARS-CoV-like viruses were isolated from four out of six masked palm civets (Paguma larvata) and one raccoon dog (Nyctereutes procyonoides). Antibodies against SARS-CoV were detected in masked palm civets, raccoon dog, and Chinese ferretbadgers (Melogale moschata). Genome sequencing indicated that the viruses isolated from civets were almost identical to those from human, suggesting a highly possible zoonotic transmission of SARS-CoV from animal(s) to human. 14 These data indicated that at least three different animal species were infected by a coronavirus that is closely related to SARS-CoV. This important study provided the first direct evidence that SARS-CoV existed in animals, pointing to an animal link of the SARS outbreaks.","In December 2019, a cluster of atypical pneumonia patients epidemiologically linked to a wholesale market in Wuhan was detected (10) . A novel beta-coronavirus named as 2019-novel coronavirus (2019-nCoV) has been identified in some of these patients. This virus is genetically similar to a bat virus in the subgenus Sarbecovirus (11) . As of 22 January 2020, over 200 cases, including 4 fatal ones, have been detected in Wuhan and other provinces (12) . In addition, confirmed patients with a recent travel history to Wuhan have been detected in multiple countries (Japan: 1, South Korea: 1, USA: 1, and Thailand: 2). Some of the confirmed cases have no epidemiological link to wet markets, suggesting the possibility of human-to-human transmissions and/or multiple spill-over events in different settings. At the time of preparing this article, a hospital outbreak had been reported (13) . The spectrum of this disease in humans is yet to be fully determined. Signs of infection are highly nonspecific, and these include respiratory symptoms, fever, cough, dyspnea, and viral pneumonia. Thus, diagnostic tests specific for this infection are urgently needed for confirming suspected cases, screening patients and conducting virus surveillance.","All specimens must be pre-packed to prevent breakage and spillage. Specimen containers should be sealed with Parafilm s and placed in ziplock bags. Place enough absorbent material to absorb the entire contents of the Secondary Container (containing Primary Container) and separate the Primary Containers (containing specimen) to prevent breakage. Send specimens with cold packs or other refrigerant blocks that are self-contained, not actual wet ice. This prevents leaking and the appearance of a spill. When large numbers of specimens are being shipped, they should be organized in a sequential manner in boxes with separate compartments for each specimen.","If the capacity of the hepatic lymphatics is exceeded, then lymph, high in protein, oozes from the hepatic capsule, presumably from the extensive lymphatic plexus there, and it may spill from the efferent lymphatics that pass from the porta hepatis to the cisterna chyli. In hepatic ascites, these efferent lymphatics become very large, numerous, and thick-walled.","Cytomegalovirus (CMV) causes congenital infections of newborns and can result in systemic disease in HIV and immunosuppressed patients. It is seen in the endothelial cells forming prominent intranuclear inclusions which spill into the cytoplasm where they form granular hematoxyphilic clusters. The CMV virus causes obvious cytomegaly in the cells it infects. All herpes viruses have an identical electron microscopic appearance of spherical, 120 nm, membrane-coated particles.","The preparedness and emergency response facilities available in laboratories include fire extinguishers for fire emergency response procedures. Wellmaintained spill kits are also available to handle chemical spills. The waste produced by laboratories has already been segregated, labeled, and stored according to type and then disposed of by a third party to prevent environmental pollution in the higher education institution area. Similar best practices have been previously observed in chemical safety and security baseline study. 18 ","Candace Mathiason (Colorado State University, Fort Collins, CO, USA) continued the theme of pathogen and host interactions through a presentation of their use of real time quaking-induced conversion (RT-QuIC) to detect blood-borne prions in buffy coat cells harvested from longitudinal chronic wasting disease (CWD) infection studies. Prion infectivity has been transmitted in blood during symptomatic and asymptomatic disease phases of Creutzfeldt-Jakob disease (vCJD)-infected humans, and CWD, bovine spongiform encephalopathy (BSE), and scrapie-infected animals. It is currently estimated that ~1:2000 individuals in the United Kingdom are asymptomatic carriers of vCJD. CWD continues to demonstrate geographic expansion, now found in captive and free-range cervid populations in North America, South Korea, and Norway. Progress in developing in vitro detection assays for hematogenous prions has been hampered by the presence of inhibitors found in blood that interfere with detection methodologies. The development of antemortem detection of blood-borne prions for humans, cervids and other susceptible species is of utmost importance for human public health, animal monitoring of prions, and improved understanding of intra/inter-host trafficking and pathogenesis. Their previous work demonstrated CWD prion infectivity and an ability to detect prions by in vitro methodology in blood throughout the protracted disease course. Here they corroborated and improved upon these findings by use of blood buffy coat cells and superparamagnetic iron oxide beads (IOMB) in combination with RT-QuIC. They demonstrated prions by IOMB-RT-QuIC in blood harvested minutes post prion exposure (15 min, 30 min, 60 min, 1 week, 2 weeks, n = 8 deer with 16 replicates/animal/time) through clinical disease (n = 7 deer, with 24\u201336 replicates/animal). This methodology will: (i) enhance our ability to determine the role of blood-borne prions in establishing and maintaining infection; as well as (ii) identify which specific blood cell phenotype subsets traffic/amplify prions through the disease course.","The highest coefficient predictor in the model for Influenza A segment 1, which codes for the 303 PB2 polymerase gene, is a motif which represents a mammalian amino acid substitution 304 experimentally observed in a mouse model 24 . This mammalian adaptation was identified as 305 relevant to the temperature sensitivity of the polymerase in H5N1. The reversion of the avian 306 conformation containing the glutamic acid residue, to the mammalian conformation containing 307 offers another angle for interpretation, especially in regards to identifying possible spill-over or 353","MERS-CoV was found and characterized because of its apparent association with severe, and therefore more obvious, illness in humans; we were the canaries in the coal mine. Sero-assays and prospective cohort studies have yet to determine the extent to which milder or asymptomatic cases contribute to MERS-CoV transmission chains. However, transmission of MERS-CoV is defined as sporadic (not sustained), intra-familial, often healthcare associated, inefficient and requiring close and prolonged contact [22, 31, 63, 93, 97, 102, 156] In a household study, 14 of 280 (5 %) contacts of 26 MERS-CoV positive index patients were RNA or antibody positive; the rate of general transmission, even in outbreaks is around 3 % [31]. It seems that the majority of human cases of MERS-CoV, even when numbers appear to increase suddenly, do not readily transmit to more than one other human so to date, the localized epidemic of MERS-CoV has not been self-sustaining [157\u2013161]. That is to say, the basic reproduction number (R0) - the average number of infections caused by one infected individual in a fully susceptible population \u2013 has been close to one throughout various clusters and outbreaks. If R0 was greater than 1, a sustained increase in case numbers would be expected. Some Ro calculations may be affected by incomplete case contact tracing, limited community testing and how a case is defined. That MERS has had a constant presence in the Arabian Peninsula since 2012 is due to ongoing, sporadic spill-over events from DCs amplified by poorly controlled hospital outbreaks.","Lyssaviruses in Europe were first reported in 1954 in Germany [19]. In 1955, a lyssavirus was isolated from insectivorous bats (Nyctalus noctula) from the FR Yugoslavia, which confirmed bat rabies [20]. Highly divergent lyssaviruses that reacted differently to monoclonal antibody panels when typed were initially discovered in 1956, originating with the description of the Lagos Bat virus in Africa. Prior to the advent of molecular testing, serological profiling using monoclonal antibodies was utilised to distinguish between lyssavirus species, and revealed virus isolates that were capable of causing rabies, but that reacted differently to defined panels of monoclonal antibodies [21]. The advancement of molecular methods, such as PCR and sequencing technologies, have superseded the antibody-based classification of new pathogens [22]. PCR and sequencing allow for the immediate genetic analysis of the suspect material, and their application to suspect material has led to the rapid typing of numerous novel lyssaviruses, initially often through genetic typing [23,24,25,26]. Although the true burden of novel lyssaviruses remains undefined, the potential for fatal infection following spill over events highlights the importance of the characterisation and classification of all lyssaviruses. The discovery of novel lyssaviruses has warranted a heightened interest in bats. As defined reservoirs of many zoonotic pathogens, the viruses harbored by bats are capable of causing explosive outbreaks of disease in human or animal populations following a cross species transmission event. In some areas, these transmission events have increased proportionally to the increased intrusion of human populations into areas of bat habitation [27], as well as the increasing popularity of leisure activities and occupations that involve entering habitats frequented by bats (e.g., caving and potholing).","Those involved in transporting specimens should be trained in safe handling practices and spill decontamination. Spills of small volume of blood/body fluids (&lt;10 ml) are cleaned with chlorine containing (5000mg/L) disinfecting wipes, while for large volume spills, higher concentrations of chlorine containing disinfectant (10,000 mg/L) or peroxyacetic acid is used. Bronchoscopes should be disinfected with 0.23% of peroxyacetic acid followed by high level of disinfection in an automatic washing and disinfection machine if available, and sterilized finally with ethylene oxide (14) . Wherever possible, dedicated medical equipment should be available while caring for patients with known or suspected COVID-19. All non-dedicated, non-disposable, medical equipment used for patient care should be disinfected according to manufacturer's policy (7, 8) .","Coronaviruses are a group of single stranded RNA viruses that infect humans and animals. The two human coronaviruses 229E and OC43 known hitherto, are causes of the common cold, rarely anything more (Holmes, 2001) . The SARS Coronavirus (SARS CoV) is genetically distinct from other human and animal coronaviruses and it appears not to have been endemic in the human population before. Thus the initial sequence information (Peiris et al., 2003a) , subsequently confirmed by the full genomic analysis of the virus (Rota et al., 2003; Marra et al., 2003) , indicated that this was a hitherto unrecognised animal virus that crossed to humans in the relatively recent past. Subsequently, the human SARS outbreak was maintained by a human-to-human transmission, presumably without need for continued reintroduction from the animal reservoir. Although not as transmissible as, for example influenza, the virus appears to have a particular propensity to transmit in a hospital setting. A significant proportion of the caseload has been caused by a relatively small number of sources and health care workers account for approximately 20% of all cases in most outbreaks (Lipsitch et al., 2003) . The virus can be cultured from the respiratory tract, gastro-intestinal tract and urine, and thus, SARS is not exclusively a respiratory disease (Peiris et al., 2003a,b) . A profuse watery diarrhoea, unrelated to antibiotics, was reported in a number of patients. Virus infection of multiple sites implies a viraemic phase, and it is of interest that in this issue of the Journal, Li and colleagues report preliminary evidence for virus replication within peripheral blood mononuclear cells (PBMC). Coronaviruses are positive strand viruses and Li and colleagues use RT-PCR to demonstrate both the genomic RNA and negative stranded RNA replicative intermediates in the first 6 days of the illness. Thereafter, the positive strand RNA was demonstrable for a further 6 days but the replicative intermediates could not. It would be desirable if the authors had controls where the RTstep is omitted, and also, if they describe whether the minus strand is still detectable if the cDNA is digested with RNAse prior to PCR. These would strengthen the argument that their minus strand signal comes exclusively from negative strand viral RNA. Attempts to culture virus from PBMC would be important confirmatory evidence, keeping in mind however, that SARS CoV is still not easy to culture from all body sites. As the authors state, the another key question is to define the cell sub-population with PBMC that the virus replicates in. Some of these questions may also be addressed through experiments in-vitro.","When the operations were performed, the Centers for Disease Control and Prevention (CDC; Atlanta, Ga., USA) had not yet prepared guidelines for the prevention of SARS transmission during Caesarean sections. Since these procedures are associated with the use of suction irrigation and diathermy, it was likely that body fluid and blood would spill and evaporate into the room during the operation. Therefore, the procedures were assumed to be aerosol generating, and previously described precautions to prevent contact and airborne infection were undertaken to prevent nosocomial SARS infection [3]. Although the CDC provided guidelines for the prevention and control of SARS infection on 6 May 2003 [4], those guidelines have, to the best of our knowledge, never been tested for the procedure described here. Following are the steps and procedures we undertook to prevent HCWs in our hospital from contracting SARS.","Equine Herpesvirus (EHV-1), a DNA virus in the Herpesvirus family, has been frequently reported in NW and OW camels, and has been experimentally reproduced in llamas from an alpaca isolate (House et al., 1991) . Infection in camelids likely develops as a \"spill-over\" from infected co-housed equids. In camelids, clinical signs are typical of equine neurological herpesviral infection. Edema, neuronal necrosis, and perivascular and meningeal lymphocytic inflammation are usually present (House et al., 1991) . Eosinophilic intranuclear inclusion bodies are typically present in endothelium and neurons. Clinical neurologic signs include blindness, which appears particularly common. Histologic lesions include necrotizing retinitis with retinal detachment, hemorrhage, choroiditis, vitritis, and optic neuritis characterized by edema and gitter cell infiltration (Rebhun et al., 1988) . Abortion has not been reported in camelids due to EHV-1. Vaccination has been proposed but there are no published studies on the effectiveness in prevention of infection or disease.","The reasons for host restriction, none or limited clinical symptoms observed in varies animal models are complexity. The interaction between the host receptor and functional proteins of SARS and MERS, respectively, plays an important and predominant role. In the context of animal models of SARS-CoV infection, researchers have compared the ACE2 amino acids that interact with the S protein RBD from several species. In agreement with the permissive nature of these species, the ACE2 residues of marmoset and hamster are similar to those of hACE2 [53]. By comparison, many residues of mouse ACE2 are different from those of hACE2, and this meets with decreased replication of SARS-CoV in mouse cells [207] and the lungs of young mice [149]. The changes at positions 353 (histidine) and 82 (asparagine) of rat ACE2 relative to hACE2 partially disrupt the S protein-DPP4 interaction and contribute to abrogation of binding. Interestingly, ferrets are permissive to SARS-CoV infection, but most of their ACE2 interaction residues are different from those of hACE2 [53], while many of the ACE2 residues between civet and ferret are the same, which may result in similar affinity [208]. For MERS-CoV, 14 residues of the S protein RBD have direct contact with 15 residues of hDPP4 [57]. Comparisons of human DPP4 binding affinity to that of other species indicated that human DPP4 had the highest affinity to the S protein of MERS-CoV, where the decreasing order of affinity is as follows: human &gt; horses &gt; camels &gt; goats &gt; bats [209]. Further evidence demonstrates that the host restriction of MERS-CoV remarkably depends on the sequence of DPP4, such as the characterization of amino acid residues at the connector of DPP4 with the RBD of S proteins in mice [187,210], hamsters, and cotton rats [210]. However, the multiplicity in severity of disease between rhesus macaques and common marmosets indicate that other host factors can perhaps affect the infection and replication of the virus, such as the presence of S-cleaving proteases [187]. In general, although the structural analysis of receptors-S protein interactions cannot fully explain all the observations for host restriction, they agree with the improved replication in several animal models and that it should be the premier and remarkable focus of small-animal model development. These special residues for host affinity are important to build up transgenic animal models enhancing the permissiveness and infection of SARS and MERS.","The concept, definitions and concerns associated with disease emergence were encapsulated in two Institute of Medicine reports, which defined the major issues and described the major causes and mechanisms leading to infectious disease emergence, as well as discussing possible strategies for recognising and counteracting the threats [3 ,4] . The association of disease emergence with anthropogenic activities is well established, especially the effects of land changes and modifications including extraction industries, rapid movement of people, globalisation of trade, human encroachment on natural environments, and climate change [5-7,8 ,9,10 ] . More than 60% of emerging diseases are zoonoses, the majority of which arise from a wildlife source [8 ] . As more information has been generated about the underlying drivers or causes of emergence, there has been an expectation that it might eventually be possible to predict or forecast the emergence of novel pathogens [11, 12, 13 ], but until now, there has been little apparent success in predicting where and when a novel pathogen might arise, nor the spill-over events which might precede emergence. Nevertheless, there has been an unprecedented search for novel pathogens over the past decade, especially in at the human-animal interface in wildlife and domestic species, fuelled in part by SARS and by post-SARS concerns raised by H5N1 highly pathogenic avian influenza and H1N1 (2009) pandemic influenza, and supported by the development of new technologies for detection and identification such as high throughput sequencing technologies and by the initiation of new pathogen discovery programs such as the PREDICT program funded by USAID [13 ] . During the decade, many new viruses have been described from wildlife belonging to a range of virus families, including Coronaviridae, Bunyaviridae, Astroviridae, Rhabdoviridae, Flaviviridae, Filoviridae, Paramyxoviridae, Adenoviridae, and Reoviridae. Most of the wildlife reservoir hosts of known viral pathogens and many novel viruses can be found in the mammalian Orders Rodentia, Chiroptera, Primates, Carnivora, as well as in birds. It is not possible to explore all of these wildlife reservoirs and hosts of novel viruses, but rather this short review will concentrate on a few specific examples chosen because they represent recent reports of diversity or geographic spread.","New threats, both man-made and naturally occurring, faced CDC in the early years of the twenty-first century. The first of these threats came in 2001, with the terrorist attacks on the World Trade Center and the Pentagon followed by anthrax attacks. In response to these events, CDC received substantial increases in funding to enhance preparedness and response capabilities within the Agency and at state and local health departments. Both the Strategic National Stockpile (SNS) of vaccines, medications, and medical supplies as well as the Laboratory Response Network were expanded to improve responses to public health emergencies. This enhanced public health capacity proved critical to national responses to infectious disease outbreaks. In 2003 alone, these included the emergence of a novel coronavirus that sparked the SARS pandemic, the introduction of monkeypox associated with the exotic pet trade, and a resurgence of West Nile virus as it moved westward across the United States. These investments were also important in CDC's responses to environmental disasters including Hurricanes Katrina and Rita in 2005, and the Haitian earthquake and Gulf of Mexico (Deepwater Horizon) oil spill, both in 2010.","In 1997, the FDA published a Final Rule, Changes to an Approved Application, which amended 21 CFR \u00a7201.12 and \u00a7314.70 to simplify and categorize manufacturing reporting requirements for changes in testing methods, equipment, facilities, or personnel. 25 Proposed changes in manufacturing methods that have a substantial potential to have an adverse effect on the safety or effectiveness of the product may not become effective until notification is given of CBER's approval. The changed created the following categories: (a) those sufficiently significant with regard to safety, purity, potency, and effectiveness of the product to require preapproval of a supplemental application before product distribution; (b) those of Feasible. This rule, referred to as the \"Animal Rule,\" allows the use of animal efficacy data in lieu of human efficacy data when human challenge studies cannot be conducted ethically and field efficacy studies are not feasible because of infectious disease epidemiology (in the case of vaccines). In these situations, certain drug and biological products (e.g., vaccines) that are intended to reduce or prevent serious or life-threatening conditions caused by lethal or permanently disabling toxic chemical, biological, radiologic, or nuclear substances may be approved for marketing based on evidence of effectiveness derived from appropriate studies in animals and additional supporting data. Safety, pharmacokinetics, and immunogenicity data are still necessary in humans. Under the animal rule, the FDA licensure of a product for which safety has been established and the requirements of 21 CFR \u00a7601.60 have been met is based upon adequate and well-controlled animal trials, when results of these animal studies establish that the product is reasonably likely to provide clinical benefit to humans. The FDA can rely on the evidence from animal studies to provide substantial evidence of the efficacy of these products when:","It has been demonstrated that the SARS-CoV possesses the ability to infect macaques, which display symptoms similar to the clinical signs of SARS patients (78), and to replicate in cats and ferrets (79). Together with the evidence implied by the phylogenetic studies, it is tempting to identify the possible animal reservoir of the coronavirus. Recent studies of domestic and wild animals in Guangdong, where the SARS epidemic was first reported, identified the existence of the SARS-CoV from several animals found in the livestock market, including Himalayan palm civets (Paguma larvata) and raccoon dogs (Nyctereutes procyonoides; ref. 80), in spite of the failure of another group to identify any SARS-CoV after the screening of more than 60 animal species (81). The genome sequences of the coronaviruses isolated from these animals are almost identical (99.8%) to that of the SARS-CoV, revealing the extremely close phylogenetic relationship between them. Another major finding from the sequence analysis highlighted a 29-bp deletion upstream the N gene, which was noted only in one Guangdong isolate available from the Gen-Bank (GD0l, accession number 278489). Such deletion leads to the fusion of the two ORFs identified in mRNA 8 into one ORF. Yet its biological significance remains to be elucidated (8). Comparison of the S gene nucleotide sequence of the animal and human SARS-CoV indicated 11 consistent nucleotide signature mutations that appeared to distinguish them. The phylogenetic analysis of the S gene sequence between human and animal SARS-CoV likely ruled out the possibility that it is a consequence of human to animal transmission, implying the infected animals may acquired the virus from a true animal source that has yet to be identified (80). This was also supported by the host-association analysis of coronaviruses based on the nucleocapsid gene (39), which pinpointed that host-shifts had played an important role in the evolution of the virus and the host. The occurrence of avian-mammal host-shift supports the hypothesis that the SARS-CoV emerged from an unknown animal coronavirus."],"title":["Bats and Viruses: Emergence of Novel Lyssaviruses and Association of Bats with Viral Zoonoses in the EU","Bats and Viruses: Emergence of Novel Lyssaviruses and Association of Bats with Viral Zoonoses in the EU","Differential stepwise evolution of SARS coronavirus functional proteins in different host species","Characterizing the Transmission Potential of Zoonotic Infections from Minor Outbreaks","MERS coronavirus: diagnostics, epidemiology and transmission","Reservoirs and vectors of emerging viruses","Bats and Viruses: Emergence of Novel Lyssaviruses and Association of Bats with Viral Zoonoses in the EU","Reservoirs and vectors of emerging viruses","Outbreak of Middle East respiratory syndrome coronavirus, Saudi Arabian experience","Molecular Diagnosis of a Novel Coronavirus (2019-nCoV) Causing an Outbreak of Pneumonia","Identification of New Respiratory Viruses in the New Millennium","A qualitative study of zoonotic risk factors among rural communities in southern China","Bats and Viruses: Emergence of Novel Lyssaviruses and Association of Bats with Viral Zoonoses in the EU","Using Routine Surveillance Data to Estimate the Epidemic Potential of Emerging Zoonoses: Application to the Emergence of US Swine Origin Influenza A H3N2v Virus","Paramyxo- and Coronaviruses in Rwandan Bats","Bats and Viruses: Emergence of Novel Lyssaviruses and Association of Bats with Viral Zoonoses in the EU","26 Evolution of SARS Coronavirus and the Relevance of Modern Molecular Epidemiology","Bats and Viruses: Emergence of Novel Lyssaviruses and Association of Bats with Viral Zoonoses in the EU","Vorpal: A novel RNA virus feature-extraction algorithm demonstrated through interpretable genotype-to-phenotype linear models","Emerging viruses of zoonotic and veterinary importance","All Hands on Deck: Transdisciplinary Approaches to Emerging Infectious Disease","Serological Screening for Coronavirus Infections in Cats","Emerging infections in animals\u2014potential new zoonoses?","2019-nCoV (Wuhan virus), a novel Coronavirus: Human-to-human transmission, travel-related cases, and vaccine readiness","COVID-19: what is next for public health?","Which Measures are Effective in Containing COVID-19? Empirical Research Based on Prevention and Control Cases in China","Detection of new genetic variants of Betacoronaviruses in Endemic Frugivorous Bats of Madagascar","2019-nCoV in context: lessons learned?","2019-nCoV in context: lessons learned?","Preparedness of institutions around the world for managing patients with Ebola virus disease: an infection control readiness checklist","Microbial molecular markers and epidemiological surveillance in the era of high throughput sequencing: an update from the IMMEM-10 conference","26 Evolution of SARS Coronavirus and the Relevance of Modern Molecular Epidemiology","Molecular Diagnosis of a Novel Coronavirus (2019-nCoV) Causing an Outbreak of Pneumonia","MIDDLE EAST RESPIRATORY SYNDROME CORONAVIRUS \u2013 MERS, MERS CoV","Peritoneum and retroperitoneum","16 Traditional stains and modern techniques for demonstrating microorganisms in histology","Evaluation of the implementation of occupational health, safety, and environment management systems in higher education laboratories","The 17th Rocky Mountain Virology Association Meeting","Vorpal: A novel RNA virus feature-extraction algorithm demonstrated through interpretable genotype-to-phenotype linear models","MERS coronavirus: diagnostics, epidemiology and transmission","Bats and Viruses: Emergence of Novel Lyssaviruses and Association of Bats with Viral Zoonoses in the EU","How to protect the protectors: 10 lessons to learn for doctors fighting the COVID-19 Coronavirus","Severe Acute Respiratory Syndrome (SARS)","Measures to Prevent Healtcare Workers from Contracting Severe Acute Respiratory Syndrome During High-Risk Surgical Procedures","Chapter 7 Camelidae","From SARS to MERS, Thrusting Coronaviruses into the Spotlight","Reservoirs and vectors of emerging viruses","Centers for Disease Control","79 Regulation and Testing of Vaccines","Molecular Advances in Severe Acute Respiratory Syndrome-associated Coronavirus (SARS-CoV)"],"url":["https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6473451/","https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6473451/","https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2676248/","https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4393285/","https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4687373/","https://doi.org/10.1016/j.coviro.2013.02.002","https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6473451/","https://doi.org/10.1016/j.coviro.2013.02.002","https://doi.org/10.1016/j.cmrp.2017.07.006","https://doi.org/10.1093/clinchem/hvaa029","https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4379558/","https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7017878/","https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6473451/","https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3589342/","https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6789848/","https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6473451/","https://doi.org/10.1016/b978-0-12-799942-5.00026-3","https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6473451/","https://doi.org/10.1101/2020.02.28.969782","https://doi.org/10.1016/j.tvjl.2017.12.022","https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7087652/","https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6723642/","https://doi.org/10.1016/j.cll.2004.05.001","https://doi.org/10.3855/jidc.12425","https://doi.org/10.1016/s0140-6736(20)30374-3","https://doi.org/10.1101/2020.03.28.20046110","https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4404003/","https://doi.org/10.1016/s2542-5196(20)30035-8","https://doi.org/10.1016/s2542-5196(20)30035-8","https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4459682/","https://doi.org/10.1016/j.resmic.2014.01.003","https://doi.org/10.1016/b978-0-12-799942-5.00026-3","https://doi.org/10.1093/clinchem/hvaa029","https://doi.org/10.1016/j.disamonth.2017.03.020","https://doi.org/10.1016/b978-070202823-6.50097-x","https://doi.org/10.1016/b978-0-7020-6864-5.00016-5","https://doi.org/10.1016/j.jchas.2018.12.006","https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5707540/","https://doi.org/10.1101/2020.02.28.969782","https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4687373/","https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6473451/","https://doi.org/10.1016/j.mjafi.2020.03.009","https://doi.org/10.1016/j.jcv.2003.08.005","https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7087897/","https://doi.org/10.1016/b978-0-12-805306-5.00007-9","https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6357155/","https://doi.org/10.1016/j.coviro.2013.02.002","https://doi.org/10.1016/b978-0-12-803678-5.00057-6","https://doi.org/10.1016/b978-0-323-35761-6.00079-1","https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5172416/"],"x":[0.950998742173126,0.6074573631249425,0.2989841518373405,0.6629986980686694,0.10541411308651583,0.08511039393778297,0.6189553058891956,0.83462653582865,0.4741606274663135,0.7024480345419809,0.35882962987857236,0.46002694969385927,0.42702171586614024,0.42058626364467677,0.9613770732024246,0.9450737299203423,0.450874779738643,0.544694211646408,0.8236463045648618,0.32325350362390637,0.8369617377865883,0.596943148377966,0.6056608810341104,0.9662864100925849,0.7743162253470872,0.2028072430170319,0.07785941224353521,0.8370013935049587,0.24031175729231857,0.41287497549214935,0.5160486979438362,0.5344747372445658,0.8548879743870715,0.3146202584829605,0.7691034757302472,0.014371468877629301,0.24538748455105452,0.43651191498753084,0.2936947939979765,0.5991880123674125,0.16834502543971397,0.8504210408597,0.8422149304861848,0.8947946404353143,0.15642901897314743,0.07044407716611334,0.8873382944199941,0.7303597799210331,0.5451176457276234,0.7088786731563522],"y":[0.5699684620365881,0.23700800366843056,0.26155619106284167,0.5830876190403215,0.9759780350172983,0.7451904320822296,0.6910448712894919,0.6780966826889167,0.10268079647839734,0.03348236340885069,0.01336930918060164,0.5283926066352513,0.2052226699899815,0.5413934582884894,0.5468777061761533,0.6857809846993871,0.6845539687348523,0.8923282147300216,0.06849730063832038,0.744542520686114,0.8987847295536107,0.14124727979097684,0.26614991300012947,0.4074813976185978,0.45506056994193034,0.6734172150432401,0.3117058295845052,0.49100901153497134,0.10769608716305878,0.5880504639021267,0.02238527052118866,0.41690801465176563,0.05229387672947605,0.4860977777107425,0.7909915744079385,0.009225479967226002,0.7162763397965015,0.5639389451957377,0.28502718858722087,0.2925278546686978,0.5755194087138709,0.20128373775611264,0.61926101851242,0.03190008636923036,0.8876434365646363,0.13574274343510406,0.8438480168849396,0.09225620122192635,0.2069875461926982,0.006058758795625452]},"selected":{"id":"9769"},"selection_policy":{"id":"9794"}},"id":"9696","type":"ColumnDataSource"},{"attributes":{"sizing_mode":"stretch_width","text":"&lt;h2&gt;Animal host(s) and any evidence of continued spill-over to humans&lt;/h2&gt;"},"id":"9771","type":"Div"},{"attributes":{"text":""},"id":"9781","type":"Title"},{"attributes":{"children":[{"id":"9765"},{"id":"9732"}]},"id":"9778","type":"Row"},{"attributes":{"js_property_callbacks":{"change:indices":[{"id":"9768"}]}},"id":"9767","type":"Selection"},{"attributes":{},"id":"9794","type":"UnionRenderers"},{"attributes":{},"id":"9801","type":"BasicTickFormatter"},{"attributes":{"children":[{"id":"9774"},{"id":"9775"},{"id":"9776"},{"id":"9777"},{"id":"9778"}]},"id":"9779","type":"Column"},{"attributes":{"columns":[{"id":"9763"},{"id":"9764"}],"height":340,"source":{"id":"9696"},"view":{"id":"9766"},"width":900},"id":"9765","type":"DataTable"},{"attributes":{"callback":null,"tooltips":[["Title","@title"],["Abstract","@text"],["Published","@date"],["Authors","@authors"],["Journal","@journal"],["URL","@url"]]},"id":"9716","type":"HoverTool"},{"attributes":{"fill_alpha":{"value":0.1},"fill_color":{"field":"color"},"line_alpha":{"value":0.1},"line_color":{"field":"color"},"radius":{"field":"rad","units":"data"},"x":{"field":"x"},"y":{"field":"y"}},"id":"9758","type":"Circle"}],"root_ids":["9779"]},"title":"Bokeh Application","version":"2.0.1"}}
        </script>
        <script type="text/javascript">
          (function() {
            var fn = function() {
              Bokeh.safely(function() {
                (function(root) {
                  function embed_document(root) {
                    
                  var docs_json = document.getElementById('10016').textContent;
                  var render_items = [{"docid":"c4d480b4-eb05-4f75-b928-38f6da1f92dc","root_ids":["9779"],"roots":{"9779":"9a11312b-3c05-48d3-a79f-d30641580089"}}];
                  root.Bokeh.embed.embed_items(docs_json, render_items);
                
                  }
                  if (root.Bokeh !== undefined) {
                    embed_document(root);
                  } else {
                    var attempts = 0;
                    var timer = setInterval(function(root) {
                      if (root.Bokeh !== undefined) {
                        clearInterval(timer);
                        embed_document(root);
                      } else {
                        attempts++;
                        if (attempts > 100) {
                          clearInterval(timer);
                          console.log("Bokeh: ERROR: Unable to run BokehJS code because BokehJS library is missing");
                        }
                      }
                    }, 10, root)
                  }
                })(window);
              });
            };
            if (document.readyState != "loading") fn();
            else document.addEventListener("DOMContentLoaded", fn);
          })();
        </script>
    
  </body>
  
</html>